<!DOCTYPE html>
<html><head>
  
  <title>Microbiology | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/systematic_mapping_of_drug_metabolism_by_the_human_gut_microbiome_54906515/">
          <h4 class="font-weight-bold">Systematic mapping of drug metabolism by the human gut microbiome, bioRxiv, 2019-02-04</h4>
          <p>The human gut microbiome harbors hundreds of bacterial species with diverse biochemical capabilities, making it one of nature&#39;s highest density, highest diversity bioreactors. Several drugs have been previously shown to be directly metabolized by the gut microbiome, but the extent of this phenomenon has not been systematically explored. Here, we develop a systematic screen for mapping the ability of the complex human gut microbiome to biochemically transform small molecules (MDM-Screen), and apply it to a library of 575 clinically used oral drugs. We show that 13% of the analyzed drugs, spanning 28 pharmacological classes, are metabolized by a single microbiome sample. In a proof-of-principle example, we show that microbiome-derived metabolism occurs in vivo, identify the genes responsible for it, and provide a possible link between its consequences and clinically observed features of drug bioavailability and toxicity. Our findings reveal a previously underappreciated role for the gut microbiome in drug metabolism, and provide a comprehensive framework for characterizing this important class of drug-microbiome interactions.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/global_phylogeography_and_ancient_evolution_of_the_widespread_human_gut_virus_crassphage_54552786/">
          <h4 class="font-weight-bold">Global phylogeography and ancient evolution of the widespread human gut virus crAssphage, bioRxiv, 2019-01-27</h4>
          <p>Microbiomes are vast communities of microbes and viruses that populate all natural ecosystems. Viruses have been considered the most variable component of microbiomes, as supported by virome surveys and examples of high genomic mosaicism. However, recent evidence suggests that the human gut virome is remarkably stable compared to other environments. Here we investigate the origin, evolution, and epidemiology of crAssphage, a widespread human gut virus. Through a global collaboratory, we obtained DNA sequences of crAssphage from over one-third of the world&#39;s countries, and showed that its phylogeography is locally clustered within countries, cities, and individuals. We also found colinear crAssphage-like genomes in both Old-World and New-World primates, challenging genomic mosaicism and suggesting that the association of crAssphage with primates may be millions of years old. We conclude that crAssphage is a benign globetrotter virus that may have co-evolved with the human lineage and an integral part of the normal human gut virome.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/mobile_genetic_element_insertions_drive_antibiotic_resistance_across_pathogens_54409093/">
          <h4 class="font-weight-bold">Mobile genetic element insertions drive antibiotic resistance across pathogens, bioRxiv, 2019-01-24</h4>
          <p>Mobile genetic elements contribute to bacterial adaptation and evolution; however, detecting these elements in a high-throughput and unbiased manner remains challenging. Here, we demonstrate a de novo approach to identify mobile elements from short-read sequencing data. The method identifies the precise site of mobile element insertion and infers the identity of the inserted sequence. This is an improvement over previous methods that either rely on curated databases of known mobile elements or rely on &#39;split-read&#39; alignments that assume the inserted element exists within the reference genome. We apply our approach to 12,419 sequenced isolates of nine prevalent bacterial pathogens, and we identify hundreds of known and novel mobile genetic elements, including many candidate insertion sequences. We find that the mobile element repertoire and insertion rate vary considerably across species, and that many of the identified mobile elements are biased toward certain target sequences, several of them being highly specific. Mobile element insertion hotspots often cluster near genes involved in mechanisms of antibiotic resistance, and such insertions are associated with antibiotic resistance in laboratory experiments and clinical isolates. Finally, we demonstrate that mutagenesis caused by these mobile elements contributes to antibiotic resistance in a genome-wide association study of mobile element insertions in pathogenic Escherichia coli. In summary, by applying a de novo approach to precisely identify mobile genetic elements and their insertion sites, we thoroughly characterize the mobile element repertoire and insertion spectrum of nine pathogenic bacterial species and find that mobile element insertions play a significant role in the evolution of clinically relevant phenotypes, such as antibiotic resistance.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/new_asgard_archaea_capable_of_anaerobic_hydrocarbon_cycling_54410688/">
          <h4 class="font-weight-bold">New Asgard archaea capable of anaerobic hydrocarbon cycling, bioRxiv, 2019-01-24</h4>
          <p>Large reservoirs of natural gas in the oceanic subsurface sustain a complex biosphere of anaerobic microbes, including recently characterized archaeal lineages that extend the potential to mediate hydrocarbon oxidation (methane and butane) beyond the Methanomicrobia. Here we describe a new archaeal phylum, Helarchaeota, belonging to the Asgard superphylum with the potential for hydrocarbon oxidation. We reconstructed Helarchaeota genomes from hydrothermal deep-sea sediment metagenomes in hydrocarbon-rich Guaymas Basin, and show that these encode novel methyl-CoM reductase-like enzymes that are similar to those found in butane-oxidizing archaea. Based on these results as well as the presence of several alkyl-CoA oxidation and Wood-Ljungdahl pathway genes in the Helarchaeota genomes, we suggest that members of the Helarchaeota have the potential to activate and subsequently anaerobically oxidize short-chain hydrocarbons. These findings link a new phylum of Asgard archaea to the microbial utilization of hydrothermally generated hydrocarbons, and extend this genomic blueprint further through the archaeal domain.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/selection_for_antibiotic_resistance_is_reduced_when_embedded_in_a_natural_microbial_community_supplemental_figure_1-4_54442691/">
          <h4 class="font-weight-bold">Selection for antibiotic resistance is reduced when embedded in a natural microbial community Supplemental Figure 1-4, bioRxiv, 2019-01-24</h4>
          <p>Antibiotic resistance has emerged as one of the most pressing, global threats to public health. In single-species experiments selection for antibiotic resistance occurs at very low antibiotic concentrations. However, it is unclear how far these findings can be extrapolated to natural environments, where species are embedded within complex communities. We competed isogenic strains of Escherichia coli, differing exclusively in a single chromosomal resistance determinant, in the presence and absence of a pig fecal microbial community across a gradient of antibiotic concentration for two relevant antibiotics gentamicin and kanamycin. We show that the minimal selective concentration was increased by more than one order of magnitude for both antibiotics when embedded in the community. We identified two general mechanisms were responsible for the increase in minimal selective concentration an increase in the cost of resistance and a protective effect of the community for the susceptible phenotype. These findings have implications for our understanding of the evolution and selection of antibiotic resistance, and can inform future risk assessment efforts on antibiotic concentrations.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/in_vivo_structure_of_the_legionella_type_ii_secretion_system_by_electron_cryotomography_supplementary_material_54157507/">
          <h4 class="font-weight-bold">In vivo structure of the Legionella type II secretion system by electron cryotomography Supplementary material, bioRxiv, 2019-01-19</h4>
          <p>The type II secretion system (T2SS) is a multi-protein envelope-spanning assembly that translocates a wide range of virulence factors, enzymes and effectors through the outer membrane (OM) of many Gram-negative bacteria. Here, using electron cryotomography and subtomogram averaging methods, we present the first in situ structure of an intact T2SS, imaged within the human pathogen Legionella pneumophila. Although the T2SS has only limited sequence and component homology with the evolutionarily-related Type IV pilus (T4P) system, we show that their overall architectures are remarkably similar. Despite similarities, there are also differences, including for instance that the T2SS-ATPase complex is usually present but disengaged from the inner membrane, the T2SS has a much longer periplasmic vestibule, and it has a short-lived flexible pseudopilus. Placing atomic models of the components into our ECT map produced a complete architectural model of the intact T2SS that provides new insights into the structure and function of its components, its position within the cell envelope, and the interactions between its different subcomplexes. Overall, these structural results strongly support the piston model for substrate extrusion.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/microbiology/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/microbiology/page/5/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/microbiology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/microbiology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/microbiology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/microbiology/page/5/" class="page-link">5</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/microbiology/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/microbiology/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/microbiology/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/microbiology/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/microbiology/page/10/" class="page-link">10</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        

        
        <li class="page-item"><a href="/tags/microbiology/page/7/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/microbiology/page/11/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
