<!DOCTYPE html>
<html><head>
  
  <title>microbiology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.68.3" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.95d2483cd3c5d2bf47c1f36307a456787d60ad958c5efe44db9afc47012b1b05b8e1f64f6ac291650439322a6c38e86eb7cd6254255516d857c23f299fb0e666.js" integrity="sha512-ldJIPNPF0r9HwfNjB6RWeH1grZWMXv5E25r8RwErGwW44fZPasKRZQQ5MipsOOhut81iVCVVFthXwj8pn7DmZg=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/assembly-free_single-molecule_nanopore_sequencing_recovers_complete_virus_genomes_from_natural_microbial_communities_59546075/">
          <h4 class="font-weight-bold">Assembly-free single-molecule nanopore sequencing recovers complete virus genomes from natural microbial communities, bioRxiv, 2019-04-26</h4>
          <p>AbstractViruses are the most abundant biological entities on Earth, and play key roles in host ecology, evolution, and horizontal gene transfer. Despite recent progress in viral metagenomics, the inherent genetic complexity of virus populations still poses technical difficulties for recovering complete virus genomes from natural assemblages. To address these challenges, we developed an assembly-free, single-molecule nanopore sequencing approach enabling direct recovery of high-quality viral genome sequences from environmental samples. Our method yielded over a thousand high quality, full-length draft virus genome sequences that could not be fully recovered using short read assembly approaches applied to the same samples. Additionally, novel DNA sequences were discovered whose repeat structures, gene contents and concatemer lengths suggested that they represent phage-inducible chromosomal islands that were packaged as concatemers within phage particles. Our new approach provided novel insight into genome structures, population biology, and ecology of naturally occurring viruses and viral parasites.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/neighbor_predation_linked_to_natural_competence_fosters_the_transfer_of_large_genomic_regions_in_vibrio_cholerae_59471825/">
          <h4 class="font-weight-bold">Neighbor predation linked to natural competence fosters the transfer of large genomic regions in Vibrio cholerae, bioRxiv, 2019-04-25</h4>
          <p>AbstractNatural competence for transformation is a primary mode of horizontal gene transfer (HGT). Competent bacteria are able to absorb free DNA from their surroundings and exchange this DNA against pieces of their own genome when sufficiently homologous. And while it is known that transformation contributes to evolution and pathogen emergence in bacteria, there are still questions regarding the general prevalence of non-degraded DNA with sufficient coding capacity. In this context, we previously showed that the naturally competent bacterium Vibrio cholerae uses its type VI secretion system (T6SS) to actively acquire DNA from non-kin neighbors under chitin-colonizing conditions. We therefore sought to further explore the role of the T6SS in acquiring DNA, the condition of the DNA released through T6SS-mediated killing versus passive cell lysis, and the extent of the transfers that occur due to these conditions. To do this, we herein measured the frequency and the extent of genetic exchanges in bacterial co-cultures on competence-inducing chitin under various DNA-acquisition conditions. We show that competent V. cholerae strains acquire DNA fragments with an average and maximum length exceeding 50 kbp and 150 kbp, respectively, and that the T6SS is of prime importance for such HGT events. Collectively, our data support the notion that the environmental lifestyle of V. cholerae fosters HGT and that the coding capacity of the exchanged genetic material is sufficient to significantly accelerate bacterial evolution.Significance StatementDNA shuffled from one organism to another in an inheritable manner is a common feature of prokaryotes. It is a significant mechanism by which bacteria acquire new phenotypes, for example by first absorbing foreign DNA and then recombining it into their genome. In this study, we show the remarkable extent of the exchanged genetic material, frequently exceeding 150 genes in a seemingly single transfer event, in Vibrio cholerae. We also show that to best preserve its length and quality, bacteria mainly acquire this DNA by killing adjacent, healthy neighbors then immediately absorbing the released DNA before it can be degraded. These new insights into this prey-killing DNA acquisition process shed light on how bacterial species evolve in the wild.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_gut_microbiota_influences_how_circulating_immune_cells_in_humans_change_from_one_day_to_the_next_59473373/">
          <h4 class="font-weight-bold">The gut microbiota influences how circulating immune cells in humans change from one day to the next, bioRxiv, 2019-04-25</h4>
          <p>ABSTRACTThe gut microbiota influences the development and homeostasis of the mammalian immune system1–3, can alter immune cell compositions in mice4–7, and is associated with responses to immunotherapy that rely on the activity of peripheral immune cells8–12. Still, our understanding of how the microbiota modulates circulatory immune cells remains limited, particularly in humans where a lack of manipulative experiments makes inference challenging. Here we overcome this challenge by studying hundreds of hospitalized—and closely monitored—bone marrow transplantation patients as they recover from chemotherapy-induced immune ablation. This aggressive treatment causes large shifts in both circulatory immune cell and microbiota populations, allowing the relationships between the two to be studied simultaneously over time with unprecedented resolution. We analyzed daily changes in white blood cell counts from 2,235 patients, and 10,680 longitudinal fecal microbiota samples to identify bacterial genera consistently associated with those changes. Bayesian inference and validation across different patient cohorts revealed consistent associations between intestinal bacteria and peripheral immune cell dynamics in the context of immunomodulatory medications, clinical metadata and homeostatic feedbacks between peripheral immune cells. The quantification of validated microbiota associations enabled us to contrast the potency of fermentatively active, obligate anaerobic bacteria with that of medications with known immunomodulatory mechanism, and this way estimate the microbiota potential to alter peripheral immune cell dynamics directly in patients. Our analysis establishes and quantifies the link between the intestinal microbiota and immune cell dynamics in humans, with implications for microbiota-driven modulation of immunity and immunotherapies that rely on circulatory immune cells.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_users_guide_to_comparative_genomics_with_enterobase._three_case_studies_micro-clades_within_salmonella_enterica_serovar_agama_ancient_and_modern_populations_of_yersinia_pestis_and_core_genomic_diversity_of_all_escherichia_59212662/">
          <h4 class="font-weight-bold">The user’s guide to comparative genomics with EnteroBase. Three case studies micro-clades within Salmonella enterica serovar Agama, ancient and modern populations of Yersinia pestis, and core genomic diversity of all Escherichia, bioRxiv, 2019-04-19</h4>
          <p>AbstractEnteroBase is an integrated software environment which supports the identification of global population structures within several bacterial genera including pathogens. It currently contains more than 300,000 genomes that have been assembled from Illumina short reads from the genera Salmonella, Escherichia, Yersinia, Clostridiodes, Helicobacter, Vibrio, and Moraxella. With the recent introduction of hierarchical clustering of core genome MLST sequence types, EnteroBase now facilitates the identification of close relatives of bacteria within those genera inside of a few hours of uploading their short reads. It also supports private collaborations between groups of users, and the comparison of genomic data that were assembled from short reads with SNP calls that were extracted from metagenomic sequences. Here we provide an overview for its users on how EnteroBase works, what it can do, and its future prospects. This user’s guide is illustrated by three case studies ranging in scale from the miniscule (local transmission of Salmonella between neighboring social groups of badgers) through pandemic transmission of plague and microevolution of Yersinia pestis over the last 5,000 years to a novel, global overview of the population structure of all of Escherichia.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/antibiotics_select_for_novel_pathways_of_resistance_in_biofilms_58836429/">
          <h4 class="font-weight-bold">Antibiotics select for novel pathways of resistance in biofilms, bioRxiv, 2019-04-11</h4>
          <p>AbstractMost bacteria in nature exist in aggregated communities known as biofilms. Bacteria within biofilms are inherently highly resistant to antibiotics. Current understanding of the evolution and mechanisms of antibiotic resistance is largely derived from work from cells in liquid culture and it is unclear whether biofilms adapt and evolve in response to sub-inhibitory concentrations of drugs. Here we used a biofilm evolution model to show that biofilms of a model food borne pathogen, Salmonella Typhimurium rapidly evolve in response to exposure to three clinically important antibiotics. Whilst the model strongly selected for improved biofilm formation in the absence of any drug, once antibiotics were introduced the need to adapt to the drug was more important than the selection for improved biofilm formation. Adaptation to antibiotic stress imposed a marked cost in biofilm formation, particularly evident for populations exposed to cefotaxime and azithromycin. We identified distinct resistance phenotypes in biofilms compared to corresponding planktonic control cultures and characterised new mechanisms of resistance to cefotaxime and azithromycin. Novel substitutions within the multidrug efflux transporter, AcrB were identified and validated as impacting drug export as well as changes in regulators of this efflux system. There were clear fitness costs identified and associated with different evolutionary trajectories. Our results demonstrate that biofilms adapt rapidly to low concentrations of antibiotics and the mechanisms of adaptation are novel. This work will be a starting point for studies to further examine biofilm specific pathways of adaptation which inform future antibiotic use.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/mitigating_pandemic_risk_with_influenza_a_virus_field_surveillance_at_a_swine-human_interface_57444712/">
          <h4 class="font-weight-bold">Mitigating Pandemic Risk with Influenza A Virus Field Surveillance at a Swine-Human Interface, bioRxiv, 2019-03-22</h4>
          <p>Working overnight at a large swine exhibition, we identified an influenza A virus (IAV) outbreak in swine, nanopore-sequenced 13 IAV genomes from samples collected, and in real-time, determined that these viruses posed a novel risk to humans due to genetic mismatches between the viruses and current pre-pandemic candidate vaccine viruses (CVV). We developed and used a portable IAV sequencing and analysis platform called Mia (Mobile Influenza Analysis) to complete and characterize full-length consensus genomes approximately 18 hours after unpacking the mobile lab. Swine are important animal IAV reservoirs that have given rise to pandemic viruses via zoonotic transmission. Genomic analyses of IAV in swine are critical to understanding pandemic risk of viruses in this reservoir, and characterization of viruses circulating in exhibition swine enables rapid comparison to current seasonal influenza vaccines and CVVs. The Mia system rapidly identified three genetically distinct swine IAV lineages from three subtypes A(H1N1), A(H3N2) and A(H1N2). Additional analysis of the HA protein sequences of the A(H1N2) viruses identified &amp;gt;30 amino acid differences between the HA1 portion of the hemagglutinin of these viruses and the most closely related pre-2009 CVV. All virus sequences were emailed to colleagues at CDC who initiated development of a synthetically derived CVV designed to provide an optimal antigenic match with the viruses detected in the exhibition. In subsequent months, this virus caused 13 infections in humans, and was the dominant variant virus in the US detected in 2018. Had this virus caused a severe outbreak or pandemic, our proactive surveillance efforts and CVV derivation would have provided an approximate 8 week time advantage for vaccine manufacturing. This is the first report of the use of field-derived nanopore sequencing data to initiate a real-time, actionable public health countermeasure.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/5/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/5/" class="page-link">5</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/microbiology/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/10/" class="page-link">10</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/7/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/16/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
