<!DOCTYPE html>
<html><head>
  
  <title>Plant Biology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/an_ancient_integration_in_a_plant_nlr_is_maintained_as_a_trans-species_polymorphism_30901766/">
          <h4 class="font-weight-bold">An ancient integration in a plant NLR is maintained as a trans-species polymorphism, bioRxiv, 2017-12-26</h4>
          <p></p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/lrx-_and_fer-dependent_extracellular_sensing_coordinates_vacuolar_size_for_cytosol_homeostasis_30164181/">
          <h4 class="font-weight-bold">LRX- and FER-dependent extracellular sensing coordinates vacuolar size for cytosol homeostasis, bioRxiv, 2017-12-09</h4>
          <p>Cellular elongation requires the defined coordination of intra- and extracellular processes. The vacuole is the biggest plant organelle and its dimension has a role in limiting cell expansion (Löfke et al., 2015; Scheuring et al., 2016). We reveal that the increase in vacuolar occupancy enables cellular elongation with relatively little enlargement of the cytosole. It remains, however, completely unknown how the vacuolar size is coordinated with other growth-relevant processes. Intriguingly, we show that extracellular constraints impact on the intracellular expansion of the vacuole. The underlying cell wall sensing mechanism requires the interaction of the extracellular leucine-rich repeat extensin (LRX) with the receptor-like kinase Feronia (FER). Our data suggests that LRX links the plasma membrane localised FER with the cell wall, allowing this module to jointly sense and convey extracellular signals to the underlying cell. This mechanism coordinates cell wall acidificationloosening with the increase in vacuolar size, contributing cytosol homeostasis during plant cell expansion.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/k-mer_grammar_uncovers_maize_regulatory_architecture_29241814/">
          <h4 class="font-weight-bold">k-mer grammar uncovers maize regulatory architecture, bioRxiv, 2017-12-06</h4>
          <p>ABSTRACTOnly a small percentage of the genome sequence is involved in regulation of gene expression, but to biochemically identify this portion is expensive and laborious. In species like maize, with diverse intergenic regions and lots of repetitive elements, this is an especially challenging problem. While regulatory regions are rare, they do have characteristic chromatin contexts and sequence organization (the grammar) with which they can be identified. We developed a computational framework to exploit this sequence arrangement. The models learn to classify regulatory regions based on sequence features - k-mers. To do this, we borrowed two approaches from the field of natural language processing (1) “bag-of-words” which is commonly used for differentially weighting key words in tasks like sentiment analyses, and (2) a vector-space model using word2vec (vector-k-mers), that captures semantic and linguistic relationships between words. We built “bag-of-k-mers” and “vector-k-mers” models that distinguish between regulatory and non-regulatory regions with an accuracy above 90%. Our “bag-of-k-mers” achieved higher overall accuracy, while the “vector-k-mers” models were more useful in highlighting key groups of sequences within the regulatory regions. These models now provide powerful tools to annotate regulatory regions in other maize lines beyond the reference, at low cost and with high accuracy.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/high_aspect_ratio_nanomaterials_enable_delivery_of_functional_genetic_material_without_dna_integration_in_mature_plants_24195892/">
          <h4 class="font-weight-bold">High Aspect Ratio Nanomaterials Enable Delivery of Functional Genetic Material Without DNA Integration in Mature Plants, bioRxiv, 2017-08-23</h4>
          <p>Genetic engineering of plants is at the core of sustainability efforts, natural product synthesis, and agricultural crop engineering. The plant cell wall is a barrier that limits the ease and throughput with which exogenous biomolecules can be delivered to plants. Current delivery methods either suffer from host range limitations, low transformation efficiencies, tissue damage, or unavoidable DNA integration into the host genome. Here, we demonstrate efficient diffusion-based biomolecule delivery into tissues and organs of intact plants of several species with a suite of pristine and chemically-functionalized high aspect ratio nanomaterials. Efficient DNA delivery and strong protein expression without transgene integration is accomplished in Nicotiana benthamiana (Nb), Eruca sativa (arugula), Triticum aestivum (wheat) and Gossypium hirsutum (cotton) leaves and arugula protoplasts. We also demonstrate a second nanoparticle-based strategy in which small interfering RNA (siRNA) is delivered to Nb leaves and silence a gene with 95% efficiency. We find that nanomaterials not only facilitate biomolecule transport into plant cells but also protect polynucleotides from nuclease degradation. Our work provides a tool for species-independent and passive delivery of genetic material, without transgene integration, into plant cells for diverse biotechnology applications.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/profiling_of_accessible_chromatin_regions_across_multiple_plant_species_and_cell_types_reveals_common_gene_regulatory_principles_and_new_control_modules_22324710/">
          <h4 class="font-weight-bold">Profiling of accessible chromatin regions across multiple plant species and cell types reveals common gene regulatory principles and new control modules, bioRxiv, 2017-07-25</h4>
          <p>ABSTRACTThe transcriptional regulatory structure of plant genomes remains poorly defined relative to animals. It is unclear how many cis-regulatory elements exist, where these elements lie relative to promoters, and how these features are conserved across plant species. We employed the Assay for Transposase-Accessible Chromatin (ATAC-seq) in four plant species (Arabidopsis thaliana, Medicago truncatula, Solanum lycopersicum, and Oryza sativa) to delineate open chromatin regions and transcription factor (TF) binding sites across each genome. Despite 10-fold variation in intergenic space among species, the majority of open chromatin regions lie within 3 kb upstream of a transcription start site in all species. We find a common set of four TFs that appear to regulate conserved gene sets in the root tips of all four species, suggesting that TF-gene networks are generally conserved. Comparative ATAC-seq profiling of Arabidopsis root hair and non-hair cell types revealed extensive similarity as well as many cell type-specific differences. Analyzing TF binding sites in differentially accessible regions identified a MYB-driven regulatory module unique to the hair cell, which appears to control both cell fate regulators and abiotic stress responses. Our analyses revealed common regulatory principles among species and shed light on the mechanisms producing cell type-specific transcriptomes during development.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/speed_breeding_a_powerful_tool_to_accelerate_crop_research_and_breeding_21692960/">
          <h4 class="font-weight-bold">Speed breeding a powerful tool to accelerate crop research and breeding, bioRxiv, 2017-07-10</h4>
          <p>The growing human population and a changing environment have raised significant concern for global food security, with the current improvement rate of several important crops inadequate to meet future demand [1]. This slow improvement rate is attributed partly to the long generation times of crop plants. Here we present a method called ‘speed breeding’, which greatly shortens generation time and accelerates breeding and research programs. Speed breeding can be used to achieve up to 6 generations per year for spring wheat (Triticum aestivum), durum wheat (T. durum), barley (Hordeum vulgare), chickpea (Cicer arietinum), and pea (Pisum sativum) and 4 generations for canola (Brassica napus), instead of 2-3 under normal glasshouse conditions. We demonstrate that speed breeding in fully-enclosed controlled-environment growth chambers can accelerate plant development for research purposes, including phenotyping of adult plant traits, mutant studies, and transformation. The use of supplemental lighting in a glasshouse environment allows rapid generation cycling through single seed descent and potential for adaptation to larger-scale crop improvement programs. Cost-saving through LED supplemental lighting is also outlined. We envisage great potential for integrating speed breeding with other modern crop breeding technologies, including high-throughput genotyping, genome editing, and genomic selection, accelerating the rate of crop improvement.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/6/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/6/" class="page-link">6</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/8/" class="page-link">8</a></li>
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/8/" rel="next" class="page-link">Next ›</a></li>
        

        
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
