<!DOCTYPE html>
<html><head>
  
  <title>Systems Biology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_deep_mutational_scan_of_an_acidic_activation_domain_30164194/">
          <h4 class="font-weight-bold">A deep mutational scan of an acidic activation domain, bioRxiv, 2017-12-09</h4>
          <p>AbstractTranscriptional activation domains are intrinsically disordered peptides with little primary sequence conservation. These properties have made it difficult to identify the sequence features that define activation domains. For example, although acidic activation domains were discovered 30 years ago, we still do not know what role, if any, acidic residues play in these peptides. To address this question we designed a rational mutagenesis scheme to independently test four sequence features theorized to control the strength of activation domains acidity (negative charge), hydrophobicity, intrinsic disorder, and short linear motifs. To test enough mutants to deconvolve these four features we developed a method to quantify the activities of thousands of activation domain variants in parallel. Our results with Gcn4, a classic acidic activation domain, suggest that acidic residues in particular regions keep two hydrophobic motifs exposed to solvent. We also found that the specific activity of the Gcn4 activation domain increases during amino acid starvation. Our results suggest that Gcn4 may have evolved to have low activity but high inducibility. Our results also demonstrate that high-throughput rational mutation scans will be powerful tools for unraveling the properties that control how intrinsically disordered proteins function.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/thanatotranscriptome_genes_actively_expressed_after_organismal_death_8651558/">
          <h4 class="font-weight-bold">Thanatotranscriptome genes actively expressed after organismal death, bioRxiv, 2016-06-11</h4>
          <p>AbstractA continuing enigma in the study of biological systems is what happens to highly ordered structures, far from equilibrium, when their regulatory systems suddenly become disabled. In life, genetic and epigenetic networks precisely coordinate the expression of genes -- but in death, it is not known if gene expression diminishes gradually or abruptly stops or if specific genes are involved. We investigated the unwinding of the clock by identifying upregulated genes, assessing their functions, and comparing their transcriptional profiles through postmortem time in two species, mouse and zebrafish. We found transcriptional abundance profiles of 1,063 genes were significantly changed after death of healthy adult animals in a time series spanning from life to 48 or 96 h postmortem. Ordination plots revealed non-random patterns in profiles by time. While most thanatotranscriptome (thanatos-, Greek defn. death) transcript levels increased within 0.5 h postmortem, some increased only at 24 and 48 h. Functional characterization of the most abundant transcripts revealed the following categories stress, immunity, inflammation, apoptosis, transport, development, epigenetic regulation, and cancer. The increase of transcript abundance was presumably due to thermodynamic and kinetic controls encountered such as the activation of epigenetic modification genes responsible for unraveling the nucleosomes, which enabled transcription of previously silenced genes (e.g., development genes). The fact that new molecules were synthesized at 48 to 96 h postmortem suggests sufficient energy and resources to maintain self-organizing processes. A step-wise shutdown occurs in organismal death that is manifested by the apparent upregulation of genes with various abundance maxima and durations. The results are of significance to transplantology and molecular biology.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2016">2016</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/massively_parallel_clonal_analysis_using_crisprcas9_induced_genetic_scars_8425528/">
          <h4 class="font-weight-bold">Massively parallel clonal analysis using CRISPRCas9 induced genetic scars, bioRxiv, 2016-06-02</h4>
          <p>A key goal of developmental biology is to understand how a single cell transforms into a full-grown organism consisting of many cells. Although impressive progress has been made in lineage tracing using imaging approaches, analysis of vertebrate lineage trees has mostly been limited to relatively small subsets of cells. Here we present scartrace, a strategy for massively parallel clonal analysis based on Cas9 induced genetic scars in the zebrafish.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2016">2016</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/revised_estimates_for_the_number_of_human_and_bacteria_cells_in_the_body_4964232/">
          <h4 class="font-weight-bold">Revised estimates for the number of human and bacteria cells in the body, bioRxiv, 2016-01-07</h4>
          <p>We critically revisit the ″common knowledge″ that bacteria outnumber human cells by a ratio of at least 101 in the human body. We found the total number of bacteria in the ″reference man″ to be 3.9·1013, with an uncertainty (SEM) of 25%, and a variation over the population (CV) of 52%. For human cells we identify the dominant role of the hematopoietic lineage to the total count of body cells (≈90%), and revise past estimates to reach a total of 3.0·1013human cells in the 70 kg ″reference man″ with 2% uncertainty and 14% CV. Our analysis updates the widely-cited 101 ratio, showing that the number of bacteria in our bodies is actually of the same order as the number of human cells. Indeed, the numbers are similar enough that each defecation event may flip the ratio to favor human cells over bacteria.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2016">2016</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/systems-biology/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/systems-biology/page/4/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="../../../../tags/systems-biology/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/systems-biology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/systems-biology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/systems-biology/page/4/" class="page-link">4</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/systems-biology/page/5/" class="page-link">5</a></li>
            
        

        

        
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
