<!DOCTYPE html>
<html><head>
  
  <title>Biorxiv | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/comparison_of_efficiency_and_specificity_of_crispr-associated_cas_nucleases_in_plants_an_expanded_toolkit_for_precision_genome_engineering_48724730/">
          <h4 class="font-weight-bold">Comparison of Efficiency and Specificity of CRISPR-Associated (Cas) Nucleases in Plants An Expanded Toolkit for Precision Genome Engineering, bioRxiv, 2018-09-21</h4>
          <p>Molecular tools adapted from bacterial CRISPR (Clustered Regulatory Interspaced Short Palindromic Repeats) systems for adaptive immunity have become widely used for plant genome engineering, both to investigate gene functions and to engineer desirable traits. A number of different Cas (CRISPR-associated) nucleases are now used but, as most studies performed to date have engineered different targets using a variety of plant species and molecular tools, it has been difficult to draw conclusions about the comparative performance of different nucleases. Due to the time and effort required to regenerate engineered plants, efficiency is critical. In addition, there have been several reports of mutations at sequences with less than perfect identity to the target. While in some plant species it is possible to remove these so-called ‘off-targets’ by backcrossing to a parental line, the specificity of genome engineering tools is important when targeting specific members of closely-related gene families, especially when recent paralogues are co-located in the genome and unlikely to segregate. Specificity is also important for species that take years to reach sexual maturity or that are clonally propagated. Here, we directly compare the efficiency and specificity of Cas nucleases from different bacterial species together with engineered variants of Cas9. We find that the nucleotide content correlates with efficiency and that Cas9 from Staphylococcus aureus is comparatively most efficient at inducing mutations. We also demonstrate that ‘high-fidelity’ variants of Cas9 can reduce off-target mutations in plants. We present these molecular tools as standardised DNA parts to facilitate their re-use.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/dissecting_heterogeneous_cell-populations_across_signaling_and_disease_conditions_with_popalign_48724548/">
          <h4 class="font-weight-bold">Dissecting heterogeneous cell-populations across signaling and disease conditions with PopAlign, bioRxiv, 2018-09-21</h4>
          <p>AbstractSingle-cell measurement techniques can now probe gene expression in heterogeneous cell populations from the human body across a range of environmental and physiological conditions. However, new mathematical and computational methods are required to represent and analyze gene expression changes that occur in complex mixtures of single cells as they respond to signals, drugs, or disease states. Here, we introduce a mathematical modeling platform, PopAlign, that automatically identifies subpopulations of cells within a heterogeneous mixture, and tracks gene expression and cell abundance changes across subpopulations by constructing and comparing probabilistic models. We apply PopAlign to discover specific categories of signaling responses within primary human immune cells as well as patient-specific disease signatures in multiple myeloma that are obscured by techniques like tSNE. PopAlign scales to comparisons involving tens to hundreds of samples, enabling large-scale studies of natural and engineered cell populations as they respond to drugs, signals or physiological change.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/objective_versus_self-reported_energy_intake_changes_during_low-carbohydrate_and_low-fat_diets_48703297/">
          <h4 class="font-weight-bold">Objective versus Self-Reported Energy Intake Changes During Low-Carbohydrate and Low-Fat Diets, bioRxiv, 2018-09-21</h4>
          <p>AbstractObjectiveTo examine objective versus self-reported energy intake changes (ΔEI) during a 12-month diet intervention.MethodsWe calculated ΔEI in subjects who participated in a 1-year randomized low-carbohydrate versus low-fat diet trial using repeated body weight measurements as inputs to an objective mathematical model (ΔEIModel) and compared these values with self-reported energy intake changes assessed by repeated 24-hr recalls (ΔEI24hrRecall).ResultsΔEI24hrRecall indicated a relatively persistent state of calorie restriction ≥500 kcald throughout the year with no significant differences between diets. ΔEIModel demonstrated large early decreases in calorie intake &amp;gt;800 kcald followed by an exponential return to approximately 100 kcald below baseline at the end of the year. The low-carbohydrate diet resulted in ΔEIModel that was 162±53 kcald lower than the low-fat diet over the first 3 months (p=0.002), but no significant diet differences were found at later times. Weight loss at 12 months was significantly related to ΔEIModel at all time intervals for both diets (p&amp;lt;0.0001).ConclusionsSelf-reported measurements of ΔEI were inaccurate. Model-based calculations of ΔEI found that instructions to follow the low-carbohydrate diet resulted in greater calorie restriction than the low-fat diet in the early phases of the intervention, but these diet differences were not sustained.What is already known about this subject?&lt;jatslist list-type=bullet&gt;&lt;jatslist-item&gt;Diet assessments that rely on self-report, such as 24hr dietary recall, are known to underestimate actual energy intake as measured by doubly labeled water. However, it is possible that repeated self-reported measurements could accurately detect changes in energy intake over time if the absolute bias of self-reported of measurements is approximately constant for each subject.&lt;jatslist-item&gt;What this study adds&lt;jatslist list-type=bullet&gt;&lt;jatslist-item&gt;We compared energy intake changes measured using repeated 24hr dietary recall measurements collected over the course of the 1-year Diet Intervention Examining The Factors Interacting with Treatment Success (DIETFITS) trial versus energy intake changes calculated using repeated body weight measurements as inputs to a validated mathematical model.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Whereas self-reported measurements indicated a relatively persistent state of calorie restriction, objective model-based measurements demonstrated a large early calorie restriction followed by an exponential rise in energy intake towards the pre-intervention baseline.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Model-based calculations, but not self-reported measurements, found that low-carbohydrate diets led to significantly greater early decreases in energy intake compared to low-fat diets, but long-term energy intake changes were not significantly different.&lt;jatslist-item&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/physiology">physiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/paleolithic_dna_from_the_caucasus_reveals_core_of_west_eurasian_ancestry_48724644/">
          <h4 class="font-weight-bold">Paleolithic DNA from the Caucasus reveals core of West Eurasian ancestry, bioRxiv, 2018-09-21</h4>
          <p>AbstractThe earliest ancient DNA data of modern humans from Europe dates to ∼40 thousand years ago1-4, but that from the Caucasus and the Near East to only ∼14 thousand years ago5,6, from populations who lived long after the Last Glacial Maximum (LGM) ∼26.5-19 thousand years ago7. To address this imbalance and to better understand the relationship of Europeans and Near Easterners, we report genome-wide data from two ∼26 thousand year old individuals from Dzudzuana Cave in Georgia in the Caucasus from around the beginning of the LGM. Surprisingly, the Dzudzuana population was more closely related to early agriculturalists from western Anatolia ∼8 thousand years ago8 than to the hunter-gatherers of the Caucasus from the same region of western Georgia of ∼13-10 thousand years ago5. Most of the Dzudzuana population’s ancestry was deeply related to the post-glacial western European hunter-gatherers of the ‘Villabruna cluster’3, but it also had ancestry from a lineage that had separated from the great majority of non-African populations before they separated from each other, proving that such ‘Basal Eurasians’6,9 were present in West Eurasia twice as early as previously recorded5,6. We document major population turnover in the Near East after the time of Dzudzuana, showing that the highly differentiated Holocene populations of the region6 were formed by ‘Ancient North Eurasian’3,9,10 admixture into the Caucasus and Iran and North African11,12 admixture into the Natufians of the Levant. We finally show that the Dzudzuana population contributed the majority of the ancestry of post-Ice Age people in the Near East, North Africa, and even parts of Europe, thereby becoming the largest single contributor of ancestry of all present-day West Eurasians.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/high-throughput_mapping_of_mesoscale_connectomes_in_individual_mice_48687703/">
          <h4 class="font-weight-bold">High-throughput mapping of mesoscale connectomes in individual mice, bioRxiv, 2018-09-20</h4>
          <p>AbstractBrain function is determined by connectivity among brain areas, and disruption of this connectivity leads to neuropsychiatric disorders. Understanding connectivity is essential to modern neuroscience, but mesoscale connectivity atlases are currently slow and expensive to generate, exist for few model systems, and require pooling across many brains. Here we present a method, muMAPseq (multisource Multiplexed Analysis of Projections by sequencing), which leverages barcoding and high-throughput sequencing to generate atlases from single animals rapidly and at low cost. We apply muMAPseq to tracing the neocortical connectome of individual mice, and demonstrate high reproducibility, and accuracy. Applying muMAPseq to the mutant BTBR mouse strain, which lacks a corpus callosum, we recapitulate its known connectopathies, and also uncover novel deficits. muMAPseq allows individual laboratories to generate atlases tailored to individuals, disease models, and new model species, and will facilitate quantitative comparative connectomics, permitting examination of how age, sex, environment, genetics and species affect neuronal wiring.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/proximity_sensors_reveal_social_information_transfer_in_maternity_colonies_of_common_noctule_bats_48701078/">
          <h4 class="font-weight-bold">Proximity sensors reveal social information transfer in maternity colonies of Common noctule bats, bioRxiv, 2018-09-20</h4>
          <p>Summary&lt;jatslist list-type=order&gt;&lt;jatslist-item&gt;Bats are a highly gregarious taxon suggesting that social information should be readily available for making decision. Social information transfer in maternity colonies might be a particularly efficient mechanism for naïve pups to acquire information on resources from informed adults. However, such behaviour is difficult to study in the wild, in particular in elusive and small-bodied animals such as bats.&lt;jatslist-item&gt;&lt;jatslist-item&gt;The goal of this study was to investigate the role of social information in acquiring access to two types of resources, which are crucial in the life of a juvenile bat suitable roosting sites and fruitful feeding grounds. We hypothesized that fledging offspring will make use of social information by following informed members of the social groups to unknown roosts or foraging sites.&lt;jatslist-item&gt;&lt;jatslist-item&gt;In the present study we applied for the first time the newly developed miniaturized proximity sensor system ‘BATS’, a fully automated system for documenting associations among individual bats both while roosting and while on the wing. We quantified associations among juveniles and other group member while switching roosts and during foraging.&lt;jatslist-item&gt;&lt;jatslist-item&gt;We found clear evidence for information transfer while switching roosts, mainly among juveniles and their genetically identified mothers. Anecdotal observations suggest intentional guidance behaviour by mothers, indicated by repeated commuting flights among the pup and the target roost. Infrequent, short meetings with colony members other than the mother indicate local enhancement at foraging sites, but no intentional information transfer.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Our study illustrates how advances in technology enable researchers to solve long-standing puzzles. Miniaturized proximity sensors facilitate the automated collection of continuous data sets and represent an ideal tool to gain novel insights into the sociobiology of elusive and small-bodied species.&lt;jatslist-item&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/animal-behavior-and-cognition">animal-behavior-and-cognition</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/biorxiv/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/biorxiv/page/117/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/114/" class="page-link">114</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/115/" class="page-link">115</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/116/" class="page-link">116</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/117/" class="page-link">117</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/biorxiv/page/118/" class="page-link">118</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/119/" class="page-link">119</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/120/" class="page-link">120</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/121/" class="page-link">121</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/122/" class="page-link">122</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/biorxiv/page/119/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/biorxiv/page/164/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
