<!DOCTYPE html>
<html><head>
  
  <title>Reports | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/widespread_divergent_transcription_from_prokaryotic_promoters_74989464/">
          <h4 class="font-weight-bold">Widespread divergent transcription from prokaryotic promoters, bioRxiv, 2020-02-03</h4>
          <p>ABSTRACTPromoters are DNA sequences that stimulate the initiation of transcription. In all prokaryotes, promoters are believed to drive transcription in a single direction. Here we show that prokaryotic promoters are frequently bidirectional and drive divergent transcription. Mechanistically, this occurs because key promoter elements have inherent symmetry and often coincide on opposite DNA strands. Reciprocal stimulation between divergent transcription start sites also contributes. Horizontally acquired DNA is enriched for bidirectional promoters suggesting that they represent an early step in prokaryotic promoter evolution.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_systematic_evaluation_of_single-cell_rna-sequencing_imputation_methods_75409420/">
          <h4 class="font-weight-bold">A Systematic Evaluation of Single-cell RNA-sequencing Imputation Methods, bioRxiv, 2020-01-30</h4>
          <p>ABSTRACTThe rapid development of single-cell RNA-sequencing (scRNA-seq) technology, with increased sparsity compared to bulk RNA-sequencing (RNA-seq), has led to the emergence of many methods for preprocessing, including imputation methods. Here, we systematically evaluate the performance of 18 state-of-the-art scRNA-seq imputation methods using cell line and tissue data measured across experimental protocols. Specifically, we assess the similarity of imputed cell profiles to bulk samples as well as investigate whether methods recover relevant biological signals or introduce spurious noise in three downstream analyses differential expression, unsupervised clustering, and inferring pseudotemporal trajectories. Broadly, we found significant variability in the performance of the methods across evaluation settings. While most scRNA-seq imputation methods recover biological expression observed in bulk RNA-seq data, the majority of the methods do not improve performance in downstream analyses compared to no imputation, in particular for clustering and trajectory analysis, and thus should be used with caution. Furthermore, we find that the performance of scRNA-seq imputation methods depends on many factors including the experimental protocol, the sparsity of the data, the number of cells in the dataset, and the magnitude of the effect sizes. We summarize our results and provide a key set of recommendations for users and investigators to navigate the current space of scRNA-seq imputation methods.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/copy_number_variants_outperform_snps_to_reveal_genotype-temperature_association_in_a_marine_species_75020180/">
          <h4 class="font-weight-bold">Copy number variants outperform SNPs to reveal genotype-temperature association in a marine species, bioRxiv, 2020-01-30</h4>
          <p>AbstractCopy number variants (CNVs) are a major component of genotypic and phenotypic variation in genomes. Yet, our knowledge on genotypic variation and evolution is often limited to single nucleotide polymorphism (SNPs) and the role of CNVs has been overlooked in non-model species, partly due to their challenging identification until recently. Here, we document the usefulness of reduced-representation sequencing data (RAD-seq) to detect and investigate copy number variants (CNVs) alongside SNPs in American lobster (Homarus americanus) populations. We conducted a comparative study to examine the potential role of SNPs and CNVs in local adaptation by sequencing 1141 lobsters from 21 sampling sites within the southern Gulf of St. Lawrence which experiences the highest yearly thermal variance of the Canadian marine coastal waters. Our results demonstrated that CNVs accounts for higher genetic differentiation than SNP markers. Contrary to SNPs for which no association was found, genetic-environment association revealed that 48 CNV candidates were significantly associated with the annual variance of sea surface temperature, leading to the genetic clustering of sampling locations despite their geographic separation. Altogether, we provide a strong empirical case that CNVs putatively contribute to local adaptation in marine species and unveil stronger spatial signal than SNPs. Our study provides the means to study CNVs in non-model species and underlines the importance to consider structural variants alongside SNPs to enhance our understanding of ecological and evolutionary processes shaping adaptive population structure.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/eqtl_catalogue_a_compendium_of_uniformly_processed_human_gene_expression_and_splicing_qtls_75134620/">
          <h4 class="font-weight-bold">eQTL Catalogue a compendium of uniformly processed human gene expression and splicing QTLs, bioRxiv, 2020-01-30</h4>
          <p>AbstractAn increasing number of gene expression quantitative trait locus (QTL) studies have made summary statistics publicly available, which can be used to gain insight into human complex traits by downstream analyses such as fine-mapping and colocalisation. However, differences between these datasets in their variants tested, allele codings, and in the transcriptional features quantified are a barrier to their widespread use. Here, we present the eQTL Catalogue, a resource which contains quality controlled, uniformly re-computed QTLs from 19 eQTL publications. In addition to gene expression QTLs, we have also identified QTLs at the level of exon expression, transcript usage, and promoter, splice junction and 3ʹ end usage. Our summary statistics can be downloaded by FTP or accessed via a REST API and are also accessible via the Open Targets Genetics Portal. We demonstrate how the eQTL Catalogue and GWAS Catalog APIs can be used to perform colocalisation analysis between GWAS and QTL results without downloading and reformatting summary statistics. New datasets will continuously be added to the eQTL Catalogue, enabling systematic interpretation of human GWAS associations across a large number of cell types and tissues. The eQTL Catalogue is available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpswww.ebi.ac.ukeqtl&gt;httpswww.ebi.ac.ukeqtl&lt;jatsext-link&gt;.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/haslr_fast_hybrid_assembly_of_long_reads_75551180/">
          <h4 class="font-weight-bold">HASLR Fast Hybrid Assembly of Long Reads, bioRxiv, 2020-01-28</h4>
          <p>AbstractThird generation sequencing technologies from platforms such as Oxford Nanopore Technologies and Pacific Biosciences have paved the way for building more contiguous assemblies and complete reconstruction of genomes. The larger effective length of the reads generated with these technologies has provided a mean to overcome the challenges of short to mid-range repeats. Currently, accurate long read assemblers are computationally expensive while faster methods are not as accurate. Therefore, there is still an unmet need for tools that are both fast and accurate for reconstructing small and large genomes. Despite the recent advances in third generation sequencing, researchers tend to generate second generation reads for many of the analysis tasks. Here, we present HASLR, a hybrid assembler which uses both second and third generation sequencing reads to efficiently generate accurate genome assemblies. Our experiments show that HASLR is not only the fastest assembler but also the one with the lowest number of misassemblies on all the samples compared to other tested assemblers. Furthermore, the generated assemblies in terms of contiguity and accuracy are on par with the other tools on most of the samples.AvailabilityHASLR is an open source tool available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsgithub.comvpc-ccghaslr&gt;httpsgithub.comvpc-ccghaslr&lt;jatsext-link&gt;.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/mapping_heritability_of_obesity_by_brain_cell_types_75587499/">
          <h4 class="font-weight-bold">Mapping heritability of obesity by brain cell types, bioRxiv, 2020-01-28</h4>
          <p>The underlying cell types mediating predisposition to obesity remain largely obscure. Here we first integrated recently published single-cell RNA-sequencing (scRNA-seq) data from &amp;gt;380 peripheral and nervous system cell types spanning 19 mouse organs with body mass index (BMI) genome-wide association study (GWAS) data from &amp;gt;450,000 individuals. Leveraging a novel strategy for integrating scRNA-seq data with GWAS data, we identified 22, exclusively neuronal, cell types from the subthalamus, midbrain, hippocampus, thalamus, cortex, pons, medulla, pallidum that were significantly enriched for BMI heritability (P&amp;lt;1.6×10-4). Using genes harboring coding mutations leading to syndromic forms of obesity, we replicate four midbrain cell types from the anterior pretectal nucleus, superior nucleus, periaqueductal gray and pallidum (P&amp;lt;1.7×10-4). Testing an additional set of 347 hypothalamic cell types, ventromedial hypothalamic steroidogenic-factor 1 (SF1) and cholecystokinin b receptor (CCKBR)-expressing neurons (P=4.9×10-5) previously implicated in energy homeostasis and glucose control and three cell types from the preoptic area of the hypothalamus and the lateral hypothalamus enriched for BMI GWAS associations (P&amp;lt;4.9×10-5). Together, our results suggest brain nuclei regulating integration of sensory stimuli, learning and memory are likely to play a key role in obesity and provide testable hypotheses for mechanistic follow-up studies.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2020">2020</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        
        <li class="page-item"><a href="/audiences/reports/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="/audiences/reports/" class="page-link">1</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/reports/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/6/" class="page-link">6</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/reports/page/3/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/reports/page/320/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
