<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_distinct_contractile_injection_system_found_in_a_majority_of_adult_human_microbiomes_72258739/">
          <h4 class="font-weight-bold">A Distinct Contractile Injection System Found in a Majority of Adult Human Microbiomes, bioRxiv, 2019-12-06</h4>
          <p>ABSTRACTAn imbalance of normal bacterial groups such as Bacteroidales within the human gut is correlated with diseases like obesity. A current grand challenge in the microbiome field is to identify factors produced by normal microbiome bacteria that cause these observed health and disease correlations. While identifying factors like a bacterial injection system could provide a missing explanation for why Bacteroidales correlates with host health, no such factor has been identified to date. The lack of knowledge about these factors is a significant barrier to improving therapies like fecal transplants that promote a healthy microbiome. Here we show that a previously ill-defined Contractile Injection System is carried in the gut microbiome of 99% of individuals from the United States and Europe. This type of Contractile Injection System, we name here Bacteroidales Injection System (BIS), is related to the contractile tails of bacteriophage (viruses of bacteria) and have been described to mediate interactions between bacteria and diverse eukaryotes like amoeba, insects and tubeworms. Our findings that BIS are ubiquitous within adult human microbiomes suggest that they shape host health by mediating interactions between Bacteroidales bacteria and the human host or its microbiome.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_minimal_crispr-cas3_system_for_genome_engineering_71822864/">
          <h4 class="font-weight-bold">A minimal CRISPR-Cas3 system for genome engineering, bioRxiv, 2019-12-03</h4>
          <p>AbstractCRISPR-Cas technologies have provided programmable gene editing tools that have revolutionized research. The leading CRISPR-Cas9 and Cas12a enzymes are ideal for programmed genetic manipulation, however, they are limited for genome-scale interventions. Here, we utilized a Cas3-based system featuring a processive nuclease, expressed endogenously or heterologously, for genome engineering purposes. Using an optimized and minimal CRISPR-Cas3 system (Type I-C) programmed with a single crRNA, large deletions ranging from 7 - 424 kb were generated in Pseudomonas aeruginosa with high efficiency and speed. By comparison, Cas9 yielded small deletions and point mutations. Cas3-generated deletion boundaries were variable in the absence of a homology-directed repair (HDR) template, and successfully and efficiently specified when present. The minimal Cas3 system is also portable; large deletions were induced with high efficiency in Pseudomonas syringae and Escherichia coli using an “all-in-one” vector. Notably, Cas3 generated bi-directional deletions originating from the programmed cut site, which was exploited to iteratively reduce a P. aeruginosa genome by 837 kb (13.5%) using 10 distinct crRNAs. We also demonstrate the utility of endogenous Cas3 systems (Type I-C and I-F) and develop an “anti-anti-CRISPR” strategy to circumvent endogenous CRISPR-Cas inhibitor proteins. CRISPR-Cas3 could facilitate rapid strain manipulation for synthetic biological and metabolic engineering purposes, genome minimization, and the analysis of large regions of unknown function.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/another_look_at_microbemetabolite_interactions_how_scale_invariant_correlations_can_outperform_a_neural_network_71403215/">
          <h4 class="font-weight-bold">Another look at microbe–metabolite interactions how scale invariant correlations can outperform a neural network, bioRxiv, 2019-12-02</h4>
          <p>AbstractMany scientists are now interested in studying the correlative relationships between microbes and metabolites. However, these kinds of analyses are complicated by the compositional (i.e., relative) nature of the data. Recently, Morton et al. proposed a neural network architecture called mmvec to predict metabolite abundances from microbe presence. They introduce this method as a scale invariant solution to the integration of multi-omics compositional data, and claim that “mmvec is the only method robust to scale deviations”. We do not doubt the utility of mmvec, but write in defense of simple linear statistics. In fact, when used correctly, correlation and proportionality can actually outperform the mmvec neural network.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/spontaneous_generation_of_face_recognition_in_untrained_deep_neural_networks_71404995/">
          <h4 class="font-weight-bold">Spontaneous generation of face recognition in untrained deep neural networks, bioRxiv, 2019-12-02</h4>
          <p>AbstractFace-selective neurons are observed in the primate visual pathway and are considered the basis of facial recognition in the brain. However, it is debated whether this neuronal selectivity can arise spontaneously, or requires training from visual experience. Here, we show that face-selective neurons arise spontaneously in random feedforward networks in the absence of learning. Using biologically inspired deep neural networks, we found that face-selective neurons arise under three different network conditions one trained using non-face natural images, one randomized after being trained, and one never trained. We confirmed that spontaneously emerged face-selective neurons show the biological view-point-invariant characteristics observed in monkeys. Such neurons suddenly vanished when feedforward weight variation declined to a certain level. Our results suggest that innate face-selectivity originates from statistical variation of the feedforward projections in hierarchical neural networks.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_hematopoietic_landscape_at_single-cell_resolution_reveals_unexpected_stem_cell_features_in_naked_mole-rats_71399778/">
          <h4 class="font-weight-bold">The hematopoietic landscape at single-cell resolution reveals unexpected stem cell features in naked mole-rats, bioRxiv, 2019-12-02</h4>
          <p>SUMMARYNaked mole-rats are the longest-lived rodents endowed with resistance to cancer and age-related diseases, yet their stem cell characteristics remain enigmatic. We profiled the naked mole-rat hematopoietic system down to single-cell resolution, and identified several unique features likely contributing to longevity. In adult naked mole-rats red blood cells are formed in spleen and marrow, a neotenic feature beneficial for hypoxic environments and to prevent anemia. Platelet numbers are lower compared to short-lived mice, which may preclude age-related platelet increase and thrombosis. T cells mature in thymus and lymph nodes, providing a supply of T cells after age-related thymus involution. The pool of quiescent stem cells is higher than in mice, and HSCs overexpress an oxidative phosphorylation signature, revealing a new paradigm of stem cell metabolism to benefit longevity and oppose oncogenesis. Our work provides a platform to study immunology and stem cell biology in an animal model of healthy aging.HIGHLIGHTS&lt;jatslist list-type=bullet&gt;&lt;jatslist-item&gt;Flow cytometry labelling panel to purify viable naked mole-rat HSPCs&lt;jatslist-item&gt;&lt;jatslist-item&gt;The spleen as the major site of erythropoiesis in the naked mole-rat&lt;jatslist-item&gt;&lt;jatslist-item&gt;Naked mole-rats show extrathymic T-cell development under homeostatic conditions&lt;jatslist-item&gt;&lt;jatslist-item&gt;Naked mole-rat hematopoietic stem cells (HSCs) have high OXPHOS activity&lt;jatslist-item&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/frequent_extrachromosomal_oncogene_amplification_drives_aggressive_tumors_71372515/">
          <h4 class="font-weight-bold">Frequent extrachromosomal oncogene amplification drives aggressive tumors, bioRxiv, 2019-11-29</h4>
          <p>Extrachromosomal DNA (ecDNA) amplification promotes high oncogene copy number, intratumoral genetic heterogeneity, and accelerated tumor evolution1–3, but its frequency and clinical impact are not well understood. Here we show, using computational analysis of whole-genome sequencing data from 1,979 cancer patients, that ecDNA amplification occurs in at least 26% of human cancers, of a wide variety of histological types, but not in whole blood or normal tissue. We demonstrate a highly significant enrichment for oncogenes on amplified ecDNA and that the most common recurrent oncogene amplifications arise on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number matched linear DNA, coupled with enhanced chromatin accessibility. Patients whose tumors have ecDNA-based oncogene amplification showed increase of cell proliferation signature activity, greater likelihood of lymph node spread at initial diagnosis, and significantly shorter survival, even when controlled for tissue type, than do patients whose cancers are not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification plays a central role in driving the poor outcome for patients with some of the most aggressive forms of cancers.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cancer-biology">cancer-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/7/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/7/" class="page-link">7</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/0-100-users/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/10/" class="page-link">10</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/11/" class="page-link">11</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/12/" class="page-link">12</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/9/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/123/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
