<!DOCTYPE html>
<html><head>
  
  <title>2018 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/speed_breeding_in_growth_chambers_and_glasshouses_for_crop_breeding_and_model_plant_research_45092716/">
          <h4 class="font-weight-bold">Speed breeding in growth chambers and glasshouses for crop breeding and model plant research, bioRxiv, 2018-07-16</h4>
          <p>1.AbstractTo meet the challenge of feeding a growing population, breeders and scientists are continuously looking for ways to increase genetic gain in crop breeding. One way this can be achieved is through “speed breeding” (SB), which shortens the breeding cycle and accelerates research studies through rapid generation advancement. The SB method can be carried out in a number of ways, one of which involves extending the duration of a plant’s daily exposure to light (photoperiod) combined with early seed harvest in order to cycle quickly from seed to seed, thereby reducing the generation times for some long-day (LD) or day-neutral crops. Here we present glasshouse and growth chamber-based SB protocols with supporting data from experimentation with several crop species. These protocols describe the growing conditions, including soil media composition, lighting, temperature and spacing, which promote rapid growth of spring and winter bread wheat, durum wheat, barley, oat, various members of the Brassica family, chickpea, pea, grasspea, quinoa and the model grass Brachypodium distachyon. Points of flexibility within the protocols are highlighted, including how plant density can be increased to efficiently scale-up plant numbers for single seed descent (SSD) purposes. Conversely, instructions on how to perform SB on a small-scale by creating a benchtop SB growth cabinet that enables optimization of parameters at a low cost are provided. We also outline the procedure for harvesting and germinating premature wheat, barley and pea seed to reduce generation time. Finally, we provide troubleshooting suggestions to avoid potential pitfalls.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genetic_diversity_patterns_and_domestication_origin_of_soybean_45029693/">
          <h4 class="font-weight-bold">Genetic Diversity Patterns and Domestication Origin of Soybean, bioRxiv, 2018-07-14</h4>
          <p>AbstractUnderstanding diversity and evolution of a crop is an essential step to implement a strategy to expand its germplasm base for crop improvement research. Samples intensively collected from Korea, which is a small but central region in the distribution geography of soybean, were genotyped to provide sufficient data to underpin genome-wide population genetic questions. After removing natural hybrids and duplicated or redundant accessions, we obtained a non-redundant set comprising 1,957 domesticated and 1,079 wild accessions to perform population structure analyses. Our analysis demonstrates that while wild soybean germplasm will require additional sampling from diverse indigenous areas to expand the germplasm base, the current domesticated soybean germplasm is saturated in terms of genetic diversity. We then showed that our genome-wide polymorphism map enabled us to detect genetic loci underling flower color, seed-coat color, and domestication syndrome. A representative soybean set consisting of 194 accessions were divided into one domesticated subpopulation and four wild subpopulations that could be traced back to their geographic collection areas. Population genomics analyses suggested that the monophyletic group of domesticated soybeans was originated in eastern Japan. The results were further substantiated by a phylogenetic tree constructed from domestication-associated single nucleotide polymorphisms identified in this study.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/what_is_a_cognitive_map_organising_knowledge_for_flexible_behaviour_44855365/">
          <h4 class="font-weight-bold">What is a cognitive map? Organising knowledge for flexible behaviour, bioRxiv, 2018-07-10</h4>
          <p>AbstractIt is proposed that a cognitive map encoding the relationships between entities in the world supports flexible behaviour, but the majority of the neural evidence for such a system comes from studies of spatial navigation. Recent work describing neuronal parallels between spatial and non-spatial behaviours has rekindled the notion of a systematic organisation of knowledge across multiple domains. We review experimental evidence and theoretical frameworks that point to principles unifying these apparently disparate functions. These principles describe how to learn and use abstract, generalisable knowledge and suggest map-like representations observed in a spatial context may be an instance of general coding mechanisms capable of organising knowledge of all kinds. We highlight how artificial agents endowed with such principles exhibit flexible behaviour and learn map-like representations observed in the brain. Finally, we speculate on how these principles may offer insight into the extreme generalisations, abstractions and inferences that characterise human cognition.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/single-cell_isoform_rna_sequencing_scisor-seq_across_thousands_of_cells_reveals_isoforms_of_cerebellar_cell_types_44708839/">
          <h4 class="font-weight-bold">Single-cell isoform RNA sequencing (ScISOr-Seq) across thousands of cells reveals isoforms of cerebellar cell types, bioRxiv, 2018-07-09</h4>
          <p>AbstractFull-length isoform sequencing has advanced our knowledge of isoform biology1–11. However, apart from applying full-length isoform sequencing to very few single cells12,13, isoform sequencing has been limited to bulk tissue, cell lines, or sorted cells. Single splicing events have been described for &amp;lt;=200 single cells with great statistical success14,15, but these methods do not describe full-length mRNAs. Single cell short-read 3’ sequencing has allowed identification of many cell sub-types16–23, but full-length isoforms for these cell types have not been profiled. Using our new method of single-cell-isoform-RNA-sequencing (ScISOr-Seq) we determine isoform-expression in thousands of individual cells from a heterogeneous bulk tissue (cerebellum), without specific antibody-fluorescence activated cell sorting. We elucidate isoform usage in high-level cell types such as neurons, astrocytes and microglia and finer sub-types, such as Purkinje cells and Granule cells, including the combination patterns of distant splice sites6–9,24,25, which for individual molecules requires long reads. We produce an enhanced genome annotation revealing cell-type specific expression of known and 16,872 novel (with respect to mouse Gencode version 10) isoforms (see isoformatlas.com).ScISOr-Seq describes isoforms from &amp;gt;1,000 single cells from bulk tissue without cell sorting by leveraging two technologies in three steps In step one, we employ microfluidics to produce amplified full-length cDNAs barcoded for their cell of origin. This cDNA is split into two pools one pool for 3’ sequencing to measure gene expression (step 2) and another pool for long-read sequencing and isoform expression (step 3). In step two, short-read 3’-sequencing provides molecular counts for each gene and cell, which allows clustering cells and assigning a cell type using cell-type specific markers. In step three, an aliquot of the same cDNAs (each barcoded for the individual cell of origin) is sequenced using Pacific Biosciences (“PacBio”)1,2,4,5,26 or Oxford Nanopore3. Since these long reads carry the single-cell barcodes identified in step two, one can determine the individual cell from which each long read originates. Since most single cells are assigned to a named cluster, we can also assign the cell’s cluster name (e.g. “Purkinje cell” or “astrocyte”) to the long read in question (Fig 1A) – without losing the cell of origin of each long read.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/cold-induced_epigenetic_programming_of_the_sperm_enhances_brown_adipose_tissue_activity_in_the_offspring_44730679/">
          <h4 class="font-weight-bold">Cold-induced epigenetic programming of the sperm enhances brown adipose tissue activity in the offspring, Nature Medicine, 2018-07-06</h4>
          <p>Recent research has focused on environmental effects that control tissue functionality and systemic metabolism. However, whether such stimuli affect human thermogenesis and body mass index (BMI) has not been explored. Here we show retrospectively that the presence of brown adipose tissue (BAT) and the season of conception are linked to BMI in humans. In mice, we demonstrate that cold exposure (CE) of males, but not females, before mating results in improved systemic metabolism and protection from diet-induced obesity of the male offspring. Integrated analyses of the DNA methylome and RNA sequencing of the sperm from male mice revealed several clusters of co-regulated differentially methylated regions (DMRs) and differentially expressed genes (DEGs), suggesting that the improved metabolic health of the offspring was due to enhanced BAT formation and increased neurogenesis. The conclusions are supported by cell-autonomous studies in the offspring that demonstrate an enhanced capacity to form mature active brown adipocytes, improved neuronal density and more norepinephrine release in BAT in response to cold stimulation. Taken together, our results indicate that in humans and in mice, seasonal or experimental CE induces an epigenetic programming of the sperm such that the offspring harbor hyperactive BAT and an improved adaptation to overnutrition and hypothermia.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/nature%20medicine">nature medicine</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/phenotypic_age_a_novel_signature_of_mortality_and_morbidity_risk_44563541/">
          <h4 class="font-weight-bold">Phenotypic Age a novel signature of mortality and morbidity risk, bioRxiv, 2018-07-06</h4>
          <p>AbstractBackgroundA person’s rate of aging has important implications for hisher risk of death and disease, thus, quantifying aging using observable characteristics has important applications for clinical, basic, and observational research. We aimed to validate a novel aging measure, “Phenotypic Age”, constructed based on routine clinical chemistry measures, by assessing its applicability for differentiating risk for morbidity and mortality in both healthy and unhealthy populations of various ages.MethodsA nationally representative US sample, NHANES III, was used to derive “Phenotypic Age” based on a linear combination of chronological age and nine multi-system clinical chemistry measures, selected via cox proportional elastic net. Mortality predictions were validated using an independent sample (NHANES IV), consisting of 11,432 participants, for whom we observed a total of 871 deaths, ascertained over 12.6 year of follow-up. Proportional hazard models and ROC curves were used to evaluate predictions.ResultsPhenotypic Age was significantly associated with all-cause mortality and cause-specific mortality. These results were robust to age and sex stratification, and remained even when excluding short-term mortality. Similarly, Phenotypic Age was associated with mortality among seemingly “healthy” participants—defined as those who were disease-free and had normal BMI at baseline—as well as the oldest-old (aged 85&#43;)—a group with high disease burden.ConclusionsPhenotypic Age is a reliable predictor of all-cause and cause-specific mortality in multiple subgroups of the population. Risk stratification by this composite measure is far superior to that of the individual measures that go into it, as well as traditional measures of health. It is able to differentiate individuals who appear healthy, who may have otherwise been missed using traditional health assessments. Further, it can differentiate risk among persons with shared disease burden. Overall, this easily measured metric may be useful in the clinical setting and facilitate secondary and tertiary prevention strategies.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/epidemiology">epidemiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/2018/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/2018/page/57/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/54/" class="page-link">54</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/55/" class="page-link">55</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/56/" class="page-link">56</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/57/" class="page-link">57</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/2018/page/58/" class="page-link">58</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/59/" class="page-link">59</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/60/" class="page-link">60</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/61/" class="page-link">61</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/62/" class="page-link">62</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/2018/page/59/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/2018/page/84/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
