<!DOCTYPE html>
<html><head>
  
  <title>2018 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/single-cell_multi-omic_profiling_of_chromatin_conformation_and_dna_methylome_53170301/">
          <h4 class="font-weight-bold">Single-cell multi-omic profiling of chromatin conformation and DNA methylome, bioRxiv, 2018-12-27</h4>
          <p>AbstractRecent advances in the development of single cell epigenomic assays have facilitated the analysis of gene regulatory landscapes in complex biological systems. Methods for detection of single-cell epigenomic variation such as DNA methylation sequencing and ATAC-seq hold tremendous promise for delineating distinct cell types and identifying their critical cis-regulatory sequences. Emerging evidence has shown that in addition to cis-regulatory sequences, dynamic regulation of 3D chromatin conformation is a critical mechanism for the modulation of gene expression during development and disease. It remains unclear whether single-cell Chromatin Conformation Capture (3C) or Hi-C profiles are suitable for cell type identification and allow the reconstruction of cell-type specific chromatin conformation maps. To address these challenges, we have developed a multi-omic method single-nucleus methyl-3C sequencing (sn-m3C-seq) to profile chromatin conformation and DNA methylation from the same cell. We have shown that bulk m3C-seq and sn-m3C-seq accurately capture chromatin organization information and robustly separate mouse cell types. We have developed a fluorescent-activated nuclei sorting strategy based on DNA content that eliminates nuclei multiplets caused by crosslinking. The sn-m3C-seq method allows high-resolution cell-type classification using two orthogonal types of epigenomic information and the reconstruction of cell-type specific chromatin conformation maps.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_mutational_landscape_of_normal_human_endometrial_epithelium_53099145/">
          <h4 class="font-weight-bold">The mutational landscape of normal human endometrial epithelium, bioRxiv, 2018-12-24</h4>
          <p>All normal somatic cells are thought to acquire mutations. However, characterisation of the patterns and consequences of somatic mutation in normal tissues is limited. Uterine endometrium is a dynamic tissue that undergoes cyclical shedding and reconstitution and is lined by a gland-forming epithelium. Whole genome sequencing of normal endometrial glands showed that most are clonal cell populations derived from a recent common ancestor with mutation burdens differing from other normal cell types and manyfold lower than endometrial cancers. Mutational signatures found ubiquitously account for most mutations. Many, in some women potentially all, endometrial glands are colonised by cell clones carrying driver mutations in cancer genes, often with multiple drivers. Total and driver mutation burdens increase with age but are also influenced by other factors including body mass index and parity. Clones with drivers often originate during early decades of life. The somatic mutational landscapes of normal cells differ between cell types and are revealing the procession of neoplastic change leading to cancer.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cancer-biology">cancer-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/estimating_heritability_of_complex_traits_in_admixed_populations_with_summary_statistics_52956033/">
          <h4 class="font-weight-bold">Estimating heritability of complex traits in admixed populations with summary statistics, bioRxiv, 2018-12-21</h4>
          <p>AbstractAll summary statistics-based methods to estimate the heritability of SNPs (hg2) rely on accurate linkage disequilibrium (LD) calculations. In admixed populations, such as African Americans and Latinos, LD estimates are influenced by admixture and can result in biased hg2 estimates. Here, we introduce covariate-adjusted LD score regression (cov-LDSC), a method to provide robust hg2 estimates from GWAS summary statistics and in-sample LD estimates in admixed populations. In simulations, we observed that unadjusted LDSC underestimates hg2 by 10%-60%; in contrast, cov-LDSC is robust to all simulation parameters. We applied cov-LDSC to approximately 170,000 Latino, 47,000 African American 135,000 European individuals in three quantitative and five dichotomous phenotypes. Our results show that most traits have high concordance of hg2 between ethnic groups; for example in the 23andMe cohort, estimates of hg2 for BMI are 0.22 ± 0.01, 0.23 ± 0.03 and 0.22 ± 0.01 in Latino, African American and European populations respectively. However, for age at menarche, we observe population specific heritability differences with estimates of hg2 of 0.10 ± 0.03, 0.33 ± 0.13 and 0.19 ± 0.01 in Latino, African American and European populations respectively.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/precision_medicine_advancements_using_whole_genome_sequencing_noninvasive_whole_body_imaging_and_functional_diagnostics_52788699/">
          <h4 class="font-weight-bold">Precision Medicine Advancements Using Whole Genome Sequencing, Noninvasive Whole Body Imaging, and Functional Diagnostics, bioRxiv, 2018-12-18</h4>
          <p>ABSTRACTWe report the results of a three-year precision medicine study that enrolled 1190 presumed healthy participants at a single research clinic. To enable a better assessment of disease risk and improve diagnosis, a precision health platform that integrates non-invasive functional measurements and clinical tests combined with whole genome sequencing (WGS) was developed. The platform included WGS, comprehensive quantitative non-contrast whole body (WB) and brain magnetic resonance imagingangiography (MRIMRA), computed tomography (CT) coronary artery calcium scoring, electrocardiogram, echocardiogram, continuous cardiac monitoring, clinical laboratory tests, and metabolomics. In our cohort, 24.3% had medically significant genetic findings (MSF) which may contribute to increased risk of disease. A total of 206 unique medically significant variants in 111 genes were identified, and forty individuals (3.4%) had more than one MSF. Phenotypic testing revealed 34.2% of our cohort had a metabolomics profile suggestive of insulin resistance, 29.2% had elevated liver fat identified by MRI, 16.4% had clinically important cardiac structure or cardiac function abnormalities on cardiac MRI or ECHO, 8.8% had a high cardiovascular risk on CT coronary artery calcium scoring (Agatston calcium score &amp;gt; 400, Relative Risk of 7.2), 8.0% had arrhythmia found on continuous rhythm monitoring, 6.5% had cardiac conduction disorders found on EKG, 2% had previously undetected tumors detected by WB MRI, and 2.5% had previously undetected aneurysms detected by non-contrast MRIMRA. Using family histories, personal histories, and test results, clinical and phenotypic findings were correlated with genomic findings in 130 study participants (63.1%) with high to moderate penetrance variants, suggesting the precision health platform improves the diagnostic process in asymptomatic individuals who were at risk. Cardiovascular and endocrine diseases achieved considerable clinical associations between MSFs and clinical phenotypes (89% and 72%, respectively). These findings demonstrate the value of integrating WGS and noninvasive clinical assessments for a rapid and integrated point-of-care clinical diagnosis of age-related diseases that contribute to premature mortality.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/self-repair_protects_microtubules_from_their_destruction_by_molecular_motors_52789425/">
          <h4 class="font-weight-bold">Self-repair protects microtubules from their destruction by molecular motors, bioRxiv, 2018-12-18</h4>
          <p>Microtubules are dynamic polymers that are used for intracellular transport and chromosome segregation during cell division. Their instability stems from the low energy of tubulin dimer interactions, which sets the growing polymer close to its disassembly conditions. Microtubules function in coordination with kinesin and dynein molecular motors, which use ATP hydrolysis to produce mechanical work and move on microtubules. This raises the possibility that the forces produced by walking motors can break dimer interactions and trigger microtubule disassembly. We tested this hypothesis by studying the interplay between microtubules and moving molecular motors in vitro. Our results show that the mechanical work of molecular motors can remove tubulin dimers from the lattice and rapidly destroy microtubules. This effect was not observed when free tubulin dimers were present in the assay. Using fluorescently labelled tubulin dimers we found that dimer removal by motors was compensated for by the insertion of free tubulin dimers into the microtubule lattice. This self-repair mechanism allows microtubules to survive the damage induced by molecular motors as they move along their tracks. Our study reveals the existence of coupling between the motion of kinesin and dynein motors and the renewal of the microtubule lattice.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/endogenous_variation_in_ventromedial_prefrontal_cortex_state_dynamics_during_naturalistic_viewing_reflects_affective_experience_52756628/">
          <h4 class="font-weight-bold">Endogenous variation in ventromedial prefrontal cortex state dynamics during naturalistic viewing reflects affective experience, bioRxiv, 2018-12-17</h4>
          <p>AbstractHow we process ongoing experiences is shaped by our personal history, current needs, and future goals. Consequently, brain regions involved in generating these subjective appraisals, such as the vmPFC, often appear to be heterogeneous across individuals even in response to the same external information. To elucidate the role of the vmPFC in processing our ongoing experiences, we developed a computational framework and analysis pipeline to characterize the spatiotemporal dynamics of individual vmPFC responses as participants viewed a 45-minute television drama. Through a combination of functional magnetic resonance imaging, facial expression tracking, and self-reported emotional experiences across four studies, our data suggest that the vmPFC slowly transitions through a series of discretized states that broadly map onto affective experiences. Although these transitions typically occur at idiosyncratic times across people, participants exhibited a marked increase in state alignment during high affectively valenced events in the show. Our work suggests that the vmPFC ascribes affective meaning to our ongoing experiences.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        
        <li class="page-item"><a href="/audiences/tags/2018/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/" class="page-link">1</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/2018/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/6/" class="page-link">6</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/2018/page/3/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/2018/page/54/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
