<!DOCTYPE html>
<html><head>
  
  <title>2018 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/single-cell_multi-omic_profiling_of_chromatin_conformation_and_dna_methylome_53170301/">
          <h4 class="font-weight-bold">Single-cell multi-omic profiling of chromatin conformation and DNA methylome, bioRxiv, 2018-12-27</h4>
          <p>AbstractRecent advances in the development of single cell epigenomic assays have facilitated the analysis of gene regulatory landscapes in complex biological systems. Methods for detection of single-cell epigenomic variation such as DNA methylation sequencing and ATAC-seq hold tremendous promise for delineating distinct cell types and identifying their critical cis-regulatory sequences. Emerging evidence has shown that in addition to cis-regulatory sequences, dynamic regulation of 3D chromatin conformation is a critical mechanism for the modulation of gene expression during development and disease. It remains unclear whether single-cell Chromatin Conformation Capture (3C) or Hi-C profiles are suitable for cell type identification and allow the reconstruction of cell-type specific chromatin conformation maps. To address these challenges, we have developed a multi-omic method single-nucleus methyl-3C sequencing (sn-m3C-seq) to profile chromatin conformation and DNA methylation from the same cell. We have shown that bulk m3C-seq and sn-m3C-seq accurately capture chromatin organization information and robustly separate mouse cell types. We have developed a fluorescent-activated nuclei sorting strategy based on DNA content that eliminates nuclei multiplets caused by crosslinking. The sn-m3C-seq method allows high-resolution cell-type classification using two orthogonal types of epigenomic information and the reconstruction of cell-type specific chromatin conformation maps.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_mutational_landscape_of_normal_human_endometrial_epithelium_53099145/">
          <h4 class="font-weight-bold">The mutational landscape of normal human endometrial epithelium, bioRxiv, 2018-12-24</h4>
          <p>All normal somatic cells are thought to acquire mutations. However, characterisation of the patterns and consequences of somatic mutation in normal tissues is limited. Uterine endometrium is a dynamic tissue that undergoes cyclical shedding and reconstitution and is lined by a gland-forming epithelium. Whole genome sequencing of normal endometrial glands showed that most are clonal cell populations derived from a recent common ancestor with mutation burdens differing from other normal cell types and manyfold lower than endometrial cancers. Mutational signatures found ubiquitously account for most mutations. Many, in some women potentially all, endometrial glands are colonised by cell clones carrying driver mutations in cancer genes, often with multiple drivers. Total and driver mutation burdens increase with age but are also influenced by other factors including body mass index and parity. Clones with drivers often originate during early decades of life. The somatic mutational landscapes of normal cells differ between cell types and are revealing the procession of neoplastic change leading to cancer.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/cancer-biology">cancer-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/estimating_heritability_of_complex_traits_in_admixed_populations_with_summary_statistics_52956033/">
          <h4 class="font-weight-bold">Estimating heritability of complex traits in admixed populations with summary statistics, bioRxiv, 2018-12-21</h4>
          <p>AbstractAll summary statistics-based methods to estimate the heritability of SNPs (hg2) rely on accurate linkage disequilibrium (LD) calculations. In admixed populations, such as African Americans and Latinos, LD estimates are influenced by admixture and can result in biased hg2 estimates. Here, we introduce covariate-adjusted LD score regression (cov-LDSC), a method to provide robust hg2 estimates from GWAS summary statistics and in-sample LD estimates in admixed populations. In simulations, we observed that unadjusted LDSC underestimates hg2 by 10%-60%; in contrast, cov-LDSC is robust to all simulation parameters. We applied cov-LDSC to approximately 170,000 Latino, 47,000 African American 135,000 European individuals in three quantitative and five dichotomous phenotypes. Our results show that most traits have high concordance of hg2 between ethnic groups; for example in the 23andMe cohort, estimates of hg2 for BMI are 0.22 ± 0.01, 0.23 ± 0.03 and 0.22 ± 0.01 in Latino, African American and European populations respectively. However, for age at menarche, we observe population specific heritability differences with estimates of hg2 of 0.10 ± 0.03, 0.33 ± 0.13 and 0.19 ± 0.01 in Latino, African American and European populations respectively.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/single-cell_analysis_of_crohns_disease_lesions_identifies_a_pathogenic_cellular_module_associated_with_resistance_to_anti-tnf_therapy_52956753/">
          <h4 class="font-weight-bold">Single-cell analysis of Crohn’s disease lesions identifies a pathogenic cellular module associated with resistance to anti-TNF therapy, bioRxiv, 2018-12-21</h4>
          <p>SummaryClinical benefits to cytokine blockade in ileal Crohn’s disease (iCD) have been limited to a subset of patients. Whether cellular and molecular heterogeneity contributes to variability in treatment responses has been unclear. Using single cell technologies combining scRNAseq, CyTOF and multiplex tissue imaging, we mapped the cellular landscape of inflamed ileum lesions, adjacent non-inflamed ileum and matched circulating blood cells of iCD patients. In inflamed tissues, we identified a pathogenic module characterized by an inflammatory mononuclear phagocyte (Inf.MNP)-associated cellular response organized around inflammatory macrophages and mature dendritic cells in a subset of iCD patients. We confirmed the Inf.MNP-associated cellular response in 4 independent iCD cohorts (n=441) and showed that presence of this pathogenic module at diagnosis correlated with primary resistance to anti-TNF therapy. Single cell mapping of iCD tissues identifies a complex cellular signature of anti-TNF resistance thereby revealing novel biomarkers of treatment response and tailored therapeutic opportunities.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/on-site_ribosome_remodeling_by_locally_synthesized_ribosomal_proteins_in_axons_52905718/">
          <h4 class="font-weight-bold">On-site ribosome remodeling by locally synthesized ribosomal proteins in axons, bioRxiv, 2018-12-20</h4>
          <p>SUMMARYRibosomes are known to be assembled in the nucleolus, yet recent studies have revealed robust enrichment and translation of mRNAs encoding ribosomal proteins (RPs) in axons, far away from neuronal cell bodies. Using subcellular proteomics and live-imaging, we show that locally synthesized RPs incorporate into axonal ribosomes in a nucleolus-independent fashion. We revealed that axonal RP translation is regulated through a novel sequence motif, CUIC, that forms a RNA-loop structure in the region immediately upstream of the initiation codon. Inhibition of axonal CUIC-regulated RP translation leads to defects in local translation activity and axon branching, demonstrating the physiological relevance of the axonal ribosome remodeling. These results indicate that axonal translation supplies cytoplasmic RPs to maintainmodify local ribosomal function far from the nucleolus.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/prioritisation_of_oncology_therapeutic_targets_using_crispr-cas9_screening_52947148/">
          <h4 class="font-weight-bold">Prioritisation of oncology therapeutic targets using CRISPR-Cas9 screening, bioRxiv, 2018-12-20</h4>
          <p>SummaryFunctional genomics approaches can overcome current limitations that hamper oncology drug development such as lack of robust target identification and clinical efficacy. Here we performed genome-scale CRISPR-Cas9 screens in 204 human cancer cell lines from 12 cancer-types and developed a data-driven framework to prioritise cancer therapeutic candidates. We integrated gene cell fitness effects with genomic biomarkers and target tractability for drug development to systematically prioritise new oncology targets in defined tissues and genotypes. Furthermore, we took one of our most promising dependencies, Werner syndrome RecQ helicase, and verified it as a candidate target for tumours with microsatellite instability. Our analysis provides a comprehensive resource of cancer dependencies, a framework to prioritise oncology targets, and nominates specific new candidates. The principles described in this study can transform the initial stages of the drug development process contributing to a new, diverse and more effective portfolio of oncology targets.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/cancer-biology">cancer-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        
        <li class="page-item"><a href="../../../../tags/2018/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="../../../../tags/2018/" class="page-link">1</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/2018/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2018/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2018/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2018/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2018/page/6/" class="page-link">6</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/2018/page/3/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/2018/page/84/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
