<!DOCTYPE html>
<html><head>
  
  <title>0-100-users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.68.3" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.95d2483cd3c5d2bf47c1f36307a456787d60ad958c5efe44db9afc47012b1b05b8e1f64f6ac291650439322a6c38e86eb7cd6254255516d857c23f299fb0e666.js" integrity="sha512-ldJIPNPF0r9HwfNjB6RWeH1grZWMXv5E25r8RwErGwW44fZPasKRZQQ5MipsOOhut81iVCVVFthXwj8pn7DmZg=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/multimodal_cell_type_correspondence_by_intersectional_mfish_in_intact_tissues_54201873/">
          <h4 class="font-weight-bold">Multimodal cell type correspondence by intersectional mFISH in intact tissues, bioRxiv, 2019-01-21</h4>
          <p>AbstractDefining a complete set of cell types within the cortex requires reconciling disparate results achieved through diverging methodologies. To address this correspondence problem, multiple methodologies must be applied to the same cells across multiple single-cell experiments. Here we present a new approach applying spatial transcriptomics using multiplexed fluorescence in situ hybridization, (mFISH) to brain tissue previously interrogated through two photon optogenetic mapping of synaptic connectivity. This approach can resolve the anatomical, transcriptomic, connectomic, electrophysiological, and morphological characteristics of single cells within the mouse cortex.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/deep_phenotyping_of_a_healthy_human_hao1_knockout_informs_therapeutic_development_for_primary_hyperoxaluria_type_1._54137671/">
          <h4 class="font-weight-bold">Deep phenotyping of a healthy human HAO1 knockout informs therapeutic development for primary hyperoxaluria type 1., bioRxiv, 2019-01-19</h4>
          <p>Primary Hyperoxaluria Type 1 (PH1) is a rare autosomal recessive metabolic disorder of oxalate metabolism leading to kidney failure as well as multi-organ damage. Overproduction of oxalate occurs in the liver due to an inherited genetic defect in the enzyme alanine-glyoxylate aminotransferase (AGXT), causing pathology due to the insolubility of calcium oxalate crystals in body fluids. The main current therapy is dual liver-kidney transplant, which incurs high morbidity and has poor availability in some health systems where PH1 is more prevalent. One approach currently in active clinical investigation targets HAO1 (hydroxyacid oxidase 1), encoding glycolate oxidase, to reduce substrate levels for oxalate production. To inform drug development, we sought individuals with reduced HAO1 function due to naturally occurring genetic variation. Analysis of loss of function variants in 141,456 sequenced individuals suggested individuals with complete HAO1 knockout would only be observed in 1 in 30 million outbred people. However in a large sequencing and health records program (Genes &amp;amp; Health), in populations with substantial autozygosity, we identified a healthy adult individual predicted to have complete knockout of HAO1 due to an ultra rare homozygous frameshift variant (rs1186715161, ENSP00000368066.3p.Leu333SerfsTer4). Primary care and hospital health records confirmed no apparently related clinical phenotype. At recall, urine and plasma oxalate levels were normal, however plasma glycolate levels (171 nmolmL) were 12 times the upper limit of normal in healthy, reference individuals (mean&#43;2sd=14 nmolmL, n=67) while her urinary glycolate levels were 6 times the upper limit of normal. Comparison with preclinical and phase 1 clinical trial data of an RNAi therapeutic targeting HAO1 (lumasiran) suggests the individual likely retains &amp;lt;2% residual glycolate oxidase activity. These results provide important data to support the safety of HAO1 inhibition as a potential chronic therapy for a devastating metabolic disease (PH1). We also suggest that the effect of glycolate oxidase suppression in any potential other roles in humans beyond glycolate oxidation do not lead to clinical phenotypes, at least in this specific individual. This demonstrates the value of studying the lifelong complete knockdown of a target protein in a living human to aid development of a potential therapeutic, both in de-risking the approach and providing potential hypotheses to optimize its development. Furthermore, therapy for PH1 is likely to be required lifelong, in contrast to data from chronicity studies in non-human species or relatively short-term therapeutic studies in people. Our approach demonstrates the potential for improved drug discovery through unlocking relevant evidence hiding in the diversity of human genetic variation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/in_vivo_structure_of_the_legionella_type_ii_secretion_system_by_electron_cryotomography_supplementary_material_54157507/">
          <h4 class="font-weight-bold">In vivo structure of the Legionella type II secretion system by electron cryotomography Supplementary material, bioRxiv, 2019-01-19</h4>
          <p>The type II secretion system (T2SS) is a multi-protein envelope-spanning assembly that translocates a wide range of virulence factors, enzymes and effectors through the outer membrane (OM) of many Gram-negative bacteria. Here, using electron cryotomography and subtomogram averaging methods, we present the first in situ structure of an intact T2SS, imaged within the human pathogen Legionella pneumophila. Although the T2SS has only limited sequence and component homology with the evolutionarily-related Type IV pilus (T4P) system, we show that their overall architectures are remarkably similar. Despite similarities, there are also differences, including for instance that the T2SS-ATPase complex is usually present but disengaged from the inner membrane, the T2SS has a much longer periplasmic vestibule, and it has a short-lived flexible pseudopilus. Placing atomic models of the components into our ECT map produced a complete architectural model of the intact T2SS that provides new insights into the structure and function of its components, its position within the cell envelope, and the interactions between its different subcomplexes. Overall, these structural results strongly support the piston model for substrate extrusion.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_dna-based_voltmeter_for_organelles_supplementary_information_54038213/">
          <h4 class="font-weight-bold">A DNA-based voltmeter for organelles Supplementary information, bioRxiv, 2019-01-17</h4>
          <p>The role of membrane potential in most intracellular organelles remains unexplored because of the lack of suitable probes. We describe a DNA-based fluorescent reporter that quantitates membrane potential and can be targeted to specific organelles in live cells. It is equipped with a voltage sensitive fluorophore, a reference fluorophore for ratiometric quantification, and acts as an endocytic tracer. We could thereby measure the membrane potential of different intracellular organelles in living cells, which has not been possible previously. Our understanding of how membrane potential regulates organelle biology is poised to expand through the use of these new sensors.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/epigenetic_suppression_of_interferon_lambda_receptor_expression_leads_to_enhanced_hunov_replication_in_vitro_54057348/">
          <h4 class="font-weight-bold">Epigenetic suppression of interferon lambda receptor expression leads to enhanced HuNoV replication in vitro, bioRxiv, 2019-01-17</h4>
          <p>Human norovirus (HuNoV) is the main cause of gastroenteritis worldwide yet no therapeutics are currently available. Here, we utilize a human norovirus replicon in human gastric tumor (HGT) cells to identify host factors involved in promoting or inhibiting HuNoV replication. We observed that an IFN-cured population of replicon-harboring HGT cells (HGT-cured) was enhanced in their ability to replicate transfected HuNoV RNA compared to parental HGT cells, suggesting that differential gene expression in HGT-cured cells created an environment favouring norovirus replication. Microarray analysis was used to identify genes differentially regulated in HGT-NV and HGT-cured compared to parental HGT cells. We found that the IFN lambda receptor alpha (IFNLR1) expression was highly reduced in HGT-NV and HGT-cured cells. All three cell lines responded to exogenous IFN-β by inducing interferon stimulated genes (ISGs), however, HGT-NV and HGT-cured failed to respond to exogenous IFN-λ. Inhibition of DNA methyltransferase activity with 5-aza-2&#39;-deoxycytidine partially reactivated IFNLR1 expression in HGT-NV and IFN-cured cells suggesting that host adaptation occurred via epigenetic reprogramming. In line with this, ectopic expression of the IFN-λ receptor alpha rescued HGT-NV and HGT-cured cells response to IFN-λ. We conclude that type III IFN is important in inhibiting HuNoV replication in vitro and that the loss of IFNLR1 enhances replication of HuNoV. This study unravels for the first time epigenetic reprogramming of the interferon lambda receptor as a new mechanism of cellular adaptation during long-term RNA virus replication and shows that an endogenous level of interferon lambda signalling is able to control human norovirus replication.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/phyloflash__rapid_ssu_rrna_profiling_and_targeted_assembly_from_metagenomes_supplementary_information_54061503/">
          <h4 class="font-weight-bold">phyloFlash — Rapid SSU rRNA profiling and targeted assembly from metagenomes Supplementary Information, bioRxiv, 2019-01-17</h4>
          <p>The SSU rRNA gene is the key marker in molecular ecology for all domains of life, but is largely absent from metagenome-assembled genomes that often are the only resource available for environmental microbes. Here we present phyloFlash, a pipeline to overcome this gap with rapid, SSU rRNA-centered taxonomic classification, targeted assembly, and graph-based binning of full metagenomic assemblies. We show that a cleanup of artifacts is pivotal even with a curated reference database. With such a filtered database, the general-purpose mapper BBmap extracts SSU rRNA reads five times faster than the rRNA-specialized tool SortMeRNA with similar sensitivity and higher selectivity on simulated metagenomes. Reference-based targeted assemblers yielded either highly fragmented assemblies or high levels of chimerism, so we employ the general-purpose genomic assembler SPAdes. Our optimized implementation is independent of reference database composition and has satisfactory levels of chimera formation. Using the phyloFlash workflow we could recover the first complete genomes of several enigmatic taxa, including Marinamargulisbacteria from surface ocean seawater. phyloFlash quickly processes Illumina (meta)genomic data, is straightforward to use, even as part of high-throughput quality control, and has user-friendly output reports. The software is available at httpsgithub.comHRGVphyloFlash (GPL3 license) and is documented with an online manual.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/53/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/50/" class="page-link">50</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/51/" class="page-link">51</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/52/" class="page-link">52</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/53/" class="page-link">53</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/54/" class="page-link">54</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/55/" class="page-link">55</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/56/" class="page-link">56</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/57/" class="page-link">57</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/58/" class="page-link">58</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/55/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/123/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
