<!DOCTYPE html>
<html><head>
  
  <title>Biorxiv | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/high-speed_multifocal_plane_fluorescence_microscopy_for_three-dimensional_visualisation_of_beating_flagella_56825995/">
          <h4 class="font-weight-bold">High-speed multifocal plane fluorescence microscopy for three-dimensional visualisation of beating flagella, bioRxiv, 2019-03-11</h4>
          <p>AbstractAnalysis of flagellum beating in three dimensions is important for understanding how cells can undergo complex flagellum-driven motility and the ability to use fluorescence microscopy for such three-dimensional analysis would be extremely powerful. Trypanosoma and Leishmania are unicellular parasites which undergo complex cell movements in three dimensions as they swim and would particularly benefit from such an analysis. Here, high-speed multifocal plane fluorescence microscopy, a technique in which a light path multi-splitter is used to visualise 4 focal planes simultaneously, was used to reconstruct the flagellum beating of Trypanosoma brucei and Leishmania mexicana in three dimensions. It was possible to use either an organic fluorescent stain or a genetically-encoded fluorescence fusion protein to visualise flagellum and cell movement in three dimensions at a 200 Hz frame rate. This high-speed multifocal plane fluorescence microscopy approach was used to address two open questions regarding Trypanosoma and Leishmania swimming To quantify the planarity of the L. mexicana flagellum beat and analyse the nature of flagellum beating during T. brucei ‘tumbling’.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/landscape_of_multi-nucleotide_variants_in_125748_human_exomes_and_15708_genomes_56814890/">
          <h4 class="font-weight-bold">Landscape of multi-nucleotide variants in 125,748 human exomes and 15,708 genomes, bioRxiv, 2019-03-11</h4>
          <p>AbstractMulti-nucleotide variants (MNVs), defined as two or more nearby variants existing on the same haplotype in an individual, are a clinically and biologically important class of genetic variation. However, existing tools for variant interpretation typically do not accurately classify MNVs, and understanding of their mutational origins remains limited. Here, we systematically survey MNVs in 125,748 whole exomes and 15,708 whole genomes from the Genome Aggregation Database (gnomAD). We identify 1,996,125 MNVs across the genome with constituent variants falling within 2 bp distance of one another, of which 31,510 exist within the same codon, including 405 predicted to result in gain of a nonsense mutation, 1,818 predicted to rescue a nonsense mutation event that would otherwise be caused by one of the constituent variants, and 16,481 additional variants predicted to alter protein sequences. We show that the distribution of MNVs is highly non-uniform across the genome, and that this non-uniformity can be largely explained by a variety of known mutational mechanisms, such as CpG deamination, replication error by polymerase zeta, or polymerase slippage at repeat junctions. We also provide an estimate of the dinucleotide mutation rate caused by polymerase zeta. Finally, we show that differential CpG methylation drives MNV differences across functional categories. Our results demonstrate the importance of incorporating haplotype-aware annotation for accurate functional interpretation of genetic variation, and refine our understanding of genome-wide mutational mechanisms of MNVs.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/metabolic_diversity_in_human_non-small_cell_lung_cancer_cells_56825986/">
          <h4 class="font-weight-bold">Metabolic Diversity in Human Non-Small Cell Lung Cancer Cells, bioRxiv, 2019-03-11</h4>
          <p>SummaryIntermediary metabolism in cancer cells is regulated by diverse cell-autonomous processes including signal transduction and gene expression patterns arising from specific oncogenotypes and cell lineages. Although it is well established that metabolic reprogramming is a hallmark of cancer, we lack a full view of the diversity of metabolic programs in cancer cells and an unbiased assessment of the associations between metabolic pathway preferences and other cell-autonomous processes. Here we quantified over 100 metabolic features, mostly from 13C enrichment of molecules from central carbon metabolism, in over 80 non-small cell lung cancer (NSCLC) cell lines cultured under identical conditions. Because these cell lines were extensively annotated for oncogenotype, gene expression, protein expression and therapeutic sensitivity, the resulting database enables the user to uncover new relationships between metabolism and these orthogonal processes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cancer-biology">cancer-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_life_of_p.i._transitions_to_independence_in_academia_56814247/">
          <h4 class="font-weight-bold">The life of P.I. Transitions to Independence in Academia, bioRxiv, 2019-03-11</h4>
          <p>The data in this report summarises the responses gathered from 365 principal investigators of academic laboratories, who started their independent positions in the UK within the last 6 years up to 2018. We find that too many new investigators express frustration and poor optimism for the future. These data also reveal, that many of these individuals lack the support required to make a successful transition to independence and that simple measures could be put in place by both funders and universities in order to better support these early career researchers. We use these data to make both recommendations of good practice and for changes to policies that would make significant improvements to those currently finding independence challenging. We find that some new investigators face significant obstacles when building momentum and hiring a research team. In particular, access to PhD students. We also find some important areas such as starting salaries where significant gender differences persist, which cannot be explained by seniority. Our data also underlines the importance of support networks, within and outside the department, and the positive influence of good mentorship through this difficult career stage.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/scientific-communication-and-education">scientific-communication-and-education</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/cytokinin_functions_as_an_asymmetric_and_anti-gravitropic_signal_in_lateral_roots_56786182/">
          <h4 class="font-weight-bold">Cytokinin functions as an asymmetric and anti-gravitropic signal in lateral roots, bioRxiv, 2019-03-10</h4>
          <p>AbstractDirectional organ growth allows the plant root system to strategically cover its surroundings. Intercellular auxin transport is aligned with the gravity vector in the primary root tips, facilitating downward organ bending at the lower root flank. Here we show that cytokinin signaling functions as a lateral root specific anti-gravitropic component, promoting the radial distribution of the root system. We performed a genome-wide association study and revealed that signal peptide processing of Cytokinin Oxidase 2 (CKX2) affects its enzymatic activity and, thereby, determines the degradation of cytokinins in natural Arabidopsis thaliana accessions. Cytokinin signaling interferes with growth at the upper lateral root flank and thereby prevents downward bending. Our interdisciplinary approach revealed that two phytohormonal cues at opposite organ flanks counterbalance each other’s negative impact on growth, suppressing organ growth towards gravity and allow for radial expansion of the root system.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/blue_light_induces_neuronal-activity-regulated_gene_expression_in_the_absence_of_optogenetic_proteins_56756504/">
          <h4 class="font-weight-bold">Blue light induces neuronal-activity-regulated gene expression in the absence of optogenetic proteins, bioRxiv, 2019-03-09</h4>
          <p>Optogenetics is widely used to control diverse cellular functions with light, requiring experimenters to expose cells to bright light. Because extended exposure to visible light can be toxic to cells, it is important to characterize the effects of light stimulation on cellular function in the absence of exogenous optogenetic proteins. Here we exposed cultured mouse cortical neurons that did not express optogenetic proteins to several hours of flashing blue, red, or green light. We found that exposing neurons to as short as one hour of blue, but not red or green, light results in the induction of neuronal-activity-regulated genes without inducing neuronal activity. Our findings suggest blue light stimulation is ill-suited to long-term optogenetic experiments, especially those that measure transcription.Significance StatementOptogenetics is widely used to control cellular functions using light. In neuroscience, channelrhodopsins, exogenous light-sensitive channels, are used to achieve light-dependent control of neuronal firing. This optogenetic control of neuronal firing requires exposing neurons to high-powered light. We ask how this light exposure, in the absence of channelrhodopsin, affects the expression of neuronal-activity-regulated genes, i.e., the genes that are transcribed in response to neuronal stimuli. Surprisingly, we find that neurons without channelrhodopsin express neuronal-activity-regulated genes in response to as short as an hour of blue, but not red or green, light exposure. These findings suggest that experimenters wishing to achieve longer-term (an hour or more) optogenetic control over neuronal firing should avoid using systems that require blue light.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/biorxiv/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/biorxiv/page/47/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biorxiv/page/44/" class="page-link">44</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biorxiv/page/45/" class="page-link">45</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biorxiv/page/46/" class="page-link">46</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biorxiv/page/47/" class="page-link">47</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/biorxiv/page/48/" class="page-link">48</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biorxiv/page/49/" class="page-link">49</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biorxiv/page/50/" class="page-link">50</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biorxiv/page/51/" class="page-link">51</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biorxiv/page/52/" class="page-link">52</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/biorxiv/page/49/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/biorxiv/page/164/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
