<!DOCTYPE html>
<html><head>
  
  <title>Genetics | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/cohesin_disrupts_polycomb-dependent_chromosome_interactions_58112915/">
          <h4 class="font-weight-bold">Cohesin disrupts polycomb-dependent chromosome interactions, bioRxiv, 2019-03-30</h4>
          <p>AbstractHow chromosome organisation is related to genome function remains poorly understood. Cohesin, loop-extrusion, and CTCF have been proposed to create structures called topologically associating domains (TADs) to regulate gene expression. Here, we examine chromosome conformation in embryonic stem cells lacking cohesin and find as in other cell types that cohesin is required to create TADs and regulate AB compartmentalisation. However, in the absence of cohesin we identify a series of long-range chromosomal interactions that persist. These correspond to regions of the genome occupied by the polycomb repressive system, depend on PRC1, and we discover that cohesin counteracts these interactions. This disruptive activity is independent of CTCF and TADs, and regulates gene repression by the polycomb system. Therefore, in contrast to the proposal that cohesin creates structure in chromosomes, we discover a new role for cohesin in disrupting polycomb-dependent chromosome interactions to regulate gene expression.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/an_approximate_full-likelihood_method_for_inferring_selection_and_allele_frequency_trajectories_from_dna_sequence_data_57983959/">
          <h4 class="font-weight-bold">An approximate full-likelihood method for inferring selection and allele frequency trajectories from DNA sequence data, bioRxiv, 2019-03-29</h4>
          <p>AbstractMost current methods for detecting natural selection from DNA sequence data are limited in that they are either based on summary statistics or a composite likelihood, and as a consequence, do not make full use of the information available in DNA sequence data. We here present a new importance sampling approach for approximating the full likelihood function for the selection coefficient. The method treats the ancestral recombination graph (ARG) as a latent variable that is integrated out using previously published Markov Chain Monte Carlo (MCMC) methods. The method can be used for detecting selection, estimating selection coefficients, testing models of changes in the strength of selection, estimating the time of the start of a selective sweep, and for inferring the allele frequency trajectory of a selected or neutral allele. We perform extensive simulations to evaluate the method and show that it uniformly improves power to detect selection compared to current popular methods such as nSL and SDS, under various demographic models and can provide reliable inferences of allele frequency trajectories under many conditions. We also explore the potential of our method to detect extremely recent changes in the strength of selection. We use the method to infer the past allele frequency trajectory for a lactase persistence SNP (MCM6) in Europeans. We also study a set of 11 pigmentation-associated variants. Several genes show evidence of strong selection particularly within the last 5,000 years, including ASIP, KITLG, and TYR. However, selection on OCA2HERC2 seems to be much older and, in contrast to previous claims, we find no evidence of selection on TYRP1.Author summaryCurrent methods to study natural selection using modern population genomic data are limited in their power and flexibility. Here, we present a new method to infer natural selection that builds on recent methodological advances in estimating genome-wide genealogies. By using importance sampling we are able to efficiently estimate the likelihood function of the selection coefficient. We show our method improves power to test for selection over competing methods across a diverse range of scenarios, and also accurately infers the selection coefficient. We also demonstrate a novel capability of our model, using it to infer the allele’s frequency over time. We validate these results with a study of a lactase persistence SNP in Europeans, and also study a set of 11 pigmentation-associated variants.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/clinical_use_of_current_polygenic_risk_scores_may_exacerbate_health_disparities_58049280/">
          <h4 class="font-weight-bold">Clinical use of current polygenic risk scores may exacerbate health disparities, Nature Genetics, 2019-03-29</h4>
          <p>Polygenic risk scores (PRS) are poised to improve biomedical outcomes via precision medicine. However, the major ethical and scientific challenge surrounding clinical implementation of PRS is that those available today are several times more accurate in individuals of European ancestry than other ancestries. This disparity is an inescapable consequence of Eurocentric biases in genome-wide association studies, thus highlighting that—unlike clinical biomarkers and prescription drugs, which may individually work better in some populations but do not ubiquitously perform far better in European populations—clinical uses of PRS today would systematically afford greater improvement for European-descent populations. Early diversifying efforts show promise in leveling this vast imbalance, even when non-European sample sizes are considerably smaller than the largest studies to date. To realize the full and equitable potential of PRS, greater diversity must be prioritized in genetic studies, and summary statistics must be publically disseminated to ensure that health disparities are not increased for those individuals already most underserved.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/nature%20genetics">nature genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/research_grade_marijuana_supplied_by_the_national_institute_on_drug_abuse_is_genetically_divergent_from_commercially_available_cannabis_57987798/">
          <h4 class="font-weight-bold">Research grade marijuana supplied by the National Institute on Drug Abuse is genetically divergent from commercially available Cannabis, bioRxiv, 2019-03-29</h4>
          <p>AbstractPublic comfort with Cannabis (marijuana and hemp) has recently increased, resulting in previously strict Cannabis regulations now allowing hemp cultivation, medical use, and in some states, recreational consumption. There is a growing interest in the potential medical benefits of the various chemical constituents produced by the Cannabis plant. Currently, the University of Mississippi, funded through the National Institutes of HealthNational Institute on Drug Abuse (NIHNIDA), is the sole Drug Enforcement Agency (DEA) licensed facility to cultivate Cannabis for research purposes. Hence, most federally funded research where participants consume Cannabis for medicinal purposes relies on NIDA-supplied product. Previous research found that cannabinoid levels in research grade marijuana supplied by NIDA did not align with commercially available Cannabis from Colorado, Washington and California. Given NIDA chemotypes were misaligned with commercial Cannabis, we sought to investigate where NIDA’s research grade marijuana falls on the genetic spectrum of Cannabis groups. NIDA research grade marijuana was found to genetically group with Hemp samples along with a small subset of commercial drug-type Cannabis. A majority of commercially available drug-type Cannabis was genetically very distinct from NIDA samples. These results suggest that subjects consuming NIDA research grade marijuana may experience different effects than average consumers.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/research_grade_marijuana_supplied_by_the_national_institute_on_drug_abuse_is_genetically_divergent_from_commercially_availablecannabis_57987798/">
          <h4 class="font-weight-bold">Research grade marijuana supplied by the National Institute on Drug Abuse is genetically divergent from commercially availableCannabis, bioRxiv, 2019-03-29</h4>
          <p>AbstractPublic comfort withCannabis(marijuana and hemp) has recently increased, resulting in previously strictCannabisregulations now allowing hemp cultivation, medical use, and in some states, recreational consumption. There is a growing interest in the potential medical benefits of the various chemical constituents produced by theCannabisplant. Currently, the University of Mississippi, funded through the National Institutes of HealthNational Institute on Drug Abuse (NIHNIDA), is the sole Drug Enforcement Agency (DEA) licensed facility to cultivateCannabisfor research purposes. Hence, most federally funded research where participants consumeCannabisfor medicinal purposes relies on NIDA-supplied product. Previous research found that cannabinoid levels in research grade marijuana supplied by NIDA did not align with commercially availableCannabisfrom Colorado, Washington and California. Given NIDA chemotypes were misaligned with commercialCannabis, we sought to investigate where NIDA’s research grade marijuana falls on the genetic spectrum ofCannabisgroups. NIDA research grade marijuana was found to genetically group with Hemp samples along with a small subset of commercial drug-typeCannabis. A majority of commercially available drug-typeCannabiswas genetically very distinct from NIDA samples. These results suggest that subjects consuming NIDA research grade marijuana may experience different effects than average consumers.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genetic_structure_in_the_paternal_lineages_of_south_east_spain_revealed_by_the_analysis_of_17_y-strs_57947784/">
          <h4 class="font-weight-bold">Genetic structure in the paternal lineages of South East Spain revealed by the analysis of 17 Y-STRs, Scientific Reports, 2019-03-26</h4>
          <p>The genetic data of 17 Y chromosome short tandem repeats in 146 unrelated donor residents in the provinces of Granada, Málaga, and Almería (GMA) were analyzed to determine the genetic legacy of the male inhabitants of the former Kingdom of Granada. A total of 139 unique haplotypes were identified. Observed allele frequencies and haplogroup frequencies were also analyzed. By AMOVA and STRUCTURE analysis, the populations of the 3 provinces could be treated genetically as a single population. The most frequent haplogroup was R1b1b2 (58.22%). By network analysis of all individuals, we observed a distribution according to haplogroup assignment. To improve the characterization of GMA population, it was compared with those of North Africa, the Iberian Peninsula, and southern Europe. In our analysis of allele frequencies and genetic distances, the GMA population lay within the Spanish population group. Further, in the STRUCTURE analysis, there was no African component in the GMA population, confirming that, based on our genetic markers, the GMA population does not reflect any male genetic influence of the North African people. The presence of African haplogroups in the GMA population is irrelevant when their frequency is compared with those in other European populations.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/scientific%20reports">scientific reports</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/genetics/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/genetics/page/10/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../../tags/genetics/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/genetics/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/genetics/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/genetics/page/10/" class="page-link">10</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/genetics/page/11/" class="page-link">11</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/genetics/page/12/" class="page-link">12</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/genetics/page/13/" class="page-link">13</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/genetics/page/14/" class="page-link">14</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/genetics/page/15/" class="page-link">15</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/genetics/page/12/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/genetics/page/36/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
