<!DOCTYPE html>
<html><head>
  
  <title>Biorxiv | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/accurate_detection_of_m6a_rna_modifications_in_native_rna_sequences_54225106/">
          <h4 class="font-weight-bold">Accurate detection of m6A RNA modifications in native RNA sequences, bioRxiv, 2019-01-21</h4>
          <p>The field of epitranscriptomics has undergone an enormous expansion in the last few years; however, a major limitation is the lack of generic methods to map RNA modifications transcriptome-wide. Here we show that using Oxford Nanopore Technologies, N6-methyladenosine (m6A) RNA modifications can be detected with high accuracy, in the form of systematic errors and decreased base-calling qualities. Our results open new avenues to investigate the universe of RNA modifications with single nucleotide resolution, in individual RNA molecules.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/pericentromeric_heterochromatin_is_hierarchically_organized_and_spatially_contacts_h3k9me23_islands_located_in_euchromatic_genome_54225676/">
          <h4 class="font-weight-bold">Pericentromeric heterochromatin is hierarchically organized and spatially contacts H3K9me23 islands located in euchromatic genome, bioRxiv, 2019-01-21</h4>
          <p>Membraneless pericentromeric heterochromatin (PCH) domains play vital roles in chromosome dynamics and genome stability. However, our current understanding of 3D genome organization does not include PCH domains because of technical challenges associated with repetitive sequences enriched in PCH genomic regions. We investigated the 3D architecture of Drosophila melanogaster PCH domains and their spatial associations with euchromatic genome by developing a novel analysis method that incorporates genome-wide Hi-C reads originating from PCH DNA. Combined with cytogenetic analysis, we reveal a hierarchical organization of the PCH domains into distinct &#39;territories,&#39; in which &#39;intra-arm&#39; interactions are the most prevalent, followed by 3D contacts between specific PCH regions on different chromosomes. Strikingly, H3K9me23-enriched regions embedded in euchromatic genome show prevalent 3D interactions with the PCH domain. These spatial contacts require H3K9me23 enrichment, are likely mediated by liquid-liquid phase separation, and influence organismal fitness. Our findings have important implications for how PCH architecture influences the function and evolution of both repetitive heterochromatin and the gene-rich euchromatin.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/prospective_brain-wide_labeling_of_neuronal_subclasses_with_enhancer-driven_aavs_54201876/">
          <h4 class="font-weight-bold">Prospective, brain-wide labeling of neuronal subclasses with enhancer-driven AAVs, bioRxiv, 2019-01-21</h4>
          <p>Labeling and perturbation of specific cell types in multicellular systems has transformed our ability to understand them. The rapid pace of cell type identification by new single-cell analysis methods has not been met with efficient access to these newly discovered types. To enable access to specific neural populations in the mouse cortex, we have collected single cell chromatin accessibility data from select cell types. We clustered the single cell data and mapped them to single cell transcriptomics to identify highly specific enhancers for cell subclasses. These enhancers, when cloned into AAVs and delivered to the brain by retro orbital injections, transgene expression in specific cell subclasses throughout the mouse brain. This approach will enable functional investigation of cell types in the mouse cortex and beyond.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/deep_phenotyping_of_a_healthy_human_hao1_knockout_informs_therapeutic_development_for_primary_hyperoxaluria_type_1._54137671/">
          <h4 class="font-weight-bold">Deep phenotyping of a healthy human HAO1 knockout informs therapeutic development for primary hyperoxaluria type 1., bioRxiv, 2019-01-19</h4>
          <p>Primary Hyperoxaluria Type 1 (PH1) is a rare autosomal recessive metabolic disorder of oxalate metabolism leading to kidney failure as well as multi-organ damage. Overproduction of oxalate occurs in the liver due to an inherited genetic defect in the enzyme alanine-glyoxylate aminotransferase (AGXT), causing pathology due to the insolubility of calcium oxalate crystals in body fluids. The main current therapy is dual liver-kidney transplant, which incurs high morbidity and has poor availability in some health systems where PH1 is more prevalent. One approach currently in active clinical investigation targets HAO1 (hydroxyacid oxidase 1), encoding glycolate oxidase, to reduce substrate levels for oxalate production. To inform drug development, we sought individuals with reduced HAO1 function due to naturally occurring genetic variation. Analysis of loss of function variants in 141,456 sequenced individuals suggested individuals with complete HAO1 knockout would only be observed in 1 in 30 million outbred people. However in a large sequencing and health records program (Genes &amp;amp; Health), in populations with substantial autozygosity, we identified a healthy adult individual predicted to have complete knockout of HAO1 due to an ultra rare homozygous frameshift variant (rs1186715161, ENSP00000368066.3p.Leu333SerfsTer4). Primary care and hospital health records confirmed no apparently related clinical phenotype. At recall, urine and plasma oxalate levels were normal, however plasma glycolate levels (171 nmolmL) were 12 times the upper limit of normal in healthy, reference individuals (mean&#43;2sd=14 nmolmL, n=67) while her urinary glycolate levels were 6 times the upper limit of normal. Comparison with preclinical and phase 1 clinical trial data of an RNAi therapeutic targeting HAO1 (lumasiran) suggests the individual likely retains &amp;lt;2% residual glycolate oxidase activity. These results provide important data to support the safety of HAO1 inhibition as a potential chronic therapy for a devastating metabolic disease (PH1). We also suggest that the effect of glycolate oxidase suppression in any potential other roles in humans beyond glycolate oxidation do not lead to clinical phenotypes, at least in this specific individual. This demonstrates the value of studying the lifelong complete knockdown of a target protein in a living human to aid development of a potential therapeutic, both in de-risking the approach and providing potential hypotheses to optimize its development. Furthermore, therapy for PH1 is likely to be required lifelong, in contrast to data from chronicity studies in non-human species or relatively short-term therapeutic studies in people. Our approach demonstrates the potential for improved drug discovery through unlocking relevant evidence hiding in the diversity of human genetic variation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/identification_and_mitigation_of_pervasive_off-target_activity_in_crispr-cas9_screens_for_essential_non-coding_elements_supplementary_information_54136426/">
          <h4 class="font-weight-bold">Identification and mitigation of pervasive off-target activity in CRISPR-Cas9 screens for essential non-coding elements Supplementary Information, bioRxiv, 2019-01-19</h4>
          <p>Pooled CRISPR-Cas9 screens have recently emerged as a powerful method for functionally characterizing regulatory elements in the non-coding genome, but off-target effects in these experiments have not been systematically evaluated. Here, we conducted a genome-scale screen for essential CTCF loop anchors in the K562 leukemia cell line. Surprisingly, the primary drivers of signal in this screen were single guide RNAs (sgRNAs) with low specificity scores. After removing these guides, we found that there were no CTCF loop anchors critical for cell growth. We also observed this effect in an independent screen fine-mapping the core motifs in enhancers of the GATA1 gene. We then conducted screens in parallel with CRISPRi and CRISPRa, which do not induce DNA damage, and found that an unexpected and distinct set of off-targets also caused strong confounding growth effects with these epigenome-editing platforms. Promisingly, strict filtering of CRISPRi libraries using GuideScan specificity scores removed these confounded sgRNAs and allowed for the identification of essential enhancers, which we validated extensively. Together, our results show off-target activity can severely limit identification of essential functional motifs by active Cas9, while strictly filtered CRISPRi screens can be reliably used for assaying larger regulatory elements.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/in_vivo_structure_of_the_legionella_type_ii_secretion_system_by_electron_cryotomography_supplementary_material_54157507/">
          <h4 class="font-weight-bold">In vivo structure of the Legionella type II secretion system by electron cryotomography Supplementary material, bioRxiv, 2019-01-19</h4>
          <p>The type II secretion system (T2SS) is a multi-protein envelope-spanning assembly that translocates a wide range of virulence factors, enzymes and effectors through the outer membrane (OM) of many Gram-negative bacteria. Here, using electron cryotomography and subtomogram averaging methods, we present the first in situ structure of an intact T2SS, imaged within the human pathogen Legionella pneumophila. Although the T2SS has only limited sequence and component homology with the evolutionarily-related Type IV pilus (T4P) system, we show that their overall architectures are remarkably similar. Despite similarities, there are also differences, including for instance that the T2SS-ATPase complex is usually present but disengaged from the inner membrane, the T2SS has a much longer periplasmic vestibule, and it has a short-lived flexible pseudopilus. Placing atomic models of the components into our ECT map produced a complete architectural model of the intact T2SS that provides new insights into the structure and function of its components, its position within the cell envelope, and the interactions between its different subcomplexes. Overall, these structural results strongly support the piston model for substrate extrusion.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/biorxiv/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/biorxiv/page/73/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biorxiv/page/70/" class="page-link">70</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biorxiv/page/71/" class="page-link">71</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biorxiv/page/72/" class="page-link">72</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biorxiv/page/73/" class="page-link">73</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/biorxiv/page/74/" class="page-link">74</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biorxiv/page/75/" class="page-link">75</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biorxiv/page/76/" class="page-link">76</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biorxiv/page/77/" class="page-link">77</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biorxiv/page/78/" class="page-link">78</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/biorxiv/page/75/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/biorxiv/page/164/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
