<!DOCTYPE html>
<html><head>
  
  <title>2017 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/reprogramming_human_t_cell_function_and_specificity_with_non-viral_genome_targeting_24597008/">
          <h4 class="font-weight-bold">Reprogramming human T cell function and specificity with non-viral genome targeting, bioRxiv, 2017-09-01</h4>
          <p>Human T cells are central to physiological immune homeostasis, which protects us from pathogens without collateral autoimmune inflammation. They are also the main effectors in most current cancer immunotherapy strategies1. Several decades of work have aimed to genetically reprogram T cells for therapeutic purposes2–5, but as human T cells are resistant to most standard methods of large DNA insertion these approaches have relied on recombinant viral vectors, which do not target transgenes to specific genomic sites6, 7. In addition, the need for viral vectors has slowed down research and clinical use as their manufacturing and testing is lengthy and expensive. Genome editing brought the promise of specific and efficient insertion of large transgenes into target cells through homology-directed repair (HDR), but to date in human T cells this still requires viral transduction8, 9. Here, we developed a non-viral, CRISPR-Cas9 genome targeting system that permits the rapid and efficient insertion of individual or multiplexed large (&amp;gt;1 kilobase) DNA sequences at specific sites in the genomes of primary human T cells while preserving cell viability and function. We successfully tested the potential therapeutic use of this approach in two settings. First, we corrected a pathogenic IL2RA mutation in primary T cells from multiple family members with monogenic autoimmune disease and demonstrated enhanced signalling function. Second, we replaced the endogenous T cell receptor (TCR) locus with a new TCR redirecting T cells to a cancer antigen. The resulting TCR-engineered T cells specifically recognized the tumour antigen, with concomitant cytokine release and tumour cell killing. Taken together, these studies provide preclinical evidence that non-viral genome targeting will enable rapid and flexible experimental manipulation and therapeutic engineering of primary human immune cells.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/inter-homologue_repair_in_fertilized_human_eggs_24442244/">
          <h4 class="font-weight-bold">Inter-homologue repair in fertilized human eggs?, bioRxiv, 2017-08-29</h4>
          <p>Many human diseases have an underlying genetic component. The development and application of methods to prevent the inheritance of damaging mutations through the human germline could have significant health benefits, and currently include preimplantation genetic diagnosis and carrier screening. Ma et al. take this a step further by attempting to remove a disease mutation from the human germline through gene editing1. They assert the following advances (i) the correction of a pathogenic gene mutation responsible for hypertrophic cardiomyopathy in human embryos using CRISPR-Cas9 and (ii) the avoidance of mosaicism in edited embryos. In the case of correction, the authors conclude that repair using the homologous chromosome was as or more frequent than mutagenic nonhomologous end-joining (NHEJ). Their conclusion is significant, if validated, because such a “self-repair” mechanism would allow gene correction without the introduction of a repair template. While the authors’ analyses relied on the failure to detect mutant alleles, here we suggest approaches to provide direct evidence for inter-homologue recombination and discuss other events consistent with the data. We also review the biological constraints on inter-homologue recombination in the early embryo.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/patterns_of_structural_variation_in_human_cancer_24405711/">
          <h4 class="font-weight-bold">Patterns of structural variation in human cancer, bioRxiv, 2017-08-28</h4>
          <p>ABSTRACTA key mutational process in cancer is structural variation, in which rearrangements delete, amplify or reorder genomic segments ranging in size from kilobases to whole chromosomes. We developed methods to group, classify and describe structural variants, applied to &amp;gt;2,500 cancer genomes. Nine signatures of structural variation emerged. Deletions have trimodal size distribution; assort unevenly across tumour types and patients; enrich in late-replicating regions; and correlate with inversions. Tandem duplications also have trimodal size distribution, but enrich in early-replicating regions, as do unbalanced translocations. Replication-based mechanisms of rearrangement generate varied chromosomal structures with low-level copy number gains and frequent inverted rearrangements. One prominent structure consists of 1-7 templates copied from distinct regions of the genome strung together within one locus. Such ‘cycles of templated insertions’ correlate with tandem duplications, frequently activating the telomerase gene, TERT, in liver cancer. Cancers access many rearrangement processes, flexibly sculpting the genome to maximise oncogenic potential.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cancer-biology">cancer-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/an_improved_atac-seq_protocol_reduces_background_and_enables_interrogation_of_frozen_tissues_24369319/">
          <h4 class="font-weight-bold">An improved ATAC-seq protocol reduces background and enables interrogation of frozen tissues, bioRxiv, 2017-08-27</h4>
          <p>ABSTRACTWe present Omni-ATAC, an improved ATAC-seq protocol for chromatin accessibility profiling that works across multiple applications with substantial improvement of signal-to-background ratio and information content. The Omni-ATAC protocol enables chromatin accessibility profiling from archival frozen tissue samples and 50 μm sections, revealing the activities of disease-associated DNA elements in distinct human brain structures. The Omni-ATAC protocol enables the interrogation of personal regulomes in tissue context and translational studies.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_neural_algorithm_for_a_fundamental_computing_problem_24305847/">
          <h4 class="font-weight-bold">A neural algorithm for a fundamental computing problem, bioRxiv, 2017-08-26</h4>
          <p>Similarity search, such as identifying similar images in a database or similar documents on the Web, is a fundamental computing problem faced by many large-scale information retrieval systems. We discovered that the fly’s olfac-tory circuit solves this problem using a novel variant of a traditional computer science algorithm (called locality-sensitive hashing). The fly’s circuit assigns similar neural activity patterns to similar input stimuli (odors), so that behav-iors learned from one odor can be applied when a similar odor is experienced. The fly’s algorithm, however, uses three new computational ingredients that depart from traditional approaches. We show that these ingredients can be translated to improve the performance of similarity search compared to tra-ditional algorithms when evaluated on several benchmark datasets. Overall, this perspective helps illuminate the logic supporting an important sensory function (olfaction), and it provides a conceptually new algorithm for solving a fundamental computational problem.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_10000_immunomes_project_a_resource_for_human_immunology_24305832/">
          <h4 class="font-weight-bold">The 10,000 Immunomes Project A resource for human immunology, bioRxiv, 2017-08-26</h4>
          <p>AbstractNew immunological assays now enable rich measurements of human immune function, but difficulty attaining enough measurements across sufficiently large and diverse cohorts has hindered describing normal human immune physiology on a large scale. Here we present the 10,000 Immunomes Project (10KIP), a diverse human immunology reference derived from over 44,000 individuals across 242 studies from ImmPort, a publicly available resource of raw immunology study data and protocols. We carefully curated datasets, aggregating subjects from healthycontrol arms and harmonizing data across studies. We demonstrate 10KIP’s utility by describing variations in serum cytokines and leukocytes by age, race, and sex; defining a baseline cell-cytokine network; and using 10KIP as a common control to describe immunologic changes in pregnancy. Subject-level data is available for interactive visualization and download at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=http10kImmunomes.org&gt;http10kImmunomes.org&lt;jatsext-link&gt;. We believe 10KIP can serve as a common control cohort and will accelerate hypothesis generation by clinical and basic immunologists across diverse populations.One Sentence SummaryAn open online resource of human immunology data from more than 10,000 normal subjects including interactive data visualization and download enables a new look at immune system differences across age and sex, rapid hypothesis generation, and creation of custom control cohorts.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/2017/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/2017/page/30/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2017/page/27/" class="page-link">27</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2017/page/28/" class="page-link">28</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2017/page/29/" class="page-link">29</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2017/page/30/" class="page-link">30</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/2017/page/31/" class="page-link">31</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2017/page/32/" class="page-link">32</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2017/page/33/" class="page-link">33</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2017/page/34/" class="page-link">34</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2017/page/35/" class="page-link">35</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/2017/page/32/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/2017/page/62/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
