<!DOCTYPE html>
<html><head>
  
  <title>Biorxiv | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/gene_regulatory_network_reconstruction_using_single-cell_rna_sequencing_of_barcoded_genotypes_in_diverse_environments_57249349/">
          <h4 class="font-weight-bold">Gene regulatory network reconstruction using single-cell RNA sequencing of barcoded genotypes in diverse environments, bioRxiv, 2019-03-19</h4>
          <p>AbstractUnderstanding how gene expression programs are controlled requires identifying regulatory relationships between transcription factors and target genes. Gene regulatory networks are typically constructed from gene expression data acquired following genetic perturbation or environmental stimulus. Single-cell RNA sequencing (scRNAseq) captures the gene expression state of thousands of individual cells in a single experiment, offering advantages in combinatorial experimental design, large numbers of independent measurements, and accessing the interaction between the cell cycle and environmental responses that is hidden by population-level analysis of gene expression. To leverage these advantages, we developed a method for transcriptionally barcoding gene deletion mutants and performing scRNAseq in budding yeast (Saccharomyces cerevisiae). We pooled diverse genotypes in 11 different environmental conditions and determined their expression state by sequencing 38,285 individual cells. We developed, and benchmarked, a framework for learning gene regulatory networks from scRNAseq data that incorporates multitask learning and constructed a global gene regulatory network comprising 12,018 interactions. Our study establishes a general approach to gene regulatory network reconstruction from scRNAseq data that can be employed in any organism.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/predicting_the_effects_of_snps_on_transcription_factor_binding_affinity_57246676/">
          <h4 class="font-weight-bold">Predicting the effects of SNPs on transcription factor binding affinity, bioRxiv, 2019-03-19</h4>
          <p>AbstractGWAS have revealed that 88% of disease associated SNPs reside in noncoding regions. However, noncoding SNPs remain understudied, partly because they are challenging to prioritize for experimental validation. To address this deficiency, we developed the SNP effect matrix pipeline (SEMpl). SEMpl estimates transcription factor binding affinity by observing differences in ChIP-seq signal intensity for SNPs within functional transcription factor binding sites genome-wide. By cataloging the effects of every possible mutation within the transcription factor binding site motif, SEMpl can predict the consequences of SNPs to transcription factor binding. This knowledge can be used to identify potential disease-causing regulatory loci.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/ribosome_profiling_at_isoform_level_reveals_an_evolutionary_conserved_impact_of_differential_splicing_on_the_proteome_57284671/">
          <h4 class="font-weight-bold">Ribosome profiling at isoform level reveals an evolutionary conserved impact of differential splicing on the proteome, bioRxiv, 2019-03-19</h4>
          <p>AbstractThe differential production of transcript isoforms from gene loci is a key cellular mechanism. Yet, its impact in protein production remains an open question. Here, we describe ORQAS (ORF quantification pipeline for alternative splicing) a new pipeline for the translation quantification of individual transcript isoforms using ribosome-protected mRNA fragments (Ribosome profiling). We found evidence of translation for 40-50% of the expressed transcript isoforms in human and mouse, with 53% of the expressed genes having more than one translated isoform in human, 33% in mouse. Differential analysis revealed that about 40% of the splicing changes at RNA level were concordant with changes in translation, with 21.7% of changes at RNA level and 17.8% at translational level conserved between human and mouse. Furthermore, orthologous cassette exons preserving the directionality of the change were found enriched in microexons in a comparison between glia and glioma, and were conserved between human and mouse. ORQAS leverages ribosome profiling to uncover a widespread and evolutionary conserved impact of differential splicing on the translation of isoforms and in particular, of microexon-containing ones. ORQAS is available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsgithub.comcomprnaorqas&gt;httpsgithub.comcomprnaorqas&lt;jatsext-link&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/stag1_and_stag2_regulate_cell_fate_decisions_in_hematopoiesis_through_non-redundant_topological_control_57258080/">
          <h4 class="font-weight-bold">Stag1 and Stag2 regulate cell fate decisions in hematopoiesis through non-redundant topological control, bioRxiv, 2019-03-19</h4>
          <p>AbstractTranscriptional regulators, including the cohesin complex member STAG2, are recurrently mutated in cancer. The role of STAG2 in gene regulation, hematopoiesis, and tumor suppression remains unresolved. We show Stag2 deletion in hematopoietic stemprogenitor cells (HSPC) results in altered hematopoietic function, increased self-renewal, and impaired differentiation. ChIP-sequencing revealed that while Stag2 and Stag1 can bind the same loci, a component of Stag2 binding sites are unoccupied by Stag1 even in Stag2-deficient HSPCs. While concurrent loss of Stag2 and Stag1 abrogated hematopoiesis, Stag2 loss alone decreased chromatin accessibility and transcription of lineage-specification genes, including Ebf1 and Pax5, leading to blunted HSPC commitment to the B-cell lineage. Our data illustrate a role for Stag2 in transformation and transcriptional dysregulation distinct from its shared role with Stag1 in chromosomal segregation.One Sentence SummaryStag1 rescues topologically associated domains in the absence of Stag2, but cannot restore chromatin architecture required for hematopoietic lineage commitment</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/the_mesospim_initiative_open-source_light-sheet_mesoscopes_for_imaging_in_cleared_tissue_57246705/">
          <h4 class="font-weight-bold">The mesoSPIM initiative open-source light-sheet mesoscopes for imaging in cleared tissue, bioRxiv, 2019-03-19</h4>
          <p></p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/ultrasound_imaging_of_gene_expression_in_mammalian_cells_57245862/">
          <h4 class="font-weight-bold">Ultrasound Imaging of Gene Expression in Mammalian Cells, bioRxiv, 2019-03-19</h4>
          <p>ABSTRACTThe study of cellular processes occurring inside intact organisms and the development of cell-based diagnostic and therapeutic agents requires methods to visualize cellular functions such as gene expression in deep tissues. Ultrasound is a widely used biomedical technology enabling deep-tissue imaging with high spatial and temporal resolution. However, no genetically encoded molecular reporters are available to connect ultrasound contrast to gene expression in mammalian cells. To address this limitation, we introduce the first mammalian acoustic reporter genes. Starting with an eleven-gene polycistronic gene cluster derived from bacteria, we engineered a eukaryotic genetic program whose introduction into mammalian cells results in the expression of a unique class of intracellular air-filled protein nanostructures called gas vesicles. The scattering of ultrasound by these nanostructures allows mammalian cells to be visualized at volumetric densities below 0.5%, enables the monitoring of dynamic circuit-driven gene expression, and permits high-resolution imaging of gene expression in living animals. These mammalian acoustic reporter genes enable previously impossible approaches to monitoring the location, viability and function of mammalian cells in vivo.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioengineering">bioengineering</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/biorxiv/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/biorxiv/page/42/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/39/" class="page-link">39</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/40/" class="page-link">40</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/41/" class="page-link">41</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/42/" class="page-link">42</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/biorxiv/page/43/" class="page-link">43</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/44/" class="page-link">44</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/45/" class="page-link">45</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/46/" class="page-link">46</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/47/" class="page-link">47</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/biorxiv/page/44/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/biorxiv/page/164/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
