<!DOCTYPE html>
<html><head>
  
  <title>100-200-users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.68.3" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.95d2483cd3c5d2bf47c1f36307a456787d60ad958c5efe44db9afc47012b1b05b8e1f64f6ac291650439322a6c38e86eb7cd6254255516d857c23f299fb0e666.js" integrity="sha512-ldJIPNPF0r9HwfNjB6RWeH1grZWMXv5E25r8RwErGwW44fZPasKRZQQ5MipsOOhut81iVCVVFthXwj8pn7DmZg=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/design_of_a_biosensor_for_direct_visualisation_of_auxin_74255335/">
          <h4 class="font-weight-bold">Design of a biosensor for direct visualisation of auxin, bioRxiv, 2020-01-20</h4>
          <p>In plants, one of the most important regulative small molecules is indole-3-acetic acid (IAA) known as auxin. Its dynamic redistribution plays an essential role in virtually every aspect of plant life, ranging from cell shape and division to organogenesis and responses to light and gravity1,2. So far, the spatial and temporal distribution of auxin at cellular resolution could not be determined directly. Instead it has been inferred from visualisation of irreversible processes involving the endogenous auxin response machinery3-7. This detection system failed to record transient changes. Here we report on a genetically encoded biosensor for quantitative in vivo visualisation of auxin distributions. The sensor is based on the E. coli tryptophan repressor (TrpR)8 whose binding pocket was engineered for specific IAA binding and coupled to fluorescent proteins to employ FRET as readout. This sensor, unlike previous systems, enables direct monitoring of the fast uptake and clearance of auxin by individual cells in the plant as well as the graded spatial distribution along the root axis and its perturbation by transport inhibitors. Thus, our auxin sensor enables mapping of auxin concentrations at (sub)cellular resolution and their changes in time and space during plant life.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/dual_proteome-scale_networks_reveal_cell-specific_remodeling_of_the_human_interactome_74254429/">
          <h4 class="font-weight-bold">Dual Proteome-scale Networks Reveal Cell-specific Remodeling of the Human Interactome, bioRxiv, 2020-01-20</h4>
          <p>SUMMARYThousands of interactions assemble proteins into modules that impart spatial and functional organization to the cellular proteome. Through affinity-purification mass spectrometry, we have created two proteome-scale, cell-line-specific interaction networks. The first, BioPlex 3.0, results from affinity purification of 10,128 human proteins – half the proteome – in 293T cells and includes 118,162 interactions among 14,586 proteins; the second results from 5,522 immunoprecipitations in HCT116 cells. These networks model the interactome at unprecedented scale, encoding protein function, localization, and complex membership. Their comparison validates thousands of interactions and reveals extensive customization of each network. While shared interactions reside in core complexes and involve essential proteins, cell-specific interactions bridge conserved complexes, likely ‘rewiring’ each cell’s interactome. Interactions are gained and lost in tandem among proteins of shared function as the proteome remodels to produce each cell’s phenotype. Viewable interactively online through BioPlexExplorer, these networks define principles of proteome organization and enable unknown protein characterization.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/accurate_and_versatile_3d_segmentation_of_plant_tissues_at_cellular_resolution_74218892/">
          <h4 class="font-weight-bold">Accurate and Versatile 3D Segmentation of Plant Tissues at Cellular Resolution, bioRxiv, 2020-01-19</h4>
          <p>ABSTRACTQuantitative analysis of plant and animal morphogenesis requires accurate segmentation of individual cells in volumetric images of growing organs. In the last years, deep learning has provided robust automated algorithms that approach human performance, with applications to bio-image analysis now starting to emerge. Here, we present PlantSeg, a pipeline for volumetric segmentation of plant tissues into cells. PlantSeg employs a convolutional neural network to predict cell boundaries and graph partitioning to segment cells based on the neural network predictions. PlantSeg was trained on fixed and live plant organs imaged with confocal and light sheet microscopes. PlantSeg delivers accurate results and generalizes well across different tissues, scales, and acquisition settings. We present results of PlantSeg applications in diverse developmental contexts. PlantSeg is free and open-source, with both a command line and a user-friendly graphical interface.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/trans-ethnic_and_ancestry-specific_blood-cell_genetics_in_746667_individuals_from_5_global_populations_74206183/">
          <h4 class="font-weight-bold">Trans-ethnic and ancestry-specific blood-cell genetics in 746,667 individuals from 5 global populations, bioRxiv, 2020-01-19</h4>
          <p>SUMMARYMost loci identified by GWAS have been found in populations of European ancestry (EA). In trans-ethnic meta-analyses for 15 hematological traits in 746,667 participants, including 184,535 non-EA individuals, we identified 5,552 trait-variant associations at P&amp;lt;5×10−9, including 71 novel loci not found in EA populations. We also identified novel ancestry-specific variants not found in EA, including an IL7 missense variant in South Asians associated with lymphocyte count in vivo and IL7 secretion levels in vitro. Fine-mapping prioritized variants annotated as functional, and generated 95% credible sets that were 30% smaller when using the trans-ethnic as opposed to the EA-only results. We explored the clinical significance and predictive value of trans-ethnic variants in multiple populations, and compared genetic architecture and the impact of natural selection on these blood phenotypes between populations. Altogether, our results for hematological traits highlight the value of a more global representation of populations in genetic studies.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/multiplexed_single-cell_metabolic_profiles_organize_the_spectrum_of_cytotoxic_human_t_cells_74169828/">
          <h4 class="font-weight-bold">Multiplexed Single-cell Metabolic Profiles Organize the Spectrum of Cytotoxic Human T Cells, bioRxiv, 2020-01-18</h4>
          <p>SummaryCellular metabolism regulates immune cell activation, differentiation and effector functions to the extent that its perturbation can augment immune responses. However, the analytical technologies available to study cellular metabolism lack single-cell resolution, obscuring metabolic heterogeneity and its connection to immune phenotype and function. To that end, we utilized high-dimensional, antibody-based technologies to simultaneously quantify the single-cell metabolic regulome in combination with phenotypic identity. Mass cytometry (CyTOF)-based application of this approach to early human T cell activation enabled the comprehensive reconstruction of the coordinated metabolic remodeling of naïve CD8&#43; T cells and aligned with conventional bulk assays for glycolysis and oxidative phosphorylation. Extending this analysis to a variety of tissue-resident immune cells revealed tissue-restricted metabolic states of human cytotoxic T cells, including metabolically repressed subsets that expressed CD39 and PD1 and that were enriched in colorectal carcinoma versus healthy adjacent tissue. Finally, combining this approach with multiplexed ion beam imaging by time-of-flight (MIBI-TOF) demonstrated the existence of spatially enriched metabolic neighborhoods, independent of cell identity and additionally revealed exclusion of metabolically repressed cytotoxic T cell states from the tumor-immune boundary in human colorectal carcinoma. Overall, we provide an approach that permits the robust approximation of metabolic states in individual cells along with multimodal analysis of cell identity and functional characteristics that can be applied to human clinical samples to study cellular metabolism how it may be perturbed to affect immunological outcomes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/investigating_the_genetic_architecture_of_non-cognitive_skills_using_gwas-by-subtraction_74072515/">
          <h4 class="font-weight-bold">Investigating the Genetic Architecture of Non-Cognitive Skills Using GWAS-by-Subtraction, bioRxiv, 2020-01-16</h4>
          <p>AbstractEducational attainment (EA) is influenced by cognitive abilities and by other characteristics and traits. However little is known about the genetic architecture of these “non-cognitive” contributions to EA. Here, we use Genomic Structural Equation Modelling and results of prior genome-wide association studies (GWASs) of EA (N = 1,131,881) and cognitive test performance (N = 257,841) to estimate SNP associations with variation in EA that is independent of cognitive ability. We identified 157 genome-wide significant loci and a polygenic architecture accounting for 57% of genetic variance in EA. Phenotypic and biological annotation revealed that (1) both cognitive and non-cognitive contributions to EA were genetically correlated with socioeconomic success and longevity; and (2) non-cognitive contributions to EA were related to personality, decision making, risk-behavior, and increased risk for psychiatric disorders; (3) non-cognitive and cognitive contributions to EA were enriched in the same tissues and cell types, but (4) showed different associations with gray-matter neuroimaging phenotypes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2020">2020</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/" class="page-link">1</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/6/" class="page-link">6</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/3/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/126/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
