<!DOCTYPE html>
<html><head>
  
  <title>2019 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_comparison_of_edna_to_camera_trapping_for_assessment_of_terrestrial_mammal_diversity_60254828/">
          <h4 class="font-weight-bold">A comparison of eDNA to camera trapping for assessment of terrestrial mammal diversity, bioRxiv, 2019-05-11</h4>
          <p>AbstractEnvironmental DNA (eDNA) is one of the most promising approaches to meet the demand for the fast and frequent monitoring of ecosystems needed to tackle the current decline in biodiversity. However, before eDNA can establish itself as a robust alternative for mammal monitoring, comparison with existing approaches is necessary, yet has not been done. Moreover, much is unknown regarding the nature, spread and persistence of DNA shed by animals into terrestrial environments, or the optimal experimental design for understanding these potential biases.To address some of these challenges, we compared the detection of terrestrial mammals using eDNA analysis of soil samples against confirmed species observations from a long-term (∼9-yr) camera trapping study. At the same time, we considered multiple experimental parameters, including two sampling designs, two DNA extraction kits and two metabarcodes of different sizes.All mammals consistently recorded with cameras were detected in eDNA. In addition, eDNA reported many small mammals not recorded by camera traps, but whose presence in the study area is otherwise documented. A long metabarcode (≈220bp) offering a high taxonomic resolution, achieved a similar efficiency as a shorter one (≈70bp) and a phosphate buffer-based extraction gave similar results as a total DNA extraction method for a fraction of the price. Our results support that eDNA-based monitoring should become a valuable part of terrestrial mammal surveys. Yet, the lack of coverage of mammal mitochondrial genomes in public databases must be addressed before eDNA can be used to its full potential.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/dsuite_-_fast_d-statistics_and_related_admixture_evidence_from_vcf_files_60277124/">
          <h4 class="font-weight-bold">Dsuite - fast D-statistics and related admixture evidence from VCF files, bioRxiv, 2019-05-11</h4>
          <p>AbstractSummaryThe D-statistic, also known as the ABBA-BABA statistic, and related statistics are commonly used to assess evidence of gene flow between populations or closely related species. While the calculations are not computationally intensive, currently available implementations require custom file formats and are impractical to evaluate all gene flow hypotheses across datasets that include many populations or species. Dsuite is a fast C&#43;&#43; implementation, allowing genome scale calculations of the D-statistic across all combinations of tens or even hundreds of populations or species directly from a variant call format (VCF) file. Furthermore, the program can estimate the admixture fraction and provide evidence of whether introgression is confined to specific loci. Thus Dsuite facilitates assessment of gene flow across large genomic datasets.Availability and implementationSource code and documentation are available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsgithub.commillanekDsuite&gt;httpsgithub.commillanekDsuite&lt;jatsext-link&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/fine-scale_haplotype_structure_reveals_strong_signatures_of_positive_selection_in_a_recombining_bacterial_pathogen_60279815/">
          <h4 class="font-weight-bold">Fine-scale haplotype structure reveals strong signatures of positive selection in a recombining bacterial pathogen, bioRxiv, 2019-05-11</h4>
          <p>ABSTRACTIdentifying the forces that create and shape ecologically meaningful variation in bacteria remains an important challenge. For recombining bacteria, the sign and strength of linkage provide a unique lens into ongoing selection. We show derived alleles less than 300bp apart in Neisseria gonorrhoeae exhibit more coupling linkage than repulsion linkage, a pattern that cannot be explained by limited recombination or neutrality as these couplings are significantly stronger for nonsynonymous alleles compared to synonymous alleles. While linkage is shaped by many evolutionary processes, extensive simulations show only two distinct forms of positive selection can drive an excess of coupling linkage between neighboring nonsynonymous alleles directional selection on introgressed alleles or selection that maintains distinct haplotypes in the presence of recombination. Our results establish a framework for identifying patterns of selection in fine-scale haplotype structure that indicate specific ecological processes in species that recombine with distantly related lineages or possess coexisting adaptive haplotypes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/paragraph_a_graph-based_structural_variant_genotyper_for_short-read_sequence_data_60281518/">
          <h4 class="font-weight-bold">Paragraph A graph-based structural variant genotyper for short-read sequence data, bioRxiv, 2019-05-11</h4>
          <p>AbstractAccurate detection and genotyping of structural variations (SVs) from short-read data is a long-standing area of development in genomics research and clinical sequencing pipelines. We introduce Paragraph, a fast and accurate genotyper that models SVs using sequence graphs and SV annotations produced by a range of methods and technologies. We demonstrate the accuracy of Paragraph on whole genome sequence data from a control sample with both short and long read sequencing data available, and then apply it at scale to a cohort of 100 samples of diverse ancestry sequenced with short-reads. Comparative analyses indicate that Paragraph has better accuracy than other existing genotypers. The Paragraph software is open-source and available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsgithub.comIlluminaparagraph&gt;httpsgithub.comIlluminaparagraph&lt;jatsext-link&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/early_origin_and_deep_conservation_of_enhancers_in_animals_60206888/">
          <h4 class="font-weight-bold">Early origin and deep conservation of enhancers in animals, bioRxiv, 2019-05-10</h4>
          <p>AbstractTranscription factors (TFs) bind DNA enhancer sequences to regulate gene transcription in animals. Unlike TFs, the evolution of enhancers has been difficult to trace because of their rapid evolution. Here, we show enhancers from the sponge Amphimedon queenslandica can drive cell type-specific reporter gene expression in zebrafish and mouse, despite sponge and vertebrate lineages diverging over 700 million years ago. Although sponge enhancers, which are present in both highly conserved syntenic gene regions (Islet–Scaper, Ccne1–Uri and Tdrd3–Diaph3) and sponge-specific intergenic regions, have no significant sequence identity with vertebrate genomic sequences, the type and frequency of TF binding motifs in the sponge enhancer allow for the identification of homologous enhancers in bilaterians. Islet enhancers identified in human and mouse Scaper genes drive zebrafish reporter expression patterns that are almost identical to the sponge Islet enhancer. The existence of homologous enhancers in these disparate metazoans suggests animal development is controlled by TF-enhancer DNA interactions that were present in the first multicellular animals.One-sentence summaryEnhancer activity is conserved across 700 million years of trans-phyletic divergence.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/fishing_for_mammals_landscape-level_monitoring_of_terrestrial_and_semi-aquatic_communities_using_edna_from_lotic_ecosystems_60205935/">
          <h4 class="font-weight-bold">Fishing for mammals landscape-level monitoring of terrestrial and semi-aquatic communities using eDNA from lotic ecosystems, bioRxiv, 2019-05-10</h4>
          <p>Abstract&lt;jatslist list-type=order&gt;&lt;jatslist-item&gt;Environmental DNA (eDNA) metabarcoding has revolutionised biomonitoring in both marine and freshwater ecosystems. However, for semi-aquatic and terrestrial animals, the application of this technique remains relatively untested.&lt;jatslist-item&gt;&lt;jatslist-item&gt;We first assess the efficiency of eDNA metabarcoding in detecting semi-aquatic and terrestrial mammals in natural lotic ecosystems in the UK by comparing sequence data recovered from water and sediment samples to the mammalian communities expected from historical data. Secondly, we evaluate the detection efficiency of eDNA samples compared to multiple conventional non-invasive survey methods (latrine surveys and camera trapping) using occupancy modelling.&lt;jatslist-item&gt;&lt;jatslist-item&gt;eDNA metabarcoding detected a large proportion of the expected mammalian community within each area. Common species in the areas were detected at the majority of sites. Several key species of conservation concern in the UK were detected by eDNA in areas where authenticated records do not currently exist, but potential false positives were also identified for several non-native species.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Water-based eDNA samples provided comparable results to conventional survey methods in per unit of survey effort for three species (water vole, field vole, and red deer) using occupancy models. The comparison between survey ‘effort’ to reach a detection probability of ≥0.95 revealed that 3-6 water replicates would be equivalent to 3-5 latrine surveys and 5-30 weeks of single camera deployment, depending on the species.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Synthesis and Applications. eDNA metabarcoding represents an extremely promising tool for monitoring mammals, allowing for the detection of multiple species simultaneously, and provides comparable results to widely-used conventional survey methods. eDNA from freshwater systems delivers a ‘terrestrial dividend’ by detecting both semi-aquatic and terrestrial mammalian communities, and provides a basis for future monitoring at a landscape level over larger spatial and temporal scales (i.e. long-term monitoring at national levels).&lt;jatslist-item&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/2019/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/2019/page/21/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/18/" class="page-link">18</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/19/" class="page-link">19</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/20/" class="page-link">20</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/21/" class="page-link">21</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/2019/page/22/" class="page-link">22</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/23/" class="page-link">23</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/24/" class="page-link">24</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/25/" class="page-link">25</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/26/" class="page-link">26</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/2019/page/23/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/2019/page/87/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
