<!DOCTYPE html>
<html><head>
  
  <title>2019 | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/clinical_use_of_current_polygenic_risk_scores_may_exacerbate_health_disparities_58049280/">
          <h4 class="font-weight-bold">Clinical use of current polygenic risk scores may exacerbate health disparities, Nature Genetics, 2019-03-29</h4>
          <p>Polygenic risk scores (PRS) are poised to improve biomedical outcomes via precision medicine. However, the major ethical and scientific challenge surrounding clinical implementation of PRS is that those available today are several times more accurate in individuals of European ancestry than other ancestries. This disparity is an inescapable consequence of Eurocentric biases in genome-wide association studies, thus highlighting that—unlike clinical biomarkers and prescription drugs, which may individually work better in some populations but do not ubiquitously perform far better in European populations—clinical uses of PRS today would systematically afford greater improvement for European-descent populations. Early diversifying efforts show promise in leveling this vast imbalance, even when non-European sample sizes are considerably smaller than the largest studies to date. To realize the full and equitable potential of PRS, greater diversity must be prioritized in genetic studies, and summary statistics must be publically disseminated to ensure that health disparities are not increased for those individuals already most underserved.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/nature%20genetics">nature genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/polygenic_architecture_of_human_neuroanatomical_diversity_57963714/">
          <h4 class="font-weight-bold">Polygenic architecture of human neuroanatomical diversity, bioRxiv, 2019-03-29</h4>
          <p>AbstractWe analysed the genomic architecture of neuroanatomical diversity using magnetic resonance imaging and SNP data from &amp;gt; 20,000 individuals. Our results replicate previous findings of a strong polygenic architecture of neuroanatomical diversity. SNPs captured from 40% to 54% of the variance in the volume of different brain regions. We observed a large correlation between chromosome length and the amount of phenotypic variance captured, r∼0.64 on average, suggesting that at a global scale causal variants are homogeneously distributed across the genome. At a more local scale, SNPs within genes (∼51%) captured ∼1.5-times more genetic variance than the rest; and SNPs with low minor allele frequency (MAF) captured significantly less variance than those with higher MAF the 40% of SNPs with MAF&amp;lt;5% captured less than one fourth of the genetic variance. We also observed extensive pleiotropy across regions, with an average genetic correlation of rG∼0.45. Across regions, genetic correlations were in general similar to phenotypic correlations. By contrast, genetic correlations were larger than phenotypic correlations for the leftright volumes of the same region, and indistinguishable from 1. Additionally, the differences in leftright volumes were not heritable, underlining the role of environmental causes in the variability of brain asymmetry. Our analysis code is available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsgithub.comneuroanatomygenomic-architecture&gt;httpsgithub.comneuroanatomygenomic-architecture&lt;jatsext-link&gt;.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/research_grade_marijuana_supplied_by_the_national_institute_on_drug_abuse_is_genetically_divergent_from_commercially_available_cannabis_57987798/">
          <h4 class="font-weight-bold">Research grade marijuana supplied by the National Institute on Drug Abuse is genetically divergent from commercially available Cannabis, bioRxiv, 2019-03-29</h4>
          <p>AbstractPublic comfort with Cannabis (marijuana and hemp) has recently increased, resulting in previously strict Cannabis regulations now allowing hemp cultivation, medical use, and in some states, recreational consumption. There is a growing interest in the potential medical benefits of the various chemical constituents produced by the Cannabis plant. Currently, the University of Mississippi, funded through the National Institutes of HealthNational Institute on Drug Abuse (NIHNIDA), is the sole Drug Enforcement Agency (DEA) licensed facility to cultivate Cannabis for research purposes. Hence, most federally funded research where participants consume Cannabis for medicinal purposes relies on NIDA-supplied product. Previous research found that cannabinoid levels in research grade marijuana supplied by NIDA did not align with commercially available Cannabis from Colorado, Washington and California. Given NIDA chemotypes were misaligned with commercial Cannabis, we sought to investigate where NIDA’s research grade marijuana falls on the genetic spectrum of Cannabis groups. NIDA research grade marijuana was found to genetically group with Hemp samples along with a small subset of commercial drug-type Cannabis. A majority of commercially available drug-type Cannabis was genetically very distinct from NIDA samples. These results suggest that subjects consuming NIDA research grade marijuana may experience different effects than average consumers.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/inhibition_of_mtorc1_signaling_in_aged_rats_counteracts_the_decline_in_muscle_mass_and_reverses_multiple_parameters_of_muscle_signaling_associated_with_sarcopenia_57951427/">
          <h4 class="font-weight-bold">Inhibition of mTORC1 signaling in aged rats counteracts the decline in muscle mass and reverses multiple parameters of muscle signaling associated with sarcopenia, bioRxiv, 2019-03-28</h4>
          <p>AbstractThere is a lack of pharmacological interventions available for sarcopenia, a progressive age-associated loss of muscle mass, leading to a decline in mobility and quality of life. We found mTORC1 (mammalian target of rapamycin complex 1), a well-established critical positive modulator of mass, to be hyperactivated in sarcopenic muscle. Furthermore, inhibition of the mTORC1 pathway counteracted sarcopenia as determined by observing an increase in muscle mass and fiber type cross sectional area, surprising because mTORC1 signaling has been shown to be required for muscle mass gains in some settings. Additionally, several genes related to senescence were downregulated, while gene expression indicators of neuromuscular junction denervation were diminished using a low dose of a rapalog. Therefore mTORC1 inhibition may delay the progression of sarcopenia by directly and indirectly modulating multiple age-associated pathways, implicating mTORC1 as a therapeutic target to treat sarcopenia.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/physiology">physiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/nanodj_a_dockerized_jupyter_notebook_for_interactive_oxford_nanopore_minion_sequence_manipulation_and_genome_assembly_57935047/">
          <h4 class="font-weight-bold">NanoDJ A Dockerized Jupyter Notebook for Interactive Oxford Nanopore MinION Sequence Manipulation and Genome Assembly, bioRxiv, 2019-03-28</h4>
          <p>AbstractBackgroundThe Oxford Nanopore Technologies (ONT) MinION portable sequencer makes it possible to use cutting-edge genomic technologies in the field and the academic classroom.ResultsWe present NanoDJ, a Jupyter notebook integration of tools for simplified manipulation and assembly of DNA sequences produced by ONT devices. It integrates basecalling, read trimming and quality control, simulation and plotting routines with a variety of widely used aligners and assemblers, including procedures for hybrid assembly.ConclusionsWith the use of Jupyter-facilitated access to self-explanatory contents of applications and the interactive visualization of results, as well as by its distribution into a Docker software container, NanoDJ is aimed to simplify and make more reproducible ONT DNA sequence analysis. The NanoDJ package code, documentation and installation instructions are freely available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsgithub.comgenomicsITERNanoDJ&gt;httpsgithub.comgenomicsITERNanoDJ&lt;jatsext-link&gt;.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/orchestrating_single-cell_analysis_with_bioconductor_57887532/">
          <h4 class="font-weight-bold">Orchestrating Single-Cell Analysis with Bioconductor, bioRxiv, 2019-03-28</h4>
          <p>AbstractRecent developments in experimental technologies such as single-cell RNA sequencing have enabled the profiling a high-dimensional number of genome-wide features in individual cells, inspiring the formation of large-scale data generation projects quantifying unprecedented levels of biological variation at the single-cell level. The data generated in such projects exhibits unique characteristics, including increased sparsity and scale, in terms of both the number of features and the number of samples. Due to these unique characteristics, specialized statistical methods are required along with fast and efficient software implementations in order to successfully derive biological insights. Bioconductor - an open-source, open-development software project based on the R programming language - has pioneered the analysis of such high-throughput, high-dimensional biological data, leveraging a rich history of software and methods development that has spanned the era of sequencing. Featuring state-of-the-art computational methods, standardized data infrastructure, and interactive data visualization tools that are all easily accessible as software packages, Bioconductor has made it possible for a diverse audience to analyze data derived from cutting-edge single-cell assays. Here, we present an overview of single-cell RNA sequencing analysis for prospective users and contributors, highlighting the contributions towards this effort made by Bioconductor.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/2019/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/2019/page/40/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/37/" class="page-link">37</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/38/" class="page-link">38</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/39/" class="page-link">39</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/40/" class="page-link">40</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/2019/page/41/" class="page-link">41</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/42/" class="page-link">42</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/43/" class="page-link">43</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/44/" class="page-link">44</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/45/" class="page-link">45</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/2019/page/42/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/2019/page/87/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
