<!DOCTYPE html>
<html><head>
  
  <title>500&#43; Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/impact_of_genetically_engineered_maize_on_agronomic_environmental_and_toxicological_traits_a_meta-analysis_of_21_years_of_field_data_33241822/">
          <h4 class="font-weight-bold">Impact of genetically engineered maize on agronomic, environmental and toxicological traits a meta-analysis of 21 years of field data, Scientific Reports, 2018-02-09</h4>
          <p>Despite the extensive cultivation of genetically engineered (GE) maize and considerable number of scientific reports on its agro-environmental impact, the risks and benefits of GE maize are still being debated and concerns about safety remain. This meta-analysis aimed at increasing knowledge on agronomic, environmental and toxicological traits of GE maize by analyzing the peer-reviewed literature (from 1996 to 2016) on yield, grain quality, non-target organisms (NTOs), target organisms (TOs) and soil biomass decomposition. Results provided strong evidence that GE maize performed better than its near isogenic line grain yield was 5.6 to 24.5% higher with lower concentrations of mycotoxins (−28.8%), fumonisin (−30.6%) and thricotecens (−36.5%). The NTOs analyzed were not affected by GE maize, except for Braconidae, represented by a parasitoid of European corn borer, the target of Lepidoptera active Bt maize. Biogeochemical cycle parameters such as lignin content in stalks and leaves did not vary, whereas biomass decomposition was higher in GE maize. The results support the cultivation of GE maize, mainly due to enhanced grain quality and reduction of human exposure to mycotoxins. Furthermore, the reduction of the parasitoid of the target and the lack of consistent effects on other NTOs are confirmed.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/scientific%20reports">scientific reports</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/human_skeletal_muscle_possesses_an_epigenetic_memory_of_hypertrophy_32417785/">
          <h4 class="font-weight-bold">Human Skeletal Muscle Possesses an Epigenetic Memory of Hypertrophy, Scientific Reports, 2018-01-24</h4>
          <p>It is unknown if adult human skeletal muscle has an epigenetic memory of earlier encounters with growth. We report, for the first time in humans, genome-wide DNA methylation (850,000 CpGs) and gene expression analysis after muscle hypertrophy (loading), return of muscle mass to baseline (unloading), followed by later hypertrophy (reloading). We discovered increased frequency of hypomethylation across the genome after reloading (18,816 CpGs) versus earlier loading (9,153 CpG sites). We also identified AXIN1, GRIK2, CAMK4, TRAF1 as hypomethylated genes with enhanced expression after loading that maintained their hypomethylated status even during unloading where muscle mass returned to control levels, indicating a memory of these genes methylation signatures following earlier hypertrophy. Further, UBR5, RPL35a, HEG1, PLA2G16, SETD3 displayed hypomethylation and enhanced gene expression following loading, and demonstrated the largest increases in hypomethylation, gene expression and muscle mass after later reloading, indicating an epigenetic memory in these genes. Finally, genes; GRIK2, TRAF1, BICC1, STAG1 were epigenetically sensitive to acute exercise demonstrating hypomethylation after a single bout of resistance exercise that was maintained 22 weeks later with the largest increase in gene expression and muscle mass after reloading. Overall, we identify an important epigenetic role for a number of largely unstudied genes in muscle hypertrophymemory.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/scientific%20reports">scientific reports</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/deep_image_reconstruction_from_human_brain_activity_31006035/">
          <h4 class="font-weight-bold">Deep image reconstruction from human brain activity, bioRxiv, 2017-12-29</h4>
          <p>Machine learning-based analysis of human functional magnetic resonance imaging (fMRI) patterns has enabled the visualization of perceptual content. However, it has been limited to the reconstruction with low-level image bases or to the matching to exemplars. Recent work showed that visual cortical activity can be decoded (translated) into hierarchical features of a deep neural network (DNN) for the same input image, providing a way to make use of the information from hierarchical visual features. Here, we present a novel image reconstruction method, in which the pixel values of an image are optimized to make its DNN features similar to those decoded from human brain activity at multiple layers. We found that the generated images resembled the stimulus images (both natural images and artificial shapes) and the subjective visual content during imagery. While our model was solely trained with natural images, our method successfully generalized the reconstruction to artificial shapes, indicating that our model indeed reconstructs or generates images from brain activity, not simply matches to exemplars. A natural image prior introduced by another deep neural network effectively rendered semantically meaningful details to reconstructions by constraining reconstructed images to be similar to natural images. Furthermore, human judgment of reconstructions suggests the effectiveness of combining multiple DNN layers to enhance visual quality of generated images. The results suggest that hierarchical visual information in the brain can be effectively combined to reconstruct perceptual and subjective images.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/content-aware_image_restoration_pushing_the_limits_of_fluorescence_microscopy_30693461/">
          <h4 class="font-weight-bold">Content-Aware Image Restoration Pushing the Limits of Fluorescence Microscopy, bioRxiv, 2017-12-20</h4>
          <p>Fluorescence microscopy is a key driver of discoveries in the life-sciences, with observable phenomena being limited by the optics of the microscope, the chemistry of the fluorophores, and the maximum photon exposure tolerated by the sample. These limits necessitate trade-offs between imaging speed, spatial resolution, light exposure, and imaging depth. In this work we show how image restoration based on deep learning extends the range of biological phenomena observable by microscopy. On seven concrete examples we demonstrate how microscopy images can be restored even if 60-fold fewer photons are used during acquisition, how near isotropic resolution can be achieved with up to 10-fold under-sampling along the axial direction, and how tubular and granular structures smaller than the diffraction limit can be resolved at 20-times higher frame-rates compared to state-of-the-art methods. All developed image restoration methods are freely available as open source software in Python, FIJI, and KNIME.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/real-time_search_of_all_bacterial_and_viral_genomic_data_30495131/">
          <h4 class="font-weight-bold">Real-time search of all bacterial and viral genomic data, bioRxiv, 2017-12-16</h4>
          <p>Genome sequencing of pathogens is now ubiquitous in microbiology, and the sequence archives are effectively no longer searchable for arbitrary sequences. Furthermore, the exponential increase of these archives is likely to be further spurred by automated diagnostics. To unlock their use for scientific research and real-time surveillance we have combined knowledge about bacterial genetic variation with ideas used in web-search, to build a DNA search engine for microbial data that can grow incrementally. We indexed the complete global corpus of bacterial and viral whole genome sequence data (447,833 genomes), using four orders of magnitude less storage than previous methods. The method allows future scaling to millions of genomes. This renders the global archive accessible to sequence search, which we demonstrate with three applications ultra-fast search for resistance genes MCR1-3, analysis of host-range for 2827 plasmids, and quantification of the rise of antibiotic resistance prevalence in the sequence archives.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/opportunities_and_obstacles_for_deep_learning_in_biology_and_medicine_20671243/">
          <h4 class="font-weight-bold">Opportunities and obstacles for deep learning in biology and medicine, bioRxiv, 2017-05-29</h4>
          <p>AbstractDeep learning, which describes a class of machine learning algorithms, has recently showed impressive results across a variety of domains. Biology and medicine are data rich, but the data are complex and often ill-understood. Problems of this nature may be particularly well-suited to deep learning techniques. We examine applications of deep learning to a variety of biomedical problems—patient classification, fundamental biological processes, and treatment of patients—and discuss whether deep learning will transform these tasks or if the biomedical sphere poses unique challenges. We find that deep learning has yet to revolutionize or definitively resolve any of these problems, but promising advances have been made on the prior state of the art. Even when improvement over a previous baseline has been modest, we have seen signs that deep learning methods may speed or aid human investigation. More work is needed to address concerns related to interpretability and how to best model each problem. Furthermore, the limited amount of labeled data for training presents problems in some domains, as do legal and privacy constraints on work with sensitive health records. Nonetheless, we foresee deep learning powering changes at both bench and bedside with the potential to transform several areas of biology and medicine.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/500&#43;-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/500&#43;-users/page/7/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/500&#43;-users/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/500&#43;-users/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/500&#43;-users/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/500&#43;-users/page/7/" class="page-link">7</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/500&#43;-users/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/500&#43;-users/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/500&#43;-users/page/10/" class="page-link">10</a></li>
            
        

        
        <li class="page-item"><a href="/audiences/tags/500&#43;-users/page/9/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/500&#43;-users/page/10/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
