<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/single-cell_in_situ_transcriptomic_map_of_astrocyte_cortical_layer_diversity_49173691/">
          <h4 class="font-weight-bold">Single-cell in situ transcriptomic map of astrocyte cortical layer diversity, bioRxiv, 2018-10-03</h4>
          <p>AbstractDuring organogenesis, patterns and gradients of gene expression underlie organization and diversified cell specification to generate complex tissue architecture. While the cerebral cortex is organized into six excitatory neuronal layers, it is unclear whether glial cells are diversified to mimic neuronal laminae or show distinct layering. To determine the molecular architecture of the mammalian cortex, we developed a high content pipeline that can quantify single-cell gene expression in situ. The Large-area Spatial Transcriptomic (LaST) map confirmed expected cortical neuron layer organization and also revealed a novel neuronal identity signature. Screening 46 candidate genes for astrocyte diversity across the cortex, we identified grey matter superficial, mid and deep astrocyte identities in gradient layer patterns that were distinct from neurons. Astrocyte layers formed in early postnatal cortex and mostly persisted in adult mouse and human cortex. Mutations that shifted neuronal post-mitotic identity or organization were sufficient to alter glial layering, indicating an instructive role for neuronal cues. In normal mouse cortex, astrocyte layer patterns showed area diversity between functionally distinct cortical regions. These findings indicate that excitatory neurons and astrocytes cells are organized into distinct lineage-associated laminae, which give rise to higher order neuroglial complexity of cortical architecture.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/quiescent_cells_actively_replenish_cenp-a_nucleosomes_to_maintain_centromere_identity_and_proliferative_potential_49147569/">
          <h4 class="font-weight-bold">Quiescent cells actively replenish CENP-A nucleosomes to maintain centromere identity and proliferative potential, bioRxiv, 2018-10-02</h4>
          <p>SummaryCentromeres provide a robust model for epigenetic inheritance as they are specified by sequence-independent mechanisms involving the histone H3-variant CENP-A. Prevailing models indicate that the high intrinsic stability of CENP-A nucleosomes maintains centromere identity indefinitely. Here, we demonstrate that CENP-A is not stable at centromeres, but is instead gradually and continuously incorporated in quiescent cells including G0-arrested tissue culture cells and prophase I-arrested oocytes. Quiescent CENP-A incorporation involves the canonical CENP-A deposition machinery, but displays distinct requirements from cell cycle-dependent deposition. We demonstrate that Plk1 is required specifically for G1 CENP-A deposition, whereas transcription promotes CENP-A incorporation in quiescent oocytes. Preventing CENP-A deposition during quiescence results in significantly reduced CENP-A levels and perturbs chromosome segregation following the resumption of cell division. In contrast to quiescent cells, terminally differentiated cells fail to maintain CENP-A levels. Our work reveals that quiescent cells actively maintain centromere identity providing an indicator of proliferative potential.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/pathogen_detection_and_microbiome_analysis_of_infected_wheat_using_a_portable_dna_sequencer_49017775/">
          <h4 class="font-weight-bold">Pathogen Detection and Microbiome Analysis of Infected Wheat Using a Portable DNA Sequencer, bioRxiv, 2018-09-29</h4>
          <p>ABSTRACTFungal diseases of plants are responsible for major losses in agriculture, highlighting the need for rapid and accurate identification of plant pathogens. Disease outcomes are often defined not only by the main pathogen but are influenced by diverse microbial communities known as the microbiome at sites of infection. Here we present the first use of whole genome shot-gun sequencing with a portable DNA sequencing device as a method for the detection of fungal pathogens from wheat (Triticum aestivum) in a standard molecular biology laboratory. The data revealed that our method is robust and applicable to the diagnosis of fungal diseases including wheat stripe rust (caused by Puccinia striiformis f. sp. tritici), septoria tritici blotch (caused by Zymoseptoria tritici) and yellow leaf spot (caused by Pyrenophora tritici repentis). We also identified the bacterial genus Pseudomonas co-present with Puccinia and Zymoseptoria but not Pyrenophora infections. One limitation of the method is the over-representation of redundant wheat genome sequences from samples. This could be addressed by long-range amplicon-based sequencing approaches in future studies, which specifically target non-host organisms. Our work outlines a new approach for detection of a broad range of plant pathogens and associated microbes using a portable sequencer in a standard laboratory, providing the basis for future development of an on-site disease monitoring system.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/reconstructing_the_human_first_trimester_fetalmaternal_interface_using_single_cell_transcriptomics_49017765/">
          <h4 class="font-weight-bold">Reconstructing the human first trimester fetal–maternal interface using single cell transcriptomics, bioRxiv, 2018-09-29</h4>
          <p>SummaryDuring the early weeks of human pregnancy, the fetal placenta implants into the uterine mucosa (decidua) where placental trophoblast cells intermingle and communicate with maternal cells. Here, we profile transcriptomes of ∼50,000 single cells from this unique microenvironment, sampling matched first trimester maternal blood and decidua, and fetal cells from the placenta itself. We define the cellular composition of human decidua, revealing five distinct subsets of decidual fibroblasts with differing growth factors and hormone production profiles, and show that fibroblast states define two distinct decidual layers. Among decidual NK cells, we resolve three subsets, each with a different immunomodulatory and chemokine profile. We develop a repository of ligand-receptor pairs (&lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpwww.CellPhoneDB.org&gt;www.CellPhoneDB.org&lt;jatsext-link&gt;) and a statistical tool to predict the probability of cell-cell interactions via these pairs, highlighting specific interactions between decidual NK cells and invading fetal extravillous trophoblast cells, maternal immune and stromal cells. Our single cell atlas of the maternal-fetal interface reveals the cellular organization and interactions critical for placentation and reproductive success.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/a_high-resolution_map_of_non-crossover_events_reveals_impacts_of_genetic_diversity_on_mammalian_meiotic_recombination_48950595/">
          <h4 class="font-weight-bold">A high-resolution map of non-crossover events reveals impacts of genetic diversity on mammalian meiotic recombination, bioRxiv, 2018-09-27</h4>
          <p>During meiotic recombination in most mammals, hundreds of programmed DNA Double-Strand Breaks (DSBs) occur across all chromosomes in each cell at sites bound by the protein PRDM9. Faithful DSB repair using the homologous chromosome is essential for fertility, yielding either non-crossovers, which are frequent but difficult to detect, or crossovers. In certain hybrid mice, high sequence divergence causes PRDM9 to bind each homologue at different sites, &#39;asymmetrically&#39;, and these mice exhibit meiotic failure and infertility, by unknown mechanisms. To investigate the impact of local sequence divergence on recombination, we intercrossed two mouse subspecies over five generations and deep-sequenced 119 offspring, whose high heterozygosity allowed detection of thousands of crossover and non-crossover events with unprecedented power and spatial resolution. Both crossovers and non-crossovers are strongly depleted at individual asymmetric sites, revealing that PRDM9 not only positions DSBs but also promotes their homologous repair by binding to the unbroken homologue at each site. Unexpectedly, we found that non-crossovers containing multiple mismatches repair by a different mechanism than single-mismatch sites, which undergo GC-biased gene conversion. These results demonstrate that local genetic diversity profoundly alters meiotic repair pathway decisions via at least two distinct mechanisms, impacting genome evolution and Prdm9-related hybrid infertility.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/cohort_profile_east_london_genes__health_elgh_a_community_based_population_genomics_and_health_study_of_british-bangladeshi_and_british-pakistani_people._48932786/">
          <h4 class="font-weight-bold">Cohort Profile East London Genes &amp; Health (ELGH), a community based population genomics and health study of British-Bangladeshi and British-Pakistani people., bioRxiv, 2018-09-27</h4>
          <p>Cohort profile in a nutshell East London Genes &amp;amp; Health (ELGH) is a large scale, community genomics and health study (to date &amp;gt;34,000 volunteers; target 100,000 volunteers). ELGH was set up in 2015 to gain deeper understanding of health and disease, and underlying genetic influences, in British-Bangladeshi and British-Pakistani people living in east London. ELGH prioritises studies in areas important to, and identified by, the community it represents. Current priorities include cardiometabolic diseases and mental illness, these being of notably high prevalence and severity. However studies in any scientific area are possible, subject to community advisory group and ethical approval. ELGH combines health data science (using linked UK National Health Service (NHS) electronic health record data) with exome sequencing and SNP array genotyping to elucidate the genetic influence on health and disease, including the contribution from high rates of parental relatedness on rare genetic variation and homozygosity (autozygosity), in two understudied ethnic groups. Linkage to longitudinal health record data enables both retrospective and prospective analyses. Through Stage 2 studies, ELGH offers researchers the opportunity to undertake recall-by-genotype andor recall-by-phenotype studies on volunteers. Sub-cohort, trial-within-cohort, and other study designs are possible. ELGH is a fully collaborative, open access resource, open to academic and life sciences industry scientific research partners.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/0-100-users/page/42/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/39/" class="page-link">39</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/40/" class="page-link">40</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/41/" class="page-link">41</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/42/" class="page-link">42</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/0-100-users/page/43/" class="page-link">43</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/44/" class="page-link">44</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/45/" class="page-link">45</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/46/" class="page-link">46</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/47/" class="page-link">47</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/0-100-users/page/44/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/0-100-users/page/64/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
