<!DOCTYPE html>
<html><head>
  
  <title>200 500 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/integrated_computational_and_experimental_identification_of_p53_kras_and_vhl_mutant_selection_associated_with_crispr-cas9_editing_47626399/">
          <h4 class="font-weight-bold">Integrated computational and experimental identification of p53, KRAS and VHL mutant selection associated with CRISPR-Cas9 editing, bioRxiv, 2018-09-04</h4>
          <p>AbstractRecent studies have reported that CRISPR-Cas9 gene editing induces a p53-dependent DNA damage response in primary cells, which may select for cells with oncogenic p53 mutations11,12. It is unclear whether these CRISPR-induced changes are applicable to different cell types, and whether CRISPR gene editing may select for other oncogenic mutations. Addressing these questions, we analyzed genome-wide CRISPR and RNAi screens to systematically chart the mutation selection potential of CRISPR knockouts across the whole exome. Our analysis suggests that CRISPR gene editing can select for mutants of KRAS and VHL, at a level comparable to that reported for p53. These predictions were further validated in a genome-wide manner by analyzing independent CRISPR screens and patients’ tumor data. Finally, we performed a new set of pooled and arrayed CRISPR screens to evaluate the competition between CRISPR-edited isogenic p53 WT and mutant cell lines, which further validated our predictions. In summary, our study systematically charts and points to the potential selection of specific cancer driver mutations during CRISPR-Cas9 gene editing.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/cancer-biology">cancer-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_genetic_architecture_of_the_human_cerebral_cortex_47606812/">
          <h4 class="font-weight-bold">The genetic architecture of the human cerebral cortex, bioRxiv, 2018-09-03</h4>
          <p>The cerebral cortex underlies our complex cognitive capabilities, yet we know little about the specific genetic loci influencing human cortical structure. To identify genetic variants, including structural variants, impacting cortical structure, we conducted a genome-wide association meta-analysis of brain MRI data from 51,662 individuals. We analysed the surface area and average thickness of the whole cortex and 34 regions with known functional specialisations. We identified 255 nominally significant loci (P ≤ 5 × 10−8); 199 survived multiple testing correction (P ≤ 8.3 × 10−10; 187 surface area; 12 thickness). We found significant enrichment for loci influencing total surface area within regulatory elements active during prenatal cortical development, supporting the radial unit hypothesis. Loci impacting regional surface area cluster near genes in Wnt signalling pathways, known to influence progenitor expansion and areal identity. Variation in cortical structure is genetically correlated with cognitive function, Parkinson’s disease, insomnia, depression and ADHD.One Sentence SummaryCommon genetic variation is associated with inter-individual variation in the structure of the human cortex, both globally and within specific regions, and is shared with genetic risk factors for some neuropsychiatric disorders.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/re-evaluating_one-step_generation_of_mice_carrying_conditional_alleles_by_crispr-cas9-mediated_genome_editing_technology_47288281/">
          <h4 class="font-weight-bold">Re-Evaluating One-step Generation of Mice Carrying Conditional Alleles by CRISPR-Cas9-Mediated Genome Editing Technology, bioRxiv, 2018-08-31</h4>
          <p>AbstractCRISPR-Cas9 gene editing technology has considerably facilitated the generation of mouse knockout alleles, relieving many of the cumbersome and time-consuming steps of traditional mouse embryonic stem cell technology. However, the generation of conditional knockout alleles remains an important challenge. An earlier study reported up to 16% efficiency in generating conditional knockout alleles in mice using 2 single guide RNAs (sgRNA) and 2 single-stranded oligonucleotides (ssODN) (2sgRNA-2ssODN). We re-evaluated this method from a large data set generated from a consortium consisting of 17 transgenic core facilities or laboratories or programs across the world. The dataset constituted 17,887 microinjected or electroporated zygotes and 1,718 live born mice, of which only 15 (0.87%) mice harbored 2 correct LoxP insertions in cis configuration indicating a very low efficiency of the method. To determine the factors required to successfully generate conditional alleles using the 2sgRNA-2ssODN approach, we performed a generalized linear regression model. We show that factors such as the concentration of the sgRNA, Cas9 protein or the distance between the placement of LoxP insertions were not predictive for the success of this technique. The major predictor affecting the method’s success was the probability of simultaneously inserting intact proximal and distal LoxP sequences, without the loss of the DNA segment between the two sgRNA cleavage sites. Our analysis of a large data set indicates that the 2sgRNA–2ssODN method generates a large number of undesired alleles (&amp;gt;99%), and a very small number of desired alleles (&amp;lt;1%) requiring, on average 1,192 zygotes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/lineage_calling_can_identify_antibiotic_resistant_clones_within_minutes_47237654/">
          <h4 class="font-weight-bold">Lineage calling can identify antibiotic resistant clones within minutes, bioRxiv, 2018-08-29</h4>
          <p>Introductory ParagraphSurveillance of circulating drug resistant bacteria is essential for healthcare providers to deliver effective empiric antibiotic therapy. However, the results of surveillance may not be available on a timescale that is optimal for guiding patient treatment. Here we present a method for inferring characteristics of an unknown bacterial sample by identifying the presence of sequence variation across the genome that is linked to a phenotype of interest, in this case drug resistance. We demonstrate an implementation of this principle using sequence k-mer content, matched to a database of known genomes. We show this technique can be applied to data from an Oxford Nanopore device in real time and is capable of identifying the presence of a known resistant strain in 5 minutes, even from a complex metagenomic sample. This flexible approach has wide application to pathogen surveillance and may be used to greatly accelerate diagnoses of resistant infections.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/rapid_heuristic_inference_of_antibiotic_resistance_and_susceptibility_by_genomic_neighbor_typing_47237654/">
          <h4 class="font-weight-bold">Rapid heuristic inference of antibiotic resistance and susceptibility by genomic neighbor typing, bioRxiv, 2018-08-29</h4>
          <p>AbstractSurveillance of drug-resistant bacteria is essential for healthcare providers to deliver effective empiric antibiotic therapy. However, traditional molecular epidemiology does not typically occur on a timescale that could impact patient treatment and outcomes. Here we present a method called ‘genomic neighbor typing’ for inferring the phenotype of a bacterial sample by identifying its closest relatives in a database of genomes with metadata. We show that this technique can infer antibiotic susceptibility and resistance for both S. pneumoniae and N. gonorrhoeae. We implemented this with rapid k-mer matching, which, when used on Oxford Nanopore MinION data, can run in real time. This resulted in determination of resistance within ten minutes (sensspec 91%100% for S. pneumoniae and 81%100% N. gonorrhoeae from isolates with a representative database) of sequencing starting, and for clinical metagenomic sputum samples (75%100% for S. pneumoniae), within four hours of sample collection. This flexible approach has wide application to pathogen surveillance and may be used to greatly accelerate appropriate empirical antibiotic treatment.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/saber_enables_highly_multiplexed_and_amplified_detection_of_dna_and_rna_in_cells_and_tissues_47155675/">
          <h4 class="font-weight-bold">SABER enables highly multiplexed and amplified detection of DNA and RNA in cells and tissues, bioRxiv, 2018-08-28</h4>
          <p>SUMMARYFluorescent in situ hybridization (FISH) reveals the abun-dance and positioning of nucleic acid sequences in fixed sam-ples and can be combined with cell segmentation to produce a powerful single cell gene expression assay. However, it re-mains difficult to label more than a few targets and to visu-alize nucleic acids in environments such as thick tissue sam-ples using conventional FISH technologies. Recently, meth-ods have been developed for multiplexed amplification of FISH signals, yet it remains challenging to achieve high lev-els of simultaneous multiplexing combined with high sam-pling efficiency and simple workflows. Here, we introduce signal amplification by exchange reaction (SABER), which endows oligo-based FISH probes with long, single-stranded DNA concatemers that serve as targets for sensitive fluores-cent detection. We establish that SABER effectively ampli-fies the signal of probes targeting nucleic acids in fixed cells and tissues, can be deployed against at least 17 targets si-multaneously, and detects mRNAs with high efficiency. As a demonstration of the utility of SABER in assays involv-ing genetic manipulations, we apply multiplexed FISH of reporters and cell type markers to the identification of en-hancers with cell type-specific activity in the mouse retina. SABER represents a simple and versatile molecular toolkit to allow rapid and cost effective multiplexed imaging.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/200-500-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/200-500-users/page/33/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../../tags/200-500-users/page/30/" class="page-link">30</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/200-500-users/page/31/" class="page-link">31</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/200-500-users/page/32/" class="page-link">32</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/200-500-users/page/33/" class="page-link">33</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/200-500-users/page/34/" class="page-link">34</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/200-500-users/page/35/" class="page-link">35</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/200-500-users/page/36/" class="page-link">36</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/200-500-users/page/37/" class="page-link">37</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/200-500-users/page/38/" class="page-link">38</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/200-500-users/page/35/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/200-500-users/page/56/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
