<!DOCTYPE html>
<html><head>
  
  <title>2019 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/microbiota_supplementation_with_bifidobacterium_and_lactobacillus_modifies_the_preterm_infant_gut_microbiota_and_metabolome_63428708/">
          <h4 class="font-weight-bold">Microbiota supplementation with Bifidobacterium and Lactobacillus modifies the preterm infant gut microbiota and metabolome, bioRxiv, 2019-07-12</h4>
          <p>AbstractSupplementation with members of the early-life microbiota or ‘probiotics’ is becoming increasingly popular to attempt to beneficially manipulate the preterm gut microbiota. We performed a large longitudinal study comprising two preterm groups; 101 orally supplemented with Bifidobacterium and Lactobacillus (BifLacto) and 133 non-supplemented (Control) matched by age, sex, birth-mode, and diet. 16S rRNA metataxonomic profiling on stool samples (n = 592) indicated a predominance of Bifidobacterium, and a reduction of pathobionts in the BifLacto group. Metabolic phenotyping found a parallel increase in fecal acetate and lactate in the BifLacto group compared to the Control group, which positively correlated with Bifidobacterium abundance consistent with the ability of the supplemented Bifidobacterium strain to metabolize human milk oligosaccharides and reduced gut pH. This study demonstrates that microbiota supplementation can modify the preterm microbiome and the gastrointestinal environment to more closely resemble that of a full-term infant.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/migratory_divides_coincide_with_species_barriers_across_replicated_avian_hybrid_zones_above_the_tibetan_plateau_63425901/">
          <h4 class="font-weight-bold">Migratory divides coincide with species barriers across replicated avian hybrid zones above the Tibetan Plateau, bioRxiv, 2019-07-12</h4>
          <p>AbstractMigratory divides are proposed to be catalysts for speciation across a diversity of taxa. However, the relative contribution of migratory behavior to reproductive isolation is difficult to test. Comparing reproductive isolation in hybrid zones with and without migratory divides offers a rare opportunity to directly examine the contribution of divergent migratory behavior to reproductive barriers. We show that across replicate sampling transects of two pairs of barn swallow (Hirundo rustica) subspecies, strong reproductive isolation coincided with an apparent migratory divide spanning 20 degrees of latitude. A third subspecies pair exhibited no evidence for a migratory divide and hybridized extensively. Within migratory divides, migratory phenotype was associated with assortative mating, implicating a central contribution of divergent migratory behavior to reproductive barriers. The remarkable geographic coincidence between migratory divides and genetic breaks supports a longstanding hypothesis that the Tibetan Plateau is a substantial barrier contributing to the diversity of Siberian avifauna.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_place-cell_representation_of_volumetric_space_in_rats_63427846/">
          <h4 class="font-weight-bold">The place-cell representation of volumetric space in rats, bioRxiv, 2019-07-12</h4>
          <p>AbstractPlace cells are spatially modulated neurons found in the hippocampus that underlie spatial memory and navigation how these neurons represent 3D space is crucial for a full understanding of spatial cognition. We wirelessly recorded place cells in rats as they explored a cubic lattice climbing frame which could be aligned or tilted with respect to gravity. Place cells represented the entire volume of the mazes their activity tended to be aligned with the maze axes, and when it was more difficult for the animals to move vertically the cells represented space less accurately and less stably. These results demonstrate that even surface-dwelling animals represent 3D space and suggests there is a fundamental relationship between environment structure, gravity, movement and spatial memory.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/animal-behavior-and-cognition">animal-behavior-and-cognition</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/volcanofinder_genomic_scans_for_adaptive_introgression_63431068/">
          <h4 class="font-weight-bold">VolcanoFinder genomic scans for adaptive introgression, bioRxiv, 2019-07-12</h4>
          <p>AbstractRecent research shows that introgression between closely-related species is an important source of adaptive alleles for a wide range of taxa. Typically, detection of adaptive introgression from genomic data relies on comparative analyses that require sequence data from both the recipient and the donor species. However, in many cases, the donor is unknown or the data is not currently available. Here, we introduce a genome-scan method—VolcanoFinder—to detect recent events of adaptive introgression using polymorphism data from the recipient species only.VolcanoFinder detects adaptive introgression sweeps from the pattern of excess intermediate-frequency polymorphism they produce in the flanking region of the genome, a pattern which appears as a volcano-shape in pairwise genetic diversity.Using coalescent theory, we derive analytical predictions for these patterns. Based on these results, we develop a composite-likelihood test to detect signatures of adaptive introgression relative to the genomic background. Simulation results show that VolcanoFinder has high statistical power to detect these signatures, even for older sweeps and for soft sweeps initiated by multiple migrant haplotypes. Finally, we implement VolcanoFinder to detect archaic introgression in European and sub-Saharan African human populations, and uncovered interesting candidates in both populations, such as TSHR in Europeans and TCHH-RPTN in Africans. We discuss their biological implications and provide guidelines for identifying and circumventing artifactual signals during empirical applications of VolcanoFinder.Author summaryThe process by which beneficial alleles are introduced into a species from a closely-related species is termed adaptive introgression. We present an analytically-tractable model for the effects of adaptive introgression on non-adaptive genetic variation in the genomic region surrounding the beneficial allele. The result we describe is a characteristic volcano-shaped pattern of increased variability that arises around the positively-selected site, and we introduce an open-source method VolcanoFinder to detect this signal in genomic data. Importantly, VolcanoFinder is a population-genetic likelihood-based approach, rather than a comparative-genomic approach, and can therefore probe genomic variation data from a single population for footprints of adaptive introgression, even from a priori unknown and possibly extinct donor species.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/cross-kingdom_recognition_of_bacterial_small_rnas_induces_transgenerational_pathogenic_avoidance_63417291/">
          <h4 class="font-weight-bold">Cross-kingdom recognition of bacterial small RNAs induces transgenerational pathogenic avoidance, bioRxiv, 2019-07-11</h4>
          <p>AbstractWe recently discovered that C. elegans can pass on a learned avoidance of pathogenic Pseudomonas aeruginosa (PA14) to four generations of its progeny. This transgenerational inheritance is bacterial species-specific, but how C. elegans recognizes and distinguishes different bacteria and transmits this information to future generations is not apparent. Here we show that small RNAs purified from pathogenic PA14 are sufficient not only to induce avoidance of pathogens in mothers, but also to confer transgenerational inheritance of this species-specific behavior for four generations, all without direct contact with pathogenic bacteria. This behavior requires the small RNA transporters SID-1 and SID-2, RNA interference pathway components, the piRNA PiwiArgonaute pathway, a functioning germline, and TGF-β ligand daf-7 expression in the ASI sensory neuron. Our results suggest that C. elegans “reads” small RNAs expressed by pathogenic bacteria, and uses this information to induce an escape behavior that lasts for four additional generations. C. elegans may have evolved this trans-kingdom signaling system to avoid pathogens in abundant classes of bacteria in its environment and its microbiome.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genome-wide_association_study_provides_new_insights_into_the_genetic_architecture_and_pathogenesis_of_heart_failure_63387281/">
          <h4 class="font-weight-bold">Genome-wide association study provides new insights into the genetic architecture and pathogenesis of heart failure, bioRxiv, 2019-07-11</h4>
          <p>AbstractHeart failure (HF) is a leading cause of morbidity and mortality worldwide1. A small proportion of HF cases are attributable to monogenic cardiomyopathies and existing genome-wide association studies (GWAS) have yielded only limited insights, leaving the observed heritability of HF largely unexplained2–4. We report the largest GWAS meta-analysis of HF to-date, comprising 47,309 cases and 930,014 controls. We identify 12 independent variant associations with HF at 11 genomic loci, all of which demonstrate one or more associations with coronary artery disease (CAD), atrial fibrillation, or reduced left ventricular function suggesting shared genetic aetiology. Expression quantitative trait analysis of non-CAD-associated loci implicate genes involved in cardiac development (MYOZ1, SYNPO2L), protein homeostasis (BAG3), and cellular senescence (CDKN1A). Using Mendelian randomisation analysis we provide new evidence supporting previously equivocal causal roles for several HF risk factors identified in observational studies, and demonstrate CAD-independent effects for atrial fibrillation, body mass index, hypertension and triglycerides. These findings extend our knowledge of the genes and pathways underlying HF and may inform the development of new therapeutic approaches.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/2019/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/2019/page/33/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/30/" class="page-link">30</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/31/" class="page-link">31</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/32/" class="page-link">32</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/33/" class="page-link">33</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/2019/page/34/" class="page-link">34</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/35/" class="page-link">35</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/36/" class="page-link">36</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/37/" class="page-link">37</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/38/" class="page-link">38</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/2019/page/35/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/2019/page/135/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
