<!DOCTYPE html>
<html><head>
  
  <title>Biophysics | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/scale-free_vertical_tracking_microscopy_towards_bridging_scales_in_biological_oceanography_59085850/">
          <h4 class="font-weight-bold">Scale-free Vertical Tracking Microscopy Towards Bridging Scales in Biological Oceanography, bioRxiv, 2019-04-16</h4>
          <p>AbstractUnderstanding key biophysical phenomena in the ocean often requires one to simultaneously focus on microscale entities, such as motile plankton and sedimenting particles, while maintaining the macroscale context of vertical transport in a highly stratified environment. This poses a conundrum How to measure single organisms, at microscale resolution, in the lab, while allowing them to freely move hundreds of meters in the vertical direction? We present a solution in the form of a scale-free, vertical tracking microscope based on a circular “hydrodynamic-treadmill”. Our technology allows us to transcend physiological and ecological scales, tracking organisms from marine zooplankton to single-cells over vertical scales of meters while resolving microflows and behavioral processes. We demonstrate measurements of sinking particles, including marine snow as they sediment tens of meters while capturing sub-particle-scale phenomena. We also demonstrate depth-patterned virtual-reality environments for novel behavioral analyses of microscale plankton. This technique offers a new experimental paradigm in microscale ocean biophysics by combining physiological-scale imaging with free movement in an ecological-scale patterned environment.One sentence summaryScale-free vertical tracking microscopy captures, for the first time, untethered behavioral dynamics at cellular resolution for marine plankton.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/tunability_of_dna_polymerase_stability_during_eukaryotic_dna_replication_58721233/">
          <h4 class="font-weight-bold">Tunability of DNA polymerase stability during eukaryotic DNA replication, bioRxiv, 2019-04-08</h4>
          <p>SummaryStructural and biochemical studies have revealed the basic principles of how the replisome duplicates genomic DNA, but little is known about its dynamics during DNA replication. We reconstitute the 34 proteins needed to form the S. cerevisiae replisome and show how changing local concentrations of the key DNA polymerases tunes the ability of the complex to efficiently recycle these proteins or to dynamically exchange them. Particularly, we demonstrate redundancy of the Pol α DNA polymerase activity in replication and show that Pol α primase and the lagging-strand Pol δ can be re-used within the replisome to support the synthesis of large numbers of Okazaki fragments. This unexpected malleability of the replisome might allow it to deal with barriers and resource challenges during replication of large genomes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/nuclear_pores_as_versatile_reference_standards_for_quantitative_superresolution_microscopy_57349188/">
          <h4 class="font-weight-bold">Nuclear pores as versatile reference standards for quantitative superresolution microscopy, bioRxiv, 2019-03-20</h4>
          <p>AbstractQuantitative fluorescence and superresolution microscopy are often limited by insufficient data quality or artifacts. In this context, it is essential to have biologically relevant control samples to benchmark and optimize the quality of microscopes, labels and imaging conditions.Here we exploit the stereotypic arrangement of proteins in the nuclear pore complex as in situ reference structures to characterize the performance of a variety of microscopy modalities. We created four genome edited cell lines in which we endogenously labeled the nucleoporin Nup96 with mEGFP, SNAP-tag or HaloTag or the photoconvertible fluorescent protein mMaple. We demonstrate their use a) as 3D resolution standards for calibration and quality control, b) to quantify absolute labeling efficiencies and c) as precise reference standards for molecular counting.These cell lines will enable the broad community to assess the quality of their microscopes and labels, and to perform quantitative, absolute measurements.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/microfluidic_protein_isolation_and_sample_preparation_for_high_resolution_cryo-em_55847155/">
          <h4 class="font-weight-bold">Microfluidic protein isolation and sample preparation for high resolution cryo-EM, bioRxiv, 2019-02-21</h4>
          <p>High-resolution structural information is essential to understand protein function. Protein-structure determination needs a considerable amount of protein, which can be challenging to produce, often involving harsh and lengthy procedures. In contrast, the several thousands to a few million protein particles required for structure-determination by cryogenic electron microscopy (cryo-EM) can be provided by miniaturized systems. Here, we present a microfluidic method for the rapid isolation of a target protein and its direct preparation for cryo-EM. Less than one microliter of cell lysate is required as starting material to solve the atomic structure of the untagged, endogenous human 20S proteasome. Our work paves the way for high-throughput structure determination of proteins from minimal amounts of cell lysate and opens new opportunities for the isolation of sensitive, endogenous protein complexes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/phase_separation_provides_a_mechanism_to_reduce_noise_in_cells_supplementary_text_54293656/">
          <h4 class="font-weight-bold">Phase separation provides a mechanism to reduce noise in cells Supplementary text, bioRxiv, 2019-01-23</h4>
          <p>A central problem in cellular control is how cells cope with the inherent noise in gene expression. Although transcriptional and posttranscriptional feedback mechanisms can suppress noise, they are often slow, and cannot explain how cells buffer acute fluctuations. Here, by using a physical model that links fluctuations in protein concentration to the theory of phase separation, we show that liquid droplets can act as fast and effective buffers for gene expression noise. We confirm our theory experimentally using an engineered phase separating protein that forms liquid-like compartments in mammalian cells. These data suggest a novel role of phase separation in biological information processing.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/direct_synthesis_of_em-visible_gold_nanoparticles_on_genetically_encoded_tags_for_single-molecule_visualization_in_cells_54062313/">
          <h4 class="font-weight-bold">Direct synthesis of EM-visible gold nanoparticles on genetically encoded tags for single-molecule visualization in cells, bioRxiv, 2019-01-18</h4>
          <p>Single-molecule visualization in cells with genetically encoded tags for electron microscopy (EM) has been a long-awaited but unimplemented tool for cell biologists. Here, we report an approach for directly synthesizing EM-visible gold nanoparticles (AuNPs) on cysteine-rich tags for single-molecule visualization in cells. We first uncovered an auto-nucleation suppression mechanism that allows specific synthesis of AuNPs on isolated cysteine-rich tags. We next exploited this mechanism to develop an approach for single-molecule detection of proteins in prokaryotic cells and achieved an unprecedented labeling efficiency. We then expanded it to more complicated eukaryotic cells and successfully detected the proteins targeted to various organelles, including the membranes of endoplasmic reticulum (ER) and nuclear envelope, ER lumen, nuclear pores, spindle pole bodies, and mitochondrial matrix. Thus, our implementation of genetically encoded tags for EM should allow cell biologists to address an enormous range of biological questions at single-molecule level in diverse cellular ultrastructural contexts without using antibodies.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/biophysics/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/biophysics/page/2/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="/audiences/tags/biophysics/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biophysics/page/2/" class="page-link">2</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/biophysics/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biophysics/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biophysics/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biophysics/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biophysics/page/7/" class="page-link">7</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        

        
        <li class="page-item"><a href="/audiences/tags/biophysics/page/4/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/biophysics/page/8/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
