<!DOCTYPE html>
<html><head>
  
  <title>plant-biology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.68.3" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.95d2483cd3c5d2bf47c1f36307a456787d60ad958c5efe44db9afc47012b1b05b8e1f64f6ac291650439322a6c38e86eb7cd6254255516d857c23f299fb0e666.js" integrity="sha512-ldJIPNPF0r9HwfNjB6RWeH1grZWMXv5E25r8RwErGwW44fZPasKRZQQ5MipsOOhut81iVCVVFthXwj8pn7DmZg=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/an_inducible_genome_editing_system_for_plants_67017400/">
          <h4 class="font-weight-bold">An inducible genome editing system for plants, bioRxiv, 2019-09-23</h4>
          <p>ABSTRACTConditional manipulation of gene expression is a key approach to investigating the primary function of a gene in a biological process. While conditional and cell-type specific overexpression systems exist for plants, there are currently no systems available to disable a gene completely and conditionally. Here, we present a novel tool with which target genes can be efficiently conditionally knocked out at any developmental stage. The target gene is manipulated using the CRISPR-Cas9 genome editing technology, and conditionality is achieved with the well-established estrogen-inducible XVE system. Target genes can also be knocked-out in a cell-type specific manner. Our tool is easy to construct and will be particularly useful for studying genes which have null-alleles that are non-viable or show strong developmental defects.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/imaging_plant_germline_differentiation_within_arabidopsis_flower_by_light_sheet_microscopy_66855348/">
          <h4 class="font-weight-bold">Imaging plant germline differentiation within Arabidopsis flower by light sheet microscopy, bioRxiv, 2019-09-20</h4>
          <p>AbstractIn higher plants, germline differentiation occurs during a relatively short period within developing flowers. Understanding of the mechanisms that govern germline differentiation lags behind other plant developmental processes. This is largely because the germline is restricted to relatively few cells buried deep within floral tissues, which makes them difficult to study. To overcome this limitation, we have developed a methodology for live imaging of the germ cell lineage within floral organs of Arabidopsis using light sheet fluorescence microscopy. We have established reporter lines, cultivation conditions, and imaging protocols for high-resolution microscopy of developing flowers continuously for up to several days. We used multiview imagining to reconstruct a three-dimensional model of a flower at subcellular resolution. We demonstrate the power of this approach by capturing male and female meiosis, asymmetric pollen division, movement of meiotic chromosomes, and unusual restitution mitosis in tapetum cells. This method will enable new avenues of research into plant sexual reproduction.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/charting_the_native_architecture_of_thylakoid_membranes_with_single-molecule_precision_65988221/">
          <h4 class="font-weight-bold">Charting the native architecture of thylakoid membranes with single-molecule precision, bioRxiv, 2019-09-05</h4>
          <p>Thylakoid membranes scaffold an assortment of large protein complexes that work together to harness the energy of light to produce oxygen, NADPH, and ATP. It has been a longstanding challenge to visualize how the intricate thylakoid network organizes these protein complexes to finely tune the photosynthetic reactions. Using cryo-electron tomography to analyze membrane surface topology, we have mapped the native molecular landscape of thylakoid membranes within green algae cells. Our tomograms provide insights into the molecular forces that drive thylakoid stacking and reveal that photosystems I and II are strictly segregated at the borders between appressed and non-appressed membrane domains. This new approach to charting thylakoid topology lays the foundation for dissecting photosynthetic regulation at the level of single protein complexes within the cell.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/structural_basis_for_recognition_of_ralf_peptides_by_lrx_proteins_during_pollen_tube_growth_63285452/">
          <h4 class="font-weight-bold">Structural basis for recognition of RALF peptides by LRX proteins during pollen tube growth, bioRxiv, 2019-07-08</h4>
          <p>AbstractPlant reproduction relies on the highly regulated growth of the pollen tube for proper sperm delivery. This process is controlled by secreted RALF signaling peptides, which have been previously shown to be perceived by CrRLK1Ls membrane receptor-kinases and leucine-rich (LRR) extensin proteins (LRXs). Here we demonstrate that RALF peptides are active as folded, disulfide bond-stabilized proteins, which can bind to the LRR domain of LRX proteins with nanomolar affinity. Crystal structures of the LRX-RALF signaling complexes reveal LRX proteins as constitutive dimers. The N-terminal LRR domain containing the RALF binding site is tightly linked to the extensin domain via a cysteine-rich tail. Our biochemical and structural work reveals a complex signaling network by which RALF ligands may instruct different signaling proteins – here CrRLK1Ls and LRXs – through structurally different binding modes to orchestrate cell wall remodeling in rapidly growing pollen tubes.SignificancePlant reproduction relies on proper pollen tube growth to reach the female tissue and release the sperm cells. This process is highly regulated by a family of secreted signaling peptides that are recognized by cell-wall monitoring proteins to enable plant fertilization. Here, we report the crystal structure of the LRX-RALF cell-wall complex and we demonstrate that RALF peptides are active as folded proteins. RALFs are autocrine signaling proteins able to instruct LRX cell-wall modules and CrRKL1L receptors, through structurally different binding modes to coordinate pollen tube integrity.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_receptor_for_herbivore-associated_molecular_patterns_mediates_plant_immunity_62572977/">
          <h4 class="font-weight-bold">A receptor for herbivore-associated molecular patterns mediates plant immunity, bioRxiv, 2019-06-23</h4>
          <p>AbstractPlant-herbivore interactions are ubiquitous across nature and drive major agricultural losses. Inducible defense responses triggered through immune recognition aid in host plant protection; however, specific ligand-receptor pairs mediating the initial perception of herbivory remain unknown. Plants in the subtribe Phaseolinae detect herbivore-associated peptides in caterpillar oral secretions and the defined ligands are proteolytic fragments of chloroplastic ATP synthase termed inceptins. Using forward genetic mapping of inceptin-induced responses, we identify a cowpea (Vigna unguiculata) leucine-rich repeat receptor-like protein as an inceptin receptor (INR) sufficient for elicitor-induced responses and enhanced defense against armyworms (Spodoptera exigua). INR defines a receptor by which plants perceive herbivore-associated molecular patterns (HAMPs) and expands the paradigm of surface immune recognition to attack with mandibles.One Sentence SummaryA plant cell surface receptor directly perceives peptides associated with caterpillar herbivory.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/compensatory_sequence_variation_between_trans-species_small_rnas_and_their_target_sites_62412912/">
          <h4 class="font-weight-bold">Compensatory sequence variation between trans-species small RNAs and their target sites, bioRxiv, 2019-06-19</h4>
          <p>AbstractTrans-species small regulatory RNAs (sRNAs) are delivered to host plants from diverse pathogens and parasites and can target host mRNAs. How trans-species sRNAs can be effective on diverse hosts has been unclear. Multiple species of the parasitic plant Cuscuta produce trans-species sRNAs that collectively target many host mRNAs. Confirmed target sites are nearly always in highly conserved, protein-coding regions of host mRNAs. Cuscuta trans-species sRNAs can be grouped into superfamilies that have variation in a three-nucleotide period. These variants compensate for synonymous-site variation in host mRNAs. By targeting host mRNAs at highly conserved protein-coding sites, and simultaneously expressing multiple variants to cover synonymous-site variation, Cuscuta trans-species sRNAs may be able to successfully target homologous mRNAs from diverse hosts.One Sentence SummaryThe parasitic plant Cuscuta produces a diverse set of sRNAs that compensate for sequence variation in mRNA targets in diverse hosts.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/" class="page-link">1</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/6/" class="page-link">6</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/3/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/8/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
