<!DOCTYPE html>
<html><head>
  
  <title>Systems Biology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/metabolic_activity_affects_response_of_single_cells_to_a_nutrient_switch_in_structured_populations_57196911/">
          <h4 class="font-weight-bold">Metabolic activity affects response of single cells to a nutrient switch in structured populations, bioRxiv, 2019-03-18</h4>
          <p>AbstractMicrobes live in ever-changing environments where they need to adapt their metabolism to different nutrient conditions. Many studies have characterized the response of genetically identical cells to nutrient switches in homogenous cultures, however in nature microbes often live in spatially structured groups such as biofilms where cells can create metabolic gradients by consuming and releasing nutrients. Consequently, cells experience different local microenvironments and vary in their phenotype. How does this phenotypic variation affect the ability of cells to cope with nutrient switches? Here we address this question by growing dense populations of Escherichia coli in microfluidic chambers and studying a switch from glucose to acetate at the single cell level. Before the switch, cells vary in their metabolic activity some grow on glucose while others cross-feed on acetate. After the switch, only few cells can resume growth after a period of lag. The probability to resume growth depends on a cells’ phenotype prior to the switch it is highest for cells crossfeeding on acetate, while it depends in a non-monotonic way on growth rate for cells growing on glucose. Our results suggest that the strong phenotypic variation in spatially structured populations might enhance their ability to cope with fluctuating environments.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/ion_beam_subcellular_tomography_55924043/">
          <h4 class="font-weight-bold">Ion beam subcellular tomography, bioRxiv, 2019-02-23</h4>
          <p>Multiplexed ion beam imaging (MIBI) has been previously used to profile multiple parameters in two dimensions in single cells within tissue slices. Here, a mathematical and technical framework for three-dimensional subcellular MIBI is presented. We term the approach ion beam tomography (IBT) wherein ion beam images are acquired iteratively across successive, multiple scans and later compiled into a 3D format. For IBT, cells were imaged at 0.2-4 pA ion current across 1,000 axial scans. Consecutive subsets of ion beam images were binned over 3 to 20 slices (above and below) to create a resolved image, wherein binning was incremented one slice at a time to yield an enhanced multi-depth data without loss of depth resolution. Algorithmic deconvolution, tailored for ion beams, was then applied to the transformed ion image series using a hybrid deblurring algorithm and an ion beam current-dependent point-spread function. Three-dimensional processing was implemented by segmentation, mesh, molecular neighborhoods, and association maps. In cultured cancer cells and tissues, IBT enabled accessible visualization of three-dimensional volumetric distributions of genomic regions, RNA transcripts, and protein factors with 65-nm lateral and 5-nm axial resolution. IBT also enabled label-free elemental mapping of cells, allowing point of source cellular component measurements not possible for most optical microscopy targets. Detailed multiparameter imaging of subcellular features at near macromolecular resolution should now be made possible by the IBT tools and reagents provided here to open novel venues for interrogating subcellular biology.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/subcellular_localization_of_drug_distribution_by_super-resolution_ion_beam_imaging_55904025/">
          <h4 class="font-weight-bold">Subcellular localization of drug distribution by super-resolution ion beam imaging, bioRxiv, 2019-02-22</h4>
          <p>Technologies that visualize multiple biomolecules at the nanometer scale in cells will enable deeper understanding of biological processes that proceed at the molecular scale. Current fluorescence-based methods for microscopy are constrained by a combination of spatial resolution limitations, limited parameters per experiment, and detector systems for the wide variety of biomolecules found in cells. We present here super-resolution ion beam imaging (srIBI), a secondary ion mass spectrometry approach capable of high-parameter imaging in 3D of targeted biological entities and exogenously added small molecules. Uniquely, the atomic constituents of the biomolecules themselves can often be used in our system as the tag. We visualized the subcellular localization of the chemotherapy drug cisplatin simultaneously with localization of five other nuclear structures, with further carbon elemental mapping and secondary electron visualization, down to ~30 nm lateral resolution. Cisplatin was preferentially enriched in nuclear speckles and excluded from closed-chromatin regions, indicative of a role for cisplatin in active regions of chromatin. These data highlight how multiplexed super-resolution techniques, such as srIBI, will enable studies of biomolecule distributions in biologically relevant subcellular microenvironments.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/full-length_mrna_sequencing_reveals_principles_of_polya_tail_length_control_55318571/">
          <h4 class="font-weight-bold">Full-length mRNA sequencing reveals principles of poly(A) tail length control, bioRxiv, 2019-02-12</h4>
          <p>Although mRNAs are key molecules for understanding life, there exists no method to determine the full-length sequence of endogenous mRNAs including their poly(A) tails. Moreover, although poly(A) tails can be modified in functionally important ways, there also exists no method to accurately sequence them. Here, we present FLAM-seq, a rapid and simple method for high-quality sequencing of entire mRNAs. We report a cDNA library preparation method coupled to single-molecule sequencing to perform FLAM-seq. Using human cell lines, brain organoids, and C. elegans we show that FLAM-seq delivers high-quality full-length mRNA sequences for thousands of different genes per sample. We find that (a) 3&#39; UTR length is correlated with poly(A) tail length, (b) alternative polyadenylation sites and alternative promoters for the same gene are linked to different tail lengths, (c) tails contain a significant number of cytosines. Thus, we provide a widely useful method and fundamental insights into poly(A) tail regulation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/short-range_interactions_govern_cellular_dynamics_in_microbial_multi-genotype_systems_54544166/">
          <h4 class="font-weight-bold">Short-range interactions govern cellular dynamics in microbial multi-genotype systems, bioRxiv, 2019-01-27</h4>
          <p>Ecosystem processes result from interaction between organisms. When interactions are local, the spatial organization of organisms defines their network of interactions, and thus influences the system&#39;s functioning. This can be especially relevant for microbial systems, which often consist of spatially structured communities of cells connected by a dense interaction network. Here we measured the spatial interaction network between cells in microbial systems and identify the factors that determine it. Combining quantitative single-cell analysis of synthetic bacterial communities with mathematical modeling, we find that cells only interact with other cells in their immediate neighbourhood. This short interaction range impacts the functioning of the whole system by reducing its ability to perform metabolic processes collectively. Our experiments and models demonstrate that the spatial scale of cell-to-cell interaction plays a fundamental role in understanding and controlling natural communities, and in engineering microbial systems for specific purposes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/statistical_physics_of_liquid_brains_51780511/">
          <h4 class="font-weight-bold">Statistical physics of liquid brains, bioRxiv, 2018-11-27</h4>
          <p>Liquid neural networks (or “liquid brains”) are a widespread class of cognitive living networks characterised by a common feature the agents (ants or immune cells, for example) move in space. Thus, no fixed, long-term agent-agent connections are maintained, in contrast with standard neural systems. How is this class of systems capable of displaying cognitive abilities, from learning to decision-making? In this paper, the collective dynamics, memory and learning properties of liquid brains is explored under the perspective of statistical physics. Using a comparative approach, we review the generic properties of three large classes of systems, namely standard neural networks (“solid brains”), ant colonies and the immune system. It is shown that, despite their intrinsic physical differences, these systems share key properties with standard neural systems in terms of formal descriptions, but strongly depart in other ways. On one hand, the attractors found in liquid brains are not always based on connection weights but instead on population abundances. However, some liquid systems use fluctuations in ways similar to those found in cortical networks, suggesting a relevant role of criticality as a way of rapidly reacting to external signals.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/systems-biology/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/systems-biology/page/2/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/systems-biology/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/systems-biology/page/2/" class="page-link">2</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/systems-biology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/systems-biology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/systems-biology/page/5/" class="page-link">5</a></li>
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/systems-biology/page/4/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/systems-biology/page/5/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
