<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/functional_integration_of_undead_neurons_in_the_olfactory_system_59827060/">
          <h4 class="font-weight-bold">Functional integration of “undead” neurons in the olfactory system, bioRxiv, 2019-05-02</h4>
          <p>AbstractProgrammed cell death (PCD) is widespread during neurodevelopment, typically countering the surpluses of neuronal production. We examined, in the Drosophila olfactory system, the potential of cells fated to die to contribute to circuit evolution. Inhibition of PCD is sufficient to generate many new cells that express neural markers and exhibit odor-evoked activity. These “undead” neurons express a subset of olfactory receptors, enriched for recent receptor duplicates, and including those normally found in other chemosensory organs and life-stages. Moreover, undead neuron axons integrate into the olfactory circuitry in the brain. Comparison of homologous olfactory lineages across drosophilids reveals natural examples of fate changes from death to a functional neuron. These results reveal the remarkable potential of alterations in patterns of PCD to evolve novel neural pathways.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/tmem119-egfp_and_tmem119-creert2_transgenic_mice_for_labeling_and_manipulating_microglia_59829485/">
          <h4 class="font-weight-bold">Tmem119-EGFP and Tmem119-CreERT2 transgenic mice for labeling and manipulating microglia, bioRxiv, 2019-05-02</h4>
          <p>AbstractMicroglia are specialized brain-resident macrophages with important functions in health and disease. To improve our understanding of these cells, the research community needs genetic tools to identify and control them in a manner that distinguishes them from closely related cell-types. We have targeted the recently discovered microglia-specific Tmem119 gene to generate knock-in mice expressing EGFP (JAX#031823) or CreERT2 (JAX#031820) for the identification and manipulation of microglia, respectively. Genetic characterization of the locus and qPCR-based analysis demonstrate correct positioning of the transgenes and intact expression of endogenous Tmem119 in the knock-in mouse models. Immunofluorescence analysis further shows that parenchymal microglia, but not other brain macrophages, are completely and faithfully labeled in the EGFP-line at different time points of development. Flow cytometry indicates highly selective expression of EGFP in CD11b&#43;CD45lo microglia. Similarly, immunofluorescence and flow cytometry analyses using a Cre-dependent reporter mouse line demonstrate activity of CreERT2 primarily in microglia upon tamoxifen administration with the caveat of activity in leptomeningeal cells. Finally, flow cytometric analyses reveal absence of EGFP expression and minimal activity of CreERT2 in blood monocytes of the Tmem119-EGFP and Tmem119-CreERT2 lines, respectively. These new transgenic lines extend the microglia toolbox by providing the currently most specific genetic labeling and control over these cells in the myeloid compartment of mice.Visual abstract&lt;jatsfig id=ufig1 position=float orientation=portrait fig-type=figure&gt;&lt;jatsgraphic xmlnsxlink=httpwww.w3.org1999xlink xlinkhref=624825v2_ufig1 position=float orientation=portrait &gt;Significance statementTools that specifically label and manipulate only microglia are currently unavailable, but are critically needed to further our understanding of this cell type. Complementing and significantly extending recently introduced microglia-specific immunostaining methods that have quickly become a new standard in the field, we generated two mouse lines that label and control gene expression in microglia with high specificity and made them publicly available. Using these readily accessible mice, the research community will be able to study microglia biology with improved specificity.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/are_place_cells_just_memory_cells_memory_compression_leads_to_spatial_tuning_and_history_dependence_59743746/">
          <h4 class="font-weight-bold">Are place cells just memory cells? Memory compression leads to spatial tuning and history dependence, bioRxiv, 2019-05-01</h4>
          <p>AbstractThe observation of place cells has suggested that the hippocampus plays a special role in encoding spatial information. However, place cell responses are modulated by several non-spatial variables, and reported to be rather unstable. Here we propose a memory model of the hippocampus that provides a novel interpretation of place cells consistent with these observations. We hypothesize that the hippocampus is a memory device that takes advantage of the correlations between sensory experiences to generate compressed representations of the episodes that are stored in memory. A simple neural network model that can efficiently compress information naturally produces place cells that are similar to those observed in experiments. It predicts that the activity of these cells is variable and that the fluctuations of the place fields encode information about the recent history of sensory experiences. Place cells may simply be a consequence of a memory compression process implemented in the hippocampus.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/long-term_exposure_to_elevated_lipoproteina_levels_parental_lifespan_and_risk_of_mortality_59696840/">
          <h4 class="font-weight-bold">Long-Term Exposure to Elevated Lipoprotein(a) Levels, Parental Lifespan and Risk of Mortality, bioRxiv, 2019-04-30</h4>
          <p>ABSTRACTBackgroundElevated Lipoprotein(a) (Lp[a]) levels are associated with a broad range of atherosclerotic cardiovascular diseases (CVD). The impact of high Lp(a) levels on human longevity is however controversial. Our objectives were to determine whether genetically-determined Lp(a) levels are associated with parental lifespan and to assess the association between measured and genetically-determined Lp(a) levels and long-term all-cause and cardiovascular mortality.MethodsWe determined the association between a genetic risk score of 26 single nucleotide polymorphisms weighted for their impact on Lp(a) levels (wGRS) and parental lifespan (at least one long-lived parent; father still alive and older than 90 or father’s age of death ≥90 or mother still alive and older than 93 or mother’s age of death ≥93) in 139,362 participants from the UK Biobank. A total of 17,686 participants were considered as having high parental lifespan. We also investigated the association between Lp(a) levels and all-cause and cardiovascular mortality in 18,720 participants from the EPIC-Norfolk study.ResultsIn the UK Biobank, increases in the wGRS (weighted for a 50 mgdL increase in Lp(a) levels) were inversely associated with a high parental lifespan (odds ratio=0.92, 95% confidence interval [CI]=0.89-0.94, p=2.7×10−8). During the 20-year follow-up of the EPIC-Norfolk study, 5686 participants died (2412 from CVD-related causes). Compared to participants with Lp(a) levels &amp;lt;50 mgdL, those with Lp(a) levels ≥50 mgdL had an increased hazard ratio (HR) for all-cause (HR=1.17, 95% CI=1.08-1.27) and cardiovascular (HR=1.54, 95% CI=1.37-1.72) mortality. Compared to individuals with Lp(a) levels below the 50th percentile of the Lp(a) distribution (in whom event rates were 29.8% and 11.3%, respectively for all-cause and cardiovascular mortality), those with Lp(a) levels equal or above the 95th percentile of the population distribution (≥70 mgdL) had HRs of 1.22 (95% CI=1.09-1.37, event rate 37.5%) and 1.71 (95% CI=1.46-2.00, event rate 20.0%), for all-cause mortality and cardiovascular mortality, respectively.ConclusionsResults of this study suggest a potentially causal effect of Lp(a) on human longevity, support the use of parental lifespan as a tool to study the genetic determinants of human longevity, and provide a rationale for a trial of Lp(a)-lowering therapy in individuals with high Lp(a) levels.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/structural_and_functional_characterization_of_g_protein-coupled_receptors_with_deep_mutational_scanning_59714625/">
          <h4 class="font-weight-bold">Structural and Functional Characterization of G Protein-Coupled Receptors with Deep Mutational Scanning, bioRxiv, 2019-04-30</h4>
          <p>AbstractIn humans, the 813 G protein-coupled receptors (GPCRs) are responsible for transducing diverse chemical stimuli to alter cell state, and are the largest class of drug targets. Their myriad structural conformations and various modes of signaling make it challenging to understand their structure and function. Here we developed a platform to characterize large libraries of GPCR variants in human cell lines with a barcoded transcriptional reporter of G-protein signal transduction. We tested 7,800 of 7,828 possible single amino acid substitutions to the beta-2 adrenergic receptor (β2AR) at four concentrations of the agonist isoproterenol. We identified residues specifically important for β2AR signaling, mutations in the human population that are potentially loss of function, and residues that modulate basal activity. Using unsupervised learning, we resolve residues critical for signaling, including all major structural motifs and molecular interfaces. We also find a previously uncharacterized structural latch spanning the first two extracellular loops that is highly conserved across Class A GPCRs and is conformationally rigid in both the inactive and active states of the receptor. More broadly, by linking deep mutational scanning with engineered transcriptional reporters, we establish a generalizable method for exploring pharmacogenomics, structure and function across broad classes of drug receptors.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/hierarchical_recurrent_state_space_models_reveal_discrete_and_continuous_dynamics_of_neural_activity_in_c._elegans_59680614/">
          <h4 class="font-weight-bold">Hierarchical recurrent state space models reveal discrete and continuous dynamics of neural activity in C. elegans, bioRxiv, 2019-04-29</h4>
          <p>AbstractModern recording techniques enable large-scale measurements of neural activity in a variety of model organisms. The dynamics of neural activity shed light on how organisms process sensory information and generate motor behavior. Here, we study these dynamics using optical recordings of neural activity in the nematode C. elegans. To understand these data, we develop state space models that decompose neural time-series into segments with simple, linear dynamics. We incorporate these models into a hierarchical framework that combines partial recordings from many worms to learn shared structure, while still allowing for individual variability. This framework reveals latent states of population neural activity, along with the discrete behavioral states that govern dynamics in this state space. We find stochastic transition patterns between discrete states and see that transition probabilities are determined by both current brain activity and sensory cues. Our methods automatically recover transition times that closely match manual labels of different behaviors, such as forward crawling, reversals, and turns. Finally, the resulting model can simulate neural data, faithfully capturing salient patterns of whole brain dynamics seen in real data.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/0-100-users/page/29/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/26/" class="page-link">26</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/27/" class="page-link">27</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/28/" class="page-link">28</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/29/" class="page-link">29</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/0-100-users/page/30/" class="page-link">30</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/31/" class="page-link">31</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/32/" class="page-link">32</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/33/" class="page-link">33</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/34/" class="page-link">34</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/0-100-users/page/31/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/0-100-users/page/123/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
