<!DOCTYPE html>
<html><head>
  
  <title>Reports | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/specialized_and_spatially_organized_coding_of_sensory_motor_and_cognitive_variables_in_midbrain_dopamine_neurons_50453798/">
          <h4 class="font-weight-bold">Specialized and spatially organized coding of sensory, motor, and cognitive variables in midbrain dopamine neurons, bioRxiv, 2018-10-29</h4>
          <p>There is increased appreciation that dopamine (DA) neurons in the midbrain respond not only to reward 1,2 and reward-predicting cues 1,3,4, but also to other variables such as distance to reward 5, movements 6–11 and behavioral choices 12–15. Based on these findings, a major open question is how the responses to these diverse variables are organized across the population of DA neurons. In other words, do individual DA neurons multiplex multiple variables, or are subsets of neurons specialized in encoding specific behavioral variables? The reason that this fundamental question has been difficult to resolve is that recordings from large populations of individual DA neurons have not been performed in a behavioral task with sufficient complexity to examine these diverse variables simultaneously. To address this gap, we used 2-photon calcium imaging through an implanted lens to record activity of &amp;gt;300 midbrain DA neurons in the VTA during a complex decision-making task. As mice navigated in a virtual reality (VR) environment, DA neurons encoded an array of sensory, motor, and cognitive variables. These responses were functionally clustered, such that subpopulations of neurons transmitted information about a subset of behavioral variables, in addition to encoding reward. These functional clusters were spatially organized, such that neighboring neurons were more likely to be part of the same cluster. Taken together with the topography between DA neurons and their projections, this specialization and anatomical organization may aid downstream circuits in correctly interpreting the wide range of signals transmitted by DA neurons.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_architecture_of_cell_differentiation_in_choanoflagellates_and_sponge_choanocytes_50443900/">
          <h4 class="font-weight-bold">The architecture of cell differentiation in choanoflagellates and sponge choanocytes, bioRxiv, 2018-10-29</h4>
          <p>SUMMARYCollar cells are ancient animal cell types which are conserved across the animal kingdom [1] and their closest relatives, the choanoflagellates [2]. However, little is known about their ancestry, their subcellular architecture, or how they differentiate. The choanoflagellate Salpingoeca rosetta [3] expresses genes necessary for animal multicellularity and development [4] and can alternate between unicellular and multicellular states [3,5], making it a powerful model to investigate the origin of animal multicellularity and mechanisms underlying cell differentiation [6,7]. To compare the subcellular architecture of solitary collar cells in S. rosetta with that of multicellular “rosettes” and collar cells in sponges, we reconstructed entire cells in 3D through transmission electron microscopy on serial ultrathin sections. Structural analysis of our 3D reconstructions revealed important differences between single and colonial choanoflagellate cells, with colonial cells exhibiting a more amoeboid morphology consistent with relatively high levels of macropinocytotic activity. Comparison of multiple reconstructed rosette colonies highlighted the variable nature of cell sizes, cell-cell contact networks and colony arrangement. Importantly, we uncovered the presence of elongated cells in some rosette colonies that likely represent a distinct and differentiated cell type. Intercellular bridges within choanoflagellate colonies displayed a variety of morphologies and connected some, but not all, neighbouring cells. Reconstruction of sponge choanocytes revealed both ultrastructural commonalities and differences in comparison to choanoflagellates. Choanocytes and colonial choanoflagellates are typified by high amoeboid cell activity. In both, the number of microvilli and volumetric proportion of the Golgi apparatus are comparable, whereas choanocytes devote less of their cell volume to the nucleus and mitochondria than choanoflagellates and more of their volume to food vacuoles. Together, our comparative reconstructions uncover the architecture of cell differentiation in choanoflagellates and sponge choanocytes and constitute an important step in reconstructing the cell biology of the last common ancestor of the animal kingdom.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/atlas_of_subcellular_rna_localization_revealed_by_apex-seq_50345186/">
          <h4 class="font-weight-bold">Atlas of Subcellular RNA Localization Revealed by APEX-seq, bioRxiv, 2018-10-27</h4>
          <p>SUMMARYWe introduce APEX-seq, a method for RNA sequencing based on spatial proximity to the peroxidase enzyme APEX2. APEX-seq in nine distinct subcellular locales produced a nanometer-resolution spatial map of the human transcriptome, revealing extensive and exquisite patterns of localization for diverse RNA classes and transcript isoforms. We uncover a radial organization of the nuclear transcriptome, which is gated at the inner surface of the nuclear pore for cytoplasmic export of processed transcripts. We identify two distinct pathways of messenger RNA localization to mitochondria, each associated with specific sets of transcripts for building complementary macromolecular machines within the organelle. APEX-seq should be widely applicable to many systems, enabling comprehensive investigations of the spatial transcriptome.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/connect-seq_to_superimpose_molecular_on_anatomical_neural_circuit_maps_50380997/">
          <h4 class="font-weight-bold">Connect-seq to superimpose molecular on anatomical neural circuit maps, bioRxiv, 2018-10-27</h4>
          <p>AbstractThe mouse brain contains ~100 million neurons interconnected in a vast array of neural circuits. The identities and functions of individual neuronal components of most circuits are undefined. Here we describe a method, termed ‘Connect-seq’, which combines retrograde viral tracing and single cell transcriptomics to uncover the molecular identities of upstream neurons in a specific circuit and the signaling molecules they use to communicate. Connect-seq can generate a molecular map that can be superimposed on a neuroanatomical map to permit molecular and genetic interrogation of how the neuronal components of a circuit control its function. Application of this method to hypothalamic neurons controlling physiological responses to fear and stress reveal subsets of upstream neurons that express diverse constellations of signaling molecules and can be distinguished by their anatomical locations.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/bacterial_contribution_to_genesis_of_the_novel_germ_line_determinant_oskar_50270763/">
          <h4 class="font-weight-bold">Bacterial contribution to genesis of the novel germ line determinant oskar, bioRxiv, 2018-10-26</h4>
          <p>New cellular functions and developmental processes can evolve by modifying the functions or regulation of preexisting genes, but the creation of new genes and their contributions to novel processes is less well understood. New genes can arise not only from mutations or rearrangements of existing sequences, but also via acquisition of foreign DNA, also called horizontal gene transfer (HGT). Here we present evidence that HGT contributed to the creation of a novel gene indispensable for reproduction in some insects. The oskar gene evolved to fulfil a crucial role in insect germ cell formation, but was long considered a novel gene with unknown evolutionary origins. Our analysis of over 100 Oskar sequences suggests that Oskar arose through a novel gene formation history involving fusion of eukaryotic and prokaryotic sequences. One of its two conserved domains (LOTUS), was likely present in the genome of a last common insect ancestor, while the second (OSK) domain appears to have been acquired through horizontal transfer of a bacterial GDSL-like lipase domain. Our evidence suggests that the bacterial contributor of the OSK domain may have been a germ line endosymbiont. This shows that gene origin processes often considered highly unusual, including HGT and de novo coding region evolution, can give rise to novel genes that can both participate in pre-existing gene regulatory networks, and also facilitate the evolution of novel developmental mechanisms.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/challenges_and_recommendations_to_improve_installability_and_archival_stability_of_omics_computational_tools_50271779/">
          <h4 class="font-weight-bold">Challenges and recommendations to improve installability and archival stability of omics computational tools, bioRxiv, 2018-10-26</h4>
          <p>AbstractDeveloping new software tools for analysis of large-scale biological data is a key component of advancing modern biomedical research. Scientific reproduction of published findings requires running computational tools on data generated by such studies, yet little attention is presently allocated to the installability and archival stability of computational software tools. Scientific journals require data and code sharing, but none currently require authors to guarantee the continuing functionality of newly published tools. We have estimated the archival stability of computational biology software tools by performing an empirical analysis of the internet presence for 36,702 omics software resources published from 2005 to 2017. We found that almost 28% of all resources are currently not accessible through URLs published in the paper they first appeared in. Among the 98 software tools selected for our installability test, 51% were deemed “easy to install,” and 28% of the tools failed to be installed at all due to problems in the implementation. Moreover, for papers introducing new software, we found that the number of citations significantly increased when authors provided an easy installation process. We propose for incorporation into journal policy several practical solutions for increasing the widespread installability and archival stability of published bioinformatics software.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/reports/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/reports/page/168/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/165/" class="page-link">165</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/166/" class="page-link">166</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/167/" class="page-link">167</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/168/" class="page-link">168</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/reports/page/169/" class="page-link">169</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/170/" class="page-link">170</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/171/" class="page-link">171</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/172/" class="page-link">172</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/173/" class="page-link">173</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/reports/page/170/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/reports/page/320/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
