<!DOCTYPE html>
<html><head>
  
  <title>Genetics | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_robust_and_efficient_method_for_mendelian_randomization_with_hundreds_of_genetic_variants_unravelling_mechanisms_linking_hdl-cholesterol_and_coronary_heart_disease_56478079/">
          <h4 class="font-weight-bold">A robust and efficient method for Mendelian randomization with hundreds of genetic variants unravelling mechanisms linking HDL-cholesterol and coronary heart disease, bioRxiv, 2019-03-05</h4>
          <p>Mendelian randomization (MR) investigations with large numbers of genetic variants are becoming increasingly common. However, the reliability of findings from a MR investigation is dependent on the validity of the genetic variants as instrumental variables. We developed a method to identify groups of genetic variants with similar causal effect estimates, which may represent distinct mechanisms by which the risk factor influences the outcome. Our contamination mixture method is a robust and efficient method for valid MR in the presence of invalid IVs. Compared to other robust methods, our method had the lowest mean squared error across a range of realistic scenarios. The method is fast and efficient, and can perform analysis with hundreds of variants in a fraction of a second. In a MR analysis for high-density lipoprotein (HDL) cholesterol and coronary heart disease (CHD) risk, the method identified 11 variants associated with increased HDL-cholesterol, decreased triglyceride levels, and decreased CHD risk that had the same directions of associations with platelet distribution width and other blood cell traits, suggesting a shared mechanism linking lipids and CHD risk relating to platelet aggregation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/functional_dissection_of_tads_reveals_non-essential_and_instructive_roles_in_regulating_gene_expression_56502379/">
          <h4 class="font-weight-bold">Functional dissection of TADs reveals non-essential and instructive roles in regulating gene expression, bioRxiv, 2019-03-05</h4>
          <p>AbstractThe genome is organized in megabase-sized three-dimensional units, called Topologically Associated Domains (TADs), that are separated by boundaries. TADs bring distant cis-regulatory elements into proximity, a process dependent on the cooperative action of cohesin and the DNA binding factor CTCF. Surprisingly, genome-wide depletion of CTCF has little effect on transcription, yet structural variations affecting TADs have been shown to cause gene misexpression and congenital disease. Here, we investigate TAD function in vivo in mice by systematically editing components of TAD organization at the Sox9Kcnj locus. We find that TADs are formed by a redundant system of CTCF sites requiring the removal of all major sites within the TAD and at the boundary for two neighboring TADs to fuse. TAD fusion resulted in leakage of regulatory activity from the Sox9 to the Kcnj TAD, but no major changes in gene expression. This indicates that TAD structures provide robustness and precision, but are not essential for developmental gene regulation. Gene misexpression and resulting disease phenotypes, however, were attained by re-directing regulatory activity through inversions andor the re-positioning of boundaries. Thus, efficient re-wiring of enhancer promoter interaction and aberrant disease causing gene activation is not induced by a mere loss of insulation but requires the re-direction of contacts.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genetic_inhibition_of_pcsk9_atherogenic_lipoprotein_concentrations_and_calcific_aortic_valve_stenosis_56256162/">
          <h4 class="font-weight-bold">Genetic inhibition of PCSK9, atherogenic lipoprotein concentrations, and calcific aortic valve stenosis, bioRxiv, 2019-03-02</h4>
          <p>Background Proprotein convertase subtilisinkexin type 9 (PCSK9) inhibition reduces plasma concentrations of low-density lipoprotein cholesterol (LDL-C), apolipoprotein B (apoB) and lipoprotein(a) [Lp(a)]. Atherogenic lipoprotein levels have been linked with calcific aortic valve stenosis (CAVS). Our objectives were to determine the association between variants in PCSK9 and lipoprotein-lipid levels, coronary artery disease (CAD) and CAVS, and to evaluate if PCSK9 could be implicated in aortic valve interstitial cells (VICs) calcification.Methods We built a genetic risk score weight for LDL-C levels (wGRS) using 10 independent PCSK9 single nucleotide polymorphisms and determined its association with lipoprotein-lipid levels in 9692 participants of the EPIC-Norfolk study. We investigated the association between the wGRS and CAD and CAVS in the UK Biobank, as well as the association between the PCSK9 R46L variant and CAVS in a meta-analysis of published prospective, population-based studies (Copenhagen studies, 1463 cases101,620 controls) and unpublished studies (UK Biobank, 1350 cases349,043 controls, Malmo Diet and Cancer study, 682 cases5963 controls and EPIC-Norfolk study, 508 cases20,421 controls). We evaluated PCSK9 expression and localization in explanted aortic valves by capillary Western blot and immunohistochemistry in patients with and without CAVS. Von Kossa staining was used to visualize aortic leaflet calcium deposits. PCSK9 expression under oxidative stress conditions in VICs was assessed.Results The wGRS was significantly associated with lower LDL-C and apoB (p&amp;lt;0.001), but not with Lp(a). In the UK Biobank, the association of PCSK9 variants with CAD were positively correlated with their effects on apoB levels. CAVS was less prevalent in carriers of the PCSK9 R46L variant [odds ratio=0.71 (95% confidence interval, 0.57-0.88), p&amp;lt;0.001]. PCSK9 expression was elevated in the aortic valves of patients with aortic sclerosis and CAVS compared to controls. In calcified leaflets, PCSK9 co-localized with calcium deposits. PCSK9 expression was induced by oxidative stress in VICs. Conclusion Genetic inhibition of PCSK9 is associated with lifelong reductions in the levels of non-Lp(a) apoB-containing lipoproteins as well as lower odds of CAD and CAVS. PCSK9 is abundant in fibrotic and calcified aortic leaflets. Oxidative stress increases PCSK9 expression in VICs. These results provide a rationale for performing randomized clinical trials of PCSK9 inhibition in CAVS.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_transcriptome-wide_mendelian_randomization_study_to_uncover_tissue-dependent_regulatory_mechanisms_across_the_human_phenome_56204012/">
          <h4 class="font-weight-bold">A transcriptome-wide Mendelian randomization study to uncover tissue-dependent regulatory mechanisms across the human phenome, bioRxiv, 2019-03-01</h4>
          <p>Background Developing insight into tissue-specific transcriptional mechanisms can help improve our understanding of how genetic variants exert their effects on complex traits and disease. By applying the principles of Mendelian randomization, we have undertaken a systematic analysis to evaluate transcriptome-wide associations between gene expression across 48 different tissue types and 395 complex traits. Results Overall, we identified 100,025 gene-trait associations based on conventional genome-wide corrections (P &amp;lt; 5 x 10-08) that also provided evidence of genetic colocalization. These results indicated that genetic variants which influence gene expression levels in multiple tissues are more likely to influence multiple complex traits. We identified many examples of tissue-specific effects, such as genetically-predicted TPO, NR3C2 and SPATA13 expression only associating with thyroid disease in thyroid tissue. Additionally, FBN2 expression was associated with both cardiovascular and lung function traits, but only when analysed in heart and lung tissue respectively. We also demonstrate that conducting phenome-wide evaluations of our results can help flag adverse on-target side effects for therapeutic intervention, as well as propose drug repositioning opportunities. Moreover, we find that exploring the tissue-dependency of associations identified by genome-wide association studies (GWAS) can help elucidate the causal genes and tissues responsible for effects, as well as uncover putative novel associations. Conclusions The atlas of tissue-dependent associations we have constructed should prove extremely valuable to future studies investigating the genetic determinants of complex disease. The follow-up analyses we have performed in this study are merely a guide for future research. Conducting similar evaluations can be undertaken systematically at httpmrcieu.mrsoftware.orgTissue_MR_atlas.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genetic_meta-analysis_of_diagnosed_alzheimers_disease_identifies_new_risk_loci_and_implicates_a%CE%B2_tau_immunity_and_lipid_processing_56189969/">
          <h4 class="font-weight-bold">Genetic meta-analysis of diagnosed Alzheimer’s disease identifies new risk loci and implicates Aβ, tau, immunity and lipid processing, Nature Genetics, 2019-02-28</h4>
          <p>IQCK</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/nature%20genetics">nature genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/biological_and_clinical_insights_from_genetics_of_insomnia_symptoms_56029109/">
          <h4 class="font-weight-bold">Biological and clinical insights from genetics of insomnia symptoms, Nature Genetics, 2019-02-25</h4>
          <p>Insomnia is a common disorder linked with adverse long-term medical and psychiatric outcomes. The underlying pathophysiological processes and causal relationships of insomnia with disease are poorly understood. Here we identified 57 loci for self-reported insomnia symptoms in the UK Biobank (n = 453,379) and confirmed their effects on self-reported insomnia symptoms in the HUNT Study (n = 14,923 cases and 47,610 controls), physician-diagnosed insomnia in the Partners Biobank (n = 2,217 cases and 14,240 controls), and accelerometer-derived measures of sleep efficiency and sleep duration in the UK Biobank (n = 83,726). Our results suggest enrichment of genes involved in ubiquitin-mediated proteolysis and of genes expressed in multiple brain regions, skeletal muscle, and adrenal glands. Evidence of shared genetic factors was found between frequent insomnia symptoms and restless legs syndrome, aging, and cardiometabolic, behavioral, psychiatric, and reproductive traits. Evidence was found for a possible causal link between insomnia symptoms and coronary artery disease, depressive symptoms, and subjective well-being.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/nature%20genetics">nature genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/genetics/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/genetics/page/12/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/genetics/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/genetics/page/10/" class="page-link">10</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/genetics/page/11/" class="page-link">11</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/genetics/page/12/" class="page-link">12</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/genetics/page/13/" class="page-link">13</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/genetics/page/14/" class="page-link">14</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/genetics/page/15/" class="page-link">15</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/genetics/page/16/" class="page-link">16</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/genetics/page/17/" class="page-link">17</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/genetics/page/14/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/genetics/page/36/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
