<!DOCTYPE html>
<html><head>
  
  <title>Genetics | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/using_dna_from_mothers_and_children_to_study_parental_investment_in_childrens_educational_attainment_52436750/">
          <h4 class="font-weight-bold">Using DNA from mothers and children to study parental investment in children&#39;s educational attainment, bioRxiv, 2018-12-10</h4>
          <p>This study tested implications of new genetic discoveries for understanding the association between parental investment and children&#39;s educational attainment. A novel design matched genetic data from 860 British mothers and their children with home-visit measures of parenting the E-Risk Study. Three findings emerged. First, both mothers&#39; and children&#39;s education-associated genetics, summarized in a genome-wide polygenic score, predicted parenting -- a gene-environment correlation. Second, accounting for genetic influences slightly reduced associations between parenting and children&#39;s attainment -- indicating some genetic confounding. Third, mothers&#39; genetics influenced children&#39;s attainment over and above genetic mother-to-child transmission, via cognitively-stimulating parenting -- an environmentally-mediated effect. Findings imply that, when interpreting parents&#39; effects on children, environmentalists must consider genetic transmission, but geneticists must also consider environmental transmission.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/models_of_archaic_admixture_and_recent_history_from_two-locus_statistics_52355933/">
          <h4 class="font-weight-bold">Models of archaic admixture and recent history from two-locus statistics, bioRxiv, 2018-12-08</h4>
          <p>AbstractWe learn about population history and underlying evolutionary biology through patterns of genetic polymorphism. Many approaches to reconstruct evolutionary histories focus on a limited number of informative statistics describing distributions of allele frequencies or patterns of linkage disequilibrium. We show that many commonly used statistics are part of a broad family of two-locus moments whose expectation can be computed jointly and rapidly under a wide range of scenarios, including complex multi-population demographies with continuous migration and admixture events. A full inspection of these statistics reveals that widely used models of human history fail to predict simple patterns of linkage disequilibrium. To jointly capture the information contained in classical and novel statistics, we implemented a tractable likelihood-based inference framework for demographic history. Using this approach, we show that human evolutionary models that include archaic admixture in Africa, Asia, and Europe provide a much better description of patterns of genetic diversity across the human genome. We estimate that an unidentified, deeply diverged population admixed with modern humans within Africa both before and after the split of African and Eurasian populations, contributing 4 - 8% genetic ancestry to individuals in world-wide populations.Author SummaryThroughout human history, populations have expanded and contracted, split and merged, and ex-changed migrants. Because these events affected genetic diversity, we can learn about human history by comparing predictions from evolutionary models to genetic data. Here, we show how to rapidly compute such predictions for a wide range of diversity measures within and across populations under complex demographic scenarios. While widely used models of human history accurately predict common measures of diversity, we show that they strongly underestimate the co-occurence of low frequency mutations within human populations in Asia, Europe, and Africa. Models allowing for archaic admixture, the relatively recent mixing of human populations with deeply diverged human lineages, resolve this discrepancy. We use such models to infer demographic models that include both recent and ancient features of human history. We recover the well-characterized admixture of Neanderthals in Eurasian populations, as well as admixture from an as-yet unknown diverged human population within Africa, further suggesting that admixture with deeply diverged lineages occurred multiple times in human history. By simultaneously testing model predictions for a broad range of diversity statistics, we can assess the robustness of common evolutionary models, identify missing historical events, and build more informed models of human demography.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/geographic_variation_and_bias_in_polygenic_scores_of_complex_diseases_and_traits_in_finland_52182179/">
          <h4 class="font-weight-bold">Geographic variation and bias in polygenic scores of complex diseases and traits in Finland, bioRxiv, 2018-12-04</h4>
          <p>AbstractPolygenic scores (PS) are becoming a useful tool to identify individuals with high genetic risk for complex diseases and several projects are currently testing their utility for translational applications. It is also tempting to use PS to assess whether genetic variation can explain a part of the geographic distribution of a phenotype. However, it is not well known how population genetic properties of the training and target samples affect the geographic distribution of PS. Here, we evaluate geographic differences, and related biases, of PS in Finland with geographically well-defined sample of 2,376 individuals from the National FINRISK study. First, we detect geographic differences in PS for coronary artery disease (CAD), rheumatoid arthritis, schizophrenia, waits-hip ratio (WHR), body-mass index (BMI) and height, but not for Crohn’s disease or ulcerative colitis. Second, we use height as a model trait to thoroughly assess the possible population genetic biases in PS and apply similar approaches to the other phenotypes. Most importantly, we detect suspiciously large accumulation of geographic differences for CAD, WHR, BMI and height, suggesting bias arising from population genetic structure rather than from a direct genotype-phenotype association. This work demonstrates how sensitive the geographic patterns of current PS are for small biases even within relatively homogenous populations and provides simple tools to identify such biases. A thorough understanding of the effects of population genetic structure on PS is essential for translational applications of PS.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/large-scale_exome_sequencing_study_implicates_both_developmental_and_functional_changes_in_the_neurobiology_of_autism_52024757/">
          <h4 class="font-weight-bold">Large-scale exome sequencing study implicates both developmental and functional changes in the neurobiology of autism, bioRxiv, 2018-12-01</h4>
          <p>SummaryWe present the largest exome sequencing study of autism spectrum disorder (ASD) to date (n=35,584 total samples, 11,986 with ASD). Using an enhanced Bayesian framework to integrate de novo and case-control rare variation, we identify 102 risk genes at a false discovery rate ≤ 0.1. Of these genes, 49 show higher frequencies of disruptive de novo variants in individuals ascertained for severe neurodevelopmental delay, while 53 show higher frequencies in individuals ascertained for ASD; comparing ASD cases with mutations in these groups reveals phenotypic differences. Expressed early in brain development, most of the risk genes have roles in regulation of gene expression or neuronal communication (i.e., mutations effect neurodevelopmental and neurophysiological changes), and 13 fall within loci recurrently hit by copy number variants. In human cortex single-cell gene expression data, expression of risk genes is enriched in both excitatory and inhibitory neuronal lineages, consistent with multiple paths to an excitatoryinhibitory imbalance underlying ASD.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/e-chrps_engineered_chromatin_remodeling_proteins_for_precise_nucleosome_positioning_51967111/">
          <h4 class="font-weight-bold">E-ChRPs Engineered Chromatin Remodeling Proteins for Precise Nucleosome Positioning, bioRxiv, 2018-11-29</h4>
          <p>SummaryRegulation of chromatin structure is essential for controlling the access of DNA to factors that require association with specific DNA sequences. The ability to alter chromatin organization in a targeted manner would provide a mechanism for directly manipulating DNA-dependent processes and should provide a means to study direct consequences of chromatin structural changes. Here we describe the development and validation of engineered chromatin remodeling proteins (E-ChRPs) for inducing programmable changes in nucleosome positioning by design. We demonstrate that E-ChRPs function both in vivo and in vitro to specifically reposition target nucleosomes and entire nucleosomal arrays, and possess the ability to evict native DNA-binding proteins through their action. E-ChRPs can be designed with a range of targeting modalities, including the SpyCatcher and dCas9 moieties, resulting in high versatility and enabling diverse future applications. Thus, engineered chromatin remodeling proteins represent a simple and robust means to probe regulation of DNA-dependent processes in different chromatin contexts.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/learning_ones_genetic_risk_changes_physiology_independent_of_actual_genetic_risk_52464320/">
          <h4 class="font-weight-bold">Learning one’s genetic risk changes physiology independent of actual genetic risk, Nature Human Behaviour, 2018-11-29</h4>
          <p>5</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/nature%20human%20behaviour">nature human behaviour</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/genetics/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/genetics/page/12/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/genetics/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/genetics/page/10/" class="page-link">10</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/genetics/page/11/" class="page-link">11</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/genetics/page/12/" class="page-link">12</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/genetics/page/13/" class="page-link">13</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/genetics/page/14/" class="page-link">14</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/genetics/page/15/" class="page-link">15</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/genetics/page/16/" class="page-link">16</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/genetics/page/17/" class="page-link">17</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/genetics/page/14/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/genetics/page/22/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
