<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/hierarchical_recurrent_state_space_models_reveal_discrete_and_continuous_dynamics_of_neural_activity_in_c._elegans_59680614/">
          <h4 class="font-weight-bold">Hierarchical recurrent state space models reveal discrete and continuous dynamics of neural activity in C. elegans, bioRxiv, 2019-04-29</h4>
          <p>AbstractModern recording techniques enable large-scale measurements of neural activity in a variety of model organisms. The dynamics of neural activity shed light on how organisms process sensory information and generate motor behavior. Here, we study these dynamics using optical recordings of neural activity in the nematode C. elegans. To understand these data, we develop state space models that decompose neural time-series into segments with simple, linear dynamics. We incorporate these models into a hierarchical framework that combines partial recordings from many worms to learn shared structure, while still allowing for individual variability. This framework reveals latent states of population neural activity, along with the discrete behavioral states that govern dynamics in this state space. We find stochastic transition patterns between discrete states and see that transition probabilities are determined by both current brain activity and sensory cues. Our methods automatically recover transition times that closely match manual labels of different behaviors, such as forward crawling, reversals, and turns. Finally, the resulting model can simulate neural data, faithfully capturing salient patterns of whole brain dynamics seen in real data.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/immunogenomic_landscape_of_hematological_malignancies_59575484/">
          <h4 class="font-weight-bold">Immunogenomic landscape of hematological malignancies, bioRxiv, 2019-04-27</h4>
          <p>SUMMARYUnderstanding factors that shape the immune landscape across hematological malignancies is essential for immunotherapy development. Here, we integrated over 8,000 transcriptomes and over 1,000 samples with multilevel genomic data of hematological cancers to investigate how immunological features are linked to cancer subtypes, genetic and epigenetic alterations, and patient survival. Infiltration of cytotoxic immune cells was associated with distinct microenvironmental responses and driver alterations in different cancers, such as TP53 in acute myeloid leukemia and DTX1 in diffuse large B cell lymphoma. Epigenetic modification of CIITA regulating antigen presentation, cancer type-specific immune checkpoints such as VISTA in myeloid malignancies, and variation in cancer antigen expression further contributed to immune heterogeneity. Prognostic models highlighted the significance of immunological properties in predicting survival. Our study represents the most comprehensive effort to date to link immunology with cancer subtypes and genomics in hematological malignancies, providing a resource to guide future studies and immunotherapy development.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/safety_and_tolerability_of_bacteriophage_therapy_in_severe_staphylococcus_aureus_infection_59575100/">
          <h4 class="font-weight-bold">Safety and Tolerability of Bacteriophage Therapy in Severe Staphylococcus aureus Infection, bioRxiv, 2019-04-27</h4>
          <p>AbstractImportanceThe effect of IV administration of a bacteriophage cocktail produced under GMP conditions on patients with severe S. aureus infection, including complicated bacteraemia, endocarditis and septic shock, is unknown.ObjectiveTo assess safety and tolerability of adjunctive bacteriophage therapy in patients with severe S. aureus infections.Design, Setting, ParticipantsObservational, open-label clinical trial of thirteen critically-ill patients admitted to a tertiary-referral hospital with S. aureus bacteraemia (including infective endocarditis, n=6) were assessed by the treating clinician and two consulting infectious diseases physicians to independently verify that routine medical and surgical therapy was optimal and that a poor outcome remained likely. Compassionate access to therapy was approved by both US and Australian regulators and by the Westmead Hospital Human Research Ethics Committee.InterventionA GMP-quality preparation of three combined Myoviridae bacteriophages with specific activity against S. aureus (AB-SA01), was administered intravenously in conjunction with optimal antibiotic therapy.Main Outcome and MeasurementsPhysiological, haematological and biochemical markers of infection, bacterial and bacteriophage kinetics in blood, development of resistance to bacteriophages, and mortality at 28 (D28) and 90 (D90) days were measured. Main outcomes were safety and tolerability.ResultsBacteriophage therapy was initiated 4-10 days after antibiotic commencement, at 109 plaque-forming units (PFU) twice daily. Infecting staphylococci were typical of common local subtypes. Initial input ratio of phages to bacteria in the bloodstream (MOIinput) was &amp;gt;100. Five of the thirteen patients died by D28 and a sixth patient suffered sudden cardiac death on D90. Bacteriophage therapy coincided with a marked reduction in staphylococcal bacterial DNA in the blood and in sepsis-associated inflammatory responses in almost all cases. No bacteriophage-attributable adverse events were identified. Development of bacteriophage resistance was not observed. Population analysis revealed no significant effect of bacteriophage therapy on the gut microflora.Conclusions and RelevanceAdjunctive bacteriophage therapy appears to be safe and well-tolerated in critically ill patients with severe S. aureus infection. Two weeks of twice daily intravenous administration may be a suitable protocol. Controlled trials are needed.Trial RegistrationWestmead Hospital Human Research Ethics Committee approval July 11, 2017; ClinicalTrials.gov Identifier &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=clintrialgov xlinkhref=NCT03395769&gt;NCT03395769&lt;jatsext-link&gt;, AB-SA01-EAP01 (January 10, 2018); Clinical Trials Notification (Australian Therapeutic Goods Association) CT-2018-CTN-02372-1 (July 23, 2018).Key PointsQuestionIs intravenous (IV) administration of investigational bacteriophage (phage) therapy safe and well-tolerated in patients with severe Staphylococcus aureus infection?FindingsThirteen patients with severe S. aureus infections received AB-SA01, a bacteriophage product prepared according to Good Manufacturing Practices (GMP), as adjunctive therapy to antibiotics. AB-SA01 was well-tolerated with no adverse events identified. Bacterial burden and inflammatory responses were reduced and no phage-resistant staphylococci were isolated during or after therapy.MeaningOur results will inform future randomised controlled trials assessing the antibacterial and anti-inflammatory potential of bacteriophages in the treatment of severe S. aureus infection.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/clinical-trials">clinical-trials</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/ctcf-dependent_chromatin_boundaries_formed_by_asymmetric_nucleosome_arrays_with_decreased_linker_length_59529123/">
          <h4 class="font-weight-bold">CTCF-dependent chromatin boundaries formed by asymmetric nucleosome arrays with decreased linker length, bioRxiv, 2019-04-26</h4>
          <p>AbstractThe CCCTC-binding factor (CTCF) organises the genome in 3D through DNA loops and in 1D by setting boundaries isolating different chromatin states, but these processes are not well understood. Here we focus on the relationship between CTCF binding and the decrease of the Nucleosome Repeat Length (NRL) for ∼20 adjacent nucleosomes, affecting up to 10% of the mouse genome. We found that the chromatin boundary near CTCF is created by the nucleosome-depleted region (NDR) asymmetrically located &amp;gt;40 nucleotides 5’-upstream from the centre of CTCF motif. The strength of CTCF binding to DNA is correlated with the decrease of NRL near CTCF and anti-correlated with the level of asymmetry of the nucleosome array. Individual chromatin remodellers have different contributions, with Snf2h having the strongest effect on the NRL decrease near CTCF and Chd4 playing a major role in the symmetry breaking. Upon differentiation of embryonic stem cells to neural progenitor cells and embryonic fibroblasts, a subset of common CTCF sites preserved in all three cell types maintains a relatively small local NRL despite genome-wide NRL increase. The sites which lost CTCF upon differentiation are characterised by nucleosome rearrangement 3’-downstream, but the boundary defined by the NDR 5’-upstream of CTCF motif remains.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/gephebase_a_database_of_genotype-phenotype_relationships_for_natural_and_domesticated_variation_in_eukaryotes_59510592/">
          <h4 class="font-weight-bold">Gephebase, a Database of Genotype-Phenotype Relationships for natural and domesticated variation in Eukaryotes, bioRxiv, 2019-04-25</h4>
          <p>AbstractGephebase is a manually-curated database compiling our accumulated knowledge of the genes and mutations that underlie natural, domesticated and experimental phenotypic variation in all Eukaryotes — mostly animals, plants and yeasts. Gephebase aims to compile studies where the genotype-phenotype association (based on linkage mapping, association mapping or a candidate gene approach) is relatively well supported or understood. Human disease and aberrant mutant phenotypes in laboratory model organisms are not included in Gephebase and can be found in other databases (eg. OMIM, OMIA, Monarch Initiative). Gephebase contains more than 1700 entries. Each entry corresponds to an allelic difference at a given gene and its associated phenotypic change(s) between two species or between two individuals of the same species, and is enriched with molecular details, taxonomic information, and bibliographic information. Users can easily browse entries for their topic of interest and perform searches at various levels, whether phenotypic, genetic, taxonomic or bibliographic (eg. transposable elements, cis-regulatory mutations, snakes, carotenoid content, an author name). Data can be searched using keywords and boolean operators and is exportable in spreadsheet format. This database allows to perform meta-analysis to extract general information and global trends about evolution, genetics, and the field of evolutionary genetics itself. Gephebase should also help breeders, conservationists and others to identify the most promising target genes for traits of interest, with potential applications such as crop improvement, parasite and pest control, bioconservation, and genetic diagnostic. It is freely available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpwww.gephebase.org&gt;www.gephebase.org&lt;jatsext-link&gt;.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/neighbor_predation_linked_to_natural_competence_fosters_the_transfer_of_large_genomic_regions_in_vibrio_cholerae_59471825/">
          <h4 class="font-weight-bold">Neighbor predation linked to natural competence fosters the transfer of large genomic regions in Vibrio cholerae, bioRxiv, 2019-04-25</h4>
          <p>AbstractNatural competence for transformation is a primary mode of horizontal gene transfer (HGT). Competent bacteria are able to absorb free DNA from their surroundings and exchange this DNA against pieces of their own genome when sufficiently homologous. And while it is known that transformation contributes to evolution and pathogen emergence in bacteria, there are still questions regarding the general prevalence of non-degraded DNA with sufficient coding capacity. In this context, we previously showed that the naturally competent bacterium Vibrio cholerae uses its type VI secretion system (T6SS) to actively acquire DNA from non-kin neighbors under chitin-colonizing conditions. We therefore sought to further explore the role of the T6SS in acquiring DNA, the condition of the DNA released through T6SS-mediated killing versus passive cell lysis, and the extent of the transfers that occur due to these conditions. To do this, we herein measured the frequency and the extent of genetic exchanges in bacterial co-cultures on competence-inducing chitin under various DNA-acquisition conditions. We show that competent V. cholerae strains acquire DNA fragments with an average and maximum length exceeding 50 kbp and 150 kbp, respectively, and that the T6SS is of prime importance for such HGT events. Collectively, our data support the notion that the environmental lifestyle of V. cholerae fosters HGT and that the coding capacity of the exchanged genetic material is sufficient to significantly accelerate bacterial evolution.Significance StatementDNA shuffled from one organism to another in an inheritable manner is a common feature of prokaryotes. It is a significant mechanism by which bacteria acquire new phenotypes, for example by first absorbing foreign DNA and then recombining it into their genome. In this study, we show the remarkable extent of the exchanged genetic material, frequently exceeding 150 genes in a seemingly single transfer event, in Vibrio cholerae. We also show that to best preserve its length and quality, bacteria mainly acquire this DNA by killing adjacent, healthy neighbors then immediately absorbing the released DNA before it can be degraded. These new insights into this prey-killing DNA acquisition process shed light on how bacterial species evolve in the wild.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/0-100-users/page/8/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/8/" class="page-link">8</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/0-100-users/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/10/" class="page-link">10</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/11/" class="page-link">11</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/12/" class="page-link">12</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/13/" class="page-link">13</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/0-100-users/page/10/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/0-100-users/page/64/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
