<!DOCTYPE html>
<html><head>
  
  <title>2019 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.68.3" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.95d2483cd3c5d2bf47c1f36307a456787d60ad958c5efe44db9afc47012b1b05b8e1f64f6ac291650439322a6c38e86eb7cd6254255516d857c23f299fb0e666.js" integrity="sha512-ldJIPNPF0r9HwfNjB6RWeH1grZWMXv5E25r8RwErGwW44fZPasKRZQQ5MipsOOhut81iVCVVFthXwj8pn7DmZg=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/clonal_replacement_of_tumor-specific_t_cells_following_pd-1_blockade_61034489/">
          <h4 class="font-weight-bold">Clonal replacement of tumor-specific T cells following PD-1 blockade, bioRxiv, 2019-05-24</h4>
          <p>AbstractImmunotherapies that block inhibitory checkpoint receptors on T cells have transformed the clinical care of cancer patients. However, which tumor-specific T cells are mobilized following checkpoint blockade remains unclear. Here, we performed paired single-cell RNA- and T cell receptor (TCR)-sequencing on 79,046 cells from site-matched tumors from patients with basal cell carcinoma (BCC) or squamous cell carcinoma (SCC) pre- and post-anti-PD-1 therapy. Tracking TCR clones and transcriptional phenotypes revealed a coupling of tumor-recognition, clonal expansion, and T cell dysfunction the T cell response to treatment was accompanied by clonal expansions of CD8&#43;CD39&#43; T cells, which co-expressed markers of chronic T cell activation and exhaustion. However, this expansion did not derive from pre-existing tumor infiltrating T cell clones; rather, it comprised novel clonotypes, which were not previously observed in the same tumor. Clonal replacement of T cells was preferentially observed in exhausted CD8&#43; T cells, compared to other distinct T cell phenotypes, and was evident in BCC and SCC patients. These results, enabled by single-cell multi-omic profiling of clinical samples, demonstrate that pre-existing tumor-specific T cells may be limited in their capacity for re-invigoration, and that the T cell response to checkpoint blockade relies on the expansion of a distinct repertoire of T cell clones that may have just recently entered the tumor.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/enabling_high-accuracy_long-read_amplicon_sequences_using_unique_molecular_identifiers_with_nanopore_or_pacbio_sequencing_61024590/">
          <h4 class="font-weight-bold">Enabling high-accuracy long-read amplicon sequences using unique molecular identifiers with Nanopore or PacBio sequencing, bioRxiv, 2019-05-24</h4>
          <p>AbstractHigh-throughput amplicon sequencing of large genomic regions remains challenging for short-read technologies. Here, we report a high-throughput amplicon sequencing approach combining unique molecular identifiers (UMIs) with Oxford Nanopore Technologies or Pacific Biosciences CCS sequencing, yielding high accuracy single-molecule consensus sequences of large genomic regions. Our approach generates amplicon and genomic sequences of &amp;gt;10,000 bp in length with a mean error-rate of 0.0049-0.0006% and chimera rate &amp;lt;0.022%.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/novel_rhabdovirus_and_an_almost_complete_drain_fly_transcriptome_recovered_from_two_independent_contaminations_of_clinical_samples_60978515/">
          <h4 class="font-weight-bold">Novel Rhabdovirus and an almost complete drain fly transcriptome recovered from two independent contaminations of clinical samples, bioRxiv, 2019-05-24</h4>
          <p>AbstractMetagenomic approaches enable an open exploration of microbial communities without requiring a priori knowledge of a sample’s composition by shotgun sequencing the total RNA or DNA of the sample. Such an approach is valuable for exploratory diagnostics of novel pathogens in clinical practice. Yet, one may also identify surprising off-target findings. Here we report a mostly complete transcriptome from a drain fly (likely Psychoda alternata) as well as a novel Rhabdovirus-like virus recovered from two independent contaminations of RNA sequencing libraries from clinical samples of cerebral spinal fluid (CSF) and serum, out of a total of 724 libraries sequenced at the same laboratory during a 2-year time span. This drain fly genome shows a considerable divergence from previously sequenced insects, which may obscure common clinical metagenomic analyses not expecting such contaminations. The classification of these contaminant sequences allowed us to identify infected drain flies as the likely origin of the novel Rhabdovirus-like sequence, which could have been erroneously linked to human pathology, had they been ignored.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/cross-species_cortical_alignment_identifies_different_types_of_neuroanatomical_reorganization_in_the_temporal_lobe_of_higher_primates_60882961/">
          <h4 class="font-weight-bold">Cross-species cortical alignment identifies different types of neuroanatomical reorganization in the temporal lobe of higher primates, bioRxiv, 2019-05-23</h4>
          <p>AbstractEvolutionary modifications of the temporo-parietal cortex are considered to be a critical adaptation of the human brain. Cortical adaptations, however, can affect different aspects of brain architecture, including areal expansion or changes in connectivity profiles. We propose to distinguishing different types of brain reorganization using a computational neuroanatomy approach. We investigate the extent to which between-species alignment based on cortical myelin can predict changes in connectivity patterns across macaque, chimpanzee and human. We show that expansion and relocation of brain areas are sufficient to predict terminations of several white matter tracts in temporo-parietal cortex, including the middle and superior longitudinal fasciculus, but not of the arcuate fasciculus. This demonstrates that the arcuate fasciculus underwent additional evolutionary modifications affecting the connectivity pattern of the temporal lobe. The presented approach can flexibly be extended to include other features of cortical organization and other species, allowing direct tests of comparative hypotheses of brain organization.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/granar_a_new_computational_tool_to_better_understand_the_functional_importance_of_root_anatomy_60880272/">
          <h4 class="font-weight-bold">GRANAR, a new computational tool to better understand the functional importance of root anatomy, bioRxiv, 2019-05-23</h4>
          <p>AbstractRoot hydraulic conductivity is an important determinant of plant water uptake capacity. In particular, the root radial conductivity is often thought to be a limiting factor along the water pathways between the soil and the leaf. The root radial conductivity is itself defined by cell scale hydraulic properties and anatomical features. However, quantifying the influence of anatomical features on the radial conductivity remains challenging due to complex, and time-consuming, experimental procedures.We present a new computation tool, the Generator of Root ANAtomy in R (GRANAR) that can be used to rapidly generate digital versions of root anatomical networks. GRANAR uses a limited set of root anatomical parameters, easily acquired with existing image analysis tools. The generated anatomical network can then be used in combination with hydraulic models to estimate the corresponding hydraulic properties.We used GRANAR to re-analyse large maize (Zea mays) anatomical datasets from the literature. Our model was successful at creating virtual anatomies for each experimental observation. We also used GRANAR to generate anatomies not observed experimentally, over wider ranges of anatomical parameters. The generated anatomies were then used to estimate the corresponding radial conductivities with the hydraulic model MECHA. This enabled us to quantify the effect of individual anatomical features on the root radial conductivity. In particular, our simulations highlight the large importance of the width of the stele and the cortex.GRANAR is an open-source project available here &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpgranar.github.io&gt;httpgranar.github.io&lt;jatsext-link&gt;One-Sentence summaryGenerator of Root ANAtomy in R (GRANAR) is a new open-source computational tool that can be used to rapidly generate digital versions of root anatomical networks.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/allelic_polymorphism_shapes_community_function_in_evolving_pseudomonas_aeruginosa_populations_60781879/">
          <h4 class="font-weight-bold">Allelic polymorphism shapes community function in evolving Pseudomonas aeruginosa populations, bioRxiv, 2019-05-22</h4>
          <p>AbstractPseudomonas aeruginosa is an opportunistic pathogen that chronically infects the lungs of individuals with cystic fibrosis (CF) by forming antibiotic resistant biofilms. Emergence of phenotypically diverse isolates within CF P. aeruginosa populations has previously been reported, however, the impact of heterogeneity on social behaviors and community function is poorly understood. Here we describe how this heterogeneity impacts on behavioral traits by evolving the strain PAO1 in biofilms grown in a synthetic sputum medium for 50 days. We measured social trait production and antibiotic resistance, and used a metagenomic approach to analyze and assess genomic changes over the duration of the evolution experiment. We found that (i) evolutionary trajectories were reproducible in independently evolving populations; (ii) over 60% of genomic diversity occurred within the first 10 days of selection. We then focused on quorum sensing (QS), a well-studied P. aeruginosa trait that is commonly mutated in strains isolated from CF lungs. We found that at the population level (i) evolution in sputum medium selected for decreased production of QS and QS-dependent traits; (ii) there was a significant correlation between lasR mutant frequency, the loss of protease and the 3O-C12-HSL signal, and an increase in resistance to clinically relevant β-lactam antibiotics, despite no previous antibiotic exposure. Overall, our findings provide insights into the impact of allelic polymorphism on community functions in diverse P. aeruginosa populations. Further, we demonstrate that P. aeruginosa population and evolutionary dynamics can impact on traits important for virulence and can lead to increased tolerance to β-lactam antibiotics.SignificancePseudomonas aeruginosa is a major cause of chronic infection of the lungs in individuals with cystic fibrosis (CF). Epidemic P. aeruginosa strains dominate and displace others in the lung and become phenotypically and genotypically diverse over time. How this heterogeneity impacts on clinically relevant traits such as virulence or antibiotic resistance has received little attention. Here we demonstrate that the accumulation of genetic variants within heterogenous P. aeruginosa populations, results in population level changes in important traits including quorum sensing and antimicrobial resistance. Our work suggests that in the future, we should consider metagenomic and metaphenotypic assessments of P. aeruginosa populations collected from CF patients, rather than focusing on single random colonies isolated from infection.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/53/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/50/" class="page-link">50</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/51/" class="page-link">51</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/52/" class="page-link">52</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/53/" class="page-link">53</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/2019/page/54/" class="page-link">54</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/55/" class="page-link">55</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/56/" class="page-link">56</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/57/" class="page-link">57</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/58/" class="page-link">58</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/55/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/135/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
