<!DOCTYPE html>
<html><head>
  
  <title>Reports | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/ultrasound_imaging_of_gene_expression_in_mammalian_cells_57245862/">
          <h4 class="font-weight-bold">Ultrasound Imaging of Gene Expression in Mammalian Cells, bioRxiv, 2019-03-19</h4>
          <p>ABSTRACTThe study of cellular processes occurring inside intact organisms and the development of cell-based diagnostic and therapeutic agents requires methods to visualize cellular functions such as gene expression in deep tissues. Ultrasound is a widely used biomedical technology enabling deep-tissue imaging with high spatial and temporal resolution. However, no genetically encoded molecular reporters are available to connect ultrasound contrast to gene expression in mammalian cells. To address this limitation, we introduce the first mammalian acoustic reporter genes. Starting with an eleven-gene polycistronic gene cluster derived from bacteria, we engineered a eukaryotic genetic program whose introduction into mammalian cells results in the expression of a unique class of intracellular air-filled protein nanostructures called gas vesicles. The scattering of ultrasound by these nanostructures allows mammalian cells to be visualized at volumetric densities below 0.5%, enables the monitoring of dynamic circuit-driven gene expression, and permits high-resolution imaging of gene expression in living animals. These mammalian acoustic reporter genes enable previously impossible approaches to monitoring the location, viability and function of mammalian cells in vivo.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioengineering">bioengineering</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/a_deep_learning_framework_for_nucleus_segmentation_using_image_style_transfer_57190621/">
          <h4 class="font-weight-bold">A deep learning framework for nucleus segmentation using image style transfer, bioRxiv, 2019-03-18</h4>
          <p>AbstractSingle cell segmentation is typically one of the first and most crucial tasks of image-based cellular analysis. We present a deep learning approach aiming towards a truly general method for localizing nuclei across a diverse range of assays and light microscopy modalities. We outperform the 739 methods submitted to the 2018 Data Science Bowl on images representing a variety of realistic conditions, some of which were not represented in the training data. The key to our approach is to adapt our model to unseen and unlabeled data using image style transfer to generate augmented training samples. This allows the model to recognize nuclei in new and different experiments without requiring expert annotations.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/metabolic_activity_affects_response_of_single_cells_to_a_nutrient_switch_in_structured_populations_57196911/">
          <h4 class="font-weight-bold">Metabolic activity affects response of single cells to a nutrient switch in structured populations, bioRxiv, 2019-03-18</h4>
          <p>AbstractMicrobes live in ever-changing environments where they need to adapt their metabolism to different nutrient conditions. Many studies have characterized the response of genetically identical cells to nutrient switches in homogenous cultures, however in nature microbes often live in spatially structured groups such as biofilms where cells can create metabolic gradients by consuming and releasing nutrients. Consequently, cells experience different local microenvironments and vary in their phenotype. How does this phenotypic variation affect the ability of cells to cope with nutrient switches? Here we address this question by growing dense populations of Escherichia coli in microfluidic chambers and studying a switch from glucose to acetate at the single cell level. Before the switch, cells vary in their metabolic activity some grow on glucose while others cross-feed on acetate. After the switch, only few cells can resume growth after a period of lag. The probability to resume growth depends on a cells’ phenotype prior to the switch it is highest for cells crossfeeding on acetate, while it depends in a non-monotonic way on growth rate for cells growing on glucose. Our results suggest that the strong phenotypic variation in spatially structured populations might enhance their ability to cope with fluctuating environments.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/plotsofdifferences__a_web_app_for_the_quantitative_comparison_of_unpaired_data_57190672/">
          <h4 class="font-weight-bold">PlotsOfDifferences – a web app for the quantitative comparison of unpaired data, bioRxiv, 2019-03-18</h4>
          <p>AbstractThe quantitative comparison of data acquired under different conditions is an important aspect of experimental science. The most widely used statistic for quantitative comparisons is the p-value. However, p-values suffer from several shortcomings. The most prominent shortcoming that is relevant for quantitative comparisons is that p-values fail to convey the magnitude of differences. The differences between conditions are best quantified by the determination of effect size. To democratize the calculation of effect size, we have developed a web-based tool. The tool uses bootstrapping to resample mean or median values for each of the conditions and these values are used to calculate the effect size and their compatibility interval. The web tool generates a graphical output, showing the bootstrap distribution of the difference next to the actual data for optimal interpretation. A tabular output with statistics and effect sizes is also generated and the table can be supplemented with p-values that are calculated with a randomization test. The app that we report here is dubbed PlotsOfDifferences and is available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpshuygens.science.uva.nlPlotsOfDifferences&gt;httpshuygens.science.uva.nlPlotsOfDifferences&lt;jatsext-link&gt;&lt;jatsfig id=ufig1 position=float fig-type=figure orientation=portrait&gt;&lt;jatsgraphic xmlnsxlink=httpwww.w3.org1999xlink xlinkhref=578575_ufig1 position=float orientation=portrait &gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/scientific-communication-and-education">scientific-communication-and-education</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/dosing_time_matters_57093657/">
          <h4 class="font-weight-bold">Dosing Time Matters, bioRxiv, 2019-03-16</h4>
          <p>AbstractTrainees in medicine are taught to diagnose and administer treatment as needed; time-of-day is rarely considered. Yet accumulating evidence shows that ∼half of human genes and physiologic functions follow daily rhythms. Circadian medicine aims to incorporate knowledge of these rhythms to enhance diagnosis and treatment. Interest in this approach goes back at least six decades, but the path to the clinic has been marked by starts, stops, and ambiguity. How do we move the field forward to impact clinical practice? To gain insight into successful strategies, we studied the results of more than 100 human trials that evaluated time-of-administration of drugs.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/epidemiology">epidemiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/evolutionary_dynamics_in_structured_populations_under_strong_population_genetic_forces_57127635/">
          <h4 class="font-weight-bold">Evolutionary dynamics in structured populations under strong population genetic forces, bioRxiv, 2019-03-16</h4>
          <p>High rates of migration between subpopulations result in little population differentiation in the long-term neutral equilibrium. However, in the short-term, even very abundant migration may not be enough for subpopulations to equilibrate immediately. In this study, we investigate dynamical patterns of short-term population differentiation in adapting populations via stochastic and analytical modeling through time. We characterize a regime in which selection and migration interact to create non-monotonic patterns of the population differentiation statistic FST when migration is weaker than selection, but stronger than drift. We demonstrate how these patterns can be leveraged to estimate high migration rates that would lead to panmixia in the long term equilibrium using an approximate Bayesian computation approach. We apply this approach to estimate fast migration in a rapidly adapting intra-host Simian-HIV population sampled from different anatomical locations. Notably, we find differences in estimated migration rates between different compartments, all above  Nem = 1. This work demonstrates how studying demographic processes on the timescale of selective sweeps illuminates processes too fast to leave signatures on neutral timescales.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/reports/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/reports/page/46/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/reports/page/43/" class="page-link">43</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/44/" class="page-link">44</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/45/" class="page-link">45</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/46/" class="page-link">46</a></li>
            
        
             
            <li class="page-item active"><a href="/reports/page/47/" class="page-link">47</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/48/" class="page-link">48</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/49/" class="page-link">49</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/50/" class="page-link">50</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/51/" class="page-link">51</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/reports/page/48/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/reports/page/172/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
