<!DOCTYPE html>
<html><head>
  
  <title>2019 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/deepfly3d_a_deep_learning-based_approach_for_3d_limb_and_appendage_tracking_in_tethered_adult_drosophila_60710317/">
          <h4 class="font-weight-bold">DeepFly3D A deep learning-based approach for 3D limb and appendage tracking in tethered, adult Drosophila, bioRxiv, 2019-05-20</h4>
          <p>AbstractStudying how neural circuits orchestrate limbed behaviors requires the precise measurement of the positions of each appendage in 3-dimensional (3D) space. Deep neural networks can estimate 2-dimensional (2D) pose in freely behaving and tethered animals. However, the unique challenges associated with transforming these 2D measurements into reliable and precise 3D poses have not been addressed for small animals including the fly, Drosophila melanogaster. Here we present DeepFly3D, a software that infers the 3D pose of tethered, adult Drosophila—or other animals—using multiple camera images. DeepFly3D does not require manual calibration, uses pictorial structures to automatically detect and correct pose estimation errors, and uses active learning to iteratively improve performance. We demonstrate more accurate unsupervised behavioral embedding using 3D joint angles rather than commonly used 2D pose data. Thus, DeepFly3D enables the automated acquisition of behavioral measurements at an unprecedented level of resolution for a variety of biological applications.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/animal-behavior-and-cognition">animal-behavior-and-cognition</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/single-cell_transcriptomics_reveals_expansion_of_cytotoxic_cd4_t-cells_in_supercentenarians_60713002/">
          <h4 class="font-weight-bold">Single-cell transcriptomics reveals expansion of cytotoxic CD4 T-cells in supercentenarians, bioRxiv, 2019-05-20</h4>
          <p>AbstractSupercentenarians, people who have reached 110 years of age, are a great model of healthy aging. Their characteristics of delayed onset of age-related diseases and compression of morbidity imply that their immune system remains functional. Here we performed single-cell transcriptome analysis of 61,202 peripheral blood mononuclear cells (PBMCs), derived from seven supercentenarians and five younger controls. We identified a marked increase of cytotoxic CD4 T-cells (CD4 CTLs) coupled with a substantial reduction of B-cells as a novel signature of supercentenarians. Furthermore, single-cell T-cell receptor sequencing of two supercentenarians revealed that CD4 CTLs had accumulated through massive clonal expansion, with the most frequent clonotypes accounting for 15% to 35% of the entire CD4 T-cell population. The CD4 CTLs exhibited substantial heterogeneity in their degree of cytotoxicity as well as a nearly identical transcriptome to that of CD8 CTLs. This indicates that CD4 CTLs utilize the transcriptional program of the CD8 lineage while retaining CD4 expression. Our study reveals that supercentenarians have unique characteristics in their circulating lymphocytes, which may represent an essential adaptation to achieve exceptional longevity by sustaining immune responses to infections and diseases.SignificanceExceptionally long-lived people such as supercentenarians tend to spend their entire lives in good health, implying that their immune system remains active to protect against infections and tumors. However, their immunological condition has been largely unexplored. We profiled thousands of circulating immune cells from supercentenarians at single-cell resolution, and identified a large number of CD4 T-cells that have cytotoxic features. This characteristic is very unique to supercentenarians, because generally CD4 T-cells have helper, but not cytotoxic, functions under physiological conditions. We further profiled their T-cell receptors, and revealed that the cytotoxic CD4 T-cells were accumulated through clonal expansion. The conversion of helper CD4 T-cells to a cytotoxic variety might be an adaptation to the late stage of aging.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/spatiotemporal_limits_of_optogenetic_manipulations_in_cortical_circuits_60710064/">
          <h4 class="font-weight-bold">Spatiotemporal limits of optogenetic manipulations in cortical circuits, bioRxiv, 2019-05-20</h4>
          <p>AbstractNeuronal inactivation is commonly used to assess the involvement of groups of neurons in specific brain functions. Optogenetic tools allow manipulations of genetically and spatially defined neuronal populations with excellent temporal resolution. However, the targeted neurons are coupled with other neural populations over multiple length scales. As a result, the effects of localized optogenetic manipulations are not limited to the targeted neurons, but produces spatially extended excitation and inhibition with rich dynamics. Here we benchmarked several optogenetic silencers in transgenic mice and with viral gene transduction, with the goal to inactivate excitatory neurons in small regions of neocortex. We analyzed the effects of the perturbations in vivo using electrophysiology. Channelrhodopsin activation of GABAergic neurons produced more effective photoinhibition of pyramidal neurons than direct photoinhibition using light-gated ion pumps. We made transgenic mice expressing the light-dependent chloride channel GtACR under the control of Cre-recombinase. Activation of GtACR produced the most potent photoinhibition. For all methods, localized photostimuli produced photoinhibition that extended substantially beyond the spread of light in tissue, although different methods had slightly different resolution limits (radius of inactivation, 0.5 mm to 1 mm). The spatial profile of photoinhibition was likely shaped by strong coupling between cortical neurons. Over some range of photostimulation, circuits produced the “paradoxical effect”, where excitation of inhibitory neurons reduced activity in these neurons, together with pyramidal neurons, a signature of inhibition-stabilized neural networks. The offset of optogenetic inactivation was followed by rebound excitation in a light dose-dependent manner, which can be mitigated by slowly varying photostimuli, but at the expense of time resolution. Our data offer guidance for the design of in vivo optogenetics experiments and suggest how these experiments can reveal operating principles of neural circuits.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_rationally_designed_and_highly_versatile_epitope_tag_for_nanobody-based_purification_detection_and_manipulation_of_proteins_60613615/">
          <h4 class="font-weight-bold">A rationally designed and highly versatile epitope tag for nanobody-based purification, detection and manipulation of proteins, bioRxiv, 2019-05-18</h4>
          <p>AbstractSpecialized epitope tags are widely used for detecting, manipulating or purifying proteins, but often their versatility is limited. Here, we introduce the ALFA-tag, a novel, rationally designed epitope tag that serves an exceptionally broad spectrum of applications in life sciences while outperforming established tags like the HA, FLAG or myc tags. The ALFA-tag forms a small and stable α-helix that is functional irrespective of its position on the target protein in prokaryotic and eukaryotic hosts. We developed a nanobody (NbALFA) binding ALFA-tagged proteins from native or fixed specimen with low picomolar affinity. It is ideally suited for super-resolution microscopy, immunoprecipitations and Western blotting, and also allows in-vivo detection of proteins. By solving the crystal structure of the complex we were able to design a nanobody mutant (NbALFAPE) that permits efficient one-step purifications of native ALFA-tagged proteins, complexes and even entire living cells using peptide elution under physiological conditions.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biochemistry">biochemistry</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/high-throughput_microcircuit_analysis_of_individual_human_brains_through_next-generation_multineuron_patch-clamp_60634870/">
          <h4 class="font-weight-bold">High-throughput microcircuit analysis of individual human brains through next-generation multineuron patch-clamp, bioRxiv, 2019-05-18</h4>
          <p>AbstractComparing neuronal microcircuits across different brain regions, species and individuals can reveal common and divergent principles of network computation. Simultaneous patch-clamp recordings from multiple neurons offer the highest temporal and subthreshold resolution to analyse local synaptic connectivity. However, its establishment is technically complex and the experimental performance is limited by high failure rates, long experimental times and small sample sizes. We introduce an in-vitro multipatch setup with an automated pipette pressure and cleaning system facilitating recordings of up to 10 neurons simultaneously and sequential patching of additional neurons. We present hardware and software solutions that increase the usability, speed and data throughput of multipatch experiments which allowed probing of 150 synaptic connections between 17 neurons in one human cortical slice and screening of over 600 connections in tissue from a single patient. This method will facilitate the systematic analysis of microcircuits and allow unprecedented comparisons at the level of individuals.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/live-cell_sted_microscopy_of_mitochondrial_cristae_60612432/">
          <h4 class="font-weight-bold">Live-cell STED microscopy of mitochondrial cristae, bioRxiv, 2019-05-18</h4>
          <p>Mitochondria are highly dynamic organelles that exhibit a complex inner architecture. They exhibit a smooth outer membrane and a highly convoluted inner membrane that forms invaginations called cristae. Imaging cristae in living cells poses a formidable challenge for light microscopy. Relying on a cell line stably expressing the mitochondrial protein Cox8A fused to the SNAP-tag and using STED (stimulated emission depletion) super-resolution microscopy, we demonstrate the visualization of cristae dynamics in cultivated human cells.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/2019/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/2019/page/55/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/52/" class="page-link">52</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/53/" class="page-link">53</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/54/" class="page-link">54</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/55/" class="page-link">55</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/2019/page/56/" class="page-link">56</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/57/" class="page-link">57</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/58/" class="page-link">58</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/59/" class="page-link">59</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/60/" class="page-link">60</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/2019/page/57/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/2019/page/135/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
