<!DOCTYPE html>
<html><head>
  
  <title>Reports | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/the_whale_shark_genome_reveals_how_genomic_and_physiological_properties_scale_with_body_size_49668029/">
          <h4 class="font-weight-bold">The whale shark genome reveals how genomic and physiological properties scale with body size, bioRxiv, 2018-10-14</h4>
          <p>AbstractThe endangered whale shark (Rhincodon typus) is the largest fish on Earth and is a long-lived member of the ancient Elasmobranchii clade. To characterize the relationship between genome features and biological traits, we sequenced and assembled the genome of the whale shark and compared its genomic and physiological features to those of 81 animals and yeast. We examined scaling relationships between body size, temperature, metabolic rates, and genomic features and found both general correlations across the animal kingdom and features specific to the whale shark genome. Among animals, increased lifespan is positively correlated to body size and metabolic rate. Several genomic features also significantly correlated with body size, including intron and gene length. Our large-scale comparative genomic analysis uncovered general features of metazoan genome architecture GC content and codon adaptation index are negatively correlated, and neural connectivity genes are longer than average genes in most genomes. Focusing on the whale shark genome, we identified multiple features that significantly correlate with lifespan. Among these were very long gene length, due to large introns highly enriched in repetitive elements such as CR1-like LINEs, and considerably longer neural genes of several types, including connectivity, activity, and neurodegeneration genes. The whale sharkâ€™s genome had an expansion of gene families related to fatty acid metabolism and neurogenesis, with the slowest evolutionary rate observed in vertebrates to date. Our comparative genomics approach uncovered multiple genetic features associated with body size, metabolic rate, and lifespan, and showed that the whale shark is a promising model for studies of neural architecture and lifespan.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/a_framework_for_intelligence_and_cortical_function_based_on_grid_cells_in_the_neocortex_49622119/">
          <h4 class="font-weight-bold">A Framework for Intelligence and Cortical Function Based on Grid Cells in the Neocortex, bioRxiv, 2018-10-13</h4>
          <p>AbstractHow the neocortex works is a mystery. In this paper we propose a novel framework for understanding its function. Grid cells are neurons in the entorhinal cortex that represent the location of an animal in its environment. Recent evidence suggests that grid cell-like neurons may also be present in the neocortex. We propose that grid cells exist throughout the neocortex, in every region and in every cortical column. They define a location-based framework for how the neocortex functions. Whereas grid cells in the entorhinal cortex represent the location of one thing, the body relative to its environment, we propose that cortical grid cells simultaneously represent the location of many things. Cortical columns in somatosensory cortex track the location of tactile features relative to the object being touched and cortical columns in visual cortex track the location of visual features relative to the object being viewed. We propose that mechanisms in the entorhinal cortex and hippocampus that evolved for learning the structure of environments are now used by the neocortex to learn the structure of objects. Having a representation of location in each cortical column suggests mechanisms for how the neocortex represents object compositionality and object behaviors. It leads to the hypothesis that every part of the neocortex learns complete models of objects and that there are many models of each object distributed throughout the neocortex. The similarity of circuitry observed in all cortical regions is strong evidence that even high-level cognitive tasks are learned and represented in a location-based framework.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/centrosome-nuclear_envelope_tethering_and_microtubule_motor-based_pulling_forces_collaborate_in_centrosome_positioning_during_mitotic_entry_49606755/">
          <h4 class="font-weight-bold">Centrosome-nuclear envelope tethering and microtubule motor-based pulling forces collaborate in centrosome positioning during mitotic entry, bioRxiv, 2018-10-13</h4>
          <p>Centrosome positioning relative to the nucleus and cell shape is highly regulated across cell types, during cell migration and during spindle formation in cell division. Across most sexually reproducing animals, centrosomes are provided to the oocyte through fertilization and must be positioned properly to establish the zygotic mitotic spindle. How centrosomes are positioned in space and time through the concerted action of key mitotic entry biochemical regulators including Protein Phosphatase 2A (PP2A-B55SUR-6), biophysical regulators including Dynein and the nuclear lamina is unclear. Here, we uncover a role for PP2A-B55SUR-6 in regulating centrosome positioning. Mechanistically, PP2A-B55SUR-6 regulates nuclear size prior to mitotic entry, in turn affecting nuclear envelope-based Dynein density and motor capacity. Using computational simulations, PP2A-B55 SUR-6 regulation of nuclear size and nuclear envelope Dynein density were both predicted to be required for proper centrosome positioning. Conversely, compromising nuclear lamina integrity led to centrosome detachment from the nuclear envelope and migration defects. Removal of PP2A-B55SUR-6 and the nuclear lamina simultaneously further disrupted centrosome positioning, leading to unseparated centrosome pairs dissociated from the nuclear envelope. Taken together, we propose a model in which centrosomes migrate and are positioned through the concerted action of nuclear envelope-based Dynein pulling forces and cen-trosome-nuclear envelope tethering.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/estimation_of_allele-specific_fitness_effects_across_human_protein-coding_sequences_and_implications_for_disease_49612069/">
          <h4 class="font-weight-bold">Estimation of allele-specific fitness effects across human protein-coding sequences and implications for disease, bioRxiv, 2018-10-13</h4>
          <p>AbstractA central challenge in human genomics is to understand the cellular, evolutionary, and clinical significance of genetic variants. Here we introduce a unified population-genetic and machine-learning model, called Linear Allele-Specific Selection InferencE (LASSIE), for estimating the fitness effects of all potential single-nucleotide variants, based on polymorphism data and predictive genomic features. We applied LASSIE to 51 high-coverage genome sequences annotated with 33 genomic features, and constructed a map of allele-specific selection coefficients across all protein-coding sequences in the human genome. We show that this map is informative about both human evolution and disease.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/in_situ_and_high-resolution_cryo-em_structure_of_the_type_vi_secretion_membrane_complex_49625246/">
          <h4 class="font-weight-bold">In situ and high-resolution Cryo-EM structure of the Type VI secretion membrane complex, bioRxiv, 2018-10-13</h4>
          <p>AbstractBacteria have evolved macromolecular machineries that secrete effectors and toxins to survive and thrive in diverse environments. The type VI secretion system (T6SS) is a contractile machine that is related to Myoviridae phages. The T6SS is composed of a baseplate that contains a spike onto which an inner tube is built, surrounded by a contractile sheath. Unlike phages that are released to and act in the extracellular medium, the T6SS is an intracellular machine inserted in the bacterial membranes by a trans-envelope complex. This membrane complex (MC) comprises three proteins TssJ, TssL and TssM. We previously reported the low-resolution negative stain electron microscopy structure of the enteroaggregative Escherichia coli MC and proposed a rotational 5-fold symmetry with a TssJTssLTssM stoichiometry of 222. Here, cryo-electron tomography analysis of the T6SS MC confirmed the 5-fold symmetry in situ and identified the regions of the structure that insert into the bacterial membranes. A high resolution model obtained by single particle cryo-electron microscopy reveals its global architecture and highlights new features five additional copies of TssJ, yielding a TssJTssLTssM stoichiometry of 322, a 11-residue loop in TssM, protruding inside the lumen of the MC and constituting a functionally important periplasmic gate, and hinge regions. Based on these data, we revisit the model on the mechanism of action of the MC during T6SS assembly and function.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/measuring_the_average_power_of_neural_oscillations_49625253/">
          <h4 class="font-weight-bold">Measuring the average power of neural oscillations, bioRxiv, 2018-10-13</h4>
          <p>AbstractBackgroundNeural oscillations are often quantified as average power relative to a cognitive, perceptual, andor behavioral task. This is commonly done using Fourier-based techniques, such as Welchâ€™s method for estimating the power spectral density, andor by estimating narrowband oscillatory power across trials, conditions, andor groups. The core assumption underlying these approaches is that the mean is an appropriate measure of central tendency. Despite the importance of this assumption, it has not been rigorously tested.New methodWe introduce extensions of common approaches that are better suited for the physiological reality of how neural oscillations often manifest as nonstationary, high-power bursts, rather than sustained rhythms. Log-transforming, or taking the median power, significantly reduces erroneously inflated power estimates.ResultsAnalyzing 101 participantsâ€™ worth of human electrophysiology, totaling 3,560 channels and over 40 hours data, we show that, in all cases examined, spectral power is not Gaussian distributed. This is true even when oscillations are prominent and sustained, such as visual cortical alpha. Power across time, at every frequency, is characterized by a substantial long tail, which implies that estimates of average power are skewed toward large, infrequent high-power oscillatory bursts.Comparison with existing methodsIn a simulated event-related experiment we show how introducing just a few high-power oscillatory bursts, as seen in real data, can, perhaps erroneously, cause significant differences between conditions using traditional methods. These erroneous effects are substantially reduced with our new methods.ConclusionsThese results call into question the validity of common statistical practices in neural oscillation research.Highlights&lt;jatslist list-type=bullet&gt;&lt;jatslist-item&gt;Analyses of oscillatory power often assume power is normally distributed.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Analyzing &amp;gt;40 hours of human MEEG and ECoG, we show that in all cases it is not.&lt;jatslist-item&gt;&lt;jatslist-item&gt;This effect is demonstrated in simple simulation of an event-related task.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Overinflated power estimates are reduced via log-transformation or median power.&lt;jatslist-item&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/reports/" rel="first" class="page-link">Â« First</a></li>
        

        
        <li class="page-item"><a href="/reports/page/112/" rel="prev" class="page-link">â€¹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/reports/page/109/" class="page-link">109</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/110/" class="page-link">110</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/111/" class="page-link">111</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/112/" class="page-link">112</a></li>
            
        
             
            <li class="page-item active"><a href="/reports/page/113/" class="page-link">113</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/114/" class="page-link">114</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/115/" class="page-link">115</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/116/" class="page-link">116</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/117/" class="page-link">117</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/reports/page/114/" rel="next" class="page-link">Next â€º</a></li>
        

        
        
        <li class="page-item"><a href="/reports/page/172/" rel="last" class="page-link">Last Â»</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
