<!DOCTYPE html>
<html><head>
  
  <title>100 200 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/spatial_metabolomics_of_in_situ_host-microbe_interactions_55762881/">
          <h4 class="font-weight-bold">Spatial metabolomics of in situ, host-microbe interactions, bioRxiv, 2019-02-20</h4>
          <p>Spatial metabolomics describes the location and chemistry of small molecules involved in metabolic phenotypes, defense molecules and chemical interactions in natural communities. Most current techniques are unable to spatially link the genotype and metabolic phenotype of microorganisms in situ at a scale relevant to microbial interactions. Here, we present a spatial metabolomics pipeline (metaFISH) that combines fluorescence in situ hybridization (FISH) microscopy and high-resolution atmospheric pressure mass spectrometry imaging (AP-MALDI-MSI) to image host-microbe symbioses and their metabolic interactions. metaFISH aligns and integrates metabolite and fluorescent images at the micrometer-scale for a spatial assignment of host and symbiont metabolites on the same tissue section. To illustrate the advantages of metaFISH, we mapped the spatial metabolome of a deep-sea mussel and its intracellular symbiotic bacteria at the scale of individual epithelial host cells. Our analytical pipeline revealed metabolic adaptations of the epithelial cells to the intracellular symbionts, a variation in metabolic phenotypes in one symbiont type, and novel symbiosis metabolites. metaFISH provides a culture-independent approach to link metabolic phenotypes to community members in situ - a powerful tool for microbiologists across fields.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/hospital_use_of_antibiotics_as_the_main_driver_of_infections_with_antibiotic-resistant_bacteria_-_a_reanalysis_of_recent_data_from_the_european_union_55695720/">
          <h4 class="font-weight-bold">Hospital use of antibiotics as the main driver of infections with antibiotic-resistant bacteria - a reanalysis of recent data from the European Union, bioRxiv, 2019-02-19</h4>
          <p>Antimicrobial resistance in bacteria causes significant morbidity worldwide. The development and acquisition of resistance to antibiotics is believed to primarily develop under the selective pressure of widespread antibiotic use in humans, however antimicrobial usage in livestock has been proposed as additional, if not principal, driver of antibiotic resistance. In this work, we correlate recent data from the European Union on antibiotic resistance rates with data on antibiotic usage in the primary care and hospital sector and data on veterinary antimicrobial consumption across the individual member states. We quantify the strength of these different potential drivers of antimicrobial resistance in order to compare their biological importance. We found that the correlation between antibiotic use in the hospital sector and antibiotic resistance rates is significantly higher than the correlation between resistance rates and any of the other two predictors. This suggests increased antibiotic use in hospitals as the main driver of the development of antibiotic resistances and necessitates further research on and a re-evaluation of the risks associated with antibiotic use in human and veterinary medicine.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/hospital_use_of_antibiotics_as_the_main_driver_of_infections_with_antibiotic-resistant_bacteria__a_reanalysis_of_recent_data_from_the_european_union_55695720/">
          <h4 class="font-weight-bold">Hospital use of antibiotics as the main driver of infections with antibiotic-resistant bacteria – a reanalysis of recent data from the European Union, bioRxiv, 2019-02-19</h4>
          <p>AbstractAntimicrobial resistance in bacteria causes significant morbidity worldwide. The development and acquisition of resistance to antibiotics is believed to primarily develop under the selective pressure of widespread antibiotic use in humans, however antimicrobial usage in livestock has been proposed as additional, if not principal, driver of antibiotic resistance. In this work, we correlate recent data from the European Union on antibiotic resistance rates with data on antibiotic usage in the primary care and hospital sector and data on veterinary antimicrobial consumption across the individual member states. We quantify the strength of these different potential drivers of antimicrobial resistance in order to compare their biological importance. We found that the correlation between antibiotic use in the hospital sector and antibiotic resistance rates is significantly higher than the correlation between resistance rates and any of the other two predictors. This suggests increased antibiotic use in hospitals as the main driver of the development of antibiotic resistances and necessitates further research on and a re-evaluation of the risks associated with antibiotic use in human and veterinary medicine.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/self-inactivating_rabies_viruses_are_just_first-generation_%CE%B4g_rabies_viruses_55698207/">
          <h4 class="font-weight-bold">Self-inactivating rabies viruses are just first-generation, ΔG rabies viruses, bioRxiv, 2019-02-19</h4>
          <p>A recent article in Cell reported a new form of modified rabies virus that was apparently capable of labeling neurons without adverse effects on neuronal physiology and circuit function. These self-inactivating rabies (SiR) viruses differed from the widely-used first-generation deletion-mutant (ΔG) rabies viruses only by the addition of a destabilization domain to the viral nucleoprotein. However, we observed that the transsynaptic tracing results from that article were inconsistent with the logic described in it, and we hypothesized that the viruses used were actually mutants that had lost the intended modification to the nucleoprotein. We obtained samples of two SiR viruses from the authors and show here that, in both SiR-CRE and SiR-FLPo, the great majority of viral particles were indeed mutants that had lost the intended modification and were therefore just first-generation, ΔG rabies viruses. We also found that SiR-CRE killed 70% of infected neurons in vivo within two weeks. We have shown elsewhere that a ΔG rabies virus encoding Cre can leave a large percentage of labeled neurons alive; we presume that Ciabatti et al. found such remaining neurons at long survival times and mistakenly concluded that they had developed a nontoxic version of rabies virus. Here we have analyzed only the two samples that were sent to MIT by Ciabatti et al., and these may not be from the same batches that were used for their paper. However, 1) both of the two viruses that we analyzed had independently lost the intended modification, 2) the mutations in the two samples were genetically quite distinct from each other yet in both cases caused the same result total or near-total loss of the C-terminal modification, and 3) the mutations that we found in these two virus samples perfectly explain the otherwise-paradoxical transsynaptic tracing results from Ciabatti et al.&#39;s paper. We suggest that the SiR strategy, or any other such attempt to attenuate a virus by addition rather than deletion, is an inherently unstable approach that can easily be evaded by mutation, as it was in this case.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/self-inactivating_rabies_viruses_are_just_first-generation_%CE%B4g_rabies_viruses_55698207/">
          <h4 class="font-weight-bold">“Self-inactivating” rabies viruses are just first-generation, ΔG rabies viruses, bioRxiv, 2019-02-19</h4>
          <p>SUMMARYA recent article in Cell reported a new form of modified rabies virus that was apparently capable of labeling neurons “without adverse effects on neuronal physiology and circuit function”. These “self-inactivating” rabies (“SiR”) viruses differed from the widely-used first-generation deletion-mutant (ΔG) rabies viruses only by the addition of a destabilization domain to the viral nucleoprotein. However, we observed that the transsynaptic tracing results from that article were inconsistent with the logic described in it, and we hypothesized that the viruses used were actually mutants that had lost the intended modification to the nucleoprotein. We obtained samples of two SiR viruses from the authors and show here that, in both “SiR-CRE” and “SiR-FLPo”, the great majority of viral particles were indeed mutants that had lost the intended modification and were therefore just first-generation, ΔG rabies viruses. We also found that SiR-CRE killed 70% of infected neurons in vivo within two weeks. We have shown elsewhere that a ΔG rabies virus encoding Cre can leave a large percentage of labeled neurons alive; we presume that Ciabatti et al. found such remaining neurons at long survival times and mistakenly concluded that they had developed a nontoxic version of rabies virus. Here we have analyzed only the two samples that were sent to MIT by Ciabatti et al., and these may not be from the same batches that were used for their paper. However, 1) both of the two viruses that we analyzed had independently lost the intended modification, 2) the mutations in the two samples were genetically quite distinct from each other yet in both cases caused the same result total or near-total loss of the C-terminal modification, and 3) the mutations that we found in these two virus samples perfectly explain the otherwise-paradoxical transsynaptic tracing results from Ciabatti et al.’s paper. We suggest that the SiR strategy, or any other such attempt to attenuate a virus by addition rather than deletion, is an inherently unstable approach that can easily be evaded by mutation, as it was in this case.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/comparative_performance_of_the_bgi_and_illumina_sequencing_technology_for_single-cell_rna-sequencing_55642362/">
          <h4 class="font-weight-bold">Comparative performance of the BGI and Illumina sequencing technology for single-cell RNA-sequencing, bioRxiv, 2019-02-17</h4>
          <p>The libraries generated by high-throughput single cell RNA-sequencing platforms such as the Chromium from 10x Genomics require considerable amounts of sequencing, typically due to the large number of cells. The ability to use this data to address biological questions is directly impacted by the quality of the sequence data. Here we have compared the performance of the Illumina NextSeq 500 and NovaSeq 6000 against the BGI MGISEQ-2000 platform using identical Single Cell libraries consisting of over 70,000 cells. Our results demonstrate a highly comparable performance between the NovaSeq 6000 and MGISEQ-2000 in sequencing quality, and cell, UMI, and gene detection. However, compared with the NextSeq 500, the MGISEQ- 2000 platform performs consistently better, identifying more cells, genes, and UMIs at equalised read depth. We were able to call an additional 1,065,659 SNPs from sequence data generated by the BGI platform, enabling an additional 14% of cells to be assigned to the correct donor from a multiplexed library. However, both the NextSeq 500 and MGISEQ-2000 detected similar frequencies of gRNAs from a pooled CRISPR single cell screen. Our study provides a benchmark for high capacity sequencing platforms applied to high-throughput single cell RNA-seq libraries.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/47/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/44/" class="page-link">44</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/45/" class="page-link">45</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/46/" class="page-link">46</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/47/" class="page-link">47</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/48/" class="page-link">48</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/49/" class="page-link">49</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/50/" class="page-link">50</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/51/" class="page-link">51</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/52/" class="page-link">52</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/49/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/126/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
