<!DOCTYPE html>
<html><head>
  
  <title>2019 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genetic_structure_in_the_paternal_lineages_of_south_east_spain_revealed_by_the_analysis_of_17_y-strs_57947784/">
          <h4 class="font-weight-bold">Genetic structure in the paternal lineages of South East Spain revealed by the analysis of 17 Y-STRs, Scientific Reports, 2019-03-26</h4>
          <p>The genetic data of 17 Y chromosome short tandem repeats in 146 unrelated donor residents in the provinces of Granada, Málaga, and Almería (GMA) were analyzed to determine the genetic legacy of the male inhabitants of the former Kingdom of Granada. A total of 139 unique haplotypes were identified. Observed allele frequencies and haplogroup frequencies were also analyzed. By AMOVA and STRUCTURE analysis, the populations of the 3 provinces could be treated genetically as a single population. The most frequent haplogroup was R1b1b2 (58.22%). By network analysis of all individuals, we observed a distribution according to haplogroup assignment. To improve the characterization of GMA population, it was compared with those of North Africa, the Iberian Peninsula, and southern Europe. In our analysis of allele frequencies and genetic distances, the GMA population lay within the Spanish population group. Further, in the STRUCTURE analysis, there was no African component in the GMA population, confirming that, based on our genetic markers, the GMA population does not reflect any male genetic influence of the North African people. The presence of African haplogroups in the GMA population is irrelevant when their frequency is compared with those in other European populations.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/scientific%20reports">scientific reports</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/recovery_of_trait_heritability_from_whole_genome_sequence_data_57697220/">
          <h4 class="font-weight-bold">Recovery of trait heritability from whole genome sequence data, bioRxiv, 2019-03-26</h4>
          <p>AbstractHeritability, the proportion of phenotypic variance explained by genetic factors, can be estimated from pedigree data 1, but such estimates are uninformative with respect to the underlying genetic architecture. Analyses of data from genome-wide association studies (GWAS) on unrelated individuals have shown that for human traits and disease, approximately one-third to two-thirds of heritability is captured by common SNPs 2–5. It is not known whether the remaining heritability is due to the imperfect tagging of causal variants by common SNPs, in particular if the causal variants are rare, or other reasons such as over-estimation of heritability from pedigree data. Here we show that pedigree heritability for height and body mass index (BMI) appears to be fully recovered from whole-genome sequence (WGS) data on 21,620 unrelated individuals of European ancestry. We assigned 47.1 million genetic variants to groups based upon their minor allele frequencies (MAF) and linkage disequilibrium (LD) with variants nearby, and estimated and partitioned variation accordingly. The estimated heritability was 0.79 (SE 0.09) for height and 0.40 (SE 0.09) for BMI, consistent with pedigree estimates. Low-MAF variants in low LD with neighbouring variants were enriched for heritability, to a greater extent for protein altering variants, consistent with negative selection thereon. Cumulatively variants in the MAF range of 0.0001 to 0.1 explained 0.54 (SE 0.05) and 0.51 (SE 0.11) of heritability for height and BMI, respectively. Our results imply that the still missing heritability of complex traits and disease is accounted for by rare variants, in particular those in regions of low LD.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/unified_rational_protein_engineering_with_sequence-only_deep_representation_learning_57770558/">
          <h4 class="font-weight-bold">Unified rational protein engineering with sequence-only deep representation learning, bioRxiv, 2019-03-26</h4>
          <p>AbstractRational protein engineering requires a holistic understanding of protein function. Here, we apply deep learning to unlabelled amino acid sequences to distill the fundamental features of a protein into a statistical representation that is semantically rich and structurally, evolutionarily, and biophysically grounded. We show that the simplest models built on top of this unified representation (UniRep) are broadly applicable and generalize to unseen regions of sequence space. Our data-driven approach reaches near state-of-the-art or superior performance predicting stability of natural and de novo designed proteins as well as quantitative function of molecularly diverse mutants. UniRep further enables two orders of magnitude cost savings in a protein engineering task. We conclude UniRep is a versatile protein summary that can be applied across protein engineering informatics.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/synthetic-biology">synthetic-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_tissue-like_platform_for_studying_engineered_quiescent_human_t-cells_interactions_with_dendritic_cells_57639426/">
          <h4 class="font-weight-bold">A tissue-like platform for studying engineered quiescent human T-cells’ interactions with dendritic cells, bioRxiv, 2019-03-25</h4>
          <p>AbstractResearch in the field of human immunology is restricted by the lack of a system that reconstitutes the in-situ activation dynamics of quiescent human antigen-specific T-cells interacting with dendritic cells. Here we report a tissue-like system that recapitulates the dynamics of engineered primary human immune cell. Our approach facilitates real-time single cell manipulations, tracking of interactions and functional responses complemented by population-based measurements of cytokines, activation status and proliferation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/bacterial_biodiversity_drives_the_evolution_of_crispr-based_phage_resistance_in_pseudomonas_aeruginosa_57638240/">
          <h4 class="font-weight-bold">Bacterial biodiversity drives the evolution of CRISPR-based phage resistance in Pseudomonas aeruginosa, bioRxiv, 2019-03-25</h4>
          <p>Approximately half of all bacterial species encode CRISPR-Cas adaptive immune systems1, which provide immunological memory by inserting short DNA sequences from phage and other parasitic DNA elements into CRISPR loci on the host genome2. Whereas CRISPR loci evolve rapidly in natural environments3, bacterial species typically evolve phage resistance by the mutation or loss of phage receptors under laboratory conditions4,5. Here, we report how this discrepancy may in part be explained by differences in the biotic complexity of in vitro and natural environments6,7. Specifically, using the opportunistic pathogen Pseudomonas aeruginosa and its phage DMS3vir, we show that coexistence with other human pathogens amplifies the fitness trade-offs associated with phage receptor mutation, and therefore tips the balance in favour of CRISPR-based resistance evolution. We also demonstrate that this has important knock-on effects for P. aeruginosa virulence, which became attenuated only if the bacteria evolved surface-based resistance. Our data reveal that the biotic complexity of microbial communities in natural environments is an important driver of the evolution of CRISPR-Cas adaptive immunity, with key implications for bacterial fitness and virulence.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/extraordinary_claims_require_extraordinary_evidence_in_the_case_of_asserted_mtdna_biparental_inheritance_57685203/">
          <h4 class="font-weight-bold">Extraordinary claims require extraordinary evidence in the case of asserted mtDNA biparental inheritance, bioRxiv, 2019-03-25</h4>
          <p>AbstractA breakthrough article published in PNAS by Luo et al. (2018) challenges a central dogma in biology which states that the mitochondrial DNA (mtDNA) is inherited exclusively from the mother. By sequencing the mitogenomes of several members of three independent families, the authors inferred an unprecedented pattern of biparental inheritance that requires the participation of an autosomal nuclear factor in the molecular process. However, a comprehensive analysis of their data reveals a number of issues that must be carefully addressed before challenging the current paradigm. Unfortunately, the methods section lacks any description of sample management, validation of their results in independent laboratories was deficient, and the reported findings have been observed at a frequency at complete variance with established evidence. Moreover, the remarkably high (and unusually homogeneous) levels of heteroplasmy reported can be readily detected using classical techniques for DNA sequencing. By reassessing the raw sequencing data with an alternative computational pipeline, we report strong correlation to the NextGENe results provided by the authors on a per sample base. However, the sequencing replicates from the same donors show aberrations in the variants detected that need further investigation to exclude contributions from other sources or methodological artifacts. Finally, applying the principle of reductio ad absurdum, we demonstrate that the nuclear factor invoked by the authors would need to be extraordinarily complex and precise in order to preclude linear accumulation of mtDNA lineages across generations. We discuss alternate scenarios that explain findings of the same nature as reported by Luo et al., in the context of in-vitro fertilization and therapeutic mtDNA replacement ooplasmic transplantation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/2019/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/2019/page/83/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/80/" class="page-link">80</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/81/" class="page-link">81</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/82/" class="page-link">82</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/83/" class="page-link">83</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/2019/page/84/" class="page-link">84</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/85/" class="page-link">85</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/86/" class="page-link">86</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/87/" class="page-link">87</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/88/" class="page-link">88</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/2019/page/85/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/2019/page/135/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
