<!DOCTYPE html>
<html><head>
  
  <title>2019 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/tree_lab_portable_genomics_for_early_detection_of_plant_viruses_and_pests_in_sub-saharan_africa_63810678/">
          <h4 class="font-weight-bold">Tree Lab Portable genomics for early detection of plant viruses and pests in Sub-Saharan Africa, bioRxiv, 2019-07-20</h4>
          <p>AbstractIn this case study we successfully teamed the PDQeX DNA purification technology developed by MicroGEM, New Zealand, with the MinION and MinIT mobile sequencing devices developed by Oxford Nanopore Technologies to produce an effective point-of-need field diagnostic system. The PDQeX extracts DNA using a cocktail of thermophilic proteinases and cell wall degrading enzymes, thermo-responsive extractor cartridges and a temperature control unit. This single-step closed system delivers purified DNA with no cross contamination. The MinIT is a newly released data processing unit that converts MinION raw signal output into base called data locally in real time, removing the need for high specification computers and large file transfers from the field. All three devices are battery powered with an exceptionally small footprint that facilitates transport and set up.To evaluate and validate capability of the system for unbiased pathogen identification by realtime sequencing in a farmer’s field setting, we analysed samples collected from cassava plants grown by subsistence farmers in three sub-Sahara African countries (Tanzania, Uganda and Kenya). A range of viral pathogens, all with similar symptoms, greatly reduce yield or completely destroy cassava crops. 800 million people worldwide depend on cassava for food and yearly income, and viral diseases are a significant constraint on its production (&lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpscassavavirusactionproject.com&gt;httpscassavavirusactionproject.com&lt;jatsext-link&gt;). Early pathogen detection at a molecular level has great potential to rescue crops within a single growing season by providing results that inform decisions on disease management, use of appropriate virus resistant or replacement planting.This case study presented conditions of working in-field with limited or no access to mains power, laboratory infrastructure, internet connectivity and highly variable ambient temperature. An additional challenge is that, generally, plant material contains inhibitors of downstream molecular processes making effective DNA purification critical. We successfully undertook real-time on-farm genome sequencing of samples collected from cassava plants on three farms, one in each country. Cassava mosaic begomoviruses were detected by sequencing leaf, stem, tuber and insect samples. The entire process, from arrival on farm to diagnosis including sample collection, processing and provisional sequencing results was complete in under 4 hours. The need for accurate, rapid and on-site diagnosis grows as globalized human activity accelerates. This technical breakthrough has applications that are relevant to human and animal health, environmental management and conservation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/comparison_of_adopted_and_non-adopted_individuals_reveals_gene-environment_interplay_for_education_in_the_uk_biobank_63738615/">
          <h4 class="font-weight-bold">Comparison of adopted and non-adopted individuals reveals gene-environment interplay for education in the UK Biobank, bioRxiv, 2019-07-19</h4>
          <p>AbstractIndividual-level polygenic scores can now explain ∼10% of the variation in number of years of completed education. However, associations between polygenic scores and education capture not only genetic propensity but information about the environment that individuals are exposed to. This is because individuals passively inherit effects of parental genotypes, since their parents typically also provide the rearing environment. In other words, the strong correlation between offspring and parent genotypes results in an association between the offspring genotypes and the rearing environment. This is termed passive gene-environment correlation. We present an approach to test for the extent of passive gene-environment correlation for education without requiring intergenerational data. Specifically, we use information from 6311 individuals in the UK Biobank who were adopted in childhood to compare genetic influence on education between adoptees and non-adopted individuals. Adoptees’ rearing environments are less correlated with their genotypes, because they do not share genes with their adoptive parents. We find that polygenic scores are twice as predictive of years of education in non-adopted individuals compared to adoptees (R2= 0.074 vs 0.037, difference test p= 8.23 × 10−24). We provide another kind of evidence for the influence of parental behaviour on offspring education individuals in the lowest decile of education polygenic score attain significantly more education if they are adopted, possibly due to educationally supportive adoptive environments. Overall, these results suggest that genetic influences on education are mediated via the home environment. As such, polygenic prediction of educational attainment represents gene-environment correlations just as much as it represents direct genetic effects.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genome_sequence_of_the_cluster_root_forming_white_lupin_63775319/">
          <h4 class="font-weight-bold">Genome sequence of the cluster root forming white lupin, bioRxiv, 2019-07-19</h4>
          <p>White lupin (Lupinus albus L.) is a legume that produces seeds recognized for their high protein content and good nutritional value (lowest glycemic index of all grains, high dietary fiber content, and zero gluten or starch)1–5. White lupin can form nitrogen-fixing nodules but has lost the ability to form mycorrhizal symbiosis with fungi6. Nevertheless, its root system is well adapted to poor soils it produces cluster roots, constituted of dozens of determinate lateral roots that improve soil exploration and phosphate remobilization7. As phosphate is a limited resource that comes from rock reserves8, the production of cluster roots is a trait of interest to improve fertilizers efficiency. Using long reads sequencing technologies, we provide a high-quality genome sequence of a modern variety of white lupin (2n=50, 451 Mb), as well as de novo assemblies of a landrace and a wild relative. We describe how domestication impacted soil exploration capacity through the early establishment of lateral and cluster roots. We identify the APETALA2 transcription factor LaPUCHI-1, homolog of the Arabidopsis morphogenesis coordinator9, as a potential regulator of this trait. Our high-quality genome and companion genomic and transcriptomic resources enable the development of modern breeding strategies to increase and stabilize yield and to develop new varieties with reduced allergenic properties (caused by conglutins10), which would favor the deployment of this promising culture.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/h3k4me3_is_neither_instructive_for_nor_informed_by_transcription_63778060/">
          <h4 class="font-weight-bold">H3K4me3 is neither instructive for, nor informed by, transcription, bioRxiv, 2019-07-19</h4>
          <p>AbstractH3K4me3 is a near-universal histone modification found predominantly at the 5’ region of genes, with a well-documented association with gene activity. H3K4me3 has been ascribed roles as both an instructor of gene expression and also a downstream consequence of expression, yet neither has been convincingly proven on a genome-wide scale. Here we test these relationships using a combination of bioinformatics, modelling and experimental data from budding yeast in which the levels of H3K4me3 have been massively ablated. We find that loss of H3K4me3 has no effect on the levels of nascent transcription or transcript in the population. Moreover, we observe no change in the rates of transcription initiation, elongation, mRNA export or turnover, or in protein levels, or cell-to-cell variation of mRNA. Loss of H3K4me3 also has no effect on the large changes in gene expression patterns that follow galactose induction. Conversely, loss of RNA polymerase from the nucleus has no effect on the pattern of H3K4me3 deposition and little effect on its levels, despite much larger changes to other chromatin features. Furthermore, large genome-wide changes in transcription, both in response to environmental stress and during metabolic cycling, are not accompanied by corresponding changes in H3K4me3. Thus, despite the correlation between H3K4me3 and gene activity, neither appear to be necessary to maintain levels of the other, nor to influence their changes in response to environmental stimuli. When we compare gene classes with very different levels of H3K4me3 but highly similar transcription levels we find that H3K4me3-marked genes are those whose expression is unresponsive to environmental changes, and that their histones are less acetylated and dynamically turned-over. Constitutive genes are generally well-expressed, which may alone explain the correlation between H3K4me3 and gene expression, while the biological role of H3K4me3 may have more to do with this distinction in gene class.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/orange_a_crisprcas9-based_genome_editing_toolbox_for_epitope_tagging_of_endogenous_proteins_in_neurons_63756313/">
          <h4 class="font-weight-bold">ORANGE A CRISPRCas9-based genome editing toolbox for epitope tagging of endogenous proteins in neurons, bioRxiv, 2019-07-19</h4>
          <p>ABSTRACTThe correct subcellular distribution of protein complexes establishes the complex morphology of neurons and is fundamental to their functioning. Thus, determining the dynamic distribution of proteins is essential to understand neuronal processes. Fluorescence imaging, in particular super-resolution microscopy, has become invaluable to investigate subcellular protein distribution. However, these approaches suffer from the limited ability to efficiently and reliably label endogenous proteins. We developed ORANGE an Open Resource for the Application of Neuronal Genome Editing, that mediates targeted genomic integration of fluorescent tags in neurons. This toolbox includes a knock-in library for in-depth investigation of endogenous protein distribution, and a detailed protocol explaining how knock-in can be developed for novel targets. In combination with super-resolution microscopy, ORANGE revealed the dynamic nanoscale organization of endogenous neuronal signaling molecules, synaptic scaffolding proteins, and neurotransmitter receptors. Thus, ORANGE enables quantitation of expression and distribution for virtually any protein in neurons at high resolution and will significantly further our understanding of neuronal cell biology.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/%CE%BCdamid_a_microfluidic_approach_for_imaging_and_sequencing_protein-dna_interactions_in_single_cells_63731175/">
          <h4 class="font-weight-bold">μDamID a microfluidic approach for imaging and sequencing protein-DNA interactions in single cells, bioRxiv, 2019-07-19</h4>
          <p>AbstractGenome regulation depends on carefully programmed protein-DNA interactions that maintain or alter gene expression states, often by influencing chromatin organization. Most studies of these interactions to date have relied on bulk methods, which in many systems cannot capture the dynamic single-cell nature of these interactions as they modulate cell states. One method allowing for sensitive single-cell mapping of protein-DNA interactions is DNA adenine methyltransferase identification (DamID), which records a protein’s DNA-binding history by methylating adenine bases in its vicinity, then selectively amplifies and sequences these methylated regions. These interaction sites can also be visualized using fluorescent proteins that bind to methyladenines. Here we combine these imaging and sequencing technologies in an integrated microfluidic platform (μDamID) that enables single-cell isolation, imaging, and sorting, followed by DamID. We apply this system to generate paired single-cell imaging and sequencing data from a human cell line, in which we map and validate interactions between DNA and nuclear lamina proteins, providing a measure of 3D chromatin organization and broad gene regulation patterns. μDamID provides the unique ability to compare paired imaging and sequencing data for each cell and between cells, enabling the joint analysis of the nuclear localization, sequence identity, and variability of protein-DNA interactions.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioengineering">bioengineering</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/2019/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/2019/page/28/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/25/" class="page-link">25</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/26/" class="page-link">26</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/27/" class="page-link">27</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/28/" class="page-link">28</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/2019/page/29/" class="page-link">29</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/30/" class="page-link">30</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/31/" class="page-link">31</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/32/" class="page-link">32</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/33/" class="page-link">33</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/2019/page/30/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/2019/page/135/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
