<!DOCTYPE html>
<html><head>
  
  <title>Evolutionary Biology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/synthesis_of_geological_and_comparative_phylogeographic_data_point_to_climate_not_mountain_uplift_as_driver_of_divergence_across_the_eastern_andean_cordillera_74062625/">
          <h4 class="font-weight-bold">Synthesis of geological and comparative phylogeographic data point to climate, not mountain uplift, as driver of divergence across the Eastern Andean Cordillera, bioRxiv, 2020-01-15</h4>
          <p>AbstractAimTo evaluate the potential role of the orogeny of the Eastern Cordillera (EC) of the Colombian Andes and the Mérida Andes (MA) of Venezuela as drivers of vicariance between populations of 37 tetrapod lineages co-distributed on both flanks, through geological reconstruction and comparative phylogeographic analyses.LocationNorthwestern South AmericaMethodsWe first reviewed and synthesized published geological data on the timing of uplift for the EC-MA. We then combined newly generated mitochondrial DNA sequence data with published datasets to create a comparative phylogeographic dataset for 37 independent tetrapod lineages. We reconstructed time-calibrated molecular phylogenies for each lineage under Bayesian inference to estimate divergence times between lineages located East and West of the Andes. We performed a comparative phylogeographic analysis of all lineages within each class of tetrapod using hierarchical approximate Bayesian computation (hABC) to test for synchronous vicariance across the EC-MA. To evaluate the potential role of life history in explaining variation in divergence times among lineages, we evaluated 13 general linear models (GLM) containing up to six variables each (maximum elevation, range size, body length, thermoregulation, type of dispersal, and taxonomic class).ResultsOur synthesis of geological evidence suggested that the EC-MA reached significant heights by 38–33 million years ago (Ma) along most of its length, and we reject the oft-cited date of 2–5 Ma. Based on mtDNA divergence from 37 lineages, however, the median estimated divergence time across the EC-MA was 3.26 Ma (SE = 2.84) in amphibians, 2.58 Ma (SE = 1.81) in birds, 2.99 Ma (SE = 4.68) in reptiles and 1.43 Ma (SE = 1.23) in mammals. Using Bayes Factors, the hypothesis for a single temporal divergence interval containing synchronous divergence events was supported for mammals and but not supported for amphibians, non-avian reptiles, or birds. Among the six life-history variables tested, only thermoregulation successfully explained variation in divergence times (minimum AICc, R2 0.10), with homeotherms showing more recent divergence relative to poikilotherms.Main conclusionsOur results reject the hypothesis of the rise Andean Cordillera as driver of vicariance of lowland population because divergence dates are too recent and too asynchronous. We discuss alternative explanations, including dispersal through mountain passes, and suggest that changes in the climatic conditions during the Pliocene and Pleistocene interacted with tetrapod physiology, promoting older divergences in amphibians and reptiles relative to mammals and birds on an already established orogen.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/recently_expanded_clonal_lineages_of_the_rice_blast_fungus_display_distinct_patterns_of_presenceabsence_of_effector_genes_73855112/">
          <h4 class="font-weight-bold">Recently expanded clonal lineages of the rice blast fungus display distinct patterns of presenceabsence of effector genes, bioRxiv, 2020-01-10</h4>
          <p>AbstractBackgroundUnderstanding the mechanisms and timescales of plant pathogen outbreaks requires a detailed genome-scale analysis of their population history. The fungus Magnaporthe (Syn. Pyricularia) oryzae —the causal agent of blast disease of cereals— is among the most destructive plant pathogens to world agriculture and a major threat to the production of rice, wheat and other cereals. Although M. oryzae is a multihost pathogen that infects more than 50 species of cereals and grasses, all rice-infecting isolates belong to a single genetically defined lineage. Here, we combined multiple genomics datasets to reconstruct the genetic history of the rice-infecting lineage of M. oryzae based on 131 isolates from 21 countries.ResultsThe global population of the rice blast fungus consists of a diverse set of individuals and three well-defined genetic groups. Multiple population genetic tests revealed that the rice-infecting lineage of the blast fungus probably originated from a recombining diverse group in South East Asia followed by three independent clonal expansions that took place over the last ∼200 years. Patterns of allele sharing identified a subpopulation from the recombining diverse group that introgressed with one of the clonal lineages before its global expansion. Remarkably, the four genetic lineages of the rice blast fungus vary in the number and patterns of presenceabsence of candidate effector genes. In particular, clonal lineages carry a reduced repertoire of effector genes compared with the diverse group, and specific combinations of effector presenceabsence define each of the pandemic clonal lineages.ConclusionsOur analyses reconstruct the genetic history of the rice-infecting lineage of M. oryzae revealing three clonal lineages associated with rice blast pandemics. Each of these lineages displays a specific pattern of presenceabsence of effector genes that may have shaped their adaptation to the rice host and their evolutionary history.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_history_of_measles_from_a_1912_genome_to_an_antique_origin_73413745/">
          <h4 class="font-weight-bold">The history of measles from a 1912 genome to an antique origin, bioRxiv, 2019-12-30</h4>
          <p>AbstractMany infectious diseases are thought to have emerged in humans after the Neolithic revolution. While it is broadly accepted that this also applies to measles, the exact date of emergence for this disease is controversial. Here, we sequenced the genome of a 1912 measles virus and used selection-aware molecular clock modeling to determine the divergence date of measles virus and rinderpest virus. This divergence date represents the earliest possible date for the establishment of measles in human populations. Our analyses show that the measles virus potentially arose as early as the 4th century BCE, rekindling the recently challenged hypothesis of an antique origin of this disease.One Sentence SummaryMeasles virus diverged from rinderpest virus in the 4th century BCE, which is compatible with an emergence of measles during Antiquity.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/quantitative_translation_of_dog-to-human_aging_by_conserved_remodeling_of_epigenetic_networks_69766930/">
          <h4 class="font-weight-bold">Quantitative translation of dog-to-human aging by conserved remodeling of epigenetic networks, bioRxiv, 2019-11-04</h4>
          <p>SUMMARYMammals progress through similar physiological stages during life, from early development to puberty, aging, and death. Yet, the extent to which this conserved physiology reflects conserved molecular events is unclear. Here, we map common epigenetic changes experienced by mammalian genomes as they age, focusing on evolutionary comparisons of humans to dogs, an emerging model of aging. Using targeted sequencing, we characterize the methylomes of 104 Labrador retrievers spanning a 16 year age range, achieving &amp;gt;150X coverage within mammalian syntenic blocks. Comparison with human methylomes reveals a nonlinear relationship which translates dog to human years, aligns the timing of major physiological milestones between the two species, and extends to mice. Conserved changes center on specific developmental gene networks which are sufficient to capture the effects of anti-aging interventions in multiple mammals. These results establish methylation not only as a diagnostic age readout but as a cross-species translator of physiological aging milestones.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/ancient_genomic_regulatory_blocks_are_a_major_source_for_gene_deserts_in_vertebrates_after_whole_genome_duplications_67129985/">
          <h4 class="font-weight-bold">Ancient genomic regulatory blocks are a major source for gene deserts in vertebrates after whole genome duplications, bioRxiv, 2019-09-26</h4>
          <p>AbstractWe investigated how the two rounds of whole genome duplication that occurred at the base of the vertebrate lineage have impacted ancient microsyntenic associations involving developmental regulators (known as genomic regulatory blocks, GRBs). We showed that the majority of GRBs present in the last common ancestor of chordates have been maintained as a single copy in humans. We found evidence that dismantling of the additional GRB copies occurred early in vertebrate evolution often through the differential retention of the regulatory gene but loss of the bystander gene’s exonic sequences. Despite the large evolutionary scale, the presence of duplicated highly conserved non-coding regions provided unambiguous proof for this scenario for dozens of ancient GRBs. Remarkably, the dismantling of ancient GRB duplicates has contributed to the creation of large gene deserts associated with regulatory genes in vertebrates, providing a widespread mechanism for the origin of these enigmatic genomic traits.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/single-cell-resolution_transcriptome_map_of_human_chimpanzee_bonobo_and_macaque_brains_66403986/">
          <h4 class="font-weight-bold">Single-cell-resolution transcriptome map of human, chimpanzee, bonobo, and macaque brains, bioRxiv, 2019-09-11</h4>
          <p>ABSTRACTIdentification of gene expression traits unique to the human brain sheds light on the mechanisms of human cognition. Here we searched for gene expression traits separating humans from other primates by analyzing 88,047 cell nuclei and 422 tissue samples representing 33 brain regions of humans, chimpanzees, bonobos, and macaques. We show that gene expression evolves rapidly within cell types, with more than two-thirds of cell type-specific differences not detected using conventional RNA sequencing of tissue samples. Neurons tend to evolve faster in all hominids, but non-neuronal cell types, such as astrocytes and oligodendrocyte progenitors, show more differences on the human lineage, including alterations of spatial distribution across neocortical layers.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        

        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/evolutionary-biology/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/evolutionary-biology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/evolutionary-biology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/evolutionary-biology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/evolutionary-biology/page/5/" class="page-link">5</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/evolutionary-biology/page/2/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/evolutionary-biology/page/20/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
