<!DOCTYPE html>
<html><head>
  
  <title>Synthetic Biology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/beditor_a_computational_workflow_for_designing_libraries_of_guide_rnas_for_crispr-mediated_base_editing_48898720/">
          <h4 class="font-weight-bold">beditor A computational workflow for designing libraries of guide RNAs for CRISPR-mediated base editing, bioRxiv, 2018-09-26</h4>
          <p>ABSTRACTCRISPR-mediated base editors have opened unique avenues for scar-free genome-wide mutagenesis. Here, we describe a comprehensive computational workflow called beditor that can be broadly adapted for designing guide RNA libraries with a range of CRISPR-mediated base editors, PAM recognition sequences and genomes of many species. Additionally, in order to assist users in selecting the best sets of guide RNAs for their experiments, a priori estimates, called beditor scores are calculated. These beditor scores are intended to select guide RNAs that conform to requirements for optimal base editing the editable base falls within maximum activity window of the CRISPR-mediated base editor and produces non-confounding mutational effects with minimal predicted off-target effects. We demonstrate the utility of the software by designing guide RNAs for base-editing to create or remove thousands of clinically important human disease mutations.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/synthetic-biology">synthetic-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/prokaryotic_nanocompartments_form_synthetic_organelles_in_a_eukaryote_31345805/">
          <h4 class="font-weight-bold">Prokaryotic nanocompartments form synthetic organelles in a eukaryote, bioRxiv, 2018-01-07</h4>
          <p>AbstractCompartmentalization of proteins into organelles is a promising strategy for enhancing the productivity of engineered eukaryotic organisms. However, approaches that co-opt endogenous organelles may be limited by the potential for unwanted crosstalk and disruption of native metabolic functions. Here, we present the construction of synthetic non-endogenous organelles in the eukaryotic yeast Saccharomyces cerevisiae, based on the prokaryotic family of self-assembling proteins known as encapsulins. We establish that encapsulins self-assemble to form nanoscale compartments in yeast, and that heterologous proteins can be selectively targeted for compartmentalization. Housing destabilized proteins within encapsulin compartments affords protection against proteolytic degradation in vivo, while the interaction between split protein components is enhanced upon co-localization within the compartment interior. Furthermore, encapsulin compartments can support enzymatic catalysis, with substrate turnover observed for an encapsulated yeast enzyme. Encapsulin compartments therefore represent a modular platform, orthogonal to existing organelles, for programming synthetic compartmentalization in eukaryotes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/synthetic-biology">synthetic-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/resetting_the_yeast_epigenome_with_human_nucleosomes_20896431/">
          <h4 class="font-weight-bold">Resetting the yeast epigenome with human nucleosomes, bioRxiv, 2017-06-08</h4>
          <p>SummaryHumans and yeast are separated by a billion years of evolution, yet their conserved core histones retain central roles in gene regulation. Here, we “reset” yeast to use core human nucleosomes in lieu of their own, an exceedingly rare event which initially took twenty days. The cells adapt, however, by acquiring suppressor mutations in cell-division genes, or by acquiring certain aneuploidy states. Robust growth was also restored by converting five histone residues back to their yeast counterparts. We reveal that humanized nucleosomes in yeast are positioned according to endogenous yeast DNA sequence and chromatin-remodeling network, as judged by a yeast-like nucleosome repeat length. However, human nucleosomes have higher DNA occupancy and reduce RNA content. Adaptation to new biological conditions presented a special challenge for these cells due to slower chromatin remodeling. This humanized yeast poses many fundamental new questions about the nature of chromatin and how it is linked to many cell processes, and provides a platform to study histone variants via yeast epigenome reprogramming.Highlights&lt;jatslist list-type=simple&gt;&lt;jatslist-item&gt;- Only 1 in 107 yeast survive with fully human nucleosomes, but they rapidly evolve&lt;jatslist-item&gt;&lt;jatslist-item&gt;- Nucleosome positioning and nucleosome repeat length is not influenced by histone type&lt;jatslist-item&gt;&lt;jatslist-item&gt;- Human nucleosomes remodel slowly and delay yeast environmental adaptation&lt;jatslist-item&gt;&lt;jatslist-item&gt;- Human core nucleosomes are more repressive and globally reduce transcription in yeast&lt;jatslist-item&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/synthetic-biology">synthetic-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        
        <li class="page-item"><a href="/audiences/tags/synthetic-biology/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="/audiences/tags/synthetic-biology/" class="page-link">1</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/synthetic-biology/page/2/" class="page-link">2</a></li>
            
        

        

        
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
