<!DOCTYPE html>
<html><head>
  
  <title>2018 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_landscape_of_somatic_mutation_in_normal_colorectal_epithelial_cells_48267271/">
          <h4 class="font-weight-bold">The landscape of somatic mutation in normal colorectal epithelial cells, bioRxiv, 2018-09-14</h4>
          <p>AbstractThe colorectal adenoma-carcinoma sequence has provided a paradigmatic framework for understanding the successive somatic genetic changes and consequent clonal expansions leading to cancer. As for most cancer types, however, understanding of the earliest phases of colorectal neoplastic change, which may occur in morphologically normal tissue, is comparatively limited because of the difficulty of detecting somatic mutations in normal cells. Each colorectal crypt is a small clone of cells derived from a single recently-existing stem cell. Here, we whole genome sequenced hundreds of normal crypts from 42 individuals. Signatures of multiple mutational processes were revealed, some ubiquitous and continuous, others only found in some individuals, in some crypts or during some phases of the cell lineage from zygote to adult cell. Likely driver mutations were present in âˆ¼1% of normal colorectal crypts in middle-aged individuals, indicating that adenomas and carcinomas are rare outcomes of a pervasive process of neoplastic change across morphologically normal colorectal epithelium.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cancer-biology">cancer-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/bio-on-magnetic-beads_bomb_open_platform_for_high-throughput_nucleic_acid_extraction_and_manipulation_48184997/">
          <h4 class="font-weight-bold">Bio-On-Magnetic-Beads (BOMB) Open platform for high-throughput nucleic acid extraction and manipulation, bioRxiv, 2018-09-13</h4>
          <p>AbstractCurrent molecular biology laboratories rely heavily on the purification and manipulation of nucleic acids. Yet, commonly used centrifuge-and column-based protocols require specialised equipment, often use toxic reagents and are not economically scalable or practical to use in a high-throughput manner. Although it has been known for some time that magnetic beads can provide an elegant answer to these issues, the development of open-source protocols based on beads has been limited. In this article, we provide step-by-step instructions for an easy synthesis of functionalised magnetic beads, and detailed protocols for their use in the high-throughput purification of plasmids, genomic DNA and total RNA from different sources, as well as environmental TNA and PCR amplicons. We also provide a bead-based protocol for bisulfite conversion, and size selection of DNA and RNA fragments. Comparison to other methods highlights the capability, versatility and extreme cost-effectiveness of using magnetic beads. These open source protocols and the associated webpage (&lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsbomb.bio&gt;httpsbomb.bio&lt;jatsext-link&gt;) can serve as a platform for further protocol customisation and community engagement.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/novel_childhood_experience_suggests_eccentricity_drives_organization_of_human_visual_cortex_48232938/">
          <h4 class="font-weight-bold">Novel childhood experience suggests eccentricity drives organization of human visual cortex, bioRxiv, 2018-09-13</h4>
          <p>The functional organization of human high-level visual cortex, such as face and place-selective regions, is strikingly consistent across individuals. A fundamental, unanswered question in neuroscience is what dimensions of visual information constrain the development and topography of this shared brain organization? To answer this question, we scanned with fMRI a unique group of adults who, as children, engaged in extensive experience with a novel stimulus, Pokemon, that varied along critical dimensions (foveal bias, rectilinearity, size, animacy) from other ecological categories such as faces and places. We find that experienced adults not only demonstrate distinct and consistent distributed cortical responses to Pokemon, but their activations suggest that it is the experienced retinal eccentricity during childhood that predicts the locus of distributed responses to Pokemon in adulthood. These data advance our understanding about how childhood experience and functional constraints shape the functional organization of the human brain.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/transient_intracellular_acidification_regulates_the_core_transcriptional_heat_shock_response_48185096/">
          <h4 class="font-weight-bold">Transient intracellular acidification regulates the core transcriptional heat shock response, bioRxiv, 2018-09-12</h4>
          <p>AbstractCellular stress induces rapid expression of genes encoding molecular chaperones. In many eukaryotes, stress also triggers transient intracellular acidification which, by unknown mechanisms, is associated with increased survival. Here, using budding yeast as a model, we discover that preventing cells from transiently acidifying during heat shock compromises induction of molecular chaperones and fitness. Prevention of acidification during stress and recovery silences induction of a canonical heat-shock protein altogether. The association between acidification, induction, and growth holds at the population and single-cell levels. Hinting at the molecular basis of these effects, the failure to acidify specifically suppresses induction of genes regulated by the conserved heat shock transcription factor Hsf1. Our results establish a central role for intracellular pH in the eukaryotic transcriptional stress response, and implicate pH-sensitive stress-sensing proteins, rather than misfolded proteins, in the activation of Hsf1 under physiological heat shock conditions.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_genomic_view_of_diversification_48122574/">
          <h4 class="font-weight-bold">The genomic view of diversification, bioRxiv, 2018-09-11</h4>
          <p>ABSTRACTEvolutionary relationships between species are traditionally represented in the form of a tree, called the species tree. The reconstruction of the species tree from molecular data is hindered by frequent conflicts between gene genealogies. A standard way of dealing with this issue is to postulate the existence of a unique species tree where disagreements between gene trees are explained by incomplete lineage sorting (ILS) due to random coalescences of gene lineages inside the edges of the species tree. This paradigm, known as the multi-species coalescent (MSC), is constantly violated by the ubiquitous presence of gene flow revealed by empirical studies, leading to topological incongruences of gene trees that cannot be explained by ILS alone. Here we argue that this paradigm should be revised in favor of a vision acknowledging the importance of gene flow and where gene histories shape the species tree rather than the opposite. We propose a new, plastic framework for modeling the joint evolution of gene and species lineages relaxing the hierarchy between the species tree and gene trees. We implement this framework in two mathematical models called the gene-based diversification models (GBD) 1) GBD-forward, following all evolving genomes and thus very intensive computationally and 2) GBD-backward, based on coalescent theory and thus more efficient. Each model features four parameters tuning colonization, mutation, gene flow and reproductive isolation. We propose a quick inference method based on the differences between gene trees and use it to evaluate the amount of gene flow in two empirical data-sets. We find that in these data-sets, gene tree distributions are better explained by the best fitting GBD model than by the best fitting MSC model. This work should pave the way for approaches of diversification using the richer signal contained in genomic evolutionary histories rather than in the mere species tree.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_data-driven_approach_to_the_automated_study_of_cross-species_homologies_48009392/">
          <h4 class="font-weight-bold">A data-driven approach to the automated study of cross-species homologies, bioRxiv, 2018-09-09</h4>
          <p>AbstractBehavioral neuroscience has made great strides in developing animal models of human behavior and psychiatric disorders. Animal models allow for the formulation of hypotheses regarding the mechanisms underlying psychiatric disorders, and the opportunity to test these hypotheses using procedures that are too invasive for human participants. However, recent scientific reviews have highlighted the low success rate of translating results from animal models into clinical interventions in humans. A potential roadblock is that bidirectional functional mappings between the human and rodent brain are incomplete. To narrow this gap, we created a framework, Neurobabel, for performing large-scale automated synthesis of human neuroimaging data and behavioral neuroscience data. By leveraging the semantics of how researchers within each field describe their studies, this framework enables region to region mapping of brain regions across species, as well as cross-species mapping of psychological functions. As a proof of concept, we utilize the framework to create a functional cross-species mapping between the amygdala and hippocampus for fear-related and spatial memories, respectively. We then proceed to address two open questions in the field (1) Do rodents have a dorsolateral prefrontal cortex? (2) Which human brain region corresponds to the rodent prelimbic cortex?</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/2018/" rel="first" class="page-link">Â« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/2018/page/41/" rel="prev" class="page-link">â€¹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/38/" class="page-link">38</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/39/" class="page-link">39</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/40/" class="page-link">40</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/41/" class="page-link">41</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/2018/page/42/" class="page-link">42</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/43/" class="page-link">43</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/44/" class="page-link">44</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/45/" class="page-link">45</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/46/" class="page-link">46</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/2018/page/43/" rel="next" class="page-link">Next â€º</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/2018/page/54/" rel="last" class="page-link">Last Â»</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
