<!DOCTYPE html>
<html><head>
  
  <title>2019 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.68.3" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.95d2483cd3c5d2bf47c1f36307a456787d60ad958c5efe44db9afc47012b1b05b8e1f64f6ac291650439322a6c38e86eb7cd6254255516d857c23f299fb0e666.js" integrity="sha512-ldJIPNPF0r9HwfNjB6RWeH1grZWMXv5E25r8RwErGwW44fZPasKRZQQ5MipsOOhut81iVCVVFthXwj8pn7DmZg=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/ohana_detecting_selection_in_multiple_populations_by_modelling_ancestral_admixture_components_55483766/">
          <h4 class="font-weight-bold">Ohana detecting selection in multiple populations by modelling ancestral admixture components, bioRxiv, 2019-02-15</h4>
          <p>One of the most powerful and commonly used methods for detecting local adaptation in the genome is the identification of extreme allele frequency differences between populations. In this paper, we present a new maximum likelihood method for finding regions under positive selection. The method is based on a Gaussian approximation to allele frequency changes and it incorporates admixture between populations. The method can analyze multiple populations simultaneously and retains power to detect selection signatures specific to ancestry components that are not representative of any extant populations. We evaluate the method using simulated data and compare it to related methods based on summary statistics. We also apply it to human genomic data and identify loci with extreme genetic differentiation between major geographic groups. Many of the genes identified are previously known selected loci relating to hair pigmentation and morphology, skin and eye pigmentation. We also identify new candidate regions, including various selected loci in the Native American component of admixed Mexican-Americans. These involve diverse biological functions, like immunity, fat distribution, food intake, vision and hair development.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/global_signal_regression_strengthens_association_between_resting-state_functional_connectivity_and_behavior_55430314/">
          <h4 class="font-weight-bold">Global Signal Regression Strengthens Association between Resting-State Functional Connectivity and Behavior, bioRxiv, 2019-02-14</h4>
          <p>Global signal regression (GSR) is one of the most debated preprocessing strategies for resting-state functional MRI. GSR effectively removes global artifacts driven by motion and respiration, but also discards globally distributed neural information and introduces negative correlations between certain brain regions. The vast majority of previous studies have focused on the effectiveness of GSR in removing imaging artifacts, as well as its potential biases. Given the growing interest in functional connectivity fingerprinting, here we considered the utilitarian question of whether GSR strengthens or weakens associations between resting-state functional connectivity (RSFC) and multiple behavioral measures across cognition, personality and emotion. By applying the variance component model to the Brain Genomics Superstruct Project (GSP), we found that behavioral variance explained by whole-brain RSFC increased by an average of 47% across 23 behavioral measures after GSR. In the Human Connectome Project (HCP), we found that behavioral variance explained by whole-brain RSFC increased by an average of 40% across 58 behavioral measures, when GSR was applied after ICA-FIX de-noising. To ensure generalizability, we repeated our analyses using kernel regression. GSR improved behavioral prediction accuracies by an average of 64% and 12% in the GSP and HCP datasets respectively. Importantly, the results were consistent across methods. A behavioral measure with greater RSFC-explained variance (using the variance component model) also exhibited greater prediction accuracy (using kernel regression). A behavioral measure with greater improvement in behavioral variance explained after GSR (using the variance component model) also enjoyed greater improvement in prediction accuracy after GSR (using kernel regression). Furthermore, GSR appeared to benefit task performance measures more than self-reported measures. Since GSR was more effective at removing motion-related and respiratory-related artifacts, GSR-related increases in variance explained and prediction accuracies were unlikely the result of motion-related or respiratory-related artifacts. However, it is worth emphasizing that the current study focused on whole-brain RSFC, so it remains unclear whether GSR improves RSFC-behavioral associations for specific connections or networks. Overall, our results suggest that at least in the case for young healthy adults, GSR strengthens the associations between RSFC and most (although not all) behavioral measures. Code for the variance component model and ridge regression can be found here httpsgithub.comThomasYeoLabCBIGtreemasterstable_projectspreprocessingLi2019_GSR.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/multi-immersion_open-top_light-sheet_microscope_for_high-throughput_imaging_of_cleared_tissues_55402438/">
          <h4 class="font-weight-bold">Multi-immersion open-top light-sheet microscope for high-throughput imaging of cleared tissues, bioRxiv, 2019-02-13</h4>
          <p>Recent advances in optical clearing and light-sheet microscopy have provided unprecedented access to structural and molecular information from intact tissues. However, current light-sheet microscopes have imposed constraints on the size, shape, number of specimens, and compatibility with various clearing protocols. Here we present a multi-immersion open-top light-sheet microscope that enables simple mounting of multiple specimens processed with a variety of protocols, which will facilitate wider adoption by preclinical researchers and clinical laboratories.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/bioengineering">bioengineering</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/resting-state_cross-frequency_coupling_networks_in_human_electrophysiological_recordings_55400952/">
          <h4 class="font-weight-bold">Resting-state cross-frequency coupling networks in human electrophysiological recordings, bioRxiv, 2019-02-13</h4>
          <p>Neuronal oscillations underlie temporal coordination of neuronal processing and their synchronization enables neuronal communication across distributed brain areas to serve a variety of sensory, motor, and cognitive functions. The regulation and integration of neuronal processing between oscillating assemblies at distinct frequencies, and thereby the coordination of distinct computational functions, is thought to be achieved via cross-frequency coupling (CFC). Although many studies have observed CFC locally within a brain region during cognitive processing, the large-scale networks of CFC have remained largely uncharted. Critically, also the validity of prior CFC observations and the presence of true neuronal CFC has been recently questioned because non-sinusoidal or non-zero-mean waveforms that are commonplace in electrophysiological data cause filtering artefacts that lead to false positive CFC findings. We used a unique dataset of stereo-electroencephalography (SEEG) and source-reconstructed magnetoencephalography (MEG) data to chart whole-brain CFC networks from human resting-state brain dynamics. Using a novel graph theoretical method to distinguish true inter-areal CFC from potentially false positive CFC, we show that the resting state is characterized by two separable forms of true inter-areal CFC phase-amplitude coupling (PAC) and nm-cross-frequency phase synchrony (CFS). PAC and CFS large-scale networks coupled prefrontal, visual and sensorimotor cortices, but with opposing anatomical architectures. Crucially also directionalities between low- and high-frequency oscillations were opposite between CFS and PAC. We also found CFC to decay as a function of distance and to be stronger in the superficial than deep layers of the cortex. In conclusion, these results provide conclusive evidence for the presence of two forms of genuine inter-areal CFC and elucidate the large-scale organization of CFC resting-state networks.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/bacterial_phage_tail-like_structure_kills_eukaryotic_cells_by_injecting_a_nuclease_effector_55318539/">
          <h4 class="font-weight-bold">Bacterial Phage Tail-like Structure Kills Eukaryotic Cells by Injecting a Nuclease Effector, bioRxiv, 2019-02-12</h4>
          <p>Many bacteria interact with target organisms using syringe-like structures called Contractile Injection Systems (CIS). CIS structurally resemble headless bacteriophages and share evolutionarily related proteins such as the tail tube, sheath, and baseplate complex. Recent evidence shows that CIS are specialized to puncture membranes and often deliver effectors to target cells. In many cases, CIS mediate trans-kingdom interactions between bacteria and eukaryotes, however the effectors delivered to target cells and their mode of action are often unknown. In this work, we establish an in vitro model to study a CIS called Metamorphosis Associated Contractile structures (MACs) that target eukaryotic cells. We show that MACs kill two eukaryotic cell lines, Fall Armyworm Sf9 cells and J774A.1 murine macrophage cells through the action of a newly identified MAC effector, termed Pne1. To our knowledge, Pne1 is the first CIS effector exhibiting nuclease activity against eukaryotic cells. Our results define a new mechanism of CIS-mediated bacteria-eukaryote interaction and are a first step toward understanding structures with the potential to be developed as novel delivery systems for eukaryotic hosts.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/exercise_twice-a-day_potentiates_skeletal_muscle_signalling_responses_associated_with_mitochondrial_biogenesis_in_humans_which_are_independent_of_lowered_muscle_glycogen_content_55355158/">
          <h4 class="font-weight-bold">Exercise twice-a-day potentiates skeletal muscle signalling responses associated with mitochondrial biogenesis in humans, which are independent of lowered muscle glycogen content, bioRxiv, 2019-02-12</h4>
          <p>Endurance exercise begun with reduced muscle glycogen stores seems to potentiate skeletal muscle protein abundance and gene expression. However, it is unknown whether this greater signalling responses is due to low muscle glycogen per se or to performing two exercise sessions in close proximity - as a first exercise session is necessary to reduce the muscle glycogen stores. In the present study, we manipulated the recovery duration between a first muscle glycogen-depleting exercise and a second exercise session, such that the second exercise session started with reduced muscle glycogen in both approaches but was performed either two or 15 h after the first exercise session (so-called twice-a-day and once-daily approaches, respectively). We found that exercise twice-a-day increased the nuclear abundance of transcription factor EB (TFEB) and nuclear factor of activated T cells (NFAT) and potentiated the transcription of peroxisome proliferator-activated receptor-ɣ coactivator 1 alpha (PGC-1a), peroxisome proliferator-activated receptor alpha (PPARa;) and peroxisome proliferator-activated receptor betadelta (PPARbd) genes, in comparison with the once-daily exercise. These results suggest that the elevated molecular signalling reported with previous train-low approaches can be attributed to performing two exercise sessions in close proximity rather than the reduced muscle glycogen content per se. The twice-a-day approach might be an effective strategy to induce adaptations related to mitochondrial biogenesis and fat oxidation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/110/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/107/" class="page-link">107</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/108/" class="page-link">108</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/109/" class="page-link">109</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/110/" class="page-link">110</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/2019/page/111/" class="page-link">111</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/112/" class="page-link">112</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/113/" class="page-link">113</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/114/" class="page-link">114</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/115/" class="page-link">115</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/112/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/135/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
