<!DOCTYPE html>
<html><head>
  
  <title>200 500 Users | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/pediatric_bone_age_assessment_using_deep_convolutional_neural_networks_30444746/">
          <h4 class="font-weight-bold">Pediatric Bone Age Assessment Using Deep Convolutional Neural Networks, bioRxiv, 2017-12-15</h4>
          <p>AbstractSkeletal bone age assessment is a common clinical practice to diagnose endocrine and metabolic disorders in child development. In this paper, we describe a fully automated deep learning approach to the problem of bone age assessment using data from the 2017 Pediatric Bone Age Challenge organized by the Radiological Society of North America. The dataset for this competition consists of 12,600 radiological images. Each radiograph in this dataset is an image of a left hand labeled with bone age and sex of a patient. Our approach utilizes several deep neural network architectures trained end-to-end. We use images of whole hands as well as specific parts of a hand for both training and prediction. This approach allows us to measure the importance of specific hand bones for automated bone age analysis. We further evaluate the performance of the suggested method in the context of skeletal development stages. Our approach outperforms other common methods for bone age assessment.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/pathology">pathology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/robust_manipulation_of_the_behavior_of_drosophila_melanogaster_by_a_fungal_pathogen_in_the_laboratory_30276844/">
          <h4 class="font-weight-bold">Robust manipulation of the behavior of Drosophila melanogaster by a fungal pathogen in the laboratory, bioRxiv, 2017-12-11</h4>
          <p>AbstractMany microbes induce striking behavioral changes in their animal hosts, but how they achieve this is poorly understood, especially at the molecular level. Mechanistic understanding has been largely constrained by the lack of a model system with advanced tools for molecular manipulation. We recently discovered a strain of the behavior-manipulating fungal pathogen Entomophthora muscae infecting wild Drosophila, and established methods to infect D. melanogaster in the lab. Lab-infected flies manifest the moribund behaviors characteristic of E. muscae infection hours before death, they climb upward, extend their proboscides and affix in place, then raise their wings, clearing a path for infectious spores to launch from their abdomens. We found that E. muscae invades the fly nervous system, suggesting a direct means by which the fungus could induce behavioral changes. Given the vast molecular toolkit available for D. melanogaster, we believe this new system will enable rapid progress in understanding the mechanistic basis of E. muscaeâ€™s behavioral manipulation in the fly.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/animal-behavior-and-cognition">animal-behavior-and-cognition</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/shared_and_distinct_transcriptomic_cell_types_across_neocortical_areas_30101217/">
          <h4 class="font-weight-bold">Shared and distinct transcriptomic cell types across neocortical areas, bioRxiv, 2017-12-07</h4>
          <p>ABSTRACTNeocortex contains a multitude of cell types segregated into layers and functionally distinct regions. To investigate the diversity of cell types across the mouse neocortex, we analyzed 12,714 cells from the primary visual cortex (VISp), and 9,035 cells from the anterior lateral motor cortex (ALM) by deep single-cell RNA-sequencing (scRNA-seq), identifying 116 transcriptomic cell types. These two regions represent distant poles of the neocortex and perform distinct functions. We define 50 inhibitory transcriptomic cell types, all of which are shared across both cortical regions. In contrast, 49 of 52 excitatory transcriptomic types were found in either VISp or ALM, with only three present in both. By combining single cell RNA-seq and retrograde labeling, we demonstrate correspondence between excitatory transcriptomic types and their region-specific long-range target specificity. This study establishes a combined transcriptomic and projectional taxonomy of cortical cell types from functionally distinct regions of the mouse cortex.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/exponential_fluorescent_amplification_of_individual_rnas_using_clampfish_probes_29216517/">
          <h4 class="font-weight-bold">Exponential fluorescent amplification of individual RNAs using clampFISH probes, bioRxiv, 2017-12-06</h4>
          <p>AbstractNon-enzymatic, high-gain signal amplification methods with single-cell, single-molecule resolution are in great need. We present click-amplifying FISH (clampFISH) for the fluorescent detection of RNA that combines the specificity of oligonucleotides with bioorthogonal click chemistry in order to achieve high specificity and extremely high-gain (&amp;gt;400x) signal amplification. We show that clampFISH signal enables detection with low magnification microscopy and separation of cells by RNA levels via flow cytometry. Additionally, we show that the modular design of clampFISH probes enables multiplexing, that the locking mechanism prevents probe detachment in expansion microscopy, and that clampFISH works in tissue samples.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioengineering">bioengineering</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/high-throughput_ani_analysis_of_90k_prokaryotic_genomes_reveals_clear_species_boundaries_29516560/">
          <h4 class="font-weight-bold">High-throughput ANI Analysis of 90K Prokaryotic Genomes Reveals Clear Species Boundaries, bioRxiv, 2017-11-28</h4>
          <p>A fundamental question in microbiology is whether there is a continuum of genetic diversity among genomes or clear species boundaries prevail instead. Answering this question requires robust measurement of whole-genome relatedness among thousands of genomes and from diverge phylogenetic lineages. Whole-genome similarity metrics such as Average Nucleotide Identity (ANI) can provide the resolution needed for this task, overcoming several limitations of traditional techniques used for the same purposes. Although the number of genomes currently available may be adequate, the associated bioinformatics tools for analysis are lagging behind these developments and cannot scale to large datasets. Here, we present a new method, FastANI, to compute ANI using alignment-free approximate sequence mapping. Our analyses demonstrate that FastANI produces an accurate ANI estimate and is up to three orders of magnitude faster when compared to an alignment (e.g., BLAST)-based approach. We leverage FastANI to compute pairwise ANI values among all prokaryotic genomes available in the NCBI database. Our results reveal a clear genetic discontinuity among the database genomes, with 99.8% of the total 8 billion genome pairs analyzed showing either &amp;gt;95% intra-species ANI or &amp;lt;83% inter-species ANI values. We further show that this discontinuity is recovered with or without the most frequently represented species in the database and is robust to historic additions in the public genome databases. Therefore, 95% ANI represents an accurate threshold for demarcating almost all currently named prokaryotic species, and wide species boundaries may exist for prokaryotes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/a_suite_of_transgenic_driver_and_reporter_mouse_lines_with_enhanced_brain_cell_type_targeting_and_functionality_29448543/">
          <h4 class="font-weight-bold">A suite of transgenic driver and reporter mouse lines with enhanced brain cell type targeting and functionality, bioRxiv, 2017-11-26</h4>
          <p>SUMMARYModern genetic approaches are powerful in providing access to diverse types of neurons within the mammalian brain and greatly facilitating the study of their function. We here report a large set of driver and reporter transgenic mouse lines, including 23 new driver lines targeting a variety of cortical and subcortical cell populations and 26 new reporter lines expressing an array of molecular tools. In particular, we describe the TIGRE2.0 transgenic platform and introduce Cre-dependent reporter lines that enable optical physiology, optogenetics, and sparse labeling of genetically-defined cell populations. TIGRE2.0 reporters broke the barrier in transgene expression level of single-copy targeted-insertion transgenesis in a wide range of neuronal types, along with additional advantage of a simplified breeding strategy compared to our first-generation TIGRE lines. These novel transgenic lines greatly expand the repertoire of high-precision genetic tools available to effectively identify, monitor, and manipulate distinct cell types in the mouse brain.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/200-500-users/" rel="first" class="page-link">Â« First</a></li>
        

        
        <li class="page-item"><a href="/tags/200-500-users/page/27/" rel="prev" class="page-link">â€¹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/24/" class="page-link">24</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/25/" class="page-link">25</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/26/" class="page-link">26</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/27/" class="page-link">27</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/200-500-users/page/28/" class="page-link">28</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/29/" class="page-link">29</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/30/" class="page-link">30</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/31/" class="page-link">31</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/32/" class="page-link">32</a></li>
            
        

        
        <li class="page-item"><a href="/tags/200-500-users/page/29/" rel="next" class="page-link">Next â€º</a></li>
        

        
        
        <li class="page-item"><a href="/tags/200-500-users/page/32/" rel="last" class="page-link">Last Â»</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
