<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/nanoscale_subcellular_architecture_revealed_by_multicolor_3d_salvaged_fluorescence_imaging_59214657/">
          <h4 class="font-weight-bold">Nanoscale subcellular architecture revealed by multicolor 3D salvaged fluorescence imaging, bioRxiv, 2019-04-19</h4>
          <p>AbstractCombining the molecular specificity of fluorescent probes with three-dimensional (3D) imaging at nanoscale resolution is critical for investigating the spatial organization and interactions of cellular organelles and protein complexes. We present a super-resolution light microscope that enables simultaneous multicolor imaging of whole mammalian cells at ~20 nm 3D resolution. We show its power for cell biology research with fluorescence images that resolved the highly convoluted Golgi apparatus and the close contacts between the endoplasmic reticulum and the plasma membrane, structures that have traditionally been the imaging realm of electron microscopy.One Sentence SummaryComplex cellular structures previously only resolved by electron microscopy can now be imaged in multiple colors by 4Pi-SMS.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/generation_of_viral_vectors_specific_to_neuronal_subtypes_of_targeted_brain_regions_by_enhancer-driven_gene_expression_edge_58992127/">
          <h4 class="font-weight-bold">Generation of viral vectors specific to neuronal subtypes of targeted brain regions by Enhancer-Driven Gene Expression (EDGE), bioRxiv, 2019-04-14</h4>
          <p>SummaryUnderstanding brain function requires understanding neural circuits at the level of specificity at which they operate. While recent years have seen the development of a variety of remarkable molecular tools for the study of neural circuits, their utility is currently limited by the inability to deploy them in specific elements of native neural circuits, i.e. particular neuronal subtypes. One can obtain a degree of specificity with neuron-specific promoters, but native promoters are almost never sufficiently specific restricting this approach to transgenic animals. We recently showed that one can obtain transgenic mice with augmented anatomical specificity in targeted brain regions by identifying cis-regulatory elements (i.e. enhancers) uniquely active in those brain regions and combining them with a heterologous promoter, an approach we call EDGE (Enhancer-Driven Gene Expression). Here we extend this strategy to the generation of viral (rAAV) vectors, showing that when combined with the right minimal promoter they largely recapitulate the specificity seen in the corresponding transgenic lines in wildtype animals, even of another species. Because active enhancers can be identified in any tissue sample, this approach promises to enable the kind of circuit-specific manipulations in any species. This should not only greatly enhance our understanding of brain function, but may one day even provide novel therapeutic avenues to correct the imbalances in neural circuits underlying many disorders of the brain.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/loss-of-function_tolerance_of_enhancers_in_the_human_genome_58986026/">
          <h4 class="font-weight-bold">Loss-of-function tolerance of enhancers in the human genome, bioRxiv, 2019-04-14</h4>
          <p>AbstractPrevious studies have surveyed the potential impact of loss-of-function (LoF) variants and identified LoF-tolerant protein-coding genes. However, the tolerance of human genomes to losing enhancers has not yet been evaluated. Here we present the catalog of LoF-tolerant enhancers using structural variants from whole-genome sequences. Using a conservative approach, we estimate that each individual human genome possesses at least 28 LoF-tolerant enhancers on average. We assessed the properties of LoF-tolerant enhancers in a unified regulatory network constructed by integrating tissue-specific enhancers and gene-gene interactions. We find that LoF-tolerant enhancers are more tissue-specific and regulate fewer and more dispensable genes. They are enriched in immune-related cells while LoF-intolerant enhancers are enriched in kidney and brainneuronal stem cells. We developed a supervised learning approach to predict the LoF-tolerance of enhancers, which achieved an AUROC of 96%. We predict 5,677 more enhancers would be likely tolerant to LoF and 75 enhancers that would be highly LoF-intolerant. Our predictions are supported by known set of disease enhancers and novel deletions from PacBio sequencing. The LoF-tolerance scores provided here will serve as an important reference for disease studies.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/light-sheet_microscopy_with_isotropic_sub-micron_resolution_and_solvent-independent_large-scale_imaging_58891468/">
          <h4 class="font-weight-bold">Light-sheet microscopy with isotropic, sub-micron resolution and solvent-independent large-scale imaging, bioRxiv, 2019-04-12</h4>
          <p>AbstractWe present cleared tissue Axially Swept Light-Sheet Microscopy (ctASLM), which achieves sub-micron isotropic resolution, high optical sectioning capability, and large field of view imaging (870×870 μm2) over a broad range of immersion media. ctASLM can image live, expanded, and both aqueous and organic chemically cleared tissue preparations and provides 2- to 5-fold better axial resolution than confocal or other reported cleared tissue light-sheet microscopes. We image millimeter-sized tissues with sub-micron 3D resolution, which enabled us to perform automated detection of cells and subcellular features such as dendritic spines.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioengineering">bioengineering</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/antibiotics_select_for_novel_pathways_of_resistance_in_biofilms_58836429/">
          <h4 class="font-weight-bold">Antibiotics select for novel pathways of resistance in biofilms, bioRxiv, 2019-04-11</h4>
          <p>AbstractMost bacteria in nature exist in aggregated communities known as biofilms. Bacteria within biofilms are inherently highly resistant to antibiotics. Current understanding of the evolution and mechanisms of antibiotic resistance is largely derived from work from cells in liquid culture and it is unclear whether biofilms adapt and evolve in response to sub-inhibitory concentrations of drugs. Here we used a biofilm evolution model to show that biofilms of a model food borne pathogen, Salmonella Typhimurium rapidly evolve in response to exposure to three clinically important antibiotics. Whilst the model strongly selected for improved biofilm formation in the absence of any drug, once antibiotics were introduced the need to adapt to the drug was more important than the selection for improved biofilm formation. Adaptation to antibiotic stress imposed a marked cost in biofilm formation, particularly evident for populations exposed to cefotaxime and azithromycin. We identified distinct resistance phenotypes in biofilms compared to corresponding planktonic control cultures and characterised new mechanisms of resistance to cefotaxime and azithromycin. Novel substitutions within the multidrug efflux transporter, AcrB were identified and validated as impacting drug export as well as changes in regulators of this efflux system. There were clear fitness costs identified and associated with different evolutionary trajectories. Our results demonstrate that biofilms adapt rapidly to low concentrations of antibiotics and the mechanisms of adaptation are novel. This work will be a starting point for studies to further examine biofilm specific pathways of adaptation which inform future antibiotic use.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/measuring_and_mitigating_pcr_bias_in_microbiome_data_58808276/">
          <h4 class="font-weight-bold">Measuring and Mitigating PCR Bias in Microbiome Data, bioRxiv, 2019-04-10</h4>
          <p>AbstractPCR amplification plays a central role in the measurement of mixed microbial communities via high-throughput sequencing. Yet PCR is also known to be a common source of bias in microbiome data. Here we present a paired modeling and experimental approach to characterize and mitigate PCR bias in microbiome studies. We use experimental data from mock bacterial communities to validate our approach and human gut microbiota samples to characterize PCR bias under real-world conditions. Our results suggest that PCR can bias estimates of microbial relative abundances by a factor of 2-4 but that this bias can be mitigated using simple Bayesian multinomial logistic-normal linear models.Author summaryHigh-throughput sequencing is often used to profile host-associated microbial communities. Many processing steps are required to transform a community of bacteria into a pool of DNA suitable for sequencing. One important step is amplification where, to create enough DNA for sequencing, DNA from many different bacteria are repeatedly copied using a technique called Polymerase Chain Reaction (PCR). However, PCR is known to introduce bias as DNA from some bacteria are more efficiently copied than others. Here we introduce an experimental procedure that allows this bias to be measured and computational techniques that allow this bias to be mitigated in sequencing data.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/10/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/10/" class="page-link">10</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/0-100-users/page/11/" class="page-link">11</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/12/" class="page-link">12</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/13/" class="page-link">13</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/14/" class="page-link">14</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/15/" class="page-link">15</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/12/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/64/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
