<!DOCTYPE html>
<html><head>
  
  <title>2020 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_standardized_and_reproducible_method_to_measure_decision-making_in_mice_74202436/">
          <h4 class="font-weight-bold">A standardized and reproducible method to measure decision-making in mice, bioRxiv, 2020-01-18</h4>
          <p>AbstractProgress in neuroscience is hindered by poor reproducibility of mouse behavior. Here we show that in a visual decision making task, reproducibility can be achieved by automating the training protocol and by standardizing experimental hardware, software, and procedures. We trained 101 mice in this task across seven laboratories at six different research institutions in three countries, and obtained 3 million mouse choices. In trained mice, variability in behavior between labs was indistinguishable from variability within labs. Psychometric curves showed no significant differences in visual threshold, bias, or lapse rates across labs. Moreover, mice across laboratories adopted similar strategies when stimulus location had asymmetrical probability that changed over time. We provide detailed instructions and open-source tools to set up and implement our method in other laboratories. These results establish a new standard for reproducibility of rodent behavior and provide accessible tools for the study of decision making in mice.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/multiplexed_single-cell_metabolic_profiles_organize_the_spectrum_of_cytotoxic_human_t_cells_74169828/">
          <h4 class="font-weight-bold">Multiplexed Single-cell Metabolic Profiles Organize the Spectrum of Cytotoxic Human T Cells, bioRxiv, 2020-01-18</h4>
          <p>SummaryCellular metabolism regulates immune cell activation, differentiation and effector functions to the extent that its perturbation can augment immune responses. However, the analytical technologies available to study cellular metabolism lack single-cell resolution, obscuring metabolic heterogeneity and its connection to immune phenotype and function. To that end, we utilized high-dimensional, antibody-based technologies to simultaneously quantify the single-cell metabolic regulome in combination with phenotypic identity. Mass cytometry (CyTOF)-based application of this approach to early human T cell activation enabled the comprehensive reconstruction of the coordinated metabolic remodeling of naïve CD8&#43; T cells and aligned with conventional bulk assays for glycolysis and oxidative phosphorylation. Extending this analysis to a variety of tissue-resident immune cells revealed tissue-restricted metabolic states of human cytotoxic T cells, including metabolically repressed subsets that expressed CD39 and PD1 and that were enriched in colorectal carcinoma versus healthy adjacent tissue. Finally, combining this approach with multiplexed ion beam imaging by time-of-flight (MIBI-TOF) demonstrated the existence of spatially enriched metabolic neighborhoods, independent of cell identity and additionally revealed exclusion of metabolically repressed cytotoxic T cell states from the tumor-immune boundary in human colorectal carcinoma. Overall, we provide an approach that permits the robust approximation of metabolic states in individual cells along with multimodal analysis of cell identity and functional characteristics that can be applied to human clinical samples to study cellular metabolism how it may be perturbed to affect immunological outcomes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/distinguishing_the_neural_correlates_of_perceptual_awareness_and_post-perceptual_processing_74118318/">
          <h4 class="font-weight-bold">Distinguishing the neural correlates of perceptual awareness and post-perceptual processing, bioRxiv, 2020-01-17</h4>
          <p>AbstractTo identify the neural correlates of perceptual awareness, researchers often compare the differences in neural activation between conditions in which an observer is or is not aware of a target stimulus. While intuitive, this approach often contains a critical limitation In order to link brain activity with perceptual awareness, observers traditionally report the contents of their perceptual experience. However, relying on observers’ reports is problematic because it makes it difficult to know if the neural responses being measured are associated with conscious perception per se or with post-perceptual processes involved in the reporting task (i.e., working memory, decision-making, etc.). To address this issue, we combined a standard visual masking paradigm with a recently developed “no-report” paradigm. In the visual masking paradigm, observers saw images of animals and objects that were visible or invisible depending on their proximity to masks. Meanwhile, on half of the trials, observers reported the contents of their perceptual experience (i.e., report condition), while on the other half of trials they refrained from reporting about their experiences (i.e., no-report condition). We used electroencephalography (EEG) to examine how visibility interacts with reporting by measuring the P3b event related potential (ERP), one of the proposed canonical “signatures” of conscious processing. Overall, we found a robust P3b in the report condition, but no P3b whatsoever in the no-report condition. This finding suggests that the P3b itself is not a neural signature of conscious processing and highlights the importance of carefully distinguishing the neural correlates of perceptual awareness from post-perceptual processing.Significance statementWhat are the neural signatures that differentiate conscious and unconscious processing in the brain? Perhaps the most well-established candidate signature is the P3b event-related potential (ERP), a late slow wave that appears when observers are aware of a stimulus, but disappears when a stimulus fails to reach awareness. Here, however, we found that the P3b does not track what observers are perceiving but instead tracks what observers are reporting. When observers are aware of simple visual stimuli, the P3b is nowhere to be found unless observers are reporting the contents of their experience. These results challenge the well-established notion of the P3b as a neural marker of awareness and highlight the need for new approaches to the neuroscience of consciousness.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/investigating_the_genetic_architecture_of_non-cognitive_skills_using_gwas-by-subtraction_74072515/">
          <h4 class="font-weight-bold">Investigating the Genetic Architecture of Non-Cognitive Skills Using GWAS-by-Subtraction, bioRxiv, 2020-01-16</h4>
          <p>AbstractEducational attainment (EA) is influenced by cognitive abilities and by other characteristics and traits. However little is known about the genetic architecture of these “non-cognitive” contributions to EA. Here, we use Genomic Structural Equation Modelling and results of prior genome-wide association studies (GWASs) of EA (N = 1,131,881) and cognitive test performance (N = 257,841) to estimate SNP associations with variation in EA that is independent of cognitive ability. We identified 157 genome-wide significant loci and a polygenic architecture accounting for 57% of genetic variance in EA. Phenotypic and biological annotation revealed that (1) both cognitive and non-cognitive contributions to EA were genetically correlated with socioeconomic success and longevity; and (2) non-cognitive contributions to EA were related to personality, decision making, risk-behavior, and increased risk for psychiatric disorders; (3) non-cognitive and cognitive contributions to EA were enriched in the same tissues and cell types, but (4) showed different associations with gray-matter neuroimaging phenotypes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_structure_and_global_distribution_of_the_endoplasmic_reticulum_network_is_actively_regulated_by_lysosomes_74074833/">
          <h4 class="font-weight-bold">The structure and global distribution of the endoplasmic reticulum network is actively regulated by lysosomes, bioRxiv, 2020-01-16</h4>
          <p>AbstractThe endoplasmic reticulum (ER) comprises morphologically and functionally distinct domains, sheets and interconnected tubules. These domains undergo dynamic reshaping, in response to changes in the cellular environment. However, the mechanisms behind this rapid remodeling within minutes are largely unknown. Here, we report that ER remodeling is actively driven by lysosomes, following lysosome repositioning in response to changes in nutritional status. The anchorage of lysosomes to ER growth tips is critical for ER tubule elongation and connection. We validate this causal link via the chemo- and optogenetically driven re-positioning of lysosomes, which leads to both a redistribution of the ER tubules and its global morphology. Lysosomes sense metabolic change in the cell and regulate ER tubule distribution accordingly. Dysfunction in this mechanism during axonal extension may lead to axonal growth defects. Our results demonstrate a critical role of lysosome-regulated ER dynamics and reshaping in nutrient responses and neuronal development.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/theoretical_and_empirical_quantification_of_the_accuracy_of_polygenic_scores_in_ancestry_divergent_populations_74072185/">
          <h4 class="font-weight-bold">Theoretical and empirical quantification of the accuracy of polygenic scores in ancestry divergent populations, bioRxiv, 2020-01-16</h4>
          <p>AbstractPolygenic scores (PGS) have been widely used to predict complex traits and risk of diseases using variants identified from genome-wide association studies (GWASs). To date, most GWASs have been conducted in populations of European ancestry, which limits the use of GWAS-derived PGS in non-European populations. Here, we develop a new theory to predict the relative accuracy (RA, relative to the accuracy in populations of the same ancestry as the discovery population) of PGS across ancestries. We used simulations and real data from the UK Biobank to evaluate our results. We found across various simulation scenarios that the RA of PGS based on trait-associated SNPs can be predicted accurately from modelling linkage disequilibrium (LD), minor allele frequencies (MAF), cross-population correlations of SNP effect sizes and heritability. Altogether, we find that LD and MAF differences between ancestries explain alone up to ~70% of the loss of RA using European-based PGS in African ancestry for traits like body mass index and height. Our results suggest that causal variants underlying common genetic variation identified in European ancestry GWASs are mostly shared across continents.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2020">2020</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/2020/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/2020/page/6/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2020/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2020/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2020/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2020/page/6/" class="page-link">6</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/2020/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2020/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2020/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2020/page/10/" class="page-link">10</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2020/page/11/" class="page-link">11</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/2020/page/8/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/2020/page/13/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
