<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/jumping_to_conclusions_general_intelligence_and_psychosis_liability_findings_from_the_multi-centre_eu-gei_case-control_study_60365531/">
          <h4 class="font-weight-bold">Jumping To Conclusions, General Intelligence, And Psychosis Liability Findings From The Multi-Centre EU-GEI Case-Control Study, bioRxiv, 2019-05-11</h4>
          <p>AbstractBackgroundThe “jumping to conclusions” (JTC) bias is associated with both psychosis and general cognition but their relationship is unclear. In this study, we set out to clarify the relationship between the JTC bias, IQ, psychosis and polygenic liability to schizophrenia and IQ.Methods817 FEP patients and 1294 population-based controls completed assessments of general intelligence (IQ), and JTC (assessed by the number of beads drawn on the probabilistic reasoning “beads” task) and provided blood or saliva samples from which we extracted DNA and computed polygenic risk scores for IQ and schizophrenia.ResultsThe estimated proportion of the total effect of casecontrol differences on JTC mediated by IQ was 79%. Schizophrenia Polygenic Risk Score (SZ PRS) was non-significantly associated with a higher number of beads drawn (B= 0.47, 95% CI −0.21 to 1.16, p=0.17); whereas IQ PRS (B=0.51, 95% CI 0.25 to 0.76, p&amp;lt;0.001) significantly predicted the number of beads drawn, and was thus associated with reduced JTC bias. The JTC was more strongly associated with higher level of psychotic-like experiences (PLE) in controls, including after controlling for IQ (B= −1.7, 95% CI −2.8 to −0.5, p=0.006), but did not relate to delusions in patients.Conclusionsthe JTC reasoning bias in psychosis is not a specific cognitive deficit but is rather a manifestation or consequence, of general cognitive impairment. Whereas, in the general population, the JTC bias is related to psychotic-like experiences, independent of IQ. The work has potential to inform interventions targeting cognitive biases in early psychosis.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/fishing_for_mammals_landscape-level_monitoring_of_terrestrial_and_semi-aquatic_communities_using_edna_from_lotic_ecosystems_60205935/">
          <h4 class="font-weight-bold">Fishing for mammals landscape-level monitoring of terrestrial and semi-aquatic communities using eDNA from lotic ecosystems, bioRxiv, 2019-05-10</h4>
          <p>Abstract&lt;jatslist list-type=order&gt;&lt;jatslist-item&gt;Environmental DNA (eDNA) metabarcoding has revolutionised biomonitoring in both marine and freshwater ecosystems. However, for semi-aquatic and terrestrial animals, the application of this technique remains relatively untested.&lt;jatslist-item&gt;&lt;jatslist-item&gt;We first assess the efficiency of eDNA metabarcoding in detecting semi-aquatic and terrestrial mammals in natural lotic ecosystems in the UK by comparing sequence data recovered from water and sediment samples to the mammalian communities expected from historical data. Secondly, we evaluate the detection efficiency of eDNA samples compared to multiple conventional non-invasive survey methods (latrine surveys and camera trapping) using occupancy modelling.&lt;jatslist-item&gt;&lt;jatslist-item&gt;eDNA metabarcoding detected a large proportion of the expected mammalian community within each area. Common species in the areas were detected at the majority of sites. Several key species of conservation concern in the UK were detected by eDNA in areas where authenticated records do not currently exist, but potential false positives were also identified for several non-native species.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Water-based eDNA samples provided comparable results to conventional survey methods in per unit of survey effort for three species (water vole, field vole, and red deer) using occupancy models. The comparison between survey ‘effort’ to reach a detection probability of ≥0.95 revealed that 3-6 water replicates would be equivalent to 3-5 latrine surveys and 5-30 weeks of single camera deployment, depending on the species.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Synthesis and Applications. eDNA metabarcoding represents an extremely promising tool for monitoring mammals, allowing for the detection of multiple species simultaneously, and provides comparable results to widely-used conventional survey methods. eDNA from freshwater systems delivers a ‘terrestrial dividend’ by detecting both semi-aquatic and terrestrial mammalian communities, and provides a basis for future monitoring at a landscape level over larger spatial and temporal scales (i.e. long-term monitoring at national levels).&lt;jatslist-item&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/quantifying_genetic_regulatory_variation_in_human_populations_improves_transcriptome_analysis_in_rare_disease_patients_60203358/">
          <h4 class="font-weight-bold">Quantifying genetic regulatory variation in human populations improves transcriptome analysis in rare disease patients, bioRxiv, 2019-05-10</h4>
          <p>AbstractTranscriptome data holds substantial promise for better interpretation of rare genetic variants in basic research and clinical settings. Here, we introduce ANalysis of Expression VAriation (ANEVA) to quantify genetic variation in gene dosage from allelic expression (AE) data in a population. Application to GTEx data showed that this variance estimate is robust across datasets and is correlated with selective constraint in a gene. We next used ANEVA variance estimates in a Dosage Outlier Test (ANEVA-DOT) to identify genes in an individual that are affected by a rare regulatory variant with an unusually strong effect. Applying ANEVA-DOT to AE data form 70 Mendelian muscular disease patients showed high accuracy in detecting genes with pathogenic variants in previously resolved cases, and lead to one confirmed and several potential new diagnoses in cases previously unresolved. Using our reference estimates from GTEx data, ANEVA-DOT can be readily incorporated in rare disease diagnostic pipelines to better utilize RNA-seq data.One Sentence SummaryNew statistical framework for modelling allelic expression characterizes genetic regulatory variation in populations and informs diagnosis in rare disease patients</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/shake-it-off_a_simple_ultrasonic_cryo-em_specimen_preparation_device_60204793/">
          <h4 class="font-weight-bold">Shake-it-off A simple ultrasonic cryo-EM specimen preparation device, bioRxiv, 2019-05-10</h4>
          <p>AbstractAlthough microscopes and image analysis software for electron cryomicroscopy (cryo-EM) have improved dramatically in recent years, specimen preparation methods have lagged behind. Most strategies still rely on blotting microscope grids with paper to produce a thin film of solution suitable for vitrification. This approach loses more than 99.9% of the applied sample and requires several seconds, leading to problematic air-water interface interactions for macromolecules in the resulting thin film of solution and complicating time-resolved studies. Recently developed self-wicking EM grids allow use of small volumes of sample, with nanowires on the grid bars removing excess solution to produce a thin film within tens of milliseconds from sample application to freezing. Here we present a simple cryo-EM specimen preparation device that uses components from an ultrasonic humidifier to transfer protein solution onto a self-wicking EM grid. The device is controlled by a Raspberry Pi single board computer and all components are either widely available or can be manufactured by online services, allowing the device to be constructed in laboratories that specialize in cryo-EM, rather than instrument design. The simple open-source design permits straightforward customization of the instrument for specialized experiments.SynopsisA method is presented for high-speed low-volume cryo-EM specimen preparation with a device constructed from readily available components.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/stress-driven_transposable_element_de-repression_dynamics_in_a_fungal_pathogen_60230982/">
          <h4 class="font-weight-bold">Stress-driven transposable element de-repression dynamics in a fungal pathogen, bioRxiv, 2019-05-10</h4>
          <p>AbstractTransposable elements (TEs) are drivers of genome evolution and affect the expression landscape of the host genome. Stress is a major factor inducing TE activity, however the regulatory mechanisms underlying de-repression are poorly understood. Key unresolved questions are whether different types of stress differentially induce TE activity and whether different TEs respond differently to the same stress. Plant pathogens are excellent models to dissect the impact of stress on TEs, because lifestyle transitions on and off the host impose exposure to a variety of stress conditions. We analyzed the TE expression landscape of four well-characterized strains of the major wheat pathogen Zymoseptoria tritici. We experimentally exposed strains to nutrient starvation and host infection stress. Contrary to expectations, we show that the two distinct conditions induce the expression of different sets of TEs. In particular, the most highly expressed TEs, including MITE and LTR-Gypsy elements, show highly distinct de-repression across stress conditions. Both the genomic context of TEs and the genetic background stress (i.e. different strains harboring the same TEs) were major predictors of de-repression dynamics under stress. Genomic defenses inducing point mutations in repetitive regions were largely ineffective to prevent TE de-repression. Consistent with TE de-repression being governed by epigenetic effects, we found that gene expression profiles under stress varied significantly depending on the proximity to the closest TEs. The unexpected complexity in TE responsiveness to stress across genetic backgrounds and genomic locations shows that species harbor substantial genetic variation to control TEs.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_megaplasmid_family_responsible_for_dissemination_of_multidrug_resistance_in_pseudomonas_60079357/">
          <h4 class="font-weight-bold">A megaplasmid family responsible for dissemination of multidrug resistance in Pseudomonas, bioRxiv, 2019-05-08</h4>
          <p>AbstractMultidrug resistance (MDR) represents a global threat to health. Although plasmids can play an important role in the dissemination of MDR, they have not been commonly linked to the emergence of antimicrobial resistance in the pathogen Pseudomonas aeruginosa. We used whole genome sequencing to characterize a collection of P. aeruginosa clinical isolates from a hospital in Thailand. Using long-read sequence data we obtained complete sequences of two closely related megaplasmids (&amp;gt;420 kb) carrying large arrays of antibiotic resistance genes located in discrete, complex and dynamic resistance regions, and revealing evidence of extensive duplication and recombination events. A comprehensive pangenomic and phylogenomic analysis indicated that 1) these large plasmids comprise a family present in different members of the Pseudomonas genus and associated with multiple sources (geographical, clinical or environmental); 2) the megaplasmids encode diverse niche-adaptive accessory traits, including multidrug resistance; 3) the pangenome of the megaplasmid family is highly flexible and diverse, comprising a substantial core genome (average of 48% of plasmid genes), but with individual members carrying large numbers of unique genes. The history of the megaplasmid family, inferred from our analysis of the available database, suggests that members carrying multiple resistance genes date back to at least the 1970s.FundingThis work was supported by the International Pseudomonas Genomics Consortium, funded by Cystic Fibrosis Canada [RCL]; and the Secretaría de Educación, Ciencia, Tecnología e Innovación (SECTEI), Mexico [AC].</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/0-100-users/page/27/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/24/" class="page-link">24</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/25/" class="page-link">25</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/26/" class="page-link">26</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/27/" class="page-link">27</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/0-100-users/page/28/" class="page-link">28</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/29/" class="page-link">29</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/30/" class="page-link">30</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/31/" class="page-link">31</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/32/" class="page-link">32</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/0-100-users/page/29/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/0-100-users/page/123/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
