<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/causal_relationships_between_obesity_and_the_leading_causes_of_death_in_women_and_men_54531769/">
          <h4 class="font-weight-bold">Causal relationships between obesity and the leading causes of death in women and men, bioRxiv, 2019-01-26</h4>
          <p>AbstractObesity traits are causally implicated with risk of cardiometabolic diseases. It remains unclear whether there are similar causal effects of obesity traits on other non-communicable diseases. Also, it is largely unexplored whether there are any sex-specific differences in the causal effects of obesity traits on cardiometabolic diseases and other leading causes of death.We constructed sex-specific genetic risk scores (GRS) for three obesity traits; body mass index (BMI), waist-hip ratio (WHR), and WHR adjusted for BMI, including 565, 324, and 337 genetic variants, respectively. These GRSs were then used as instrumental variables to assess associations between the obesity traits and leading causes of mortality in the UK Biobank using Mendelian randomization. We also investigated associations with potential mediators, including smoking, glycemic and blood pressure traits. Sex-differences were subsequently assessed by Cochran’s Q-test (Phet).A Mendelian randomization analysis of 228,466 women and 195,041 men showed that obesity causes coronary artery disease, stroke (particularly ischemic), chronic obstructive pulmonary disease, lung cancer, type 2 and 1 diabetes mellitus, non-alcoholic fatty liver disease, chronic liver disease, and acute and chronic renal failure. Higher BMI led to higher risk of type 2 diabetes in women than in men (Phet=1.4×10-5). Waist-hip-ratio led to a higher risk of chronic obstructive pulmonary disease (Phet=3.7×10-6) and higher risk of chronic renal failure (Phet=1.0×10-4) in men than women.Obesity traits have an etiological role in the majority of the leading global causes of death. Sex differences exist in the effects of obesity traits on risk of type 2 diabetes, chronic obstructive pulmonary disease, and renal failure, which may have downstream implications for public health.Author summaryObesity is increasing globally and has been linked to major causes of death, such as diabetes and heart disease. Still, the causal effects of obesity on other leading causes of death is relatively unexplored. It is also unclear if any such effects differ between men and women.Mendelian randomization is a method that explores causal relationships between traits using genetic data. Using Mendelian randomization, we investigated the effects of obesity traits on leading causes of death and assessed if any such effects differ between men and women.We found that obesity increases the risks of heart disease, stroke, chronic obstructive pulmonary disease, lung cancer, diabetes, kidney disease, non-alcoholic fatty liver disease and chronic liver disease. Higher body mass index led to a higher risk of type 2 diabetes in women than in men, whereas a higher waist-hip ratio increased risks of chronic obstructive pulmonary disease and chronic kidney disease more in men than in women.In summary, obesity traits are causally involved in the majority of the leading causes of death, and some obesity traits affect disease risk differently in men and women. This has potential implications for public health strategies and indicates that sex-specific preventative measure may be needed.AbbreviationsBMI, Body mass index; CAD coronary artery disease; CLD, chronic liver disease; COPD, chronic obstructive pulmonary disease; DBP, diastolic blood pressure; FG, fasting glucose; FI, fasting insulin; GIANT, Genetic Investigation of ANthropometric Traits; GRS, genetic risk score; GWAS, genomewide association study; MAGIC, the Meta-Analyses of Glucose and Insulin-related traits Consortium; MR, Mendelian randomization; NAFLD, non-alcoholic fatty liver disease; OR, odds ratio; T1D, type 1 diabetes; T2D, type 2 diabetes; SBP, systolic blood pressure; SD, standard deviation; SNP, single nucleotide polymorphism; WHO, the World Health Organization; WHR, waist-hip-ratio; WHRadjBMI, waist-hip-ratio adjusted for body mass index.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/epidemiology">epidemiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/rna_interactions_with_ctcf_are_essential_for_its_proper_function_supplemental_figures_1-5_54452778/">
          <h4 class="font-weight-bold">RNA interactions with CTCF are essential for its proper function Supplemental Figures 1-5, bioRxiv, 2019-01-25</h4>
          <p>The function of the CCCTC-binding factor (CTCF) in the organization of the genome has become an important area of investigation, but the mechanisms of how CTCF dynamically contributes to genome organization is not clear. We previously discovered that CTCF binds to large numbers of endogenous RNAs; promoting its oligomerization. Here we found that inhibition of transcription or interfering with CTCF ability to bind RNA through mutations of two of its 11 zinc fingers that are not involved with CTCF binding to its cognate site in vitro, zinc finger-1 (ZF1) or -10 (ZF10), disrupt CTCF association to chromatin. These mutations alter gene expression profiles as CTCF mutants lose their ability to promote local insulation. Our results highlight the importance of RNA as a structural component of the genome, in part by affecting the association of CTCF with chromatin and likely its interaction with other factors.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/simultaneous_quantification_of_protein-dna_contacts_and_transcriptomes_in_single_cells_supplemental_figures_54446601/">
          <h4 class="font-weight-bold">Simultaneous quantification of protein-DNA contacts and transcriptomes in single cells Supplemental Figures, bioRxiv, 2019-01-25</h4>
          <p>The epigenome plays a critical role in regulating gene expression in mammalian cells. However, understanding how cell-to-cell heterogeneity in the epigenome influences gene expression variability remains a major challenge. Here we report a novel method for simultaneous single-cell quantification of protein-DNA contacts with DamID and transcriptomics (scDamID&amp;amp;T). This method enables quantifying the impact of protein-DNA contacts on gene expression from the same cell. By profiling lamina-associated domains (LADs) in human cells, we reveal different dependencies between genome-nuclear lamina (NL) association and gene expression in single cells. In addition, we introduce the E. coli methyltransferase, Dam, as an in vivo marker of chromatin accessibility in single cells and show that scDamID&amp;amp;T can be utilized as a general technology to identify cell types in silico while simultaneously determining the underlying gene-regulatory landscape. With this strategy the effect of chromatin states, transcription factor binding, and genome organization on the acquisition of cell-type specific transcriptional programs can be quantified.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/accurate_estimation_of_snp-heritability_from_biobank-scale_data_irrespective_of_genetic_architecture_54380447/">
          <h4 class="font-weight-bold">Accurate estimation of SNP-heritability from biobank-scale data irrespective of genetic architecture, bioRxiv, 2019-01-24</h4>
          <p>AbstractThe proportion of phenotypic variance attributable to the additive effects of a given set of genotyped SNPs (i.e. SNP-heritability) is a fundamental quantity in the study of complex traits. Recent works have shown that existing methods to estimate genome-wide SNP-heritability often yield biases when their assumptions are violated. While various approaches have been proposed to account for frequency- and LD-dependent genetic architectures, it remains unclear which estimates of SNP-heritability reported in the literature are reliable. Here we show that genome-wide SNP-heritability can be accurately estimated from biobank-scale data irrespective of the underlying genetic architecture of the trait, without specifying a heritability model or partitioning SNPs by minor allele frequency andor LD. We use theoretical justifications coupled with extensive simulations starting from real genotypes from the UK Biobank (N = 337K) to show that, unlike existing methods, our closed-form estimator for SNP-heritability is highly accurate across a wide range of architectures. We provide estimates of SNP-heritability for 22 complex traits and diseases in the UK Biobank and show that, consistent with our results in simulations, existing biobank-scale methods yield estimates up to 30% different from our theoretically-justified approach.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/behavior_influences_range_limits_and_patterns_of_coexistence_across_an_elevational_gradient_in_tropical_bird_diversity_54385470/">
          <h4 class="font-weight-bold">Behavior influences range limits and patterns of coexistence across an elevational gradient in tropical bird diversity, bioRxiv, 2019-01-24</h4>
          <p>Does competition influence patterns of coexistence between closely related taxa? Here we address this basic question in ecology by analyzing patterns of range overlap between related bird species (sister pairs) distributed along a Neotropical elevational gradient. We explicitly contrast the behavioral dimension of interspecific competition (interference competition) with similarity in resource acquisition traits (exploitative competition). We find that behavioral interactions are generally important in setting elevational range limits and preventing coexistence of closely related species. Specifically, close relatives that defend year-round territories tend to live in non-overlapping elevational distributions, while close relatives that do not defend territories tend to broadly overlap in distribution. In contrast, neither similarity in beak morphology nor evolutionary relatedness was associated with patterns of range limitation. Our main result is that interference competition can be an important driver of species ranges at the scale of entire diverse assemblages. Consequently, we suggest that behavioral dimensions of the niche should be more broadly incorporated in macroecological studies.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/mobile_genetic_element_insertions_drive_antibiotic_resistance_across_pathogens_54409093/">
          <h4 class="font-weight-bold">Mobile genetic element insertions drive antibiotic resistance across pathogens, bioRxiv, 2019-01-24</h4>
          <p>Mobile genetic elements contribute to bacterial adaptation and evolution; however, detecting these elements in a high-throughput and unbiased manner remains challenging. Here, we demonstrate a de novo approach to identify mobile elements from short-read sequencing data. The method identifies the precise site of mobile element insertion and infers the identity of the inserted sequence. This is an improvement over previous methods that either rely on curated databases of known mobile elements or rely on &#39;split-read&#39; alignments that assume the inserted element exists within the reference genome. We apply our approach to 12,419 sequenced isolates of nine prevalent bacterial pathogens, and we identify hundreds of known and novel mobile genetic elements, including many candidate insertion sequences. We find that the mobile element repertoire and insertion rate vary considerably across species, and that many of the identified mobile elements are biased toward certain target sequences, several of them being highly specific. Mobile element insertion hotspots often cluster near genes involved in mechanisms of antibiotic resistance, and such insertions are associated with antibiotic resistance in laboratory experiments and clinical isolates. Finally, we demonstrate that mutagenesis caused by these mobile elements contributes to antibiotic resistance in a genome-wide association study of mobile element insertions in pathogenic Escherichia coli. In summary, by applying a de novo approach to precisely identify mobile genetic elements and their insertion sites, we thoroughly characterize the mobile element repertoire and insertion spectrum of nine pathogenic bacterial species and find that mobile element insertions play a significant role in the evolution of clinically relevant phenotypes, such as antibiotic resistance.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/27/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/24/" class="page-link">24</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/25/" class="page-link">25</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/26/" class="page-link">26</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/27/" class="page-link">27</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/0-100-users/page/28/" class="page-link">28</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/29/" class="page-link">29</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/30/" class="page-link">30</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/31/" class="page-link">31</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/32/" class="page-link">32</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/29/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/64/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
