<!DOCTYPE html>
<html><head>
  
  <title>plant-biology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.68.3" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.95d2483cd3c5d2bf47c1f36307a456787d60ad958c5efe44db9afc47012b1b05b8e1f64f6ac291650439322a6c38e86eb7cd6254255516d857c23f299fb0e666.js" integrity="sha512-ldJIPNPF0r9HwfNjB6RWeH1grZWMXv5E25r8RwErGwW44fZPasKRZQQ5MipsOOhut81iVCVVFthXwj8pn7DmZg=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/chloroplasts_navigate_towards_the_pathogen_interface_to_counteract_infection_by_the_irish_potato_famine_pathogen_53711731/">
          <h4 class="font-weight-bold">Chloroplasts navigate towards the pathogen interface to counteract infection by the Irish potato famine pathogen, bioRxiv, 2019-01-10</h4>
          <p>Chloroplasts are light harvesting organelles that arose from ancient endosymbiotic cyanobacteria. Upon immune activation, chloroplasts switch off photosynthesis, produce anti-microbial compounds, and develop tubular extensions called stromules. We report that chloroplasts navigate to the pathogen interface to counteract infection by the Irish potato famine pathogen Phytophthora infestans, physically associating with the specialised membrane that engulfs pathogen haustoria. Outer envelope protein, chloroplast unusual positioning1 (CHUP1), anchors chloroplasts to the host-pathogen interface. Stromules are induced during infection in a CHUP1-dependent manner, embracing haustoria and interconnecting chloroplasts, to form dynamic organelle clusters. Infection-triggered reprogramming of chloroplasts relies on surface immune signalling, whereas pathogen effectors subvert these immune pulses. Chloroplast are deployed focally, and coordinate to restrict pathogen entry into plant cells, a process actively countered by parasite effectors.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/plants_emit_informative_airborne_sounds_under_stress_53244340/">
          <h4 class="font-weight-bold">Plants emit informative airborne sounds under stress, bioRxiv, 2018-12-29</h4>
          <p>Stressed plants show altered phenotypes, including changes in color, smell, and shape. Yet, the possibility that plants emit airborne sounds when stressed, similarly to many animals, has not been investigated. Here we show, to our knowledge for the first time, that stressed plants emit airborne sounds that can be recorded remotely, both in acoustic chambers and in greenhouses. We recorded ~65 dBSPL ultrasonic sounds 10 cm from tomato and tobacco plants, implying that these sounds could be detected by some organisms from up to several meters away. We developed machine learning models that were capable of distinguishing between plant sounds and general noises, and identifying the condition of the plants dry, cut, or intact, based solely on the emitted sounds. Our results suggest that animals, humans, and possibly even other plants, could use sounds emitted by a plant to gain information about the plant&#39;s condition. More investigation on plant bioacoustics in general and on sound emission in plants in particular may open new avenues for understanding plants and their interactions with the environment, and it may also have a significant impact on agriculture.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genetic_determination_of_stomatal_patterning_in_winter_wheat_triticum_aestivum_l._52515038/">
          <h4 class="font-weight-bold">Genetic determination of stomatal patterning in winter wheat (Triticum aestivum L.), bioRxiv, 2018-12-11</h4>
          <p>Leaf stomata are microscopic pores mediating plant-environment interactions. Their role in carbon uptake and transpiration make them prime candidates for improving water use efficiency (WUE). Stomatal density (SD), the number of stomata per unit area, has been shown to be negatively correlated with WUE. However, little is known about the genetic basis of SD in wheat (Triticum aestivum L.), and to what extant genetic variation exists in contemporary wheat germplasm. Here, we evaluated stomatal patterning over two growing seasons in a set of 333 wheat lines, representing the European winter wheat germplasm. Stomatal patterning was mainly determined by two underlying traits, the distance between files of stomata and the distance between stomata within a file. By haplotype association mapping, quantitative trait loci for SD were consistently detected in both seasons on wheat chromosomes (CHR) 2A, 3A and 7B. The single nucleotide polymorphism markers most significantly associated with SD coincided with the genes INDUCER OF CBF EXPRESSION 1 (ICE1) and STOMATAL CYTOKINESIS-DEFECTIVE 1 (SCD1) on CHR 3A, and genes involved in ethylene and auxin signaling on CHR 2A and 7B, respectively. Our study unlocks the phenotypic and genotypic variation for stomatal patterning traits in contemporary wheat germplasm. It provides gene targets for functional validation and practical tools to manipulate SD using marker-assisted selection for crop improvement.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/crispr-tsko_facilitates_efficient_cell_type-_tissue-_or_organ-specific_mutagenesis_in_arabidopsis_51476356/">
          <h4 class="font-weight-bold">CRISPR-TSKO facilitates efficient cell type-, tissue-, or organ-specific mutagenesis in Arabidopsis, bioRxiv, 2018-11-21</h4>
          <p>AbstractDetailed functional analyses of many fundamentally-important plant genes via conventional loss-of-function approaches are impeded by severe pleiotropic phenotypes. In particular, mutations in genes that are required for basic cellular functions andor reproduction often interfere with the generation of homozygous mutant plants, precluding further functional studies. To overcome this limitation, we devised a CRISPR-based tissue-specific knockout system, CRISPR-TSKO, enabling the generation of somatic mutations in particular plant cell types, tissues, and organs. In Arabidopsis, CRISPR-TSKO mutations in essential genes caused well-defined, localized phenotypes in the root cap, stomatal lineage, or entire lateral roots. The underlying modular cloning system allows for efficient selection, identification, and functional analysis of mutant lines directly in the first transgenic generation. The efficacy of CRISPR-TSKO opens new avenues to discover and analyze gene functions in spatial and temporal contexts of plant life while avoiding pleiotropic effects of system-wide loss of gene function.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/plant_extracellular_vesicles_contain_diverse_small_rna_species_and_are_enriched_in_10_to_17_nucleotide_tiny_rnas_51342374/">
          <h4 class="font-weight-bold">Plant Extracellular Vesicles Contain Diverse Small RNA Species and Are Enriched in 10 to 17 Nucleotide “Tiny” RNAs, bioRxiv, 2018-11-17</h4>
          <p>ABSTRACTSmall RNAs (sRNAs) that are 21 to 24 nucleotides (nt) in length are found in most eukaryotic organisms and regulate numerous biological functions, including transposon silencing, development, reproduction, and stress responses, typically via control of the stability andor translation of target mRNAs. Major classes of sRNAs in plants include microRNAs (miRNAs) and small interfering RNAs (siRNAs); sRNAs are known to travel as a silencing signal from cell to cell, root to shoot, and even between host and pathogen. In mammals, sRNAs are transported inside extracellular vesicles (EVs), which are mobile lipid compartments that participate in intercellular communication. In addition to sRNAs, EVs carry proteins, lipids, metabolites, and potentially other types of nucleic acids. Here we report that plant EVs also contain diverse species of sRNA. We found that specific miRNAs and siRNAs are preferentially loaded into plant EVs. We also report a previously overlooked class of “tiny RNAs” (10 to 17 nt) that are highly enriched in EVs. This new RNA category of unknown function has a broad and very diverse genome origin and might correspond to degradation products.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/pathogen_detection_and_microbiome_analysis_of_infected_wheat_using_a_portable_dna_sequencer_49017775/">
          <h4 class="font-weight-bold">Pathogen Detection and Microbiome Analysis of Infected Wheat Using a Portable DNA Sequencer, bioRxiv, 2018-09-29</h4>
          <p>ABSTRACTFungal diseases of plants are responsible for major losses in agriculture, highlighting the need for rapid and accurate identification of plant pathogens. Disease outcomes are often defined not only by the main pathogen but are influenced by diverse microbial communities known as the microbiome at sites of infection. Here we present the first use of whole genome shot-gun sequencing with a portable DNA sequencing device as a method for the detection of fungal pathogens from wheat (Triticum aestivum) in a standard molecular biology laboratory. The data revealed that our method is robust and applicable to the diagnosis of fungal diseases including wheat stripe rust (caused by Puccinia striiformis f. sp. tritici), septoria tritici blotch (caused by Zymoseptoria tritici) and yellow leaf spot (caused by Pyrenophora tritici repentis). We also identified the bacterial genus Pseudomonas co-present with Puccinia and Zymoseptoria but not Pyrenophora infections. One limitation of the method is the over-representation of redundant wheat genome sequences from samples. This could be addressed by long-range amplicon-based sequencing approaches in future studies, which specifically target non-host organisms. Our work outlines a new approach for detection of a broad range of plant pathogens and associated microbes using a portable sequencer in a standard laboratory, providing the basis for future development of an on-site disease monitoring system.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/4/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/4/" class="page-link">4</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/8/" class="page-link">8</a></li>
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/6/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/8/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
