<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/rootnav_2.0_deep_learning_for_automatic_navigation_of_complex_plant_root_architectures_63809339/">
          <h4 class="font-weight-bold">RootNav 2.0 Deep Learning for Automatic Navigation of Complex Plant Root Architectures, bioRxiv, 2019-07-20</h4>
          <p>AbstractWe present a new image analysis approach that provides fully-automatic extraction of complex root system architectures from a range of plant species in varied imaging setups. Driven by modern deep-learning approaches, RootNav 2.0 replaces previously manual and semi-automatic feature extraction with an extremely deep multi-task Convolutional Neural Network architecture. The network has been designed to explicitly combine local pixel information with global scene information in order to accurately segment small root features across high-resolution images. In addition, the network simultaneously locates seeds, and first and second order root tips to drive a search algorithm seeking optimal paths throughout the image, extracting accurate architectures without user interaction. The proposed method is evaluated on images of wheat (Triticum aestivum L.) from a seedling assay. The results are compared with semi-automatic analysis via the original RootNav tool, demonstrating comparable accuracy, with a 10-fold increase in speed. We then demonstrate the ability of the network to adapt to different plant species via transfer learning, offering similar accuracy when transferred to an Arabidopsis thaliana plate assay. We transfer for a final time to images of Brassica napus from a hydroponic assay, and still demonstrate good accuracy despite many fewer training images. The tool outputs root architectures in the widely accepted RSML standard, for which numerous analysis packages exist (&lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httprootsystemml.github.io&gt;httprootsystemml.github.io&lt;jatsext-link&gt;), as well as segmentation masks compatible with other automated measurement tools.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genome_sequence_of_the_cluster_root_forming_white_lupin_63775319/">
          <h4 class="font-weight-bold">Genome sequence of the cluster root forming white lupin, bioRxiv, 2019-07-19</h4>
          <p>White lupin (Lupinus albus L.) is a legume that produces seeds recognized for their high protein content and good nutritional value (lowest glycemic index of all grains, high dietary fiber content, and zero gluten or starch)1–5. White lupin can form nitrogen-fixing nodules but has lost the ability to form mycorrhizal symbiosis with fungi6. Nevertheless, its root system is well adapted to poor soils it produces cluster roots, constituted of dozens of determinate lateral roots that improve soil exploration and phosphate remobilization7. As phosphate is a limited resource that comes from rock reserves8, the production of cluster roots is a trait of interest to improve fertilizers efficiency. Using long reads sequencing technologies, we provide a high-quality genome sequence of a modern variety of white lupin (2n=50, 451 Mb), as well as de novo assemblies of a landrace and a wild relative. We describe how domestication impacted soil exploration capacity through the early establishment of lateral and cluster roots. We identify the APETALA2 transcription factor LaPUCHI-1, homolog of the Arabidopsis morphogenesis coordinator9, as a potential regulator of this trait. Our high-quality genome and companion genomic and transcriptomic resources enable the development of modern breeding strategies to increase and stabilize yield and to develop new varieties with reduced allergenic properties (caused by conglutins10), which would favor the deployment of this promising culture.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/%CE%BCdamid_a_microfluidic_approach_for_imaging_and_sequencing_protein-dna_interactions_in_single_cells_63731175/">
          <h4 class="font-weight-bold">μDamID a microfluidic approach for imaging and sequencing protein-DNA interactions in single cells, bioRxiv, 2019-07-19</h4>
          <p>AbstractGenome regulation depends on carefully programmed protein-DNA interactions that maintain or alter gene expression states, often by influencing chromatin organization. Most studies of these interactions to date have relied on bulk methods, which in many systems cannot capture the dynamic single-cell nature of these interactions as they modulate cell states. One method allowing for sensitive single-cell mapping of protein-DNA interactions is DNA adenine methyltransferase identification (DamID), which records a protein’s DNA-binding history by methylating adenine bases in its vicinity, then selectively amplifies and sequences these methylated regions. These interaction sites can also be visualized using fluorescent proteins that bind to methyladenines. Here we combine these imaging and sequencing technologies in an integrated microfluidic platform (μDamID) that enables single-cell isolation, imaging, and sorting, followed by DamID. We apply this system to generate paired single-cell imaging and sequencing data from a human cell line, in which we map and validate interactions between DNA and nuclear lamina proteins, providing a measure of 3D chromatin organization and broad gene regulation patterns. μDamID provides the unique ability to compare paired imaging and sequencing data for each cell and between cells, enabling the joint analysis of the nuclear localization, sequence identity, and variability of protein-DNA interactions.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/bioengineering">bioengineering</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/peripheral_blood_cell_immunophenotyping_reveals_distinct_subgroups_of_inflamed_depression_63721395/">
          <h4 class="font-weight-bold">Peripheral blood cell immunophenotyping reveals distinct subgroups of inflamed depression, bioRxiv, 2019-07-18</h4>
          <p>AbstractDepression has been associated with increased inflammatory proteins but changes in circulating immune cells are less well defined. We used multi-parametric flow cytometry to investigate 14 subsets of peripheral blood cells in 206 cases of major depressive disorder (MDD) and 77 age- and sex-matched controls. There were significant case-control differences, by univariate and multivariate analysis cases showed increased immune cell counts, especially neutrophils, CD4&#43; T cells and monocytes, and increased inflammatory proteins (C-reactive protein and interleukin-6). Within-group analysis demonstrated significant association between the severity of depressive symptoms and increased myeloid and CD4&#43; cell counts. MDD cases could be partitioned into two groups by forced binary clustering of cell counts the inflamed depression group (N=81 out of 206; 39%) had increased monocyte, CD4&#43; and neutrophil counts, increased C-reactive protein (CRP) and interleukin 6 (IL-6), and was more depressed than the uninflamed majority of cases. Relaxing the presumption of a binary classification, data-driven clustering identified four subgroups of MDD cases two of these subgroups (N=38 and N=100; 67% collectively) were associated with increased inflammatory proteins and more severe depression, but differed from each other in the relative weighting of myeloid and lymphoid cell counts. Case-control and within-group results were robust to statistical control for the potentially confounding effects of age, sex, BMI, recent infection status, and tobacco use. Peripheral blood immunophenotyping can be used to identify a candidate cellular biomarker of inflamed depression, and to further decompose that binary partition, suggesting that there is more than one mechanistic pathway underlying inflamed depression.One Sentence SummaryTwo subgroups of depressed cases (about two-thirds of all 206 cases) were identified by peripheral blood biomarker evidence of distinctive cellular immunophenotypes, biased towards the myeloid or lymphoid lineages in different subgroups, but consistently associated with increased blood concentrations of inflammatory proteins and greater severity of depressive symptoms.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/dna_mismatches_reveal_widespread_conformational_penalties_in_protein-dna_recognition_63670576/">
          <h4 class="font-weight-bold">DNA mismatches reveal widespread conformational penalties in protein-DNA recognition, bioRxiv, 2019-07-17</h4>
          <p>ABSTRACTTranscription-factor (TF) proteins recognize specific genomic sequences, despite an overwhelming excess of non-specific DNA, to regulate complex gene expression programs1–3. While there have been significant advances in understanding how DNA sequence and shape contribute to recognition, some fundamental aspects of protein-DNA binding remain poorly understood2,3. Many DNA-binding proteins induce changes in the DNA structure outside the intrinsic B-DNA envelope. How the energetic cost associated with distorting DNA contributes to recognition has proven difficult to study and measure experimentally because the distorted DNA structures exist as low-abundance conformations in the naked B-DNA ensemble4–10. Here, we use a novel high-throughput assay called SaMBA (Saturation Mismatch-Binding Assay) to investigate the role of DNA conformational penalties in TF-DNA recognition. The approach introduces mismatched base-pairs (i.e. mispairs) within TF binding sites to pre-induce a variety of DNA structural distortions much larger than those induced by changes in Watson-Crick sequence. Strikingly, while most mismatches either weakened TF binding (~70%) or had negligible effects (~20%), approximately 10% of mismatches increased binding and at least one mismatch was found that increased the binding affinity for each of 21 examined TFs. Mismatches also converted sites from the non-specific affinity range into specific sites, and high-affinity sites into “super-sites” stronger than any known canonical binding site. These findings reveal a complex binding landscape that cannot be explained based on DNA sequence alone. Analysis of crystal structures together with NMR and molecular dynamics simulations revealed that many of the mismatches that increase binding induce distortions similar to those induced by TF binding, thus pre-paying some of the energetic cost to deform the DNA. Our work indicates that conformational penalties are a major determinant of protein-DNA recognition, and reveals mechanisms by which mismatches can recruit TFs and thus modulate replication and repair activities in the cell11,12.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/gene_networks_with_transcriptional_bursting_recapitulate_rare_transient_coordinated_expression_states_in_cancer_63645752/">
          <h4 class="font-weight-bold">Gene networks with transcriptional bursting recapitulate rare transient coordinated expression states in cancer, bioRxiv, 2019-07-17</h4>
          <p>SUMMARYNon-genetic transcriptional variability at the single-cell level is a potential mechanism for therapy resistance in melanoma. Specifically, rare subpopulations of melanoma cells occupy a transient pre-resistant state characterized by coordinated high expression of several genes. Importantly, these rare cells are able to survive drug treatment and develop resistance. How might these extremely rare states arise and disappear within the population? It is unclear whether the canonical stochastic models of probabilistic transcriptional pulsing can explain this behavior, or if it requires special, hitherto unidentified molecular mechanisms. Here we use mathematical modeling to show that a minimal network comprising of transcriptional bursting and interactions between genes can give rise to rare coordinated high states. We next show that although these states occur across networks of different sizes, they depend strongly on three (out of seven) model parameters and require network connectivity to be ≤ 6. Interestingly, we find that while entry into the rare coordinated high state is initiated by a long transcriptional burst that also triggers entry of other genes, the exit from it occurs through the independent inactivation of individual genes. Finally, our model predicts that increased network connectivity can lead to transcriptionally stable states, which we verify using network inference analysis of experimental data. In sum, we demonstrate that established principles of gene regulation are sufficient to describe this new class of rare cell variability and argue for its general existence in other biological contexts.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/16/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/13/" class="page-link">13</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/14/" class="page-link">14</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/15/" class="page-link">15</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/16/" class="page-link">16</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/17/" class="page-link">17</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/18/" class="page-link">18</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/19/" class="page-link">19</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/20/" class="page-link">20</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/21/" class="page-link">21</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/18/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/123/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
