<!DOCTYPE html>
<html><head>
  
  <title>100 200 Users | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/a_positively_selected_common_missense_variant_in_fbn1_confers_a_2.2_centimeter_reduction_of_height_in_the_peruvian_population_56071057/">
          <h4 class="font-weight-bold">A positively selected, common, missense variant in FBN1 confers a 2.2 centimeter reduction of height in the Peruvian population, bioRxiv, 2019-02-26</h4>
          <p>Peruvians are among the shortest people in the world. To understand the genetic basis of short stature in Peru, we examined an ethnically diverse group of Peruvians and identified a novel, population-specific, missense variant in FBN1 (E1297G) that is significantly associated with lower height in the Peruvian population. Each copy of the minor allele (frequency = 4.7%) reduces height by 2.2 cm (4.4 cm in homozygous individuals). This is the largest effect size known for a common height-associated variant. This variant shows strong evidence of positive selection within the Peruvian population and is significantly more frequent in Native American populations from coastal regions of Peru compared to populations from the Andes or the Amazon, suggesting that short stature in Peruvians is the result of adaptation to the coastal environment.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/brain_aging_in_major_depressive_disorder_results_from_the_enigma_major_depressive_disorder_working_group_56073512/">
          <h4 class="font-weight-bold">Brain Aging in Major Depressive Disorder Results from the ENIGMA Major Depressive Disorder working group, bioRxiv, 2019-02-26</h4>
          <p>Background Major depressive disorder (MDD) is associated with an increased risk of brain atrophy, aging-related diseases, and mortality. We examined potential advanced brain aging in MDD patients, and whether this process is associated with clinical characteristics in a large multi-center international dataset. Methods We performed a mega-analysis by pooling brain measures derived from T1-weighted MRI scans from 29 samples worldwide. Normative brain aging was estimated by predicting chronological age (10-75 years) from 7 subcortical volumes, 34 cortical thickness and 34 surface area, lateral ventricles and total intracranial volume measures separately in 1,147 male and 1,386 female controls from the ENIGMA MDD working group. The learned model parameters were applied to 1,089 male controls and 1,167 depressed males, and 1,326 female controls and 2,044 depressed females to obtain independent unbiased brain-based age predictions. The difference between predicted brain age and chronological age was calculated to indicate brain predicted age difference (brain-PAD). Findings On average, MDD patients showed a higher brain-PAD of &#43;0.90 (SE 0.21) years (Cohen&#39;s d=0.12, 95% CI 0.06-0.17) compared to controls. Relative to controls, first-episode and currently depressed patients showed higher brain-PAD (&#43;1.2 [0.3] years), and the largest effect was observed in those with late-onset depression (&#43;1.7 [0.7] years). In addition, higher brain-PAD was associated with higher self-reported depressive symptomatology (b=0.05, p=0.004). Interpretation This highly powered collaborative effort showed subtle patterns of abnormal structural brain aging in MDD. Substantial within-group variance and overlap between groups were observed. Longitudinal studies of MDD and somatic health outcomes are needed to further assess the predictive value of these brain-PAD estimates.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/measuring_single_cell_divisions_in_human_cancers_from_multi-region_sequencing_data_56045617/">
          <h4 class="font-weight-bold">Measuring single cell divisions in human cancers from multi-region sequencing data, bioRxiv, 2019-02-26</h4>
          <p>Cancer is driven by complex evolutionary dynamics involving billions of cells. Increasing effort has been dedicated to sequence single tumour cells, but obtaining robust measurements remains challenging. Here we show that multi-region sequencing of bulk tumour samples contains quantitative information on single-cell divisions that is accessible if combined with evolutionary theory. Using high-throughput data from 16 human cancers, we measured the in vivo per-cell point mutation rate (mean 1.69*10^(-8) bp per cell division) and per-cell survival rate (mean 0.57) in individual patient tumours from colon, lung and renal cancers. Per-cell mutation rates varied 50-fold between individuals, and per-cell survival rates were between nearly-homeostatic and almost perfect cell doublings, equating to tumour ages between 1 and 19 years. Furthermore, reanalysing a recent dataset of 89 whole-genome sequenced healthy haematopoietic stem cells, we find 1.14 mutations per genome per cell division and near perfect cell doublings (per-cell survival rate 0.96) during early haematopoietic development. Our analysis measures in vivo the most fundamental properties of human cancer and healthy somatic evolution at single-cell resolution within single individuals.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cancer-biology">cancer-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/accurate_inference_of_tree_topologies_from_multiple_sequence_alignments_using_deep_learning_55999685/">
          <h4 class="font-weight-bold">Accurate inference of tree topologies from multiple sequence alignments using deep learning, bioRxiv, 2019-02-25</h4>
          <p>AbstractReconstructing the phylogenetic relationships between species is one of the most formidable tasks in evolutionary biology. Multiple methods exist to reconstruct phylogenetic trees, each with their own strengths and weaknesses. Both simulation and empirical studies have identified several “zones” of parameter space where accuracy of some methods can plummet, even for four-taxon trees. Further, some methods can have undesirable statistical properties such as statistical inconsistency andor the tendency to be positively misleading (i.e. assert strong support for the incorrect tree topology). Recently, deep learning techniques have made inroads on a number of both new and longstanding problems in biological research. Here we designed a deep convolutional neural network (CNN) to infer quartet topologies from multiple sequence alignments. This CNN can readily be trained to make inferences using both gapped and ungapped data. We show that our approach is highly accurate on simulated data, often outperforming traditional methods, and is remarkably robust to bias-inducing regions of parameter space such as the Felsenstein zone and the Farris zone. We also demonstrate that the confidence scores produced by our CNN can more accurately assess support for the chosen topology than bootstrap and posterior probability scores from traditional methods. While numerous practical challenges remain, these findings suggest that deep learning approaches such as ours have the potential to produce more accurate phylogenetic inferences.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/antibiotic_production_in_streptomyces_is_organized_by_a_division_of_labour_through_terminal_genomic_differentiation_56033162/">
          <h4 class="font-weight-bold">Antibiotic production in Streptomyces is organized by a division of labour through terminal genomic differentiation, bioRxiv, 2019-02-25</h4>
          <p>AbstractOne of the hallmark behaviors of social groups is division of labour, where different group members become specialized to carry out complementary tasks. By dividing labour, cooperative groups of individuals increase their efficiency, thereby raising group fitness even if these specialized behaviors reduce the fitness of individual group members. Here we provide evidence that antibiotic production in colonies of the multicellular bacterium Streptomyces coelicolor is coordinated by a division of labour. We show that S. coelicolor colonies are genetically heterogeneous due to massive amplifications and deletions to the chromosome. Cells with gross chromosomal changes produce an increased diversity of secondary metabolites and secrete significantly more antibiotics; however, these changes come at the cost of dramatically reduced individual fitness, providing direct evidence for a trade-off between secondary metabolite production and fitness. Finally, we show that colonies containing mixtures of mutant strains and their parents produce significantly more antibiotics, while colony-wide spore production remains unchanged. Our work demonstrates that by generating mutants that are specialized to hyper-produce antibiotics, streptomycetes reduce the colony-wide fitness costs of secreted secondary metabolites while maximizing the yield and diversity of these products.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/biological_and_clinical_insights_from_genetics_of_insomnia_symptoms_56029109/">
          <h4 class="font-weight-bold">Biological and clinical insights from genetics of insomnia symptoms, Nature Genetics, 2019-02-25</h4>
          <p>Insomnia is a common disorder linked with adverse long-term medical and psychiatric outcomes. The underlying pathophysiological processes and causal relationships of insomnia with disease are poorly understood. Here we identified 57 loci for self-reported insomnia symptoms in the UK Biobank (n = 453,379) and confirmed their effects on self-reported insomnia symptoms in the HUNT Study (n = 14,923 cases and 47,610 controls), physician-diagnosed insomnia in the Partners Biobank (n = 2,217 cases and 14,240 controls), and accelerometer-derived measures of sleep efficiency and sleep duration in the UK Biobank (n = 83,726). Our results suggest enrichment of genes involved in ubiquitin-mediated proteolysis and of genes expressed in multiple brain regions, skeletal muscle, and adrenal glands. Evidence of shared genetic factors was found between frequent insomnia symptoms and restless legs syndrome, aging, and cardiometabolic, behavioral, psychiatric, and reproductive traits. Evidence was found for a possible causal link between insomnia symptoms and coronary artery disease, depressive symptoms, and subjective well-being.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/nature%20genetics">nature genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/100-200-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/100-200-users/page/23/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/100-200-users/page/20/" class="page-link">20</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/100-200-users/page/21/" class="page-link">21</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/100-200-users/page/22/" class="page-link">22</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/100-200-users/page/23/" class="page-link">23</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/100-200-users/page/24/" class="page-link">24</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/100-200-users/page/25/" class="page-link">25</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/100-200-users/page/26/" class="page-link">26</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/100-200-users/page/27/" class="page-link">27</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/100-200-users/page/28/" class="page-link">28</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/100-200-users/page/25/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/100-200-users/page/68/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
