<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/animal_fungi_and_plant_genome_sequences_harbour_different_non-canonical_splice_sites_59403390/">
          <h4 class="font-weight-bold">Animal, fungi, and plant genome sequences harbour different non-canonical splice sites, bioRxiv, 2019-04-23</h4>
          <p>AbstractMost protein encoding genes in eukaryotes contain introns which are inter-woven with exons. After transcription, introns need to be removed in order to generate the final mRNA which can be translated into an amino acid sequence by the ribosome. Precise excision of introns by the spliceosome requires conserved dinucleotides which mark the splice sites. However, there are variations of the highly conserved combination of GT at the 5’ end and AG at the 3’ end of an intron in the genome. GC-AG and AT-AC are two major non-canonical splice site combinations which are known for many years. During the last few years, various minor non-canonical splice site combinations were detected with all possible dinucleotide permutations. Here we expand systematic investigations of non-canonical splice site combinations in plant genomes to all eukaryotes by analysing fungal and animal genome sequences. Comparisons of splice site combinations between these three kingdoms revealed several differences such as a substantially increased CT-AC frequency in fungal genomes. In addition, high numbers of GA-AG splice site combinations were observed in two animal species. In depth investigation of splice site usage based on RNA-Seq read mappings indicates a generally higher flexibility of the 3’ splice site compared to the 5’ splice site.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/excess_significance_bias_in_repetitive_transcranial_magnetic_stimulation_literature_for_neuropsychiatric_disorders_59362941/">
          <h4 class="font-weight-bold">Excess significance bias in repetitive transcranial magnetic stimulation literature for neuropsychiatric disorders, bioRxiv, 2019-04-23</h4>
          <p>ABSTRACTIntroductionRepetitive transcranial magnetic stimulation (rTMS) has been widely tested and promoted for use in multiple neuropsychiatric conditions, but as for many other medical devices, some gaps may exist in the literature and the evidence base for rTMS clinical efficacy remains under debate. We aimed to empirically test for an excess number of statistically significant results in the literature on rTMS therapeutic efficacy across a wide range of meta-analyses and to characterize the power of studies included in these meta-analyses.MethodsBased on power calculations, we computed the expected number of “positive” datasets for a medium effect-size (standardized mean difference, SMD=0.30) and compared it with the number of observed “positive” datasets. Sensitivity analyses considered small (SMD=0.20), modest (SMD=0.50), and large (SMD=0.80) effect sizes.Results14 meta-analyses with 228 datasets (110 for neurological disorders and 118 for psychiatric disorders) were assessed. For SMD=0.3, the number of observed “positive” studies (n=94) was larger than expected (n=35). We found evidence for an excess of significant findings overall (p&amp;lt;0.0001) and in 814 meta-analyses. Evidence for an excess of significant findings was also observed for SMD=0.5 for neurological disorders. 0 (0 %), 0 (0 %), 3 (1 %), and 53 (23 %) of the 228 datasets had power &amp;gt;0.80, respectively for SMDs of 0.30, 0.20, 0.50, and 0.80.ConclusionMost studies in the rTMS literature are underpowered. This results in fragmentation and waste of research efforts. The somewhat high frequency of “positive” results seems spurious and may reflect bias.Trial Registration PROSPERO 2017 CRD42017056694</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/scientific-communication-and-education">scientific-communication-and-education</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/modeltest-ng_a_new_and_scalable_tool_for_the_selection_of_dna_and_protein_evolutionary_models_59369428/">
          <h4 class="font-weight-bold">ModelTest-NG a new and scalable tool for the selection of DNA and protein evolutionary models, bioRxiv, 2019-04-23</h4>
          <p>AbstractModelTest-NG is a re-implementation from scratch of jModelTest and ProtTest, two popular tools for selecting the best-fit nucleotide and amino acid substitution models, respectively. ModelTest-NG is one to two orders of magnitude faster than jModelTest and ProtTest but equally accurate, and introduces several new features, such as ascertainment bias correction, mixture and FreeRate models, or the automatic processing of partitioned datasets. ModelTest-NG is available under a GNU GPL3 license at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsgithub.comddarribamodeltest&gt;httpsgithub.comddarribamodeltest&lt;jatsext-link&gt;.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/pathwaypca_an_r_package_for_integrative_pathway_analysis_with_modern_pca_methodology_and_gene_selection_59368395/">
          <h4 class="font-weight-bold">pathwayPCA an R package for integrative pathway analysis with modern PCA methodology and gene selection, bioRxiv, 2019-04-23</h4>
          <p>ABSTRACTWith the advance in high-throughput technology for molecular assays, multi-omics datasets have become increasingly available. However, most currently available pathway analysis software provide little or no functionalities for analyzing multiple types of -omics data simultaneously. In addition, most tools do not provide sample-specific estimates of pathway activities, which are important for precision medicine. To address these challenges, we present pathwayPCA, a unique R package for integrative pathway analysis that utilizes modern statistical methodology including supervised PCA and adaptive elastic-net PCA for principal component analysis. pathwayPCA can analyze continuous, binary, and survival outcomes in studies with multiple covariate andor interaction effects. We provide three case studies to illustrate pathway analysis with gene selection, integrative analysis of multi-omics datasets to identify driver genes, estimating and visualizing sample-specific pathway activities in ovarian cancer, and identifying sex-specific pathway effects in kidney cancer. pathwayPCA is an open source R package, freely available to the research community. We expect pathwayPCA to be a useful tool for empowering the wide scientific community on the analyses and interpretation of the wealth of multiomics data recently made available by TCGA, CPTAC and other large consortiums.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/nanoscale_subcellular_architecture_revealed_by_multicolor_3d_salvaged_fluorescence_imaging_59214657/">
          <h4 class="font-weight-bold">Nanoscale subcellular architecture revealed by multicolor 3D salvaged fluorescence imaging, bioRxiv, 2019-04-19</h4>
          <p>AbstractCombining the molecular specificity of fluorescent probes with three-dimensional (3D) imaging at nanoscale resolution is critical for investigating the spatial organization and interactions of cellular organelles and protein complexes. We present a super-resolution light microscope that enables simultaneous multicolor imaging of whole mammalian cells at ~20 nm 3D resolution. We show its power for cell biology research with fluorescence images that resolved the highly convoluted Golgi apparatus and the close contacts between the endoplasmic reticulum and the plasma membrane, structures that have traditionally been the imaging realm of electron microscopy.One Sentence SummaryComplex cellular structures previously only resolved by electron microscopy can now be imaged in multiple colors by 4Pi-SMS.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/chitin_perception_in_plasmodesmata_identifies_subcellular_context-specific_immune_signalling_in_plants_59151351/">
          <h4 class="font-weight-bold">Chitin perception in plasmodesmata identifies subcellular, context-specific immune signalling in plants, bioRxiv, 2019-04-17</h4>
          <p>AbstractThe plasma membrane (PM) that lines plasmodesmata has a distinct protein and lipid composition, underpinning specific regulation of these connections between cells. The plasmodesmal PM can integrate extracellular signals differently from the cellular PM, but it is not known how this specificity is established or how a single stimulus can trigger independent signalling cascades in neighbouring membrane domains. Here we have used the fungal elicitor chitin to investigate signal integration and responses at the plasmodesmal PM. We found that the plasmodesmal PM employs a receptor complex composed of the LysM receptors LYM2 and LYK4 which respectively change their location and interactions in response to chitin. Downstream, signalling is transmitted via a specific phosphorylation signature of an NADPH oxidase and localised callose synthesis that causes plasmodesmata closure. This demonstrates the plasmodesmal PM deploys both plasmodesmata-specific components and differential activation of PM-common components to independently integrate an immune signal.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/0-100-users/page/31/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/28/" class="page-link">28</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/29/" class="page-link">29</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/30/" class="page-link">30</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/31/" class="page-link">31</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/0-100-users/page/32/" class="page-link">32</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/33/" class="page-link">33</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/34/" class="page-link">34</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/35/" class="page-link">35</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/36/" class="page-link">36</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/0-100-users/page/33/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/0-100-users/page/123/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
