<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_comprehensive_assessment_of_demographic_environmental_and_host_genetic_associations_with_gut_microbiome_diversity_in_healthy_individuals_56150428/">
          <h4 class="font-weight-bold">A Comprehensive Assessment of Demographic, Environmental and Host Genetic Associations with Gut Microbiome Diversity in Healthy Individuals, bioRxiv, 2019-02-28</h4>
          <p>Background The gut microbiome is an important determinant of human health. Its composition has been shown to be influenced by multiple environmental factors and likely by host genetic variation. In the framework of the Milieu Intérieur Consortium, a total of 1,000 healthy individuals of western European ancestry, with a 11 sex ratio and evenly stratified across five decades of life (age 20 - 69), were recruited. We generated 16S ribosomal RNA profiles from stool samples for 858 participants. We investigated genetic and non-genetic factors that contribute to individual differences in fecal microbiome composition.Results Among 110 demographic, clinical and environmental factors, 11 were identified as significantly correlated with α-diversity, β-diversity or abundance of specific microbial communities in multivariable models. Age and blood alanine aminotransferase levels showed the strongest associations with microbiome diversity. In total, all non-genetic factors explained 16.4% of the variance. We then searched for associations between &amp;gt;5 million single nucleotide polymorphisms and the same indicators of fecal microbiome diversity, including the significant non-genetic factors as covariates. No genome-wide significant associations were identified after correction for multiple testing. A small fraction of previously reported associations between human genetic variants and specific taxa could be replicated in our cohort, while no replication was observed for any of the diversity metrics.Conclusion In a well-characterized cohort of healthy individuals, we identified several non-genetic variables associated with fecal microbiome diversity. In contrast, host genetics only had a negligible influence. Demographic and environmental factors are thus the main contributors to fecal microbiome composition in healthy individuals.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/interplay_between_the_human_gut_microbiome_and_host_metabolism_56164411/">
          <h4 class="font-weight-bold">Interplay between the human gut microbiome and host metabolism, bioRxiv, 2019-02-28</h4>
          <p>The human gut is inhabited by a complex and metabolically active microbial ecosystem regulating host health. While many studies have focused on the effect of individual microbial taxa, the metabolic potential of the entire gut microbial ecosystem has been largely under-explored. We characterised the gut microbiome of 1,004 twins via whole shotgun metagenomic sequencing (average 39M reads per sample). We observed greater similarity, across unrelated individuals, for functional metabolic pathways (82%) than for taxonomic composition (43%). We conducted a microbiota-wide association study linking both taxonomic information and microbial metabolic pathways with 673 blood and 713 faecal metabolites (Metabolon, Inc.). Metabolic pathways associated with 34% of blood and 95% of faecal metabolites, with over 18,000 significant associations, while species-level results identified less than 3,000 associations, suggesting that coordinated action of multiple taxa is required to affect the metabolome. Finally, we estimated that the microbiome mediated a crosstalk between 71% of faecal and 15% of blood metabolites, highlighting six key species (unclassified Subdoligranulum spp., Faecalibacterium prausnitzii, Roseburia inulinivorans, Methanobrevibacter smithii, Eubacterium rectale, and Akkermansia muciniphila). Because of the large inter-person variability in microbiome composition, our results underline the importance of studying gut microbial metabolic pathways rather than focusing purely on taxonomy to find therapeutic and diagnostic targets.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/interdependent_phenotypic_and_biogeographic_evolution_driven_by_biotic_interactions_56047585/">
          <h4 class="font-weight-bold">Interdependent Phenotypic and Biogeographic Evolution Driven by Biotic Interactions, bioRxiv, 2019-02-26</h4>
          <p>Biotic interactions are hypothesized to be one of the main processes shaping trait and biogeographic evolution during lineage diversification. Theoretical and empirical evidence suggests that species with similar ecological requirements either spatially exclude each other, by preventing the colonization of competitors or by driving coexisting populations to extinction, or show niche divergence when in sympatry. However, the extent and generality of the effect of interspecific competition in trait and biogeographic evolution has been limited by a dearth of appropriate process-generating models to directly test the effect of biotic interactions. Here, we formulate a phylogenetic parametric model that allows interdependence between trait and biogeographic evolution, thus enabling a direct test of central hypotheses on how biotic interactions shape these evolutionary processes. We adopt a Bayesian data augmentation approach to estimate the joint posterior distribution of trait histories, range histories, and co-evolutionary process parameters under this analytically intractable model. Through simulations, we show that our model is capable of distinguishing alternative scenarios of biotic interactions. We apply our model to the radiation of Darwin&#39;s finches---a classic example of adaptive divergence---and find support for in situ trait divergence in beak size, convergence in traits such as beak shape and tarsus length, and strong competitive exclusion throughout their evolutionary history. Our modeling framework opens new possibilities for testing more complex hypotheses about the processes underlying lineage diversification. More generally, it provides a robust probabilistic methodology to model correlated evolution of continuous and discrete characters.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/samovar_single-sample_mosaic_snv_calling_with_linked_reads_56051779/">
          <h4 class="font-weight-bold">Samovar Single-sample mosaic SNV calling with linked reads, bioRxiv, 2019-02-26</h4>
          <p>We present Samovar, a mosaic single-nucleotide variant (SNV) caller for linked-read whole-genome shotgun sequencing data. Samovar scores candidate sites using a random forest model trained using the input dataset that considers read quality, phasing, and linked-read characteristics. We show Samovar calls mosaic SNVs within a single sample with accuracy comparable to what previously required trios or matched tumornormal pairs and outperform single-sample mosaic variant callers at MAF 5%-50% with at least 30x coverage. Furthermore, we use Samovar to find somatic variants in whole genome sequencing of both tumor and normal from 13 pediatric cancer cases that can be corroborated with high recall with whole exome sequencing. Samovar is available open-source at httpsgithub.comcdarbysamovar under the MIT license.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/task-evoked_activity_quenches_neural_correlations_and_variability_in_large-scale_brain_systems_56050434/">
          <h4 class="font-weight-bold">Task-evoked activity quenches neural correlations and variability in large-scale brain systems, bioRxiv, 2019-02-26</h4>
          <p>Many studies of large-scale neural systems have emphasized the importance of communication through increased inter-region correlations (functional connectivity) during task states relative to resting state. In contrast, local circuit studies have demonstrated that task states reduce correlations among local neural populations, likely enhancing their information coding. Here we sought to adjudicate between these conflicting perspectives, assessing whether large-scale system correlations tend to increase or decrease during task states. To establish a mechanistic framework for interpreting changes in neural correlations, we conceptualized neural populations as having a sigmoidal neural transfer function. In a computational model we found that this straightforward assumption predicts reductions in neural populations&#39; dynamic output range as task-evoked activity levels increase, reducing responsiveness to inputs from other regions (i.e., reduced correlations). We demonstrated this empirically in large-scale neural populations across two highly distinct data sets human functional magnetic resonance imaging data and non-human primate spiking data. We found that task states increased global neural activity, while globally quenching neural variability and correlations. Further, this global reduction of neural correlations led to an overall increase in dimensionality (reflecting less information redundancy) during task states, providing an information-theoretic explanation for task-induced correlation reductions. Together, our results provide an integrative mechanistic account that encompasses measures of large-scale neural activity, variability, and correlations during resting and task states.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/cooler_scalable_storage_for_hi-c_data_and_other_genomically-labeled_arrays_55927091/">
          <h4 class="font-weight-bold">Cooler scalable storage for Hi-C data and other genomically-labeled arrays, bioRxiv, 2019-02-23</h4>
          <p>Most existing coverage-based (epi)genomic datasets are one-dimensional, but newer technologies probing interactions (physical, genetic, etc.) produce quantitative maps with two-dimensional genomic coordinate systems. Storage and computational costs mount sharply with data resolution when such maps are stored in dense form. Hence, there is a pressing need to develop data storage strategies that handle the full range of useful resolutions in multidimensional genomic datasets by taking advantage of their sparse nature, while supporting efficient compression and providing fast random access to facilitate development of scalable algorithms for data analysis. We developed a file format called cooler, based on a sparse data model, that can support genomically-labeled matrices at any resolution. It has the flexibility to accommodate various descriptions of the data axes (genomic coordinates, tracks and bin annotations), resolutions, data density patterns, and metadata. Cooler is based on HDF5 and is supported by a Python library and command line suite to create, read, inspect and manipulate cooler data collections. The format has been adopted as a standard by the NIH 4D Nucleome Consortium. Cooler is cross-platform, BSD-licensed, and can be installed from the Python Package Index or the bioconda repository. The source code is maintained on Github at httpsgithub.commirnylabcooler.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/44/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/41/" class="page-link">41</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/42/" class="page-link">42</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/43/" class="page-link">43</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/44/" class="page-link">44</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/0-100-users/page/45/" class="page-link">45</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/46/" class="page-link">46</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/47/" class="page-link">47</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/48/" class="page-link">48</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/49/" class="page-link">49</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/46/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/123/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
