<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/stronger_and_higher_proportion_of_beneficial_amino_acid_changing_mutations_in_humans_compared_to_mice_and_flies_48909889/">
          <h4 class="font-weight-bold">Stronger and higher proportion of beneficial amino acid changing mutations in humans compared to mice and flies, bioRxiv, 2018-09-26</h4>
          <p>ABSTRACTQuantifying and comparing the amount of adaptive evolution among different species is key to understanding evolutionary processes. Previous studies have shown differences in adaptive evolution across species, however their specific causes remain elusive. Here, we use improved modeling of weakly deleterious mutations and the demographic history of the outgroup species and estimate that 30â€“34% of nonsynonymous substitutions between humans and outgroup species have been fixed by positive selection. This estimate is much higher than previous estimates, which did not account for the population size of the outgroup species. Next, we directly estimate the proportion and selection coefficients of newly arising strongly beneficial nonsynonymous mutations in humans, mice, and D. melanogaster by examining patterns of polymorphism and divergence. We develop a novel composite likelihood framework to test whether these parameters differ across species. Overall, we reject a model with the same proportion and the same selection coefficients of beneficial mutations across species, and estimate that humans have a higher proportion of beneficial mutations compared to Drosophila and mice. We demonstrate that this result cannot be attributed to biased gene conversion. In summary, we find the proportion of beneficial mutations is higher in humans than in D. melanogaster or mice, suggesting that organismal complexity, which increases the number of steps required in adaptive walks, may be a key predictor of the amount of adaptive evolution within a species.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_ecological_drivers_of_variation_in_global_language_diversity_48910924/">
          <h4 class="font-weight-bold">The ecological drivers of variation in global language diversity, bioRxiv, 2018-09-26</h4>
          <p>AbstractLanguage diversity is distributed unevenly over the globe. Why do some areas have so many different languages and other areas so few? Intriguingly, patterns of language diversity resemble biodiversity patterns, leading to suggestions that similar mechanisms may underlie both linguistic and biological diversification. Here we present the first global analysis of language diversity that identifies the relative importance of two key ecological mechanisms suggested to promote language diversification - isolation and ecological risk - after correcting for spatial autocorrelation and phylogenetic non-independence. We find significant effects of climate on language diversity consistent with the ecological risk hypothesis that areas of high year-round productivity lead to more languages by supporting human cultural groups with smaller distributions. Climate has a much stronger effect on language diversity than landscape features that might contribute to isolation of cultural groups, such as altitudinal variation, river density, or landscape roughness. The association between biodiversity and language diversity appears to be an incidental effect of their covariation with climate, rather than a causal link between the two. While climate and landscape provide strong explanatory signal for variation in language diversity, we identify a number of areas of high unexplained language diversity, with more languages than would be predicted from environmental features alone; notably New Guinea, the Himalayan foothills, West Africa, and Mesoamerica. Additional processes may be at play in generating higher than expected language diversity in these regions.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/beditor_a_computational_workflow_for_designing_libraries_of_guide_rnas_for_crispr-mediated_base_editing_48898720/">
          <h4 class="font-weight-bold">beditor A computational workflow for designing libraries of guide RNAs for CRISPR-mediated base editing, bioRxiv, 2018-09-26</h4>
          <p>ABSTRACTCRISPR-mediated base editors have opened unique avenues for scar-free genome-wide mutagenesis. Here, we describe a comprehensive computational workflow called beditor that can be broadly adapted for designing guide RNA libraries with a range of CRISPR-mediated base editors, PAM recognition sequences and genomes of many species. Additionally, in order to assist users in selecting the best sets of guide RNAs for their experiments, a priori estimates, called beditor scores are calculated. These beditor scores are intended to select guide RNAs that conform to requirements for optimal base editing the editable base falls within maximum activity window of the CRISPR-mediated base editor and produces non-confounding mutational effects with minimal predicted off-target effects. We demonstrate the utility of the software by designing guide RNAs for base-editing to create or remove thousands of clinically important human disease mutations.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/synthetic-biology">synthetic-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/amygdala_neuronal_ensembles_dynamically_encode_behavioral_states_48886404/">
          <h4 class="font-weight-bold">Amygdala neuronal ensembles dynamically encode behavioral states, bioRxiv, 2018-09-25</h4>
          <p>AbstractInternal states, including affective or homeostatic states, are important behavioral motivators. The amygdala is a key brain region involved in the regulation of motivated behaviors, yet how distinct internal states are represented in amygdala circuits is unknown. Here, by imaging somatic neural calcium dynamics in freely moving mice, we identify changes in the relative activity levels of two major, non-overlapping populations of principal neurons in the basal nucleus of the amygdala (BA) that predict switches between exploratory and non-exploratory (defensive, anxiety-like) behavioral states across different environments. Moreover, the amygdala widely broadcasts internal state information via several output pathways to larger brain networks, and sensory responses in BA occur independently of behavioral state encoding. Thus, the brain processes external stimuli and internal states in an orthogonal manner, which may facilitate rapid and flexible selection of appropriate, state-dependent behavioral responses.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/layer-dependent_activity_in_human_prefrontal_cortex_during_working_memory_48847582/">
          <h4 class="font-weight-bold">Layer-dependent activity in human prefrontal cortex during working memory, bioRxiv, 2018-09-24</h4>
          <p>Working memory involves a series of functions encoding a stimulus, maintaining or manipulating its representation over a delay, and finally making a behavioral response. While working memory engages dorsolateral prefrontal cortex (dlPFC), few studies have investigated whether these subfunctions are localized to different cortical depths in this region, and none have done so in humans. Here, we use high-resolution functional MRI to interrogate the layer specificity of neural activity during different epochs of a working memory task in dlPFC. We detect activity timecourses that follow the hypothesized patterns superficial layers are preferentially active during the delay period, while deeper layers are preferentially active during the response. Results demonstrate that layer-specific fMRI can be used in higher-order brain regions to non-invasively map cognitive information processing along cortical circuitry in humans.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/parliament2_fast_structural_variant_calling_using_optimized_combinations_of_callers_48807637/">
          <h4 class="font-weight-bold">Parliament2 Fast Structural Variant Calling Using Optimized Combinations of Callers, bioRxiv, 2018-09-23</h4>
          <p>AbstractHere we present Parliament2 â€“ a structural variant caller which combines multiple best-in-class structural variant callers to create a highly accurate callset. This captures more events than the individual callers achieve independently. Parliament2 uses a call-overlap-genotype approach that is highly extensible to new methods and presents users the choice to run some or all of Breakdancer, Breakseq, CNVnator, Delly, Lumpy, and Manta to run. Parliament2 applies an additional parallelization framework to speed certain callers and executes these in parallel, taking advantage of the different resource requirements to complete structural variant calling much faster than running the programs individually. Parliament2 is available as a Docker container, which pre-installs all required dependencies. This allows users to run any caller with easy installation and execution. This Docker container can easily be deployed in cloud or local environments and is available as an app on DNAnexus.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/0-100-users/" rel="first" class="page-link">Â« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/44/" rel="prev" class="page-link">â€¹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/41/" class="page-link">41</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/42/" class="page-link">42</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/43/" class="page-link">43</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/44/" class="page-link">44</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/0-100-users/page/45/" class="page-link">45</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/46/" class="page-link">46</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/47/" class="page-link">47</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/48/" class="page-link">48</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/49/" class="page-link">49</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/46/" rel="next" class="page-link">Next â€º</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/64/" rel="last" class="page-link">Last Â»</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
