<!DOCTYPE html>
<html><head>
  
  <title>2019 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.68.3" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.95d2483cd3c5d2bf47c1f36307a456787d60ad958c5efe44db9afc47012b1b05b8e1f64f6ac291650439322a6c38e86eb7cd6254255516d857c23f299fb0e666.js" integrity="sha512-ldJIPNPF0r9HwfNjB6RWeH1grZWMXv5E25r8RwErGwW44fZPasKRZQQ5MipsOOhut81iVCVVFthXwj8pn7DmZg=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/pirate_a_fast_and_scalable_pangenomics_toolbox_for_clustering_diverged_orthologues_in_bacteria_58599091/">
          <h4 class="font-weight-bold">PIRATE A fast and scalable pangenomics toolbox for clustering diverged orthologues in bacteria, bioRxiv, 2019-04-05</h4>
          <p>AbstractCataloguing the distribution of genes within natural bacterial populations is essential for understanding evolutionary processes and the genetic basis of adaptation. Here we present a pangenomics toolbox, PIRATE (Pangenome Iterative Refinement And Threshold Evaluation), which identifies and classifies orthologous gene families in bacterial pangenomes over a wide range of sequence similarity thresholds. PIRATE builds upon recent scalable software developments to allow for the rapid interrogation of thousands of isolates. PIRATE clusters genes (or other annotated features) over a wide range of amino-acid or nucleotide identity thresholds and uses the clustering information to rapidly classify paralogous gene families into either putative fissionfusion events or gene duplications. Furthermore, PIRATE orders the pangenome using a directed graph, provides a measure of allelic variation and estimates sequence divergence for each gene family. We demonstrate that PIRATE scales linearly with both number of samples and computation resources, allowing for analysis of large genomic datasets, and compares favorably to other popular tools. PIRATE provides a robust framework for analysing bacterial pangenomes, from largely clonal to panmictic species.AvailabilityPIRATE is implemented in Perl and is freely available under an GNU GPL 3 open source license from &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsgithub.comSionBaylissPIRATE&gt;httpsgithub.comSionBaylissPIRATE&lt;jatsext-link&gt;.&lt;jatssec sec-type=supplementary-material&gt;Supplementary InformationSupplementary data is available online.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/single-cell_transcriptomics_of_the_naked_mole-rat_reveals_unexpected_features_of_mammalian_immunity_58550675/">
          <h4 class="font-weight-bold">Single-cell transcriptomics of the naked mole-rat reveals unexpected features of mammalian immunity, bioRxiv, 2019-04-05</h4>
          <p>AbstractUsing single-cell transcriptional profiling we mapped the immune system of the naked mole-rat (Heterocephalus glaber), a small but long-lived and cancer-resistant subterranean rodent. Both splenic and circulating immune cells were examined in healthy young animals and following an infection-mimicking lipopolysaccharide challenge. Our study revealed that the naked mole-rat immune system is characterized by a high myeloid to lymphoid cell ratio that includes a novel, lipopolysaccharide responsive, granulocyte cell subset not found in the mouse. Conversely, we find that naked mole-rats do not have a cell subset that corresponds to natural killer cells as defined in other well-characterized mammalian species. Supporting this finding, we show that the naked mole-rat genome has not expanded any of the gene families encoding diverse natural killer cell receptors, which are the genomic hallmarks of species in which natural killer cells have been described. These unusual features suggest an atypical mode of immunosurveillance and a greater reliance on myeloid-biased innate immunity.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_elixir_core_data_resources_fundamental_infrastructure_for_the_life_sciences_58598348/">
          <h4 class="font-weight-bold">The ELIXIR Core Data Resources fundamental infrastructure for the life sciences, bioRxiv, 2019-04-05</h4>
          <p>AbstractMotivationLife science research in academia, industry, agriculture, and the health sector is critically dependent on free and open data resources. ELIXIR, the European Research Infrastructure for life sciences data, has undertaken the task of identifying the set of Core Data Resources within Europe that are of most fundamental importance to the life science community for the long-term preservation of biological data. Having defined the Core Data Resources, we explored characteristics of the usage, impact and sustainability of the set as a whole to assess the value and importance of these resources as an infrastructure, to understand sustainability to the infrastructure, and to demonstrate a model for assessing Core Data Resources worldwide.ResultsThe nineteen resources designated as Core Data Resources by ELIXIR together form a data infrastructure in Europe that is a subset of the wider worldwide open life sciences data infrastructure. These resources are of crucial importance to research throughout the world. We show that, from 2013 to 2017, data managed by the Core Data Resources tripled and usage doubled while staff numbers increased by only a sixth. Additionally, support for the Core Data Resources is precarious, with all resources together having assured funding for less than a third of current staff after only three years.Our findings demonstrate the importance of the ELIXIR Core Data Resources as repositories for research data and the knowledge generated from those data, while also demonstrating the precarious nature of the funding environment for this infrastructure. The ELIXIR Core Data Resources are part of a larger worldwide life sciences data resources ecosystem. Both within Europe and as part of the Global Biodata Coalition, ELIXIR will work for longer-term support for the worldwide life sciences data resource infrastructure and for the subset of that infrastructure that is the ELIXIR Core Data Resources.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/training_and_inferring_neural_network_function_with_multi-agent_reinforcement_learning_58604703/">
          <h4 class="font-weight-bold">Training and inferring neural network function with multi-agent reinforcement learning, bioRxiv, 2019-04-05</h4>
          <p>AbstractA central goal in systems neuroscience is to understand the functions performed by neural circuits. Previous top-down models addressed this question by comparing the behaviour of an ideal model circuit, optimised to perform a given function, with neural recordings. However, this requires guessing in advance what function is being performed, which may not be possible for many neural systems. To address this, we propose a new framework for optimising a recurrent network using multi-agent reinforcement learning (RL). In this framework, a reward function quantifies how desirable each state of the network is for performing a given function. Each neuron is treated as an ‘agent’, which optimises its responses so as to drive the network towards rewarded states. Three applications follow from this. First, one can use multi-agent RL algorithms to optimise a recurrent neural network to perform diverse functions (e.g. efficient sensory coding or motor control). Second, one could use inverse RL to infer the function of a recorded neural network from data. Third, the theory predicts how neural networks should adapt their dynamics to maintain the same function when the external environment or network structure changes. This could lead to theoretical predictions about how neural network dynamics adapt to deal with cell death andor varying sensory stimulus statistics.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/large-scale_death_of_retinal_astrocytes_during_normal_development_mediated_by_microglia_58453924/">
          <h4 class="font-weight-bold">Large-scale death of retinal astrocytes during normal development mediated by microglia, bioRxiv, 2019-04-04</h4>
          <p>Naturally-occurring cell death is a fundamental developmental mechanism for regulating cell numbers and sculpting developing organs. This is particularly true in the central nervous system, where large numbers of neurons and oligodendrocytes are eliminated via apoptosis during normal development. Given the profound impact of death upon these two major cell populations, it is surprising that developmental death of another major cell type – the astrocyte – has rarely been studied. It is presently unclear whether astrocytes are subject to significant amounts of developmental death, or how it occurs. Here we address these questions using mouse retinal astrocytes as our model system. We show that the total number of retinal astrocytes declines by over 3-fold during a death period spanning postnatal days 5-14. Surprisingly, these astrocytes do not die by apoptosis, the canonical mechanism underlying the vast majority of developmental cell death. Instead, we find that microglia kill and engulf astrocytes to mediate their developmental removal. Genetic ablation of microglia inhibits astrocyte death, leading to a larger astrocyte population size at the end of the death period. However, astrocyte death is not completely blocked in the absence of microglia, apparently due to the ability of astrocytes to engulf each other. Nevertheless, mice lacking microglia showed significant anatomical changes to the retinal astrocyte network, with functional consequences for the astrocyte-associated vasculature leading to retinal hemorrhage. These results establish a novel modality for naturally-occurring cell death, and demonstrate its importance for formation and integrity of the retinal gliovascular network.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/suppression_of_unwanted_crisprcas9_editing_by_co-administration_of_catalytically_inactivating_truncated_guide_rnas_58453110/">
          <h4 class="font-weight-bold">Suppression of unwanted CRISPRCas9 editing by co-administration of catalytically inactivating truncated guide RNAs, bioRxiv, 2019-04-04</h4>
          <p>AbstractCRISPRCas9 nucleases are powerful genome engineering tools, but unwanted cleavage at off-target and previously edited sites remains a major concern. Numerous strategies to reduce unwanted cleavage have been devised, but all are imperfect. Here, we report off-target sites can be shielded from the active Cas9•single guide RNA (sgRNA) complex through the co-administration of dead-RNAs (dRNAs), truncated guide RNAs that direct Cas9 binding but not cleavage. dRNAs can effectively suppress a wide-range of off-targets with minimal optimization while preserving on-target editing, and they can be multiplexed to suppress several off-targets simultaneously. dRNAs can be combined with high-specificity Cas9 variants, which often do not eliminate all unwanted editing. Moreover, dRNAs can prevent cleavage of homology-directed repair (HDR)-corrected sites, facilitating “scarless” editing by eliminating the need for blocking mutations. Thus, we enable precise genome editing by establishing a novel and flexible approach for suppressing unwanted editing of both off-targets and HDR-corrected sites.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/78/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/75/" class="page-link">75</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/76/" class="page-link">76</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/77/" class="page-link">77</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/78/" class="page-link">78</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/2019/page/79/" class="page-link">79</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/80/" class="page-link">80</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/81/" class="page-link">81</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/82/" class="page-link">82</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/83/" class="page-link">83</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/80/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/135/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
