<!DOCTYPE html>
<html><head>
  
  <title>Reports | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/revealing_multi-scale_population_structure_in_large_cohorts_48806746/">
          <h4 class="font-weight-bold">Revealing multi-scale population structure in large cohorts, bioRxiv, 2018-09-23</h4>
          <p>Genetic structure in large cohorts results from technical, sampling and demographic variation. Visualisation is therefore a first step in most genomic analyses. However, existing data exploration methods struggle with unbalanced sampling and the many scales of population structure. We investigate an approach to dimension reduction of genomic data that combines principal components analysis (PCA) with uniform manifold approximation and projection (UMAP) to succinctly illustrate population structure in large cohorts and capture their relationships on local and global scales. Using data from large-scale genomic datasets, we demonstrate that PCA-UMAP effectively clusters closely related individuals while placing them in a global continuum of genetic variation. This approach reveals previously overlooked subpopulations within the American Hispanic population and fine-scale relationships between geography, genotypes, and phenotypes in the UK population. This opens new lines of investigation for demographic research and statistical genetics. Given its small computational cost, PCA-UMAP also provides a general-purpose approach to exploratory analysis in population-scale datasets.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/vision_using_multiple_distinct_rod_opsins_in_deep-sea_fishes_48810320/">
          <h4 class="font-weight-bold">Vision using multiple distinct rod opsins in deep-sea fishes, bioRxiv, 2018-09-23</h4>
          <p>AbstractVertebrate vision is accomplished through a set of light-sensitive photopigments, which are located in the photoreceptors of the retina and consist of a visual opsin protein bound to a chromophore. In dim-light, vertebrates generally rely upon a single rod opsin (RH1) for obtaining visual information. By inspecting 101 fish genomes, we found that three deep-sea teleost lineages have independently expanded their RH1 gene repertoires. Amongst these, the silver spinyfin (Diretmus argenteus Johnson 1863) stands out as having the highest number of visual opsins known for animals to date (2 cone and 38 rod opsins). Spinyfins simultaneously express up to 14 RH1s encoding for photopigments with different peak spectral sensitivities (λmax=448-513 nm) that cover the range of the residual daylight, as well as the bioluminescence spectrum present in the deep-sea. Our findings present novel molecular and functional evidence for the recurrent evolution of multiple rod opsin-based vision in vertebrates.SHORT ABSTRACTContrary to the single rod opsin used by most vertebrates, some fishes use multiple rod opsins for vision in the dimly lit deep-sea.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/a_multi-domain_task_battery_reveals_functional_boundaries_in_the_human_cerebellum_48724700/">
          <h4 class="font-weight-bold">A Multi-Domain Task Battery Reveals Functional Boundaries in the Human Cerebellum, bioRxiv, 2018-09-21</h4>
          <p>AbstractThere is compelling evidence that the human cerebellum is engaged in a wide array of motor and cognitive tasks. A fundamental question centers on whether the cerebellum is organized into distinct functional sub-regions. To address this question, we employed a rich task battery, designed to tap into a broad range of cognitive processes. During four functional magnetic resonance imaging (fMRI) sessions, participants performed a battery of 26 diverse tasks comprising 47 unique conditions. Using the data from this multi-domain task battery (MDTB), we derived a comprehensive functional parcellation of the cerebellar cortex and evaluated it by predicting functional boundaries in a novel set of tasks. The new parcellation successfully identified distinct functional sub-regions, providing significant improvements over existing parcellations derived from task-free data. Lobular boundaries, commonly used to summarize functional data, did not coincide with functional subdivisions. This multi-domain task approach offers novel insights into the functional heterogeneity of the cerebellar cortex.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/comparison_of_efficiency_and_specificity_of_crispr-associated_cas_nucleases_in_plants_an_expanded_toolkit_for_precision_genome_engineering_48724730/">
          <h4 class="font-weight-bold">Comparison of Efficiency and Specificity of CRISPR-Associated (Cas) Nucleases in Plants An Expanded Toolkit for Precision Genome Engineering, bioRxiv, 2018-09-21</h4>
          <p>Molecular tools adapted from bacterial CRISPR (Clustered Regulatory Interspaced Short Palindromic Repeats) systems for adaptive immunity have become widely used for plant genome engineering, both to investigate gene functions and to engineer desirable traits. A number of different Cas (CRISPR-associated) nucleases are now used but, as most studies performed to date have engineered different targets using a variety of plant species and molecular tools, it has been difficult to draw conclusions about the comparative performance of different nucleases. Due to the time and effort required to regenerate engineered plants, efficiency is critical. In addition, there have been several reports of mutations at sequences with less than perfect identity to the target. While in some plant species it is possible to remove these so-called ‘off-targets’ by backcrossing to a parental line, the specificity of genome engineering tools is important when targeting specific members of closely-related gene families, especially when recent paralogues are co-located in the genome and unlikely to segregate. Specificity is also important for species that take years to reach sexual maturity or that are clonally propagated. Here, we directly compare the efficiency and specificity of Cas nucleases from different bacterial species together with engineered variants of Cas9. We find that the nucleotide content correlates with efficiency and that Cas9 from Staphylococcus aureus is comparatively most efficient at inducing mutations. We also demonstrate that ‘high-fidelity’ variants of Cas9 can reduce off-target mutations in plants. We present these molecular tools as standardised DNA parts to facilitate their re-use.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/dissecting_heterogeneous_cell-populations_across_signaling_and_disease_conditions_with_popalign_48724548/">
          <h4 class="font-weight-bold">Dissecting heterogeneous cell-populations across signaling and disease conditions with PopAlign, bioRxiv, 2018-09-21</h4>
          <p>AbstractSingle-cell measurement techniques can now probe gene expression in heterogeneous cell populations from the human body across a range of environmental and physiological conditions. However, new mathematical and computational methods are required to represent and analyze gene expression changes that occur in complex mixtures of single cells as they respond to signals, drugs, or disease states. Here, we introduce a mathematical modeling platform, PopAlign, that automatically identifies subpopulations of cells within a heterogeneous mixture, and tracks gene expression and cell abundance changes across subpopulations by constructing and comparing probabilistic models. We apply PopAlign to discover specific categories of signaling responses within primary human immune cells as well as patient-specific disease signatures in multiple myeloma that are obscured by techniques like tSNE. PopAlign scales to comparisons involving tens to hundreds of samples, enabling large-scale studies of natural and engineered cell populations as they respond to drugs, signals or physiological change.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/objective_versus_self-reported_energy_intake_changes_during_low-carbohydrate_and_low-fat_diets_48703297/">
          <h4 class="font-weight-bold">Objective versus Self-Reported Energy Intake Changes During Low-Carbohydrate and Low-Fat Diets, bioRxiv, 2018-09-21</h4>
          <p>AbstractObjectiveTo examine objective versus self-reported energy intake changes (ΔEI) during a 12-month diet intervention.MethodsWe calculated ΔEI in subjects who participated in a 1-year randomized low-carbohydrate versus low-fat diet trial using repeated body weight measurements as inputs to an objective mathematical model (ΔEIModel) and compared these values with self-reported energy intake changes assessed by repeated 24-hr recalls (ΔEI24hrRecall).ResultsΔEI24hrRecall indicated a relatively persistent state of calorie restriction ≥500 kcald throughout the year with no significant differences between diets. ΔEIModel demonstrated large early decreases in calorie intake &amp;gt;800 kcald followed by an exponential return to approximately 100 kcald below baseline at the end of the year. The low-carbohydrate diet resulted in ΔEIModel that was 162±53 kcald lower than the low-fat diet over the first 3 months (p=0.002), but no significant diet differences were found at later times. Weight loss at 12 months was significantly related to ΔEIModel at all time intervals for both diets (p&amp;lt;0.0001).ConclusionsSelf-reported measurements of ΔEI were inaccurate. Model-based calculations of ΔEI found that instructions to follow the low-carbohydrate diet resulted in greater calorie restriction than the low-fat diet in the early phases of the intervention, but these diet differences were not sustained.What is already known about this subject?&lt;jatslist list-type=bullet&gt;&lt;jatslist-item&gt;Diet assessments that rely on self-report, such as 24hr dietary recall, are known to underestimate actual energy intake as measured by doubly labeled water. However, it is possible that repeated self-reported measurements could accurately detect changes in energy intake over time if the absolute bias of self-reported of measurements is approximately constant for each subject.&lt;jatslist-item&gt;What this study adds&lt;jatslist list-type=bullet&gt;&lt;jatslist-item&gt;We compared energy intake changes measured using repeated 24hr dietary recall measurements collected over the course of the 1-year Diet Intervention Examining The Factors Interacting with Treatment Success (DIETFITS) trial versus energy intake changes calculated using repeated body weight measurements as inputs to a validated mathematical model.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Whereas self-reported measurements indicated a relatively persistent state of calorie restriction, objective model-based measurements demonstrated a large early calorie restriction followed by an exponential rise in energy intake towards the pre-intervention baseline.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Model-based calculations, but not self-reported measurements, found that low-carbohydrate diets led to significantly greater early decreases in energy intake compared to low-fat diets, but long-term energy intake changes were not significantly different.&lt;jatslist-item&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/physiology">physiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/reports/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/reports/page/123/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/reports/page/120/" class="page-link">120</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/121/" class="page-link">121</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/122/" class="page-link">122</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/123/" class="page-link">123</a></li>
            
        
             
            <li class="page-item active"><a href="/reports/page/124/" class="page-link">124</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/125/" class="page-link">125</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/126/" class="page-link">126</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/127/" class="page-link">127</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/128/" class="page-link">128</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/reports/page/125/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/reports/page/172/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
