<!DOCTYPE html>
<html><head>
  
  <title>Biorxiv | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/increased_antibiotic_susceptibility_in_neisseria_gonorrhoeae_through_adaptation_to_the_cervical_environment_73769545/">
          <h4 class="font-weight-bold">Increased antibiotic susceptibility in Neisseria gonorrhoeae through adaptation to the cervical environment, bioRxiv, 2020-01-09</h4>
          <p>AbstractNeisseria gonorrhoeae is an urgent public health threat due to rapidly increasing incidence and antibiotic resistance. In contrast with the trend of increasing resistance, clinical isolates that have reverted to susceptibility regularly appear, prompting questions about which pressures compete with antibiotics to shape gonococcal evolution. Here, we used genome-wide association on the largest collection of N. gonorrhoeae isolates to date (n=4882) to identify loss-of-function (LOF) mutations in the efflux pump mtrCDE operon as a mechanism of increased antibiotic susceptibility and demonstrate that these mutations are overrepresented in cervical isolates relative to urethral isolates (odds ratio (OR) = 3.74, 95% CI [1.98-6.70]). In support of a model in which pump expression incurs a fitness cost in this niche, cervical isolates were also enriched relative to urethral isolates in LOF mutations in the mtrCDE activator mtrA (OR = 8.60, 95% CI [4.96-14.57]) and in farA, a subunit of the FarAB efflux pump (OR = 6.25, 95% CI [3.90-9.83]). In total, approximately 1 in 3 cervical isolates (36.4%) contained a LOF mutation in either the efflux pump components mtrC or farA or the activator mtrA. Our findings extend beyond N. gonorrhoeae to other Neisseria mtrC LOF mutations are rare (&amp;lt;1%) in the primarily nasopharyngeal-colonizing N. meningitidis in a collection of 14,798 genomes but enriched in a heterosexual urethritis-associated lineage (8.6%, p = 9.90×10-5), indicating that efflux pump downregulation contributes broadly to the adaptation of pathogenic Neisseria to the female urogenital tract. Overall, our findings highlight the impact of integrating microbial population genomics with host metadata and demonstrate how host environmental pressures can lead to increased antibiotic susceptibility.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/individual_differences_among_deep_neural_network_models_73799089/">
          <h4 class="font-weight-bold">Individual differences among deep neural network models, bioRxiv, 2020-01-09</h4>
          <p>AbstractDeep neural networks (DNNs) excel at visual recognition tasks and are increasingly used as a modelling framework for neural computations in the primate brain. However, each DNN instance, just like each individual brain, has a unique connectivity and representational profile. Here, we investigate individual differences among DNN instances that arise from varying only the random initialization of the network weights. Using representational similarity analysis, we demonstrate that this minimal change in initial conditions prior to training leads to substantial differences in intermediate and higher-level network representations, despite achieving indistinguishable network-level classification performance. We locate the origins of the effects in an under-constrained alignment of category exemplars, rather than a misalignment of category centroids. Furthermore, while network regularization can increase the consistency of learned representations, considerable differences remain. These results suggest that computational neuroscientists working with DNNs should base their inferences on multiple networks instances instead of single off-the-shelf networks.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/three_rules_explain_transgenerational_small_rna_inheritance_in_c._elegans_73799005/">
          <h4 class="font-weight-bold">Three Rules Explain Transgenerational Small RNA Inheritance in C. elegans, bioRxiv, 2020-01-09</h4>
          <p>Life experiences trigger transgenerational small RNA-based responses in C. elegans nematodes. Dedicated machinery ensures that heritable effects would re-set, typically after a few generations. Here we show that isogenic individuals differ dramatically in the persistence of transgenerational responses. By examining lineages composed of &amp;gt;20,000 worms we reveal 3 inheritance rules (1) Once a response is initiated, each isogenic mother stochastically assumes an “inheritance state”, establishing a commitment that determines the fate of the inheritance. (2) The response that each mother transfers is uniform in each generation of her descendants. (3) The likelihood that an RNAi response would transmit to the progeny increases the more generations the response lasts, according to a “hot hand” principle. Mechanistically, the different parental “inheritance states” correspond to global changes in the expression levels of endogenous small RNAs, immune response genes, and targets of the conserved transcription factor HSF-1. We show that these rules predict the descendants’ developmental rate and resistance to stress.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_simple_cost-effective_and_robust_method_for_rrna_depletion_in_rna-sequencing_studies_73726276/">
          <h4 class="font-weight-bold">A simple, cost-effective, and robust method for rRNA depletion in RNA-sequencing studies, bioRxiv, 2020-01-07</h4>
          <p>AbstractThe profiling of gene expression by RNA-sequencing (RNA-seq) has enabled powerful studies of global transcriptional patterns in all organisms, including bacteria. Because the vast majority of RNA in bacteria is ribosomal RNA (rRNA), it is standard practice to deplete the rRNA from a total RNA sample such that the reads in an RNA-seq experiment derive predominantly from mRNA. One of the most commonly used commercial kits for rRNA depletion, the Ribo-Zero kit from Illumina, was recently discontinued. Here, we report the development a simple, cost-effective, and robust method for depleting rRNA that can be easily implemented by any lab or facility. We first developed an algorithm for designing biotinylated oligonucleotides that will hybridize tightly and specifically to the 23S, 16S, and 5S rRNAs from any species of interest. Precipitation of these oligonucleotides bound to rRNA by magnetic streptavidin beads then depletes rRNA from a complex, total RNA sample such that ~75-80% of reads in a typical RNA-seq experiment derive from mRNA. Importantly, we demonstrate a high correlation of RNA abundance or fold-change measurements in RNA-seq experiments between our method and the previously available Ribo-Zero kit. Complete details on the methodology are provided, including open-source software for designing oligonucleotides optimized for any bacterial species or metagenomic sample of interest.ImportanceThe ability to examine global patterns of gene expression in microbes through RNA-sequencing has fundamentally transformed microbiology. However, RNA-seq depends critically on the removal of ribosomal RNA from total RNA samples. Otherwise, rRNA would comprise upwards of 90% of the reads in a typical RNA-seq experiment, limiting the reads coming from messenger RNA or requiring high total read depth. A commonly used, kit for rRNA subtraction from Illumina was recently discontinued. Here, we report the development of a ‘do-it-yourself’ kit for rapid, cost-effective, and robust depletion of rRNA from total RNA. We present an algorithm for designing biotinylated oligonucleotides that will hybridize to the rRNAs from a target set of species. We then demonstrate that the designed oligos enable sufficient rRNA depletion to produce RNA-seq data with 75-80% of reads comming from mRNA. The methodology presented should enable RNA-seq studies on any species or metagenomic sample of interest.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/engineering_e._coli_for_magnetic_control_and_the_spatial_localization_of_functions_73711135/">
          <h4 class="font-weight-bold">Engineering E. coli for magnetic control and the spatial localization of functions, bioRxiv, 2020-01-07</h4>
          <p>AbstractThe fast-developing field of synthetic biology enables broad applications of programmed microorganisms including the development of whole-cell biosensors, delivery vehicles for therapeutics, or diagnostic agents. However, the lack of spatial control required for localizing microbial functions could limit their use and induce their dilution leading to ineffective action or dissemination. To overcome this limitation, the integration of magnetic properties into living systems enables a contact-less and orthogonal method for spatiotemporal control. Here, we generated a magnetic-sensing Escherichia coli by driving the formation of iron-rich bodies into bacteria. We found that these bacteria could be spatially controlled by magnetic forces and sustained cell growth and division, by transmitting asymmetrically their magnetic properties to one daughter cell. We combined the spatial control of bacteria with genetically encoded-adhesion properties to achieve the magnetic capture of specific target bacteria as well as the spatial modulation of human cell invasions.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/synthetic-biology">synthetic-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/glacier_ice_archives_fifteen-thousand-year-old_viruses_73713223/">
          <h4 class="font-weight-bold">Glacier ice archives fifteen-thousand-year-old viruses, bioRxiv, 2020-01-07</h4>
          <p>AbstractWhile glacier ice cores provide climate information over tens to hundreds of thousands of years, study of microbes is challenged by ultra-low-biomass conditions, and virtually nothing is known about co-occurring viruses. Here we establish ultra-clean microbial and viral sampling procedures and apply them to two ice cores from the Guliya ice cap (northwestern Tibetan Plateau, China) to study these archived communities. This method reduced intentionally contaminating bacterial, viral, and free DNA to background levels in artificial-ice-core control experiments, and was then applied to two authentic ice cores to profile their microbes and viruses. The microbes differed significantly across the two ice cores, presumably representing the very different climate conditions at the time of deposition that is similar to findings in other cores. Separately, viral particle enrichment and ultra-low-input quantitative viral metagenomic sequencing from ∼520 and ∼15,000 years old ice revealed 33 viral populations (i.e., species-level designations) that represented four known genera and likely 28 novel viral genera (assessed by gene-sharing networks). In silico host predictions linked 18 of the 33 viral populations to co-occurring abundant bacteria, including Methylobacterium, Sphingomonas, and Janthinobacterium, indicating that viruses infected several abundant microbial groups. Depth-specific viral communities were observed, presumably reflecting differences in the environmental conditions among the ice samples at the time of deposition. Together, these experiments establish a clean procedure for studying microbial and viral communities in low-biomass glacier ice and provide baseline information for glacier viruses, some of which appear to be associated with the dominant microbes in these ecosystems.ImportanceThis study establishes ultra-clean microbial and viral sampling procedures for glacier ice, which complements prior in silico decontamination methods and expands, for the first time, the clean procedures to viruses. Application of these methods to glacier ice confirmed prior common microbiological findings for a new ice core climate record, and provides a first window into viral genomes and their ecology from glacier ice across two time horizons, and emphasizes their likely impact on abundant microbial groups. Together these efforts provide clean sampling approaches and foundational datasets that should enable simultaneous access to an archived virosphere in glacier ice.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2020">2020</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/10/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/10/" class="page-link">10</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/11/" class="page-link">11</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/12/" class="page-link">12</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/13/" class="page-link">13</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/14/" class="page-link">14</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/15/" class="page-link">15</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/12/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/312/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
