<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/compositional_data_analysis_is_necessary_for_simulating_and_analyzing_rna-seq_data_56315555/">
          <h4 class="font-weight-bold">Compositional Data Analysis is necessary for simulating and analyzing RNA-Seq data, bioRxiv, 2019-03-02</h4>
          <p>Seq techniques (e.g. RNA-Seq) generate compositional datasets, i.e. the number of fragments sequenced is not proportional to the total RNA present. Thus, datasets carry only relative information, even though absolute RNA copy numbers are often of interest. Current normalization methods assume most features are not changing, which can lead to misleading conclusions when there are large shifts. However, there are few real datasets and no simulation protocols currently available that can directly benchmark methods when such large shifts occur.We present absSimSeq, an R package that simulates compositional data in the form of RNA-Seq reads. We tested several tools used for RNA-Seq differential analysis sleuth, DESeq2, edgeR, limma, sleuth and ALDEx2 (which explicitly takes a compositional approach). For these tools, we compared their standard normalization to either “compositional normalization”, which uses log-ratios to anchor the data on a set of negative control features, or RUVSeq, another tool that directly uses negative control features.We show that common normalizations result in reduced performance with current methods when there is a large change in the total RNA per cell. Performance improves when spike-ins are included and used by a compositional approach, even if the spike-ins have substantial variation. In contrast, RUVSeq, which normalizes count data rather than compositional data, has poor performance. Further, we show that previous criticisms of spike-ins did not take into account the compositional nature of the data. We conclude that absSimSeq can generate more representative datasets for testing performance, and that spike-ins should be more broadly used in a compositional manner to minimize misleading conclusions from differential analyses.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genetic_inhibition_of_pcsk9_atherogenic_lipoprotein_concentrations_and_calcific_aortic_valve_stenosis_56256162/">
          <h4 class="font-weight-bold">Genetic inhibition of PCSK9, atherogenic lipoprotein concentrations, and calcific aortic valve stenosis, bioRxiv, 2019-03-02</h4>
          <p>Background Proprotein convertase subtilisinkexin type 9 (PCSK9) inhibition reduces plasma concentrations of low-density lipoprotein cholesterol (LDL-C), apolipoprotein B (apoB) and lipoprotein(a) [Lp(a)]. Atherogenic lipoprotein levels have been linked with calcific aortic valve stenosis (CAVS). Our objectives were to determine the association between variants in PCSK9 and lipoprotein-lipid levels, coronary artery disease (CAD) and CAVS, and to evaluate if PCSK9 could be implicated in aortic valve interstitial cells (VICs) calcification.Methods We built a genetic risk score weight for LDL-C levels (wGRS) using 10 independent PCSK9 single nucleotide polymorphisms and determined its association with lipoprotein-lipid levels in 9692 participants of the EPIC-Norfolk study. We investigated the association between the wGRS and CAD and CAVS in the UK Biobank, as well as the association between the PCSK9 R46L variant and CAVS in a meta-analysis of published prospective, population-based studies (Copenhagen studies, 1463 cases101,620 controls) and unpublished studies (UK Biobank, 1350 cases349,043 controls, Malmo Diet and Cancer study, 682 cases5963 controls and EPIC-Norfolk study, 508 cases20,421 controls). We evaluated PCSK9 expression and localization in explanted aortic valves by capillary Western blot and immunohistochemistry in patients with and without CAVS. Von Kossa staining was used to visualize aortic leaflet calcium deposits. PCSK9 expression under oxidative stress conditions in VICs was assessed.Results The wGRS was significantly associated with lower LDL-C and apoB (p&amp;lt;0.001), but not with Lp(a). In the UK Biobank, the association of PCSK9 variants with CAD were positively correlated with their effects on apoB levels. CAVS was less prevalent in carriers of the PCSK9 R46L variant [odds ratio=0.71 (95% confidence interval, 0.57-0.88), p&amp;lt;0.001]. PCSK9 expression was elevated in the aortic valves of patients with aortic sclerosis and CAVS compared to controls. In calcified leaflets, PCSK9 co-localized with calcium deposits. PCSK9 expression was induced by oxidative stress in VICs. Conclusion Genetic inhibition of PCSK9 is associated with lifelong reductions in the levels of non-Lp(a) apoB-containing lipoproteins as well as lower odds of CAD and CAVS. PCSK9 is abundant in fibrotic and calcified aortic leaflets. Oxidative stress increases PCSK9 expression in VICs. These results provide a rationale for performing randomized clinical trials of PCSK9 inhibition in CAVS.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_mechanism_to_minimize_errors_during_non-homologous_end_joining_56200760/">
          <h4 class="font-weight-bold">A mechanism to minimize errors during non-homologous end joining, bioRxiv, 2019-03-01</h4>
          <p>SUMMARYEnzymatic processing of DNA underlies all DNA repair, yet inappropriate DNA processing must be avoided. In vertebrates, double-strand breaks are repaired predominantly by non-homologous end-joining (NHEJ), which directly ligates DNA ends. NHEJ has the potential to be highly mutagenic because it employs DNA polymerases, nucleases, and other enzymes that modify incompatible DNA ends to allow their ligation. Using a biochemical system that recapitulates key features of cellular NHEJ, we show that end-processing requires formation of a “short-range synaptic complex” in which DNA ends are closely aligned in a ligation-competent state. Furthermore, single-molecule imaging directly demonstrates that processing occurs within the short-range complex. This confinement of end processing to a ligation-competent complex ensures that DNA ends undergo ligation as soon as they become compatible, thereby minimizing mutagenesis. Our results illustrate how the coordination of enzymatic catalysis with higher-order structural organization of substrate maximizes the fidelity of DNA repair.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/ctcf_confers_local_nucleosome_resiliency_after_dna_replication_and_during_mitosis_56227964/">
          <h4 class="font-weight-bold">CTCF confers local nucleosome resiliency after DNA replication and during mitosis, bioRxiv, 2019-03-01</h4>
          <p>The access of Transcription Factors (TFs) to their cognate DNA binding motifs requires a precise control over nucleosome positioning. This is especially important following DNA replication and during mitosis, both resulting in profound changes in nucleosome organization over TF binding regions. Using mouse Embryonic Stem (ES) cells, we show that the TF CTCF displaces nucleosomes from its binding site and locally organizes large and phased nucleosomal arrays, not only in interphase steady-state but also immediately after replication and during mitosis. While regions bound by other TFs, such as Oct4 and Sox2, display major rearrangement, the post-replication and mitotic nucleosome organization activity of CTCF is not likely to be unique Esrrb binding regions are also characterized by persistent nucleosome positioning. Therefore, we propose that selected TFs, such as CTCF and Esrrb, govern the inheritance of nucleosome positioning at gene regulatory regions throughout the ES cell-cycle.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/high-density_spatial_transcriptomics_arrays_for_in_situ_tissue_profiling_56205399/">
          <h4 class="font-weight-bold">High-density spatial transcriptomics arrays for in situ tissue profiling, bioRxiv, 2019-03-01</h4>
          <p>AbstractTissue function relies on the precise spatial organization of cells characterized by distinct molecular profiles. Single-cell RNA-Seq captures molecular profiles but not spatial organization. Conversely, spatial profiling assays to date have lacked global transcriptome information, throughput or single-cell resolution. Here, we develop High-Density Spatial Transcriptomics (HDST), a method for RNA-Seq at high spatial resolution. Spatially barcoded reverse transcription oligonucleotides are coupled to beads that are randomly deposited into tightly packed individual microsized wells on a slide. The position of each bead is decoded with sequential hybridization using complementary oligonucleotides providing a unique bead-specific spatial address. We then capture, and spatially in situ barcode, RNA from the histological tissue sections placed on the HDST array. HDST recovers hundreds of thousands of transcript-coupled spatial barcodes per experiment at 2 μm resolution. We demonstrate HDST in the mouse brain, use it to resolve spatial expression patterns and cell types, and show how to combine it with histological stains to relate expression patterns to tissue architecture and anatomy. HDST opens the way to spatial analysis of tissues at high resolution.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/non-homologous_end_joining_minimizes_errors_by_coordinating_dna_processing_with_ligation_56200760/">
          <h4 class="font-weight-bold">Non-homologous end joining minimizes errors by coordinating DNA processing with ligation, bioRxiv, 2019-03-01</h4>
          <p>Genome stability requires efficient and faithful repair of DNA double-strand breaks (DSBs). The predominant DSB repair pathway in human cells is non-homologous end-joining (NHEJ), which directly ligates DNA ends1–5. Broken DNA ends at DSBs are chemically diverse, and many are not compatible for direct ligation by the NHEJ-associated DNA Ligase IV (Lig4). To solve this problem, NHEJ end-processing enzymes including polymerases and nucleases modify ends until they are ligatable. How cells regulate end processing to minimize unnecessary genomic alterations6 during repair of pathological DSBs remains unknown. Using a biochemical system that recapitulates key features of cellular NHEJ, we previously demonstrated that DNA ends are initially tethered at a distance, followed by Lig4-mediated formation of a “short-range synaptic complex” in which DNA ends are closely aligned for ligation7. Here, we show that a wide variety of end-processing activities all depend on formation of the short-range complex. Moreover, using real-time single molecule imaging, we find that end processing occurs within the short-range complex. Confining end processing to the Lig4-dependent short-range synaptic complex promotes immediate ligation of compatible ends and ensures that incompatible ends are ligated as soon as they become compatible, thereby minimizing end processing. Our results elucidate how NHEJ exploits end processing to achieve versatility while minimizing errors that cause genome instability.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/43/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/40/" class="page-link">40</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/41/" class="page-link">41</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/42/" class="page-link">42</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/43/" class="page-link">43</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/0-100-users/page/44/" class="page-link">44</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/45/" class="page-link">45</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/46/" class="page-link">46</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/47/" class="page-link">47</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/48/" class="page-link">48</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/45/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/123/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
