<!DOCTYPE html>
<html><head>
  
  <title>Developmental Biology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/chromatin_accessibility_established_by_pou5f3_sox19b_and_nanog_primes_genes_for_activity_during_zebrafish_genome_activation_60528681/">
          <h4 class="font-weight-bold">Chromatin accessibility established by Pou5f3, Sox19b and Nanog primes genes for activity during zebrafish genome activation, bioRxiv, 2019-05-16</h4>
          <p>ABSTRACTIn many organisms, early embryonic development is driven by maternally provided factors until the controlled onset of transcription during zygotic genome activation. The regulation of chromatin accessibility and its relationship to gene activity during this transition remains poorly understood. Here, we generated chromatin accessibility maps from genome activation until the onset of lineage specification. During this period, chromatin accessibility increases at regulatory elements. This increase is independent of RNA polymerase II-mediated transcription, with the exception of the hyper-transcribed miR-430 locus. Instead, accessibility often precedes the transcription of associated genes. Loss of the maternal transcription factors Pou5f3, Sox19b, and Nanog, which are known to be required for zebrafish genome activation, results in decreased accessibility at regulatory elements. Importantly, the accessibility of regulatory regions, especially when established by Pou5f3, Sox19b and Nanog, is predictive for future transcription. Our results show that the maternally provided transcription factors Pou5f3, Sox19b, and Nanog open up chromatin and prime genes for activity during zygotic genome activation in zebrafish.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/developmentally_regulated_shh_expression_is_robust_to_tad_perturbations_59072674/">
          <h4 class="font-weight-bold">Developmentally regulated Shh expression is robust to TAD perturbations, bioRxiv, 2019-04-16</h4>
          <p>AbstractTopologically Associating Domains (TADs) have been proposed to both guide and constrain enhancer activity. Shh is located within a TAD known to contain all its enhancers. To investigate the importance of chromatin conformation and TAD integrity on developmental gene regulation, we have manipulated the Shh TAD – creating internal deletions, deleting CTCF sites including those at TAD boundaries, as well as larger deletions and inversions of TAD boundaries. Chromosome conformation capture and fluorescence in situ hybridisation assays were used the investigate changes in chromatin conformation that result from these manipulations. Our data suggest that the substantial alteration of TAD structure has no readily detectable effect on Shh expression patterns during development – except where enhancers are deleted - and results in no detectable phenotypes. Only in the case of a larger deletion of one TAD boundary could some ectopic influence of the Shh limb enhancer be detected on a gene - Mnx1 in the neighbouring TAD. Our data suggests that, contrary to expectations, the developmental regulation of Shh expression is remarkably robust to TAD perturbations.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/parallel_control_of_mechanosensory_hair_cell_orientation_by_the_pcp_and_wnt_pathways_54413002/">
          <h4 class="font-weight-bold">Parallel control of mechanosensory hair cell orientation by the PCP and Wnt pathways, bioRxiv, 2019-01-24</h4>
          <p>Cell polarity plays a crucial role during development of vertebrates and invertebrates. Planar Cell Polarity (PCP) is defined as the coordinated polarity of cells within a tissue axis and is essential for processes such as gastrulation, neural tube closure or hearing. Wnt ligands can be instructive or permissive during PCP-dependent processes, and Wnt pathway mutants are often classified as PCP mutants due to the complexity and the similarities between their phenotypes. Our studies of the zebrafish sensory lateral line reveal that disruptions of the PCP and Wnt pathways have differential effects on hair cell orientations. While mutations in PCP genes cause random orientations of hair cells, mutations in Wnt pathway members induce hair cells to adopt a concentric pattern. We show that PCP signaling is normal in hair cells of Wnt pathway mutants and that the concentric hair cell phenotype is due to altered organization of the surrounding support cells. Thus, the PCP and Wnt pathways work in parallel, as separate pathways to establish proper hair cell orientation. Our data suggest that coordinated support cell organization is established during the formation of lateral line primordia, much earlier than the appearance of hair cells. Together, these finding reveal that hair cell orientation defects are not solely explained by defects in PCP signaling and that some hair cell phenotypes warrant reevaluation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/single_cell_multi-omics_profiling_reveals_a_hierarchical_epigenetic_landscape_during_mammalian_germ_layer_specification_supplementary_figures_53889608/">
          <h4 class="font-weight-bold">Single cell multi-omics profiling reveals a hierarchical epigenetic landscape during mammalian germ layer specification Supplementary Figures, bioRxiv, 2019-01-14</h4>
          <p>Formation of the three primary germ layers during gastrulation is an essential step in the establishment of the vertebrate body plan. Recent studies employing single cell RNA-sequencing have identified major transcriptional changes associated with germ layer specification. Global epigenetic reprogramming accompanies these changes, but the role of the epigenome in regulating early cell fate choice remains unresolved, and the coordination between different epigenetic layers is unclear. Here we describe the first single cell triple-omics map of chromatin accessibility, DNA methylation and RNA expression during the exit from pluripotency and the onset of gastrulation in mouse embryos. We find dynamic dependencies between the different molecular layers, with evidence for distinct modes of epigenetic regulation. The initial exit from pluripotency coincides with the establishment of a global repressive epigenetic landscape, followed by the emergence of local lineage-specific epigenetic patterns during gastrulation. Notably, cells committed to mesoderm and endoderm undergo widespread coordinated epigenetic rearrangements, driven by loss of methylation in enhancer marks and a concomitant increase of chromatin accessibility. In striking contrast, the epigenetic landscape of ectodermal cells is already established in the early epiblast. Hence, regulatory elements associated with each germ layer are either epigenetically primed or epigenetically remodelled prior to overt cell fate decisions during gastrulation, providing the molecular logic for a hierarchical emergence of the primary germ layers.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/stem_cell_differentiation_trajectories_in_hydra_resolved_at_single-cell_resolution_50682487/">
          <h4 class="font-weight-bold">Stem cell differentiation trajectories in Hydra resolved at single-cell resolution, bioRxiv, 2018-11-03</h4>
          <p>AbstractThe adult Hydra polyp continuously renews all of its cells using three separate stem cell populations, but the genetic pathways enabling homeostatic tissue maintenance are not well understood. We used Drop-seq to sequence transcriptomes of 24,985 single Hydra cells and identified the molecular signatures of a broad spectrum of cell states, from stem cells to terminally differentiated cells. We constructed differentiation trajectories for each cell lineage and identified the transcription factors expressed along these trajectories, thus creating a comprehensive molecular map of all developmental lineages in the adult animal. We unexpectedly found that neuron and gland cell differentiation transits through a common progenitor state, suggesting a shared evolutionary history for these secretory cell types. Finally, we have built the first gene expression map of the Hydra nervous system. By producing a comprehensive molecular description of the adult Hydra polyp, we have generated a resource for addressing fundamental questions regarding the evolution of developmental processes and nervous system function.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/reconstructing_the_human_first_trimester_fetalmaternal_interface_using_single_cell_transcriptomics_49017765/">
          <h4 class="font-weight-bold">Reconstructing the human first trimester fetal–maternal interface using single cell transcriptomics, bioRxiv, 2018-09-29</h4>
          <p>SummaryDuring the early weeks of human pregnancy, the fetal placenta implants into the uterine mucosa (decidua) where placental trophoblast cells intermingle and communicate with maternal cells. Here, we profile transcriptomes of ∼50,000 single cells from this unique microenvironment, sampling matched first trimester maternal blood and decidua, and fetal cells from the placenta itself. We define the cellular composition of human decidua, revealing five distinct subsets of decidual fibroblasts with differing growth factors and hormone production profiles, and show that fibroblast states define two distinct decidual layers. Among decidual NK cells, we resolve three subsets, each with a different immunomodulatory and chemokine profile. We develop a repository of ligand-receptor pairs (&lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpwww.CellPhoneDB.org&gt;www.CellPhoneDB.org&lt;jatsext-link&gt;) and a statistical tool to predict the probability of cell-cell interactions via these pairs, highlighting specific interactions between decidual NK cells and invading fetal extravillous trophoblast cells, maternal immune and stromal cells. Our single cell atlas of the maternal-fetal interface reveals the cellular organization and interactions critical for placentation and reproductive success.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/developmental-biology/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/developmental-biology/page/2/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="/audiences/tags/developmental-biology/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/developmental-biology/page/2/" class="page-link">2</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/developmental-biology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/developmental-biology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/developmental-biology/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/developmental-biology/page/6/" class="page-link">6</a></li>
            
        

        
        <li class="page-item"><a href="/audiences/tags/developmental-biology/page/4/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/developmental-biology/page/6/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
