<!DOCTYPE html>
<html><head>
  
  <title>200 500 Users | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/maternal_gut_and_breast_milk_microbiota_affect_infant_gut_antibiotic_resistome_and_mobile_genetic_elements_48827891/">
          <h4 class="font-weight-bold">Maternal gut and breast milk microbiota affect infant gut antibiotic resistome and mobile genetic elements, Nature Communications, 2018-09-18</h4>
          <p>The infant gut microbiota has a high abundance of antibiotic resistance genes (ARGs) compared to adults, even in the absence of antibiotic exposure. Here we study potential sources of infant gut ARGs by performing metagenomic sequencing of breast milk, as well as infant and maternal gut microbiomes. We find that fecal ARG and mobile genetic element (MGE) profiles of infants are more similar to those of their own mothers than to those of unrelated mothers. MGEs in mothersâ€™ breast milk are also shared with their own infants. Termination of breastfeeding and intrapartum antibiotic prophylaxis of mothers, which have the potential to affect microbial community composition, are associated with higher abundances of specific ARGs, the composition of which is largely shaped by bacterial phylogeny in the infant gut. Our results suggest that infants inherit the legacy of past antibiotic consumption of their mothers via transmission of genes, but microbiota composition still strongly impacts the overall resistance load.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/nature%20communications">nature communications</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/genetic_analysis_of_over_1_million_people_identifies_535_new_loci_associated_with_blood_pressure_traits_48542474/">
          <h4 class="font-weight-bold">Genetic analysis of over 1 million people identifies 535 new loci associated with blood pressure traits, Nature Genetics, 2018-09-17</h4>
          <p>High blood pressure is a highly heritable and modifiable risk factor for cardiovascular disease. We report the largest genetic association study of blood pressure traits (systolic, diastolic and pulse pressure) to date in over 1 million people of European ancestry. We identify 535 novel blood pressure loci that not only offer new biological insights into blood pressure regulation but also highlight shared genetic architecture between blood pressure and lifestyle exposures. Our findings identify new biological pathways for blood pressure regulation with potential for improved cardiovascular disease prevention in the future.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/nature%20genetics">nature genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/a_comprehensive_analysis_of_rna_sequences_reveals_macroscopic_somatic_clonal_expansion_across_normal_tissues_48266255/">
          <h4 class="font-weight-bold">A comprehensive analysis of RNA sequences reveals macroscopic somatic clonal expansion across normal tissues, bioRxiv, 2018-09-14</h4>
          <p>Cancer genome studies have significantly advanced our knowledge of somatic mutations. However, how these mutations accumulate in normal cells and whether they promote pre-cancerous lesions remains poorly understood. Here we perform a comprehensive analysis of normal tissues by utilizing RNA sequencing data from ~6,700 samples across 29 normal tissues collected as part of the Genotype-Tissue Expression (GTEx) project. We identify somatic mutations using a newly developed pipeline, RNA-MuTect, for calling somatic mutations directly from RNA-seq samples and their matched-normal DNA. When applied to the GTEx dataset, we detect multiple variants across different tissues and find that mutation burden is associated with both the age of the individual and tissue proliferation rate. We also detect hotspot cancer mutations that share tissue specificity with their matched cancer type. This study is the first to analyze a large number of samples across multiple normal tissues, identifying clones with genomic aberrations observed in cancer.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/predicting_mrna_abundance_directly_from_genomic_sequence_using_deep_convolutional_neural_networks_48267284/">
          <h4 class="font-weight-bold">Predicting mRNA abundance directly from genomic sequence using deep convolutional neural networks, bioRxiv, 2018-09-14</h4>
          <p>Algorithms that accurately predict gene structure from primary sequence alone were transformative for annotating the human genome. Can we also predict the expression levels of genes based solely on genome sequence? Here we sought to apply deep convolutional neural networks towards this goal. Surprisingly, a model that includes only promoter sequences and features associated with mRNA stability explains 59% and 71% of variation in steady-state mRNA levels in human and mouse, respectively. This model, which we call Xpresso, more than doubles the accuracy of alternative sequence-based models, and isolates rules as predictive as models relying on ChIP-seq data. Xpresso recapitulates genome-wide patterns of transcriptional activity and predicts the influence of enhancers, heterochromatic domains, and microRNAs. Model interpretation reveals that promoter-proximal CpG dinucleotides strongly predict transcriptional activity. Looking forward, we propose the accurate prediction of cell type-specific gene expression based solely on primary sequence as a grand challenge for the field.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/bio-on-magnetic-beads_bomb_open_platform_for_high-throughput_nucleic_acid_extraction_and_manipulation_48184997/">
          <h4 class="font-weight-bold">Bio-On-Magnetic-Beads (BOMB) Open platform for high-throughput nucleic acid extraction and manipulation, bioRxiv, 2018-09-13</h4>
          <p>AbstractCurrent molecular biology laboratories rely heavily on the purification and manipulation of nucleic acids. Yet, commonly used centrifuge-and column-based protocols require specialised equipment, often use toxic reagents and are not economically scalable or practical to use in a high-throughput manner. Although it has been known for some time that magnetic beads can provide an elegant answer to these issues, the development of open-source protocols based on beads has been limited. In this article, we provide step-by-step instructions for an easy synthesis of functionalised magnetic beads, and detailed protocols for their use in the high-throughput purification of plasmids, genomic DNA and total RNA from different sources, as well as environmental TNA and PCR amplicons. We also provide a bead-based protocol for bisulfite conversion, and size selection of DNA and RNA fragments. Comparison to other methods highlights the capability, versatility and extreme cost-effectiveness of using magnetic beads. These open source protocols and the associated webpage (&lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsbomb.bio&gt;httpsbomb.bio&lt;jatsext-link&gt;) can serve as a platform for further protocol customisation and community engagement.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/re-evaluating_one-step_generation_of_mice_carrying_conditional_alleles_by_crispr-cas9-mediated_genome_editing_technology_47288281/">
          <h4 class="font-weight-bold">Re-Evaluating One-step Generation of Mice Carrying Conditional Alleles by CRISPR-Cas9-Mediated Genome Editing Technology, bioRxiv, 2018-08-31</h4>
          <p>AbstractCRISPR-Cas9 gene editing technology has considerably facilitated the generation of mouse knockout alleles, relieving many of the cumbersome and time-consuming steps of traditional mouse embryonic stem cell technology. However, the generation of conditional knockout alleles remains an important challenge. An earlier study reported up to 16% efficiency in generating conditional knockout alleles in mice using 2 single guide RNAs (sgRNA) and 2 single-stranded oligonucleotides (ssODN) (2sgRNA-2ssODN). We re-evaluated this method from a large data set generated from a consortium consisting of 17 transgenic core facilities or laboratories or programs across the world. The dataset constituted 17,887 microinjected or electroporated zygotes and 1,718 live born mice, of which only 15 (0.87%) mice harbored 2 correct LoxP insertions in cis configuration indicating a very low efficiency of the method. To determine the factors required to successfully generate conditional alleles using the 2sgRNA-2ssODN approach, we performed a generalized linear regression model. We show that factors such as the concentration of the sgRNA, Cas9 protein or the distance between the placement of LoxP insertions were not predictive for the success of this technique. The major predictor affecting the methodâ€™s success was the probability of simultaneously inserting intact proximal and distal LoxP sequences, without the loss of the DNA segment between the two sgRNA cleavage sites. Our analysis of a large data set indicates that the 2sgRNAâ€“2ssODN method generates a large number of undesired alleles (&amp;gt;99%), and a very small number of desired alleles (&amp;lt;1%) requiring, on average 1,192 zygotes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/200-500-users/" rel="first" class="page-link">Â« First</a></li>
        

        
        <li class="page-item"><a href="/tags/200-500-users/page/23/" rel="prev" class="page-link">â€¹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/20/" class="page-link">20</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/21/" class="page-link">21</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/22/" class="page-link">22</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/23/" class="page-link">23</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/200-500-users/page/24/" class="page-link">24</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/25/" class="page-link">25</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/26/" class="page-link">26</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/27/" class="page-link">27</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/28/" class="page-link">28</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/200-500-users/page/25/" rel="next" class="page-link">Next â€º</a></li>
        

        
        
        <li class="page-item"><a href="/tags/200-500-users/page/32/" rel="last" class="page-link">Last Â»</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
