<!DOCTYPE html>
<html><head>
  
  <title>Developmental Biology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/liposome-based_transfection_enhances_rnai_and_crispr-mediated_mutagenesis_in_non-model_nematode_systems_48947328/">
          <h4 class="font-weight-bold">Liposome-based transfection enhances RNAi and CRISPR-mediated mutagenesis in non-model nematode systems, bioRxiv, 2018-09-27</h4>
          <p>AbstractNematodes belong to one of the most diverse animal phyla. However, functional genomic studies in nematodes, other than in a few species, have often been limited in their reliability and success. Here we report that by combining liposome-based technology with microinjection, we were able to establish a wide range of genomic techniques in the newly described nematode genus Auanema. The method also allowed heritable changes in dauer larvae of Auanema, despite the immaturity of the gonad at the time of the microinjection. As proof of concept for potential functional studies in other nematode species, we also induced RNAi in the free-living nematode Pristionchus pacificus and targeted the human parasite Strongyloides stercoralis.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/integrin-mediated_attachment_of_the_blastoderm_to_the_vitelline_envelope_impacts_gastrulation_of_insects_48653929/">
          <h4 class="font-weight-bold">Integrin-mediated attachment of the blastoderm to the vitelline envelope impacts gastrulation of insects, bioRxiv, 2018-09-19</h4>
          <p>AbstractDuring gastrulation, physical forces reshape the simple embryonic tissue to form a complex body plan of multicellular organisms1. These forces often cause large-scale asymmetric movements of the embryonic tissue2,3. In many embryos, the tissue undergoing gastrulation movements is surrounded by a rigid protective shell4,5. While it is well recognized that gastrulation movements depend on forces generated by tissue-intrinsic contractility6,7, it is not known if interactions between the tissue and the protective shell provide additional forces that impact gastrulation. Here we show that a particular part of the blastoderm tissue of the red flour beetle Tribolium castaneum tightly adheres in a temporally coordinated manner to the vitelline envelope surrounding the embryo. This attachment generates an additional force that counteracts the tissue-intrinsic contractile forces to create asymmetric tissue movements. Furthermore, this localized attachment is mediated by a specific integrin, and its knock-down leads to a gastrulation phenotype consistent with complete loss of attachment. Moreover, analysis of another integrin in the fruit fly Drosophila melanogaster suggests that gastrulation in this organism also relies on adhesion between the blastoderm and the vitelline. Together, our findings reveal a conserved mechanism whereby the spatiotemporal pattern of tissue adhesion to the vitelline envelope provides controllable counter-forces that shape gastrulation movements in insects.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/molecular_recording_of_mammalian_embryogenesis_46004647/">
          <h4 class="font-weight-bold">Molecular recording of mammalian embryogenesis, bioRxiv, 2018-08-03</h4>
          <p>Understanding the emergence of complex multicellular organisms from single totipotent cells, or ontogenesis, represents a foundational question in biology. The study of mammalian development is particularly challenging due to the difficulty of monitoring embryos in utero, the variability of progenitor field sizes, and the indeterminate relationship between the generation of uncommitted progenitors and their progression to subsequent stages. Here, we present a flexible, high information, multi-channel molecular recorder with a single cell (sc) readout and apply it as an evolving lineage tracer to define a mouse cell fate map from fertilization through gastrulation. By combining lineage information with scRNA-seq profiles, we recapitulate canonical developmental relationships between different tissue types and reveal an unexpected transcriptional convergence of endodermal cells from extra-embryonic and embryonic origins, illustrating how lineage information complements scRNA-seq to define cell types. Finally, we apply our cell fate map to estimate the number of embryonic progenitor cells and the degree of asymmetric partitioning within the pluripotent epiblast during specification. Our approach enables massively parallel, high-resolution recording of lineage and other information in mammalian systems to facilitate a quantitative framework for describing developmental processes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/altering_the_temporal_regulation_of_one_transcription_factor_drives_sensory_trade-offs_43772463/">
          <h4 class="font-weight-bold">Altering the temporal regulation of one transcription factor drives sensory trade-offs, bioRxiv, 2018-06-16</h4>
          <p>SUMMARYSize trade-offs of visual versus olfactory organs is a pervasive feature of animal evolution. Comparing Drosophila species, we find that larger eyes correlate with smaller antennae, where olfactory organs reside, and narrower faces. We demonstrate that this tradeoff arises through differential subdivision of the head primordium into visual versus non-visual fields. Specification of the visual field requires a highly-conserved eye development gene called eyeless in flies and Pax6 in humans. We discover that changes in the temporal regulation of eyeless expression during development is a conserved mechanism for sensory trade-offs within and between Drosophila species. We identify a natural single nucleotide polymorphism in the cis-regulatory region of eyeless that is sufficient to alter its temporal regulation and eye size. Because Pax6 is a conserved regulator of sensory placode subdivision, we propose that alterations in the mutual repression between sensory territories is a conserved mechanism for sensory trade-offs in animals.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/third-generation_in_situ_hybridization_chain_reaction_multiplexed_quantitative_sensitive_versatile_robust_34588711/">
          <h4 class="font-weight-bold">Third-generation in situ hybridization chain reaction multiplexed, quantitative, sensitive, versatile, robust, bioRxiv, 2018-03-20</h4>
          <p>ABSTRACTIn situ hybridization based on the mechanism of hybridization chain reaction (HCR) has addressed multi-decade challenges to imaging mRNA expression in diverse organisms, offering a unique combination of multiplexing, quantitation, sensitivity, resolution, and versatility. Here, with third-generation in situ HCR, we augment these capabilities using probes and amplifiers that combine to provide automatic background suppression throughout the protocol, ensuring that even if reagents bind non-specifically within the sample they will not generate amplified background. Automatic background suppression dramatically enhances performance and robustness, combining the benefits of higher signal-to-background with the convenience of using unoptimized probe sets for new targets and organisms. In situ HCR v3.0 enables multiplexed quantitative mRNA imaging with subcellular resolution in the anatomical context of whole-mount vertebrate embryos, multiplexed quantitative mRNA flow cytometry for high-throughput single-cell expression profiling, and multiplexed quantitative single-molecule mRNA imaging in thick autofluorescent samples.SUMMARYIn situ hybridization chain reaction (HCR) v3.0 exploits automatic background suppression to enable multiplexed quantitative mRNA imaging and flow cytometry with dramatically enhanced ease-of-use and performance.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/efficient_expression_of_genes_in_the_drosophila_germline_using_a_uas-promoter_free_of_interference_by_hsp70_pirnas_33866774/">
          <h4 class="font-weight-bold">Efficient expression of genes in the Drosophila germline using a UAS-promoter free of interference by Hsp70 piRNAs, bioRxiv, 2018-03-02</h4>
          <p>ABSTRACTControlling the expression of genes using a binary system involving the yeast GAL4 transcription factor has been a mainstay of Drosophila melanogaster developmental genetics for twenty-five years. However, most existing GAL4 expression constructs only function effectively in somatic cells, but not in germ cells during oogenesis, for unknown reasons. A special UAS promoter, UASp was created that does express during oogenesis, but the need to use different constructs for somatic and female germline cells has remained a significant technical limitation. Here we show that the expression problem of UASt and many other Drosophila molecular tools in germline cells is caused by their core Hsp70 promoter sequences, which are targeted in female germ cells by Hsp70-directed piRNAs generated from endogenous Hsp70 gene sequences. In a genetic background lacking genomic Hsp70 genes and associated piRNAs, UASt-based constructs function effectively during oogenesis. By reducing Hsp70 sequences targeted by piRNAs, we created UASz, which functions better than UASp in the germline and like UASt in somatic cells.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/developmental-biology/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/developmental-biology/page/3/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/developmental-biology/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/developmental-biology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/developmental-biology/page/3/" class="page-link">3</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/developmental-biology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/developmental-biology/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/developmental-biology/page/6/" class="page-link">6</a></li>
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/developmental-biology/page/5/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/developmental-biology/page/6/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
