<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/punctuated_evolution_shaped_modern_vertebrate_diversity_21156696/">
          <h4 class="font-weight-bold">Punctuated evolution shaped modern vertebrate diversity, bioRxiv, 2017-06-19</h4>
          <p>AbstractThe relative importance of different modes of evolution in shaping phenotypic diversity remains a hotly debated question. Fossil data suggest that stasis may be a common mode of evolution, while modern data suggest very fast rates of evolution. One way to reconcile these observations is to imagine that evolution is punctuated, rather than gradual, on geological time scales. To test this hypothesis, we developed a novel maximum likelihood framework for fitting Lévy processes to comparative morphological data. This class of stochastic processes includes both a gradual and punctuated component. We found that a plurality of modern vertebrate clades examined are best fit by punctuated processes over models of gradual change, gradual stasis, and adaptive radiation. When we compare our results to theoretical expectations of the rate and speed of regime shifts for models that detail fitness landscape dynamics, we find that our quantitative results are broadly compatible with both microevolutionary models and with observations from the fossil record.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/platform_for_rapid_nanobody_discovery_in_vitro_21125069/">
          <h4 class="font-weight-bold">Platform for rapid nanobody discovery in vitro, bioRxiv, 2017-06-17</h4>
          <p>AbstractCamelid single-domain antibody fragments (“nanobodies”) provide the remarkable specificity of antibodies within a single immunoglobulin VHH domain. This unique feature enables applications ranging from their use as biochemical tools to therapeutic agents. Virtually all nanobodies reported to date have been obtained by animal immunization, a bottleneck restricting many applications of this technology. To solve this problem, we developed a fully in vitro platform for nanobody discovery based on yeast surface display of a synthetic nanobody scaffold. This platform provides a facile and cost-effective method for rapidly isolating nanobodies targeting a diverse range of antigens. We provide a blueprint for identifying nanobodies starting from both purified and non-purified antigens, and in addition, we demonstrate application of the platform to discover rare conformationally-selective nanobodies to a lipid flippase and a G protein-coupled receptor. To facilitate broad deployment of this platform, we have made the library and all associated protocols publicly available.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biochemistry">biochemistry</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/identification_of_a_novel_interspecific_hybrid_yeast_from_a_metagenomic_spontaneously_inoculated_beer_sample_using_hi-c_21094275/">
          <h4 class="font-weight-bold">Identification of a novel interspecific hybrid yeast from a metagenomic spontaneously inoculated beer sample using Hi-C, bioRxiv, 2017-06-16</h4>
          <p>AbstractInterspecific hybridization is a common mechanism enabling genetic diversification and adaptation; however, the detection of hybrid species has been quite difficult. The identification of microbial hybrids is made even more complicated, as most environmental microbes are resistant to culturing and must be studied in their native mixed communities. We have previously adapted the chromosome conformation capture method Hi-C to the assembly of genomes from mixed populations. Here, we show the method’s application in assembling genomes directly from an uncultured, mixed population from a spontaneously inoculated beer sample. Our assembly method has enabled us to de-convolute 4 bacterial and 4 yeast genomes from this sample, including a putative yeast hybrid. Downstream isolation and analysis of this hybrid confirmed its genome to consist of Pichia membranifaciens and that of another related, but undescribed yeast. Our work shows that Hi-C-based metagenomic methods can overcome the limitation of traditional sequencing methods in studying complex mixtures of genomes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/beyond_consensus_embracing_heterogeneity_in_curated_neuroimaging_meta-analysis_21038399/">
          <h4 class="font-weight-bold">Beyond Consensus Embracing Heterogeneity in Curated Neuroimaging Meta-Analysis, bioRxiv, 2017-06-14</h4>
          <p>Coordinate-based meta-analysis can provide important insights into mind-brain relationships. A popular approach for curated small-scale meta-analysis is activation likelihood estimation (ALE), which identifies brain regions consistently activated across a selected set of experiments, such as within a functional domain or mental disorder. ALE can also be utilized in meta-analytic co-activation modeling (MACM) to identify brain regions consistently co-activated with a seed region. Therefore, ALE aims to find consensus across experiments, treating heterogeneity across experiments as noise. However, heterogeneity within an ALE analysis of a functional domain might indicate the presence of functional sub-domains. Similarly, heterogeneity within a MACM analysis might indicate the involvement of a seed region in multiple co-activation patterns that are dependent on task contexts. Here, we demonstrate the use of the author-topic model to automatically determine if heterogeneities within ALE-type meta-analyses can be robustly explained by a small number of latent patterns. In the first application, the author-topic modeling of experiments involving self-generated thought (N = 179) revealed cognitive components fractionating the default network. In the second application, the author-topic model revealed that the left inferior frontal junction (IFJ) participated in multiple task-dependent co-activation patterns (N = 323). Furthermore, the author-topic model estimates compared favorably with spatial independent component analysis in both simulation and real data. Overall, the results suggest that the author-topic model is a flexible tool for exploring heterogeneity in ALE-type meta-analyses that might arise from functional sub-domains, mental disorder subtypes or task-dependent co-activation patterns. Code for this study is publicly available (httpsgithub.comThomasYeoLabCBIGtreemasterstable_projectsmeta-analysisNgo2019_AuthorTopic).</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/graphtyper_population-scale_genotyping_using_pangenome_graphs_20936228/">
          <h4 class="font-weight-bold">Graphtyper Population-scale genotyping using pangenome graphs, bioRxiv, 2017-06-10</h4>
          <p>AbstractA fundamental requisite for genetic studies is an accurate determination of sequence variation. While human genome sequence diversity is increasingly well characterized, there is a need for efficient ways to utilize this knowledge in sequence analysis. Here we present Graphtyper, a publicly available novel algorithm and software for discovering and genotyping sequence variants. Graphtyper realigns short-read sequence data to a pangenome, a variation-aware graph structure that encodes sequence variation within a population by representing possible haplotypes as graph paths. Our results show that Graphtyper is fast, highly scalable, and provides sensitive and accurate genotype calls. Graphtyper genotyped 89.4 million sequence variants in whole-genomes of 28,075 Icelanders using less than 100,000 CPU days, including detailed genotyping of six human leukocyte antigen (HLA) genes. We show that Graphtyper is a valuable tool in characterizing sequence variation in population-scale sequencing studies.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/highly_parallel_genome_variant_engineering_with_crisprcas9_in_eukaryotic_cells_20911698/">
          <h4 class="font-weight-bold">Highly parallel genome variant engineering with CRISPRCas9 in eukaryotic cells, bioRxiv, 2017-06-09</h4>
          <p>AbstractDirect measurement of functional effects of DNA sequence variants throughout a genome is a major challenge. We developed a method that uses CRISPRCas9 to engineer many specific variants of interest in parallel in the budding yeast Saccharomyces cerevisiae, and to screen them for functional effects. We used the method to examine the functional consequences of premature termination codons (PTCs) at different locations within all annotated essential genes in yeast. We found that most PTCs were highly deleterious unless they occurred close to the C-terminal end and did not interrupt an annotated protein domain. Surprisingly, we discovered that some putatively essential genes are dispensable, while others have large dispensable regions. This approach can be used to profile the effects of large classes of variants in a high-throughput manner.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/101/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/98/" class="page-link">98</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/99/" class="page-link">99</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/100/" class="page-link">100</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/101/" class="page-link">101</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/102/" class="page-link">102</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/103/" class="page-link">103</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/104/" class="page-link">104</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/105/" class="page-link">105</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/106/" class="page-link">106</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/103/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/123/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
