<!DOCTYPE html>
<html><head>
  
  <title>Biochemistry | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/crispr-cas12a_target_binding_unleashes_single-stranded_dnase_activity_29659753/">
          <h4 class="font-weight-bold">CRISPR-Cas12a target binding unleashes single-stranded DNase activity, bioRxiv, 2017-11-30</h4>
          <p>AbstractCRISPR-Cas12a (Cpf1) proteins are RNA-guided DNA targeting enzymes that bind and cut DNA as components of bacterial adaptive immune systems. Like CRISPR-Cas9, Cas12a can be used as a powerful genome editing tool based on its ability to induce genetic changes in cells at sites of double-stranded DNA (dsDNA) cuts. Here we show that RNA-guided DNA binding unleashes robust, non-specific single-stranded DNA (ssDNA) cleavage activity in Cas12a sufficient to completely degrade both linear and circular ssDNA molecules within minutes. This activity, catalyzed by the same active site responsible for site-specific dsDNA cutting, indiscriminately shreds ssDNA with rapid multiple-turnover cleavage kinetics. Activation of ssDNA cutting requires faithful recognition of a DNA target sequence matching the 20-nucleotide guide RNA sequence with specificity sufficient to distinguish between closely related viral serotypes. We find that target-dependent ssDNA degradation, not observed for CRISPR-Cas9 enzymes, is a fundamental property of type V CRISPR-Cas12 proteins, revealing a fascinating parallel with the RNA-triggered general RNase activity of the type VI CRISPR-Cas13 enzymes.One Sentence SummaryCas12a (Cpf1) and related type V CRISPR interference proteins possess non-specific, single-stranded DNase activity upon activation by guide RNA-dependent DNA binding.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biochemistry">biochemistry</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/enhanced_proofreading_governs_crispr-cas9_targeting_accuracy_21571824/">
          <h4 class="font-weight-bold">Enhanced proofreading governs CRISPR-Cas9 targeting accuracy, bioRxiv, 2017-07-07</h4>
          <p>The RNA-guided CRISPR-Cas9 nuclease from Streptococcus pyogenes (SpCas9) has been widely repurposed for genome editing1-4. High-fidelity (SpCas9-HF1) and enhanced specificity (eSpCas9(1.1)) variants exhibit substantially reduced off-target cleavage in human cells, but the mechanism of target discrimination and the potential to further improve fidelity were unknown5-9. Using single-molecule Förster resonance energy transfer (smFRET) experiments, we show that both SpCas9-HF1 and eSpCas9(1.1) are trapped in an inactive state10 when bound to mismatched targets. We find that a non-catalytic domain within Cas9, REC3, recognizes target mismatches and governs the HNH nuclease to regulate overall catalytic competence. Exploiting this observation, we identified residues within REC3 involved in mismatch sensing and designed a new hyper-accurate Cas9 variant (HypaCas9) that retains robust on-target activity in human cells. These results offer a more comprehensive model to rationalize and modify the balance between target recognition and nuclease activation for precision genome editing.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biochemistry">biochemistry</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/crisprcas9-apex-mediated_proximity_labeling_enables_discovery_of_proteins_associated_with_a_predefined_genomic_locus_in_living_cells_21516410/">
          <h4 class="font-weight-bold">CRISPRCas9-APEX-mediated proximity labeling enables discovery of proteins associated with a predefined genomic locus in living cells, bioRxiv, 2017-07-05</h4>
          <p>AbstractThe activation or repression of a gene’s expression is primarily controlled by changes in the proteins that occupy its regulatory elements. The most common method to identify proteins associated with genomic loci is chromatin immunoprecipitation (ChIP). While having greatly advanced our understanding of gene expression regulation, ChIP requires specific, high quality, IP-competent antibodies against nominated proteins, which can limit its utility and scope for discovery. Thus, a method able to discover and identify proteins associated with a particular genomic locus within the native cellular context would be extremely valuable. Here, we present a novel technology combining recent advances in chemical biology, genome targeting, and quantitative mass spectrometry to develop genomic locus proteomics, a method able to identify proteins which occupy a specific genomic locus.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biochemistry">biochemistry</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/platform_for_rapid_nanobody_discovery_in_vitro_21125069/">
          <h4 class="font-weight-bold">Platform for rapid nanobody discovery in vitro, bioRxiv, 2017-06-17</h4>
          <p>AbstractCamelid single-domain antibody fragments (“nanobodies”) provide the remarkable specificity of antibodies within a single immunoglobulin VHH domain. This unique feature enables applications ranging from their use as biochemical tools to therapeutic agents. Virtually all nanobodies reported to date have been obtained by animal immunization, a bottleneck restricting many applications of this technology. To solve this problem, we developed a fully in vitro platform for nanobody discovery based on yeast surface display of a synthetic nanobody scaffold. This platform provides a facile and cost-effective method for rapidly isolating nanobodies targeting a diverse range of antigens. We provide a blueprint for identifying nanobodies starting from both purified and non-purified antigens, and in addition, we demonstrate application of the platform to discover rare conformationally-selective nanobodies to a lipid flippase and a G protein-coupled receptor. To facilitate broad deployment of this platform, we have made the library and all associated protocols publicly available.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biochemistry">biochemistry</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/dna-dependent_rna_cleavage_by_the_natronobacterium_gregoryi_argonaute_15616406/">
          <h4 class="font-weight-bold">DNA-dependent RNA cleavage by the Natronobacterium gregoryi Argonaute, bioRxiv, 2017-01-21</h4>
          <p>AbstractWe show here that, unlike most other prokaryotic Argonaute (Ago) proteins, which are DNA-guided endonucleases, the Natronobacterium gregoryi-derived Ago (NgAgo) can function as a DNA-guided endoribonuclease, cleaving RNA, rather than DNA, in a targeted manner. The NgAgo protein, in complex with 5’-hydroxylated or 5’-phosphrylated oligodeoxyribonucleotides (ODNs) of variable lengths, split RNA targets into two or more fragments in vitro, suggesting its physiological role in bacteria and demonstrating a potential for degrading RNA molecules such as mRNA or lncRNA in eukaryotic cells in a targeted manner.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biochemistry">biochemistry</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biochemistry/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biochemistry/page/2/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biochemistry/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biochemistry/page/2/" class="page-link">2</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/biochemistry/page/3/" class="page-link">3</a></li>
            
        

        

        
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
