<!DOCTYPE html>
<html><head>
  
  <title>2018 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/brain-score_which_artificial_neural_network_for_object_recognition_is_most_brain-like_47699841/">
          <h4 class="font-weight-bold">Brain-Score Which Artificial Neural Network for Object Recognition is most Brain-Like?, bioRxiv, 2018-09-05</h4>
          <p>The internal representations of early deep artificial neural networks (ANNs) were found to be remarkably similar to the internal neural representations measured experimentally in the primate brain. Here we ask, as deep ANNs have continued to evolve, are they becoming more or less brain-like? ANNs that are most functionally similar to the brain will contain mechanisms that are most like those used by the brain. We therefore developed Brain-Score – a composite of multiple neural and behavioral benchmarks that score any ANN on how similar it is to the brain’s mechanisms for core object recognition – and we deployed it to evaluate a wide range of state-of-the-art deep ANNs. Using this scoring system, we here report that (1) DenseNet-169, CORnet-S and ResNet-101 are the most brain-like ANNs. (2) There remains considerable variability in neural and behavioral responses that is not predicted by any ANN, suggesting that no ANN model has yet captured all the relevant mechanisms. (3) Extending prior work, we found that gains in ANN ImageNet performance led to gains on Brain-Score. However, correlation weakened at ≥ 70% top-1 ImageNet performance, suggesting that additional guidance from neuroscience is needed to make further advances in capturing brain mechanisms. (4) We uncovered smaller (i.e. less complex) ANNs that are more brain-like than many of the best-performing ImageNet models, which suggests the opportunity to simplify ANNs to better understand the ventral stream. The scoring system used here is far from complete. However, we propose that evaluating and tracking model-benchmark correspondences through a Brain-Score that is regularly updated with new brain data is an exciting opportunity experimental benchmarks can be used to guide machine network evolution, and machine networks are mechanistic hypotheses of the brain’s network and thus drive next experiments. To facilitate both of these, we release &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpBrain-Score.org&gt;Brain-Score.org&lt;jatsext-link&gt; a platform that hosts the neural and behavioral benchmarks, where ANNs for visual processing can be submitted to receive a Brain-Score and their rank relative to other models, and where new experimental data can be naturally incorporated.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/integrated_computational_and_experimental_identification_of_p53_kras_and_vhl_mutant_selection_associated_with_crispr-cas9_editing_47626399/">
          <h4 class="font-weight-bold">Integrated computational and experimental identification of p53, KRAS and VHL mutant selection associated with CRISPR-Cas9 editing, bioRxiv, 2018-09-04</h4>
          <p>AbstractRecent studies have reported that CRISPR-Cas9 gene editing induces a p53-dependent DNA damage response in primary cells, which may select for cells with oncogenic p53 mutations11,12. It is unclear whether these CRISPR-induced changes are applicable to different cell types, and whether CRISPR gene editing may select for other oncogenic mutations. Addressing these questions, we analyzed genome-wide CRISPR and RNAi screens to systematically chart the mutation selection potential of CRISPR knockouts across the whole exome. Our analysis suggests that CRISPR gene editing can select for mutants of KRAS and VHL, at a level comparable to that reported for p53. These predictions were further validated in a genome-wide manner by analyzing independent CRISPR screens and patients’ tumor data. Finally, we performed a new set of pooled and arrayed CRISPR screens to evaluate the competition between CRISPR-edited isogenic p53 WT and mutant cell lines, which further validated our predictions. In summary, our study systematically charts and points to the potential selection of specific cancer driver mutations during CRISPR-Cas9 gene editing.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/cancer-biology">cancer-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_genetic_architecture_of_the_human_cerebral_cortex_47606812/">
          <h4 class="font-weight-bold">The genetic architecture of the human cerebral cortex, bioRxiv, 2018-09-03</h4>
          <p>The cerebral cortex underlies our complex cognitive capabilities, yet we know little about the specific genetic loci influencing human cortical structure. To identify genetic variants, including structural variants, impacting cortical structure, we conducted a genome-wide association meta-analysis of brain MRI data from 51,662 individuals. We analysed the surface area and average thickness of the whole cortex and 34 regions with known functional specialisations. We identified 255 nominally significant loci (P ≤ 5 × 10−8); 199 survived multiple testing correction (P ≤ 8.3 × 10−10; 187 surface area; 12 thickness). We found significant enrichment for loci influencing total surface area within regulatory elements active during prenatal cortical development, supporting the radial unit hypothesis. Loci impacting regional surface area cluster near genes in Wnt signalling pathways, known to influence progenitor expansion and areal identity. Variation in cortical structure is genetically correlated with cognitive function, Parkinson’s disease, insomnia, depression and ADHD.One Sentence SummaryCommon genetic variation is associated with inter-individual variation in the structure of the human cortex, both globally and within specific regions, and is shared with genetic risk factors for some neuropsychiatric disorders.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/complex_cell-state_changes_revealed_by_single_cell_rna_sequencing_of_76149_microglia_throughout_the_mouse_lifespan_and_in_the_injured_brain_47436475/">
          <h4 class="font-weight-bold">Complex cell-state changes revealed by single cell RNA sequencing of 76,149 microglia throughout the mouse lifespan and in the injured brain, bioRxiv, 2018-08-31</h4>
          <p>Microglia, the resident immune cells of the brain, rapidly change states in response to their environment, but we lack molecular and functional signatures of different microglial populations. In this study, we analyzed the RNA expression patterns of more than 76,000 individual microglia during development, old age and after brain injury. Analysis uncovered at least nine transcriptionally distinct microglial states, which expressed unique sets of genes and were localized in the brain using specific markers. The greatest microglial heterogeneity was found at young ages; however, several states - including chemokine-enriched inflammatory microglia - persisted throughout the lifespan or increased in the aged brain. Multiple reactive microglial subtypes were also found following demyelinating injury in mice, at least one of which was also found in human MS lesions. These unique microglia signatures can be used to better understand microglia function and to identify and manipulate specific subpopulations in health and disease.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/increased_frequency_of_travel_in_the_presence_of_cross-immunity_may_act_to_decrease_the_chance_of_a_global_pandemic_47444563/">
          <h4 class="font-weight-bold">Increased frequency of travel in the presence of cross-immunity may act to decrease the chance of a global pandemic, bioRxiv, 2018-08-31</h4>
          <p>The high frequency of modern travel has led to concerns about a devastating pandemic since a lethal pathogen strain could spread worldwide quickly. Many historical pandemics have arisen following pathogen evolution to a more virulent form. However, some pathogen strains invoke immune responses that provide partial cross-immunity against infection with related strains. Here, we consider a mathematical model of successive outbreaks of two strains a low virulence strain outbreak followed by a high virulence strain outbreak. Under these circumstances, we investigate the impacts of varying travel rates and cross-immunity on the probability that a major epidemic of the high virulence strain occurs, and the size of that outbreak. Frequent travel between subpopulations can lead to widespread immunity to the high virulence strain, driven by exposure to the low virulence strain. As a result, major epidemics of the high virulence strain are less likely, and can potentially be smaller, with more connected subpopulations. Cross-immunity may be a factor contributing to the absence of a global pandemic as severe as the 1918 influenza pandemic in the century since.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/epidemiology">epidemiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/re-evaluating_one-step_generation_of_mice_carrying_conditional_alleles_by_crispr-cas9-mediated_genome_editing_technology_47288281/">
          <h4 class="font-weight-bold">Re-Evaluating One-step Generation of Mice Carrying Conditional Alleles by CRISPR-Cas9-Mediated Genome Editing Technology, bioRxiv, 2018-08-31</h4>
          <p>AbstractCRISPR-Cas9 gene editing technology has considerably facilitated the generation of mouse knockout alleles, relieving many of the cumbersome and time-consuming steps of traditional mouse embryonic stem cell technology. However, the generation of conditional knockout alleles remains an important challenge. An earlier study reported up to 16% efficiency in generating conditional knockout alleles in mice using 2 single guide RNAs (sgRNA) and 2 single-stranded oligonucleotides (ssODN) (2sgRNA-2ssODN). We re-evaluated this method from a large data set generated from a consortium consisting of 17 transgenic core facilities or laboratories or programs across the world. The dataset constituted 17,887 microinjected or electroporated zygotes and 1,718 live born mice, of which only 15 (0.87%) mice harbored 2 correct LoxP insertions in cis configuration indicating a very low efficiency of the method. To determine the factors required to successfully generate conditional alleles using the 2sgRNA-2ssODN approach, we performed a generalized linear regression model. We show that factors such as the concentration of the sgRNA, Cas9 protein or the distance between the placement of LoxP insertions were not predictive for the success of this technique. The major predictor affecting the method’s success was the probability of simultaneously inserting intact proximal and distal LoxP sequences, without the loss of the DNA segment between the two sgRNA cleavage sites. Our analysis of a large data set indicates that the 2sgRNA–2ssODN method generates a large number of undesired alleles (&amp;gt;99%), and a very small number of desired alleles (&amp;lt;1%) requiring, on average 1,192 zygotes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/2018/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/2018/page/49/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2018/page/46/" class="page-link">46</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2018/page/47/" class="page-link">47</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2018/page/48/" class="page-link">48</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2018/page/49/" class="page-link">49</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/2018/page/50/" class="page-link">50</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2018/page/51/" class="page-link">51</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2018/page/52/" class="page-link">52</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2018/page/53/" class="page-link">53</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2018/page/54/" class="page-link">54</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/2018/page/51/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/2018/page/84/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
