<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/galactose-modified_duocarmycin_prodrugs_as_senolytics_65413535/">
          <h4 class="font-weight-bold">Galactose-modified duocarmycin prodrugs as senolytics, bioRxiv, 2019-08-25</h4>
          <p>SUMMARYSenescence is a stable growth arrest that impairs the replication of damaged, old or preneoplastic cells, therefore contributing to tissue homeostasis. Senescent cells accumulate during ageing and are associated with diseases, such as cancer, fibrosis and many age-related pathologies. Recent evidence suggests that the selective elimination of senescent cells can be effective on the treatment of many of these senescence-associated diseases. A universal characteristic of senescent cells is that they display elevated activity of the lysosomal β-galactosidase this has been exploited as a marker for senescence (senescence-associated β-galactosidase activity). Consequently, we hypothesised that galactose-modified cytotoxic prodrugs will be preferentially processed by senescent cells, resulting in their selective killing. Here, we show that different galactose-modified duocarmycin (GMD) derivatives preferentially kill senescent cells. GMD prodrugs induce selective apoptosis of senescent cells in a lysosomal β-galactosidase (GLB1)-dependent manner. GMD prodrugs can eliminate a broad range of senescent cells in culture, and treatment with a GMD prodrug enhances the elimination of bystander senescent cells that accumulate upon whole body irradiation or doxorubicin treatment of mice. Moreover, taking advantage of a mouse model of human adamantinomatous craniopharyngioma (ACP), we show that treatment with a GMD pro-drug result selectively reduced the number of β-catenin-positive preneoplastic senescent cells, what could have therapeutic implications. In summary, the above results show that galactose-modified duocarmycin prodrugs behave as senolytics, suggesting that they could be used to treat a wide range of senescence-related pathologies.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/global_adaptation_confounds_the_search_for_local_adaptation_65264544/">
          <h4 class="font-weight-bold">Global adaptation confounds the search for local adaptation, bioRxiv, 2019-08-21</h4>
          <p>AbstractSpatially varying selection promotes variance in allele frequencies, increasing genetic differentiation between the demes of a metapopulation. For that reason, outliers in the genome wide distribution of summary statistics measuring genetic differentiation, such as FST, are often interpreted as evidence for alleles which contribute to local adaptation. However, in spatially structured populations, the spread of beneficial mutations with spatially uniform effects can also induce transient genetic differentiation and numerous theoretical studies have suggested that species-wide, or global, adaptation makes a substantial contribution to molecular evolution. In this study, we ask whether such global adaptation affects the genome-wide distribution of FST and generates statistical outliers which could be mistaken for local adaptation. Using forward-in-time population genetic simulations assuming parameters for the rate and strength of beneficial mutations similar to those that have been estimated for natural populations, we show the spread of globally beneficial in parapatric populations can readily generate FST outliers, which may be misinterpreted as evidence for local adaptation. The spread of beneficial mutations causes selective sweeps at flanking sites, so the effects of global versus local adaptation may be distinguished by examining patterns of nucleotide diversity along with FST. Our study suggests that global adaptation should be considered in the interpretation of genome-scan results and the design of future studies aimed at understanding the genetic basis of local adaptation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/prevalence_estimate_of_blood_doping_in_elite_track_and_field_at_the_introduction_of_the_athlete_biological_passport_65166375/">
          <h4 class="font-weight-bold">Prevalence estimate of blood doping in elite track and field at the introduction of the Athlete Biological Passport, bioRxiv, 2019-08-19</h4>
          <p>AbstractIn elite sport, the Athlete Biological Passport (ABP) was invented to tackle cheaters by monitoring closely changes in biological parameters, flagging atypical variations. The haematological module of the ABP was indeed adopted in 2011 by the International Association of Athletics Federations (IAAF). This study estimates the prevalence of blood doping based on haematological parameters in a large cohort of track &amp;amp; field athletes measured at two international major events (2011 &amp;amp; 2013 IAAF World Championships) with a hypothesized decrease in prevalence due to the ABP introduction.A total of 3683 blood samples were collected and analysed from all participating athletes originating from 209 countries. The estimate of doping prevalence was obtained by using a Bayesian network with seven variables, as well as “doping” as a variable mimicking doping with low-doses of recombinant human erythropoietin (rhEPO), to generate reference cumulative distribution functions (CDFs) for the Abnormal Blood Profile Score (ABPS) from the ABP.Our results from robust haematological parameters indicate an estimation of an overall blood doping prevalence of 18% in average in endurance athletes (95% Confidence Interval (C.I.) 14-22%). A higher prevalence was observed in female athletes (22%, C.I. 16-28%) than in male athletes (15%, C.I. 9-20%). In conclusion, this study presents the first comparison of blood doping prevalence in elite athletes based on biological measurements from major international events that may help scientists and experts to use the ABP in a more efficient and deterrent way.What are the new findings ?&lt;jatslist list-type=bullet&gt;&lt;jatslist-item&gt;This study presents the first comparison of blood doping prevalence in elite track &amp;amp; field athletes based on biological measurements from major international events&lt;jatslist-item&gt;&lt;jatslist-item&gt;Our results from robust haematological parameters indicate an estimation of an overall blood doping prevalence of 18% in average in endurance athletes.&lt;jatslist-item&gt;&lt;jatslist-item&gt;The confidence intervals for blood doping prevalence range from 9-28% with wide discrepancies between certain countries.&lt;jatslist-item&gt;How might it impact on clinical practice in the near future&lt;jatslist list-type=bullet&gt;&lt;jatslist-item&gt;The further development of the Athlete Biological Passport with a careful monitoring of biological parameters still represents the most consistent approach to thwart athletes using undetectable prohibited substances or methods.&lt;jatslist-item&gt;&lt;jatslist-item&gt;This study describes a method to define blood doping prevalence with the analysis of robust haematological parameters&lt;jatslist-item&gt;&lt;jatslist-item&gt;Estimates of doping prevalence in subpopulations of athletes may represent a valuable tool for the antidoping authorities to perform a risk assessment in their sport.&lt;jatslist-item&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/physiology">physiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/ancestral_reconstruction_of_sunflower_karyotypes_reveals_non-random_chromosomal_evolution_65028683/">
          <h4 class="font-weight-bold">Ancestral reconstruction of sunflower karyotypes reveals non-random chromosomal evolution, bioRxiv, 2019-08-16</h4>
          <p>AbstractMapping the chromosomal rearrangements between species can inform our understanding of genome evolution, reproductive isolation, and speciation. Here we present a novel algorithm for identifying regions of synteny in pairs of genetic maps, which is implemented in the accompanying R package, syntR. The syntR algorithm performs as well as previous ad-hoc methods while being systematic, repeatable, and is applicable to mapping chromosomal rearrangements in any group of species. In addition, we present a systematic survey of chromosomal rearrangements in the annual sunflowers, which is a group known for extreme karyotypic diversity. We build high-density genetic maps for two subspecies of the prairie sunflower, Helianthus petiolaris ssp. petiolaris and H. petiolaris ssp. fallax.Using syntR, and we identify blocks of synteny between these two subspecies and previously published high-density genetic maps. We reconstruct ancestral karyotypes for annual sunflowers using those synteny blocks and conservatively estimate that there have been 7.9 chromosomal rearrangements per million years – a high rate of chromosomal evolution. Although the rate of inversion is even higher than the rate of translocation in this group, we further find that every extant karyotype is distinguished by between 1 and 3 translocations involving only 8 of the 17 chromosomes. This non-random exchange suggests that specific chromosomes are prone to translocation and may thus contribute disproportionately to widespread hybrid sterility in sunflowers. These data deepen our understanding of chromosome evolution and confirm that Helianthus has an exceptional rate of chromosomal rearrangement that may facilitate similarly rapid diversification.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/plasmodium_vivaxmalaria_viewed_through_the_lens_of_an_eradicated_european_strain_65028992/">
          <h4 class="font-weight-bold">Plasmodium vivaxMalaria viewed through the lens of an eradicated European strain, bioRxiv, 2019-08-16</h4>
          <p>AbstractThe protozoanPlasmodium vivaxis responsible for 42% of all cases of malaria outside Africa. The parasite is currently largely restricted to tropical and subtropical latitudes in Asia, Oceania and the Americas. Though, it was historically present in most of Europe before being finally eradicated during the second half of the 20th century. The lack of genomic information on the extinct European lineage has prevented a clear understanding of historical population structuring and past migrations ofP. vivax. We used medical microscope slides prepared in 1944 from malaria-affected patients from the Ebro Delta in Spain, one of the last footholds of malaria in Europe, to generate a genome of a EuropeanP. vivaxstrain. Population genetics and phylogenetic analyses placed this strain basal to a cluster including samples from the Americas. This genome allowed us to calibrate a genomic mutation rate forP. vivax, and to estimate the mean age of the last common ancestor between European and American strains to the 15th century. This date points to an introduction of the parasite during the European colonisation of the Americas. In addition, we found that some known variants for resistance to anti-malarial drugs, including Chloroquine and Sulfadoxine, were already present in this European strain, predating their use. Our results shed light on the evolution of an important human pathogen and illustrate the value of antique medical collections as a resource for retrieving genomic information on pathogens from the past.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/modulation_of_sensory_behavior_and_food_choice_by_an_enteric_bacteria-produced_neurotransmitter_65002334/">
          <h4 class="font-weight-bold">Modulation of sensory behavior and food choice by an enteric bacteria-produced neurotransmitter, bioRxiv, 2019-08-15</h4>
          <p>AbstractAnimals coexist in commensal, pathogenic or mutualistic relationships with complex communities of diverse organisms including microbes1. Some bacteria produce bioactive neurotransmitters which have been proposed to modulate host nervous system activity and behaviors2. However, the mechanistic basis of this microbiota-brain modulation and its physiological relevance is largely unknown. Here we show that in C. elegans, the neuromodulator tyramine (TA) produced by gut-colonizing commensal Providencia bacteria can bypass the requirement for host TA biosynthesis to manipulate a host sensory decision. Bacterially-produced TA is likely converted to octopamine (OA) by the host tyramine beta-hydroxylase enzyme. OA, in turn, targets the OCTR-1 receptor on the ASHASI sensory neurons to modulate an aversive olfactory response. We identify genes required for TA biosynthesis in Providencia, and show that these genes are necessary for modulation of host behavior. We further find that C. elegans colonized by Providencia preferentially select these bacteria in food choice assays, and that this selection bias requires bacterially-produced TA. Our results demonstrate that a neurotransmitter produced by gut microbiota mimics the functions of the cognate host molecule to override host control of a sensory decision, thereby promoting fitness of both host and microbe.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/13/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/10/" class="page-link">10</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/11/" class="page-link">11</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/12/" class="page-link">12</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/13/" class="page-link">13</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/14/" class="page-link">14</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/15/" class="page-link">15</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/16/" class="page-link">16</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/17/" class="page-link">17</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/18/" class="page-link">18</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/15/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/123/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
