<!DOCTYPE html>
<html><head>
  
  <title>2019 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/high_genetic_diversity_can_contribute_to_extinction_in_small_populations_62521534/">
          <h4 class="font-weight-bold">High genetic diversity can contribute to extinction in small populations, bioRxiv, 2019-06-22</h4>
          <p>AbstractHuman-driven habitat fragmentation and loss has led to a proliferation of small and isolated plant and animal populations that may be threatened with extinction by genetic factors. The prevailing approach for managing these populations is to maintain high genetic diversity, which is often equated with fitness. Increasingly, this is being done using genetic rescue, where individuals from populations with high genetic diversity are translocated to small populations with high levels of inbreeding. However, the potentially negative consequences of this approach have recently been highlighted by the demise of the gray wolf population on Isle Royale, which only briefly recovered after genetic rescue by a migrant from the large mainland wolf population and then declined to the brink of extinction. Here, we use ecologically-motivated population genetic simulations to show that extinction risk in small populations is often increased by maximizing genetic diversity but is consistently decreased by minimizing deleterious variation. Surprisingly, we find that small populations that are founded or rescued by individuals from large populations with high genetic diversity have an elevated risk of extinction due to the high levels of recessive deleterious variation harbored by large populations. By contrast, we show that genetic rescue or founding from small or moderate-sized populations leads to decreased extinction risk due to greater purging of strongly deleterious variants. Our findings challenge the traditional conservation paradigm that focuses on genetic diversity in assessing extinction risk in favor of a new view that emphasizes minimizing deleterious variation. These insights have immediate implications for managing small and isolated populations in the increasingly fragmented landscape of the Anthropocene.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/high_precision_coding_in_mouse_visual_cortex_62520267/">
          <h4 class="font-weight-bold">High precision coding in mouse visual cortex, bioRxiv, 2019-06-22</h4>
          <p>ABSTRACTSingle neurons in visual cortex provide unreliable measurements of visual features due to their high trial-to-trial variability. It is not known if this “noise” extends its effects over large neural populations to impair the global encoding of sensory stimuli. We recorded simultaneously from ∼20,000 neurons in mouse visual cortex and found that the neural population had discrimination thresholds of 0.3° in an orientation decoding task. These thresholds are ∼100 times smaller than those reported behaviorally in mice. The discrepancy between neural and behavioral discrimination could not be explained by the types of stimuli we used, by behavioral states or by the sequential nature of trial-by-trial perceptual learning tasks. These results imply that the limits of sensory perception in mice are not set by neural noise in sensory cortex, but by the limitations of downstream decoders.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/highly_multiplexed_spatial_mapping_of_microbial_communities_62519865/">
          <h4 class="font-weight-bold">Highly Multiplexed Spatial Mapping of Microbial Communities, bioRxiv, 2019-06-22</h4>
          <p>ABSTRACTMapping the complex biogeography of microbial communities in situ with high taxonomic and spatial resolution poses a major challenge because of the high density and rich diversity of species in environmental microbiomes and the limitations of optical imaging technology. Here, we introduce High Phylogenetic Resolution microbiome mapping by Fluorescence In-Situ Hybridization (HiPR-FISH), a versatile and cost-effective technology that uses binary encoding and spectral imaging and machine learning based decoding to create micron-scale maps of the locations and identities of hundreds of microbial species in complex communities. We demonstrate the ability of 10-bit HiPR-FISH to distinguish 1023 E. coli strains, each fluorescently labeled with a unique binary barcode. HiPR-FISH, in conjunction with custom algorithms for automated probe design and segmentation of single-cells in the native context of tissues, reveals the intricate spatial architectures formed by bacteria in the human oral plaque microbiome and disruption of spatial networks in the mouse gut microbiome in response to antibiotic treatment. HiPR-FISH provides a framework for analyzing the spatial organization of microbial communities in tissues and the environment at single cell resolution.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/super-resolution_imaging_reveals_3d_structure_and_organizing_mechanism_of_accessible_chromatin_62523093/">
          <h4 class="font-weight-bold">Super-resolution Imaging Reveals 3D Structure and Organizing Mechanism of Accessible Chromatin, bioRxiv, 2019-06-22</h4>
          <p>Access to cis-regulatory elements packaged in chromatin is essential for directing gene expression and cell viability. Here, we report a super-resolution imaging strategy, 3D ATAC-PALM, that enables direct visualization of the entire accessible genome. We found that active chromosomal segments are organized into spatially-segregated accessible chromatin domains (ACDs). Rapid depletion of CTCF or Cohesin (RAD21 subunit) induced enhanced ACD clustering, reduced physical separation between intrachromosomal ACDs, and differentially regulated ACD compaction. Experimental perturbations and polymer modeling suggest that dynamic protein-protein and protein-DNA interactions within ACDs couple with loop extrusion to organize ACD topology. Dysorganization of ACDs upon CTCF or Cohesin loss alters transcription factor binding and target search dynamics in living cells. These results uncover fundamental mechanisms underpinning the formation of 3D genome architecture and its pivotal function in transcriptional regulation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_creatures_of_the_human_cortical_somatosensory_system_62520606/">
          <h4 class="font-weight-bold">The “creatures” of the human cortical somatosensory system, bioRxiv, 2019-06-22</h4>
          <p>AbstractPenfield’s description of the “homunculus”, a “grotesque creature” with large lips and hands and small trunk and legs depicting the representation of body-parts within the primary somatosensory cortex (S1), is one of the most prominent contributions to the neurosciences. Since then, numerous studies have identified additional body-parts representations outside of S1. Nevertheless, it has been implicitly assumed that S1’s homunculus is representative of the entire somatosensory cortex. Therefore, the distribution of body-parts representations in other brain regions, the property that gave Penfield’s homunculus its famous “grotesque” appearance, has been overlooked. We used whole-body somatosensory stimulation, functional MRI and a new cortical parcellation to quantify the organization of the cortical somatosensory representation. Our analysis showed first, an extensive somatosensory response over the cortex; and second, that the proportional representation of body-parts differs substantially between major neuroanatomical regions and from S1, with, for instance, much larger trunk representation at higher brain regions, potentially in relation to the regions’ functional specialization. These results extend Penfield’s initial findings to the higher level of somatosensory processing and suggest a major role for somatosensation in human cognition.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_genomic_impact_of_european_colonization_of_the_americas_62499079/">
          <h4 class="font-weight-bold">The genomic impact of European colonization of the Americas, bioRxiv, 2019-06-21</h4>
          <p>AbstractThe human genetic diversity of the Americas has been shaped by several events of gene flow that have continued since the Colonial Era and the Atlantic slave trade. Moreover, multiple waves of migration followed by local admixture occurred in the last two centuries, the impact of which has been largely unexplored.Here we compiled a genome-wide dataset of ∼12,000 individuals from twelve American countries and ∼6,000 individuals from worldwide populations and applied haplotype-based methods to investigate how historical movements from outside the New World affected i) the genetic structure, ii) the admixture profile, iii) the demographic history and iv) sex-biased gene-flow dynamics, of the Americas.We revealed a high degree of complexity underlying the genetic contribution of European and African populations in North and South America, from both geographic and temporal perspectives, identifying previously unreported sources related to Italy, the Middle East and to specific regions of Africa.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/2019/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/2019/page/4/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/4/" class="page-link">4</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/2019/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/9/" class="page-link">9</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/2019/page/6/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/2019/page/87/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
