<!DOCTYPE html>
<html><head>
  
  <title>Developmental Biology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_single-cell_transcriptome_atlas_for_zebrafish_development_65164614/">
          <h4 class="font-weight-bold">A Single-Cell Transcriptome Atlas for Zebrafish Development, bioRxiv, 2019-08-19</h4>
          <p>ABSTRACTThe ability to define cell types and how they change during organogenesis is central to our understanding of animal development and human disease. Despite the crucial nature of this knowledge, we have yet to fully characterize all distinct cell types and the gene expression differences that generate cell types during development. To address this knowledge gap, we produced an Atlas using single-cell RNA-sequencing methods to investigate gene expression from the pharyngula to early larval stages in developing zebrafish. Our single-cell transcriptome Atlas encompasses transcriptional profiles from 44,102 cells across four days of development using duplicate experiments that confirmed high reproducibility. We annotated 220 identified clusters and highlighted several strategies for interrogating changes in gene expression associated with the development of zebrafish embryos at single-cell resolution. Furthermore, we highlight the power of this analysis to assign new cell-type or developmental stage-specific expression information to many genes, including those that are currently known only by sequence andor that lack expression information altogether. The resulting Atlas is a resource of biologists to generate hypotheses for genetic (mutant) or functional analysis, to launch an effort to define the diversity of cell-types during zebrafish organogenesis, and to examine the transcriptional profiles that produce each cell type over developmental time.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/nad&#43;_repletion_rescues_female_fertility_during_reproductive_ageing_64487836/">
          <h4 class="font-weight-bold">NAD&#43; repletion rescues female fertility during reproductive ageing, bioRxiv, 2019-08-03</h4>
          <p>AbstractFemale infertility is a common and devastating condition with life-long health, emotional and social consequences. There is currently no pharmacological therapy for preserving oocyte quality during aging, which is the strongest risk factor for infertility. This leads to an age dependent decline in natural conception and IVF success rates (1). Here, we show that this is due in part to declining levels of the metabolic cofactor nicotinamide adenine dinucleotide (NAD&#43;), and that restoring NAD&#43; levels with its metabolic precursor nicotinamide mononucleotide (NMN) rejuvenates oocyte quality and quantity in aged animals, leading to improved fertility. These benefits extend to the developing embryo, where NMN supplementation in embryo culture media following IVF enhances blastocyst formation in older mice. The NAD&#43; dependent deacylase SIRT2 is sufficient, but not essential, to recapitulate the benefits of in vivo NMN treatment, and transgenic overexpression of SIRT2 maintains oocyte spindle assembly, accurate chromosome segregation, decreased oxidative stress and overall fertility with ageing. Pharmacological elevation of NAD&#43; may be an effective, non-invasive strategy for restoring and maintaining female fertility during ageing, and for improving the success of IVF.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/precision_of_tissue_patterning_is_controlled_by_dynamical_properties_of_gene_regulatory_networks_64403267/">
          <h4 class="font-weight-bold">Precision of Tissue Patterning is Controlled by Dynamical Properties of Gene Regulatory Networks, bioRxiv, 2019-08-01</h4>
          <p>AbstractDuring development, gene regulatory networks allocate cell fates by partitioning tissues into spatially organised domains of gene expression. How the sharp boundaries that delineate these gene expression patterns arise, despite the stochasticity associated with gene regulation, is poorly understood. We show, in the vertebrate neural tube, using perturbations of coding and regulatory regions, that the structure of the regulatory network contributes to boundary precision. This is achieved, not by reducing noise in individual genes, but by the configuration of the network modulating the ability of stochastic fluctuations to initiate gene expression changes. We use a computational screen to identify the properties of a network that influence boundary precision, revealing two dynamical mechanisms by which small gene circuits attenuate the effect of noise to increase patterning precision. These results establish design principles of gene regulatory networks that produce precise patterns of gene expression.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/supercentenarians_and_the_oldest-old_are_concentrated_into_regions_with_no_birth_certificates_and_short_lifespans_63629545/">
          <h4 class="font-weight-bold">Supercentenarians and the oldest-old are concentrated into regions with no birth certificates and short lifespans, bioRxiv, 2019-07-16</h4>
          <p>AbstractThe observation of individuals attaining remarkable ages, and their concentration into geographic sub-regions or ‘blue zones’, has generated considerable scientific interest. Proposed drivers of remarkable longevity include high vegetable intake, strong social connections, and genetic markers. Here, we reveal new predictors of remarkable longevity and ‘supercentenarian’ status. In the United States, supercentenarian status is predicted by the absence of vital registration. The state-specific introduction of birth certificates is associated with a 69-82% fall in the number of supercentenarian records. In Italy, which has more uniform vital registration, remarkable longevity is instead predicted by low per capita incomes and a short life expectancy. Finally, the designated ‘blue zones’ of Sardinia, Okinawa, and Ikaria corresponded to regions with low incomes, low literacy, high crime rate and short life expectancy relative to their national average. As such, relative poverty and short lifespan constitute unexpected predictors of centenarian and supercentenarian status, and support a primary role of fraud and error in generating remarkable human age records.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/single-cell_genomic_atlas_of_great_ape_cerebral_organoids_uncovers_human-specific_features_of_brain_development_62783876/">
          <h4 class="font-weight-bold">Single-cell genomic atlas of great ape cerebral organoids uncovers human-specific features of brain development, bioRxiv, 2019-06-28</h4>
          <p>ABSTRACTThe human brain has changed dramatically since humans diverged from our closest living relatives, chimpanzees and the other great apes1–5. However, the genetic and developmental programs underlying this divergence are not fully understood6–8. Here, we have analyzed stem cell-derived cerebral organoids using single-cell transcriptomics (scRNA-seq) and accessible chromatin profiling (scATAC-seq) to explore gene regulatory changes that are specific to humans. We first analyze cell composition and reconstruct differentiation trajectories over the entire course of human cerebral organoid development from pluripotency, through neuroectoderm and neuroepithelial stages, followed by divergence into neuronal fates within the dorsal and ventral forebrain, midbrain and hindbrain regions. We find that brain region composition varies in organoids from different iPSC lines, yet regional gene expression patterns are largely reproducible across individuals. We then analyze chimpanzee and macaque cerebral organoids and find that human neuronal development proceeds at a delayed pace relative to the other two primates. Through pseudotemporal alignment of differentiation paths, we identify human-specific gene expression resolved to distinct cell states along progenitor to neuron lineages in the cortex. We find that chromatin accessibility is dynamic during cortex development, and identify instances of accessibility divergence between human and chimpanzee that correlate with human-specific gene expression and genetic change. Finally, we map human-specific expression in adult prefrontal cortex using single-nucleus RNA-seq and find developmental differences that persist into adulthood, as well as cell state-specific changes that occur exclusively in the adult brain. Our data provide a temporal cell atlas of great ape forebrain development, and illuminate dynamic gene regulatory features that are unique to humans.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/species-specific_oscillation_periods_of_human_and_mouse_segmentation_clocks_are_due_to_cell_autonomous_differences_in_biochemical_reaction_parameters_61120294/">
          <h4 class="font-weight-bold">Species-specific oscillation periods of human and mouse segmentation clocks are due to cell autonomous differences in biochemical reaction parameters, bioRxiv, 2019-05-26</h4>
          <p>AbstractWhile the mechanisms of embryonic development are similar between mouse and human, the tempo is in general slower in human. The cause of interspecies differences in developmental time remains a mystery partly due to lack of an appropriate model system1. Since murine and human embryos differ in their sizes, geometries, and nutrients, we use in vitro differentiation of pluripotent stem cells (PSCs) to compare the same type of cells between the species in similar culture conditions. As an example of well-defined developmental time, we focus on the segmentation clock, oscillatory gene expression that regulates the timing of sequential formation of body segments2–4. In this way we recapitulate the murine and human segmentation clocks in vitro, showing that the species-specific oscillation periods are derived from cell autonomous differences in the speeds of biochemical reactions. Presomitic mesoderm (PSM)-like cells induced from murine and human PSCs displayed the oscillatory expression of HES7, the core gene of the segmentation clock5,6, with oscillation periods of 2-3 hours (mouse PSM) and 5-6 hours (human PSM). Swapping HES7 loci between murine and human genomes did not change the oscillation periods dramatically, denying the possibility that interspecies differences in the sequences of HES7 loci might be the cause of the observed period difference. Instead, we found that the biochemical reactions that determine the oscillation period, such as the degradation of HES7 and delays in its expression, are slower in human PSM compared with those in mouse PSM. With the measured biochemical parameters, our mathematical model successfully accounted for the 2-3-fold period difference between mouse and human. We further demonstrate that the concept of slower biochemical reactions in human cells is generalizable to several other genes, as well as to another cell type. These results collectively indicate that differences in the speeds of biochemical reactions between murine and human cells give rise to the interspecies period difference of the segmentation clock and may contribute to other interspecies differences in developmental time.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/developmental-biology/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/developmental-biology/" class="page-link">1</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/developmental-biology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/developmental-biology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/developmental-biology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/developmental-biology/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/developmental-biology/page/6/" class="page-link">6</a></li>
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/developmental-biology/page/3/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/developmental-biology/page/6/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
