<!DOCTYPE html>
<html><head>
  
  <title>2018 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.68.3" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.95d2483cd3c5d2bf47c1f36307a456787d60ad958c5efe44db9afc47012b1b05b8e1f64f6ac291650439322a6c38e86eb7cd6254255516d857c23f299fb0e666.js" integrity="sha512-ldJIPNPF0r9HwfNjB6RWeH1grZWMXv5E25r8RwErGwW44fZPasKRZQQ5MipsOOhut81iVCVVFthXwj8pn7DmZg=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/transposon_accumulation_lines_uncover_histone_h2a.z-driven_integration_bias_towards_environmentally_responsive_genes_49937639/">
          <h4 class="font-weight-bold">Transposon accumulation lines uncover histone H2A.Z-driven integration bias towards environmentally responsive genes, bioRxiv, 2018-10-19</h4>
          <p>Inherited transposition events are important drivers of genome evolution but because transposable element (TE) mobilization is usually rare, its impact on the creation of genetic variation remains poorly characterized. Here, we used a population of A. thaliana epigenetic recombinant inbred lines (epiRILs) to characterize &amp;gt;8000 de novo insertions produced by several TEs families also active in nature. Integration was strongly biased towards genes, with evident deleterious effects. Biases were TE family-specific and associated with distinct chromatin features. Notably, we demonstrate that the histone variant H2A.Z guides the preferential integration of Ty1Copia LTR-retrotransposons within environmentally responsive genes and that this guiding function is evolutionary conserved. Finally, we uncover an important role for epigenetic silencing in exacerbating or alleviating the effects of TE insertions on target genes. These findings establish chromatin as a major determinant of the spectrum and functional impact of TE-generated mutations, with important implications for adaptation and evolution.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/unraveling_the_polygenic_architecture_of_complex_traits_using_blood_eqtl_meta-analysis_49939673/">
          <h4 class="font-weight-bold">Unraveling the polygenic architecture of complex traits using blood eQTL meta-analysis, bioRxiv, 2018-10-19</h4>
          <p>While many disease-associated variants have been identified through genome-wide association studies, their downstream molecular consequences remain unclear. To identify these effects, we performed cis- and trans-expression quantitative trait locus (eQTL) analysis in blood from 31,684 individuals through the eQTLGen Consortium. We observed that cis-eQTLs can be detected for 88% of the studied genes, but that they have a different genetic architecture compared to disease-associated variants, limiting our ability to use cis-eQTLs to pinpoint causal genes within susceptibility loci. In contrast, trans-eQTLs (detected for 37% of 10,317 studied trait-associated variants) were more informative. Multiple unlinked variants, associated to the same complex trait, often converged on trans-genes that are known to play central roles in disease etiology. We observed the same when ascertaining the effect of polygenic scores calculated for 1,263 genome-wide association study (GWAS) traits. Expression levels of 13% of the studied genes correlated with polygenic scores, and many resulting genes are known to drive these traits.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/unraveling_the_polygenic_architecture_of_complex_traits_using_blood_eqtl_metaanalysis_49939673/">
          <h4 class="font-weight-bold">Unraveling the polygenic architecture of complex traits using blood eQTL metaanalysis, bioRxiv, 2018-10-19</h4>
          <p>SummaryWhile many disease-associated variants have been identified through genome-wide association studies, their downstream molecular consequences remain unclear.To identify these effects, we performed cis- and trans-expression quantitative trait locus (eQTL) analysis in blood from 31,684 individuals through the eQTLGen Consortium.We observed that cis-eQTLs can be detected for 88% of the studied genes, but that they have a different genetic architecture compared to disease-associated variants, limiting our ability to use cis-eQTLs to pinpoint causal genes within susceptibility loci.In contrast, trans-eQTLs (detected for 37% of 10,317 studied trait-associated variants) were more informative. Multiple unlinked variants, associated to the same complex trait, often converged on trans-genes that are known to play central roles in disease etiology.We observed the same when ascertaining the effect of polygenic scores calculated for 1,263 genome-wide association study (GWAS) traits. Expression levels of 13% of the studied genes correlated with polygenic scores, and many resulting genes are known to drive these traits.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_stable_long-term_cortical_signature_underlying_consistent_behavior_49899064/">
          <h4 class="font-weight-bold">A stable, long-term cortical signature underlying consistent behavior, bioRxiv, 2018-10-18</h4>
          <p>AbstractAnimals readily execute learned motor behaviors in a consistent manner over long periods of time, yet similarly stable neural correlates remained elusive up to now. How does the cortex achieve this stable control? Using the sensorimotor system as a model of cortical processing, we investigated the hypothesis that the dynamics of neural latent activity, which capture the dominant co-variation patterns within the neural population, are preserved across time. We recorded from populations of neurons in premotor, primary motor, and somatosensory cortices for up to two years as monkeys performed a reaching task. Intriguingly, despite steady turnover in the recorded neurons, the low-dimensional latent dynamics remained stable. Such stability allowed reliable decoding of behavioral features for the entire timespan, while fixed decoders based on the recorded neural activity degraded substantially. We posit that latent cortical dynamics within the manifold are the fundamental and stable building blocks underlying consistent behavioral execution.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/diversification_and_collapse_of_a_telomere_elongation_mechanism_49862531/">
          <h4 class="font-weight-bold">Diversification and collapse of a telomere elongation mechanism, bioRxiv, 2018-10-18</h4>
          <p>AbstractIn virtually all eukaryotes, telomerase counteracts chromosome erosion by adding repetitive sequence to terminal ends. Drosophila melanogaster instead relies on specialized retrotransposons that insert preferentially at telomeres. This exchange of goods between host and mobile element—wherein the mobile element provides an essential genome service and the host provides a hospitable niche for mobile element propagation—has been called a ‘genomic symbiosis’. However, these telomere-specialized, ‘jockey’ family elements may actually evolve to selfishly over-replicate in the genomes that they ostensibly serve. Under this intra-genomic conflict model, we expect rapid diversification of telomere-specialized retrotransposon lineages and possibly, the breakdown of this tenuous relationship. Here we report data consistent with both predictions. Searching the raw reads of the 15-million-year-old ‘melanogaster species group’, we generated de novo jockey retrotransposon consensus sequences and used phylogenetic tree-building to delineate four distinct telomere-associated lineages. Recurrent gains, losses, and replacements account for this striking retrotransposon lineage diversity. Moreover, an ancestrally telomere-specialized element has ‘escaped,’ residing now throughout the genome of D. rhopaloa. In D. biarmipes, telomere-specialized elements have disappeared completely. De novo assembly of long-reads and cytogenetics confirmed this species-specific collapse of retrotransposon-dependent telomere elongation. Instead, telomere-restricted satellite DNA and DNA transposon fragments occupy its terminal ends. We infer that D. biarmipes relies instead on a recombination-based mechanism conserved from yeast to flies to humans. Combined with previous reports of adaptive evolution at host proteins that regulate telomere length, telomere-associated retrotransposon diversification and disappearance offer compelling evidence that intra-genomic conflict shapes Drosophila telomere evolution.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/mutation_detection_in_thousands_of_acute_myeloid_leukemia_cells_using_single_cell_rna-sequencing_49857784/">
          <h4 class="font-weight-bold">Mutation detection in thousands of acute myeloid leukemia cells using single cell RNA-sequencing, bioRxiv, 2018-10-18</h4>
          <p>AbstractVirtually all tumors are genetically heterogeneous, containing subclonal populations of cells that are defined by distinct mutations1. Subclones can have unique phenotypes that influence disease progression2, but these phenotypes are difficult to characterize subclones usually cannot be physically purified, and bulk gene expression measurements obscure interclonal differences. Single-cell RNA-sequencing has revealed transcriptional heterogeneity within a variety of tumor types, but it is unclear how this expression heterogeneity relates to subclonal genetic events – for example, whether particular expression clusters correspond to mutationally defined subclones3,4,5,6-9. To address this question, we developed an approach that integrates enhanced whole genome sequencing (eWGS) with the 10x Genomics Chromium Single Cell 5’ Gene Expression workflow (scRNA-seq) to directly link expressed mutations with transcriptional profiles at single cell resolution. Using bone marrow samples from five cases of primary human Acute Myeloid Leukemia (AML), we generated WGS and scRNA-seq data for each case. Duplicate single cell libraries representing a median of 20,474 cells per case were generated from the bone marrow of each patient. Although the libraries were 5’ biased, we detected expressed mutations in cDNAs at distances up to 10 kbp from the 5’ ends of well-expressed genes, allowing us to identify hundreds to thousands of cells with AML-specific somatic mutations in every case. This data made it possible to distinguish AML cells (including normal-karyotype AML cells) from surrounding normal cells, to study tumor differentiation and intratumoral expression heterogeneity, to identify expression signatures associated with subclonal mutations, and to find cell surface markers that could be used to purify subclones for further study. The data also revealed transcriptional heterogeneity that occurred independently of subclonal mutations, suggesting that additional factors drive epigenetic heterogeneity. This integrative approach for connecting genotype to phenotype in AML cells is broadly applicable for analysis of any sample that is phenotypically and genetically heterogeneous.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/cancer-biology">cancer-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/26/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/23/" class="page-link">23</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/24/" class="page-link">24</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/25/" class="page-link">25</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/26/" class="page-link">26</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/2018/page/27/" class="page-link">27</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/28/" class="page-link">28</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/29/" class="page-link">29</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/30/" class="page-link">30</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/31/" class="page-link">31</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/28/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/84/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
