<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/disorganization_of_the_histone_core_promotes_organization_of_heterochromatin_into_phase-separated_droplets_51385915/">
          <h4 class="font-weight-bold">Disorganization of the histone core promotes organization of heterochromatin into phase-separated droplets, bioRxiv, 2018-11-19</h4>
          <p>AbstractThe heterochromatin protein HP1 is proposed to enable chromatin compaction via liquid droplet formation. Yet, a connection between phase separation and chromatin compaction has not been experimentally demonstrated. More fundamentally, how HP1 action at the level of a single nucleosome drives chromatin compaction remains poorly understood. Here we directly demonstrate that the S. pombe HP1 protein, Swi6, compacts arrays of multiple nucleosomes into phase-separated droplets. Using hydrogen-deuterium exchange, NMR, and mass-spectrometry, we further find that Swi6 substantially increases the accessibility and dynamics of buried histone residues within a mononucleosome. Restraining these dynamics via site-specific disulfide bonds impairs the compaction of nucleosome arrays into phase-separated droplets. Our results indicate that chromatin compaction and phase separation can be highly coupled processes. Further, we find that such coupling is promoted by a counter-intuitive function of Swi6, namely disorganization of the octamer core. Phase separation is canonically mediated by weak and dynamic multivalent interactions. We propose that dynamic exposure of buried histone residues increases opportunities for multivalent interactions between nucleosomes, thereby coupling chromatin compaction to phase separation. We anticipate that this new model for chromatin organization may more generally explain the formation of highly compacted chromatin assemblies beyond heterochromatin.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/variability_of_bacterial_behavior_in_the_mammalian_gut_captured_using_a_growth-linked_single-cell_synthetic_gene_oscillator_51342001/">
          <h4 class="font-weight-bold">Variability of bacterial behavior in the mammalian gut captured using a growth-linked single-cell synthetic gene oscillator, bioRxiv, 2018-11-17</h4>
          <p>AbstractThe dynamics of the bacterial population that comprises the gut microbiota plays key roles in overall mammalian health. However, a detailed understanding of bacterial growth within the gut is limited by the inherent complexity and inaccessibility of the gut environment. Here, we deploy an improved synthetic genetic oscillator to investigate dynamics of bacterial colonization and growth in the mammalian gut under both healthy and disease conditions. The synthetic oscillator, when introduced into both Escherichia coli and Salmonella Typhimurium maintains regular oscillations with a constant period in generations across growth conditions. We determine the phase of oscillation from individual bacteria using image analysis of resultant colonies and thereby infer the number of cell divisions elapsed. In doing so, we demonstrate robust functionality and controllability of the oscillator circuit’s activity during bacterial growth in vitro, in a simulated murine gut microfluidic environment, and in vivo within the mouse gut. We determine different dynamics of bacterial colonization and growth in the gut under normal and inflammatory conditions. Our results show that a precise genetic oscillator can function in a complex environment and reveal single cell behavior under diverse conditions where disease may create otherwise impossible-to-quantify variability in growth across the population.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/synthetic-biology">synthetic-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/single_particle_cryo-em_reconstruction_of_52_kda_streptavidin_at_3.2_angstrom_resolution_51142456/">
          <h4 class="font-weight-bold">Single particle cryo-EM reconstruction of 52 kDa streptavidin at 3.2 Angstrom resolution, bioRxiv, 2018-11-13</h4>
          <p>AbstractThe fast development of single particle cryo-EM has made it more feasible to obtain the 3D structure of well-behaved macromolecules with molecular weight higher than 300 kDa at ~3 Å resolution. It remains a challenge to obtain high resolution structure of molecules smaller than 100 kDa using single particle cryo-EM, mainly due to the low contrast of the molecules embedded in vitreous ice. In this work, we applied the Cs-corrector-VPP coupled cryo-EM to study 52 kDa streptavidin (SA) protein supported on a thin layer of graphene film and embedded in vitreous ice. We were able to solve both the apo-SA and biotin-bound SA at near-atomic resolution using single particle cryo-EM. We demonstrated that the method is capable to determine the structure of molecule as small as 39 kDa and potentially even smaller molecules. Furthermore, we found that using the graphene film to avoid the adsorption to the air-water interface is critical to maintain the protein’s high-resolution structural information.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/target-specific_co-transmission_of_acetylcholine_and_gaba_from_a_subset_of_cortical_vip&#43;_interneurons_51111647/">
          <h4 class="font-weight-bold">Target-specific co-transmission of acetylcholine and GABA from a subset of cortical VIP&#43; interneurons, bioRxiv, 2018-11-13</h4>
          <p>AbstractThe modulation of cortex by acetylcholine (ACh) is typically thought to originate from long-range projections arising in the basal forebrain. However, a subset of VIP interneurons express ChAT, the synthetic enzyme for ACh, and are a potential local source of cortical ACh. Which neurotransmitters these VIPChAT interneurons (VCINs) release is unclear, and which post-synaptic cell types these transmitters target is not known. Using quantitative molecular analysis of VCIN pre-synaptic terminals, we show expression of the molecular machinery to release both ACh and GABA, with ACh release restricted to a subset of boutons. A systematic survey of potential post-synaptic cell types shows that VCINs release GABA primarily onto other inhibitory interneuron subtypes, while ACh neurotransmission is notably sparse, with most ACh release onto layer 1 interneurons and other VCINs. Therefore, VCINs are an alternative source of cortical ACh signaling that supplement GABA-mediated disinhibition with highly targeted excitation through ACh.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/simultaneous_mesoscopic_and_two-photon_imaging_of_neuronal_activity_in_cortical_circuits_51078977/">
          <h4 class="font-weight-bold">Simultaneous mesoscopic and two-photon imaging of neuronal activity in cortical circuits, bioRxiv, 2018-11-12</h4>
          <p>AbstractSpontaneous and sensory-evoked activity propagates across spatial scales in the mammalian cortex but technical challenges have generally precluded establishing conceptual links between the function of local circuits of neurons and brain-wide network dynamics. To solve this problem, we developed a method for simultaneous cellular-resolution two-photon calcium imaging of a local microcircuit and mesoscopic widefield calcium imaging of the entire cortical mantle in awake, behaving mice. Our method employs an orthogonal axis design whereby the mesoscopic objective is oriented downward directly above the brain and the two-photon objective is oriented horizontally, with imaging performed through a glass right angle microprism implanted in the skull. In support of this method, we introduce a suite of analysis tools for relating the activity of individual cells to distal cortical areas, as well as a viral method for robust and widespread gene delivery in the juvenile mouse brain. We use these methods to characterize the diversity of associations of individual, genetically-defined neurons with cortex-wide network motifs.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/a_polyploid_admixed_origin_of_beer_yeasts_derived_from_european_and_asian_wine_populations_51006931/">
          <h4 class="font-weight-bold">A polyploid admixed origin of beer yeasts derived from European and Asian wine populations, bioRxiv, 2018-11-09</h4>
          <p>AbstractStrains of Saccharomyces cerevisiae used to make beer, bread and wine are genetically and phenotypically distinct from wild populations associated with trees. The origins of these domesticated populations are not always clear; human-associated migration and admixture with wild populations have had a strong impact on S. cerevisiae population structure. We examined the population genetic history of beer strains and find that ale strains and the S. cerevisiae portion of allotetraploid lager strains were derived from admixture between populations closely related to European grape wine strains and Asian rice wine strains. Similar to both lager and baking strains, ale strains are polyploid, providing them with a passive means of remaining isolated from other populations and providing us with a living relic of their ancestral hybridization. To reconstruct their polyploid origin we phased the genomes of two ale strains and found ale haplotypes to both be recombinants between European and Asian alleles and to also contain novel alleles derived from extinct or as yet uncharacterized populations. We conclude that modern beer strains are the product of a historical melting pot of fermentation technology.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/0-100-users/page/33/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/30/" class="page-link">30</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/31/" class="page-link">31</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/32/" class="page-link">32</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/33/" class="page-link">33</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/0-100-users/page/34/" class="page-link">34</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/35/" class="page-link">35</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/36/" class="page-link">36</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/37/" class="page-link">37</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/38/" class="page-link">38</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/0-100-users/page/35/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/0-100-users/page/64/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
