<!DOCTYPE html>
<html><head>
  
  <title>2019 | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/determining_sufficient_sequencing_depth_in_rna-seq_differential_expression_studies_60401132/">
          <h4 class="font-weight-bold">Determining sufficient sequencing depth in RNA-Seq differential expression studies, bioRxiv, 2019-05-13</h4>
          <p>AbstractRNA-Seq studies require a sufficient read depth to detect biologically important genes. Sequencing below this threshold will reduce statistical power while sequencing above will provide only marginal improvements in power and incur unnecessary sequencing costs. Although existing methodologies can help assess whether there is sufficient read depth, they are unable to guide how many additional reads should be sequenced to reach this threshold. We provide a new method called superSeq that models the relationship between statistical power and read depth. We apply the superSeq framework to 393 RNA-Seq experiments (1,021 total contrasts) in the Expression Atlas and find the model accurately predicts the increase in statistical power gained by increasing the read depth. Based on our analysis, we find that most published studies (&amp;gt; 70%) are undersequenced, i.e., their statistical power can be improved by increasing the sequencing read depth. In addition, the extent of saturation is highly dependent on statistical methodology only 9.5%, 29.5%, and 26.6% of contrasts are saturated when using DESeq2, edgeR, and limma, respectively. Finally, we also find that there is no clear minimum per-transcript read depth to guarantee saturation for an entire technology. Therefore, our framework not only delineates key differences among methods and their impact on determining saturation, but will also be needed even as technology improves and the read depth of experiments increases. Researchers can thus use superSeq to calculate the read depth to achieve required statistical power while avoiding unnecessary sequencing costs.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/logomaker_beautiful_sequence_logos_in_python_60401379/">
          <h4 class="font-weight-bold">Logomaker Beautiful sequence logos in python, bioRxiv, 2019-05-13</h4>
          <p>AbstractSequence logos are visually compelling ways of illustrating the biological properties of DNA, RNA, and protein sequences, yet it is currently difficult to generate such logos within the Python programming environment. Here we introduce Logomaker, a Python API for creating publication-quality sequence logos. Logomaker can produce both standard and highly customized logos from any matrix-like array of numbers. Logos are rendered as vector graphics that are easy to stylize using standard matplotlib functions. Methods for creating logos from multiple-sequence alignments are also included.Availability and ImplementationLogomaker can be installed using the pip package manager and is compatible with both Python 2.7 and Python 3.6. Source code is available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpgithub.comjbkinneylogomaker&gt;httpgithub.comjbkinneylogomaker&lt;jatsext-link&gt;.Supplemental InformationDocumentation is provided at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httplogomaker.readthedocs.io&gt;httplogomaker.readthedocs.io&lt;jatsext-link&gt;.Contactjkinney@cshl.edu.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/precise_tuning_of_cortical_contractility_regulates_cell_shape_during_cytokinesis_60423481/">
          <h4 class="font-weight-bold">Precise tuning of cortical contractility regulates cell shape during cytokinesis, bioRxiv, 2019-05-13</h4>
          <p>ABSTRACTThe mechanical properties of the cellular cortex regulate shape changes during cell division, cell migration and tissue morphogenesis. During cell division, contractile force generated by the molecular motor myosin II (MII) at the equatorial cortex drives cleavage furrow ingression. Cleavage furrow ingression in turn increases stresses at the polar cortex, where contractility must be regulated to maintain cell shape during cytokinesis. How polar cortex contractility controls cell shape is poorly understood. We show a balance between MII paralogs allows a fine-tuning of cortex tension at the polar cortex to maintain cell shape during cytokinesis, with MIIA driving cleavage furrow ingression and bleb formation, and MIIB serving as a stabilizing motor and mediating completion of cytokinesis. As the majority of non-muscle contractile systems are cortical, this tuning mechanism will likely be applicable to numerous processes driven by MII contractility.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/proximity_labeling_of_protein_complexes_and_cell_type-specific_organellar_proteomes_in_arabidopsis_enabled_by_turboid_60400444/">
          <h4 class="font-weight-bold">Proximity labeling of protein complexes and cell type-specific organellar proteomes in Arabidopsis enabled by TurboID, bioRxiv, 2019-05-13</h4>
          <p>AbstractDefining specific protein interactions and spatially or temporally restricted local proteomes improves our understanding of all cellular processes, but obtaining such data is challenging, especially for rare proteins, cell types, or events. Proximity labeling enables discovery of protein neighborhoods defining functional complexes andor organellar protein compositions. Recent technological improvements, namely two highly active biotin ligase variants (TurboID and miniTurboID), allowed us to address two challenging questions in plants (1) what are in vivo partners of a low abundant key developmental transcription factor and (2) what is the nuclear proteome of a rare cell type? Proteins identified with FAMA-TurboID include known interactors of this stomatal transcription factor and novel proteins that could facilitate its activator and repressor functions. Directing TurboID to stomatal nuclei enabled purification of cell type- and subcellular compartment-specific proteins. Broad tests of TurboID and miniTurboID in Arabidopsis and N. benthamiana and versatile vectors enable customization by plant researchers.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/the_tomato_pan-genome_uncovers_new_genes_and_a_rare_allele_regulating_fruit_flavor_60397352/">
          <h4 class="font-weight-bold">The tomato pan-genome uncovers new genes and a rare allele regulating fruit flavor, Nature Genetics, 2019-05-13</h4>
          <p>TomLoxC</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/nature%20genetics">nature genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/ultrastructure_of_the_axonal_periodic_scaffold_reveals_a_braid-like_organization_of_actin_rings_60400156/">
          <h4 class="font-weight-bold">Ultrastructure of the axonal periodic scaffold reveals a braid-like organization of actin rings, bioRxiv, 2019-05-13</h4>
          <p>AbstractRecent super-resolution microscopy studies have unveiled a periodic scaffold of actin rings regularly spaced by spectrins under the plasma membrane of axons. However, ultrastructural details are unknown, limiting a molecular and mechanistic understanding of these enigmatic structures. Here, we combine platinum-replica electron and optical super-resolution micros-copy to investigate the cortical cytoskeleton of axons at the ultrastructural level. We resolve actin rings as braids made of two long, intertwined actin filaments connected by a dense mesh of aligned spectrins. This molecular arrangement contrasts with the currently assumed model of actin rings made of short, capped actin filaments. We propose that braided rings explain the stability of the actin-spectrin scaffold and ultimately help preserving the axon integrity.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/2019/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/2019/page/20/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/17/" class="page-link">17</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/18/" class="page-link">18</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/19/" class="page-link">19</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/20/" class="page-link">20</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/2019/page/21/" class="page-link">21</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/22/" class="page-link">22</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/23/" class="page-link">23</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/24/" class="page-link">24</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/25/" class="page-link">25</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/2019/page/22/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/2019/page/87/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
