<!DOCTYPE html>
<html><head>
  
  <title>2019 | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/reconstruction_of_1000_projection_neurons_reveals_new_cell_types_and_organization_of_long-range_connectivity_in_the_mouse_brain_54827539/">
          <h4 class="font-weight-bold">Reconstruction of 1,000 projection neurons reveals new cell types and organization of long-range connectivity in the mouse brain, bioRxiv, 2019-02-02</h4>
          <p>Neuronal cell types are the nodes of neural circuits that determine the flow of information within the brain. Neuronal morphology, especially the shape of the axonal arbor, provides an essential descriptor of cell type and reveals how individual neurons route their output across the brain. Despite the importance of morphology, few projection neurons in the mouse brain have been reconstructed in their entirety. Here we present a robust and efficient platform for imaging and reconstructing complete neuronal morphologies, including axonal arbors that span substantial portions of the brain. We used this platform to reconstruct more than 1,000 projection neurons in the motor cortex, thalamus, subiculum, and hypothalamus. Together, the reconstructed neurons comprise more than 75 meters of axonal length and are available in a searchable online database. Axonal shapes revealed previously unknown subtypes of projection neurons and suggest organizational principles of long-range connectivity.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/self-reporting_transposons_enable_simultaneous_readout_of_gene_expression_and_transcription_factor_binding_in_single_cells_54834788/">
          <h4 class="font-weight-bold">Self-reporting transposons enable simultaneous readout of gene expression and transcription factor binding in single cells, bioRxiv, 2019-02-02</h4>
          <p>In situ assays of transcription factor (TF) binding are confounded by cellular heterogeneity and represent averaged profiles in complex tissues. Single cell RNA-seq (scRNA-seq) is capable of resolving different cell types based on gene expression profiles, but no technology exists to directly link specific cell types to the binding pattern of TFs in those cell types. Here, we present self-reporting transposons (SRTs) and their use in single cell calling cards (scCC), a novel assay for simultaneously capturing gene expression profiles and mapping TF binding sites in single cells. First, we show how the genomic locations of SRTs can be recovered from mRNA. Next, we demonstrate that SRTs deposited by the piggyBac transposase can be used to map the genome-wide localization of the TFs SP1, through a direct fusion of the two proteins, and BRD4, through its native affinity for piggyBac. We then present the scCC method, which maps SRTs from scRNA-seq libraries, thus enabling concomitant identification of cell types and TF binding sites in those same cells. As a proof-of-concept, we show recovery of cell type-specific BRD4 and SP1 binding sites from cultured cells. Finally, we map Brd4 binding sites in the mouse cortex at single cell resolution, thus establishing a new technique for studying TF biology in situ.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/synquant_an_automatic_tool_to_quantify_synapses_from_microscopy_images_54857304/">
          <h4 class="font-weight-bold">SynQuant An Automatic Tool to Quantify Synapses from Microscopy Images, bioRxiv, 2019-02-02</h4>
          <p>AbstractMotivationSynapses are essential to neural signal transmission. Therefore, quantification of synapses and related neurites from images is vital to gain insights into the underlying pathways of brain functionality and diseases. Despite the wide availability of synaptic punctum imaging data, several issues are impeding satisfactory quantification of these structures by current tools. First, the antibodies used for labeling synapses are not perfectly specific to synapses. These antibodies may exist in neurites or other cell compartments. Second, the brightness for different neurites and synaptic puncta is heterogeneous due to the variation of antibody concentration and synapse-intrinsic differences. Third, images often have low signal to noise ratio due to constraints of experiment facilities and availability of sensitive antibodies. These issues make the detection of synapses challenging and necessitates developing a new tool to easily and accurately quantify synapses.ResultsWe present an automatic probability-principled synapse detection algorithm and integrate it into our synapse quantification tool SynQuant. Derived from the theory of order statistics, our method controls the false discovery rate and improves the power of detecting synapses. SynQuant is unsupervised, works for both 2D and 3D data, and can handle multiple staining channels. Through extensive experiments on one synthetic and three real data sets with ground truth annotation or manual labeling, SynQuant was demonstrated to outperform peer specialized synapse detection tools as well as generic spot detection methods, including 4 unsupervised and 11 variants of 3 supervised methods.AvailabilityJava source code, Fiji plug-in, and test data available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsgithub.comyu-lab-vtSynQuant&gt;httpsgithub.comyu-lab-vtSynQuant&lt;jatsext-link&gt;.Contactyug@vt.edu</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/freeze-frame_imaging_of_synaptic_activity_using_syntagma_54819656/">
          <h4 class="font-weight-bold">Freeze-frame imaging of synaptic activity using SynTagMA, bioRxiv, 2019-02-01</h4>
          <p>AbstractInformation within the brain travels from neuron to neuron across billions of synapses. At any given moment, only a small subset of neurons and synapses are active, but finding the active synapses in brain tissue has been a technical challenge. To tag active synapses in a user-defined time window, we developed SynTagMA. Upon 405 nm illumination, this genetically encoded marker of activity converts from green to red fluorescence if, and only if, it is bound to calcium. Targeted to presynaptic terminals, preSynTagMA allows discrimination between active and silent axons. Targeted to excitatory postsynapses, postSynTagMA creates a snapshot of synapses active just before photoconversion. To analyze large datasets, we developed software to identify and track the fluorescence of thousands of individual synapses in tissue. Together, these tools provide a high throughput method for repeatedly mapping active neurons and synapses in cell culture, slice preparations, and in vivo during behavior.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/patterns_of_genetic_differentiation_and_the_footprints_of_historical_migrations_in_the_iberian_peninsula_54801340/">
          <h4 class="font-weight-bold">Patterns of genetic differentiation and the footprints of historical migrations in the Iberian Peninsula, Nature Communications, 2019-02-01</h4>
          <p>Reconquista</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/nature%20communications">nature communications</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/the_arabidopsis_thaliana_pan-nlrome_54771722/">
          <h4 class="font-weight-bold">The Arabidopsis thaliana pan-NLRome, bioRxiv, 2019-02-01</h4>
          <p>Disease is both among the most important selection pressures in nature and among the main causes of yield loss in agriculture. In plants, resistance to disease is often conferred by Nucleotide-binding Leucine-rich Repeat (NLR) proteins. These proteins function as intracellular immune receptors that recognize pathogen proteins and their effects on the plant. Consistent with evolutionarily dynamic interactions between plants and pathogens, NLRs are known to be encoded by one of the most variable gene families in plants, but the true extent of intraspecific NLR diversity has been unclear. Here, we define the majority of the Arabidopsis thaliana species-wide &#39;NLRome&#39;. From NLR sequence enrichment and long-read sequencing of 65 diverse A. thaliana accessions, we infer that the pan-NLRome saturates with approximately 40 accessions. Despite the high diversity of NLRs, half of the pan-NLRome is present in most accessions. We chart the architectural diversity of NLR proteins, identify novel architectures, and quantify the selective forces that act on specific NLRs, domains, and positions. Our study provides a blueprint for defining the pan-NLRome of plant species.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/2019/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/2019/page/71/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/68/" class="page-link">68</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/69/" class="page-link">69</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/70/" class="page-link">70</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/71/" class="page-link">71</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/2019/page/72/" class="page-link">72</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/73/" class="page-link">73</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/74/" class="page-link">74</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/75/" class="page-link">75</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/76/" class="page-link">76</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/2019/page/73/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/2019/page/87/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
