<!DOCTYPE html>
<html><head>
  
  <title>Cell Biology | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/muscle_specific_stress_fibers_give_rise_to_sarcomeres_and_are_mechanistically_distinct_from_stress_fibers_in_non-muscle_cells_30617152/">
          <h4 class="font-weight-bold">Muscle specific stress fibers give rise to sarcomeres and are mechanistically distinct from stress fibers in non-muscle cells, bioRxiv, 2017-12-19</h4>
          <p>AbstractThe sarcomere is the basic contractile unit within cardiomyocytes driving heart muscle contraction. We sought to test the mechanisms regulating thin (i.e., actin) and thick (i.e., myosin) filament assembly during sarcomere formation. Thus, we developed an assay using human cardiomyocytes to test de novo sarcomere assembly. Using this assay, we report a population of muscle-specific stress fibers are essential sarcomere precursors. We show sarcomeric actin filaments arise directly from these muscle stress fibers. This process requires formin-mediated but not Arp23-mediated actin polymerization and nonmuscle myosin IIB but not non-muscle myosin IIA. Furthermore, we show a short species of β cardiac myosin II filaments grows to form ~1.5 long filaments that then “stitch” together to form the stack of filaments at the core of the sarcomere (i.e., A-band). Interestingly, these are different from mechanisms that have previously been reported during stress fiber assembly in non-muscle cells. Thus, we provide a new model of cardiac sarcomere assembly based on distinct mechanisms of stress fiber regulation between non-muscle and muscle cells.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/transcription_organizes_euchromatin_similar_to_an_active_microemulsion_30486047/">
          <h4 class="font-weight-bold">Transcription organizes euchromatin similar to an active microemulsion, bioRxiv, 2017-12-16</h4>
          <p>Chromatin is organized into heterochromatin, which is transcriptionally inactive, and euchromatin, which can switch between transcriptionally active and inactive states. This switch in euchromatin activity is accompanied by changes in its spatial distribution. How euchromatin rearrangements are established is unknown. Here we use super-resolution and live-cell microscopy to show that transcriptionally inactive euchromatin moves away from transcriptionally active euchromatin. This movement is driven by the formation of RNA-enriched microenvironments that exclude inactive euchromatin. Using theory, we show that the segregation into RNA-enriched microenvironments and euchromatin domains can be considered an active microemulsion. The tethering of transcripts to chromatin via RNA polymerase II forms effective amphiphiles that intersperse the two segregated phases. Taken together with previous experiments, our data suggest that chromatin is organized in the following way heterochromatin segregates from euchromatin by phase separation, while transcription organizes euchromatin similar to an active microemulsion.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/partially_redundant_actin_genes_in_chlamydomonas_control_flagellum-directed_traffic_and_transition_zone_organization_29742284/">
          <h4 class="font-weight-bold">Partially redundant actin genes in Chlamydomonas control flagellum-directed traffic and transition zone organization, bioRxiv, 2017-12-01</h4>
          <p>ABSTRACTFlagella of the unicellular green alga Chlamydomonas reinhardtii are nearly identical to cilia of mammalian cells and provide an excellent model to study ciliogenesis. These biflagellated cells have two actin genes one encoding a conventional actin (IDA5) and the other encoding a divergent novel actin-like protein (NAP1). Previously, we described a role for actin in the regulation of flagella-building intraflagellar transport machinery. Here, we probe how actin redundancy contributes to this process using a nap1 mutant Chlamydomonas strain. Disruption of a single actin allows normal or slower incorporation but complete flagellar assembly. However, when we disrupt both actins using Latrunculin B (LatB) treatment on the nap1 mutant background, we find flagellar growth from newly synthesized limiting flagellar proteins is actin-dependent. Upon total actin disruption during flagellar assembly, transmission electron microscopy identified an accumulation of Golgi-adjacent vesicles, suggesting impaired vesicular trafficking may be the mechanism by which actin supports flagellar growth from new flagellar proteins. We also find there is a mislocalization of a key transition zone gating and ciliopathy protein, NPHP-4. Extended (2 hour) treatment with LatB, a condition under which NAP1 is upregulated, restores NPHP-4 localization. This suggests NAP1 can perform the functions of conventional actin at the transition zone. Our experiments demonstrate that each stage of flagellar biogenesis requires redundant actin function to varying degrees, with an absolute requirement for these actins in transport of Golgi-adjacent vesicles and flagellar incorporation of newly synthesized proteins.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/strong_binding_activity_of_few_transcription_factors_is_a_major_determinant_of_open_chromatin_27571437/">
          <h4 class="font-weight-bold">Strong binding activity of few transcription factors is a major determinant of open chromatin, bioRxiv, 2017-10-18</h4>
          <p>AbstractIt is well established that transcription factors (TFs) play crucial roles in determining cell identity, and that a large fraction of all TFs are expressed in most cell types. In order to globally characterize activities of TFs in cells, we have developed a novel massively parallel protein activity assay, Active TF Identification (ATI) that measures DNA-binding activity of all TFs from any species or tissue type. In contrast to previous studies based on mRNA expression or protein abundance, we found that a set of TFs binding to only around ten distinct motifs display strong DNA-binding activity in any given cell or tissue type. Mass spectrometric identification of TFs revealed that within these highly active TFs, there were both housekeeping TFs, which were universally found in all cell types, and specific TFs, which were highly enriched in known factors that determine the fate of the analyzed tissue or cell type. The importance of a small subset of TFs for determining the overall accessible chromatin landscape of a cell suggests that gene regulatory logic may be simpler than what has previously been appreciated.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/robust_and_bright_genetically_encoded_fluorescent_markers_for_highlighting_structures_and_compartments_in_mammalian_cells_21571833/">
          <h4 class="font-weight-bold">Robust and Bright Genetically Encoded Fluorescent Markers for Highlighting Structures and Compartments in Mammalian Cells, bioRxiv, 2017-07-07</h4>
          <p>To increase our understanding of cells, there is a need for specific markers to identify biomolecules, cellular structures and compartments. One type of markers comprises genetically encoded fluorescent probes that are linked with protein domains, peptides andor signal sequences. These markers are encoded on a plasmid and they allow straightforward, convenient labeling of cultured mammalian cells by introducing the plasmid into the cells. Ideally, the fluorescent marker combines favorable spectroscopic properties (brightness, photostability) with specific labeling of the structure or compartment of interest. Here, we report on our ongoing efforts to generate robust and bright genetically encoded fluorescent markers for highlighting structures and compartments in living cells.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/mitochondria_are_physiologically_maintained_at_close_to_50_c_19791234/">
          <h4 class="font-weight-bold">Mitochondria are physiologically maintained at close to 50 °C, bioRxiv, 2017-05-03</h4>
          <p>AbstractIn endothermic species, heat released as a product of metabolism ensures stable internal temperature throughout the organism, despite varying environmental conditions. Mitochondria are major actors in this thermogenic process. Part of the energy released by the oxidation of respiratory substrates drives ATP synthesis and metabolite transport, while a noticeable proportion is released as heat. Using a temperature-sensitive fluorescent probe targeted to mitochondria, we measured mitochondrial temperature in situ under different physiological conditions. At a constant external temperature of 38 °C, mitochondria were more than 10 °C warmer when the respiratory chain was fully functional, both in HEK293cells and primary skin fibroblasts. This differential was abolished in cells lacking mitochondrial DNA or by respiratory inhibitors, but preserved or enhanced by expressing thermogenic enzymes such as the alternative oxidase or the uncoupling protein 1. The activity of various RC enzymes was maximal at, or slightly above, 50 °C. Our study prompts a re-examination of the literature on mitochondria, taking account of the inferred high temperature.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/cell-biology/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/cell-biology/page/5/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/cell-biology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/cell-biology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/cell-biology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/cell-biology/page/5/" class="page-link">5</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/cell-biology/page/6/" class="page-link">6</a></li>
            
        

        

        
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
