<!DOCTYPE html>
<html><head>
  
  <title>Molecular Biology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/widespread_divergent_transcription_from_prokaryotic_promoters_74989464/">
          <h4 class="font-weight-bold">Widespread divergent transcription from prokaryotic promoters, bioRxiv, 2020-02-03</h4>
          <p>ABSTRACTPromoters are DNA sequences that stimulate the initiation of transcription. In all prokaryotes, promoters are believed to drive transcription in a single direction. Here we show that prokaryotic promoters are frequently bidirectional and drive divergent transcription. Mechanistically, this occurs because key promoter elements have inherent symmetry and often coincide on opposite DNA strands. Reciprocal stimulation between divergent transcription start sites also contributes. Horizontally acquired DNA is enriched for bidirectional promoters suggesting that they represent an early step in prokaryotic promoter evolution.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_minimal_crispr-cas3_system_for_genome_engineering_71822864/">
          <h4 class="font-weight-bold">A minimal CRISPR-Cas3 system for genome engineering, bioRxiv, 2019-12-03</h4>
          <p>AbstractCRISPR-Cas technologies have provided programmable gene editing tools that have revolutionized research. The leading CRISPR-Cas9 and Cas12a enzymes are ideal for programmed genetic manipulation, however, they are limited for genome-scale interventions. Here, we utilized a Cas3-based system featuring a processive nuclease, expressed endogenously or heterologously, for genome engineering purposes. Using an optimized and minimal CRISPR-Cas3 system (Type I-C) programmed with a single crRNA, large deletions ranging from 7 - 424 kb were generated in Pseudomonas aeruginosa with high efficiency and speed. By comparison, Cas9 yielded small deletions and point mutations. Cas3-generated deletion boundaries were variable in the absence of a homology-directed repair (HDR) template, and successfully and efficiently specified when present. The minimal Cas3 system is also portable; large deletions were induced with high efficiency in Pseudomonas syringae and Escherichia coli using an “all-in-one” vector. Notably, Cas3 generated bi-directional deletions originating from the programmed cut site, which was exploited to iteratively reduce a P. aeruginosa genome by 837 kb (13.5%) using 10 distinct crRNAs. We also demonstrate the utility of endogenous Cas3 systems (Type I-C and I-F) and develop an “anti-anti-CRISPR” strategy to circumvent endogenous CRISPR-Cas inhibitor proteins. CRISPR-Cas3 could facilitate rapid strain manipulation for synthetic biological and metabolic engineering purposes, genome minimization, and the analysis of large regions of unknown function.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_proximity_biotinylation_map_of_a_human_cell_67927593/">
          <h4 class="font-weight-bold">A proximity biotinylation map of a human cell, bioRxiv, 2019-10-08</h4>
          <p>Compartmentalization is an essential characteristic of eukaryotic cells, ensuring that cellular processes are partitioned to defined subcellular locations. High throughput microscopy1 and biochemical fractionation coupled with mass spectrometry2-6 have helped to define the proteomes of multiple organelles and macromolecular structures. However, many compartments have remained refractory to such methods, partly due to lysis and purification artefacts and poor subcompartment resolution. Recently developed proximity-dependent biotinylation approaches such as BioID and APEX provide an alternative avenue for defining the composition of cellular compartments in living cells (e.g. 7-10). Here we report an extensive BioID-based proximity map of a human cell, comprising 192 markers from 32 different compartments that identifies 35,902 unique high confidence proximity interactions and localizes 4,145 proteins expressed in HEK293 cells. The recall of our localization predictions is on par with or better than previous large-scale mass spectrometry and microscopy approaches, but with higher localization specificity. In addition to assigning compartment and subcompartment localization for many previously unlocalized proteins, our data contain fine-grained localization information that, for example, allowed us to identify proteins with novel roles in mitochondrial dynamics. As a community resource, we have created humancellmap.org, a website that allows exploration of our data in detail, and aids with the analysis of BioID experiments.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/mammalian_y_rnas_are_modified_at_discrete_guanosine_residues_with_n-glycans_67473699/">
          <h4 class="font-weight-bold">Mammalian Y RNAs are modified at discrete guanosine residues with N-glycans, bioRxiv, 2019-09-30</h4>
          <p>ABSTRACTGlycans modify lipids and proteins to mediate inter- and intramolecular interactions across all domains of life. RNA, another multifaceted biopolymer, is not thought to be a major target of glycosylation. Here, we challenge this view with evidence that mammalian cells use RNA as a third scaffold for glycosylation in the secretory pathway. Using a battery of chemical and biochemical approaches, we find that a select group of small noncoding RNAs including Y RNAs are modified with complex, sialylated N-glycans (glycoRNAs). These glycoRNA are present in multiple cell types and mammalian species, both in cultured cells andin vivo. Finally, we find that RNA glycosylation depends on the canonical N-glycan biosynthetic machinery within the ERGolgi luminal spaces. Collectively, these findings suggest the existence of a ubiquitous interface of RNA biology and glycobiology suggesting an expanded role for glycosylation beyond canonical lipid and protein scaffolds.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/fmrp_promotes_rna_localization_to_neuronal_projections_through_interactions_between_its_rgg_domain_and_g-quadruplex_rna_sequences_67386952/">
          <h4 class="font-weight-bold">FMRP promotes RNA localization to neuronal projections through interactions between its RGG domain and G-quadruplex RNA sequences, bioRxiv, 2019-09-28</h4>
          <p>ABSTRACTThe sorting of RNA molecules to distinct subcellular locations facilitates the activity of spatially restricted processes through local protein synthesis. This process affects thousands of transcripts yet precisely how these RNAs are trafficked to their destinations remains generally unclear. Here we have analyzed subcellular transcriptomes of FMRP-null mouse neuronal cells to identify transcripts that depend on FMRP for efficient transport to neurites. We found that these FMRP RNA localization targets contain a large enrichment of G-quadruplex sequences, particularly in their 3′ UTRs, suggesting that FMRP recognizes these sequences to promote the localization of transcripts that contain them. Fractionation of neurons derived from human Fragile X Syndrome patients revealed a high degree of conservation in the identity of FMRP localization targets between human and mouse as well as an enrichment of G-quadruplex sequences in human FMRP RNA localization targets. Using high-throughput RNAprotein interaction assays and single-molecule RNA FISH, we identified the RGG domain of FMRP as important for both interaction with G-quadruplex RNA sequences and the neuronal transport of G-quadruplex-containing transcripts. Finally, we used ribosome footprinting to identify translational regulatory targets of FMRP. The translational regulatory targets were not enriched for G-quadruplex sequences and were largely distinct from the RNA localization targets of FMRP, indicating that the two functions can be biochemically separated and are mediated through different target recognition mechanisms. These results establish a molecular mechanism underlying FMRP-mediated neuronal RNA localization and provide a framework for the elucidation of similar mechanisms governed by other RNA-binding proteins.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/probe-seq_enables_transcriptional_profiling_of_specific_cell_types_from_heterogeneous_tissue_by_rna-based_isolation_65018921/">
          <h4 class="font-weight-bold">Probe-Seq enables transcriptional profiling of specific cell types from heterogeneous tissue by RNA-based isolation, bioRxiv, 2019-08-15</h4>
          <p>ABSTRACTRecent transcriptional profiling technologies are uncovering previously-undefined cell populations and molecular markers at an unprecedented pace. While single cell RNA (scRNA) sequencing is an attractive approach for unbiased transcriptional profiling of all cell types, a complementary method to isolate and sequence specific cell populations from heterogeneous tissue remains challenging. Here, we developed Probe-Seq, which allows deep transcriptional profiling of specific cell types isolated using RNA as the defining feature. Dissociated cells are labelled using fluorescent in situ hybridization (FISH) for RNA, and then isolated by fluorescent activated cell sorting (FACS). We used Probe-Seq to purify and profile specific cell types from mouse, human, and chick retinas, as well as the Drosophila midgut. Probe-Seq is compatible with frozen nuclei, making cell types within archival tissue immediately accessible. As it can be multiplexed, combinations of markers can be used to create specificity. Multiplexing also allows for the isolation of multiple cell types from one cell preparation. Probe-Seq should enable RNA profiling of specific cell types from any organism.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        

        
             
            <li class="page-item active"><a href="/audiences/tags/molecular-biology/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/molecular-biology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/molecular-biology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/molecular-biology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/molecular-biology/page/5/" class="page-link">5</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/molecular-biology/page/2/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/molecular-biology/page/9/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
