<!DOCTYPE html>
<html><head>
  
  <title>Molecular Biology | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/super-resolution_imaging_reveals_3d_structure_and_organizing_mechanism_of_accessible_chromatin_62523093/">
          <h4 class="font-weight-bold">Super-resolution Imaging Reveals 3D Structure and Organizing Mechanism of Accessible Chromatin, bioRxiv, 2019-06-22</h4>
          <p>Access to cis-regulatory elements packaged in chromatin is essential for directing gene expression and cell viability. Here, we report a super-resolution imaging strategy, 3D ATAC-PALM, that enables direct visualization of the entire accessible genome. We found that active chromosomal segments are organized into spatially-segregated accessible chromatin domains (ACDs). Rapid depletion of CTCF or Cohesin (RAD21 subunit) induced enhanced ACD clustering, reduced physical separation between intrachromosomal ACDs, and differentially regulated ACD compaction. Experimental perturbations and polymer modeling suggest that dynamic protein-protein and protein-DNA interactions within ACDs couple with loop extrusion to organize ACD topology. Dysorganization of ACDs upon CTCF or Cohesin loss alters transcription factor binding and target search dynamics in living cells. These results uncover fundamental mechanisms underpinning the formation of 3D genome architecture and its pivotal function in transcriptional regulation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/enabling_high-accuracy_long-read_amplicon_sequences_using_unique_molecular_identifiers_and_nanopore_sequencing_61024590/">
          <h4 class="font-weight-bold">Enabling high-accuracy long-read amplicon sequences using unique molecular identifiers and Nanopore sequencing, bioRxiv, 2019-05-24</h4>
          <p>AbstractHigh-throughput amplicon sequencing of large genomic regions represents a challenge for existing short-read technologies. Long-read technologies can in theory sequence large genomic regions, but they currently suffer from high error rates. Here, we report a high-throughput amplicon sequencing approach that combines unique molecular identifiers (UMIs) with Oxford Nanopore sequencing to generate single-molecule consensus sequences of large genomic regions. We demonstrate the approach by generating nearly 10,000 full-length ribosomal RNA (rRNA) operons of roughly 4,400 bp in length from a mock microbial community consisting of eight bacterial species using a single Oxford Nanopore MinION flowcell. The mean error rate of the consensus sequences was 0.03%, with no detectable chimeras due to a rigorous UMI-barcode filtering strategy. The simplicity and accessibility of this method paves way for widespread use of high-accuracy amplicon sequencing in a variety of genomic applications.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/cryo-em_structure_of_the_clpxp_protein_degradation_machinery_60489906/">
          <h4 class="font-weight-bold">Cryo-EM structure of the ClpXP protein degradation machinery, bioRxiv, 2019-05-15</h4>
          <p>AbstractThe ClpXP machinery is a two component protease complex performing targeted protein degradation in bacteria and eukaryotes. The complex consists of the AAA&#43; chaperone ClpX and the peptidase ClpP. The hexameric ClpX utilizes the energy of ATP binding and hydrolysis to engage, unfold and translocate substrates into the catalytic chamber of tetradecameric ClpP where they are degraded. Formation of the complex involves a symmetry mismatch, since hexameric AAA&#43; rings bind axially to the opposing stacked heptameric rings of the tetradecameric ClpP. Here we present the first high-resolution cryo-EM structure of ClpXP from Listeria monocytogenes. We unravel the heptamer-hexamer binding interface and provide novel insights into the ClpX-ClpP crosstalk and activation mechanism. The comparison with available crystal structures of ClpP and ClpX in different states allows us to understand important aspects of ClpXP’s complex mode of action and provides a structural framework for future pharmacological applications.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/structural_and_functional_characterization_of_g_protein-coupled_receptors_with_deep_mutational_scanning_59714625/">
          <h4 class="font-weight-bold">Structural and Functional Characterization of G Protein-Coupled Receptors with Deep Mutational Scanning, bioRxiv, 2019-04-30</h4>
          <p>AbstractIn humans, the 813 G protein-coupled receptors (GPCRs) are responsible for transducing diverse chemical stimuli to alter cell state, and are the largest class of drug targets. Their myriad structural conformations and various modes of signaling make it challenging to understand their structure and function. Here we developed a platform to characterize large libraries of GPCR variants in human cell lines with a barcoded transcriptional reporter of G-protein signal transduction. We tested 7,800 of 7,828 possible single amino acid substitutions to the beta-2 adrenergic receptor (β2AR) at four concentrations of the agonist isoproterenol. We identified residues specifically important for β2AR signaling, mutations in the human population that are potentially loss of function, and residues that modulate basal activity. Using unsupervised learning, we resolve residues critical for signaling, including all major structural motifs and molecular interfaces. We also find a previously uncharacterized structural latch spanning the first two extracellular loops that is highly conserved across Class A GPCRs and is conformationally rigid in both the inactive and active states of the receptor. More broadly, by linking deep mutational scanning with engineered transcriptional reporters, we establish a generalizable method for exploring pharmacogenomics, structure and function across broad classes of drug receptors.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/native_molecule_sequencing_by_nano-id_reveals_synthesis_and_stability_of_rna_isoforms_58710054/">
          <h4 class="font-weight-bold">Native molecule sequencing by nano-ID reveals synthesis and stability of RNA isoforms, bioRxiv, 2019-04-08</h4>
          <p>AbstractEukaryotic genes often generate a variety of RNA isoforms that can lead to functionally distinct protein variants. The synthesis and stability of RNA isoforms is however poorly characterized. The reason for this is that current methods to quantify RNA metabolism use ‘short-read’ sequencing that cannot detect RNA isoforms. Here we present nanopore sequencing-based Isoform Dynamics (nano-ID), a method that detects newly synthesized RNA isoforms and monitors isoform metabolism. nano-ID combines metabolic RNA labeling, ‘long-read’ nanopore sequencing of native RNA molecules and machine learning. Application of nano-ID to the heat shock response in human cells reveals that many RNA isoforms change their synthesis rate, stability, and splicing pattern. nano-ID also shows that the metabolism of individual RNA isoforms differs strongly from that estimated for the combined RNA signal at a specific gene locus. And although combined RNA stability correlates with poly(A)-tail length, individual RNA isoforms can deviate significantly. nano-ID enables studies of RNA metabolism on the level of single RNA molecules and isoforms in different cell states and conditions.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/suppression_of_unwanted_crisprcas9_editing_by_co-administration_of_catalytically_inactivating_truncated_guide_rnas_58453110/">
          <h4 class="font-weight-bold">Suppression of unwanted CRISPRCas9 editing by co-administration of catalytically inactivating truncated guide RNAs, bioRxiv, 2019-04-04</h4>
          <p>AbstractCRISPRCas9 nucleases are powerful genome engineering tools, but unwanted cleavage at off-target and previously edited sites remains a major concern. Numerous strategies to reduce unwanted cleavage have been devised, but all are imperfect. Here, we report off-target sites can be shielded from the active Cas9•single guide RNA (sgRNA) complex through the co-administration of dead-RNAs (dRNAs), truncated guide RNAs that direct Cas9 binding but not cleavage. dRNAs can effectively suppress a wide-range of off-targets with minimal optimization while preserving on-target editing, and they can be multiplexed to suppress several off-targets simultaneously. dRNAs can be combined with high-specificity Cas9 variants, which often do not eliminate all unwanted editing. Moreover, dRNAs can prevent cleavage of homology-directed repair (HDR)-corrected sites, facilitating “scarless” editing by eliminating the need for blocking mutations. Thus, we enable precise genome editing by establishing a novel and flexible approach for suppressing unwanted editing of both off-targets and HDR-corrected sites.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        

        
             
            <li class="page-item active"><a href="/tags/molecular-biology/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/molecular-biology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/molecular-biology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/molecular-biology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/molecular-biology/page/5/" class="page-link">5</a></li>
            
        

        
        <li class="page-item"><a href="/tags/molecular-biology/page/2/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/molecular-biology/page/5/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
