<!DOCTYPE html>
<html><head>
  
  <title>Molecular Biology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/native_molecule_sequencing_by_nano-id_reveals_synthesis_and_stability_of_rna_isoforms_58710054/">
          <h4 class="font-weight-bold">Native molecule sequencing by nano-ID reveals synthesis and stability of RNA isoforms, bioRxiv, 2019-04-08</h4>
          <p>AbstractEukaryotic genes often generate a variety of RNA isoforms that can lead to functionally distinct protein variants. The synthesis and stability of RNA isoforms is however poorly characterized. The reason for this is that current methods to quantify RNA metabolism use ‘short-read’ sequencing that cannot detect RNA isoforms. Here we present nanopore sequencing-based Isoform Dynamics (nano-ID), a method that detects newly synthesized RNA isoforms and monitors isoform metabolism. nano-ID combines metabolic RNA labeling, ‘long-read’ nanopore sequencing of native RNA molecules and machine learning. Application of nano-ID to the heat shock response in human cells reveals that many RNA isoforms change their synthesis rate, stability, and splicing pattern. nano-ID also shows that the metabolism of individual RNA isoforms differs strongly from that estimated for the combined RNA signal at a specific gene locus. And although combined RNA stability correlates with poly(A)-tail length, individual RNA isoforms can deviate significantly. nano-ID enables studies of RNA metabolism on the level of single RNA molecules and isoforms in different cell states and conditions.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/suppression_of_unwanted_crisprcas9_editing_by_co-administration_of_catalytically_inactivating_truncated_guide_rnas_58453110/">
          <h4 class="font-weight-bold">Suppression of unwanted CRISPRCas9 editing by co-administration of catalytically inactivating truncated guide RNAs, bioRxiv, 2019-04-04</h4>
          <p>AbstractCRISPRCas9 nucleases are powerful genome engineering tools, but unwanted cleavage at off-target and previously edited sites remains a major concern. Numerous strategies to reduce unwanted cleavage have been devised, but all are imperfect. Here, we report off-target sites can be shielded from the active Cas9•single guide RNA (sgRNA) complex through the co-administration of dead-RNAs (dRNAs), truncated guide RNAs that direct Cas9 binding but not cleavage. dRNAs can effectively suppress a wide-range of off-targets with minimal optimization while preserving on-target editing, and they can be multiplexed to suppress several off-targets simultaneously. dRNAs can be combined with high-specificity Cas9 variants, which often do not eliminate all unwanted editing. Moreover, dRNAs can prevent cleavage of homology-directed repair (HDR)-corrected sites, facilitating “scarless” editing by eliminating the need for blocking mutations. Thus, we enable precise genome editing by establishing a novel and flexible approach for suppressing unwanted editing of both off-targets and HDR-corrected sites.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_heterochromatin-specific_rna_export_pathway_facilitates_pirna_production_58354059/">
          <h4 class="font-weight-bold">A heterochromatin-specific RNA export pathway facilitates piRNA production, bioRxiv, 2019-04-02</h4>
          <p>PIWI-interacting RNAs (piRNAs) guide transposon silencing in animals. The 22-30nt piRNAs are processed in the cytoplasm from long non-coding RNAs. How piRNA precursors, which often lack RNA processing hallmarks of export-competent transcripts, achieve nuclear export is unknown. Here, we uncover the RNA export pathway specific for piRNA precursors in the Drosophila germline. This pathway requires Nxf3-Nxt1, a variant of the hetero-dimeric mRNA export receptor Nxf1-Nxt1. Nxf3 interacts with UAP56, a nuclear RNA helicase essential for mRNA export, and CG13741Bootlegger, which recruits Nxf3-Nxt1 and UAP56 to heterochromatic piRNA source loci. Upon RNA cargo binding, Nxf3 achieves nuclear export via the exportin Crm1, and accumulates together with Bootlegger in peri-nuclear nuage, suggesting that after export, Nxf3-Bootlegger delivers precursor transcripts to the piRNA processing sites. Our findings indicate that the piRNA pathway bypasses nuclear RNA surveillance systems to achieve export of heterochromatic, unprocessed transcripts to the cytoplasm, a strategy also exploited by retroviruses.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/muscle_fiber_hypertrophy_in_response_to_6_weeks_of_high-volume_resistance_training_in_trained_young_men_is_largely_attributed_to_sarcoplasmic_hypertrophy_58360274/">
          <h4 class="font-weight-bold">Muscle fiber hypertrophy in response to 6 weeks of high-volume resistance training in trained young men is largely attributed to sarcoplasmic hypertrophy, bioRxiv, 2019-04-02</h4>
          <p>ABSTRACTCellular adaptations that occur during skeletal muscle hypertrophy in response to high-volume resistance training are not well-characterized. Therefore, we sought to explore how actin, myosin, sarcoplasmic protein, mitochondrial, and glycogen concentrations were altered in individuals that exhibited mean skeletal muscle fiber cross-sectional area (fCSA) hypertrophy following 6 weeks of high-volume resistance training. Thirty-one previously resistance-trained, college-aged males (mean ± standard deviation 21±2 years, 5±3 training years) had vastus lateralis (VL) muscle biopsies obtained prior to training (PRE), at week 3 (W3), and at week 6 (W6). Muscle tissue from 15 subjects exhibiting PRE to W6 VL mean fCSA increases ranging from 320-1600 μm2 was further interrogated using various biochemical and histological assays as well as proteomic analysis. Seven of these individuals donated a VL biopsy after refraining from training 8 days following the last training session (W7) to determine how deloading affected biomarkers. The 15 fCSA hypertrophic responders experienced a &#43;23% increase in mean fCSA from PRE to W6 (p&amp;lt;0.001) and, while muscle glycogen concentrations remained unaltered, citrate synthase activity levels decreased by 24% (p&amp;lt;0.001) suggesting mitochondrial volume decreased. Interestingly, both myosin and actin concentrations decreased ~30% from PRE to W6 (p&amp;lt;0.05). Phalloidin-actin staining similarly revealed actin concentrations per fiber decreased from PRE to W6. Proteomic analysis of the sarcoplasmic fraction from PRE to W6 indicated 40 proteins were up-regulated (p&amp;lt;0.05), KEGG analysis indicated that the glycolysisgluconeogenesis pathway was upregulated (FDR sig. &amp;lt;0.001), and DAVID indicated that the following functionally-annotated pathways were upregulated (FDR value &amp;lt;0.05) a) glycolysis (8 proteins), b) acetylation (23 proteins), c) gluconeogenesis (5 proteins) and d) cytoplasm (20 proteins). At W7, sarcoplasmic protein concentrations remained higher than PRE (&#43;66%, p&amp;lt;0.05), and both actin and myosin concentrations remained lower than PRE (~−50%, p&amp;lt;0.05). These data suggest that short-term high-volume resistance training may a) reduce muscle fiber actin and myosin protein concentrations in spite of increasing fCSA, and b) promote sarcoplasmic expansion coincident with a coordinated up-regulation of sarcoplasmic proteins involved in glycolysis and other metabolic processes related to ATP generation. Interestingly, these effects seem to persist up to 8 days following training.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/crisprcas9-based_mutagenesis_frequently_provokes_on-target_mrna_misregulation_57333383/">
          <h4 class="font-weight-bold">CRISPRCas9-based mutagenesis frequently provokes on-target mRNA misregulation, bioRxiv, 2019-03-20</h4>
          <p>The introduction of insertion-deletions (INDELs) by activation of the error-prone non-homologous end-joining (NHEJ) pathway underlies the mechanistic basis of CRISPRCas9-directed genome editing. The ability of CRISPRCas9 to achieve gene elimination (knockouts) is largely attributed to the emergence of a pre-mature termination codon (PTC) from a frameshift-inducing INDEL that elicits non-sense mediated decay (NMD) of the mutant mRNA. Yet, the impact on gene expression as a consequence of CRISPRCas9-introduced INDELs into RNA regulatory sequences has been largely left uninvestigated. By tracking DNA-mRNA-protein relationships in a collection of CRISPRCas9-edited cell lines that harbor frameshift-inducing INDELs in various targeted genes, we detected the production of foreign mRNAs or proteins in ∼50% of the cell lines. We demonstrate that these aberrant protein products are derived from the introduction of INDELs that promote internal ribosomal entry, convert pseudo-mRNAs into protein encoding molecules, or induce exon skipping by disruption of exon splicing enhancers (ESEs). Our results using CRISPRCas9-introduced INDELs reveal facets of an epigenetic genome buffering apparatus that likely evolved to mitigate the impact of such mutations introduced by pathogens and aberrant DNA damage repair, and that more recently pose challenges to manipulating gene expression outcomes using INDEL-based mutagenesis.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_comparison_of_dna_stains_and_staining_methods_for_agarose_gel_electrophoresis_56625796/">
          <h4 class="font-weight-bold">A comparison of DNA stains and staining methods for Agarose Gel Electrophoresis, bioRxiv, 2019-03-07</h4>
          <p>ABSTRACTNucleic acid stains are necessary for Agarose Gel Electrophoresis (AGE). The commonly used but mutagenic Ethidium Bromide is being usurped by a range of safer but more expensive alternatives. These safe stains vary in cost, sensitivity and the impedance of DNA as it migrates through the gel. Modified protocols developed to reduce cost increase this variability. In this study, five Gel stains (GelRed™, GelGreen™, SYBR™ safe, SafeView and EZ-Vision®In-Gel Solution) two premixed loading dyes (SafeWhite, EZ-Vision®One) and four methods (pre-loading at 100x, pre-loading at 10x, precasting and post-staining) are evaluated for sensitivity and effect on DNA migration. GelRed™ was found to be the most sensitive while the EZ-Vision® dyes and SafeWhite had no discernible effect on DNA migration. Homemade loading dyes were as effective as readymade ones at less than 4% of the price. This method used less than 1% of the dye needed for the manufacturer recommended protocols. Thus, with careful consideration of stain and method, Gel stain expenditure can be reduced by over 99%.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/molecular-biology/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/molecular-biology/page/2/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="../../../../tags/molecular-biology/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/molecular-biology/page/2/" class="page-link">2</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/molecular-biology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/molecular-biology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/molecular-biology/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/molecular-biology/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/molecular-biology/page/7/" class="page-link">7</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/molecular-biology/page/4/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/molecular-biology/page/9/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
