<!DOCTYPE html>
<html><head>
  
  <title>microbiology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.68.3" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.95d2483cd3c5d2bf47c1f36307a456787d60ad958c5efe44db9afc47012b1b05b8e1f64f6ac291650439322a6c38e86eb7cd6254255516d857c23f299fb0e666.js" integrity="sha512-ldJIPNPF0r9HwfNjB6RWeH1grZWMXv5E25r8RwErGwW44fZPasKRZQQ5MipsOOhut81iVCVVFthXwj8pn7DmZg=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_membrane-depolarising_toxin_substrate_of_the_staphylococcus_aureus_type_vii_protein_secretion_system_targets_eukaryotes_and_bacteria_49785415/">
          <h4 class="font-weight-bold">A membrane-depolarising toxin substrate of the Staphylococcus aureus Type VII protein secretion system targets eukaryotes and bacteria, bioRxiv, 2018-10-16</h4>
          <p>SummaryThe type VII protein secretion system (T7SS) is conserved across Staphylococcus aureus strains and plays important roles in virulence and interbacterial competition. To date only one T7SS substrate protein, encoded in a subset of strains, has been functionally characterized. Here, using an unbiased proteomic approach, we identify TspA as a further T7SS substrate. TspA, encoded distantly from the T7SS gene cluster, is found across all S. aureus strains. Heterologous expression of TspA indicates that it has a toxic C-terminal domain that depolarizes membranes. The membrane depolarizing activity is alleviated by co-production of the TsaI immunity protein. Using a zebrafish hindbrain ventricle infection model, we demonstrate that the T7SS of strain RN6390 contributes to zebrafish mortality, and deletion of tspA leads to increased bacterial clearance in vivo. The toxin domain of TspA is highly polymorphic and S. aureus strains encode multiple tsaI homologues at the tspA locus, suggestive of additional roles in intra-species competition. In agreement, we demonstrate TspA-dependent growth inhibition of RN6390 by strain COL in the zebrafish infection model, that is alleviated by the presence of TsaI homologues. This is the first T7SS substrate protein shown to have activity against both eukaryotes and prokaryotes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/in_situ_and_high-resolution_cryo-em_structure_of_the_type_vi_secretion_membrane_complex_49625246/">
          <h4 class="font-weight-bold">In situ and high-resolution Cryo-EM structure of the Type VI secretion membrane complex, bioRxiv, 2018-10-13</h4>
          <p>AbstractBacteria have evolved macromolecular machineries that secrete effectors and toxins to survive and thrive in diverse environments. The type VI secretion system (T6SS) is a contractile machine that is related to Myoviridae phages. The T6SS is composed of a baseplate that contains a spike onto which an inner tube is built, surrounded by a contractile sheath. Unlike phages that are released to and act in the extracellular medium, the T6SS is an intracellular machine inserted in the bacterial membranes by a trans-envelope complex. This membrane complex (MC) comprises three proteins TssJ, TssL and TssM. We previously reported the low-resolution negative stain electron microscopy structure of the enteroaggregative Escherichia coli MC and proposed a rotational 5-fold symmetry with a TssJTssLTssM stoichiometry of 222. Here, cryo-electron tomography analysis of the T6SS MC confirmed the 5-fold symmetry in situ and identified the regions of the structure that insert into the bacterial membranes. A high resolution model obtained by single particle cryo-electron microscopy reveals its global architecture and highlights new features five additional copies of TssJ, yielding a TssJTssLTssM stoichiometry of 322, a 11-residue loop in TssM, protruding inside the lumen of the MC and constituting a functionally important periplasmic gate, and hinge regions. Based on these data, we revisit the model on the mechanism of action of the MC during T6SS assembly and function.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/single-cell_virus_sequencing_of_influenza_infections_that_trigger_innate_immunity_49343272/">
          <h4 class="font-weight-bold">Single-cell virus sequencing of influenza infections that trigger innate immunity, bioRxiv, 2018-10-07</h4>
          <p>SUMMARYThe outcome of viral infection is extremely heterogeneous, with infected cells only sometimes activating innate immunity. Here we develop a new approach to assess how the genetic variation inherent in viral populations contributes to this heterogeneity. We do this by determining both the transcriptome and full-length sequences of all viral genes in single influenza-infected cells. Most cells are infected by virions with defects such as amino-acid mutations, internal deletions, or failure to express a gene. We identify instances of each type of defect that increase the likelihood that a cell activates an innate-immune response. However, immune activation remains stochastic in cells infected by virions with these defects, and sometimes occurs even when a cell is infected by a virion that expresses unmutated copies of all genes. Our work shows that viral genetic variation substantially contributes to but does not fully explain the heterogeneity in single influenza-infected cells.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/co-option_and_detoxification_of_a_phage_lysin_for_housekeeping_function_48500288/">
          <h4 class="font-weight-bold">Co-option and Detoxification of a Phage Lysin for Housekeeping Function, bioRxiv, 2018-09-16</h4>
          <p>SummaryTemperate phages constitute a potentially beneficial genetic reservoir for bacterial innovation despite being selfish entities encoding an infection cycle inherently at odds with bacterial fitness. These phages integrate their genomes into the bacterial host during infection, donating new, but deleterious, genetic material the phage genome encodes toxic genes, such as lysins, that kill the bacterium during the phage infection cycle. Remarkably, some bacteria have exploited the destructive properties of phage genes for their own benefit by co-opting them as toxins for functions related to bacterial warfare, virulence, and secretion. However, do toxic phage genes ever become raw material for functional innovation? Here we report on a toxic phage gene whose product has lost its toxicity and has become a domain of a core cellular factor, SpmX, throughout the bacterial order Caulobacterales. Using a combination of phylogenetics, bioinformatics, structural biology, cell biology, and biochemistry, we have investigated the origin and function of SpmX and determined that its occurrence is the result of the detoxification of a phage peptidoglycan hydrolase gene. We show that the retained, attenuated activity of the phage-derived domain plays an important role in proper cell morphology and developmental regulation in representatives of this large bacterial clade. To our knowledge, this is the first observation of phage gene domestication in which a toxic phage gene has been co-opted for a housekeeping function.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/widespread_methane_formation_by_cyanobacteria_in_aquatic_and_terrestrial_ecosystems_47078636/">
          <h4 class="font-weight-bold">Widespread methane formation by Cyanobacteria in aquatic and terrestrial ecosystems, bioRxiv, 2018-08-25</h4>
          <p>AbstractEvidence is accumulating to challenge the paradigm that biogenic methanogenesis, traditionally considered a strictly anerobic process, is exclusive to Archaea. Here we demonstrate that Cyanobacteria living in marine, freshwater and terrestrial environments produce methane at substantial rates under light and dark oxic and anoxic conditions, forming a link between light driven primary productivity and methane production in globally relevant group of phototrophs. Biogenic methane production was enhanced during oxygenic photosynthesis and directly attributed to the cyanobacteria by applying stable isotope labelling techniques. We suggest that formation of methane by Cyanobacteria may contribute to methane accumulation in oxygen-saturated surface waters of marine and freshwater ecosystems. Moreover, in these environments, cyanobacterial blooms already do, and might further occur more frequently during future global warming and thus have a direct feedback on climate change. We further highlight that cyanobacterial methane production not only affects recent and future global methane budgets, but also has implications for inferences on Earth’s methane budget for the last 3.5 billion years, when this phylum is thought to have first evolved.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/high-throughput_amplicon_sequencing_of_the_full-length_16s_rrna_gene_with_single-nucleotide_resolution_46552121/">
          <h4 class="font-weight-bold">High-throughput amplicon sequencing of the full-length 16S rRNA gene with single-nucleotide resolution, bioRxiv, 2018-08-16</h4>
          <p>AbstractTargeted PCR amplification and high-throughput sequencing (amplicon sequencing) of 16S rRNA gene fragments is widely used to profile microbial communities. New long-read sequencing technologies can sequence the entire 16S rRNA gene, but higher error rates have limited their attractiveness when accuracy is important. Here we present a high-throughput amplicon sequencing methodology based on PacBio circular consensus sequencing and the DADA2 sample inference method that measures the full-length 16S rRNA gene with single-nucleotide resolution and a near-zero error rate.In two artificial communities of known composition, our method recovered the full complement of full-length 16S sequence variants from expected community members without residual errors. The measured abundances of intra-genomic sequence variants were in the integral ratios expected from the genuine allelic variants within a genome. The full-length 16S gene sequences recovered by our approach allowed E. coli strains to be correctly classified to the O157H7 and K12 sub-species clades. In human fecal samples, our method showed strong technical replication and was able to recover the full complement of 16S rRNA alleles in several E. coli strains.There are likely many applications beyond microbial profiling for which high-throughput amplicon sequencing of complete genes with single-nucleotide resolution will be of use.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/12/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/10/" class="page-link">10</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/11/" class="page-link">11</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/12/" class="page-link">12</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/microbiology/page/13/" class="page-link">13</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/14/" class="page-link">14</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/15/" class="page-link">15</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/16/" class="page-link">16</a></li>
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/14/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/16/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
