<!DOCTYPE html>
<html><head>
  
  <title>200 500 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/ultra-high_throughput_single-cell_rna_sequencing_by_combinatorial_fluidic_indexing_72967508/">
          <h4 class="font-weight-bold">Ultra-high throughput single-cell RNA sequencing by combinatorial fluidic indexing, bioRxiv, 2019-12-19</h4>
          <p>AbstractCell atlas projects and single-cell CRISPR screens hit the limits of current technology, as they require cost-effective profiling for millions of individual cells. To satisfy these enormous throughput requirements, we developed “single-cell combinatorial fluidic indexing” (scifi) and applied it to single-cell RNA sequencing. The resulting scifi-RNA-seq assay combines one-step combinatorial pre-indexing of single-cell transcriptomes with subsequent single-cell RNA-seq using widely available droplet microfluidics. Pre-indexing allows us to load multiple cells per droplet, which increases the throughput of droplet-based single-cell RNA-seq up to 15-fold, and it provides a straightforward way of multiplexing hundreds of samples in a single scifi-RNA-seq experiment. Compared to multi-round combinatorial indexing, scifi-RNA-seq provides an easier, faster, and more efficient workflow, thereby enabling massive-scale scRNA-seq experiments for a broad range of applications ranging from population genomics to drug screens with scRNA-seq readout. We benchmarked scifi-RNA-seq on various human and mouse cell lines, and we demonstrated its feasibility for human primary material by profiling TCR activation in T cells.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/late_life_metformin_treatment_limits_cell_survival_and_shortens_lifespan_by_triggering_an_aging-associated_failure_of_energy_metabolism_71919297/">
          <h4 class="font-weight-bold">Late life metformin treatment limits cell survival and shortens lifespan by triggering an aging-associated failure of energy metabolism, bioRxiv, 2019-12-04</h4>
          <p>SummaryThe diabetes drug metformin is to be clinically tested in aged humans to achieve health span extension, but little is known about responses of old non-diabetic individuals to this drug. By in vitro and in vivo tests we found that metformin shortens life span and limits cell survival when provided in late life, contrary to its positive early life effects. Mechanistically, metformin exacerbates aging-associated mitochondrial dysfunction towards respiratory failure, aggravated by the inability of old cells to upregulate glycolysis in response to metformin, leading to ATP exhaustion. The beneficial dietary restriction effect of metformin on lipid reserves is abrogated in old animals, contributing to metabolic failure, while ectopic stabilization of cellular ATP levels alleviates late life metformin toxicity in vitro and in vivo. The toxicity is also suspended in nematodes carrying diabetes-like insulin receptor insufficiency and showing prolonged resilience to metabolic stress induced by metformin. In sum, we uncovered an alarming metabolic decay triggered by metformin in late life which may limit its benefits for non-diabetic elderly patients. Novel regulators of life extension by metformin are also presented.Highlights&lt;jatslist list-type=bullet&gt;&lt;jatslist-item&gt;Late life metformin treatment limits cell survival and shortens lifespan.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Metformin exacerbates aging-associated mitochondrial dysfunction causing fatal ATP exhaustion.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Old cells fail to upregulate glycolysis as a compensatory response to metformin.&lt;jatslist-item&gt;&lt;jatslist-item&gt;The dietary restriction (DR) mimetic response to metformin is abrogated in old animals.&lt;jatslist-item&gt;&lt;jatslist-item&gt;PKA and not AMPK pathway instigates the early life DR response to metformin.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Stabilization of cellular ATP levels alleviates late life metformin toxicity in vitro and in vivo.&lt;jatslist-item&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/physiology">physiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/in_situ_cryo-electron_tomography_reveals_filamentous_actin_within_the_microtubule_lumen_70722436/">
          <h4 class="font-weight-bold">In situ cryo-electron tomography reveals filamentous actin within the microtubule lumen, bioRxiv, 2019-11-18</h4>
          <p>AbstractMicrotubules and filamentous (F-) actin engage in complex interactions to drive many cellular processes from subcellular organisation to cell division and migration. This is thought to be largely controlled by proteins that interface between the two structurally distinct cytoskeletal components. Here, we use cryo-electron tomography to demonstrate that the microtubule lumen can be occupied by extended segments of F-actin in small-molecule induced, microtubule-based cellular projections. We uncover an unexpected versatility in cytoskeletal form that may prompt a significant development of our current models of cellular architecture and offer a new experimental approach for the in-situ study of microtubule structure and contents.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/quantitative_translation_of_dog-to-human_aging_by_conserved_remodeling_of_epigenetic_networks_69766930/">
          <h4 class="font-weight-bold">Quantitative translation of dog-to-human aging by conserved remodeling of epigenetic networks, bioRxiv, 2019-11-04</h4>
          <p>SUMMARYMammals progress through similar physiological stages during life, from early development to puberty, aging, and death. Yet, the extent to which this conserved physiology reflects conserved molecular events is unclear. Here, we map common epigenetic changes experienced by mammalian genomes as they age, focusing on evolutionary comparisons of humans to dogs, an emerging model of aging. Using targeted sequencing, we characterize the methylomes of 104 Labrador retrievers spanning a 16 year age range, achieving &amp;gt;150X coverage within mammalian syntenic blocks. Comparison with human methylomes reveals a nonlinear relationship which translates dog to human years, aligns the timing of major physiological milestones between the two species, and extends to mice. Conserved changes center on specific developmental gene networks which are sufficient to capture the effects of anti-aging interventions in multiple mammals. These results establish methylation not only as a diagnostic age readout but as a cross-species translator of physiological aging milestones.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/generalizing_rna_velocity_to_transient_cell_states_through_dynamical_modeling_69439202/">
          <h4 class="font-weight-bold">Generalizing RNA velocity to transient cell states through dynamical modeling, bioRxiv, 2019-10-29</h4>
          <p>AbstractThe introduction of RNA velocity in single cells has opened up new ways of studying cellular differentiation. The originally proposed framework obtains velocities as the deviation of the observed ratio of spliced and unspliced mRNA from an inferred steady state. Errors in velocity estimates arise if the central assumptions of a common splicing rate and the observation of the full splicing dynamics with steady-state mRNA levels are violated. With scVelo (&lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsscvelo.org&gt;httpsscvelo.org&lt;jatsext-link&gt;), we address these restrictions by solving the full transcriptional dynamics of splicing kinetics using a likelihood-based dynamical model. This generalizes RNA velocity to a wide variety of systems comprising transient cell states, which are common in development and in response to perturbations. We infer gene-specific rates of transcription, splicing and degradation, and recover the latent time of the underlying cellular processes. This latent time represents the cell’s internal clock and is based only on its transcriptional dynamics. Moreover, scVelo allows us to identify regimes of regulatory changes such as stages of cell fate commitment and, therein, systematically detects putative driver genes. We demonstrate that scVelo enables disentangling heterogeneous subpopulation kinetics with unprecedented resolution in hippocampal dentate gyrus neurogenesis and pancreatic endocrinogenesis. We anticipate that scVelo will greatly facilitate the study of lineage decisions, gene regulation, and pathway activity identification.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_simple_method_for_spray-on_gene_editing_in_planta_69298477/">
          <h4 class="font-weight-bold">A simple method for spray-on gene editing in planta, bioRxiv, 2019-10-26</h4>
          <p>Potential innovation in Plant research using gene-edited and genetically modified plants is currently being hindered by inefficient and costly plant transformation. We show that carbon dots formed from natural materials (quasi-spherical, &amp;lt;10nm nanoparticles) can act as a fast vehicle for carrying plasmids into mature plant cells, resulting in transient plant transformation in a number of important crop species with no negative impacts on photosynthesis or growth. We further show that GFP, Cas9, and gRNA introduced into wheat via foliar application (spraying on) of plasmid coated carbon dots are expressed and, in the case of Cas9, make genome edits in SPO11 genes. Therefore, we present a protocol for spray-on gene editing that is simple, inexpensive, fast, transforms in planta, and is applicable to multiple crop species. We believe this technique creates many opportunities for the future of plant transformation in research and shows great promise for plant protein production systems.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        
        <li class="page-item"><a href="../../../../tags/200-500-users/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="../../../../tags/200-500-users/" class="page-link">1</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/200-500-users/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/200-500-users/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/200-500-users/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/200-500-users/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/200-500-users/page/6/" class="page-link">6</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/200-500-users/page/3/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/200-500-users/page/56/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
