<!DOCTYPE html>
<html><head>
  
  <title>Neuroscience | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/dopamine_transients_delivered_in_learning_contexts_do_not_act_as_model-free_prediction_errors_56909657/">
          <h4 class="font-weight-bold">Dopamine transients delivered in learning contexts do not act as model-free prediction errors, bioRxiv, 2019-03-12</h4>
          <p>AbstractDopamine neurons fire transiently in response to unexpected rewards. These neural correlates are proposed to signal the reward prediction error described in model-free reinforcement learning algorithms. This error term represents the unpredicted or ‘excess’ value of the rewarding event. In model-free reinforcement learning, this value is then stored as part of the learned value of any antecedent cues, contexts or events, making them intrinsically valuable, independent of the specific rewarding event that caused the prediction error. In support of equivalence between dopamine transients and this model-free error term, proponents cite causal optogenetic studies showing that artificially induced dopamine transients cause lasting changes in behavior. Yet none of these studies directly demonstrate the presence of cached value under conditions appropriate for associative learning. To address this gap in our knowledge, we conducted three studies where we optogenetically activated dopamine neurons while rats were learning associative relationships, both with and without reward. In each experiment, the antecedent cues failed to acquired value and instead entered into value-independent associative relationships with the other cues or rewards. These results show that dopamine transients, constrained within appropriate learning situations, support valueless associative learning.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/blue_light_induces_neuronal-activity-regulated_gene_expression_in_the_absence_of_optogenetic_proteins_56756504/">
          <h4 class="font-weight-bold">Blue light induces neuronal-activity-regulated gene expression in the absence of optogenetic proteins, bioRxiv, 2019-03-09</h4>
          <p>Optogenetics is widely used to control diverse cellular functions with light, requiring experimenters to expose cells to bright light. Because extended exposure to visible light can be toxic to cells, it is important to characterize the effects of light stimulation on cellular function in the absence of exogenous optogenetic proteins. Here we exposed cultured mouse cortical neurons that did not express optogenetic proteins to several hours of flashing blue, red, or green light. We found that exposing neurons to as short as one hour of blue, but not red or green, light results in the induction of neuronal-activity-regulated genes without inducing neuronal activity. Our findings suggest blue light stimulation is ill-suited to long-term optogenetic experiments, especially those that measure transcription.Significance StatementOptogenetics is widely used to control cellular functions using light. In neuroscience, channelrhodopsins, exogenous light-sensitive channels, are used to achieve light-dependent control of neuronal firing. This optogenetic control of neuronal firing requires exposing neurons to high-powered light. We ask how this light exposure, in the absence of channelrhodopsin, affects the expression of neuronal-activity-regulated genes, i.e., the genes that are transcribed in response to neuronal stimuli. Surprisingly, we find that neurons without channelrhodopsin express neuronal-activity-regulated genes in response to as short as an hour of blue, but not red or green, light exposure. These findings suggest that experimenters wishing to achieve longer-term (an hour or more) optogenetic control over neuronal firing should avoid using systems that require blue light.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/whisking_asymmetry_signals_motor_preparation_and_the_behavioral_state_of_mice_56668364/">
          <h4 class="font-weight-bold">Whisking asymmetry signals motor preparation and the behavioral state of mice, bioRxiv, 2019-03-08</h4>
          <p>AbstractA central function of the brain is to plan, predict and imagine the effect of movement in a dynamically changing environment. Here we show that in mice head fixed in a plus-maze, floating on air, and trained to pick lanes based on visual stimuli, the asymmetric movement and position of whiskers on the two sides of the face signals whether the animal is moving, turning, expecting reward or licking. We show that 1) we can decode and predict the behavioral state of the animal based on this asymmetry, 2) that tactile input from whiskers indicates little about the behavioral state, and 3) that movement of the nose correlates with asymmetry, indicating that facial expression of the mouse is itself correlated with behavioral state. Amazingly, the movement of whiskers – a behavior that is not instructed or necessary in the task--informs an observer about what a mouse is doing in the maze. Thus, these mobile tactile sensors reflect a behavioral and movement-preparation state of the mouse.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/genomic_decoding_of_neuronal_depolarization_by_stimulus-specific_npas4_heterodimers_56397312/">
          <h4 class="font-weight-bold">Genomic decoding of neuronal depolarization by stimulus-specific NPAS4 heterodimers, bioRxiv, 2019-03-04</h4>
          <p>Cells regulate gene expression in response to salient external stimuli. In neurons, depolarization leads to the expression of inducible transcription factors (ITFs) that direct subsequent gene regulation. Depolarization encodes both neuronal action potential (AP) output and synaptic inputs, via excitatory postsynaptic potentials (EPSPs). However, it is unclear if different types of depolarizing signals can be transformed by an ITF into distinct modes of genomic regulation. Here, we show that APs and EPSPs in the murine hippocampus trigger two spatially segregated and molecularly distinct mechanisms that lead to the expression of the ITF NPAS4. These two pathways culminate in the assembly of unique, stimulus-specific NPAS4 heterodimers that exhibit distinctive DNA binding patterns. Thus, NPAS4 independently communicates increases in neuronal spiking output and synaptic inputs to the nucleus, enabling gene regulation to be tailored to the type of depolarizing activity experienced by a neuron.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/brain_aging_in_major_depressive_disorder_results_from_the_enigma_major_depressive_disorder_working_group_56073512/">
          <h4 class="font-weight-bold">Brain Aging in Major Depressive Disorder Results from the ENIGMA Major Depressive Disorder working group, bioRxiv, 2019-02-26</h4>
          <p>Background Major depressive disorder (MDD) is associated with an increased risk of brain atrophy, aging-related diseases, and mortality. We examined potential advanced brain aging in MDD patients, and whether this process is associated with clinical characteristics in a large multi-center international dataset. Methods We performed a mega-analysis by pooling brain measures derived from T1-weighted MRI scans from 29 samples worldwide. Normative brain aging was estimated by predicting chronological age (10-75 years) from 7 subcortical volumes, 34 cortical thickness and 34 surface area, lateral ventricles and total intracranial volume measures separately in 1,147 male and 1,386 female controls from the ENIGMA MDD working group. The learned model parameters were applied to 1,089 male controls and 1,167 depressed males, and 1,326 female controls and 2,044 depressed females to obtain independent unbiased brain-based age predictions. The difference between predicted brain age and chronological age was calculated to indicate brain predicted age difference (brain-PAD). Findings On average, MDD patients showed a higher brain-PAD of &#43;0.90 (SE 0.21) years (Cohen&#39;s d=0.12, 95% CI 0.06-0.17) compared to controls. Relative to controls, first-episode and currently depressed patients showed higher brain-PAD (&#43;1.2 [0.3] years), and the largest effect was observed in those with late-onset depression (&#43;1.7 [0.7] years). In addition, higher brain-PAD was associated with higher self-reported depressive symptomatology (b=0.05, p=0.004). Interpretation This highly powered collaborative effort showed subtle patterns of abnormal structural brain aging in MDD. Substantial within-group variance and overlap between groups were observed. Longitudinal studies of MDD and somatic health outcomes are needed to further assess the predictive value of these brain-PAD estimates.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/task-evoked_activity_quenches_neural_correlations_and_variability_in_large-scale_brain_systems_56050434/">
          <h4 class="font-weight-bold">Task-evoked activity quenches neural correlations and variability in large-scale brain systems, bioRxiv, 2019-02-26</h4>
          <p>Many studies of large-scale neural systems have emphasized the importance of communication through increased inter-region correlations (functional connectivity) during task states relative to resting state. In contrast, local circuit studies have demonstrated that task states reduce correlations among local neural populations, likely enhancing their information coding. Here we sought to adjudicate between these conflicting perspectives, assessing whether large-scale system correlations tend to increase or decrease during task states. To establish a mechanistic framework for interpreting changes in neural correlations, we conceptualized neural populations as having a sigmoidal neural transfer function. In a computational model we found that this straightforward assumption predicts reductions in neural populations&#39; dynamic output range as task-evoked activity levels increase, reducing responsiveness to inputs from other regions (i.e., reduced correlations). We demonstrated this empirically in large-scale neural populations across two highly distinct data sets human functional magnetic resonance imaging data and non-human primate spiking data. We found that task states increased global neural activity, while globally quenching neural variability and correlations. Further, this global reduction of neural correlations led to an overall increase in dimensionality (reflecting less information redundancy) during task states, providing an information-theoretic explanation for task-induced correlation reductions. Together, our results provide an integrative mechanistic account that encompasses measures of large-scale neural activity, variability, and correlations during resting and task states.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/neuroscience/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/neuroscience/page/7/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/neuroscience/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/neuroscience/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/neuroscience/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/neuroscience/page/7/" class="page-link">7</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/neuroscience/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/neuroscience/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/neuroscience/page/10/" class="page-link">10</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/neuroscience/page/11/" class="page-link">11</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/neuroscience/page/12/" class="page-link">12</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/neuroscience/page/9/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/neuroscience/page/29/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
