<!DOCTYPE html>
<html><head>
  
  <title>neuroscience | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.68.3" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.95d2483cd3c5d2bf47c1f36307a456787d60ad958c5efe44db9afc47012b1b05b8e1f64f6ac291650439322a6c38e86eb7cd6254255516d857c23f299fb0e666.js" integrity="sha512-ldJIPNPF0r9HwfNjB6RWeH1grZWMXv5E25r8RwErGwW44fZPasKRZQQ5MipsOOhut81iVCVVFthXwj8pn7DmZg=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_evolutionary_history_of_common_genetic_variants_influencing_human_cortical_surface_area_63648431/">
          <h4 class="font-weight-bold">The Evolutionary History of Common Genetic Variants Influencing Human Cortical Surface Area, bioRxiv, 2019-07-17</h4>
          <p>AbstractStructural brain changes along the lineage that led to modern Homo sapiens have contributed to our unique cognitive and social abilities. However, the evolutionarily relevant molecular variants impacting key aspects of neuroanatomy are largely unknown. Here, we integrate evolutionary annotations of the genome at diverse timescales with common variant associations from large-scale neuroimaging genetic screens in living humans, to reveal how selective pressures have shaped neocortical surface area. We show that variation within human gained enhancers active in the developing brain is associated with global surface area as well as that of specific regions. Moreover, we find evidence of recent polygenic selection over the past 2,000 years influencing surface area of multiple cortical regions, including those involved in spoken language and visual processing.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/striatal_activity_reflects_cortical_activity_patterns_63627968/">
          <h4 class="font-weight-bold">Striatal activity reflects cortical activity patterns, bioRxiv, 2019-07-16</h4>
          <p>The dorsal striatum is organized into domains that drive characteristic behaviors1–7, and receive inputs from different parts of the cortex8,9 which modulate similar behaviors10–12. Striatal responses to cortical inputs, however, can be affected by changes in connection strength13–15, local striatal circuitry16,17, and thalamic inputs18,19. Therefore, it is unclear whether the pattern of activity across striatal domains mirrors that across the cortex20–23 or differs from it24–28. Here we use simultaneous large-scale recordings in the cortex and the striatum to show that striatal activity can be accurately predicted by spatiotemporal activity patterns in the cortex. The relationship between activity in the cortex and the striatum was spatially consistent with corticostriatal anatomy, and temporally consistent with a feedforward drive. Each striatal domain exhibited specific sensorimotor responses that predictably followed activity in the associated cortical regions, and the corticostriatal relationship remained unvaried during passive states or performance of a task probing visually guided behavior. However, the task’s visual stimuli and corresponding behavioral responses evoked relatively more activity in the striatum than in associated cortical regions. This increased striatal activity involved an additive offset in firing rate, which was independent of task engagement but only present in animals that had learned the task. Thus, striatal activity largely reflects patterns of cortical activity, deviating from them in a simple additive fashion for learned stimuli or actions.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/structures_of_virus-like_capsids_formed_by_the_drosophila_neuronal_arc_proteins_63385121/">
          <h4 class="font-weight-bold">Structures of virus-like capsids formed by the Drosophila neuronal Arc proteins, bioRxiv, 2019-07-10</h4>
          <p>AbstractThe neuronal protein Arc is a critical mediator of synaptic plasticity. Arc originated in tetrapods and flies through domestication of retrotransposon Gag genes. Recent studies have suggested that Arc mediates intercellular mRNA transfer and like Gag, can form capsid-like structures. Here we report that Drosophila proteins dArc1 and dArc2 assemble virus-like capsids. We determine the capsid structures to 2.8 Å and 3.7 Å resolution, respectively, finding similarity to capsids of retroviruses and retrotransposons. Differences between dArc1 and dArc2 capsids, including the presence of a structured zinc-finger pair in dArc1, are consistent with differential RNA-binding specificity. Our data support a model in which ancestral capsid-forming and RNA-binding properties of Arc remain under positive selection pressure and have been repurposed to function in neuronal signalling.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/hierarchical_compression_reveals_sub-second_to_day-long_structure_in_larval_zebrafish_behaviour_63261654/">
          <h4 class="font-weight-bold">Hierarchical Compression Reveals Sub-Second to Day-Long Structure in Larval Zebrafish Behaviour, bioRxiv, 2019-07-08</h4>
          <p>AbstractAnimal behaviour is dynamic, evolving over multiple timescales from milliseconds to days and even across a lifetime. To understand the mechanisms governing these dynamics, it is necessary to capture multi-timescale structure from behavioural data. Here, we develop computational tools and study the behaviour of hundreds of larval zebrafish tracked continuously across multiple 24-hour daynight cycles. We extracted millions of movements and pauses, termed bouts, and used unsupervised learning to reduce each larva’s behaviour to an alternating sequence of active and inactive bout types, termed modules. Through hierarchical compression, we identified recurrent behavioural patterns, termed motifs. Module and motif usage varied across the daynight cycle, revealing structure at sub-second to day-long timescales. We further demonstrate that module and motif analysis can uncover novel pharmacological and genetic mutant phenotypes. Overall, our work reveals the organisation of larval zebrafish behaviour at multiple timescales and provides tools to identify structure from large-scale behavioural datasets.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/reconciling_dimensional_and_categorical_models_of_autism_heterogeneity_a_brain_connectomics__behavioral_study_63138882/">
          <h4 class="font-weight-bold">Reconciling Dimensional and Categorical Models of Autism Heterogeneity a Brain Connectomics &amp; Behavioral Study, bioRxiv, 2019-07-04</h4>
          <p>AbstractBackgroundHeterogeneity in autism spectrum disorder (ASD) has hindered the development of biomarkers, thus motivating subtyping efforts. Most subtyping studies divide ASD individuals into non-overlapping (categorical) subgroups. However, continuous inter-individual variation in ASD suggests the need for a dimensional approach.MethodsA Bayesian model was employed to decompose resting-state functional connectivity (RSFC) of ASD individuals into multiple abnormal RSFC patterns, i.e., categorical subtypes henceforth referred to as “factors”. Importantly, the model allowed each individual to express one or more factors to varying degrees (dimensional subtyping). The model was applied to 306 ASD individuals (age 5.2-57 years) from two multisite repositories. Posthoc analyses associated factors with symptoms and demographics.ResultsAnalyses yielded three factors with dissociable whole-brain hypohyper RSFC patterns. Most participants expressed multiple (categorical) factors, suggestive of a mosaic of subtypes within individuals. All factors shared abnormal RSFC involving the default network, but the directionality (hypohyper RSFC) differed across factors. Factor 1 was associated with core ASD symptoms, while factor 2 was associated with comorbid symptoms. Older males preferentially expressed factor 3. Factors were robust across multiple control analyses and not associated with IQ, nor head motion.ConclusionsThere exist at least three ASD factors with dissociable patterns of whole-brain RSFC, behaviors and demographics. Heterogeneous default network hypohyper RSFC across the factors might explain previously reported inconsistencies. The factors differentiated between core ASD and comorbid symptoms - a less appreciated domain of heterogeneity in ASD. These factors are co-expressed in ASD individuals with different degrees, thus reconciling categorical and dimensional perspectives of ASD heterogeneity.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/ninscope_a_versatile_miniscope_for_multi-region_circuit_investigations_62916136/">
          <h4 class="font-weight-bold">NINscope a versatile miniscope for multi-region circuit investigations, bioRxiv, 2019-07-01</h4>
          <p>AbstractMiniaturized fluorescence microscopes (miniscopes) have been instrumental to monitor neural activity during unrestrained behavior and their open-source versions have helped to distribute them at an affordable cost. Generally, the footprint and weight of open-source miniscopes is sacrificed for added functionality. Here, we present NINscope a light-weight, small footprint, open-source miniscope that incorporates a high-sensitivity image sensor, an inertial measurement unit (IMU), and an LED driver for an external optogenetic probe. We highlight the advantages of NINscope by performing the first simultaneous cellular resolution (dual scope) recordings from cerebellum and cerebral cortex in unrestrained mice, revealing that the activity of both regions generally precedes the onset of behavioral acceleration. We further demonstrate the optogenetic stimulation capabilities of NINscope and show that cerebral cortical activity can be driven strongly by cerebellar stimulation. To validate the performance of our miniscope to image from deep-brain regions, we recorded in the dorsal striatum and, using the IMU to assess turning movements, replicate previous studies that show encoding of action space in this subcortical region. Finally, we combine optogenetic stimulation of distinct cortical regions projecting to the dorsal striatum, to probe functional connectivity. In combination with cross-platform control software, NINscope is a versatile addition to the expanding toolbox of open-source miniscopes and will aid multi-region circuit investigations during unrestrained behavior.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/7/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/7/" class="page-link">7</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/10/" class="page-link">10</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/11/" class="page-link">11</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/12/" class="page-link">12</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/9/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/53/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
