<!DOCTYPE html>
<html><head>
  
  <title>Reports | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/teaching_r_in_the_undergraduate_ecology_classroom_approaches_lessons_learned_and_recommendations_61899821/">
          <h4 class="font-weight-bold">Teaching R in the undergraduate ecology classroom approaches, lessons learned, and recommendations, bioRxiv, 2019-06-11</h4>
          <p>AbstractEcology requires training in data management and analysis. In this paper, we present data from the last 10 years demonstrating the increase in the use of R, an open-source programming environment, in ecology and its prevalence as a required skill in job descriptions. Because of its transparent and flexible nature, R is increasingly used for data management and analysis in the field of ecology. Consequently, job postings targeting candidates with a bachelor’s degree and a required knowledge of R have increased over the past ten years. We discuss our experiences teaching undergraduates R in two advanced ecology classes using different approaches. One approach, in a course with a field lab, focused on collecting, cleaning, and preparing data for analysis. The other approach, in a course without a field lab, focused on analyzing existing data sets and applying the results to content discussed in the lecture portion of the course. Our experiences determined that each approach had strengths and weaknesses. We recommend that above all, instructors of ecology and related subjects should be encouraged to include R in their coursework. Furthermore, instructors should be aware of the following learning R is a separate skill from learning statistics; writing R assignments is a significant time sink for course preparation; and, there is a tradeoff between teaching R and teaching content. Determining how one’s course fits into the curriculum and identifying resources outside of the classroom for students’ continued practice will ensure that R training is successful and will extend beyond a one-semester course.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/scientific-communication-and-education">scientific-communication-and-education</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/wild_pollinator_activities_negatively_related_to_honey_bee_colony_densities_in_urban_context_61913412/">
          <h4 class="font-weight-bold">Wild pollinator activities negatively related to honey bee colony densities in urban context, bioRxiv, 2019-06-11</h4>
          <p>AbstractAs pollinator decline is increasingly reported in natural and agricultural environments, cities are perceived as shelters for pollinators because of low pesticide exposure and high floral diversity throughout the year. This has led to the development of environmental policies supporting pollinators in urban areas. However, policies are often restricted to the promotion of honey bee colony installations, which resulted in a strong increase in apiary numbers in cities. Recently, competition for floral resources between wild pollinators and honey bees has been highlighted in semi-natural contexts, but whether urban beekeeping could impact wild pollinators remains unknown. Here, we show that in the city of Paris (France), wild pollinator visitation rates is negatively correlated to honey bee colony densities present in the surrounding (500m – slope = −0.614; p = 0.001 – and 1000m – slope = −0.489; p = 0.005). More particularly, large solitary bees and beetles were significantly affected at 500m (respectively slope = −0.425, p = 0.007 and slope = - 0.671, p = 0.002) and bumblebees were significantly affected at 1000m (slope = - 0.451, p = 0.012). Further, lower interaction evenness in plant-pollinator networks was observed with honey bee colony densities within 1000 meter buffers (slope = −0.487, p = 0.008). Finally, honey bees tended to focus their foraging activity on managed rather than spontaneous plant species (student t-test, p = 0.001) whereas wild pollinators equally visited managed and spontaneous species. We advocate responsible practices mitigating the introduction of high density of hives in urban environments. Future studies are needed to deepen our knowledge about the potential negative interactions between wild and domesticated pollinators.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_robust_benchmark_for_germline_structural_variant_detection_61877665/">
          <h4 class="font-weight-bold">A robust benchmark for germline structural variant detection, bioRxiv, 2019-06-10</h4>
          <p>AbstractNew technologies and analysis methods are enabling genomic structural variants (SVs) to be detected with ever-increasing accuracy, resolution, and comprehensiveness. Translating these methods to routine research and clinical practice requires robust benchmark sets. We developed the first benchmark set for identification of both false negative and false positive germline SVs, which complements recent efforts emphasizing increasingly comprehensive characterization of SVs. To create this benchmark for a broadly consented son in a Personal Genome Project trio with broadly available cells and DNA, the Genome in a Bottle (GIAB) Consortium integrated 19 sequence-resolved variant calling methods, both alignment- and de novo assembly-based, from short-, linked-, and long-read sequencing, as well as optical and electronic mapping. The final benchmark set contains 12745 isolated, sequence-resolved insertion and deletion calls ≥50 base pairs (bp) discovered by at least 2 technologies or 5 callsets, genotyped as heterozygous or homozygous variants by long reads. The Tier 1 benchmark regions, for which any extra calls are putative false positives, cover 2.66 Gbp and 9641 SVs supported by at least one diploid assembly. Support for SVs was assessed using svviz with short-, linked-, and long-read sequence data. In general, there was strong support from multiple technologies for the benchmark SVs, with 90 % of the Tier 1 SVs having support in reads from more than one technology. The Mendelian genotype error rate was 0.3 %, and genotype concordance with manual curation was &amp;gt;98.7 %. We demonstrate the utility of the benchmark set by showing it reliably identifies both false negatives and false positives in high-quality SV callsets from short-, linked-, and long-read sequencing and optical mapping.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/optogenetic_control_of_wnt_signaling_for_modeling_early_embryogenic_patterning_with_human_pluripotent_stem_cells_61878281/">
          <h4 class="font-weight-bold">Optogenetic control of Wnt signaling for modeling early embryogenic patterning with human pluripotent stem cells, bioRxiv, 2019-06-10</h4>
          <p>ABSTRACTThe processes of cell proliferation, differentiation, migration, and self-organization during early embryonic development are governed by dynamic, spatially and temporally varying morphogen signals. Analogous tissue patterns emerge spontaneously in embryonic stem cell (ESC) models for gastrulation, but mechanistic insight into this self-organization is limited by a lack of molecular methods to precisely control morphogen signal dynamics. Here we combine optogenetic stimulation and single-cell imaging approaches to study self-organization of human pluripotent stem cells. Precise control of morphogen signal dynamics, achieved through activation of canonical Wntβ-catenin signaling over a broad high dynamic range (&amp;gt;500-fold) using an optoWnt optogenetic system, drove broad transcriptional changes and mesendoderm differentiation of human ESCs at high efficiency (&amp;gt;95% cells). Furthermore, activating Wnt signaling in subpopulations of ESCs in 2D and 3D cultures induced cell self-organization and morphogenesis reminiscent of human gastrulation, including changes in cell migration and epithelial to mesenchymal transition. Our findings thus reveal an instructive role for Wnt in directing cell patterning in this ESC model for gastrulation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioengineering">bioengineering</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/single-cell_mass-spectrometry_quantifies_the_emergence_of_macrophage_heterogeneity_61883494/">
          <h4 class="font-weight-bold">Single-cell mass-spectrometry quantifies the emergence of macrophage heterogeneity, bioRxiv, 2019-06-10</h4>
          <p>The fate and physiology of individual cells are controlled by protein interactions. Yet, our ability to quantitatively analyze proteins in single cells has remained limited. To overcome this barrier, we developed SCoPE2. It lowers cost and hands-on time by introducing automated and miniaturized sample preparation while substantially increasing quantitative accuracy. These advances enabled us to analyze the emergence of cellular heterogeneity as homogeneous monocytes differentiated into macrophage-like cells in the absence of polarizing cytokines. SCoPE2 quantified over 2,700 proteins in 1,018 single monocytes and macrophages in ten days of instrument time, and the quantified proteins allowed us to discern single cells by cell type. Furthermore, the data uncovered a continuous gradient of proteome states for the macrophage-like cells, suggesting that macrophage heterogeneity may emerge even in the absence of polarizing cytokines. Parallel measurements of transcripts by 10x Genomics scRNA-seq suggest that SCoPE2 samples 20-fold more copies per gene, thus supporting quantification with improved count statistics. Joint analysis of the data indicated that most genes had similar responses at the protein and RNA levels, though the responses of hundreds of genes differed. Our methodology lays the foundation for automated and quantitative single-cell analysis of proteins by mass-spectrometry.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/clij_gpu-accelerated_image_processing_for_everyone_61880256/">
          <h4 class="font-weight-bold">CLIJ GPU-accelerated image processing for everyone, bioRxiv, 2019-06-09</h4>
          <p>AbstractGraphics processing units (GPU) allow image processing at unprecedented speed. We present CLIJ, a Fiji plugin enabling end-users with entry level experience in programming to benefit from GPU-accelerated image processing. Freely programmable workflows can speed up image processing in Fiji by factor 10 and more using high-end GPU hardware and on affordable mobile computers with built-in GPUs.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/reports/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/reports/page/57/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/54/" class="page-link">54</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/55/" class="page-link">55</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/56/" class="page-link">56</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/57/" class="page-link">57</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/reports/page/58/" class="page-link">58</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/59/" class="page-link">59</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/60/" class="page-link">60</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/61/" class="page-link">61</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/62/" class="page-link">62</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/reports/page/59/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/reports/page/320/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
