<!DOCTYPE html>
<html><head>
  
  <title>Biorxiv | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/mammary_mechanobiology_mechanically-activated_ion_channels_in_lactation_and_involution_61136380/">
          <h4 class="font-weight-bold">Mammary mechanobiology mechanically-activated ion channels in lactation and involution, bioRxiv, 2019-05-27</h4>
          <p>AbstractA mother’s ability to produce a nutritionally-complete neonatal food source has provided a powerful evolutionary advantage to mammals. Milk production by secretory mammary epithelial cells is adaptive, its release is exquisitely timed and its own glandular stagnation with the permanent cessation of suckling triggers the programmed cell death and tissue remodeling that enables female mammals to nurse successive progeny. Both chemical and mechanical signals control epithelial expansion, function and remodeling. Despite this duality of input, however, the nature and function of mechanical forces in the mammary gland remain unknown. Here, we characterize the mammary force landscape and the capacity of luminal and basal epithelial cells to experience and exert force. We explore the molecular instruments for force-sensing in the mammary gland and the physiological requirement for PIEZO1 in lactation and involution. Our study supports the existence of a multifaceted system of chemical and mechanical sensing in the mammary gland, and a protective redundancy that ensures continued lactational competence and offspring survival.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/population_history_and_genetic_adaptation_of_the_fulani_nomads_inferences_from_genome-wide_data_and_the_lactase_persistence_trait_61155493/">
          <h4 class="font-weight-bold">Population history and genetic adaptation of the Fulani nomads Inferences from genome-wide data and the lactase persistence trait, bioRxiv, 2019-05-27</h4>
          <p>AbstractHuman population history in the Holocene was profoundly impacted by changes in lifestyle following the invention and adoption of food-production practices. These changes triggered significant increases in population sizes and expansions over large distances. Here we investigate the population history of the Fulani, a pastoral population extending throughout the African SahelSavannah belt. Based on genome-wide analyses we propose that ancestors of the Fulani population experienced admixture between a West African group and a group carrying both European and North African ancestries. This admixture was likely coupled with newly adopted herding practices, as it resulted in signatures of genetic adaptation in contemporary Fulani genomes, including the control element of the LCT gene enabling carriers to digest lactose throughout their lives. The lactase persistence (LP) trait in the Fulani is conferred by the presence of the allele T-13910, which is also present at high frequencies in Europe. We establish that the T-13910 LP allele in Fulani individuals analysed in this study lies on a European haplotype background thus excluding parallel convergent evolution. Our findings further suggest that Eurasian admixture and the European LP allele was introduced into the Fulani through contact with a North African populations. We furthermore confirm the link between the lactose digestion phenotype in the Fulani to the MCM6LCT locus by reporting the first Genome Wide Association study (GWAS) of the lactase persistence trait. We also further explored signals of recent adaptation in the Fulani and identified additional candidates for selection to adapt to herding life-styles.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_advantages_and_disadvantages_of_short-_and_long-read_metagenomics_to_infer_bacterial_and_eukaryotic_community_composition_61155939/">
          <h4 class="font-weight-bold">The advantages and disadvantages of short- and long-read metagenomics to infer bacterial and eukaryotic community composition, bioRxiv, 2019-05-27</h4>
          <p>AbstractBackgroundThe first step in understanding ecological community diversity and dynamics is quantifying community membership. An increasingly common method for doing so is through metagenomics. Because of the rapidly increasing popularity of this approach, a large number of computational tools and pipelines are available for analysing metagenomic data. However, the majority of these tools have been designed and benchmarked using highly accurate short read data (i.e. illumina), with few studies benchmarking classification accuracy for long error-prone reads (PacBio or Oxford Nanopore). In addition, few tools have been benchmarked for non-microbial communities.ResultsHere we use simulated error prone Oxford Nanopore and high accuracy Illumina read sets to systematically investigate the effects of sequence length and taxon type on classification accuracy for metagenomic data from both microbial and non-microbial communities. We show that very generally, classification accuracy is far lower for non-microbial communities, even at low taxonomic resolution (e.g. family rather than genus).ConclusionsWe then show that for two popular taxonomic classifiers, long error-prone reads can significantly increase classification accuracy, and this is most pronounced for non-microbial communities. This work provides insight on the expected accuracy for metagenomic analyses for different taxonomic groups, and establishes the point at which read length becomes more important than error rate for assigning the correct taxon.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/species-specific_oscillation_periods_of_human_and_mouse_segmentation_clocks_are_due_to_cell_autonomous_differences_in_biochemical_reaction_parameters_61120294/">
          <h4 class="font-weight-bold">Species-specific oscillation periods of human and mouse segmentation clocks are due to cell autonomous differences in biochemical reaction parameters, bioRxiv, 2019-05-26</h4>
          <p>AbstractWhile the mechanisms of embryonic development are similar between mouse and human, the tempo is in general slower in human. The cause of interspecies differences in developmental time remains a mystery partly due to lack of an appropriate model system1. Since murine and human embryos differ in their sizes, geometries, and nutrients, we use in vitro differentiation of pluripotent stem cells (PSCs) to compare the same type of cells between the species in similar culture conditions. As an example of well-defined developmental time, we focus on the segmentation clock, oscillatory gene expression that regulates the timing of sequential formation of body segments2–4. In this way we recapitulate the murine and human segmentation clocks in vitro, showing that the species-specific oscillation periods are derived from cell autonomous differences in the speeds of biochemical reactions. Presomitic mesoderm (PSM)-like cells induced from murine and human PSCs displayed the oscillatory expression of HES7, the core gene of the segmentation clock5,6, with oscillation periods of 2-3 hours (mouse PSM) and 5-6 hours (human PSM). Swapping HES7 loci between murine and human genomes did not change the oscillation periods dramatically, denying the possibility that interspecies differences in the sequences of HES7 loci might be the cause of the observed period difference. Instead, we found that the biochemical reactions that determine the oscillation period, such as the degradation of HES7 and delays in its expression, are slower in human PSM compared with those in mouse PSM. With the measured biochemical parameters, our mathematical model successfully accounted for the 2-3-fold period difference between mouse and human. We further demonstrate that the concept of slower biochemical reactions in human cells is generalizable to several other genes, as well as to another cell type. These results collectively indicate that differences in the speeds of biochemical reactions between murine and human cells give rise to the interspecies period difference of the segmentation clock and may contribute to other interspecies differences in developmental time.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_shared_genetic_basis_for_personality_traits_and_local_cortical_grey_matter_structure_61066707/">
          <h4 class="font-weight-bold">A shared genetic basis for personality traits and local cortical grey matter structure?, bioRxiv, 2019-05-25</h4>
          <p>AbstractPersonality traits are key indices of inter-individual variation. Personality is heritable and has been associated with brain structure and function. To date, it is unknown whether the relation between personality and brain macrostructure can be explained by genetic factors. In a large-scale twin sample (Human Connectome Project), we performed genetic correlation analyses to evaluate whether personality traits (NEO-FFI) and local brain structure have a shared genetic basis. We found a genetic overlap between personality traits and local brain structure in 11 of 22 observed phenotypic associations in predominantly frontal cortices. In these regions the proportion of phenotypic covariance accounted for by shared genetic effects was between 82 and 100%. Second, in the case of Agreeableness, Conscientiousness, and Openness, the phenotypic correlation between personality and local brain structure was observed to reflect genetic, more than environmental, factors. These observations indicate that genetic factors influence the relationship between personality traits and local brain structure. Importantly, observed associations between personality traits and cortical thickness did only partially replicate in two independent large-scale samples of unrelated individuals. Taken together, our findings demonstrate that genes impact the relationship between personality and local brain structure, but that phenotypic associations are, to a large extent, non-generalizable. These observations provide a novel perspective on the nature and nurture of the biological basis of personality.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/algorithms_for_efficiently_collapsing_reads_with_unique_molecular_identifiers_61056743/">
          <h4 class="font-weight-bold">Algorithms for efficiently collapsing reads with Unique Molecular Identifiers, bioRxiv, 2019-05-25</h4>
          <p>AbstractBackgroundUnique Molecular Identifiers (UMI) are used in many experiments to find and remove PCR duplicates. Although there are many tools for solving the problem of deduplicating reads based on their finding reads with the same alignment coordinates and UMIs, many tools either cannot handle substitution errors, or require expensive pairwise UMI comparisons that do not efficiently scale to larger datasets.ResultsWe formulate the problem of deduplicating UMIs in a manner that enables optimizations to be made, and more efficient data structures to be used. We implement our data structures and optimizations in a tool called UMICollapse, which is able to deduplicate over one million unique UMIs of length 9 at a single alignment position in around 26 seconds.ConclusionsWe present a new formulation of the UMI deduplication problem, and show that it can be solved faster, with more sophisticated data structures.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/biorxiv/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/biorxiv/page/62/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../../tags/biorxiv/page/59/" class="page-link">59</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/biorxiv/page/60/" class="page-link">60</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/biorxiv/page/61/" class="page-link">61</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/biorxiv/page/62/" class="page-link">62</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/biorxiv/page/63/" class="page-link">63</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/biorxiv/page/64/" class="page-link">64</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/biorxiv/page/65/" class="page-link">65</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/biorxiv/page/66/" class="page-link">66</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/biorxiv/page/67/" class="page-link">67</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/biorxiv/page/64/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/biorxiv/page/312/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
