<!DOCTYPE html>
<html><head>
  
  <title>2018 | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/increased_frequency_of_travel_in_the_presence_of_cross-immunity_may_act_to_decrease_the_chance_of_a_global_pandemic_47444563/">
          <h4 class="font-weight-bold">Increased frequency of travel in the presence of cross-immunity may act to decrease the chance of a global pandemic, bioRxiv, 2018-08-31</h4>
          <p>The high frequency of modern travel has led to concerns about a devastating pandemic since a lethal pathogen strain could spread worldwide quickly. Many historical pandemics have arisen following pathogen evolution to a more virulent form. However, some pathogen strains invoke immune responses that provide partial cross-immunity against infection with related strains. Here, we consider a mathematical model of successive outbreaks of two strains a low virulence strain outbreak followed by a high virulence strain outbreak. Under these circumstances, we investigate the impacts of varying travel rates and cross-immunity on the probability that a major epidemic of the high virulence strain occurs, and the size of that outbreak. Frequent travel between subpopulations can lead to widespread immunity to the high virulence strain, driven by exposure to the low virulence strain. As a result, major epidemics of the high virulence strain are less likely, and can potentially be smaller, with more connected subpopulations. Cross-immunity may be a factor contributing to the absence of a global pandemic as severe as the 1918 influenza pandemic in the century since.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/epidemiology">epidemiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/re-evaluating_one-step_generation_of_mice_carrying_conditional_alleles_by_crispr-cas9-mediated_genome_editing_technology_47288281/">
          <h4 class="font-weight-bold">Re-Evaluating One-step Generation of Mice Carrying Conditional Alleles by CRISPR-Cas9-Mediated Genome Editing Technology, bioRxiv, 2018-08-31</h4>
          <p>AbstractCRISPR-Cas9 gene editing technology has considerably facilitated the generation of mouse knockout alleles, relieving many of the cumbersome and time-consuming steps of traditional mouse embryonic stem cell technology. However, the generation of conditional knockout alleles remains an important challenge. An earlier study reported up to 16% efficiency in generating conditional knockout alleles in mice using 2 single guide RNAs (sgRNA) and 2 single-stranded oligonucleotides (ssODN) (2sgRNA-2ssODN). We re-evaluated this method from a large data set generated from a consortium consisting of 17 transgenic core facilities or laboratories or programs across the world. The dataset constituted 17,887 microinjected or electroporated zygotes and 1,718 live born mice, of which only 15 (0.87%) mice harbored 2 correct LoxP insertions in cis configuration indicating a very low efficiency of the method. To determine the factors required to successfully generate conditional alleles using the 2sgRNA-2ssODN approach, we performed a generalized linear regression model. We show that factors such as the concentration of the sgRNA, Cas9 protein or the distance between the placement of LoxP insertions were not predictive for the success of this technique. The major predictor affecting the method’s success was the probability of simultaneously inserting intact proximal and distal LoxP sequences, without the loss of the DNA segment between the two sgRNA cleavage sites. Our analysis of a large data set indicates that the 2sgRNA–2ssODN method generates a large number of undesired alleles (&amp;gt;99%), and a very small number of desired alleles (&amp;lt;1%) requiring, on average 1,192 zygotes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/gender_and_international_diversity_improves_equity_in_peer_review_47238859/">
          <h4 class="font-weight-bold">Gender and international diversity improves equity in peer review, bioRxiv, 2018-08-29</h4>
          <p>AbstractThe robustness of scholarly peer review has been challenged by evidence of disparities in publication outcomes based on author gender and nationality. To address this, we examined the peer review outcomes of 23,876 initial submissions and 7,192 full submissions that were submitted to the biosciences journal eLife between 2012 and 2017. Women and authors from nations outside of North America and Europe were underrepresented both as gatekeepers (editors and peer reviewers) and authors. We found evidence of a homophilic relationship between the demographics of the gatekeepers and authors in determining the outcome of peer review; that is, gatekeepers favored manuscripts from authors of the same gender and from the same country. The acceptance rate for manuscripts with male last authors was higher than for female last authors, and this gender inequity was greatest when the team of reviewers was all male; mixed-gender gatekeeper teams lead to more equitable peer review outcomes. Homogeny between the country affiliation of the gatekeeper and the corresponding author also lend to improved acceptance rates for many countries. We conclude with a discussion of mechanisms that could contribute to this effect, directions for future research, and policy implications. Code and anonymized data have been made available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsgithub.commurraydselife-analysis&gt;httpsgithub.commurraydselife-analysis&lt;jatsext-link&gt;Author summaryPeer review, the primary method by which scientific work is evaluated and developed, is ideally a fair and equitable process in which scientific work is judged solely on its own merit. However, the integrity of peer review has been called into question based on evidence that outcomes often differ between between male and female authors, and for authors in different countries. We investigated such a disparity at the biosciences journal eLife by analyzing the demographics of authors and gatekeepers (editors and peer reviewers), and peer review outcomes of all submissions between 2012 and 2017. We found evidence of disparity in outcomes that favored women and those affiliated within North America and Europe, and that these groups were over-represented among authors and gatekeepers. The gender disparity was greatest when reviewers were all male; mixed-gender reviewer teams lead to more equitable outcomes. Similarly, for some countries manuscripts were more likely to be accepted when reviewed by a gatekeeper from the same country as the author. Our results indicate that author and gatekeeper characteristics are associated with the outcomes of scientific peer review. We discuss mechanisms that could contribute to this effect, directions for future research, and policy implications.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/scientific-communication-and-education">scientific-communication-and-education</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/high-throughput_single-cell_transcriptome_profiling_of_plant_cell_types_47225996/">
          <h4 class="font-weight-bold">High-throughput single-cell transcriptome profiling of plant cell types, bioRxiv, 2018-08-29</h4>
          <p>AbstractSingle-cell transcriptome analysis of heterogeneous tissues can provide high-resolution windows into the genomic basis and spatiotemporal dynamics of developmental processes. Here we demonstrate the feasibility of high-throughput single-cell RNA sequencing of plant tissue using the Drop-seq approach. Profiling of &amp;gt;4,000 individual cells from the Arabidopsis root provides transcriptomes and marker genes for a diversity of cell types and illuminates the gene expression changes that occur across endodermis development.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/lineage_calling_can_identify_antibiotic_resistant_clones_within_minutes_47237654/">
          <h4 class="font-weight-bold">Lineage calling can identify antibiotic resistant clones within minutes, bioRxiv, 2018-08-29</h4>
          <p>Introductory ParagraphSurveillance of circulating drug resistant bacteria is essential for healthcare providers to deliver effective empiric antibiotic therapy. However, the results of surveillance may not be available on a timescale that is optimal for guiding patient treatment. Here we present a method for inferring characteristics of an unknown bacterial sample by identifying the presence of sequence variation across the genome that is linked to a phenotype of interest, in this case drug resistance. We demonstrate an implementation of this principle using sequence k-mer content, matched to a database of known genomes. We show this technique can be applied to data from an Oxford Nanopore device in real time and is capable of identifying the presence of a known resistant strain in 5 minutes, even from a complex metagenomic sample. This flexible approach has wide application to pathogen surveillance and may be used to greatly accelerate diagnoses of resistant infections.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/high-throughput_amplicon_sequencing_of_the_full-length_16s_rrna_gene_with_single-nucleotide_resolution_46552121/">
          <h4 class="font-weight-bold">High-throughput amplicon sequencing of the full-length 16S rRNA gene with single-nucleotide resolution, bioRxiv, 2018-08-16</h4>
          <p>AbstractTargeted PCR amplification and high-throughput sequencing (amplicon sequencing) of 16S rRNA gene fragments is widely used to profile microbial communities. New long-read sequencing technologies can sequence the entire 16S rRNA gene, but higher error rates have limited their attractiveness when accuracy is important. Here we present a high-throughput amplicon sequencing methodology based on PacBio circular consensus sequencing and the DADA2 sample inference method that measures the full-length 16S rRNA gene with single-nucleotide resolution and a near-zero error rate.In two artificial communities of known composition, our method recovered the full complement of full-length 16S sequence variants from expected community members without residual errors. The measured abundances of intra-genomic sequence variants were in the integral ratios expected from the genuine allelic variants within a genome. The full-length 16S gene sequences recovered by our approach allowed E. coli strains to be correctly classified to the O157H7 and K12 sub-species clades. In human fecal samples, our method showed strong technical replication and was able to recover the full complement of 16S rRNA alleles in several E. coli strains.There are likely many applications beyond microbial profiling for which high-throughput amplicon sequencing of complete genes with single-nucleotide resolution will be of use.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/2018/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/2018/page/42/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/2018/page/39/" class="page-link">39</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2018/page/40/" class="page-link">40</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2018/page/41/" class="page-link">41</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2018/page/42/" class="page-link">42</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/2018/page/43/" class="page-link">43</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2018/page/44/" class="page-link">44</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2018/page/45/" class="page-link">45</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2018/page/46/" class="page-link">46</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2018/page/47/" class="page-link">47</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/2018/page/44/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/2018/page/54/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
