<!DOCTYPE html>
<html><head>
  
  <title>Reports | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/deep_learning_at_base-resolution_reveals_motif_syntax_of_the_cis-regulatory_code_65278904/">
          <h4 class="font-weight-bold">Deep learning at base-resolution reveals motif syntax of the cis-regulatory code, bioRxiv, 2019-08-22</h4>
          <p>AbstractGenes are regulated through enhancer sequences, in which transcription factor binding motifs and their specific arrangements (syntax) form a cis-regulatory code. To understand the relationship between motif syntax and transcription factor binding, we train a deep learning model that uses DNA sequence to predict base-resolution binding profiles of four pluripotency transcription factors Oct4, Sox2, Nanog, and Klf4. We interpret the model to accurately map hundreds of thousands of motifs in the genome, learn novel motif representations and identify rules by which motifs and syntax influence transcription factor binding. We find that instances of strict motif spacing are largely due to retrotransposons, but that soft motif syntax influences motif interactions at protein and nucleosome range. Most strikingly, Nanog binding is driven by motifs with a strong preference for ∼10.5 bp spacings corresponding to helical periodicity. Interpreting deep learning models applied to high-resolution binding data is a powerful and versatile approach to uncover the motifs and syntax of cis-regulatory sequences.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_curated_database_reveals_trends_in_single-cell_transcriptomics_65259847/">
          <h4 class="font-weight-bold">A curated database reveals trends in single-cell transcriptomics, bioRxiv, 2019-08-21</h4>
          <p>The more than 500 single-cell transcriptomics studies that have been published to date constitute a valuable and vast resource for biological discovery. While various “atlas” projects have collated some of the associated datasets, most questions related to specific tissue types, species, or other attributes of studies require identifying papers through manual and challenging literature search. To facilitate discovery with published single-cell transcriptomics data, we have assembled a near exhaustive, manually curated database of single-cell transcriptomics studies with key information descriptions of the type of data and technologies used, along with descriptors of the biological systems studied. Additionally, the database contains summarized information about analysis in the papers, allowing for analysis of trends in the field. As an example, we show that the number of cell types identified in scRNA-seq studies is proportional to the number of cells analysed. The database is available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpswww.nxn.sesingle-cell-studiesgui&gt;www.nxn.sesingle-cell-studiesgui&lt;jatsext-link&gt;.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/global_adaptation_confounds_the_search_for_local_adaptation_65264544/">
          <h4 class="font-weight-bold">Global adaptation confounds the search for local adaptation, bioRxiv, 2019-08-21</h4>
          <p>AbstractSpatially varying selection promotes variance in allele frequencies, increasing genetic differentiation between the demes of a metapopulation. For that reason, outliers in the genome wide distribution of summary statistics measuring genetic differentiation, such as FST, are often interpreted as evidence for alleles which contribute to local adaptation. However, in spatially structured populations, the spread of beneficial mutations with spatially uniform effects can also induce transient genetic differentiation and numerous theoretical studies have suggested that species-wide, or global, adaptation makes a substantial contribution to molecular evolution. In this study, we ask whether such global adaptation affects the genome-wide distribution of FST and generates statistical outliers which could be mistaken for local adaptation. Using forward-in-time population genetic simulations assuming parameters for the rate and strength of beneficial mutations similar to those that have been estimated for natural populations, we show the spread of globally beneficial in parapatric populations can readily generate FST outliers, which may be misinterpreted as evidence for local adaptation. The spread of beneficial mutations causes selective sweeps at flanking sites, so the effects of global versus local adaptation may be distinguished by examining patterns of nucleotide diversity along with FST. Our study suggests that global adaptation should be considered in the interpretation of genome-scan results and the design of future studies aimed at understanding the genetic basis of local adaptation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/atlas_a_snakemake_workflow_for_assembly_annotation_and_genomic_binning_of_metagenome_sequence_data_65212890/">
          <h4 class="font-weight-bold">ATLAS a Snakemake workflow for assembly, annotation, and genomic binning of metagenome sequence data, bioRxiv, 2019-08-20</h4>
          <p>AbstractBackgroundMetagenomics and metatranscriptomics studies provide valuable insight into the composition and function of microbial populations from diverse environments, however the data processing pipelines that rely on mapping reads to gene catalogs or genome databases for cultured strains yield results that underrepresent the genes and functional potential of uncultured microbes. Recent improvements in sequence assembly methods have eased the reliance on genome databases, thereby allowing the recovery of genomes from uncultured microbes. However, configuring these tools, linking them with advanced binning and annotation tools, and maintaining provenance of the processing continues to be challenging for researchers.ResultsHere we present ATLAS, a software package for customizable data processing from raw sequence reads to functional and taxonomic annotations using state-of-the-art tools to assemble, annotate, quantify, and bin metagenome and metatranscriptome data. Genome-centric resolution and abundance estimates are provided for each sample in a dataset. ATLAS is written in Python and the workflow implemented in Snakemake; it operates in a Linux environment, and is compatible with Python 3.5&#43; and Anaconda 3&#43; versions. The source code for ATLAS is freely available, distributed under a BSD-3 license.ConclusionATLAS provides a user-friendly, modular and customizable Snakemake workflow for metagenome and metatranscriptome data processing; it is easily installable with conda and maintained as open-source on GitHub at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsgithub.commetagenome-atlasatlas&gt;httpsgithub.commetagenome-atlasatlas&lt;jatsext-link&gt;.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_single-cell_transcriptome_atlas_for_zebrafish_development_65164614/">
          <h4 class="font-weight-bold">A Single-Cell Transcriptome Atlas for Zebrafish Development, bioRxiv, 2019-08-19</h4>
          <p>ABSTRACTThe ability to define cell types and how they change during organogenesis is central to our understanding of animal development and human disease. Despite the crucial nature of this knowledge, we have yet to fully characterize all distinct cell types and the gene expression differences that generate cell types during development. To address this knowledge gap, we produced an Atlas using single-cell RNA-sequencing methods to investigate gene expression from the pharyngula to early larval stages in developing zebrafish. Our single-cell transcriptome Atlas encompasses transcriptional profiles from 44,102 cells across four days of development using duplicate experiments that confirmed high reproducibility. We annotated 220 identified clusters and highlighted several strategies for interrogating changes in gene expression associated with the development of zebrafish embryos at single-cell resolution. Furthermore, we highlight the power of this analysis to assign new cell-type or developmental stage-specific expression information to many genes, including those that are currently known only by sequence andor that lack expression information altogether. The resulting Atlas is a resource of biologists to generate hypotheses for genetic (mutant) or functional analysis, to launch an effort to define the diversity of cell-types during zebrafish organogenesis, and to examine the transcriptional profiles that produce each cell type over developmental time.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/prevalence_estimate_of_blood_doping_in_elite_track_and_field_at_the_introduction_of_the_athlete_biological_passport_65166375/">
          <h4 class="font-weight-bold">Prevalence estimate of blood doping in elite track and field at the introduction of the Athlete Biological Passport, bioRxiv, 2019-08-19</h4>
          <p>AbstractIn elite sport, the Athlete Biological Passport (ABP) was invented to tackle cheaters by monitoring closely changes in biological parameters, flagging atypical variations. The haematological module of the ABP was indeed adopted in 2011 by the International Association of Athletics Federations (IAAF). This study estimates the prevalence of blood doping based on haematological parameters in a large cohort of track &amp;amp; field athletes measured at two international major events (2011 &amp;amp; 2013 IAAF World Championships) with a hypothesized decrease in prevalence due to the ABP introduction.A total of 3683 blood samples were collected and analysed from all participating athletes originating from 209 countries. The estimate of doping prevalence was obtained by using a Bayesian network with seven variables, as well as “doping” as a variable mimicking doping with low-doses of recombinant human erythropoietin (rhEPO), to generate reference cumulative distribution functions (CDFs) for the Abnormal Blood Profile Score (ABPS) from the ABP.Our results from robust haematological parameters indicate an estimation of an overall blood doping prevalence of 18% in average in endurance athletes (95% Confidence Interval (C.I.) 14-22%). A higher prevalence was observed in female athletes (22%, C.I. 16-28%) than in male athletes (15%, C.I. 9-20%). In conclusion, this study presents the first comparison of blood doping prevalence in elite athletes based on biological measurements from major international events that may help scientists and experts to use the ABP in a more efficient and deterrent way.What are the new findings ?&lt;jatslist list-type=bullet&gt;&lt;jatslist-item&gt;This study presents the first comparison of blood doping prevalence in elite track &amp;amp; field athletes based on biological measurements from major international events&lt;jatslist-item&gt;&lt;jatslist-item&gt;Our results from robust haematological parameters indicate an estimation of an overall blood doping prevalence of 18% in average in endurance athletes.&lt;jatslist-item&gt;&lt;jatslist-item&gt;The confidence intervals for blood doping prevalence range from 9-28% with wide discrepancies between certain countries.&lt;jatslist-item&gt;How might it impact on clinical practice in the near future&lt;jatslist list-type=bullet&gt;&lt;jatslist-item&gt;The further development of the Athlete Biological Passport with a careful monitoring of biological parameters still represents the most consistent approach to thwart athletes using undetectable prohibited substances or methods.&lt;jatslist-item&gt;&lt;jatslist-item&gt;This study describes a method to define blood doping prevalence with the analysis of robust haematological parameters&lt;jatslist-item&gt;&lt;jatslist-item&gt;Estimates of doping prevalence in subpopulations of athletes may represent a valuable tool for the antidoping authorities to perform a risk assessment in their sport.&lt;jatslist-item&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/physiology">physiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/reports/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/reports/page/31/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/28/" class="page-link">28</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/29/" class="page-link">29</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/30/" class="page-link">30</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/31/" class="page-link">31</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/reports/page/32/" class="page-link">32</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/33/" class="page-link">33</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/34/" class="page-link">34</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/35/" class="page-link">35</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/36/" class="page-link">36</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/reports/page/33/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/reports/page/320/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
