<!DOCTYPE html>
<html><head>
  
  <title>2019 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/peptide-based_quorum_sensing_systems_in_paenibacillus_polymyxa_66520456/">
          <h4 class="font-weight-bold">Peptide-based quorum sensing systems in Paenibacillus polymyxa, bioRxiv, 2019-09-13</h4>
          <p>AbstractPaenibacillus polymyxa is an agriculturally important plant growth-promoting rhizobacterium. Many Paenibacillus species are known to be engaged in complex bacteria-bacteria and bacteria-host interactions, which in other species were shown to necessitate quorum sensing communication. However, to date no quorum sensing systems have been described in Paenibacillus. Here we show that the type strain P. polymyxa ATCC 842 encodes at least 16 peptide-based communication systems. Each of these systems is comprised of a pro-peptide that is secreted to the growth medium and processed to generate a mature short peptide. Each peptide has a cognate intracellular receptor of the RRNPP family, and we show that external addition of P. polymyxa communication peptides leads to reprogramming of the transcriptional response. We found that these quorum sensing systems are conserved across hundreds of species belonging to the Paenibacillaceae family, with some species encoding more than 25 different peptide-receptor pairs, representing a record number of quorum sensing systems encoded in a single genome.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/ancient_dna_reconstructs_the_genetic_legacies_of_pre-contact_puerto_rico_communities_66507190/">
          <h4 class="font-weight-bold">Ancient DNA reconstructs the genetic legacies of pre-contact Puerto Rico communities, bioRxiv, 2019-09-12</h4>
          <p>AbstractIndigenous peoples have occupied the island of Puerto Rico since at least 3000 B.C. Due to the demographic shifts that occurred after European contact, the origin(s) of these ancient populations, and their genetic relationship to present-day islanders, are unclear. We use ancient DNA to characterize the population history and genetic legacies of pre-contact Indigenous communities from Puerto Rico. Bone, tooth and dental calculus samples were collected from 124 individuals from three pre-contact archaeological sites Tibes, Punta Candelero and Paso del Indio. Despite poor DNA preservation, we used target enrichment and high-throughput sequencing to obtain complete mitochondrial genomes (mtDNA) from 45 individuals and autosomal genotypes from two individuals. We found a high proportion of Native American mtDNA haplogroups A2 and C1 in the pre-contact Puerto Rico sample (40% and 44%, respectively). This distribution, as well as the haplotypes represented, support a primarily Amazonian South American origin for these populations, and mirrors the Native American mtDNA diversity patterns found in present-day islanders. Three mtDNA haplotypes from pre-contact Puerto Rico persist among Puerto Ricans and other Caribbean islanders, indicating that present-day populations are reservoirs of pre-contact mtDNA diversity. Lastly, we find similarity in autosomal ancestry patterns between pre-contact individuals from Puerto Rico and the Bahamas, suggesting a shared component of Indigenous Caribbean ancestry with close affinity to South American populations. Our findings contribute to a more complete reconstruction of pre-contact Caribbean population history and explore the role of Indigenous peoples in shaping the biocultural diversity of present-day Puerto Ricans and other Caribbean islanders.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/dynamic_arousal_signals_construct_memories_of_time_and_events_66445403/">
          <h4 class="font-weight-bold">Dynamic arousal signals construct memories of time and events, bioRxiv, 2019-09-11</h4>
          <p>AbstractEveryday life unfolds continuously, yet we tend to remember past experiences as discrete event sequences or episodes. Although this phenomenon has been well documented, the brain mechanisms that support the transformation of continuous experience into memorable episodes remain unknown. Here we show that a sudden change in context, or ‘event boundary’, elicits a burst of autonomic arousal, as indexed by pupil dilation. These boundaries during dynamic experience also led to the segmentation of adjacent episodes in later memory, evidenced by changes in memory for the timing, order, and perceptual details of recent event sequences. Critically, we find that distinct cognitive components of this pupil response were associated with both subjective (temporal distance judgements) and objective (temporal order discrimination) measures of episodic memory, suggesting that multiple arousal-mediated cognitive processes help construct meaningful mnemonic events. Together, these findings reveal that arousal processes may play a fundamental role in everyday memory organization.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/hormonal_and_neural_correlates_of_care_in_active_versus_observing_poison_frog_parents_66442344/">
          <h4 class="font-weight-bold">Hormonal and neural correlates of care in active versus observing poison frog parents, bioRxiv, 2019-09-11</h4>
          <p>The occasional reversal of sex-typical behavior suggests that many of the neural circuits underlying behavior are conserved between males and females and can be activated in response to the appropriate social condition or stimulus. Most poison frog species (Family Dendrobatidae) exhibit male uniparental care, but flexible compensation has been observed in some species, where females will take over parental care duties when males disappear. We investigated hormonal and neural correlates of sex-typical and sex-reversed parental care in a typically male uniparental species, the Dyeing Poison Frog (Dendrobates tinctorius). We first characterized hormone levels and whole brain gene expression across parental care stages during sex-typical care. Surprisingly, hormonal changes and brain gene expression differences associated with active parental behavior in males were mirrored in their non-caregiving female partners. To further explore the disconnect between neuroendocrine patterns and behavior, we characterized hormone levels and neural activity patterns in females performing sex-reversed parental care. In contrast to hormone and gene expression patterns, we found that patterns of neural activity were linked to the active performance of parental behavior, with sex-reversed tadpole transporting females exhibiting neural activity patterns more similar to those of transporting males than non-caregiving females. We suggest that parallels in hormones and brain gene expression in active and observing parents are related to females ability to flexibly take over parental care in the absence of their male partners.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/how_neurons_move_during_action_potentials_66443380/">
          <h4 class="font-weight-bold">How neurons move during action potentials, bioRxiv, 2019-09-11</h4>
          <p>AbstractNeurons undergo nanometer-scale deformations during action potentials, and the underlying mechanism has been actively debated for decades. Previous observations were limited to a single spot or the cell boundary, while movement across the entire neuron during the action potential remained unclear.We report full-field imaging of cellular deformations accompanying the action potential in mammalian neuron somas (−1.8nm~1.3nm) and neurites (−0.7nm~0.9nm), using fast quantitative phase imaging with a temporal resolution of 0.1ms and an optical pathlength sensitivity of &amp;lt;4pm per pixel. Spike-triggered average, synchronized to electrical recording, demonstrates that the time course of the optical phase changes matches the dynamics of the electrical signal, with the optical signal revealing the intracellular potential rather than its time derivative detected via extracellular electrodes. Using 3D cellular morphology extracted via confocal microscopy, we demonstrate that the voltage-dependent changes in the membrane tension induced by ionic repulsion can explain the magnitude, time course and spatial features of the phase imaging. Our full-field observations of the spike-induced deformations in mammalian neurons opens the door to non-invasive label-free imaging of neural signaling.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/single-cell-resolution_transcriptome_map_of_human_chimpanzee_bonobo_and_macaque_brains_66403986/">
          <h4 class="font-weight-bold">Single-cell-resolution transcriptome map of human, chimpanzee, bonobo, and macaque brains, bioRxiv, 2019-09-11</h4>
          <p>ABSTRACTIdentification of gene expression traits unique to the human brain sheds light on the mechanisms of human cognition. Here we searched for gene expression traits separating humans from other primates by analyzing 88,047 cell nuclei and 422 tissue samples representing 33 brain regions of humans, chimpanzees, bonobos, and macaques. We show that gene expression evolves rapidly within cell types, with more than two-thirds of cell type-specific differences not detected using conventional RNA sequencing of tissue samples. Neurons tend to evolve faster in all hominids, but non-neuronal cell types, such as astrocytes and oligodendrocyte progenitors, show more differences on the human lineage, including alterations of spatial distribution across neocortical layers.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/2019/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/2019/page/12/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/10/" class="page-link">10</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/11/" class="page-link">11</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/12/" class="page-link">12</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/2019/page/13/" class="page-link">13</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/14/" class="page-link">14</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/15/" class="page-link">15</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/16/" class="page-link">16</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/17/" class="page-link">17</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/2019/page/14/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/2019/page/135/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
