<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_comparison_of_dna_stains_and_staining_methods_for_agarose_gel_electrophoresis_56625796/">
          <h4 class="font-weight-bold">A comparison of DNA stains and staining methods for Agarose Gel Electrophoresis, bioRxiv, 2019-03-07</h4>
          <p>ABSTRACTNucleic acid stains are necessary for Agarose Gel Electrophoresis (AGE). The commonly used but mutagenic Ethidium Bromide is being usurped by a range of safer but more expensive alternatives. These safe stains vary in cost, sensitivity and the impedance of DNA as it migrates through the gel. Modified protocols developed to reduce cost increase this variability. In this study, five Gel stains (GelRed™, GelGreen™, SYBR™ safe, SafeView and EZ-Vision®In-Gel Solution) two premixed loading dyes (SafeWhite, EZ-Vision®One) and four methods (pre-loading at 100x, pre-loading at 10x, precasting and post-staining) are evaluated for sensitivity and effect on DNA migration. GelRed™ was found to be the most sensitive while the EZ-Vision® dyes and SafeWhite had no discernible effect on DNA migration. Homemade loading dyes were as effective as readymade ones at less than 4% of the price. This method used less than 1% of the dye needed for the manufacturer recommended protocols. Thus, with careful consideration of stain and method, Gel stain expenditure can be reduced by over 99%.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/absence_of_entourage_terpenoids_commonly_found_in_cannabis_sativa_do_not_modulate_the_functional_activity_of_%CE%B49-thc_at_human_cb1and_cb2_receptors_56582898/">
          <h4 class="font-weight-bold">Absence of entourage Terpenoids commonly found in Cannabis sativa do not modulate the functional activity of Δ9-THC at human CB1and CB2 receptors, bioRxiv, 2019-03-06</h4>
          <p>AbstractIntroductionCompounds present in Cannabis sativa such as phytocannabinoids and terpenoids, may act in concert to elicit therapeutic effects. Cannabinoids such as Δ9-tetrahydrocannabinol (Δ9-THC) directly activate cannabinoid receptor 1 (CB1) and cannabinoid receptor 2 (CB2), however, it is not known if terpenoids present in Cannabis also affect cannabinoid receptor signalling. Therefore, we examined 6 common terpenoids alone, and in combination with cannabinoid receptor agonists, on CB1 and CB2 signalling in vitro.Materials and MethodsPotassium channel activity in AtT20 FlpIn cells transfected with human CB1 or CB2 receptors was measured in real-time using FLIPR® membrane potential dye in a FlexStation 3 plate reader. Terpenoids were tested individually and in combination for periods up to 30 minutes. Endogenous somatostatin receptors served as a control for direct effects of drugs on potassium channels.Resultsα-Pinene, β-pinene, β-caryophyllene, linalool, limonene and β-myrcene (up to 30-100 µM) did not change membrane potential in AtT20 cells expressing CB1 or CB2, or affect the response to a maximally effective concentration of the synthetic cannabinoid CP55,940. The presence of individual or a combination of terpenoids did not affect the hyperpolarization produced by Δ9-THC (10µM) (CB1 control, 59±7%; with terpenoids (10 µM each) 55±4%; CB2 Δ9-THC 16±5%, with terpenoids (10 µM each) 17±4%). To investigate possible effect on desensitization of CB1 responses, all six terpenoids were added together with Δ9-THC and signalling measured continuously over 30 min. Terpenoids did not affect desensitization, after 30 minutes the control hyperpolarization recovered by 63±6%, in the presence of the terpenoids recovery was 61±5%.DiscussionNone of the six of the most common terpenoids in Cannabis directly activated CB1 or CB2, or modulated the signalling of the phytocannabinoid agonist Δ9-THC. These results suggest that if a phytocannabinoid-terpenoid entourage effect exists, it is not at the CB1 or CB2 receptor level. It remains possible that terpenoids activate CB1 and CB2 signalling pathways that do not involve potassium channels, however, it seems more likely that they may act at different molecular target(s) in the neuronal circuits important for the behavioural effect of Cannabis.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/pharmacology-and-toxicology">pharmacology-and-toxicology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/functional_metagenomics-guided_discovery_of_potent_cas9_inhibitors_in_the_human_microbiome_56580602/">
          <h4 class="font-weight-bold">Functional metagenomics-guided discovery of potent Cas9 inhibitors in the human microbiome, bioRxiv, 2019-03-06</h4>
          <p>AbstractCRISPR-Cas systems protect bacteria and archaea from phages and other mobile genetic elements, which use small anti-CRISPR (Acr) proteins to overcome CRISPR-Cas immunity. Because they are difficult to identify, the natural diversity and impact of Acrs on microbial ecosystems is underappreciated. To overcome this discovery bottleneck, we developed a high-throughput functional selection that isolates acr genes based on their ability to inhibit CRISPR-Cas function. Using this selection, we discovered ten DNA fragments from human oral and fecal metagenomes that antagonize Streptococcus pyogenes Cas9 (SpyCas9). The most potent acr discovered, acrIIA11, was recovered from a Lachnospiraceae phage and is among the strongest known SpyCas9 inhibitors. AcrIIA11 homologs are distributed across multiple bacterial phyla and many divergent homologs inhibit SpyCas9. We show that AcrIIA11 antagonizes SpyCas9 using a different mechanism than that of previously characterized inhibitors. Our study highlights the power of functional selections to uncover widespread Cas9 inhibitors within diverse microbiomes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_robust_and_efficient_method_for_mendelian_randomization_with_hundreds_of_genetic_variants_unravelling_mechanisms_linking_hdl-cholesterol_and_coronary_heart_disease_56478079/">
          <h4 class="font-weight-bold">A robust and efficient method for Mendelian randomization with hundreds of genetic variants unravelling mechanisms linking HDL-cholesterol and coronary heart disease, bioRxiv, 2019-03-05</h4>
          <p>Mendelian randomization (MR) investigations with large numbers of genetic variants are becoming increasingly common. However, the reliability of findings from a MR investigation is dependent on the validity of the genetic variants as instrumental variables. We developed a method to identify groups of genetic variants with similar causal effect estimates, which may represent distinct mechanisms by which the risk factor influences the outcome. Our contamination mixture method is a robust and efficient method for valid MR in the presence of invalid IVs. Compared to other robust methods, our method had the lowest mean squared error across a range of realistic scenarios. The method is fast and efficient, and can perform analysis with hundreds of variants in a fraction of a second. In a MR analysis for high-density lipoprotein (HDL) cholesterol and coronary heart disease (CHD) risk, the method identified 11 variants associated with increased HDL-cholesterol, decreased triglyceride levels, and decreased CHD risk that had the same directions of associations with platelet distribution width and other blood cell traits, suggesting a shared mechanism linking lipids and CHD risk relating to platelet aggregation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/chromatin_arranges_in_filaments_of_blobs_with_nanoscale_functional_zonation_56456514/">
          <h4 class="font-weight-bold">Chromatin arranges in filaments of blobs with nanoscale functional zonation, bioRxiv, 2019-03-05</h4>
          <p>Three-dimensional (3D) chromatin organisation plays a key role in regulating genome function in higher eukaryotes. Despite recognition that the genome partitions into ~1Mb-sized topological associated domains (TADs) based on ensemble Hi-C measurements, many features of the physical organisation at the single cell level remain underexplored. Using 3D super-resolution microscopy, we reveal a sequential curvilinear arrangement of globular chromatin domains with viscoelastic properties (‘blobs’) juxtaposed to an RNA-populated interchromatin (IC) network. Quantitative mapping of genome function markers uncovers a zonal distribution, with RNA-binding factors concentrated in the IC, confinement of structural proteins and transcriptionally activepermissive marks to chromatin domain surfaces, and enrichment of repressive marks towards the interior. This correlation between nanoscale topology and genome function is relaxed in postreplicative chromatin, accentuated in replicative senescence, persists upon ATP depletion and hyperosmolarity induced chromatin condensation and, remarkably, after inactivation of cohesin. Our findings support a model of a higher-order chromatin architecture on the size level of TADs that creates and modulates distinct functional environments through a combination of biophysical parameters such as density and ATP-driven processes such as replication and transcription, but independent of cohesin.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/functional_dissection_of_tads_reveals_non-essential_and_instructive_roles_in_regulating_gene_expression_56502379/">
          <h4 class="font-weight-bold">Functional dissection of TADs reveals non-essential and instructive roles in regulating gene expression, bioRxiv, 2019-03-05</h4>
          <p>AbstractThe genome is organized in megabase-sized three-dimensional units, called Topologically Associated Domains (TADs), that are separated by boundaries. TADs bring distant cis-regulatory elements into proximity, a process dependent on the cooperative action of cohesin and the DNA binding factor CTCF. Surprisingly, genome-wide depletion of CTCF has little effect on transcription, yet structural variations affecting TADs have been shown to cause gene misexpression and congenital disease. Here, we investigate TAD function in vivo in mice by systematically editing components of TAD organization at the Sox9Kcnj locus. We find that TADs are formed by a redundant system of CTCF sites requiring the removal of all major sites within the TAD and at the boundary for two neighboring TADs to fuse. TAD fusion resulted in leakage of regulatory activity from the Sox9 to the Kcnj TAD, but no major changes in gene expression. This indicates that TAD structures provide robustness and precision, but are not essential for developmental gene regulation. Gene misexpression and resulting disease phenotypes, however, were attained by re-directing regulatory activity through inversions andor the re-positioning of boundaries. Thus, efficient re-wiring of enhancer promoter interaction and aberrant disease causing gene activation is not induced by a mere loss of insulation but requires the re-direction of contacts.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/18/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/15/" class="page-link">15</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/16/" class="page-link">16</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/17/" class="page-link">17</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/18/" class="page-link">18</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/0-100-users/page/19/" class="page-link">19</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/20/" class="page-link">20</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/21/" class="page-link">21</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/22/" class="page-link">22</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/23/" class="page-link">23</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/20/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/64/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
