<!DOCTYPE html>
<html><head>
  
  <title>2019 | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/localization_microscopy_at_doubled_precision_with_patterned_illumination_55804298/">
          <h4 class="font-weight-bold">Localization microscopy at doubled precision with patterned illumination, bioRxiv, 2019-02-20</h4>
          <p>MINFLUX offers a breakthrough in single molecule localization precision, but suffers from a tiny field of-view and a lack of practical parallelism. Here, we combine centroid estimation and illumination pattern induced photon count variations in a conventional widefield imaging setup to extract position information over a typical micron sized field-of-view. We show a near twofold improvement in precision over standard localization with the same photon count on DNA-origami nano-structures.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/spatial_metabolomics_of_in_situ_host-microbe_interactions_55762881/">
          <h4 class="font-weight-bold">Spatial metabolomics of in situ, host-microbe interactions, bioRxiv, 2019-02-20</h4>
          <p>Spatial metabolomics describes the location and chemistry of small molecules involved in metabolic phenotypes, defense molecules and chemical interactions in natural communities. Most current techniques are unable to spatially link the genotype and metabolic phenotype of microorganisms in situ at a scale relevant to microbial interactions. Here, we present a spatial metabolomics pipeline (metaFISH) that combines fluorescence in situ hybridization (FISH) microscopy and high-resolution atmospheric pressure mass spectrometry imaging (AP-MALDI-MSI) to image host-microbe symbioses and their metabolic interactions. metaFISH aligns and integrates metabolite and fluorescent images at the micrometer-scale for a spatial assignment of host and symbiont metabolites on the same tissue section. To illustrate the advantages of metaFISH, we mapped the spatial metabolome of a deep-sea mussel and its intracellular symbiotic bacteria at the scale of individual epithelial host cells. Our analytical pipeline revealed metabolic adaptations of the epithelial cells to the intracellular symbionts, a variation in metabolic phenotypes in one symbiont type, and novel symbiosis metabolites. metaFISH provides a culture-independent approach to link metabolic phenotypes to community members in situ - a powerful tool for microbiologists across fields.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/hospital_use_of_antibiotics_as_the_main_driver_of_infections_with_antibiotic-resistant_bacteria_-_a_reanalysis_of_recent_data_from_the_european_union_55695720/">
          <h4 class="font-weight-bold">Hospital use of antibiotics as the main driver of infections with antibiotic-resistant bacteria - a reanalysis of recent data from the European Union, bioRxiv, 2019-02-19</h4>
          <p>Antimicrobial resistance in bacteria causes significant morbidity worldwide. The development and acquisition of resistance to antibiotics is believed to primarily develop under the selective pressure of widespread antibiotic use in humans, however antimicrobial usage in livestock has been proposed as additional, if not principal, driver of antibiotic resistance. In this work, we correlate recent data from the European Union on antibiotic resistance rates with data on antibiotic usage in the primary care and hospital sector and data on veterinary antimicrobial consumption across the individual member states. We quantify the strength of these different potential drivers of antimicrobial resistance in order to compare their biological importance. We found that the correlation between antibiotic use in the hospital sector and antibiotic resistance rates is significantly higher than the correlation between resistance rates and any of the other two predictors. This suggests increased antibiotic use in hospitals as the main driver of the development of antibiotic resistances and necessitates further research on and a re-evaluation of the risks associated with antibiotic use in human and veterinary medicine.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/self-inactivating_rabies_viruses_are_just_first-generation_%CE%B4g_rabies_viruses_55698207/">
          <h4 class="font-weight-bold">Self-inactivating rabies viruses are just first-generation, ΔG rabies viruses, bioRxiv, 2019-02-19</h4>
          <p>A recent article in Cell reported a new form of modified rabies virus that was apparently capable of labeling neurons without adverse effects on neuronal physiology and circuit function. These self-inactivating rabies (SiR) viruses differed from the widely-used first-generation deletion-mutant (ΔG) rabies viruses only by the addition of a destabilization domain to the viral nucleoprotein. However, we observed that the transsynaptic tracing results from that article were inconsistent with the logic described in it, and we hypothesized that the viruses used were actually mutants that had lost the intended modification to the nucleoprotein. We obtained samples of two SiR viruses from the authors and show here that, in both SiR-CRE and SiR-FLPo, the great majority of viral particles were indeed mutants that had lost the intended modification and were therefore just first-generation, ΔG rabies viruses. We also found that SiR-CRE killed 70% of infected neurons in vivo within two weeks. We have shown elsewhere that a ΔG rabies virus encoding Cre can leave a large percentage of labeled neurons alive; we presume that Ciabatti et al. found such remaining neurons at long survival times and mistakenly concluded that they had developed a nontoxic version of rabies virus. Here we have analyzed only the two samples that were sent to MIT by Ciabatti et al., and these may not be from the same batches that were used for their paper. However, 1) both of the two viruses that we analyzed had independently lost the intended modification, 2) the mutations in the two samples were genetically quite distinct from each other yet in both cases caused the same result total or near-total loss of the C-terminal modification, and 3) the mutations that we found in these two virus samples perfectly explain the otherwise-paradoxical transsynaptic tracing results from Ciabatti et al.&#39;s paper. We suggest that the SiR strategy, or any other such attempt to attenuate a virus by addition rather than deletion, is an inherently unstable approach that can easily be evaded by mutation, as it was in this case.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/the_distribution_of_fitness_effects_among_synonymous_mutations_in_a_gene_under_selection_55697006/">
          <h4 class="font-weight-bold">The distribution of fitness effects among synonymous mutations in a gene under selection, bioRxiv, 2019-02-19</h4>
          <p>AbstractThe fitness effects of synonymous mutations, nucleotide changes that do not alter the encoded amino acid, have often been assumed to be neutral, but a growing body of evidence suggests otherwise. We used site-directed mutagenesis coupled with direct measures of competitive fitness to estimate the distribution of fitness effects among synonymous mutations for a gene under selection. Synonymous mutations had highly variable fitness effects, both deleterious and beneficial, resembling those of nonsynonymous mutations in the same gene. This variation in fitness was underlain by changes in transcription linked to the creation of internal promoter sites. A positive correlation between fitness and the presence of synonymous substitutions across a phylogeny of related Pseudomonads suggests these mutations may be common in nature. Taken together, our results provide the most compelling evidence to date that synonymous mutations with non-neutral fitness effects may in fact be commonplace.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/transcript_expression-aware_annotation_improves_rare_variant_discovery_and_interpretation_55759141/">
          <h4 class="font-weight-bold">Transcript expression-aware annotation improves rare variant discovery and interpretation, bioRxiv, 2019-02-19</h4>
          <p>The acceleration of DNA sequencing in patients and population samples has resulted in unprecedented catalogues of human genetic variation, but the interpretation of rare genetic variants discovered using such technologies remains extremely challenging. A striking example of this challenge is the existence of disruptive variants in dosage-sensitive disease genes, even in apparently healthy individuals. Through manual curation of putative loss of function (pLoF) variants in haploinsufficient disease genes in the Genome Aggregation Database (gnomAD), we show that one explanation for this paradox involves alternative mRNA splicing, which allows exons of a gene to be expressed at varying levels across cell types. Currently, no existing annotation tool systematically incorporates this exon expression information into variant interpretation. Here, we develop a transcript-level annotation metric, the proportion expressed across transcripts (pext), which summarizes isoform quantifications for variants. We calculate this metric using 11,706 tissue samples from the Genotype Tissue Expression project (GTEx) and show that it clearly differentiates between weakly and highly evolutionarily conserved exons, a proxy for functional importance. We demonstrate that expression-based annotation selectively filters 22.4% of falsely annotated pLoF variants found in haploinsufficient disease genes in gnomAD, while removing less than 4% of high-confidence pathogenic variants in the same genes. Finally, we apply our expression filter to the analysis of de novo variants in patients with autism spectrum disorder (ASD) and developmental disorders and intellectual disability (DDID) to show that pLoF variants in weakly expressed regions have effect sizes similar to those of synonymous variants, while pLoF variants in highly expressed exons are most strongly enriched among cases versus controls. Our annotation is fast, flexible, and generalizable, making it possible for any variant file to be annotated with any isoform expression dataset, and will be valuable for rare disease diagnosis, rare variant burden analyses in complex disorders, and curation and prioritization of variants in recall-by-genotype studies.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/2019/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/2019/page/62/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/59/" class="page-link">59</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/60/" class="page-link">60</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/61/" class="page-link">61</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/62/" class="page-link">62</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/2019/page/63/" class="page-link">63</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/64/" class="page-link">64</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/65/" class="page-link">65</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/66/" class="page-link">66</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/67/" class="page-link">67</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/2019/page/64/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/2019/page/87/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
