<!DOCTYPE html>
<html><head>
  
  <title>100 200 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/orchestrating_single-cell_analysis_with_bioconductor_57887532/">
          <h4 class="font-weight-bold">Orchestrating Single-Cell Analysis with Bioconductor, bioRxiv, 2019-03-28</h4>
          <p>AbstractRecent developments in experimental technologies such as single-cell RNA sequencing have enabled the profiling a high-dimensional number of genome-wide features in individual cells, inspiring the formation of large-scale data generation projects quantifying unprecedented levels of biological variation at the single-cell level. The data generated in such projects exhibits unique characteristics, including increased sparsity and scale, in terms of both the number of features and the number of samples. Due to these unique characteristics, specialized statistical methods are required along with fast and efficient software implementations in order to successfully derive biological insights. Bioconductor - an open-source, open-development software project based on the R programming language - has pioneered the analysis of such high-throughput, high-dimensional biological data, leveraging a rich history of software and methods development that has spanned the era of sequencing. Featuring state-of-the-art computational methods, standardized data infrastructure, and interactive data visualization tools that are all easily accessible as software packages, Bioconductor has made it possible for a diverse audience to analyze data derived from cutting-edge single-cell assays. Here, we present an overview of single-cell RNA sequencing analysis for prospective users and contributors, highlighting the contributions towards this effort made by Bioconductor.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/unified_rational_protein_engineering_with_sequence-only_deep_representation_learning_57770558/">
          <h4 class="font-weight-bold">Unified rational protein engineering with sequence-only deep representation learning, bioRxiv, 2019-03-26</h4>
          <p>AbstractRational protein engineering requires a holistic understanding of protein function. Here, we apply deep learning to unlabelled amino acid sequences to distill the fundamental features of a protein into a statistical representation that is semantically rich and structurally, evolutionarily, and biophysically grounded. We show that the simplest models built on top of this unified representation (UniRep) are broadly applicable and generalize to unseen regions of sequence space. Our data-driven approach reaches near state-of-the-art or superior performance predicting stability of natural and de novo designed proteins as well as quantitative function of molecularly diverse mutants. UniRep further enables two orders of magnitude cost savings in a protein engineering task. We conclude UniRep is a versatile protein summary that can be applied across protein engineering informatics.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/synthetic-biology">synthetic-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/extraordinary_claims_require_extraordinary_evidence_in_the_case_of_asserted_mtdna_biparental_inheritance_57685203/">
          <h4 class="font-weight-bold">Extraordinary claims require extraordinary evidence in the case of asserted mtDNA biparental inheritance, bioRxiv, 2019-03-25</h4>
          <p>AbstractA breakthrough article published in PNAS by Luo et al. (2018) challenges a central dogma in biology which states that the mitochondrial DNA (mtDNA) is inherited exclusively from the mother. By sequencing the mitogenomes of several members of three independent families, the authors inferred an unprecedented pattern of biparental inheritance that requires the participation of an autosomal nuclear factor in the molecular process. However, a comprehensive analysis of their data reveals a number of issues that must be carefully addressed before challenging the current paradigm. Unfortunately, the methods section lacks any description of sample management, validation of their results in independent laboratories was deficient, and the reported findings have been observed at a frequency at complete variance with established evidence. Moreover, the remarkably high (and unusually homogeneous) levels of heteroplasmy reported can be readily detected using classical techniques for DNA sequencing. By reassessing the raw sequencing data with an alternative computational pipeline, we report strong correlation to the NextGENe results provided by the authors on a per sample base. However, the sequencing replicates from the same donors show aberrations in the variants detected that need further investigation to exclude contributions from other sources or methodological artifacts. Finally, applying the principle of reductio ad absurdum, we demonstrate that the nuclear factor invoked by the authors would need to be extraordinarily complex and precise in order to preclude linear accumulation of mtDNA lineages across generations. We discuss alternate scenarios that explain findings of the same nature as reported by Luo et al., in the context of in-vitro fertilization and therapeutic mtDNA replacement ooplasmic transplantation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/serum_flt3_ligand_is_biomarker_of_progenitor_cell_mass_and_prognosis_in_acute_myeloid_leukemia_57683166/">
          <h4 class="font-weight-bold">Serum Flt3 ligand is biomarker of progenitor cell mass and prognosis in acute myeloid leukemia, bioRxiv, 2019-03-25</h4>
          <p>AbstractFms-like tyrosine kinase 3 (Flt3) is a hematopoietic growth factor receptor expressed on lymphomyeloid progenitors and frequently, by AML blasts. Its ligand, Flt3L, has non-hematopoietic and lymphoid origins, is detectable during homeostasis and increases to high levels in states of hypoplasia due to genetic defects or treatment with cytoreductive agents. Measurement of Flt3L by ELISA reveals that Flt3&#43;AML, is associated with depletion of Flt3L to undetectable levels. After induction chemotherapy, Flt3L is restored in patients entering CR, but remains depressed in those with refractory disease. Weekly sampling reveals marked differences in the kinetics of Flt3L response during the first 6 weeks of treatment, proportionate to the clearance of blasts and cellularity of the BM. In the UK NCRI AML17 trial, Flt3L was measured at day 26 in a subgroup of 135 patients with Flt3 mutation randomized to the tyrosine kinase inhibitor lestaurtinib. In these patients, attainment of CR was associated with higher Flt3L at day 26 (Mann-Whitney p &amp;lt; 0.0001). Day 26 Flt3L was also associated with survival Flt3L ≤ 291pgml was associated with inferior event-free survival; and, Flt3L &amp;gt;1185pgml was associated with higher overall survival (p = 0.0119). Serial measurement of Flt3L in patients who had received a hematopoietic stem cell transplant for AML further illustrated the potential value of declining Flt3L to identify relapse. Together these observations suggest that measurement of Flt3L provides a non-invasive estimate of progenitor cell mass in most patients with AML, with the potential to inform clinical decisions.Graphical abstract&lt;jatsfig id=ufig1 position=float fig-type=figure orientation=portrait&gt;&lt;jatsgraphic xmlnsxlink=httpwww.w3.org1999xlink xlinkhref=588319_ufig1 position=float orientation=portrait &gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/cancer-biology">cancer-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/dietary_intake_regulates_the_circulating_inflammatory_monocyte_pool_57444705/">
          <h4 class="font-weight-bold">Dietary intake regulates the circulating inflammatory monocyte pool, bioRxiv, 2019-03-22</h4>
          <p>SUMMARYCaloric restriction is known to improve inflammatory and autoimmune diseases. However, the mechanisms by which reduced caloric intake modulates inflammation are poorly understood. Here we show that short-term fasting reduced monocyte metabolic and inflammatory activity and drastically reduced the number of circulating monocytes. Regulation of peripheral monocyte numbers was dependent on dietary glucose and protein levels. Specifically, we found that activation of the low-energy sensor 5’-AMP-activated protein kinase (AMPK) in hepatocytes and suppression of systemic CCL2 production by peroxisome proliferator-activator receptor alpha (PPARα) reduced monocyte mobilization from the bone marrow. Importantly, while caloric restriction improves chronic inflammatory diseases, fasting did not compromise monocyte emergency mobilization during acute infectious inflammation and tissue repair. These results reveal that caloric intake and liver energy sensors dictate the blood and tissue immune tone and link dietary habits to inflammatory disease outcome.Highlights&lt;jatslist list-type=bullet&gt;&lt;jatslist-item&gt;Fasting reduces the numbers of peripheral pro-inflammatory monocytes in healthy humans and mice.&lt;jatslist-item&gt;&lt;jatslist-item&gt;A hepatic AMPK-PPARα energy-sensing axis controls homeostatic monocyte numbers via regulation of steady-state CCL2 production.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Fasting reduces monocyte metabolic and inflammatory activity.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Fasting improves chronic inflammatory diseases but does not compromise monocyte emergency mobilization during acute infectious inflammation and tissue repair.&lt;jatslist-item&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/relationship_between_cardiac_cycle_and_the_timing_of_actions_during_action_execution_and_observation_57446137/">
          <h4 class="font-weight-bold">Relationship between cardiac cycle and the timing of actions during action execution and observation, bioRxiv, 2019-03-22</h4>
          <p>AbstractPrevious research suggests that there may be a relationship between the timing of motor events and phases of the cardiac cycle. However, this relationship has thus far only been researched using simple isolated movements such as key-presses in reaction-time tasks and only in a single subject acting alone. Here, we investigated how the cardiac cycle relates to ongoing self-paced movements in both action execution and observation using a novel dyadic paradigm. We recorded electrocardiography (ECG) in 26 subjects who formed 13 dyads containing an action executioner and observer as they performed a self-paced sequence of movements. We demonstrated that heartbeats are timed to movements during both action execution and observation. Specifically, movements were more likely to culminate between heartbeats than simultaneously with the heartbeat. The same pattern was observed for action observation, with the observer’s heartbeats occurring off-phase with movement culmination. These findings demonstrate that there is synchronicity between an action executioner’s cardiac cycle and the timing of their movements, and that the same relationship is mirrored in an observer. This suggests that interpersonal synchronicity may be caused by the mirroring of a phasic relationship between movement and the heart.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/38/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/35/" class="page-link">35</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/36/" class="page-link">36</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/37/" class="page-link">37</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/38/" class="page-link">38</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/39/" class="page-link">39</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/40/" class="page-link">40</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/41/" class="page-link">41</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/42/" class="page-link">42</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/43/" class="page-link">43</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/40/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/126/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
