<!DOCTYPE html>
<html><head>
  
  <title>Biophysics | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/dna-loop_extruding_condensin_complexes_can_traverse_one_another_62727406/">
          <h4 class="font-weight-bold">DNA-loop extruding condensin complexes can traverse one another, bioRxiv, 2019-06-27</h4>
          <p>Condensin, a key member of the Structure Maintenance of Chromosome (SMC) protein complexes, has recently been shown to be a motor that extrudes loops of DNA1. It remains unclear, however, how condensin complexes work together to collectively package DNA into the chromosomal architecture. Here, we use time-lapse single-molecule visualization to study mutual interactions between two DNA-loop-extruding yeast condensins. We find that these one-side-pulling motor proteins are able to dynamically change each other’s DNA loop sizes, even when located large distances apart. When coming into close proximity upon forming a loop within a loop, condensin complexes are, surprisingly, able to traverse each other and form a new type of loop structure, which we term Z loop – three double-stranded DNA helices aligned in parallel with one condensin at each edge. These Z-loops can fill gaps left by single loops and can form symmetric dimer motors that reel in DNA from both sides. These new findings indicate that condensin may achieve chromosomal compaction using a variety of looping structures.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/motility_induced_fracture_reveals_a_ductile_to_brittle_crossover_in_the_epithelial_tissues_of_a_simple_animal_62434365/">
          <h4 class="font-weight-bold">Motility induced fracture reveals a ductile to brittle crossover in the epithelial tissues of a simple animal, bioRxiv, 2019-06-20</h4>
          <p>ABSTRACTAnimals are characterized by their movement, and their tissues are continuously subjected to dynamic force loading while they crawl, walk, run or swim1. Tissue mechanics fundamentally determine the ecological niches that can be endured by a living organism2. While epithelial tissues provide an important barrier function in animals, they are subjected to extreme strains during day to day physiological activities, such as breathing1, feeding3, and defense response4. How-ever, failure or inability to withstand to these extreme strains can result in epithelial fractures5, 6 and associated diseases7, 8. From a materials science perspective, how properties of living cells and their interactions prescribe larger scale tissue rheology and adaptive response in dynamic force landscapes remains an important frontier9. Motivated by pushing tissues to the limits of their integrity, we carry out a multi-modal study of a simple yet highly dynamic organism, the Trichoplax Adhaerens10–12, across four orders of magnitude in length (1 µm to 10 mm), and six orders in time (0.1 sec to 10 hours). We report the discovery of abrupt, bulk epithelial tissue fractures (∼10 sec) induced by the organism’s own motility. Coupled with rapid healing (∼10 min), this discovery accounts for dramatic shape change and physiological asexual division in this early-divergent metazoan. We generalize our understanding of this phenomena by codifying it in a heuristic model, highlighting the fundamental questions underlying the debondingbonding criterion in a soft-active-living material by evoking the concept of an ‘epithelial alloy’. Using a suite of quantitative experimental and numerical techniques, we demonstrate a force-driven ductile to brittle material transition governing the morphodynamics of tissues pushed to the edge of rupture. This work contributes to an important discussion at the core of developmental biology13–17, with important applications to an emerging paradigm in materials and tissue engineering5, 18–20, wound healing and medicine8, 21, 22.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/multi-scale_spatial_heterogeneity_enhances_particle_clearance_in_airway_ciliary_arrays_61877693/">
          <h4 class="font-weight-bold">Multi-scale spatial heterogeneity enhances particle clearance in airway ciliary arrays, bioRxiv, 2019-06-09</h4>
          <p>Mucus clearance constitutes the primary defence of the respiratory system against viruses, bacteria and environmental insults [1]. This transport across the entire airway emerges from the integrated activity of thousands of multiciliated cells, each containing hundreds of cilia, which together must coordinate their spatial arrangement, alignment and motility [2, 3]. The mechanisms of fluid transport have been studied extensively at the level of an individual cilium [4, 5], collectively moving metachronal waves [6–10], and more generally the hydrodynamics of active matter [11, 12]. However, the connection between local cilia architecture and the topology of the flows they generate remains largely unexplored. Here, we image the mouse airway from the sub-cellular (nm) to the organ scales (mm), characterising quantitatively its ciliary arrangement and the generated flows. Locally we measure heterogeneity in both cilia organisation and flow structure, but across the trachea fluid transport is coherent. To examine this result, a hydrodynamic model was developed for a systematic exploration of different tissue architectures. Surprisingly, we find that disorder enhances particle clearance, whether it originates from fluctuations, heterogeneity in multiciliated cell arrangement or ciliary misalignment. This resembles elements of ‘stochastic resonance’ [13–15] in a self-assembled biological system. Taken together, our results shed light on how the microstructure of an active carpet [16, 17] determines its emergent dynamics. Furthermore, this work is also directly applicable to human airway pathologies [1], which are the third leading cause of deaths worldwide [18].</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/shake-it-off_a_simple_ultrasonic_cryo-em_specimen_preparation_device_60204793/">
          <h4 class="font-weight-bold">Shake-it-off A simple ultrasonic cryo-EM specimen preparation device, bioRxiv, 2019-05-10</h4>
          <p>AbstractAlthough microscopes and image analysis software for electron cryomicroscopy (cryo-EM) have improved dramatically in recent years, specimen preparation methods have lagged behind. Most strategies still rely on blotting microscope grids with paper to produce a thin film of solution suitable for vitrification. This approach loses more than 99.9% of the applied sample and requires several seconds, leading to problematic air-water interface interactions for macromolecules in the resulting thin film of solution and complicating time-resolved studies. Recently developed self-wicking EM grids allow use of small volumes of sample, with nanowires on the grid bars removing excess solution to produce a thin film within tens of milliseconds from sample application to freezing. Here we present a simple cryo-EM specimen preparation device that uses components from an ultrasonic humidifier to transfer protein solution onto a self-wicking EM grid. The device is controlled by a Raspberry Pi single board computer and all components are either widely available or can be manufactured by online services, allowing the device to be constructed in laboratories that specialize in cryo-EM, rather than instrument design. The simple open-source design permits straightforward customization of the instrument for specialized experiments.SynopsisA method is presented for high-speed low-volume cryo-EM specimen preparation with a device constructed from readily available components.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/nanoscale_subcellular_architecture_revealed_by_multicolor_3d_salvaged_fluorescence_imaging_59214657/">
          <h4 class="font-weight-bold">Nanoscale subcellular architecture revealed by multicolor 3D salvaged fluorescence imaging, bioRxiv, 2019-04-19</h4>
          <p>AbstractCombining the molecular specificity of fluorescent probes with three-dimensional (3D) imaging at nanoscale resolution is critical for investigating the spatial organization and interactions of cellular organelles and protein complexes. We present a super-resolution light microscope that enables simultaneous multicolor imaging of whole mammalian cells at ~20 nm 3D resolution. We show its power for cell biology research with fluorescence images that resolved the highly convoluted Golgi apparatus and the close contacts between the endoplasmic reticulum and the plasma membrane, structures that have traditionally been the imaging realm of electron microscopy.One Sentence SummaryComplex cellular structures previously only resolved by electron microscopy can now be imaged in multiple colors by 4Pi-SMS.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/scale-free_vertical_tracking_microscopy_towards_bridging_scales_in_biological_oceanography_59085850/">
          <h4 class="font-weight-bold">Scale-free Vertical Tracking Microscopy Towards Bridging Scales in Biological Oceanography, bioRxiv, 2019-04-16</h4>
          <p>AbstractUnderstanding key biophysical phenomena in the ocean often requires one to simultaneously focus on microscale entities, such as motile plankton and sedimenting particles, while maintaining the macroscale context of vertical transport in a highly stratified environment. This poses a conundrum How to measure single organisms, at microscale resolution, in the lab, while allowing them to freely move hundreds of meters in the vertical direction? We present a solution in the form of a scale-free, vertical tracking microscope based on a circular “hydrodynamic-treadmill”. Our technology allows us to transcend physiological and ecological scales, tracking organisms from marine zooplankton to single-cells over vertical scales of meters while resolving microflows and behavioral processes. We demonstrate measurements of sinking particles, including marine snow as they sediment tens of meters while capturing sub-particle-scale phenomena. We also demonstrate depth-patterned virtual-reality environments for novel behavioral analyses of microscale plankton. This technique offers a new experimental paradigm in microscale ocean biophysics by combining physiological-scale imaging with free movement in an ecological-scale patterned environment.One sentence summaryScale-free vertical tracking microscopy captures, for the first time, untethered behavioral dynamics at cellular resolution for marine plankton.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        

        
             
            <li class="page-item active"><a href="/audiences/tags/biophysics/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biophysics/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biophysics/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biophysics/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biophysics/page/5/" class="page-link">5</a></li>
            
        

        
        <li class="page-item"><a href="/audiences/tags/biophysics/page/2/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/biophysics/page/5/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
