<!DOCTYPE html>
<html><head>
  
  <title>Immunology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/serosolver_an_open_source_tool_to_infer_epidemiological_and_immunological_dynamics_from_serological_data_64738862/">
          <h4 class="font-weight-bold">Serosolver an open source tool to infer epidemiological and immunological dynamics from serological data, bioRxiv, 2019-08-09</h4>
          <p>AbstractWe present a flexible, open source R package designed to obtain additional biological and epidemiological insights from commonly available serological datasets. Analysis of serological responses against pathogens with multiple strains such as influenza pose a specific statistical challenge because observed antibody responses measured in serological assays depend both on unobserved prior infections and the resulting cross-reactive antibody dynamics that these infections generate. We provide a general modelling framework to jointly infer these two typically confounded biological processes using antibody titres against current and historical strains. We do this by linking latent infection dynamics with a mechanistic model of antibody dynamics that generates expected antibody titres over time. This makes it possible to use observations of antibodies in serological assays to infer an individual’s infection history as well as the parameters of the antibody process model. Our aim is to provide a flexible inference package that can be applied to a range of datasets studying different viruses over different timescales. We present two case studies to illustrate how our model can infer key immunological parameters, such as antibody titre boosting, waning and cross-reaction, and well as latent epidemiological processes such as attack rates and age-stratified infection risk.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/peripheral_blood_cell_immunophenotyping_reveals_distinct_subgroups_of_inflamed_depression_63721395/">
          <h4 class="font-weight-bold">Peripheral blood cell immunophenotyping reveals distinct subgroups of inflamed depression, bioRxiv, 2019-07-18</h4>
          <p>AbstractDepression has been associated with increased inflammatory proteins but changes in circulating immune cells are less well defined. We used multi-parametric flow cytometry to investigate 14 subsets of peripheral blood cells in 206 cases of major depressive disorder (MDD) and 77 age- and sex-matched controls. There were significant case-control differences, by univariate and multivariate analysis cases showed increased immune cell counts, especially neutrophils, CD4&#43; T cells and monocytes, and increased inflammatory proteins (C-reactive protein and interleukin-6). Within-group analysis demonstrated significant association between the severity of depressive symptoms and increased myeloid and CD4&#43; cell counts. MDD cases could be partitioned into two groups by forced binary clustering of cell counts the inflamed depression group (N=81 out of 206; 39%) had increased monocyte, CD4&#43; and neutrophil counts, increased C-reactive protein (CRP) and interleukin 6 (IL-6), and was more depressed than the uninflamed majority of cases. Relaxing the presumption of a binary classification, data-driven clustering identified four subgroups of MDD cases two of these subgroups (N=38 and N=100; 67% collectively) were associated with increased inflammatory proteins and more severe depression, but differed from each other in the relative weighting of myeloid and lymphoid cell counts. Case-control and within-group results were robust to statistical control for the potentially confounding effects of age, sex, BMI, recent infection status, and tobacco use. Peripheral blood immunophenotyping can be used to identify a candidate cellular biomarker of inflamed depression, and to further decompose that binary partition, suggesting that there is more than one mechanistic pathway underlying inflamed depression.One Sentence SummaryTwo subgroups of depressed cases (about two-thirds of all 206 cases) were identified by peripheral blood biomarker evidence of distinctive cellular immunophenotypes, biased towards the myeloid or lymphoid lineages in different subgroups, but consistently associated with increased blood concentrations of inflammatory proteins and greater severity of depressive symptoms.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/systems-level_immunomonitoring_using_self-sampled_capillary_blood_63285927/">
          <h4 class="font-weight-bold">Systems-level immunomonitoring using self-sampled capillary blood, bioRxiv, 2019-07-08</h4>
          <p>AbstractComprehensive profiling of the human immune system in patients with cancer, autoimmune disease and during infections are providing valuable information that help us understand disease states and discriminate productive from inefficient immune responses and identify possible targets for immune modulation. Recent technical advances now allow for all immune cell populations and hundreds of plasma proteins to be detected using small volume blood samples. To democratize such systems-immunological analyses, further simplified blood sampling and preservation will be important. Here we describe that blood obtained via a nearly painless self-sampling device of 100 microliter of capillary blood that is preserved and frozen, can simplify systems-level immunomonitoring studies.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/transcriptome_dynamics_reveals_progressive_transition_from_effector_to_memory_in_cd4&#43;_t_cells_62407963/">
          <h4 class="font-weight-bold">Transcriptome Dynamics Reveals Progressive Transition from Effector to Memory in CD4&#43; T cells, bioRxiv, 2019-06-19</h4>
          <p>AbstractCD4&#43; T cells are repositories of immune memory, conferring enhanced immunity to many infectious agents. Studies of acute viral and bacterial infection suggest that memory CD4&#43; T cells develop directly from effectors. However, delineating these dynamic developmental pathways has been challenging. Here, we used high-resolution single-cell RNA-seq and temporal mixture modelling to examine the fate of Th1 and Tfh effector cells during non-lethal Plasmodium infection in mice. We observed linear Th1 and Tfh pathways towards memory, characterized by progressive halving in the numbers of genes expressed, and partial transcriptomic coalescence. Low-level persisting infection diverted but did not block these pathways. We observed in the Th1-pathway a linear transition from Th1 through a Tr1 state to TEM cells, which were then poised for Th1 re-call. The Tfh-pathway exhibited a modest Th1-signature throughout, with little evidence of Tr1 development, and co-expression of TCM and memory Tfh markers. Thus, we present a high-resolution atlas of transcriptome dynamics for naïve to memory transitions in CD4&#43; T cells. We also defined a subset of memory-associated genes, including transcription factors Id2 and Maf, whose expression increased progressively against the background of transcriptomic quiescence. Single-cell ATAC-seq revealed substantial heterogeneity in chromatin accessibility in single effectors, which was extensively, though incompletely reset and homogenized in memory. Our data reveal that linear transitions from effector to memory occur in a progressive manner over several weeks, suggesting opportunities for manipulating CD4&#43; T cell memory after primary infection.Highlights&lt;jatslist list-type=bullet&gt;&lt;jatslist-item&gt;scRNA-seq reveals progressive transition from effector to memory in CD4&#43; T cells.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Transcriptome dynamics suggest linear not branching models for memory development.&lt;jatslist-item&gt;&lt;jatslist-item&gt;A subset of genes associates with gradual onset of CD4&#43; T cell memory.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Th1Tfh predisposition varies among clonotypes with identical antigen-specificity.&lt;jatslist-item&gt;&lt;jatslist-item&gt;scATAC-seq uncovers non-coding “memory” elements in the genome.&lt;jatslist-item&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/netosis_proceeds_by_cytoskeleton_and_endomembrane_disassembly_and_pad4-mediated_chromatin_de-condensation_and_nuclear_envelope_rupture_61630599/">
          <h4 class="font-weight-bold">NETosis proceeds by cytoskeleton and endomembrane disassembly and PAD4-mediated chromatin de-condensation and nuclear envelope rupture, bioRxiv, 2019-06-07</h4>
          <p>AbstractNeutrophil extracellular traps (NETs) are web-like DNA structures decorated with histones and cytotoxic proteins that are released by activated neutrophils to trap and neutralize pathogens during the innate immune response, but also form in and exacerbate sterile inflammation. Peptidylarginine deiminase 4 (PAD4) citrullinates histones and is required for NET formation (NETosis) in mouse neutrophils. While the in vivo impact of NETs is accumulating, the cellular events driving NETosis and the role of PAD4 in these events are unclear. We performed high resolution time-lapse microscopy of mouse and human neutrophils (PMN) and differentiated HL-60 neutrophil-like cells (dHL-60) labelled with fluorescent markers of organelles and stimulated with ionomycin or lipopolysaccharides to induce NETosis. Upon stimulation, cells exhibited rapid disassembly of the actin cytoskeleton, followed by shedding of plasma membrane microvesicles, disassembly and remodeling of the microtubule and vimentin cytoskeletons, ER vesiculation, chromatin de-condensation and nuclear rounding, progressive plasma membrane and nuclear envelope (NE) permeabilization, nuclear lamin meshwork and then NE rupture to release DNA into the cytoplasm, and finally plasma membrane rupture and discharge of extracellular DNA. Inhibition of actin disassembly blocked NET release. Mouse and dHL-60 cells bearing genetic alteration of PAD4 showed that chromatin de-condensation, lamin meshwork and NE rupture and extracellular DNA release required the enzymatic and nuclear localization activities of PAD4. Thus, NETosis proceeds by a step-wise sequence of cellular events culminating in the PAD4-mediated expulsion of DNA.Significance StatementNeutrophils are white blood cells specialized as the first line of host defense in the immune system. One way they protect organisms is through NETosis, in which they expel their DNA to form a web-like trap that ensnares pathogens and promotes clotting. However, NETs also mediate sterile inflammation, causing damage to the body. We used high-resolution live-cell microscopy to perform the first systematic characterization of the timing of dynamic cellular events leading to NETosis in human and mouse neutrophils and a neutrophil-like cell line. We discovered that NETosis proceeds by a step-wise sequence of cellular events that is conserved across species, and requires the activity of the PAD4 enzyme for DNA to be released from the nucleus and cell membrane.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/fate_mapping_via_ms4a3_expression_history_traces_monocyte-derived_cells_61185605/">
          <h4 class="font-weight-bold">Fate mapping via Ms4a3 expression history traces monocyte-derived cells, bioRxiv, 2019-05-28</h4>
          <p>SUMMARYMost tissue-resident macrophage (RTM) populations are seeded by waves of embryonic hematopoiesis and are self-maintained independently of a bone-marrow contribution during adulthood. A proportion of RTMs, however, is constantly replaced by blood monocytes and their functions compared to embryonic RTM remains unclear. The kinetics and extent of the contribution of circulating monocytes to RTM replacement during homeostasis, inflammation and disease is highly debated. Here, we identified Ms4a3 as a specific marker expressed by granulocyte-monocyte progenitors (GMPs) and subsequently generated Ms4a3TdT reporter and Ms4a3Cre-RosaTdT fate mapper models to follow monocytes and their progenies. Our Ms4a3Cre-RosaTdT model traced efficiently blood monocytes (97%) and granulocytes (100%), but no lymphocytes or tissue dendritic cells. Using this model, we precisely quantified the contribution of monocytes to the RTM pool during homeostasis and inflammation. The unambiguous identification of monocyte-derived cells will permit future studies of their function under any condition.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        
        <li class="page-item"><a href="/audiences/tags/immunology/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="/audiences/tags/immunology/" class="page-link">1</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/immunology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/immunology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/immunology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/immunology/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/immunology/page/6/" class="page-link">6</a></li>
            
        

        
        <li class="page-item"><a href="/audiences/tags/immunology/page/3/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/immunology/page/6/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
