<!DOCTYPE html>
<html><head>
  
  <title>2019 | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/structural_and_functional_characterization_of_g_protein-coupled_receptors_with_deep_mutational_scanning_59714625/">
          <h4 class="font-weight-bold">Structural and Functional Characterization of G Protein-Coupled Receptors with Deep Mutational Scanning, bioRxiv, 2019-04-30</h4>
          <p>AbstractIn humans, the 813 G protein-coupled receptors (GPCRs) are responsible for transducing diverse chemical stimuli to alter cell state, and are the largest class of drug targets. Their myriad structural conformations and various modes of signaling make it challenging to understand their structure and function. Here we developed a platform to characterize large libraries of GPCR variants in human cell lines with a barcoded transcriptional reporter of G-protein signal transduction. We tested 7,800 of 7,828 possible single amino acid substitutions to the beta-2 adrenergic receptor (β2AR) at four concentrations of the agonist isoproterenol. We identified residues specifically important for β2AR signaling, mutations in the human population that are potentially loss of function, and residues that modulate basal activity. Using unsupervised learning, we resolve residues critical for signaling, including all major structural motifs and molecular interfaces. We also find a previously uncharacterized structural latch spanning the first two extracellular loops that is highly conserved across Class A GPCRs and is conformationally rigid in both the inactive and active states of the receptor. More broadly, by linking deep mutational scanning with engineered transcriptional reporters, we establish a generalizable method for exploring pharmacogenomics, structure and function across broad classes of drug receptors.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/hierarchical_recurrent_state_space_models_reveal_discrete_and_continuous_dynamics_of_neural_activity_in_c._elegans_59680614/">
          <h4 class="font-weight-bold">Hierarchical recurrent state space models reveal discrete and continuous dynamics of neural activity in C. elegans, bioRxiv, 2019-04-29</h4>
          <p>AbstractModern recording techniques enable large-scale measurements of neural activity in a variety of model organisms. The dynamics of neural activity shed light on how organisms process sensory information and generate motor behavior. Here, we study these dynamics using optical recordings of neural activity in the nematode C. elegans. To understand these data, we develop state space models that decompose neural time-series into segments with simple, linear dynamics. We incorporate these models into a hierarchical framework that combines partial recordings from many worms to learn shared structure, while still allowing for individual variability. This framework reveals latent states of population neural activity, along with the discrete behavioral states that govern dynamics in this state space. We find stochastic transition patterns between discrete states and see that transition probabilities are determined by both current brain activity and sensory cues. Our methods automatically recover transition times that closely match manual labels of different behaviors, such as forward crawling, reversals, and turns. Finally, the resulting model can simulate neural data, faithfully capturing salient patterns of whole brain dynamics seen in real data.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/co-reviewing_and_ghostwriting_by_early_career_researchers_in_the_peer_review_of_manuscripts_59574336/">
          <h4 class="font-weight-bold">Co-reviewing and ghostwriting by early career researchers in the peer review of manuscripts, bioRxiv, 2019-04-27</h4>
          <p>AbstractThe goal of this study is to shed light on the involvement of early career researchers (ECRs) during peer review of manuscripts for publication in journals. In particular, we sought to better understand how commonly ECRs contribute ideas andor text to peer review reports when they are not the invited reviewer (“co-review”), and how commonly ECRs do not receive named credit to the journal editorial staff for these scholarly efforts (“ghostwrite”). First, we evaluated 1,952 publications in the peer-reviewed literature generated by exhaustive search terms that combined synonyms of “early career researcher” and “peer review” and found no previous studies about ECRs ghostwriting peer review reports. We then surveyed 498 researchers about their experiences with, and opinions about, co-reviewing and ghostwriting as ECRs. Three quarters of those surveyed have co-reviewed and most find it to be a beneficial (95% agree) and ethical (73% agree) form of training in peer review. Co-reviewing is the second most commonly reported form of training in peer review besides receiving reviews on one’s own papers. Half of survey respondents have ghostwritten a peer review report, despite the 45ths majority opinion that ghostwriting is unethical. Survey respondents report that the three major barriers to including co-reviewer names on peer review reports are a lack of communication between PIs and ECRs; a false belief that co-authorship is for manuscripts but not peer review reports; and prohibitive journal policies that are out of alignment with current practice and opinions about best practice. We therefore propose recommendations for changing this status quo, to discourage unethical ghostwriting of peer review reports and encourage quality co-reviewing experiences as normal training in peer review.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/scientific-communication-and-education">scientific-communication-and-education</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/fast_and_robust_animal_pose_estimation_59580878/">
          <h4 class="font-weight-bold">Fast and robust animal pose estimation, bioRxiv, 2019-04-27</h4>
          <p>AbstractQuantitative behavioral measurements are important for answering questions across scientific disciplines—from neuroscience to ecology. State-of-the-art deep-learning methods offer major advances in data quality and detail by allowing researchers to automatically estimate locations of an animal’s body parts directly from images or videos. However, currently-available animal pose estimation methods have limitations in speed and robustness. Here we introduce an easy-to-use open-source software toolkit, DeepPoseKit, that addresses these problems. Using modern desktop hardware, our methods perform real-time measurements at ∼30–110-Hz with offline performance &amp;gt;1000-Hz—approximately 2× faster than current methods. We also achieve these results without significantly increasing measurement error compared to the most-accurate methods currently available. We demonstrate the versatility of our approach with multiple challenging animal pose estimation tasks in laboratory and field settings—including groups of interacting individuals. Our work reduces barriers to using advanced tools for measuring behavior and has broad applicability across the behavioral sciences.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/animal-behavior-and-cognition">animal-behavior-and-cognition</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/immunogenomic_landscape_of_hematological_malignancies_59575484/">
          <h4 class="font-weight-bold">Immunogenomic landscape of hematological malignancies, bioRxiv, 2019-04-27</h4>
          <p>SUMMARYUnderstanding factors that shape the immune landscape across hematological malignancies is essential for immunotherapy development. Here, we integrated over 8,000 transcriptomes and over 1,000 samples with multilevel genomic data of hematological cancers to investigate how immunological features are linked to cancer subtypes, genetic and epigenetic alterations, and patient survival. Infiltration of cytotoxic immune cells was associated with distinct microenvironmental responses and driver alterations in different cancers, such as TP53 in acute myeloid leukemia and DTX1 in diffuse large B cell lymphoma. Epigenetic modification of CIITA regulating antigen presentation, cancer type-specific immune checkpoints such as VISTA in myeloid malignancies, and variation in cancer antigen expression further contributed to immune heterogeneity. Prognostic models highlighted the significance of immunological properties in predicting survival. Our study represents the most comprehensive effort to date to link immunology with cancer subtypes and genomics in hematological malignancies, providing a resource to guide future studies and immunotherapy development.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/safety_and_tolerability_of_bacteriophage_therapy_in_severe_staphylococcus_aureus_infection_59575100/">
          <h4 class="font-weight-bold">Safety and Tolerability of Bacteriophage Therapy in Severe Staphylococcus aureus Infection, bioRxiv, 2019-04-27</h4>
          <p>AbstractImportanceThe effect of IV administration of a bacteriophage cocktail produced under GMP conditions on patients with severe S. aureus infection, including complicated bacteraemia, endocarditis and septic shock, is unknown.ObjectiveTo assess safety and tolerability of adjunctive bacteriophage therapy in patients with severe S. aureus infections.Design, Setting, ParticipantsObservational, open-label clinical trial of thirteen critically-ill patients admitted to a tertiary-referral hospital with S. aureus bacteraemia (including infective endocarditis, n=6) were assessed by the treating clinician and two consulting infectious diseases physicians to independently verify that routine medical and surgical therapy was optimal and that a poor outcome remained likely. Compassionate access to therapy was approved by both US and Australian regulators and by the Westmead Hospital Human Research Ethics Committee.InterventionA GMP-quality preparation of three combined Myoviridae bacteriophages with specific activity against S. aureus (AB-SA01), was administered intravenously in conjunction with optimal antibiotic therapy.Main Outcome and MeasurementsPhysiological, haematological and biochemical markers of infection, bacterial and bacteriophage kinetics in blood, development of resistance to bacteriophages, and mortality at 28 (D28) and 90 (D90) days were measured. Main outcomes were safety and tolerability.ResultsBacteriophage therapy was initiated 4-10 days after antibiotic commencement, at 109 plaque-forming units (PFU) twice daily. Infecting staphylococci were typical of common local subtypes. Initial input ratio of phages to bacteria in the bloodstream (MOIinput) was &amp;gt;100. Five of the thirteen patients died by D28 and a sixth patient suffered sudden cardiac death on D90. Bacteriophage therapy coincided with a marked reduction in staphylococcal bacterial DNA in the blood and in sepsis-associated inflammatory responses in almost all cases. No bacteriophage-attributable adverse events were identified. Development of bacteriophage resistance was not observed. Population analysis revealed no significant effect of bacteriophage therapy on the gut microflora.Conclusions and RelevanceAdjunctive bacteriophage therapy appears to be safe and well-tolerated in critically ill patients with severe S. aureus infection. Two weeks of twice daily intravenous administration may be a suitable protocol. Controlled trials are needed.Trial RegistrationWestmead Hospital Human Research Ethics Committee approval July 11, 2017; ClinicalTrials.gov Identifier &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=clintrialgov xlinkhref=NCT03395769&gt;NCT03395769&lt;jatsext-link&gt;, AB-SA01-EAP01 (January 10, 2018); Clinical Trials Notification (Australian Therapeutic Goods Association) CT-2018-CTN-02372-1 (July 23, 2018).Key PointsQuestionIs intravenous (IV) administration of investigational bacteriophage (phage) therapy safe and well-tolerated in patients with severe Staphylococcus aureus infection?FindingsThirteen patients with severe S. aureus infections received AB-SA01, a bacteriophage product prepared according to Good Manufacturing Practices (GMP), as adjunctive therapy to antibiotics. AB-SA01 was well-tolerated with no adverse events identified. Bacterial burden and inflammatory responses were reduced and no phage-resistant staphylococci were isolated during or after therapy.MeaningOur results will inform future randomised controlled trials assessing the antibacterial and anti-inflammatory potential of bacteriophages in the treatment of severe S. aureus infection.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/clinical-trials">clinical-trials</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/2019/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/2019/page/26/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/23/" class="page-link">23</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/24/" class="page-link">24</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/25/" class="page-link">25</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/26/" class="page-link">26</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/2019/page/27/" class="page-link">27</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/28/" class="page-link">28</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/29/" class="page-link">29</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/30/" class="page-link">30</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/31/" class="page-link">31</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/2019/page/28/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/2019/page/87/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
