<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/diversification_and_collapse_of_a_telomere_elongation_mechanism_49862531/">
          <h4 class="font-weight-bold">Diversification and collapse of a telomere elongation mechanism, bioRxiv, 2018-10-18</h4>
          <p>AbstractIn virtually all eukaryotes, telomerase counteracts chromosome erosion by adding repetitive sequence to terminal ends. Drosophila melanogaster instead relies on specialized retrotransposons that insert preferentially at telomeres. This exchange of goods between host and mobile element—wherein the mobile element provides an essential genome service and the host provides a hospitable niche for mobile element propagation—has been called a ‘genomic symbiosis’. However, these telomere-specialized, ‘jockey’ family elements may actually evolve to selfishly over-replicate in the genomes that they ostensibly serve. Under this intra-genomic conflict model, we expect rapid diversification of telomere-specialized retrotransposon lineages and possibly, the breakdown of this tenuous relationship. Here we report data consistent with both predictions. Searching the raw reads of the 15-million-year-old ‘melanogaster species group’, we generated de novo jockey retrotransposon consensus sequences and used phylogenetic tree-building to delineate four distinct telomere-associated lineages. Recurrent gains, losses, and replacements account for this striking retrotransposon lineage diversity. Moreover, an ancestrally telomere-specialized element has ‘escaped,’ residing now throughout the genome of D. rhopaloa. In D. biarmipes, telomere-specialized elements have disappeared completely. De novo assembly of long-reads and cytogenetics confirmed this species-specific collapse of retrotransposon-dependent telomere elongation. Instead, telomere-restricted satellite DNA and DNA transposon fragments occupy its terminal ends. We infer that D. biarmipes relies instead on a recombination-based mechanism conserved from yeast to flies to humans. Combined with previous reports of adaptive evolution at host proteins that regulate telomere length, telomere-associated retrotransposon diversification and disappearance offer compelling evidence that intra-genomic conflict shapes Drosophila telomere evolution.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/an_analysis_of_decision_under_risk_in_rats_49861621/">
          <h4 class="font-weight-bold">An Analysis of Decision Under Risk in Rats, bioRxiv, 2018-10-17</h4>
          <p>AbstractProspect Theory is the predominant behavioral economic theory describing decision-making under risk. It accounts for near universal aspects of human choice behavior whose prevalence may reflect fundamental neural mechanisms. We now apply Prospect Theory’s framework to rodents, using a task in which rats chose between guaranteed and probabilistic rewards. Like humans, rats distorted probabilities and showed diminishing marginal sensitivity, in which they were less sensitive to differences in larger rewards. They exhibited reference dependence, in which the valence of outcomes (gain or loss) was determined by an internal reference point reflecting reward history. The similarities between rats and humans suggest conserved neural substrates, and enable application of powerful molecularcircuit tools to study mechanisms of psychological phenomena from behavioral economics.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/high_throughput_droplet_single-cell_genotyping_of_transcriptomes_got_reveals_the_cell_identity_dependency_of_the_impact_of_somatic_mutations_49824772/">
          <h4 class="font-weight-bold">High throughput droplet single-cell Genotyping of Transcriptomes (GoT) reveals the cell identity dependency of the impact of somatic mutations, bioRxiv, 2018-10-17</h4>
          <p>AbstractDefining the transcriptomic identity of clonally related malignant cells is challenging in the absence of cell surface markers that distinguish cancer clones from one another or from admixed non-neoplastic cells. While single-cell methods have been devised to capture both the transcriptome and genotype, these methods are not compatible with droplet-based single-cell transcriptomics, limiting their throughput. To overcome this limitation, we present single-cell Genotyping of Transcriptomes (GoT), which integrates cDNA genotyping with high-throughput droplet-based single-cell RNA-seq. We further demonstrate that multiplexed GoT can interrogate multiple genotypes for distinguishing subclonal transcriptomic identity. We apply GoT to 26,039 CD34&#43; cells across six patients with myeloid neoplasms, in which the complex process of hematopoiesis is corrupted by CALR-mutated stem and progenitor cells. We define high-resolution maps of malignant versus normal hematopoietic progenitors, and show that while mutant cells are comingled with wildtype cells throughout the hematopoietic progenitor landscape, their frequency increases with differentiation. We identify the unfolded protein response as a predominant outcome of CALR mutations, with significant cell identity dependency. Furthermore, we identify that CALR mutations lead to NF-κB pathway upregulation specifically in uncommitted early stem cells. Collectively, GoT provides high-throughput linkage of single-cell genotypes with transcriptomes and reveals that the transcriptional output of somatic mutations is heavily dependent on the native cell identity.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cancer-biology">cancer-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/maternal_and_fetal_genetic_effects_on_birth_weight_and_their_relevance_to_cardio-metabolic_risk_factors_49861927/">
          <h4 class="font-weight-bold">Maternal and fetal genetic effects on birth weight and their relevance to cardio-metabolic risk factors, bioRxiv, 2018-10-17</h4>
          <p>AbstractBirth weight (BW) variation is influenced by fetal and maternal genetic and non-genetic factors, and has been reproducibly associated with future cardio-metabolic health outcomes. These associations have been proposed to reflect the lifelong consequences of an adverse intrauterine environment. In earlier work, we demonstrated that much of the negative correlation between BW and adult cardio-metabolic traits could instead be attributable to shared genetic effects. However, that work and other previous studies did not systematically distinguish the direct effects of an individual’s own genotype on BW and subsequent disease risk from indirect effects of their mother’s correlated genotype, mediated by the intrauterine environment. Here, we describe expanded genome-wide association analyses of own BW (n=321,223) and offspring BW (n=230,069 mothers), which identified 278 independent association signals influencing BW (214 novel). We used structural equation modelling to decompose the contributions of direct fetal and indirect maternal genetic influences on BW, implicating fetal- and maternal-specific mechanisms. We used Mendelian randomization to explore the causal relationships between factors influencing BW through fetal or maternal routes, for example, glycemic traits and blood pressure. Direct fetal genotype effects dominate the shared genetic contribution to the association between lower BW and higher type 2 diabetes risk, whereas the relationship between lower BW and higher later blood pressure (BP) is driven by a combination of indirect maternal and direct fetal genetic effects indirect effects of maternal BP-raising genotypes act to reduce offspring BW, but only direct fetal genotype effects (once inherited) increase the offspring’s later BP. Instrumental variable analysis using maternal BW-lowering genotypes to proxy for an adverse intrauterine environment provided no evidence that it causally raises offspring BP. In successfully separating fetal from maternal genetic effects, this work represents an important advance in genetic studies of perinatal outcomes, and shows that the association between lower BW and higher adult BP is attributable to genetic effects, and not to intrauterine programming.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/tads_pair_homologous_chromosomes_to_promote_interchromosomal_gene_regulation_49824201/">
          <h4 class="font-weight-bold">TADs pair homologous chromosomes to promote interchromosomal gene regulation, bioRxiv, 2018-10-17</h4>
          <p>AbstractHomologous chromosomes colocalize to regulate gene expression in processes including genomic imprinting and X-inactivation, but the mechanisms driving these interactions are poorly understood. In Drosophila, homologous chromosomes pair throughout development, promoting an interchromosomal gene regulatory mechanism called transvection. Despite over a century of study, the molecular features that facilitate chromosome-wide pairing are unknown. The “button” model of pairing proposes that specific regions along chromosomes pair with a higher affinity than their surrounding regions, but only a handful of DNA elements that drive homologous pairing between chromosomes have been described. Here, we identify button loci interspersed across the fly genome that have the ability to pair with their homologous sequences. Buttons are characterized by topologically associated domains (TADs), which drive pairing with their endogenous loci from multiple locations in the genome. Fragments of TADs do not pair, suggesting a model in which combinations of elements interspersed along the full length of a TAD are required for pairing. Though DNA-binding insulator proteins are not associated with pairing, buttons are enriched for insulator cofactors, suggesting that these proteins may mediate higher order interactions between homologous TADs. Using a TAD spanning the spinelessd gene as a paradigm, we find that pairing is necessary but not sufficient for transvection. spineless pairing and transvection are cell-type-specific, suggesting that local buttoning and unbuttoning regulates transvection efficiency between cell types. Together, our data support a model in which specialized TADs button homologous chromosomes together to facilitate cell-type-specific interchromosomal gene regulation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/uncertainty_in_rna-seq_gene_expression_data_49861898/">
          <h4 class="font-weight-bold">Uncertainty in RNA-seq gene expression data, bioRxiv, 2018-10-17</h4>
          <p>RNA-sequencing data is widely used to identify disease biomarkers and therapeutic targets. Here, using data from five RNA-seq processing pipelines applied to 6,690 human tumor and normal tissues, we show that for &amp;gt;12% of protein-coding genes, in at least 1% of samples, current best-in-class RNA-seq processing pipelines differ in their abundance estimates by more than four-fold using the same samples and the same set of RNA-seq reads, raising clinical concern.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/37/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/34/" class="page-link">34</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/35/" class="page-link">35</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/36/" class="page-link">36</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/37/" class="page-link">37</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/0-100-users/page/38/" class="page-link">38</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/39/" class="page-link">39</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/40/" class="page-link">40</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/41/" class="page-link">41</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/42/" class="page-link">42</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/39/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/64/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
