<!DOCTYPE html>
<html><head>
  
  <title>Evolutionary Biology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/global_adaptation_confounds_the_search_for_local_adaptation_65264544/">
          <h4 class="font-weight-bold">Global adaptation confounds the search for local adaptation, bioRxiv, 2019-08-21</h4>
          <p>AbstractSpatially varying selection promotes variance in allele frequencies, increasing genetic differentiation between the demes of a metapopulation. For that reason, outliers in the genome wide distribution of summary statistics measuring genetic differentiation, such as FST, are often interpreted as evidence for alleles which contribute to local adaptation. However, in spatially structured populations, the spread of beneficial mutations with spatially uniform effects can also induce transient genetic differentiation and numerous theoretical studies have suggested that species-wide, or global, adaptation makes a substantial contribution to molecular evolution. In this study, we ask whether such global adaptation affects the genome-wide distribution of FST and generates statistical outliers which could be mistaken for local adaptation. Using forward-in-time population genetic simulations assuming parameters for the rate and strength of beneficial mutations similar to those that have been estimated for natural populations, we show the spread of globally beneficial in parapatric populations can readily generate FST outliers, which may be misinterpreted as evidence for local adaptation. The spread of beneficial mutations causes selective sweeps at flanking sites, so the effects of global versus local adaptation may be distinguished by examining patterns of nucleotide diversity along with FST. Our study suggests that global adaptation should be considered in the interpretation of genome-scan results and the design of future studies aimed at understanding the genetic basis of local adaptation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/ancestral_reconstruction_of_sunflower_karyotypes_reveals_non-random_chromosomal_evolution_65028683/">
          <h4 class="font-weight-bold">Ancestral reconstruction of sunflower karyotypes reveals non-random chromosomal evolution, bioRxiv, 2019-08-16</h4>
          <p>AbstractMapping the chromosomal rearrangements between species can inform our understanding of genome evolution, reproductive isolation, and speciation. Here we present a novel algorithm for identifying regions of synteny in pairs of genetic maps, which is implemented in the accompanying R package, syntR. The syntR algorithm performs as well as previous ad-hoc methods while being systematic, repeatable, and is applicable to mapping chromosomal rearrangements in any group of species. In addition, we present a systematic survey of chromosomal rearrangements in the annual sunflowers, which is a group known for extreme karyotypic diversity. We build high-density genetic maps for two subspecies of the prairie sunflower, Helianthus petiolaris ssp. petiolaris and H. petiolaris ssp. fallax.Using syntR, and we identify blocks of synteny between these two subspecies and previously published high-density genetic maps. We reconstruct ancestral karyotypes for annual sunflowers using those synteny blocks and conservatively estimate that there have been 7.9 chromosomal rearrangements per million years – a high rate of chromosomal evolution. Although the rate of inversion is even higher than the rate of translocation in this group, we further find that every extant karyotype is distinguished by between 1 and 3 translocations involving only 8 of the 17 chromosomes. This non-random exchange suggests that specific chromosomes are prone to translocation and may thus contribute disproportionately to widespread hybrid sterility in sunflowers. These data deepen our understanding of chromosome evolution and confirm that Helianthus has an exceptional rate of chromosomal rearrangement that may facilitate similarly rapid diversification.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/anomalous_phylogenetic_behavior_of_ribosomal_proteins_in_metagenome_assembled_genomes_64786597/">
          <h4 class="font-weight-bold">Anomalous phylogenetic behavior of ribosomal proteins in metagenome assembled genomes, bioRxiv, 2019-08-10</h4>
          <p>SummaryMetagenomic studies have claimed the existence of novel lineages with unprecedented properties never before observed in prokaryotes. Such lineages include Asgard archaea1–3, which are purported to represent archaea with eukaryotic cell complexity, and the Candidate Phyla Radiation (CPR), a novel domain level taxon erected solely on the basis of metagenomic data4. However, it has escaped the attention of most biologists that these metagenomic sequences are not assembled into genomes by sequence overlap, as for cultured archaea and bacteria. Instead, short contigs are sorted into computer files by a process called binning in which they receive taxonomic assignment on the basis of sequence properties like GC content, dinucleotide frequencies, and stoichiometric co-occurrence across samples. Consequently, they are not genome sequences as we know them, reflecting the gene content of real organisms. Rather they are metagenome assembled genomes (MAGs). Debates that Asgard data are contaminated with individual eukaryotic sequences5–7 are overshadowed by the more pressing issue that no evidence exists to indicate that any sequences in binned Asgard MAGs actually stem from the same chromosome, as opposed to simply stemming from the same environment. Here we show that Asgard and CPR MAGs fail spectacularly to meet the most basic phylogenetic criterion8 fulfilled by genome sequences of all cultured prokaryotes investigated to date the ribosomal proteins of Asgard and CPR MAGs do not share common evolutionary histories. Their phylogenetic behavior is anomalous to a degree never observed with genomes of real organisms. CPR and Asgard MAGs are binning artefacts, assembled from environments where up to 90% of the DNA is from dead cells9–12. Asgard and CPR MAGs are unnatural constructs, genome-like patchworks of genes that have been stitched together into computer files by binning.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/plumage_redness_signals_mitochondrial_function_in_the_house_finch_64738134/">
          <h4 class="font-weight-bold">Plumage redness signals mitochondrial function in the House Finch, bioRxiv, 2019-08-09</h4>
          <p>AbstractCarotenoid coloration is widely recognized as a signal of individual condition in various animals, but despite decades of study, the mechanisms that link carotenoid coloration to condition remain unresolved. Most birds with red feathers convert yellow dietary carotenoids to red carotenoids in an oxidation process requiring the gene encoding the putative cytochrome P450 enzyme CYP2J19. Here, we tested the hypothesis that the process of carotenoid oxidation and feather pigmentation is functionally linked to mitochondrial performance. Consistent with this hypothesis, we observed high levels of red ketolated carotenoids associated with the hepatic mitochondria of molting wild house finches (Haemorhous mexicanus), and upon fractionation, we found the highest concentration of ketolated carotenoids in the inner mitochondrial membrane. We further found that the redness of growing feathers was positively related to the performance of liver mitochondria. Structural modeling of CYP2J19 supports a direct role of this protein in carotenoid ketolation that may be functionally linked to cellular respiration. These observations suggest that feather coloration serves as a signal of core functionality through inexorable links to cellular respiration in the mitochondria.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/phylogenies_of_extant_species_are_consistent_with_an_infinite_array_of_diversification_histories_64388458/">
          <h4 class="font-weight-bold">Phylogenies of extant species are consistent with an infinite array of diversification histories, bioRxiv, 2019-08-01</h4>
          <p>AbstractTime-calibrated molecular phylogenies of extant species (extant timetrees) are widely used for estimating the dynamics of diversification rates (1–6) and testing for associations between these rates and environmental factors (5, 7) or species traits (8). However, there has been considerable debate surrounding the reliability of these inferences in the absence of fossil data (9–13), and to date this critical question remains unresolved. Here we mathematically clarify the precise information that can be extracted from extant timetrees under the generalized birth-death model, which underlies the majority of existing estimation methods. We prove that for a given extant timetree and a candidate diversification scenario, there exists an infinite number of alternative diversification scenarios that are equally likely to have generated a given tree. These “congruent” scenarios cannot possibly be distinguished using extant timetrees alone, even in the presence of infinite data. Importantly, congruent diversification scenarios can exhibit markedly different and yet plausible diversification dynamics, suggesting that many previous studies may have over-interpreted phylogenetic evidence. We show that sets of congruent models can be uniquely described using composite variables, which contain all available information about past dynamics of diversification (14); this suggests an alternative paradigm for learning about the past from extant timetrees.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/differential_gene_expression_and_gene_variants_drive_color_and_pattern_development_in_divergent_color_morphs_of_a_mimetic_poison_frog_63857422/">
          <h4 class="font-weight-bold">Differential gene expression and gene variants drive color and pattern development in divergent color morphs of a mimetic poison frog, bioRxiv, 2019-07-22</h4>
          <p>AbstractEvolutionary biologists have long investigated the ecological contexts, evolutionary forces, and proximate mechanisms that produce the diversity of animal coloration we see in the natural world. In aposematic species, color and pattern is directly tied to survival and thus understanding the origin of the phenotype has been a focus of both theoretical and empirical inquiry. In order to better understand this diversity, we examined gene expression in skin tissue during development in four different color morphs of the aposematic mimic poison frog, Ranitomeya imitator. We identified a suite of candidate color-related genes a priori and identified the pattern of expression in these genes over time, differences in expression of these genes between the mimetic morphs, and genetic variants that differ between color morphs. We identified several candidate color genes that are differentially expressed over time or across populations, as well as a number of color genes with fixed genetic variants between color morphs. Many of the color genes we discovered in our dataset are involved in the canonical Wnt signaling pathway, including several fixed SNPs between color morphs. Further, many genes in this pathway were differentially expressed at different points in development (e.g., lef1, tyr, tyrp1). Importantly, Wnt signaling pathway genes are overrepresented relative to expression in Xenopus tropicalis. Taken together, this provides evidence that the Wnt signaling pathway is contributing to color pattern production in R. imitator, and is an excellent candidate for producing some of the differences in color pattern between morphs. In addition, we found evidence that sepiapterin reductase is likely important in the production of yellow-green coloration in this adaptive radiation. Finally, two iridophore genes (arfap1, gart) draw a strong parallel to previous work in another dendrobatid, indicating that these genes are also strong candidates for differential color production. We have used high throughput sequencing throughout development to examine the evolution of coloration in a rapid mimetic adaptive radiation and found that these divergent color patterns are likely to be affected by a combination of developmental patterns of gene expression, color morph-specific gene expression, and color morph-specific gene variants.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        
        <li class="page-item"><a href="/audiences/tags/evolutionary-biology/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="/audiences/tags/evolutionary-biology/" class="page-link">1</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/evolutionary-biology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/evolutionary-biology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/evolutionary-biology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/evolutionary-biology/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/evolutionary-biology/page/6/" class="page-link">6</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/evolutionary-biology/page/3/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/evolutionary-biology/page/20/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
