<!DOCTYPE html>
<html><head>
  
  <title>100 200 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/towards_inferring_causal_gene_regulatory_networks_from_single_cell_expression_measurements_48894487/">
          <h4 class="font-weight-bold">Towards inferring causal gene regulatory networks from single cell expression Measurements, bioRxiv, 2018-09-27</h4>
          <p>AbstractSingle-cell transcriptome sequencing now routinely samples thousands of cells, potentially providing enough data to reconstruct causal gene regulatory networks from observational data. Here, we present Scribe, a toolkit for detecting and visualizing causal regulatory interactions between genes and explore the potential for single-cell experiments to power network reconstruction. Scribe employs Restricted Directed Information to determine causality by estimating the strength of information transferred from a potential regulator to its downstream target. We apply Scribe and other leading approaches for causal network reconstruction to several types of single-cell measurements and show that there is a dramatic drop in performance for pseudotime” ordered single-cell data compared to true time series data. We demonstrate that performing causal inference requires temporal coupling between measurements. We show that methods such as “RNA velocity” restore some degree of coupling through an analysis of chromaffin cell fate commitment. These analyses therefore highlight an important shortcoming in experimental and computational methods for analyzing gene regulation at single-cell resolution and point the way towards overcoming it.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/universal_light-sheet_generation_with_field_synthesis_48909175/">
          <h4 class="font-weight-bold">Universal Light-Sheet Generation with Field Synthesis, bioRxiv, 2018-09-26</h4>
          <p>AbstractWe introduce Field Synthesis, a theorem that can be used to synthesize any scanned or dithered light-sheet, including those used in lattice light-sheet microscopy (LLSM), from an incoherent superposition of one-dimensional intensity distributions. This user-friendly and modular approach offers a drastically simplified optical design, higher light-throughput, simultaneous multicolor illumination, and a 100% spatial duty cycle, thereby providing uncompromised biological imaging with decreased rates of photobleaching.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/bioengineering">bioengineering</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/high-throughput_targeted_long-read_single_cell_sequencing_reveals_the_clonal_and_transcriptional_landscape_of_lymphocytes_48842849/">
          <h4 class="font-weight-bold">High-throughput targeted long-read single cell sequencing reveals the clonal and transcriptional landscape of lymphocytes, bioRxiv, 2018-09-25</h4>
          <p>AbstractHigh-throughput single-cell RNA-Sequencing is a powerful technique for gene expression profiling of complex and heterogeneous cellular populations such as the immune system. However, these methods only provide short-read sequence from one end of a cDNA template, making them poorly suited to the investigation of gene-regulatory events such as mRNA splicing, adaptive immune responses or somatic genome evolution. To address this challenge, we have developed a method that combines targeted long-read sequencing with short-read based transcriptome profiling of barcoded single cell libraries generated by droplet-based partitioning. We use Repertoire And Gene Expression sequencing (RAGE-seq) to accurately characterize full-length T cell (TCR) and B cell (BCR) receptor sequences and transcriptional profiles of more than 7,138 lymphocytes sampled from the primary tumour and draining lymph node of a breast cancer patient. With this method we show that somatic mutation, alternate splicing and clonal evolution of T and B lymphocytes can be tracked across these tissue compartments. Our results demonstrate that RAGE-Seq is an accessible and cost-effective method for high-throughput deep single cell profiling, applicable to a wide range of biological challenges.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_transposable_element_insertion_is_the_switch_between_alternative_life_history_strategies_48826894/">
          <h4 class="font-weight-bold">A transposable element insertion is the switch between alternative life history strategies, bioRxiv, 2018-09-24</h4>
          <p>Tradeoffs affect resource allocation during development and result in fitness consequences that drive the evolution of life history strategies. Yet despite their importance, we know little about the mechanisms underlying life history tradeoffs in wild populations. Many species of Colias butterflies exhibit an alternative life history strategy (ALHS) where females divert resources from wing pigment synthesis to reproductive and somatic development. Due to this reallocation, a wing color polymorphism is associated with the ALHS individuals have either yelloworange or white wings. Here we map the genetic basis of the ALHS switch in Colias crocea to a transposable element insertion downstream of the Colias homolog of BarH-1, a homeobox transcription factor. Using CRISPRCas9 gene editing, antibody staining, and electron microscopy we find morph-specific specific expression of BarH-1 suppresses the formation of pigment granules in wing scales. Lipid and transcriptome analyses reveal physiological differences associated with the ALHS. These findings characterize a novel mechanism for a female-limited ALHS and show that the switch arises via recruitment of a transcription factor previously known for its function in cell fate determination in pigment cells of the retina.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/mito-tag_mice_enable_rapid_isolation_and_multimodal_profiling_of_mitochondria_from_specific_cell_types_in_vivo_48845489/">
          <h4 class="font-weight-bold">MITO-Tag Mice enable rapid isolation and multimodal profiling of mitochondria from specific cell types in vivo, bioRxiv, 2018-09-24</h4>
          <p>ABSTRACTMitochondria are metabolic organelles that are essential for mammalian life, but the dynamics of mitochondrial metabolism within mammalian tissues in vivo remains incompletely understood. While whole-tissue metabolite profiling has been useful for studying metabolism in vivo, such an approach lacks resolution at the cellular and subcellular level. In vivo methods for interrogating organellar metabolites in specific cell-types within mammalian tissues have been limited. To address this, we built on prior work in which we exploited a mitochondrially-localized 3XHA epitope-tag (“MITO-Tag”) for the fast isolation of mitochondria from cultured cells to now generate “MITO-Tag Mice.” Affording spatiotemporal control over MITO-Tag expression, these transgenic animals enable the rapid, cell-type-specific immunoisolation of mitochondria from tissues, which we verified using a combination of proteomic and metabolomic approaches. Using MITO-Tag Mice and targeted and untargeted metabolite profiling, we identified changes during fasted and refed conditions in a diverse array of mitochondrial metabolites in hepatocytes and found metabolites that behaved differently at the mitochondrial versus whole-tissue level. MITO-Tag Mice should have utility for studying mitochondrial physiology and our strategy should be generally applicable for studying other mammalian organelles in specific cell-types in vivo.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/how_to_make_a_rodent_giant_genomic_basis_and_tradeoffs_of_gigantism_in_the_capybara_the_worlds_largest_rodent_48807334/">
          <h4 class="font-weight-bold">How to make a rodent giant Genomic basis and tradeoffs of gigantism in the capybara, the world’s largest rodent, bioRxiv, 2018-09-23</h4>
          <p>AbstractGigantism is the result of one lineage within a clade evolving extremely large body size relative to its small-bodied ancestors, a phenomenon observed numerous times in animals. Theory predicts that the evolution of giants should be constrained by two tradeoffs. First, because body size is negatively correlated with population size, purifying selection is expected to be less efficient in species of large body size, leading to a genome-wide elevation of the ratio of non-synonymous to synonymous substitution rates (dNdS) or mutation load. Second, gigantism is achieved through higher number of cells and higher rates of cell proliferation, thus increasing the likelihood of cancer. However, the incidence of cancer in gigantic animals is lower than the theoretical expectation, a phenomenon referred to as Peto’s Paradox. To explore the genetic basis of gigantism in rodents and uncover genomic signatures of gigantism-related tradeoffs, we sequenced the genome of the capybara, the world’s largest living rodent. We found that dNdS is elevated genome wide in the capybara, relative to other rodents, implying a higher mutation load. Conversely, a genome-wide scan for adaptive protein evolution in the capybara highlighted several genes involved in growth regulation by the insulininsulin-like growth factor signaling (IIS) pathway. Capybara-specific gene-family expansions included a putative novel anticancer adaptation that involves T cell-mediated tumor suppression, offering a potential resolution to Peto’s Paradox in this lineage. Gene interaction network analyses also revealed that size regulators function simultaneously as growth factors and oncogenes, creating an evolutionary conflict. Based on our findings, we hypothesize that gigantism in the capybara likely involved three evolutionary steps 1) Increase in body size by cell proliferation through the ISS pathway, 2) coupled evolution of growth-regulatory and cancer-suppression mechanisms, possibly driven by intragenomic conflict, and 3) establishment of the T cell-mediated tumor suppression pathway as an anticancer adaptation. Interestingly, increased mutation load appears to be an inevitable outcome of an increase in body size.Author SummaryThe existence of gigantic animals presents an evolutionary puzzle. Larger animals have more cells and undergo exponentially more cell divisions, thus, they should have enormous rates of cancer. Moreover, large animals also have smaller populations making them vulnerable to extinction. So, how do gigantic animals such as elephants and blue whales protect themselves from cancer, and what are the consequences of evolving a large size on the ‘genetic health’ of a species? To address these questions we sequenced the genome of the capybara, the world’s largest rodent, and performed comparative genomic analyses to identify the genes and pathways involved in growth regulation and cancer suppression. We found that the insulin-signaling pathway was involved in the evolution of gigantism in the capybara. We also found a putative novel anticancer mechanism mediated by the detection of tumors by T-cells, offering a potential solution to how capybaras mitigated the tradeoff imposed by cancer. Furthermore, we show that capybara genome harbors a higher proportion of slightly deleterious mutations relative to all other rodent genomes. Overall, this study provides insights at the genomic level into the evolution of a complex and extreme phenotype, and offers a detailed picture of how the evolution of a giant body size in the capybara has shaped its genome.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/100-200-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/100-200-users/page/74/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../../tags/100-200-users/page/71/" class="page-link">71</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/100-200-users/page/72/" class="page-link">72</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/100-200-users/page/73/" class="page-link">73</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/100-200-users/page/74/" class="page-link">74</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/100-200-users/page/75/" class="page-link">75</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/100-200-users/page/76/" class="page-link">76</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/100-200-users/page/77/" class="page-link">77</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/100-200-users/page/78/" class="page-link">78</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/100-200-users/page/79/" class="page-link">79</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/100-200-users/page/76/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/100-200-users/page/126/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
