<!DOCTYPE html>
<html><head>
  
  <title>2018 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/variability_of_bacterial_behavior_in_the_mammalian_gut_captured_using_a_growth-linked_single-cell_synthetic_gene_oscillator_51342001/">
          <h4 class="font-weight-bold">Variability of bacterial behavior in the mammalian gut captured using a growth-linked single-cell synthetic gene oscillator, bioRxiv, 2018-11-17</h4>
          <p>AbstractThe dynamics of the bacterial population that comprises the gut microbiota plays key roles in overall mammalian health. However, a detailed understanding of bacterial growth within the gut is limited by the inherent complexity and inaccessibility of the gut environment. Here, we deploy an improved synthetic genetic oscillator to investigate dynamics of bacterial colonization and growth in the mammalian gut under both healthy and disease conditions. The synthetic oscillator, when introduced into both Escherichia coli and Salmonella Typhimurium maintains regular oscillations with a constant period in generations across growth conditions. We determine the phase of oscillation from individual bacteria using image analysis of resultant colonies and thereby infer the number of cell divisions elapsed. In doing so, we demonstrate robust functionality and controllability of the oscillator circuit’s activity during bacterial growth in vitro, in a simulated murine gut microfluidic environment, and in vivo within the mouse gut. We determine different dynamics of bacterial colonization and growth in the gut under normal and inflammatory conditions. Our results show that a precise genetic oscillator can function in a complex environment and reveal single cell behavior under diverse conditions where disease may create otherwise impossible-to-quantify variability in growth across the population.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/synthetic-biology">synthetic-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/epigenetically_reprogrammed_methylation_landscape_drives_the_dna_self-assembly_and_serves_as_a_universal_cancer_biomarker_52179027/">
          <h4 class="font-weight-bold">Epigenetically reprogrammed methylation landscape drives the DNA self-assembly and serves as a universal cancer biomarker, Nature Communications, 2018-11-15</h4>
          <p>Epigenetic reprogramming in cancer genomes creates a distinct methylation landscape encompassing clustered methylation at regulatory regions separated by large intergenic tracks of hypomethylated regions. This methylation landscape that we referred to as Methylscape is displayed by most cancer types, thus may serve as a universal cancer biomarker. To-date most research has focused on the biological consequences of DNA Methylscape changes whereas its impact on DNA physicochemical properties remains unexplored. Herein, we examine the effect of levels and genomic distribution of methylcytosines on the physicochemical properties of DNA to detect the Methylscape biomarker. We find that DNA polymeric behaviour is strongly affected by differential patterning of methylcytosine, leading to fundamental differences in DNA solvation and DNA-gold affinity between cancerous and normal genomes. We exploit these Methylscape differences to develop simple, highly sensitive and selective electrochemical or colorimetric one-step assays for the detection of cancer. These assays are quick, i.e., analysis time ≤10 minutes, and require minimal sample preparation and small DNA input.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/nature%20communications">nature communications</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/scaling_computational_genomics_to_millions_of_individuals_with_gpus_51176671/">
          <h4 class="font-weight-bold">Scaling computational genomics to millions of individuals with GPUs, bioRxiv, 2018-11-14</h4>
          <p>Current genomics methods were designed to handle tens to thousands of samples, but will soon need to scale to millions to keep up with the pace of data and hypothesis generation in biomedical science. Moreover, costs associated with processing these growing datasets will become prohibitive without improving the computational efficiency and scalability of methods. Here, we show that recently developed machine-learning libraries (TensorFlow and PyTorch) facilitate implementation of genomics methods for GPUs and significantly accelerate computations. To demonstrate this, we re-implemented methods for two commonly performed computational genomics tasks QTL mapping and Bayesian non-negative matrix factorization. Our implementations ran &amp;gt; 200 times faster than current CPU-based versions, and these analyses are ~5-10 fold cheaper on GPUs due to the vastly shorter runtimes. We anticipate that the accessibility of these libraries, and the improvements in run-time will lead to a transition to GPU-based implementations for a wide range of computational genomics methods.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/unbiased_detection_of_crispr_off-targets_in_vivo_using_discover-seq_51176196/">
          <h4 class="font-weight-bold">Unbiased detection of CRISPR off-targets in vivo using DISCOVER-Seq, bioRxiv, 2018-11-14</h4>
          <p>AbstractGenome editing using nucleases such as CRISPR-Cas induces programmable DNA damage at a target genomic site but can also affect off-target sites. Here, we develop a powerful, sensitive assay for the unbiased identification of off-target sites that we term DISCOVER-Seq. This approach takes advantage of the recruitment of endogenous DNA repair factors for genome-wide identification of Cas-induced double-strand breaks. One such factor, MRE11, is recruited precisely to double-strand breaks, enabling molecular characterization of nuclease cut sites with single-base resolution. DISCOVER-Seq detects off-targets in cellular models and in vivo upon adenoviral gene editing of mouse livers, paving the way for real-time off-target discovery during therapeutic gene editing. DISCOVER-Seq is furthermore applicable to multiple types of Cas nucleases and provides an unprecedented view of events that precede repair of the affected sites.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/assembly_of_a_pan-genome_from_deep_sequencing_of_910_humans_of_african_descent_51402455/">
          <h4 class="font-weight-bold">Assembly of a pan-genome from deep sequencing of 910 humans of African descent, Nature Genetics, 2018-11-13</h4>
          <p>We used a deeply sequenced dataset of 910 individuals, all of African descent, to construct a set of DNA sequences that is present in these individuals but missing from the reference human genome. We aligned 1.19 trillion reads from the 910 individuals to the reference genome (GRCh38), collected all reads that failed to align, and assembled these reads into contiguous sequences (contigs). We then compared all contigs to one another to identify a set of unique sequences representing regions of the African pan-genome missing from the reference genome. Our analysis revealed 296,485,284 bp in 125,715 distinct contigs present in the populations of African descent, demonstrating that the African pan-genome contains ~10% more DNA than the current human reference genome. Although the functional significance of nearly all of this sequence is unknown, 387 of the novel contigs fall within 315 distinct protein-coding genes, and the rest appear to be intergenic.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/nature%20genetics">nature genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/single_particle_cryo-em_reconstruction_of_52_kda_streptavidin_at_3.2_angstrom_resolution_51142456/">
          <h4 class="font-weight-bold">Single particle cryo-EM reconstruction of 52 kDa streptavidin at 3.2 Angstrom resolution, bioRxiv, 2018-11-13</h4>
          <p>AbstractThe fast development of single particle cryo-EM has made it more feasible to obtain the 3D structure of well-behaved macromolecules with molecular weight higher than 300 kDa at ~3 Å resolution. It remains a challenge to obtain high resolution structure of molecules smaller than 100 kDa using single particle cryo-EM, mainly due to the low contrast of the molecules embedded in vitreous ice. In this work, we applied the Cs-corrector-VPP coupled cryo-EM to study 52 kDa streptavidin (SA) protein supported on a thin layer of graphene film and embedded in vitreous ice. We were able to solve both the apo-SA and biotin-bound SA at near-atomic resolution using single particle cryo-EM. We demonstrated that the method is capable to determine the structure of molecule as small as 39 kDa and potentially even smaller molecules. Furthermore, we found that using the graphene film to avoid the adsorption to the air-water interface is critical to maintain the protein’s high-resolution structural information.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/14/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/11/" class="page-link">11</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/12/" class="page-link">12</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/13/" class="page-link">13</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/14/" class="page-link">14</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/2018/page/15/" class="page-link">15</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/16/" class="page-link">16</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/17/" class="page-link">17</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/18/" class="page-link">18</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/19/" class="page-link">19</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/16/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/84/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
