<!DOCTYPE html>
<html><head>
  
  <title>Reports | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/high-resolution_structure_determination_of_sub-100_kilodalton_complexes_using_conventional_cryo-em_52341824/">
          <h4 class="font-weight-bold">High-resolution structure determination of sub-100 kilodalton complexes using conventional cryo-EM, bioRxiv, 2018-12-07</h4>
          <p>Determining high-resolution structures of biological macromolecules with masses of less than 100 kilodaltons (kDa) has long been a goal of the cryo-electron microscopy (cryo-EM) community. While the Volta Phase Plate has enabled cryo-EM structure determination of biological specimens of this size range, use of this instrumentation is not yet fully automated and can present technical challenges. Here, we show that conventional defocus-based cryo-EM methodologies can be used to determine the high-resolution structures of specimens amassing less than 100 kDa using a transmission electron microscope operating at 200 keV coupled with a direct electron detector. Our ~2.9 Å structure of alcohol dehydrogenase (82 kDa) proves that bound ligands can be resolved with high fidelity, indicating that these methodologies can be used to investigate the molecular details of drug-target interactions. Our ~2.8 Å and ~3.2 Å resolution structures of methemoglobin demonstrate that distinct conformational states can be identified within a dataset for proteins as small as 64 kDa. Furthermore, we provide the first sub-nanometer cryo-EM structure of a protein smaller than 50 kDa.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/a_comparison_of_gene_expression_and_dna_methylation_patterns_across_tissues_and_species_52189381/">
          <h4 class="font-weight-bold">A comparison of gene expression and DNA methylation patterns across tissues and species, bioRxiv, 2018-12-05</h4>
          <p>AbstractPreviously published comparative functional genomic data sets from primates using frozen tissue samples, including many data sets from our own group, were collected and analyzed using non-optimal study designs and analysis approaches. In addition, when samples from multiple tissues were studied in a comparative framework, individual and tissue were confounded. We designed a multi-tissue comparative study of gene expression and DNA methylation in primates that minimizes confounding effects by using a balanced design with respect to species, tissues, and individuals. We also developed a comparative analysis pipeline that minimizes biases due to sequence divergence. We thus present the most comprehensive catalog of similarities and differences in gene expression and methylation levels between livers, kidneys, hearts, and lungs, in humans, chimpanzees, and rhesus macaques. We estimate that overall, only between 7 to 11% (depending on the tissue) of inter-species differences in gene expression levels can be accounted for by corresponding differences in promoter DNA methylation. However, gene expression divergence in conserved tissue-specific genes can be explained by corresponding inter-species methylation changes more often. We end the paper by providing recommendations for effective study design and best practices for meta-data recording for comparative functional genomic studies in primates.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/natural_selection_contributed_to_immunological_differences_between_human_hunter-gatherers_and_agriculturalists_52201024/">
          <h4 class="font-weight-bold">Natural selection contributed to immunological differences between human hunter-gatherers and agriculturalists, bioRxiv, 2018-12-05</h4>
          <p>AbstractThe shift from a hunter-gatherer (HG) to an agricultural (AG) mode of subsistence is believed to have been associated with profound changes in the burden and diversity of pathogens across human populations. Yet, the extent to which the advent of agriculture may have impacted the evolution of the human immune system remains unknown. Here we present a comparative study of variation in the transcriptional responses of peripheral blood mononuclear cells to bacterial and viral stimuli between Batwa rainforest hunter-gatherers and Bakiga agriculturalists from Uganda. We observed increased divergence between hunter-gatherers and agriculturalists in the transcriptional response to viruses compared to that for bacterial stimuli. We demonstrate that a significant fraction of these transcriptional differences are under genetic control, and we show that positive natural selection has helped to shape population differences in immune regulation. Across the set of genetic variants underlying inter-population immune response differences, however, the signatures of positive selection were disproportionately observed in the rainforest hunter-gatherers. This result is counter to expectations based on the popularized notion that shifts in pathogen exposure due to the advent of agriculture imposed radically heightened selective pressures in agriculturalist populations.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/new_methods_to_calculate_concordance_factors_for_phylogenomic_datasets_52232751/">
          <h4 class="font-weight-bold">New methods to calculate concordance factors for phylogenomic datasets, bioRxiv, 2018-12-05</h4>
          <p>Summary We introduce and implement two measures for quantifying genealogical concordance in phylogenomic datasets the gene concordance factor (gCF) and the site concordance factor (sCF). For every branch of a reference tree, gCF is defined as the percentage of decisive gene trees containing that branch. This measure is already in wide usage, but here we introduce a package that calculates it while accounting for variable taxon coverage among gene trees. sCF is a new measure defined as the percentage of decisive sites supporting a branch in the reference tree. gCF and sCF complement classical measures of branch support in phylogenetics by providing a full description of underlying disagreement among loci and sites. Availability and Implementation An easy to use implementation and tutorial is freely available in the IQ-TREE software (httpwww.iqtree.org). Supplementary information Data are available at httpsdoi.org10.5281zenodo.1949290</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/posture_controls_mechanical_tuning_in_the_black_widow_spider_mechanosensory_system_52202031/">
          <h4 class="font-weight-bold">Posture controls mechanical tuning in the black widow spider mechanosensory system, bioRxiv, 2018-12-05</h4>
          <p>Spiders rely on mechanical vibration sensing for sexual signalling, prey capture and predator evasion. The sensory organs underlying vibration detection, called slit sensilla, resemble cracks in the spider&#39;s exoskeleton, and are distributed all over the spider body. Those crucial to sensing web- and other substrate-borne vibrations are called lyriform organs and are densely distributed around leg joints. It has been shown that forces that cause bending at leg joints also activate these lyriform organs. Little is known of how the biomechanics of the body of a freely-suspended spider in its natural posture interact with vibrations introduced into the body and how this affects vibration perception. Female black widow spiders, in particular, have a striking body-form; their long thin legs support a large pendulous abdomen. Here, we show that in their natural posture, the large abdominal mass of black widow females, interacts with the spring-like behaviour of their leg joints and determines the mechanical behaviour of different leg joints. Furthermore, we find that adopting different body postures enables females to alter both the level and tuning of the mechanical input to lyriform organs. Therefore, we suggest that posture may be used to flexibly and reversibly focus attention to different classes or components of web vibration. Postural effects thus emphasize the dynamic loop of interactions between behaviour and perception, i.e. between &#39;brain&#39; and body.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/animal-behavior-and-cognition">animal-behavior-and-cognition</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/geographic_variation_and_bias_in_polygenic_scores_of_complex_diseases_and_traits_in_finland_52182179/">
          <h4 class="font-weight-bold">Geographic variation and bias in polygenic scores of complex diseases and traits in Finland, bioRxiv, 2018-12-04</h4>
          <p>AbstractPolygenic scores (PS) are becoming a useful tool to identify individuals with high genetic risk for complex diseases and several projects are currently testing their utility for translational applications. It is also tempting to use PS to assess whether genetic variation can explain a part of the geographic distribution of a phenotype. However, it is not well known how population genetic properties of the training and target samples affect the geographic distribution of PS. Here, we evaluate geographic differences, and related biases, of PS in Finland with geographically well-defined sample of 2,376 individuals from the National FINRISK study. First, we detect geographic differences in PS for coronary artery disease (CAD), rheumatoid arthritis, schizophrenia, waits-hip ratio (WHR), body-mass index (BMI) and height, but not for Crohn’s disease or ulcerative colitis. Second, we use height as a model trait to thoroughly assess the possible population genetic biases in PS and apply similar approaches to the other phenotypes. Most importantly, we detect suspiciously large accumulation of geographic differences for CAD, WHR, BMI and height, suggesting bias arising from population genetic structure rather than from a direct genotype-phenotype association. This work demonstrates how sensitive the geographic patterns of current PS are for small biases even within relatively homogenous populations and provides simple tools to identify such biases. A thorough understanding of the effects of population genetic structure on PS is essential for translational applications of PS.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/reports/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/reports/page/91/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/reports/page/88/" class="page-link">88</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/89/" class="page-link">89</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/90/" class="page-link">90</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/91/" class="page-link">91</a></li>
            
        
             
            <li class="page-item active"><a href="/reports/page/92/" class="page-link">92</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/93/" class="page-link">93</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/94/" class="page-link">94</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/95/" class="page-link">95</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/96/" class="page-link">96</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/reports/page/93/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/reports/page/172/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
