<!DOCTYPE html>
<html><head>
  
  <title>2018 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/base_editing_generates_substantial_off-target_single_nucleotide_variants_51825046/">
          <h4 class="font-weight-bold">Base editing generates substantial off-target single nucleotide variants, bioRxiv, 2018-11-27</h4>
          <p>AbstractGenome editing tools including CRISPRCas9 and base editors hold great promise for correcting pathogenic mutations. Unbiased genome-wide off-target effects of the editing in mammalian cells is required before clinical applications, but determination of the extent of off-target effects has been difficult due to the existence of single nucleotide polymorphisms (SNPs) in individuals. Here, we developed a method named GOTI (Genome-wide Off-target analysis by Two-cell embryo Injection) to detect off-target mutations without interference of SNPs. We applied GOTI to both the CRISPR-Cas9 and base editing (BE3) systems by editing one blastomere of the two-cell mouse embryo and then compared whole genome sequences of progeny-cell populations at E14.5 stage. Sequence analysis of edited and non-edited cell progenies showed that undesired off-target single nucleotide variants (SNVs) are rare (average 10.5) in CRISPR-edited mouse embryos, with a frequency close to the spontaneous mutation rate. By contrast, BE3 editing induced over 20-fold higher SNVs (average 283), raising the concern of using base-editing approaches for biomedical application.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/climate_or_disturbance_temperate_forest_structural_change_and_carbon_sink_potential_51853375/">
          <h4 class="font-weight-bold">Climate or disturbance temperate forest structural change and carbon sink potential, bioRxiv, 2018-11-27</h4>
          <p>ABSTRACTAnticipating forest responses to changing climate and disturbance regimes requires understanding long-term successional processes and aggregating these local processes into global relevance. Estimates of existing forest structure and biomass are improving globally; however, vegetation models continue to show substantial spread in predictions of future land carbon uptake and the roles of forest structural change and demography are increasingly being recognized as important. To identify mechanisms that drive change in tree size, density, and carbon, we need a better understanding of forest structural trajectories and the factors that affect those trajectories. Here we reveal a coherent, cyclic pattern of structural change in temperate forests, as predicted by successional theory, and identify significant sensitivity to climatic precipitation and temperature anomalies using large datasets and empirical modeling. For example, in the eastern US above average temperature (&#43;1°C) was associated with a 27% (−0.4±0.1 Mg C ha-1 yr-1) reduction in productivity attributed to higher rates of disease (&#43;23%), weather disturbance (&#43;57%), and sapling mortality. Projections of future vegetative carbon sink potential suggests biomass would be lowest on managed lands (72±2 Mg C ha-1) and highest when larger trees survive in undisturbed conditions (153±21 Mg C ha-1). Overall, the indirect effects of disturbance and mortality were considerably larger than the direct effects of climate on productivity when predicting future vegetative carbon sinks. Results provide robust comparisons for global vegetation models, and valuable projections for management and carbon mitigation efforts.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/naught_all_zeros_in_sequence_count_data_are_the_same_51782152/">
          <h4 class="font-weight-bold">Naught all zeros in sequence count data are the same, bioRxiv, 2018-11-27</h4>
          <p>AbstractDue to the advent and utility of high-throughput sequencing, modern biomedical research abounds with multivariate count data. Yet such sequence count data is often extremely sparse; that is, much of the data is zero values. Such zero values are well known to cause problems for statistical analyses. In this work we provide a systematic description of different processes that can give rise to zero values as well as the types of methods for addressing zeros in sequence count studies. Importantly, we systematically review how various models perform on each type of zero generating process. Our results demonstrate that zero-inflated models can have substantial biases in both simulated and real data settings. Additionally, we find that zeros due to biological absences can, for many applications, be approximated as originating from under sampling. Beyond these results, this work provides a paired categorization scheme for models and zero generating processes to facilitate discussions and future research into the analysis of sequence count data.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/quantitative_pcr_provides_a_simple_and_accessible_method_for_quantitative_microbiome_profiling_51829165/">
          <h4 class="font-weight-bold">Quantitative PCR provides a simple and accessible method for quantitative microbiome profiling, bioRxiv, 2018-11-27</h4>
          <p>AbstractThe use of relative next generation sequencing (NGS) abundance data can lead to misinterpretations of microbial community structures as the increase of one taxon leads to concurrent decrease of the other(s). To overcome compositionality, we provide a quantitative NGS solution, which is achieved by adjusting the relative 16S rRNA gene amplicon NGS data with quantitative PCR (qPCR-based) total bacterial counts. By comparing the enumeration of dominant bacterial groups on different taxonomic levels in human fecal samples using taxon-specific 16S rRNA gene-targeted qPCR we show that quantitative NGS is able to estimate absolute bacterial abundances accurately. We also observed a higher degree of correspondence in the estimated microbe-metabolite relationship when quantitative NGS was applied. Being conceptually and methodologically analogous to amplicon-based NGS, our qPCR-based method can be readily incorporated into the standard, high-throughput NGS sample processing pipeline for more accurate description of interactions within and between the microbes and host.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/statistical_physics_of_liquid_brains_51780511/">
          <h4 class="font-weight-bold">Statistical physics of liquid brains, bioRxiv, 2018-11-27</h4>
          <p>Liquid neural networks (or “liquid brains”) are a widespread class of cognitive living networks characterised by a common feature the agents (ants or immune cells, for example) move in space. Thus, no fixed, long-term agent-agent connections are maintained, in contrast with standard neural systems. How is this class of systems capable of displaying cognitive abilities, from learning to decision-making? In this paper, the collective dynamics, memory and learning properties of liquid brains is explored under the perspective of statistical physics. Using a comparative approach, we review the generic properties of three large classes of systems, namely standard neural networks (“solid brains”), ant colonies and the immune system. It is shown that, despite their intrinsic physical differences, these systems share key properties with standard neural systems in terms of formal descriptions, but strongly depart in other ways. On one hand, the attractors found in liquid brains are not always based on connection weights but instead on population abundances. However, some liquid systems use fluctuations in ways similar to those found in cortical networks, suggesting a relevant role of criticality as a way of rapidly reacting to external signals.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/using_deeplabcut_for_3d_markerless_pose_estimation_across_species_and_behaviors_51658173/">
          <h4 class="font-weight-bold">Using DeepLabCut for 3D markerless pose estimation across species and behaviors, bioRxiv, 2018-11-26</h4>
          <p>Noninvasive behavioral tracking of animals during experiments is crucial to many scientific pursuits. Extracting the poses of animals without using markers is often essential for measuring behavioral effects in biomechanics, genetics, ethology &amp;amp; neuroscience. Yet, extracting detailed poses without markers in dynamically changing backgrounds has been challenging. We recently introduced an open source toolbox called DeepLabCut that builds on a state-of-the-art human pose estimation algorithm to allow a user to train a deep neural network using limited training data to precisely track user-defined features that matches human labeling accuracy. Here, with this paper we provide an updated toolbox that is self contained within a Python package that includes new features such as graphical user interfaces and active-learning based network refinement. Lastly, we provide a step-by-step guide for using DeepLabCut.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/2018/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/2018/page/10/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/10/" class="page-link">10</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/2018/page/11/" class="page-link">11</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/12/" class="page-link">12</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/13/" class="page-link">13</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/14/" class="page-link">14</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/15/" class="page-link">15</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/2018/page/12/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/2018/page/84/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
