<!DOCTYPE html>
<html><head>
  
  <title>Genomics | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/mutational_signatures_are_jointly_shaped_by_dna_damage_and_repair_62847858/">
          <h4 class="font-weight-bold">Mutational signatures are jointly shaped by DNA damage and repair, bioRxiv, 2019-06-29</h4>
          <p>SummaryMutations arise when DNA lesions escape DNA repair. To delineate the contributions of DNA damage and DNA repair deficiency to mutagenesis we sequenced 2,721 genomes of 54 C. elegans strains, each deficient for a specific DNA repair gene and wild-type, upon exposure to 12 different genotoxins. Combining genotoxins and repair deficiency leads to differential mutation rates or new mutational signatures in more than one third of experiments. Translesion synthesis polymerase deficiencies show dramatic and diverging effects. Knockout of Polκ dramatically exacerbates the mutagenicity of alkylating agents; conversely, Polζ deficiency reduces alkylation- and UV-induced substitution rates. Examples of DNA damage-repair deficiency interactions are also found in cancer genomes, although cases of hypermutation are surprisingly rare despite signs of positive selection in a number of DNA repair genes. Nevertheless, cancer risk may be substantially elevated even by small increases in mutagenicity according to evolutionary multi-hit theory. Overall, our data underscore how mutagenesis is a joint product of DNA damage and DNA repair, implying that mutational signatures may be more variable than currently anticipated.Highlights&lt;jatslist list-type=bullet&gt;&lt;jatslist-item&gt;Combining exposure to DNA damaging agents and DNA repair deficiency in C. elegans leads to altered mutation rates and new mutational signatures&lt;jatslist-item&gt;&lt;jatslist-item&gt;Mutagenic effects of genotoxic exposures are generally exacerbated by DNA repair deficiency&lt;jatslist-item&gt;&lt;jatslist-item&gt;Mutagenesis of UVB and alkylating agents is reduced in translesion synthesis polymerase deficiencies&lt;jatslist-item&gt;&lt;jatslist-item&gt;Human cancer genomes contain examples of DNA damagerepair interactions, but mutations in DNA repair genes usually only associate with moderate mutator phenotypes, in line with evolutionary theory&lt;jatslist-item&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/insights_into_human_genetic_variation_and_population_history_from_929_diverse_genomes_62783252/">
          <h4 class="font-weight-bold">Insights into human genetic variation and population history from 929 diverse genomes, bioRxiv, 2019-06-28</h4>
          <p>AbstractGenome sequences from diverse human groups are needed to understand the structure of genetic variation in our species and the history of, and relationships between, different populations. We present 929 high-coverage genome sequences from 54 diverse human populations, 26 of which are physically phased using linked-read sequencing. Analyses of these genomes reveal an excess of previously undocumented private genetic variation in southern and central Africa and in Oceania and the Americas, but an absence of fixed, private variants between major geographical regions. We also find deep and gradual population separations within Africa, contrasting population size histories between hunter-gatherer and agriculturalist groups in the last 10,000 years, a potentially major population growth episode after the peopling of the Americas, and a contrast between single Neanderthal but multiple Denisovan source populations contributing to present-day human populations. We also demonstrate benefits to the study of population relationships of genome sequences over ascertained array genotypes. These genome sequences are freely available as a resource with no access or analysis restrictions.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/radicl-seq_identifies_general_and_cell_type-specific_principles_of_genome-wide_rna-chromatin_interactions_62782165/">
          <h4 class="font-weight-bold">RADICL-seq identifies general and cell type-specific principles of genome-wide RNA-chromatin interactions, bioRxiv, 2019-06-28</h4>
          <p>AbstractMammalian genomes encode tens of thousands of noncoding RNAs. Most noncoding transcripts exhibit nuclear localization and several have been shown to play a role in the regulation of gene expression and chromatin remodelling. To investigate the function of such RNAs, methods to massively map the genomic interacting sites of multiple transcripts have been developed. However, they still present some limitations. Here, we introduce RNA And DNA Interacting Complexes Ligated and sequenced (RADICL-seq), a technology that maps genome-wide RNA-chromatin interactions in intact nuclei. RADICL-seq is a proximity ligation-based methodology that reduces the bias for nascent transcription, while increasing genomic coverage and unique mapping rate efficiency compared to existing methods. RADICL-seq identifies distinct patterns of genome occupancy for different classes of transcripts as well as cell type-specific RNA-chromatin interactions, and emphasizes the role of transcription in the establishment of chromatin structure.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/the_mutational_footprints_of_cancer_therapies_62782167/">
          <h4 class="font-weight-bold">The mutational footprints of cancer therapies, bioRxiv, 2019-06-28</h4>
          <p>Some cancer therapies damage DNA and cause mutations both in cancer and healthy cells of the patient1. These therapy-induced mutations may underlie some of the long-term and late side effects of the treatment, such as mental disabilities, organ toxicities and secondary neoplasms. Currently we ignore the mutation pattern and burden caused by different cancer treatments. Here we identify mutational signatures, or footprints of six widely-used anti-cancer therapies with the study of whole-genomes from more than 3500 metastatic tumors originated in different organs. These include previously known and new mutational signatures generated by platinum-based drugs, and a novel signature of treatment with nucleoside metabolic inhibitors. Exploiting these mutational footprints, we estimate the contribution of different treatments to the mutation burden of tumors and their risk of causing coding and likely driver mutations in the genome. In summary, the mutational footprints identified here open a window to precisely appraise the mutational risk of different cancer therapies to understand their late side effects.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/the_genomic_impact_of_european_colonization_of_the_americas_62499079/">
          <h4 class="font-weight-bold">The genomic impact of European colonization of the Americas, bioRxiv, 2019-06-21</h4>
          <p>AbstractThe human genetic diversity of the Americas has been shaped by several events of gene flow that have continued since the Colonial Era and the Atlantic slave trade. Moreover, multiple waves of migration followed by local admixture occurred in the last two centuries, the impact of which has been largely unexplored.Here we compiled a genome-wide dataset of ∼12,000 individuals from twelve American countries and ∼6,000 individuals from worldwide populations and applied haplotype-based methods to investigate how historical movements from outside the New World affected i) the genetic structure, ii) the admixture profile, iii) the demographic history and iv) sex-biased gene-flow dynamics, of the Americas.We revealed a high degree of complexity underlying the genetic contribution of European and African populations in North and South America, from both geographic and temporal perspectives, identifying previously unreported sources related to Italy, the Middle East and to specific regions of Africa.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/passenger_hotspot_mutations_in_cancer_62409958/">
          <h4 class="font-weight-bold">Passenger Hotspot Mutations in Cancer, bioRxiv, 2019-06-19</h4>
          <p>AbstractHotspots, or mutations that recur at the same genomic site across multiple tumors, have been conventionally interpreted as strong universal evidence of somatic positive selection, unequivocally pinpointing genes driving tumorigenesis. Here, we demonstrate that this convention is falsely premised on an inaccurate statistical model of background mutagenesis. Many hotspots are in fact passenger events, recurring at sites that are simply inherently more mutable rather than under positive selection, which current background models do not account for. We thus detail a log-normal-Poisson (LNP) background model that accounts for variation in site-specific mutability in a manner consistent with models of mutagenesis, use this model to show that the tendency to generate passenger hotspots pervades all common mutational processes, and apply it to a ~10, 000 patient cohort from The Cancer Genome Atlas to nominate driver hotspots with far fewer false positives compared to conventional methods. As the biomedical community faces critical decisions in prioritizing putative driver mutations for deep experimental characterization to assess therapeutic potential, we offer our findings as a guide to avoid wasting valuable scientific resources on passenger hotspots.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        

        
             
            <li class="page-item active"><a href="/tags/genomics/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/genomics/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/genomics/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/genomics/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/genomics/page/5/" class="page-link">5</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/genomics/page/2/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/genomics/page/29/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
