<!DOCTYPE html>
<html><head>
  
  <title>Plant Biology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/comparison_of_efficiency_and_specificity_of_crispr-associated_cas_nucleases_in_plants_an_expanded_toolkit_for_precision_genome_engineering_48724730/">
          <h4 class="font-weight-bold">Comparison of Efficiency and Specificity of CRISPR-Associated (Cas) Nucleases in Plants An Expanded Toolkit for Precision Genome Engineering, bioRxiv, 2018-09-21</h4>
          <p>Molecular tools adapted from bacterial CRISPR (Clustered Regulatory Interspaced Short Palindromic Repeats) systems for adaptive immunity have become widely used for plant genome engineering, both to investigate gene functions and to engineer desirable traits. A number of different Cas (CRISPR-associated) nucleases are now used but, as most studies performed to date have engineered different targets using a variety of plant species and molecular tools, it has been difficult to draw conclusions about the comparative performance of different nucleases. Due to the time and effort required to regenerate engineered plants, efficiency is critical. In addition, there have been several reports of mutations at sequences with less than perfect identity to the target. While in some plant species it is possible to remove these so-called ‘off-targets’ by backcrossing to a parental line, the specificity of genome engineering tools is important when targeting specific members of closely-related gene families, especially when recent paralogues are co-located in the genome and unlikely to segregate. Specificity is also important for species that take years to reach sexual maturity or that are clonally propagated. Here, we directly compare the efficiency and specificity of Cas nucleases from different bacterial species together with engineered variants of Cas9. We find that the nucleotide content correlates with efficiency and that Cas9 from Staphylococcus aureus is comparatively most efficient at inducing mutations. We also demonstrate that ‘high-fidelity’ variants of Cas9 can reduce off-target mutations in plants. We present these molecular tools as standardised DNA parts to facilitate their re-use.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/optimization_of_t-dna_architecture_for_cas9-mediated_mutagenesis_in_arabidopsis_48543705/">
          <h4 class="font-weight-bold">Optimization of T-DNA architecture for Cas9-mediated mutagenesis in Arabidopsis, bioRxiv, 2018-09-17</h4>
          <p>ABSTRACTBacterial CRISPR systems have been widely adopted to create operator-specified site-specific nucleases. Such nuclease action commonly results in loss-of-function alleles, facilitating functional analysis of genes and gene families We conducted a systematic comparison of components and T-DNA architectures for CRISPR-mediated gene editing in Arabidopsis, testing multiple promoters, terminators, sgRNA backbones and Cas9 alleles. We identified a T-DNA architecture that usually results in stable (i.e. homozygous) mutations in the first generation after transformation. Notably, the transcription of sgRNA and Cas9 in head-to-head divergent orientation usually resulted in highly active lines. Our Arabidopsis data may prove useful for optimization of CRISPR methods in other plants.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/high-throughput_single-cell_transcriptome_profiling_of_plant_cell_types_47225996/">
          <h4 class="font-weight-bold">High-throughput single-cell transcriptome profiling of plant cell types, bioRxiv, 2018-08-29</h4>
          <p>AbstractSingle-cell transcriptome analysis of heterogeneous tissues can provide high-resolution windows into the genomic basis and spatiotemporal dynamics of developmental processes. Here we demonstrate the feasibility of high-throughput single-cell RNA sequencing of plant tissue using the Drop-seq approach. Profiling of &amp;gt;4,000 individual cells from the Arabidopsis root provides transcriptomes and marker genes for a diversity of cell types and illuminates the gene expression changes that occur across endodermis development.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/speed_breeding_in_growth_chambers_and_glasshouses_for_crop_breeding_and_model_plant_research_45092716/">
          <h4 class="font-weight-bold">Speed breeding in growth chambers and glasshouses for crop breeding and model plant research, bioRxiv, 2018-07-16</h4>
          <p>1.AbstractTo meet the challenge of feeding a growing population, breeders and scientists are continuously looking for ways to increase genetic gain in crop breeding. One way this can be achieved is through “speed breeding” (SB), which shortens the breeding cycle and accelerates research studies through rapid generation advancement. The SB method can be carried out in a number of ways, one of which involves extending the duration of a plant’s daily exposure to light (photoperiod) combined with early seed harvest in order to cycle quickly from seed to seed, thereby reducing the generation times for some long-day (LD) or day-neutral crops. Here we present glasshouse and growth chamber-based SB protocols with supporting data from experimentation with several crop species. These protocols describe the growing conditions, including soil media composition, lighting, temperature and spacing, which promote rapid growth of spring and winter bread wheat, durum wheat, barley, oat, various members of the Brassica family, chickpea, pea, grasspea, quinoa and the model grass Brachypodium distachyon. Points of flexibility within the protocols are highlighted, including how plant density can be increased to efficiently scale-up plant numbers for single seed descent (SSD) purposes. Conversely, instructions on how to perform SB on a small-scale by creating a benchtop SB growth cabinet that enables optimization of parameters at a low cost are provided. We also outline the procedure for harvesting and germinating premature wheat, barley and pea seed to reduce generation time. Finally, we provide troubleshooting suggestions to avoid potential pitfalls.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genetic_diversity_patterns_and_domestication_origin_of_soybean_45029693/">
          <h4 class="font-weight-bold">Genetic Diversity Patterns and Domestication Origin of Soybean, bioRxiv, 2018-07-14</h4>
          <p>AbstractUnderstanding diversity and evolution of a crop is an essential step to implement a strategy to expand its germplasm base for crop improvement research. Samples intensively collected from Korea, which is a small but central region in the distribution geography of soybean, were genotyped to provide sufficient data to underpin genome-wide population genetic questions. After removing natural hybrids and duplicated or redundant accessions, we obtained a non-redundant set comprising 1,957 domesticated and 1,079 wild accessions to perform population structure analyses. Our analysis demonstrates that while wild soybean germplasm will require additional sampling from diverse indigenous areas to expand the germplasm base, the current domesticated soybean germplasm is saturated in terms of genetic diversity. We then showed that our genome-wide polymorphism map enabled us to detect genetic loci underling flower color, seed-coat color, and domestication syndrome. A representative soybean set consisting of 194 accessions were divided into one domesticated subpopulation and four wild subpopulations that could be traced back to their geographic collection areas. Population genomics analyses suggested that the monophyletic group of domesticated soybeans was originated in eastern Japan. The results were further substantiated by a phylogenetic tree constructed from domestication-associated single nucleotide polymorphisms identified in this study.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/crispr-cas9_interference_in_cassava_linked_to_the_evolution_of_editing-resistant_geminiviruses_40711777/">
          <h4 class="font-weight-bold">CRISPR-Cas9 interference in cassava linked to the evolution of editing-resistant geminiviruses, bioRxiv, 2018-05-04</h4>
          <p>ABSTRACTWe used CRISPR-Cas9 in the staple food crop cassava with the aim of engineering resistance to African cassava mosaic virus, a member of a widespread and important family of plant-pathogenic DNA viruses. We found that between 33 and 48% of edited virus genomes evolved a conserved single-nucleotide mutation that confers resistance to CRISPR-Cas9 cleavage. Our study highlights the potential for virus escape from this technology. Care should be taken to design CRISPR-Cas9 experiments that minimize the risk of virus escape.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/5/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/5/" class="page-link">5</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/8/" class="page-link">8</a></li>
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/7/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/plant-biology/page/8/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
