<!DOCTYPE html>
<html><head>
  
  <title>Systems Biology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/full-length_mrna_sequencing_reveals_principles_of_polya_tail_length_control_55318571/">
          <h4 class="font-weight-bold">Full-length mRNA sequencing reveals principles of poly(A) tail length control, bioRxiv, 2019-02-12</h4>
          <p>Although mRNAs are key molecules for understanding life, there exists no method to determine the full-length sequence of endogenous mRNAs including their poly(A) tails. Moreover, although poly(A) tails can be modified in functionally important ways, there also exists no method to accurately sequence them. Here, we present FLAM-seq, a rapid and simple method for high-quality sequencing of entire mRNAs. We report a cDNA library preparation method coupled to single-molecule sequencing to perform FLAM-seq. Using human cell lines, brain organoids, and C. elegans we show that FLAM-seq delivers high-quality full-length mRNA sequences for thousands of different genes per sample. We find that (a) 3&#39; UTR length is correlated with poly(A) tail length, (b) alternative polyadenylation sites and alternative promoters for the same gene are linked to different tail lengths, (c) tails contain a significant number of cytosines. Thus, we provide a widely useful method and fundamental insights into poly(A) tail regulation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/short-range_interactions_govern_cellular_dynamics_in_microbial_multi-genotype_systems_54544166/">
          <h4 class="font-weight-bold">Short-range interactions govern cellular dynamics in microbial multi-genotype systems, bioRxiv, 2019-01-27</h4>
          <p>Ecosystem processes result from interaction between organisms. When interactions are local, the spatial organization of organisms defines their network of interactions, and thus influences the system&#39;s functioning. This can be especially relevant for microbial systems, which often consist of spatially structured communities of cells connected by a dense interaction network. Here we measured the spatial interaction network between cells in microbial systems and identify the factors that determine it. Combining quantitative single-cell analysis of synthetic bacterial communities with mathematical modeling, we find that cells only interact with other cells in their immediate neighbourhood. This short interaction range impacts the functioning of the whole system by reducing its ability to perform metabolic processes collectively. Our experiments and models demonstrate that the spatial scale of cell-to-cell interaction plays a fundamental role in understanding and controlling natural communities, and in engineering microbial systems for specific purposes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/lineage_tracing_on_transcriptional_landscapes_links_state_to_fate_during_differentiation_51070913/">
          <h4 class="font-weight-bold">Lineage tracing on transcriptional landscapes links state to fate during differentiation, bioRxiv, 2018-11-11</h4>
          <p>AbstractA challenge in stem cell biology is to associate molecular differences among progenitor cells with their capacity to generate mature cell types. Though the development of single cell assays allows for the capture of progenitor cell states in great detail, these assays cannot definitively link cell states to their long-term fate. Here, we use expressed DNA barcodes to clonally trace single cell transcriptomes dynamically during differentiation and apply this approach to the study of hematopoiesis. Our analysis identifies functional boundaries of cell potential early in the hematopoietic hierarchy and locates them on a continuous transcriptional landscape. We reconstruct a developmental hierarchy showing separate ontogenies for granulocytic subtypes and two routes to monocyte differentiation that leave a persistent imprint on mature cells. Finally, we use our approach to benchmark methods of dynamic inference from single-cell snapshots, and provide evidence of strong early fate biases dependent on cellular properties hidden from single-cell RNA sequencing.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/charting_a_tissue_from_single-cell_transcriptomes_50502918/">
          <h4 class="font-weight-bold">Charting a tissue from single-cell transcriptomes, bioRxiv, 2018-10-30</h4>
          <p>AbstractMassively multiplexed sequencing of RNA in individual cells is transforming basic and clinical life sciences. However, in standard experiments, tissues must first be dissociated. Thus, after sequencing, information about the spatial relationships between cells is lost although this knowledge is crucial for understanding cellular and tissue-level function. Recent attempts to overcome this fundamental challenge rely on employing additional in situ gene expression imaging data which can guide spatial mapping of sequenced cells. Here we present a conceptually different approach that allows to reconstruct spatial positions of cells in a variety of tissues without using reference imaging data. We first show for several complex biological systems that distances of single cells in expression space monotonically increase with their physical distances across tissues. We therefore seek to map cells to tissue space such that this principle is optimally preserved, while matching existing imaging data when available. We show that this optimization problem can be cast as a generalized optimal transport problem and solved efficiently. We apply our approach successfully to reconstruct the mammalian liver and intestinal epithelium as well as fly and zebrafish embryos. Our results demonstrate a simple spatial expression organization principle and that this principle (or future refined principles) can be used to infer, for individual cells, meaningful spatial position probabilities from the sequencing data alone.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_interaction_landscape_between_transcription_factors_and_the_nucleosome_31012336/">
          <h4 class="font-weight-bold">The interaction landscape between transcription factors and the nucleosome, bioRxiv, 2017-12-29</h4>
          <p>Nucleosomes cover most of the genome and are thought to be displaced by transcription factors (TFs) in regions that direct gene expression. However, the modes of interaction between TFs and nucleosomal DNA remain largely unknown. Here, we use nucleosome consecutive affinity-purification systematic evolution of ligands by exponential enrichment (NCAP-SELEX) to systematically explore interactions between the nucleosome and 220 TFs representing diverse structural families. Consistently with earlier observations, we find that the vast majority of TFs have less access to nucleosomal DNA than to free DNA. The motifs recovered from TFs bound to nucleosomal and free DNA are generally similar; however, steric hindrance and scaffolding by the nucleosome result in specific positioning and orientation of the motifs. Many TFs preferentially bind close to the end of nucleosomal DNA, or to periodic positions at its solvent-exposed side. TFs often also bind nucleosomal DNA in a particular orientation, because the nucleosome breaks the local rotational symmetry of DNA. Some TFs also specifically interact with DNA located at the dyad position where only one DNA gyre is wound, whereas other TFs prefer sites spanning two DNA gyres and bind specifically to each of them. Our work reveals striking differences in TF binding to free and nucleosomal DNA, and uncovers a rich interaction landscape between the TFs and the nucleosome.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_null_additivity_of_multi-drug_combinations_30901769/">
          <h4 class="font-weight-bold">The null additivity of multi-drug combinations, bioRxiv, 2017-12-26</h4>
          <p>AbstractFrom natural ecology 1–4 to clinical therapy 5–8, cells are often exposed to mixtures of multiple drugs. Two competing null models are used to predict the combined effect of drugs response additivity (Bliss) and dosage additivity (Loewe) 9–11. Here, noting that these models diverge with increased number of drugs, we contrast their predictions with measurements of Escherichia coli growth under combinations of up to 10 different antibiotics. As the number of drugs increases, Bliss maintains accuracy while Loewe systematically loses its predictive power. The total dosage required for growth inhibition, which Loewe predicts should be fixed, steadily increases with the number of drugs, following a square root scaling. This scaling is explained by an approximation to Bliss where, inspired by RA Fisher’s classical geometric model 12, dosages of independent drugs adds up as orthogonal vectors rather than linearly. This dose-orthogonality approximation provides results similar to Bliss, yet uses the dosage language as Loewe and is hence easier to implement and intuit. The rejection of dosage additivity in favor of effect additivity and dosage orthogonality provides a framework for understanding how multiple drugs and stressors add up in nature and the clinic.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        
        <li class="page-item"><a href="/audiences/tags/systems-biology/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="/audiences/tags/systems-biology/" class="page-link">1</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/systems-biology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/systems-biology/page/3/" class="page-link">3</a></li>
            
        

        
        <li class="page-item"><a href="/audiences/tags/systems-biology/page/3/" rel="next" class="page-link">Next ›</a></li>
        

        
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
