<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/evidence_for_rapid_phenotypic_and_behavioral_change_in_a_recently_established_cavefish_population_61186125/">
          <h4 class="font-weight-bold">Evidence for rapid phenotypic and behavioral change in a recently established cavefish population, bioRxiv, 2019-05-28</h4>
          <p>AbstractSubstantial morphological and behavioral shifts often accompany rapid environmental change, yet, little is known about the early stages of cave colonization. Relative to surface streams, caves are extreme environments with perpetual darkness and low nutrient availability. The Mexican tetra (Astyanax mexicanus), has repeatedly colonized caves throughout Mexico, suggesting an ability to adapt to these conditions. Here, we survey for phenotypic and behavioral differences between a surface population and a cave population of A. mexicanus that has recently colonized Honey Creek Cave, Comal County, Texas, likely within the last century. We found that fish from Honey Creek Cave and fish from Honey Creek surface populations differ significantly in morphological traits including length, coloration, body condition, eye size, and dorsal fin placement. Cavefish also exhibit an increased number of superficial neuromasts relative to surface fish. Behaviorally, cavefish consume fewer worms when trials are performed in both lighted and darkened conditions. Cavefish are more aggressive than surface fish and exhibit fewer behaviors associated with stress. Further in contrast to surface fish, cavefish prefer the edges to the center of an arena and are qualitatively more likely to investigate a novel object placed in the tank. While cavefish and surface fish were wild-caught and developmental environment likely play a role in shaping these differences, our work demonstrates morphological and behavioral shifts for Texas cavefish and offers an exciting opportunity for future work to explore the genetic and environmental contributions to early cave colonization.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/animal-behavior-and-cognition">animal-behavior-and-cognition</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/fate_mapping_via_ms4a3_expression_history_traces_monocyte-derived_cells_61185605/">
          <h4 class="font-weight-bold">Fate mapping via Ms4a3 expression history traces monocyte-derived cells, bioRxiv, 2019-05-28</h4>
          <p>SUMMARYMost tissue-resident macrophage (RTM) populations are seeded by waves of embryonic hematopoiesis and are self-maintained independently of a bone-marrow contribution during adulthood. A proportion of RTMs, however, is constantly replaced by blood monocytes and their functions compared to embryonic RTM remains unclear. The kinetics and extent of the contribution of circulating monocytes to RTM replacement during homeostasis, inflammation and disease is highly debated. Here, we identified Ms4a3 as a specific marker expressed by granulocyte-monocyte progenitors (GMPs) and subsequently generated Ms4a3TdT reporter and Ms4a3Cre-RosaTdT fate mapper models to follow monocytes and their progenies. Our Ms4a3Cre-RosaTdT model traced efficiently blood monocytes (97%) and granulocytes (100%), but no lymphocytes or tissue dendritic cells. Using this model, we precisely quantified the contribution of monocytes to the RTM pool during homeostasis and inflammation. The unambiguous identification of monocyte-derived cells will permit future studies of their function under any condition.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/episcanpy_integrated_single-cell_epigenomic_analysis_61069578/">
          <h4 class="font-weight-bold">EpiScanpy integrated single-cell epigenomic analysis, bioRxiv, 2019-05-25</h4>
          <p>ABSTRACTEpigenetic single-cell measurements reveal a layer of regulatory information not accessible to single-cell transcriptomics, however single-cell-omics analysis tools mainly focus on gene expression data. To address this issue, we present epiScanpy, a computational framework for the analysis of single-cell DNA methylation and single-cell ATAC-seq data. EpiScanpy makes the many existing RNA-seq workflows from scanpy available to large-scale single-cell data from other -omics modalities. We introduce and compare multiple feature space constructions for epigenetic data and show the feasibility of common clustering, dimension reduction and trajectory learning techniques. We benchmark epiScanpy by interrogating different single-cell brain mouse atlases of DNA methylation, ATAC-seq and transcriptomics. We find that differentially methylated and differentially open markers between cell clusters enrich transcriptome-based cell type labels by orthogonal epigenetic information.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_genetic_makeup_of_the_electrocardiogram_61070956/">
          <h4 class="font-weight-bold">The genetic makeup of the electrocardiogram, bioRxiv, 2019-05-25</h4>
          <p>AbstractSince its original description in 1893 by Willem van Einthoven, the electrocardiogram (ECG) has been instrumental in the recognition of a wide array of cardiac disorders1,2. Although many electrocardiographic patterns have been well described, the underlying biology is incompletely understood. Genetic associations of particular features of the ECG have been identified by genome wide studies. This snapshot approach only provides fragmented information of the underlying genetic makeup of the ECG. Here, we follow the effecs of individual genetic variants through the complete cardiac cycle the ECG represents. We found that genetic variants have unique morphological signatures not identfied by previous analyses. By exploiting identified abberations of these morphological signatures, we show that novel genetic loci can be identified for cardiac disorders. Our results demonstrate how an integrated approach to analyse high-dimensional data can further our understanding of the ECG, adding to the earlier undertaken snapshot analyses of individual ECG components. We anticipate that our comprehensive resource will fuel in silico explorations of the biological mechanisms underlying cardiac traits and disorders represented on the ECG. For example, known disease causing variants can be used to identify novel morphological ECG signatures, which in turn can be utilized to prioritize genetic variants or genes for functional validation. Furthermore, the ECG plays a major role in the development of drugs, a genetic assessment of the entire ECG can drive such developments.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/novel_rhabdovirus_and_an_almost_complete_drain_fly_transcriptome_recovered_from_two_independent_contaminations_of_clinical_samples_60978515/">
          <h4 class="font-weight-bold">Novel Rhabdovirus and an almost complete drain fly transcriptome recovered from two independent contaminations of clinical samples, bioRxiv, 2019-05-24</h4>
          <p>AbstractMetagenomic approaches enable an open exploration of microbial communities without requiring a priori knowledge of a sample’s composition by shotgun sequencing the total RNA or DNA of the sample. Such an approach is valuable for exploratory diagnostics of novel pathogens in clinical practice. Yet, one may also identify surprising off-target findings. Here we report a mostly complete transcriptome from a drain fly (likely Psychoda alternata) as well as a novel Rhabdovirus-like virus recovered from two independent contaminations of RNA sequencing libraries from clinical samples of cerebral spinal fluid (CSF) and serum, out of a total of 724 libraries sequenced at the same laboratory during a 2-year time span. This drain fly genome shows a considerable divergence from previously sequenced insects, which may obscure common clinical metagenomic analyses not expecting such contaminations. The classification of these contaminant sequences allowed us to identify infected drain flies as the likely origin of the novel Rhabdovirus-like sequence, which could have been erroneously linked to human pathology, had they been ignored.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/root_development_is_maintained_by_specific_bacteria-bacteria_interactions_within_a_complex_microbiome_60956302/">
          <h4 class="font-weight-bold">Root development is maintained by specific bacteria-bacteria interactions within a complex microbiome, bioRxiv, 2019-05-24</h4>
          <p>AbstractPlants grow within a complex web of species interacting with each other and with the plant via a wide repertoire of chemical signals. To model plant-microbe-microbe-environment interactions, we inoculated seedlings with a defined 185-member bacterial synthetic community (SynCom), and manipulated the abiotic environment to enable classification of the SynCom to modules of co-occurring strains. We deconstructed the SynCom based on these modules, identifying a single bacterial genus, Variovorax, which reverts phenotypic effects on root development induced by a wide diversity of bacterial strains and by the entire 185-member community. Variovorax use mechanisms related to auxin and ethylene manipulation to balance this ecologically realistic root community’s effects on root development. We demonstrate metabolic signal interference within a complex model community, defining Variovorax as determinants of bacteria-plant communication networks.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/3/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/3/" class="page-link">3</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/0-100-users/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/8/" class="page-link">8</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/5/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/64/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
