<!DOCTYPE html>
<html><head>
  
  <title>Ecology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/psychoactive_plant-_and_mushroom-associated_alkaloids_from_two_behavior_modifying_cicada_pathogens_45499444/">
          <h4 class="font-weight-bold">Psychoactive plant- and mushroom-associated alkaloids from two behavior modifying cicada pathogens, bioRxiv, 2018-07-25</h4>
          <p>AbstractEntomopathogenic fungi routinely kill their hosts before releasing infectious spores, but select species keep insects alive while sporulating, which enhances dispersal. Transcriptomics and metabolomics studies of entomopathogens with post-mortem dissemination from their parasitized hosts have unraveled infection processes and host responses, yet mechanisms underlying active spore transmission by Entomophthoralean fungi in living insects remain elusive. Here we report the discovery, through metabolomics, of the plant-associated amphetamine, cathinone, in four Massospora cicadina-infected periodical cicada populations, and the mushroom-associated tryptamine, psilocybin, in annual cicadas infected with Massospora platypediae or Massospora levispora, which appear to represent a single fungal species. The absence of some fungal enzymes necessary for cathinone and psilocybin biosynthesis along with the inability to detect intermediate metabolites or gene orthologs are consistent with possibly novel biosynthesis pathways in Massospora. The neurogenic activities of these compounds suggest the extended phenotype of Massospora that modifies cicada behavior to maximize dissemination is chemically-induced.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_network_structure_of_cancer_ecosystems_31042414/">
          <h4 class="font-weight-bold">The network structure of cancer ecosystems, bioRxiv, 2017-12-30</h4>
          <p>Ever since Paget’s seed-and-soil and Ewing’s connectivity hypotheses to explain tumor metastasis (1,2), it has become clear that cancer progression can be envisaged as an ecological phenomenon. This connection has flourished during the past two decades (3–7), giving rise to important insights into the ecology and evolution of cancer progression, with therapeutic implications (8–10). Here, we take a metapopulation view of metastasis (i.e. the migration to and colonization of, habitat patches) and represent it as a bipartite network, distinguishing source patches, or organs that host a primary tumor, and acceptor patches, or organs colonized ultimately from the source through metastasis. Using 20,326, biomedical records obtained from literature, we show that (i) the network structure of cancer ecosystems is non-random, exhibiting a nested subset pattern as has been found both in the distribution of species across islands and island-like habitats (11–13), and in the distribution of among species interactions across different ecological networks (14–16); (ii) similar to ecological networks, there is a heterogeneous distribution of degree (i.e., number of connections associated with a source or acceptor organ); (iii) there is a significant correlation between metastatic incidence (or the frequency with which tumor cells from a source organ colonize an acceptor one) and arterial blood supply, suggesting that more irrigated organs have a higher probability of developing metastasis or being invaded; (iv) there is a positive correlation between metastatic incidence and acceptor organ degree (or number of different tumor-bearing source organs that generate metastasis in a given acceptor organ), and a negative one between acceptor organ degree and number of stem cell divisions, implying that there are preferred sink organs for metastasis and that this could be related to average acceptor organ cell longevity; (v) there is a negative association between organ cell turnover and source organ degree, implying that organs with rapid cell turnovers tend to generate more metastasis, a process akin to the phenomenon of propagule pressure in ecology (17); and (vi) the cancer ecosystem network exhibits a modular structure in both source and acceptor patches, suggesting that some of them share more connections among themselves than with the rest of the network. We show that both niche-related processes occurring at the organ level as well as spatial connectivity and propagule pressure contribute to metastaticspread and result in a non-random cancer network, which exhibits a truncated power law degree distribution, clustering and a nested subset structure. The similarity between the cancer network and ecological networks highlights the importance of ecological approaches in increasing our understanding of patterns in cancer incidence and dynamics, which may lead to new strategies to control tumor spread within the human ecosystem.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/phylofactorization_a_graph-partitioning_algorithm_to_identify_phylogenetic_scales_of_ecological_data_30533791/">
          <h4 class="font-weight-bold">Phylofactorization a graph-partitioning algorithm to identify phylogenetic scales of ecological data, bioRxiv, 2017-12-17</h4>
          <p>AbstractThe problem of pattern and scale is a central challenge in ecology. The problem of scale is central to community ecology, where functional ecological groups are aggregated and treated as a unit underlying an ecological pattern, such as aggregation of “nitrogen fixing trees” into a total abundance of a trait underlying ecosystem physiology. With the emergence of massive community ecological datasets, from microbiomes to breeding bird surveys, there is a need to objectively identify the scales of organization pertaining to well-defined patterns in community ecological data.The phylogeny is a scaffold for identifying key phylogenetic scales associated with macroscopic patterns. Phylofactorization was developed to objectively identify phylogenetic scales underlying patterns in relative abundance data. However, many ecological data, such as presence-absences and counts, are not relative abundances, yet it is still desireable and informative to identify phylogenetic scales underlying a pattern of interest. Here, we generalize phylofactorization beyond relative abundances to a graph-partitioning algorithm for any community ecological data.Generalizing phylofactorization connects many tools from data analysis to phylogenetically-informe analysis of community ecological data. Two-sample tests identify three phylogenetic factors of mammalian body mass which arose during the K-Pg extinction event, consistent with other analyses of mammalian body mass evolution. Projection of data onto coordinates defined by the phylogeny yield a phylogenetic principal components analysis which refines our understanding of the major sources of variation in the human gut microbiome. These same coordinates allow generalized additive modeling of microbes in Central Park soils and confirm that a large clade of Acidobacteria thrive in neutral soils. Generalized linear and additive modeling of exponential family random variables can be performed by phylogenetically-constrained reduced-rank regression or stepwise factor contrasts. We finish with a discussion of how phylofac-torization produces an ecological species concept with a phylogenetic constraint. All of these tools can be implemented with a new R package available online.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/analyzing_ecological_networks_of_species_interactions_16858420/">
          <h4 class="font-weight-bold">Analyzing ecological networks of species interactions, bioRxiv, 2017-03-01</h4>
          <p>Networks provide one of the best representations for ecological communities, composed of many species with sometimes complex connections between them. Yet the methodological literature allowing one to analyze and extract meaning from ecological networks is dense, fragmented, and unwelcoming. We provide a general overview to the field of using networks in community ecology, outlining both the intent of the different measures, their assumptions, and the contexts in which they can be used. When methodologically justified, we suggest good practices to use in the analysis of ecological networks. We anchor this synopsis with examples from empirical studies, and conclude by highlighting what identified as needed future developments in the field.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/relic_dna_is_abundant_in_soil_and_obscures_estimates_of_soil_microbial_diversity_6201648/">
          <h4 class="font-weight-bold">Relic DNA is abundant in soil and obscures estimates of soil microbial diversity, bioRxiv, 2016-03-17</h4>
          <p>AbstractIt is implicitly assumed that the microbial DNA recovered from soil originates from living cells. However, because relic DNA (DNA from dead cells) can persist in soil for weeks to years, it could impact DNA-based analyses of microbial diversity. We examined a wide range of soils and found that, on average, 40% of prokaryotic and fungal DNA was derived from the relic DNA pool. Relic DNA inflated the observed prokaryotic and fungal diversity by as much as 55%, and caused misestimation of taxon abundances, including taxa integral to key ecosystem processes. These findings imply that relic DNA can obscure treatment effects, spatiotemporal patterns, and relationships between taxa and environmental conditions. Moreover, relic DNA may represent a historical record of microbes formerly living in soil.One Sentence SummarySoils can harbor substantial amounts of DNA from dead microbial cells; this ‘relic’ DNA inflates estimates of microbial diversity and obscures assessments of community structure.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2016">2016</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/model-based_projections_of_zika_virus_infections_in_childbearing_women_in_the_americas_5524326/">
          <h4 class="font-weight-bold">Model-based projections of Zika virus infections in childbearing women in the Americas, bioRxiv, 2016-02-13</h4>
          <p>AbstractZika virus is a mosquito-borne pathogen that is rapidly spreading across the Americas. Due to associations between Zika virus infection and a range of fetal maladies1,2, the epidemic trajectory of this viral infection poses a significant concern for the nearly 15 million children born in the Americas each year. Ascertaining the portion of this population that is truly at risk is an important priority. One recent estimate3suggested that 5.42 million childbearing women live in areas of the Americas that are suitable for Zika occurrence. To improve on that estimate, which did not take into account the protective effects of herd immunity, we developed a new approach that combines classic results from epidemiological theory with seroprevalence data and highly spatially resolved data about drivers of transmission to make location-specific projections of epidemic attack rates. Our results suggest that 1.65 (1.45–2.06) million childbearing women and 93.4 (81.6–117.1) million people in total could become infected before the first wave of the epidemic concludes. Based on current estimates of rates of adverse fetal outcomes among infected women2,4,5, these results suggest that tens of thousands of pregnancies could be negatively impacted by the first wave of the epidemic. These projections constitute a revised upper limit of populations at risk in the current Zika epidemic, and our approach offers a new way to make rapid assessments of the threat posed by emerging infectious diseases more generally.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2016">2016</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/ecology/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/ecology/page/3/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/ecology/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/ecology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/ecology/page/3/" class="page-link">3</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/ecology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/ecology/page/5/" class="page-link">5</a></li>
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/ecology/page/5/" rel="next" class="page-link">Next ›</a></li>
        

        
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
