<!DOCTYPE html>
<html><head>
  
  <title>100 200 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/dopamine_modulates_the_size_of_striatal_projection_neuron_ensembles_72268452/">
          <h4 class="font-weight-bold">Dopamine modulates the size of striatal projection neuron ensembles, bioRxiv, 2019-12-06</h4>
          <p>SUMMARYDopamine (DA) is a critical modulator of brain circuits that control voluntary movements, but our understanding of its influence on the activity of target neurons in vivo remains limited. Here, we use two-photon Ca2&#43; imaging to simultaneously monitor the activity of direct and indirect-pathway spiny projection neurons (SPNs) in the striatum of behaving mice during acute and prolonged manipulations of DA signaling. We find that, contrary to prevailing models, DA does not modulate activity rates in either pathway strongly or differentially. Instead, DA exerts a prominent influence on the overall number of direct and indirect pathway SPNs recruited during behavior. Chronic loss of midbrain DA neurons in a model of Parkinson’s disease selectively impacts direct pathway ensembles and profoundly alters how they respond to DA elevation. Our results indicate that DA regulates striatal output by dynamically reconfiguring its sparse ensemble code and provide novel insights into the pathophysiology of Parkinson’s disease.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/early_replacement_of_west_eurasian_male_y_chromosomes_from_the_east_72335288/">
          <h4 class="font-weight-bold">Early replacement of West Eurasian male Y chromosomes from the east, bioRxiv, 2019-12-06</h4>
          <p>AbstractThe genomes of humans outside Africa originated almost entirely from a single migration out ∼50,000-60,000 years ago1,2, followed closely by mixture with Neanderthals contributing ∼2% to all non-Africans3,4. However, the details of this initial migration remain poorly-understood because no ancient DNA analyses are available from this key time period, and present-day autosomal data are uninformative due to subsequent population movementsreshaping5. One locus, however, does retain extensive information from this early period the Y-chromosome, where a detailed calibrated phylogeny has been constructed6. Three present-day Y lineages were carried by the initial migration the rare haplogroup D, the moderately rare C, and the very common FT lineage which now dominates most non-African populations6,7. We show that phylogenetic analyses of haplogroup C, D and FT sequences, including very rare deep-rooting lineages, together with phylogeographic analyses of ancient and present-day non-African Y-chromosomes, all point to EastSouth-east Asia as the origin 50,000-55,000 years ago of all known non-African male lineages (apart from recent migrants). This implies that the initial Y lineages in populations between Africa and eastern Asia have been entirely replaced by lineages from the east, contrasting with the expectations of the serial-founder model8,9, and thus informing and constraining models of the initial expansion.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/prokaryotic_single-cell_rna_sequencing_by_in_situ_combinatorial_indexing_72333206/">
          <h4 class="font-weight-bold">Prokaryotic Single-Cell RNA Sequencing by In Situ Combinatorial Indexing, bioRxiv, 2019-12-06</h4>
          <p>AbstractDespite longstanding appreciation of gene expression heterogeneity in isogenic bacterial populations, affordable and scalable technologies for studying single bacterial cells have been limited. While single-cell RNA sequencing (scRNA-seq) has revolutionized studies of transcriptional heterogeneity in diverse eukaryotic systems, application of scRNA-seq to prokaryotic cells has been hindered by their low levels of mRNA, lack of mRNA polyadenylation, and thick cell walls. Here, we present Prokaryotic Expression-profiling by Tagging RNA In Situ and sequencing (PETRI-seq), a high-throughput prokaryotic scRNA-seq pipeline that overcomes these obstacles. PETRI-seq uses in situ combinatorial indexing to barcode transcripts from tens of thousands of cells in a single experiment. We have demonstrated that PETRI-seq effectively captures single cell transcriptomes of Gram-negative and Gram-positive bacteria with high purity and little bias. Although bacteria express only thousands of mRNAs per cell, captured mRNA levels were sufficient to distinguish between the transcriptional states of single cells within isogenic populations. In E. coli, we were able to identify single cells in either stationary or exponential phase and define consensus transcriptomes for these sub-populations. In wild type S. aureus, we detected a rare population of cells undergoing prophage induction. We anticipate that PETRI-seq will be widely useful for studying transcriptional heterogeneity in microbial communities.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_nucleus_measures_shape_deformation_for_cellular_proprioception_and_regulates_adaptive_morphodynamics_72222727/">
          <h4 class="font-weight-bold">The nucleus measures shape deformation for cellular proprioception and regulates adaptive morphodynamics, bioRxiv, 2019-12-05</h4>
          <p>AbstractThe physical microenvironment regulates cell behavior during tissue development and homeostasis. How single cells decode information about their geometrical shape under mechanical stress and physical space constraints within their local environment remains largely unknown. Here we show that the nucleus, the biggest cellular organelle, functions as a non-dissipative cellular shape deformation gauge that enables cells to continuously measure shape variations on the time scale of seconds. Inner nuclear membrane unfolding together with the relative spatial intracellular positioning of the nucleus provides physical information on the amplitude and type of cellular shape deformation. This adaptively activates a calcium-dependent mechano-transduction pathway, controlling the level of actomyosin contractility and migration plasticity. Our data support that the nucleus establishes a functional module for cellular proprioception that enables cells to sense shape variations for adapting cellular behaviour to their microenvironment.One Sentence SummaryThe nucleus functions as an active deformation sensor that enables cells to adapt their behavior to the tissue microenvironment.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/experience_dependent_contextual_codes_in_the_hippocampus_71917134/">
          <h4 class="font-weight-bold">Experience dependent contextual codes in the hippocampus, bioRxiv, 2019-12-04</h4>
          <p>AbstractThe hippocampus is a medial temporal lobe brain structure that contains circuitry and neural representations capable of supporting declarative memory. Hippocampal place cells fire in one or few restricted spatial locations in a given environment. Between environmental contexts, place cell firing fields remap (turning onoff or moving to a new spatial location), providing a unique population-wide neural code for context specificity. However, the manner by which features associated with a given context combine to drive place cell remapping remains a matter of debate. Here we show that remapping of neural representations in region CA1 of the hippocampus is strongly driven by prior beliefs about the frequency of certain contexts, and that remapping is equivalent to an optimal estimate of the identity of the current context under that prior. This prior-driven remapping is learned early in training and remains robust to changes in behavioral task-demands. Furthermore, a simple associative learning mechanism is sufficient to reproduce these results. Our findings demonstrate that place cell remapping is a generalization of representing an animal’s location. Rather than simply representing location in physical space, the hippocampus represents an optimal estimate of location in a multi-dimensional stimulus space.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genetic_dominance_governs_the_evolution_and_spread_of_mobile_genetic_elements_in_bacteria_71919044/">
          <h4 class="font-weight-bold">Genetic dominance governs the evolution and spread of mobile genetic elements in bacteria, bioRxiv, 2019-12-04</h4>
          <p>AbstractMobile genetic elements (MGEs), such as plasmids, promote bacterial evolution through horizontal gene transfer (HGT). However, the rules governing the repertoire of traits encoded on MGEs remain unclear. In this study, we uncovered the central role of genetic dominance shaping genetic cargo in MGEs, using antibiotic resistance as a model system. MGEs are typically present in more than one copy per host bacterium and, as a consequence, genetic dominance favors the fixation of dominant mutations over recessive ones. Moreover, genetic dominance also determines the phenotypic effects of horizontally acquired MGE-encoded genes, silencing recessive alleles if the recipient bacterium already carries a wild-type copy of the gene. The combination of these two effects governs the catalogue of genes encoded on MGEs, dictating bacterial evolution through HGT.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/100-200-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/100-200-users/page/5/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/5/" class="page-link">5</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/100-200-users/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/10/" class="page-link">10</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/100-200-users/page/7/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/100-200-users/page/126/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
