<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/layer-dependent_activity_in_human_prefrontal_cortex_during_working_memory_48847582/">
          <h4 class="font-weight-bold">Layer-dependent activity in human prefrontal cortex during working memory, bioRxiv, 2018-09-24</h4>
          <p>Working memory involves a series of functions encoding a stimulus, maintaining or manipulating its representation over a delay, and finally making a behavioral response. While working memory engages dorsolateral prefrontal cortex (dlPFC), few studies have investigated whether these subfunctions are localized to different cortical depths in this region, and none have done so in humans. Here, we use high-resolution functional MRI to interrogate the layer specificity of neural activity during different epochs of a working memory task in dlPFC. We detect activity timecourses that follow the hypothesized patterns superficial layers are preferentially active during the delay period, while deeper layers are preferentially active during the response. Results demonstrate that layer-specific fMRI can be used in higher-order brain regions to non-invasively map cognitive information processing along cortical circuitry in humans.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/parliament2_fast_structural_variant_calling_using_optimized_combinations_of_callers_48807637/">
          <h4 class="font-weight-bold">Parliament2 Fast Structural Variant Calling Using Optimized Combinations of Callers, bioRxiv, 2018-09-23</h4>
          <p>AbstractHere we present Parliament2 – a structural variant caller which combines multiple best-in-class structural variant callers to create a highly accurate callset. This captures more events than the individual callers achieve independently. Parliament2 uses a call-overlap-genotype approach that is highly extensible to new methods and presents users the choice to run some or all of Breakdancer, Breakseq, CNVnator, Delly, Lumpy, and Manta to run. Parliament2 applies an additional parallelization framework to speed certain callers and executes these in parallel, taking advantage of the different resource requirements to complete structural variant calling much faster than running the programs individually. Parliament2 is available as a Docker container, which pre-installs all required dependencies. This allows users to run any caller with easy installation and execution. This Docker container can easily be deployed in cloud or local environments and is available as an app on DNAnexus.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/vision_using_multiple_distinct_rod_opsins_in_deep-sea_fishes_48810320/">
          <h4 class="font-weight-bold">Vision using multiple distinct rod opsins in deep-sea fishes, bioRxiv, 2018-09-23</h4>
          <p>AbstractVertebrate vision is accomplished through a set of light-sensitive photopigments, which are located in the photoreceptors of the retina and consist of a visual opsin protein bound to a chromophore. In dim-light, vertebrates generally rely upon a single rod opsin (RH1) for obtaining visual information. By inspecting 101 fish genomes, we found that three deep-sea teleost lineages have independently expanded their RH1 gene repertoires. Amongst these, the silver spinyfin (Diretmus argenteus Johnson 1863) stands out as having the highest number of visual opsins known for animals to date (2 cone and 38 rod opsins). Spinyfins simultaneously express up to 14 RH1s encoding for photopigments with different peak spectral sensitivities (λmax=448-513 nm) that cover the range of the residual daylight, as well as the bioluminescence spectrum present in the deep-sea. Our findings present novel molecular and functional evidence for the recurrent evolution of multiple rod opsin-based vision in vertebrates.SHORT ABSTRACTContrary to the single rod opsin used by most vertebrates, some fishes use multiple rod opsins for vision in the dimly lit deep-sea.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/comparison_of_efficiency_and_specificity_of_crispr-associated_cas_nucleases_in_plants_an_expanded_toolkit_for_precision_genome_engineering_48724730/">
          <h4 class="font-weight-bold">Comparison of Efficiency and Specificity of CRISPR-Associated (Cas) Nucleases in Plants An Expanded Toolkit for Precision Genome Engineering, bioRxiv, 2018-09-21</h4>
          <p>Molecular tools adapted from bacterial CRISPR (Clustered Regulatory Interspaced Short Palindromic Repeats) systems for adaptive immunity have become widely used for plant genome engineering, both to investigate gene functions and to engineer desirable traits. A number of different Cas (CRISPR-associated) nucleases are now used but, as most studies performed to date have engineered different targets using a variety of plant species and molecular tools, it has been difficult to draw conclusions about the comparative performance of different nucleases. Due to the time and effort required to regenerate engineered plants, efficiency is critical. In addition, there have been several reports of mutations at sequences with less than perfect identity to the target. While in some plant species it is possible to remove these so-called ‘off-targets’ by backcrossing to a parental line, the specificity of genome engineering tools is important when targeting specific members of closely-related gene families, especially when recent paralogues are co-located in the genome and unlikely to segregate. Specificity is also important for species that take years to reach sexual maturity or that are clonally propagated. Here, we directly compare the efficiency and specificity of Cas nucleases from different bacterial species together with engineered variants of Cas9. We find that the nucleotide content correlates with efficiency and that Cas9 from Staphylococcus aureus is comparatively most efficient at inducing mutations. We also demonstrate that ‘high-fidelity’ variants of Cas9 can reduce off-target mutations in plants. We present these molecular tools as standardised DNA parts to facilitate their re-use.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/dissecting_heterogeneous_cell-populations_across_signaling_and_disease_conditions_with_popalign_48724548/">
          <h4 class="font-weight-bold">Dissecting heterogeneous cell-populations across signaling and disease conditions with PopAlign, bioRxiv, 2018-09-21</h4>
          <p>AbstractSingle-cell measurement techniques can now probe gene expression in heterogeneous cell populations from the human body across a range of environmental and physiological conditions. However, new mathematical and computational methods are required to represent and analyze gene expression changes that occur in complex mixtures of single cells as they respond to signals, drugs, or disease states. Here, we introduce a mathematical modeling platform, PopAlign, that automatically identifies subpopulations of cells within a heterogeneous mixture, and tracks gene expression and cell abundance changes across subpopulations by constructing and comparing probabilistic models. We apply PopAlign to discover specific categories of signaling responses within primary human immune cells as well as patient-specific disease signatures in multiple myeloma that are obscured by techniques like tSNE. PopAlign scales to comparisons involving tens to hundreds of samples, enabling large-scale studies of natural and engineered cell populations as they respond to drugs, signals or physiological change.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/proximity_sensors_reveal_social_information_transfer_in_maternity_colonies_of_common_noctule_bats_48701078/">
          <h4 class="font-weight-bold">Proximity sensors reveal social information transfer in maternity colonies of Common noctule bats, bioRxiv, 2018-09-20</h4>
          <p>Summary&lt;jatslist list-type=order&gt;&lt;jatslist-item&gt;Bats are a highly gregarious taxon suggesting that social information should be readily available for making decision. Social information transfer in maternity colonies might be a particularly efficient mechanism for naïve pups to acquire information on resources from informed adults. However, such behaviour is difficult to study in the wild, in particular in elusive and small-bodied animals such as bats.&lt;jatslist-item&gt;&lt;jatslist-item&gt;The goal of this study was to investigate the role of social information in acquiring access to two types of resources, which are crucial in the life of a juvenile bat suitable roosting sites and fruitful feeding grounds. We hypothesized that fledging offspring will make use of social information by following informed members of the social groups to unknown roosts or foraging sites.&lt;jatslist-item&gt;&lt;jatslist-item&gt;In the present study we applied for the first time the newly developed miniaturized proximity sensor system ‘BATS’, a fully automated system for documenting associations among individual bats both while roosting and while on the wing. We quantified associations among juveniles and other group member while switching roosts and during foraging.&lt;jatslist-item&gt;&lt;jatslist-item&gt;We found clear evidence for information transfer while switching roosts, mainly among juveniles and their genetically identified mothers. Anecdotal observations suggest intentional guidance behaviour by mothers, indicated by repeated commuting flights among the pup and the target roost. Infrequent, short meetings with colony members other than the mother indicate local enhancement at foraging sites, but no intentional information transfer.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Our study illustrates how advances in technology enable researchers to solve long-standing puzzles. Miniaturized proximity sensors facilitate the automated collection of continuous data sets and represent an ideal tool to gain novel insights into the sociobiology of elusive and small-bodied species.&lt;jatslist-item&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/animal-behavior-and-cognition">animal-behavior-and-cognition</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/73/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/70/" class="page-link">70</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/71/" class="page-link">71</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/72/" class="page-link">72</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/73/" class="page-link">73</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/0-100-users/page/74/" class="page-link">74</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/75/" class="page-link">75</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/76/" class="page-link">76</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/77/" class="page-link">77</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/78/" class="page-link">78</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/75/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/123/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
