<!DOCTYPE html>
<html><head>
  
  <title>Reports | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/single-cell_in_situ_transcriptomic_map_of_astrocyte_cortical_layer_diversity_49173691/">
          <h4 class="font-weight-bold">Single-cell in situ transcriptomic map of astrocyte cortical layer diversity, bioRxiv, 2018-10-03</h4>
          <p>AbstractDuring organogenesis, patterns and gradients of gene expression underlie organization and diversified cell specification to generate complex tissue architecture. While the cerebral cortex is organized into six excitatory neuronal layers, it is unclear whether glial cells are diversified to mimic neuronal laminae or show distinct layering. To determine the molecular architecture of the mammalian cortex, we developed a high content pipeline that can quantify single-cell gene expression in situ. The Large-area Spatial Transcriptomic (LaST) map confirmed expected cortical neuron layer organization and also revealed a novel neuronal identity signature. Screening 46 candidate genes for astrocyte diversity across the cortex, we identified grey matter superficial, mid and deep astrocyte identities in gradient layer patterns that were distinct from neurons. Astrocyte layers formed in early postnatal cortex and mostly persisted in adult mouse and human cortex. Mutations that shifted neuronal post-mitotic identity or organization were sufficient to alter glial layering, indicating an instructive role for neuronal cues. In normal mouse cortex, astrocyte layer patterns showed area diversity between functionally distinct cortical regions. These findings indicate that excitatory neurons and astrocytes cells are organized into distinct lineage-associated laminae, which give rise to higher order neuroglial complexity of cortical architecture.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/nanoj_a_high-performance_open-source_super-resolution_microscopy_toolbox_49119416/">
          <h4 class="font-weight-bold">NanoJ a high-performance open-source super-resolution microscopy toolbox, bioRxiv, 2018-10-02</h4>
          <p>Super-resolution microscopy has become essential for the study of nanoscale biological processes. This type of imaging often requires the use of specialised image analysis tools to process a large volume of recorded data and extract quantitative information. In recent years, our team has built an open-source image analysis framework for super-resolution microscopy designed to combine high performance and ease of use. We named it NanoJ - a reference to the popular ImageJ software it was developed for. In this paper, we highlight the current capabilities of NanoJ for several essential processing steps spatio-temporal alignment of raw data (NanoJ-Core), super-resolution image reconstruction (NanoJ-SRRF), image quality assessment (NanoJ-SQUIRREL), structural modelling (NanoJ-VirusMapper) and control of the sample environment (NanoJ-Fluidics). We expect to expand NanoJ in the future through the development of new tools designed to improve quantitative data analysis and measure the reliability of fluorescent microscopy studies.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/quiescent_cells_actively_replenish_cenp-a_nucleosomes_to_maintain_centromere_identity_and_proliferative_potential_49147569/">
          <h4 class="font-weight-bold">Quiescent cells actively replenish CENP-A nucleosomes to maintain centromere identity and proliferative potential, bioRxiv, 2018-10-02</h4>
          <p>SummaryCentromeres provide a robust model for epigenetic inheritance as they are specified by sequence-independent mechanisms involving the histone H3-variant CENP-A. Prevailing models indicate that the high intrinsic stability of CENP-A nucleosomes maintains centromere identity indefinitely. Here, we demonstrate that CENP-A is not stable at centromeres, but is instead gradually and continuously incorporated in quiescent cells including G0-arrested tissue culture cells and prophase I-arrested oocytes. Quiescent CENP-A incorporation involves the canonical CENP-A deposition machinery, but displays distinct requirements from cell cycle-dependent deposition. We demonstrate that Plk1 is required specifically for G1 CENP-A deposition, whereas transcription promotes CENP-A incorporation in quiescent oocytes. Preventing CENP-A deposition during quiescence results in significantly reduced CENP-A levels and perturbs chromosome segregation following the resumption of cell division. In contrast to quiescent cells, terminally differentiated cells fail to maintain CENP-A levels. Our work reveals that quiescent cells actively maintain centromere identity providing an indicator of proliferative potential.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/there_is_no_single_functional_atlas_even_for_a_single_individual_parcellation_of_the_human_brain_is_state_dependent_49104845/">
          <h4 class="font-weight-bold">There is no single functional atlas even for a single individual Parcellation of the human brain is state dependent, bioRxiv, 2018-10-02</h4>
          <p>AbstractThe goal of human brain mapping has long been to delineate the functional subunits in the brain and elucidate the functional role of each of these brain regions. Recent work has focused on whole-brain parcellation of functional Magnetic Resonance Imaging (fMRI) data to identify these subunits and create a functional atlas. Functional connectivity approaches to understand the brain at the network level require such an atlas to assess connections between parcels and extract network properties. While no single functional atlas has emerged as the dominant atlas to date, there remains an underlying assumption that such an atlas exists. Using fMRI data from a highly sampled subject as well as two independent replication data sets, we demonstrate that functional parcellations based on fMRI connectivity data reconfigure substantially and in a meaningful manner, according to brain state.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_spatial_atlas_of_inhibitory_cell_types_in_mouse_hippocampus_49099053/">
          <h4 class="font-weight-bold">A spatial atlas of inhibitory cell types in mouse hippocampus, bioRxiv, 2018-10-01</h4>
          <p>Understanding the function of a tissue requires knowing the spatial organization of its constituent cell types. In the cerebral cortex, single-cell RNA sequencing (scRNA-seq) has revealed the genome-wide expression patterns that define its many, closely related cell types, but cannot reveal their spatial arrangement. Here we introduce probabilistic cell typing by in situ sequencing (pciSeq), an approach that leverages prior scRNA-seq classification to identify cell types using multiplexed in situ RNA detection. We applied this method to map the inhibitory neurons of hippocampal area CA1, a cell system critical for memory function, for which ground truth is available from extensive prior work identifying the laminar organization of subtly differing cell types. Our method confidently identified 16 interneuron classes, in a spatial arrangement closely matching ground truth. This method will allow identifying the spatial organization of fine cell types across the brain and other tissues.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/pathogen_detection_and_microbiome_analysis_of_infected_wheat_using_a_portable_dna_sequencer_49017775/">
          <h4 class="font-weight-bold">Pathogen Detection and Microbiome Analysis of Infected Wheat Using a Portable DNA Sequencer, bioRxiv, 2018-09-29</h4>
          <p>ABSTRACTFungal diseases of plants are responsible for major losses in agriculture, highlighting the need for rapid and accurate identification of plant pathogens. Disease outcomes are often defined not only by the main pathogen but are influenced by diverse microbial communities known as the microbiome at sites of infection. Here we present the first use of whole genome shot-gun sequencing with a portable DNA sequencing device as a method for the detection of fungal pathogens from wheat (Triticum aestivum) in a standard molecular biology laboratory. The data revealed that our method is robust and applicable to the diagnosis of fungal diseases including wheat stripe rust (caused by Puccinia striiformis f. sp. tritici), septoria tritici blotch (caused by Zymoseptoria tritici) and yellow leaf spot (caused by Pyrenophora tritici repentis). We also identified the bacterial genus Pseudomonas co-present with Puccinia and Zymoseptoria but not Pyrenophora infections. One limitation of the method is the over-representation of redundant wheat genome sequences from samples. This could be addressed by long-range amplicon-based sequencing approaches in future studies, which specifically target non-host organisms. Our work outlines a new approach for detection of a broad range of plant pathogens and associated microbes using a portable sequencer in a standard laboratory, providing the basis for future development of an on-site disease monitoring system.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/reports/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/reports/page/118/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/115/" class="page-link">115</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/116/" class="page-link">116</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/117/" class="page-link">117</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/118/" class="page-link">118</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/reports/page/119/" class="page-link">119</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/120/" class="page-link">120</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/121/" class="page-link">121</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/122/" class="page-link">122</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/123/" class="page-link">123</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/reports/page/120/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/reports/page/172/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
