<!DOCTYPE html>
<html><head>
  
  <title>100 200 Users | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/alignment_and_mapping_methodology_influence_transcript_abundance_estimation_61457922/">
          <h4 class="font-weight-bold">Alignment and mapping methodology influence transcript abundance estimation, bioRxiv, 2019-06-03</h4>
          <p>AbstractThe accuracy of transcript quantification using RNA-seq data depends on many factors, such as the choice of alignment or mapping method and the quantification model being adopted. While the choice of quantification model has been shown to be important, considerably less attention has been given to comparing the effect of various read alignment approaches on quantification accuracy.We investigate the effect of mapping and alignment on the accuracy of transcript quantification in both simulated and experimental data, as well as the effect on subsequent differential gene expression analysis. We observe that, even when the quantification model itself is held fixed, the effect of choosing a different alignment methodology, or aligning reads using different parameters, on quantification estimates can sometimes be large, and can affect downstream analyses as well. These effects can go unnoticed when assessment is focused too heavily on simulated data, where the alignment task is often simpler than in experimentally-acquired samples. We discuss best practices regarding alignment for the purposes of quantification, and also introduce a new hybrid alignment methodology, called selective alignment (SA), to overcome the shortcomings of lightweight approaches without incurring the computational cost of traditional alignment.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/unconventional_cell_division_cycles_from_marine-derived_yeasts_61412659/">
          <h4 class="font-weight-bold">Unconventional cell division cycles from marine-derived yeasts, bioRxiv, 2019-06-02</h4>
          <p>AbstractFungi have been found in every marine habitat that has been explored, however, the diversity and functions of fungi in the ocean are poorly understood. In this study, fungi were cultured from the marine environment in the vicinity of Woods Hole, MA, USA including from plankton, sponge and coral. Our sampling resulted in 36 unique species across 20 genera. We observed many isolates by time-lapse differential interference contrast (DIC) microscopy and analyzed modes of growth and division. Several black yeasts displayed highly unconventional cell division cycles compared to those of traditional model yeast systems. Black yeasts have been found in habitats inhospitable to other life and are known for halotolerance, virulence, and stress-resistance. We find that this group of yeasts also shows remarkable plasticity in terms of cell size control, modes of cell division, and cell polarity. Unexpected behaviors include division through a combination of fission and budding, production of multiple simultaneous buds, and cell division by sequential orthogonal septations. These marine-derived yeasts reveal alternative mechanisms for cell division cycles that seem likely to expand the repertoire of rules established from classic model system yeasts.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/deep_learning_does_not_outperform_classical_machine_learning_for_cell-type_annotation_61335515/">
          <h4 class="font-weight-bold">Deep learning does not outperform classical machine learning for cell-type annotation, bioRxiv, 2019-05-31</h4>
          <p>AbstractDeep learning has revolutionized image analysis and natural language processing with remarkable accuracies in prediction tasks, such as image labeling or word identification. The origin of this revolution was arguably the deep learning approach by the Hinton lab in 2012, which halved the error rate of existing classifiers in the then 2-year-old ImageNet database1. In hindsight, the combination of algorithmic and hardware advances with the appearance of large and well-labeled datasets has led up to this seminal contribution.The emergence of large amounts of data from single-cell RNA-seq and the recent global effort to chart all cell types in the Human Cell Atlas has attracted an interest in deep-learning applications. However, all current approaches are unsupervised, i.e., learning of latent spaces without using any cell labels, even though supervised learning approaches are often more powerful in feature learning and the most popular approach in the current AI revolution by far.Here, we ask why this is the case. In particular we ask whether supervised deep learning can be used for cell annotation, i.e. to predict cell-type labels from single-cell gene expression profiles. After evaluating 6 classification methods across 14 datasets, we notably find that deep learning does not outperform classical machine-learning methods in the task. Thus, cell-type prediction based on gene-signature derived cell-type labels is potentially too simplistic a task for complex non-linear methods, which demands better labels of functional single-cell readouts. We, therefore, are still waiting for the “ImageNet moment” in single-cell genomics.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/genomic_diversity_affects_the_accuracy_of_bacterial_snp_calling_pipelines_61335502/">
          <h4 class="font-weight-bold">Genomic diversity affects the accuracy of bacterial SNP calling pipelines, bioRxiv, 2019-05-31</h4>
          <p>AbstractBackgroundAccurately identifying SNPs from bacterial sequencing data is an essential requirement for using genomics to track transmission and predict important phenotypes such as antimicrobial resistance. However, most previous performance evaluations of SNP calling have been restricted to eukaryotic (human) data. Additionally, bacterial SNP calling requires choosing an appropriate reference genome to align reads to, which, together with the bioinformatic pipeline, affects the accuracy and completeness of a set of SNP calls obtained.This study evaluates the performance of 41 SNP calling pipelines using simulated data from 254 strains of 10 clinically common bacteria and real data from environmentally-sourced and genomically diverse isolates within the genera Citrobacter, Enterobacter, Escherichia and Klebsiella.ResultsWe evaluated the performance of 41 SNP calling pipelines, aligning reads to genomes of the same or a divergent strain. Irrespective of pipeline, a principal determinant of reliable SNP calling was reference genome selection. Across multiple taxa, there was a strong inverse relationship between pipeline sensitivity and precision, and the Mash distance (a proxy for average nucleotide divergence) between reads and reference genome. The effect was especially pronounced for diverse, recombinogenic, bacteria such as Escherichia coli, but less dominant for clonal species such as Mycobacterium tuberculosis.ConclusionsThe accuracy of SNP calling for a given species is compromised by increasing intra-species diversity. When reads were aligned to the same genome from which they were sequenced, among the highest performing pipelines was NovoalignGATK. However, across the full range of (divergent) genomes, among the consistently highest-performing pipelines was Snippy.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/accelerating_sequence_alignment_to_graphs_61175973/">
          <h4 class="font-weight-bold">Accelerating Sequence Alignment to Graphs, bioRxiv, 2019-05-28</h4>
          <p>AbstractAligning DNA sequences to an annotated reference is a key step for genotyping in biology. Recent scientific studies have demonstrated improved inference by aligning reads to a variation graph, i.e., a reference sequence augmented with known genetic variations. Given a variation graph in the form of a directed acyclic string graph, the sequence to graph alignment problem seeks to find the best matching path in the graph for an input query sequence. Solving this problem exactly using a sequential dynamic programming algorithm takes quadratic time in terms of the graph size and query length, making it difficult to scale to high throughput DNA sequencing data. In this work, we propose the first parallel algorithm for computing sequence to graph alignments that leverages multiple cores and single-instruction multiple-data (SIMD) operations. We take advantage of the available inter-task parallelism, and provide a novel blocked approach to compute the score matrix while ensuring high memory locality. Using a 48-core Intel Xeon Skylake processor, the proposed algorithm achieves peak performance of 317 billion cell updates per second (GCUPS), and demonstrates near linear weak and strong scaling on up to 48 cores. It delivers significant performance gains compared to existing algorithms, and results in run-time reduction from multiple days to three hours for the problem of optimally aligning high coverage long (PacBioONT) or short (Illumina) DNA reads to an MHC human variation graph containing 10 million vertices.AvailabilityThe implementation of our algorithm is available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsgithub.comParBLiSSPaSGAL&gt;httpsgithub.comParBLiSSPaSGAL&lt;jatsext-link&gt;. Data sets used for evaluation are accessible using &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsalurulab.cc.gatech.eduPaSGAL&gt;httpsalurulab.cc.gatech.eduPaSGAL&lt;jatsext-link&gt;.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/estimations_of_the_weather_effects_on_brain_functions_using_functional_mri__a_cautionary_tale_61184981/">
          <h4 class="font-weight-bold">Estimations of the weather effects on brain functions using functional MRI – a cautionary tale, bioRxiv, 2019-05-28</h4>
          <p>AbstractThe influences of environmental factors such as weather on human brain are still largely unknown. A few neuroimaging studies have demonstrated seasonal effects, but were limited by their cross-sectional design or sample sizes. Most importantly, the stability of MRI scanner hasn’t been taken into account, which may also be affected by environments. In the current study, we analyzed longitudinal resting-state functional MRI (fMRI) data from eight individuals, where the participants were scanned over months to years. We applied machine learning regression to use different resting-state parameters, including amplitude of low-frequency fluctuations (ALFF), regional homogeneity (ReHo), and functional connectivity matrix, to predict different weather and environmental parameters. For a careful control, the raw EPI and the anatomical images were also used in the prediction analysis. We first found that daylight length and temperatures could be reliability predicted using cross-validation using resting-state parameters. However, similar prediction accuracies could also achieved by using one frame of EPI image, and even higher accuracies could be achieved by using segmented or even the raw anatomical images. Finally, we verified that the signals outside of the brain in the anatomical images and signals in phantom scans could also achieve higher prediction accuracies, suggesting that the predictability may be due to the baseline signals of the MRI scanner. After all, we did not identify detectable influences of weather on brain functions other than the influences on the stability of MRI scanners. The results highlight the difficulty of studying long term effects on brain using MRI.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/100-200-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/100-200-users/page/4/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="/tags/100-200-users/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/100-200-users/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/100-200-users/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/100-200-users/page/4/" class="page-link">4</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/100-200-users/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/100-200-users/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/100-200-users/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/100-200-users/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/100-200-users/page/9/" class="page-link">9</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/100-200-users/page/6/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/100-200-users/page/68/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
