<!DOCTYPE html>
<html><head>
  
  <title>Cell Biology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/lrrk2_mediates_tubulation_and_vesicle_sorting_from_membrane_damaged_lysosomes_75420545/">
          <h4 class="font-weight-bold">LRRK2 mediates tubulation and vesicle sorting from membrane damaged lysosomes, bioRxiv, 2020-01-25</h4>
          <p>ABSTRACTMutations in the leucine rich repeat kinase 2 (LRRK2) gene are a cause of familial and sporadic Parkinson’s disease (PD). Nonetheless, the biological functions of LRRK2 remain incompletely understood. Here, we observed that LRRK2 is recruited to lysosomes that have a ruptured membrane. Using unbiased proteomics, we observed that LRRK2 is able to recruit the motor adaptor protein JIP4 to permeabilized lysosomes in a kinase-dependent manner through the phosphorylation of RAB35 and RAB10. Super-resolution live cell imaging microscopy and FIB-SEM revealed that once at the lysosomal membrane, JIP4 promotes the formation of LAMP1-negative lysosomal tubules that release membranous content from ruptured lysosomes. Released vesicular structures are able to interact with other lysosomes. Thus, we described a new process that uses lysosomal tubulation to release vesicular structures from permeabilized lysosomes. LRRK2 orchestrates this process that we name LYTL (LYsosomal Tubulationsorting driven by LRRK2) that, given the central role of the lysosome in PD, is likely to be disease relevant.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_structure_and_global_distribution_of_the_endoplasmic_reticulum_network_is_actively_regulated_by_lysosomes_74074833/">
          <h4 class="font-weight-bold">The structure and global distribution of the endoplasmic reticulum network is actively regulated by lysosomes, bioRxiv, 2020-01-16</h4>
          <p>AbstractThe endoplasmic reticulum (ER) comprises morphologically and functionally distinct domains, sheets and interconnected tubules. These domains undergo dynamic reshaping, in response to changes in the cellular environment. However, the mechanisms behind this rapid remodeling within minutes are largely unknown. Here, we report that ER remodeling is actively driven by lysosomes, following lysosome repositioning in response to changes in nutritional status. The anchorage of lysosomes to ER growth tips is critical for ER tubule elongation and connection. We validate this causal link via the chemo- and optogenetically driven re-positioning of lysosomes, which leads to both a redistribution of the ER tubules and its global morphology. Lysosomes sense metabolic change in the cell and regulate ER tubule distribution accordingly. Dysfunction in this mechanism during axonal extension may lead to axonal growth defects. Our results demonstrate a critical role of lysosome-regulated ER dynamics and reshaping in nutrient responses and neuronal development.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/unconventional_kinetochore_kinases_kkt2_and_kkt3_have_a_unique_zinc_finger_that_promotes_their_kinetochore_localization_72748577/">
          <h4 class="font-weight-bold">Unconventional kinetochore kinases KKT2 and KKT3 have a unique zinc finger that promotes their kinetochore localization, bioRxiv, 2019-12-14</h4>
          <p>AbstractChromosome segregation in eukaryotes is driven by the kinetochore, a macromolecular protein complex that assembles onto centromeric DNA and binds spindle microtubules. Cells must tightly control the number and position of kinetochores so that all chromosomes assemble a single kinetochore. A central player in this process is the centromere-specific histone H3 variant CENP-A, which localizes specifically within centromeres and promotes kinetochore assembly. However, CENP-A is absent from several eukaryotic lineages including kinetoplastids, a group of evolutionarily divergent eukaryotes that have an unconventional set of kinetochore proteins. It remains unknown how kinetoplastids specify kinetochore positions or promote kinetochore assembly in the absence of CENP-A. Here we studied two homologous kinetoplastid kinases (KKT2 and KKT3) that localize constitutively at centromeres. KKT2 and KKT3 central domains were sufficient for centromere localization in Trypanosoma brucei. Crystal structures of the KKT2 central domain from two divergent kinetoplastids revealed a unique zinc finger domain, which promotes its kinetochore localization in T. brucei. Mutations in the equivalent zinc finger domain of KKT3 abolished its kinetochore localization and function. This study lays the foundation for understanding the mechanism of kinetochore specification and assembly in kinetoplastids.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_nucleus_measures_shape_deformation_for_cellular_proprioception_and_regulates_adaptive_morphodynamics_72222727/">
          <h4 class="font-weight-bold">The nucleus measures shape deformation for cellular proprioception and regulates adaptive morphodynamics, bioRxiv, 2019-12-05</h4>
          <p>AbstractThe physical microenvironment regulates cell behavior during tissue development and homeostasis. How single cells decode information about their geometrical shape under mechanical stress and physical space constraints within their local environment remains largely unknown. Here we show that the nucleus, the biggest cellular organelle, functions as a non-dissipative cellular shape deformation gauge that enables cells to continuously measure shape variations on the time scale of seconds. Inner nuclear membrane unfolding together with the relative spatial intracellular positioning of the nucleus provides physical information on the amplitude and type of cellular shape deformation. This adaptively activates a calcium-dependent mechano-transduction pathway, controlling the level of actomyosin contractility and migration plasticity. Our data support that the nucleus establishes a functional module for cellular proprioception that enables cells to sense shape variations for adapting cellular behaviour to their microenvironment.One Sentence SummaryThe nucleus functions as an active deformation sensor that enables cells to adapt their behavior to the tissue microenvironment.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/in_situ_cryo-electron_tomography_reveals_filamentous_actin_within_the_microtubule_lumen_70722436/">
          <h4 class="font-weight-bold">In situ cryo-electron tomography reveals filamentous actin within the microtubule lumen, bioRxiv, 2019-11-18</h4>
          <p>AbstractMicrotubules and filamentous (F-) actin engage in complex interactions to drive many cellular processes from subcellular organisation to cell division and migration. This is thought to be largely controlled by proteins that interface between the two structurally distinct cytoskeletal components. Here, we use cryo-electron tomography to demonstrate that the microtubule lumen can be occupied by extended segments of F-actin in small-molecule induced, microtubule-based cellular projections. We uncover an unexpected versatility in cytoskeletal form that may prompt a significant development of our current models of cellular architecture and offer a new experimental approach for the in-situ study of microtubule structure and contents.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/correlative_three-dimensional_super-resolution_and_block_face_electron_microscopy_of_whole_vitreously_frozen_cells_66817456/">
          <h4 class="font-weight-bold">Correlative three-dimensional super-resolution and block face electron microscopy of whole vitreously frozen cells, bioRxiv, 2019-09-20</h4>
          <p>AbstractLiving cells function through the spatial compartmentalization of thousands of distinct proteins serving a multitude of diverse biochemical needs. Correlative super-resolution (SR) fluorescence and electron microscopy (EM) has emerged as a pathway to directly view nanoscale protein relationships to the underlying global ultrastructure, but has traditionally suffered from tradeoffs of structure preservation, fluorescence retention, resolution, and field of view. We developed a platform for three-dimensional correlative cryogenic SR and focused ion beam milled block-face EM across entire vitreously frozen cells that addresses these issues by preserving native ultrastructure and enabling independent SR and EM workflow optimization. Application to a variety of biological systems revealed a number of unexpected protein-ultrastructure relationships and underscored the value of a comprehensive multimodal view of ultrastructural variability across whole cells.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        

        
             
            <li class="page-item active"><a href="/audiences/tags/cell-biology/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/cell-biology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/cell-biology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/cell-biology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/cell-biology/page/5/" class="page-link">5</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/cell-biology/page/2/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/cell-biology/page/10/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
