<!DOCTYPE html>
<html><head>
  
  <title>2018 | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/a_metaproteomics_method_to_determine_carbon_sources_and_assimilation_pathways_of_species_in_microbial_communities_31461621/">
          <h4 class="font-weight-bold">A metaproteomics method to determine carbon sources and assimilation pathways of species in microbial communities, bioRxiv, 2018-01-10</h4>
          <p>AbstractMeasurements of the carbon stable isotope ratio (δ13C) are widely used in biology to address major questions regarding food sources and metabolic pathways used by organisms. Measurement of these so called stable carbon isotope fingerprints (SIFs) for microbes involved in biogeochemical cycling and microbiota of plants and animals have led to major discoveries in environmental microbiology. Currently, obtaining SIFs for microbial communities is challenging as the available methods either only provide limited taxonomic resolution, such as with the use of lipid biomarkers, or are limited in throughput, such as NanoSIMS imaging of single cells.Here we present “direct Protein-SIF” and the Calis-p software package (&lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpssourceforge.netprojectscalis-p&gt;httpssourceforge.netprojectscalis-p&lt;jatsext-link&gt;), which enable high-throughput measurements of accurate δ13C values for individual species within a microbial community. We benchmark the method using 20 pure culture microorganisms and show that the method reproducibly provides SIF values consistent with gold standard bulk measurements performed with an isotope ratio mass spectrometer. Using mock community samples, we show that SIF values can also be obtained for individual species within a microbial community. Finally, a case study of an obligate bacteria-animal symbiosis showed that direct Protein-SIF confirms previous physiological hypotheses and can provide unexpected new insights into the symbionts’ metabolism. This confirms the usefulness of this new approach to accurately determine δ13C values for different species in microbial community samples.&lt;jatssec id=s1&gt;SignificanceTo understand the roles that microorganisms play in diverse environments such as the open ocean and the human intestinal tract, we need an understanding of their metabolism and physiology. A variety of methods such as metagenomics and metaproteomics exist to assess the metabolism of environmental microorganisms based on gene content and gene expression. These methods often only provide indirect evidence for which substrates are used by a microorganism in a community. The direct Protein-SIF method that we developed allows linking microbial species in communities to the environmental carbon sources they consume by determining their stable carbon isotope signature. Direct Protein-SIF also allows assessing which carbon fixation pathway is used by autotrophic microorganisms that directly assimilate CO2.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/insect_wings_and_body_wall_evolved_from_ancient_leg_segments_31454380/">
          <h4 class="font-weight-bold">Insect wings and body wall evolved from ancient leg segments, bioRxiv, 2018-01-10</h4>
          <p>AbstractThe origin of insect wings has long been debated. Central to this debate is whether wings evolved from an epipod (outgrowth, e.g., a gill) on ancestral crustacean leg segments, or represent a novel outgrowth from the dorsal body wall that co-opted some of the genes used to pattern the epipods. To determine whether wings can be traced to ancestral, pre-insect structures, or arose by co-option, comparisons are necessary between insects and arthropods more representative of the ancestral state, where the hypothesized proximal leg region is not fused to the body wall. To do so, we examined the function of five leg patterning genes in the crustacean Parhyale hawaiensis and compared this to previous functional data from insects. By comparing gene knockout phenotypes of leg patterning genes in a crustacean with those of insects, we show that two ancestral crustacean leg segments were incorporated into the insect body, moving the leg’s epipod dorsally, up onto the back to form insect wings. Thus, our data shows that much of the body wall of insects, including the entire wing, is derived from these two ancestral proximal leg segments. This model explains all observations in favor of either the body wall origin or proximal leg origin of insect wings. Thus, our results show that insect wings are not novel structures, but instead evolved from existing, ancestral structures.One Sentence SummaryCRISPR-Cas9 knockout of leg gap genes in a crustacean reveals that insect wings are not novel structures, they evolved from crustacean leg segments</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/speciation_genes_are_more_likely_to_have_discordant_gene_trees_31423756/">
          <h4 class="font-weight-bold">Speciation genes are more likely to have discordant gene trees, bioRxiv, 2018-01-09</h4>
          <p>AbstractSpeciation genes are responsible for reproductive isolation between species. By directly participating in the process of speciation, the genealogies of isolating loci have been thought to more faithfully represent species trees. The unique properties of speciation genes may provide valuable evolutionary insights and help determine the true history of species divergence. Here, we formally analyze whether genealogies from loci participating in Dobzhansky-Muller (DM) incompatibilities are more likely to be concordant with the species tree under incomplete lineage sorting (ILS). Individual loci differ stochastically from the true history of divergence with a predictable frequency due to ILS, and these expectations—combined with the DM model of intrinsic reproductive isolation from epistatic interactions—can be used to examine the probability of concordance at isolating loci. Contrary to existing verbal models, we find that reproductively isolating loci that follow the DM model are often more likely to have discordant gene trees. These results are dependent on the pattern of isolation observed between three species, the time between speciation events, and the time since the last speciation event. Results supporting a higher probability of discordance are found for both derived-derived and derived-ancestral DM pairs, and regardless of whether incompatibilities are allowed or prohibited from segregating in the same population. Our overall results suggest that DM loci are unlikely to be especially useful for reconstructing species relationships, even in the presence of gene flow between incipient species, and may in fact be positively misleading.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/visualization_of_prc2-dinucleosome_interactions_leading_to_epigenetic_repression_31435909/">
          <h4 class="font-weight-bold">Visualization of PRC2-Dinucleosome Interactions Leading to Epigenetic Repression, bioRxiv, 2018-01-09</h4>
          <p>AbstractEpigenetic regulation is mediated by protein complexes that couple recognition of chromatin marks to activity or recruitment of chromatin-modifying enzymes. Polycomb repressive complex 2 (PRC2), a gene silencer that methylates lysine 27 of histone H3, is stimulated upon recognition of its own catalytic product, and has been shown to be more active on dinucleosomes than H3 tails or single nucleosomes. These properties likely facilitate local H3K27me23 spreading causing heterochromatin formation and gene repression. Here, cryo-EM reconstructions of human PRC2 bound to dinucleosomes show how a single PRC2, interacting with nucleosomal DNA, precisely positions the H3 tails to recognize a H3K27me3 mark in one nucleosome and is stimulated to modify a neighboring nucleosome. The geometry of the PRC2-DNA interactions allow PRC2 to tolerate different dinucleosome geometries due to varying lengths of the linker DNA. Our structures are the first to illustrate how an epigenetic regulator engages with a complex chromatin substrate.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/genome-wide_signals_of_drift_and_local_adaptation_during_rapid_lineage_divergence_in_a_songbird_31377972/">
          <h4 class="font-weight-bold">Genome-wide signals of drift and local adaptation during rapid lineage divergence in a songbird, bioRxiv, 2018-01-08</h4>
          <p>AbstractThe formation of independent evolutionary lineages involves neutral and selective factors, and understanding their relative roles in population divergence is a fundamental goal of speciation research. Correlations between allele frequencies and environmental variability can reveal the role of selection, yet the relative contribution of drift can be difficult to establish. Recently diversified systems such as that of the Oregon junco (Aves Emberizidae) of western North America provide ideal scenarios to apply genetic-environment association analyses (GEA) while controlling for population structure. Genome-wide SNP loci analyses revealed marked genetic structure consisting of differentiated populations in isolated, dry southern mountain ranges, and more admixed recently expanded populations in humid northern latitudes. We used correlations between genomic and environmental variance to test for three specific modes of evolutionary divergence (i) drift in geographic isolation, (ii) differentiation along continuous selective gradients, and (iii) isolation by adaptation. We found evidence of strong drift in southern mountains, but also signals of local adaptation in several populations, driven by temperature, precipitation, elevation and vegetation, especially when controlling for population history. We identified numerous variants under selection scattered across the genome, suggesting that local adaptation can promote rapid differentiation over short periods when acting over multiple independent loci.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/inferring_species_trees_using_integrative_models_of_species_evolution_31374017/">
          <h4 class="font-weight-bold">Inferring Species Trees Using Integrative Models of Species Evolution, bioRxiv, 2018-01-08</h4>
          <p>AbstractBayesian methods can be used to accurately estimate species tree topologies, times and other parameters, but only when the models of evolution which are available and utilized sufficiently account for the underlying evolutionary processes. Multispecies coalescent (MSC) models have been shown to accurately account for the evolution of genes within species in the absence of strong gene flow between lineages, and fossilized birth-death (FBD) models have been shown to estimate divergence times from fossil data in good agreement with expert opinion. Until now dating analyses using the MSC have been based on a fixed clock or informally derived node priors instead of the FBD. On the other hand, dating analyses using an FBD process have concatenated all gene sequences and ignored coalescence processes. To address these mirror-image deficiencies in evolutionary models, we have developed an integrative model of evolution which combines both the FBD and MSC models. By applying concatenation and the MSC (without employing the FBD process) to an exemplar data set consisting of molecular sequence data and morphological characters from the dog and fox subfamily Caninae, we show that concatenation causes predictable biases in estimated branch lengths. We then applied concatenation using the FBD process and the combined FBD-MSC model to show that the same biases are still observed when the FBD process is employed. These biases can be avoided by using the FBD-MSC model, which coherently models fossilization and gene evolution, and does not require an a priori substitution rate estimate to calibrate the molecular clock. We have implemented the FBD-MSC in a new version of StarBEAST2, a package developed for the BEAST2 phylogenetic software.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/2018/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/2018/page/51/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/2018/page/48/" class="page-link">48</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2018/page/49/" class="page-link">49</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2018/page/50/" class="page-link">50</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2018/page/51/" class="page-link">51</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/2018/page/52/" class="page-link">52</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2018/page/53/" class="page-link">53</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2018/page/54/" class="page-link">54</a></li>
            
        

        
        <li class="page-item"><a href="/tags/2018/page/53/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/2018/page/54/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
