<!DOCTYPE html>
<html><head>
  
  <title>Genetics | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/super-mendelian_inheritance_mediated_by_crisprcas9_in_the_female_mouse_germline_44494359/">
          <h4 class="font-weight-bold">Super-Mendelian inheritance mediated by CRISPRCas9 in the female mouse germline, bioRxiv, 2018-07-04</h4>
          <p>AbstractA gene drive biases the transmission of a particular allele of a gene such that it is inherited at a greater frequency than by random assortment. Recently, a highly efficient gene drive was developed in insects, which leverages the sequence-targeted DNA cleavage activity of CRISPRCas9 and endogenous homology directed repair mechanisms to convert heterozygous genotypes to homozygosity. If implemented in laboratory rodents, this powerful system would enable the rapid assembly of genotypes that involve multiple genes (e.g., to model multigenic human diseases). Such complex genetic models are currently precluded by time, cost, and a requirement for a large number of animals to obtain a few individuals of the desired genotype. However, the efficiency of a CRISPRCas9 gene drive system in mammals has not yet been determined. Here, we utilize an active genetic “CopyCat” element embedded in the mouse Tyrosinase gene to detect genotype conversions after Cas9 activity in the embryo and in the germline. Although Cas9 efficiently induces double strand DNA breaks in the early embryo and is therefore highly mutagenic, these breaks are not resolved by homology directed repair. However, when Cas9 expression is limited to the developing female germline, resulting double strand breaks are resolved by homology directed repair that copies the CopyCat allele from the donor to the receiver chromosome and leads to its super-Mendelian inheritance. These results demonstrate that the CRISPRCas9 gene drive mechanism can be implemented to simplify complex genetic crosses in laboratory mice and also contribute valuable data to the ongoing debate about applications to combat invasive rodent populations in island communities.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/robust_genome_editing_with_short_single-stranded_and_long_partially_single-stranded_dna_donors_in_c._elegans_43927432/">
          <h4 class="font-weight-bold">Robust genome editing with short single-stranded and long, partially single-stranded DNA donors in C. elegans, bioRxiv, 2018-06-20</h4>
          <p>AbstractCRISPR-based genome editing using ribonucleoprotein (RNP) complexes and synthetic single stranded oligodeoxynucleotide (ssODN) donors can be highly effective. However, reproducibility can vary, and precise, targeted integration of longer constructs – such as green fluorescent protein (GFP) tags remains challenging in many systems. Here we describe a streamlined and optimized editing protocol for the nematode C. elegans. We demonstrate its efficacy, flexibility, and cost-effectiveness by affinity-tagging all twelve of the Worm-specific Argonaute (WAGO) proteins in C. elegans using ssODN donors. In addition, we describe a novel PCR-based partially single-stranded “hybrid” donor design that yields high efficiency editing with large (kilobase-scale) constructs. We use these hybrid donors to introduce fluorescent protein tags into multiple loci achieving editing efficiencies that approach those previously obtained only with much shorter ssODN donors. The principals and strategies described here are likely to translate to other systems and should allow researchers to reproducibly and efficiently obtain both long and short precision genome edits.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genetic_compensation_is_triggered_by_mutant_mrna_degradation_42383030/">
          <h4 class="font-weight-bold">Genetic compensation is triggered by mutant mRNA degradation, bioRxiv, 2018-05-22</h4>
          <p>Genetic compensation by transcriptional modulation of related gene(s) (also known as transcriptional adaptation) has been reported in numerous systems1–3; however, whether and how such a response can be activated in the absence of protein feedback loops is unknown. Here, we develop and analyze several models of transcriptional adaptation in zebrafish and mouse that we show are not caused by loss of protein function. We find that the increase in transcript levels is due to enhanced transcription, and observe a correlation between the levels of mutant mRNA decay and transcriptional upregulation of related genes. To assess the role of mutant mRNA degradation in triggering transcriptional adaptation, we use genetic and pharmacological approaches and find that mRNA degradation is indeed required for this process. Notably, uncapped RNAs, themselves subjected to rapid degradation, can also induce transcriptional adaptation. Next, we generate alleles that fail to transcribe the mutated gene and find that they do not show transcriptional adaptation, and exhibit more severe phenotypes than those observed in alleles displaying mutant mRNA decay. Transcriptome analysis of these different alleles reveals the upregulation of hundreds of genes with enrichment for those showing sequence similarity with the mutated gene’s mRNA, suggesting a model whereby mRNA degradation products induce the response via sequence similarity. These results expand the role of the mRNA surveillance machinery in buffering against mutations by triggering the transcriptional upregulation of related genes. Besides implications for our understanding of disease-causing mutations, our findings will help design mutant alleles with minimal transcriptional adaptation-derived compensation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genomic_sem_provides_insights_into_the_multivariate_genetic_architecture_of_complex_traits_38614501/">
          <h4 class="font-weight-bold">Genomic SEM Provides Insights into the Multivariate Genetic Architecture of Complex Traits, bioRxiv, 2018-04-21</h4>
          <p>AbstractMethods for using GWAS to estimate genetic correlations between pairwise combinations of traits have produced “atlases” of genetic architecture. Genetic atlases reveal pervasive pleiotropy, and genome-wide significant loci are often shared across different phenotypes. We introduce genomic structural equation modeling (Genomic SEM), a multivariate method for analyzing the joint genetic architectures of complex traits. Using formal methods for modeling covariance structure, Genomic SEM synthesizes genetic correlations and SNP-heritabilities inferred from GWAS summary statistics of individual traits from samples with varying and unknown degrees of overlap. Genomic SEM can be used to identify variants with effects on general dimensions of cross-trait liability, boost power for discovery, and calculate more predictive polygenic scores. Finally, Genomic SEM can be used to identify loci that cause divergence between traits, aiding the search for what uniquely differentiates highly correlated phenotypes. We demonstrate several applications of Genomic SEM, including a joint analysis of GWAS summary statistics from five genetically correlated psychiatric traits. We identify 27 independent SNPs not previously identified in the univariate GWASs, 5 of which have been reported in other published GWASs of the included traits. Polygenic scores derived from Genomic SEM consistently outperform polygenic scores derived from GWASs of the individual traits. Genomic SEM is flexible, open ended, and allows for continuous innovations in how multivariate genetic architecture is modeled.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/large-scale_neuroimaging_and_genetic_study_reveals_genetic_architecture_of_brain_white_matter_microstructure_34840408/">
          <h4 class="font-weight-bold">Large-scale neuroimaging and genetic study reveals genetic architecture of brain white matter microstructure, bioRxiv, 2018-03-26</h4>
          <p>AbstractMicrostructural changes of white matter (WM) tracts are known to be associated with various neuropsychiatric disordersdiseases. Heritability of structural changes of WM tracts has been examined using diffusion tensor imaging (DTI) in family-based studies for different age groups. The availability of genetic and DTI data from recent large population-based studies offers opportunity to further improve our understanding of genetic contributions. Here, we analyzed the genetic architecture of WM tracts using DTI and single-nucleotide polymorphism (SNP) data of unrelated individuals in the UK Biobank (n ∼ 8000). The DTI parameters were generated using the ENIGMA-DTI pipeline. We found that DTI parameters are substantially heritable on most WM tracts. We observed a highly polygenic or omnigenic architecture of genetic influence across the genome as well as the enrichment of SNPs in active chromatin regions. Our bivariate analyses showed strong genetic correlations for several pairs of WM tracts as well as pairs of DTI parameters. We performed voxel-based analysis to illustrate the pattern of genetic effects on selected parts of the tract-based spatial statistics skeleton. Comparing the estimates from the UK Biobank to those from small population-based studies, we illustrated that sufficiently large sample size is essential for genetic architecture discovery in imaging genetics. We confirmed this finding with a simulation study.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/meta-analysis_of_genome-wide_association_studies_for_height_and_body_mass_index_in_700000_individuals_of_european_ancestry_33863466/">
          <h4 class="font-weight-bold">Meta-analysis of genome-wide association studies for height and body mass index in ∼700,000 individuals of European ancestry, bioRxiv, 2018-03-03</h4>
          <p>Genome-wide association studies (GWAS) stand as powerful experimental designs for identifying DNA variants associated with complex traits and diseases. In the past decade, both the number of such studies and their sample sizes have increased dramatically. Recent GWAS of height and body mass index (BMI) in ∼250,000 European participants have led to the discovery of ∼700 and ∼100 nearly independent SNPs associated with these traits, respectively. Here we combine summary statistics from those two studies with GWAS of height and BMI performed in ∼450,000 UK Biobank participants of European ancestry. Overall, our combined GWAS meta-analysis reaches N∼700,000 individuals and substantially increases the number of GWAS signals associated with these traits. We identified 3,290 and 716 near-independent SNPs associated with height and BMI, respectively (at a revised genome-wide significance threshold of p&amp;lt;1 × 10−8), including 1,185 height-associated SNPs and 554 BMI-associated SNPs located within loci not previously identified by these two GWAS. The genome-wide significant SNPs explain ∼24.6% of the variance of height and ∼5% of the variance of BMI in an independent sample from the Health and Retirement Study (HRS). Correlations between polygenic scores based upon these SNPs with actual height and BMI in HRS participants were 0.44 and 0.20, respectively. From analyses of integrating GWAS and eQTL data by Summary-data based Mendelian Randomization (SMR), we identified an enrichment of eQTLs amongst lead height and BMI signals, prioritisting 684 and 134 genes, respectively. Our study demonstrates that, as previously predicted, increasing GWAS sample sizes continues to deliver, by discovery of new loci, increasing prediction accuracy and providing additional data to achieve deeper insight into complex trait biology. All summary statistics are made available for follow up studies.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/genetics/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/genetics/page/24/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../../tags/genetics/page/21/" class="page-link">21</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/genetics/page/22/" class="page-link">22</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/genetics/page/23/" class="page-link">23</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/genetics/page/24/" class="page-link">24</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/genetics/page/25/" class="page-link">25</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/genetics/page/26/" class="page-link">26</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/genetics/page/27/" class="page-link">27</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/genetics/page/28/" class="page-link">28</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/genetics/page/29/" class="page-link">29</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/genetics/page/26/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/genetics/page/36/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
