<!DOCTYPE html>
<html><head>
  
  <title>2018 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/inference_of_recombination_maps_from_a_single_pair_of_genomes_and_its_application_to_archaic_samples_50219053/">
          <h4 class="font-weight-bold">Inference of recombination maps from a single pair of genomes and its application to archaic samples, bioRxiv, 2018-10-25</h4>
          <p>ABSTRACTUnderstanding the causes and consequences of recombination rate evolution is a fundamental goal in genetics that requires recombination maps from across the tree of life. Since statistical inference of recombination maps typically depends on large samples, reaching out studies to non-model organisms requires alternative tools. Here we extend the sequentially Markovian coalescent model to jointly infer demography and the variation in recombination along a pair of genomes. Using extensive simulations and sequence data from humans, fruit-flies and a fungal pathogen, we demonstrate that iSMC accurately infers recombination maps under a wide range of scenarios – remarkably, even from a single pair of unphased genomes. We exploit this possibility and reconstruct the recombination maps of archaic hominids. We report that the evolution of the recombination landscape follows the established phylogeny of Neandertals, Denisovans and modern human populations, as expected if the genomic distribution of crossovers in hominids is largely neutral.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/microbiota_profiling_with_long_amplicons_using_nanopore_sequencing_full-length_16s_rrna_gene_and_whole_rrn_operon_50175188/">
          <h4 class="font-weight-bold">Microbiota profiling with long amplicons using Nanopore sequencing full-length 16S rRNA gene and whole rrn operon, bioRxiv, 2018-10-24</h4>
          <p>Background Profiling microbiome on low biomass samples is challenging for metagenomics since these samples are prone to present DNA from other sources, such as the host or the environment. The usual approach is sequencing specific hypervariable regions of the 16S rRNA gene, which fails to assign taxonomy to genus and species level. Here, we aim to assess long-amplicon PCR-based approaches for assigning taxonomy at the genus and species level. We use Nanopore sequencing with two different markers full-length 16S rRNA (~1,500 bp) and the whole rrn operon (16S rRNA gene - ITS - 23S rRNA gene; 4,500 bp).Methods We sequenced a clinical isolate of Staphylococcus pseudintermedius, two mock communities (HM-783D, Bei Resources; D6306, ZymoBIOMICS) and two pools of low-biomass samples (dog skin). Nanopore sequencing was performed on MinION (Oxford Nanopore Technologies) using 1D PCR barcoding kit. Sequences were pre-processed, and data were analyzed using WIMP workflow on EPI2ME (ONT) or Minimap2 software with rrn database.Results Full-length 16S rRNA and the rrn operon retrieved the microbiota composition from the bacterial isolate, the mock communities and the complex skin samples, even at the genus and species level. For Staphylococcus pseudintermedius isolate, when using EPI2ME, the amplicons were assigned to the correct bacterial species in ~98% of the cases with rrn operon as the marker, and ~68% of the cases with 16S rRNA gene respectively. In both skin microbiota samples, we detected many species with an environmental origin. In chin, we found different Pseudomonas species in high abundance, whereas in the dorsal skin there were more taxa with lower abundances.Conclusions Both full-length 16S rRNA and the rrn operon retrieved the microbiota composition of simple and complex microbial communities, even from the low-biomass samples such as dog skin. For an increased resolution at the species level, rrn operon would be the best choice.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/origins_and_evolution_of_the_global_rna_virome_50175926/">
          <h4 class="font-weight-bold">Origins and Evolution of the Global RNA Virome, bioRxiv, 2018-10-24</h4>
          <p>AbstractViruses with RNA genomes dominate the eukaryotic virome, reaching enormous diversity in animals and plants. The recent advances of metaviromics prompted us to perform a detailed phylogenomic reconstruction of the evolution of the dramatically expanded global RNA virome. The only universal gene among RNA viruses is the RNA-dependent RNA polymerase (RdRp). We developed an iterative computational procedure that alternates the RdRp phylogenetic tree construction with refinement of the underlying multiple sequence alignments. The resulting tree encompasses 4,617 RNA virus RdRps and consists of 5 major branches, 2 of which include positive-sense RNA viruses, 1 is a mix of positive-sense (&#43;) RNA and double-stranded (ds) RNA viruses, and 2 consist of dsRNA and negative-sense (−) RNA viruses, respectively. This tree topology implies that dsRNA viruses evolved from &#43;RNA viruses on at least two independent occasions, whereas -RNA viruses evolved from dsRNA viruses. Reconstruction of RNA virus evolution using the RdRp tree as the scaffold suggests that the last common ancestors of the major branches of &#43;RNA viruses encoded only the RdRp and a single jelly-roll capsid protein. Subsequent evolution involved independent capture of additional genes, particularly, those encoding distinct RNA helicases, enabling replication of larger RNA genomes and facilitating virus genome expression and virus-host interactions. Phylogenomic analysis reveals extensive gene module exchange among diverse viruses and horizontal virus transfer between distantly related hosts. Although the network of evolutionary relationships within the RNA virome is bound to further expand, the present results call for a thorough reevaluation of the RNA virus taxonomy.IMPORTANCEThe majority of the diverse viruses infecting eukaryotes have RNA genomes, including numerous human, animal, and plant pathogens. Recent advances of metagenomics have led to the discovery of many new groups of RNA viruses in a wide range of hosts. These findings enable a far more complete reconstruction of the evolution of RNA viruses than what was attainable previously. This reconstruction reveals the relationships between different Baltimore Classes of viruses and indicates extensive transfer of viruses between distantly related hosts, such as plants and animals. These results call for a major revision of the existing taxonomy of RNA viruses.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/fix_your_membrane_receptor_imaging_actin_cytoskeleton_and_cd4_membrane_organization_disruption_by_chemical_fixation_50127531/">
          <h4 class="font-weight-bold">Fix your membrane receptor imaging Actin cytoskeleton and CD4 membrane organization disruption by chemical fixation, bioRxiv, 2018-10-23</h4>
          <p>Single-molecule localization microscopy (SMLM) techniques allow near molecular scale resolution (~ 20nm) as well as precise and robust analysis of protein organization at different scales. SMLM hardware, analytics and probes have been the focus of a variety of studies and are now commonly used in laboratories across the world. Protocol reliability and artefact identification are increasingly seen as important aspects of super-resolution microscopy. The reliability of these approaches thus requires in-depth evaluation so that biological findings are based on solid foundations. Here we explore how different fixation approaches that disrupt or preserve the actin cytoskeleton affect membrane protein organization. Using CD4 as a model, we show that fixation-mediated disruption of the actin cytoskeleton correlates with changes in CD4 membrane organization. We highlight how these artefacts are easy to overlook and how careful sample preparation is essential for extracting meaningful results from super-resolution microscopy.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/latent_developmental_potential_to_form_limb-like_skeletal_structures_in_zebrafish_50117443/">
          <h4 class="font-weight-bold">Latent developmental potential to form limb-like skeletal structures in zebrafish, bioRxiv, 2018-10-23</h4>
          <p>AbstractThe evolution of fins into limbs was a key transition in vertebrate history. A hallmark of this transition is the addition of multiple long bones to the proximal-distal axis of paired appendages. Whereas limb skeletons are often elaborate and diverse, teleost pectoral fins retain a simple endoskeleton. Fins and limbs share many core developmental processes, but how these programs were reshaped to produce limbs from fins during evolution remains enigmatic. Here we identify zebrafish mutants that form supernumerary long bones along the proximal-distal axis of pectoral fins with limb-like patterning. These new skeletal elements are integrated into the fin, as they are connected to the musculature, form joints, and articulate with neighboring bones. This phenotype is caused by activating mutations in previously unrecognized regulators of appendage development, vav2 and waslb, which we show function in a common pathway. We find that this pathway functions in appendage development across vertebrates, and loss of Wasl in developing limbs results in patterning defects identical to those seen in Hoxall knockout mice. Concordantly, formation of supernumerary fin long bones requires the function of hoxall paralogs, indicating developmental homology with the forearm and the existence of a latent functional Hox code patterning the fin endoskeleton. Our findings reveal an inherent limb-like patterning ability in fins that can be activated by simple genetic perturbation, resulting in the elaboration of the endoskeleton.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/polygenic_adaptation_from_sweeps_to_subtle_frequency_shifts_50132793/">
          <h4 class="font-weight-bold">Polygenic Adaptation From sweeps to subtle frequency shifts, bioRxiv, 2018-10-23</h4>
          <p>Evolutionary theory has produced two conflicting paradigms for the adaptation of a polygenic trait. While population genetics views adaptation as a sequence of selective sweeps at single loci underlying the trait, quantitative genetics posits a collective response, where phenotypic adaptation results from subtle allele frequency shifts at many loci. Yet, a synthesis of these views is largely missing and the population genetic factors that favor each scenario are not well understood. Here, we study the architecture of adaptation of a binary polygenic trait (such as resistance) with negative epistasis among the loci of its basis. The genetic structure of this trait allows for a full range of potential architectures of adaptation, ranging from sweeps to small frequency shifts. By combining computer simulations and a newly devised analytical framework based on Yule branching processes, we gain a detailed understanding of the adaptation dynamics for this trait. Our key analytical result is an expression for the joint distribution of mutant alleles at the end of the adaptive phase. This distribution characterizes the polygenic pattern of adaptation at the underlying genotype when phenotypic adaptation has been accomplished. We find that a single compound parameter, the population-scaled background mutation rate Θbg, explains the main differences among these patterns. For a focal locus, Θbg measures the mutation rate at all redundant loci in its genetic background that offer alternative ways for adaptation. For adaptation starting from mutation-selection-drift balance, we observe different patterns in three parameter regions. Adaptation proceeds by sweeps for small Θbg ≲0.1, while small polygenic allele frequency shifts require large Θbg ≳100. In the large intermediate regime, we observe a heterogeneous pattern of partial sweeps at several interacting loci.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/2018/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/2018/page/20/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/17/" class="page-link">17</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/18/" class="page-link">18</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/19/" class="page-link">19</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/20/" class="page-link">20</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/2018/page/21/" class="page-link">21</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/22/" class="page-link">22</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/23/" class="page-link">23</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/24/" class="page-link">24</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2018/page/25/" class="page-link">25</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/2018/page/22/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/2018/page/54/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
