<!DOCTYPE html>
<html><head>
  
  <title>Reports | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.68.3" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.95d2483cd3c5d2bf47c1f36307a456787d60ad958c5efe44db9afc47012b1b05b8e1f64f6ac291650439322a6c38e86eb7cd6254255516d857c23f299fb0e666.js" integrity="sha512-ldJIPNPF0r9HwfNjB6RWeH1grZWMXv5E25r8RwErGwW44fZPasKRZQQ5MipsOOhut81iVCVVFthXwj8pn7DmZg=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/detection_of_human_adaptation_during_the_past_2000_years_7282628/">
          <h4 class="font-weight-bold">Detection of human adaptation during the past 2,000 years, bioRxiv, 2016-05-08</h4>
          <p>AbstractDetection of recent natural selection is a challenging problem in population genetics, as standard methods generally integrate over long timescales. Here we introduce the Singleton Density Score (SDS), a powerful measure to infer very recent changes in allele frequencies from contemporary genome sequences. When applied to data from the UK10K Project, SDS reflects allele frequency changes in the ancestors of modern Britons during the past 2,000 years. We see strong signals of selection at lactase and HLA, and in favor of blond hair and blue eyes. Turning to signals of polygenic adaptation we find, remarkably, that recent selection for increased height has driven allele frequency shifts across most of the genome. Moreover, we report suggestive new evidence for polygenic shifts affecting many other complex traits. Our results suggest that polygenic adaptation has played a pervasive role in shaping genotypic and phenotypic variation in modern humans.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2016">2016</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/ld_hub_a_centralized_database_and_web_interface_to_perform_ld_score_regression_that_maximizes_the_potential_of_summary_level_gwas_data_for_snp_heritability_and_genetic_correlation_analysis_7135656/">
          <h4 class="font-weight-bold">LD Hub a centralized database and web interface to perform LD score regression that maximizes the potential of summary level GWAS data for SNP heritability and genetic correlation analysis, bioRxiv, 2016-05-04</h4>
          <p>AbstractMotivationLD score regression is a reliable and efficient method of using genome-wide association study (GWAS) summary-level results data to estimate the SNP heritability of complex traits and diseases, partition this heritability into functional categories, and estimate the genetic correlation between different phenotypes. Because the method relies on summary level results data, LD score regression is computationally tractable even for very large sample sizes. However, publicly available GWAS summary-level data are typically stored in different databases and have different formats, making it difficult to apply LD score regression to estimate genetic correlations across many different traits simultaneously.ResultsIn this manuscript, we describe LD Hub – a centralized database of summary-level GWAS results for 177 diseasestraits from different publicly available resourcesconsortia and a web interface that automates the LD score regression analysis pipeline. To demonstrate functionality and validate our software, we replicated previously reported LD score regression analyses of 49 traitsdiseases using LD Hub; and estimated SNP heritability and the genetic correlation across the different phenotypes. We also present new results obtained by uploading a recent atopic dermatitis GWAS meta-analysis to examine the genetic correlation between the condition and other potentially related traits. In response to the growing availability of publicly accessible GWAS summary-level results data, our database and the accompanying web interface will ensure maximal uptake of the LD score regression methodology, provide a useful database for the public dissemination of GWAS results, and provide a method for easily screening hundreds of traits for overlapping genetic aetiologies.Availability and implementationThe web interface and instructions for using LD Hub are available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpldsc.broadinstitute.org&gt;httpldsc.broadinstitute.org&lt;jatsext-link&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2016">2016</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/is_most_published_research_really_false_7043540/">
          <h4 class="font-weight-bold">Is most published research really false?, bioRxiv, 2016-04-28</h4>
          <p>AbstractThere has been an increasing concern that most published medical findings are false. But what does it mean to be false? Here we describe the range of definitions of false discoveries in the scientific literature. We summarize the philosophical, statistical, and experimental evidence for each type of false discovery. We discuss common underpinning problems with the scientific and data analytic practices and point to tools and behaviors that can be implemented to reduce the problems with published scientific results.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/scientific-communication-and-education">scientific-communication-and-education</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2016">2016</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/dengue_virus_antibodies_enhance_zika_virus_infection_6955826/">
          <h4 class="font-weight-bold">Dengue Virus Antibodies Enhance Zika Virus Infection, bioRxiv, 2016-04-26</h4>
          <p>AbstractBackgroundFor decades, human infections with Zika virus (ZIKV), a mosquito-transmitted flavivirus, were sporadic, associated with mild disease, and went underreported since symptoms were similar to other acute febrile diseases endemic in the same regions. Recent reports of severe disease associated with ZIKV, including Guillain-Barré syndrome and severe fetal abnormalities, have greatly heightened awareness. Given its recent history of rapid spread in immune naïve populations, it is anticipated that ZIKV will continue to spread in the Americas and globally in regions where competent Aedes mosquito vectors are found. Globally, dengue virus (DENV) is the most common mosquito-transmitted human flavivirus and is both well-established and the source of outbreaks in areas of recent ZIKV introduction. DENV and ZIKV are closely related, resulting in substantial antigenic overlap. Through a mechanism known as antibody-dependent enhancement (ADE), anti-DENV antibodies can enhance the infectivity of DENV for certain classes of immune cells, causing increased viral production that correlates with severe disease outcomes. Similarly, ZIKV has been shown to undergo ADE in response to antibodies generated by other flaviviruses. However, response to DENV antibodies has not yet been investigated.Methodology  Principal FindingsWe tested the neutralizing and enhancing potential of well-characterized broadly neutralizing human anti-DENV monoclonal antibodies (HMAbs) and human DENV immune sera against ZIKV using neutralization and ADE assays. We show that anti-DENV HMAbs, cross-react, do not neutralize, and greatly enhance ZIKV infection in vitro. DENV immune sera had varying degrees of neutralization against ZIKV and similarly enhanced ZIKV infection.Conclusions  SignificanceOur results suggest that pre-existing DENV immunity will enhance ZIKV infection in vivo and may increase disease severity. A clear understanding of the interplay between ZIKV and DENV will be critical in informing public health responses in regions where these viruses co-circulate and will be particularly valuable for ZIKV and DENV vaccine design and implementation strategies.Author SummaryRecent reports of severe disease, including developmental problems in newborns, have greatly heightened public health awareness of Zika virus (ZIKV), a mosquito-transmitted virus for which there is no vaccine or treatment. It is anticipated that ZIKV will continue to spread in the Americas and globally in regions where competent mosquitoes are found. Dengue virus (DENV), a closely related mosquito-transmitted virus is well-established in regions of recent ZIKV introduction and spread. It is increasingly common that individuals living in these regions may have had a prior DENV infection or may be infected with DENV and ZIKV at the same time. However, very little is known about the impact of DENV infections on ZIKV disease severity. In this study, we tested the ability of antibodies against DENV to prevent or enhance ZIKV infection in cell culture-based assays. We found that DENV antibodies can greatly enhance ZIKV infection in cells.Our results suggest that ZIKV infection in individuals that had a prior DENV infection may experience more severe clinical manifestations. The results of this study provide a better understanding of the interplay between ZIKV and DENV infections that can serve to inform public health responses and vaccine strategies.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2016">2016</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/first_detection_of_zika_virus_in_neotropical_primates_in_brazil_a_possible_new_reservoir_6845621/">
          <h4 class="font-weight-bold">First detection of Zika virus in neotropical primates in Brazil a possible new reservoir, bioRxiv, 2016-04-21</h4>
          <p>Samples from sera and oral swabs from fifteen marmosets (Callithrix jacchus) and nine capuchin-monkeys (Sapajus libidinosus) captured in Ceara State in Brazil were tested for Zika virus.  Samples were positive by Real time PCR and sequencing of the amplified product from a capuchin monkey showed 100% similarity to other ZIKV from South America. This is the first report on ZIKV detection among Neotropical primates.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2016">2016</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/prevalence_phenotype_and_architecture_of_developmental_disorders_caused_by_de_novo_mutation_the_deciphering_developmental_disorders_study_6845275/">
          <h4 class="font-weight-bold">Prevalence, phenotype and architecture of developmental disorders caused by de novo mutation The Deciphering Developmental Disorders Study, bioRxiv, 2016-04-21</h4>
          <p>AbstractIndividuals with severe, undiagnosed developmental disorders (DDs) are enriched for damaging de novo mutations (DNMs) in developmentally important genes. We exome sequenced 4,293 families with individuals with DDs, and meta-analysed these data with published data on 3,287 individuals with similar disorders. We show that the most significant factors influencing the diagnostic yield of de novo mutations are the sex of the affected individual, the relatedness of their parents and the age of both father and mother. We identified 94 genes enriched for damaging de novo mutation at genome-wide significance (P &amp;lt; 7 × 10−7), including 14 genes for which compelling data for causation was previously lacking. We have characterised the phenotypic diversity among these genetic disorders. We demonstrate that, at current cost differentials, exome sequencing has much greater power than genome sequencing for novel gene discovery in genetically heterogeneous disorders. We estimate that 42% of our cohort carry pathogenic DNMs (single nucleotide variants and indels) in coding sequences, with approximately half operating by a loss-of-function mechanism, and the remainder resulting in altered-function (e.g. activating, dominant negative). We established that most haplo insufficient developmental disorders have already been identified, but that many altered-function disorders remain to be discovered. Extrapolating from the DDD cohort to the general population, we estimate that developmental disorders caused by DNMs have an average birth prevalence of 1 in 213 to 1 in 448 (0.22-0.47% of live births), depending on parental age.Abbreviations&lt;jatsdef-list&gt;&lt;jatsdef-item&gt;PTVProtein-Truncating Variant&lt;jatsdef-item&gt;&lt;jatsdef-item&gt;DNMDe Novo Mutation&lt;jatsdef-item&gt;&lt;jatsdef-item&gt;DDDevelopmental Disorder&lt;jatsdef-item&gt;&lt;jatsdef-item&gt;DDDDeciphering Developmental Disorders study&lt;jatsdef-item&gt;&lt;jatsdef-list&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2016">2016</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/reports/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/reports/page/305/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/reports/page/302/" class="page-link">302</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/reports/page/303/" class="page-link">303</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/reports/page/304/" class="page-link">304</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/reports/page/305/" class="page-link">305</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/reports/page/306/" class="page-link">306</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/reports/page/307/" class="page-link">307</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/reports/page/308/" class="page-link">308</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/reports/page/309/" class="page-link">309</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/reports/page/310/" class="page-link">310</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/reports/page/307/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/reports/page/320/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
