<!DOCTYPE html>
<html><head>
  
  <title>2019 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/multiparametric_phenotyping_of_compound_effects_on_patient_derived_organoids_61886352/">
          <h4 class="font-weight-bold">Multiparametric phenotyping of compound effects on patient derived organoids, bioRxiv, 2019-06-07</h4>
          <p>AbstractPatient derived organoids (PDOs) closely resemble individual tumor biology and allow testing of small molecules ex vivo. To systematically dissect compound effects on 3D organoids, we developed a high-throughput imaging and quantitative analysis approach. We generated PDOs from colorectal cancer patients, treated them with &amp;gt;500 small molecules and captured &amp;gt;3 million images by confocal microscopy. We developed the software framework SCOPE to measure compound induced re-organization of PDOs. We found diverse, but re-occurring phenotypes that clustered by compound mode-of-action. Complex phenotypes were not congruent with PDO viability and many were specific to subsets of PDO lines or were influenced by recurrent mutations. We further analyzed specific phenotypes induced by compound classes and found GSK3 inhibitors to disassemble PDOs via focal adhesion signaling or that MEK inhibition led to bloating of PDOs by enhancing of stemness. Finally, by viability classification, we show heterogeneous susceptibilities of PDOs to clinical anticancer drugs.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/netosis_proceeds_by_cytoskeleton_and_endomembrane_disassembly_and_pad4-mediated_chromatin_de-condensation_and_nuclear_envelope_rupture_61630599/">
          <h4 class="font-weight-bold">NETosis proceeds by cytoskeleton and endomembrane disassembly and PAD4-mediated chromatin de-condensation and nuclear envelope rupture, bioRxiv, 2019-06-07</h4>
          <p>AbstractNeutrophil extracellular traps (NETs) are web-like DNA structures decorated with histones and cytotoxic proteins that are released by activated neutrophils to trap and neutralize pathogens during the innate immune response, but also form in and exacerbate sterile inflammation. Peptidylarginine deiminase 4 (PAD4) citrullinates histones and is required for NET formation (NETosis) in mouse neutrophils. While the in vivo impact of NETs is accumulating, the cellular events driving NETosis and the role of PAD4 in these events are unclear. We performed high resolution time-lapse microscopy of mouse and human neutrophils (PMN) and differentiated HL-60 neutrophil-like cells (dHL-60) labelled with fluorescent markers of organelles and stimulated with ionomycin or lipopolysaccharides to induce NETosis. Upon stimulation, cells exhibited rapid disassembly of the actin cytoskeleton, followed by shedding of plasma membrane microvesicles, disassembly and remodeling of the microtubule and vimentin cytoskeletons, ER vesiculation, chromatin de-condensation and nuclear rounding, progressive plasma membrane and nuclear envelope (NE) permeabilization, nuclear lamin meshwork and then NE rupture to release DNA into the cytoplasm, and finally plasma membrane rupture and discharge of extracellular DNA. Inhibition of actin disassembly blocked NET release. Mouse and dHL-60 cells bearing genetic alteration of PAD4 showed that chromatin de-condensation, lamin meshwork and NE rupture and extracellular DNA release required the enzymatic and nuclear localization activities of PAD4. Thus, NETosis proceeds by a step-wise sequence of cellular events culminating in the PAD4-mediated expulsion of DNA.Significance StatementNeutrophils are white blood cells specialized as the first line of host defense in the immune system. One way they protect organisms is through NETosis, in which they expel their DNA to form a web-like trap that ensnares pathogens and promotes clotting. However, NETs also mediate sterile inflammation, causing damage to the body. We used high-resolution live-cell microscopy to perform the first systematic characterization of the timing of dynamic cellular events leading to NETosis in human and mouse neutrophils and a neutrophil-like cell line. We discovered that NETosis proceeds by a step-wise sequence of cellular events that is conserved across species, and requires the activity of the PAD4 enzyme for DNA to be released from the nucleus and cell membrane.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/rna_velocity_and_protein_acceleration_from_single-cell_multiomics_experiments_61628780/">
          <h4 class="font-weight-bold">RNA velocity and protein acceleration from single-cell multiomics experiments, bioRxiv, 2019-06-07</h4>
          <p>AbstractThe simultaneous quantification of protein and RNA makes possible the inference of past, present and future cell states from single experimental snapshots. To enable such temporal analysis from multimodal single-cell experiments, we introduce an extension of the RNA velocity method that leverages estimates of unprocessed transcript and protein abundances to extrapolate cell states. We apply the model to four datasets and demonstrate consistency among landscapes and phase portraits.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_murine_transcriptome_reveals_global_aging_nodes_with_organ-specific_phase_and_amplitude_61878680/">
          <h4 class="font-weight-bold">The murine transcriptome reveals global aging nodes with organ-specific phase and amplitude, bioRxiv, 2019-06-07</h4>
          <p>Aging is the single greatest cause of disease and death worldwide, and so understanding the associated processes could vastly improve quality of life. While the field has identified major categories of aging damage such as altered intercellular communication, loss of proteostasis, and eroded mitochondrial function1, these deleterious processes interact with extraordinary complexity within and between organs. Yet, a comprehensive analysis of aging dynamics organism-wide is lacking. Here we performed RNA-sequencing of 17 organs and plasma proteomics at 10 ages across the mouse lifespan. We uncover previously unknown linear and non-linear expression shifts during aging, which cluster in strikingly consistent trajectory groups with coherent biological functions, including extracellular matrix regulation, unfolded protein binding, mitochondrial function, and inflammatory and immune response. Remarkably, these gene sets are expressed similarly across tissues, differing merely in age of onset and amplitude. Especially pronounced is widespread immune cell activation, detectable first in white adipose depots in middle age. Single-cell RNA-sequencing confirms the accumulation of adipose T and B cells, including immunoglobulin J-expressing plasma cells, which also accrue concurrently across diverse organs. Finally, we show how expression shifts in distinct tissues are highly correlated with corresponding protein levels in plasma, thus potentially contributing to aging of the systemic circulation. Together, these data demonstrate a similar yet asynchronous inter- and intra-organ progression of aging, thereby providing a foundation to track systemic sources of declining health at old age.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/chromatinization_of_escherichia_coli_with_archaeal_histones_61507230/">
          <h4 class="font-weight-bold">Chromatinization of Escherichia coli with archaeal histones, bioRxiv, 2019-06-04</h4>
          <p>ABSTRACTNucleosomes restrict DNA accessibility throughout eukaryotic genomes, with repercussions for replication, transcription, and other DNA-templated processes. How this globally restrictive organization emerged from a presumably more open ancestral state remains poorly understood. Here, to better understand the challenges associated with establishing globally restrictive chromatin, we express histones in a naïve bacterial system that has not evolved to deal with nucleosomal structures Escherichia coli. We find that histone proteins from the archaeon Methanothermus fervidus assemble on the E. coli chromosome in vivo and protect DNA from micrococcal nuclease digestion, allowing us to map binding footprints genome-wide. We provide evidence that nucleosome occupancy along the E. coli genome tracks intrinsic sequence preferences but is disturbed by ongoing transcription and replication. Notably, we show that higher nucleosome occupancy at promoters and across gene bodies is associated with lower transcript levels, consistent with local repressive effects. Surprisingly, however, this sudden enforced chromatinization has only mild repercussions for growth, suggesting that histones can become established as ubiquitous chromatin proteins without interfering critically with key DNA-templated processes. Our results have implications for the evolvability of transcriptional ground states and highlight chromatinization by archaeal histones as a potential avenue for controlling genome accessibility in synthetic prokaryotic systems.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/on_the_emergence_of_candida_auris_climate_change_azoles_swamps_and_birds_61471872/">
          <h4 class="font-weight-bold">On the emergence of Candida auris climate change, azoles, swamps and birds, bioRxiv, 2019-06-04</h4>
          <p>The most enigmatic aspect of the rise of Candida auris as a human pathogen is that it emerged simultaneously in three continents with each clade being genetically distinct. Although new pathogenic fungal species are described regularly, these are mostly species associated with single cases in individuals who are immunosuppressed. In this study, we used phylogenetic analysis to compare C. auris with temperature susceptibility of close relatives and use these results to argue that it may be the first example of a new fungal disease emerging from climate change with the caveat that many other factors could have contributed.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/2019/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/2019/page/9/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/9/" class="page-link">9</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/2019/page/10/" class="page-link">10</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/11/" class="page-link">11</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/12/" class="page-link">12</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/13/" class="page-link">13</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/14/" class="page-link">14</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/2019/page/11/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/2019/page/87/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
