<!DOCTYPE html>
<html><head>
  
  <title>Ecology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/wild_pollinator_activities_negatively_related_to_honey_bee_colony_densities_in_urban_context_61913412/">
          <h4 class="font-weight-bold">Wild pollinator activities negatively related to honey bee colony densities in urban context, bioRxiv, 2019-06-11</h4>
          <p>AbstractAs pollinator decline is increasingly reported in natural and agricultural environments, cities are perceived as shelters for pollinators because of low pesticide exposure and high floral diversity throughout the year. This has led to the development of environmental policies supporting pollinators in urban areas. However, policies are often restricted to the promotion of honey bee colony installations, which resulted in a strong increase in apiary numbers in cities. Recently, competition for floral resources between wild pollinators and honey bees has been highlighted in semi-natural contexts, but whether urban beekeeping could impact wild pollinators remains unknown. Here, we show that in the city of Paris (France), wild pollinator visitation rates is negatively correlated to honey bee colony densities present in the surrounding (500m – slope = −0.614; p = 0.001 – and 1000m – slope = −0.489; p = 0.005). More particularly, large solitary bees and beetles were significantly affected at 500m (respectively slope = −0.425, p = 0.007 and slope = - 0.671, p = 0.002) and bumblebees were significantly affected at 1000m (slope = - 0.451, p = 0.012). Further, lower interaction evenness in plant-pollinator networks was observed with honey bee colony densities within 1000 meter buffers (slope = −0.487, p = 0.008). Finally, honey bees tended to focus their foraging activity on managed rather than spontaneous plant species (student t-test, p = 0.001) whereas wild pollinators equally visited managed and spontaneous species. We advocate responsible practices mitigating the introduction of high density of hives in urban environments. Future studies are needed to deepen our knowledge about the potential negative interactions between wild and domesticated pollinators.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_comparison_of_edna_to_camera_trapping_for_assessment_of_terrestrial_mammal_diversity_60254828/">
          <h4 class="font-weight-bold">A comparison of eDNA to camera trapping for assessment of terrestrial mammal diversity, bioRxiv, 2019-05-11</h4>
          <p>AbstractEnvironmental DNA (eDNA) is one of the most promising approaches to meet the demand for the fast and frequent monitoring of ecosystems needed to tackle the current decline in biodiversity. However, before eDNA can establish itself as a robust alternative for mammal monitoring, comparison with existing approaches is necessary, yet has not been done. Moreover, much is unknown regarding the nature, spread and persistence of DNA shed by animals into terrestrial environments, or the optimal experimental design for understanding these potential biases.To address some of these challenges, we compared the detection of terrestrial mammals using eDNA analysis of soil samples against confirmed species observations from a long-term (∼9-yr) camera trapping study. At the same time, we considered multiple experimental parameters, including two sampling designs, two DNA extraction kits and two metabarcodes of different sizes.All mammals consistently recorded with cameras were detected in eDNA. In addition, eDNA reported many small mammals not recorded by camera traps, but whose presence in the study area is otherwise documented. A long metabarcode (≈220bp) offering a high taxonomic resolution, achieved a similar efficiency as a shorter one (≈70bp) and a phosphate buffer-based extraction gave similar results as a total DNA extraction method for a fraction of the price. Our results support that eDNA-based monitoring should become a valuable part of terrestrial mammal surveys. Yet, the lack of coverage of mammal mitochondrial genomes in public databases must be addressed before eDNA can be used to its full potential.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/fishing_for_mammals_landscape-level_monitoring_of_terrestrial_and_semi-aquatic_communities_using_edna_from_lotic_ecosystems_60205935/">
          <h4 class="font-weight-bold">Fishing for mammals landscape-level monitoring of terrestrial and semi-aquatic communities using eDNA from lotic ecosystems, bioRxiv, 2019-05-10</h4>
          <p>Abstract&lt;jatslist list-type=order&gt;&lt;jatslist-item&gt;Environmental DNA (eDNA) metabarcoding has revolutionised biomonitoring in both marine and freshwater ecosystems. However, for semi-aquatic and terrestrial animals, the application of this technique remains relatively untested.&lt;jatslist-item&gt;&lt;jatslist-item&gt;We first assess the efficiency of eDNA metabarcoding in detecting semi-aquatic and terrestrial mammals in natural lotic ecosystems in the UK by comparing sequence data recovered from water and sediment samples to the mammalian communities expected from historical data. Secondly, we evaluate the detection efficiency of eDNA samples compared to multiple conventional non-invasive survey methods (latrine surveys and camera trapping) using occupancy modelling.&lt;jatslist-item&gt;&lt;jatslist-item&gt;eDNA metabarcoding detected a large proportion of the expected mammalian community within each area. Common species in the areas were detected at the majority of sites. Several key species of conservation concern in the UK were detected by eDNA in areas where authenticated records do not currently exist, but potential false positives were also identified for several non-native species.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Water-based eDNA samples provided comparable results to conventional survey methods in per unit of survey effort for three species (water vole, field vole, and red deer) using occupancy models. The comparison between survey ‘effort’ to reach a detection probability of ≥0.95 revealed that 3-6 water replicates would be equivalent to 3-5 latrine surveys and 5-30 weeks of single camera deployment, depending on the species.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Synthesis and Applications. eDNA metabarcoding represents an extremely promising tool for monitoring mammals, allowing for the detection of multiple species simultaneously, and provides comparable results to widely-used conventional survey methods. eDNA from freshwater systems delivers a ‘terrestrial dividend’ by detecting both semi-aquatic and terrestrial mammalian communities, and provides a basis for future monitoring at a landscape level over larger spatial and temporal scales (i.e. long-term monitoring at national levels).&lt;jatslist-item&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/shared_morphological_consequences_of_global_warming_in_north_american_migratory_birds_59211896/">
          <h4 class="font-weight-bold">Shared morphological consequences of global warming in North American migratory birds, bioRxiv, 2019-04-19</h4>
          <p>ABSTRACTIncreasing temperatures associated with climate change are predicted to cause reductions in body size, a key determinant of animal physiology and ecology. Using a four-decade specimen series of 70,716 individuals of 52 North American migratory bird species, we demonstrate that increasing annual summer temperature over the 40-year period drove consistent reductions in body size across these diverse taxa. Concurrently, wing length – which impacts nearly all aspects of avian ecology and behavior – has consistently increased across taxa. Our findings suggest that warming-induced body size reduction is a general response to climate change, and reveal a similarly consistent shift in an ecologically-important dimension of body shape. We hypothesize that increasing wing length represents a compensatory adaptation to maintain migration as reductions in body size have increased the metabolic cost of flight. An improved understanding of warming-induced morphological changes, and their limits, are important for predicting biotic responses to global change.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/best_practices_for_making_reliable_inferences_from_citizen_science_data_case_study_using_ebird_to_estimate_species_distributions_56909054/">
          <h4 class="font-weight-bold">Best practices for making reliable inferences from citizen science data case study using eBird to estimate species distributions, bioRxiv, 2019-03-12</h4>
          <p>AbstractCitizen science data are valuable for addressing a wide range of ecological research questions, and there has been a rapid increase in the scope and volume of data available. However, data from large-scale citizen science projects typically present a number of challenges that can inhibit robust ecological inferences. These challenges include species bias, spatial bias, variation in effort, and variation in observer skill.To demonstrate key challenges in analysing citizen science data, we use the example of estimating species distributions with data from eBird, a large semi-structured citizen science project. We estimate three widely applied metrics for describing species distributions encounter rate, occupancy probability, and relative abundance. For each method, we outline approaches for data processing and modelling that are suitable for using citizen science data for estimating species distributions.Model performance improved when data processing and analytical methods addressed the challenges arising from citizen science data. The largest gains in model performance were achieved with two key processes 1) the use of complete checklists rather than presence-only data, and 2) the use of covariates describing variation in effort and detectability for each checklist. Including these covariates accounted for heterogeneity in detectability and reporting, and resulted in substantial differences in predicted distributions. The data processing and analytical steps we outlined led to improved model performance across a range of sample sizes.When using citizen science data it is imperative to carefully consider the appropriate data processing and analytical procedures required to address the bias and variation. Here, we describe the consequences and utility of applying our suggested approach to semi-structured citizen science data to estimate species distributions. The methods we have outlined are also likely to improve other forms of inference and will enable researchers to conduct robust analyses and harness the vast ecological knowledge that exists within citizen science data.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/semi-quantitative_characterisation_of_mixed_pollen_samples_using_minion_sequencing_and_reverse_metagenomics_revmet_55573826/">
          <h4 class="font-weight-bold">Semi-quantitative characterisation of mixed pollen samples using MinION sequencing and Reverse Metagenomics (RevMet), bioRxiv, 2019-02-16</h4>
          <p>The ability to identify and quantify the constituent plant species that make up a mixed-species sample of pollen has important applications in ecology, conservation, and agriculture. Recently, metabarcoding protocols have been developed for pollen that can identify constituent plant species, but there are strong reasons to doubt that metabarcoding can accurately quantify their relative abundances. A PCR-free, shotgun metagenomics approach has greater potential for accurately quantifying species relative abundances, but applying metagenomics to eukaryotes is challenging due to low numbers of reference genomes. We have developed a pipeline, RevMet (Reverse Metagenomics), that allows reliable and semi-quantitative characterization of the species composition of mixed-species eukaryote samples, such as bee-collected pollen, without requiring reference genomes. Instead, reference species are represented only by &#39;genome skims&#39; low-cost, low-coverage, shortread sequence datasets. The skims are mapped to individual long reads sequenced from mixed-species samples using the MinION, a portable nanopore sequencing device, and each long read is uniquely assigned to a plant species. We genome-skimmed 49 wild UK plant species, validated our pipeline with mock DNA mixtures of known composition, and then applied RevMet to pollen loads collected from wild bees. We demonstrate that RevMet can identify plant species present in mixed-species samples at proportions of DNA &amp;gt;1%, with few false positives and false negatives, and reliably differentiate species represented by high versus low amounts of DNA in a sample. The RevMet pipeline could readily be adapted to generate semi-quantitative datasets for a wide range of mixed eukaryote samples, which could include characterising diets, quantifying allergenic pollen from air samples, quantifying soil fauna, and identifying the compositions of algal and diatom communities. Our per-sample costs were GBP 90 per genome skim and GBP 60 per pollen sample, and new versions of sequencers available now will further reduce these costs.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        

        
             
            <li class="page-item active"><a href="/audiences/tags/ecology/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/ecology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/ecology/page/3/" class="page-link">3</a></li>
            
        

        
        <li class="page-item"><a href="/audiences/tags/ecology/page/2/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/ecology/page/3/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
