<!DOCTYPE html>
<html><head>
  
  <title>Ecology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/glacier_ice_archives_fifteen-thousand-year-old_viruses_73713223/">
          <h4 class="font-weight-bold">Glacier ice archives fifteen-thousand-year-old viruses, bioRxiv, 2020-01-07</h4>
          <p>AbstractWhile glacier ice cores provide climate information over tens to hundreds of thousands of years, study of microbes is challenged by ultra-low-biomass conditions, and virtually nothing is known about co-occurring viruses. Here we establish ultra-clean microbial and viral sampling procedures and apply them to two ice cores from the Guliya ice cap (northwestern Tibetan Plateau, China) to study these archived communities. This method reduced intentionally contaminating bacterial, viral, and free DNA to background levels in artificial-ice-core control experiments, and was then applied to two authentic ice cores to profile their microbes and viruses. The microbes differed significantly across the two ice cores, presumably representing the very different climate conditions at the time of deposition that is similar to findings in other cores. Separately, viral particle enrichment and ultra-low-input quantitative viral metagenomic sequencing from ∼520 and ∼15,000 years old ice revealed 33 viral populations (i.e., species-level designations) that represented four known genera and likely 28 novel viral genera (assessed by gene-sharing networks). In silico host predictions linked 18 of the 33 viral populations to co-occurring abundant bacteria, including Methylobacterium, Sphingomonas, and Janthinobacterium, indicating that viruses infected several abundant microbial groups. Depth-specific viral communities were observed, presumably reflecting differences in the environmental conditions among the ice samples at the time of deposition. Together, these experiments establish a clean procedure for studying microbial and viral communities in low-biomass glacier ice and provide baseline information for glacier viruses, some of which appear to be associated with the dominant microbes in these ecosystems.ImportanceThis study establishes ultra-clean microbial and viral sampling procedures for glacier ice, which complements prior in silico decontamination methods and expands, for the first time, the clean procedures to viruses. Application of these methods to glacier ice confirmed prior common microbiological findings for a new ice core climate record, and provides a first window into viral genomes and their ecology from glacier ice across two time horizons, and emphasizes their likely impact on abundant microbial groups. Together these efforts provide clean sampling approaches and foundational datasets that should enable simultaneous access to an archived virosphere in glacier ice.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genomic_evidence_for_global_ocean_plankton_biogeography_shaped_by_large-scale_current_systems_72345163/">
          <h4 class="font-weight-bold">Genomic evidence for global ocean plankton biogeography shaped by large-scale current systems, bioRxiv, 2019-12-07</h4>
          <p>AbstractBiogeographical studies have traditionally focused on readily visible organisms, but recent technological advances are enabling analyses of the large-scale distribution of microscopic organisms, whose biogeographical patterns have long been debated1,2. The most prominent global biogeography of marine plankton was derived by Longhurst3 based on parameters principally associated with photosynthetic plankton. Localized studies of selected plankton taxa or specific organismal sizes1,4–7 have mapped community structure and begun to assess the roles of environment and ocean current transport in shaping these patterns2,8. Here we assess global plankton biogeography and its relation to the biological, chemical and physical context of the ocean (the ‘seascape’) by analyzing 24 terabases of metagenomic sequence data and 739 million metabarcodes from the Tara Oceans expedition in light of environmental data and simulated ocean current transport. In addition to significant local heterogeneity, viral, prokaryotic and eukaryotic plankton communities all display near steady-state, large-scale, size-dependent biogeographical patterns. Correlation analyses between plankton transport time and metagenomic or environmental dissimilarity reveal the existence of basin-scale biological and environmental continua emerging within the main current systems. Across oceans, there is a measurable, continuous change within communities and environmental factors up to an average of 1.5 years of travel time. Modulation of plankton communities during transport varies with organismal size, such that the distribution of smaller plankton best matches Longhurst biogeochemical provinces, whereas larger plankton group into larger provinces. Together these findings provide an integrated framework to interpret plankton community organization in its physico-chemical context, paving the way to a better understanding of oceanic ecosystem functioning in a changing global environment.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/species_better_track_the_shifting_isotherms_in_the_oceans_than_on_lands_66445867/">
          <h4 class="font-weight-bold">Species better track the shifting isotherms in the oceans than on lands, bioRxiv, 2019-09-11</h4>
          <p>Despite mounting evidence of species redistribution as climate warms, our knowledge of the coupling between species range shifts and isotherm shifts is limited. Compiling a global geo-database of 30,534 range shifts from 12,415 taxa, we show that only marine taxa closely track the shifting isotherms. In the oceans, the velocity of isotherm shifts interacts synergistically with anthropogenic disturbances and baseline temperatures such that isotherm tracking by marine life happens either in warm and undisturbed waters (e.g. Central Pacific Basin) or in colder waters where human activities are more pronounced (e.g. North Sea). On lands, increasing anthropogenic activities and temperatures negatively impact the capacity of terrestrial taxa to track isotherm shifts in latitude and elevation, respectively. This suggests that terrestrial taxa are lagging behind the shifting isotherms, most likely due to their wider thermal safety margin, more constrained physical environment for dispersal and higher availability of thermal microrefugia at shorter distances.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/assessing_the_potential_of_environmental_dna_metabarcoding_for_monitoring_neotropical_mammals_a_case_study_in_the_amazon_and_atlantic_forest_brazil_65710239/">
          <h4 class="font-weight-bold">Assessing the potential of environmental DNA metabarcoding for monitoring Neotropical mammals a case study in the Amazon and Atlantic Forest, Brazil, bioRxiv, 2019-08-31</h4>
          <p>AbstractThe application of environmental DNA (eDNA) metabarcoding as a biomonitoring tool has greatly increased in the last decade. However, most studies have focused on aquatic macro-organisms in temperate areas (e.g., fishes). We apply eDNA metabarcoding to detect the mammalian community in two high-biodiversity regions of Brazil, the Amazon and Atlantic Forest. We identified critically endangered and endangered mammalian species in the Atlantic Forest and Amazon respectively and found congruence with species identified via camera trapping in the Atlantic Forest. In light of our results, we highlight the potential and challenges of eDNA monitoring for mammals in these high biodiverse areas.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/land_use_impacts_poison_frog_chemical_defenses_through_changes_in_leaf_litter_ant_communities_65690558/">
          <h4 class="font-weight-bold">Land use impacts poison frog chemical defenses through changes in leaf litter ant communities, bioRxiv, 2019-08-31</h4>
          <p>AbstractMuch of the world’s biodiversity is held within tropical rainforests, which are increasingly fragmented by agricultural practices. In these threatened landscapes, there are many organisms that acquire chemical defenses from their diet and are therefore intimately connected with their local food webs. Poison frogs (Family Dendrobatidae) are one such example, as they acquire alkaloid-based chemical defenses from their diet of leaf litter ants and mites. It is currently unknown how habitat fragmentation impacts chemical defense across trophic levels, from arthropods to frogs. Thus, we examined the chemical defenses and diets of the Diablito poison frog (Oophaga sylvatica), and the diversity of their leaf litter ant communities in secondary forest and reclaimed cattle pasture. We found that forest and pasture frogs differed in diet and alkaloid profiles, where forest frogs contained more of specific alkaloids and ate more ants. We also found that the leaf litter of forest and pasture frog habitats differed in ant community structure. Finally, ant species composition of frog diets resembled the surrounding leaf litter, but diets were less variable. This suggests that frogs tend to consume particular ant species within each habitat. To better understand how ants contribute to the alkaloid chemical profiles of frogs, we chemically profiled several ant species and found some alkaloids to be common across many ant species while others are restricted to a few species. Our experiments are the first to link anthropogenic land use changes to dendrobatid poison frog chemical defenses through variation in leaf litter communities, which has implications for conservation management of these threatened amphibians.ResumenLos bosques tropicales, que mantienen la mayor parte de la biodiversidad del planeta, están cada vez más fragmentados por diferentes prácticas agrícolas. En estos paisajes amenazados por la actividad humana hay organismos que acumulan defensas químicas a partir de su dieta. Las ranas venenosas (Familia Dendrobatidae) son un ejemplo de esto, y adquieren de su dieta, formada principalmente de hormigas de la hojarasca y de ácaros, diferentes alcaloides que pueden ser usados como defensas químicas. Las ranas venenosas están por lo tanto íntimamente conectadas a las redes tróficas locales. Actualmente es desconocido como la fragmentación del hábitat modifica la defensa por químicos a través de cambios en las cadena tróficas. Por lo tanto, examinamos las defensas químicas y dietas de la rana diablito (Oophaga sylvatica), y las comunidades de hormigas de la hojarasca, en un bosque secundario y en un pastizal cercano. Encontramos que las dietas y los perfiles de alcaloides en la piel de las ranas en bosque y en pastizal eran diferentes, con las ranas de bosque comiendo más hormigas y acumulando más alcaloides específicos. También encontramos que la hojarasca del bosque y del pastizal mantenían comunidades de hormigas con estructuras distintas. Finalmente, la composición de las hormigas en las dietas se parecían que la composición de hormigas en sus alrededores, pero dietas menos variables. Ello sugiere que las ranas tienden a consumir sólo ciertas especies de hormigas en cada hábitat. Para entender mejor cómo las hormigas contribuyen a los perfiles de alcaloides de las ranas, obtuvimos perfiles de los alcaloides presentes en algunas especies de hormigas y encontramos que algunos alcaloides son comunes a muchas especies de hormigas, y otros alcaloides están restringidos a pocas especies. Nuestros experimentos son los primeros en vincular los cambios antropogénicos en el uso de suelo con cambios en las defensas químicas de las ranas venenosas a través de cambios en las comunidades de la hojarasca, lo que tiene implicaciones para la conservación de estos anfibios altamente amenazados.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genetic_tool_development_in_marine_protists_emerging_model_organisms_for_experimental_cell_biology_64435858/">
          <h4 class="font-weight-bold">Genetic tool development in marine protists Emerging model organisms for experimental cell biology, bioRxiv, 2019-08-02</h4>
          <p>ABSTRACTMarine microbial eukaryotes underpin the largest food web on the planet and influence global biogeochemical cycles that maintain habitability. They are also remarkably diverse and provide insights into evolution, including the origins of complex life forms, as revealed through genome analyses. However, their genetic tractability has been limited to a few species that do not represent the broader diversity of eukaryotic life or some of the most environmentally relevant taxa. Here, we report on genetic systems developed as a community resource for experimental cell biology of aquatic protists from across the eukaryotic tree and primarily from marine habitats. We present evidence for foreign DNA delivery and expression in 14 species never before transformed, report on the advancement of genetic systems in 7 species, review of an already published transformation protocol in 1 species and discuss why the transformation of 17 additional species has not been achieved yet. For all protists studied in this community effort, we outline our methods, constructs, and genome-editing approaches in the context of published systems. The reported breakthroughs on genetic manipulation position the community to dissect cellular mechanisms from a breadth of protists, which will collectively provide insights into ancestral eukaryotic lifeforms, protein diversification and evolution of cellular pathways.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        

        
             
            <li class="page-item active"><a href="/audiences/tags/ecology/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/ecology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/ecology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/ecology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/ecology/page/5/" class="page-link">5</a></li>
            
        

        
        <li class="page-item"><a href="/audiences/tags/ecology/page/2/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/ecology/page/5/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
