<!DOCTYPE html>
<html><head>
  
  <title>Reports | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/single_cell_rna-seq_reveals_ectopic_and_aberrant_lung_resident_cell_populations_in_idiopathic_pulmonary_fibrosis_66040758/">
          <h4 class="font-weight-bold">Single Cell RNA-seq reveals ectopic and aberrant lung resident cell populations in Idiopathic Pulmonary Fibrosis, bioRxiv, 2019-09-06</h4>
          <p>AbstractWe provide a single cell atlas of Idiopathic Pulmonary Fibrosis (IPF), a fatal interstitial lung disease, focusing on resident lung cell populations. By profiling 312,928 cells from 32 IPF, 29 healthy control and 18 chronic obstructive pulmonary disease (COPD) lungs, we demonstrate that IPF is characterized by changes in discrete subpopulations of cells in the three major parenchymal compartments the epithelium, endothelium and stroma. Among epithelial cells, we identify a novel population of IPF enriched aberrant basaloid cells that co-express basal epithelial markers, mesenchymal markers, senescence markers, developmental transcription factors and are located at the edge of myofibroblast foci in the IPF lung. Among vascular endothelial cells in the in IPF lung parenchyma we identify an expanded cell population transcriptomically identical to vascular endothelial cells normally restricted to the bronchial circulation. We confirm the presence of both populations by immunohistochemistry and independent datasets. Among stromal cells we identify fibroblasts and myofibroblasts in both control and IPF lungs and leverage manifold-based algorithms diffusion maps and diffusion pseudotime to infer the origins of the activated IPF myofibroblast. Our work provides a comprehensive catalogue of the aberrant cellular transcriptional programs in IPF, demonstrates a new framework for analyzing complex disease with scRNAseq, and provides the largest lung disease single-cell atlas to date.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/charting_the_native_architecture_of_thylakoid_membranes_with_single-molecule_precision_65988221/">
          <h4 class="font-weight-bold">Charting the native architecture of thylakoid membranes with single-molecule precision, bioRxiv, 2019-09-05</h4>
          <p>Thylakoid membranes scaffold an assortment of large protein complexes that work together to harness the energy of light to produce oxygen, NADPH, and ATP. It has been a longstanding challenge to visualize how the intricate thylakoid network organizes these protein complexes to finely tune the photosynthetic reactions. Using cryo-electron tomography to analyze membrane surface topology, we have mapped the native molecular landscape of thylakoid membranes within green algae cells. Our tomograms provide insights into the molecular forces that drive thylakoid stacking and reveal that photosystems I and II are strictly segregated at the borders between appressed and non-appressed membrane domains. This new approach to charting thylakoid topology lays the foundation for dissecting photosynthetic regulation at the level of single protein complexes within the cell.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genewalk_identifies_relevant_gene_functions_for_a_biological_context_using_network_representation_learning_65971626/">
          <h4 class="font-weight-bold">GeneWalk identifies relevant gene functions for a biological context using network representation learning, bioRxiv, 2019-09-05</h4>
          <p>AbstractThe primary bottleneck in high-throughput genomics experiments is identifying the most important genes and their relevant functions from a list of gene hits. Existing methods such as Gene Ontology (GO) enrichment analysis provide insight at the gene set level. For individual genes, GO annotations are static and biological context can only be added by manual literature searches. Here, we introduce GeneWalk (&lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpgithub.comchurchmanlabgenewalk&gt;github.comchurchmanlabgenewalk&lt;jatsext-link&gt;), a method that identifies individual genes and their relevant functions under a particular experimental condition. After automatic assembly of an experiment-specific gene regulatory network, GeneWalk quantifies the similarity between vector representations of each gene and its GO annotations through representation learning, yielding annotation significance scores that reflect their functional relevance for the experimental context. We demonstrate the use of GeneWalk analysis of RNA-seq and nascent transcriptome (NET-seq) data from human cells and mouse brains, validating the methodology. By performing gene- and condition-specific functional analysis that converts a list of genes into data-driven hypotheses, GeneWalk accelerates the interpretation of high-throughput genetics experiments.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/profiling_cellular_diversity_in_sponges_informs_animal_cell_type_and_nervous_system_evolution_65970795/">
          <h4 class="font-weight-bold">Profiling cellular diversity in sponges informs animal cell type and nervous system evolution, bioRxiv, 2019-09-05</h4>
          <p>AbstractThe evolutionary origin of metazoan cell types such as neurons, muscles, digestive, and immune cells, remains unsolved. Using whole-body single-cell RNA sequencing in a sponge, an animal without nervous system and musculature, we identify 18 distinct cell types comprising four major families. This includes nitric-oxide sensitive contractile cells, digestive cells active in macropinocytosis, and a family of amoeboid-neuroid cells involved in innate immunity. We uncover ‘presynaptic’ genes in an amoeboid-neuroid cell type, and ‘postsynaptic’ genes in digestive choanocytes, suggesting asymmetric and targeted communication. Corroborating this, long neurite-like extensions from neuroid cells directly contact and enwrap choanocyte microvillar collars. Our data indicate a link between neuroid and immune functions in sponges, and suggest that a primordial neuro-immune system cleared intruders and controlled ciliary beating for feeding.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/stimjim_open_source_hardware_for_precise_electrical_stimulation_65925120/">
          <h4 class="font-weight-bold">Stimjim open source hardware for precise electrical stimulation, bioRxiv, 2019-09-05</h4>
          <p>AbstractElectrical stimulation is a simple and powerful tool to perturb and evoke neuronal activity in order to understand the function of neurons and neural circuits. Despite this, devices that can provide precise current or voltage stimulation are expensive and closed-source. Here, we introduce Stimjim, a capable and inexpensive ($200 USD) open-source instrument for electrical stimulation that combines both function generation and electrical isolation. Stimjim provides microsecond temporal resolution with microampere or millivolt scale precision on two electrically isolated output channels. We demonstrate Stimjim’s utility both in vitro by precisely stimulating brain slices, and in vivo by training mice to perform intracranial self-stimulation (ICSS) for brain stimulation reward. During ICSS, Stimjim enables the experimenter to smoothly tune the strength of reward-seeking behavior by varying either the output frequency or amplitude. We envision Stimjim will enable new kinds of experiments due to its open-source and scalable nature.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_novel_weight_lifting_task_for_investigating_effort_and_persistence_in_rats_65896639/">
          <h4 class="font-weight-bold">A novel weight lifting task for investigating effort and persistence in rats, bioRxiv, 2019-09-04</h4>
          <p>AbstractHere we present a novel effort-based task for laboratory rats the weight lifting task (WLT). Studies of effort expenditure in rodents have typically involved climbing barriers within T-mazes or operant lever pressing paradigms. These task designs have been successful for neuropharmacological and neurophysiological investigations, but both tasks involve simple action patterns prone to automatization. Furthermore, high climbing barriers present risk of injury to animals andor tethered recording equipment. In the WLT, a rat is placed in a large rectangular arena and tasked with pulling a rope 30 cm to trigger food delivery at a nearby spout; weights can be added to the rope in 45 g increments to increase the intensity of effort. As compared to lever pressing and barrier jumping, 30 cm of rope pulling is a multi-step action sequence requiring sustained effort. The actions are carried out on the single plane of the arena floor, making it safer for the animal and more suitable for tethered equipment and video tracking. A microcontroller and associated sensors enable precise timestamping of specific behaviors to synchronize with electrophysiological recordings. The rope and reward spout are spatially segregated to allow for spatial discrimination of the effort zone and the reward zone. We validated the task across five cohorts of rats (total n=35) and report consistent behavioral metrics. The WLT is well-suited for neuropharmacological andor in vivo neurophysiological investigations surrounding effortful behaviors, particularly when wanting to probe different aspects of effort expenditure (intensity vs. duration).</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/animal-behavior-and-cognition">animal-behavior-and-cognition</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/reports/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/reports/page/27/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/24/" class="page-link">24</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/25/" class="page-link">25</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/26/" class="page-link">26</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/27/" class="page-link">27</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/reports/page/28/" class="page-link">28</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/29/" class="page-link">29</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/30/" class="page-link">30</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/31/" class="page-link">31</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/32/" class="page-link">32</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/reports/page/29/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/reports/page/320/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
