<!DOCTYPE html>
<html><head>
  
  <title>Reports | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/assembly-free_single-molecule_nanopore_sequencing_recovers_complete_virus_genomes_from_natural_microbial_communities_59546075/">
          <h4 class="font-weight-bold">Assembly-free single-molecule nanopore sequencing recovers complete virus genomes from natural microbial communities, bioRxiv, 2019-04-26</h4>
          <p>AbstractViruses are the most abundant biological entities on Earth, and play key roles in host ecology, evolution, and horizontal gene transfer. Despite recent progress in viral metagenomics, the inherent genetic complexity of virus populations still poses technical difficulties for recovering complete virus genomes from natural assemblages. To address these challenges, we developed an assembly-free, single-molecule nanopore sequencing approach enabling direct recovery of high-quality viral genome sequences from environmental samples. Our method yielded over a thousand high quality, full-length draft virus genome sequences that could not be fully recovered using short read assembly approaches applied to the same samples. Additionally, novel DNA sequences were discovered whose repeat structures, gene contents and concatemer lengths suggested that they represent phage-inducible chromosomal islands that were packaged as concatemers within phage particles. Our new approach provided novel insight into genome structures, population biology, and ecology of naturally occurring viruses and viral parasites.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/ctcf-dependent_chromatin_boundaries_formed_by_asymmetric_nucleosome_arrays_with_decreased_linker_length_59529123/">
          <h4 class="font-weight-bold">CTCF-dependent chromatin boundaries formed by asymmetric nucleosome arrays with decreased linker length, bioRxiv, 2019-04-26</h4>
          <p>AbstractThe CCCTC-binding factor (CTCF) organises the genome in 3D through DNA loops and in 1D by setting boundaries isolating different chromatin states, but these processes are not well understood. Here we focus on the relationship between CTCF binding and the decrease of the Nucleosome Repeat Length (NRL) for ∼20 adjacent nucleosomes, affecting up to 10% of the mouse genome. We found that the chromatin boundary near CTCF is created by the nucleosome-depleted region (NDR) asymmetrically located &amp;gt;40 nucleotides 5’-upstream from the centre of CTCF motif. The strength of CTCF binding to DNA is correlated with the decrease of NRL near CTCF and anti-correlated with the level of asymmetry of the nucleosome array. Individual chromatin remodellers have different contributions, with Snf2h having the strongest effect on the NRL decrease near CTCF and Chd4 playing a major role in the symmetry breaking. Upon differentiation of embryonic stem cells to neural progenitor cells and embryonic fibroblasts, a subset of common CTCF sites preserved in all three cell types maintains a relatively small local NRL despite genome-wide NRL increase. The sites which lost CTCF upon differentiation are characterised by nucleosome rearrangement 3’-downstream, but the boundary defined by the NDR 5’-upstream of CTCF motif remains.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/rapid_low-cost_detection_of_water_contaminants_using_regulated_in_vitro_transcription_59548555/">
          <h4 class="font-weight-bold">Rapid, Low-Cost Detection of Water Contaminants Using Regulated In Vitro Transcription, bioRxiv, 2019-04-26</h4>
          <p>ABSTRACTSynthetic biology has enabled the development of powerful nucleic acid diagnostic technologies for detecting pathogens and human health biomarkers. Here we expand the reach of synthetic biology-enabled diagnostics by developing a cell-free biosensing platform that uses RNA output sensors activated by ligand induction (ROSALIND) to detect harmful contaminants in aqueous samples. ROSALIND consists of three programmable components highly-processive RNA polymerases, allosteric transcription factors, and synthetic DNA transcription templates. Together, these components allosterically regulate the in vitro transcription of a fluorescence-activating RNA aptamer in the absence of a target compound, transcription is blocked, while in its presence a fluorescent signal is produced. We demonstrate that ROSALIND can be configured to detect a range of water contaminants, including antibiotics, toxic small molecules, and metals. Our cell-free biosensing platform, which can be freeze-dried for field deployment, creates a new capability for point-of-use monitoring of molecular species to address growing global crises in water quality and human health.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/synthetic-biology">synthetic-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/gephebase_a_database_of_genotype-phenotype_relationships_for_natural_and_domesticated_variation_in_eukaryotes_59510592/">
          <h4 class="font-weight-bold">Gephebase, a Database of Genotype-Phenotype Relationships for natural and domesticated variation in Eukaryotes, bioRxiv, 2019-04-25</h4>
          <p>AbstractGephebase is a manually-curated database compiling our accumulated knowledge of the genes and mutations that underlie natural, domesticated and experimental phenotypic variation in all Eukaryotes — mostly animals, plants and yeasts. Gephebase aims to compile studies where the genotype-phenotype association (based on linkage mapping, association mapping or a candidate gene approach) is relatively well supported or understood. Human disease and aberrant mutant phenotypes in laboratory model organisms are not included in Gephebase and can be found in other databases (eg. OMIM, OMIA, Monarch Initiative). Gephebase contains more than 1700 entries. Each entry corresponds to an allelic difference at a given gene and its associated phenotypic change(s) between two species or between two individuals of the same species, and is enriched with molecular details, taxonomic information, and bibliographic information. Users can easily browse entries for their topic of interest and perform searches at various levels, whether phenotypic, genetic, taxonomic or bibliographic (eg. transposable elements, cis-regulatory mutations, snakes, carotenoid content, an author name). Data can be searched using keywords and boolean operators and is exportable in spreadsheet format. This database allows to perform meta-analysis to extract general information and global trends about evolution, genetics, and the field of evolutionary genetics itself. Gephebase should also help breeders, conservationists and others to identify the most promising target genes for traits of interest, with potential applications such as crop improvement, parasite and pest control, bioconservation, and genetic diagnostic. It is freely available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpwww.gephebase.org&gt;www.gephebase.org&lt;jatsext-link&gt;.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/neighbor_predation_linked_to_natural_competence_fosters_the_transfer_of_large_genomic_regions_in_vibrio_cholerae_59471825/">
          <h4 class="font-weight-bold">Neighbor predation linked to natural competence fosters the transfer of large genomic regions in Vibrio cholerae, bioRxiv, 2019-04-25</h4>
          <p>AbstractNatural competence for transformation is a primary mode of horizontal gene transfer (HGT). Competent bacteria are able to absorb free DNA from their surroundings and exchange this DNA against pieces of their own genome when sufficiently homologous. And while it is known that transformation contributes to evolution and pathogen emergence in bacteria, there are still questions regarding the general prevalence of non-degraded DNA with sufficient coding capacity. In this context, we previously showed that the naturally competent bacterium Vibrio cholerae uses its type VI secretion system (T6SS) to actively acquire DNA from non-kin neighbors under chitin-colonizing conditions. We therefore sought to further explore the role of the T6SS in acquiring DNA, the condition of the DNA released through T6SS-mediated killing versus passive cell lysis, and the extent of the transfers that occur due to these conditions. To do this, we herein measured the frequency and the extent of genetic exchanges in bacterial co-cultures on competence-inducing chitin under various DNA-acquisition conditions. We show that competent V. cholerae strains acquire DNA fragments with an average and maximum length exceeding 50 kbp and 150 kbp, respectively, and that the T6SS is of prime importance for such HGT events. Collectively, our data support the notion that the environmental lifestyle of V. cholerae fosters HGT and that the coding capacity of the exchanged genetic material is sufficient to significantly accelerate bacterial evolution.Significance StatementDNA shuffled from one organism to another in an inheritable manner is a common feature of prokaryotes. It is a significant mechanism by which bacteria acquire new phenotypes, for example by first absorbing foreign DNA and then recombining it into their genome. In this study, we show the remarkable extent of the exchanged genetic material, frequently exceeding 150 genes in a seemingly single transfer event, in Vibrio cholerae. We also show that to best preserve its length and quality, bacteria mainly acquire this DNA by killing adjacent, healthy neighbors then immediately absorbing the released DNA before it can be degraded. These new insights into this prey-killing DNA acquisition process shed light on how bacterial species evolve in the wild.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/rna_transcribed_from_heterochromatic_simple-tandem_repeats_are_required_for_male_fertility_and_histone-protamine_exchange_in_drosophila_melanogaster_59473338/">
          <h4 class="font-weight-bold">RNA transcribed from heterochromatic simple-tandem repeats are required for male fertility and histone-protamine exchange in Drosophila melanogaster, bioRxiv, 2019-04-25</h4>
          <p>AbstractLong arrays of simple, tandemly repeated DNA sequences (known as satellites) are enriched in centromeres1 and pericentromeric regions2, and contribute to chromosome segregation and other heterochromatin functions3,4. Surprisingly, satellite DNAs are expressed in many multicellular eukaryotes, and their aberrant transcription may contribute to carcinogenesis and cellular toxicity5-7. Satellite transcription andor RNAs may also promote centromere and heterochromatin activities 8-12. However, we lack direct evidence that satellite DNA transcripts are required for normal cell or organismal functions. Here, we show that satellite RNAs derived from AAGAG tandem repeats are transcribed in many cell types throughout Drosophila melanogaster development, enriched in neuronal tissues and testes, localized within heterochromatic regions, and important for viability. Strikingly, we find that AAGAG transcripts are necessary for male fertility and are specifically required for normal histone-protamine exchange and sperm chromatin organization. Since AAGAG RNA-dependent events happen late in spermatogenesis when the transcripts are not detected, we speculate that AAGAG RNA functions in primary spermatocytes to ‘prime’ post-meiosis steps in sperm maturation. In addition to demonstrating specific essential functions for AAGAG RNAs, comparisons between closely related Drosophila species suggest that satellite repeats and their transcription evolve quickly to generate new functions.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/reports/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/reports/page/27/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/reports/page/24/" class="page-link">24</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/25/" class="page-link">25</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/26/" class="page-link">26</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/27/" class="page-link">27</a></li>
            
        
             
            <li class="page-item active"><a href="/reports/page/28/" class="page-link">28</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/29/" class="page-link">29</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/30/" class="page-link">30</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/31/" class="page-link">31</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/32/" class="page-link">32</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/reports/page/29/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/reports/page/172/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
