<!DOCTYPE html>
<html><head>
  
  <title>Bioengineering | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/reading_canonical_and_modified_nucleotides_in_16s_ribosomal_rna_using_nanopore_direct_rna_sequencing_19696288/">
          <h4 class="font-weight-bold">Reading canonical and modified nucleotides in 16S ribosomal RNA using nanopore direct RNA sequencing, bioRxiv, 2017-04-30</h4>
          <p>The ribosome small subunit is expressed in all living cells. It performs numerous essential functions during translation, including formation of the initiation complex and proofreading of base-pairs between mRNA codons and tRNA anticodons. The core constituent of the small ribosomal subunit is a ∼1.5 kb RNA strand in prokaryotes (16S rRNA) and a homologous ∼1.8 kb RNA strand in eukaryotes (18S rRNA). Traditional sequencing-by-synthesis (SBS) of rRNA genes or rRNA cDNA copies has achieved wide use as a ‘molecular chronometer’ for phylogenetic studies 1, and as a tool for identifying infectious organisms in the clinic 2. However, epigenetic modifications on rRNA are erased by SBS methods. Here we describe direct MinION nanopore sequencing of individual, full-length 16S rRNA absent reverse transcription or amplification. As little as 5 picograms (∼10 attomole) of E. coli 16S rRNA was detected in 4.5 micrograms of total human RNA. Nanopore ionic current traces that deviated from canonical patterns revealed conserved 16S rRNA base modifications, and a 7-methylguanosine modification that confers aminoglycoside resistance to some pathological E. coli strains. This direct RNA sequencing technology has promise for rapid identification of microbes in the environment and in patient samples.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/bioengineering">bioengineering</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/scaling_up_dna_data_storage_and_random_access_retrieval_17090763/">
          <h4 class="font-weight-bold">Scaling up DNA data storage and random access retrieval, bioRxiv, 2017-03-08</h4>
          <p>Current storage technologies can no longer keep pace with exponentially growing amounts of data. 1 Synthetic DNA offers an attractive alternative due to its potential information density of ~ 1018 Bmm3, 107 times denser than magnetic tape, and potential durability of thousands of years.2 Recent advances in DNA data storage have highlighted technical challenges, in particular, coding and random access, but have stored only modest amounts of data in synthetic DNA. 3,4,5 This paper demonstrates an end-to-end approach toward the viability of DNA data storage with large-scale random access. We encoded and stored 35 distinct files, totaling 200MB of data, in more than 13 million DNA oligonucleotides (about 2 billion nucleotides in total) and fully recovered the data with no bit errors, representing an advance of almost an order of magnitude compared to prior work. 6 Our data curation focused on technologically advanced data types and historical relevance, including the Universal Declaration of Human Rights in over 100 languages,7 a high-definition music video of the band OK Go,8 and a CropTrust database of the seeds stored in the Svalbard Global Seed Vault.9 We developed a random access methodology based on selective amplification, for which we designed and validated a large library of primers, and successfully retrieved arbitrarily chosen items from a subset of our pool containing 10.3 million DNA sequences. Moreover, we developed a novel coding scheme that dramatically reduces the physical redundancy (sequencing read coverage) required for error-free decoding to a median of 5x, while maintaining levels of logical redundancy comparable to the best prior codes. We further stress-tested our coding approach by successfully decoding a file using the more error-prone nanopore-based sequencing. We provide a detailed analysis of errors in the process of writing, storing, and reading data from synthetic DNA at a large scale, which helps characterize DNA as a storage medium and justify our coding approach. Thus, we have demonstrated a significant improvement in data volume, random access, and encodingdecoding schemes that contribute to a whole-system vision for DNA data storage.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/bioengineering">bioengineering</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/multiplexed_confocal_and_super-resolution_fluorescence_imaging_of_cytoskeletal_and_neuronal_synapse_proteins_16758800/">
          <h4 class="font-weight-bold">Multiplexed confocal and super-resolution fluorescence imaging of cytoskeletal and neuronal synapse proteins, bioRxiv, 2017-02-26</h4>
          <p>ABSTRACTNeuronal synapses contain dozens of protein species whose expression levels and localizations are key determinants of synaptic transmission and plasticity. The spectral properties of fluorophores used in conventional microscopy limit the number of measured proteins to four species within a given sample. The ability to perform high-throughput confocal or super-resolution imaging of many proteins simultaneously without limitation in target number imposed by this spectral limit would enable large-scale characterization of synaptic protein networks in situ. Here, we introduce PRISM Probe-based Imaging for Sequential Multiplexing, a method that sequentially utilizes either high affinity Locked Nucleic Acid (LNA) or low affinity DNA probes to enable diffraction-limited confocal and PAINT-based super-resolution imaging. High-affinity LNA probes offer high-throughput, confocal-based imaging compared with PAINT, which uses low affinity probes to realize localization-based super-resolution imaging. Simultaneous immunostaining of all targets is performed prior to imaging, followed by sequential LNADNA probe exchange that requires only minutes under mild wash conditions. We apply PRISM to quantify the co-expression levels and nanometer-scale organization of one dozen cytoskeletal and synaptic proteins within individual neuronal synapses. Our approach is scalable to dozens of target proteins and is compatible with high-content screening platforms commonly used to interrogate phenotypic changes associated with genetic and drug perturbations in a variety of cell types.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/bioengineering">bioengineering</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/paperfuge_an_ultra-low_cost_hand-powered_centrifuge_inspired_by_the_mechanics_of_a_whirligig_toy_10905666/">
          <h4 class="font-weight-bold">Paperfuge An ultra-low cost, hand-powered centrifuge inspired by the mechanics of a whirligig toy, bioRxiv, 2016-09-01</h4>
          <p>AbstractSample preparation, including separation of plasma from whole blood or isolation of parasites, is an unmet challenge in many point of care (POC) diagnostics and requires centrifugation as the first key step. From the context of global health applications, commercial centrifuges are expensive, bulky and electricity-powered, leading to a critical bottle-neck in the development of decentralized, electricity-free POC diagnostic devices. By uncovering the fundamental mechanics of an ancient whirligig toy (3300 B.C.E), we design an ultra-low cost (20 cents), light-weight (2 g), human-powered centrifuge that is made out of paper (“paperfuge”). To push the operating limits of this unconventional centrifuge, we present an experimentally-validated theoretical model that describes the paperfuge as a non-linear, non-conservative oscillator system. We use this model to inform our design process, achieving speeds of 125,000 rpm and equivalent centrifugal forces of 30,000 g, with theoretical limits predicting one million rpm. We harness these speeds to separate pure plasma in less than 1.5 minutes and isolate malaria parasites in 15 minutes from whole human blood. By expanding the materials used, we implement centrifugal microfluidics using PDMS, plastic and 3D-printed devices, ultimately opening up new opportunities for electricity-free POC diagnostics, especially in resource-poor settings.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/bioengineering">bioengineering</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2016">2016</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/bioengineering/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/bioengineering/page/3/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/bioengineering/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/bioengineering/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/bioengineering/page/3/" class="page-link">3</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/bioengineering/page/4/" class="page-link">4</a></li>
            
        

        

        
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
