<!DOCTYPE html>
<html><head>
  
  <title>Genetics | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_method_for_genome-wide_genealogy_estimation_for_thousands_of_samples_55484262/">
          <h4 class="font-weight-bold">A method for genome-wide genealogy estimation for thousands of samples, bioRxiv, 2019-02-15</h4>
          <p>Knowledge of genome-wide genealogies for thousands of individuals would simplify most evolutionary analyses for humans and other species, but has remained computationally infeasible. We developed a method, Relate, scaling to &amp;gt; 10,000 sequences while simultaneously estimating branch lengths, mutational ages,  and variable historical population sizes, as well as allowing for data errors. Application to 1000 Genomes Project haplotypes produces joint genealogical histories for 26 human populations. Highly diverged lineages are present in all groups, but most frequent in Africa. Outside Africa, these mainly reflect ancient introgression from groups related to Neanderthals and Denisovans, while African signals instead reflect unknown events, unique to that continent. Our approach allows more powerful inferences of natural selection than previously possible. We identify multiple novel regions under strong positive selection, and multi-allelic traits including hair colour, BMI, and blood pressure, showing strong evidence of directional selection, varying among human groups.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/cannabis_use_depression_and_self-harm_phenotypic_and_genetic_relationships_55486314/">
          <h4 class="font-weight-bold">Cannabis use, depression and self-harm phenotypic and genetic relationships, bioRxiv, 2019-02-15</h4>
          <p>Background and Aims The use of cannabis has previously been linked to both depression and self-harm, however the role of genetics in this relationship are unclear. We aimed to examine the phenotypic and genetic relationships between these traits.Design Genetic and cross-sectional phenotypic data collected through UK Biobank, together with consortia genome-wide association study summary statistics. These data were used to assess the phenotypic and genetic relationship between cannabis use, depression and self harm.Setting UK, with additional international consortia dataParticipants N=126,291 British adults aged between 40 and 70 years, recruited into UK Biobank.Measurements Genome-wide genetic data, phenotypic data on lifetime history of cannabis use, depression and self-harm.Findings In UK Biobank, cannabis use is associated with increased likelihood of depression (OR=1.64, 95% CI=1.59-1.70, p=1.19x10-213) and self-harm (OR=2.85, 95% CI=2.69-3.01, p=3.46x10-304). The strength of this phenotypic association is stronger when more severe trait definitions of cannabis use and depression are considered. Additionally, significant genetic correlations are seen between cannabis use and depression using consortia summary statistics (rg=0.289, SE=0.036, p=1.45x10-15). Polygenic risk scores for cannabis use and depression both explain a small but significant proportion of variance in cannabis use, depression and self harm within a UK Biobank target sample. However, two-sample Mendelian randomisation analyses were not significant.Conclusions Cannabis use is both phenotypically and genetically associated with depression and self harm. Future work dissecting the causal mechanism linking these traits may have implications for cannabis users.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/new_insights_into_malaria_susceptibility_from_the_genomes_of_17000_individuals_from_africa_asia_and_oceania_54946042/">
          <h4 class="font-weight-bold">New insights into malaria susceptibility from the genomes of 17,000 individuals from Africa, Asia, and Oceania, bioRxiv, 2019-02-04</h4>
          <p>We conducted a genome-wide association study of host resistance to severe Plasmodium falciparum malaria in over 17,000 individuals from 11 malaria- endemic countries, undertaking a wide ranging analysis which identifies five replicable associations with genome-wide levels of evidence. Our findings include a newly implicated variant on chromosome 6 associated with risk of cerebral malaria, and the discovery of an erythroid-specific transcription start site underlying the association in ATP2B4. Previously reported HLA associations cannot be replicated in this dataset. We estimate substantial heritability of severe malaria (h2 ~ 23%), of which around 10% is explained by the currently identified associations. Our dataset will provide a major building block for future research on the genetic determinants of disease in these diverse human populations.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/resistance_gene_cloning_from_a_wild_crop_relative_by_sequence_capture_and_association_genetics_54941534/">
          <h4 class="font-weight-bold">Resistance gene cloning from a wild crop relative by sequence capture and association genetics, Nature Biotechnology, 2019-02-04</h4>
          <p>Disease resistance (R) genes from wild relatives could be used to engineer broad-spectrum resistance in domesticated crops. We combined association genetics with R gene enrichment sequencing (AgRenSeq) to exploit pan-genome variation in wild diploid wheat and rapidly clone four stem rust resistance genes. AgRenSeq enables R gene cloning in any crop that has a diverse germplasm panel.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/nature%20biotechnology">nature biotechnology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/patterns_of_genetic_differentiation_and_the_footprints_of_historical_migrations_in_the_iberian_peninsula_54801340/">
          <h4 class="font-weight-bold">Patterns of genetic differentiation and the footprints of historical migrations in the Iberian Peninsula, Nature Communications, 2019-02-01</h4>
          <p>Reconquista</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/nature%20communications">nature communications</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/activity-by-contact_model_of_enhancer_specificity_from_thousands_of_crispr_perturbations_54544414/">
          <h4 class="font-weight-bold">Activity-by-Contact model of enhancer specificity from thousands of CRISPR perturbations, bioRxiv, 2019-01-27</h4>
          <p>Mammalian genomes harbor millions of noncoding elements called enhancers that quantitatively regulate gene expression, but it remains unclear which enhancers regulate which genes. Here we describe an experimental approach, based on CRISPR interference, RNA FISH, and flow cytometry (CRISPRi-FlowFISH), to perturb enhancers in the genome, and apply it to test &amp;gt;3,000 potential regulatory enhancer-gene connections across multiple genomic loci. A simple equation based on a mechanistic model for enhancer function performed remarkably well at predicting the complex patterns of regulatory connections we observe in our CRISPR dataset. This Activity-by-Contact (ABC) model involves multiplying measures of enhancer activity and enhancer-promoter 3D contacts, and can predict enhancer-gene connections in a given cell type based on chromatin state maps. Together, CRISPRi-FlowFISH and the ABC model provide a systematic approach to map and predict which enhancers regulate which genes, and will help to interpret the functions of the thousands of disease risk variants in the noncoding genome.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/genetics/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/genetics/page/14/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../../tags/genetics/page/11/" class="page-link">11</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/genetics/page/12/" class="page-link">12</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/genetics/page/13/" class="page-link">13</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/genetics/page/14/" class="page-link">14</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/genetics/page/15/" class="page-link">15</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/genetics/page/16/" class="page-link">16</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/genetics/page/17/" class="page-link">17</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/genetics/page/18/" class="page-link">18</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/genetics/page/19/" class="page-link">19</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/genetics/page/16/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/genetics/page/36/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
