<!DOCTYPE html>
<html><head>
  
  <title>2018 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/protein-coding_variation_and_introgression_of_regulatory_alleles_drive_plumage_pattern_diversity_in_the_rock_pigeon_31301251/">
          <h4 class="font-weight-bold">Protein-coding variation and introgression of regulatory alleles drive plumage pattern diversity in the rock pigeon, bioRxiv, 2018-01-06</h4>
          <p>ABSTRACTBirds and other vertebrates display stunning variation in pigmentation patterning, yet the genes controlling this diversity remain largely unknown. Rock pigeons (Columba livia) are fundamentally one of four color pattern phenotypes, in decreasing order of melanism T-check, checker, bar (ancestral), or barless. Using whole-genome scans, we identified NDP as a candidate gene for this variation. Allele-specific expression differences in NDP indicate cis-regulatory differences between ancestral and melanistic alleles. Sequence comparisons suggest that derived alleles originated in the speckled pigeon (Columba guinea), providing a striking example of introgression of alleles that are favored by breeders and are potentially advantageous in the wild. In contrast, barless rock pigeons have an increased incidence of vision defects and, like two human families with hereditary blindness, carry start-codon mutations in NDP. In summary, we find unexpected links between color pattern, introgression, and vision defects associated with regulatory and coding variation at a single locus.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genome-wide_association_study_of_1_million_people_identifies_111_loci_for_atrial_fibrillation_31258135/">
          <h4 class="font-weight-bold">Genome-wide association study of 1 million people identifies 111 loci for atrial fibrillation, bioRxiv, 2018-01-05</h4>
          <p>SummaryTo understand the genetic variation underlying atrial fibrillation (AF), the most common cardiac arrhythmia, we performed a genome-wide association study (GWAS) of &amp;gt; 1 million people, including 60,620 AF cases and 970,216 controls. We identified 163 independent risk variants at 111 loci and prioritized 165 candidate genes likely to be involved in AF. Many of the identified risk variants fall near genes where more deleterious mutations have been reported to cause serious heart defects in humans or mice (MYH6, NKX2-5, PITX2, TBC1D32, TBX5),1,2 or near genes important for striated muscle function and integrity (e.g. MYH7, PKP2, SSPN, SGCA). Experiments in rabbits with heart failure and left atrial dilation identified a heterogeneous distributed molecular switch from MYH6 to MYH7 in the left atrium, which resulted in contractile and functional heterogeneity and may predispose to initiation and maintenance of atrial arrhythmia.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/acoustically_targeted_chemogenetics_for_noninvasive_control_of_neural_circuits_31144328/">
          <h4 class="font-weight-bold">Acoustically Targeted Chemogenetics for Noninvasive Control of Neural Circuits, bioRxiv, 2018-01-02</h4>
          <p>ABSTRACTNeurological and psychiatric diseases often involve the dysfunction of specific neural circuits in particular regions of the brain. Existing treatments, including drugs and implantable brain stimulators, aim to modulate the activity of these circuits, but are typically not cell type-specific, lack spatial targeting or require invasive procedures. Here, we introduce an approach to modulating neural circuits noninvasively with spatial, cell-type and temporal specificity. This approach, called acoustically targeted chemogenetics, or ATAC, uses transient ultrasonic opening of the blood brain barrier to transduce neurons at specific locations in the brain with virally-encoded engineered G-protein-coupled receptors, which subsequently respond to systemically administered bio-inert compounds to activate or inhibit the activity of these neurons. We demonstrate this concept in mice by using ATAC to noninvasively modify and subsequently activate or inhibit excitatory neurons within the hippocampus, showing that this enables pharmacological control of memory formation. This technology allows a brief, noninvasive procedure to make one or more specific brain regions capable of being selectively modulated using orally bioavailable compounds, thereby overcoming some of the key limitations of conventional brain therapies.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/defects_in_the_neuroendocrine_axis_cause_global_development_delay_in_a_drosophila_model_of_ngly1_deficiency_31142136/">
          <h4 class="font-weight-bold">Defects in the neuroendocrine axis cause global development delay in a Drosophila model of NGLY1 Deficiency, bioRxiv, 2018-01-02</h4>
          <p>ABSTRACTN-glycanase 1 (NGLY1) Deficiency is a rare monogenic multi-system disorder first described in 2014. NGLY1 is evolutionarily conserved in model organisms, including the Drosophila melanogaster NGLY1 homolog, Pngl. Here we conducted a natural history study and chemical-modifier screen on a new fly model of NGLY1 Deficiency engineered with a nonsense mutation in Pngl at codon 420, resulting in truncation of the C-terminal carbohydrate-binding PAW domain. Homozygous mutant animals exhibit global development delay, pupal lethality and small body size as adults. We developed a 96-well-plate, image-based, quantitative assay of Drosophila larval size for use in a screen of the 2,650-member Microsource Spectrum compound library of FDA approved drugs, bioactive tool compounds, and natural products. We found that the cholesterol-derived ecdysteroid molting hormone 20-hydroxyecdysone (20E) rescued the global developmental delay in mutant homozygotes. Targeted expression of a human NGLY1 transgene to tissues involved in ecdysteroidogenesis, e.g., prothoracic gland, also rescues global developmental delay in mutant homozygotes. Finally, the proteasome inhibitor bortezomib is a potent enhancer of global developmental delay in our fly model, evidence of a defective proteasome “bounce-back” response that is also observed in nematode and cellular models of NGLY1 Deficiency. Together, these results demonstrate the therapeutic relevance of a new fly model of NGLY1 Deficiency for drug discovery, biomarker discovery, pharmacodynamics studies, and gene modifier screens.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/comparison_of_computational_methods_for_imputing_single-cell_rna-sequencing_data_31110841/">
          <h4 class="font-weight-bold">Comparison of computational methods for imputing single-cell RNA-sequencing data, bioRxiv, 2018-01-01</h4>
          <p>AbstractSingle-cell RNA-sequencing (scRNA-seq) is a recent breakthrough technology, which paves the way for measuring RNA levels at single cell resolution to study precise biological functions. One of the main challenges when analyzing scRNA-seq data is the presence of zeros or dropout events, which may mislead downstream analyses. To compensate the dropout effect, several methods have been developed to impute gene expression since the first Bayesian-based method being proposed in 2016. However, these methods have shown very diverse characteristics in terms of model hypothesis and imputation performance. Thus, large-scale comparison and evaluation of these methods is urgently needed now. To this end, we compared eight imputation methods, evaluated their power in recovering original real data, and performed broad analyses to explore their effects on clustering cell types, detecting differentially expressed genes, and reconstructing lineage trajectories in the context of both simulated and real data. Simulated datasets and case studies highlight that there are no one method performs the best in all the situations. Some defects of these methods such as scalability, robustness and unavailability in some situations need to be addressed in future studies.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/deepgs_predicting_phenotypes_from_genotypes_using_deep_learning_31104270/">
          <h4 class="font-weight-bold">DeepGS Predicting phenotypes from genotypes using Deep Learning, bioRxiv, 2018-01-01</h4>
          <p>AbstractMotivationGenomic selection (GS) is a new breeding strategy by which the phenotypes of quantitative traits are usually predicted based on genome-wide markers of genotypes using conventional statistical models. However, the GS prediction models typically make strong assumptions and perform linear regression analysis, limiting their accuracies since they do not capture the complex, non-linear relationships within genotypes, and between genotypes and phenotypes.ResultsWe present a deep learning method, named DeepGS, to predict phenotypes from genotypes. Using a deep convolutional neural network, DeepGS uses hidden variables that jointly represent features in genotypic markers when making predictions; it also employs convolution, sampling and dropout strategies to reduce the complexity of high-dimensional marker data. We used a large GS dataset to train DeepGS and compare its performance with other methods. In terms of mean normalized discounted cumulative gain value, DeepGS achieves an increase of 27.70%~246.34% over a conventional neural network in selecting top-ranked 1% individuals with high phenotypic values for the eight tested traits. Additionally, compared with the widely used method RR-BLUP, DeepGS still yields a relative improvement ranging from 1.44% to 65.24%. Through extensive simulation experiments, we also demonstrated the effectiveness and robustness of DeepGS for the absent of outlier individuals and subsets of genotypic markers. Finally, we illustrated the complementarity of DeepGS and RR-BLUP with an ensemble learning approach for further improving prediction performance.AvailabilityDeepGS is provided as an open source R package available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsgithub.comcma2015DeepGS&gt;httpsgithub.comcma2015DeepGS&lt;jatsext-link&gt;.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/2018/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/2018/page/82/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2018/page/79/" class="page-link">79</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2018/page/80/" class="page-link">80</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2018/page/81/" class="page-link">81</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2018/page/82/" class="page-link">82</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/2018/page/83/" class="page-link">83</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2018/page/84/" class="page-link">84</a></li>
            
        

        
        <li class="page-item"><a href="../../../../tags/2018/page/84/" rel="next" class="page-link">Next ›</a></li>
        

        
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
