<!DOCTYPE html>
<html><head>
  
  <title>Neuroscience | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/geosmin_attracts_aedes_aegypti_mosquitoes_to_oviposition_sites_58558884/">
          <h4 class="font-weight-bold">Geosmin attracts Aedes aegypti mosquitoes to oviposition sites, bioRxiv, 2019-04-05</h4>
          <p>Geosmin is one of the most recognizable and common microbial smells on the planet. Some insects, like mosquitoes, require microbial-rich environments for their progeny, whereas for other insects such microbes may prove dangerous. In the vinegar fly Drosophila melanogaster, geosmin is decoded in a remarkably precise fashion and induces aversion, presumably signaling the presence of harmful microbes. We have here investigated the effect of geosmin on the behavior of the yellow fever mosquito Aedes aegypti. In contrast to flies, geosmin is not aversive in mosquitoes but stimulates egg-laying site selection. Female mosquitoes could associate geosmin with microbes, including cyanobacteria consumed by larvae, who also find geosmin – as well as geosmin producing cyanobacteria – attractive. Using in vivo multiphoton imaging from mosquitoes with pan-neural expression of the calcium reporter GCaMP6s, we show that Ae. aegypti code geosmin in a similar fashion to flies, i.e. with extreme sensitivity and with a high degree of selectivity. We further demonstrate that geosmin can be used as bait under field conditions, and finally we show that geosmin, which is both expensive and difficult to obtain, can be substituted by beetroot peel extract, providing a cheap and viable mean of mosquito control and surveillance in developing countries.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/inferring_the_function_performed_by_a_recurrent_neural_network_58604703/">
          <h4 class="font-weight-bold">Inferring the function performed by a recurrent neural network, bioRxiv, 2019-04-05</h4>
          <p>AbstractA central goal in systems neuroscience is to understand the functions performed by neural circuits. Previous top-down models addressed this question by comparing the behaviour of an ideal model circuit, optimised to perform a given function, with neural recordings. However, this requires guessing in advance what function is being performed, which may not be possible for many neural systems. Here, we propose an alternative approach that uses recorded neural responses to directly infer the function performed by a neural network. We assume that the goal of the network can be expressed via a reward function, which describes how desirable each state of the network is for carrying out a given objective. This allows us to frame the problem of optimising each neuron’s responses by viewing neurons as agents in a reinforcement learning (RL) paradigm; likewise the problem of inferring the reward function from the observed dynamics can be treated using inverse RL. Our framework encompasses previous influential theories of neural coding, such as efficient coding and attractor network models, as special cases, given specific choices of reward function. Finally, we can use the reward function inferred from recorded neural responses to make testable predictions about how the network dynamics will adapt depending on contextual changes, such as cell death andor varying input statistics, so as to carry out the same underlying function with different constraints.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/large-scale_death_of_retinal_astrocytes_during_normal_development_mediated_by_microglia_58453924/">
          <h4 class="font-weight-bold">Large-scale death of retinal astrocytes during normal development mediated by microglia, bioRxiv, 2019-04-04</h4>
          <p>Naturally-occurring cell death is a fundamental developmental mechanism for regulating cell numbers and sculpting developing organs. This is particularly true in the central nervous system, where large numbers of neurons and oligodendrocytes are eliminated via apoptosis during normal development. Given the profound impact of death upon these two major cell populations, it is surprising that developmental death of another major cell type – the astrocyte – has rarely been studied. It is presently unclear whether astrocytes are subject to significant amounts of developmental death, or how it occurs. Here we address these questions using mouse retinal astrocytes as our model system. We show that the total number of retinal astrocytes declines by over 3-fold during a death period spanning postnatal days 5-14. Surprisingly, these astrocytes do not die by apoptosis, the canonical mechanism underlying the vast majority of developmental cell death. Instead, we find that microglia kill and engulf astrocytes to mediate their developmental removal. Genetic ablation of microglia inhibits astrocyte death, leading to a larger astrocyte population size at the end of the death period. However, astrocyte death is not completely blocked in the absence of microglia, apparently due to the ability of astrocytes to engulf each other. Nevertheless, mice lacking microglia showed significant anatomical changes to the retinal astrocyte network, with functional consequences for the astrocyte-associated vasculature leading to retinal hemorrhage. These results establish a novel modality for naturally-occurring cell death, and demonstrate its importance for formation and integrity of the retinal gliovascular network.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/margo_massively_automated_real-time_gui_for_object-tracking_a_platform_for_high-throughput_ethology_58113148/">
          <h4 class="font-weight-bold">MARGO (Massively Automated Real-time GUI for Object-tracking), a platform for high-throughput ethology, bioRxiv, 2019-03-30</h4>
          <p>AbstractFast object tracking in real time allows convenient tracking of very large numbers of animals and closed-loop experiments that control stimuli for multiple animals in parallel. We developed MARGO, a real-time animal tracking suite for custom behavioral experiments. We demonstrated that MARGO can rapidly and accurately track large numbers of animals in parallel over very long timescales. We incorporated control of peripheral hardware, and implemented a flexible software architecture for defining new experimental routines. These features enable closed-loop delivery of stimuli to many individuals simultaneously. We highlight MARGO’s ability to coordinate tracking and hardware control with two custom behavioral assays (measuring phototaxis and optomotor response) and one optogenetic operant conditioning assay. There are currently several open source animal trackers. MARGO’s strengths are 1) robustness, 2) high throughput, 3) flexible control of hardware and 4) real-time closed-loop control of sensory and optogenetic stimuli, all of which are optimized for large-scale experimentation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/eztrack_an_open-source_video_analysis_pipeline_for_the_investigation_of_animal_behavior_58113144/">
          <h4 class="font-weight-bold">ezTrack An open-source video analysis pipeline for the investigation of animal behavior, bioRxiv, 2019-03-30</h4>
          <p>AbstractTracking small animal behavior by video is one of the most common tasks in the fields of neuroscience and psychology. Although commercial software exists for the execution of this task, commercial software often presents enormous cost to the researcher, and can also entail purchasing specific hardware setups that are not only expensive but lack adaptability. Moreover, the inaccessibility of the code underlying this software renders them inflexible. Alternatively, available open source options frequently require extensive model training and can be challenging for those inexperienced with programming. Here we present an open source and platform independent set of behavior analysis pipelines using interactive Python (iPythonJupyter Notebook) that researchers with no prior programming experience can use. Two modules are described. One module can be used for the positional analysis of an individual animal across a session (i.e., location tracking), amenable to a wide range of behavioral tasks including conditioned place preference, water maze, light-dark box, open field, and elevated plus maze, to name but a few. A second module is described for the analysis of conditioned freezing behavior. For both modules, a range of interactive plots and visualizations are available to confirm that chosen parameters produce results that conform to the user’s approval. In addition, batch processing tools for the fast analysis of multiple videos is provided, and frame-by-frame output makes aligning the data with neural recording data simple. Lastly, options for cropping video frames to mitigate the influence of fiberopticelectrophysiology cables, analyzing specified portions of time in a video, and defining regions of interest, can be implemented with ease.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/polygenic_architecture_of_human_neuroanatomical_diversity_57963714/">
          <h4 class="font-weight-bold">Polygenic architecture of human neuroanatomical diversity, bioRxiv, 2019-03-29</h4>
          <p>AbstractWe analysed the genomic architecture of neuroanatomical diversity using magnetic resonance imaging and SNP data from &amp;gt; 20,000 individuals. Our results replicate previous findings of a strong polygenic architecture of neuroanatomical diversity. SNPs captured from 40% to 54% of the variance in the volume of different brain regions. We observed a large correlation between chromosome length and the amount of phenotypic variance captured, r∼0.64 on average, suggesting that at a global scale causal variants are homogeneously distributed across the genome. At a more local scale, SNPs within genes (∼51%) captured ∼1.5-times more genetic variance than the rest; and SNPs with low minor allele frequency (MAF) captured significantly less variance than those with higher MAF the 40% of SNPs with MAF&amp;lt;5% captured less than one fourth of the genetic variance. We also observed extensive pleiotropy across regions, with an average genetic correlation of rG∼0.45. Across regions, genetic correlations were in general similar to phenotypic correlations. By contrast, genetic correlations were larger than phenotypic correlations for the leftright volumes of the same region, and indistinguishable from 1. Additionally, the differences in leftright volumes were not heritable, underlining the role of environmental causes in the variability of brain asymmetry. Our analysis code is available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsgithub.comneuroanatomygenomic-architecture&gt;httpsgithub.comneuroanatomygenomic-architecture&lt;jatsext-link&gt;.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/neuroscience/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/neuroscience/page/5/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/neuroscience/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/neuroscience/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/neuroscience/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/neuroscience/page/5/" class="page-link">5</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/neuroscience/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/neuroscience/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/neuroscience/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/neuroscience/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/neuroscience/page/10/" class="page-link">10</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/neuroscience/page/7/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/neuroscience/page/29/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
