<!DOCTYPE html>
<html><head>
  
  <title>neuroscience | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.68.3" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.95d2483cd3c5d2bf47c1f36307a456787d60ad958c5efe44db9afc47012b1b05b8e1f64f6ac291650439322a6c38e86eb7cd6254255516d857c23f299fb0e666.js" integrity="sha512-ldJIPNPF0r9HwfNjB6RWeH1grZWMXv5E25r8RwErGwW44fZPasKRZQQ5MipsOOhut81iVCVVFthXwj8pn7DmZg=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/all-optical_electrophysiology_reveals_brain-state_dependent_changes_in_hippocampal_subthreshold_dynamics_and_excitability_34302488/">
          <h4 class="font-weight-bold">All-optical electrophysiology reveals brain-state dependent changes in hippocampal subthreshold dynamics and excitability, bioRxiv, 2018-03-14</h4>
          <p>AbstractA technology to record membrane potential from multiple neurons, simultaneously, in behaving animals will have a transformative impact on neuroscience research1. Parallel recordings could reveal the subthreshold potentials and intercellular correlations that underlie network behavior2. Paired stimulation and recording can further reveal the input-output properties of individual cells or networks in the context of different brain states3. Genetically encoded voltage indicators are a promising tool for these purposes, but were so far limited to single-cell recordings with marginal signal to noise ratio (SNR) in vivo4-6. We developed improved near infrared voltage indicators, high speed microscopes and targeted gene expression schemes which enabled recordings of supra- and subthreshold voltage dynamics from multiple neurons simultaneously in mouse hippocampus, in vivo. The reporters revealed sub-cellular details of back-propagating action potentials, correlations in sub-threshold voltage between multiple cells, and changes in dynamics associated with transitions from resting to locomotion. In combination with optogenetic stimulation, the reporters revealed brain state-dependent changes in neuronal excitability, reflecting the interplay of excitatory and inhibitory synaptic inputs. These tools open the possibility for detailed explorations of network dynamics in the context of behavior.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/spatial_organization_of_the_somatosensory_cortex_revealed_by_cyclic_smfish_33929641/">
          <h4 class="font-weight-bold">Spatial organization of the somatosensory cortex revealed by cyclic smFISH, bioRxiv, 2018-03-05</h4>
          <p>The global efforts towards the creation of a molecular census of the brain using single-cell transcriptomics is generating a large catalog of molecularly defined cell types lacking spatial information. Thus, new methods are needed to map a large number of cell-specific markers simultaneously on large tissue areas. Here, we developed a cyclic single molecule fluorescence in situ hybridization methodology and defined the cellular organization of the somatosensory cortex using markers identified by single-cell transcriptomics.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/unsupervised_discovery_of_temporal_sequences_in_high-dimensional_datasets_with_applications_to_neuroscience_33858378/">
          <h4 class="font-weight-bold">Unsupervised discovery of temporal sequences in high-dimensional datasets, with applications to neuroscience, bioRxiv, 2018-03-03</h4>
          <p>AbstractIdentifying low-dimensional features that describe large-scale neural recordings is a major challenge in neuroscience. Repeated temporal patterns (sequences) are thought to be a salient feature of neural dynamics, but are not succinctly captured by traditional dimensionality reduction techniques. Here we describe a software toolbox—called seqNMF—with new methods for extracting informative, non-redundant, sequences from high-dimensional neural data, testing the significance of these extracted patterns, and assessing the prevalence of sequential structure in data. We test these methods on simulated data under multiple noise conditions, and on several real neural and behavioral data sets. In hippocampal data, seqNMF identifies neural sequences that match those calculated manually by reference to behavioral events. In songbird data, seqNMF discovers neural sequences in untutored birds that lack stereotyped songs. Thus, by identifying temporal structure directly from neural data, seqNMF enables dissection of complex neural circuits without relying on temporal references from stimuli or behavioral outputs.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/an_animal-actuated_rotational_head-fixation_system_for_2-photon_imaging_during_2-d_navigation_33829308/">
          <h4 class="font-weight-bold">An animal-actuated rotational head-fixation system for 2-photon imaging during 2-d navigation, bioRxiv, 2018-03-02</h4>
          <p>AbstractUnderstanding how the biology of the brain gives rise to the computations that drive behavior requires high fidelity, large scale, and subcellular measurements of neural activity. 2-photon microscopy is the primary tool that satisfies these requirements, particularly for measurements during behavior. However, this technique requires rigid head-fixation, constraining the behavioral repertoire of experimental subjects. Increasingly, complex task paradigms are being used to investigate the neural substrates of complex behaviors, including navigation of complex environments, resolving uncertainty between multiple outcomes, integrating unreliable information over time, andor building internal models of the world. In rodents, planning and decision making processes are often expressed via head and body motion. This produces a significant limitation for head-fixed two-photon imaging. We therefore developed a system that overcomes a major problem of head-fixation the lack of rotational vestibular input. The system measures rotational strain exerted by mice on the head restraint, which consequently drives a motor, rotating the constraint system and dissipating the strain. This permits mice to rotate their heads in the azimuthal plane with negligible inertia and friction. This stable rotating head-fixation system allows mice to explore physical or virtual 2-D environments. To demonstrate the performance of our system, we conducted 2-photon GCaMP6f imaging in somas and dendrites of pyramidal neurons in mouse retrosplenial cortex. We show that the subcellular resolution of the system’s 2-photon imaging is comparable to that of conventional head-fixed experiments. Additionally, this system allows the attachment of heavy instrumentation to the animal, making it possible to extend the approach to large-scale electrophysiology experiments in the future. Our method enables the use of state-of-the-art imaging techniques while animals perform more complex and naturalistic behaviors than currently possible, with broad potential applications in systems neuroscience.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/end-to-end_deep_image_reconstruction_from_human_brain_activity_33734694/">
          <h4 class="font-weight-bold">End-to-end deep image reconstruction from human brain activity, bioRxiv, 2018-02-28</h4>
          <p>AbstractDeep neural networks (DNNs) have recently been applied successfully to brain decoding and image reconstruction from functional magnetic resonance imaging (fMRI) activity. However, direct training of a DNN with fMRI data is often avoided because the size of available data is thought to be insufficient to train a complex network with numerous parameters. Instead, a pre-trained DNN has served as a proxy for hierarchical visual representations, and fMRI data were used to decode individual DNN features of a stimulus image using a simple linear model, which were then passed to a reconstruction module. Here, we present our attempt to directly train a DNN model with fMRI data and the corresponding stimulus images to build an end-to-end reconstruction model. We trained a generative adversarial network with an additional loss term defined in a high-level feature space (feature loss) using up to 6,000 training data points (natural images and the fMRI responses). The trained deep generator network was tested on an independent dataset, directly producing a reconstructed image given an fMRI pattern as the input. The reconstructions obtained from the proposed method showed resemblance with both natural and artificial test stimuli. The accuracy increased as a function of the training data size, though not outperforming the decoded feature-based method with the available data size. Ablation analyses indicated that the feature loss played a critical role to achieve accurate reconstruction. Our results suggest a potential for the end-to-end framework to learn a direct mapping between brain activity and perception given even larger datasets.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/motor_cortex_is_an_input-driven_dynamical_system_controlling_dexterous_movement_33259776/">
          <h4 class="font-weight-bold">Motor cortex is an input-driven dynamical system controlling dexterous movement, bioRxiv, 2018-02-16</h4>
          <p>AbstractSkillful control of movement is central to our ability to sense and manipulate the world. A large body of work in nonhuman primates has demonstrated that motor cortex provides flexible, time-varying activity patterns that control the arm during reaching and grasping. Previous studies have suggested that these patterns are generated by strong local recurrent dynamics operating autonomously from inputs during movement execution. An alternative possibility is that motor cortex requires coordination with upstream brain regions throughout the entire movement in order to yield these patterns. Here, we developed an experimental preparation in the mouse to directly test these possibilities using optogenetics and electrophysiology during a skilled reach-to-grab-to-eat task. To validate this preparation, we first established that a specific, time-varying pattern of motor cortical activity was required to produce coordinated movement. Next, in order to disentangle the contribution of local recurrent motor cortical dynamics from external input, we optogenetically held the recurrent contribution constant, then observed how motor cortical activity recovered following the end of this perturbation. Both the neural responses and hand trajectory varied from trial to trial, and this variability reflected variability in external inputs. To directly probe the role of these inputs, we used optogenetics to perturb activity in the thalamus. Thalamic perturbation at the start of the trial prevented movement initiation, and perturbation at any stage of the movement prevented progression of the hand to the target; this demonstrates that input is required throughout the movement. By comparing motor cortical activity with and without thalamic perturbation, we were able to estimate the effects of external inputs on motor cortical population activity. Thus, unlike pattern-generating circuits that are local and autonomous, such as those in the spinal cord that generate left-right alternation during locomotion, the pattern generator for reaching and grasping is distributed across multiple, strongly-interacting brain regions.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/37/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/34/" class="page-link">34</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/35/" class="page-link">35</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/36/" class="page-link">36</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/37/" class="page-link">37</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/38/" class="page-link">38</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/39/" class="page-link">39</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/40/" class="page-link">40</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/41/" class="page-link">41</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/42/" class="page-link">42</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/39/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/53/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
