<!DOCTYPE html>
<html><head>
  
  <title>Biorxiv | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/feeding_kinematics_and_morphology_of_the_alligator_gar_atractosteus_spatula_lac%C3%A9p%C3%A8de_1803_56154162/">
          <h4 class="font-weight-bold">Feeding kinematics and morphology of the alligator gar (Atractosteus spatula, Lacépède, 1803), bioRxiv, 2019-02-28</h4>
          <p>Modern (lepisosteid) gars are a small clade of seven species and two genera that occupy an important position on the actinopterygian phylogenetic tree as members of the Holostei (Amia&#43; gars), sister-group of the teleost radiation. Often referred to as living fossils, these taxa preserve many plesiomorphic characteristics used to interpret and reconstruct early osteichthyan feeding conditions. Less attention, however, has been paid to the functional implications of gar-specific morphology, thought to be related to an exclusively ram-based, lateral-snapping mode of prey capture. Previous studies of feeding kinematics in gars have focused solely on members of the narrow-snouted Lepisosteus genus, and here we expand that dataset to include a member of the broad-snouted sister-genus and largest species of gar, the alligator gar (Atractosteus spatula, Lacépède, 1803). High-speed videography reveals that the feeding system of alligator gars is capable of rapid expansion from anterior-to-posterior, precisely timed in a way that appears to counteract the effects of a bow-wave during ram-feeding and generate a unidirectional flow of water through the feeding system. Reconstructed cranial anatomy based on contrast-enhanced micro-CT data show that a lateral-sliding palatoquadrate, flexible intrasuspensorial joint, pivoting interhyal, and retractable pectoral girdle are all responsible for increasing the range of motion and expansive capabilities of the gar cranial linkage system. Muscular reconstructions and manipulation experiments show that, while the sternohyoideus is the primary input to the feeding system (similar to other basal actinopterygians), additional input from the hyoid constrictors and hypaxials play an important role in decoupling and modulating between the dual roles of the sternohyoideus hyoid retraction (jaw opening) and hyoid rotation (pharyngeal expansion) respectively. The data presented here demonstrate an intricate feeding mechanism, capable of precise control with plesiomorphic muscles, that represents one of the many ways the ancestral osteichthyan feeding mechanism has been modified for prey capture.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/human_loss-of-function_variants_suggest_that_partial_lrrk2_inhibition_is_a_safe_therapeutic_strategy_for_parkinsons_disease_56151907/">
          <h4 class="font-weight-bold">Human loss-of-function variants suggest that partial LRRK2 inhibition is a safe therapeutic strategy for Parkinsons disease, bioRxiv, 2019-02-28</h4>
          <p>Human genetic variants causing loss of function (LoF) of protein-coding genes provide natural in vivo models of gene inactivation, which are powerful indicators of gene function and the potential toxicity of therapeutic inhibitors targeting these genes. Gain of kinase function variants in LRRK2 are known to significantly increase the risk of Parkinsons disease suggesting that inhibition of LRRK2 kinase activity is a promising therapeutic strategy. Whilst preclinical studies in model organisms have raised some on-target toxicity concerns, the biological consequences of LRRK2 inhibition have not been well characterized in humans. Here we systematically analyse LoF variants in LRRK2 observed across 141,456 individuals sequenced in the Genome Aggregation Database (gnomAD) and over 4 million participants in the 23andMe genotyped dataset, to assess their impact at a molecular and phenotypic level. After thorough variant curation, we identify 1,358 individuals with high-confidence predicted LoF variants in LRRK2, several with experimental validation. We show that heterozygous LoF of LRRK2 reduces LRRK2 protein level by ~50% but is not associated with reduced life expectancy, or with any specific phenotype or disease state. These data suggest that therapeutics that downregulate LRRK2 levels or kinase activity by up to 50% are unlikely to have major on-target safety liabilities. Our results demonstrate the value of large scale genomic databases and phenotyping of human LoF carriers for target validation in drug discovery.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/interplay_between_the_human_gut_microbiome_and_host_metabolism_56164411/">
          <h4 class="font-weight-bold">Interplay between the human gut microbiome and host metabolism, bioRxiv, 2019-02-28</h4>
          <p>The human gut is inhabited by a complex and metabolically active microbial ecosystem regulating host health. While many studies have focused on the effect of individual microbial taxa, the metabolic potential of the entire gut microbial ecosystem has been largely under-explored. We characterised the gut microbiome of 1,004 twins via whole shotgun metagenomic sequencing (average 39M reads per sample). We observed greater similarity, across unrelated individuals, for functional metabolic pathways (82%) than for taxonomic composition (43%). We conducted a microbiota-wide association study linking both taxonomic information and microbial metabolic pathways with 673 blood and 713 faecal metabolites (Metabolon, Inc.). Metabolic pathways associated with 34% of blood and 95% of faecal metabolites, with over 18,000 significant associations, while species-level results identified less than 3,000 associations, suggesting that coordinated action of multiple taxa is required to affect the metabolome. Finally, we estimated that the microbiome mediated a crosstalk between 71% of faecal and 15% of blood metabolites, highlighting six key species (unclassified Subdoligranulum spp., Faecalibacterium prausnitzii, Roseburia inulinivorans, Methanobrevibacter smithii, Eubacterium rectale, and Akkermansia muciniphila). Because of the large inter-person variability in microbiome composition, our results underline the importance of studying gut microbial metabolic pathways rather than focusing purely on taxonomy to find therapeutic and diagnostic targets.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/a_positively_selected_common_missense_variant_in_fbn1_confers_a_2.2_centimeter_reduction_of_height_in_the_peruvian_population_56071057/">
          <h4 class="font-weight-bold">A positively selected, common, missense variant in FBN1 confers a 2.2 centimeter reduction of height in the Peruvian population, bioRxiv, 2019-02-26</h4>
          <p>Peruvians are among the shortest people in the world. To understand the genetic basis of short stature in Peru, we examined an ethnically diverse group of Peruvians and identified a novel, population-specific, missense variant in FBN1 (E1297G) that is significantly associated with lower height in the Peruvian population. Each copy of the minor allele (frequency = 4.7%) reduces height by 2.2 cm (4.4 cm in homozygous individuals). This is the largest effect size known for a common height-associated variant. This variant shows strong evidence of positive selection within the Peruvian population and is significantly more frequent in Native American populations from coastal regions of Peru compared to populations from the Andes or the Amazon, suggesting that short stature in Peruvians is the result of adaptation to the coastal environment.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/brain_aging_in_major_depressive_disorder_results_from_the_enigma_major_depressive_disorder_working_group_56073512/">
          <h4 class="font-weight-bold">Brain Aging in Major Depressive Disorder Results from the ENIGMA Major Depressive Disorder working group, bioRxiv, 2019-02-26</h4>
          <p>Background Major depressive disorder (MDD) is associated with an increased risk of brain atrophy, aging-related diseases, and mortality. We examined potential advanced brain aging in MDD patients, and whether this process is associated with clinical characteristics in a large multi-center international dataset. Methods We performed a mega-analysis by pooling brain measures derived from T1-weighted MRI scans from 29 samples worldwide. Normative brain aging was estimated by predicting chronological age (10-75 years) from 7 subcortical volumes, 34 cortical thickness and 34 surface area, lateral ventricles and total intracranial volume measures separately in 1,147 male and 1,386 female controls from the ENIGMA MDD working group. The learned model parameters were applied to 1,089 male controls and 1,167 depressed males, and 1,326 female controls and 2,044 depressed females to obtain independent unbiased brain-based age predictions. The difference between predicted brain age and chronological age was calculated to indicate brain predicted age difference (brain-PAD). Findings On average, MDD patients showed a higher brain-PAD of &#43;0.90 (SE 0.21) years (Cohen&#39;s d=0.12, 95% CI 0.06-0.17) compared to controls. Relative to controls, first-episode and currently depressed patients showed higher brain-PAD (&#43;1.2 [0.3] years), and the largest effect was observed in those with late-onset depression (&#43;1.7 [0.7] years). In addition, higher brain-PAD was associated with higher self-reported depressive symptomatology (b=0.05, p=0.004). Interpretation This highly powered collaborative effort showed subtle patterns of abnormal structural brain aging in MDD. Substantial within-group variance and overlap between groups were observed. Longitudinal studies of MDD and somatic health outcomes are needed to further assess the predictive value of these brain-PAD estimates.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/interdependent_phenotypic_and_biogeographic_evolution_driven_by_biotic_interactions_56047585/">
          <h4 class="font-weight-bold">Interdependent Phenotypic and Biogeographic Evolution Driven by Biotic Interactions, bioRxiv, 2019-02-26</h4>
          <p>Biotic interactions are hypothesized to be one of the main processes shaping trait and biogeographic evolution during lineage diversification. Theoretical and empirical evidence suggests that species with similar ecological requirements either spatially exclude each other, by preventing the colonization of competitors or by driving coexisting populations to extinction, or show niche divergence when in sympatry. However, the extent and generality of the effect of interspecific competition in trait and biogeographic evolution has been limited by a dearth of appropriate process-generating models to directly test the effect of biotic interactions. Here, we formulate a phylogenetic parametric model that allows interdependence between trait and biogeographic evolution, thus enabling a direct test of central hypotheses on how biotic interactions shape these evolutionary processes. We adopt a Bayesian data augmentation approach to estimate the joint posterior distribution of trait histories, range histories, and co-evolutionary process parameters under this analytically intractable model. Through simulations, we show that our model is capable of distinguishing alternative scenarios of biotic interactions. We apply our model to the radiation of Darwin&#39;s finches---a classic example of adaptive divergence---and find support for in situ trait divergence in beak size, convergence in traits such as beak shape and tarsus length, and strong competitive exclusion throughout their evolutionary history. Our modeling framework opens new possibilities for testing more complex hypotheses about the processes underlying lineage diversification. More generally, it provides a robust probabilistic methodology to model correlated evolution of continuous and discrete characters.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/biorxiv/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/biorxiv/page/54/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/51/" class="page-link">51</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/52/" class="page-link">52</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/53/" class="page-link">53</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/54/" class="page-link">54</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/biorxiv/page/55/" class="page-link">55</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/56/" class="page-link">56</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/57/" class="page-link">57</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/58/" class="page-link">58</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/59/" class="page-link">59</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/biorxiv/page/56/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/biorxiv/page/164/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
