<!DOCTYPE html>
<html><head>
  
  <title>Reports | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/nuclei_multiplexing_with_barcoded_antibodies_for_single-nucleus_genomics_51602206/">
          <h4 class="font-weight-bold">Nuclei multiplexing with barcoded antibodies for single-nucleus genomics, bioRxiv, 2018-11-23</h4>
          <p>AbstractSingle-nucleus RNA-Seq (snRNA-seq) enables the interrogation of cellular states in complex tissues that are challenging to dissociate, including frozen clinical samples. This opens the way, in principle, to large studies, such as those required for human genetics, clinical trials, or precise cell atlases of large organs. However, such applications are currently limited by batch effects, sequential processing, and costs. To address these challenges, we present an approach for multiplexing snRNA-seq, using sample-barcoded antibodies against the nuclear pore complex to uniquely label nuclei from distinct samples. Comparing human brain cortex samples profiled in multiplex with or without hashing antibodies, we demonstrate that nucleus hashing does not significantly alter the recovered transcriptome profiles. We further developed demuxEM, a novel computational tool that robustly detects inter-sample nucleus multiplets and assigns singlets to their samples of origin by antibody barcodes, and validated its accuracy using gender-specific gene expression, species-mixing and natural genetic variation. Nucleus hashing significantly reduces cost per nucleus, recovering up to about 5 times as many single nuclei per microfluidc channel. Our approach provides a robust technique for diverse studies including tissue atlases of isogenic model organisms or from a single larger human organ, multiple biopsies or longitudinal samples of one donor, and large-scale perturbation screens.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/c._elegans_pathogenic_learning_confers_multigenerational_pathogen_avoidance_51528447/">
          <h4 class="font-weight-bold">C. elegans pathogenic learning confers multigenerational pathogen avoidance, bioRxiv, 2018-11-22</h4>
          <p>AbstractThe ability to pass on learned information to progeny could present an evolutionary advantage for many generations. While apparently evolutionarily conserved1–12, transgenerational epigenetic inheritance (TEI) is not well understood at the molecular or behavioral levels. Here we describe our discovery that C. elegans can pass on a learned pathogenic avoidance behavior to their progeny for several generations through epigenetic mechanisms. Although worms are initially attracted to the gram-negative bacteria P. aeruginosa (PA14)13, they can learn to avoid this pathogen13. We found that prolonged PA14 exposure results in transmission of avoidance behavior to progeny that have themselves never been exposed to PA14, and this behavior persists through the fourth generation. This form of transgenerational inheritance of bacterial avoidance is specific to pathogenic P. aeruginosa, requires physical contact and infection, and is distinct from CREB-dependent long-term associative memory and larval imprinting. The TGF-β ligand daf-7, whose expression increases in the ASJ upon initial exposure to PA1414, is highly expressed in the ASI neurons of progeny of trained mothers until the fourth generation, correlating with transgenerational avoidance behavior. Mutants of histone modifiers and small RNA mediators display defects in naïve PA14 attraction and aversive learning. By contrast, the germline-expressed PRG-1Piwi homolog15 is specifically required for transgenerational inheritance of avoidance behavior. Our results demonstrate a novel and natural paradigm of TEI that may optimize progeny decisions and subsequent survival in the face of changing environmental conditions.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/hunger_for_knowledge_how_the_irresistible_lure_of_curiosity_is_generated_in_the_brain_51561076/">
          <h4 class="font-weight-bold">Hunger for Knowledge How the Irresistible Lure of Curiosity is Generated in the Brain, bioRxiv, 2018-11-22</h4>
          <p>Introductory ParagraphCuriosity is often portrayed as a desirable feature of human faculty. For example, a meta-analysis revealed that curiosity predicts academic performance above and beyond intelligence1, corroborating findings that curiosity supported long-term consolidation of learning2,3. However, curiosity may come at a cost of strong seductive power that sometimes puts people in a harmful situation. Here, with a set of three behavioural and two neuroimaging experiments including novel stimuli that strongly trigger curiosity (i.e. magic tricks), we examined the psychological and neural mechanisms underlying the irresistible lure of curiosity. We consistently demonstrated that across different samples people were indeed willing to gamble to expose themselves to physical risks (i.e. electric shocks) in order to satisfy their curiosity for trivial knowledge that carries no apparent instrumental values. Also, underlying this seductive power of curiosity is its incentive salience properties, which share common neural mechanisms with extrinsic incentives (i.e. hunger for foods). In particular, the two independent fMRI experiments using different kinds of curiosity-stimulating stimuli found replicable results that acceptance (compared to rejection) of curiosityincentive-driven gambles was accompanied with an enhanced activity in the striatum.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_primer_on_deep_learning_in_genomics_51770010/">
          <h4 class="font-weight-bold">A primer on deep learning in genomics, Nature Genetics, 2018-11-21</h4>
          <p>Deep learning methods are a class of machine learning techniques capable of identifying highly complex patterns in large datasets. Here, we provide a perspective and primer on deep learning applications for genome analysis. We discuss successful applications in the fields of regulatory genomics, variant calling and pathogenicity scores. We include general guidance for how to effectively use deep learning methods as well as a practical guide to tools and resources. This primer is accompanied by an interactive online tutorial.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/nature%20genetics">nature genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/crispr-tsko_facilitates_efficient_cell_type-_tissue-_or_organ-specific_mutagenesis_in_arabidopsis_51476356/">
          <h4 class="font-weight-bold">CRISPR-TSKO facilitates efficient cell type-, tissue-, or organ-specific mutagenesis in Arabidopsis, bioRxiv, 2018-11-21</h4>
          <p>AbstractDetailed functional analyses of many fundamentally-important plant genes via conventional loss-of-function approaches are impeded by severe pleiotropic phenotypes. In particular, mutations in genes that are required for basic cellular functions andor reproduction often interfere with the generation of homozygous mutant plants, precluding further functional studies. To overcome this limitation, we devised a CRISPR-based tissue-specific knockout system, CRISPR-TSKO, enabling the generation of somatic mutations in particular plant cell types, tissues, and organs. In Arabidopsis, CRISPR-TSKO mutations in essential genes caused well-defined, localized phenotypes in the root cap, stomatal lineage, or entire lateral roots. The underlying modular cloning system allows for efficient selection, identification, and functional analysis of mutant lines directly in the first transgenic generation. The efficacy of CRISPR-TSKO opens new avenues to discover and analyze gene functions in spatial and temporal contexts of plant life while avoiding pleiotropic effects of system-wide loss of gene function.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_database_of_egg_size_and_shape_from_more_than_6700_insect_species_51435213/">
          <h4 class="font-weight-bold">A database of egg size and shape from more than 6,700 insect species, bioRxiv, 2018-11-20</h4>
          <p>1AbstractOffspring size is a fundamental trait in disparate biological fields of study. This trait can be measured as the size of plant seeds, animal eggs, or live young, and it influences ecological interactions, organism fitness, maternal investment, and embryonic development. Although multiple evolutionary processes have been predicted to drive the evolution of offspring size, the phylogenetic distribution of this trait remains poorly understood, due to the difficulty of reliably collecting and comparing offspring size data from many species. Here we present a database of 10,449 morphological descriptions of insect eggs, with records for 6,706 unique insect species and representatives from every extant hexapod order. The dataset includes eggs whose volumes span more than eight orders of magnitude. We created this database by partially automating the extraction of egg traits from the primary literature. In the process, we overcame challenges associated with large-scale phenotyping by designing and employing custom bioinformatic solutions to common problems. We matched the taxa in this database to the currently accepted scientific names in taxonomic and genetic databases, which will facilitate the use of this data for testing pressing evolutionary hypotheses in offspring size evolution.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/reports/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/reports/page/158/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/155/" class="page-link">155</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/156/" class="page-link">156</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/157/" class="page-link">157</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/158/" class="page-link">158</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/reports/page/159/" class="page-link">159</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/160/" class="page-link">160</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/161/" class="page-link">161</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/162/" class="page-link">162</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/163/" class="page-link">163</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/reports/page/160/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/reports/page/320/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
