<!DOCTYPE html>
<html><head>
  
  <title>2020 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.68.3" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.95d2483cd3c5d2bf47c1f36307a456787d60ad958c5efe44db9afc47012b1b05b8e1f64f6ac291650439322a6c38e86eb7cd6254255516d857c23f299fb0e666.js" integrity="sha512-ldJIPNPF0r9HwfNjB6RWeH1grZWMXv5E25r8RwErGwW44fZPasKRZQQ5MipsOOhut81iVCVVFthXwj8pn7DmZg=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/aesthetic_preference_for_art_emerges_from_a_weighted_integration_over_hierarchically_structured_visual_features_in_the_brain_75739737/">
          <h4 class="font-weight-bold">Aesthetic preference for art emerges from a weighted integration over hierarchically structured visual features in the brain, bioRxiv, 2020-02-10</h4>
          <p>AbstractIt is an open question whether preferences for visual art can be lawfully predicted from the basic constituent elements of a visual image. Moreover, little is known about how such preferences are actually constructed in the brain. Here we developed and tested a computational framework to gain an understanding of how the human brain constructs aesthetic value. We show that it is possible to explain human preferences for a piece of art based on an analysis of features present in the image. This was achieved by analyzing the visual properties of drawings and photographs by multiple means, ranging from image statistics extracted by computer vision tools, subjective human ratings about attributes, to a deep convolutional neural network. Crucially, it is possible to predict subjective value ratings not only within but also across individuals, speaking to the possibility that much of the variance in human visual preference is shared across individuals. Neuroimaging data revealed that preference computations occur in the brain by means of a graded hierarchical representation of lower and higher level features in the visual system. These features are in turn integrated to compute an overall subjective preference in the parietal and prefrontal cortex. Our findings suggest that rather than being idiosyncratic, human preferences for art can be explained at least in part as a product of a systematic neural integration over underlying visual features of an image. This work not only advances our understanding of the brain-wide computations underlying value construction but also brings new mechanistic insights to the study of visual aesthetics and art appreciation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/boss-runs_a_flexible_and_practical_dynamic_read_sampling_framework_for_nanopore_sequencing_75419138/">
          <h4 class="font-weight-bold">BOSS-RUNS a flexible and practical dynamic read sampling framework for nanopore sequencing, bioRxiv, 2020-02-08</h4>
          <p>AbstractReal-time selective sequencing of individual DNA fragments, or ‘Read Until’, allows the focusing of Oxford Nanopore Technology sequencing on pre-selected genomic regions. This can lead to large improvements in DNA sequencing performance in many scenarios where only part of the DNA content of a sample is of interest. This approach is based on the idea of deciding whether to sequence a fragment completely after having sequenced only a small initial part of it. If, based on this small part, the fragment is not deemed of (sufficient) interest it is rejected and sequencing is continued on a new fragment. To date, only simple decision strategies based on location within a genome have been proposed to determine what fragments are of interest. We present a new mathematical model and algorithm for the real-time assessment of the value of prospective fragments. Our decision framework is based not only on which genomic regions are a priori interesting, but also on which fragments have so far been sequenced, and so on the current information available regarding the genome being sequenced. As such, our strategy can adapt dynamically during each run, focusing sequencing efforts in areas of highest uncertainty (typically areas currently low coverage). We show that our approach can lead to considerable savings of time and materials, providing high-confidence genome reconstruction sooner than a standard sequencing run, and resulting in more homogeneous coverage across the genome, even when entire genomes are of interest.Author SummaryAn existing technique called ‘Read Until’ allows selective sequencing of DNA fragments with an Oxford Nanopore Technology (ONT) sequencer. With Read Until it is possible to enrich coverage of areas of interest within a sequenced genome. We propose a new use of this technique combining a mathematical model of read utility and an algorithm to select an optimal dynamic decision strategy (i.e. one that can be updated in real time, and so react to the data generated so far in an experiment), we show that it possible to improve the efficiency of a sequencing run by focusing effort on areas of highest uncertainty.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/clonal_evolution_of_acute_myeloid_leukemia_revealed_by_high-throughput_single-cell_genomics_75412873/">
          <h4 class="font-weight-bold">Clonal Evolution of Acute Myeloid Leukemia Revealed by High-Throughput Single-Cell Genomics, bioRxiv, 2020-02-08</h4>
          <p>SummaryOne of the pervasive features of cancer is the diversity of mutations found in malignant cells within the same tumor; a phenomenon called clonal diversity or intratumor heterogeneity. Clonal diversity allows tumors to adapt to the selective pressure of treatment and likely contributes to the development of treatment resistance and cancer recurrence. Thus, the ability to precisely delineate the clonal substructure of a tumor, including the evolutionary history of its development and the co-occurrence of its mutations, is necessary to understand and overcome treatment resistance. However, DNA sequencing of bulk tumor samples cannot accurately resolve complex clonal architectures. Here, we performed high-throughput single-cell DNA sequencing to quantitatively assess the clonal architecture of acute myeloid leukemia (AML). We sequenced a total of 556,951 cells from 77 patients with AML for 19 genes known to be recurrently mutated in AML. The data revealed clonal relationship among AML driver mutations and identified mutations that often co-occurred (e.g., NPM1FLT3-ITD, DNMT3ANPM1, SRSF2IDH2, and WT1FLT3-ITD) and those that were mutually exclusive (e.g., NRASKRAS, FLT3-D835ITD, and IDH1IDH2) at single-cell resolution. Reconstruction of the tumor phylogeny uncovered history of tumor development that is characterized by linear and branching clonal evolution patterns with latter involving functional convergence of separately evolved clones. Analysis of longitudinal samples revealed remodeling of clonal architecture in response to therapeutic pressure that is driven by clonal selection. Furthermore, in this AML cohort, higher clonal diversity (≥4 subclones) was associated with significantly worse overall survival. These data portray clonal relationship, architecture, and evolution of AML driver genes with unprecedented resolution, and illuminate the role of clonal diversity in therapeutic resistance, relapse and clinical outcome in AML.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/cancer-biology">cancer-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/direct_imaging_of_liquid_domains_in_membranes_by_cryo_electron_tomography_75079476/">
          <h4 class="font-weight-bold">Direct Imaging of Liquid Domains in Membranes by Cryo Electron Tomography, bioRxiv, 2020-02-06</h4>
          <p>ABSTRACTImages of micron-scale domains in lipid bilayers have provided the gold standard of model-free evidence to understand the domains’ shapes, sizes, and distributions. Corresponding techniques to directly and quantitatively assess smaller (nanoscale and submicron) liquid domains have been lacking, leading to an inability to answer key questions. For example, researchers commonly seek to correlate activities of membrane proteins with attributes of the domains in which they reside; doing so hinges on identification and characterization of membrane domains. Although some features of membrane domains can be probed by indirect methods, these methods are often constrained by the limitation that data must be analyzed in the context of models that require multiple assumptions or parameters. Here, we address this challenge by developing and testing two new methods of identifying submicron domains in biomimetic membranes. Both methods leverage cryo-electron tomograms of ternary membranes under native solution conditions. The first method is optimized for probe-free applications domains are directly distinguished from the surrounding membrane by their thickness. This technique measures area fractions of domains with quantitative accuracy, in excellent agreement with known phase diagrams. The second method is optimized for applications in which a single label is deployed for imaging membranes by both high-resolution cryo-electron tomography and diffraction-limited optical microscopy. For this method, we test a panel of probes, find that a trimeric mCherry label performs best, and specify criteria for developing future high-performance, dual-use probes. These developments have led to the first direct and quantitative imaging of submicron membrane domains under native conditions.SIGNIFICANCE STATEMENTFluorescence micrographs that capture the sizes, shapes, and distributions of liquid domains in model membranes have provided high standards of evidence to prove (and disprove) theories of how micron-scale domains form and grow. Corresponding theories about smaller domains have remained untested, partly because experimental methods of identifying submicron domains in vesicles under native solvent conditions have not been available. Here we introduce two such methods. Both leverage cryo-electron tomography to observe membrane features far smaller than the diffraction limit of light. The first method is probe-free and identifies differences in thicknesses between liquid domains and their surrounding membranes. The second method identifies membrane regions labeled by an electron-dense, fluorescent protein, which enables direct comparison of fluorescence micrographs with cryo-electron tomograms.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/direct_label-free_imaging_of_nanodomains_in_biomimetic_and_biological_membranes_by_cryogenic_electron_microscopy_75079451/">
          <h4 class="font-weight-bold">Direct label-free imaging of nanodomains in biomimetic and biological membranes by cryogenic electron microscopy, bioRxiv, 2020-02-06</h4>
          <p>ABSTRACTThe nanoscale organization of biological membranes into structurally and compositionally distinct lateral domains is believed to be central to membrane function. The nature of this organization has remained elusive due to a lack of methods to directly probe nanoscopic membrane features. We show here that cryogenic electron microscopy (cryoEM) can be used to directly image coexisting nanoscopic domains in synthetic and bio-derived membranes without extrinsic probes. Analyzing a series of single-component liposomes composed of synthetic lipids of varying lengths, we demonstrate that cryoEM can distinguish bilayer thickness differences as small as 0.5 Å, comparable to the resolution of small-angle scattering methods. Simulated images from computational models reveal that features in cryoEM images result from a complex interplay between the atomic distribution normal to the plane of the bilayer and imaging parameters. Simulations of phase separated bilayers were used to predict two sources of contrast between coexisting ordered and disordered phases within a single liposome, namely differences in membrane thickness and molecular density. We observe both sources of contrast in biomimetic membranes composed of saturated lipids, unsaturated lipids, and cholesterol. When extended to isolated mammalian plasma membranes, these methods reveal similar nanoscale lateral heterogeneities. The methods reported here for direct, probe-free imaging of nanodomains in unperturbed membranes open new avenues for investigation of nanoscopic membrane organization.SIGNIFICANCEWe have used cryoEM to achieve direct, probe-free imaging of lateral domains in biomimetic lipid membranes under native conditions and to characterize differences in their structures. First, measurements of membrane thickness in laterally uniform single-component membranes show that cryoEM is capable of sub-angstrom resolution of interleaflet membrane thickness. All-atom simulations are used to predict the cryo-EM appearance of submicron domains in vesicles with coexisting liquid domains and these are quantitatively validated by direct imaging of phase separated membranes. We then extend this approach to observe nanoscopic domains in isolated cellular membranes, comprising the first direct imaging of nanodomains in biomembranes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/predicting_commercially_available_antiviral_drugs_that_may_act_on_the_novel_coronavirus_2019-ncov_wuhan_china_through_a_drug-target_interaction_deep_learning_model_75023403/">
          <h4 class="font-weight-bold">Predicting commercially available antiviral drugs that may act on the novel coronavirus (2019-nCoV), Wuhan, China through a drug-target interaction deep learning model, bioRxiv, 2020-02-03</h4>
          <p>AbstractThe infection of a novel coronavirus found in Wuhan of China (2019-nCoV) is rapidly spreading, and the incidence rate is increasing worldwide. Due to the lack of effective treatment options for 2019-nCoV, various strategies are being tested in China, including drug repurposing. In this study, we used our pretrained deep learning-based drug-target interaction model called Molecule Transformer-Drug Target Interaction (MT-DTI) to identify commercially available drugs that could act on viral proteins of 2019-nCoV. The result showed that atazanavir, an antiretroviral medication used to treat and prevent the human immunodeficiency virus (HIV), is the best chemical compound, showing a inhibitory potency with Kd of 94.94 nM against the 2019-nCoV 3C-like proteinase, followed by efavirenz (199.17 nM), ritonavir (204.05 nM), and dolutegravir (336.91 nM). Interestingly, lopinavir, ritonavir, and darunavir are all designed to target viral proteinases. However, in our prediction, they may also bind to the replication complex components of 2019-nCoV with an inhibitory potency with Kd &amp;lt; 1000 nM. In addition, we also found that several antiviral agents, such as Kaletra, could be used for the treatment of 2019-nCoV, although there is no real-world evidence supporting the prediction. Overall, we suggest that the list of antiviral drugs identified by the MT-DTI model should be considered, when establishing effective treatment strategies for 2019-nCoV.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2020">2020</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        

        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/2020/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2020/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2020/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2020/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2020/page/5/" class="page-link">5</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2020/page/2/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2020/page/12/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
