<!DOCTYPE html>
<html><head>
  
  <title>microbiology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.68.3" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.95d2483cd3c5d2bf47c1f36307a456787d60ad958c5efe44db9afc47012b1b05b8e1f64f6ac291650439322a6c38e86eb7cd6254255516d857c23f299fb0e666.js" integrity="sha512-ldJIPNPF0r9HwfNjB6RWeH1grZWMXv5E25r8RwErGwW44fZPasKRZQQ5MipsOOhut81iVCVVFthXwj8pn7DmZg=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/membrane-bounded_nucleoid_discovered_in_a_cultivated_bacterium_of_the_candidate_phylum_atribacteria_64732123/">
          <h4 class="font-weight-bold">Membrane-bounded nucleoid discovered in a cultivated bacterium of the candidate phylum ‘Atribacteria’, bioRxiv, 2019-08-08</h4>
          <p>AbstractA key feature that differentiates prokaryotes from eukaryotes is the absence of an intracellular membrane surrounding the chromosomal DNA. Here, we report isolation of an anaerobic bacterium that possesses an additional intracytoplasmic membrane surrounding a nucleoid, affiliates with the yet-to-be-cultivated ubiquitous phylum ‘Ca. Atribacteria’, and possesses unique genomic features likely associated with organization of complex cellular structure. Exploration of the uncharted microorganism overturned the prevailing dogma of prokaryotic cell structure.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/global_genetic_cartography_of_urban_metagenomes_and_anti-microbial_resistance_64599287/">
          <h4 class="font-weight-bold">Global Genetic Cartography of Urban Metagenomes and Anti-Microbial Resistance, bioRxiv, 2019-08-06</h4>
          <p>AbstractAlthough studies have shown that urban environments and mass-transit systems have distinct genetic profiles, there are no systematic studies of these dense, humanmicrobial ecosystems around the world. To address this gap in knowledge, we created a global metagenomic and antimicrobial resistance (AMR) atlas of urban mass transit systems from 58 cities, spanning 3,741 samples and 4,424 taxonomically-defined microorganisms collected for from 2015-2017. The map provides annotated, geospatial details about microbial strains, functional genetics, antimicrobial resistance, and novel genetic elements, including 10,928 novel predicted viral species. Urban microbiomes often resemble human commensal microbiomes from the skin and airways, but also contain a consistent “core” of 61 species which are predominantly not human commensal species. Conversely, samples may be accurately (91.4%) classified to their city-of-origin using a linear support vector machine over taxa. These data also show that AMR density across cities varies by several orders of magnitude, including many AMRs present on plasmids with specific cosmopolitan distributions. Together, these results constitute a high-resolution global metagenomic atlas, which enables the discovery of new genetic components of the built human environment, highlights potential forensic applications, and provides an essential first draft of the global AMR burden of the world’s cities.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/isolation_of_an_archaeon_at_the_prokaryote-eukaryote_interface_64641879/">
          <h4 class="font-weight-bold">Isolation of an archaeon at the prokaryote-eukaryote interface, bioRxiv, 2019-08-06</h4>
          <p>AbstractThe origin of eukaryotes remains enigmatic. Current data suggests that eukaryotes may have risen from an archaeal lineage known as “Asgard archaea”. Despite the eukaryote-like genomic features found in these archaea, the evolutionary transition from archaea to eukaryotes remains unclear due to the lack of cultured representatives and corresponding physiological insight. Here we report the decade-long isolation of a Lokiarchaeota-related Asgard archaeon from deep marine sediment. The archaeon, “Candidatus Prometheoarchaeum syntrophicum strain MK-D1”, is an anaerobic, extremely slow-growing, small cocci (∼550 nm), that degrades amino acids through syntrophy. Although eukaryote-like intracellular complexities have been proposed for Asgard archaea, the isolate has no visible organella-like structure. Ca. P. syntrophicum instead displays morphological complexity – unique long, and often, branching protrusions. Based on cultivation and genomics, we propose an “Entangle-Engulf-Enslave (E3) model” for eukaryogenesis through archaea-alphaproteobacteria symbiosis mediated by the physical complexities and metabolic dependency of the hosting archaeon.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/non-antibiotic_pharmaceuticals_can_enhance_the_spread_of_antibiotic_resistance_via_conjugation_64599273/">
          <h4 class="font-weight-bold">Non-antibiotic pharmaceuticals can enhance the spread of antibiotic resistance via conjugation, bioRxiv, 2019-08-06</h4>
          <p>AbstractAntibiotic resistance is a global threat for public health. It is widely acknowledged that antibiotics at sub-inhibitory concentrations are important in disseminating antibiotic resistance via horizontal gene transfer. While there is high use of non-antibiotic human-targeted pharmaceuticals in our societies, the potential contribution of these on the spread of antibiotic resistance has been overlooked so far. Here, we report that commonly consumed non-antibiotic pharmaceuticals, including nonsteroidal anti-inflammatories (ibuprofen, naproxen, diclofenac), a lipid-lowering drug (gemfibrozil), and a β-blocker (propanolol), at clinically and environmentally relevant concentrations, significantly accelerated the conjugation of plasmid-borne antibiotic resistance genes. We looked at the response to these drugs by the bacteria involved in the gene transfer through various analyses that included monitoring reactive oxygen species (ROS) and cell membrane permeability by flow cytometry, cell arrangement, and whole-genome RNA and protein sequencing. We found the enhanced conjugation correlated well with increased production of ROS and cell membrane permeability. We also detected closer cell-to-cell contact and upregulated conjugal genes. Additionally, these non-antibiotic pharmaceuticals caused the bacteria to have responses similar to those detected when exposed to antibiotics, such as inducing the SOS response, and enhancing efflux pumps. The findings advance our understanding of the bacterial transfer of antibiotic resistance genes, and importantly emphasize concerns of non-antibiotic human-targeted pharmaceuticals for enhancing the spread of antibiotic resistance.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/unlinked_rrna_genes_are_widespread_among_bacteria_and_archaea_63648382/">
          <h4 class="font-weight-bold">Unlinked rRNA genes are widespread among Bacteria and Archaea, bioRxiv, 2019-07-17</h4>
          <p>AbstractRibosomes are essential to cellular life and the genes for their RNA components are the most conserved and transcribed genes in Bacteria and Archaea. These ribosomal rRNA genes are typically organized into a single operon, an arrangement that is thought to facilitate gene regulation. In reality, some Bacteria and Archaea do not share this canonical rRNA arrangement-their 16S and 23S rRNA genes are not co-located, but are instead separated across the genome and referred to as “unlinked”. This rearrangement has previously been treated as a rare exception or a byproduct of genome degradation in obligate intracellular bacteria. Here, we leverage complete genome and long-read metagenomic data to show that unlinked 16S and 23S rRNA genes are much more common than previously thought. Unlinked rRNA genes occur in many phyla, most significantly within Deinococcus-Thermus, Chloroflexi, Planctomycetes, and Euryarchaeota, and occur in differential frequencies across natural environments. We found that up to 41% of the taxa in soil, including dominant taxa, had unlinked rRNA genes, in contrast to the human gut, where all sequenced rRNA genes were linked. The frequency of unlinked rRNA genes may reflect meaningful life history traits, as they tend to be associated with a mix of slow-growing free-living species and obligatory intracellular species. Unlinked rRNA genes are also associated with changes in RNA metabolism, notably the loss of RNaseIII. We propose that unlinked rRNA genes may confer selective advantages in some environments, though the specific nature of these advantages remains undetermined and worthy of further investigation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/assembly_of_hundreds_of_novel_bacterial_genomes_from_the_chicken_caecum_63470531/">
          <h4 class="font-weight-bold">Assembly of hundreds of novel bacterial genomes from the chicken caecum, bioRxiv, 2019-07-12</h4>
          <p>AbstractChickens are a highly important source of protein for a large proportion of the human population. The caecal microbiota plays a crucial role in chicken nutrition through the production of short chain fatty acids, nitrogen recycling and amino acid production. In this study we sequenced DNA from caecal contents samples taken from 24 chickens belonging to either a fast or slower growing breed consuming either a vegetable-only diet or a diet containing fish meal. We utilised 1.6T of Illumina data to construct 469 draft metagenome-assembled bacterial genomes, including 460 novel strains, 283 novel species and 42 novel genera. We compared our genomes to data from eight EU countries and show that these genomes are abundant within European chicken flocks. We also compared the abundance of our genomes, and the carbohydrate active enzymes they produce, between our chicken groups and demonstrate that there are both breed- and diet-specific microbiomes, as well as an overlapping core microbiome. This data will form the basis for future studies examining the composition and function of the chicken caecal microbiota.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/2/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/2/" class="page-link">2</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/microbiology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/7/" class="page-link">7</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/4/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/16/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
