<!DOCTYPE html>
<html><head>
  
  <title>Evolutionary Biology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/antibiotic_production_in_streptomyces_is_organized_by_a_division_of_labour_through_terminal_genomic_differentiation_56033162/">
          <h4 class="font-weight-bold">Antibiotic production in Streptomyces is organized by a division of labour through terminal genomic differentiation, bioRxiv, 2019-02-25</h4>
          <p>AbstractOne of the hallmark behaviors of social groups is division of labour, where different group members become specialized to carry out complementary tasks. By dividing labour, cooperative groups of individuals increase their efficiency, thereby raising group fitness even if these specialized behaviors reduce the fitness of individual group members. Here we provide evidence that antibiotic production in colonies of the multicellular bacterium Streptomyces coelicolor is coordinated by a division of labour. We show that S. coelicolor colonies are genetically heterogeneous due to massive amplifications and deletions to the chromosome. Cells with gross chromosomal changes produce an increased diversity of secondary metabolites and secrete significantly more antibiotics; however, these changes come at the cost of dramatically reduced individual fitness, providing direct evidence for a trade-off between secondary metabolite production and fitness. Finally, we show that colonies containing mixtures of mutant strains and their parents produce significantly more antibiotics, while colony-wide spore production remains unchanged. Our work demonstrates that by generating mutants that are specialized to hyper-produce antibiotics, streptomycetes reduce the colony-wide fitness costs of secreted secondary metabolites while maximizing the yield and diversity of these products.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/antibiotic_production_is_organized_by_a_division_of_labour_in_streptomyces_56033162/">
          <h4 class="font-weight-bold">Antibiotic production is organized by a division of labour in Streptomyces, bioRxiv, 2019-02-25</h4>
          <p>AbstractOne of the hallmark behaviors of social groups is division of labour, where different group members become specialized to carry out complementary tasks. By dividing labour, cooperative groups of individuals increase their efficiency, thereby raising group fitness even if these specialized behaviors reduce the fitness of individual group members. Here we provide evidence that antibiotic production in colonies of the multicellular bacterium Streptomyces coelicolor is coordinated by a division of labour. We show that S. coelicolor colonies are genetically heterogenous due to massive amplifications and deletions to the chromosome. Cells with gross chromosomal changes produce an increased diversity of secondary metabolites and secrete significantly more antibiotics; however, these changes come at the cost of dramatically reduced individual fitness, providing direct evidence for a trade-off between secondary metabolite production and fitness. Finally, we show that colonies containing mixtures of mutant strains and their parents produce significantly more antibiotics, while colony-wide spore production remains unchanged. Our work demonstrates that by generating mutants that are specialized to hyper-produce antibiotics, streptomycetes reduce the colony-wide fitness costs of secreted secondary metabolites while maximizing the yield and diversity of these products.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_linked_selection_signature_of_rapid_adaptation_in_temporal_genomic_data_55999683/">
          <h4 class="font-weight-bold">The Linked Selection Signature of Rapid Adaptation in Temporal Genomic Data, bioRxiv, 2019-02-25</h4>
          <p>Populations can adapt over short, ecological timescales via standing genetic variation. Genomic data collected over tens of generations in both natural and lab populations is increasingly used to find selected loci underpinning such rapid adaptation. Although selection on large effect loci may be detectable in such data, often the fitness differences between individuals have a polygenic architecture, such that selection at any one locus leads to allele frequency changes that are too subtle to distinguish from genetic drift. However, one promising signal comes from the fact that selection on polygenic traits leads to heritable fitness backgrounds that neutral alleles can become stochastically associated with. These associations perturb neutral allele frequency trajectories, creating autocovariance across generations that can be directly measured from temporal genomic data. We develop theory that predicts the magnitude of these temporal autocovariances, showing that it is determined by the level of additive genetic variation, recombination, and linkage disequilibria in a region. Furthermore, by using analytic expressions for the temporal variances and autocovariances in allele frequency, we demonstrate one can estimate the additive genetic variation for fitness and the drift-effective population size from temporal genomic data. Finally, we also show how the proportion of total variation in allele frequency change due to linked selection can be estimated from temporal data. Temporal genomic data offers strong opportunities to identify the role linked selection has on genome-wide diversity over short timescales, and can help bridge population genetic and quantitative genetic studies of adaptation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_bayesian_approach_for_estimating_branch-specific_speciation_and_extinction_rates_55822262/">
          <h4 class="font-weight-bold">A Bayesian Approach for Estimating Branch-Specific Speciation and Extinction Rates, bioRxiv, 2019-02-21</h4>
          <p>Species richness varies considerably among the tree of life which can only be explained by heterogeneous rates of diversification (speciation and extinction). Previous approaches use phylogenetic trees to estimate branch-specific diversification rates. However, all previous approaches disregard diversification-rate shifts on extinct lineages although 99% of species that ever existed are now extinct. Here we describe a lineage-specific birth-death-shift process where lineages, both extant and extinct, may have heterogeneous rates of diversification. To facilitate probability computation we discretize the base distribution on speciation and extinction rates into k rate categories. The fixed number of rate categories allows us to extend the theory of state-dependent speciation and extinction models (e.g., BiSSE and MuSSE) to compute the probability of an observed phylogeny given the set of speciation and extinction rates. To estimate branch-specific diversification rates, we develop two independent and theoretically equivalent approaches numerical integration with stochastic character mapping and data-augmentation with reversible-jump Markov chain Monte Carlo sampling. We validate the implementation of the two approaches in RevBayes using simulated data and an empirical example study of primates. In the empirical example, we show that estimates of the number of diversification-rate shifts are, unsurprisingly, very sensitive to the choice of prior distribution. Instead, branch-specific diversification rate estimates are less sensitive to the assumed prior distribution on the number of diversification-rate shifts and consistently infer an increased rate of diversification for Old World Monkeys. Additionally, we observe that as few as 10 diversification-rate categories are sufficient to approximate a continuous base distribution on diversification rates. In conclusion, our implementation of the lineage-specific birth-death-shift model in RevBayes provides biologists with a method to estimate branch-specific diversification rates under a mathematically consistent model.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_distribution_of_fitness_effects_among_synonymous_mutations_in_a_gene_under_selection_55697006/">
          <h4 class="font-weight-bold">The distribution of fitness effects among synonymous mutations in a gene under selection, bioRxiv, 2019-02-19</h4>
          <p>AbstractThe fitness effects of synonymous mutations, nucleotide changes that do not alter the encoded amino acid, have often been assumed to be neutral, but a growing body of evidence suggests otherwise. We used site-directed mutagenesis coupled with direct measures of competitive fitness to estimate the distribution of fitness effects among synonymous mutations for a gene under selection. Synonymous mutations had highly variable fitness effects, both deleterious and beneficial, resembling those of nonsynonymous mutations in the same gene. This variation in fitness was underlain by changes in transcription linked to the creation of internal promoter sites. A positive correlation between fitness and the presence of synonymous substitutions across a phylogeny of related Pseudomonads suggests these mutations may be common in nature. Taken together, our results provide the most compelling evidence to date that synonymous mutations with non-neutral fitness effects may in fact be commonplace.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/extensive_loss_of_cell_cycle_and_dna_repair_genes_in_an_ancient_lineage_of_bipolar_budding_yeasts_55315095/">
          <h4 class="font-weight-bold">Extensive loss of cell cycle and DNA repair genes in an ancient lineage of bipolar budding yeasts, bioRxiv, 2019-02-12</h4>
          <p>Cell cycle checkpoints and DNA repair processes protect organisms from potentially lethal mutational damage. Compared to other budding yeasts in the subphylum Saccharomycotina, we noticed that a lineage in the genus Hanseniaspora exhibited very high evolutionary rates, low GC content, small genome sizes, and lower gene numbers. To better understand Hanseniaspora evolution, we analyzed 25 genomes, including 11 newly sequenced, representing 18  21 known species in the genus. Our phylogenomic analyses identify two Hanseniaspora lineages, the fast-evolving lineage (FEL), which began diversifying ~87 million years ago (mya), and the slow-evolving lineage (SEL), which began diversifying ~54 mya. Remarkably, both lineages lost genes associated with the cell cycle and genome integrity, but these losses were greater in the FEL. For example, all species lost the cell cycle regulator WHI5, and the FEL lost components of the spindle checkpoint pathway (e.g., MAD1, MAD2) and DNA damage checkpoint pathway (e.g., MEC3, RAD9). Similarly, both lineages lost genes involved in DNA repair pathways, including the DNA glycosylase gene MAG1, which is part of the base excision repair pathway, and the DNA photolyase gene PHR1, which is involved in pyrimidine dimer repair. Strikingly, the FEL lost 33 additional genes, including polymerases (i.e., POL4 and POL32) and telomere-associated genes (e.g., RIF1, RFA3, CDC13, PBP2). Echoing these losses, molecular evolutionary analyses reveal that, compared to the SEL, the FEL stem lineage underwent a burst of accelerated evolution, which resulted in greater mutational loads, homopolymer instabilities, and higher fractions of mutations associated with the common endogenously damaged base, 8-oxoguanine. We conclude that Hanseniaspora is an ancient lineage that has diversified and thrived, despite lacking many otherwise highly conserved cell cycle and genome integrity genes and pathways, and may represent a novel system for studying cellular life without them.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/evolutionary-biology/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/evolutionary-biology/page/7/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../../tags/evolutionary-biology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/evolutionary-biology/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/evolutionary-biology/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/evolutionary-biology/page/7/" class="page-link">7</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/evolutionary-biology/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/evolutionary-biology/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/evolutionary-biology/page/10/" class="page-link">10</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/evolutionary-biology/page/11/" class="page-link">11</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/evolutionary-biology/page/12/" class="page-link">12</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/evolutionary-biology/page/9/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/evolutionary-biology/page/20/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
