<!DOCTYPE html>
<html><head>
  
  <title>Evolutionary Biology | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/identifying_loci_under_positive_selection_in_complex_population_histories_50269396/">
          <h4 class="font-weight-bold">Identifying loci under positive selection in complex population histories, bioRxiv, 2018-10-26</h4>
          <p>AbstractDetailed modeling of a species’ history is of prime importance for understanding how natural selection operates over time. Most methods designed to detect positive selection along sequenced genomes, however, use simplified representations of past histories as null models of genetic drift. Here, we present the first method that can detect signatures of strong local adaptation across the genome using arbitrarily complex admixture graphs, which are typically used to describe the history of past divergence and admixture events among any number of populations. The method—called Graph-aware Retrieval of Selective Sweeps (GRoSS)—has good power to detect loci in the genome with strong evidence for past selective sweeps and can also identify which branch of the graph was most affected by the sweep. As evidence of its utility, we apply the method to bovine, codfish and human population genomic data containing multiple population panels related in complex ways. We find new candidate genes for important adaptive functions, including immunity and metabolism in under-studied human populations, as well as muscle mass, milk production and tameness in specific bovine breeds. We are also able to pinpoint the emergence of large regions of differentiation due to inversions in the history of Atlantic codfish.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/inference_of_recombination_maps_from_a_single_pair_of_genomes_and_its_application_to_archaic_samples_50219053/">
          <h4 class="font-weight-bold">Inference of recombination maps from a single pair of genomes and its application to archaic samples, bioRxiv, 2018-10-25</h4>
          <p>ABSTRACTUnderstanding the causes and consequences of recombination rate evolution is a fundamental goal in genetics that requires recombination maps from across the tree of life. Since statistical inference of recombination maps typically depends on large samples, reaching out studies to non-model organisms requires alternative tools. Here we extend the sequentially Markovian coalescent model to jointly infer demography and the variation in recombination along a pair of genomes. Using extensive simulations and sequence data from humans, fruit-flies and a fungal pathogen, we demonstrate that iSMC accurately infers recombination maps under a wide range of scenarios – remarkably, even from a single pair of unphased genomes. We exploit this possibility and reconstruct the recombination maps of archaic hominids. We report that the evolution of the recombination landscape follows the established phylogeny of Neandertals, Denisovans and modern human populations, as expected if the genomic distribution of crossovers in hominids is largely neutral.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/latent_developmental_potential_to_form_limb-like_skeletal_structures_in_zebrafish_50117443/">
          <h4 class="font-weight-bold">Latent developmental potential to form limb-like skeletal structures in zebrafish, bioRxiv, 2018-10-23</h4>
          <p>AbstractThe evolution of fins into limbs was a key transition in vertebrate history. A hallmark of this transition is the addition of multiple long bones to the proximal-distal axis of paired appendages. Whereas limb skeletons are often elaborate and diverse, teleost pectoral fins retain a simple endoskeleton. Fins and limbs share many core developmental processes, but how these programs were reshaped to produce limbs from fins during evolution remains enigmatic. Here we identify zebrafish mutants that form supernumerary long bones along the proximal-distal axis of pectoral fins with limb-like patterning. These new skeletal elements are integrated into the fin, as they are connected to the musculature, form joints, and articulate with neighboring bones. This phenotype is caused by activating mutations in previously unrecognized regulators of appendage development, vav2 and waslb, which we show function in a common pathway. We find that this pathway functions in appendage development across vertebrates, and loss of Wasl in developing limbs results in patterning defects identical to those seen in Hoxall knockout mice. Concordantly, formation of supernumerary fin long bones requires the function of hoxall paralogs, indicating developmental homology with the forearm and the existence of a latent functional Hox code patterning the fin endoskeleton. Our findings reveal an inherent limb-like patterning ability in fins that can be activated by simple genetic perturbation, resulting in the elaboration of the endoskeleton.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/polygenic_adaptation_from_sweeps_to_subtle_frequency_shifts_50132793/">
          <h4 class="font-weight-bold">Polygenic Adaptation From sweeps to subtle frequency shifts, bioRxiv, 2018-10-23</h4>
          <p>Evolutionary theory has produced two conflicting paradigms for the adaptation of a polygenic trait. While population genetics views adaptation as a sequence of selective sweeps at single loci underlying the trait, quantitative genetics posits a collective response, where phenotypic adaptation results from subtle allele frequency shifts at many loci. Yet, a synthesis of these views is largely missing and the population genetic factors that favor each scenario are not well understood. Here, we study the architecture of adaptation of a binary polygenic trait (such as resistance) with negative epistasis among the loci of its basis. The genetic structure of this trait allows for a full range of potential architectures of adaptation, ranging from sweeps to small frequency shifts. By combining computer simulations and a newly devised analytical framework based on Yule branching processes, we gain a detailed understanding of the adaptation dynamics for this trait. Our key analytical result is an expression for the joint distribution of mutant alleles at the end of the adaptive phase. This distribution characterizes the polygenic pattern of adaptation at the underlying genotype when phenotypic adaptation has been accomplished. We find that a single compound parameter, the population-scaled background mutation rate Θbg, explains the main differences among these patterns. For a focal locus, Θbg measures the mutation rate at all redundant loci in its genetic background that offer alternative ways for adaptation. For adaptation starting from mutation-selection-drift balance, we observe different patterns in three parameter regions. Adaptation proceeds by sweeps for small Θbg ≲0.1, while small polygenic allele frequency shifts require large Θbg ≳100. In the large intermediate regime, we observe a heterogeneous pattern of partial sweeps at several interacting loci.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/bridging_the_divide_bacteria_synthesizing_archaeal_membrane_lipids_49944733/">
          <h4 class="font-weight-bold">Bridging the divide bacteria synthesizing archaeal membrane lipids, bioRxiv, 2018-10-19</h4>
          <p>Archaea synthesize membranes of isoprenoid lipids that are ether-linked to glycerol, while BacteriaEukarya produce membranes consisting of ester-bound fatty acids. This dichotomy in membrane lipid composition or ‘lipid divide’ is believed to have arisen after the Last Universal Common Ancestor (LUCA). A leading hypothesis is that LUCA possessed a ‘mixed heterochiral archaealbacterial membrane’, however no natural microbial representatives supporting this scenario have been shown to exist today. Here, we demonstrate that bacteria of the Fibrobacteres-Chlorobi-Bacteroidetes (FCB) group superphylum and related candidate phyla encode a complete pathway for archaeal membrane lipid biosynthesis in addition to the bacterial fatty acid membrane pathway. Key genes were expressed in the environment and their recombinant expression in E. coli resulted in the formation of a ‘mixed archaealbacterial membrane’. Our results support the existence of ‘mixed membranes’ in natural environments and their stability over large evolutionary timescales, thereby bridging a once-thought fundamental divide in biology.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/diversification_and_collapse_of_a_telomere_elongation_mechanism_49862531/">
          <h4 class="font-weight-bold">Diversification and collapse of a telomere elongation mechanism, bioRxiv, 2018-10-18</h4>
          <p>AbstractIn virtually all eukaryotes, telomerase counteracts chromosome erosion by adding repetitive sequence to terminal ends. Drosophila melanogaster instead relies on specialized retrotransposons that insert preferentially at telomeres. This exchange of goods between host and mobile element—wherein the mobile element provides an essential genome service and the host provides a hospitable niche for mobile element propagation—has been called a ‘genomic symbiosis’. However, these telomere-specialized, ‘jockey’ family elements may actually evolve to selfishly over-replicate in the genomes that they ostensibly serve. Under this intra-genomic conflict model, we expect rapid diversification of telomere-specialized retrotransposon lineages and possibly, the breakdown of this tenuous relationship. Here we report data consistent with both predictions. Searching the raw reads of the 15-million-year-old ‘melanogaster species group’, we generated de novo jockey retrotransposon consensus sequences and used phylogenetic tree-building to delineate four distinct telomere-associated lineages. Recurrent gains, losses, and replacements account for this striking retrotransposon lineage diversity. Moreover, an ancestrally telomere-specialized element has ‘escaped,’ residing now throughout the genome of D. rhopaloa. In D. biarmipes, telomere-specialized elements have disappeared completely. De novo assembly of long-reads and cytogenetics confirmed this species-specific collapse of retrotransposon-dependent telomere elongation. Instead, telomere-restricted satellite DNA and DNA transposon fragments occupy its terminal ends. We infer that D. biarmipes relies instead on a recombination-based mechanism conserved from yeast to flies to humans. Combined with previous reports of adaptive evolution at host proteins that regulate telomere length, telomere-associated retrotransposon diversification and disappearance offer compelling evidence that intra-genomic conflict shapes Drosophila telomere evolution.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/evolutionary-biology/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/evolutionary-biology/page/7/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/evolutionary-biology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/evolutionary-biology/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/evolutionary-biology/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/evolutionary-biology/page/7/" class="page-link">7</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/evolutionary-biology/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/evolutionary-biology/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/evolutionary-biology/page/10/" class="page-link">10</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/evolutionary-biology/page/11/" class="page-link">11</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/evolutionary-biology/page/12/" class="page-link">12</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        

        
        <li class="page-item"><a href="/tags/evolutionary-biology/page/9/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/evolutionary-biology/page/14/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
