<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/allododecaploid_yeasts_synthetic_hybrids_of_six_species_58388689/">
          <h4 class="font-weight-bold">Allododecaploid yeasts synthetic hybrids of six species, bioRxiv, 2019-04-03</h4>
          <p>AbstractPolyploidy generates diversity by increasing the number of copies of each chromosome. Many plants, animals, fungi, and other eukaryotes are ancient or recent polyploids, including some of the best-known evolutionary radiations, crops, and industrial organisms. Polyploidy facilitates differentiation and adaptation to new environments, but the tools to test its limits are lacking. Here we develop an iterative Hybrid Production (iHyPr) method to produce allododecaploid yeast strains with a base ploidy of 12n. Chromosomal instability increased dramatically as additional copies of the genome were added. These six-species hybrids rapidly improved their fitness during adaptive laboratory evolution. This new method for making synthetic hybrids will enable basic research on polyploidy, cancer, and chromosome biology, as well as more applied research on biofuels, bioproducts, and synthetic biology.One sentence summaryWe constructed six-species synthetic hybrids and showed that they were chromosomally unstable but able to adapt rapidly.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/interactions_between_the_gut_microbiome_and_host_gene_regulation_in_cystic_fibrosis_58395178/">
          <h4 class="font-weight-bold">Interactions between the gut microbiome and host gene regulation in cystic fibrosis, bioRxiv, 2019-04-03</h4>
          <p>AbstractCystic Fibrosis (CF) is the most common autosomal recessive genetic disease in Caucasians. It is caused by mutations in the CFTR gene, leading to poor hydration of mucus and impairment of the respiratory, digestive, and reproductive organ functions. Advancements in medical care have lead to markedly increased longevity of patients with CF, but new complications have emerged, such as early onset of colorectal cancer (CRC). Although the pathogenesis of CRC in CF remains unclear, altered host-microbe interactions might play a critical role. Here, we characterize the changes in the gut microbiome and host gene expression in colonic mucosa of CF patients relative to healthy controls. We find that CF patients show decreased microbial diversity, decreased abundance of taxa such as Butyricimonas, Sutterella, and Ruminococcaceae, and increased abundance of other taxa, such as Actinobacteria and Firmicutes. We find that 1543 genes, including CFTR, show differential expression in the colon of CF patients compared to healthy controls. Interestingly, we find that these genes are enriched with functions related to gastrointestinal and colorectal cancer, such as metastasis of CRC, tumor suppression, cellular dysfunction, p53 and mTOR signaling pathways. Lastly, we modeled associations between relative abundances of specific bacterial taxa in the gut mucosa and host gene expression, and identified CRC-related genes, including LCN2 and DUOX2, for which gene expression is correlated with the abundance of CRC-associated bacteria, such as Ruminococcaceae and Veillonella. Our results provide new insight into the role of host-microbe interactions in the etiology of CRC in CF.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/synthetic_hybrids_of_six_yeast_species_58388689/">
          <h4 class="font-weight-bold">Synthetic hybrids of six yeast species, bioRxiv, 2019-04-03</h4>
          <p>AbstractAllopolyploidy generates diversity by increasing the number of copies and sources of chromosomes. Many of the best-known evolutionary radiations, crops, and industrial organisms are ancient or recent allopolyploids. Allopolyploidy promotes differentiation and facilitates adaptation to new environments, but the tools to test its limits are lacking. Here we develop an iterative method to combine the genomes of multiple budding yeast species, generating Saccharomyces allopolyploids of an unprecedented scale. Chromosomal instability and cell size increased dramatically as additional copies of the genome were added, but we were able to construct synthetic hybrids of up to six species. The six-species hybrids initially grew slowly, but they rapidly adapted when selection to a novel environment was applied, even as they retained traits from multiple species. These new synthetic yeast hybrids have potential applications for the study of polyploidy, genome stability, chromosome segregation, cancer, and bioenergy.One sentence summaryWe constructed six-species synthetic hybrids and showed that they were chromosomally unstable but able to adapt rapidly.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_heterochromatin-specific_rna_export_pathway_facilitates_pirna_production_58354059/">
          <h4 class="font-weight-bold">A heterochromatin-specific RNA export pathway facilitates piRNA production, bioRxiv, 2019-04-02</h4>
          <p>PIWI-interacting RNAs (piRNAs) guide transposon silencing in animals. The 22-30nt piRNAs are processed in the cytoplasm from long non-coding RNAs. How piRNA precursors, which often lack RNA processing hallmarks of export-competent transcripts, achieve nuclear export is unknown. Here, we uncover the RNA export pathway specific for piRNA precursors in the Drosophila germline. This pathway requires Nxf3-Nxt1, a variant of the hetero-dimeric mRNA export receptor Nxf1-Nxt1. Nxf3 interacts with UAP56, a nuclear RNA helicase essential for mRNA export, and CG13741Bootlegger, which recruits Nxf3-Nxt1 and UAP56 to heterochromatic piRNA source loci. Upon RNA cargo binding, Nxf3 achieves nuclear export via the exportin Crm1, and accumulates together with Bootlegger in peri-nuclear nuage, suggesting that after export, Nxf3-Bootlegger delivers precursor transcripts to the piRNA processing sites. Our findings indicate that the piRNA pathway bypasses nuclear RNA surveillance systems to achieve export of heterochromatic, unprocessed transcripts to the cytoplasm, a strategy also exploited by retroviruses.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/cohesin_disrupts_polycomb-dependent_chromosome_interactions_58112915/">
          <h4 class="font-weight-bold">Cohesin disrupts polycomb-dependent chromosome interactions, bioRxiv, 2019-03-30</h4>
          <p>AbstractHow chromosome organisation is related to genome function remains poorly understood. Cohesin, loop-extrusion, and CTCF have been proposed to create structures called topologically associating domains (TADs) to regulate gene expression. Here, we examine chromosome conformation in embryonic stem cells lacking cohesin and find as in other cell types that cohesin is required to create TADs and regulate AB compartmentalisation. However, in the absence of cohesin we identify a series of long-range chromosomal interactions that persist. These correspond to regions of the genome occupied by the polycomb repressive system, depend on PRC1, and we discover that cohesin counteracts these interactions. This disruptive activity is independent of CTCF and TADs, and regulates gene repression by the polycomb system. Therefore, in contrast to the proposal that cohesin creates structure in chromosomes, we discover a new role for cohesin in disrupting polycomb-dependent chromosome interactions to regulate gene expression.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/common_dna_sequence_variation_influences_3-dimensional_conformation_of_the_human_genome_58114643/">
          <h4 class="font-weight-bold">Common DNA sequence variation influences 3-dimensional conformation of the human genome, bioRxiv, 2019-03-30</h4>
          <p>ABSTRACTThe 3-dimensional (3D) conformation of chromatin inside the nucleus is integral to a variety of nuclear processes including transcriptional regulation, DNA replication, and DNA damage repair. Aberrations in 3D chromatin conformation have been implicated in developmental abnormalities and cancer. Despite the importance of 3D chromatin conformation to cellular function and human health, little is known about how 3D chromatin conformation varies in the human population, or whether DNA sequence variation between individuals influences 3D chromatin conformation. To address these questions, we performed Hi-C on Lymphoblastoid Cell Lines (LCLs) from 20 individuals. We identified thousands of regions across the genome where 3D chromatin conformation varies between individuals and found that this conformational variation is often accompanied by variation in gene expression, histone modifications, and transcription factor (TF) binding. Moreover, we found that DNA sequence variation influences several features of 3D chromatin conformation including loop strength, contact insulation, contact directionality and density of localciscontacts. We mapped hundreds of Quantitative Trait Loci (QTLs) associated with 3D chromatin features and found evidence that some of these same variants are associated at modest levels with other molecular phenotypes as well as complex disease risk. Our results demonstrate that common DNA sequence variants can influence 3D chromatin conformation, pointing to a more pervasive role for 3D chromatin conformation in human phenotypic variation than previously recognized.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/35/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/32/" class="page-link">32</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/33/" class="page-link">33</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/34/" class="page-link">34</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/35/" class="page-link">35</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/0-100-users/page/36/" class="page-link">36</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/37/" class="page-link">37</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/38/" class="page-link">38</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/39/" class="page-link">39</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/40/" class="page-link">40</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/37/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/123/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
