<!DOCTYPE html>
<html><head>
  
  <title>2019 | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/systematic_mapping_of_drug_metabolism_by_the_human_gut_microbiome_54906515/">
          <h4 class="font-weight-bold">Systematic mapping of drug metabolism by the human gut microbiome, bioRxiv, 2019-02-04</h4>
          <p>The human gut microbiome harbors hundreds of bacterial species with diverse biochemical capabilities, making it one of nature&#39;s highest density, highest diversity bioreactors. Several drugs have been previously shown to be directly metabolized by the gut microbiome, but the extent of this phenomenon has not been systematically explored. Here, we develop a systematic screen for mapping the ability of the complex human gut microbiome to biochemically transform small molecules (MDM-Screen), and apply it to a library of 575 clinically used oral drugs. We show that 13% of the analyzed drugs, spanning 28 pharmacological classes, are metabolized by a single microbiome sample. In a proof-of-principle example, we show that microbiome-derived metabolism occurs in vivo, identify the genes responsible for it, and provide a possible link between its consequences and clinically observed features of drug bioavailability and toxicity. Our findings reveal a previously underappreciated role for the gut microbiome in drug metabolism, and provide a comprehensive framework for characterizing this important class of drug-microbiome interactions.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/a_primal_role_for_balance_in_the_development_of_coordinated_locomotion_54826173/">
          <h4 class="font-weight-bold">A primal role for balance in the development of coordinated locomotion, bioRxiv, 2019-02-02</h4>
          <p>Mature locomotion requires that animal nervous systems coordinate distinct groups of muscles. The pressures that guide the development of coordination are not well understood. We studied vertical locomotion in developing zebrafish to understand how and why coordination might emerge. We found that zebrafish used their pectoral fins and bodies synergistically to climb. As they developed, zebrafish came to coordinate their fins and bodies to climb with increasing postural stability. Fin-body synergies were absent in mutants without vestibular sensation, linking balance and coordination. Similarly, synergies were systematically altered following cerebellar lesions, identifying a neural substrate regulating fin-body coordination. Computational modeling illustrated how coordinated climbing could improve balance as zebrafish mature. Together these findings link the sense of balance to the maturation of coordinated locomotion. As they develop, zebrafish improve postural stability by optimizing fin-body coordination. We therefore propose that the need to balance drives the development of coordinated locomotion.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/an_olfactory_receptor_gene_underlies_reproductive_isolation_in_perfume-collecting_orchid_bees_supplemental_materials_54834197/">
          <h4 class="font-weight-bold">An olfactory receptor gene underlies reproductive isolation in perfume-collecting orchid bees Supplemental Materials, bioRxiv, 2019-02-02</h4>
          <p>Speciation is facilitated by the evolution of reproductive barriers that prevent or reduce hybridization among diverging lineages. However, the genetic mechanisms that control the evolution of reproductive barriers remain elusive, particularly in natural populations. We identify a gene associated with divergence in chemical courtship signaling in a pair of nascent orchid bee lineages. Male orchid bees collect perfume compounds from flowers and other sources to subsequently expose during courtship display, thereby conveying information on species identity. We show that these two lineages exhibit differentiated perfume blends and that this change is associated with the rapid evolution of a single odorant receptor gene. Our study suggests that reproductive isolation evolved through divergence of a major barrier gene involved in chemically mediated pre-mating isolation via genetic coupling.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/assessing_graph-based_read_mappers_against_a_novel_baseline_approach_highlights_strengths_and_weaknesses_of_the_current_generation_of_methods_54835128/">
          <h4 class="font-weight-bold">Assessing graph-based read mappers against a novel baseline approach highlights strengths and weaknesses of the current generation of methods, bioRxiv, 2019-02-02</h4>
          <p>AbstractGraph-based reference genomes have become popular as they allow read mapping and follow-up analyses in settings where the exact haplotypes underlying a high-throughput sequencing experiment are not precisely known. Two recent papers show that mapping to graph-based reference genomes can improve accuracy as compared to methods using linear references. Both of these methods index the sequences for most paths up to a certain length in the graph in order to enable direct mapping of reads containing common variants. However, the combinatorial explosion of possible paths through nearby variants also leads to a huge search space and an increased chance of false positive alignments to highly variable regions.We here assess three prominent graph-based read mappers against a novel hybrid baseline approach that combines an initial path determination with a tuned linear read mapping method. We show, using a previously proposed benchmark, that this simple approach is able to improve accuracy of read-mapping to graph-based reference genomes.Our method is implemented in a tool Two-step Graph Mapper, which is available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsgithub.comuio-bmitwo_step_graph_mapper&gt;httpsgithub.comuio-bmitwo_step_graph_mapper&lt;jatsext-link&gt; along with data and scripts for reproducing the experiments.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/islands_of_retroelements_are_the_major_components_of_drosophila_centromeres_54834204/">
          <h4 class="font-weight-bold">Islands of retroelements are the major components of Drosophila centromeres, bioRxiv, 2019-02-02</h4>
          <p>Centromeres are essential chromosomal regions that mediate kinetochore assembly and spindle attachments during cell division. Despite their functional conservation, centromeres are amongst the most rapidly evolving genomic regions and can shape karyotype evolution and speciation across taxa. Although significant progress has been made in identifying centromere-associated proteins, the highly repetitive centromeres of metazoans have been refractory to DNA sequencing and assembly, leaving large gaps in our understanding of their functional organization and evolution. Here, we identify the sequence composition and organization of the centromeres of Drosophila melanogaster by combining long-read sequencing, chromatin immunoprecipitation for the centromeric histone CENP-A, and high-resolution chromatin fiber imaging. Contrary to previous models that heralded satellite repeats as the major functional components, we demonstrate that functional centromeres form on islands of complex DNA sequences enriched in retroelements that are flanked by large arrays of satellite repeats. Each centromere displays distinct size and arrangement of its DNA elements but is similar in composition overall. We discover that a specific retroelement, G2Jockey-3, is the most highly enriched sequence in CENP-A chromatin and is the only element shared among all centromeres. G2Jockey-3 is also associated with CENP-A in the sister species Drosophila simulans, revealing an unexpected conservation despite the reported turnover of centromeric satellite DNA. Our work reveals the DNA sequence identity of the active centromeres of a premier model organism and implicates retroelements as conserved features of centromeric DNA.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/millefy_visualizing_cell-to-cell_heterogeneity_in_read_coverage_of_single-cell_rna_sequencing_datasets_54835125/">
          <h4 class="font-weight-bold">Millefy visualizing cell-to-cell heterogeneity in read coverage of single-cell RNA sequencing datasets, bioRxiv, 2019-02-02</h4>
          <p>Background Read coverage of RNA sequencing data reflects gene expression and RNA processing events. Single-cell RNA sequencing (scRNA-seq) methods, particularly full-length ones, provide read coverage of many individual cells and have the potential to reveal cellular heterogeneity in RNA transcription and processing. However, visualization tools suited to highlighting cell-to-cell heterogeneity in read coverage are still lacking.Results Here, we have developed Millefy, a tool for visualizing read coverage of scRNA-seq data in genomic contexts. Millefy is designed to show read coverage of all individual cells at once in genomic contexts and to highlight cell-to-cell heterogeneity in read coverage. By visualizing read coverage of all cells as a heat map and dynamically reordering cells based on diffusion maps, Millefy facilitates discovery of local region-specific, cell-to-cell heterogeneity in read coverage, including variability of transcribed regions. Conclusions Millefy simplifies the examination of cellular heterogeneity in RNA transcription and processing events using scRNA-seq data. Millefy is available as an R package (httpsgithub.comyuifumillefy) and a Docker image to help use Millefy on the Jupyter notebook (httpshub.docker.comryuifudatascience-notebook-millefy).</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/2019/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/2019/page/70/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/67/" class="page-link">67</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/68/" class="page-link">68</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/69/" class="page-link">69</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/70/" class="page-link">70</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/2019/page/71/" class="page-link">71</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/72/" class="page-link">72</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/73/" class="page-link">73</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/74/" class="page-link">74</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/75/" class="page-link">75</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/2019/page/72/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/2019/page/87/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
