<!DOCTYPE html>
<html><head>
  
  <title>Microbiology | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/hospital_use_of_antibiotics_as_the_main_driver_of_infections_with_antibiotic-resistant_bacteria_-_a_reanalysis_of_recent_data_from_the_european_union_55695720/">
          <h4 class="font-weight-bold">Hospital use of antibiotics as the main driver of infections with antibiotic-resistant bacteria - a reanalysis of recent data from the European Union, bioRxiv, 2019-02-19</h4>
          <p>Antimicrobial resistance in bacteria causes significant morbidity worldwide. The development and acquisition of resistance to antibiotics is believed to primarily develop under the selective pressure of widespread antibiotic use in humans, however antimicrobial usage in livestock has been proposed as additional, if not principal, driver of antibiotic resistance. In this work, we correlate recent data from the European Union on antibiotic resistance rates with data on antibiotic usage in the primary care and hospital sector and data on veterinary antimicrobial consumption across the individual member states. We quantify the strength of these different potential drivers of antimicrobial resistance in order to compare their biological importance. We found that the correlation between antibiotic use in the hospital sector and antibiotic resistance rates is significantly higher than the correlation between resistance rates and any of the other two predictors. This suggests increased antibiotic use in hospitals as the main driver of the development of antibiotic resistances and necessitates further research on and a re-evaluation of the risks associated with antibiotic use in human and veterinary medicine.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/evolutionary_dynamics_of_phage_resistance_in_bacterial_biofilms_55644817/">
          <h4 class="font-weight-bold">Evolutionary dynamics of phage resistance in bacterial biofilms, bioRxiv, 2019-02-17</h4>
          <p>Interactions among bacteria and their viral predators, the bacteriophages, are likely among the most common ecological phenomena on Earth. The constant threat of phage infection to bacterial hosts, and the imperative of achieving infection on the part of phages, drives an evolutionary contest in which phage-resistant bacteria emerge, often followed by phages with new routes of infection. This process has received abundant theoretical and experimental attention for decades and forms an important basis for molecular genetics and theoretical ecology and evolution. However, at present, we know very little about the nature of phage-bacteria interaction -- and the evolution of phage resistance -- inside the surface-bound communities that microbes usually occupy in natural environments. These communities, termed biofilms, are encased in a matrix of secreted polymers produced by their microbial residents. Biofilms are spatially constrained such that interactions become limited to neighbors or near-neighbors; diffusion of solutes and particulates is reduced; and there is pronounced heterogeneity in nutrient access and therefore physiological state. These factors can dramatically impact the way phage infections proceed even in simple, single-strain biofilms, but we still know little of their effect on phage resistance evolutionary dynamics. Here we explore this problem using a computational simulation framework customized for implementing phage infection inside multi-strain biofilms. Our simulations predict that it is far easier for phage-susceptible and phage- resistant bacteria to coexist inside biofilms relative to planktonic culture, where phages and hosts are well-mixed. We characterize the negative frequency dependent selection that underlies this coexistence, and we then test and confirm this prediction using an experimental model of biofilm growth measured with confocal microscopy at single-cell and single-phage resolution.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/cryptic_inoviruses_are_pervasive_in_bacteria_and_archaea_across_earths_biomes_55575446/">
          <h4 class="font-weight-bold">Cryptic inoviruses are pervasive in bacteria and archaea across Earth&#39;s biomes, bioRxiv, 2019-02-16</h4>
          <p>Bacteriophages from the Inoviridae family (inoviruses) are characterized by their unique morphology, genome content, and infection cycle. To date, a relatively small number of inovirus isolates have been extensively studied, either for biotechnological applications such as phage display, or because of their impact on the toxicity of known bacterial pathogens including Vibrio cholerae and Neisseria meningitidis. Here we show that the current 56 members of the Inoviridae family represent a minute fraction of a highly diverse group of inoviruses. Using a new machine learning approach leveraging a combination of marker gene and genome features, we identified 10,295 inovirus-like genomes from microbial genomes and metagenomes. Collectively, these represent six distinct proposed inovirus families infecting nearly all bacterial phyla across virtually every ecosystem. Putative inoviruses were also detected in several archaeal genomes, suggesting that these viruses may have occasionally transferred from bacterial to archaeal hosts. Finally, we identified an expansive diversity of inovirus-encoded  toxin-antitoxin and gene expression modulation systems, alongside evidence of both synergistic (CRISPR evasion) and antagonistic (superinfection exclusion) interactions with co-infecting viruses which we experimentally validated in a Pseudomonas model. Capturing this previously obscured component of the global virosphere sparks new avenues for microbial manipulation approaches and innovative biotechnological applications.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/enterococcus_faecium_genome_dynamics_during_long-term_asymptomatic_patient_gut_colonization_55572932/">
          <h4 class="font-weight-bold">Enterococcus faecium genome dynamics during long-term asymptomatic patient gut colonization, bioRxiv, 2019-02-16</h4>
          <p>Background E. faecium is a gut commensal of humans and animals. In addition, it has recently emerged as an important nosocomial pathogen through the acquisition of genetic elements that confer resistance to antibiotics and virulence. We performed a whole-genome sequencing based study on 96 multidrug-resistant E. faecium strains that asymptomatically colonized five patients with the aim to describe the genome dynamics of this species. Results The patients were hospitalized on multiple occasions and isolates were collected over periods ranging from 15 months to 6.5 years. Ninety-five of the sequenced isolates belonged to E. faecium clade A1, which was previously determined to be responsible for the vast majority of clinical infections. The clade A1 strains clustered into six clonal groups of highly similar isolates, three of which entirely consisted of isolates from a single patient. We also found evidence of concurrent colonization of patients by multiple distinct lineages and transfer of strains between patients during hospitalisation. We estimated the evolutionary rate of two clonal groups that colonized a single patient at 12.6 and 25.2 single nucleotide polymorphisms (SNPs)genomeyear. A detailed analysis of the accessory genome of one of the clonal groups revealed considerable variation due to gene gain and loss events, including the chromosomal acquisition of a 37 kbp prophage and the loss of an element containing carbohydrate metabolism-related genes. We determined the presence and location of twelve different Insertion Sequence (IS) elements, with ISEfa5 showing a unique pattern of location in 24 of the 25 isolates, suggesting widespread ISEfa5 excision and insertion into the genome during gut colonization. Conclusions Our findings show that the E. faecium genome is highly dynamic during asymptomatic colonization of the patient gut. We observe considerable genomic flexibility due to frequent horizontal gene transfer and recombination, which can contribute to the generation of genetic diversity within the species and, ultimately, can contribute to its success as a nosocomial pathogen.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/bacterial_phage_tail-like_structure_kills_eukaryotic_cells_by_injecting_a_nuclease_effector_55318539/">
          <h4 class="font-weight-bold">Bacterial Phage Tail-like Structure Kills Eukaryotic Cells by Injecting a Nuclease Effector, bioRxiv, 2019-02-12</h4>
          <p>Many bacteria interact with target organisms using syringe-like structures called Contractile Injection Systems (CIS). CIS structurally resemble headless bacteriophages and share evolutionarily related proteins such as the tail tube, sheath, and baseplate complex. Recent evidence shows that CIS are specialized to puncture membranes and often deliver effectors to target cells. In many cases, CIS mediate trans-kingdom interactions between bacteria and eukaryotes, however the effectors delivered to target cells and their mode of action are often unknown. In this work, we establish an in vitro model to study a CIS called Metamorphosis Associated Contractile structures (MACs) that target eukaryotic cells. We show that MACs kill two eukaryotic cell lines, Fall Armyworm Sf9 cells and J774A.1 murine macrophage cells through the action of a newly identified MAC effector, termed Pne1. To our knowledge, Pne1 is the first CIS effector exhibiting nuclease activity against eukaryotic cells. Our results define a new mechanism of CIS-mediated bacteria-eukaryote interaction and are a first step toward understanding structures with the potential to be developed as novel delivery systems for eukaryotic hosts.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/emergence_of_stable_coexistence_in_a_complex_microbial_community_through_metabolic_cooperation_and_spatio-temporal_niche_partitioning_55007744/">
          <h4 class="font-weight-bold">Emergence of stable coexistence in a complex microbial community through metabolic cooperation and spatio-temporal niche partitioning, bioRxiv, 2019-02-06</h4>
          <p>Microbial communities in nature often feature complex compositional dynamics yet also stable coexistence of diverse species. The mechanistic underpinnings of such dynamic stability remain unclear as system-wide studies have been limited to small engineered communities or synthetic assemblies. Here we show how kefir, a natural milk-fermenting community, realizes stable coexistence through spatio-temporal orchestration of species and metabolite dynamics. During milk fermentation, kefir grains (a polysaccharide matrix synthesized by kefir microbes) grow in mass but remain unchanged in composition. In contrast, the milk is colonized in a dynamic fashion with early members opening metabolic niches for the followers. Through large-scale mapping of metabolic preferences and inter-species interactions, we show how microbes poorly suited for milk survive in, and even dominate the community through metabolic cooperation and uneven partitioning between the grain and the liquid phase. Overall, our findings reveal how spatio-temporal dynamics promote stable coexistence and have implications for deciphering and modulating complex microbial ecosystems.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/microbiology/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/microbiology/page/4/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="/tags/microbiology/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/microbiology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/microbiology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/microbiology/page/4/" class="page-link">4</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/microbiology/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/microbiology/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/microbiology/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/microbiology/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/microbiology/page/9/" class="page-link">9</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        

        
        <li class="page-item"><a href="/tags/microbiology/page/6/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/microbiology/page/11/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
