<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/a_comparison_of_gene_expression_and_dna_methylation_patterns_across_tissues_and_species_52189381/">
          <h4 class="font-weight-bold">A comparison of gene expression and DNA methylation patterns across tissues and species, bioRxiv, 2018-12-05</h4>
          <p>AbstractPreviously published comparative functional genomic data sets from primates using frozen tissue samples, including many data sets from our own group, were collected and analyzed using non-optimal study designs and analysis approaches. In addition, when samples from multiple tissues were studied in a comparative framework, individual and tissue were confounded. We designed a multi-tissue comparative study of gene expression and DNA methylation in primates that minimizes confounding effects by using a balanced design with respect to species, tissues, and individuals. We also developed a comparative analysis pipeline that minimizes biases due to sequence divergence. We thus present the most comprehensive catalog of similarities and differences in gene expression and methylation levels between livers, kidneys, hearts, and lungs, in humans, chimpanzees, and rhesus macaques. We estimate that overall, only between 7 to 11% (depending on the tissue) of inter-species differences in gene expression levels can be accounted for by corresponding differences in promoter DNA methylation. However, gene expression divergence in conserved tissue-specific genes can be explained by corresponding inter-species methylation changes more often. We end the paper by providing recommendations for effective study design and best practices for meta-data recording for comparative functional genomic studies in primates.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/natural_selection_contributed_to_immunological_differences_between_human_hunter-gatherers_and_agriculturalists_52201024/">
          <h4 class="font-weight-bold">Natural selection contributed to immunological differences between human hunter-gatherers and agriculturalists, bioRxiv, 2018-12-05</h4>
          <p>AbstractThe shift from a hunter-gatherer (HG) to an agricultural (AG) mode of subsistence is believed to have been associated with profound changes in the burden and diversity of pathogens across human populations. Yet, the extent to which the advent of agriculture may have impacted the evolution of the human immune system remains unknown. Here we present a comparative study of variation in the transcriptional responses of peripheral blood mononuclear cells to bacterial and viral stimuli between Batwa rainforest hunter-gatherers and Bakiga agriculturalists from Uganda. We observed increased divergence between hunter-gatherers and agriculturalists in the transcriptional response to viruses compared to that for bacterial stimuli. We demonstrate that a significant fraction of these transcriptional differences are under genetic control, and we show that positive natural selection has helped to shape population differences in immune regulation. Across the set of genetic variants underlying inter-population immune response differences, however, the signatures of positive selection were disproportionately observed in the rainforest hunter-gatherers. This result is counter to expectations based on the popularized notion that shifts in pathogen exposure due to the advent of agriculture imposed radically heightened selective pressures in agriculturalist populations.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/climate_or_disturbance_temperate_forest_structural_change_and_carbon_sink_potential_51853375/">
          <h4 class="font-weight-bold">Climate or disturbance temperate forest structural change and carbon sink potential, bioRxiv, 2018-11-27</h4>
          <p>ABSTRACTAnticipating forest responses to changing climate and disturbance regimes requires understanding long-term successional processes and aggregating these local processes into global relevance. Estimates of existing forest structure and biomass are improving globally; however, vegetation models continue to show substantial spread in predictions of future land carbon uptake and the roles of forest structural change and demography are increasingly being recognized as important. To identify mechanisms that drive change in tree size, density, and carbon, we need a better understanding of forest structural trajectories and the factors that affect those trajectories. Here we reveal a coherent, cyclic pattern of structural change in temperate forests, as predicted by successional theory, and identify significant sensitivity to climatic precipitation and temperature anomalies using large datasets and empirical modeling. For example, in the eastern US above average temperature (&#43;1°C) was associated with a 27% (−0.4±0.1 Mg C ha-1 yr-1) reduction in productivity attributed to higher rates of disease (&#43;23%), weather disturbance (&#43;57%), and sapling mortality. Projections of future vegetative carbon sink potential suggests biomass would be lowest on managed lands (72±2 Mg C ha-1) and highest when larger trees survive in undisturbed conditions (153±21 Mg C ha-1). Overall, the indirect effects of disturbance and mortality were considerably larger than the direct effects of climate on productivity when predicting future vegetative carbon sinks. Results provide robust comparisons for global vegetation models, and valuable projections for management and carbon mitigation efforts.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/quantitative_pcr_provides_a_simple_and_accessible_method_for_quantitative_microbiome_profiling_51829165/">
          <h4 class="font-weight-bold">Quantitative PCR provides a simple and accessible method for quantitative microbiome profiling, bioRxiv, 2018-11-27</h4>
          <p>AbstractThe use of relative next generation sequencing (NGS) abundance data can lead to misinterpretations of microbial community structures as the increase of one taxon leads to concurrent decrease of the other(s). To overcome compositionality, we provide a quantitative NGS solution, which is achieved by adjusting the relative 16S rRNA gene amplicon NGS data with quantitative PCR (qPCR-based) total bacterial counts. By comparing the enumeration of dominant bacterial groups on different taxonomic levels in human fecal samples using taxon-specific 16S rRNA gene-targeted qPCR we show that quantitative NGS is able to estimate absolute bacterial abundances accurately. We also observed a higher degree of correspondence in the estimated microbe-metabolite relationship when quantitative NGS was applied. Being conceptually and methodologically analogous to amplicon-based NGS, our qPCR-based method can be readily incorporated into the standard, high-throughput NGS sample processing pipeline for more accurate description of interactions within and between the microbes and host.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/comparative_assessment_of_long-read_error-correction_software_applied_to_rna-sequencing_data_51603919/">
          <h4 class="font-weight-bold">Comparative assessment of long-read error-correction software applied to RNA-sequencing data, bioRxiv, 2018-11-23</h4>
          <p>AbstractMotivationLong-read sequencing technologies offer promising alternatives to high-throughput short read sequencing, especially in the context of RNA-sequencing. However these technologies are currently hindered by high error rates in the output data that affect analyses such as the identification of isoforms, exon boundaries, open reading frames, and the creation of gene catalogues. Due to the novelty of such data, computational methods are still actively being developed and options for the error-correction of RNA-sequencing long reads remain limited.ResultsIn this article, we evaluate the extent to which existing long-read DNA error correction methods are capable of correcting cDNA Nanopore reads. We provide an automatic and extensive benchmark tool that not only reports classical error-correction metrics but also the effect of correction on gene families, isoform diversity, bias towards the major isoform, and splice site detection. We find that long read error-correction tools that were originally developed for DNA are also suitable for the correction of RNA-sequencing data, especially in terms of increasing base-pair accuracy. Yet investigators should be warned that the correction process perturbs gene family sizes and isoform diversity. This work provides guidelines on which (or whether) error-correction tools should be used, depending on the application type.Benchmarking software&lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsgitlab.comleoislLR_EC_analyser&gt;httpsgitlab.comleoislLR_EC_analyser&lt;jatsext-link&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/tracing_diagnosis_trajectories_over_millions_of_inpatients_reveal_an_unexpected_association_between_schizophrenia_and_rhabdomyolysis_51407516/">
          <h4 class="font-weight-bold">Tracing diagnosis trajectories over millions of inpatients reveal an unexpected association between schizophrenia and rhabdomyolysis, bioRxiv, 2018-11-20</h4>
          <p>AbstractWhile it has been technically feasible to create longitudinal representations of individual health at a nationwide scale, the use of these techniques to identify novel disease associations for the risk stratification of patients has had limited success. Here, we created a large-scale US longitudinal disease network of traced readmission patterns (i.e., disease trajectories), merging data from over 10.4 million inpatients from 350 California hospitals through the Healthcare Cost and Utilization Project between 1980 and 2010. We were able to create longitudinal representations of disease progression mapping over 300 common diseases, including the well-known complication of heart failure after acute myocardial infarction. Surprisingly, out of these generated disease trajectories, we discovered an unknown association between schizophrenia, a chronic mental disorder, and rhabdomyolysis, a rare disease of muscle breakdown. It was found that 92 of 3674 patients (2.5%) with schizophrenia were readmitted for rhabdomyolysis (relative risk, 2.21 [1.80–2.71, confidence interval = 0.95] P-value 9.54E-15), which has a general population incidence of 1 in 10,000. We validated this association using independent electronic health records from over 830,000 patients treated over seven years at the University of California, San Francisco (UCSF) medical center. A case review of 29 patients at UCSF who were treated for schizophrenia and who went on to develop rhabdomyolysis demonstrated that the majority of cases (62%) are idiopathic, which suggests a biological connection between these two diseases. Together, these findings demonstrate the power of using public disease registries in combination with electronic medical records to discover novel disease associations.One Sentence SummaryBased on the longitudinal health records from millions of California inpatient discharges, we created a temporal network that enabled us to understand statewide patterns of hospital readmissions, which led to the novel finding that hospitalization for schizophrenia is significantly associated with rehospitalization for rhabdomyolysis.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/0-100-users/page/32/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/29/" class="page-link">29</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/30/" class="page-link">30</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/31/" class="page-link">31</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/32/" class="page-link">32</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/0-100-users/page/33/" class="page-link">33</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/34/" class="page-link">34</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/35/" class="page-link">35</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/36/" class="page-link">36</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/37/" class="page-link">37</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/0-100-users/page/34/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/0-100-users/page/64/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
