<!DOCTYPE html>
<html><head>
  
  <title>Neuroscience | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/facilitating_open-science_with_realistic_fmri_simulation_validation_and_application_54629057/">
          <h4 class="font-weight-bold">Facilitating open-science with realistic fMRI simulation validation and application, bioRxiv, 2019-01-29</h4>
          <p>Background With advances in methods for collecting and analyzing fMRI data, there is a concurrent need to understand how to reliably evaluate and optimally use these methods. Simulations of fMRI data can aid in both the evaluation of complex designs and the analysis of data. New Method We present fmrisim, a new Python package for standardized, realistic simulation of fMRI data. This package is part of BrainIAK a recently released open-source Python toolbox for advanced neuroimaging analyses. We describe how to use fmrisim to extract noise properties from real fMRI data and then create a synthetic dataset with matched noise properties and a user-specified signal. Results We validate the noise generated by fmrisim to show that it can approximate the noise properties of real data. We further show how fmrisim can help researchers find the optimal design in terms of power. Comparison with other methods fmrisim ports the functionality of other packages to the Python platform while extending what is available in order to make it seamless to simulate realistic fMRI data. Conclusions The fmrisim package holds promise for improving the design of fMRI experiments, which may facilitate both the pre-registration</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/high-pass_filtering_artifacts_in_multivariate_classification_of_neural_time_series_data_54543179/">
          <h4 class="font-weight-bold">High-pass filtering artifacts in multivariate classification of neural time series data, bioRxiv, 2019-01-27</h4>
          <p>The application of time-resolved multivariate pattern classification analyses (MVPA) to EEG and MEG data has become increasingly popular. Traditionally, such time series data are high-pass filtered before analyses, in order to remove slow drifts. Here we show that high-pass filtering should be applied with extreme caution in MVPA, as it may easily create artifacts that result in displacement of decoding accuracy, leading to statistically significant above-chance classification during time periods in which the source is clearly not in brain activity. In both real and simulated EEG data, we show that spurious decoding may emerge with filter cut-off settings from as modest as 0.1 Hz. We provide an alternative method of removing slow drift noise, referred to as robust detrending (de Cheveigne &amp;amp; Arzounian, 2018), which, when applied in concert with masking of cortical events does not result in the temporal displacement of information. We show that temporal generalization may benefit from robust detrending, without any of the unwanted side effects introduced by filtering. However, we conclude that for sufficiently clean data sets, no filtering or detrending at all may work sufficiently well. Implications for other types of data are discussed, followed by a number of recommendations.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/c99_not_beta-amyloid_is_associated_with_selective_death_of_vulnerable_neurons_in_alzheimers_disease._supplementary_54295140/">
          <h4 class="font-weight-bold">C99, not beta-amyloid, is associated with selective death of vulnerable neurons in Alzheimer&#39;s disease. Supplementary, bioRxiv, 2019-01-23</h4>
          <p>Amyloid Precursor Protein (APP) and its cleavage product beta-amyloid are widely believed to be key players in the development of Alzheimer&#39;s disease (AD). However, the distribution of amyloid deposits in the brain does not correlate well with disease progression. Therefore, it seemed possible that APP metabolites other than beta-amyloid might make a strong contribution to AD pathology. We developed a sensitive assay adapted to the detection of C99, an intermediate in the conversion of APP to beta-amyloid. Brain tissue sections were obtained from patients suffering from sporadic AD and non-demented controls. Our results demonstrate that C99 levels, but not Abeta levels, correlate with the degree of vulnerability to neurodegeneration and cognitive impairment in patients suffering from AD.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/prospective_brain-wide_labeling_of_neuronal_subclasses_with_enhancer-driven_aavs_54201876/">
          <h4 class="font-weight-bold">Prospective, brain-wide labeling of neuronal subclasses with enhancer-driven AAVs, bioRxiv, 2019-01-21</h4>
          <p>Labeling and perturbation of specific cell types in multicellular systems has transformed our ability to understand them. The rapid pace of cell type identification by new single-cell analysis methods has not been met with efficient access to these newly discovered types. To enable access to specific neural populations in the mouse cortex, we have collected single cell chromatin accessibility data from select cell types. We clustered the single cell data and mapped them to single cell transcriptomics to identify highly specific enhancers for cell subclasses. These enhancers, when cloned into AAVs and delivered to the brain by retro orbital injections, transgene expression in specific cell subclasses throughout the mouse brain. This approach will enable functional investigation of cell types in the mouse cortex and beyond.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/evolving_super_stimuli_for_real_neurons_using_deep_generative_networks_54060728/">
          <h4 class="font-weight-bold">Evolving super stimuli for real neurons using deep generative networks, bioRxiv, 2019-01-17</h4>
          <p>Finding the best stimulus for a neuron is challenging because it is impossible to test all possible stimuli. Here we used a vast, unbiased, and diverse hypothesis space encoded by a generative deep neural network model to investigate neuronal selectivity in inferotemporal cortex without making any assumptions about natural features or categories. A genetic algorithm, guided by neuronal responses, searched this space for optimal stimuli. Evolved synthetic images evoked higher firing rates than even the best natural images and revealed diagnostic features, independently of category or feature selection. This approach provides a way to investigate neural selectivity in any modality that can be represented by a neural network and challenges our understanding of neural coding in visual cortex.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/single-cell_transcriptomic_evidence_for_dense_intracortical_neuropeptide_networks_53890159/">
          <h4 class="font-weight-bold">Single-Cell Transcriptomic Evidence for Dense Intracortical Neuropeptide Networks, bioRxiv, 2019-01-14</h4>
          <p>BrieflyAnalysis of single-cell RNA-Seq data from mouse neocortex exposes evidence for local neuropeptidergic modulation networks that involve every cortical neuron directly.Data Highlights&lt;jatslist list-type=bullet&gt;&lt;jatslist-item&gt;At least 98% of mouse neocortical neurons express one or more of 18 neuropeptide precursor proteins (NPP) genes.&lt;jatslist-item&gt;&lt;jatslist-item&gt;At least 98% of cortical neurons express one or more of 29 neuropeptide-selective G-protein-coupled receptor (NP-GPCR) genes.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Neocortical expression of these 18 NPP and 29 NP-GPCR genes is highly neuron-type-specific and permits exceptionally powerful differentiation of transcriptomic neuron types.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Neuron-type-specific expression of 37 cognate NPP  NP-GPCR gene pairs predicts modulatory connectivity within 37 or more neuron-type-specific intracortical networks.&lt;jatslist-item&gt;SummarySeeking insight into homeostasis, modulation and plasticity of cortical synaptic networks, we analyzed results from deep RNA-Seq analysis of 22,439 individual mouse neocortical neurons. This work exposes transcriptomic evidence that all cortical neurons participate directly in highly multiplexed networks of modulatory neuropeptide (NP) signaling. The evidence begins with a discovery that transcripts of one or more neuropeptide precursor (NPP) and one or more neuropeptide-selective G-protein-coupled receptor (NP-GPCR) genes are highly abundant in nearly all cortical neurons. Individual neurons express diverse subsets of NP signaling genes drawn from a palette encoding 18 NPPs and 29 NP-GPCRs. Remarkably, these 47 genes comprise 37 cognate NPPNP-GPCR pairs, implying a strong likelihood of dense, cortically localized neuropeptide signaling. Here we use neuron-type-specific NP gene expression signatures to put forth specific, testable predictions regarding 37 peptidergic neuromodulatory networks that may play prominent roles in cortical homeostasis and plasticity.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/neuroscience/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/neuroscience/page/10/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/neuroscience/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/neuroscience/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/neuroscience/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/neuroscience/page/10/" class="page-link">10</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/neuroscience/page/11/" class="page-link">11</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/neuroscience/page/12/" class="page-link">12</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/neuroscience/page/13/" class="page-link">13</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/neuroscience/page/14/" class="page-link">14</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/neuroscience/page/15/" class="page-link">15</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/neuroscience/page/12/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/neuroscience/page/29/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
