<!DOCTYPE html>
<html><head>
  
  <title>2017 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/platform_for_rapid_nanobody_discovery_in_vitro_21125069/">
          <h4 class="font-weight-bold">Platform for rapid nanobody discovery in vitro, bioRxiv, 2017-06-17</h4>
          <p>AbstractCamelid single-domain antibody fragments (“nanobodies”) provide the remarkable specificity of antibodies within a single immunoglobulin VHH domain. This unique feature enables applications ranging from their use as biochemical tools to therapeutic agents. Virtually all nanobodies reported to date have been obtained by animal immunization, a bottleneck restricting many applications of this technology. To solve this problem, we developed a fully in vitro platform for nanobody discovery based on yeast surface display of a synthetic nanobody scaffold. This platform provides a facile and cost-effective method for rapidly isolating nanobodies targeting a diverse range of antigens. We provide a blueprint for identifying nanobodies starting from both purified and non-purified antigens, and in addition, we demonstrate application of the platform to discover rare conformationally-selective nanobodies to a lipid flippase and a G protein-coupled receptor. To facilitate broad deployment of this platform, we have made the library and all associated protocols publicly available.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biochemistry">biochemistry</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/identification_of_a_novel_interspecific_hybrid_yeast_from_a_metagenomic_spontaneously_inoculated_beer_sample_using_hi-c_21094275/">
          <h4 class="font-weight-bold">Identification of a novel interspecific hybrid yeast from a metagenomic spontaneously inoculated beer sample using Hi-C, bioRxiv, 2017-06-16</h4>
          <p>AbstractInterspecific hybridization is a common mechanism enabling genetic diversification and adaptation; however, the detection of hybrid species has been quite difficult. The identification of microbial hybrids is made even more complicated, as most environmental microbes are resistant to culturing and must be studied in their native mixed communities. We have previously adapted the chromosome conformation capture method Hi-C to the assembly of genomes from mixed populations. Here, we show the method’s application in assembling genomes directly from an uncultured, mixed population from a spontaneously inoculated beer sample. Our assembly method has enabled us to de-convolute 4 bacterial and 4 yeast genomes from this sample, including a putative yeast hybrid. Downstream isolation and analysis of this hybrid confirmed its genome to consist of Pichia membranifaciens and that of another related, but undescribed yeast. Our work shows that Hi-C-based metagenomic methods can overcome the limitation of traditional sequencing methods in studying complex mixtures of genomes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/dark_control_towards_a_unified_account_of_default_mode_function_by_markov_decision_processes_21053315/">
          <h4 class="font-weight-bold">Dark Control Towards a Unified Account of Default Mode Function by Markov Decision Processes, bioRxiv, 2017-06-15</h4>
          <p>AbstractThe default mode network (DMN) is believed to subserve the baseline mental activity in humans. Its highest energy consumption compared to other brain networks and its intimate coupling with conscious awareness are both pointing to an overarching function. Many research streams speak in favor of an evolutionarily adaptive role in envisioning experience to anticipate the future. In the present work, we propose a process model that tries to explain how the DMN may implement continuous evaluation and prediction of the environment to guide behavior. Specifically, we answer the question whether the neurobiological properties of the DMN collectively provide the computational building blocks necessary for a Markov Decision Process. We argue that our formal account of DMN function naturally accommodates as special cases previous interpretations based on (1) predictive coding, (2) semantic associations, and (3) a sentinel role. Moreover, this process model for the neural optimization of complex behavior in the DMN offers parsimonious explanations for recent experimental findings in animals and humans.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/high_contiguity_arabidopsis_thaliana_genome_assembly_with_a_single_nanopore_flow_cell_21065652/">
          <h4 class="font-weight-bold">High contiguity Arabidopsis thaliana genome assembly with a single nanopore flow cell, bioRxiv, 2017-06-15</h4>
          <p>AbstractWhile many evolutionary questions can be answered by short read re-sequencing, presenceabsence polymorphisms of genes andor transposons have been largely ignored in large-scale intraspecific evolutionary studies. To enable the rigorous analysis of such variants, multiple high quality and contiguous genome assemblies are essential. Similarly, while genome assemblies based on short reads have made genomics accessible for non-reference species, these assemblies have limitations due to low contiguity. Long-read sequencers and long-read technologies have ushered in a new era of genome sequencing where the lengths of reads exceed those of most repeats. However, because these technologies are not only costly, but also time and compute intensive, it has been unclear how scalable they are. Here we demonstrate a fast and cost effective reference assembly for an Arabidopsis thaliana accession using the USB-sized Oxford Nanopore MinION sequencer and typical consumer computing hardware (4 Cores, 16Gb RAM). We assemble the accession KBS-Mac-74 into 62 contigs with an N50 length of 12.3 Mb covering 100% (119 Mb) of the non-repetitive genome. We demonstrate that the polished KBS-Mac-74 assembly is highly contiguous with BioNano optical genome maps, and of high per-base quality against a likewise polished Pacific Biosciences long-read assembly. The approach we implemented took a total of four days at a cost of less than 1,000 USD for sequencing consumables including instrument depreciation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/an_open_resource_for_transdiagnostic_research_in_pediatric_mental_health_and_learning_disorders_21035196/">
          <h4 class="font-weight-bold">An open resource for transdiagnostic research in pediatric mental health and learning disorders, bioRxiv, 2017-06-14</h4>
          <p>ABSTRACTTechnological and methodological innovations are equipping researchers with unprecedented capabilities for detecting and characterizing pathologic processes in the developing human brain. As a result, ambitions to achieve clinically useful tools to assist in the diagnosis and management of mental health and learning disorders are gaining momentum. To this end, it is critical to accrue large-scale multimodal datasets that capture a broad range of commonly encountered clinical psychopathology. The Child Mind Institute has launched the Healthy Brain Network (HBN), an ongoing initiative focused on creating and sharing a biobank of data from 10,000 New York area participants (ages 5-21). The HBN Biobank houses data about psychiatric, behavioral, cognitive, and lifestyle phenotypes, as well as multimodal brain imaging (resting and naturalistic viewing fMRI, diffusion MRI, morphometric MRI), electroencephalography, eye-tracking, voice and video recordings, genetics, and actigraphy. Here, we present the rationale, design and implementation of HBN protocols. We describe the first data release (n = 664) and the potential of the biobank to advance related areas (e.g., biophysical modeling, voice analysis).</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/beyond_consensus_embracing_heterogeneity_in_curated_neuroimaging_meta-analysis_21038399/">
          <h4 class="font-weight-bold">Beyond Consensus Embracing Heterogeneity in Curated Neuroimaging Meta-Analysis, bioRxiv, 2017-06-14</h4>
          <p>Coordinate-based meta-analysis can provide important insights into mind-brain relationships. A popular approach for curated small-scale meta-analysis is activation likelihood estimation (ALE), which identifies brain regions consistently activated across a selected set of experiments, such as within a functional domain or mental disorder. ALE can also be utilized in meta-analytic co-activation modeling (MACM) to identify brain regions consistently co-activated with a seed region. Therefore, ALE aims to find consensus across experiments, treating heterogeneity across experiments as noise. However, heterogeneity within an ALE analysis of a functional domain might indicate the presence of functional sub-domains. Similarly, heterogeneity within a MACM analysis might indicate the involvement of a seed region in multiple co-activation patterns that are dependent on task contexts. Here, we demonstrate the use of the author-topic model to automatically determine if heterogeneities within ALE-type meta-analyses can be robustly explained by a small number of latent patterns. In the first application, the author-topic modeling of experiments involving self-generated thought (N = 179) revealed cognitive components fractionating the default network. In the second application, the author-topic model revealed that the left inferior frontal junction (IFJ) participated in multiple task-dependent co-activation patterns (N = 323). Furthermore, the author-topic model estimates compared favorably with spatial independent component analysis in both simulation and real data. Overall, the results suggest that the author-topic model is a flexible tool for exploring heterogeneity in ALE-type meta-analyses that might arise from functional sub-domains, mental disorder subtypes or task-dependent co-activation patterns. Code for this study is publicly available (httpsgithub.comThomasYeoLabCBIGtreemasterstable_projectsmeta-analysisNgo2019_AuthorTopic).</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2017">2017</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/2017/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/2017/page/39/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2017/page/36/" class="page-link">36</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2017/page/37/" class="page-link">37</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2017/page/38/" class="page-link">38</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2017/page/39/" class="page-link">39</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/2017/page/40/" class="page-link">40</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2017/page/41/" class="page-link">41</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2017/page/42/" class="page-link">42</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2017/page/43/" class="page-link">43</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2017/page/44/" class="page-link">44</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/2017/page/41/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/2017/page/62/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
