<!DOCTYPE html>
<html><head>
  
  <title>2018 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/an_rna-binding_region_regulates_ctcf_clustering_and_chromatin_looping_52608001/">
          <h4 class="font-weight-bold">An RNA-binding region regulates CTCF clustering and chromatin looping, bioRxiv, 2018-12-13</h4>
          <p>Mammalian genomes are folded into Topologically Associating Domains (TADs), consisting of cell-type specific chromatin loops anchored by CTCF and cohesin. Since CTCF and cohesin are expressed ubiquitously, how cell-type specific CTCF-mediated loops are formed poses a paradox. Here we show RNase-sensitive CTCF self-association in vitro and that an RNA-binding region (RBR) mediates CTCF clustering in vivo. Intriguingly, deleting the RBR abolishes or impairs almost half of all chromatin loops in mouse embryonic stem cells. Disrupted loop formation correlates with abrogated clustering and diminished chromatin binding of the RBR mutant CTCF protein, which in turn results in a failure to halt cohesin-mediated extrusion. Thus, CTCF loops fall into at least 2 classes RBR-independent and RBR-dependent loops. We suggest that evidence for distinct classes of RBR-dependent loops may provide a mechanism for establishing cell-specific CTCF loops regulated by RNAs and other RBR partner.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/comprehensive_immune_monitoring_of_clinical_trials_to_advance_human_immunotherapy_52608023/">
          <h4 class="font-weight-bold">Comprehensive Immune Monitoring of Clinical Trials to Advance Human Immunotherapy, bioRxiv, 2018-12-13</h4>
          <p>SummaryThe success of immunotherapy has led to a myriad of new clinical trials. Connected to these trials are efforts to discover biomarkers providing mechanistic insight and predictive signatures for personalization. Still, the plethora of immune monitoring technologies can face investigator bias, missing unanticipated cellular responses in limited clinical material. We here present a mass cytometry workflow for standardized, systems-level biomarker discovery in immunotherapy trials. To broadly enumerate human immune cell identity and activity, we established and extensively assessed a reference panel of 33 antibodies to cover major cell subsets, simultaneously quantifying activation and immune checkpoint molecules in a single assay. The resulting assay enumerated ≥ 98% of peripheral immune cells with ≥ 4 positively identifying antigens. Robustness and reproducibility were demonstrated on multiple samples types, across research centers and by orthogonal measurements. Using automated analysis, we monitored complex immune dynamics, identifying signatures in bone-marrow transplantation associated graft-versus-host disease. This validated and available workflow ensures comprehensive immunophenotypic analysis, data comparability and will accelerate biomarker discovery in immunomodulatory therapeutics.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/identifying_facial_phenotypes_of_genetic_disorders_using_deep_learning_53588150/">
          <h4 class="font-weight-bold">Identifying facial phenotypes of genetic disorders using deep learning, Nature Medicine, 2018-12-13</h4>
          <p>6</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/nature%20medicine">nature medicine</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_fluorescent_reporter_enables_instantaneous_measurement_of_cell_cycle_speed_in_live_cells_52553188/">
          <h4 class="font-weight-bold">A fluorescent reporter enables instantaneous measurement of cell cycle speed in live cells, bioRxiv, 2018-12-12</h4>
          <p>AbstractPeriodicity is a fundamental property of biological oscillators such as the mitotic cell cycle. In this context, periodicity refers to the time interval between the same phases of two consecutive cell cycles. The length of this interval, or the cell cycle speed, varies widely depending on cell type and the pathophysiological conditions. The relevance of cell cycle speed in various biological contexts has not been well-studied, partially due to the lack of experimental approaches that capture this parameter. Here, we describe a genetically encoded live-cell reporter of cell cycle speed. This reporter is based on the color-changing Fluorescent Timer (FT) protein, which emits blue fluorescence when newly synthesized before maturing into a red fluorescent protein. Its ability to report cell cycle speed exploits the different half-life of the blue vs. red form of the same molecule, as predicted by mathematical modeling. When a Histone H2B-FT fusion protein is expressed at steady-state in heterogeneously dividing cells, faster-cycling cells can be distinguished from slower-cycling ones by differences in their intracellular ratio between the blue and red fluorescent wavelengths. Cell cycle perturbation experiments demonstrate that the H2B-FT is a bona fide reporter of cell cycle speed in multiple cultured cell lines. In vivo, the bluered profile faithfully tracked with known proliferation kinetics of various hematopoietic stem and progenitor cells, when expressed either from lentiviral vectors or from a targeted knock-in allele. As the H2B-FT is compatible with flow cytometry, it provides a strategy to physically separate subpopulations of live cells cycling at different rates for downstream analysis. We anticipate this system to be useful in diverse cell types and tissue contexts for dissecting the role of cell cycle speed in development and disease.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/resolving_cell_cycle_speed_in_one_snapshot_with_a_live-cell_fluorescent_reporter_52553188/">
          <h4 class="font-weight-bold">Resolving cell cycle speed in one snapshot with a live-cell fluorescent reporter, bioRxiv, 2018-12-12</h4>
          <p>SummaryCell proliferation changes concomitantly with fate transitions during reprogramming, differentiation, regeneration, and oncogenesis. Methods to resolve cell cycle length heterogeneity in real-time are currently lacking. Here, we describe a genetically encoded fluorescent reporter that captures live cell cycle speed using a single measurement. This reporter is based on the color-changing Fluorescent Timer (FT) protein, which emits blue fluorescence when newly synthesized before maturing into a red fluorescent protein. We generated a mouse strain expressing an H2B-FT fusion reporter from a universally active locus, and demonstrate that faster-cycling cells can be distinguished from slower-cycling ones based on the intracellular fluorescence ratio between the FT’s blue and red states. Using this reporter, we reveal the native cell cycle speed distributions of fresh hematopoietic cells, and demonstrate its utility in analyzing cell proliferation in solid tissues. This system is broadly applicable for dissecting functional heterogeneity associated with cell cycle dynamics in complex tissues.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/cryptic_and_extensive_hybridization_between_ancient_lineages_of_american_crows_52485986/">
          <h4 class="font-weight-bold">Cryptic and extensive hybridization between ancient lineages of American crows, bioRxiv, 2018-12-11</h4>
          <p>AbstractMost species and therefore most hybrid zones have historically been described using phenotypic characters. However, both speciation and hybridization can occur with negligible morphological differentiation. The Northwestern Crow (Corvus caurinus) and American Crow (Corvus brachyrhynchos) are sister taxonomic species with a continuous distribution that lack reliable traditional characters for identification. In this first population genomic study of Northwestern and American crows, we use genomic SNPs (nuDNA) and mtDNA to investigate whether these crows are genetically differentiated and the extent to which they may hybridize. We found that American and Northwestern crows have distinct evolutionary histories, supported by two nuDNA ancestry clusters and two 1.1%-divergent mtDNA clades dating to the late Pleistocene, when glacial advances may have isolated crow populations in separate refugia. We document extensive hybridization, with geographic overlap of mtDNA clades and admixture of nuDNA across &amp;gt;1,400 km of western Washington and western British Columbia. This broad hybrid zone consists of late-generation hybrids and backcrosses, not recent (e.g., F1) hybrids. Nuclear DNA and mtDNA clines were both centered in southwestern British Columbia, farther north than previously postulated. The mtDNA cline was narrower than the nuDNA cline, consistent with Haldane’s rule but not sex-biased dispersal. Overall, our results suggest a history of reticulate evolution in American and Northwestern crows, consistent with potentially recurring neutral expansion(s) from Pleistocene glacial refugia followed by lineage fusion(s). However, we do not rule out a contributing role for more recent potential drivers of hybridization, such as expansion into human-modified habitats.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/2018/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/2018/page/3/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="../../../../tags/2018/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2018/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2018/page/3/" class="page-link">3</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/2018/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2018/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2018/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2018/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2018/page/8/" class="page-link">8</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/2018/page/5/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/2018/page/84/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
