<!DOCTYPE html>
<html><head>
  
  <title>biorxiv | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.68.3" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.95d2483cd3c5d2bf47c1f36307a456787d60ad958c5efe44db9afc47012b1b05b8e1f64f6ac291650439322a6c38e86eb7cd6254255516d857c23f299fb0e666.js" integrity="sha512-ldJIPNPF0r9HwfNjB6RWeH1grZWMXv5E25r8RwErGwW44fZPasKRZQQ5MipsOOhut81iVCVVFthXwj8pn7DmZg=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/land_use_impacts_poison_frog_chemical_defenses_through_changes_in_leaf_litter_ant_communities_65690558/">
          <h4 class="font-weight-bold">Land use impacts poison frog chemical defenses through changes in leaf litter ant communities, bioRxiv, 2019-08-31</h4>
          <p>AbstractMuch of the world’s biodiversity is held within tropical rainforests, which are increasingly fragmented by agricultural practices. In these threatened landscapes, there are many organisms that acquire chemical defenses from their diet and are therefore intimately connected with their local food webs. Poison frogs (Family Dendrobatidae) are one such example, as they acquire alkaloid-based chemical defenses from their diet of leaf litter ants and mites. It is currently unknown how habitat fragmentation impacts chemical defense across trophic levels, from arthropods to frogs. Thus, we examined the chemical defenses and diets of the Diablito poison frog (Oophaga sylvatica), and the diversity of their leaf litter ant communities in secondary forest and reclaimed cattle pasture. We found that forest and pasture frogs differed in diet and alkaloid profiles, where forest frogs contained more of specific alkaloids and ate more ants. We also found that the leaf litter of forest and pasture frog habitats differed in ant community structure. Finally, ant species composition of frog diets resembled the surrounding leaf litter, but diets were less variable. This suggests that frogs tend to consume particular ant species within each habitat. To better understand how ants contribute to the alkaloid chemical profiles of frogs, we chemically profiled several ant species and found some alkaloids to be common across many ant species while others are restricted to a few species. Our experiments are the first to link anthropogenic land use changes to dendrobatid poison frog chemical defenses through variation in leaf litter communities, which has implications for conservation management of these threatened amphibians.ResumenLos bosques tropicales, que mantienen la mayor parte de la biodiversidad del planeta, están cada vez más fragmentados por diferentes prácticas agrícolas. En estos paisajes amenazados por la actividad humana hay organismos que acumulan defensas químicas a partir de su dieta. Las ranas venenosas (Familia Dendrobatidae) son un ejemplo de esto, y adquieren de su dieta, formada principalmente de hormigas de la hojarasca y de ácaros, diferentes alcaloides que pueden ser usados como defensas químicas. Las ranas venenosas están por lo tanto íntimamente conectadas a las redes tróficas locales. Actualmente es desconocido como la fragmentación del hábitat modifica la defensa por químicos a través de cambios en las cadena tróficas. Por lo tanto, examinamos las defensas químicas y dietas de la rana diablito (Oophaga sylvatica), y las comunidades de hormigas de la hojarasca, en un bosque secundario y en un pastizal cercano. Encontramos que las dietas y los perfiles de alcaloides en la piel de las ranas en bosque y en pastizal eran diferentes, con las ranas de bosque comiendo más hormigas y acumulando más alcaloides específicos. También encontramos que la hojarasca del bosque y del pastizal mantenían comunidades de hormigas con estructuras distintas. Finalmente, la composición de las hormigas en las dietas se parecían que la composición de hormigas en sus alrededores, pero dietas menos variables. Ello sugiere que las ranas tienden a consumir sólo ciertas especies de hormigas en cada hábitat. Para entender mejor cómo las hormigas contribuyen a los perfiles de alcaloides de las ranas, obtuvimos perfiles de los alcaloides presentes en algunas especies de hormigas y encontramos que algunos alcaloides son comunes a muchas especies de hormigas, y otros alcaloides están restringidos a pocas especies. Nuestros experimentos son los primeros en vincular los cambios antropogénicos en el uso de suelo con cambios en las defensas químicas de las ranas venenosas a través de cambios en las comunidades de la hojarasca, lo que tiene implicaciones para la conservación de estos anfibios altamente amenazados.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/ca2&#43;_signaling_in_astrocytes_is_sleep-wake_state_specific_and_modulates_sleep_65632238/">
          <h4 class="font-weight-bold">Ca2&#43; signaling in astrocytes is sleep-wake state specific and modulates sleep, bioRxiv, 2019-08-30</h4>
          <p>SummaryAstrocytic Ca2&#43; signaling has been intensively studied in health and disease but remains uncharacterized in sleep. Here, we employed a novel activity-based algorithm to assess astrocytic Ca2&#43; signals in the barrel cortex of awake and naturally sleeping mice while monitoring neuronal Ca2&#43; activity, brain rhythms and behavior. We discovered that Ca2&#43; signaling in astrocytes exhibits distinct features across the sleep-wake cycle and is reduced in sleep compared to wakefulness. Moreover, an increase in astrocytic Ca2&#43; signaling precedes transitions from slow-wave sleep to wakefulness, with a peak upon awakening exceeding the levels during whisking and locomotion. Genetic ablation of a key astrocytic Ca2&#43; signaling pathway resulted in fragmentation of slow-wave sleep, yet increased the frequency of sleep spindles. Our findings suggest a role for astrocytic Ca2&#43; signaling in modulating sleep.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/insight_into_the_genomic_history_of_the_near_east_from_whole-genome_sequences_and_genotypes_of_yemenis_65613878/">
          <h4 class="font-weight-bold">Insight into the genomic history of the Near East from whole-genome sequences and genotypes of Yemenis, bioRxiv, 2019-08-29</h4>
          <p>AbstractWe report high-coverage whole-genome sequencing data from 46 Yemeni individuals as well as genome-wide genotyping data from 169 Yemenis from diverse locations. We use this dataset to define the genetic diversity in Yemen and how it relates to people elsewhere in the Near East. Yemen is a vast region with substantial cultural and geographic diversity, but we found little genetic structure correlating with geography among the Yemenis – probably reflecting continuous movement of people between the regions. African ancestry from admixture in the past 800 years is widespread in Yemen and is the main contributor to the country’s limited genetic structure, with some individuals in Hudayda and Hadramout having up to 20% of their genetic ancestry from Africa. In contrast, individuals from Maarib appear to have been genetically isolated from the African gene flow and thus have genomes likely to reflect Yemen’s ancestry before the admixture. This ancestry was comparable to the ancestry present during the Bronze Age in the distant Northern regions of the Near East. After the Bronze Age, the South and North of the Near East therefore followed different genetic trajectories in the North the Levantines admixed with a Eurasian population carrying steppe ancestry whose impact never reached as far south as the Yemen, where people instead admixed with Africans leading to the genetic structure observed in the Near East today.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/optimization_of_ascas12a_for_combinatorial_genetic_screens_in_human_cells_65564690/">
          <h4 class="font-weight-bold">Optimization of AsCas12a for combinatorial genetic screens in human cells, bioRxiv, 2019-08-28</h4>
          <p>ABSTRACTCas12a enzymes have attractive properties for scalable delivery of multiplexed perturbations, yet widespread usage has lagged behind Cas9-based strategies. Here we describe the optimization of Cas12a from Acidaminococcus (AsCas12a) for use in pooled genetic screens in human cells. By assaying the activity of thousands of guides, we confirm on-target design rules and extend them to an enhanced activity variant, enAsCas12a. We also develop the first comprehensive set of off-target rules for Cas12a, and demonstrate that we can predict and exclude promiscuous guides. Finally, to enable efficient higher-order multiplexing via lentiviral delivery, we screen thousands of direct repeat variants and identify 38 that outperform the wildtype sequence. We validate this optimized AsCas12a toolkit by targeting 12 synthetic lethal gene pairs with up to 400 guide pairs each, and demonstrate effective triple knockout via flow cytometry. These results establish AsCas12a as a robust system for combinatorial applications of CRISPR technology.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_molecular_cell_atlas_of_the_human_lung_from_single_cell_rna_sequencing_65524741/">
          <h4 class="font-weight-bold">A molecular cell atlas of the human lung from single cell RNA sequencing, bioRxiv, 2019-08-27</h4>
          <p>AbstractAlthough single cell RNA sequencing studies have begun providing compendia of cell expression profiles, it has proven more difficult to systematically identify and localize all molecular cell types in individual organs to create a full molecular cell atlas. Here we describe droplet- and plate-based single cell RNA sequencing applied to ∼70,000 human lung and blood cells, combined with a multi-pronged cell annotation approach, which have allowed us to define the gene expression profiles and anatomical locations of 58 cell populations in the human lung, including 41 of 45 previously known cell types or subtypes and 14 new ones. This comprehensive molecular atlas elucidates the biochemical functions of lung cell types and the cell-selective transcription factors and optimal markers for making and monitoring them; defines the cell targets of circulating hormones and predicts local signaling interactions including sources and targets of chemokines in immune cell trafficking and expression changes on lung homing; and identifies the cell types directly affected by lung disease genes. Comparison to mouse identified 17 molecular types that appear to have been gained or lost during lung evolution and others whose expression profiles have been substantially altered, revealing extensive plasticity of cell types and cell-type-specific gene expression during organ evolution including expression switches between cell types. This lung atlas provides the molecular foundation for investigating how lung cell identities, functions, and interactions are achieved in development and tissue engineering and altered in disease and evolution.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genomescope_2.0_and_smudgeplots_reference-free_profiling_of_polyploid_genomes_65479920/">
          <h4 class="font-weight-bold">GenomeScope 2.0 and Smudgeplots Reference-free profiling of polyploid genomes, bioRxiv, 2019-08-27</h4>
          <p>AbstractAn important assessment prior to genome assembly and related analyses is genome profiling, where the k-mer frequencies within raw sequencing reads are analyzed to estimate major genome characteristics such as genome size, heterozygosity, and repetitiveness. Here we introduce GenomeScope 2.0 (&lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsgithub.comtbenavi1genomescope2.0&gt;httpsgithub.comtbenavi1genomescope2.0&lt;jatsext-link&gt;), which applies combinatorial theory to establish a detailed mathematical model of how k-mer frequencies are distributed in heterozygous and polyploid genomes. We describe and evaluate a practical implementation of the polyploid-aware mixture model that, within seconds, accurately infers genome properties across thousands of simulated and eleven real datasets spanning a broad range of complexity. We also present a new method called Smudgeplots (&lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsgithub.comKamilSJaronsmudgeplot&gt;httpsgithub.comKamilSJaronsmudgeplot&lt;jatsext-link&gt;) to visualize and infer the ploidy and genome structure of a genome by analyzing heterozygous k-mer pairs. We successfully apply the approach to systems of known variable ploidy levels in the Meloidogyne genus and also the extreme case of octoploid Fragaria x ananassa.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/29/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/26/" class="page-link">26</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/27/" class="page-link">27</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/28/" class="page-link">28</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/29/" class="page-link">29</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/30/" class="page-link">30</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/31/" class="page-link">31</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/32/" class="page-link">32</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/33/" class="page-link">33</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/34/" class="page-link">34</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/31/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/312/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
