<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/specialized_and_spatially_organized_coding_of_sensory_motor_and_cognitive_variables_in_midbrain_dopamine_neurons_50453798/">
          <h4 class="font-weight-bold">Specialized and spatially organized coding of sensory, motor, and cognitive variables in midbrain dopamine neurons, bioRxiv, 2018-10-29</h4>
          <p>There is increased appreciation that dopamine (DA) neurons in the midbrain respond not only to reward 1,2 and reward-predicting cues 1,3,4, but also to other variables such as distance to reward 5, movements 6–11 and behavioral choices 12–15. Based on these findings, a major open question is how the responses to these diverse variables are organized across the population of DA neurons. In other words, do individual DA neurons multiplex multiple variables, or are subsets of neurons specialized in encoding specific behavioral variables? The reason that this fundamental question has been difficult to resolve is that recordings from large populations of individual DA neurons have not been performed in a behavioral task with sufficient complexity to examine these diverse variables simultaneously. To address this gap, we used 2-photon calcium imaging through an implanted lens to record activity of &amp;gt;300 midbrain DA neurons in the VTA during a complex decision-making task. As mice navigated in a virtual reality (VR) environment, DA neurons encoded an array of sensory, motor, and cognitive variables. These responses were functionally clustered, such that subpopulations of neurons transmitted information about a subset of behavioral variables, in addition to encoding reward. These functional clusters were spatially organized, such that neighboring neurons were more likely to be part of the same cluster. Taken together with the topography between DA neurons and their projections, this specialization and anatomical organization may aid downstream circuits in correctly interpreting the wide range of signals transmitted by DA neurons.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/connect-seq_to_superimpose_molecular_on_anatomical_neural_circuit_maps_50380997/">
          <h4 class="font-weight-bold">Connect-seq to superimpose molecular on anatomical neural circuit maps, bioRxiv, 2018-10-27</h4>
          <p>AbstractThe mouse brain contains ~100 million neurons interconnected in a vast array of neural circuits. The identities and functions of individual neuronal components of most circuits are undefined. Here we describe a method, termed ‘Connect-seq’, which combines retrograde viral tracing and single cell transcriptomics to uncover the molecular identities of upstream neurons in a specific circuit and the signaling molecules they use to communicate. Connect-seq can generate a molecular map that can be superimposed on a neuroanatomical map to permit molecular and genetic interrogation of how the neuronal components of a circuit control its function. Application of this method to hypothalamic neurons controlling physiological responses to fear and stress reveal subsets of upstream neurons that express diverse constellations of signaling molecules and can be distinguished by their anatomical locations.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/etdb-caltech_a_blockchain-based_distributed_public_database_for_electron_tomography_50267721/">
          <h4 class="font-weight-bold">ETDB-Caltech a blockchain-based distributed public database for electron tomography, bioRxiv, 2018-10-26</h4>
          <p>AbstractThree-dimensional electron microscopy techniques like electron tomography provide valuable insights into cellular structures, and present significant challenges for data storage and dissemination. Here we explored a novel method to publicly release more than 11,000 such datasets, more than 30 TB in total, collected by our group. Our method, based on a peer-to-peer file sharing network built around a blockchain ledger, offers a distributed solution to data storage. In addition, we offer a user-friendly browser-based interface, &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsetdb.caltech.edu&gt;httpsetdb.caltech.edu&lt;jatsext-link&gt;, for anyone interested to explore and download our data. We discuss the relative advantages and disadvantages of this system and provide tools for other groups to mine our data andor use the same approach to share their own imaging datasets.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/inference_of_recombination_maps_from_a_single_pair_of_genomes_and_its_application_to_archaic_samples_50219053/">
          <h4 class="font-weight-bold">Inference of recombination maps from a single pair of genomes and its application to archaic samples, bioRxiv, 2018-10-25</h4>
          <p>ABSTRACTUnderstanding the causes and consequences of recombination rate evolution is a fundamental goal in genetics that requires recombination maps from across the tree of life. Since statistical inference of recombination maps typically depends on large samples, reaching out studies to non-model organisms requires alternative tools. Here we extend the sequentially Markovian coalescent model to jointly infer demography and the variation in recombination along a pair of genomes. Using extensive simulations and sequence data from humans, fruit-flies and a fungal pathogen, we demonstrate that iSMC accurately infers recombination maps under a wide range of scenarios – remarkably, even from a single pair of unphased genomes. We exploit this possibility and reconstruct the recombination maps of archaic hominids. We report that the evolution of the recombination landscape follows the established phylogeny of Neandertals, Denisovans and modern human populations, as expected if the genomic distribution of crossovers in hominids is largely neutral.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/independent_domestication_events_in_the_blue-cheese_fungus_penicillium_roqueforti_50177653/">
          <h4 class="font-weight-bold">Independent domestication events in the blue-cheese fungus Penicillium roqueforti, bioRxiv, 2018-10-24</h4>
          <p>AbstractDomestication provides an excellent framework for studying adaptive divergence. Using population genomics and phenotypic assays, we reconstructed the domestication history of the blue cheese mold Penicillium roqueforti. We showed that this fungus was domesticated twice independently. The population used in Roquefort originated from an old domestication event associated with weak bottlenecks and exhibited traits beneficial for pre-industrial cheese production (slower growth in cheese and greater spore production on bread, the traditional multiplication medium). The other cheese population originated more recently from the selection of a single clonal lineage, was associated to all types of blue cheese worldwide but Roquefort, and displayed phenotypes more suited for industrial cheese production (high lipolytic activity, efficient cheese cavity colonization ability and salt tolerance). We detected genomic regions affected by recent positive selection and putative horizontal gene transfers. This study sheds light on the processes of rapid adaptation and raises questions about genetic resource conservation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/latent_developmental_potential_to_form_limb-like_skeletal_structures_in_zebrafish_50117443/">
          <h4 class="font-weight-bold">Latent developmental potential to form limb-like skeletal structures in zebrafish, bioRxiv, 2018-10-23</h4>
          <p>AbstractThe evolution of fins into limbs was a key transition in vertebrate history. A hallmark of this transition is the addition of multiple long bones to the proximal-distal axis of paired appendages. Whereas limb skeletons are often elaborate and diverse, teleost pectoral fins retain a simple endoskeleton. Fins and limbs share many core developmental processes, but how these programs were reshaped to produce limbs from fins during evolution remains enigmatic. Here we identify zebrafish mutants that form supernumerary long bones along the proximal-distal axis of pectoral fins with limb-like patterning. These new skeletal elements are integrated into the fin, as they are connected to the musculature, form joints, and articulate with neighboring bones. This phenotype is caused by activating mutations in previously unrecognized regulators of appendage development, vav2 and waslb, which we show function in a common pathway. We find that this pathway functions in appendage development across vertebrates, and loss of Wasl in developing limbs results in patterning defects identical to those seen in Hoxall knockout mice. Concordantly, formation of supernumerary fin long bones requires the function of hoxall paralogs, indicating developmental homology with the forearm and the existence of a latent functional Hox code patterning the fin endoskeleton. Our findings reveal an inherent limb-like patterning ability in fins that can be activated by simple genetic perturbation, resulting in the elaboration of the endoskeleton.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/0-100-users/page/62/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/59/" class="page-link">59</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/60/" class="page-link">60</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/61/" class="page-link">61</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/62/" class="page-link">62</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/0-100-users/page/63/" class="page-link">63</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/64/" class="page-link">64</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/65/" class="page-link">65</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/66/" class="page-link">66</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/67/" class="page-link">67</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/0-100-users/page/64/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/0-100-users/page/123/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
