<!DOCTYPE html>
<html><head>
  
  <title>Molecular Biology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/principles_of_meiotic_chromosome_assembly_49623064/">
          <h4 class="font-weight-bold">Principles of Meiotic Chromosome Assembly, bioRxiv, 2018-10-13</h4>
          <p>During meiotic prophase, chromosomes organise into a series of chromatin loops emanating from a proteinaceous axis, but the mechanisms of assembly remain unclear. Here we elucidate how this elaborate three-dimensional chromosome organisation is underpinned by genomic sequence in Saccharomyces cerevisiae. Entering meiosis, strong cohesin-dependent grid-like Hi-C interaction patterns emerge, reminiscent of mammalian interphase organisation, but with distinct regulation. Meiotic patterns agree with simulations of loop extrusion limited by barriers, yet are patterned by convergent transcription rather than binding of the mammalian interphase factor, CTCF, which is absent in S. cerevisiae - thereby both challenging and extending current paradigms of local chromosome organisation. While grid-like interactions emerge independently of meiotic chromosome synapsis, synapsis itself generates additional compaction that matures differentially according to telomere proximity and chromosome size. Collectively, our results elucidate fundamental principles of chromosome assembly and demonstrate the essential role of cohesin within this evolutionarily conserved process.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/altered_chromatin_localization_of_hybrid_lethality_proteins_in_drosophila_49389027/">
          <h4 class="font-weight-bold">Altered chromatin localization of hybrid lethality proteins in Drosophila, bioRxiv, 2018-10-09</h4>
          <p>AbstractUnderstanding hybrid incompatibilities is a fundamental pursuit in evolutionary genetics. In crosses between Drosophila melanogaster females and Drosophila simulans males, the interaction of at least three genes is necessary for hybrid male lethality Hmr mel, Lhr sim, and gfzf sim. All three hybrid incompatibility genes are chromatin associated factors. While HMR and LHR physically bind each other and function together in a single complex, the connection between either of these proteins and gfzf remains mysterious. Here, we investigate the allele specific chromatin binding patterns of gfzf. First, our cytological analyses show that there is little difference in protein localization of GFZF between the two species except at telomeric sequences. In particular, GFZF binds the telomeric retrotransposon repeat arrays, and the differential binding of GFZF at telomeres reflects the rapid changes in sequence composition at telomeres between D. melanogaster and D. simulans. Second, we investigate the patterns of GFZF and HMR co-localization and find that the two proteins do not normally co-localize in D. melanogaster. However, in inter-species hybrids, HMR shows extensive mis-localization to GFZF sites, and this altered localization requires the presence of gfzf sim. Third, we find by ChIP-Seq that over-expression of HMR and LHR within species is sufficient to cause HMR to mis-localize to GFZF binding sites, indicating that HMR has a natural low affinity for GFZF sites. Together, these studies provide the first insights into the different properties of gfzf between D. melanogaster and D. simulans as well as a molecular interaction between gfzf and Hmr in the form of altered protein localization.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/slow_transcriptional_elongation_causes_embryonic_lethality_and_perturbs_kinetic_coupling_of_long_neural_genes_48912400/">
          <h4 class="font-weight-bold">Slow transcriptional elongation causes embryonic lethality and perturbs kinetic coupling of long neural genes, bioRxiv, 2018-09-26</h4>
          <p>The rate of RNA Polymerase II (RNAPII) elongation has an important role in the control of Alternative splicing (AS); however, the in vivo consequences of an altered elongation rate are unknown. Here, we generated mouse embryonic stem cells (ESCs) knocked-in for a slow elongating form of RNAPII. We show that a reduced transcriptional elongation rate results in early embryonic lethality in mice and impairs the differentiation of ESCs into the neural lineage. This is accompanied by changes in splicing and in gene expression in ESCs and along the pathway of neuronal differentiation. In particular, we found a crucial role for RNAPII elongation rate in transcription and splicing of long neuronal genes involved in synapse signaling. The impact of the kinetic coupling of RNAPII elongation rate with AS is more predominant in ESC-differentiated neurons than in pluripotent cells. Our results demonstrate the requirement for an appropriate transcriptional elongation rate to ensure proper gene expression and to regulate AS during development.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/mito-tag_mice_enable_rapid_isolation_and_multimodal_profiling_of_mitochondria_from_specific_cell_types_in_vivo_48845489/">
          <h4 class="font-weight-bold">MITO-Tag Mice enable rapid isolation and multimodal profiling of mitochondria from specific cell types in vivo, bioRxiv, 2018-09-24</h4>
          <p>ABSTRACTMitochondria are metabolic organelles that are essential for mammalian life, but the dynamics of mitochondrial metabolism within mammalian tissues in vivo remains incompletely understood. While whole-tissue metabolite profiling has been useful for studying metabolism in vivo, such an approach lacks resolution at the cellular and subcellular level. In vivo methods for interrogating organellar metabolites in specific cell-types within mammalian tissues have been limited. To address this, we built on prior work in which we exploited a mitochondrially-localized 3XHA epitope-tag (“MITO-Tag”) for the fast isolation of mitochondria from cultured cells to now generate “MITO-Tag Mice.” Affording spatiotemporal control over MITO-Tag expression, these transgenic animals enable the rapid, cell-type-specific immunoisolation of mitochondria from tissues, which we verified using a combination of proteomic and metabolomic approaches. Using MITO-Tag Mice and targeted and untargeted metabolite profiling, we identified changes during fasted and refed conditions in a diverse array of mitochondrial metabolites in hepatocytes and found metabolites that behaved differently at the mitochondrial versus whole-tissue level. MITO-Tag Mice should have utility for studying mitochondrial physiology and our strategy should be generally applicable for studying other mammalian organelles in specific cell-types in vivo.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/bio-on-magnetic-beads_bomb_open_platform_for_high-throughput_nucleic_acid_extraction_and_manipulation_48184997/">
          <h4 class="font-weight-bold">Bio-On-Magnetic-Beads (BOMB) Open platform for high-throughput nucleic acid extraction and manipulation, bioRxiv, 2018-09-13</h4>
          <p>AbstractCurrent molecular biology laboratories rely heavily on the purification and manipulation of nucleic acids. Yet, commonly used centrifuge-and column-based protocols require specialised equipment, often use toxic reagents and are not economically scalable or practical to use in a high-throughput manner. Although it has been known for some time that magnetic beads can provide an elegant answer to these issues, the development of open-source protocols based on beads has been limited. In this article, we provide step-by-step instructions for an easy synthesis of functionalised magnetic beads, and detailed protocols for their use in the high-throughput purification of plasmids, genomic DNA and total RNA from different sources, as well as environmental TNA and PCR amplicons. We also provide a bead-based protocol for bisulfite conversion, and size selection of DNA and RNA fragments. Comparison to other methods highlights the capability, versatility and extreme cost-effectiveness of using magnetic beads. These open source protocols and the associated webpage (&lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsbomb.bio&gt;httpsbomb.bio&lt;jatsext-link&gt;) can serve as a platform for further protocol customisation and community engagement.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/precise_temporal_regulation_of_alternative_splicing_during_neural_development_31667630/">
          <h4 class="font-weight-bold">Precise temporal regulation of alternative splicing during neural development, bioRxiv, 2018-01-15</h4>
          <p>AbstractAlternative splicing (AS) is a crucial step of gene expression that must be tightly controlled, but the precise timing of dynamic splicing switches during neural development and the underlying regulatory mechanisms are poorly understood. Here we systematically analyzed the temporal regulation of AS in a large number of transcriptome profiles of developing mouse cortices, in vivo purified neuronal subtypes, and neurons differentiated in vitro. Our analysis revealed early- and late-switch exons in genes with distinct functions, and these switches accurately define neuronal maturation stages. Integrative modeling suggests that these switches are under direct and combinatorial regulation by distinct sets of neuronal RNA-binding proteins including Nova, Rbfox, Mbnl and Ptbp. Surprisingly, various neuronal subtypes in the sensory systems lack Nova andor Rbfox expression. These neurons retain the “immature” splicing program in early-switch exons, affecting numerous synaptic genes. These results provide new insights into the organization and regulation of the neurodevelopmental transcriptome.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/molecular-biology/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/molecular-biology/page/3/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="/audiences/tags/molecular-biology/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/molecular-biology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/molecular-biology/page/3/" class="page-link">3</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/molecular-biology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/molecular-biology/page/5/" class="page-link">5</a></li>
            
        

        
        <li class="page-item"><a href="/audiences/tags/molecular-biology/page/5/" rel="next" class="page-link">Next ›</a></li>
        

        
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
