<!DOCTYPE html>
<html><head>
  
  <title>Reports | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_high-resolution_chromosome-assigned_komodo_dragon_genome_reveals_adaptations_in_the_cardiovascular_muscular_and_chemosensory_systems_of_monitor_lizards_55575714/">
          <h4 class="font-weight-bold">A high-resolution, chromosome-assigned Komodo dragon genome reveals adaptations in the cardiovascular, muscular, and chemosensory systems of monitor lizards, bioRxiv, 2019-02-16</h4>
          <p>Monitor lizards are unique among ectothermic reptiles in that they have a high aerobic capacity and distinctive cardiovascular physiology which resembles that of endothermic mammals. We have sequenced the genome of the Komodo dragon (Varanus komodoensis), the largest extant monitor lizard, and present a high resolution de novo chromosome-assigned genome assembly for V. komodoensis, generated with a hybrid approach of long-range sequencing and single molecule physical mapping. Comparing the genome of V. komodoensis with those of related species showed evidence of positive selection in pathways related to muscle energy metabolism, cardiovascular homeostasis, and thrombosis. We also found species-specific expansions of a chemoreceptor gene family related to pheromone and kairomone sensing in V. komodoensis and several other lizard lineages. Together, these evolutionary signatures of adaptation reveal genetic underpinnings of the unique Komodo sensory, cardiovascular, and muscular systems, and suggest that selective pressure altered thrombosis genes to help Komodo dragons evade the anticoagulant effects of their own saliva. As the only sequenced monitor lizard genome, the Komodo dragon genome is an important resource for understanding the biology of this lineage and of reptiles worldwide.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/cryptic_inoviruses_are_pervasive_in_bacteria_and_archaea_across_earths_biomes_55575446/">
          <h4 class="font-weight-bold">Cryptic inoviruses are pervasive in bacteria and archaea across Earth&#39;s biomes, bioRxiv, 2019-02-16</h4>
          <p>Bacteriophages from the Inoviridae family (inoviruses) are characterized by their unique morphology, genome content, and infection cycle. To date, a relatively small number of inovirus isolates have been extensively studied, either for biotechnological applications such as phage display, or because of their impact on the toxicity of known bacterial pathogens including Vibrio cholerae and Neisseria meningitidis. Here we show that the current 56 members of the Inoviridae family represent a minute fraction of a highly diverse group of inoviruses. Using a new machine learning approach leveraging a combination of marker gene and genome features, we identified 10,295 inovirus-like genomes from microbial genomes and metagenomes. Collectively, these represent six distinct proposed inovirus families infecting nearly all bacterial phyla across virtually every ecosystem. Putative inoviruses were also detected in several archaeal genomes, suggesting that these viruses may have occasionally transferred from bacterial to archaeal hosts. Finally, we identified an expansive diversity of inovirus-encoded  toxin-antitoxin and gene expression modulation systems, alongside evidence of both synergistic (CRISPR evasion) and antagonistic (superinfection exclusion) interactions with co-infecting viruses which we experimentally validated in a Pseudomonas model. Capturing this previously obscured component of the global virosphere sparks new avenues for microbial manipulation approaches and innovative biotechnological applications.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/cryptic_inoviruses_are_pervasive_in_bacteria_and_archaea_across_earths_biomes_55575446/">
          <h4 class="font-weight-bold">Cryptic inoviruses are pervasive in bacteria and archaea across Earthâ€™s biomes, bioRxiv, 2019-02-16</h4>
          <p>AbstractBacteriophages from the Inoviridae family (inoviruses) are characterized by their unique morphology, genome content, and infection cycle. To date, a relatively small number of inovirus isolates have been extensively studied, either for biotechnological applications such as phage display, or because of their impact on the toxicity of known bacterial pathogens including Vibrio cholerae and Neisseria meningitidis. Here we show that the current 56 members of the Inoviridae family represent a minute fraction of a highly diverse group of inoviruses. Using a new machine learning approach leveraging a combination of marker gene and genome features, we identified 10,295 inovirus-like genomes from microbial genomes and metagenomes. Collectively, these represent six distinct proposed inovirus families infecting nearly all bacterial phyla across virtually every ecosystem. Putative inoviruses were also detected in several archaeal genomes, suggesting that these viruses may have occasionally transferred from bacterial to archaeal hosts. Finally, we identified an expansive diversity of inovirus-encoded toxin-antitoxin and gene expression modulation systems, alongside evidence of both synergistic (CRISPR evasion) and antagonistic (superinfection exclusion) interactions with co-infecting viruses which we experimentally validated in a Pseudomonas model. Capturing this previously obscured component of the global virosphere sparks new avenues for microbial manipulation approaches and innovative biotechnological applications.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/enterococcus_faecium_genome_dynamics_during_long-term_asymptomatic_patient_gut_colonization_55572932/">
          <h4 class="font-weight-bold">Enterococcus faecium genome dynamics during long-term asymptomatic patient gut colonization, bioRxiv, 2019-02-16</h4>
          <p>Background E. faecium is a gut commensal of humans and animals. In addition, it has recently emerged as an important nosocomial pathogen through the acquisition of genetic elements that confer resistance to antibiotics and virulence. We performed a whole-genome sequencing based study on 96 multidrug-resistant E. faecium strains that asymptomatically colonized five patients with the aim to describe the genome dynamics of this species. Results The patients were hospitalized on multiple occasions and isolates were collected over periods ranging from 15 months to 6.5 years. Ninety-five of the sequenced isolates belonged to E. faecium clade A1, which was previously determined to be responsible for the vast majority of clinical infections. The clade A1 strains clustered into six clonal groups of highly similar isolates, three of which entirely consisted of isolates from a single patient. We also found evidence of concurrent colonization of patients by multiple distinct lineages and transfer of strains between patients during hospitalisation. We estimated the evolutionary rate of two clonal groups that colonized a single patient at 12.6 and 25.2 single nucleotide polymorphisms (SNPs)genomeyear. A detailed analysis of the accessory genome of one of the clonal groups revealed considerable variation due to gene gain and loss events, including the chromosomal acquisition of a 37 kbp prophage and the loss of an element containing carbohydrate metabolism-related genes. We determined the presence and location of twelve different Insertion Sequence (IS) elements, with ISEfa5 showing a unique pattern of location in 24 of the 25 isolates, suggesting widespread ISEfa5 excision and insertion into the genome during gut colonization. Conclusions Our findings show that the E. faecium genome is highly dynamic during asymptomatic colonization of the patient gut. We observe considerable genomic flexibility due to frequent horizontal gene transfer and recombination, which can contribute to the generation of genetic diversity within the species and, ultimately, can contribute to its success as a nosocomial pathogen.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/semi-quantitative_characterisation_of_mixed_pollen_samples_using_minion_sequencing_and_reverse_metagenomics_revmet_55573826/">
          <h4 class="font-weight-bold">Semi-quantitative characterisation of mixed pollen samples using MinION sequencing and Reverse Metagenomics (RevMet), bioRxiv, 2019-02-16</h4>
          <p>The ability to identify and quantify the constituent plant species that make up a mixed-species sample of pollen has important applications in ecology, conservation, and agriculture. Recently, metabarcoding protocols have been developed for pollen that can identify constituent plant species, but there are strong reasons to doubt that metabarcoding can accurately quantify their relative abundances. A PCR-free, shotgun metagenomics approach has greater potential for accurately quantifying species relative abundances, but applying metagenomics to eukaryotes is challenging due to low numbers of reference genomes. We have developed a pipeline, RevMet (Reverse Metagenomics), that allows reliable and semi-quantitative characterization of the species composition of mixed-species eukaryote samples, such as bee-collected pollen, without requiring reference genomes. Instead, reference species are represented only by &#39;genome skims&#39; low-cost, low-coverage, shortread sequence datasets. The skims are mapped to individual long reads sequenced from mixed-species samples using the MinION, a portable nanopore sequencing device, and each long read is uniquely assigned to a plant species. We genome-skimmed 49 wild UK plant species, validated our pipeline with mock DNA mixtures of known composition, and then applied RevMet to pollen loads collected from wild bees. We demonstrate that RevMet can identify plant species present in mixed-species samples at proportions of DNA &amp;gt;1%, with few false positives and false negatives, and reliably differentiate species represented by high versus low amounts of DNA in a sample. The RevMet pipeline could readily be adapted to generate semi-quantitative datasets for a wide range of mixed eukaryote samples, which could include characterising diets, quantifying allergenic pollen from air samples, quantifying soil fauna, and identifying the compositions of algal and diatom communities. Our per-sample costs were GBP 90 per genome skim and GBP 60 per pollen sample, and new versions of sequencers available now will further reduce these costs.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_method_for_genome-wide_genealogy_estimation_for_thousands_of_samples_55484262/">
          <h4 class="font-weight-bold">A method for genome-wide genealogy estimation for thousands of samples, bioRxiv, 2019-02-15</h4>
          <p>Knowledge of genome-wide genealogies for thousands of individuals would simplify most evolutionary analyses for humans and other species, but has remained computationally infeasible. We developed a method, Relate, scaling to &amp;gt; 10,000 sequences while simultaneously estimating branch lengths, mutational ages,  and variable historical population sizes, as well as allowing for data errors. Application to 1000 Genomes Project haplotypes produces joint genealogical histories for 26 human populations. Highly diverged lineages are present in all groups, but most frequent in Africa. Outside Africa, these mainly reflect ancient introgression from groups related to Neanderthals and Denisovans, while African signals instead reflect unknown events, unique to that continent. Our approach allows more powerful inferences of natural selection than previously possible. We identify multiple novel regions under strong positive selection, and multi-allelic traits including hair colour, BMI, and blood pressure, showing strong evidence of directional selection, varying among human groups.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../reports/" rel="first" class="page-link">Â« First</a></li>
        

        
        <li class="page-item"><a href="../../../reports/page/121/" rel="prev" class="page-link">â€¹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/118/" class="page-link">118</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/119/" class="page-link">119</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/120/" class="page-link">120</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/121/" class="page-link">121</a></li>
            
        
             
            <li class="page-item active"><a href="../../../reports/page/122/" class="page-link">122</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/123/" class="page-link">123</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/124/" class="page-link">124</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/125/" class="page-link">125</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/126/" class="page-link">126</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../reports/page/123/" rel="next" class="page-link">Next â€º</a></li>
        

        
        
        <li class="page-item"><a href="../../../reports/page/320/" rel="last" class="page-link">Last Â»</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
