<!DOCTYPE html>
<html><head>
  
  <title>Reports | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.68.3" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.95d2483cd3c5d2bf47c1f36307a456787d60ad958c5efe44db9afc47012b1b05b8e1f64f6ac291650439322a6c38e86eb7cd6254255516d857c23f299fb0e666.js" integrity="sha512-ldJIPNPF0r9HwfNjB6RWeH1grZWMXv5E25r8RwErGwW44fZPasKRZQQ5MipsOOhut81iVCVVFthXwj8pn7DmZg=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/saver_gene_expression_recovery_for_umi-based_single_cell_rna_sequencing_20204200/">
          <h4 class="font-weight-bold">SAVER Gene expression recovery for UMI-based single cell RNA sequencing, bioRxiv, 2017-05-17</h4>
          <p>AbstractRapid advances in massively parallel single cell RNA sequencing (scRNA-seq) is paving the way for high-resolution single cell profiling of biological samples. In most scRNA-seq studies, only a small fraction of the transcripts present in each cell are sequenced. The efficiency, that is, the proportion of transcripts in the cell that are sequenced, can be especially low in highly parallelized experiments where the number of reads allocated for each cell is small. This leads to unreliable quantification of lowly and moderately expressed genes, resulting in extremely sparse data and hindering downstream analysis. To address this challenge, we introduce SAVER (Single-cell Analysis Via Expression Recovery), an expression recovery method for scRNA-seq that borrows information across genes and cells to impute the zeros as well as to improve the expression estimates for all genes. We show, by comparison to RNA fluorescence in situ hybridization (FISH) and by data down-sampling experiments, that SAVER reliably recovers cell-specific gene expression concentrations, cross-cell gene expression distributions, and gene-to-gene and cell-to-cell correlations. This improves the power and accuracy of any downstream analysis involving genes with low to moderate expression.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/no_compelling_evidence_that_preferences_for_facial_masculinity_track_changes_in_womens_hormonal_status_20043216/">
          <h4 class="font-weight-bold">No compelling evidence that preferences for facial masculinity track changes in women’s hormonal status, bioRxiv, 2017-05-13</h4>
          <p>AbstractAlthough widely cited as strong evidence that sexual selection has shaped human facial attractiveness judgments, evidence that preferences for masculine characteristics in men’s faces are related to women’s hormonal status is equivocal and controversial. Consequently, we conducted the largest ever longitudinal study of the hormonal correlates of women’s preferences for facial masculinity (N=584). Analyses showed no compelling evidence that preferences for facial masculinity were related to changes in women’s salivary steroid hormone levels. Furthermore, both within-subject and between-subject comparisons showed no evidence that oral contraceptive use decreased masculinity preferences. However, women generally preferred masculinized over feminized versions of men’s faces, particularly when assessing men’s attractiveness for short-term, rather than long-term, relationships. Our results do not support the hypothesized link between women’s preferences for facial masculinity and their hormonal status.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/animal-behavior-and-cognition">animal-behavior-and-cognition</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/crispr-cas9_genome_editing_in_human_cells_works_via_the_fanconi_anemia_pathway_20023519/">
          <h4 class="font-weight-bold">CRISPR-Cas9 genome editing in human cells works via the Fanconi Anemia pathway, bioRxiv, 2017-05-12</h4>
          <p>AbstractCRISPR-Cas9 genome editing creates targeted double strand breaks (DSBs) in eukaryotic cells that are processed by cellular DNA repair pathways. Co-administration of single stranded oligonucleotide donor DNA (ssODN) during editing can result in high-efficiency (&amp;gt;20%) incorporation of ssODN sequences into the break site. This process is commonly referred to as homology directed repair (HDR) and here referred to as single stranded template repair (SSTR) to distinguish it from repair using a double stranded DNA donor (dsDonor). The high efficacy of SSTR makes it a promising avenue for the treatment of genetic diseases1,2, but the genetic basis of SSTR editing is still unclear, leaving its use a mostly empiric process. To determine the pathways underlying SSTR in human cells, we developed a coupled knockdown-editing screening system capable of interrogating multiple editing outcomes in the context of thousands of individual gene knockdowns. Unexpectedly, we found that SSTR requires multiple components of the Fanconi Anemia (FA) repair pathway, but does not require Rad51-mediated homologous recombination, distinguishing SSTR from repair using dsDonors. Knockdown of FA genes impacts SSTR without altering break repair by non-homologous end joining (NHEJ) in multiple human cell lines and in neonatal dermal fibroblasts. Our results establish an unanticipated and central role for the FA pathway in templated repair from single stranded DNA by human cells. Therapeutic genome editing has been proposed to treat genetic disorders caused by deficiencies in DNA repair, including Fanconi Anemia. Our data imply that patient genotype andor transcriptome profoundly impact the effectiveness of gene editing treatments and that adjuvant treatments to bias cells towards FA repair pathways could have considerable therapeutic value.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_beaker_phenomenon_and_the_genomic_transformation_of_northwest_europe_20019834/">
          <h4 class="font-weight-bold">The Beaker Phenomenon and the Genomic Transformation of Northwest Europe, bioRxiv, 2017-05-12</h4>
          <p>Bell Beaker pottery spread across western and central Europe beginning around 2750 BCE before disappearing between 2200–1800 BCE. The mechanism of its expansion is a topic of long-standing debate, with support for both cultural diffusion and human migration. We present new genome-wide ancient DNA data from 170 Neolithic, Copper Age and Bronze Age Europeans, including 100 Beaker-associated individuals. In contrast to the Corded Ware Complex, which has previously been identified as arriving in central Europe following migration from the east, we observe limited genetic affinity between Iberian and central European Beaker Complex-associated individuals, and thus exclude migration as a significant mechanism of spread between these two regions. However, human migration did have an important role in the further dissemination of the Beaker Complex, which we document most clearly in Britain using data from 80 newly reported individuals dating to 3900–1200 BCE. British Neolithic farmers were genetically similar to contemporary populations in continental Europe and in particular to Neolithic Iberians, suggesting that a portion of the farmer ancestry in Britain came from the Mediterranean rather than the Danubian route of farming expansion. Beginning with the Beaker period, and continuing through the Bronze Age, all British individuals harboured high proportions of Steppe ancestry and were genetically closely related to Beaker-associated individuals from the Lower Rhine area. We use these observations to show that the spread of the Beaker Complex to Britain was mediated by migration from the continent that replaced &amp;gt;90% of Britain’s Neolithic gene pool within a few hundred years, continuing the process that brought Steppe ancestry into central and northern Europe 400 years earlier.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_genomic_history_of_southeastern_europe_20019436/">
          <h4 class="font-weight-bold">The Genomic History Of Southeastern Europe, bioRxiv, 2017-05-12</h4>
          <p>AbstractFarming was first introduced to southeastern Europe in the mid-7th millennium BCE – brought by migrants from Anatolia who settled in the region before spreading throughout Europe. To clarify the dynamics of the interaction between the first farmers and indigenous hunter-gatherers where they first met, we analyze genome-wide ancient DNA data from 223 individuals who lived in southeastern Europe and surrounding regions between 12,000 and 500 BCE. We document previously uncharacterized genetic structure, showing a West-East cline of ancestry in hunter-gatherers, and show that some Aegean farmers had ancestry from a different lineage than the northwestern Anatolian lineage that formed the overwhelming ancestry of other European farmers. We show that the first farmers of northern and western Europe passed through southeastern Europe with limited admixture with local hunter-gatherers, but that some groups mixed extensively, with relatively sex-balanced admixture compared to the male-biased hunter-gatherer admixture that prevailed later in the North and West. Southeastern Europe continued to be a nexus between East and West after farming arrived, with intermittent genetic contact from the Steppe up to 2,000 years before the migration that replaced much of northern Europe’s population.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_next_generation_connectivity_map_l1000_platform_and_the_first_1000000_profiles_20025345/">
          <h4 class="font-weight-bold">A Next Generation Connectivity Map L1000 Platform And The First 1,000,000 Profiles, bioRxiv, 2017-05-11</h4>
          <p>SUMMARYWe previously piloted the concept of a Connectivity Map (CMap), whereby genes, drugs and disease states are connected by virtue of common gene-expression signatures. Here, we report more than a 1,000-fold scale-up of the CMap as part of the NIH LINCS Consortium, made possible by a new, low-cost, high throughput reduced representation expression profiling method that we term L1000. We show that L1000 is highly reproducible, comparable to RNA sequencing, and suitable for computational inference of the expression levels of 81% of non-measured transcripts. We further show that the expanded CMap can be used to discover mechanism of action of small molecules, functionally annotate genetic variants of disease genes, and inform clinical trials. The 1.3 million L1000 profiles described here, as well as tools for their analysis, are available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsclue.io&gt;httpsclue.io&lt;jatsext-link&gt;.HIGHLIGHTS&lt;jatslist list-type=bullet&gt;&lt;jatslist-item&gt;A new gene expression profiling method, L1000, dramatically lowers cost&lt;jatslist-item&gt;&lt;jatslist-item&gt;The Connectivity Map database now includes 1.3 million publicly accessible L1000 perturbational profiles&lt;jatslist-item&gt;&lt;jatslist-item&gt;This expanded Connectivity Map facilitates discovery of small molecule mechanism of action and functional annotation of genetic variants&lt;jatslist-item&gt;&lt;jatslist-item&gt;The work establishes feasibility and utility of a truly comprehensive Connectivity Map&lt;jatslist-item&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/reports/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/reports/page/275/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/reports/page/272/" class="page-link">272</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/reports/page/273/" class="page-link">273</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/reports/page/274/" class="page-link">274</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/reports/page/275/" class="page-link">275</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/reports/page/276/" class="page-link">276</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/reports/page/277/" class="page-link">277</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/reports/page/278/" class="page-link">278</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/reports/page/279/" class="page-link">279</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/reports/page/280/" class="page-link">280</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/reports/page/277/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/reports/page/320/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
