<!DOCTYPE html>
<html><head>
  
  <title>Microbiology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/next-generation_sequencing_of_double_stranded_rna_is_greatly_improved_by_treatment_with_the_inexpensive_denaturing_reagent_dmso_60745250/">
          <h4 class="font-weight-bold">Next-generation sequencing of double stranded RNA is greatly improved by treatment with the inexpensive denaturing reagent DMSO, bioRxiv, 2019-05-21</h4>
          <p>AbstractDouble stranded RNA (dsRNA) is the genetic material of important viruses and a key component of RNA interference-based immunity in eukaryotes. Previous studies have noted difficulties in determining the sequence of dsRNA molecules that have affected studies of immune function and estimates of viral diversity in nature. Dimethyl sulfoxide (DMSO) has been used to denature dsRNA prior to the reverse transcription stage to improve RT-PCR and Sanger sequencing. We systematically tested the utility of DMSO to improve sequencing yield of a dsRNA virus (Φ6) in a short-read next generation sequencing platform. DMSO treatment improved sequencing read recovery by over two orders of magnitude, even when RNA and cDNA concentrations were below the limit of detection. We also tested the effects of DMSO on a mock eukaryotic viral community and found that dsRNA virus reads increased with DMSO treatment. Furthermore, we provide evidence that DMSO treatment does not adversely affect recovery of reads from a single-stranded RNA viral genome (Influenza ACalifornia072009). We suggest that up to 50% DMSO treatment be used prior to cDNA synthesis when samples of interest are composed of or may contain dsRNA.Data SummarySequence data was deposited in the NCBI Short Read Archive (accession numbers PRJNA527100, PRJNA527101, PRJNA527098). Data and code for analysis is available on GitHub (&lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsgithub.comawilcox83dsRNA-sequencing&gt;httpsgithub.comawilcox83dsRNA-sequencing&lt;jatsext-link&gt;. doi10.5281zenodo.1453423). Protocol for dsRNA sequencing is posted on protocols.io (doi10.17504protocols.io.ugnetve).</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/rare_microbes_from_diverse_earth_biomes_dominate_community_activity_60565210/">
          <h4 class="font-weight-bold">Rare microbes from diverse Earth biomes dominate community activity, bioRxiv, 2019-05-17</h4>
          <p>AbstractMicrobes are the Earth’s most numerous organisms and are instrumental in driving major global biological and chemical processes. Microbial activity is a crucial component of all ecosystems, as microbes have the potential to control any major biochemical process. In recent years, considerable strides have been made in describing the community structure, i.e. diversity and abundance, of microbes from the Earth’s major biomes. In virtually all environments studied, a few highly abundant taxa dominate the structure of microbial communities. Still, microbial diversity is high and is concentrated in the less abundant, or rare, fractions of the community, i.e. the “long tail” of the abundance distribution. The relationship between microbial community structure and activity, specifically the role of rare microbes, and its connection to ecosystem function, is not fully understood. We analyzed 12.3 million metagenomic and metatranscriptomic sequence assemblies and their genes from environmental, human, and engineered microbiomes, and show that microbial activity is dominated by rare microbes (96% of total activity) across all measured biomes. Further, rare microbial activity was comprised of traits that are fundamental to ecosystem and organismal health, e.g. biogeochemical cycling and infectious disease. The activity of rare microbes was also tightly coupled to temperature, revealing a link between basic biological processes, e.g. reaction rates, and community activity. Our study provides a broadly applicable and predictable paradigm that implicates rare microbes as the main microbial drivers of ecosystem function and organismal health.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/long_metabarcoding_of_the_eukaryotic_rdna_operon_to_phylogenetically_and_taxonomically_resolve_environmental_diversity_59961244/">
          <h4 class="font-weight-bold">Long metabarcoding of the eukaryotic rDNA operon to phylogenetically and taxonomically resolve environmental diversity, bioRxiv, 2019-05-06</h4>
          <p>AbstractHigh-throughput environmental DNA metabarcoding has revolutionized the analysis of microbial diversity, but this approach is generally restricted to amplicon sizes below 500 base pairs. These short regions contain limited phylogenetic signal, which makes it impractical to use environmental DNA in full phylogenetic inferences. However, new long-read sequencing technologies such as the Pacific Biosciences platform may provide sufficiently large sequence lengths to overcome the poor phylogenetic resolution of short amplicons. To test this idea, we amplified soil DNA and used PacBio Circular Consensus Sequencing (CCS) to obtain a ~4500 bp region of the eukaryotic rDNA operon spanning most of the small (18S) and large subunit (28S) ribosomal RNA genes. The CCS reads were first treated with a novel curation workflow that generated 650 high-quality OTUs containing the physically linked 18S and 28S regions of the long amplicons. In order to assign taxonomy to these OTUs, we developed a phylogeny-aware approach based on the 18S region that showed greater accuracy and sensitivity than similarity-based and phylogenetic placement-based methods using shorter reads. The taxonomically-annotated OTUs were then combined with available 18S and 28S reference sequences to infer a well-resolved phylogeny spanning all major groups of eukaryotes, allowing to accurately derive the evolutionary origin of environmental diversity. A total of 1019 sequences were included, of which a majority (58%) corresponded to the new long environmental CCS reads. Comparisons to the 18S-only region of our amplicons revealed that the combined 18S-28S genes globally increased the phylogenetic resolution, recovering specific groupings otherwise missing. The long-reads also allowed to directly investigate the relationships among environmental sequences themselves, which represents a key advantage over the placement of short reads on a reference phylogeny. Altogether, our results show that long amplicons can be treated in a full phylogenetic framework to provide greater taxonomic resolution and a robust evolutionary perspective to environmental DNA.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/assembly-free_single-molecule_nanopore_sequencing_recovers_complete_virus_genomes_from_natural_microbial_communities_59546075/">
          <h4 class="font-weight-bold">Assembly-free single-molecule nanopore sequencing recovers complete virus genomes from natural microbial communities, bioRxiv, 2019-04-26</h4>
          <p>AbstractViruses are the most abundant biological entities on Earth, and play key roles in host ecology, evolution, and horizontal gene transfer. Despite recent progress in viral metagenomics, the inherent genetic complexity of virus populations still poses technical difficulties for recovering complete virus genomes from natural assemblages. To address these challenges, we developed an assembly-free, single-molecule nanopore sequencing approach enabling direct recovery of high-quality viral genome sequences from environmental samples. Our method yielded over a thousand high quality, full-length draft virus genome sequences that could not be fully recovered using short read assembly approaches applied to the same samples. Additionally, novel DNA sequences were discovered whose repeat structures, gene contents and concatemer lengths suggested that they represent phage-inducible chromosomal islands that were packaged as concatemers within phage particles. Our new approach provided novel insight into genome structures, population biology, and ecology of naturally occurring viruses and viral parasites.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/neighbor_predation_linked_to_natural_competence_fosters_the_transfer_of_large_genomic_regions_in_vibrio_cholerae_59471825/">
          <h4 class="font-weight-bold">Neighbor predation linked to natural competence fosters the transfer of large genomic regions in Vibrio cholerae, bioRxiv, 2019-04-25</h4>
          <p>AbstractNatural competence for transformation is a primary mode of horizontal gene transfer (HGT). Competent bacteria are able to absorb free DNA from their surroundings and exchange this DNA against pieces of their own genome when sufficiently homologous. And while it is known that transformation contributes to evolution and pathogen emergence in bacteria, there are still questions regarding the general prevalence of non-degraded DNA with sufficient coding capacity. In this context, we previously showed that the naturally competent bacterium Vibrio cholerae uses its type VI secretion system (T6SS) to actively acquire DNA from non-kin neighbors under chitin-colonizing conditions. We therefore sought to further explore the role of the T6SS in acquiring DNA, the condition of the DNA released through T6SS-mediated killing versus passive cell lysis, and the extent of the transfers that occur due to these conditions. To do this, we herein measured the frequency and the extent of genetic exchanges in bacterial co-cultures on competence-inducing chitin under various DNA-acquisition conditions. We show that competent V. cholerae strains acquire DNA fragments with an average and maximum length exceeding 50 kbp and 150 kbp, respectively, and that the T6SS is of prime importance for such HGT events. Collectively, our data support the notion that the environmental lifestyle of V. cholerae fosters HGT and that the coding capacity of the exchanged genetic material is sufficient to significantly accelerate bacterial evolution.Significance StatementDNA shuffled from one organism to another in an inheritable manner is a common feature of prokaryotes. It is a significant mechanism by which bacteria acquire new phenotypes, for example by first absorbing foreign DNA and then recombining it into their genome. In this study, we show the remarkable extent of the exchanged genetic material, frequently exceeding 150 genes in a seemingly single transfer event, in Vibrio cholerae. We also show that to best preserve its length and quality, bacteria mainly acquire this DNA by killing adjacent, healthy neighbors then immediately absorbing the released DNA before it can be degraded. These new insights into this prey-killing DNA acquisition process shed light on how bacterial species evolve in the wild.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_gut_microbiota_influences_circulatory_immune_cell_dynamics_in_humans_59473373/">
          <h4 class="font-weight-bold">The gut microbiota influences circulatory immune cell dynamics in humans, bioRxiv, 2019-04-25</h4>
          <p>ABSTRACTThe gut microbiota influences the development and homeostasis of the mammalian immune system1–3 and can alter immune cell compositions in mice4–7. However, our understanding of how the microbiota modulates immunity remains limited, particularly in humans where a lack of manipulative experiments makes inference challenging. Here we overcome this challenge by studying hundreds of hospitalized—and closely monitored—bone marrow transplantation patients as they recover from chemotherapy-induced immune ablation. This aggressive treatment causes large shifts in both white blood cell and microbiota populations allowing the relationships between the two to be studied simultaneously over time, with unprecedented resolution. Our analysis shows that the ecological diversity of the gut microbiota had an immunosuppressive effect on circulating lymphocyte counts similar in magnitude to that of anti-inflammatory and immunosuppressive drugs administered to cancer patients. Moreover, by controlling for drug treatments and clinical metadata, we identified several microbiota components strongly associated with white blood cell dynamics Streptococcaceae associated with lymphocyte increase and Actinomycetaceae with lymphocyte suppression, Prevotellaceae with monocyte increase and Enterobacteriaceae with monocyte suppression, Ruminococcaceae with neutrophil increase and Lachnospiraceae and Bacteroidaceae with neutrophil suppression. Our analysis establishes a direct link between the intestinal microbiota and systemic immunity in humans, with implications on clinical outcomes and microbiota-based therapies.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        
        <li class="page-item"><a href="/audiences/tags/microbiology/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="/audiences/tags/microbiology/" class="page-link">1</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/microbiology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/microbiology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/microbiology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/microbiology/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/microbiology/page/6/" class="page-link">6</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/microbiology/page/3/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/microbiology/page/11/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
