<!DOCTYPE html>
<html><head>
  
  <title>microbiology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.68.3" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.95d2483cd3c5d2bf47c1f36307a456787d60ad958c5efe44db9afc47012b1b05b8e1f64f6ac291650439322a6c38e86eb7cd6254255516d857c23f299fb0e666.js" integrity="sha512-ldJIPNPF0r9HwfNjB6RWeH1grZWMXv5E25r8RwErGwW44fZPasKRZQQ5MipsOOhut81iVCVVFthXwj8pn7DmZg=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/microbiota_supplementation_with_bifidobacterium_and_lactobacillus_modifies_the_preterm_infant_gut_microbiota_and_metabolome_63428708/">
          <h4 class="font-weight-bold">Microbiota supplementation with Bifidobacterium and Lactobacillus modifies the preterm infant gut microbiota and metabolome, bioRxiv, 2019-07-12</h4>
          <p>AbstractSupplementation with members of the early-life microbiota or ‘probiotics’ is becoming increasingly popular to attempt to beneficially manipulate the preterm gut microbiota. We performed a large longitudinal study comprising two preterm groups; 101 orally supplemented with Bifidobacterium and Lactobacillus (BifLacto) and 133 non-supplemented (Control) matched by age, sex, birth-mode, and diet. 16S rRNA metataxonomic profiling on stool samples (n = 592) indicated a predominance of Bifidobacterium, and a reduction of pathobionts in the BifLacto group. Metabolic phenotyping found a parallel increase in fecal acetate and lactate in the BifLacto group compared to the Control group, which positively correlated with Bifidobacterium abundance consistent with the ability of the supplemented Bifidobacterium strain to metabolize human milk oligosaccharides and reduced gut pH. This study demonstrates that microbiota supplementation can modify the preterm microbiome and the gastrointestinal environment to more closely resemble that of a full-term infant.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/intestinal_delta-6-desaturase_activity_determines_host_range_for_toxoplasma_sexual_reproduction_62936604/">
          <h4 class="font-weight-bold">Intestinal delta-6-desaturase activity determines host range for Toxoplasma sexual reproduction, bioRxiv, 2019-07-01</h4>
          <p>AbstractMany eukaryotic microbes have complex lifecycles that include both sexual and asexual phases with strict species-specificity. While the asexual cycle of the protistan parasite Toxoplasma gondii can occur in any warm-blooded mammal, the sexual cycle is restricted to the feline intestine1. The molecular determinants that identify cats as the definitive host for T. gondii are unknown. Here, we defined the mechanism of species specificity for T. gondii sexual development and break the species barrier to allow the sexual cycle to occur in mice. We determined that T. gondii sexual development occurs when cultured feline intestinal epithelial cells are supplemented with linoleic acid. Felines are the only mammals that lack delta-6-desaturase activity in their intestines, which is required for linoleic acid metabolism, resulting in systemic excess of linoleic acid2, 3. We found that inhibition of murine delta-6-desaturase and supplementation of their diet with linoleic acid allowed T. gondii sexual development in mice. This mechanism of species specificity is the first defined for a parasite sexual cycle. This work highlights how host diet and metabolism shape coevolution with microbes. The key to unlocking the species boundaries for other eukaryotic microbes may also rely on the lipid composition of their environments as we see increasing evidence for the importance of host lipid metabolism during parasitic lifecycles4, 5. Pregnant women are advised against handling cat litter as maternal infection with T. gondii can be transmitted to the fetus with potentially lethal outcomes. Knowing the molecular components that create a conducive environment for T. gondii sexual reproduction will allow for development of therapeutics that prevent shedding of T. gondii parasites. Finally, given the current reliance on companion animals to study T. gondii sexual development, this work will allow the T. gondii field to use of alternative models in future studies.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/highly_multiplexed_spatial_mapping_of_microbial_communities_62519865/">
          <h4 class="font-weight-bold">Highly Multiplexed Spatial Mapping of Microbial Communities, bioRxiv, 2019-06-22</h4>
          <p>ABSTRACTMapping the complex biogeography of microbial communities in situ with high taxonomic and spatial resolution poses a major challenge because of the high density and rich diversity of species in environmental microbiomes and the limitations of optical imaging technology. Here, we introduce High Phylogenetic Resolution microbiome mapping by Fluorescence In-Situ Hybridization (HiPR-FISH), a versatile and cost-effective technology that uses binary encoding and spectral imaging and machine learning based decoding to create micron-scale maps of the locations and identities of hundreds of microbial species in complex communities. We demonstrate the ability of 10-bit HiPR-FISH to distinguish 1023 E. coli strains, each fluorescently labeled with a unique binary barcode. HiPR-FISH, in conjunction with custom algorithms for automated probe design and segmentation of single-cells in the native context of tissues, reveals the intricate spatial architectures formed by bacteria in the human oral plaque microbiome and disruption of spatial networks in the mouse gut microbiome in response to antibiotic treatment. HiPR-FISH provides a framework for analyzing the spatial organization of microbial communities in tissues and the environment at single cell resolution.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/comprehensive_ecosystem-specific_16s_rrna_gene_databases_with_automated_taxonomy_assignment_autotax_provide_species-level_resolution_in_microbial_ecology_62362886/">
          <h4 class="font-weight-bold">Comprehensive ecosystem-specific 16S rRNA gene databases with automated taxonomy assignment (AutoTax) provide species-level resolution in microbial ecology, bioRxiv, 2019-06-18</h4>
          <p>AbstractHigh-throughput 16S rRNA gene amplicon sequencing is an indispensable method for studying the diversity and dynamics of microbial communities. However, this method is presently hampered by the lack of high-identity reference sequences for many environmental microbes in the public 16S rRNA gene reference databases, and by the lack of a systematic and comprehensive taxonomic classification for most environmental bacteria. Here we combine high-quality and high-throughput full-length 16S rRNA gene sequencing with a novel sequence identity-based approach for automated taxonomy assignment (AutoTax) to create robust, near-complete 16S rRNA gene databases for complex environmental ecosystems. To demonstrate the benefit of the approach, we created an ecosystem-specific database for wastewater treatment systems and anaerobic digesters. The novel approach allows consistent species-level classification of 16S rRNA amplicons sequence variants (ASVs) and the design of highly specific oligonucleotide probes for fluorescence in situ hybridization, which can reveal in situ properties of microbes at unprecedented taxonomic resolution.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/on_the_emergence_of_candida_auris_climate_change_azoles_swamps_and_birds_61471872/">
          <h4 class="font-weight-bold">On the emergence of Candida auris climate change, azoles, swamps and birds, bioRxiv, 2019-06-04</h4>
          <p>The most enigmatic aspect of the rise of Candida auris as a human pathogen is that it emerged simultaneously in three continents with each clade being genetically distinct. Although new pathogenic fungal species are described regularly, these are mostly species associated with single cases in individuals who are immunosuppressed. In this study, we used phylogenetic analysis to compare C. auris with temperature susceptibility of close relatives and use these results to argue that it may be the first example of a new fungal disease emerging from climate change with the caveat that many other factors could have contributed.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/consistent_metagenome-derived_metrics_verify_and_define_bacterial_species_boundaries_61070986/">
          <h4 class="font-weight-bold">Consistent metagenome-derived metrics verify and define bacterial species boundaries, bioRxiv, 2019-05-25</h4>
          <p>AbstractLongstanding questions relate to the existence of naturally distinct bacterial species and genetic approaches to distinguish them. Bacterial genomes in public databases form distinct groups, but these databases are subject to isolation and deposition biases. We compared 5,203 bacterial genomes from 1,457 environmental metagenomic samples to test for distinct clouds of diversity, and evaluated metrics that could be used to define the species boundary. Bacterial genomes from the human gut, soil, and the ocean all exhibited gaps in whole-genome average nucleotide identities (ANI) near the previously suggested species threshold of 95% ANI. While genome-wide ratios of non-synonymous and synonymous nucleotide differences (dNdS) decrease until ANI values approach ∼98%, estimates for homologous recombination approached zero at ∼95% ANI, supporting breakdown of recombination due to sequence divergence as a species-forming force. We evaluated 107 genome-based metrics for their ability to distinguish species when full genomes are not recovered. Full length 16S rRNA genes were least useful because they were under-recovered from metagenomes, but many ribosomal proteins displayed both high metagenomic recoverability and species-discrimination power. Taken together, our results verify the existence of sequence-discrete microbial species in metagenome-derived genomes and highlight the usefulness of ribosomal genes for gene-level species discrimination.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/3/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/3/" class="page-link">3</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/microbiology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/8/" class="page-link">8</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/5/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/16/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
