<!DOCTYPE html>
<html><head>
  
  <title>0-100-users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.68.3" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.95d2483cd3c5d2bf47c1f36307a456787d60ad958c5efe44db9afc47012b1b05b8e1f64f6ac291650439322a6c38e86eb7cd6254255516d857c23f299fb0e666.js" integrity="sha512-ldJIPNPF0r9HwfNjB6RWeH1grZWMXv5E25r8RwErGwW44fZPasKRZQQ5MipsOOhut81iVCVVFthXwj8pn7DmZg=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_revised_understanding_of_tribolium_morphogenesis_further_reconciles_short_and_long_germ_development_30343453/">
          <h4 class="font-weight-bold">A revised understanding of Tribolium morphogenesis further reconciles short and long germ development, bioRxiv, 2017-12-13</h4>
          <p>AbstractIn Drosophila melanogaster, the germband forms directly on the egg surface and solely consists of embryonic tissue. In contrast, most insect embryos undergo a complicated set of tissue rearrangements to generate a condensed, multi-layered germband. The ventral side of the germband is embryonic, while the dorsal side is thought to be an extraembryonic tissue called the amnion. While this tissue organisation has been accepted for decades, and has been widely reported in insects, its accuracy has not been directly tested in any species. Using live cell tracking and differential cell labelling in the short germ beetle Tribolium castaneum, I show that most of the cells previously thought to be amnion actually give rise to large parts of the embryo. This process occurs via the dorsal-to-ventral flow of cells and contributes to germband extension. In addition, I show that true ‘amnion’ cells in Tribolium originate from a small region of the blastoderm. Together, my findings show that development in the short germ embryos of Tribolium and the long germ embryos of Drosophila is more similar than previously proposed. Dorsal-to-ventral cell flow also occurs in Drosophila during germband extension, and I argue that the flow is driven by a conserved set of underlying morphogenetic events in both species. Furthermore, the revised Tribolium fatemap that I present is far more similar to that of Drosophila than the classic Tribolium fatemap. Lastly, my findings show that there is no qualitative difference between the tissue structure of the cellularised blastoderm and the shortintermediate germ germband. As such, the same tissue patterning mechanisms could function continuously throughout the cellularised blastoderm and germband stages, and easily shift between them over evolutionary time.Author summaryIn many animals, certain groups of cells in the embryo do not directly contribute to adult structures. Instead, these cells generate so-called ‘extra-embryonic tissues’ that support and facilitate development, but degenerate prior to birthhatching. In most insect species, embryos are described as having two major extra-embryonic tissues; the serosa, which encapsulates the entire embryo and yolk, and the amnion, which covers one side of the embryo. This tissue structure has been widely reported for over a century, but detailed studies on the amnion are lacking. Working in the beetle Tribolium castaneum, I used long-term fluorescent live imaging, cell tracking and differential cell labelling to investigate amnion development. In contrast to our current understanding, I show that most cells previously thought to be amnion actually form large parts of the embryo. In addition, I show how these cells ‘flow’ as a whole tissue and contribute to elongation of the embryo, and how only a relatively small number of cells form the actual amnion. Lastly, I describe how my findings show that despite exhibiting substantial differences in overall structure, embryos of Tribolium and the fruit fly, Drosophila melanogaster, utilise a conserved set of morphogenetic processes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/rarefaction_alpha_diversity_and_statistics_30290617/">
          <h4 class="font-weight-bold">Rarefaction, alpha diversity, and statistics, bioRxiv, 2017-12-12</h4>
          <p>AbstractUnderstanding the drivers of microbial diversity is a fundamental question in microbial ecology. Extensive literature discusses different methods for describing microbial diversity and documenting its effects on ecosystem function. However, it is widely believed that diversity depends on the number of reads that are sequenced. I discuss a statistical perspective on diversity, framing the diversity of an environment as an unknown parameter, and discussing the bias and variance of plug-in and rarefied estimates. I argue that by failing to account for both bias and variance, we invalidate analysis of alpha diversity. I describe the state of the statistical literature for addressing these problems, and suggest that measurement error modeling can address issues with variance, but bias corrections need to be utilized as well. I encourage microbial ecologists to avoid motivating their investigations with alpha diversity analyses that do not use valid statistical methodology.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/optogenetic_dissection_of_descending_behavioral_control_in_drosophila_30220788/">
          <h4 class="font-weight-bold">Optogenetic dissection of descending behavioral control in Drosophila, bioRxiv, 2017-12-10</h4>
          <p>AbstractIn most animals, the brain makes behavioral decisions that are transmitted by descending neurons to the nerve cord circuitry that produces behaviors. In insects, only a few descending neurons have been associated with specific behaviors. To explore how these neurons control an insect’s movements, we developed a novel method to systematically assay the behavioral effects of activating individual neurons on freely behaving terrestrial D. melanogaster. We calculated a two-dimensional representation of the entire behavior space explored by these flies and associated descending neurons with specific behaviors by identifying regions of this space that were visited with increased frequency during optogenetic activation. Applying this approach across a population of descending neurons, we found, that (1) activation of most of the descending neurons drove stereotyped behaviors, (2) in many cases multiple descending neurons activated similar behaviors, and (3) optogenetically-activated behaviors were often dependent on the behavioral state prior to activation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/optogenetic_dissection_of_descending_behavioral_control_indrosophila_30220788/">
          <h4 class="font-weight-bold">Optogenetic dissection of descending behavioral control inDrosophila, bioRxiv, 2017-12-10</h4>
          <p>AbstractIn most animals, the brain makes behavioral decisions that are transmitted by descending neurons to the nerve cord circuitry that produces behaviors. In insects, only a few descending neurons have been associated with specific behaviors. To explore how these neurons control an insect’s movements, we developed a novel method to systematically assay the behavioral effects of activating individual neurons on freely behaving terrestrialD. melanogaster. We calculated a two-dimensional representation of the entire behavior space explored by these flies and associated descending neurons with specific behaviors by identifying regions of this space that were visited with increased frequency during optogenetic activation. Applying this approach across a population of descending neurons, we found, that (1) activation of most of the descending neurons drove stereotyped behaviors, (2) in many cases multiple descending neurons activated similar behaviors, and (3) optogenetically-activated behaviors were often dependent on the behavioral state prior to activation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_deep_mutational_scan_of_an_acidic_activation_domain_30164194/">
          <h4 class="font-weight-bold">A deep mutational scan of an acidic activation domain, bioRxiv, 2017-12-09</h4>
          <p>AbstractTranscriptional activation domains are intrinsically disordered peptides with little primary sequence conservation. These properties have made it difficult to identify the sequence features that define activation domains. For example, although acidic activation domains were discovered 30 years ago, we still do not know what role, if any, acidic residues play in these peptides. To address this question we designed a rational mutagenesis scheme to independently test four sequence features theorized to control the strength of activation domains acidity (negative charge), hydrophobicity, intrinsic disorder, and short linear motifs. To test enough mutants to deconvolve these four features we developed a method to quantify the activities of thousands of activation domain variants in parallel. Our results with Gcn4, a classic acidic activation domain, suggest that acidic residues in particular regions keep two hydrophobic motifs exposed to solvent. We also found that the specific activity of the Gcn4 activation domain increases during amino acid starvation. Our results suggest that Gcn4 may have evolved to have low activity but high inducibility. Our results also demonstrate that high-throughput rational mutation scans will be powerful tools for unraveling the properties that control how intrinsically disordered proteins function.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/generative_adversarial_networks_for_reconstructing_natural_images_from_brain_activity_30177133/">
          <h4 class="font-weight-bold">Generative adversarial networks for reconstructing natural images from brain activity, bioRxiv, 2017-12-09</h4>
          <p>AbstractWe explore a method for reconstructing visual stimuli from brain activity. Using large databases of natural images we trained a deep convolutional generative adversarial network capable of generating gray scale photos, similar to stimUli presented during two functional magnetic resonance imaging experiments. Using a linear model we learned to predict the generative model’s latent space from measured brain activity. The objective was to create an image similar to the presented stimulus image through the previously trained generator. Using this approach we were able to reconstruct structural and some semantic features of a proportion of the natural images sets. A behavioral test showed that subjects were capable of identifying a reconstruction of the original stimuhis in 67.2% and 66.4% of the cases in a pairwise comparison for the two natural image datasets respectively. our approach does not require end-to-end training of a large generative model on limited neuroimaging data. Rapid advances in generative modeling promise further improvements in reconstruction performance.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2017">2017</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/90/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/87/" class="page-link">87</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/88/" class="page-link">88</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/89/" class="page-link">89</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/90/" class="page-link">90</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/91/" class="page-link">91</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/92/" class="page-link">92</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/93/" class="page-link">93</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/94/" class="page-link">94</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/95/" class="page-link">95</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/92/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/123/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
