<!DOCTYPE html>
<html><head>
  
  <title>Developmental Biology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/crispr-cas13d_induces_efficient_mrna_knock-down_in_animal_embryos_74016936/">
          <h4 class="font-weight-bold">CRISPR-Cas13d induces efficient mRNA knock-down in animal embryos, bioRxiv, 2020-01-14</h4>
          <p>AbstractEarly embryonic development is driven exclusively by maternal gene products deposited into the oocyte. Although critical in establishing early developmental programs, maternal gene functions have remained elusive due to a paucity of techniques for their systematic disruption and assessment. CRISPR-Cas13 systems have recently been employed to induce RNA degradation in yeast, plants and mammalian cell lines. However, no systematic study of the potential of Cas13 has been carried out in an animal system. Here, we show that CRISPR-Cas13d is an effective and precise system to deplete specific mRNA transcripts in zebrafish embryos. We demonstrate that both zygotically-expressed and maternally-provided transcripts are efficiently targeted, resulting in an 80% average decrease in transcript level and the recapitulation of well-known embryonic phenotypes. Moreover, we show that this system can be used in medaka, killifish and mouse embryos. Altogether our results demonstrate that CRISPR-Cas13d is an efficient knock-down platform to interrogate gene function in animal embryos.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/single_cell_epigenomic_atlas_of_the_developing_human_brain_and_organoids_73456347/">
          <h4 class="font-weight-bold">Single cell epigenomic atlas of the developing human brain and organoids, bioRxiv, 2020-01-01</h4>
          <p>AbstractDynamic changes in chromatin accessibility coincide with important aspects of neuronal differentiation, such as fate specification and arealization and confer cell type-specific associations to neurodevelopmental disorders. However, studies of the epigenomic landscape of the developing human brain have yet to be performed at single-cell resolution. Here, we profiled chromatin accessibility of &amp;gt;75,000 cells from eight distinct areas of developing human forebrain using single cell ATAC-seq (scATACseq). We identified thousands of loci that undergo extensive cell type-specific changes in accessibility during corticogenesis. Chromatin state profiling also reveals novel distinctions between neural progenitor cells from different cortical areas not seen in transcriptomic profiles and suggests a role for retinoic acid signaling in cortical arealization. Comparison of the cell type-specific chromatin landscape of cerebral organoids to primary developing cortex found that organoids establish broad cell type-specific enhancer accessibility patterns similar to the developing cortex, but lack many putative regulatory elements identified in homologous primary cell types. Together, our results reveal the important contribution of chromatin state to the emerging patterns of cell type diversity and cell fate specification and provide a blueprint for evaluating the fidelity and robustness of cerebral organoids as a model for cortical development.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/species-specific_developmental_timing_is_associated_with_global_differences_in_protein_stability_in_mouse_and_human_73458058/">
          <h4 class="font-weight-bold">Species-specific developmental timing is associated with global differences in protein stability in mouse and human, bioRxiv, 2020-01-01</h4>
          <p>ABSTRACTWhat determines the pace of embryonic development? Although many molecular mechanisms controlling developmental processes are evolutionarily conserved, the speed at which these operate can vary substantially between species. For example, the same genetic programme, comprising sequential changes in transcriptional states, governs the differentiation of motor neurons in mouse and human, but the tempo at which it operates differs between species. Using in vitro directed differentiation of embryonic stem cells to motor neurons, we show that the programme runs twice as fast in mouse as in human. We provide evidence that this is neither due to differences in signalling, nor the genomic sequence of genes or their regulatory elements. Instead, we find an approximately two-fold increase in protein stability and cell cycle duration in human cells compared to mouse. This can account for the slower pace of human development, indicating that global differences in key kinetic parameters play a major role in interspecies differences in developmental tempo.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/self-organised_symmetry_breaking_in_zebrafish_reveals_feedback_from_morphogenesis_to_pattern_formation_66593193/">
          <h4 class="font-weight-bold">Self-organised symmetry breaking in zebrafish reveals feedback from morphogenesis to pattern formation, bioRxiv, 2019-09-14</h4>
          <p>A fundamental question in developmental biology is how the early embryo breaks initial symmetry to establish the spatial coordinate system later important for the organisation of the embryonic body plan. In zebrafish, this is thought to depend on the inheritance of maternal mRNAs [1–3], cortical rotation to generate a dorsal pole of beta-catenin activity [4–8] and the release of Nodal signals from the yolk syncytial layer (YSL) [9–12]. Recent work aggregating mouse embryonic stem cells has shown that symmetry breaking can occur in the absence of extra-embryonic tissue [19,20]. To test whether this is also true in zebrafish, we separated embryonic cells from the yolk and allowed them to develop as aggregates. These aggregates break symmetry autonomously to form elongated structures with an anterior-posterior pattern. Extensive cell mixing shows that any pre-existing asymmetry is lost prior to the breaking morphological symmetry, revealing that the maternal pre-pattern is not strictly required for early embryo patterning. Following early signalling events after isolation of embryonic cells reveals that a pole of Nodal activity precedes and is required for elongation. The blocking of PCP-dependent convergence and extension movements disrupts the establishment of opposing poles of BMP and WntTCF activity and the patterning of anterior-posterior neural tissue. These results lead us to suggest that convergence and extension plays a causal role in the establishment of morphogen gradients and pattern formation during zebrafish gastrulation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/profiling_cellular_diversity_in_sponges_informs_animal_cell_type_and_nervous_system_evolution_65970795/">
          <h4 class="font-weight-bold">Profiling cellular diversity in sponges informs animal cell type and nervous system evolution, bioRxiv, 2019-09-05</h4>
          <p>AbstractThe evolutionary origin of metazoan cell types such as neurons, muscles, digestive, and immune cells, remains unsolved. Using whole-body single-cell RNA sequencing in a sponge, an animal without nervous system and musculature, we identify 18 distinct cell types comprising four major families. This includes nitric-oxide sensitive contractile cells, digestive cells active in macropinocytosis, and a family of amoeboid-neuroid cells involved in innate immunity. We uncover ‘presynaptic’ genes in an amoeboid-neuroid cell type, and ‘postsynaptic’ genes in digestive choanocytes, suggesting asymmetric and targeted communication. Corroborating this, long neurite-like extensions from neuroid cells directly contact and enwrap choanocyte microvillar collars. Our data indicate a link between neuroid and immune functions in sponges, and suggest that a primordial neuro-immune system cleared intruders and controlled ciliary beating for feeding.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/interspecies_transcriptome_analyses_identify_genes_that_control_the_development_and_evolution_of_limb_skeletal_proportion_65742453/">
          <h4 class="font-weight-bold">Interspecies transcriptome analyses identify genes that control the development and evolution of limb skeletal proportion, bioRxiv, 2019-09-01</h4>
          <p>AbstractDespite the great diversity of vertebrate limb proportion and our deep understanding of the genetic mechanisms that drive skeletal elongation, little is known about how individual bones reach different lengths in any species. Here, we directly compare the transcriptomes of homologous growth cartilages of the mouse (Mus musculus) and bipedal jerboa (Jaculus jaculus), which has extremely long metatarsals of the feet and ‘mouse-like’ arms. When we intersected gene expression differences in metatarsals of the two species with expression differences in forearms, we found that about 10% of all orthologous genes are associated with disproportionate elongation of jerboa feet. Among these, Shox2, has gained expression in jerboa metatarsals where it is not expressed in other vertebrates that have been assessed. This transcription factor is necessary for proximal limb elongation, and we show that it is sufficient to increase mouse distal limb length. Unexpectedly, we also found evidence that jerboa foot elongation occurs in part by releasing latent growth potential that is repressed in mouse feet. In jerboa metatarsals, we observed higher expression of Crabp1, an antagonist of growth inhibitory retinoic acid, lower expression of Gdf10, an inhibitory TGFβ ligand, and lower expression of Mab21L2, a BMP signaling inhibitor that we show is sufficient to reduce limb bone elongation. By intersecting our data with prior expression analyses in other systems, we identify mechanisms that may both establish limb proportion during development and diversify proportion during evolution. The genes we identified here therefore provide a framework to understand the modular genetic control of skeletal growth and the remarkable malleability of vertebrate limb proportion.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        

        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/developmental-biology/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/developmental-biology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/developmental-biology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/developmental-biology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/developmental-biology/page/5/" class="page-link">5</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/developmental-biology/page/2/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/developmental-biology/page/6/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
