<!DOCTYPE html>
<html><head>
  
  <title>Immunology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/rnase_l_reprograms_translation_by_widespread_mrna_turnover_escaped_by_antiviral_mrnas_52201035/">
          <h4 class="font-weight-bold">RNase L reprograms translation by widespread mRNA turnover escaped by antiviral mRNAs, bioRxiv, 2018-12-05</h4>
          <p>SUMMARYIn response to foreign and endogenous double-stranded RNA (dsRNA), protein kinase R (PKR) and ribonuclease L (RNase L) reprogram translation in mammalian cells. PKR inhibits translation initiation through eIF2α phosphorylation, which triggers stress granule (SG) formation and promotes translation of stress responsive mRNAs. The mechanisms of RNase L-driven translation repression, its contribution to SG assembly, and its regulation of dsRNA stress-induced mRNAs are unknown. We demonstrate that RNase L drives translational shut-off in response to dsRNA by promoting widespread turnover of mRNAs. This alters stress granule assembly and reprograms translation by only allowing for the translation of mRNAs resistant to RNase L degradation, including numerous antiviral mRNAs such as IFN-β. Individual cells differentially activate dsRNA responses revealing variation that can affect cellular outcomes. This identifies bulk mRNA degradation and the resistance of antiviral mRNAs as the mechanism by which RNaseL reprograms translation in response to dsRNA.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genome-wide_crispr_screens_in_primary_human_t_cells_reveal_key_regulators_of_immune_function_46004726/">
          <h4 class="font-weight-bold">Genome-wide CRISPR Screens in Primary Human T Cells Reveal Key Regulators of Immune Function, bioRxiv, 2018-08-03</h4>
          <p>SUMMARYHuman T cells are central effectors of immunity and cancer immunotherapy. CRISPR-based functional studies in T cells could prioritize novel targets for drug development and improve the design of genetically reprogrammed cell-based therapies. However, large-scale CRISPR screens have been challenging in primary human cells. We developed a new method, sgRNA lentiviral infection with Cas9 protein electroporation (SLICE), to identify regulators of stimulation responses in primary human T cells. Genome-wide loss-of-function screens identified essential T cell receptor signaling components and genes that negatively tune proliferation following stimulation. Targeted ablation of individual candidate genes validated hits and identified perturbations that enhanced cancer cell killing. SLICE coupled with single-cell RNA-Seq revealed signature stimulation-response gene programs altered by key genetic perturbations. SLICE genome-wide screening was also adaptable to identify mediators of immunosuppression, revealing genes controlling response to adenosine signaling. The SLICE platform enables unbiased discovery and characterization of functional gene targets in primary cells.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/identification_of_pre-existing_adaptive_immunity_to_cas9_proteins_in_humans_31294982/">
          <h4 class="font-weight-bold">Identification of Pre-Existing Adaptive Immunity to Cas9 Proteins in Humans, bioRxiv, 2018-01-06</h4>
          <p>AbstractThe CRISPR-Cas9 system has proven to be a powerful tool for genome editing, allowing for the precise modification of specific DNA sequences within a cell. Many efforts are currently underway to use the CRISPR-Cas9 system for the therapeutic correction of human genetic diseases. The most widely used homologs of the Cas9 protein are derived from the bacteria Staphylococcus aureus (S. aureus) and Streptococcus pyogenes (S. pyogenes). Based on the fact that these two bacterial species cause infections in the human population at high frequencies, we looked for the presence of pre-existing adaptive immune responses to their respective Cas9 homologs, SaCas9 (S. aureus homolog of Cas9) and SpCas9 (S. pyogenes homolog of Cas9). To determine the presence of anti-Cas9 antibodies, we probed for the two homologs using human serum and were able to detect antibodies against both, with 79% of donors staining against SaCas9 and 65% of donors staining against SpCas9. Upon investigating the presence of antigen-specific T-cells against the two homologs in human peripheral blood, we found anti-SaCas9 T-cells in 46% of donors. Upon isolating, expanding, and conducting antigen re-stimulation experiments on several of these donors’ anti-SaCas9 T-cells, we observed an SaCas9-specific response confirming that these T-cells were antigen-specific. We were unable to detect antigen-specific T-cells against SpCas9, although the sensitivity of the assay precludes us from concluding that such T-cells do not exist. Together, this data demonstrates that there are pre-existing humoral and cell-mediated adaptive immune responses to Cas9 in humans, a factor which must be taken into account as the CRISPR-Cas9 system moves forward into clinical trials.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/single-cell_map_of_diverse_immune_phenotypes_driven_by_the_tumor_microenvironment_29430006/">
          <h4 class="font-weight-bold">Single-cell Map of Diverse Immune Phenotypes Driven by the Tumor Microenvironment, bioRxiv, 2017-11-26</h4>
          <p>SUMMARYKnowledge of immune cell phenotypes in the tumor microenvironment is essential for understanding mechanisms of cancer progression and immunotherapy response. We created an immune map of breast cancer using single-cell RNA-seq data from 45,000 immune cells from eight breast carcinomas, as well as matched normal breast tissue, blood, and lymph node. We developed a preprocessing pipeline, SEQC, and a Bayesian clustering and normalization method, Biscuit, to address computational challenges inherent to single-cell data. Despite significant similarity between normal and tumor tissue-resident immune cells, we observed continuous tumor-specific phenotypic expansions driven by environmental cues. Analysis of paired single-cell RNA and T cell receptor (TCR) sequencing data from 27,000 additional T cells revealed the combinatorial impact of TCR utilization on phenotypic diversity. Our results support a model of continuous activation in T cells and do not comport with the macrophage polarization model in cancer, with important implications for characterizing tumor-infiltrating immune cells.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_10000_immunomes_project_a_resource_for_human_immunology_24305832/">
          <h4 class="font-weight-bold">The 10,000 Immunomes Project A resource for human immunology, bioRxiv, 2017-08-26</h4>
          <p>AbstractNew immunological assays now enable rich measurements of human immune function, but difficulty attaining enough measurements across sufficiently large and diverse cohorts has hindered describing normal human immune physiology on a large scale. Here we present the 10,000 Immunomes Project (10KIP), a diverse human immunology reference derived from over 44,000 individuals across 242 studies from ImmPort, a publicly available resource of raw immunology study data and protocols. We carefully curated datasets, aggregating subjects from healthycontrol arms and harmonizing data across studies. We demonstrate 10KIP’s utility by describing variations in serum cytokines and leukocytes by age, race, and sex; defining a baseline cell-cytokine network; and using 10KIP as a common control to describe immunologic changes in pregnancy. Subject-level data is available for interactive visualization and download at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=http10kImmunomes.org&gt;http10kImmunomes.org&lt;jatsext-link&gt;. We believe 10KIP can serve as a common control cohort and will accelerate hypothesis generation by clinical and basic immunologists across diverse populations.One Sentence SummaryAn open online resource of human immunology data from more than 10,000 normal subjects including interactive data visualization and download enables a new look at immune system differences across age and sex, rapid hypothesis generation, and creation of custom control cohorts.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/tumor-infiltrating_immune_repertoires_captured_by_single-cell_barcoding_in_emulsion_19894024/">
          <h4 class="font-weight-bold">Tumor-infiltrating immune repertoires captured by single-cell barcoding in emulsion, bioRxiv, 2017-05-06</h4>
          <p>AbstractTumor-infiltrating lymphocytes (TILs) are critical to anti-cancer immune responses, but their diverse phenotypes and functions remain poorly understood and challenging to study. We therefore developed a single-cell barcoding technology for deep characterization of TILs without the need for cell-sorting or culture. Our emulsion-based method captures full-length, natively paired B-cell and T-cell receptor (BCR and TCR) sequences from lymphocytes among millions of input cells. We validated the method with 3 million B-cells from healthy human blood and 350,000 B-cells from an HIV elite controller, before processing 400,000 cells from an unsorted dissociated ovarian adenocarcinoma and recovering paired BCRs and TCRs from over 11,000 TILs. We then extended the barcoding method to detect DNA-labeled antibodies, allowing ultra-high throughput, simultaneous protein detection and RNA sequencing from single cells.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/immunology/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/immunology/page/4/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="/audiences/tags/immunology/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/immunology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/immunology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/immunology/page/4/" class="page-link">4</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/immunology/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/immunology/page/6/" class="page-link">6</a></li>
            
        

        
        <li class="page-item"><a href="/audiences/tags/immunology/page/6/" rel="next" class="page-link">Next ›</a></li>
        

        
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
