<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/gapless_assembly_of_maize_chromosomes_using_long_read_technologies_74057430/">
          <h4 class="font-weight-bold">Gapless assembly of maize chromosomes using long read technologies, bioRxiv, 2020-01-15</h4>
          <p>AbstractCreating gapless telomere-to-telomere assemblies of complex genomes is one of the ultimate challenges in genomics. We used long read technologies and an optical map based approach to produce a maize genome assembly composed of only 63 contigs. The B73-Ab10 genome includes gapless assemblies of chromosome 3 (236 Mb) and chromosome 9 (162 Mb), multiple highly repetitive centromeres and heterochromatic knobs, and 53 Mb of the Ab10 meiotic drive haplotype.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genetic_associations_at_regulatory_phenotypes_improve_fine-mapping_of_causal_variants_for_twelve_immune-mediated_diseases_74062956/">
          <h4 class="font-weight-bold">Genetic associations at regulatory phenotypes improve fine-mapping of causal variants for twelve immune-mediated diseases, bioRxiv, 2020-01-15</h4>
          <p>AbstractThe identification of causal genetic variants for common diseases improves understanding of disease biology. Here we use data from the BLUEPRINT project to identify regulatory quantitative trait loci (QTL) for three primary human immune cell types and use these to fine-map putative causal variants for twelve immune-mediated diseases. We identify 340 unique, non major histocompatibility complex (MHC) disease loci that colocalise with high (&amp;gt;98%) posterior probability with regulatory QTLs, and apply Bayesian frameworks to fine-map associations at each locus. We show that fine-mapping applied to regulatory QTLs yields smaller credible set sizes and higher posterior probabilities for candidate causal variants compared to disease summary statistics. We also describe a systematic under-representation of insertiondeletion (INDEL) polymorphisms in credible sets derived from publicly available disease meta-analysis when compared to QTLs based on genome-sequencing data. Overall, our findings suggest that fine-mapping applied to disease-colocalising regulatory QTLs can enhance the discovery of putative causal disease variants and provide insights into the underlying causal genes and molecular mechanisms.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/recent_fluctuations_in_mexican_american_genomes_have_altered_the_genetic_architecture_of_biomedical_traits_74022833/">
          <h4 class="font-weight-bold">Recent fluctuations in Mexican American genomes have altered the genetic architecture of biomedical traits, bioRxiv, 2020-01-15</h4>
          <p>AbstractHispanicsLatinos are a diverse group of admixed populations with African, European, and Native American ancestries. They remain understudied, and thus little is known about the genetic architecture of phenotypic variation in these populations. Using genome-wide genotype data from the Hispanic Community Health StudyStudy of Latinos, we find that Native American ancestry has increased over time across HispanicLatino populations, particularly in Mexican Americans where Native American ancestry increased by an average of ∼20% over the 50-year period spanning 1940s-1990s. We find similar patterns across American cities, and replicate our observations in an independent sample of Mexican Americans. These dynamic ancestry patterns are a result of a complex interaction of several population and cultural factors, including strong ancestry-related assortative mating and subtle shifts in migration with differences in subcontinental Native American ancestry over time. These factors have shaped patterns of genetic variation, including an increase in runs of homozygosity in Native American ancestral tracts, and also influenced the genetic architecture of complex traits within the Mexican American population. We show for height, a trait correlated with ancestry, polygenic risk scores based on summary statistics from a European-based genome-wide association study perform poorly in Mexican Americans. Our findings reveal temporal changes in population structure within HispanicsLatinos that may influence biomedical traits, demonstrating a crucial need to improve our understanding of the genetic diversity of admixed populations.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/synthesis_of_geological_and_comparative_phylogeographic_data_point_to_climate_not_mountain_uplift_as_driver_of_divergence_across_the_eastern_andean_cordillera_74062625/">
          <h4 class="font-weight-bold">Synthesis of geological and comparative phylogeographic data point to climate, not mountain uplift, as driver of divergence across the Eastern Andean Cordillera, bioRxiv, 2020-01-15</h4>
          <p>AbstractAimTo evaluate the potential role of the orogeny of the Eastern Cordillera (EC) of the Colombian Andes and the Mérida Andes (MA) of Venezuela as drivers of vicariance between populations of 37 tetrapod lineages co-distributed on both flanks, through geological reconstruction and comparative phylogeographic analyses.LocationNorthwestern South AmericaMethodsWe first reviewed and synthesized published geological data on the timing of uplift for the EC-MA. We then combined newly generated mitochondrial DNA sequence data with published datasets to create a comparative phylogeographic dataset for 37 independent tetrapod lineages. We reconstructed time-calibrated molecular phylogenies for each lineage under Bayesian inference to estimate divergence times between lineages located East and West of the Andes. We performed a comparative phylogeographic analysis of all lineages within each class of tetrapod using hierarchical approximate Bayesian computation (hABC) to test for synchronous vicariance across the EC-MA. To evaluate the potential role of life history in explaining variation in divergence times among lineages, we evaluated 13 general linear models (GLM) containing up to six variables each (maximum elevation, range size, body length, thermoregulation, type of dispersal, and taxonomic class).ResultsOur synthesis of geological evidence suggested that the EC-MA reached significant heights by 38–33 million years ago (Ma) along most of its length, and we reject the oft-cited date of 2–5 Ma. Based on mtDNA divergence from 37 lineages, however, the median estimated divergence time across the EC-MA was 3.26 Ma (SE = 2.84) in amphibians, 2.58 Ma (SE = 1.81) in birds, 2.99 Ma (SE = 4.68) in reptiles and 1.43 Ma (SE = 1.23) in mammals. Using Bayes Factors, the hypothesis for a single temporal divergence interval containing synchronous divergence events was supported for mammals and but not supported for amphibians, non-avian reptiles, or birds. Among the six life-history variables tested, only thermoregulation successfully explained variation in divergence times (minimum AICc, R2 0.10), with homeotherms showing more recent divergence relative to poikilotherms.Main conclusionsOur results reject the hypothesis of the rise Andean Cordillera as driver of vicariance of lowland population because divergence dates are too recent and too asynchronous. We discuss alternative explanations, including dispersal through mountain passes, and suggest that changes in the climatic conditions during the Pliocene and Pleistocene interacted with tetrapod physiology, promoting older divergences in amphibians and reptiles relative to mammals and birds on an already established orogen.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/using_natural_language_processing_to_learn_the_grammar_of_glycans_73910354/">
          <h4 class="font-weight-bold">Using Natural Language Processing to Learn the Grammar of Glycans, bioRxiv, 2020-01-12</h4>
          <p>AbstractWhile nucleic acids and proteins receive ample attention, progress on understanding the structural and functional roles of carbohydrates has lagged behind. Here, we develop a language model for glycans, SweetTalk, taking into account glycan connectivity and composition. We use this model to investigate motifs in glycan substructures, classify them according to their O-N-linkage, and predict their immunogenicity with an accuracy of ∼92%, opening up the potential for rational glycoengineering.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/quantifying_the_spatiotemporal_dynamics_of_ires_versus_cap_translation_with_single-molecule_resolution_in_living_cells_73809213/">
          <h4 class="font-weight-bold">Quantifying the spatiotemporal dynamics of IRES versus Cap translation with single-molecule resolution in living cells, bioRxiv, 2020-01-10</h4>
          <p>ABSTRACTViruses use IRES sequences within their RNA to hijack translation machinery and thereby rapidly replicate in host cells. While this process has been extensively studied in bulk assays, the dynamics of hijacking at the single-molecule level remain unexplored in living cells. To achieve this, we developed a bicistronic biosensor encoding complementary repeat epitopes in two ORFs, one translated in a Cap-dependent manner and the other translated in an IRES-mediated manner. Using a pair of complementary probes that bind the epitopes co-translationally, our biosensor lights up in different colors depending on which ORF is being translated. In combination with single-molecule tracking and computational modeling, we measured the relative kinetics of Cap versus IRES translation and show (1) Two non-overlapping ORFs can be simultaneously translated within a single mRNA; (2) EMCV IRES-mediated translation sites recruit ribosomes less efficiently than Cap-dependent translation sites but are otherwise nearly indistinguishable, having similar mobilities, sizes, spatial distributions, and ribosomal initiation and elongation rates; (3) Both Cap-dependent and IRES-mediated ribosomes tend to stretch out translation sites; (4) Although the IRES recruits two to three times fewer ribosomes than the Cap in normal conditions, the balance shifts dramatically in favor of the IRES during oxidative and ER stresses that mimic viral infection; and (5) Translation of the IRES is enhanced by translation of the Cap, demonstrating upstream translation can positively impact the downstream translation of a non-overlapping ORF. With the ability to simultaneously quantify two distinct translation mechanisms in physiologically relevant live-cell environments, we anticipate bicistronic biosensors like the one we developed here will become powerful new tools to dissect both canonical and non-canonical translation dynamics with single-molecule precision.Graphical Abstract&lt;jatsfig id=ufig1 position=float orientation=portrait fig-type=figure&gt;&lt;jatsgraphic xmlnsxlink=httpwww.w3.org1999xlink xlinkhref=900829v1_ufig1 position=float orientation=portrait &gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biochemistry">biochemistry</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2020">2020</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/5/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/5/" class="page-link">5</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/0-100-users/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/10/" class="page-link">10</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/7/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/123/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
