<!DOCTYPE html>
<html><head>
  
  <title>Reports | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/maple_a_modular_automated_platform_for_large-scale_experiments_a_low-cost_robot_for_integrated_animal-handling_and_phenotyping_30874038/">
          <h4 class="font-weight-bold">MAPLE a Modular Automated Platform for Large-scale Experiments, a low-cost robot for integrated animal-handling and phenotyping, bioRxiv, 2017-12-25</h4>
          <p>AbstractGenetic model system animals have significant scientific value in part because of large-scale experiments like screens, but performing such experiments over long time periods by hand is arduous and risks errors. Thus the field is poised to benefit from automation, just as molecular biology did from liquid-handling robots. We developed a Modular Automated Platform for Large-scale Experiments (MAPLE), a Drosophila-handling robot capable of conducting lab tasks and experiments. We demonstrate MAPLE’s ability to accelerate the collection of virgin female flies (a pervasive experimental chore in fly genetics) and assist high-throughput phenotyping assays. Using MAPLE to autonomously run a novel social interaction experiment, we found that 1) pairs of flies exhibit persistent idiosyncrasies in affiliative behavior, 2) these dyad-specific interactions require olfactory and visual cues, and 3) social interaction network structure is topologically stable over time. These diverse examples demonstrate MAPLE’s versatility as a general platform for conducting fly science automatically.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioengineering">bioengineering</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/discovery_and_characterization_of_coding_and_non-coding_driver_mutations_in_more_than_2500_whole_cancer_genomes_30832106/">
          <h4 class="font-weight-bold">Discovery and characterization of coding and non-coding driver mutations in more than 2,500 whole cancer genomes, bioRxiv, 2017-12-24</h4>
          <p>AbstractDiscovery of cancer drivers has traditionally focused on the identification of protein-coding genes. Here we present a comprehensive analysis of putative cancer driver mutations in both protein-coding and non-coding genomic regions across &amp;gt;2,500 whole cancer genomes from the Pan-Cancer Analysis of Whole Genomes (PCAWG) Consortium. We developed a statistically rigorous strategy for combining significance levels from multiple driver discovery methods and demonstrate that the integrated results overcome limitations of individual methods. We combined this strategy with careful filtering and applied it to protein-coding genes, promoters, untranslated regions (UTRs), distal enhancers and non-coding RNAs. These analyses redefine the landscape of non-coding driver mutations in cancer genomes, confirming a few previously reported elements and raising doubts about others, while identifying novel candidate elements across 27 cancer types. Novel recurrent events were found in the promoters or 5’UTRs of TP53, RFTN1, RNF34, and MTG2, in the 3’UTRs of NFKBIZ and TOB1, and in the non-coding RNA RMRP. We provide evidence that the previously reported non-coding RNAs NEAT1 and MALAT1 may be subject to a localized mutational process. Perhaps the most striking finding is the relative paucity of point mutations driving cancer in non-coding genes and regulatory elements. Though we have limited power to discover infrequent non-coding drivers in individual cohorts, combined analysis of promoters of known cancer genes show little excess of mutations beyond TERT.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/emergence_of_reward_expectation_signals_in_identified_dopamine_neurons_30829742/">
          <h4 class="font-weight-bold">Emergence of reward expectation signals in identified dopamine neurons, bioRxiv, 2017-12-23</h4>
          <p>AbstractCoherent control of purposive actions emerges from the coordination of multiple brain circuits during learning. Dissociable brain circuits and cell-types are thought to preferentially participate in distinct learning mechanisms. For example, the activity of midbrain dopamine (mDA) neurons is proposed to primarily, or even exclusively, reflect reward prediction error signals in well-trained animals. To study the specific contribution of individual circuits requires observing changes before tight functional coordination is achieved. However, little is known about the detailed timing of the emergence of reward-related representations in dopaminergic neurons. Here we recorded activity of identified dopaminergic neurons as naïve mice learned a novel stimulus-reward association. We found that at early stages of learning mDA neuron activity reflected both external (sensory) and internal (action initiation) causes of reward expectation. The increasingly precise correlation of action initiation with sensory stimuli rather than an evaluation of outcomes governed mDA neuron activity. Thus, our data demonstrate that mDA neuron activity early in learning does not reflect errors, but is more akin to a Hebbian learning signal - providing new insight into a critical computation in a highly conserved, essential learning circuit.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/a_single-cell_catalogue_of_regulatory_states_in_the_ageing_drosophila_brain_30763906/">
          <h4 class="font-weight-bold">A single-cell catalogue of regulatory states in the ageing Drosophila brain, bioRxiv, 2017-12-22</h4>
          <p>SummaryThe diversity of cell types and regulatory states in the brain, and how these change during ageing, remains largely unknown. Here, we present a single-cell transcriptome catalogue of the entire adult Drosophila melanogaster brain sampled across its lifespan. Both neurons and glia age through a process of “regulatory erosion”, characterized by a strong decline of RNA content, and accompanied by increasing transcriptional and chromatin noise. We identify more than 50 cell types by specific transcription factors and their downstream gene regulatory networks. In addition to neurotransmitter types and neuroblast lineages, we find a novel neuronal cell state driven by datilografo and prospero. This state relates to neuronal birth order, the metabolic profile, and the activity of a neuron. Our single-cell brain catalogue reveals extensive regulatory heterogeneity linked to ageing and brain function and will serve as a reference for future studies of genetic variation and disease mutations.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/cell_hashing_with_barcoded_antibodies_enables_multiplexing_and_doublet_detection_for_single_cell_genomics_30805042/">
          <h4 class="font-weight-bold">Cell “hashing” with barcoded antibodies enables multiplexing and doublet detection for single cell genomics, bioRxiv, 2017-12-22</h4>
          <p>ABSTRACTDespite rapid developments in single cell sequencing technology, sample-specific batch effects, detection of cell doublets, and the cost of generating massive datasets remain outstanding challenges. Here, we introduce cell “hashing”, where oligo-tagged antibodies against ubiquitously expressed surface proteins are used to uniquely label cells from distinct samples, which can be subsequently pooled. By sequencing these tags alongside the cellular transcriptome, we can assign each cell to its sample of origin, and robustly identify doublets originating from multiple samples. We demonstrate our approach by pooling eight human PBMC samples on a single run of the 10x Chromium system, substantially reducing our per-cell costs for library generation. Cell “hashing” is inspired by, and complementary to, elegant multiplexing strategies based on genetic variation, which we also leverage to validate our results. We therefore envision that our approach will help to generalize the benefits of single cell multiplexing to diverse samples and experimental designs.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/cellular_diversity_in_the_drosophila_midbrain_revealed_by_single-cell_transcriptomics_30761876/">
          <h4 class="font-weight-bold">Cellular diversity in the Drosophila midbrain revealed by single-cell transcriptomics, bioRxiv, 2017-12-22</h4>
          <p>AbstractTo understand the brain, molecular details need to be overlaid onto neural wiring diagrams so that synaptic mode, neuromodulation and critical signaling operations can be considered. Single-cell transcriptomics provide a unique opportunity to collect this information. Here we present an initial analysis of thousands of individual cells from Drosophila midbrain, that were acquired using Drop-Seq. A number of approaches permitted the assignment of transcriptional profiles to several major brain regions and cell-types. Expression of biosynthetic enzymes and reuptake mechanisms allows all the neurons to be typed according to the neurotransmitter or neuromodulator that they produce and presumably release. Some neuropeptides are preferentially co-expressed in neurons using a particular fast-acting transmitter, or monoamine. Neuromodulatory and neurotransmitter receptor subunit expression illustrates the potential of these molecules in generating complexity in neural circuit function. This cell atlas dataset provides an important resource to link molecular operations to brain regions and complex neural processes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/reports/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/reports/page/142/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/reports/page/139/" class="page-link">139</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/140/" class="page-link">140</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/141/" class="page-link">141</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/142/" class="page-link">142</a></li>
            
        
             
            <li class="page-item active"><a href="/reports/page/143/" class="page-link">143</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/144/" class="page-link">144</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/145/" class="page-link">145</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/146/" class="page-link">146</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/147/" class="page-link">147</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/reports/page/144/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/reports/page/172/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
