<!DOCTYPE html>
<html><head>
  
  <title>200 500 Users | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/on_the_emergence_of_candida_auris_climate_change_azoles_swamps_and_birds_61471872/">
          <h4 class="font-weight-bold">On the emergence of Candida auris climate change, azoles, swamps and birds, bioRxiv, 2019-06-04</h4>
          <p>The most enigmatic aspect of the rise of Candida auris as a human pathogen is that it emerged simultaneously in three continents with each clade being genetically distinct. Although new pathogenic fungal species are described regularly, these are mostly species associated with single cases in individuals who are immunosuppressed. In this study, we used phylogenetic analysis to compare C. auris with temperature susceptibility of close relatives and use these results to argue that it may be the first example of a new fungal disease emerging from climate change with the caveat that many other factors could have contributed.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/multiscale_activity_imaging_in_the_mammary_gland_reveals_how_oxytocin_enables_lactation_61465240/">
          <h4 class="font-weight-bold">Multiscale activity imaging in the mammary gland reveals how oxytocin enables lactation, bioRxiv, 2019-06-03</h4>
          <p>AbstractThe mammary epithelium is indispensable for the continued survival of more than 5000 mammalian species. For some, the volume of milk ejected in a single day exceeds their entire blood volume. Here, we unveil the spatiotemporal properties of physiological signals that orchestrate milk ejection. Using quantitative, multidimensional imaging of mammary cell ensembles, we reveal how stimulus-evoked Ca2&#43; oscillations couple to contraction in basal epithelial cells. Moreover, we show that Ca2&#43;-dependent contractions generate the requisite force to physically-deform the innermost layer of luminal cells, forcing them to discharge the fluid that they produced and housed. Through the collective action of thousands of these biological positive-displacement pumps, each linked to a contractile ductal network, milk is delivered into the mouth of the dependent neonate, seconds after the command.One Sentence SummaryThis study provides a window into the organization, dynamics and role of epithelial Ca2&#43; responses in the organ principally responsible for sustaining neonatal life in mammals.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/neanderthal-denisovan_ancestors_interbred_with_a_distantly-related_hominin_61425591/">
          <h4 class="font-weight-bold">Neanderthal-Denisovan ancestors interbred with a distantly-related hominin, bioRxiv, 2019-06-02</h4>
          <p>Previous research has shown that modern Eurasians interbred with their Neanderthal and Denisovan predecessors. We show here that hundreds of thousands of years earlier, the ancestors of Neanderthals and Denisovans interbred with their own Eurasian predecessors—members of a “superarchaic” population that separated from other humans about 2 mya. The superarchaic population was large, with an effective size between 20 and 50 thousand individuals. We confirm previous findings that (1) Denisovans also interbred with superarchaics, (2) Neanderthals and Denisovans separated early in the middle Pleistocene, (3) their ancestors endured a bottleneck of population size, and (4) the Neanderthal population was large at first but then declined in size. We provide qualified support for the view that (5) Neanderthals interbred with the ancestors of modern humans.One-sentence summaryWe document the earliest known interbreeding between ancient human populations and an expansion out of Africa early in the middle Pleistocene.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/the_mastery_rubric_for_bioinformatics_supporting_design_and_evaluation_of_career-spanning_education_and_training_61414123/">
          <h4 class="font-weight-bold">The Mastery Rubric for Bioinformatics supporting design and evaluation of career-spanning education and training, bioRxiv, 2019-06-02</h4>
          <p>AbstractAs the life sciences have become more data intensive, the pressure to incorporate the requisite training into life-science education and training programs has increased. To facilitate curriculum development, various sets of (bio)informatics competencies have been articulated; however, these have proved difficult to implement in practice. Addressing this issue, we have created a curriculum-design and -evaluation tool to support the development of specific Knowledge, Skills and Abilities (KSAs) that reflect the scientific method and promote both bioinformatics practice and the achievement of competencies. Twelve KSAs were extracted via formal analysis, and stages along a developmental trajectory, from uninitiated student to independent practitioner, were identified. Demonstration of each KSA by a performer at each stage was initially described (Performance Level Descriptors, PLDs), evaluated, and revised at an international workshop. This work was subsequently extended and further refined to yield the Mastery Rubric for Bioinformatics (MR-Bi). The MR-Bi was validated by demonstrating alignment between the KSAs and competencies, and its consistency with principles of adult learning. The MR-Bi tool provides a formal framework to support curriculum building, training, and self-directed learning. It prioritizes the development of independence and scientific reasoning, and is structured to allow individuals (regardless of career stage, disciplinary background, or skill level) to locate themselves within the framework. The KSAs and their PLDs promote scientific problem formulation and problem solving, lending the MR-Bi durability and flexibility. With its explicit developmental trajectory, the tool can be used by developing or practicing scientists to direct their (and their team’s) acquisition of new, or to deepen existing, bioinformatics KSAs. The MR-Bi can thereby contribute to the cultivation of a next generation of bioinformaticians who are able to design reproducible and rigorous research, and to critically analyze results from their own, and others’, work.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/scientific-communication-and-education">scientific-communication-and-education</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/rock_the_chalk_a_five-year_comparative_analysis_of_a_large_microbiology_lecture_course_reveals_improved_outcomes_of_chalk-talk_compared_to_powerpoint_61181592/">
          <h4 class="font-weight-bold">Rock the Chalk A five-year comparative analysis of a large microbiology lecture course reveals improved outcomes of chalk-talk compared to PowerPoint, bioRxiv, 2019-05-28</h4>
          <p>AbstractThe rise of electronic assisted presentation programs such as PowerPoint in undergraduate large lecture biology classes has displaced more traditional hand-drawn lectures such as the blackboard or overhead projectors, referred here as “chalk-talk” approaches. But which method is more effective in a large lecture microbiology classroom is unclear. Here I present data from a large microbial genetics lecture course taken during a five-year span comparing PowerPoint to chalk-talk lecturing methods. The results indicate that the chalk-talk approach was preferred by the students and rated higher in all measured metrics including course enjoyment, learning of key concepts, and course outcomes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/scientific-communication-and-education">scientific-communication-and-education</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/whole-genome_deep-learning_analysis_identifies_contribution_of_noncoding_mutations_to_autism_risk_61167647/">
          <h4 class="font-weight-bold">Whole-genome deep-learning analysis identifies contribution of noncoding mutations to autism risk, Nature Genetics, 2019-05-27</h4>
          <p>We address the challenge of detecting the contribution of noncoding mutations to disease with a deep-learning-based framework that predicts the specific regulatory effects and the deleterious impact of genetic variants. Applying this framework to 1,790 autism spectrum disorder (ASD) simplex families reveals a role in disease for noncoding mutations—ASD probands harbor both transcriptional- and post-transcriptional-regulation-disrupting de novo mutations of significantly higher functional impact than those in unaffected siblings. Further analysis suggests involvement of noncoding mutations in synaptic transmission and neuronal development and, taken together with previous studies, reveals a convergent genetic landscape of coding and noncoding mutations in ASD. We demonstrate that sequences carrying prioritized mutations identified in probands possess allele-specific regulatory activity, and we highlight a link between noncoding mutations and heterogeneity in the IQ of ASD probands. Our predictive genomics framework illuminates the role of noncoding mutations in ASD and prioritizes mutations with high impact for further study, and is broadly applicable to complex human diseases.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/nature%20genetics">nature genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/200-500-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/200-500-users/page/2/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="/tags/200-500-users/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/2/" class="page-link">2</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/200-500-users/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/7/" class="page-link">7</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/200-500-users/page/4/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/200-500-users/page/32/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
