<!DOCTYPE html>
<html><head>
  
  <title>200 500 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/precise_tuning_of_cortical_contractility_regulates_cell_shape_during_cytokinesis_60423481/">
          <h4 class="font-weight-bold">Precise tuning of cortical contractility regulates cell shape during cytokinesis, bioRxiv, 2019-05-13</h4>
          <p>ABSTRACTThe mechanical properties of the cellular cortex regulate shape changes during cell division, cell migration and tissue morphogenesis. During cell division, contractile force generated by the molecular motor myosin II (MII) at the equatorial cortex drives cleavage furrow ingression. Cleavage furrow ingression in turn increases stresses at the polar cortex, where contractility must be regulated to maintain cell shape during cytokinesis. How polar cortex contractility controls cell shape is poorly understood. We show a balance between MII paralogs allows a fine-tuning of cortex tension at the polar cortex to maintain cell shape during cytokinesis, with MIIA driving cleavage furrow ingression and bleb formation, and MIIB serving as a stabilizing motor and mediating completion of cytokinesis. As the majority of non-muscle contractile systems are cortical, this tuning mechanism will likely be applicable to numerous processes driven by MII contractility.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_tomato_pan-genome_uncovers_new_genes_and_a_rare_allele_regulating_fruit_flavor_60397352/">
          <h4 class="font-weight-bold">The tomato pan-genome uncovers new genes and a rare allele regulating fruit flavor, Nature Genetics, 2019-05-13</h4>
          <p>TomLoxC</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/nature%20genetics">nature genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/ultrastructure_of_the_axonal_periodic_scaffold_reveals_a_braid-like_organization_of_actin_rings_60400156/">
          <h4 class="font-weight-bold">Ultrastructure of the axonal periodic scaffold reveals a braid-like organization of actin rings, bioRxiv, 2019-05-13</h4>
          <p>AbstractRecent super-resolution microscopy studies have unveiled a periodic scaffold of actin rings regularly spaced by spectrins under the plasma membrane of axons. However, ultrastructural details are unknown, limiting a molecular and mechanistic understanding of these enigmatic structures. Here, we combine platinum-replica electron and optical super-resolution micros-copy to investigate the cortical cytoskeleton of axons at the ultrastructural level. We resolve actin rings as braids made of two long, intertwined actin filaments connected by a dense mesh of aligned spectrins. This molecular arrangement contrasts with the currently assumed model of actin rings made of short, capped actin filaments. We propose that braided rings explain the stability of the actin-spectrin scaffold and ultimately help preserving the axon integrity.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_comparison_of_edna_to_camera_trapping_for_assessment_of_terrestrial_mammal_diversity_60254828/">
          <h4 class="font-weight-bold">A comparison of eDNA to camera trapping for assessment of terrestrial mammal diversity, bioRxiv, 2019-05-11</h4>
          <p>AbstractEnvironmental DNA (eDNA) is one of the most promising approaches to meet the demand for the fast and frequent monitoring of ecosystems needed to tackle the current decline in biodiversity. However, before eDNA can establish itself as a robust alternative for mammal monitoring, comparison with existing approaches is necessary, yet has not been done. Moreover, much is unknown regarding the nature, spread and persistence of DNA shed by animals into terrestrial environments, or the optimal experimental design for understanding these potential biases.To address some of these challenges, we compared the detection of terrestrial mammals using eDNA analysis of soil samples against confirmed species observations from a long-term (∼9-yr) camera trapping study. At the same time, we considered multiple experimental parameters, including two sampling designs, two DNA extraction kits and two metabarcodes of different sizes.All mammals consistently recorded with cameras were detected in eDNA. In addition, eDNA reported many small mammals not recorded by camera traps, but whose presence in the study area is otherwise documented. A long metabarcode (≈220bp) offering a high taxonomic resolution, achieved a similar efficiency as a shorter one (≈70bp) and a phosphate buffer-based extraction gave similar results as a total DNA extraction method for a fraction of the price. Our results support that eDNA-based monitoring should become a valuable part of terrestrial mammal surveys. Yet, the lack of coverage of mammal mitochondrial genomes in public databases must be addressed before eDNA can be used to its full potential.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/variable_prediction_accuracy_of_polygenic_scores_within_an_ancestry_group_60037925/">
          <h4 class="font-weight-bold">Variable prediction accuracy of polygenic scores within an ancestry group, bioRxiv, 2019-05-07</h4>
          <p>AbstractFields as diverse as human genetics and sociology are increasingly using polygenic scores based on genome-wide association studies (GWAS) for phenotypic prediction. However, recent work has shown that polygenic scores have limited portability across groups of different genetic ancestries, restricting the contexts in which they can be used reliably and potentially creating serious inequities in future clinical applications. Using the UK Biobank data, we demonstrate that even within a single ancestry group, the prediction accuracy of polygenic scores depends on characteristics such as the age or sex composition of the individuals in which the GWAS and the prediction were conducted, and on the GWAS study design. Our findings highlight both the complexities of interpreting polygenic scores and underappreciated obstacles to their broad use.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/screening_human_embryos_for_polygenic_traits_has_limited_utility_59961829/">
          <h4 class="font-weight-bold">Screening human embryos for polygenic traits has limited utility, bioRxiv, 2019-05-06</h4>
          <p>AbstractGenome-wide association studies have led to the development of polygenic score (PS) predictors that explain increasing proportions of the variance in human complex traits. In parallel, progress in preimplantation genetic testing now allows genome-wide genotyping of embryos generated via in vitro fertilization (IVF). Jointly, these developments suggest the possibility of screening embryos for polygenic traits such as height or cognitive function. There are clear ethical, legal, and societal concerns regarding such a procedure, but these cannot be properly discussed in the absence of data on the expected outcomes of screening. Here, we use theory, simulations, and real data to evaluate the potential gain of PS-based embryo selection, defined as the expected difference in trait value between the top-scoring embryo and an average, unselected embryo. We observe that the gain increases very slowly with the number of embryos, but more rapidly with increased variance explained by the PS. Given currently available polygenic predictors and typical IVF yields, the average gain due to selection would be ≈2.5cm if selecting for height, and ≈2.5 IQ (intelligence quotient) points if selecting for cognitive function. These mean values are accompanied by wide confidence intervals; in real data drawn from nuclear families with up to 20 offspring each, we observe that the offspring with the highest PS for height was the tallest only in 25% of the families. We discuss prospects and limitations of PS-based embryo selection for the foreseeable future.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/200-500-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/200-500-users/page/11/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../../tags/200-500-users/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/200-500-users/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/200-500-users/page/10/" class="page-link">10</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/200-500-users/page/11/" class="page-link">11</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/200-500-users/page/12/" class="page-link">12</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/200-500-users/page/13/" class="page-link">13</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/200-500-users/page/14/" class="page-link">14</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/200-500-users/page/15/" class="page-link">15</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/200-500-users/page/16/" class="page-link">16</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/200-500-users/page/13/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/200-500-users/page/56/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
