<!DOCTYPE html>
<html><head>
  
  <title>Cell Biology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/etdb-caltech_a_blockchain-based_distributed_public_database_for_electron_tomography_50267721/">
          <h4 class="font-weight-bold">ETDB-Caltech a blockchain-based distributed public database for electron tomography, bioRxiv, 2018-10-26</h4>
          <p>AbstractThree-dimensional electron microscopy techniques like electron tomography provide valuable insights into cellular structures, and present significant challenges for data storage and dissemination. Here we explored a novel method to publicly release more than 11,000 such datasets, more than 30 TB in total, collected by our group. Our method, based on a peer-to-peer file sharing network built around a blockchain ledger, offers a distributed solution to data storage. In addition, we offer a user-friendly browser-based interface, &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsetdb.caltech.edu&gt;httpsetdb.caltech.edu&lt;jatsext-link&gt;, for anyone interested to explore and download our data. We discuss the relative advantages and disadvantages of this system and provide tools for other groups to mine our data andor use the same approach to share their own imaging datasets.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/fix_your_membrane_receptor_imaging_actin_cytoskeleton_and_cd4_membrane_organization_disruption_by_chemical_fixation_50127531/">
          <h4 class="font-weight-bold">Fix your membrane receptor imaging Actin cytoskeleton and CD4 membrane organization disruption by chemical fixation, bioRxiv, 2018-10-23</h4>
          <p>Single-molecule localization microscopy (SMLM) techniques allow near molecular scale resolution (~ 20nm) as well as precise and robust analysis of protein organization at different scales. SMLM hardware, analytics and probes have been the focus of a variety of studies and are now commonly used in laboratories across the world. Protocol reliability and artefact identification are increasingly seen as important aspects of super-resolution microscopy. The reliability of these approaches thus requires in-depth evaluation so that biological findings are based on solid foundations. Here we explore how different fixation approaches that disrupt or preserve the actin cytoskeleton affect membrane protein organization. Using CD4 as a model, we show that fixation-mediated disruption of the actin cytoskeleton correlates with changes in CD4 membrane organization. We highlight how these artefacts are easy to overlook and how careful sample preparation is essential for extracting meaningful results from super-resolution microscopy.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/4d_imaging_and_analysis_of_multicellular_tumour_spheroid_cell_migration_and_invasion_49708910/">
          <h4 class="font-weight-bold">4D imaging and analysis of multicellular tumour spheroid cell migration and invasion, bioRxiv, 2018-10-15</h4>
          <p>Studying and characterising tumour cell migration is critical for understanding disease progression and for assessing drug efficacy. Whilst tumour cell migration occurs fundamentally in 3 spatial dimensions (3D), for practical reasons, most migration studies to date have performed analysis in 2D. Here we imaged live multicellular tumour spheroids with lightsheet fluorescence microscopy to determine cellular migration and invasion in 3D over time (4D). We focused on glioblastoma, which are aggressive brain tumours, where cell invasion into the surrounding normal brain remains a major clinical challenge. We developed a workflow for analysing complex 3D cell movement, taking into account migration within the spheroid as well as invasion into the surrounding matrix. This provided metrics characterising cell motion, which we used to evaluate the efficacy of chemother-apeutics on invasion. These rich datasets open avenues for further studies on drug efficacy, microenvironment composition, as well as collective cell migration and metastatic potential.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/centrosome-nuclear_envelope_tethering_and_microtubule_motor-based_pulling_forces_collaborate_in_centrosome_positioning_during_mitotic_entry_49606755/">
          <h4 class="font-weight-bold">Centrosome-nuclear envelope tethering and microtubule motor-based pulling forces collaborate in centrosome positioning during mitotic entry, bioRxiv, 2018-10-13</h4>
          <p>Centrosome positioning relative to the nucleus and cell shape is highly regulated across cell types, during cell migration and during spindle formation in cell division. Across most sexually reproducing animals, centrosomes are provided to the oocyte through fertilization and must be positioned properly to establish the zygotic mitotic spindle. How centrosomes are positioned in space and time through the concerted action of key mitotic entry biochemical regulators including Protein Phosphatase 2A (PP2A-B55SUR-6), biophysical regulators including Dynein and the nuclear lamina is unclear. Here, we uncover a role for PP2A-B55SUR-6 in regulating centrosome positioning. Mechanistically, PP2A-B55SUR-6 regulates nuclear size prior to mitotic entry, in turn affecting nuclear envelope-based Dynein density and motor capacity. Using computational simulations, PP2A-B55 SUR-6 regulation of nuclear size and nuclear envelope Dynein density were both predicted to be required for proper centrosome positioning. Conversely, compromising nuclear lamina integrity led to centrosome detachment from the nuclear envelope and migration defects. Removal of PP2A-B55SUR-6 and the nuclear lamina simultaneously further disrupted centrosome positioning, leading to unseparated centrosome pairs dissociated from the nuclear envelope. Taken together, we propose a model in which centrosomes migrate and are positioned through the concerted action of nuclear envelope-based Dynein pulling forces and cen-trosome-nuclear envelope tethering.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/cytoskeletal_tension_actively_sustains_the_migratory_t_cell_synaptic_contact_49371934/">
          <h4 class="font-weight-bold">Cytoskeletal tension actively sustains the migratory T cell synaptic contact, bioRxiv, 2018-10-08</h4>
          <p>SummaryWhen migratory T cells encounter antigen presenting cells (APCs), they arrest and form radially symmetric, stable intercellular junctions termed immunological synapses which facilitate exchange of crucial biochemical information and are critical for T cell immunity. While the cellular processes underlying synapse formation have been well-characterized, those that maintain the symmetry, and thereby the stability of the synapse remain unknown. Here we identify an antigen-triggered mechanism that actively promotes T cell synapse symmetry by generating cytoskeletal tension in the plane of the synapse through focal nucleation of actin via Wiskott -Aldrich syndrome Protein (WASP), and contraction of the resultant actin filaments by myosin II. Following T cell activation, WASP is degraded, leading to cytoskeletal rearrangement and tension decay, which result in synapse breaking. Thus, our study identifies and characterizes a mechanical program within otherwise highly motile T cells that sustains the symmetry and stability of the T cell-APC synaptic contact.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/microscopy-based_chromosome_conformation_capture_enables_simultaneous_visualization_of_genome_organization_and_transcription_in_intact_organisms_49190629/">
          <h4 class="font-weight-bold">Microscopy-based chromosome conformation capture enables simultaneous visualization of genome organization and transcription in intact organisms, bioRxiv, 2018-10-04</h4>
          <p>Eukaryotic chromosomes are organized in multiple scales, from nucleosomes to chromosome territories. Recently, genome-wide methods identified an intermediate level of chromosome organization, topologically associating domains (TADs), that play key roles in transcriptional regulation. However, these methods cannot directly examine the interplay between transcriptional activation and chromosome architecture while maintaining spatial information. Here, we present a multiplexed, sequential imaging approach (Hi-M) that permits the simultaneous detection of chromosome organization and transcription in single nuclei. This allowed us to unveil the changes in 3D chromatin organization occurring upon transcriptional activation and homologous chromosome un-pairing during the awakening of the zygotic genome in intact Drosophila embryos. Excitingly, the ability of Hi-M to explore the multi-scale chromosome architecture with spatial resolution at different stages of development or during the cell cycle will be key to understand the mechanisms and consequences of the 4D organization of the genome.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/cell-biology/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/cell-biology/page/6/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/cell-biology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/cell-biology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/cell-biology/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/cell-biology/page/6/" class="page-link">6</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/cell-biology/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/cell-biology/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/cell-biology/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/cell-biology/page/10/" class="page-link">10</a></li>
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/cell-biology/page/8/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/cell-biology/page/10/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
