<!DOCTYPE html>
<html><head>
  
  <title>0-100-users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.68.3" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.95d2483cd3c5d2bf47c1f36307a456787d60ad958c5efe44db9afc47012b1b05b8e1f64f6ac291650439322a6c38e86eb7cd6254255516d857c23f299fb0e666.js" integrity="sha512-ldJIPNPF0r9HwfNjB6RWeH1grZWMXv5E25r8RwErGwW44fZPasKRZQQ5MipsOOhut81iVCVVFthXwj8pn7DmZg=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/single-cell_analysis_of_crohns_disease_lesions_identifies_a_pathogenic_cellular_module_associated_with_resistance_to_anti-tnf_therapy_52956753/">
          <h4 class="font-weight-bold">Single-cell analysis of Crohn’s disease lesions identifies a pathogenic cellular module associated with resistance to anti-TNF therapy, bioRxiv, 2018-12-21</h4>
          <p>SummaryClinical benefits to cytokine blockade in ileal Crohn’s disease (iCD) have been limited to a subset of patients. Whether cellular and molecular heterogeneity contributes to variability in treatment responses has been unclear. Using single cell technologies combining scRNAseq, CyTOF and multiplex tissue imaging, we mapped the cellular landscape of inflamed ileum lesions, adjacent non-inflamed ileum and matched circulating blood cells of iCD patients. In inflamed tissues, we identified a pathogenic module characterized by an inflammatory mononuclear phagocyte (Inf.MNP)-associated cellular response organized around inflammatory macrophages and mature dendritic cells in a subset of iCD patients. We confirmed the Inf.MNP-associated cellular response in 4 independent iCD cohorts (n=441) and showed that presence of this pathogenic module at diagnosis correlated with primary resistance to anti-TNF therapy. Single cell mapping of iCD tissues identifies a complex cellular signature of anti-TNF resistance thereby revealing novel biomarkers of treatment response and tailored therapeutic opportunities.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/precision_medicine_advancements_using_whole_genome_sequencing_noninvasive_whole_body_imaging_and_functional_diagnostics_52788699/">
          <h4 class="font-weight-bold">Precision Medicine Advancements Using Whole Genome Sequencing, Noninvasive Whole Body Imaging, and Functional Diagnostics, bioRxiv, 2018-12-18</h4>
          <p>ABSTRACTWe report the results of a three-year precision medicine study that enrolled 1190 presumed healthy participants at a single research clinic. To enable a better assessment of disease risk and improve diagnosis, a precision health platform that integrates non-invasive functional measurements and clinical tests combined with whole genome sequencing (WGS) was developed. The platform included WGS, comprehensive quantitative non-contrast whole body (WB) and brain magnetic resonance imagingangiography (MRIMRA), computed tomography (CT) coronary artery calcium scoring, electrocardiogram, echocardiogram, continuous cardiac monitoring, clinical laboratory tests, and metabolomics. In our cohort, 24.3% had medically significant genetic findings (MSF) which may contribute to increased risk of disease. A total of 206 unique medically significant variants in 111 genes were identified, and forty individuals (3.4%) had more than one MSF. Phenotypic testing revealed 34.2% of our cohort had a metabolomics profile suggestive of insulin resistance, 29.2% had elevated liver fat identified by MRI, 16.4% had clinically important cardiac structure or cardiac function abnormalities on cardiac MRI or ECHO, 8.8% had a high cardiovascular risk on CT coronary artery calcium scoring (Agatston calcium score &amp;gt; 400, Relative Risk of 7.2), 8.0% had arrhythmia found on continuous rhythm monitoring, 6.5% had cardiac conduction disorders found on EKG, 2% had previously undetected tumors detected by WB MRI, and 2.5% had previously undetected aneurysms detected by non-contrast MRIMRA. Using family histories, personal histories, and test results, clinical and phenotypic findings were correlated with genomic findings in 130 study participants (63.1%) with high to moderate penetrance variants, suggesting the precision health platform improves the diagnostic process in asymptomatic individuals who were at risk. Cardiovascular and endocrine diseases achieved considerable clinical associations between MSFs and clinical phenotypes (89% and 72%, respectively). These findings demonstrate the value of integrating WGS and noninvasive clinical assessments for a rapid and integrated point-of-care clinical diagnosis of age-related diseases that contribute to premature mortality.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/large-scale_analyses_of_human_microbiomes_reveal_thousands_of_small_novel_genes_and_their_predicted_functions_52616107/">
          <h4 class="font-weight-bold">Large-scale analyses of human microbiomes reveal thousands of small, novel genes and their predicted functions, bioRxiv, 2018-12-14</h4>
          <p>AbstractSmall proteins likely abound in prokaryotes, and may mediate much of the communication that occurs between organisms within a microbiome and their host. Unfortunately, small proteins are traditionally overlooked in biology, in part due to the computational and experimental difficulties in detecting them. To systematically identify novel small proteins, we carried out a large comparative genomics study on 1,773 HMP human-associated metagenomes from four different body sites (mouth, gut, skin and vagina). We describe more than four thousand conserved protein families, the majority of which are novel; ~30% of these protein families are predicted to be secreted or transmembrane. Over 90% of the small protein families have no known domain, and almost half are not represented in reference genomes, emphasizing the incompleteness of knowledge in this space. Our analysis exposes putative novel ‘housekeeping’ small protein families, including a potential novel ribosomally associated protein, as well as ‘mammalian-specific’ or ‘human-specific’ protein families. By analyzing the genomic neighborhood of small genes, we pinpoint a subset of families that are potentially associated with defense against bacteriophage. Finally, we identify families that may be subject to horizontal transfer and are thus potentially involved in adaptation of bacteria to the changing human environment. Our study suggest that small proteins are highly abundant and that those of the human microbiome, in particular, may perform diverse functions that have not been previously reported.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/population_structure_of_modern-day_italians_reveals_patterns_of_ancient_and_archaic_ancestries_in_southern_europe_52624180/">
          <h4 class="font-weight-bold">Population structure of modern-day Italians reveals patterns of ancient and archaic ancestries in Southern Europe, bioRxiv, 2018-12-14</h4>
          <p>European populations display low genetic diversity as the result of long term blending of the small number of ancient founding ancestries. However it is still unclear how the combination of ancient ancestries related to early European foragers, Neolithic farmers and Bronze Age nomadic pastoralists can fully explain genetic variation across Europe. Populations in natural crossroads like the Italian peninsula are expected to recapitulate the overall continental diversity, but to date have been systematically understudied. Here we characterised the ancestry profiles of modern-day Italian populations using a genome-wide dataset representative of modern and ancient samples from across Italy, Europe and the rest of the world. Italian genomes captured several ancient signatures, including a non-steppe related substantial ancestry contribution ultimately from the Caucasus. Differences in ancestry composition as the result of migration and admixture generated in Italy the largest degree of population structure detected so far in the continent and shaped the amount of Neanderthal DNA present in modern-day populations.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/structure_of_a_cytochrome-based_bacterial_nanowire_52488682/">
          <h4 class="font-weight-bold">Structure of a cytochrome-based bacterial nanowire, bioRxiv, 2018-12-11</h4>
          <p>AbstractElectrically conductive pili from Geobacter species, termed bacterial “nanowires”, are intensely studied for their biological significance and potential in the development of new materials. We have characterized a unique nanowire from conductive G. sulfurreducens pili preparations by cryo-electron microscopy composed solely of the c-type cytochrome OmcS. We present here, at 3.4 Å resolution, a novel structure of a cytochrome-based filament and discuss its possible role in long-range biological electron transport.Summary sentenceCryo-electron microscopy reveals the remarkable assembly of a c-type cytochrome into filaments comprising a heme-based bacterial nanowire.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/assignment_of_virus_and_antimicrobial_resistance_genes_to_microbial_hosts_in_a_complex_microbial_community_by_combined_long-read_assembly_and_proximity_ligation_52411464/">
          <h4 class="font-weight-bold">Assignment of virus and antimicrobial resistance genes to microbial hosts in a complex microbial community by combined long-read assembly and proximity ligation, bioRxiv, 2018-12-09</h4>
          <p>AbstractThe characterization of microbial communities by metagenomic approaches has been enhanced by recent improvements in short-read sequencing efficiency and assembly algorithms. We describe the results of adding long-read sequencing to the mix of technologies used to assemble a highly complex cattle rumen microbial community, and compare the assembly to current short read-based methods applied to the same sample. Contigs in the long-read assembly were 7-fold longer on average, and contained 7-fold more complete open reading frames (ORF), than the short read assembly, despite having three-fold lower sequence depth. The linkages between long-read contigs, provided by proximity ligation data, supported identification of 188 novel viral-host associations in the rumen microbial community that suggest cross-species infectivity of specific viral strains. The improved contiguity of the long-read assembly also identified 94 antimicrobial resistance genes, compared to only seven alleles identified in the short-read assembly. Overall, we demonstrate a combination of experimental and computational methods that work synergistically to improve characterization of biological features in a highly complex rumen microbial community.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/56/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/53/" class="page-link">53</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/54/" class="page-link">54</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/55/" class="page-link">55</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/56/" class="page-link">56</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/57/" class="page-link">57</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/58/" class="page-link">58</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/59/" class="page-link">59</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/60/" class="page-link">60</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/61/" class="page-link">61</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/58/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/123/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
