<!DOCTYPE html>
<html><head>
  
  <title>Biophysics | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/disorganization_of_the_histone_core_promotes_organization_of_heterochromatin_into_phase-separated_droplets_51385915/">
          <h4 class="font-weight-bold">Disorganization of the histone core promotes organization of heterochromatin into phase-separated droplets, bioRxiv, 2018-11-19</h4>
          <p>AbstractThe heterochromatin protein HP1 is proposed to enable chromatin compaction via liquid droplet formation. Yet, a connection between phase separation and chromatin compaction has not been experimentally demonstrated. More fundamentally, how HP1 action at the level of a single nucleosome drives chromatin compaction remains poorly understood. Here we directly demonstrate that the S. pombe HP1 protein, Swi6, compacts arrays of multiple nucleosomes into phase-separated droplets. Using hydrogen-deuterium exchange, NMR, and mass-spectrometry, we further find that Swi6 substantially increases the accessibility and dynamics of buried histone residues within a mononucleosome. Restraining these dynamics via site-specific disulfide bonds impairs the compaction of nucleosome arrays into phase-separated droplets. Our results indicate that chromatin compaction and phase separation can be highly coupled processes. Further, we find that such coupling is promoted by a counter-intuitive function of Swi6, namely disorganization of the octamer core. Phase separation is canonically mediated by weak and dynamic multivalent interactions. We propose that dynamic exposure of buried histone residues increases opportunities for multivalent interactions between nucleosomes, thereby coupling chromatin compaction to phase separation. We anticipate that this new model for chromatin organization may more generally explain the formation of highly compacted chromatin assemblies beyond heterochromatin.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/single_particle_cryo-em_reconstruction_of_52_kda_streptavidin_at_3.2_angstrom_resolution_51142456/">
          <h4 class="font-weight-bold">Single particle cryo-EM reconstruction of 52 kDa streptavidin at 3.2 Angstrom resolution, bioRxiv, 2018-11-13</h4>
          <p>AbstractThe fast development of single particle cryo-EM has made it more feasible to obtain the 3D structure of well-behaved macromolecules with molecular weight higher than 300 kDa at ~3 Å resolution. It remains a challenge to obtain high resolution structure of molecules smaller than 100 kDa using single particle cryo-EM, mainly due to the low contrast of the molecules embedded in vitreous ice. In this work, we applied the Cs-corrector-VPP coupled cryo-EM to study 52 kDa streptavidin (SA) protein supported on a thin layer of graphene film and embedded in vitreous ice. We were able to solve both the apo-SA and biotin-bound SA at near-atomic resolution using single particle cryo-EM. We demonstrated that the method is capable to determine the structure of molecule as small as 39 kDa and potentially even smaller molecules. Furthermore, we found that using the graphene film to avoid the adsorption to the air-water interface is critical to maintain the protein’s high-resolution structural information.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/the_dynamic_conformational_landscapes_of_the_protein_methyltransferase_setd8_49606751/">
          <h4 class="font-weight-bold">The Dynamic Conformational Landscapes of the Protein Methyltransferase SETD8, bioRxiv, 2018-10-13</h4>
          <p>Elucidating conformational heterogeneity of proteins is essential for understanding protein functions and developing exogenous ligands for chemical perturbation. While structural biology methods can provide atomic details of static protein structures, these approaches cannot in general resolve less populated, functionally relevant conformations and uncover conformational kinetics. Here we demonstrate a new paradigm for illuminating dynamic conformational landscapes of target proteins. SETD8 (Pr-SET7SET8KMT5A) is a biologically relevant protein lysine methyltransferase for in vivo monomethylation of histone H4 lysine 20 and nonhistone targets. Utilizing covalent chemical inhibitors and depleting native ligands to trap hidden high-energy conformational states, we obtained diverse novel X-ray structures of SETD8. These structures were used to seed massively distributed molecular simulations that generated six milliseconds of trajectory data of SETD8 in the presence or absence of its cofactor. We used an automated machine learning approach to reveal slow conformational motions and thus distinct conformational states of SETD8, and validated the resulting dynamic conformational landscapes with multiple biophysical methods. The resulting models provide unprecedented mechanistic insight into how protein dynamics plays a role in SAM binding and thus catalysis, and how this function can be modulated by diverse cancer-associated mutants. These findings set up the foundation for revealing enzymatic mechanisms and developing inhibitors in the context of conformational landscapes of target proteins.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/collective_intercellular_communication_through_ultra-fast_hydrodynamic_trigger_waves_48929251/">
          <h4 class="font-weight-bold">Collective intercellular communication through ultra-fast hydrodynamic trigger waves, bioRxiv, 2018-09-27</h4>
          <p>The biophysical relationships between sensors and actuators have been fundamental to the development of complex life forms; abundant flows are generated and persist in aquatic environments by swimming organisms, while responding promptly to external stimuli is key to survival. Here, akin to a chain reaction, we present the discovery of hydrodynamic trigger waves in cellular communities of the protist Spirostomum ambiguum, propagating hundreds of times faster than the swimming speed. Coiling its cytoskeleton, Spirostomum can contract its long body by 50% within milliseconds, with accelerations reaching 14g-forces. Surprisingly, a single cellular contraction (transmitter) is shown to generate long-ranged vortex flows at intermedi- ate Reynolds numbers, which can trigger neighbouring cells, in turn. To measure the sensitivity to hydrodynamic signals (receiver), we further present a high-throughput suction-flow device to probe mechanosensitive ion channel gating by back-calculating the microscopic forces on the cell mem- brane. These ultra-fast hydrodynamic trigger waves are analysed and modelled quantitatively in a universal framework of antenna and percolation theory. A phase transition is revealed, requiring a critical colony density to sustain collective communication. Our results suggest that this signalling could help organise cohabiting communities over large distances, influencing long-term behaviour through gene expression, comparable to quorum sensing. More immediately, as contractions release toxins, synchronised discharges could also facilitate the repulsion of large predators, or conversely immobilise large prey. We postulate that beyond protists numerous other freshwater and marine organisms could coordinate with variations of hydrodynamic trigger waves.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/relion-3_new_tools_for_automated_high-resolution_cryo-em_structure_determination_48653558/">
          <h4 class="font-weight-bold">RELION-3 new tools for automated high-resolution cryo-EM structure determination, bioRxiv, 2018-09-19</h4>
          <p>AbstractHere, we describe the third major release of relion. CPU-based vector acceleration has been added in addition to GPU support, which provides flexibility in use of resources and avoids memory limitations. Reference-free autopicking with Laplacian-of-Gaussian filtering and execution of jobs from python allows non-interactive processing during acquisition, including 2D-classification, de novo model generation and 3D-classification. Perparticle refinement of CTF parameters and correction of estimated beam tilt provides higher-resolution reconstructions when particles are at different heights in the ice, andor coma-free alignment has not been optimal. Ewald sphere curvature correction improves resolution for large particles. We illustrate these developments with publicly available data sets together with a Bayesian approach to beam-induced motion correction it leads to resolution improvements of 0.2-0.7 Å compared to previous relion versions.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/determining_cellular_ctcf_and_cohesin_abundances_to_constrain_3d_genome_models_45155888/">
          <h4 class="font-weight-bold">Determining cellular CTCF and cohesin abundances to constrain 3D genome models, bioRxiv, 2018-07-18</h4>
          <p>Achieving a quantitative and predictive understanding of 3D genome architecture remains a major challenge, as it requires quantitative measurements of the key proteins involved. Here we report the quantification of CTCF and cohesin, two causal regulators of topological associating domains (TADs) in mammalian cells. Extending our previous imaging studies (Hansen 2017), we estimate bounds on the density of putatively DNA loop-extruding cohesin complexes and CTCF binding site occupancy. Furthermore, co-immunoprecipitation studies of an endogenously tagged subunit (Rad21) suggest the presence of cohesin dimers andor oligomers. Finally, based on our cell lines with accurately measured protein abundances, we report a method to conveniently determine the number of molecules of any Halo-tagged protein in the cell. We anticipate that our results and the established tool for measuring cellular protein abundances will advance a more quantitative understanding of 3D genome organization, and facilitate protein quantification, key for understanding diverse biological processes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/biophysics/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/biophysics/page/3/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="/tags/biophysics/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biophysics/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biophysics/page/3/" class="page-link">3</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/biophysics/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biophysics/page/5/" class="page-link">5</a></li>
            
        

        
        <li class="page-item"><a href="/tags/biophysics/page/5/" rel="next" class="page-link">Next ›</a></li>
        

        
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
