<!DOCTYPE html>
<html><head>
  
  <title>Neuroscience | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/tmem119-egfp_and_tmem119-creert2_transgenic_mice_for_labeling_and_manipulating_microglia_59829485/">
          <h4 class="font-weight-bold">Tmem119-EGFP and Tmem119-CreERT2 transgenic mice for labeling and manipulating microglia, bioRxiv, 2019-05-02</h4>
          <p>AbstractMicroglia are specialized brain-resident macrophages with important functions in health and disease. To improve our understanding of these cells, the research community needs genetic tools to identify and control them in a manner that distinguishes them from closely related cell-types. We have targeted the recently discovered microglia-specific Tmem119 gene to generate knock-in mice expressing EGFP (JAX#031823) or CreERT2 (JAX#031820) for the identification and manipulation of microglia, respectively. Genetic characterization of the locus and qPCR-based analysis demonstrate correct positioning of the transgenes and intact expression of endogenous Tmem119 in the knock-in mouse models. Immunofluorescence analysis further shows that parenchymal microglia, but not other brain macrophages, are completely and faithfully labeled in the EGFP-line at different time points of development. Flow cytometry indicates highly selective expression of EGFP in CD11b&#43;CD45lo microglia. Similarly, immunofluorescence and flow cytometry analyses using a Cre-dependent reporter mouse line demonstrate activity of CreERT2 primarily in microglia upon tamoxifen administration with the caveat of activity in leptomeningeal cells. Finally, flow cytometric analyses reveal absence of EGFP expression and minimal activity of CreERT2 in blood monocytes of the Tmem119-EGFP and Tmem119-CreERT2 lines, respectively. These new transgenic lines extend the microglia toolbox by providing the currently most specific genetic labeling and control over these cells in the myeloid compartment of mice.Visual abstract&lt;jatsfig id=ufig1 position=float orientation=portrait fig-type=figure&gt;&lt;jatsgraphic xmlnsxlink=httpwww.w3.org1999xlink xlinkhref=624825v2_ufig1 position=float orientation=portrait &gt;Significance statementTools that specifically label and manipulate only microglia are currently unavailable, but are critically needed to further our understanding of this cell type. Complementing and significantly extending recently introduced microglia-specific immunostaining methods that have quickly become a new standard in the field, we generated two mouse lines that label and control gene expression in microglia with high specificity and made them publicly available. Using these readily accessible mice, the research community will be able to study microglia biology with improved specificity.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/are_place_cells_just_memory_cells_memory_compression_leads_to_spatial_tuning_and_history_dependence_59743746/">
          <h4 class="font-weight-bold">Are place cells just memory cells? Memory compression leads to spatial tuning and history dependence, bioRxiv, 2019-05-01</h4>
          <p>AbstractThe observation of place cells has suggested that the hippocampus plays a special role in encoding spatial information. However, place cell responses are modulated by several non-spatial variables, and reported to be rather unstable. Here we propose a memory model of the hippocampus that provides a novel interpretation of place cells consistent with these observations. We hypothesize that the hippocampus is a memory device that takes advantage of the correlations between sensory experiences to generate compressed representations of the episodes that are stored in memory. A simple neural network model that can efficiently compress information naturally produces place cells that are similar to those observed in experiments. It predicts that the activity of these cells is variable and that the fluctuations of the place fields encode information about the recent history of sensory experiences. Place cells may simply be a consequence of a memory compression process implemented in the hippocampus.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/hierarchical_recurrent_state_space_models_reveal_discrete_and_continuous_dynamics_of_neural_activity_in_c._elegans_59680614/">
          <h4 class="font-weight-bold">Hierarchical recurrent state space models reveal discrete and continuous dynamics of neural activity in C. elegans, bioRxiv, 2019-04-29</h4>
          <p>AbstractModern recording techniques enable large-scale measurements of neural activity in a variety of model organisms. The dynamics of neural activity shed light on how organisms process sensory information and generate motor behavior. Here, we study these dynamics using optical recordings of neural activity in the nematode C. elegans. To understand these data, we develop state space models that decompose neural time-series into segments with simple, linear dynamics. We incorporate these models into a hierarchical framework that combines partial recordings from many worms to learn shared structure, while still allowing for individual variability. This framework reveals latent states of population neural activity, along with the discrete behavioral states that govern dynamics in this state space. We find stochastic transition patterns between discrete states and see that transition probabilities are determined by both current brain activity and sensory cues. Our methods automatically recover transition times that closely match manual labels of different behaviors, such as forward crawling, reversals, and turns. Finally, the resulting model can simulate neural data, faithfully capturing salient patterns of whole brain dynamics seen in real data.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/population_imaging_of_neural_activity_in_awake_behaving_mice_in_multiple_brain_regions_59416424/">
          <h4 class="font-weight-bold">Population imaging of neural activity in awake behaving mice in multiple brain regions, bioRxiv, 2019-04-24</h4>
          <p>AbstractA longstanding goal in neuroscience has been to image membrane voltage, with high temporal precision and sensitivity, in awake behaving mammals. Here, we report a genetically encoded voltage indicator, SomArchon, which exhibits millisecond response times and compatibility with optogenetic control, and which increases the sensitivity, signal-to-noise ratio, and number of neurons observable, by manyfold over previous reagents. SomArchon only requires conventional one-photon microscopy to achieve these high performance characteristics. These improvements enable population analysis of neural activity, both at the subthreshold and spiking levels, in multiple brain regions – cortex, hippocampus, and striatum – of awake behaving mice. Using SomArchon, we detect both positive and negative responses of striatal neurons during movement, highlighting the power of voltage imaging to reveal bidirectional modulation. We also examine how the intracellular subthreshold theta oscillations of hippocampal neurons govern spike output, finding that nearby cells can exhibit highly correlated subthreshold activities, even as they generate highly divergent spiking patterns.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/alzheimers_patient_brain_myeloid_cells_exhibit_enhanced_aging_and_unique_transcriptional_activation_59250676/">
          <h4 class="font-weight-bold">Alzheimer’s patient brain myeloid cells exhibit enhanced aging and unique transcriptional activation, bioRxiv, 2019-04-19</h4>
          <p>AbstractGene expression changes in brain microglia from mouse models of Alzheimer’s disease (AD) are highly characterized and reflect specific myeloid cell activation states that could modulate AD risk or progression. While some groups have produced valuable expression profiles for human brain cells1–4, the cellular clarity with which we now view transcriptional responses in mouse AD models has not yet been realized for human AD tissues due to limited availability of fresh tissue samples and technological hurdles of recovering transcriptomic data with cell-type resolution from frozen samples. We developed a novel method for isolating multiple cell types from frozen post-mortem specimens of superior frontal gyrus for RNA-Seq and identified 66 genes differentially expressed between AD and control subjects in the myeloid cell compartment. Myeloid cells sorted from fusiform gyrus of the same subjects showed similar changes, and whole tissue RNA analyses further corroborated our findings. The changes we observed did not resemble the “damage-associated microglia” (DAM) profile described in mouse AD models5, or other known activation states from other disease models. Instead, roughly half of the changes were consistent with an “enhanced human aging” phenotype, whereas the other half, including the AD risk gene APOE, were altered in AD myeloid cells but not differentially expressed with age. We refer to this novel profile in human Alzheimer’s microgliamyeloid cells as the HAM signature. These results, which can be browsed at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpresearch-pub.gene.comBrainMyeloidLandscapereviewVersion&gt;research-pub.gene.comBrainMyeloidLandscapereviewVersion&lt;jatsext-link&gt;, highlight considerable differences between myeloid activation in mouse models and human disease, and provide a genome-wide picture of brain myeloid activation in human AD.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/automated_analysis_of_whole_brain_vasculature_using_machine_learning_59217253/">
          <h4 class="font-weight-bold">Automated analysis of whole brain vasculature using machine learning, bioRxiv, 2019-04-19</h4>
          <p>SUMMARYTissue clearing methods enable imaging of intact biological specimens without sectioning. However, reliable and scalable analysis of such large imaging data in 3D remains a challenge. Towards this goal, we developed a deep learning-based framework to quantify and analyze the brain vasculature, named Vessel Segmentation &amp;amp; Analysis Pipeline (VesSAP). Our pipeline uses a fully convolutional network with a transfer learning approach for segmentation. We systematically analyzed vascular features of the whole brains including their length, bifurcation points and radius at the micrometer scale by registering them to the Allen mouse brain atlas. We reported the first evidence of secondary intracranial collateral vascularization in CD1-Elite mice and found reduced vascularization in the brainstem as compared to the cerebrum. VesSAP thus enables unbiased and scalable quantifications for the angioarchitecture of the cleared intact mouse brain and yields new biological insights related to the vascular brain function.GRAPHICAL ABSTRACT&lt;jatsfig id=ufig1 position=float fig-type=figure orientation=portrait&gt;Supporting material of VesSAP is available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpDISCOtechnologies.orgVesSAP&gt;httpDISCOtechnologies.orgVesSAP&lt;jatsext-link&gt;&lt;jatsgraphic xmlnsxlink=httpwww.w3.org1999xlink xlinkhref=613257_ufig1 position=float orientation=portrait &gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/neuroscience/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/neuroscience/page/13/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/neuroscience/page/10/" class="page-link">10</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/neuroscience/page/11/" class="page-link">11</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/neuroscience/page/12/" class="page-link">12</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/neuroscience/page/13/" class="page-link">13</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/neuroscience/page/14/" class="page-link">14</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/neuroscience/page/15/" class="page-link">15</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/neuroscience/page/16/" class="page-link">16</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/neuroscience/page/17/" class="page-link">17</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/neuroscience/page/18/" class="page-link">18</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/neuroscience/page/15/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/neuroscience/page/53/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
