<!DOCTYPE html>
<html><head>
  
  <title>Neuroscience | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/neocortical_layer_4_in_adult_mouse_differs_in_major_cell_types_and_circuit_organization_between_primary_sensory_areas_53203823/">
          <h4 class="font-weight-bold">Neocortical layer 4 in adult mouse differs in major cell types and circuit organization between primary sensory areas, bioRxiv, 2018-12-28</h4>
          <p>AbstractLayer 4 (L4) of mammalian neocortex plays a crucial role in cortical information processing, yet a complete census of its cell types and connectivity remains elusive. Using whole-cell recordings with morphological recovery, we identified one major excitatory and seven inhibitory types of neurons in L4 of adult mouse visual cortex (V1). Nearly all excitatory neurons were pyramidal and all somatostatin-positive (SOM&#43;) non-fast-spiking neurons were Martinotti cells. In contrast, in somatosensory cortex (S1), excitatory neurons were mostly stellate and SOM&#43; neurons were non-Martinotti. These morphologically distinct SOM&#43; interneurons corresponded to different transcriptomic cell types and were differentially integrated into the local circuit with only S1 neurons receiving local excitatory input. We propose that cell-type specific circuit motifs, such as the Martinottipyramidal and non-Martinottistellate pairs, are optionally used across the cortex as building blocks to assemble cortical circuits.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/endogenous_variation_in_ventromedial_prefrontal_cortex_state_dynamics_during_naturalistic_viewing_reflects_affective_experience_52756628/">
          <h4 class="font-weight-bold">Endogenous variation in ventromedial prefrontal cortex state dynamics during naturalistic viewing reflects affective experience, bioRxiv, 2018-12-17</h4>
          <p>AbstractHow we process ongoing experiences is shaped by our personal history, current needs, and future goals. Consequently, brain regions involved in generating these subjective appraisals, such as the vmPFC, often appear to be heterogeneous across individuals even in response to the same external information. To elucidate the role of the vmPFC in processing our ongoing experiences, we developed a computational framework and analysis pipeline to characterize the spatiotemporal dynamics of individual vmPFC responses as participants viewed a 45-minute television drama. Through a combination of functional magnetic resonance imaging, facial expression tracking, and self-reported emotional experiences across four studies, our data suggest that the vmPFC slowly transitions through a series of discretized states that broadly map onto affective experiences. Although these transitions typically occur at idiosyncratic times across people, participants exhibited a marked increase in state alignment during high affectively valenced events in the show. Our work suggests that the vmPFC ascribes affective meaning to our ongoing experiences.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/chemogenetic_ligands_for_translational_neurotheranostics_52352703/">
          <h4 class="font-weight-bold">Chemogenetic ligands for translational neurotheranostics, bioRxiv, 2018-12-08</h4>
          <p>AbstractDesigner Receptors Exclusively Activated by Designer Drugs (DREADDs) are a popular chemogenetic technology for manipulation of neuronal activity in uninstrumented awake animals with potential for precision medicine-based clinical theranostics. DREADD ligands developed to date are not appropriate for such translational applications. The prototypical DREADD agonist clozapine N-oxide (CNO) lacks brain entry and converts to clozapine. The second-generation DREADD agonist, Compound 21 (C21), was developed to overcome these limitations. We found that C21 has low brain penetrance, weak affinity, and low in vivo DREADD occupancy. To address these drawbacks, we developed two new DREADD agonists, JHU37152 and JHU37160, and the first dedicated positron emission tomography (PET) DREADD radiotracer, [18F]JHU37107. JHU37152 and JHU37160 exhibit high in vivo DREADD potency. [18F]JHU37107 combined with PET allows for DREADD detection in locally-targeted neurons and at their long-range projections, enabling for the first time, noninvasive and longitudinal neuronal projection mapping and potential for neurotheranostic applications.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/intelligible_speech_synthesis_from_neural_decoding_of_spoken_sentences_52011021/">
          <h4 class="font-weight-bold">Intelligible speech synthesis from neural decoding of spoken sentences, bioRxiv, 2018-11-30</h4>
          <p>The ability to read out, or decode, mental content from brain activity has significant practical and scientific implications. For example, technology that translates cortical activity into speech would be transformative for people unable to communicate as a result of neurological impairment. Decoding speech from neural activity is challenging because speaking requires extremely precise and dynamic control of multiple vocal tract articulators on the order of milliseconds. Here, we designed a neural decoder that explicitly leverages the continuous kinematic and sound representations encoded in cortical activity to generate fluent and intelligible speech. A recurrent neural network first decoded vocal tract physiological signals from direct cortical recordings, and then transformed them to acoustic speech output. Robust decoding performance was achieved with as little as 25 minutes of training data. Naive listeners were able to accurately identify these decoded sentences. Additionally, speech decoding was not only effective for audibly produced speech, but also when participants silently mimed speech. These results advance the development of speech neuroprosthetic technology to restore spoken communication in patients with disabling neurological disorders.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/hunger_for_knowledge_how_the_irresistible_lure_of_curiosity_is_generated_in_the_brain_51561076/">
          <h4 class="font-weight-bold">Hunger for Knowledge How the Irresistible Lure of Curiosity is Generated in the Brain, bioRxiv, 2018-11-22</h4>
          <p>Introductory ParagraphCuriosity is often portrayed as a desirable feature of human faculty. For example, a meta-analysis revealed that curiosity predicts academic performance above and beyond intelligence1, corroborating findings that curiosity supported long-term consolidation of learning2,3. However, curiosity may come at a cost of strong seductive power that sometimes puts people in a harmful situation. Here, with a set of three behavioural and two neuroimaging experiments including novel stimuli that strongly trigger curiosity (i.e. magic tricks), we examined the psychological and neural mechanisms underlying the irresistible lure of curiosity. We consistently demonstrated that across different samples people were indeed willing to gamble to expose themselves to physical risks (i.e. electric shocks) in order to satisfy their curiosity for trivial knowledge that carries no apparent instrumental values. Also, underlying this seductive power of curiosity is its incentive salience properties, which share common neural mechanisms with extrinsic incentives (i.e. hunger for foods). In particular, the two independent fMRI experiments using different kinds of curiosity-stimulating stimuli found replicable results that acceptance (compared to rejection) of curiosityincentive-driven gambles was accompanied with an enhanced activity in the striatum.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/deep_neural_networks_and_kernel_regression_achieve_comparable_accuracies_for_functional_connectivity_prediction_of_behavior_and_demographics_51420417/">
          <h4 class="font-weight-bold">Deep Neural Networks and Kernel Regression Achieve Comparable Accuracies for Functional Connectivity Prediction of Behavior and Demographics, bioRxiv, 2018-11-20</h4>
          <p>AbstractThere is significant interest in the development and application of deep neural networks (DNNs) to neuroimaging data. A growing literature suggests that DNNs outperform their classical counterparts in a variety of neuroimaging applications, yet there are few direct comparisons of relative utility. Here, we compared the performance of three DNN architectures and a classical machine learning algorithm (kernel regression) in predicting individual phenotypes from whole-brain resting-state functional connectivity (RSFC) patterns. One of the DNNs was a generic fully-connected feedforward neural network, while the other two DNNs were recently published approaches specifically designed to exploit the structure of connectome data. By using a combined sample of almost 10,000 participants from the Human Connectome Project (HCP) and UK Biobank, we showed that the three DNNs and kernel regression achieved similar performance across a wide range of behavioral and demographic measures. Furthermore, the generic feedforward neural network exhibited similar performance to the two state-of-the-art connectome-specific DNNs. When predicting fluid intelligence in the UK Biobank, performance of all algorithms dramatically improved when sample size increased from 100 to 1000 subjects. Improvement was smaller, but still significant, when sample size increased from 1000 to 5000 subjects. Importantly, kernel regression was competitive across all sample sizes. Overall, our study suggests that kernel regression is as effective as DNNs for RSFC-based behavioral prediction, while incurring significantly lower computational costs. Therefore, kernel regression might serve as a useful baseline algorithm for future studies.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/neuroscience/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/neuroscience/page/13/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/neuroscience/page/10/" class="page-link">10</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/neuroscience/page/11/" class="page-link">11</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/neuroscience/page/12/" class="page-link">12</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/neuroscience/page/13/" class="page-link">13</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/neuroscience/page/14/" class="page-link">14</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/neuroscience/page/15/" class="page-link">15</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/neuroscience/page/16/" class="page-link">16</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/neuroscience/page/17/" class="page-link">17</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/neuroscience/page/18/" class="page-link">18</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/neuroscience/page/15/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/neuroscience/page/29/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
