<!DOCTYPE html>
<html><head>
  
  <title>200 500 Users | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/highly-accurate_long-read_sequencing_improves_variant_detection_and_assembly_of_a_human_genome_supplementary_material_53881485/">
          <h4 class="font-weight-bold">Highly-accurate long-read sequencing improves variant detection and assembly of a human genome Supplementary Material, bioRxiv, 2019-01-13</h4>
          <p>The major DNA sequencing technologies in use today produce either highly-accurate short reads or noisy long reads. We developed a protocol based on single-molecule, circular consensus sequencing (CCS) to generate highly-accurate (99.8%) long reads averaging 13.5 kb and applied it to sequence the well-characterized human HG002NA24385. We optimized existing tools to comprehensively detect variants, achieving precision and recall above 99.91% for SNVs, 95.98% for indels, and 95.99% for structural variants. We estimate that 2,434 discordances are correctable mistakes in the high-quality Genome in a Bottle benchmark. Nearly all (99.64%) variants are phased into haplotypes, which further improves variant detection. De novo assembly produces a highly contiguous and accurate genome with contig N50 above 15 Mb and concordance of 99.998%. CCS reads match short reads for small variant detection, while enabling structural variant detection and de novo assembly at similar contiguity and markedly higher concordance than noisy long reads.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/false-positive_neuroimaging_undisclosed_flexibility_in_testing_spatial_hypotheses_allows_presenting_anything_as_a_replicated_finding_53697323/">
          <h4 class="font-weight-bold">False-positive neuroimaging Undisclosed flexibility in testing spatial hypotheses allows presenting anything as a replicated finding, bioRxiv, 2019-01-09</h4>
          <p>AbstractHypothesis testing in neuroimaging studies relies heavily on treating named anatomical regions (e.g., “the amygdala”) as unitary entities. Though data collection and analyses are conducted at the voxel level, inferences are often based on anatomical regions. The discrepancy between the unit of analysis and the unit of inference leads to ambiguity and flexibility in analyses that can create a false sense of reproducibility. For example, hypothesizing effects on “amygdala activity” does not provide a falsifiable and reproducible definition of precisely which voxels or which patterns of activation should be observed. Rather, it comprises a large number of unspecified sub-hypotheses, leaving room for flexible interpretation of findings, which we refer to as “model degrees of freedom.” From a survey of 135 functional Magnetic Resonance Imaging studies in which researchers claimed replications of previous findings, we found that 42.2% of the studies did not report any quantitative evidence for replication such as activation peaks. Only 14.1% of the papers used exact coordinate-based or a priori pattern-based models. Of the studies that reported peak information, 42.9% of the ‘replicated’ findings had peak coordinates more than 15 mm away from the ‘original’ findings, suggesting that different brain locations were activated, even when studies claimed to replicate prior results. To reduce the flexible and qualitative region-level tests in neuroimaging studies, we recommend adopting quantitative spatial models and tests to assess the spatial reproducibility of findings. Techniques reviewed here include permutation tests on peak distance, Bayesian MANOVA, and a priori multivariate pattern-based models. These practices will help researchers to establish precise and falsifiable spatial hypotheses, promoting a cumulative science of neuroimaging.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/optical_clearing_of_living_brains_with_magical_to_extend_in_vivo_imaging_53697807/">
          <h4 class="font-weight-bold">Optical clearing of living brains with MAGICAL to extend in vivo imaging, bioRxiv, 2019-01-09</h4>
          <p>To understand brain functions, it is important to observe directly how multiple neural circuits are performing in living brains. However, due to tissue opaqueness, observable depth and spatiotemporal resolution are severely degraded in vivo. Here, we propose an optical brain clearing method for in vivo fluorescence microscopy, termed MAGICAL (Magical Additive Glycerol Improves Clear Alive Luminance). MAGICAL enabled two-photon microscopy to capture vivid images with fast speed, at cortical layer V and hippocampal CA1 in vivo. Moreover, MAGICAL promoted conventional confocal microscopy to visualize finer neuronal structures including synaptic boutons and spines in unprecedented deep regions, without intensive illumination leading to phototoxic effects. Fluorescence Emission Spectrum Transmissive Analysis (FESTA) showed that MAGICAL improved in vivo transmittance of shorter wavelength light, which is vulnerable to optical scattering thus unsuited for in vivo microscopy. These results suggest that MAGICAL would transparentize living brains via scattering reduction.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/repurposing_large_health_insurance_claims_data_to_estimate_genetic_and_environmental_contributions_in_560_phenotypes_53917059/">
          <h4 class="font-weight-bold">Repurposing large health insurance claims data to estimate genetic and environmental contributions in 560 phenotypes, Nature Genetics, 2019-01-08</h4>
          <p>h</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/nature%20genetics">nature genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/ecological_causes_of_uneven_diversification_and_richness_in_the_mammal_tree_of_life_53474875/">
          <h4 class="font-weight-bold">Ecological causes of uneven diversification and richness in the mammal tree of life, bioRxiv, 2019-01-05</h4>
          <p>AbstractThe uneven distribution of species in the tree of life is rooted in unequal speciation and extinction among groups. Yet the causes of differential diversification are little known despite their relevance for sustaining biodiversity into the future. Here we investigate rates of species diversification across extant Mammalia, a compelling system that includes our own closest relatives. We develop a new phylogeny of nearly all ∼6000 species using a 31-gene supermatrix and fossil node- and tip-dating approaches to establish a robust evolutionary timescale for mammals. Our findings link the causes of uneven modern species richness with ecologically-driven variation in rates of speciation andor extinction, including 24 detected shifts in net diversification. Speciation rates are a stronger predictor of among-clade richness than clade age, countering claims of clock-like speciation in large phylogenies. Surprisingly, speciation rate heterogeneity in recent radiations shows limited association with latitude, despite the well-known increase in species richness toward the equator. Instead, we find a deeper-time association where clades of high-latitude species have the highest speciation rates, suggesting that species durations are shorter (turnover is higher) outside than inside the tropics. At shallower timescales (i.e., young clades), diurnality and low vagility are both linked to greater speciation rates and extant richness. We suggest that high turnover among small-ranged allopatric species has erased the signal of vagility in older clades, while diurnality has adaptively promoted lineage persistence. These findings highlight the underappreciated joint roles of ephemeral (turnover-based) and adaptive (persistence-based) processes of diversification, which manifest in recent and more ancient evolutionary radiations of mammals to explain modern diversity.Author SummaryThe over 6000 living species in the mammalian tree of life are distributed unevenly among branches so that similarly aged groups sometimes differ many fold in species richness (e.g., ∼2500 rodent species versus 8 pangolins). Why differential bursts of species diversification occur, and how long they persist, has implications for sustaining biodiversity. Here we develop a robust evolutionary timescale for most extant species, recovering signatures of rate-variable diversification linked to ecological factors. Mammals with low dispersal or that are day-active show the fastest recent speciation rates, consistent with mechanisms of allopatric isolation and ecological opportunity, respectively. Speciation is surprisingly faster in extra-tropical than tropical lineages, suggesting that longer species durations for tropical lineages underpin the latitudinal diversity gradient in mammal.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/recording_of_sonic_attacks_on_u.s._diplomats_in_cuba_spectrally_matches_the_echoing_call_of_a_caribbean_cricket_53461299/">
          <h4 class="font-weight-bold">Recording of sonic attacks on U.S. diplomats in Cuba spectrally matches the echoing call of a Caribbean cricket, bioRxiv, 2019-01-04</h4>
          <p>Beginning in late 2016, diplomats posted to the United States embassy in Cuba began to experience unexplained health problems including ear pain, tinnitus, vertigo, and cognitive difficulties which reportedly began after they heard strange noises in their homes or hotel rooms. In response, the U.S. government dramatically reduced the number of diplomats posted at the U.S. embassy in Havana. U.S. officials initially believed a sonic attack might be responsible for their ailments. The sound linked to these attacks, which has been described as a high-pitched beam of sound, was recorded by U.S. personnel in Cuba and released by the Associated Press (AP). Because these recordings are the only available non-medical evidence of the sonic attacks, much attention has focused on identifying health problems and the origin of the acoustic signal. As shown here, the calling song of the Indies short-tailed cricket (Anurogryllus celerinictus) matches, in nuanced detail, the AP recording in duration, pulse repetition rate, power spectrum, pulse rate stability, and oscillations per pulse. The AP recording also exhibits frequency decay in individual pulses, a distinct acoustic signature of cricket sound production. While the temporal pulse structure in the recording is unlike any natural insect source, when the cricket call is played on a loudspeaker and recorded indoors, the interaction of reflected sound pulses yields a sound virtually indistinguishable from the AP sample. This provides strong evidence that an echoing cricket call, rather than a sonic attack or other technological device, is responsible for the sound in the released recording. Although the causes of the health problems reported by embassy personnel are beyond the scope of this paper, our findings highlight the need for more rigorous research into the source of these ailments, including the potential psychogenic effects, as well as possible physiological explanations unrelated to sonic attacks.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/200-500-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/200-500-users/page/15/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/12/" class="page-link">12</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/13/" class="page-link">13</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/14/" class="page-link">14</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/15/" class="page-link">15</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/200-500-users/page/16/" class="page-link">16</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/17/" class="page-link">17</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/18/" class="page-link">18</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/19/" class="page-link">19</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/200-500-users/page/20/" class="page-link">20</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/200-500-users/page/17/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/200-500-users/page/32/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
