<!DOCTYPE html>
<html><head>
  
  <title>Biorxiv | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/genome-wide_quantification_of_the_effects_of_dna_methylation_on_human_gene_regulation_20880241/">
          <h4 class="font-weight-bold">Genome-wide quantification of the effects of DNA methylation on human gene regulation, bioRxiv, 2017-06-08</h4>
          <p>AbstractChanges in DNA methylation are important in development and disease, but not all regulatory elements act in a methylation-dependent (MD) manner. Here, we developed mSTARR-seq, a high-throughput approach to quantify the effects of DNA methylation on regulatory element function. We assay MD activity in 14% of the euchromatic human genome, identify 2,143 MD regulatory elements, and predict MD activity using sequence and chromatin state information. We identify transcription factors associated with higher activity in unmethylated or methylated states, including an association between pioneer transcription factors and methylated DNA. Finally, we use mSTARR-seq to predict DNA methylation-gene expression correlations in primary cells. Our findings provide a map of MD regulatory activity across the human genome, facilitating interpretation of the many emerging associations between methylation and trait variation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/non-invasive_laminar_inference_with_meg_comparison_of_methods_and_source_inversion_algorithms_20883873/">
          <h4 class="font-weight-bold">Non-invasive laminar inference with MEG Comparison of methods and source inversion algorithms, bioRxiv, 2017-06-08</h4>
          <p>AbstractMagnetoencephalography (MEG) is a direct measure of neuronal current flow; its anatomical resolution is therefore not constrained by physiology but rather by data quality and the models used to explain these data. Recent simulation work has shown that it is possible to distinguish between signals arising in the deep and superficial cortical laminae given accurate knowledge of these surfaces with respect to the MEG sensors. This previous work has focused around a single inversion scheme (multiple sparse priors) and a single global parametric fit metric (free energy). In this paper we use several different source inversion algorithms and both local and global, as well as parametric and non-parametric fit metrics in order to demonstrate the robustness of the discrimination between layers. We find that only algorithms with some sparsity constraint can successfully be used to make laminar discrimination. Importantly, local t-statistics, global cross-validation and free energy all provide robust and mutually corroborating metrics of fit. We show that discrimination accuracy is affected by patch size estimates, cortical surface features, and lead field strength, which suggests several possible future improvements to this technique. This study demonstrates the possibility of determining the laminar origin of MEG sensor activity, and thus directly testing theories of human cognition that involve laminar- and frequency- specific mechanisms. This possibility can now be achieved using recent developments in high precision MEG, most notably the use of subject-specific head-casts, which allow for significant increases in data quality and therefore anatomically precise MEG recordings.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/resetting_the_yeast_epigenome_with_human_nucleosomes_20896431/">
          <h4 class="font-weight-bold">Resetting the yeast epigenome with human nucleosomes, bioRxiv, 2017-06-08</h4>
          <p>SummaryHumans and yeast are separated by a billion years of evolution, yet their conserved core histones retain central roles in gene regulation. Here, we “reset” yeast to use core human nucleosomes in lieu of their own, an exceedingly rare event which initially took twenty days. The cells adapt, however, by acquiring suppressor mutations in cell-division genes, or by acquiring certain aneuploidy states. Robust growth was also restored by converting five histone residues back to their yeast counterparts. We reveal that humanized nucleosomes in yeast are positioned according to endogenous yeast DNA sequence and chromatin-remodeling network, as judged by a yeast-like nucleosome repeat length. However, human nucleosomes have higher DNA occupancy and reduce RNA content. Adaptation to new biological conditions presented a special challenge for these cells due to slower chromatin remodeling. This humanized yeast poses many fundamental new questions about the nature of chromatin and how it is linked to many cell processes, and provides a platform to study histone variants via yeast epigenome reprogramming.Highlights&lt;jatslist list-type=simple&gt;&lt;jatslist-item&gt;- Only 1 in 107 yeast survive with fully human nucleosomes, but they rapidly evolve&lt;jatslist-item&gt;&lt;jatslist-item&gt;- Nucleosome positioning and nucleosome repeat length is not influenced by histone type&lt;jatslist-item&gt;&lt;jatslist-item&gt;- Human nucleosomes remodel slowly and delay yeast environmental adaptation&lt;jatslist-item&gt;&lt;jatslist-item&gt;- Human core nucleosomes are more repressive and globally reduce transcription in yeast&lt;jatslist-item&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/synthetic-biology">synthetic-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/ancient_genomes_from_southern_africa_pushes_modern_human_divergence_beyond_260000_years_ago_20841926/">
          <h4 class="font-weight-bold">Ancient genomes from southern Africa pushes modern human divergence beyond 260,000 years ago, bioRxiv, 2017-06-06</h4>
          <p>Southern Africa is consistently placed as one of the potential regions for the evolution of          Homo sapiens          . To examine the region&#39;s human prehistory prior to the arrival of migrants from East and West Africa or Eurasia in the last 1,700 years, we generated and analyzed genome sequence data from seven ancient individuals from KwaZulu-Natal, South Africa. Three Stone Age hunter-gatherers date to ~2,000 years ago, and we show that they were related to current-day southern San groups such as the Karretjie People. Four Iron Age farmers (300-500 years old) have genetic signatures similar to present day Bantu-speakers. The genome sequence (13x coverage) of a juvenile boy from Ballito Bay, who lived ~2,000 years ago, demonstrates that southern African Stone Age hunter-gatherers were not impacted by recent admixture; however, we estimate that all modern-day Khoekhoe and San groups have been influenced by 9-22% genetic admixture from East AfricanEurasian pastoralist groups arriving &amp;gt;1,000 years ago, including the Ju|&#39;hoansi San, previously thought to have very low levels of admixture. Using traditional and new approaches, we estimate the population divergence time between the Ballito Bay boy and other groups to beyond 260,000 years ago. These estimates dramatically increases the deepest divergence amongst modern humans, coincide with the onset of the Middle Stone Age in sub-Saharan Africa, and coincide with anatomical developments of archaic humans into modern humans as represented in the local fossil record. Cumulatively, cross-disciplinary records increasingly point to southern Africa as a potential (not necessarily exclusive) &#39;hot spot&#39; for the evolution of our species.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/detecting_polygenic_adaptation_in_admixture_graphs_20819640/">
          <h4 class="font-weight-bold">Detecting polygenic adaptation in admixture graphs, bioRxiv, 2017-06-06</h4>
          <p>AbstractAn open question in human evolution is the importance of polygenic adaptation adaptive changes in the mean of a multifactorial trait due to shifts in allele frequencies across many loci. In recent years, several methods have been developed to detect polygenic adaptation using loci identified in genome-wide association studies (GWAS). Though powerful, these methods suffer from limited interpretability they can detect which sets of populations have evidence for polygenic adaptation, but are unable to reveal where in the history of multiple populations these processes occurred. To address this, we created a method to detect polygenic adaptation in an admixture graph, which is a representation of the historical divergences and admixture events relating different populations through time. We developed a Markov chain Monte Carlo (MCMC) algorithm to infer branch-specific parameters reflecting the strength of selection in each branch of a graph. Additionally, we developed a set of summary statistics that are fast to compute and can indicate which branches are most likely to have experienced polygenic adaptation. We show via simulations that this method - which we call PolyGraph - has good power to detect polygenic adaptation, and applied it to human population genomic data from around the world. We also provide evidence that variants associated with several traits, including height, educational attainment, and self-reported unibrow, have been influenced by polygenic adaptation in different populations during human evolution.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/distinct_neuronal_activity_patterns_induce_different_gene_expression_programs_20841401/">
          <h4 class="font-weight-bold">Distinct neuronal activity patterns induce different gene expression programs, bioRxiv, 2017-06-06</h4>
          <p>SUMMARYBrief and sustained neuronal activity patterns can have opposite effects on synaptic strength that both require activity-regulated gene (ARG) expression. However, whether distinct patterns of activity induce different sets of ARGs is unknown. In genome-scale experiments, we reveal that a neuron’s activity-pattern history can be predicted from the ARGs it expresses. Surprisingly, brief activity selectively induces a small subset of the ARG program that that corresponds precisely to the first of three temporal waves of genes induced by sustained activity. These first-wave genes are distinguished by an open chromatin state, proximity to rapidly activated enhancers, and a requirement for MAPKERK signaling for their induction. MAPKERK mediates rapid RNA polymerase recruitment to promoters, as well as enhancer RNA induction but not histone acetylation at enhancers. Thus, the same mechanisms that establish the multi-wave temporal structure of ARG induction also enable different sets of genes to be induced by distinct activity patterns.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/biorxiv/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/biorxiv/page/156/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/153/" class="page-link">153</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/154/" class="page-link">154</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/155/" class="page-link">155</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/156/" class="page-link">156</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/biorxiv/page/157/" class="page-link">157</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/158/" class="page-link">158</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/159/" class="page-link">159</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/160/" class="page-link">160</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/161/" class="page-link">161</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/biorxiv/page/158/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/biorxiv/page/164/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
