<!DOCTYPE html>
<html><head>
  
  <title>0-100-users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.68.3" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.95d2483cd3c5d2bf47c1f36307a456787d60ad958c5efe44db9afc47012b1b05b8e1f64f6ac291650439322a6c38e86eb7cd6254255516d857c23f299fb0e666.js" integrity="sha512-ldJIPNPF0r9HwfNjB6RWeH1grZWMXv5E25r8RwErGwW44fZPasKRZQQ5MipsOOhut81iVCVVFthXwj8pn7DmZg=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/antibody_against_envelope_protein_from_human_endogenous_retrovirus_activates_neutrophils_in_systemic_lupus_erythematosus_66923674/">
          <h4 class="font-weight-bold">Antibody against envelope protein from human endogenous retrovirus activates neutrophils in systemic lupus erythematosus, bioRxiv, 2019-09-21</h4>
          <p>AbstractNeutrophil activation and the formation of neutrophil extracellular trap (NET) are hallmarks of innate immune activation in systemic lupus erythematosus (SLE) and contribute to the systemic interferon signature. Here we report that the expression of an endogenous retrovirus (ERV) locus ERV-K102, encoding an envelope protein, was significantly elevated in SLE patient blood and was correlated with higher interferon status. Induction of ERV-K102 expression most strongly correlated with reduced transcript levels of epigenetic silencing factors. SLE IgG promoted phagocytosis of ERV-K102 envelope protein by neutrophils through immune complex formation. ERV immune complex phagocytosis resulted in subsequent NET formation consisting of DNA, neutrophil elastase, and citrullinated histone H3. Finally, analysis of anti-ERV-K102 IgG in SLE patients showed that IgG2 likely mediates this effect. Together, we identified an immunostimulatory ERV-K envelope protein elevated in SLE that may be a target of SLE IgG and able to promote neutrophil activation.eTOC summaryUsing ERVmap, the authors determined that the expression of ERV-K102 locus was elevated in SLE patient blood and correlated with the interferon signature. The envelope protein encoded by this locus activates human neutrophils through immune complex formation with SLE IgG.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/gene_capture_by_transposable_elements_leads_to_epigenetic_conflict_66909156/">
          <h4 class="font-weight-bold">Gene capture by transposable elements leads to epigenetic conflict, bioRxiv, 2019-09-21</h4>
          <p>ABSTRACTPlant transposable elements (TEs) regularly capture fragments of host genes. When the host employs siRNAs to silence these TEs, siRNAs homologous to the captured regions may target both the TEs and the genes, potentially leading to their silencing. This epigenetic cross-talk establishes an intragenomic conflict silencing the TEs comes with the potential cost of silencing the genes. If the genes are important, however, natural selection will act to maintain function by moderating the silencing response. Such moderation may advantage the TEs. Here, we examined the potential for these epigenetic conflicts by focusing on three TE families in maize - Helitrons, Pack-MULEs and Sirevirus LTR retrotransposons. We documented 1,508 TEs with fragments captured from 2,019 donor genes and characterized the epigenetic profiles of both. Consistent with epigenetic conflict, donor genes mapped more siRNAs and were more methylated than ‘free’ genes that had no evidence of exon capture. However, these patterns differed between syntelog vs. transposed donor genes. Syntelog genes appeared to maintain function, consistent with moderation of the epigenetic response for important genes before reaching a deleterious threshold, while transposed genes bore the signature of silencing and potential pseudogenization. Intriguingly, transposed genes were overrepresented among donor genes, suggesting a link between capture and gene movement. We also investigated the potential for TEs to gain an advantage. TEs with captured fragments were older, mapped fewer siRNAs and had lower levels of methylation than ‘free’ TEs without gene fragments, but they showed no obvious evidence of increased copy numbers. Altogether, our results demonstrate that TE capture triggers an epigenetic conflict when genes are important, contrasting the loss of function for genes that are not under strong selective constraint. The evidence for an advantage to TEs is currently less obvious.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_genome_of_caenorhabditis_bovis_66855615/">
          <h4 class="font-weight-bold">The genome of Caenorhabditis bovis, bioRxiv, 2019-09-20</h4>
          <p>AbstractThe free-living nematode Caenorhabditis elegans is a key laboratory model for metazoan biology. C. elegans is also used as a model for parasitic nematodes despite being only distantly related to most parasitic species. All ∼65 Caenorhabditis species currently in culture are free-living with most having been isolated from decaying plant or fungal matter. Caenorhabditis bovis is a particularly unusual species, having been isolated several times from the inflamed ears of Zebu cattle in Eastern Africa where it is believed to be the cause of bovine parasitic otitis. C. bovis is therefore of particular interest to researchers interested in the evolution of nematode parasitism and in Caenorhabditis diversity. However, as C. bovis is not in laboratory culture, it remains little studied and details of its prevalence, role in bovine parasitic otitis and relationships to other Caenorhabditis species are scarce. Here, by sampling livestock markets and slaughterhouses in Western Kenya, we successfully reisolate C. bovis from the ear of adult female Zebu. We sequence the genome of C. bovis using the Oxford Nanopore MinION platform in a nearby field laboratory and use the data to generate a chromosome-scale draft genome sequence. We exploit this draft genome to reconstruct the phylogenetic relationships of C. bovis to other Caenorhabditis species and reveal the changes in genome size and content that have occurred during its evolution. We also identify expansions in several gene families that have been implicated in parasitism in other nematode species, including those associated with resistance to antihelminthic drugs. The high-quality draft genome and our analyses thereof represent a significant advancement in our understanding of this unusual Caenorhabditis species.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/precision_calcium_imaging_of_dense_neural_populations_via_a_cell_body-targeted_calcium_indicator_66752354/">
          <h4 class="font-weight-bold">Precision calcium imaging of dense neural populations via a cell body-targeted calcium indicator, bioRxiv, 2019-09-18</h4>
          <p>AbstractMethods for one-photon fluorescent imaging of calcium dynamics in vivo are popular due to their ability to simultaneously capture the dynamics of hundreds of neurons across large fields of view, at a low equipment complexity and cost. In contrast to two-photon methods, however, one-photon methods suffer from higher levels of crosstalk between cell bodies and the surrounding neuropil, resulting in decreased signal-to-noise and artifactual correlations of neural activity. Here, we address this problem by engineering cell body-targeted variants of the fluorescent calcium indicator GCaMP6f. We screened fusions of GCaMP6f to both natural as well as engineered peptides, and identified fusions that localized GCaMP6f to within approximately 50 microns of the cell body of neurons in live mice and larval zebrafish. One-photon imaging of soma-targeted GCaMP6f in dense neural circuits reported fewer artifactual spikes from neuropil, increased signal-to-noise ratio, and decreased artifactual correlation across neurons. Thus, soma-targeting of fluorescent calcium indicators increases neuronal signal fidelity and may facilitate even greater usage of simple, powerful, one-photon methods of population imaging of neural calcium dynamics.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/an_improved_de_novo_assembly_and_annotation_of_the_tomato_reference_genome_using_single-molecule_sequencing_hi-c_proximity_ligation_and_optical_maps_66592888/">
          <h4 class="font-weight-bold">An improved de novo assembly and annotation of the tomato reference genome using single-molecule sequencing, Hi-C proximity ligation and optical maps, bioRxiv, 2019-09-14</h4>
          <p>AbstractThe original Heinz 1706 reference genome was produced by a large team of scientists from across the globe from a variety of input sources that included 454 sequences in addition to full-length BACs, BAC and fosmid ends sequenced with Sanger technology. We present here the latest tomato reference genome (SL4.0) assembled de novo from PacBio long reads and scaffolded using Hi-C contact maps. The assembly was validated using Bionano optical maps and 10X linked-read sequences. This assembly is highly contiguous with fewer gaps compared to previous genome builds and almost all scaffolds have been anchored and oriented to the 12 tomato chromosomes. We have found more repeats compared to the previous versions and one of the largest repeat classes identified are the LTR retrotransposons. We also describe updates to the reference genome and annotation since the last publication. The corresponding ITAG4.0 annotation has 4,794 novel genes along with 29,281 genes preserved from ITAG2.4. Most of the updated genes have extensions in the 5’ and 3’ UTRs resulting in doubling of annotated UTRs per gene. The genome and annotation can be accessed using SGN through BLAST database, Pathway database (SolCyc), Apollo, JBrowse genome browser and FTP available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpssolgenomics.net&gt;httpssolgenomics.net&lt;jatsext-link&gt;.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/self-organised_symmetry_breaking_in_zebrafish_reveals_feedback_from_morphogenesis_to_pattern_formation_66593193/">
          <h4 class="font-weight-bold">Self-organised symmetry breaking in zebrafish reveals feedback from morphogenesis to pattern formation, bioRxiv, 2019-09-14</h4>
          <p>A fundamental question in developmental biology is how the early embryo breaks initial symmetry to establish the spatial coordinate system later important for the organisation of the embryonic body plan. In zebrafish, this is thought to depend on the inheritance of maternal mRNAs [1–3], cortical rotation to generate a dorsal pole of beta-catenin activity [4–8] and the release of Nodal signals from the yolk syncytial layer (YSL) [9–12]. Recent work aggregating mouse embryonic stem cells has shown that symmetry breaking can occur in the absence of extra-embryonic tissue [19,20]. To test whether this is also true in zebrafish, we separated embryonic cells from the yolk and allowed them to develop as aggregates. These aggregates break symmetry autonomously to form elongated structures with an anterior-posterior pattern. Extensive cell mixing shows that any pre-existing asymmetry is lost prior to the breaking morphological symmetry, revealing that the maternal pre-pattern is not strictly required for early embryo patterning. Following early signalling events after isolation of embryonic cells reveals that a pole of Nodal activity precedes and is required for elongation. The blocking of PCP-dependent convergence and extension movements disrupts the establishment of opposing poles of BMP and WntTCF activity and the patterning of anterior-posterior neural tissue. These results lead us to suggest that convergence and extension plays a causal role in the establishment of morphogen gradients and pattern formation during zebrafish gastrulation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/9/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/9/" class="page-link">9</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/10/" class="page-link">10</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/11/" class="page-link">11</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/12/" class="page-link">12</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/13/" class="page-link">13</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/14/" class="page-link">14</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/11/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/123/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
