<!DOCTYPE html>
<html><head>
  
  <title>2019 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/margo_massively_automated_real-time_gui_for_object-tracking_a_platform_for_high-throughput_ethology_58113148/">
          <h4 class="font-weight-bold">MARGO (Massively Automated Real-time GUI for Object-tracking), a platform for high-throughput ethology, bioRxiv, 2019-03-30</h4>
          <p>AbstractFast object tracking in real time allows convenient tracking of very large numbers of animals and closed-loop experiments that control stimuli for multiple animals in parallel. We developed MARGO, a real-time animal tracking suite for custom behavioral experiments. We demonstrated that MARGO can rapidly and accurately track large numbers of animals in parallel over very long timescales. We incorporated control of peripheral hardware, and implemented a flexible software architecture for defining new experimental routines. These features enable closed-loop delivery of stimuli to many individuals simultaneously. We highlight MARGO’s ability to coordinate tracking and hardware control with two custom behavioral assays (measuring phototaxis and optomotor response) and one optogenetic operant conditioning assay. There are currently several open source animal trackers. MARGO’s strengths are 1) robustness, 2) high throughput, 3) flexible control of hardware and 4) real-time closed-loop control of sensory and optogenetic stimuli, all of which are optimized for large-scale experimentation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/single_nucleotide_mapping_of_the_locally_accessible_trait_space_in_yeast_reveals_pareto_fronts_that_constrain_initial_adaptation_58119203/">
          <h4 class="font-weight-bold">Single Nucleotide Mapping of the Locally Accessible Trait Space in Yeast Reveals Pareto Fronts that Constrain Initial Adaptation, bioRxiv, 2019-03-30</h4>
          <p>AbstractTradeoffs constrain the improvement of performance of multiple traits simultaneously. Such tradeoffs define Pareto fronts, which represent a set of optimal individuals that cannot be improved in any one trait without reducing performance in another. Surprisingly, experimental evolution often yields genotypes with improved performance in all measured traits, perhaps indicating an absence of tradeoffs at least in the short-term. Here we densely sample adaptive mutations in S. cerevisiae to ask whether first-step adaptive mutations result in tradeoffs during the growth cycle. We isolated thousands of adaptive clones evolved under carefully chosen conditions and quantified their performances in each part of the growth cycle. We too find that some first-step adaptive mutations can improve all traits to a modest extent. However, our dense sampling allowed us to identify tradeoffs and establish the existence of Pareto fronts between fermentation and respiration, and between respiration and stationary phases. Moreover, we establish that no single mutation in the ancestral genome can circumvent the detected tradeoffs. Finally, we sequenced hundreds of these adaptive clones, revealing novel targets of adaptation and defining the genetic basis of the identified tradeoffs.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/eztrack_an_open-source_video_analysis_pipeline_for_the_investigation_of_animal_behavior_58113144/">
          <h4 class="font-weight-bold">ezTrack An open-source video analysis pipeline for the investigation of animal behavior, bioRxiv, 2019-03-30</h4>
          <p>AbstractTracking small animal behavior by video is one of the most common tasks in the fields of neuroscience and psychology. Although commercial software exists for the execution of this task, commercial software often presents enormous cost to the researcher, and can also entail purchasing specific hardware setups that are not only expensive but lack adaptability. Moreover, the inaccessibility of the code underlying this software renders them inflexible. Alternatively, available open source options frequently require extensive model training and can be challenging for those inexperienced with programming. Here we present an open source and platform independent set of behavior analysis pipelines using interactive Python (iPythonJupyter Notebook) that researchers with no prior programming experience can use. Two modules are described. One module can be used for the positional analysis of an individual animal across a session (i.e., location tracking), amenable to a wide range of behavioral tasks including conditioned place preference, water maze, light-dark box, open field, and elevated plus maze, to name but a few. A second module is described for the analysis of conditioned freezing behavior. For both modules, a range of interactive plots and visualizations are available to confirm that chosen parameters produce results that conform to the user’s approval. In addition, batch processing tools for the fast analysis of multiple videos is provided, and frame-by-frame output makes aligning the data with neural recording data simple. Lastly, options for cropping video frames to mitigate the influence of fiberopticelectrophysiology cables, analyzing specified portions of time in a video, and defining regions of interest, can be implemented with ease.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/an_approximate_full-likelihood_method_for_inferring_selection_and_allele_frequency_trajectories_from_dna_sequence_data_57983959/">
          <h4 class="font-weight-bold">An approximate full-likelihood method for inferring selection and allele frequency trajectories from DNA sequence data, bioRxiv, 2019-03-29</h4>
          <p>AbstractMost current methods for detecting natural selection from DNA sequence data are limited in that they are either based on summary statistics or a composite likelihood, and as a consequence, do not make full use of the information available in DNA sequence data. We here present a new importance sampling approach for approximating the full likelihood function for the selection coefficient. The method treats the ancestral recombination graph (ARG) as a latent variable that is integrated out using previously published Markov Chain Monte Carlo (MCMC) methods. The method can be used for detecting selection, estimating selection coefficients, testing models of changes in the strength of selection, estimating the time of the start of a selective sweep, and for inferring the allele frequency trajectory of a selected or neutral allele. We perform extensive simulations to evaluate the method and show that it uniformly improves power to detect selection compared to current popular methods such as nSL and SDS, under various demographic models and can provide reliable inferences of allele frequency trajectories under many conditions. We also explore the potential of our method to detect extremely recent changes in the strength of selection. We use the method to infer the past allele frequency trajectory for a lactase persistence SNP (MCM6) in Europeans. We also study a set of 11 pigmentation-associated variants. Several genes show evidence of strong selection particularly within the last 5,000 years, including ASIP, KITLG, and TYR. However, selection on OCA2HERC2 seems to be much older and, in contrast to previous claims, we find no evidence of selection on TYRP1.Author summaryCurrent methods to study natural selection using modern population genomic data are limited in their power and flexibility. Here, we present a new method to infer natural selection that builds on recent methodological advances in estimating genome-wide genealogies. By using importance sampling we are able to efficiently estimate the likelihood function of the selection coefficient. We show our method improves power to test for selection over competing methods across a diverse range of scenarios, and also accurately infers the selection coefficient. We also demonstrate a novel capability of our model, using it to infer the allele’s frequency over time. We validate these results with a study of a lactase persistence SNP in Europeans, and also study a set of 11 pigmentation-associated variants.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/clinical_use_of_current_polygenic_risk_scores_may_exacerbate_health_disparities_58049280/">
          <h4 class="font-weight-bold">Clinical use of current polygenic risk scores may exacerbate health disparities, Nature Genetics, 2019-03-29</h4>
          <p>Polygenic risk scores (PRS) are poised to improve biomedical outcomes via precision medicine. However, the major ethical and scientific challenge surrounding clinical implementation of PRS is that those available today are several times more accurate in individuals of European ancestry than other ancestries. This disparity is an inescapable consequence of Eurocentric biases in genome-wide association studies, thus highlighting that—unlike clinical biomarkers and prescription drugs, which may individually work better in some populations but do not ubiquitously perform far better in European populations—clinical uses of PRS today would systematically afford greater improvement for European-descent populations. Early diversifying efforts show promise in leveling this vast imbalance, even when non-European sample sizes are considerably smaller than the largest studies to date. To realize the full and equitable potential of PRS, greater diversity must be prioritized in genetic studies, and summary statistics must be publically disseminated to ensure that health disparities are not increased for those individuals already most underserved.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/nature%20genetics">nature genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/polygenic_architecture_of_human_neuroanatomical_diversity_57963714/">
          <h4 class="font-weight-bold">Polygenic architecture of human neuroanatomical diversity, bioRxiv, 2019-03-29</h4>
          <p>AbstractWe analysed the genomic architecture of neuroanatomical diversity using magnetic resonance imaging and SNP data from &amp;gt; 20,000 individuals. Our results replicate previous findings of a strong polygenic architecture of neuroanatomical diversity. SNPs captured from 40% to 54% of the variance in the volume of different brain regions. We observed a large correlation between chromosome length and the amount of phenotypic variance captured, r∼0.64 on average, suggesting that at a global scale causal variants are homogeneously distributed across the genome. At a more local scale, SNPs within genes (∼51%) captured ∼1.5-times more genetic variance than the rest; and SNPs with low minor allele frequency (MAF) captured significantly less variance than those with higher MAF the 40% of SNPs with MAF&amp;lt;5% captured less than one fourth of the genetic variance. We also observed extensive pleiotropy across regions, with an average genetic correlation of rG∼0.45. Across regions, genetic correlations were in general similar to phenotypic correlations. By contrast, genetic correlations were larger than phenotypic correlations for the leftright volumes of the same region, and indistinguishable from 1. Additionally, the differences in leftright volumes were not heritable, underlining the role of environmental causes in the variability of brain asymmetry. Our analysis code is available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsgithub.comneuroanatomygenomic-architecture&gt;httpsgithub.comneuroanatomygenomic-architecture&lt;jatsext-link&gt;.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/2019/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/2019/page/81/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/78/" class="page-link">78</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/79/" class="page-link">79</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/80/" class="page-link">80</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/81/" class="page-link">81</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/2019/page/82/" class="page-link">82</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/83/" class="page-link">83</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/84/" class="page-link">84</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/85/" class="page-link">85</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/2019/page/86/" class="page-link">86</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/2019/page/83/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/2019/page/135/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
