<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_creatures_of_the_human_cortical_somatosensory_system_62520606/">
          <h4 class="font-weight-bold">The “creatures” of the human cortical somatosensory system, bioRxiv, 2019-06-22</h4>
          <p>AbstractPenfield’s description of the “homunculus”, a “grotesque creature” with large lips and hands and small trunk and legs depicting the representation of body-parts within the primary somatosensory cortex (S1), is one of the most prominent contributions to the neurosciences. Since then, numerous studies have identified additional body-parts representations outside of S1. Nevertheless, it has been implicitly assumed that S1’s homunculus is representative of the entire somatosensory cortex. Therefore, the distribution of body-parts representations in other brain regions, the property that gave Penfield’s homunculus its famous “grotesque” appearance, has been overlooked. We used whole-body somatosensory stimulation, functional MRI and a new cortical parcellation to quantify the organization of the cortical somatosensory representation. Our analysis showed first, an extensive somatosensory response over the cortex; and second, that the proportional representation of body-parts differs substantially between major neuroanatomical regions and from S1, with, for instance, much larger trunk representation at higher brain regions, potentially in relation to the regions’ functional specialization. These results extend Penfield’s initial findings to the higher level of somatosensory processing and suggest a major role for somatosensation in human cognition.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_live-cell_screen_for_altered_erk_dynamics_reveals_principles_of_proliferative_control_62418075/">
          <h4 class="font-weight-bold">A live-cell screen for altered Erk dynamics reveals principles of proliferative control, bioRxiv, 2019-06-20</h4>
          <p>Complex, time-varying responses have been observed widely in cell signaling, but how specific dynamics are generated or regulated is largely unknown. One major obstacle has been that high-throughput screens for identifying pathway components are typically incompatible with the live-cell assays used to monitor dynamics. Here, we address this challenge by performing a drug screen for altered Erk signaling dynamics in primary mouse keratinocytes. We screened a library of 429 kinase inhibitors, monitoring Erk activity over 5 h in more than 80,000 single live cells. The screen revealed both known and uncharacterized modulators of Erk dynamics, including inhibitors of non-EGFR receptor tyrosine kinases (RTKs) that increased Erk pulse frequency and overall activity. Using drug treatment and direct optogenetic control, we demonstrate that drug-induced changes to Erk dynamics alter the conditions under which cells proliferate. Our work opens the door to high-throughput screens using live-cell biosensors and reveals that cell proliferation integrates information from Erk dynamics as well as additional permissive cues.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/higher_fitness_yeast_genotypes_are_less_robust_to_deleterious_mutations_62449614/">
          <h4 class="font-weight-bold">Higher fitness yeast genotypes are less robust to deleterious mutations, bioRxiv, 2019-06-20</h4>
          <p>AbstractNatural selection drives populations towards higher fitness, but second-order selection for adaptability and mutational robustness can also influence the dynamics of adaptation. In many microbial systems, diminishing returns epistasis contributes to a tendency for more-fit genotypes to be less adaptable, but no analogous patterns for robustness are known. To understand how robustness varies across genotypes, we measure the fitness effects of hundreds of individual insertion mutations in a panel of yeast strains. We find that more-fit strains are less robust they have distributions of fitness effects (DFEs) with lower mean and higher variance. These shifts in the DFE arise because many mutations have more strongly deleterious effects in faster-growing strains. This negative correlation between fitness and robustness implies that second-order selection for robustness will tend to conflict with first-order selection for fitness.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/compensatory_sequence_variation_between_trans-species_small_rnas_and_their_target_sites_62412912/">
          <h4 class="font-weight-bold">Compensatory sequence variation between trans-species small RNAs and their target sites, bioRxiv, 2019-06-19</h4>
          <p>AbstractTrans-species small regulatory RNAs (sRNAs) are delivered to host plants from diverse pathogens and parasites and can target host mRNAs. How trans-species sRNAs can be effective on diverse hosts has been unclear. Multiple species of the parasitic plant Cuscuta produce trans-species sRNAs that collectively target many host mRNAs. Confirmed target sites are nearly always in highly conserved, protein-coding regions of host mRNAs. Cuscuta trans-species sRNAs can be grouped into superfamilies that have variation in a three-nucleotide period. These variants compensate for synonymous-site variation in host mRNAs. By targeting host mRNAs at highly conserved protein-coding sites, and simultaneously expressing multiple variants to cover synonymous-site variation, Cuscuta trans-species sRNAs may be able to successfully target homologous mRNAs from diverse hosts.One Sentence SummaryThe parasitic plant Cuscuta produces a diverse set of sRNAs that compensate for sequence variation in mRNA targets in diverse hosts.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/passenger_hotspot_mutations_in_cancer_62409958/">
          <h4 class="font-weight-bold">Passenger Hotspot Mutations in Cancer, bioRxiv, 2019-06-19</h4>
          <p>AbstractHotspots, or mutations that recur at the same genomic site across multiple tumors, have been conventionally interpreted as strong universal evidence of somatic positive selection, unequivocally pinpointing genes driving tumorigenesis. Here, we demonstrate that this convention is falsely premised on an inaccurate statistical model of background mutagenesis. Many hotspots are in fact passenger events, recurring at sites that are simply inherently more mutable rather than under positive selection, which current background models do not account for. We thus detail a log-normal-Poisson (LNP) background model that accounts for variation in site-specific mutability in a manner consistent with models of mutagenesis, use this model to show that the tendency to generate passenger hotspots pervades all common mutational processes, and apply it to a ~10, 000 patient cohort from The Cancer Genome Atlas to nominate driver hotspots with far fewer false positives compared to conventional methods. As the biomedical community faces critical decisions in prioritizing putative driver mutations for deep experimental characterization to assess therapeutic potential, we offer our findings as a guide to avoid wasting valuable scientific resources on passenger hotspots.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/transcriptome_dynamics_reveals_progressive_transition_from_effector_to_memory_in_cd4&#43;_t_cells_62407963/">
          <h4 class="font-weight-bold">Transcriptome Dynamics Reveals Progressive Transition from Effector to Memory in CD4&#43; T cells, bioRxiv, 2019-06-19</h4>
          <p>AbstractCD4&#43; T cells are repositories of immune memory, conferring enhanced immunity to many infectious agents. Studies of acute viral and bacterial infection suggest that memory CD4&#43; T cells develop directly from effectors. However, delineating these dynamic developmental pathways has been challenging. Here, we used high-resolution single-cell RNA-seq and temporal mixture modelling to examine the fate of Th1 and Tfh effector cells during non-lethal Plasmodium infection in mice. We observed linear Th1 and Tfh pathways towards memory, characterized by progressive halving in the numbers of genes expressed, and partial transcriptomic coalescence. Low-level persisting infection diverted but did not block these pathways. We observed in the Th1-pathway a linear transition from Th1 through a Tr1 state to TEM cells, which were then poised for Th1 re-call. The Tfh-pathway exhibited a modest Th1-signature throughout, with little evidence of Tr1 development, and co-expression of TCM and memory Tfh markers. Thus, we present a high-resolution atlas of transcriptome dynamics for naïve to memory transitions in CD4&#43; T cells. We also defined a subset of memory-associated genes, including transcription factors Id2 and Maf, whose expression increased progressively against the background of transcriptomic quiescence. Single-cell ATAC-seq revealed substantial heterogeneity in chromatin accessibility in single effectors, which was extensively, though incompletely reset and homogenized in memory. Our data reveal that linear transitions from effector to memory occur in a progressive manner over several weeks, suggesting opportunities for manipulating CD4&#43; T cell memory after primary infection.Highlights&lt;jatslist list-type=bullet&gt;&lt;jatslist-item&gt;scRNA-seq reveals progressive transition from effector to memory in CD4&#43; T cells.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Transcriptome dynamics suggest linear not branching models for memory development.&lt;jatslist-item&gt;&lt;jatslist-item&gt;A subset of genes associates with gradual onset of CD4&#43; T cell memory.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Th1Tfh predisposition varies among clonotypes with identical antigen-specificity.&lt;jatslist-item&gt;&lt;jatslist-item&gt;scATAC-seq uncovers non-coding “memory” elements in the genome.&lt;jatslist-item&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/0-100-users/page/21/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/18/" class="page-link">18</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/19/" class="page-link">19</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/20/" class="page-link">20</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/21/" class="page-link">21</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/0-100-users/page/22/" class="page-link">22</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/23/" class="page-link">23</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/24/" class="page-link">24</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/25/" class="page-link">25</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/26/" class="page-link">26</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/0-100-users/page/23/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/0-100-users/page/123/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
