<!DOCTYPE html>
<html><head>
  
  <title>2018 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/latent_developmental_potential_to_form_limb-like_skeletal_structures_in_zebrafish_50117443/">
          <h4 class="font-weight-bold">Latent developmental potential to form limb-like skeletal structures in zebrafish, bioRxiv, 2018-10-23</h4>
          <p>AbstractThe evolution of fins into limbs was a key transition in vertebrate history. A hallmark of this transition is the addition of multiple long bones to the proximal-distal axis of paired appendages. Whereas limb skeletons are often elaborate and diverse, teleost pectoral fins retain a simple endoskeleton. Fins and limbs share many core developmental processes, but how these programs were reshaped to produce limbs from fins during evolution remains enigmatic. Here we identify zebrafish mutants that form supernumerary long bones along the proximal-distal axis of pectoral fins with limb-like patterning. These new skeletal elements are integrated into the fin, as they are connected to the musculature, form joints, and articulate with neighboring bones. This phenotype is caused by activating mutations in previously unrecognized regulators of appendage development, vav2 and waslb, which we show function in a common pathway. We find that this pathway functions in appendage development across vertebrates, and loss of Wasl in developing limbs results in patterning defects identical to those seen in Hoxall knockout mice. Concordantly, formation of supernumerary fin long bones requires the function of hoxall paralogs, indicating developmental homology with the forearm and the existence of a latent functional Hox code patterning the fin endoskeleton. Our findings reveal an inherent limb-like patterning ability in fins that can be activated by simple genetic perturbation, resulting in the elaboration of the endoskeleton.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/polygenic_adaptation_from_sweeps_to_subtle_frequency_shifts_50132793/">
          <h4 class="font-weight-bold">Polygenic Adaptation From sweeps to subtle frequency shifts, bioRxiv, 2018-10-23</h4>
          <p>Evolutionary theory has produced two conflicting paradigms for the adaptation of a polygenic trait. While population genetics views adaptation as a sequence of selective sweeps at single loci underlying the trait, quantitative genetics posits a collective response, where phenotypic adaptation results from subtle allele frequency shifts at many loci. Yet, a synthesis of these views is largely missing and the population genetic factors that favor each scenario are not well understood. Here, we study the architecture of adaptation of a binary polygenic trait (such as resistance) with negative epistasis among the loci of its basis. The genetic structure of this trait allows for a full range of potential architectures of adaptation, ranging from sweeps to small frequency shifts. By combining computer simulations and a newly devised analytical framework based on Yule branching processes, we gain a detailed understanding of the adaptation dynamics for this trait. Our key analytical result is an expression for the joint distribution of mutant alleles at the end of the adaptive phase. This distribution characterizes the polygenic pattern of adaptation at the underlying genotype when phenotypic adaptation has been accomplished. We find that a single compound parameter, the population-scaled background mutation rate Θbg, explains the main differences among these patterns. For a focal locus, Θbg measures the mutation rate at all redundant loci in its genetic background that offer alternative ways for adaptation. For adaptation starting from mutation-selection-drift balance, we observe different patterns in three parameter regions. Adaptation proceeds by sweeps for small Θbg ≲0.1, while small polygenic allele frequency shifts require large Θbg ≳100. In the large intermediate regime, we observe a heterogeneous pattern of partial sweeps at several interacting loci.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_hippocampus_is_necessary_for_the_sleep-dependent_consolidation_of_a_task_that_does_not_require_the_hippocampus_for_initial_learning_50127132/">
          <h4 class="font-weight-bold">The hippocampus is necessary for the sleep-dependent consolidation of a task that does not require the hippocampus for initial learning, bioRxiv, 2018-10-23</h4>
          <p>AbstractDuring sleep, the hippocampus plays an active role in consolidating memories that depend on it for initial encoding. There are hints in the literature that the hippocampus may have a broader influence, contributing to the consolidation of memories that may not initially require the area. We tested this possibility by evaluating learning and consolidation of the motor sequence task (MST) in hippocampal amnesics and demographically matched control participants. While the groups showed similar initial learning, only controls exhibited evidence of sleep-dependent consolidation. These results demonstrate that the hippocampus can be required for normal consolidation of a task without being required for its acquisition, suggesting that the area plays a broader role in coordinating sleep-dependent memory consolidation than has previously been assumed.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/adding_function_to_the_genome_of_african_salmonella_st313_50075876/">
          <h4 class="font-weight-bold">Adding function to the genome of African Salmonella ST313, bioRxiv, 2018-10-22</h4>
          <p>AbstractSalmonella Typhimurium ST313 causes invasive nontyphoidal Salmonella (iNTS) disease in sub-Saharan Africa, targeting susceptible HIV&#43;, malarial or malnourished individuals. An in-depth genomic comparison between the ST313 isolate D23580, and the well-characterized ST19 isolate 474 that causes gastroenteritis across the globe, revealed extensive synteny. To understand how the 856 nucleotide variations generated phenotypic differences, we devised a large-scale experimental approach that involved the global gene expression analysis of strains D23580 and 474 grown in sixteen infection-relevant growth conditions. Comparison of transcriptional patterns identified virulence and metabolic genes that were differentially expressed between D23580 versus 474, many of which were validated by proteomics. We also uncovered the S. Typhimurium D23580 and 474 genes that showed expression differences during infection of murine macrophages. Our comparative transcriptomic data are presented in a new enhanced version of the Salmonella expression compendium SalComD23580 &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpbioinf.gen.tcd.iecgi-binsalcom_v2.pl&gt;bioinf.gen.tcd.iecgi-binsalcom_v2.pl&lt;jatsext-link&gt;. We discovered that the ablation of melibiose utilization was caused by 3 independent SNP mutations in D23580 that are shared across ST313 lineage 2, suggesting that the ability to catabolise this carbon source has been negatively selected during ST313 evolution. The data revealed a novel plasmid maintenance system involving a plasmid-encoded CysS cysteinyl-tRNA synthetase, highlighting the power of large-scale comparative multi-condition analyses to pinpoint key phenotypic differences between bacterial pathovariants.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/navigome_navigating_the_human_phenome_50081743/">
          <h4 class="font-weight-bold">Navigome Navigating the Human Phenome, bioRxiv, 2018-10-22</h4>
          <p>AbstractWe now have access to a sufficient number of genome-wide association studies (GWAS) to cluster phenotypes into genetic-informed categories and to navigate the “phenome” space of human traits. Using a collection of 465 GWAS, we generated genetic correlations, pathways, gene-wise and tissue-wise associations using MAGMA and S-PrediXcan for 465 human traits. Testing 7267 biological pathways, we found that only 898 were significantly associated with any trait. Similarly, out of ~20,000 tested protein-coding genes, 12,311 genes exhibited an association. Based on the genetic correlations between all traits, we constructed a phenome map using t-distributed stochastic neighbor embedding (t-SNE), where each of the 465 traits can be visualized as an individual point. This map reveals well-defined clusters of traits such as educationhigh longevity, lower longevity, height, body composition, and depressionanxietyneuroticism. These clusters are enriched in specific groups of pathways, such as lipid pathways in the lower longevity cluster, and neuronal pathways for body composition or education clusters. The map and all other analyses are available in the Navigome web interface (&lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsphenviz.navigome.com&gt;httpsphenviz.navigome.com&lt;jatsext-link&gt;).</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/polygenic_scores_for_major_depressive_disorder_and_depressive_symptoms_predict_response_to_lithium_in_patients_with_bipolar_disorder_50081597/">
          <h4 class="font-weight-bold">Polygenic scores for major depressive disorder and depressive symptoms predict response to lithium in patients with bipolar disorder, bioRxiv, 2018-10-22</h4>
          <p>AbstractBackgroundLithium is a first-line medication for bipolar disorder (BD), but only ~30% of patients respond optimally to the drug. Since genetic factors are known to mediate lithium treatment response, we hypothesized whether polygenic susceptibility to the spectrum of depression traits is associated with treatment outcomes in patients with BD. In addition, we explored the potential molecular underpinnings of this relationship.MethodsWeighted polygenic scores (PGSs) were computed for major depressive disorder (MDD) and depressive symptoms (DS) in BD patients from the Consortium on Lithium Genetics (ConLi&#43;Gen; n=2,586) who received lithium treatment. Lithium treatment outcome was assessed using the ALDA scale. Summary statistics from genome-wide association studies (GWAS) in MDD (130,664 cases and 330,470 controls) and DS (n=161,460) were used for PGS weighting. Associations between PGSs of depression traits and lithium treatment response were assessed by binary logistic regression. We also performed a cross-trait meta-GWAS, followed by Ingenuity® Pathway Analysis.OutcomesBD patients with a low polygenic load for depressive traits were more likely to respond well to lithium, compared to patients with high polygenic load (MDD OR =1.64 [95%CI 1.26-2.15], lowest vs highest PGS quartiles; DS OR=1.53 [95%CI 1.18-2.00]). Associations were significant for type 1, but not type 2 BD. Cross-trait GWAS and functional characterization implicated voltage-gated potassium channels, insulin-related pathways, mitogen-activated protein-kinase (MAPK) signaling, and miRNA expression.InterpretationGenetic loading to depression traits in BD patients lower their odds of responding optimally to lithium. Our findings support the emerging concept of a lithium-responsive biotype in BD.FundingSee attached details</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/24/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/21/" class="page-link">21</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/22/" class="page-link">22</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/23/" class="page-link">23</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/24/" class="page-link">24</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/2018/page/25/" class="page-link">25</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/26/" class="page-link">26</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/27/" class="page-link">27</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/28/" class="page-link">28</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/29/" class="page-link">29</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/26/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2018/page/84/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
