<!DOCTYPE html>
<html><head>
  
  <title>Reports | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/root_development_is_maintained_by_specific_bacteria-bacteria_interactions_within_a_complex_microbiome_60956302/">
          <h4 class="font-weight-bold">Root development is maintained by specific bacteria-bacteria interactions within a complex microbiome, bioRxiv, 2019-05-24</h4>
          <p>AbstractPlants grow within a complex web of species interacting with each other and with the plant via a wide repertoire of chemical signals. To model plant-microbe-microbe-environment interactions, we inoculated seedlings with a defined 185-member bacterial synthetic community (SynCom), and manipulated the abiotic environment to enable classification of the SynCom to modules of co-occurring strains. We deconstructed the SynCom based on these modules, identifying a single bacterial genus, Variovorax, which reverts phenotypic effects on root development induced by a wide diversity of bacterial strains and by the entire 185-member community. Variovorax use mechanisms related to auxin and ethylene manipulation to balance this ecologically realistic root community’s effects on root development. We demonstrate metabolic signal interference within a complex model community, defining Variovorax as determinants of bacteria-plant communication networks.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/granar_a_new_computational_tool_to_better_understand_the_functional_importance_of_root_anatomy_60880272/">
          <h4 class="font-weight-bold">GRANAR, a new computational tool to better understand the functional importance of root anatomy, bioRxiv, 2019-05-23</h4>
          <p>AbstractRoot hydraulic conductivity is an important determinant of plant water uptake capacity. In particular, the root radial conductivity is often thought to be a limiting factor along the water pathways between the soil and the leaf. The root radial conductivity is itself defined by cell scale hydraulic properties and anatomical features. However, quantifying the influence of anatomical features on the radial conductivity remains challenging due to complex, and time-consuming, experimental procedures.We present a new computation tool, the Generator of Root ANAtomy in R (GRANAR) that can be used to rapidly generate digital versions of root anatomical networks. GRANAR uses a limited set of root anatomical parameters, easily acquired with existing image analysis tools. The generated anatomical network can then be used in combination with hydraulic models to estimate the corresponding hydraulic properties.We used GRANAR to re-analyse large maize (Zea mays) anatomical datasets from the literature. Our model was successful at creating virtual anatomies for each experimental observation. We also used GRANAR to generate anatomies not observed experimentally, over wider ranges of anatomical parameters. The generated anatomies were then used to estimate the corresponding radial conductivities with the hydraulic model MECHA. This enabled us to quantify the effect of individual anatomical features on the root radial conductivity. In particular, our simulations highlight the large importance of the width of the stele and the cortex.GRANAR is an open-source project available here &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpgranar.github.io&gt;httpgranar.github.io&lt;jatsext-link&gt;One-Sentence summaryGenerator of Root ANAtomy in R (GRANAR) is a new open-source computational tool that can be used to rapidly generate digital versions of root anatomical networks.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/allelic_polymorphism_shapes_collective_phenotypes_in_evolving_pseudomonas_aeruginosa_populations_60781879/">
          <h4 class="font-weight-bold">Allelic polymorphism shapes collective phenotypes in evolving Pseudomonas aeruginosa populations, bioRxiv, 2019-05-22</h4>
          <p>AbstractPseudomonas aeruginosa is an opportunistic pathogen that chronically infects cystic fibrosis (CF) lungs by forming antibiotic resistant biofilms. Emergence of phenotypically diverse isolates within P. aeruginosa infections has previously been reported, however, the impact of heterogeneity on social behaviors is poorly understood. Here we test how diversity impacts on traits by evolving the strain PAO1 in biofilms grown in a synthetic sputum medium for 50 days. We measured social trait production and used a metagenomic approach to analyze and assess genomic changes over the duration of the evolution experiment. We found that (i) evolutionary trajectories were reproducible in independently evolving populations; (ii) over 60% of genomic diversity occurred within the first 10 days of selection. We then focused on quorum sensing (QS), a well-studied P. aeruginosa trait that is commonly mutated in strains isolated from CF lungs. We found that at the population level (i) evolution in sputum medium selected for decreased production of QS and QS-dependent traits due to accumulation of a SNP in the lasR gene; (ii) there was a significant correlation between lasR mutant frequency, the loss of protease and the 3O-C12-HSL signal, and an increase in resistance to clinically relevant β-lactam antibiotics, despite no previous antibiotic exposure. Overall, our findings provide insights into the impact of allele polymorphism on the collective phenotypes of diverse P. aeruginosa populations. Further, we demonstrate that P. aeruginosa population and evolutionary dynamics can impact on traits important for virulence and could also provide reasons for antibiotic resistance found in CF infections.SignificancePseudomonas aeruginosa is a major cause of chronic infection of the lungs in individuals with cystic fibrosis (CF). Epidemic P. aeruginosa strains dominate and displace others in the lung and become phenotypically and genotypically diverse over time. How this diversity impacts on clinically relevant traits such as virulence or antibiotic resistance has received little attention. Here we demonstrate that the accumulation and frequency of genetic variants within heterogenous P. aeruginosa populations, results in population level changes in important traits including quorum sensing and antimicrobial resistance. Our work suggests that in the future, we should consider metagenomic and metaphenotypic assessments of P. aeruginosa populations collected from CF patients, rather than focusing on single random colonies isolated from infection.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/cellular_and_molecular_probing_of_intact_transparent_human_organs_60742793/">
          <h4 class="font-weight-bold">Cellular and Molecular Probing of Intact Transparent Human Organs, bioRxiv, 2019-05-21</h4>
          <p>SUMMARYOptical tissue transparency permits cellular and molecular investigation of complex tissues in 3D, a fundamental need in biomedical sciences. Adult human organs are particularly challenging for this approach, owing to the accumulation of dense and sturdy molecules in decades-aged human tissues. Here, we introduce SHANEL method utilizing a new tissue permeabilization approach to clear and label stiff human organs. We used SHANEL to generate the first intact transparent adult human brain and kidney, and perform 3D histology using antibodies and dyes in centimeters depth. Thereby, we revealed structural details of sclera, iris and suspensory ligament in the human eye, and the vessels and glomeruli in the human kidney. We also applied SHANEL on transgenic pig organs to map complex structures of EGFP expressing beta cells in &amp;gt;10 cm size pancreas. Overall, SHANEL is a robust and unbiased technology to chart the cellular and molecular architecture of intact large mammalian organs.Graphical Abstract&lt;jatsfig id=ufig1 position=float orientation=portrait fig-type=figure&gt;&lt;jatsgraphic xmlnsxlink=httpwww.w3.org1999xlink xlinkhref=643908v1_ufig1 position=float orientation=portrait &gt;Supplementary Movies of SHANEL are available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpdiscotechnologies.orgSHANEL&gt;httpdiscotechnologies.orgSHANEL&lt;jatsext-link&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/next-generation_sequencing_of_double_stranded_rna_is_greatly_improved_by_treatment_with_the_inexpensive_denaturing_reagent_dmso_60745250/">
          <h4 class="font-weight-bold">Next-generation sequencing of double stranded RNA is greatly improved by treatment with the inexpensive denaturing reagent DMSO, bioRxiv, 2019-05-21</h4>
          <p>AbstractDouble stranded RNA (dsRNA) is the genetic material of important viruses and a key component of RNA interference-based immunity in eukaryotes. Previous studies have noted difficulties in determining the sequence of dsRNA molecules that have affected studies of immune function and estimates of viral diversity in nature. Dimethyl sulfoxide (DMSO) has been used to denature dsRNA prior to the reverse transcription stage to improve RT-PCR and Sanger sequencing. We systematically tested the utility of DMSO to improve sequencing yield of a dsRNA virus (Φ6) in a short-read next generation sequencing platform. DMSO treatment improved sequencing read recovery by over two orders of magnitude, even when RNA and cDNA concentrations were below the limit of detection. We also tested the effects of DMSO on a mock eukaryotic viral community and found that dsRNA virus reads increased with DMSO treatment. Furthermore, we provide evidence that DMSO treatment does not adversely affect recovery of reads from a single-stranded RNA viral genome (Influenza ACalifornia072009). We suggest that up to 50% DMSO treatment be used prior to cDNA synthesis when samples of interest are composed of or may contain dsRNA.Data SummarySequence data was deposited in the NCBI Short Read Archive (accession numbers PRJNA527100, PRJNA527101, PRJNA527098). Data and code for analysis is available on GitHub (&lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsgithub.comawilcox83dsRNA-sequencing&gt;httpsgithub.comawilcox83dsRNA-sequencing&lt;jatsext-link&gt;. doi10.5281zenodo.1453423). Protocol for dsRNA sequencing is posted on protocols.io (doi10.17504protocols.io.ugnetve).</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/benchmarking_algorithms_for_gene_regulatory_network_inference_from_single-cell_transcriptomic_data_60720620/">
          <h4 class="font-weight-bold">Benchmarking algorithms for gene regulatory network inference from single-cell transcriptomic data, bioRxiv, 2019-05-20</h4>
          <p>AbstractWe present a comprehensive evaluation of state-of-the-art algorithms for inferring gene regulatory networks (GRNs) from single-cell gene expression data. We develop a systematic framework called BEELINE for this purpose. We use synthetic networks with predictable cellular trajectories as well as curated Boolean models to serve as the ground truth for evaluating the accuracy of GRN inference algorithms. We develop a strategy to simulate single-cell gene expression data from these two types of networks that avoids the pitfalls of previously-used methods. We selected 12 representative GRN inference algorithms. We found that the accuracy of these methods (measured in terms of AUROC and AUPRC) was moderate, by and large, although the methods were better in recovering interactions in the synthetic networks than the Boolean models. Techniques that did not require pseudotime-ordered cells were more accurate, in general. The observation that the endpoints of many false positive edges were connected by paths of length two in the Boolean models suggested that indirect effects may be predominant in the outputs of the algorithms we tested. The predicted networks were considerably inconsistent with each other, indicating that combining GRN inference algorithms using ensembles is likely to be challenging. Based on the results, we present some recommendations to users of GRN inference algorithms, including suggestions on how to create simulated gene expression datasets for testing them. BEELINE, which is available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpgithub.commurali-groupBEELINE&gt;httpgithub.commurali-groupBEELINE&lt;jatsext-link&gt; under an open-source license, will aid in the future development of GRN inference algorithms for single-cell transcriptomic data.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/reports/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/reports/page/15/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/12/" class="page-link">12</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/13/" class="page-link">13</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/14/" class="page-link">14</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/15/" class="page-link">15</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/reports/page/16/" class="page-link">16</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/17/" class="page-link">17</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/18/" class="page-link">18</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/19/" class="page-link">19</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/20/" class="page-link">20</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/reports/page/17/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/reports/page/172/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
