<!DOCTYPE html>
<html><head>
  
  <title>Microbiology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/assignment_of_virus_and_antimicrobial_resistance_genes_to_microbial_hosts_in_a_complex_microbial_community_by_combined_long-read_assembly_and_proximity_ligation_52411464/">
          <h4 class="font-weight-bold">Assignment of virus and antimicrobial resistance genes to microbial hosts in a complex microbial community by combined long-read assembly and proximity ligation, bioRxiv, 2018-12-09</h4>
          <p>AbstractThe characterization of microbial communities by metagenomic approaches has been enhanced by recent improvements in short-read sequencing efficiency and assembly algorithms. We describe the results of adding long-read sequencing to the mix of technologies used to assemble a highly complex cattle rumen microbial community, and compare the assembly to current short read-based methods applied to the same sample. Contigs in the long-read assembly were 7-fold longer on average, and contained 7-fold more complete open reading frames (ORF), than the short read assembly, despite having three-fold lower sequence depth. The linkages between long-read contigs, provided by proximity ligation data, supported identification of 188 novel viral-host associations in the rumen microbial community that suggest cross-species infectivity of specific viral strains. The improved contiguity of the long-read assembly also identified 94 antimicrobial resistance genes, compared to only seven alleles identified in the short-read assembly. Overall, we demonstrate a combination of experimental and computational methods that work synergistically to improve characterization of biological features in a highly complex rumen microbial community.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_genomic_and_proteomic_landscape_of_the_rumen_microbiome_revealed_by_comprehensive_genome-resolved_metagenomics_52390814/">
          <h4 class="font-weight-bold">The genomic and proteomic landscape of the rumen microbiome revealed by comprehensive genome-resolved metagenomics, bioRxiv, 2018-12-08</h4>
          <p>AbstractRuminants provide essential nutrition for billions of people worldwide. The rumen is a specialised stomach adapted to the breakdown of plant-derived complex polysaccharides, and collectively the rumen microbiota encode the thousands of enzymes responsible. Here we present a comprehensive analysis of over 6.5 terabytes of Illumina and Nanopore sequence data, including assembly of 4941 metagenome-assembled genomes, and several single-contig, whole-chromosome assemblies of novel rumen bacteria. We also present the largest dataset of predicted proteins from the rumen, and provide rich annotation against public datasets. Together these data will form an essential part of future studies of rumen microbiome structure and function.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/insights_from_bacillus_anthracis_strains_isolated_from_permafrost_in_the_tundra_zone_of_russia_52150760/">
          <h4 class="font-weight-bold">Insights from Bacillus anthracis strains isolated from permafrost in the tundra zone of Russia, bioRxiv, 2018-12-04</h4>
          <p>This article describes Bacillus anthracis strains isolated during an outbreak of anthrax on the Yamal Peninsula in the summer of 2016 and independently in Yakutia in 2015. A common feature of these strains is their conservation in permafrost, from which they were extracted either due to the thawing of permafrost (Yamal strains) or as the result of paleontological excavations (Yakut strains). All strains isolated on the Yamal share an identical genotype belonging to lineage B.Br.001002, pointing to a common source of infection in a territory over 250 km in length. In contrast, during the excavations in Yakutia, three genetically different strains were recovered from a single pit. One strain belongs to B.Br.001002, as the Yamal strains. Despite the remoteness of Yamal from Yakutia, whole genome sequence analysis showed that the B.Br.001002 strains are very closely related. The two other strains contribute to two different branches of A.Br.008011, one of the remarkable polytomies described so far in B. anthracis population. The geographic distribution of the strains belonging to this polytomy is suggesting that this polytomy emerged in the thirteenth century, in combination with the constitution of a unified Mongol empire extending from China to Eastern Europe. We propose an evolutionary model for B. anthracis recent evolution in which the B lineage spread throughout Eurasia and was subsequently replaced by the A lineage except in some geographically isolated areas.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/quantitative_pcr_provides_a_simple_and_accessible_method_for_quantitative_microbiome_profiling_51829165/">
          <h4 class="font-weight-bold">Quantitative PCR provides a simple and accessible method for quantitative microbiome profiling, bioRxiv, 2018-11-27</h4>
          <p>AbstractThe use of relative next generation sequencing (NGS) abundance data can lead to misinterpretations of microbial community structures as the increase of one taxon leads to concurrent decrease of the other(s). To overcome compositionality, we provide a quantitative NGS solution, which is achieved by adjusting the relative 16S rRNA gene amplicon NGS data with quantitative PCR (qPCR-based) total bacterial counts. By comparing the enumeration of dominant bacterial groups on different taxonomic levels in human fecal samples using taxon-specific 16S rRNA gene-targeted qPCR we show that quantitative NGS is able to estimate absolute bacterial abundances accurately. We also observed a higher degree of correspondence in the estimated microbe-metabolite relationship when quantitative NGS was applied. Being conceptually and methodologically analogous to amplicon-based NGS, our qPCR-based method can be readily incorporated into the standard, high-throughput NGS sample processing pipeline for more accurate description of interactions within and between the microbes and host.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/microbiota_profiling_with_long_amplicons_using_nanopore_sequencing_full-length_16s_rrna_gene_and_whole_rrn_operon_50175188/">
          <h4 class="font-weight-bold">Microbiota profiling with long amplicons using Nanopore sequencing full-length 16S rRNA gene and whole rrn operon, bioRxiv, 2018-10-24</h4>
          <p>Background Profiling microbiome on low biomass samples is challenging for metagenomics since these samples are prone to present DNA from other sources, such as the host or the environment. The usual approach is sequencing specific hypervariable regions of the 16S rRNA gene, which fails to assign taxonomy to genus and species level. Here, we aim to assess long-amplicon PCR-based approaches for assigning taxonomy at the genus and species level. We use Nanopore sequencing with two different markers full-length 16S rRNA (~1,500 bp) and the whole rrn operon (16S rRNA gene - ITS - 23S rRNA gene; 4,500 bp).Methods We sequenced a clinical isolate of Staphylococcus pseudintermedius, two mock communities (HM-783D, Bei Resources; D6306, ZymoBIOMICS) and two pools of low-biomass samples (dog skin). Nanopore sequencing was performed on MinION (Oxford Nanopore Technologies) using 1D PCR barcoding kit. Sequences were pre-processed, and data were analyzed using WIMP workflow on EPI2ME (ONT) or Minimap2 software with rrn database.Results Full-length 16S rRNA and the rrn operon retrieved the microbiota composition from the bacterial isolate, the mock communities and the complex skin samples, even at the genus and species level. For Staphylococcus pseudintermedius isolate, when using EPI2ME, the amplicons were assigned to the correct bacterial species in ~98% of the cases with rrn operon as the marker, and ~68% of the cases with 16S rRNA gene respectively. In both skin microbiota samples, we detected many species with an environmental origin. In chin, we found different Pseudomonas species in high abundance, whereas in the dorsal skin there were more taxa with lower abundances.Conclusions Both full-length 16S rRNA and the rrn operon retrieved the microbiota composition of simple and complex microbial communities, even from the low-biomass samples such as dog skin. For an increased resolution at the species level, rrn operon would be the best choice.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/origins_and_evolution_of_the_global_rna_virome_50175926/">
          <h4 class="font-weight-bold">Origins and Evolution of the Global RNA Virome, bioRxiv, 2018-10-24</h4>
          <p>AbstractViruses with RNA genomes dominate the eukaryotic virome, reaching enormous diversity in animals and plants. The recent advances of metaviromics prompted us to perform a detailed phylogenomic reconstruction of the evolution of the dramatically expanded global RNA virome. The only universal gene among RNA viruses is the RNA-dependent RNA polymerase (RdRp). We developed an iterative computational procedure that alternates the RdRp phylogenetic tree construction with refinement of the underlying multiple sequence alignments. The resulting tree encompasses 4,617 RNA virus RdRps and consists of 5 major branches, 2 of which include positive-sense RNA viruses, 1 is a mix of positive-sense (&#43;) RNA and double-stranded (ds) RNA viruses, and 2 consist of dsRNA and negative-sense (−) RNA viruses, respectively. This tree topology implies that dsRNA viruses evolved from &#43;RNA viruses on at least two independent occasions, whereas -RNA viruses evolved from dsRNA viruses. Reconstruction of RNA virus evolution using the RdRp tree as the scaffold suggests that the last common ancestors of the major branches of &#43;RNA viruses encoded only the RdRp and a single jelly-roll capsid protein. Subsequent evolution involved independent capture of additional genes, particularly, those encoding distinct RNA helicases, enabling replication of larger RNA genomes and facilitating virus genome expression and virus-host interactions. Phylogenomic analysis reveals extensive gene module exchange among diverse viruses and horizontal virus transfer between distantly related hosts. Although the network of evolutionary relationships within the RNA virome is bound to further expand, the present results call for a thorough reevaluation of the RNA virus taxonomy.IMPORTANCEThe majority of the diverse viruses infecting eukaryotes have RNA genomes, including numerous human, animal, and plant pathogens. Recent advances of metagenomics have led to the discovery of many new groups of RNA viruses in a wide range of hosts. These findings enable a far more complete reconstruction of the evolution of RNA viruses than what was attainable previously. This reconstruction reveals the relationships between different Baltimore Classes of viruses and indicates extensive transfer of viruses between distantly related hosts, such as plants and animals. These results call for a major revision of the existing taxonomy of RNA viruses.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/microbiology/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/microbiology/page/11/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../../tags/microbiology/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/microbiology/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/microbiology/page/10/" class="page-link">10</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/microbiology/page/11/" class="page-link">11</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/microbiology/page/12/" class="page-link">12</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/microbiology/page/13/" class="page-link">13</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/microbiology/page/14/" class="page-link">14</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/microbiology/page/15/" class="page-link">15</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/microbiology/page/16/" class="page-link">16</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        

        
        <li class="page-item"><a href="../../../../tags/microbiology/page/13/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/microbiology/page/17/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
