<!DOCTYPE html>
<html><head>
  
  <title>Genomics | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/mapping_histone_modifications_in_low_cell_number_and_single_cells_using_antibody-guided_chromatin_tagmentation_act-seq_56670663/">
          <h4 class="font-weight-bold">Mapping Histone Modifications in Low Cell Number and Single Cells Using Antibody-guided Chromatin Tagmentation (ACT-seq), bioRxiv, 2019-03-08</h4>
          <p>ABSTRACTModern next-generation sequencing-based methods have empowered researchers to assay the epigenetic states of individual cells. Existing techniques for profiling epigenetic marks in single cells often require the use and optimization of time-intensive procedures such as drop fluidics, chromatin fragmentation, and end repair. Here we describe ACT-seq, a novel and streamlined method for mapping genome-wide distributions of histone tail modifications, histone variants, and chromatin-binding proteins in a small number of or single cells. ACT-seq utilizes a fusion of Tn5 transposase to Protein A that is targeted to chromatin by a specific antibody, allowing chromatin fragmentation and sequence tag insertion specifically at genomic sites presenting the relevant antigen. The Tn5 transposase enables the use of an index multiplexing strategy (iACT-seq), which enables construction of thousands of single-cell libraries in one day by a single researcher without the need for drop-based fluidics or visual sorting. We conclude that ACT-seq present an attractive alternative to existing techniques for mapping epigenetic marks in single cells.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/cuttag_for_efficient_epigenomic_profiling_of_small_samples_and_single_cells_56616542/">
          <h4 class="font-weight-bold">CUT&amp;Tag for efficient epigenomic profiling of small samples and single cells, bioRxiv, 2019-03-07</h4>
          <p>AbstractMany chromatin features play critical roles in regulating gene expression. A complete understanding of gene regulation will require the mapping of specific chromatin features in small samples of cells at high resolution. Here we describe Cleavage Under Targets and Tagmentation (CUT&amp;amp;Tag), an enzyme-tethering strategy that provides efficient high-resolution sequencing libraries for profiling diverse chromatin components. In CUT&amp;amp;Tag, a chromatin protein is bound in situ by a specific antibody, which then tethers a protein A-Tn5 transposase fusion protein. Activation of the transposase efficiently generates fragment libraries with high resolution and exceptionally low background. All steps from live cells to sequencing-ready libraries can be performed in a single tube on the benchtop or a microwell in a high-throughput pipeline, and the entire procedure can be performed in one day. We demonstrate the utility of CUT&amp;amp;Tag by profiling histone modifications, RNA Polymerase II and transcription factors on low cell numbers and single cells.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/sequencing_of_53831_diverse_genomes_from_the_nhlbi_topmed_program_56616543/">
          <h4 class="font-weight-bold">Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program, bioRxiv, 2019-03-07</h4>
          <p>Summary paragraphThe Trans-Omics for Precision Medicine (TOPMed) program seeks to elucidate the genetic architecture and disease biology of heart, lung, blood, and sleep disorders, with the ultimate goal of improving diagnosis, treatment, and prevention. The initial phases of the program focus on whole genome sequencing of individuals with rich phenotypic data and diverse backgrounds. Here, we describe TOPMed goals and design as well as resources and early insights from the sequence data. The resources include a variant browser, a genotype imputation panel, and sharing of genomic and phenotypic data via dbGaP. In 53,581 TOPMed samples, &amp;gt;400 million single-nucleotide and insertiondeletion variants were detected by alignment with the reference genome. Additional novel variants are detectable through assembly of unmapped reads and customized analysis in highly variable loci. Among the &amp;gt;400 million variants detected, 97% have frequency &amp;lt;1% and 46% are singletons. These rare variants provide insights into mutational processes and recent human evolutionary history. The nearly complete catalog of genetic variation in TOPMed studies provides unique opportunities for exploring the contributions of rare and non-coding sequence variants to phenotypic variation. Furthermore, combining TOPMed haplotypes with modern imputation methods improves the power and extends the reach of nearly all genome-wide association studies to include variants down to ~0.01% in frequency.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/the_evolutionary_dynamics_and_fitness_landscape_of_clonal_haematopoiesis_56620613/">
          <h4 class="font-weight-bold">The evolutionary dynamics and fitness landscape of clonal haematopoiesis, bioRxiv, 2019-03-07</h4>
          <p>Somatic mutations acquired in healthy tissues as we age are major determinants of cancer risk. Whether variants confer a fitness advantage or rise to detectable frequencies by chance, however, remains largely unknown. Here, by combining blood sequencing data from ∼50,000 individuals, we reveal how mutation, genetic drift and fitness differences combine to shape the genetic diversity of healthy blood (‘clonal haematopoiesis’). By analysing the spectrum of variant allele frequencies we quantify fitness advantages for key pathogenic variants and genes and provide bounds on the number of haematopoietic stem cells. Positive selection, not drift, is the major force shaping clonal haematopoiesis. The remarkably wide variation in variant allele frequencies observed across individuals is driven by chance differences in the timing of mutation acquisition combined with differences in the cell-intrinsic fitness effect of variants. Contrary to the widely held view that clonal haematopoiesis is driven by ageing-related alterations in the stem cell niche, the data are consistent with the age dependence being driven simply by continuing risk of mutations and subsequent clonal expansions that lead to increased detectability at older ages.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/chromatin_arranges_in_filaments_of_blobs_with_nanoscale_functional_zonation_56456514/">
          <h4 class="font-weight-bold">Chromatin arranges in filaments of blobs with nanoscale functional zonation, bioRxiv, 2019-03-05</h4>
          <p>Three-dimensional (3D) chromatin organisation plays a key role in regulating genome function in higher eukaryotes. Despite recognition that the genome partitions into ~1Mb-sized topological associated domains (TADs) based on ensemble Hi-C measurements, many features of the physical organisation at the single cell level remain underexplored. Using 3D super-resolution microscopy, we reveal a sequential curvilinear arrangement of globular chromatin domains with viscoelastic properties (‘blobs’) juxtaposed to an RNA-populated interchromatin (IC) network. Quantitative mapping of genome function markers uncovers a zonal distribution, with RNA-binding factors concentrated in the IC, confinement of structural proteins and transcriptionally activepermissive marks to chromatin domain surfaces, and enrichment of repressive marks towards the interior. This correlation between nanoscale topology and genome function is relaxed in postreplicative chromatin, accentuated in replicative senescence, persists upon ATP depletion and hyperosmolarity induced chromatin condensation and, remarkably, after inactivation of cohesin. Our findings support a model of a higher-order chromatin architecture on the size level of TADs that creates and modulates distinct functional environments through a combination of biophysical parameters such as density and ATP-driven processes such as replication and transcription, but independent of cohesin.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/ctcf_confers_local_nucleosome_resiliency_after_dna_replication_and_during_mitosis_56227964/">
          <h4 class="font-weight-bold">CTCF confers local nucleosome resiliency after DNA replication and during mitosis, bioRxiv, 2019-03-01</h4>
          <p>The access of Transcription Factors (TFs) to their cognate DNA binding motifs requires a precise control over nucleosome positioning. This is especially important following DNA replication and during mitosis, both resulting in profound changes in nucleosome organization over TF binding regions. Using mouse Embryonic Stem (ES) cells, we show that the TF CTCF displaces nucleosomes from its binding site and locally organizes large and phased nucleosomal arrays, not only in interphase steady-state but also immediately after replication and during mitosis. While regions bound by other TFs, such as Oct4 and Sox2, display major rearrangement, the post-replication and mitotic nucleosome organization activity of CTCF is not likely to be unique Esrrb binding regions are also characterized by persistent nucleosome positioning. Therefore, we propose that selected TFs, such as CTCF and Esrrb, govern the inheritance of nucleosome positioning at gene regulatory regions throughout the ES cell-cycle.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/genomics/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/genomics/page/9/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/genomics/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/genomics/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/genomics/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/genomics/page/9/" class="page-link">9</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/genomics/page/10/" class="page-link">10</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/genomics/page/11/" class="page-link">11</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/genomics/page/12/" class="page-link">12</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/genomics/page/13/" class="page-link">13</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/genomics/page/14/" class="page-link">14</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/genomics/page/11/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/genomics/page/29/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
