<!DOCTYPE html>
<html><head>
  
  <title>100 200 Users | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/a_comprehensive_examination_of_nanopore_native_rna_sequencing_for_characterization_of_complex_transcriptomes_56864682/">
          <h4 class="font-weight-bold">A comprehensive examination of Nanopore native RNA sequencing for characterization of complex transcriptomes, bioRxiv, 2019-03-12</h4>
          <p>AbstractA platform for highly parallel direct sequencing of native RNA strands was recently described by Oxford Nanopore Technologies (ONT); in order to assess overall performance in transcript-level investigations, the technology was applied for sequencing sets of synthetic transcripts as well as a yeast transcriptome. However, despite initial efforts it remains crucial to further investigate characteristics of ONT native RNA sequencing when applied to much more complex transcriptomes. Here we thus undertook extensive native RNA sequencing of polyA&#43; RNA from two human cell lines, and thereby analysed ~5.2 million aligned native RNA reads which consisted of a total of ~4.6 billion bases. To enable informative comparisons, we also performed relevant ONT direct cDNA- and Illumina-sequencing. We find that while native RNA sequencing does enable some of the anticipated advantages, key unexpected aspects hamper its performance, most notably the quite frequent inability to obtain full-length transcripts from single reads, as well as difficulties to unambiguously infer their true transcript of origin. While characterising issues that need to be addressed when investigating more complex transcriptomes, our study highlights that with some defined improvements, native RNA sequencing could be an important addition to the mammalian transcriptomics toolbox.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/best_practices_for_making_reliable_inferences_from_citizen_science_data_case_study_using_ebird_to_estimate_species_distributions_56909054/">
          <h4 class="font-weight-bold">Best practices for making reliable inferences from citizen science data case study using eBird to estimate species distributions, bioRxiv, 2019-03-12</h4>
          <p>AbstractCitizen science data are valuable for addressing a wide range of ecological research questions, and there has been a rapid increase in the scope and volume of data available. However, data from large-scale citizen science projects typically present a number of challenges that can inhibit robust ecological inferences. These challenges include species bias, spatial bias, variation in effort, and variation in observer skill.To demonstrate key challenges in analysing citizen science data, we use the example of estimating species distributions with data from eBird, a large semi-structured citizen science project. We estimate three widely applied metrics for describing species distributions encounter rate, occupancy probability, and relative abundance. For each method, we outline approaches for data processing and modelling that are suitable for using citizen science data for estimating species distributions.Model performance improved when data processing and analytical methods addressed the challenges arising from citizen science data. The largest gains in model performance were achieved with two key processes 1) the use of complete checklists rather than presence-only data, and 2) the use of covariates describing variation in effort and detectability for each checklist. Including these covariates accounted for heterogeneity in detectability and reporting, and resulted in substantial differences in predicted distributions. The data processing and analytical steps we outlined led to improved model performance across a range of sample sizes.When using citizen science data it is imperative to carefully consider the appropriate data processing and analytical procedures required to address the bias and variation. Here, we describe the consequences and utility of applying our suggested approach to semi-structured citizen science data to estimate species distributions. The methods we have outlined are also likely to improve other forms of inference and will enable researchers to conduct robust analyses and harness the vast ecological knowledge that exists within citizen science data.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/dopamine_transients_delivered_in_learning_contexts_do_not_act_as_model-free_prediction_errors_56909657/">
          <h4 class="font-weight-bold">Dopamine transients delivered in learning contexts do not act as model-free prediction errors, bioRxiv, 2019-03-12</h4>
          <p>AbstractDopamine neurons fire transiently in response to unexpected rewards. These neural correlates are proposed to signal the reward prediction error described in model-free reinforcement learning algorithms. This error term represents the unpredicted or ‘excess’ value of the rewarding event. In model-free reinforcement learning, this value is then stored as part of the learned value of any antecedent cues, contexts or events, making them intrinsically valuable, independent of the specific rewarding event that caused the prediction error. In support of equivalence between dopamine transients and this model-free error term, proponents cite causal optogenetic studies showing that artificially induced dopamine transients cause lasting changes in behavior. Yet none of these studies directly demonstrate the presence of cached value under conditions appropriate for associative learning. To address this gap in our knowledge, we conducted three studies where we optogenetically activated dopamine neurons while rats were learning associative relationships, both with and without reward. In each experiment, the antecedent cues failed to acquired value and instead entered into value-independent associative relationships with the other cues or rewards. These results show that dopamine transients, constrained within appropriate learning situations, support valueless associative learning.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/the_life_history_of_human_foraging_cross-cultural_and_individual_variation_56908823/">
          <h4 class="font-weight-bold">The Life History Of Human Foraging Cross-Cultural And Individual Variation, bioRxiv, 2019-03-12</h4>
          <p>ABSTRACTHuman adaptation depends upon the integration of slow life history, complex production skills, and extensive sociality. Refining and testing models of the evolution of human life history and cultural learning will benefit from increasingly accurate measurement of knowledge, skills, and rates of production with age. We pursue this goal by inferring individual hunters’ of hunting skill gain and loss from approximately 23,000 hunting records generated by more than 1,800 individuals at 40 locations. The model provides an improved picture of ages of peak productivity as well as variation within and among ages. The data reveal an average age of peak productivity between 30 and 35 years of age, though high skill is maintained throughout much of adulthood. In addition, there is substantial variation both among individuals and sites. Within study sites, variation among individuals depends more upon heterogeneity in rates of decline than in rates of increase. This analysis sharpens questions about the co-evolution of human life history and cultural adaptation. It also demonstrates new statistical algorithms and models that expand the potential inferences drawn from detailed quantitative data collected in the field.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/animal-behavior-and-cognition">animal-behavior-and-cognition</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/convergent_gene_loss_in_aquatic_plants_predicts_new_components_of_plant_immunity_and_drought_response_56826634/">
          <h4 class="font-weight-bold">Convergent gene loss in aquatic plants predicts new components of plant immunity and drought response, bioRxiv, 2019-03-11</h4>
          <p>AbstractThe transition of plants from sea to land sparked an arms race with pathogens. The increased susceptibility of land plants is largely thought to be due to their dependence on micro-organisms for nutrients; the ensuing co-evolution has shaped the plant immune system. By profiling the immune receptors across flowering plants, we identified species with low numbers of NLR immune receptors. Interestingly, four of these species represent distinct lineages of monocots and dicots that returned to the aquatic lifestyle. Both aquatic monocot and dicot species lost the same well-known downstream immune signalling complex (EDS1-PAD4). This observation inspired us to look for other genes with a similar loss pattern and allowed us to predict putative new components of plant immunity. Gene expression analyses confirmed that a group of these genes was differentially expressed under pathogen infection. Excitingly, another subset of these genes was differentially expressed upon drought. Collectively, our study reveals the minimal plant immune system required for life under water, and highlights additional components required for the life of land plants.Author summaryPlant resistance to pathogens is commonly mediated by a complex gene family, known as NLRs. Upon pathogen infection, changes in the cellular environment trigger NLR activation and subsequent defence responses. Despite the dependence of agricultural practices on NLR genes to control pathogen load, relatively little is known about this gene family outside of model crop species. In this study, we identified a convergent reduction in the NLR gene family among two lineages of aquatic plants. Furthermore, we established that NLR reduction occurred in conjunction with the loss of a common immune signalling pathway. Subsequently, we identified other genes convergently lost in aquatic species and propose these as candidate components of the plant immune signalling pathway. In addition, we revealed components of the agronomically important drought response to be lost in aquatic plants. This study adds to our understanding of the complex interactions between environment and response to biotic stress, widely known as the disease triangle. The pathways identified in this study shed further light on the link between responses to drought and disease.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/high-speed_multifocal_plane_fluorescence_microscopy_for_three-dimensional_visualisation_of_beating_flagella_56825995/">
          <h4 class="font-weight-bold">High-speed multifocal plane fluorescence microscopy for three-dimensional visualisation of beating flagella, bioRxiv, 2019-03-11</h4>
          <p>AbstractAnalysis of flagellum beating in three dimensions is important for understanding how cells can undergo complex flagellum-driven motility and the ability to use fluorescence microscopy for such three-dimensional analysis would be extremely powerful. Trypanosoma and Leishmania are unicellular parasites which undergo complex cell movements in three dimensions as they swim and would particularly benefit from such an analysis. Here, high-speed multifocal plane fluorescence microscopy, a technique in which a light path multi-splitter is used to visualise 4 focal planes simultaneously, was used to reconstruct the flagellum beating of Trypanosoma brucei and Leishmania mexicana in three dimensions. It was possible to use either an organic fluorescent stain or a genetically-encoded fluorescence fusion protein to visualise flagellum and cell movement in three dimensions at a 200 Hz frame rate. This high-speed multifocal plane fluorescence microscopy approach was used to address two open questions regarding Trypanosoma and Leishmania swimming To quantify the planarity of the L. mexicana flagellum beat and analyse the nature of flagellum beating during T. brucei ‘tumbling’.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/100-200-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/100-200-users/page/20/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/100-200-users/page/17/" class="page-link">17</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/100-200-users/page/18/" class="page-link">18</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/100-200-users/page/19/" class="page-link">19</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/100-200-users/page/20/" class="page-link">20</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/100-200-users/page/21/" class="page-link">21</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/100-200-users/page/22/" class="page-link">22</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/100-200-users/page/23/" class="page-link">23</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/100-200-users/page/24/" class="page-link">24</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/100-200-users/page/25/" class="page-link">25</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/100-200-users/page/22/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/100-200-users/page/68/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
