<!DOCTYPE html>
<html><head>
  
  <title>Biophysics | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/sub-2_%C3%A5_ewald_curvature_corrected_single-particle_cryo-em_38641204/">
          <h4 class="font-weight-bold">Sub-2 Å Ewald Curvature Corrected Single-Particle Cryo-EM, bioRxiv, 2018-04-21</h4>
          <p>AbstractSingle-particle cryogenic electron microscopy (cryo-EM) provides a powerful methodology for structural biologists, but the resolutions typically attained with experimentally determined structures have lagged behind microscope capabilities. Here, we have exploited several technical solutions to improve resolution, including sub-Angstrom pixelation, per-particle CTF refinement, and most notably a correction for Ewald sphere curvature. The application of these methods on micrographs recorded on a base model Titan Krios enabled structure determination at ∼1.86-Å resolution of an adeno-associated virus serotype 2 variant (AAV2), an important gene-delivery vehicle.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/heterochromatin_drives_organization_of_conventional_and_inverted_nuclei_31471754/">
          <h4 class="font-weight-bold">Heterochromatin drives organization of conventional and inverted nuclei, bioRxiv, 2018-01-10</h4>
          <p>AbstractThe mammalian cell nucleus displays a remarkable spatial segregation of active euchromatic from inactive heterochromatic genomic regions. In conventional nuclei, euchromatin is localized in the nuclear interior and heterochromatin at the nuclear periphery. In contrast, rod photoreceptors in nocturnal mammals have inverted nuclei, with a dense heterochromatic core and a thin euchromatic outer shell. This inverted architecture likely converts rod nuclei into microlenses to facilitate nocturnal vision, and may relate to the absence of particular proteins that tether heterochromatin to the lamina. However, both the mechanism of inversion and the role of interactions between different types of chromatin and the lamina in nuclear organization remain unknown. To elucidate this mechanism we performed Hi-C and microscopy on cells with inverted nuclei and their conventional counterparts. Strikingly, despite the inversion evident in microscopy, both types of nuclei display similar Hi-C maps. To resolve this paradox we developed a polymer model of chromosomes and found a universal mechanism that reconciles Hi-C and microscopy for both inverted and conventional nuclei. Based solely on attraction between heterochromatic regions, this mechanism is sufficient to drive phase separation of euchromatin and heterochromatin and faithfully reproduces the 3D organization of inverted nuclei. When interactions between heterochromatin and the lamina are added, the same model recreates the conventional nuclear organization. To further test our models, we eliminated lamina interactions in models of conventional nuclei and found that this triggers a spontaneous process of inversion that qualitatively reproduces the pathway of morphological changes during nuclear inversion in vivo. Together, our experiments and modeling suggest that interactions among heterochromatic regions are central to phase separation of the active and inactive genome in inverted and conventional nuclei, while interactions with the lamina are essential for building the conventional architecture from these segregated phases. Ultimately our data suggest that an inverted organization constitutes the default state of nuclear architecture.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/visualization_of_prc2-dinucleosome_interactions_leading_to_epigenetic_repression_31435909/">
          <h4 class="font-weight-bold">Visualization of PRC2-Dinucleosome Interactions Leading to Epigenetic Repression, bioRxiv, 2018-01-09</h4>
          <p>AbstractEpigenetic regulation is mediated by protein complexes that couple recognition of chromatin marks to activity or recruitment of chromatin-modifying enzymes. Polycomb repressive complex 2 (PRC2), a gene silencer that methylates lysine 27 of histone H3, is stimulated upon recognition of its own catalytic product, and has been shown to be more active on dinucleosomes than H3 tails or single nucleosomes. These properties likely facilitate local H3K27me23 spreading causing heterochromatin formation and gene repression. Here, cryo-EM reconstructions of human PRC2 bound to dinucleosomes show how a single PRC2, interacting with nucleosomal DNA, precisely positions the H3 tails to recognize a H3K27me3 mark in one nucleosome and is stimulated to modify a neighboring nucleosome. The geometry of the PRC2-DNA interactions allow PRC2 to tolerate different dinucleosome geometries due to varying lengths of the linker DNA. Our structures are the first to illustrate how an epigenetic regulator engages with a complex chromatin substrate.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/high_accuracy_measurements_of_nanometer-scale_distances_between_fluorophores_at_the_single-molecule_level_30486039/">
          <h4 class="font-weight-bold">High accuracy measurements of nanometer-scale distances between fluorophores at the single-molecule level, bioRxiv, 2017-12-16</h4>
          <p>To uncover the mechanisms of molecular machines it is useful to probe their structural conformations. Single-molecule Förster resonance energy transfer (smFRET) is a powerful tool for measuring intra-molecular shape changes of single-molecules, but is confined to distances of 2-8 nm. Current super-resolution measurements are error prone at &amp;lt;25 nm. Thus, reliable high-throughput distance information between 8-25 nm is currently difficult to achieve. Here, we describe methods that utilize information about localization and imaging errors to measure distances between two different color fluorophores with ∼1 nm accuracy at any distance &amp;gt;2 nm, using a standard TIRF microscope and open-source software. We applied our two-color localization method to uncover a ∼4 nm conformational change in the “stalk” of the motor protein dynein, revealing unexpected flexibility in this antiparallel coiled-coil domain. These new methods enable high-accuracy distance measurements of single-molecules that can be used over a wide range of length scales.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/biomolecular_simulations_under_realistic_macroscopic_salt_conditions_29643349/">
          <h4 class="font-weight-bold">Biomolecular simulations under realistic macroscopic salt conditions, bioRxiv, 2017-11-30</h4>
          <p>Biomolecular simulations are typically performed in an aqueous environment where the number of ions remains fixed for the duration of the simulation, generally with either a minimally neutralizing ion environment or a number of salt pairs intended to match the macroscopic salt concentration. In contrast, real biomolecules experience local ion environments where the salt concentration is dynamic and may differ from bulk. The degree of salt concentration variability and average deviation from the macroscopic concentration remains, as yet, unknown. Here, we describe the theory and implementation of a Monte Carlo osmostat that can be added to explicit solvent molecular dynamics or Monte Carlo simulations to sample from a semigrand canonical ensemble in which the number of salt pairs fluctuates dynamically during the simulation. The osmostat reproduce the correct equilibrium statistics for a simulation volume that can exchange ions with a large reservoir at a defined macroscopic salt concentration. To achieve useful Monte Carlo acceptance rates, the method makes use of nonequilibrium candidate Monte Carlo (NCMC) moves in which monovalent ions and water molecules are alchemically transmuted using short nonequilibrium trajectories, with a modified Metropolis-Hastings criterion ensuring correct equilibrium statistics for an (Δ𝜇, 𝑁, 𝑝, 𝑇) ensemble. We demonstrate how typical protein (DHFR and the tyrosine kinase Src) and nucleic acid (Drew-Dickerson B-DNA dodecamer) systems exhibit salt concentration distributions that significantly differ from fixed-salt bulk simulations and display fluctuations that are on the same order of magnitude as the average.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/nonequilibrium_entropic_bounds_for_darwinian_replicators_29448506/">
          <h4 class="font-weight-bold">Nonequilibrium entropic bounds for Darwinian replicators, bioRxiv, 2017-11-26</h4>
          <p>Life evolved on our planet by means of a combination of Darwinian selection and innovations leading to higher levels of complexity. The emergence and selection of replicating entities is a central problem in prebiotic evolution. Theoretical models have shown how populations of different types of replicating entities exclude or coexist with other classes of replicators. Models are typically kinetic, based on standard replicator equations. On the other hand, the presence of thermodynamical constrains for these systems remain an open question. This is largely due to the lack of a general theory of out of statistical methods for systems far from equilibrium. Nonetheless, a first approach to this problem has been put forward in a series of novel developements in non-equilibrium physics, under the rubric of the extended second law of thermodynamics. The work presented here is twofold firstly, we review this theoretical framework and provide a brief description of the three fundamental replicator types in prebiotic evolution parabolic, malthusian and hyperbolic. Finally, we employ these previously mentioned techinques to explore how replicators are constrained by thermodynamics.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/biophysics/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/biophysics/page/6/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biophysics/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biophysics/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biophysics/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biophysics/page/6/" class="page-link">6</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/biophysics/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biophysics/page/8/" class="page-link">8</a></li>
            
        

        
        <li class="page-item"><a href="/audiences/tags/biophysics/page/8/" rel="next" class="page-link">Next ›</a></li>
        

        
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
