<!DOCTYPE html>
<html><head>
  
  <title>200 500 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/highly_multiplexed_in_situ_protein_imaging_with_signal_amplification_by_immuno-saber_53240706/">
          <h4 class="font-weight-bold">Highly multiplexed in situ protein imaging with signal amplification by Immuno-SABER, bioRxiv, 2018-12-29</h4>
          <p>AbstractProbing the molecular organization of tissues requires in situ analysis by microscopy. However current limitations in multiplexing, sensitivity, and throughput collectively constitute a major barrier for comprehensive single-cell profiling of proteins. Here, we report Immunostaining with Signal Amplification By Exchange Reaction (Immuno-SABER), a rapid, highly multiplexed signal amplification method that simultaneously tackles these key challenges. Immuno-SABER utilizes DNA-barcoded antibodies and provides a method for highly multiplexed signal amplification via modular orthogonal DNA concatemers generated by Primer Exchange Reaction. This approach offers the capability to preprogram and control the amplification level independently for multiple targets without in situ enzymatic reactions, and the intrinsic scalability to rapidly amplify and image a large number of protein targets. We validated our approach in diverse sample types including cultured cells, cryosections, FFPE sections, and whole mount tissues. We demonstrated independently tunable 5-180-fold amplification for multiple targets, covering the full signal range conventionally achieved by secondary antibodies to tyramide signal amplification, as well as simultaneous signal amplification for 10 different proteins using standard equipment and workflow. We further combined Immuno-SABER with Expansion Microscopy to enable rapid and highly multiplexed super-resolution tissue imaging. Overall, Immuno-SABER presents an effective and accessible platform for rapid, multiplexed imaging of proteins across scales with high sensitivity.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/human_microbiome_aging_clocks_based_on_deep_learning_and_tandem_of_permutation_feature_importance_and_accumulated_local_effects_53245742/">
          <h4 class="font-weight-bold">Human microbiome aging clocks based on deep learning and tandem of permutation feature importance and accumulated local effects, bioRxiv, 2018-12-29</h4>
          <p>The human gut microbiome is a complex ecosystem that both affects and is affected by its host status. Previous analyses of gut microflora revealed associations between specific microbes and host health and disease status, genotype and diet. Here, we developed a method of predicting the biological age of the host based on the microbiological profiles of gut microbiota using a curated dataset of 1,165 healthy individuals (1,663 microbiome samples). Our predictive model, a human microbiome clock, has an architecture of a deep neural network and achieves the accuracy of 3.94 years mean absolute error in cross-validation. The performance of the deep microbiome clock was also evaluated on several additional populations. We further introduce a platform for biological interpretation of individual microbial features used in age models, which relies on permutation feature importance and accumulated local effects. This approach has allowed us to define two lists of 95 intestinal biomarkers of human aging. We further show that this list can be reduced to 39 taxa that convey the most information on their host&#39;s aging. Overall, we show that (a) microbiological profiles can be used to predict human age; and (b) microbial features selected by models are age-related.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/prioritisation_of_oncology_therapeutic_targets_using_crispr-cas9_screening_52947148/">
          <h4 class="font-weight-bold">Prioritisation of oncology therapeutic targets using CRISPR-Cas9 screening, bioRxiv, 2018-12-20</h4>
          <p>SummaryFunctional genomics approaches can overcome current limitations that hamper oncology drug development such as lack of robust target identification and clinical efficacy. Here we performed genome-scale CRISPR-Cas9 screens in 204 human cancer cell lines from 12 cancer-types and developed a data-driven framework to prioritise cancer therapeutic candidates. We integrated gene cell fitness effects with genomic biomarkers and target tractability for drug development to systematically prioritise new oncology targets in defined tissues and genotypes. Furthermore, we took one of our most promising dependencies, Werner syndrome RecQ helicase, and verified it as a candidate target for tumours with microsatellite instability. Our analysis provides a comprehensive resource of cancer dependencies, a framework to prioritise oncology targets, and nominates specific new candidates. The principles described in this study can transform the initial stages of the drug development process contributing to a new, diverse and more effective portfolio of oncology targets.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cancer-biology">cancer-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/an_rna-binding_region_regulates_ctcf_clustering_and_chromatin_looping_52608001/">
          <h4 class="font-weight-bold">An RNA-binding region regulates CTCF clustering and chromatin looping, bioRxiv, 2018-12-13</h4>
          <p>Mammalian genomes are folded into Topologically Associating Domains (TADs), consisting of cell-type specific chromatin loops anchored by CTCF and cohesin. Since CTCF and cohesin are expressed ubiquitously, how cell-type specific CTCF-mediated loops are formed poses a paradox. Here we show RNase-sensitive CTCF self-association in vitro and that an RNA-binding region (RBR) mediates CTCF clustering in vivo. Intriguingly, deleting the RBR abolishes or impairs almost half of all chromatin loops in mouse embryonic stem cells. Disrupted loop formation correlates with abrogated clustering and diminished chromatin binding of the RBR mutant CTCF protein, which in turn results in a failure to halt cohesin-mediated extrusion. Thus, CTCF loops fall into at least 2 classes RBR-independent and RBR-dependent loops. We suggest that evidence for distinct classes of RBR-dependent loops may provide a mechanism for establishing cell-specific CTCF loops regulated by RNAs and other RBR partner.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/fast_and_accurate_large_multiple_sequence_alignments_using_root-to-leave_regressive_computation_52359507/">
          <h4 class="font-weight-bold">Fast and accurate large multiple sequence alignments using root-to-leave regressive computation, bioRxiv, 2018-12-08</h4>
          <p>AbstractInferences derived from large multiple alignments of biological sequences are critical to many areas of biology, including evolution, genomics, biochemistry, and structural biology. However, the complexity of the alignment problem imposes the use of approximate solutions. The most common is the progressive algorithm, which starts by aligning the most similar sequences, incorporating the remaining ones following the order imposed by a guide-tree. We developed and validated on protein sequences a regressive algorithm that works the other way around, aligning first the most dissimilar sequences. Our algorithm produces more accurate alignments than non-regressive methods, especially on datasets larger than 10,000 sequences. By design, it can run any existing alignment method in linear time thus allowing the scale-up required for extremely large genomic analyses.One Sentence SummaryInitiating alignments with the most dissimilar sequences allows slow and accurate methods to be used on large datasets</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/large-scale_exome_sequencing_study_implicates_both_developmental_and_functional_changes_in_the_neurobiology_of_autism_52024757/">
          <h4 class="font-weight-bold">Large-scale exome sequencing study implicates both developmental and functional changes in the neurobiology of autism, bioRxiv, 2018-12-01</h4>
          <p>SummaryWe present the largest exome sequencing study of autism spectrum disorder (ASD) to date (n=35,584 total samples, 11,986 with ASD). Using an enhanced Bayesian framework to integrate de novo and case-control rare variation, we identify 102 risk genes at a false discovery rate ≤ 0.1. Of these genes, 49 show higher frequencies of disruptive de novo variants in individuals ascertained for severe neurodevelopmental delay, while 53 show higher frequencies in individuals ascertained for ASD; comparing ASD cases with mutations in these groups reveals phenotypic differences. Expressed early in brain development, most of the risk genes have roles in regulation of gene expression or neuronal communication (i.e., mutations effect neurodevelopmental and neurophysiological changes), and 13 fall within loci recurrently hit by copy number variants. In human cortex single-cell gene expression data, expression of risk genes is enriched in both excitatory and inhibitory neuronal lineages, consistent with multiple paths to an excitatoryinhibitory imbalance underlying ASD.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/200-500-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/200-500-users/page/25/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/200-500-users/page/22/" class="page-link">22</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/200-500-users/page/23/" class="page-link">23</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/200-500-users/page/24/" class="page-link">24</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/200-500-users/page/25/" class="page-link">25</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/200-500-users/page/26/" class="page-link">26</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/200-500-users/page/27/" class="page-link">27</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/200-500-users/page/28/" class="page-link">28</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/200-500-users/page/29/" class="page-link">29</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/200-500-users/page/30/" class="page-link">30</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/200-500-users/page/27/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/200-500-users/page/56/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
