<!DOCTYPE html>
<html><head>
  
  <title>Reports | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/compositional_data_analysis_is_necessary_for_simulating_and_analyzing_rna-seq_data_56315555/">
          <h4 class="font-weight-bold">Compositional Data Analysis is necessary for simulating and analyzing RNA-Seq data, bioRxiv, 2019-03-02</h4>
          <p>Seq techniques (e.g. RNA-Seq) generate compositional datasets, i.e. the number of fragments sequenced is not proportional to the total RNA present. Thus, datasets carry only relative information, even though absolute RNA copy numbers are often of interest. Current normalization methods assume most features are not changing, which can lead to misleading conclusions when there are large shifts. However, there are few real datasets and no simulation protocols currently available that can directly benchmark methods when such large shifts occur.We present absSimSeq, an R package that simulates compositional data in the form of RNA-Seq reads. We tested several tools used for RNA-Seq differential analysis sleuth, DESeq2, edgeR, limma, sleuth and ALDEx2 (which explicitly takes a compositional approach). For these tools, we compared their standard normalization to either “compositional normalization”, which uses log-ratios to anchor the data on a set of negative control features, or RUVSeq, another tool that directly uses negative control features.We show that common normalizations result in reduced performance with current methods when there is a large change in the total RNA per cell. Performance improves when spike-ins are included and used by a compositional approach, even if the spike-ins have substantial variation. In contrast, RUVSeq, which normalizes count data rather than compositional data, has poor performance. Further, we show that previous criticisms of spike-ins did not take into account the compositional nature of the data. We conclude that absSimSeq can generate more representative datasets for testing performance, and that spike-ins should be more broadly used in a compositional manner to minimize misleading conclusions from differential analyses.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genetic_inhibition_of_pcsk9_atherogenic_lipoprotein_concentrations_and_calcific_aortic_valve_stenosis_56256162/">
          <h4 class="font-weight-bold">Genetic inhibition of PCSK9, atherogenic lipoprotein concentrations, and calcific aortic valve stenosis, bioRxiv, 2019-03-02</h4>
          <p>Background Proprotein convertase subtilisinkexin type 9 (PCSK9) inhibition reduces plasma concentrations of low-density lipoprotein cholesterol (LDL-C), apolipoprotein B (apoB) and lipoprotein(a) [Lp(a)]. Atherogenic lipoprotein levels have been linked with calcific aortic valve stenosis (CAVS). Our objectives were to determine the association between variants in PCSK9 and lipoprotein-lipid levels, coronary artery disease (CAD) and CAVS, and to evaluate if PCSK9 could be implicated in aortic valve interstitial cells (VICs) calcification.Methods We built a genetic risk score weight for LDL-C levels (wGRS) using 10 independent PCSK9 single nucleotide polymorphisms and determined its association with lipoprotein-lipid levels in 9692 participants of the EPIC-Norfolk study. We investigated the association between the wGRS and CAD and CAVS in the UK Biobank, as well as the association between the PCSK9 R46L variant and CAVS in a meta-analysis of published prospective, population-based studies (Copenhagen studies, 1463 cases101,620 controls) and unpublished studies (UK Biobank, 1350 cases349,043 controls, Malmo Diet and Cancer study, 682 cases5963 controls and EPIC-Norfolk study, 508 cases20,421 controls). We evaluated PCSK9 expression and localization in explanted aortic valves by capillary Western blot and immunohistochemistry in patients with and without CAVS. Von Kossa staining was used to visualize aortic leaflet calcium deposits. PCSK9 expression under oxidative stress conditions in VICs was assessed.Results The wGRS was significantly associated with lower LDL-C and apoB (p&amp;lt;0.001), but not with Lp(a). In the UK Biobank, the association of PCSK9 variants with CAD were positively correlated with their effects on apoB levels. CAVS was less prevalent in carriers of the PCSK9 R46L variant [odds ratio=0.71 (95% confidence interval, 0.57-0.88), p&amp;lt;0.001]. PCSK9 expression was elevated in the aortic valves of patients with aortic sclerosis and CAVS compared to controls. In calcified leaflets, PCSK9 co-localized with calcium deposits. PCSK9 expression was induced by oxidative stress in VICs. Conclusion Genetic inhibition of PCSK9 is associated with lifelong reductions in the levels of non-Lp(a) apoB-containing lipoproteins as well as lower odds of CAD and CAVS. PCSK9 is abundant in fibrotic and calcified aortic leaflets. Oxidative stress increases PCSK9 expression in VICs. These results provide a rationale for performing randomized clinical trials of PCSK9 inhibition in CAVS.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/mash_screen_high-throughput_sequence_containment_estimation_for_genome_discovery_56276933/">
          <h4 class="font-weight-bold">Mash Screen High-throughput sequence containment estimation for genome discovery, bioRxiv, 2019-03-02</h4>
          <p>The MinHash algorithm has proven effective for rapidly estimating the resemblance of two genomes or metagenomes. However, this method cannot reliably estimate the containment of a genome within a metagenome. Here we describe an online algorithm capable of measuring the containment of genomes and proteomes within either assembled or unassembled sequencing read sets. We describe several use cases, including contamination screening and retrospective analysis of metagenomes for novel genome discovery. Using this tool, we provide containment estimates for every NCBI RefSeq genome within every SRA metagenome, and demonstrate the identification of a novel polyomavirus species from a public metagenome.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_mechanism_to_minimize_errors_during_non-homologous_end_joining_56200760/">
          <h4 class="font-weight-bold">A mechanism to minimize errors during non-homologous end joining, bioRxiv, 2019-03-01</h4>
          <p>SUMMARYEnzymatic processing of DNA underlies all DNA repair, yet inappropriate DNA processing must be avoided. In vertebrates, double-strand breaks are repaired predominantly by non-homologous end-joining (NHEJ), which directly ligates DNA ends. NHEJ has the potential to be highly mutagenic because it employs DNA polymerases, nucleases, and other enzymes that modify incompatible DNA ends to allow their ligation. Using a biochemical system that recapitulates key features of cellular NHEJ, we show that end-processing requires formation of a “short-range synaptic complex” in which DNA ends are closely aligned in a ligation-competent state. Furthermore, single-molecule imaging directly demonstrates that processing occurs within the short-range complex. This confinement of end processing to a ligation-competent complex ensures that DNA ends undergo ligation as soon as they become compatible, thereby minimizing mutagenesis. Our results illustrate how the coordination of enzymatic catalysis with higher-order structural organization of substrate maximizes the fidelity of DNA repair.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_transcriptome-wide_mendelian_randomization_study_to_uncover_tissue-dependent_regulatory_mechanisms_across_the_human_phenome_56204012/">
          <h4 class="font-weight-bold">A transcriptome-wide Mendelian randomization study to uncover tissue-dependent regulatory mechanisms across the human phenome, bioRxiv, 2019-03-01</h4>
          <p>Background Developing insight into tissue-specific transcriptional mechanisms can help improve our understanding of how genetic variants exert their effects on complex traits and disease. By applying the principles of Mendelian randomization, we have undertaken a systematic analysis to evaluate transcriptome-wide associations between gene expression across 48 different tissue types and 395 complex traits. Results Overall, we identified 100,025 gene-trait associations based on conventional genome-wide corrections (P &amp;lt; 5 x 10-08) that also provided evidence of genetic colocalization. These results indicated that genetic variants which influence gene expression levels in multiple tissues are more likely to influence multiple complex traits. We identified many examples of tissue-specific effects, such as genetically-predicted TPO, NR3C2 and SPATA13 expression only associating with thyroid disease in thyroid tissue. Additionally, FBN2 expression was associated with both cardiovascular and lung function traits, but only when analysed in heart and lung tissue respectively. We also demonstrate that conducting phenome-wide evaluations of our results can help flag adverse on-target side effects for therapeutic intervention, as well as propose drug repositioning opportunities. Moreover, we find that exploring the tissue-dependency of associations identified by genome-wide association studies (GWAS) can help elucidate the causal genes and tissues responsible for effects, as well as uncover putative novel associations. Conclusions The atlas of tissue-dependent associations we have constructed should prove extremely valuable to future studies investigating the genetic determinants of complex disease. The follow-up analyses we have performed in this study are merely a guide for future research. Conducting similar evaluations can be undertaken systematically at httpmrcieu.mrsoftware.orgTissue_MR_atlas.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_unicellular_relative_of_animals_generates_an_epithelium-like_cell_layer_by_actomyosin-dependent_cellularization_56227250/">
          <h4 class="font-weight-bold">A unicellular relative of animals generates an epithelium-like cell layer by actomyosin-dependent cellularization, bioRxiv, 2019-03-01</h4>
          <p>In animals, cellularization of a coenocyte is a specialized form of cytokinesis that results in the formation of a polarized epithelium during early embryonic development. It is characterized by coordinated assembly of an actomyosin network, which drives inward membrane invaginations. However, whether coordinated cellularization driven by membrane invagination exists outside animals is not known. To that end, we investigate cellularization in the ichthyosporean Sphaeroforma arctica, a close unicellular relative of animals. We show that the process of cellularization involves coordinated inward plasma membrane invaginations dependent on an actomyosin network, and reveal the temporal order of its assembly. This leads to the formation of a polarized layer of cells resembling an epithelium. We show that this epithelium-like stage is associated with tightly regulated transcriptional activation of genes involved in cell adhesion. Hereby we demonstrate the presence of a self-organized, clonally-generated, polarized layer of cells in a unicellular relative of animals.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/reports/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/reports/page/55/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/52/" class="page-link">52</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/53/" class="page-link">53</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/54/" class="page-link">54</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/55/" class="page-link">55</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/reports/page/56/" class="page-link">56</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/57/" class="page-link">57</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/58/" class="page-link">58</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/59/" class="page-link">59</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/60/" class="page-link">60</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/reports/page/57/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/reports/page/172/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
