<!DOCTYPE html>
<html><head>
  
  <title>500&#43; Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/high_precision_coding_in_visual_cortex_62520267/">
          <h4 class="font-weight-bold">High precision coding in visual cortex, bioRxiv, 2019-06-22</h4>
          <p>Single neurons in visual cortex provide unreliable measurements of visual features due to their high trial-to-trial variability. It is not known if this “noise” extends its effects over large neural populations to impair the global encoding of stimuli. We recorded simultaneously from ∼20,000 neurons in mouse primary visual cortex (V1) and found that the neural populations had discrimination thresholds of ∼0.34° in an orientation decoding task. These thresholds were nearly 100 times smaller than those reported behaviorally in mice. The discrepancy between neural and behavioral discrimination could not be explained by the types of stimuli we used, by behavioral states or by the sequential nature of perceptual learning tasks. Furthermore, higher-order visual areas lateral to V1 could be decoded equally well. These results imply that the limits of sensory perception in mice are not set by neural noise in sensory cortex, but by the limitations of downstream decoders.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/aequorea_victorias_secrets_62432700/">
          <h4 class="font-weight-bold">Aequorea victoria’s secrets, bioRxiv, 2019-06-20</h4>
          <p>Using mRNA-Seq and de novo transcriptome assembly, we identified, cloned and characterized nine previously undiscovered fluorescent protein (FP) homologs from Aequorea victoria and a related Aequorea species, with most sequences highly divergent from avGFP. Among these FPs are the brightest GFP homolog yet characterized and a reversibly photochromic FP that responds to UV and blue light. Beyond green emitters, Aequorea species express purple- and blue-pigmented chromoproteins (CPs) with absorbances ranging from green to far-red, including two that are photoconvertible. X-ray crystallography revealed that Aequorea CPs contain a chemically novel chromophore with an unexpected crosslink to the main polypeptide chain. Because of the unique attributes of several of these newly discovered FPs, we expect that Aequorea will, once again, give rise to an entirely new generation of useful probes for bioimaging and biosensing.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biochemistry">biochemistry</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genetic_analyses_of_diverse_populations_improves_discovery_for_complex_traits_62414002/">
          <h4 class="font-weight-bold">Genetic analyses of diverse populations improves discovery for complex traits, Nature, 2019-06-19</h4>
          <p>13</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/nature">nature</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/chromatinization_of_escherichia_coli_with_archaeal_histones_61507230/">
          <h4 class="font-weight-bold">Chromatinization of Escherichia coli with archaeal histones, bioRxiv, 2019-06-04</h4>
          <p>ABSTRACTNucleosomes restrict DNA accessibility throughout eukaryotic genomes, with repercussions for replication, transcription, and other DNA-templated processes. How this globally restrictive organization emerged from a presumably more open ancestral state remains poorly understood. Here, to better understand the challenges associated with establishing globally restrictive chromatin, we express histones in a naïve bacterial system that has not evolved to deal with nucleosomal structures Escherichia coli. We find that histone proteins from the archaeon Methanothermus fervidus assemble on the E. coli chromosome in vivo and protect DNA from micrococcal nuclease digestion, allowing us to map binding footprints genome-wide. We provide evidence that nucleosome occupancy along the E. coli genome tracks intrinsic sequence preferences but is disturbed by ongoing transcription and replication. Notably, we show that higher nucleosome occupancy at promoters and across gene bodies is associated with lower transcript levels, consistent with local repressive effects. Surprisingly, however, this sudden enforced chromatinization has only mild repercussions for growth, suggesting that histones can become established as ubiquitous chromatin proteins without interfering critically with key DNA-templated processes. Our results have implications for the evolvability of transcriptional ground states and highlight chromatinization by archaeal histones as a potential avenue for controlling genome accessibility in synthetic prokaryotic systems.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/7_tesla_mri_of_the_ex_vivo_human_brain_at_100_micron_resolution_61336697/">
          <h4 class="font-weight-bold">7 Tesla MRI of the ex vivo human brain at 100 micron resolution, bioRxiv, 2019-05-31</h4>
          <p>AbstractWe present an ultra-high resolution MRI dataset of an ex vivo human brain specimen. The brain specimen was donated by a 58-year-old woman who had no history of neurological disease and died of non-neurological causes. After fixation in 10% formalin, the specimen was imaged on a 7 Tesla MRI scanner at 100 μm isotropic resolution using a custom-built 31-channel receive array coil. Single-echo multi-flip Fast Low-Angle SHot (FLASH) data were acquired over 100 hours of scan time (25 hours per flip angle), allowing derivation of a T1 parameter map and synthesized FLASH volumes. This dataset provides an unprecedented view of the three-dimensional neuroanatomy of the human brain. To optimize the utility of this resource, we warped the dataset into standard stereotactic space. We now distribute the dataset in both native space and stereotactic space to the academic community via multiple platforms. We envision that this dataset will have a broad range of investigational, educational, and clinical applications that will advance understanding of human brain anatomy in health and disease.&lt;jatstable-wrap id=utbl1 orientation=portrait position=float&gt;&lt;jatsgraphic xmlnsxlink=httpwww.w3.org1999xlink xlinkhref=649822v2_utbl1 position=float orientation=portrait &gt;&lt;jatstable-wrap&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/mammary_mechanobiology_mechanically-activated_ion_channels_in_lactation_and_involution_61136380/">
          <h4 class="font-weight-bold">Mammary mechanobiology mechanically-activated ion channels in lactation and involution, bioRxiv, 2019-05-27</h4>
          <p>AbstractA mother’s ability to produce a nutritionally-complete neonatal food source has provided a powerful evolutionary advantage to mammals. Milk production by secretory mammary epithelial cells is adaptive, its release is exquisitely timed and its own glandular stagnation with the permanent cessation of suckling triggers the programmed cell death and tissue remodeling that enables female mammals to nurse successive progeny. Both chemical and mechanical signals control epithelial expansion, function and remodeling. Despite this duality of input, however, the nature and function of mechanical forces in the mammary gland remain unknown. Here, we characterize the mammary force landscape and the capacity of luminal and basal epithelial cells to experience and exert force. We explore the molecular instruments for force-sensing in the mammary gland and the physiological requirement for PIEZO1 in lactation and involution. Our study supports the existence of a multifaceted system of chemical and mechanical sensing in the mammary gland, and a protective redundancy that ensures continued lactational competence and offspring survival.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/500&#43;-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/500&#43;-users/page/3/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="/audiences/tags/500&#43;-users/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/500&#43;-users/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/500&#43;-users/page/3/" class="page-link">3</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/500&#43;-users/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/500&#43;-users/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/500&#43;-users/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/500&#43;-users/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/500&#43;-users/page/8/" class="page-link">8</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/500&#43;-users/page/5/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/500&#43;-users/page/16/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
