<!DOCTYPE html>
<html><head>
  
  <title>biorxiv | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.68.3" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.95d2483cd3c5d2bf47c1f36307a456787d60ad958c5efe44db9afc47012b1b05b8e1f64f6ac291650439322a6c38e86eb7cd6254255516d857c23f299fb0e666.js" integrity="sha512-ldJIPNPF0r9HwfNjB6RWeH1grZWMXv5E25r8RwErGwW44fZPasKRZQQ5MipsOOhut81iVCVVFthXwj8pn7DmZg=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genotype-phenotype_relationships_in_children_with_copy_number_variants_associated_with_high_neuropsychiatric_risk_findings_from_the_intellectual_disability__mental_health_assessing_the_genomic_impact_on_neurodevelopment_imagine-id_study_54765026/">
          <h4 class="font-weight-bold">Genotype-phenotype relationships in children with copy number variants associated with high neuropsychiatric risk Findings from the Intellectual Disability &amp; Mental Health Assessing the Genomic Impact on Neurodevelopment (IMAGINE-ID) study, bioRxiv, 2019-01-31</h4>
          <p>AbstractBackgroundA variety of copy number variants are associated with a high risk of neurodevelopmental and psychiatric disorders (ND-CNVs). Different ND-CNVs could lead to distinct and specific patterns of cognitive and behavioural outcomes, but supporting evidence is currently lacking.Methods258 children with ND-CNVs (13 CNVs across 9 loci) were systematically assessed for psychiatric disorders as well as broader traits of neurodevelopmental, cognitive and psychopathological origin. A comparison was made with 106 non-carrier control siblings, in order to test the hypothesis that phenotypes would differ by genotype, both quantitatively, in terms of severity, and qualitatively in the pattern of associated impairments.Outcomes79.8% of ND-CNVs carriers met criteria for one or more psychiatric disorders (OR=13.8 compared to controls) the risk of ADHD (OR=6.9), ODD (OR=3.6), anxiety disorders (OR=2.9), and ASD traits (OR=44.1) was particularly high. ND-CNVs carriers were impaired across all neurodevelopmental, cognitive, and psychopathological traits relative to controls. Only moderate quantitative and qualitative differences in phenotypic profile were found between genotypes. In general, the range of phenotypes was broadly similar for all ND-CNV genotypes. Traits did show some evidence of genotypic specificity, however the specific genotype accounted for a low proportion of variance in outcome (5-20% depending on trait).InterpretationThe 13 ND-CNVs studied have a similar range of adverse effects on childhood neurodevelopment, despite subtle quantitative and qualitative differences. Our findings suggest that genomic risk for neuropsychiatric disorder has pleiotropic effects on multiple processes and neural circuits, and provides important implications for research into genotype-phenotype relationships within psychiatry.FundingThe Medical Research Council and the Medical Research FoundationResearch in contextEvidence before this studySeveral copy number variants (CNVs) have been associated with high risk of development of child and adult neuropsychiatric disorders. Increasingly young children with developmental delay referred for genetic testing are being diagnosed with neurodevelopmental and psychiatric risk CNVs (referred to as ND-CNVs hereafter). It remains unclear whether different genotypes are associated with specific cognitive and behavioural phenotypes or whether these outcomes are non-specific. We searched PubMed for studies published from database inception until January 10th, 2019 that investigated the relationship between CNVs and cognitive and behavioural outcomes. Search terms included “CNV”, “genomics”, “1q21.1”, “2p16.3”, “NRXN1”, “9q34”, “Kleefstra Syndrome”, “15q11.2”, “15q13.3”, “16p11.2”, “22q11.2”, “psychiatry”, and “cognition”. Preliminary studies have indicated that deletions and duplications at the same loci may differ in cognitive and behavioural phenotypes. However, to date, there have been limited studies that contrasted the phenotypes of ND-CNVs across several loci on a range of cognitive and behavioural domains.Added value of this studyWe found that young people carrying a ND-CNV were at considerably increased risk for neuropsychiatric disorder and impairments across a range of neurodevelopmental, psychopathological, cognitive, social, sleep and motor traits. Within ND-CNV carriers, comparisons between genotypes indicated moderate quantitative and qualitative differences in overall phenotypic profile, with evidence that severity of impairment was similar across all genotypes for some traits (e.g. mood problems, sleep impairments, peer problems, and sustained attention) whereas for other traits there was evidence of genotype specific effects on severity (e.g., IQ, spatial planning, processing speed, subclinical psychotic experiences, ASD traits, motor coordination total psychiatric symptomatology, particularly anxiety, ADHD, and conduct related traits). However the proportion of variance explained by genotype was low, 5-20% depending on trait, indicating that overall ND-CNVs lead to similar neurodevelopmental outcomes. It is important that genotype-phenotype relationships are viewed through a developmental lens as some phenotypic outcomes were found to be associated with age.Implications of all the available evidenceChildren who carry a ND-CNV represent a patient group that warrants clinical and educational attention for a broad range of cognitive and behavioural impairments. Although qualitative and quantitative differences exist between ND-CNVs, our findings point to commonalities in clinical outcomes with neurodevelopmental impairments being present across all ND-CNVs. This group of young people could benefit from the development of a general intervention plan, to which genotype-specific recommendations can be added where needed. Our findings do not support a model whereby different ND-CNVs represent discrete forms of neuropsychiatric disorder and suggest that multiple processes and neural circuits are affected by ND-CNVs. The pleiotropic effects of ND-CNVs emphasises that research aiming to identify causal pathways between genetic variation and psychiatric outcomes via intermediary (or endo-)phenotypes needs to take a global perspective and not be narrowly focused on single phenotypes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/distinct_characteristics_of_genes_associated_with_phenome-wide_variation_in_maize_zea_mays_54676152/">
          <h4 class="font-weight-bold">Distinct characteristics of genes associated with phenome-wide variation in maize (Zea mays), bioRxiv, 2019-01-30</h4>
          <p>ABSTRACTNaturally occurring functional genetic variation is often employed to identify genetic loci that regulate specific traits. Existing approaches to link functional genetic variation to quantitative phenotypic outcomes typically evaluate one or several traits at a time. Advances in high throughput phenotyping now enable datasets which include information on dozens or hundreds of traits scored across multiple environments. Here, we develop an approach to use data from many phenotypic traits simultaneously to identify causal genetic loci. Using data for 260 traits scored across a maize diversity panel, we demonstrate that a distinct set of genes are identified relative to conventional genome wide association. The genes identified using this many-trait approach are more likely to be independently validated than the genes identified by conventional analysis of the same dataset. Genes identified by the new many-trait approach share a number of molecular, population genetic, and evolutionary features with a gold standard set of genes characterized through forward genetics. These features, as well as substantially stronger functional enrichment and purification, separate them from both genes identified by conventional genome wide association and from the overall population of annotated gene models. These results are consistent with a large subset of annotated gene models in maize playing little or no role in determining organismal phenotypes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/is_it_time_to_change_the_reference_genome_54673121/">
          <h4 class="font-weight-bold">Is it time to change the reference genome?, bioRxiv, 2019-01-30</h4>
          <p>The use of the human reference genome has shaped methods and data across modern genomics.  This has offered many benefits while creating a few constraints. In the following piece, we outline the history, properties, and pitfalls of the current human reference genome. In a few illustrative analyses, we focus on its use for variant-calling, highlighting its nearness to a type specimen. We suggest that switching to a consensus reference offers important advantages over the current reference with few disadvantages.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/constant_sub-second_cycling_between_representations_of_possible_futures_in_the_hippocampus_54628652/">
          <h4 class="font-weight-bold">Constant sub-second cycling between representations of possible futures in the hippocampus, bioRxiv, 2019-01-29</h4>
          <p>Cognitive faculties such as imagination, planning, and decision-making entail the ability to project into the future. Crucially, animal behavior in natural settings implies that the brain can generate representations of future scenarios not only quickly but also constantly over time, as external events continually unfold. Despite this insight, how the brain accomplishes this remains unknown. Here we report neural activity in the hippocampus encoding two future scenarios (two upcoming maze paths) in constant alternation at 8 Hz one scenario per 8 Hz cycle (125 ms). We further found that the underlying cycling dynamic generalized across multiple hippocampal representations (location and direction) relevant to future behavior. These findings identify an extremely fast and regular dynamical process capable of representing future possibilities.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/evaluating_potential_drug_targets_through_human_loss-of-function_genetic_variation_54627010/">
          <h4 class="font-weight-bold">Evaluating potential drug targets through human loss-of-function genetic variation, bioRxiv, 2019-01-29</h4>
          <p>AbstractHuman genetics has informed the clinical development of new drugs, and is beginning to influence the selection of new drug targets. Large-scale DNA sequencing studies have created a catalogue of naturally occurring genetic variants predicted to cause loss of function in human genes, which in principle should provide powerful in vivo models of human genetic “knockouts” to complement model organism knockout studies and inform drug development. Here, we consider the use of predicted loss-of-function (pLoF) variation catalogued in the Genome Aggregation Database (gnomAD) for the evaluation of genes as potential drug targets. Many drug targets, including the targets of highly successful inhibitors such as aspirin and statins, are under natural selection at least as extreme as known haploinsufficient genes, with pLoF variants almost completely depleted from the population. Thus, metrics of gene essentiality should not be used to eliminate genes from consideration as potential targets. The identification of individual humans harboring “knockouts” (biallelic gene inactivation), followed by individual recall and deep phenotyping, is highly valuable to study gene function. In most genes, pLoF alleles are sufficiently rare that ascertainment will be largely limited to heterozygous individuals in outbred populations. Sampling of diverse bottlenecked populations and consanguineous individuals will aid in identification of total “knockouts”. Careful filtering and curation of pLoF variants in a gene of interest is necessary in order to identify true LoF individuals for follow-up, and the positional distribution or frequency of true LoF variants may reveal important disease biology. Our analysis suggests that the value of pLoF variant data for drug discovery lies in deep curation informed by the nature of the drug and its indication, as well as the biology of the gene, followed by recall-by-genotype studies in targeted populations.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/facilitating_open-science_with_realistic_fmri_simulation_validation_and_application_54629057/">
          <h4 class="font-weight-bold">Facilitating open-science with realistic fMRI simulation validation and application, bioRxiv, 2019-01-29</h4>
          <p>Background With advances in methods for collecting and analyzing fMRI data, there is a concurrent need to understand how to reliably evaluate and optimally use these methods. Simulations of fMRI data can aid in both the evaluation of complex designs and the analysis of data. New Method We present fmrisim, a new Python package for standardized, realistic simulation of fMRI data. This package is part of BrainIAK a recently released open-source Python toolbox for advanced neuroimaging analyses. We describe how to use fmrisim to extract noise properties from real fMRI data and then create a synthetic dataset with matched noise properties and a user-specified signal. Results We validate the noise generated by fmrisim to show that it can approximate the noise properties of real data. We further show how fmrisim can help researchers find the optimal design in terms of power. Comparison with other methods fmrisim ports the functionality of other packages to the Python platform while extending what is available in order to make it seamless to simulate realistic fMRI data. Conclusions The fmrisim package holds promise for improving the design of fMRI experiments, which may facilitate both the pre-registration</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/126/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/123/" class="page-link">123</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/124/" class="page-link">124</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/125/" class="page-link">125</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/126/" class="page-link">126</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/127/" class="page-link">127</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/128/" class="page-link">128</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/129/" class="page-link">129</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/130/" class="page-link">130</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/131/" class="page-link">131</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/128/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biorxiv/page/312/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
