<!DOCTYPE html>
<html><head>
  
  <title>microbiology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.68.3" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.95d2483cd3c5d2bf47c1f36307a456787d60ad958c5efe44db9afc47012b1b05b8e1f64f6ac291650439322a6c38e86eb7cd6254255516d857c23f299fb0e666.js" integrity="sha512-ldJIPNPF0r9HwfNjB6RWeH1grZWMXv5E25r8RwErGwW44fZPasKRZQQ5MipsOOhut81iVCVVFthXwj8pn7DmZg=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/interplay_between_the_human_gut_microbiome_and_host_metabolism_56164411/">
          <h4 class="font-weight-bold">Interplay between the human gut microbiome and host metabolism, bioRxiv, 2019-02-28</h4>
          <p>The human gut is inhabited by a complex and metabolically active microbial ecosystem regulating host health. While many studies have focused on the effect of individual microbial taxa, the metabolic potential of the entire gut microbial ecosystem has been largely under-explored. We characterised the gut microbiome of 1,004 twins via whole shotgun metagenomic sequencing (average 39M reads per sample). We observed greater similarity, across unrelated individuals, for functional metabolic pathways (82%) than for taxonomic composition (43%). We conducted a microbiota-wide association study linking both taxonomic information and microbial metabolic pathways with 673 blood and 713 faecal metabolites (Metabolon, Inc.). Metabolic pathways associated with 34% of blood and 95% of faecal metabolites, with over 18,000 significant associations, while species-level results identified less than 3,000 associations, suggesting that coordinated action of multiple taxa is required to affect the metabolome. Finally, we estimated that the microbiome mediated a crosstalk between 71% of faecal and 15% of blood metabolites, highlighting six key species (unclassified Subdoligranulum spp., Faecalibacterium prausnitzii, Roseburia inulinivorans, Methanobrevibacter smithii, Eubacterium rectale, and Akkermansia muciniphila). Because of the large inter-person variability in microbiome composition, our results underline the importance of studying gut microbial metabolic pathways rather than focusing purely on taxonomy to find therapeutic and diagnostic targets.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/necrotizing_enterocolitis_is_preceded_by_increased_gut_bacterial_replication_klebsiella_and_fimbriae-encoding_bacteria_that_may_stimulate_tlr4_receptors_55933703/">
          <h4 class="font-weight-bold">Necrotizing enterocolitis is preceded by increased gut bacterial replication, Klebsiella, and fimbriae-encoding bacteria that may stimulate TLR4 receptors, bioRxiv, 2019-02-23</h4>
          <p>Necrotizing enterocolitis (NEC) is a devastating intestinal disease that occurs primarily in premature infants. We performed genome-resolved metagenomic analysis of 1,163 fecal samples from premature infants to identify microbial features predictive of NEC. Features considered include genes, bacterial strain types, eukaryotes, bacteriophages, plasmids and growth rates. A machine learning classifier found that samples collected prior to NEC diagnosis harbored significantly more Klebsiella, bacteria encoding fimbriae, and bacteria encoding secondary metabolite gene clusters related to quorum sensing and bacteriocin production. Notably, replication rates of all bacteria, especially Enterobacteriaceae, were significantly higher two days before NEC diagnosis. The findings uncover biomarkers that could lead to early detection of NEC and targets for microbiome-based therapeutics.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/evolution-guided_design_of_super-restrictor_antiviral_proteins_reveals_a_breadth-versus-specificity_tradeoff_55902383/">
          <h4 class="font-weight-bold">Evolution-guided design of super-restrictor antiviral proteins reveals a breadth-versus-specificity tradeoff, bioRxiv, 2019-02-22</h4>
          <p>Host-virus evolutionary arms-races are driven by antagonistic interactions and often manifest as recurrent amino acid changes (i.e., positive selection) at their protein-protein interaction interfaces. Here, we investigated whether combinatorial mutagenesis of positions under positive selection in a host antiviral protein could enhance its restrictive properties. We tested ~800 variants of the human MxA protein, generated by combinatorial mutagenesis, for their ability to restrict Thogoto orthomyxovirus (THOV). We identified MxA ‘super-restrictors&#39; with increased binding to THOV NP target protein and 10-fold higher anti-THOV restriction relative to wild-type human MxA, the most potent naturally-occurring anti-THOV restrictor identified. However, MxA super-restrictors of THOV were impaired in their restriction of influenza A virus. Our findings thus reveal a breadth-versus-specificity tradeoff that constrains the adaptive landscape of antiviral proteins.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/discovery_of_several_thousand_highly_diverse_circular_dna_viruses_55772625/">
          <h4 class="font-weight-bold">Discovery of several thousand highly diverse circular DNA viruses, bioRxiv, 2019-02-20</h4>
          <p>SummaryAlthough it is suspected that there are millions of distinct viral species, fewer than 9,000 are catalogued in GenBank’s RefSeq database. We selectively enriched for and amplified the genomes of circular DNA viruses in over 70 animal samples, ranging from cultured soil nematodes to human tissue specimens. A bioinformatics pipeline, Cenote-Taker, was developed to automatically annotate over 2,500 circular genomes in a GenBank-compliant format. The new genomes belong to dozens of established and emerging viral families. Some appear to be the result of previously undescribed recombination events between ssDNA viruses and ssRNA viruses. In addition, hundreds of circular DNA elements that do not encode any discernable similarities to previously characterized sequences were identified. To characterize these “dark matter” sequences, we used an artificial neural network to identify candidate viral capsid proteins, several of which formed virus-like particles when expressed in culture. These data further the understanding of viral sequence diversity and allow for high throughput documentation of the virosphere.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/spatial_metabolomics_of_in_situ_host-microbe_interactions_55762881/">
          <h4 class="font-weight-bold">Spatial metabolomics of in situ, host-microbe interactions, bioRxiv, 2019-02-20</h4>
          <p>Spatial metabolomics describes the location and chemistry of small molecules involved in metabolic phenotypes, defense molecules and chemical interactions in natural communities. Most current techniques are unable to spatially link the genotype and metabolic phenotype of microorganisms in situ at a scale relevant to microbial interactions. Here, we present a spatial metabolomics pipeline (metaFISH) that combines fluorescence in situ hybridization (FISH) microscopy and high-resolution atmospheric pressure mass spectrometry imaging (AP-MALDI-MSI) to image host-microbe symbioses and their metabolic interactions. metaFISH aligns and integrates metabolite and fluorescent images at the micrometer-scale for a spatial assignment of host and symbiont metabolites on the same tissue section. To illustrate the advantages of metaFISH, we mapped the spatial metabolome of a deep-sea mussel and its intracellular symbiotic bacteria at the scale of individual epithelial host cells. Our analytical pipeline revealed metabolic adaptations of the epithelial cells to the intracellular symbionts, a variation in metabolic phenotypes in one symbiont type, and novel symbiosis metabolites. metaFISH provides a culture-independent approach to link metabolic phenotypes to community members in situ - a powerful tool for microbiologists across fields.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/hospital_use_of_antibiotics_as_the_main_driver_of_infections_with_antibiotic-resistant_bacteria_-_a_reanalysis_of_recent_data_from_the_european_union_55695720/">
          <h4 class="font-weight-bold">Hospital use of antibiotics as the main driver of infections with antibiotic-resistant bacteria - a reanalysis of recent data from the European Union, bioRxiv, 2019-02-19</h4>
          <p>Antimicrobial resistance in bacteria causes significant morbidity worldwide. The development and acquisition of resistance to antibiotics is believed to primarily develop under the selective pressure of widespread antibiotic use in humans, however antimicrobial usage in livestock has been proposed as additional, if not principal, driver of antibiotic resistance. In this work, we correlate recent data from the European Union on antibiotic resistance rates with data on antibiotic usage in the primary care and hospital sector and data on veterinary antimicrobial consumption across the individual member states. We quantify the strength of these different potential drivers of antimicrobial resistance in order to compare their biological importance. We found that the correlation between antibiotic use in the hospital sector and antibiotic resistance rates is significantly higher than the correlation between resistance rates and any of the other two predictors. This suggests increased antibiotic use in hospitals as the main driver of the development of antibiotic resistances and necessitates further research on and a re-evaluation of the risks associated with antibiotic use in human and veterinary medicine.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/7/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/7/" class="page-link">7</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/microbiology/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/10/" class="page-link">10</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/11/" class="page-link">11</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/12/" class="page-link">12</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/9/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/microbiology/page/16/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
