<!DOCTYPE html>
<html><head>
  
  <title>Reports | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/comparative_assessment_of_long-read_error-correction_software_applied_to_rna-sequencing_data_51603919/">
          <h4 class="font-weight-bold">Comparative assessment of long-read error-correction software applied to RNA-sequencing data, bioRxiv, 2018-11-23</h4>
          <p>AbstractMotivationLong-read sequencing technologies offer promising alternatives to high-throughput short read sequencing, especially in the context of RNA-sequencing. However these technologies are currently hindered by high error rates in the output data that affect analyses such as the identification of isoforms, exon boundaries, open reading frames, and the creation of gene catalogues. Due to the novelty of such data, computational methods are still actively being developed and options for the error-correction of RNA-sequencing long reads remain limited.ResultsIn this article, we evaluate the extent to which existing long-read DNA error correction methods are capable of correcting cDNA Nanopore reads. We provide an automatic and extensive benchmark tool that not only reports classical error-correction metrics but also the effect of correction on gene families, isoform diversity, bias towards the major isoform, and splice site detection. We find that long read error-correction tools that were originally developed for DNA are also suitable for the correction of RNA-sequencing data, especially in terms of increasing base-pair accuracy. Yet investigators should be warned that the correction process perturbs gene family sizes and isoform diversity. This work provides guidelines on which (or whether) error-correction tools should be used, depending on the application type.Benchmarking software&lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsgitlab.comleoislLR_EC_analyser&gt;httpsgitlab.comleoislLR_EC_analyser&lt;jatsext-link&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/discovery_of_the_first_genome-wide_significant_risk_loci_for_attention_deficithyperactivity_disorder_51769413/">
          <h4 class="font-weight-bold">Discovery of the first genome-wide significant risk loci for attention deficithyperactivity disorder, Nature Genetics, 2018-11-23</h4>
          <p>Attention deficithyperactivity disorder (ADHD) is a highly heritable childhood behavioral disorder affecting 5% of children and 2.5% of adults. Common genetic variants contribute substantially to ADHD susceptibility, but no variants have been robustly associated with ADHD. We report a genome-wide association meta-analysis of 20,183 individuals diagnosed with ADHD and 35,191 controls that identifies variants surpassing genome-wide significance in 12 independent loci, finding important new information about the underlying biology of ADHD. Associations are enriched in evolutionarily constrained genomic regions and loss-of-function intolerant genes and around brain-expressed regulatory marks. Analyses of three replication studies a cohort of individuals diagnosed with ADHD, a self-reported ADHD sample and a meta-analysis of quantitative measures of ADHD symptoms in the population, support these findings while highlighting study-specific differences on genetic overlap with educational attainment. Strong concordance with GWAS of quantitative population measures of ADHD symptoms supports that clinical diagnosis of ADHD is an extreme expression of continuous heritable traits.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/nature%20genetics">nature genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/c._elegans_pathogenic_learning_confers_multigenerational_pathogen_avoidance_51528447/">
          <h4 class="font-weight-bold">C. elegans pathogenic learning confers multigenerational pathogen avoidance, bioRxiv, 2018-11-22</h4>
          <p>AbstractThe ability to pass on learned information to progeny could present an evolutionary advantage for many generations. While apparently evolutionarily conserved1–12, transgenerational epigenetic inheritance (TEI) is not well understood at the molecular or behavioral levels. Here we describe our discovery that C. elegans can pass on a learned pathogenic avoidance behavior to their progeny for several generations through epigenetic mechanisms. Although worms are initially attracted to the gram-negative bacteria P. aeruginosa (PA14)13, they can learn to avoid this pathogen13. We found that prolonged PA14 exposure results in transmission of avoidance behavior to progeny that have themselves never been exposed to PA14, and this behavior persists through the fourth generation. This form of transgenerational inheritance of bacterial avoidance is specific to pathogenic P. aeruginosa, requires physical contact and infection, and is distinct from CREB-dependent long-term associative memory and larval imprinting. The TGF-β ligand daf-7, whose expression increases in the ASJ upon initial exposure to PA1414, is highly expressed in the ASI neurons of progeny of trained mothers until the fourth generation, correlating with transgenerational avoidance behavior. Mutants of histone modifiers and small RNA mediators display defects in naïve PA14 attraction and aversive learning. By contrast, the germline-expressed PRG-1Piwi homolog15 is specifically required for transgenerational inheritance of avoidance behavior. Our results demonstrate a novel and natural paradigm of TEI that may optimize progeny decisions and subsequent survival in the face of changing environmental conditions.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/hunger_for_knowledge_how_the_irresistible_lure_of_curiosity_is_generated_in_the_brain_51561076/">
          <h4 class="font-weight-bold">Hunger for Knowledge How the Irresistible Lure of Curiosity is Generated in the Brain, bioRxiv, 2018-11-22</h4>
          <p>Introductory ParagraphCuriosity is often portrayed as a desirable feature of human faculty. For example, a meta-analysis revealed that curiosity predicts academic performance above and beyond intelligence1, corroborating findings that curiosity supported long-term consolidation of learning2,3. However, curiosity may come at a cost of strong seductive power that sometimes puts people in a harmful situation. Here, with a set of three behavioural and two neuroimaging experiments including novel stimuli that strongly trigger curiosity (i.e. magic tricks), we examined the psychological and neural mechanisms underlying the irresistible lure of curiosity. We consistently demonstrated that across different samples people were indeed willing to gamble to expose themselves to physical risks (i.e. electric shocks) in order to satisfy their curiosity for trivial knowledge that carries no apparent instrumental values. Also, underlying this seductive power of curiosity is its incentive salience properties, which share common neural mechanisms with extrinsic incentives (i.e. hunger for foods). In particular, the two independent fMRI experiments using different kinds of curiosity-stimulating stimuli found replicable results that acceptance (compared to rejection) of curiosityincentive-driven gambles was accompanied with an enhanced activity in the striatum.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/a_primer_on_deep_learning_in_genomics_51770010/">
          <h4 class="font-weight-bold">A primer on deep learning in genomics, Nature Genetics, 2018-11-21</h4>
          <p>Deep learning methods are a class of machine learning techniques capable of identifying highly complex patterns in large datasets. Here, we provide a perspective and primer on deep learning applications for genome analysis. We discuss successful applications in the fields of regulatory genomics, variant calling and pathogenicity scores. We include general guidance for how to effectively use deep learning methods as well as a practical guide to tools and resources. This primer is accompanied by an interactive online tutorial.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/nature%20genetics">nature genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/crispr-tsko_facilitates_efficient_cell_type-_tissue-_or_organ-specific_mutagenesis_in_arabidopsis_51476356/">
          <h4 class="font-weight-bold">CRISPR-TSKO facilitates efficient cell type-, tissue-, or organ-specific mutagenesis in Arabidopsis, bioRxiv, 2018-11-21</h4>
          <p>AbstractDetailed functional analyses of many fundamentally-important plant genes via conventional loss-of-function approaches are impeded by severe pleiotropic phenotypes. In particular, mutations in genes that are required for basic cellular functions andor reproduction often interfere with the generation of homozygous mutant plants, precluding further functional studies. To overcome this limitation, we devised a CRISPR-based tissue-specific knockout system, CRISPR-TSKO, enabling the generation of somatic mutations in particular plant cell types, tissues, and organs. In Arabidopsis, CRISPR-TSKO mutations in essential genes caused well-defined, localized phenotypes in the root cap, stomatal lineage, or entire lateral roots. The underlying modular cloning system allows for efficient selection, identification, and functional analysis of mutant lines directly in the first transgenic generation. The efficacy of CRISPR-TSKO opens new avenues to discover and analyze gene functions in spatial and temporal contexts of plant life while avoiding pleiotropic effects of system-wide loss of gene function.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/reports/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/reports/page/95/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/reports/page/92/" class="page-link">92</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/93/" class="page-link">93</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/94/" class="page-link">94</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/95/" class="page-link">95</a></li>
            
        
             
            <li class="page-item active"><a href="/reports/page/96/" class="page-link">96</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/97/" class="page-link">97</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/98/" class="page-link">98</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/99/" class="page-link">99</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/100/" class="page-link">100</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/reports/page/97/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/reports/page/172/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
