<!DOCTYPE html>
<html><head>
  
  <title>Biorxiv | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/serum_flt3_ligand_is_biomarker_of_progenitor_cell_mass_and_prognosis_in_acute_myeloid_leukemia_57683166/">
          <h4 class="font-weight-bold">Serum Flt3 ligand is biomarker of progenitor cell mass and prognosis in acute myeloid leukemia, bioRxiv, 2019-03-25</h4>
          <p>AbstractFms-like tyrosine kinase 3 (Flt3) is a hematopoietic growth factor receptor expressed on lymphomyeloid progenitors and frequently, by AML blasts. Its ligand, Flt3L, has non-hematopoietic and lymphoid origins, is detectable during homeostasis and increases to high levels in states of hypoplasia due to genetic defects or treatment with cytoreductive agents. Measurement of Flt3L by ELISA reveals that Flt3&#43;AML, is associated with depletion of Flt3L to undetectable levels. After induction chemotherapy, Flt3L is restored in patients entering CR, but remains depressed in those with refractory disease. Weekly sampling reveals marked differences in the kinetics of Flt3L response during the first 6 weeks of treatment, proportionate to the clearance of blasts and cellularity of the BM. In the UK NCRI AML17 trial, Flt3L was measured at day 26 in a subgroup of 135 patients with Flt3 mutation randomized to the tyrosine kinase inhibitor lestaurtinib. In these patients, attainment of CR was associated with higher Flt3L at day 26 (Mann-Whitney p &amp;lt; 0.0001). Day 26 Flt3L was also associated with survival Flt3L ≤ 291pgml was associated with inferior event-free survival; and, Flt3L &amp;gt;1185pgml was associated with higher overall survival (p = 0.0119). Serial measurement of Flt3L in patients who had received a hematopoietic stem cell transplant for AML further illustrated the potential value of declining Flt3L to identify relapse. Together these observations suggest that measurement of Flt3L provides a non-invasive estimate of progenitor cell mass in most patients with AML, with the potential to inform clinical decisions.Graphical abstract&lt;jatsfig id=ufig1 position=float fig-type=figure orientation=portrait&gt;&lt;jatsgraphic xmlnsxlink=httpwww.w3.org1999xlink xlinkhref=588319_ufig1 position=float orientation=portrait &gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cancer-biology">cancer-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/bipartite_anchoring_of_scream_enforces_stomatal_initiation_by_coupling_map_kinases_to_speechless_57606705/">
          <h4 class="font-weight-bold">Bipartite anchoring of SCREAM enforces stomatal initiation by coupling MAP Kinases to SPEECHLESS, bioRxiv, 2019-03-24</h4>
          <p>AbstractCell-fate in eukaryotes is regulated by MAP Kinases (MAPKs) that translate external cues to cellular responses. In plants, two MAPKs, MPK36, regulate diverse processes of development, environmental response, and immunity. Yet, the mechanism bridging these shared signaling components with a specific target remains unresolved. Focusing on the development of stomata, epidermal valves for gas exchange and transpiration, we report here that the bHLH protein SCREAM functions as a scaffold by recruiting MPK36 to downregulate SPEECHLESS, a transcription factor initiating stomatal cell lineages. SCREAM directly binds with MPK36 through an evolutionarily-conserved yet unconventional bipartite motif. Mutations in this motif abrogate association, phosphorylation and degradation of SCREAM, unmask hidden non-redundancies between MPK3 and MPK6, and result in uncontrolled stomatal differentiation. Structural analyses of MPK6 at the 2.75Å resolution unraveled bipartite binding of SCREAM with MPK6, that is distinct from an upstream MAPKK. Our findings elucidate, at the atomic resolution, the mechanism directly linking extrinsic signals to transcriptional reprogramming during the establishment of stomatal cell-fate, and highlight a unique substrate-binding mode adopted by plant MAPKs.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/spatial_structure_governs_the_mode_of_tumour_evolution_57587794/">
          <h4 class="font-weight-bold">Spatial structure governs the mode of tumour evolution, bioRxiv, 2019-03-24</h4>
          <p>AbstractCharacterizing the mode – the way, manner, or pattern – of evolution in tumours is important for clinical forecasting and optimizing cancer treatment. DNA sequencing studies have inferred various modes, including branching, punctuated and neutral evolution, but it is unclear why a particular pattern predominates in any given tumour.1, 2 Here we propose that differences in tumour architecture alone can explain the variety of observed patterns. We examine this hypothesis using spatially explicit population genetic models and demonstrate that, within biologically relevant parameter ranges, human tumours are expected to exhibit four distinct onco-evolutionary modes (oncoevotypes) rapid clonal expansion (predicted in leukaemia); progressive diversification (in colorectal adenomas and early-stage colorectal carcinomas); branching evolution (in invasive glandular tumours); and effectively almost neutral evolution (in certain non-glandular and poorly differentiated solid tumours). We thus provide a simple, mechanistic explanation for a wide range of empirical observations. Oncoevotypes are governed by the mode of cell dispersal and the range of cell-cell interaction, which we show are essential factors in accurately characterizing, forecasting and controlling tumour evolution.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cancer-biology">cancer-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/the_ability_of_single_genes_vs_full_genomes_to_resolve_time_and_space_in_outbreak_analysis_57504868/">
          <h4 class="font-weight-bold">The ability of single genes vs full genomes to resolve time and space in outbreak analysis, bioRxiv, 2019-03-24</h4>
          <p>AbstractInexpensive pathogen genome sequencing has had a transformative effect on the field of phylodynamics, where ever increasing volumes of data have promised real-time insight into outbreaks of infectious disease. As well as the sheer volume of pathogen isolates being sequenced, the sequencing of whole pathogen genomes, rather than select loci, has allowed phylogenetic analyses to be carried out at finer time scales, often approaching serial intervals for infections caused by rapidly evolving RNA viruses. Despite its utility, whole genome sequencing of pathogens has not been adopted universally and targeted sequencing of loci is common in some pathogen-specific fields. In this study we aim to highlight the utility of sequencing whole genomes of pathogens by re-analysing a well-characterised collection of Ebola virus sequences in the form of complete viral genomes (~19kb long) or the rapidly evolving glycoprotein (GP, ~2kb long) gene. We quantify changes in phylogenetic, temporal, and spatial inference resolution as a result of this reduction in data and compare these to theoretical expectations. We propose a simple intuitive metric for quantifying temporal resolution, i.e. the time scale over which sequence data might be informative of various processes as a quick back-of-the-envelope calculation of statistical power available to molecular clock analyses.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/epidemiology">epidemiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/dietary_intake_regulates_the_circulating_inflammatory_monocyte_pool_57444705/">
          <h4 class="font-weight-bold">Dietary intake regulates the circulating inflammatory monocyte pool, bioRxiv, 2019-03-22</h4>
          <p>SUMMARYCaloric restriction is known to improve inflammatory and autoimmune diseases. However, the mechanisms by which reduced caloric intake modulates inflammation are poorly understood. Here we show that short-term fasting reduced monocyte metabolic and inflammatory activity and drastically reduced the number of circulating monocytes. Regulation of peripheral monocyte numbers was dependent on dietary glucose and protein levels. Specifically, we found that activation of the low-energy sensor 5’-AMP-activated protein kinase (AMPK) in hepatocytes and suppression of systemic CCL2 production by peroxisome proliferator-activator receptor alpha (PPARα) reduced monocyte mobilization from the bone marrow. Importantly, while caloric restriction improves chronic inflammatory diseases, fasting did not compromise monocyte emergency mobilization during acute infectious inflammation and tissue repair. These results reveal that caloric intake and liver energy sensors dictate the blood and tissue immune tone and link dietary habits to inflammatory disease outcome.Highlights&lt;jatslist list-type=bullet&gt;&lt;jatslist-item&gt;Fasting reduces the numbers of peripheral pro-inflammatory monocytes in healthy humans and mice.&lt;jatslist-item&gt;&lt;jatslist-item&gt;A hepatic AMPK-PPARα energy-sensing axis controls homeostatic monocyte numbers via regulation of steady-state CCL2 production.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Fasting reduces monocyte metabolic and inflammatory activity.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Fasting improves chronic inflammatory diseases but does not compromise monocyte emergency mobilization during acute infectious inflammation and tissue repair.&lt;jatslist-item&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/mitigating_pandemic_risk_with_influenza_a_virus_field_surveillance_at_a_swine-human_interface_57444712/">
          <h4 class="font-weight-bold">Mitigating Pandemic Risk with Influenza A Virus Field Surveillance at a Swine-Human Interface, bioRxiv, 2019-03-22</h4>
          <p>Working overnight at a large swine exhibition, we identified an influenza A virus (IAV) outbreak in swine, nanopore-sequenced 13 IAV genomes from samples collected, and in real-time, determined that these viruses posed a novel risk to humans due to genetic mismatches between the viruses and current pre-pandemic candidate vaccine viruses (CVV). We developed and used a portable IAV sequencing and analysis platform called Mia (Mobile Influenza Analysis) to complete and characterize full-length consensus genomes approximately 18 hours after unpacking the mobile lab. Swine are important animal IAV reservoirs that have given rise to pandemic viruses via zoonotic transmission. Genomic analyses of IAV in swine are critical to understanding pandemic risk of viruses in this reservoir, and characterization of viruses circulating in exhibition swine enables rapid comparison to current seasonal influenza vaccines and CVVs. The Mia system rapidly identified three genetically distinct swine IAV lineages from three subtypes A(H1N1), A(H3N2) and A(H1N2). Additional analysis of the HA protein sequences of the A(H1N2) viruses identified &amp;gt;30 amino acid differences between the HA1 portion of the hemagglutinin of these viruses and the most closely related pre-2009 CVV. All virus sequences were emailed to colleagues at CDC who initiated development of a synthetically derived CVV designed to provide an optimal antigenic match with the viruses detected in the exhibition. In subsequent months, this virus caused 13 infections in humans, and was the dominant variant virus in the US detected in 2018. Had this virus caused a severe outbreak or pandemic, our proactive surveillance efforts and CVV derivation would have provided an approximate 8 week time advantage for vaccine manufacturing. This is the first report of the use of field-derived nanopore sequencing data to initiate a real-time, actionable public health countermeasure.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/biorxiv/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/biorxiv/page/39/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/36/" class="page-link">36</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/37/" class="page-link">37</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/38/" class="page-link">38</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/39/" class="page-link">39</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/biorxiv/page/40/" class="page-link">40</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/41/" class="page-link">41</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/42/" class="page-link">42</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/43/" class="page-link">43</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/biorxiv/page/44/" class="page-link">44</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/biorxiv/page/41/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/biorxiv/page/164/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
