<!DOCTYPE html>
<html><head>
  
  <title>Reports | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/scaling_computational_genomics_to_millions_of_individuals_with_gpus_51176671/">
          <h4 class="font-weight-bold">Scaling computational genomics to millions of individuals with GPUs, bioRxiv, 2018-11-14</h4>
          <p>Current genomics methods were designed to handle tens to thousands of samples, but will soon need to scale to millions to keep up with the pace of data and hypothesis generation in biomedical science. Moreover, costs associated with processing these growing datasets will become prohibitive without improving the computational efficiency and scalability of methods. Here, we show that recently developed machine-learning libraries (TensorFlow and PyTorch) facilitate implementation of genomics methods for GPUs and significantly accelerate computations. To demonstrate this, we re-implemented methods for two commonly performed computational genomics tasks QTL mapping and Bayesian non-negative matrix factorization. Our implementations ran &amp;gt; 200 times faster than current CPU-based versions, and these analyses are ~5-10 fold cheaper on GPUs due to the vastly shorter runtimes. We anticipate that the accessibility of these libraries, and the improvements in run-time will lead to a transition to GPU-based implementations for a wide range of computational genomics methods.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/unbiased_detection_of_crispr_off-targets_in_vivo_using_discover-seq_51176196/">
          <h4 class="font-weight-bold">Unbiased detection of CRISPR off-targets in vivo using DISCOVER-Seq, bioRxiv, 2018-11-14</h4>
          <p>AbstractGenome editing using nucleases such as CRISPR-Cas induces programmable DNA damage at a target genomic site but can also affect off-target sites. Here, we develop a powerful, sensitive assay for the unbiased identification of off-target sites that we term DISCOVER-Seq. This approach takes advantage of the recruitment of endogenous DNA repair factors for genome-wide identification of Cas-induced double-strand breaks. One such factor, MRE11, is recruited precisely to double-strand breaks, enabling molecular characterization of nuclease cut sites with single-base resolution. DISCOVER-Seq detects off-targets in cellular models and in vivo upon adenoviral gene editing of mouse livers, paving the way for real-time off-target discovery during therapeutic gene editing. DISCOVER-Seq is furthermore applicable to multiple types of Cas nucleases and provides an unprecedented view of events that precede repair of the affected sites.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/assembly_of_a_pan-genome_from_deep_sequencing_of_910_humans_of_african_descent_51402455/">
          <h4 class="font-weight-bold">Assembly of a pan-genome from deep sequencing of 910 humans of African descent, Nature Genetics, 2018-11-13</h4>
          <p>We used a deeply sequenced dataset of 910 individuals, all of African descent, to construct a set of DNA sequences that is present in these individuals but missing from the reference human genome. We aligned 1.19 trillion reads from the 910 individuals to the reference genome (GRCh38), collected all reads that failed to align, and assembled these reads into contiguous sequences (contigs). We then compared all contigs to one another to identify a set of unique sequences representing regions of the African pan-genome missing from the reference genome. Our analysis revealed 296,485,284 bp in 125,715 distinct contigs present in the populations of African descent, demonstrating that the African pan-genome contains ~10% more DNA than the current human reference genome. Although the functional significance of nearly all of this sequence is unknown, 387 of the novel contigs fall within 315 distinct protein-coding genes, and the rest appear to be intergenic.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/nature%20genetics">nature genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/single_particle_cryo-em_reconstruction_of_52_kda_streptavidin_at_3.2_angstrom_resolution_51142456/">
          <h4 class="font-weight-bold">Single particle cryo-EM reconstruction of 52 kDa streptavidin at 3.2 Angstrom resolution, bioRxiv, 2018-11-13</h4>
          <p>AbstractThe fast development of single particle cryo-EM has made it more feasible to obtain the 3D structure of well-behaved macromolecules with molecular weight higher than 300 kDa at ~3 Å resolution. It remains a challenge to obtain high resolution structure of molecules smaller than 100 kDa using single particle cryo-EM, mainly due to the low contrast of the molecules embedded in vitreous ice. In this work, we applied the Cs-corrector-VPP coupled cryo-EM to study 52 kDa streptavidin (SA) protein supported on a thin layer of graphene film and embedded in vitreous ice. We were able to solve both the apo-SA and biotin-bound SA at near-atomic resolution using single particle cryo-EM. We demonstrated that the method is capable to determine the structure of molecule as small as 39 kDa and potentially even smaller molecules. Furthermore, we found that using the graphene film to avoid the adsorption to the air-water interface is critical to maintain the protein’s high-resolution structural information.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/target-specific_co-transmission_of_acetylcholine_and_gaba_from_a_subset_of_cortical_vip&#43;_interneurons_51111647/">
          <h4 class="font-weight-bold">Target-specific co-transmission of acetylcholine and GABA from a subset of cortical VIP&#43; interneurons, bioRxiv, 2018-11-13</h4>
          <p>AbstractThe modulation of cortex by acetylcholine (ACh) is typically thought to originate from long-range projections arising in the basal forebrain. However, a subset of VIP interneurons express ChAT, the synthetic enzyme for ACh, and are a potential local source of cortical ACh. Which neurotransmitters these VIPChAT interneurons (VCINs) release is unclear, and which post-synaptic cell types these transmitters target is not known. Using quantitative molecular analysis of VCIN pre-synaptic terminals, we show expression of the molecular machinery to release both ACh and GABA, with ACh release restricted to a subset of boutons. A systematic survey of potential post-synaptic cell types shows that VCINs release GABA primarily onto other inhibitory interneuron subtypes, while ACh neurotransmission is notably sparse, with most ACh release onto layer 1 interneurons and other VCINs. Therefore, VCINs are an alternative source of cortical ACh signaling that supplement GABA-mediated disinhibition with highly targeted excitation through ACh.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/characterization_of_proprioceptive_system_dynamics_in_behaving_drosophila_larvae_using_high-speed_volumetric_microscopy_51038018/">
          <h4 class="font-weight-bold">Characterization of proprioceptive system dynamics in behaving Drosophila larvae using high-speed volumetric microscopy, bioRxiv, 2018-11-12</h4>
          <p>SummaryProprioceptors provide feedback about body position that is essential for coordinated movement. Proprioceptive sensing of the position of rigid joints has been described in detail in several systems, however it is not known how animals with an elastic skeleton encode their body positions. Understanding how diverse larval body positions are dynamically encoded requires knowledge of proprioceptor activity patterns in vivo during natural movement. Here we applied high-speed volumetric SCAPE microscopy to simultaneously track the position, physical deformation, and temporal patterns of intracellular calcium activity of multidendritic proprioceptors in crawling Drosophila larvae. During the periodic segment contraction and relaxation that occurs during crawling, proprioceptors with diverse morphologies showed sequential onset of activity throughout each periodic episode. A majority of these proprioceptors showed activity during segment contraction with one neuron type activated by segment extension. Different timing of activity of contraction-sensing proprioceptors was related to distinct dendrite terminal targeting, providing a continuum of position encoding during all phases of crawling. These dynamics could endow different proprioceptors with specific roles in monitoring the progression of contraction waves, as well as body shape during other behaviors. We provide activity measurements during exploration as one example. Our results provide powerful new insights into the body-wide neuronal dynamics of the proprioceptive system in crawling Drosophila, and demonstrate the utility of our approach for characterization of neural encoding throughout the nervous system of a freely behaving animal.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../reports/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../reports/page/161/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/158/" class="page-link">158</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/159/" class="page-link">159</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/160/" class="page-link">160</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/161/" class="page-link">161</a></li>
            
        
             
            <li class="page-item active"><a href="../../../reports/page/162/" class="page-link">162</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/163/" class="page-link">163</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/164/" class="page-link">164</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/165/" class="page-link">165</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/166/" class="page-link">166</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../reports/page/163/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../reports/page/320/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
