<!DOCTYPE html>
<html><head>
  
  <title>Evolutionary Biology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/assortative_mating_in_hybrid_zones_is_remarkably_ineffective_in_promoting_speciation_60565197/">
          <h4 class="font-weight-bold">Assortative mating in hybrid zones is remarkably ineffective in promoting speciation, bioRxiv, 2019-05-17</h4>
          <p>AbstractAssortative mating and other forms of partial prezygotic isolation are often viewed as being more important than partial postzygotic isolation (low fitness of hybrids) early in the process of speciation. Here we simulate secondary contact between two populations (‘species’) to examine effects of pre- and postzygotic isolation in preventing blending. A small reduction in hybrid fitness (e.g., 10%) produces a narrower hybrid zone than a strong but imperfect mating preference (e.g., 10x stronger preference for conspecific over heterospecific mates). This is because, in the latter case, rare F1 hybrids find each other attractive (due to assortative mating), leading to the gradual buildup of a full continuum of intermediates between the two species. The cline is narrower than would result from purely neutral diffusion over the same number of generations, but this effect is due to the frequency-dependent mating disadvantage of individuals of rare mating types. Hybrids tend to pay this cost of rarity more than pure individuals, meaning there is an induced postzygotic isolation effect of assortative mating. When this induced mating disadvantage is removed, partial assortative mating does not prevent eventual blending of the species. These results prompt a questioning of the concept of partial prezygotic isolation, since it is not very isolating unless there is also postzygotic isolation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/fine-scale_haplotype_structure_reveals_strong_signatures_of_positive_selection_in_a_recombining_bacterial_pathogen_60279815/">
          <h4 class="font-weight-bold">Fine-scale haplotype structure reveals strong signatures of positive selection in a recombining bacterial pathogen, bioRxiv, 2019-05-11</h4>
          <p>ABSTRACTIdentifying the forces that create and shape ecologically meaningful variation in bacteria remains an important challenge. For recombining bacteria, the sign and strength of linkage provide a unique lens into ongoing selection. We show derived alleles less than 300bp apart in Neisseria gonorrhoeae exhibit more coupling linkage than repulsion linkage, a pattern that cannot be explained by limited recombination or neutrality as these couplings are significantly stronger for nonsynonymous alleles compared to synonymous alleles. While linkage is shaped by many evolutionary processes, extensive simulations show only two distinct forms of positive selection can drive an excess of coupling linkage between neighboring nonsynonymous alleles directional selection on introgressed alleles or selection that maintains distinct haplotypes in the presence of recombination. Our results establish a framework for identifying patterns of selection in fine-scale haplotype structure that indicate specific ecological processes in species that recombine with distantly related lineages or possess coexisting adaptive haplotypes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/early_origin_and_deep_conservation_of_enhancers_in_animals_60206888/">
          <h4 class="font-weight-bold">Early origin and deep conservation of enhancers in animals, bioRxiv, 2019-05-10</h4>
          <p>AbstractTranscription factors (TFs) bind DNA enhancer sequences to regulate gene transcription in animals. Unlike TFs, the evolution of enhancers has been difficult to trace because of their rapid evolution. Here, we show enhancers from the sponge Amphimedon queenslandica can drive cell type-specific reporter gene expression in zebrafish and mouse, despite sponge and vertebrate lineages diverging over 700 million years ago. Although sponge enhancers, which are present in both highly conserved syntenic gene regions (Islet–Scaper, Ccne1–Uri and Tdrd3–Diaph3) and sponge-specific intergenic regions, have no significant sequence identity with vertebrate genomic sequences, the type and frequency of TF binding motifs in the sponge enhancer allow for the identification of homologous enhancers in bilaterians. Islet enhancers identified in human and mouse Scaper genes drive zebrafish reporter expression patterns that are almost identical to the sponge Islet enhancer. The existence of homologous enhancers in these disparate metazoans suggests animal development is controlled by TF-enhancer DNA interactions that were present in the first multicellular animals.One-sentence summaryEnhancer activity is conserved across 700 million years of trans-phyletic divergence.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/matryoshka_rna_virus_1_a_novel_rna_virus_associated_with_plasmodium_parasites_in_human_malaria_59782036/">
          <h4 class="font-weight-bold">Matryoshka RNA virus 1 a novel RNA virus associated with Plasmodium parasites in human malaria, bioRxiv, 2019-05-01</h4>
          <p>AbstractParasites of the genus Plasmodium cause human malaria. Yet nothing is known about the viruses that infect these divergent eukaryotes. We investigated the Plasmodium virome by performing a meta-transcriptomic analysis of blood samples from malaria patients infected with P. vivax, P. falciparum or P. knowlesi. This revealed a novel bi-segmented narna-like RNA virus restricted to P. vivax and named Matryoshka RNA virus 1 (MaRNAV-1) to reflect its “Russian doll” nature a virus, infecting a parasite, infecting an animal. MaRNAV-1 was abundant in geographically diverse P. vivax from humans and mosquitoes. Notably, a related virus (MaRNAV-2) was identified in Australian birds infected with a Leucocytozoon - eukaryotic parasites that group with Plasmodium in the Apicomplexa subclass hematozoa. This is the first report of a Plasmodium virus. As well as broadening our understanding of the eukaryotic virosphere, the restriction to P. vivax may help understand P. vivax-specific biology in humans and mosquitoes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/diversity_begets_diversity_in_microbiomes_59216528/">
          <h4 class="font-weight-bold">Diversity begets diversity in microbiomes, bioRxiv, 2019-04-19</h4>
          <p>AbstractMicrobes are embedded in complex microbiomes where they engage in a wide array of inter- and intra-specific interactions1–4. However, whether these interactions are a significant driver of natural biodiversity is not well understood. Two contrasting hypotheses have been put forward to explain how species interactions could influence diversification. ‘Ecological Controls’ (EC) predicts a negative diversity-diversification relationship, where the evolution of novel types becomes constrained as available niches become filled5. In contrast, ‘Diversity Begets Diversity’ (DBD) predicts a positive relationship, with diversity promoting diversification via niche construction and other species interactions6. Using the Earth Microbiome Project, the largest standardized survey of global biodiversity to date7, we provide support for DBD as the dominant driver of microbiome diversity. Only in the most diverse microbiomes does DBD reach a plateau, consistent with increasingly saturated niche space. Genera that are strongly associated with a particular biome show a stronger DBD relationship than non-residents, consistent with prolonged evolutionary interactions driving diversification. Genera with larger genomes also experience a stronger DBD response, which could be due to a higher potential for metabolic interactions and niche construction offered by more diverse gene repertoires. Our results demonstrate that the rate at which microbiomes accumulate diversity is crucially dependent on resident diversity. This fits a scenario in which species interactions are important drivers of microbiome diversity. Further (population genomic or metagenomic) data are needed to elucidate the nature of these biotic interactions in order to more fully inform predictive models of biodiversity and ecosystem stability4,5.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/whole_genome_phylogenies_reflect_long-tailed_distributions_of_recombination_rates_in_many_bacterial_species_58699019/">
          <h4 class="font-weight-bold">Whole genome phylogenies reflect long-tailed distributions of recombination rates in many bacterial species, bioRxiv, 2019-04-08</h4>
          <p>AbstractAlthough homologous recombination is accepted to be common in bacteria, so far it has been challenging to accurately quantify its impact on genome evolution within bacterial species. We here introduce methods that use the statistics of single-nucleotide polymorphism (SNP) splits in the core genome alignment of a set of strains to show that, for many bacterial species, recombination dominates genome evolution. Each genomic locus has been overwritten so many times by recombination that it is impossible to reconstruct the clonal phylogeny and, instead of a consensus phylogeny, the phylogeny typically changes many thousands of times along the core genome alignment.We also show how SNP splits can be used to quantify the relative rates with which different subsets of strains have recombined in the past. We find that virtually every strain has a unique pattern of recombination frequencies with other strains and that the relative rates with which different subsets of strains share SNPs follow long-tailed distributions. Our findings show that bacterial populations are neither clonal nor freely recombining, but structured such that recombination rates between different lineages vary along a continuum spanning several orders of magnitude, with a unique pattern of rates for each lineage. Thus, rather than reflecting clonal ancestry, whole genome phylogenies reflect these long-tailed distributions of recombination rates.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/evolutionary-biology/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/evolutionary-biology/page/4/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="../../../../tags/evolutionary-biology/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/evolutionary-biology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/evolutionary-biology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/evolutionary-biology/page/4/" class="page-link">4</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/evolutionary-biology/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/evolutionary-biology/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/evolutionary-biology/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/evolutionary-biology/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/evolutionary-biology/page/9/" class="page-link">9</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/evolutionary-biology/page/6/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/evolutionary-biology/page/20/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
