<!DOCTYPE html>
<html><head>
  
  <title>Biophysics | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/nanoj-squirrel_quantitative_mapping_and_minimisation_of_super-resolution_optical_imaging_artefacts_21479543/">
          <h4 class="font-weight-bold">NanoJ-SQUIRREL quantitative mapping and minimisation of super-resolution optical imaging artefacts, bioRxiv, 2017-07-03</h4>
          <p>Most super-resolution microscopy methods depend on steps that contribute to the formation of image artefacts. Here we present NanoJ-SQUIRREL, an ImageJ-based analytical approach providing a quantitative assessment of super-resolution image quality. By comparing diffraction-limited images and super-resolution equivalents of the same focal volume, this approach generates a quantitative map of super-resolution defects, as well as methods for their correction. To illustrate its broad applicability to super-resolution approaches we apply our method to Localization Microscopy, STED and SIM images of a variety of in-cell structures including microtubules, poxviruses, neuronal actin rings and clathrin coated pits. We particularly focus on single-molecule localisation microscopy, where super-resolution reconstructions often feature imperfections not present in the original data. By showing the quantitative evolution of data quality over these varied sample preparation, acquisition and super-resolution methods we display the potential of NanoJ-SQUIRREL to guide optimization of superresolution imaging parameters.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/achieving_better_than_3_%C3%A5_resolution_by_single_particle_cryo-em_at_200_kev_20490363/">
          <h4 class="font-weight-bold">Achieving better than 3 Å resolution by single particle cryo-EM at 200 keV, bioRxiv, 2017-05-26</h4>
          <p>AbstractTechnical and methodological advances in single-particle cryo-electron microscopy (cryo-EM) have expanded the technique into a resolution regime that was previously only attainable by X-ray crystallography. Although single-particle cryo-EM has proven to be a useful technique for determining the structures of biomedically relevant molecules at near-atomic resolution, nearly 98% of the structures resolved to better than 4 Å resolution have been determined using 300 keV transmission electron microscopes (TEMs). We demonstrate that it is possible to obtain cryo-EM reconstructions of macromolecular complexes at a range of sizes to better than 3 Å resolution using a 200 keV TEM. These structures are of sufficient quality to unambiguously assign amino acid rotameric conformations and identify ordered water molecules, features previously thought only to be resolvable using TEMs operating at 300 keV.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/direct_visualization_of_transcriptional_activation_by_physical_enhancerpromoter_proximity_15397862/">
          <h4 class="font-weight-bold">Direct visualization of transcriptional activation by physical enhancer–promoter proximity, bioRxiv, 2017-01-12</h4>
          <p>A long-standing question in metazoan gene regulation is how remote enhancers communicate with their target promoters over long distances. Combining genome editing and quantitative live imaging we simultaneously visualize physical enhancer–promoter communication and transcription in Drosophila embryos. Enhancers regulating pair rule stripes of even-skipped expression activate transcription of a reporter gene over a distance of 150 kb. We show in individual cells that activation only occurs after the enhancer comes into close proximity with its regulatory target and that upon dissociation transcription ceases almost immediately. We further observe distinct topological conformations of the eve locus, depending on the spatial identity of the activating stripe enhancer. In addition, long-range activation results in transcriptional competition at the endogenous eve locus, causing corresponding developmental defects. Overall, we demonstrate that sustained physical proximity and enhancer–promoter engagement are required for enhancer action, and we provide a path to probe the implications of long-range regulation on cellular fates.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/cryo-em_structure_of_haemoglobin_at_3.2_%C3%A5_determined_with_the_volta_phase_plate_13652006/">
          <h4 class="font-weight-bold">Cryo-EM structure of haemoglobin at 3.2 Å determined with the Volta phase plate, bioRxiv, 2016-11-18</h4>
          <p>With the advent of direct electron detectors, the perspectives of cryo-electron microscopy (cryo-EM) have changed in a profound way1. These cameras are superior to previous detectors in coping with the intrinsically low contrast of radiation-sensitive organic materials embedded in amorphous ice, and so they have enabled the structure determination of several macromolecular assemblies to atomic or near-atomic resolution. According to one theoretical estimation, a few thousand images should suffice for calculating the structure of proteins as small as 17 kDa at 3 Å resolution2. In practice, however, we are still far away from this theoretical ideal. Thus far, protein complexes that have been successfully reconstructed to high-resolution by single particle analysis (SPA) have molecular weights of ~100 kDa or larger3. Here, we report the use of Volta phase plate in determining the structure of human haemoglobin (64 kDa) at 3.2 Å. Our results demonstrate that this method can be applied to complexes that are significantly smaller than those previously studied by conventional defocus-based approaches. Cryo-EM is now close to becoming a fast and cost-effective alternative to crystallography for high-resolution protein structure determination.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2016">2016</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/biophysics/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/biophysics/page/7/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biophysics/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biophysics/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biophysics/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biophysics/page/7/" class="page-link">7</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/biophysics/page/8/" class="page-link">8</a></li>
            
        

        

        
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
