<!DOCTYPE html>
<html><head>
  
  <title>Developmental Biology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/insect_wings_and_body_wall_evolved_from_ancient_leg_segments_31454380/">
          <h4 class="font-weight-bold">Insect wings and body wall evolved from ancient leg segments, bioRxiv, 2018-01-10</h4>
          <p>AbstractThe origin of insect wings has long been debated. Central to this debate is whether wings evolved from an epipod (outgrowth, e.g., a gill) on ancestral crustacean leg segments, or represent a novel outgrowth from the dorsal body wall that co-opted some of the genes used to pattern the epipods. To determine whether wings can be traced to ancestral, pre-insect structures, or arose by co-option, comparisons are necessary between insects and arthropods more representative of the ancestral state, where the hypothesized proximal leg region is not fused to the body wall. To do so, we examined the function of five leg patterning genes in the crustacean Parhyale hawaiensis and compared this to previous functional data from insects. By comparing gene knockout phenotypes of leg patterning genes in a crustacean with those of insects, we show that two ancestral crustacean leg segments were incorporated into the insect body, moving the leg’s epipod dorsally, up onto the back to form insect wings. Thus, our data shows that much of the body wall of insects, including the entire wing, is derived from these two ancestral proximal leg segments. This model explains all observations in favor of either the body wall origin or proximal leg origin of insect wings. Thus, our results show that insect wings are not novel structures, but instead evolved from existing, ancestral structures.One Sentence SummaryCRISPR-Cas9 knockout of leg gap genes in a crustacean reveals that insect wings are not novel structures, they evolved from crustacean leg segments</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/contact-dependent_cell-cell_communications_drive_morphological_invariance_during_ascidian_embryogenesis_30875487/">
          <h4 class="font-weight-bold">Contact-dependent cell-cell communications drive morphological invariance during ascidian embryogenesis, bioRxiv, 2017-12-25</h4>
          <p>ABSTRACTCanalization of developmental processes ensures the reproducibility and robustness of embryogenesis within each species. In its extreme form, found in ascidians, early embryonic cell lineages are invariant between embryos within and between species, despite rapid genomic divergence. To resolve this paradox, we used live light-sheet imaging to quantify individual cell behaviors in digitalized embryos and explore the forces that canalize their development. This quantitative approach revealed that individual cell geometries and cell contacts are strongly constrained, and that these constraints are tightly linked to the control of fate specification by local cell inductions. While in vertebrates ligand concentration usually controls cell inductions, we found that this role is fulfilled in ascidians by the area of contacts between signaling and responding cells. We propose that the duality between geometric and genetic control of inductions contributes to the counterintuitive inverse correlation between geometric and genetic variability during embryogenesis.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/high_throughput_single_cell_rna-seq_of_developing_mouse_kidney_and_human_kidney_organoids_reveals_a_roadmap_for_recreating_the_kidney_30555182/">
          <h4 class="font-weight-bold">High throughput single cell RNA-seq of developing mouse kidney and human kidney organoids reveals a roadmap for recreating the kidney, bioRxiv, 2017-12-17</h4>
          <p>AbstractRecent advances in our capacity to differentiate human pluripotent stem cells to human kidney tissue are moving the field closer to novel approaches for renal replacement. Such protocols have relied upon our current understanding of the molecular basis of mammalian kidney morphogenesis. To date this has depended upon population based-profiling of non-homogenous cellular compartments. In order to improve our resolution of individual cell transcriptional profiles during kidney morphogenesis, we have performed 10x Chromium single cell RNA-seq on over 6000 cells from the E18.5 developing mouse kidney, as well as more than 7000 cells from human iPSC-derived kidney organoids. We identified 16 clusters of cells representing all major cell lineages in the E18.5 mouse kidney. The differentially expressed genes from individual murine clusters were then used to guide the classification of 16 cell clusters within human kidney organoids, revealing the presence of distinguishable stromal, endothelial, nephron, podocyte and nephron progenitor populations. Despite the congruence between developing mouse and human organoid, our analysis suggested limited nephron maturation and the presence of ‘off target’ populations in human kidney organoids, including unidentified stromal populations and evidence of neural clusters. This may reflect unique human kidney populations, mixed cultures or aberrant differentiation in vitro. Analysis of clusters within the mouse data revealed novel insights into progenitor maintenance and cellular maturation in the major renal lineages and will serve as a roadmap to refine directed differentiation approaches in human iPSC-derived kidney organoids.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_revised_understanding_of_tribolium_morphogenesis_further_reconciles_short_and_long_germ_development_30343453/">
          <h4 class="font-weight-bold">A revised understanding of Tribolium morphogenesis further reconciles short and long germ development, bioRxiv, 2017-12-13</h4>
          <p>AbstractIn Drosophila melanogaster, the germband forms directly on the egg surface and solely consists of embryonic tissue. In contrast, most insect embryos undergo a complicated set of tissue rearrangements to generate a condensed, multi-layered germband. The ventral side of the germband is embryonic, while the dorsal side is thought to be an extraembryonic tissue called the amnion. While this tissue organisation has been accepted for decades, and has been widely reported in insects, its accuracy has not been directly tested in any species. Using live cell tracking and differential cell labelling in the short germ beetle Tribolium castaneum, I show that most of the cells previously thought to be amnion actually give rise to large parts of the embryo. This process occurs via the dorsal-to-ventral flow of cells and contributes to germband extension. In addition, I show that true ‘amnion’ cells in Tribolium originate from a small region of the blastoderm. Together, my findings show that development in the short germ embryos of Tribolium and the long germ embryos of Drosophila is more similar than previously proposed. Dorsal-to-ventral cell flow also occurs in Drosophila during germband extension, and I argue that the flow is driven by a conserved set of underlying morphogenetic events in both species. Furthermore, the revised Tribolium fatemap that I present is far more similar to that of Drosophila than the classic Tribolium fatemap. Lastly, my findings show that there is no qualitative difference between the tissue structure of the cellularised blastoderm and the shortintermediate germ germband. As such, the same tissue patterning mechanisms could function continuously throughout the cellularised blastoderm and germband stages, and easily shift between them over evolutionary time.Author summaryIn many animals, certain groups of cells in the embryo do not directly contribute to adult structures. Instead, these cells generate so-called ‘extra-embryonic tissues’ that support and facilitate development, but degenerate prior to birthhatching. In most insect species, embryos are described as having two major extra-embryonic tissues; the serosa, which encapsulates the entire embryo and yolk, and the amnion, which covers one side of the embryo. This tissue structure has been widely reported for over a century, but detailed studies on the amnion are lacking. Working in the beetle Tribolium castaneum, I used long-term fluorescent live imaging, cell tracking and differential cell labelling to investigate amnion development. In contrast to our current understanding, I show that most cells previously thought to be amnion actually form large parts of the embryo. In addition, I show how these cells ‘flow’ as a whole tissue and contribute to elongation of the embryo, and how only a relatively small number of cells form the actual amnion. Lastly, I describe how my findings show that despite exhibiting substantial differences in overall structure, embryos of Tribolium and the fruit fly, Drosophila melanogaster, utilise a conserved set of morphogenetic processes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/continuous_addition_of_progenitors_forms_the_cardiac_ventricle_in_zebrafish_30125771/">
          <h4 class="font-weight-bold">Continuous addition of progenitors forms the cardiac ventricle in zebrafish, bioRxiv, 2017-12-08</h4>
          <p>AbstractThe vertebrate heart develops from several progenitor lineages. After early-differentiating first heart field (FHF) progenitors form the linear heart tube, late-differentiating second heart field (SHF) progenitors extend atrium, ventricle, and form the inflow and outflow tracts (IFTOFT). However, the position and migration of late-differentiating progenitors during heart formation remains unclear. Here, we tracked zebrafish heart development using transgenics based on the cardiopharyngeal transcription factor gene tbx1. Live-imaging uncovered a tbx1 reporter-expressing cell sheath that from anterior lateral plate mesoderm continuously disseminates towards the forming heart tube. High-speed imaging and optogenetic lineage tracing corroborated that the zebrafish ventricle forms through continuous addition from the undifferentiated progenitor sheath followed by late-phase accrual of the bulbus arteriosus (BA). FGF inhibition during sheath migration reduced ventricle size and abolished BA formation, refining the window of FGF action during OFT formation. Our findings consolidate previous end-point analyses and establish zebrafish ventricle formation as a continuous process.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/epigraph_an_open-source_platform_to_quantify_epithelial_organization_28875557/">
          <h4 class="font-weight-bold">EpiGraph an open-source platform to quantify epithelial organization, bioRxiv, 2017-11-14</h4>
          <p>SUMMARYDuring development, cells must coordinate their differentiation with their growth and organization to form complex multicellular structures such as tissues and organs. Healthy tissues must maintain these structures during homeostasis. Epithelia are packed ensembles of cells from which the different tissues of the organism will originate during embryogenesis. A large barrier to the analysis of the morphogenetic changes in epithelia is the lack of simple tools that enable the quantification of cell arrangements. Here we present EpiGraph, an image analysis tool that quantifies epithelial organization. Our method combines computational geometry and graph theory to measure the degree of order of any packed tissue. EpiGraph goes beyond the traditional polygon distribution analysis, capturing other organizational traits that improve the characterization of epithelia. EpiGraph can objectively compare the rearrangements of epithelial cells during development and homeostasis to quantify how the global ensemble is affected. Importantly, it has been implemented in the open-access platform FIJI. This makes EpiGraph very user friendly, with no programming skills required.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/developmental-biology/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/developmental-biology/page/4/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="/audiences/tags/developmental-biology/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/developmental-biology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/developmental-biology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/developmental-biology/page/4/" class="page-link">4</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/developmental-biology/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/developmental-biology/page/6/" class="page-link">6</a></li>
            
        

        
        <li class="page-item"><a href="/audiences/tags/developmental-biology/page/6/" rel="next" class="page-link">Next ›</a></li>
        

        
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
