<!DOCTYPE html>
<html><head>
  
  <title>Biochemistry | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/structure_of_a_bacterial_atp_synthase_50887616/">
          <h4 class="font-weight-bold">Structure of a bacterial ATP synthase, bioRxiv, 2018-11-06</h4>
          <p>ATP synthases produce ATP from ADP and inorganic phosphate with energy from a transmembrane proton motive force. Bacterial ATP synthases have been studied extensively because they are the simplest form of the enzyme and because of the relative ease of genetic manipulation of these complexes. We expressed the Bacillus PS3 ATP synthase in Eschericia coli, purified it, and imaged it by cryo-EM, allowing us to build atomic models of the complex in three rotational states. The position of subunit e shows how it is able to inhibit ATP hydrolysis while allowing ATP synthesis. The architecture of the membrane region shows how the simple bacterial ATP synthase is able to perform the same core functions as the equivalent, but more complicated, mitochondrial complex. The structures reveal the path of transmembrane proton translocation and provide a model for understanding decades of biochemical analysis interrogating the roles of specific residues in the enzyme.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biochemistry">biochemistry</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/structure_of_transcribing_rna_polymerase_ii-nucleosome_complex_49346096/">
          <h4 class="font-weight-bold">Structure of transcribing RNA polymerase II-nucleosome complex, bioRxiv, 2018-10-07</h4>
          <p>Transcription of eukaryotic protein-coding genes requires passage of RNA polymerase II (Pol II) through chromatin. Pol II passage is impaired by nucleosomes and requires elongation factors that help Pol II to efficiently overcome the nucleosomal barrier1-4. How the Pol II machinery transcribes through a nucleosome remains unclear because structural studies have been limited to Pol II elongation complexes formed on DNA templates lacking nucleosomes5. Here we report the cryo-electron microscopy (cryo-EM) structure of transcribing Pol II from the yeast Saccharomyces cerevisiae engaged with a downstream nucleosome core particle (NCP) at an overall resolution of 4.4 Å with resolutions ranging from 4-6 Å in Pol II and 6-8 Å in the NCP. Pol II and the NCP adopt a defined orientation that could not be predicted from modelling. Pol II contacts DNA of the incoming NCP on both sides of the nucleosomal dyad with its domains ‘clamp head’ and ‘lobe’. Comparison of the Pol II-NCP structure to known structures of Pol II complexes reveals that the elongation factors TFIIS, DSIF, NELF, PAF1 complex, and SPT6 can be accommodated on the Pol II surface in the presence of the oriented nucleosome. Further structural comparisons show that the chromatin remodelling enzyme Chd1, which is also required for efficient Pol II passage6,7, could bind the oriented nucleosome with its motor domain. The DNA-binding region of Chd1 must however be released from DNA when Pol II approaches the nucleosome, and based on published data8,9 this is predicted to stimulate Chd1 activity and to facilitate Pol II passage. Our results provide a starting point for a mechanistic analysis of chromatin transcription.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biochemistry">biochemistry</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/active_and_repressed_chromatin_domains_exhibit_distinct_nucleosome_segregation_during_dna_replication_48460139/">
          <h4 class="font-weight-bold">Active and repressed chromatin domains exhibit distinct nucleosome segregation during DNA replication, bioRxiv, 2018-09-16</h4>
          <p>SummaryChromatin domains and their associated structures must be faithfully inherited through cellular division to maintain cellular identity. Yet, accessing the localized strategies preserving chromatin domain inheritance, specifically the transfer of parental, pre-existing nucleosomes with their associated post-translational modifications (PTMs) during DNA replication is challenging in living cells. We devised an inducible, proximity-dependent labeling system to irreversibly mark replication-dependent H3.1 and H3.2 histone-containing nucleosomes at single desired loci in mouse embryonic stem cells such that their fate after DNA replication could be followed. Strikingly, repressed chromatin domains are preserved through the local re-deposition of parental nucleosomes. In contrast, nucleosomes decorating active chromatin domains do not exhibit such preservation. Notably, altering cell fate leads to an adjustment in the positional inheritance of parental nucleosomes that reflects the corresponding changes in chromatin structure. These findings point to important mechanisms that contribute to parental nucleosome segregation to preserve cellular identity.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biochemistry">biochemistry</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_chemical_biology_view_of_bioactive_small_molecules_and_a_binder-based_approach_to_connect_biology_to_precision_medicines_46229676/">
          <h4 class="font-weight-bold">A chemical biology view of bioactive small molecules and a binder-based approach to connect biology to precision medicines, bioRxiv, 2018-08-07</h4>
          <p></p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biochemistry">biochemistry</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/homology_directed_repair_by_cas9donor_co-localization_in_mammalian_cells_31753504/">
          <h4 class="font-weight-bold">Homology Directed Repair by Cas9Donor Co-localization in Mammalian Cells, bioRxiv, 2018-01-17</h4>
          <p>AbstractHomology directed repair (HDR) induced by site specific DNA double strand breaks (DSB) with CRISPRCas9 is a precision gene editing approach that occurs at low frequency in comparison to indel forming non homologous end joining (NHEJ). In order to obtain high HDR percentages in mammalian cells, we engineered Cas9 protein fused to a high-affinity monoavidin domain to deliver biotinylated donor DNA to a DSB site. In addition, we used the cationic polymer, polyethylenimine, to deliver Cas9 RNP-donor DNA complex into the cell. Combining these strategies improved HDR percentages of up to 90% in three tested loci (CXCR4, EMX1, and TLR) in standard HEK293 cells. Our approach offers a cost effective, simple and broadly applicable gene editing method, thereby expanding the CRISPRCas9 genome editing toolbox.SummaryPrecision gene editing occurs at a low percentage in mammalian cells using Cas9. Colocalization of donor with Cas9MAV and PEI delivery raises HDR occurrence.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biochemistry">biochemistry</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/classification_of_single_particles_from_human_cell_extract_reveals_distinct_structures_31667604/">
          <h4 class="font-weight-bold">Classification of Single Particles from Human Cell Extract Reveals Distinct Structures, bioRxiv, 2018-01-15</h4>
          <p>SummaryMulti-protein complexes are necessary for nearly all cellular processes, and understanding their structure is required for elucidating their function. Current high-resolution strategies in structural biology are effective, but lag behind other fields (e.g. genomics and proteomics) due to their reliance on purified samples rather than characterizing heterogeneous mixtures. Here, we present a method combining single particle analysis by electron microscopy with protein identification by mass spectrometry to structurally characterize macromolecular complexes from extracts of human cells. We obtain three-dimensional structures of native proteasomes directly from ab initio classification of a heterogeneous mixture of protein complexes. In addition, we find an ~1 MDa size structure of unknown composition and reference our proteomics data to suggest possible identities. Our study shows the power of using a shotgun approach to electron microscopy (shotgun EM) when coupled with mass spectrometry as a tool to uncover the structures of macromolecular machines in parallel.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biochemistry">biochemistry</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biochemistry/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biochemistry/" class="page-link">1</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/biochemistry/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biochemistry/page/3/" class="page-link">3</a></li>
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/biochemistry/page/3/" rel="next" class="page-link">Next ›</a></li>
        

        
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
