<!DOCTYPE html>
<html><head>
  
  <title>Immunology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/human_immune_system_variation_during_one_year_74997012/">
          <h4 class="font-weight-bold">Human immune system variation during one year, bioRxiv, 2020-01-23</h4>
          <p>SUMMARYThe human immune system varies extensively between individuals, but variation within individuals over time has not been well characterized. Systems-level analyses allow for simultaneous quantification of many interacting immune system components, and the inference of global regulatory principles. Here we present a longitudinal, systems-level analysis in 99 healthy adults, 50 to 65 years of age and sampled every 3rd month during one year. We describe the structure of inter-individual variation and characterize extreme phenotypes along a principal curve. From coordinated measurement fluctuations, we infer relationships between 115 immune cell populations and 750 plasma proteins constituting the blood immune system. While most individuals have stable immune systems, the degree of longitudinal variability is an individual feature. The most variable individuals, in the absence of overt infections, exhibited markers of poor metabolic health suggestive of a functional link between metabolic and immunologic homeostatic regulation.HIGHLIGHTSLongitudinal variation in immune cell composition during one yearInter-individual variation can be described along a principal curveImmune cell and protein relationships are inferredVariability over time is an individual feature correlating with markers of poor metabolic health</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/multiplexed_single-cell_metabolic_profiles_organize_the_spectrum_of_cytotoxic_human_t_cells_74169828/">
          <h4 class="font-weight-bold">Multiplexed Single-cell Metabolic Profiles Organize the Spectrum of Cytotoxic Human T Cells, bioRxiv, 2020-01-18</h4>
          <p>SummaryCellular metabolism regulates immune cell activation, differentiation and effector functions to the extent that its perturbation can augment immune responses. However, the analytical technologies available to study cellular metabolism lack single-cell resolution, obscuring metabolic heterogeneity and its connection to immune phenotype and function. To that end, we utilized high-dimensional, antibody-based technologies to simultaneously quantify the single-cell metabolic regulome in combination with phenotypic identity. Mass cytometry (CyTOF)-based application of this approach to early human T cell activation enabled the comprehensive reconstruction of the coordinated metabolic remodeling of naïve CD8&#43; T cells and aligned with conventional bulk assays for glycolysis and oxidative phosphorylation. Extending this analysis to a variety of tissue-resident immune cells revealed tissue-restricted metabolic states of human cytotoxic T cells, including metabolically repressed subsets that expressed CD39 and PD1 and that were enriched in colorectal carcinoma versus healthy adjacent tissue. Finally, combining this approach with multiplexed ion beam imaging by time-of-flight (MIBI-TOF) demonstrated the existence of spatially enriched metabolic neighborhoods, independent of cell identity and additionally revealed exclusion of metabolically repressed cytotoxic T cell states from the tumor-immune boundary in human colorectal carcinoma. Overall, we provide an approach that permits the robust approximation of metabolic states in individual cells along with multimodal analysis of cell identity and functional characteristics that can be applied to human clinical samples to study cellular metabolism how it may be perturbed to affect immunological outcomes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/perturbation_of_effector_and_regulatory_t_cell_subsets_in_myalgic_encephalomyelitischronic_fatigue_syndrome_mecfs_73284635/">
          <h4 class="font-weight-bold">Perturbation of effector and regulatory T cell subsets in Myalgic EncephalomyelitisChronic Fatigue Syndrome (MECFS), bioRxiv, 2019-12-26</h4>
          <p>AbstractMyalgic encephalomyelitischronic fatigue syndrome (MECFS) is a debilitating disorder of unknown etiology, and diagnosis of the disease is largely based on clinical symptoms. We hypothesized that immunological disruption is the major driver of this disease and analyzed a large cohort of MECFS patient or control blood samples for differences in T cell subset frequencies and functions. We found that the ratio of CD4&#43; to CD8&#43; T cells and the proportion of CD8&#43; effector memory T cells were increased, whereas NK cells were reduced in MECFS patients younger than 50 years old compared to a healthy control group. Remarkably, major differences were observed in Th1, Th2, Th17 and mucosal-associated invariant T (MAIT) T cell subset functions across all ages of patients compared to healthy subjects. While CCR6&#43; Th17 cells in MECFS secreted less IL-17 compared to controls, their overall frequency was higher. Similarly, MAIT cells from patients secreted lower IFNγ, GranzymeA and IL-17 upon activation. Together, these findings suggest chronic stimulation of these T cell populations in MECFS patients. In contrast, the frequency of regulatory T cells (Tregs), which control excessive immune activation, was higher in MECFS patients. Finally, using a machine learning algorithm called random forest, we determined that the set of T cell parameters analyzed could identify more than 90% of the subjects in the MECFS cohort as patients (93% true positive rate or sensitivity). In conclusion, these multiple and major perturbations or dysfunctions in T cell subsets in MECFS patients suggest potential chronic infections or microbiome dysbiosis. These findings also have implications for development of MECFS specific immune biomarkers and reveal potential targets for novel therapeutic interventions.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_hematopoietic_landscape_at_single-cell_resolution_reveals_unexpected_stem_cell_features_in_naked_mole-rats_71399778/">
          <h4 class="font-weight-bold">The hematopoietic landscape at single-cell resolution reveals unexpected stem cell features in naked mole-rats, bioRxiv, 2019-12-02</h4>
          <p>SUMMARYNaked mole-rats are the longest-lived rodents endowed with resistance to cancer and age-related diseases, yet their stem cell characteristics remain enigmatic. We profiled the naked mole-rat hematopoietic system down to single-cell resolution, and identified several unique features likely contributing to longevity. In adult naked mole-rats red blood cells are formed in spleen and marrow, a neotenic feature beneficial for hypoxic environments and to prevent anemia. Platelet numbers are lower compared to short-lived mice, which may preclude age-related platelet increase and thrombosis. T cells mature in thymus and lymph nodes, providing a supply of T cells after age-related thymus involution. The pool of quiescent stem cells is higher than in mice, and HSCs overexpress an oxidative phosphorylation signature, revealing a new paradigm of stem cell metabolism to benefit longevity and oppose oncogenesis. Our work provides a platform to study immunology and stem cell biology in an animal model of healthy aging.HIGHLIGHTS&lt;jatslist list-type=bullet&gt;&lt;jatslist-item&gt;Flow cytometry labelling panel to purify viable naked mole-rat HSPCs&lt;jatslist-item&gt;&lt;jatslist-item&gt;The spleen as the major site of erythropoiesis in the naked mole-rat&lt;jatslist-item&gt;&lt;jatslist-item&gt;Naked mole-rats show extrathymic T-cell development under homeostatic conditions&lt;jatslist-item&gt;&lt;jatslist-item&gt;Naked mole-rat hematopoietic stem cells (HSCs) have high OXPHOS activity&lt;jatslist-item&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/antibody_against_envelope_protein_from_human_endogenous_retrovirus_activates_neutrophils_in_systemic_lupus_erythematosus_66923674/">
          <h4 class="font-weight-bold">Antibody against envelope protein from human endogenous retrovirus activates neutrophils in systemic lupus erythematosus, bioRxiv, 2019-09-21</h4>
          <p>AbstractNeutrophil activation and the formation of neutrophil extracellular trap (NET) are hallmarks of innate immune activation in systemic lupus erythematosus (SLE) and contribute to the systemic interferon signature. Here we report that the expression of an endogenous retrovirus (ERV) locus ERV-K102, encoding an envelope protein, was significantly elevated in SLE patient blood and was correlated with higher interferon status. Induction of ERV-K102 expression most strongly correlated with reduced transcript levels of epigenetic silencing factors. SLE IgG promoted phagocytosis of ERV-K102 envelope protein by neutrophils through immune complex formation. ERV immune complex phagocytosis resulted in subsequent NET formation consisting of DNA, neutrophil elastase, and citrullinated histone H3. Finally, analysis of anti-ERV-K102 IgG in SLE patients showed that IgG2 likely mediates this effect. Together, we identified an immunostimulatory ERV-K envelope protein elevated in SLE that may be a target of SLE IgG and able to promote neutrophil activation.eTOC summaryUsing ERVmap, the authors determined that the expression of ERV-K102 locus was elevated in SLE patient blood and correlated with the interferon signature. The envelope protein encoded by this locus activates human neutrophils through immune complex formation with SLE IgG.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_compact_vocabulary_of_paratope-epitope_interactions_enables_predictability_of_antibody-antigen_binding_66345444/">
          <h4 class="font-weight-bold">A compact vocabulary of paratope-epitope interactions enables predictability of antibody-antigen binding, bioRxiv, 2019-09-10</h4>
          <p>Antibody-antigen binding relies on the specific interaction of amino acids at the paratope-epitope interface. It has been a long-standing question whether antibody-antigen binding is predictable. A fundamental premise for the predictability of paratope-epitope interactions is the existence of structural units that are universally shared among antibody-antigen binding complexes. Here, we screened the largest available set of non-redundant antibody-antigen structures for binding patterns and identified structural interaction motifs, which together compose a vocabulary of paratope-epitope interactions that is universally shared among investigated antibody-antigen structures. The vocabulary (i) is compact, less than 104 motifs, (ii) is immunity-specific and distinct from non-immune protein-protein interactions, (iii) mediates specific oligo- and polyreactive interactions between paratope-epitope pairs, and (iv) enables the machine learnability of paratope-epitope interactions. Collectively, our results demonstrate the predictability of antibody-antigen binding.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        

        
             
            <li class="page-item active"><a href="../../tags/immunology/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="../../tags/immunology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="../../tags/immunology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="../../tags/immunology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="../../tags/immunology/page/5/" class="page-link">5</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        

        
        <li class="page-item"><a href="../../tags/immunology/page/2/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../tags/immunology/page/6/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
