<!DOCTYPE html>
<html><head>
  
  <title>2019 | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/phase_separation_provides_a_mechanism_to_reduce_noise_in_cells_supplementary_text_54293656/">
          <h4 class="font-weight-bold">Phase separation provides a mechanism to reduce noise in cells Supplementary text, bioRxiv, 2019-01-23</h4>
          <p>A central problem in cellular control is how cells cope with the inherent noise in gene expression. Although transcriptional and posttranscriptional feedback mechanisms can suppress noise, they are often slow, and cannot explain how cells buffer acute fluctuations. Here, by using a physical model that links fluctuations in protein concentration to the theory of phase separation, we show that liquid droplets can act as fast and effective buffers for gene expression noise. We confirm our theory experimentally using an engineered phase separating protein that forms liquid-like compartments in mammalian cells. These data suggest a novel role of phase separation in biological information processing.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/the_malaria_cell_atlas_a_comprehensive_reference_of_single_parasite_transcriptomes_across_the_complete_plasmodium_life_cycle_file_s1_54299376/">
          <h4 class="font-weight-bold">The Malaria Cell Atlas a comprehensive reference of single parasite transcriptomes across the complete Plasmodium life cycle file S1, bioRxiv, 2019-01-23</h4>
          <p>Malaria parasites adopt a remarkable variety of morphological life stages as they transition through multiple mammalian host and mosquito vector environments. Here we profile the single-cell transcriptomes of thousands of individual parasites, deriving the first high-resolution transcriptional atlas of the entire Plasmodium berghei life cycle. We then use our atlas to precisely define developmental stages of single cells from three different human malaria parasite species, including parasites isolated directly from infected individuals. The Malaria Cell Atlas provides both a comprehensive view of gene usage in a complex eukaryotic parasite and an open access reference data set for the study of malaria parasites.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/accurate_detection_of_m6a_rna_modifications_in_native_rna_sequences_54225106/">
          <h4 class="font-weight-bold">Accurate detection of m6A RNA modifications in native RNA sequences, bioRxiv, 2019-01-21</h4>
          <p>The field of epitranscriptomics has undergone an enormous expansion in the last few years; however, a major limitation is the lack of generic methods to map RNA modifications transcriptome-wide. Here we show that using Oxford Nanopore Technologies, N6-methyladenosine (m6A) RNA modifications can be detected with high accuracy, in the form of systematic errors and decreased base-calling qualities. Our results open new avenues to investigate the universe of RNA modifications with single nucleotide resolution, in individual RNA molecules.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/direct_estimation_of_mutations_in_great_apes_reconciles_phylogenetic_dating_54236027/">
          <h4 class="font-weight-bold">Direct estimation of mutations in great apes reconciles phylogenetic dating, Nature Ecology &amp; Evolution, 2019-01-21</h4>
          <p>−9</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/nature%20ecology%20&amp;%20evolution">nature ecology &amp; evolution</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/pericentromeric_heterochromatin_is_hierarchically_organized_and_spatially_contacts_h3k9me23_islands_located_in_euchromatic_genome_54225676/">
          <h4 class="font-weight-bold">Pericentromeric heterochromatin is hierarchically organized and spatially contacts H3K9me23 islands located in euchromatic genome, bioRxiv, 2019-01-21</h4>
          <p>Membraneless pericentromeric heterochromatin (PCH) domains play vital roles in chromosome dynamics and genome stability. However, our current understanding of 3D genome organization does not include PCH domains because of technical challenges associated with repetitive sequences enriched in PCH genomic regions. We investigated the 3D architecture of Drosophila melanogaster PCH domains and their spatial associations with euchromatic genome by developing a novel analysis method that incorporates genome-wide Hi-C reads originating from PCH DNA. Combined with cytogenetic analysis, we reveal a hierarchical organization of the PCH domains into distinct &#39;territories,&#39; in which &#39;intra-arm&#39; interactions are the most prevalent, followed by 3D contacts between specific PCH regions on different chromosomes. Strikingly, H3K9me23-enriched regions embedded in euchromatic genome show prevalent 3D interactions with the PCH domain. These spatial contacts require H3K9me23 enrichment, are likely mediated by liquid-liquid phase separation, and influence organismal fitness. Our findings have important implications for how PCH architecture influences the function and evolution of both repetitive heterochromatin and the gene-rich euchromatin.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/prospective_brain-wide_labeling_of_neuronal_subclasses_with_enhancer-driven_aavs_54201876/">
          <h4 class="font-weight-bold">Prospective, brain-wide labeling of neuronal subclasses with enhancer-driven AAVs, bioRxiv, 2019-01-21</h4>
          <p>Labeling and perturbation of specific cell types in multicellular systems has transformed our ability to understand them. The rapid pace of cell type identification by new single-cell analysis methods has not been met with efficient access to these newly discovered types. To enable access to specific neural populations in the mouse cortex, we have collected single cell chromatin accessibility data from select cell types. We clustered the single cell data and mapped them to single cell transcriptomics to identify highly specific enhancers for cell subclasses. These enhancers, when cloned into AAVs and delivered to the brain by retro orbital injections, transgene expression in specific cell subclasses throughout the mouse brain. This approach will enable functional investigation of cell types in the mouse cortex and beyond.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/2019/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/2019/page/77/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/74/" class="page-link">74</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/75/" class="page-link">75</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/76/" class="page-link">76</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/77/" class="page-link">77</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/2019/page/78/" class="page-link">78</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/79/" class="page-link">79</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/80/" class="page-link">80</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/81/" class="page-link">81</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/2019/page/82/" class="page-link">82</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/2019/page/79/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/2019/page/87/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
