<!DOCTYPE html>
<html><head>
  
  <title>Biorxiv | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/resolving_cell_cycle_speed_in_one_snapshot_with_a_live-cell_fluorescent_reporter_52553188/">
          <h4 class="font-weight-bold">Resolving cell cycle speed in one snapshot with a live-cell fluorescent reporter, bioRxiv, 2018-12-12</h4>
          <p>SummaryCell proliferation changes concomitantly with fate transitions during reprogramming, differentiation, regeneration, and oncogenesis. Methods to resolve cell cycle length heterogeneity in real-time are currently lacking. Here, we describe a genetically encoded fluorescent reporter that captures live cell cycle speed using a single measurement. This reporter is based on the color-changing Fluorescent Timer (FT) protein, which emits blue fluorescence when newly synthesized before maturing into a red fluorescent protein. We generated a mouse strain expressing an H2B-FT fusion reporter from a universally active locus, and demonstrate that faster-cycling cells can be distinguished from slower-cycling ones based on the intracellular fluorescence ratio between the FT’s blue and red states. Using this reporter, we reveal the native cell cycle speed distributions of fresh hematopoietic cells, and demonstrate its utility in analyzing cell proliferation in solid tissues. This system is broadly applicable for dissecting functional heterogeneity associated with cell cycle dynamics in complex tissues.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/cryptic_and_extensive_hybridization_between_ancient_lineages_of_american_crows_52485986/">
          <h4 class="font-weight-bold">Cryptic and extensive hybridization between ancient lineages of American crows, bioRxiv, 2018-12-11</h4>
          <p>AbstractMost species and therefore most hybrid zones have historically been described using phenotypic characters. However, both speciation and hybridization can occur with negligible morphological differentiation. The Northwestern Crow (Corvus caurinus) and American Crow (Corvus brachyrhynchos) are sister taxonomic species with a continuous distribution that lack reliable traditional characters for identification. In this first population genomic study of Northwestern and American crows, we use genomic SNPs (nuDNA) and mtDNA to investigate whether these crows are genetically differentiated and the extent to which they may hybridize. We found that American and Northwestern crows have distinct evolutionary histories, supported by two nuDNA ancestry clusters and two 1.1%-divergent mtDNA clades dating to the late Pleistocene, when glacial advances may have isolated crow populations in separate refugia. We document extensive hybridization, with geographic overlap of mtDNA clades and admixture of nuDNA across &amp;gt;1,400 km of western Washington and western British Columbia. This broad hybrid zone consists of late-generation hybrids and backcrosses, not recent (e.g., F1) hybrids. Nuclear DNA and mtDNA clines were both centered in southwestern British Columbia, farther north than previously postulated. The mtDNA cline was narrower than the nuDNA cline, consistent with Haldane’s rule but not sex-biased dispersal. Overall, our results suggest a history of reticulate evolution in American and Northwestern crows, consistent with potentially recurring neutral expansion(s) from Pleistocene glacial refugia followed by lineage fusion(s). However, we do not rule out a contributing role for more recent potential drivers of hybridization, such as expansion into human-modified habitats.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genetic_determination_of_stomatal_patterning_in_winter_wheat_triticum_aestivum_l._52515038/">
          <h4 class="font-weight-bold">Genetic determination of stomatal patterning in winter wheat (Triticum aestivum L.), bioRxiv, 2018-12-11</h4>
          <p>Leaf stomata are microscopic pores mediating plant-environment interactions. Their role in carbon uptake and transpiration make them prime candidates for improving water use efficiency (WUE). Stomatal density (SD), the number of stomata per unit area, has been shown to be negatively correlated with WUE. However, little is known about the genetic basis of SD in wheat (Triticum aestivum L.), and to what extant genetic variation exists in contemporary wheat germplasm. Here, we evaluated stomatal patterning over two growing seasons in a set of 333 wheat lines, representing the European winter wheat germplasm. Stomatal patterning was mainly determined by two underlying traits, the distance between files of stomata and the distance between stomata within a file. By haplotype association mapping, quantitative trait loci for SD were consistently detected in both seasons on wheat chromosomes (CHR) 2A, 3A and 7B. The single nucleotide polymorphism markers most significantly associated with SD coincided with the genes INDUCER OF CBF EXPRESSION 1 (ICE1) and STOMATAL CYTOKINESIS-DEFECTIVE 1 (SCD1) on CHR 3A, and genes involved in ethylene and auxin signaling on CHR 2A and 7B, respectively. Our study unlocks the phenotypic and genotypic variation for stomatal patterning traits in contemporary wheat germplasm. It provides gene targets for functional validation and practical tools to manipulate SD using marker-assisted selection for crop improvement.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/real-time_capture_of_horizontal_gene_transfers_from_gut_microbiota_by_engineered_crispr-cas_acquisition_52487723/">
          <h4 class="font-weight-bold">Real-time capture of horizontal gene transfers from gut microbiota by engineered CRISPR-Cas acquisition, bioRxiv, 2018-12-11</h4>
          <p>AbstractHorizontal gene transfer (HGT) is central to the adaptation and evolution of bacteria. However, our knowledge about the flow of genetic material within complex microbiomes is lacking; most studies of HGT rely on bioinformatic analyses of genetic elements maintained on evolutionary timescales or experimental measurements of phenotypically trackable markers (e.g. antibiotic resistance). Consequently, our knowledge of the capacity and dynamics of HGT in complex communities is limited. Here, we utilize the CRISPR-Cas spacer acquisition process to detect HGT events from complex microbiota in real-time and at nucleotide resolution. In this system, a recording strain is exposed to a microbial sample, spacers are acquired from foreign transferred elements and permanently stored in genomic CRISPR arrays. Subsequently, sequencing and analysis of these spacers enables identification of the transferred elements. This approach allowed us to quantify transfer frequencies of individual mobile elements without the need for phenotypic markers or post-transfer replication. We show that HGT in human clinical fecal samples can be extensive and rapid, often involving multiple different plasmid types, with the IncX type being the most actively transferred. Importantly, the vast majority of transferred elements did not carry readily selectable phenotypic markers, highlighting the utility of our approach to reveal previously hidden real-time dynamics of mobile gene pools within complex microbiomes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/structure_of_a_cytochrome-based_bacterial_nanowire_52488682/">
          <h4 class="font-weight-bold">Structure of a cytochrome-based bacterial nanowire, bioRxiv, 2018-12-11</h4>
          <p>AbstractElectrically conductive pili from Geobacter species, termed bacterial “nanowires”, are intensely studied for their biological significance and potential in the development of new materials. We have characterized a unique nanowire from conductive G. sulfurreducens pili preparations by cryo-electron microscopy composed solely of the c-type cytochrome OmcS. We present here, at 3.4 Å resolution, a novel structure of a cytochrome-based filament and discuss its possible role in long-range biological electron transport.Summary sentenceCryo-electron microscopy reveals the remarkable assembly of a c-type cytochrome into filaments comprising a heme-based bacterial nanowire.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/using_dna_from_mothers_and_children_to_study_parental_investment_in_childrens_educational_attainment_52436750/">
          <h4 class="font-weight-bold">Using DNA from mothers and children to study parental investment in children&#39;s educational attainment, bioRxiv, 2018-12-10</h4>
          <p>This study tested implications of new genetic discoveries for understanding the association between parental investment and children&#39;s educational attainment. A novel design matched genetic data from 860 British mothers and their children with home-visit measures of parenting the E-Risk Study. Three findings emerged. First, both mothers&#39; and children&#39;s education-associated genetics, summarized in a genome-wide polygenic score, predicted parenting -- a gene-environment correlation. Second, accounting for genetic influences slightly reduced associations between parenting and children&#39;s attainment -- indicating some genetic confounding. Third, mothers&#39; genetics influenced children&#39;s attainment over and above genetic mother-to-child transmission, via cognitively-stimulating parenting -- an environmentally-mediated effect. Findings imply that, when interpreting parents&#39; effects on children, environmentalists must consider genetic transmission, but geneticists must also consider environmental transmission.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/biorxiv/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/biorxiv/page/145/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biorxiv/page/142/" class="page-link">142</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biorxiv/page/143/" class="page-link">143</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biorxiv/page/144/" class="page-link">144</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biorxiv/page/145/" class="page-link">145</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/biorxiv/page/146/" class="page-link">146</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biorxiv/page/147/" class="page-link">147</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biorxiv/page/148/" class="page-link">148</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biorxiv/page/149/" class="page-link">149</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/biorxiv/page/150/" class="page-link">150</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/biorxiv/page/147/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/biorxiv/page/312/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
