<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/principles_of_meiotic_chromosome_assembly_49623064/">
          <h4 class="font-weight-bold">Principles of Meiotic Chromosome Assembly, bioRxiv, 2018-10-13</h4>
          <p>During meiotic prophase, chromosomes organise into a series of chromatin loops emanating from a proteinaceous axis, but the mechanisms of assembly remain unclear. Here we elucidate how this elaborate three-dimensional chromosome organisation is underpinned by genomic sequence in Saccharomyces cerevisiae. Entering meiosis, strong cohesin-dependent grid-like Hi-C interaction patterns emerge, reminiscent of mammalian interphase organisation, but with distinct regulation. Meiotic patterns agree with simulations of loop extrusion limited by barriers, yet are patterned by convergent transcription rather than binding of the mammalian interphase factor, CTCF, which is absent in S. cerevisiae - thereby both challenging and extending current paradigms of local chromosome organisation. While grid-like interactions emerge independently of meiotic chromosome synapsis, synapsis itself generates additional compaction that matures differentially according to telomere proximity and chromosome size. Collectively, our results elucidate fundamental principles of chromosome assembly and demonstrate the essential role of cohesin within this evolutionarily conserved process.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_flexiscope_a_low_cost_flexible_convertible_and_modular_microscope_with_automated_scanning_and_micromanipulation_49607835/">
          <h4 class="font-weight-bold">The Flexiscope a Low Cost, Flexible, Convertible, and Modular Microscope with Automated Scanning and Micromanipulation, bioRxiv, 2018-10-13</h4>
          <p>AbstractWith technologies rapidly evolving, many research institutions are now opting to invest in costly, high-quality, specialised microscopes which are shared by many researchers. As a consequence, the user does not have the ability to adapt a microscope to their specific needs and limitations in experimental design are introduced. A flexible work-horse microscopy system is a valuable tool in any laboratory to meet the diverse needs of a research team and promote innovation in experimental design. We have developed the Flexiscope; a multi-functional, adaptable, efficient and high performance microscopyelectrophysiology system for everyday applications in a neurobiology laboratory. The core optical components are relatively constant in the three configurations described here; an upright configuration, an inverted configuration and an uprightelectrophysiology configuration. We have provided a comprehensive description of the Flexiscope. We show that this method is capable of oblique infrared illumination imaging, multi-channel fluorescent imaging, and automated 3D scanning of larger specimens. Image quality is conserved across the three configurations of the microscope, and conversion between configurations is possible quickly and easily, while the motion control system can be repurposed to allow sub-micron computer-controlled micromanipulation. The Flexiscope provides similar performance and usability to commercially available systems. However, as it can be easily reconfigured for multiple roles, it can remove the need to purchase multiple microscopes, giving significant cost savings. The modular re-configurable nature allows the user to customise the system to their specific needs and adaptupgrade the system as challenges arise.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_multi-state_birth-death_model_for_bayesian_inference_of_lineage-specific_birth_and_death_rates_49532735/">
          <h4 class="font-weight-bold">A Multi-State Birth-Death model for Bayesian inference of lineage-specific birth and death rates, bioRxiv, 2018-10-11</h4>
          <p>AbstractHeterogeneous populations can lead to important differences in birth and death rates across a phylogeny Taking this heterogeneity into account is thus critical to obtain accurate estimates of the underlying population dynamics. We present a new multi-state birth-death model (MSBD) that can estimate lineage-specific birth and death rates. For species phylogenies, this corresponds to estimating lineage-dependent speciation and extinction rates. Contrary to existing models, we do not require a prior hypothesis on a trait driving the rate differences and we allow the same rates to be present in different parts of the phylogeny. Using simulated datasets, we show that the MSBD model can reliably infer the presence of multiple evolutionary regimes, their positions in the tree, and the birth and death rates associated with each. We also present a re-analysis of two empirical datasets and compare the results obtained by MSBD and by the existing software BAMM. The MSBD model is implemented as a package in the Bayesian inference software BEAST2, which allows joint inference of the phylogeny and the model parameters.Significance statementPhylogenetic trees can inform about the underlying speciation and extinction processes within a species clade. Many different factors, for instance environmental changes or morphological changes, can lead to differences in macroevolutionary dynamics within a clade. We present here a new multi-state birth-death (MSBD) model that can detect these differences and estimate both the position of changes in the tree and the associated macroevolutionary parameters. The MSBD model does not require a prior hypothesis on which trait is driving the changes in dynamics and is thus applicable to a wide range of datasets. It is implemented as an extension to the existing framework BEAST2.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/bayesian_estimation_of_species_divergence_times_using_correlated_quantitative_characters_49533279/">
          <h4 class="font-weight-bold">Bayesian Estimation of Species Divergence Times Using Correlated Quantitative Characters, bioRxiv, 2018-10-11</h4>
          <p>Discrete morphological data have been widely used to study species evolution, but the use of quantitative (or continuous) morphological characters is less common. Here, we implement a Bayesian method to estimate species divergence times using quantitative characters. Quantitative character evolution is modelled using Brownian diffusion with character correlation and character variation within populations. Through simulations, we demonstrate that ignoring the population variation (or population noise) and the correlation among characters leads to biased estimates of divergence times and rate, especially if the correlation and population noise are high. We apply our new method to the analysis of quantitative characters (cranium landmarks) and molecular data from carnivoran mammals. Our results show that time estimates are affected by whether the correlations and population noise are accounted for or ignored in the analysis. The estimates are also affected by the type of data analysed, with analyses of morphological characters only, molecular data only, or a combination of both; showing noticeable differences among the time estimates. Rate variation of morphological characters among the carnivoran species appears to be very high, with Bayesian model selection indicating that the independent-rates model fits the morphological data better than the autocorrelated-rates model. We suggest that using morphological continuous characters, together with molecular data, can bring a new perspective to the study of species evolution. Our new model is implemented in the MCMCtree computer program for Bayesian inference of divergence times.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/existence_and_implications_of_population_variance_structure_49513562/">
          <h4 class="font-weight-bold">Existence and implications of population variance structure, bioRxiv, 2018-10-11</h4>
          <p>AbstractIdentifying the genetic and environmental factors underlying phenotypic differences between populations is fundamental to multiple research communities. To date, studies have focused on the relationship between population and phenotypic mean. Here we consider the relationship between population and phenotypic variance, i.e., “population variance structure.” In addition to gene-gene and gene-environment interaction, we show that population variance structure is a direct consequence of natural selection. We develop the ancestry double generalized linear model (ADGLM), a statistical framework to jointly model population mean and variance effects. We apply ADGLM to several deeply phenotyped datasets and observe ancestry-variance associations with 12 of 44 tested traits in ~113K British individuals and 3 of 14 tested traits in ~3K Mexican, Puerto Rican, and African-American individuals. We show through extensive simulations that population variance structure can both bias and reduce the power of genetic association studies, even when principal components or linear mixed models are used. ADGLM corrects this bias and improves power relative to previous methods in both simulated and real datasets. Additionally, ADGLM identifies 17 novel genotype-variance associations across six phenotypes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/first_detection_of_a_highly_invasive_freshwater_amphipod_crangonyx_floridanus_in_the_united_kingdom_49513116/">
          <h4 class="font-weight-bold">First detection of a highly invasive freshwater amphipod (Crangonyx floridanus) in the United Kingdom, bioRxiv, 2018-10-11</h4>
          <p>AbstractThe freshwater gammarid, Crangonyx floridanus, originates from North America but has invaded and subsequently spread rapidly throughout Japan. We provide here the first genetic and microscopic evidence that C. floridanus has now also reached the United Kingdom. We found this species in two locations separated by more than 200 km (Lake Windermere in the North of the UK and Smestow Brook, West Midlands). The current distribution of C. floridanus is currently unknown, however both sites are well connected to other river and channel systems therefore the chance of further spread is high. Genetic analyses of C. floridanus indicate that British inland waters are colonised by the same linage, which also has invaded Japan. We recommend further work to assess the distribution of this species and its impact on the local fauna and flora.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/0-100-users/page/67/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/64/" class="page-link">64</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/65/" class="page-link">65</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/66/" class="page-link">66</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/67/" class="page-link">67</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/0-100-users/page/68/" class="page-link">68</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/69/" class="page-link">69</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/70/" class="page-link">70</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/71/" class="page-link">71</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/72/" class="page-link">72</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/0-100-users/page/69/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/0-100-users/page/123/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
