<!DOCTYPE html>
<html><head>
  
  <title>Cell Biology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/stag1_and_stag2_regulate_cell_fate_decisions_in_hematopoiesis_through_non-redundant_topological_control_57258080/">
          <h4 class="font-weight-bold">Stag1 and Stag2 regulate cell fate decisions in hematopoiesis through non-redundant topological control, bioRxiv, 2019-03-19</h4>
          <p>AbstractTranscriptional regulators, including the cohesin complex member STAG2, are recurrently mutated in cancer. The role of STAG2 in gene regulation, hematopoiesis, and tumor suppression remains unresolved. We show Stag2 deletion in hematopoietic stemprogenitor cells (HSPC) results in altered hematopoietic function, increased self-renewal, and impaired differentiation. ChIP-sequencing revealed that while Stag2 and Stag1 can bind the same loci, a component of Stag2 binding sites are unoccupied by Stag1 even in Stag2-deficient HSPCs. While concurrent loss of Stag2 and Stag1 abrogated hematopoiesis, Stag2 loss alone decreased chromatin accessibility and transcription of lineage-specification genes, including Ebf1 and Pax5, leading to blunted HSPC commitment to the B-cell lineage. Our data illustrate a role for Stag2 in transformation and transcriptional dysregulation distinct from its shared role with Stag1 in chromosomal segregation.One Sentence SummaryStag1 rescues topologically associated domains in the absence of Stag2, but cannot restore chromatin architecture required for hematopoietic lineage commitment</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/high-speed_multifocal_plane_fluorescence_microscopy_for_three-dimensional_visualisation_of_beating_flagella_56825995/">
          <h4 class="font-weight-bold">High-speed multifocal plane fluorescence microscopy for three-dimensional visualisation of beating flagella, bioRxiv, 2019-03-11</h4>
          <p>AbstractAnalysis of flagellum beating in three dimensions is important for understanding how cells can undergo complex flagellum-driven motility and the ability to use fluorescence microscopy for such three-dimensional analysis would be extremely powerful. Trypanosoma and Leishmania are unicellular parasites which undergo complex cell movements in three dimensions as they swim and would particularly benefit from such an analysis. Here, high-speed multifocal plane fluorescence microscopy, a technique in which a light path multi-splitter is used to visualise 4 focal planes simultaneously, was used to reconstruct the flagellum beating of Trypanosoma brucei and Leishmania mexicana in three dimensions. It was possible to use either an organic fluorescent stain or a genetically-encoded fluorescence fusion protein to visualise flagellum and cell movement in three dimensions at a 200 Hz frame rate. This high-speed multifocal plane fluorescence microscopy approach was used to address two open questions regarding Trypanosoma and Leishmania swimming To quantify the planarity of the L. mexicana flagellum beat and analyse the nature of flagellum beating during T. brucei ‘tumbling’.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/about_samples_giving_examples_optimized_single_molecule_localization_microscopy_56507313/">
          <h4 class="font-weight-bold">About samples, giving examples Optimized Single Molecule Localization Microscopy, bioRxiv, 2019-03-05</h4>
          <p>AbstractSuper-resolution microscopy has profoundly transformed how we study the architecture of cells, revealing unknown structures and refining our view of cellular assemblies. Among the various techniques, the resolution of Single Molecule Localization Microscopy (SMLM) can reach the size of macromolecular complexes and offer key insights on their nanoscale arrangement in situ. SMLM is thus a demanding technique and taking advantage of its full potential requires specifically optimized procedures. Here we describe how we perform the successive steps of an SMLM workflow, focusing on single-color Stochastic Optical Reconstruction Microscopy (STORM) as well as multicolor DNA Points Accumulation for imaging in Nanoscale Topography (DNA-PAINT) of fixed samples. We provide detailed procedures for careful sample fixation and immunostaining of typical cellular structures cytoskeleton, clathrin-coated pits, and organelles. We then offer guidelines for optimal imaging and processing of SMLM data in order to optimize reconstruction quality and avoid the generation of artifacts. We hope that the tips and tricks we discovered over the years and detail here will be useful for researchers looking to make the best possible SMLM images, a pre-requisite for meaningful biological discovery.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/dna_methylation_directs_polycomb-dependent_3d_genome_re-_organisation_in_naive_pluripotency_supplementary_information_54303043/">
          <h4 class="font-weight-bold">DNA methylation directs polycomb-dependent 3D genome re- organisation in naive pluripotency Supplementary information, bioRxiv, 2019-01-23</h4>
          <p>The DNA hypomethylation that occurs when embryonic stem cells (ESCs) are directed to the ground state of naive pluripotency by culturing in 2i conditions results in redistribution of polycomb (H3K27me3) away from its target loci. Here we demonstrate that 3D genome organisation is also altered in 2i. We found chromatin decompaction at polycomb target loci as well as loss of long-range polycomb interactions. By preventing DNA hypomethylation during the transition to the ground-state, we are able to restore the H3K27me3 distribution, and polycomb-mediated 3D genome organisation that is characteristic of primed ESCs grown in serum, to ESCs in 2i. However, these cells retain the functional characteristics of 2i ground state ESCs. Our findings demonstrate the central role of DNA methylation in shaping major aspects of 3D genome organisation but caution against assuming causal roles for the epigenome and 3D genome in gene regulation and function in ESCs.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/multiplexed_electron_microscopy_by_fluorescent_barcoding_allows_screening_for_ultrastructural_phenotype_53694291/">
          <h4 class="font-weight-bold">Multiplexed electron microscopy by fluorescent barcoding allows screening for ultrastructural phenotype, bioRxiv, 2019-01-09</h4>
          <p>Genetic screens performed using high-throughput fluorescent microscopes have generated large datasets that have contributed many insights into cell biology. However, such approaches typically cannot tackle questions requiring knowledge of ultrastructure below the resolution limit of fluorescent microscopy. Electron microscopy (EM) is not subject to this resolution limit, generating detailed images of cellular ultrastructure, but requires time consuming preparation of individual samples, limiting its throughput. Here we overcome this obstacle and describe a robust method for screening by high-throughput electron microscopy. Our approach uses combinations of fluorophores as barcodes to mark the genotype of each cell in mixed populations, and correlative light and electron microscopy to read the fluorescent barcode of each cell before it is imaged by electron microscopy. Coupled with an easy-to-use software workflow for correlation, segmentation and computer image analysis, our method allows to extract and analyze multiple cell populations from each EM sample preparation. We demonstrate the method on several organelles with samples that each contain up to 15 different yeast variants. The methodology is not restricted to yeast, can be scaled to higher-throughput, and can be utilized in multiple ways to enable electron microscopy to become a powerful screening methodology.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/microtubule_plus-end_dynamics_link_wound_repair_to_the_innate_immune_response_53552872/">
          <h4 class="font-weight-bold">Microtubule plus-end dynamics link wound repair to the innate immune response, bioRxiv, 2019-01-07</h4>
          <p>As a first line of defence against the environment, the epidermis protect animals from infection and physical damage. In C. elegans, wounding the epidermal epithelium triggers both an immune reaction and a repair response. Exactly how these are controlled, and the degree to which they are inter-connected remains unclear. To address these questions, we established a simple system for simultaneously inflicting precise laser wounds and imaging at high spatial and temporal resolution. We show that in C. elegans, wounding provokes a rapid sealing of the plasma membrane, involving reorganisation of phosphatidylinositol 4,5-bisphosphate domains. This is followed by a radial recruitment at the wound site of EBP-2EB1, a protein that binds the plus ends of microtubules. EB1 recruitment is accompanied by a reorganisation of microtubules, required for the subsequent recruitment of actin and wound closure. It is also required for the directed trafficking towards the site of injury of the key signaling protein SNF-12. In the absence of SNF-12 recruitment, there is an abrogation of the immune response. Our results suggest that microtubule dynamics coordinate the cytoskeletal changes required for wound repair and the concomitant activation of the innate immune response.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/cell-biology/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/cell-biology/page/2/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="/audiences/tags/cell-biology/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/cell-biology/page/2/" class="page-link">2</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/cell-biology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/cell-biology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/cell-biology/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/cell-biology/page/6/" class="page-link">6</a></li>
            
        

        
        <li class="page-item"><a href="/audiences/tags/cell-biology/page/4/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/cell-biology/page/6/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
