<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/rate_variation_in_the_evolution_of_non-coding_dna_associated_with_social_evolution_in_bees_50953621/">
          <h4 class="font-weight-bold">Rate variation in the evolution of non-coding DNA associated with social evolution in bees, bioRxiv, 2018-11-08</h4>
          <p>The evolutionary origins of eusociality represent increases in complexity from individual to caste-based, group reproduction. These behavioral transitions have been hypothesized to go hand-in-hand with an increased ability to regulate when and where genes are expressed. Bees have convergently evolved eusociality up to five times, providing a framework to test this hypothesis. To examine potential links between putative gene regulatory elements and social evolution, we compare alignable, non-coding sequences in eleven diverse bee species, encompassing three independent origins of reproductive division of labor and two elaborations of eusocial complexity. We find that rates of evolution in a number of non-coding sequences correlate with key social transitions in bees. Interestingly, while we find little evidence for convergent rate changes associated with independent origins of social behavior, a number of molecular pathways exhibit convergent rate changes in conjunction with subsequent elaborations of social organization. We also present evidence that many novel non-coding regions may have been recruited alongside the origin of sociality in corbiculate bees; these loci could represent gene regulatory elements associated with division of labor within this group. Thus, our findings are consistent with the hypothesis that gene regulatory innovations are associated with the evolution of eusociality and illustrate how a thorough examination of both coding and non-coding sequence can provide a more complete understanding of the molecular mechanisms underlying behavioral evolution.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/a_genome-wide_algal_mutant_library_reveals_a_global_view_of_genes_required_for_eukaryotic_photosynthesis_50963501/">
          <h4 class="font-weight-bold">A genome-wide algal mutant library reveals a global view of genes required for eukaryotic photosynthesis, bioRxiv, 2018-11-07</h4>
          <p>Photosynthetic organisms provide food and energy for nearly all life on Earth, yet half of their protein-coding genes remain uncharacterized1,2. Characterization of these genes could be greatly accelerated by new genetic resources for unicellular organisms that complement the use of multicellular plants by enabling higher-throughput studies. Here, we generated a genome-wide, indexed library of mapped insertion mutants for the flagship unicellular alga Chlamydomonas reinhardtii (Chlamydomonas hereafter). The 62,389 mutants in the library, covering 83% of nuclear, protein-coding genes, are available to the community. Each mutant contains unique DNA barcodes, allowing the collection to be screened as a pool. We leveraged this feature to perform a genome-wide survey of genes required for photosynthesis, which identified 303 candidate genes. Characterization of one of these genes, the conserved predicted phosphatase CPL3, showed it is important for accumulation of multiple photosynthetic protein complexes. Strikingly, 21 of the 43 highest-confidence genes are novel, opening new opportunities for advances in our understanding of this biogeochemically fundamental process. This library is the first genome-wide mapped mutant resource in any unicellular photosynthetic organism, and will accelerate the characterization of thousands of genes in algae, plants and animals.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/comparative_analysis_of_sequencing_technologies_platforms_for_single-cell_transcriptomics_50830362/">
          <h4 class="font-weight-bold">Comparative analysis of sequencing technologies platforms for single-cell transcriptomics, bioRxiv, 2018-11-06</h4>
          <p>AbstractAll single-cell RNA-seq protocols and technologies require library preparation prior to sequencing on a platform such as Illumina. Here, we present the first report to utilize the BGISEQ-500 platform for scRNA-seq, and compare the sensitivity and accuracy to Illumina sequencing. We generate a scRNA-seq resource of 468 unique single-cells and 1,297 matched single cDNA samples, performing SMARTer and Smart-seq2 protocols on mESCs and K562 cells with RNA spike-ins. We sequence these libraries on both BGISEQ-500 and Illumina HiSeq platforms using single- and paired-end reads. The two platforms have comparable sensitivity and accuracy in terms of quantification of gene expression, and low technical variability. Our study provides a standardised scRNA-seq resource to benchmark new scRNA-seq library preparation protocols and sequencing platforms.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/investigating_causal_relationships_between_sleep_traits_and_risk_of_breast_cancer_a_mendelian_randomization_study_50836751/">
          <h4 class="font-weight-bold">Investigating causal relationships between sleep traits and risk of breast cancer a Mendelian randomization study, bioRxiv, 2018-11-06</h4>
          <p>AbstractObjectiveTo examine whether sleep traits have a causal effect on risk of breast cancer.DesignMultivariable regression, one- and two-sample Mendelian randomization.SettingThe UK Biobank prospective cohort study and the Breast Cancer Association Consortium (BCAC) case-control genome-wide association study.Participants156,848 women in the multivariable regression and one-sample Mendelian randomization analysis in UK Biobank (7,784 with a breast cancer diagnosis) and 122,977 breast cancer cases and 105,974 controls from BCAC in the two-sample Mendelian randomization analysis.ExposuresSelf-reported chronotype (morningevening preference), insomnia symptoms and sleep duration in multivariable regression, and genetic variants robustly associated with these sleep traits.Main outcome measuresBreast cancer (prevalent and incident cases in UK Biobank, prevalent cases only in BCAC).ResultsIn multivariable regression analysis using data on breast cancer incidence in UK Biobank, morning preference was inversely associated with breast cancer (HR 0.95, 95% CI 0.93, 0.98 per category increase) while there was little evidence for an association with sleep duration and insomnia symptoms. Using 341 single nucleotide polymorphisms (SNPs) associated with chronotype, 91 SNPs associated sleep duration and 57 SNPs associated with insomnia symptoms, one-sample MR analysis in UK Biobank provided some supportive evidence for a protective effect of morning preference on breast cancer risk (HR 0.85, 95% 0.70, 1.03 per category increase) but imprecise estimates for sleep duration and insomnia symptoms. Two-sample MR using data from BCAC supported findings for a protective effect of morning preference (OR 0.88, 95% CI 0.82, 0.93 per category increase) and adverse effect of increased sleep duration (OR 1.19, 95% CI 1.02, 1.39 per hour increase) on breast cancer (both estrogen receptor positive and negative), while there was inconsistent evidence for insomnia symptoms. Results were largely robust to sensitivity analyses accounting for horizontal pleiotropy.ConclusionsWe found consistent evidence for a protective effect of morning preference and suggestive evidence for an adverse effect of sleep duration on breast cancer risk.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/epidemiology">epidemiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/on_the_adaptive_behavior_of_head-fixed_flies_navigating_in_two-dimensional_visual_virtual_reality_50722645/">
          <h4 class="font-weight-bold">On the adaptive behavior of head-fixed flies navigating in two-dimensional, visual virtual reality, bioRxiv, 2018-11-05</h4>
          <p>AbstractA navigating animal’s sensory experience is shaped not just by its surroundings, but by its movements within them, which in turn are influenced by its past experiences. Studying the intertwined roles of sensation, experience and directed action in navigation has been made easier by the development of virtual reality (VR) environments for head-fixed animals, which allow for quantitative measurements of behavior in well-controlled sensory conditions. VR has long featured in studies of Drosophila melanogaster, but these experiments have typically relied on one-dimensional (1D) VR, effectively allowing the fly to change only its heading in a visual scene, and not its position. Here we explore how flies navigate in a two-dimensional (2D) visual VR environment that more closely resembles their experience during free behavior. We show that flies’ interaction with landmarks in 2D environments cannot be automatically derived from their behavior in simpler 1D environments. Using a novel paradigm, we then demonstrate that flies in 2D VR adapt their behavior in a visual environment in response to optogenetically delivered appetitive and aversive stimuli. Much like free-walking flies after encounters with food, head-fixed flies respond to optogenetic activation of sugar-sensing neurons by initiating a local search behavior. Finally, by pairing optogenetic activation of heat-sensing cells to the flies’ presence near visual landmarks of specific shapes, we elicit selective learned avoidance of landmarks associated with aversive “virtual heat”. These head-fixed paradigms set the stage for an interrogation of fly brain circuitry underlying flexible navigation in complex visual environments.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/animal-behavior-and-cognition">animal-behavior-and-cognition</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/tartan_underlies_the_evolution_of_male_drosophila_genital_morphology_50816397/">
          <h4 class="font-weight-bold">tartan underlies the evolution of male Drosophila genital morphology, bioRxiv, 2018-11-05</h4>
          <p>AbstractMale genital structures are among the most rapidly evolving morphological traits and are often the only features that can distinguish closely related species. This process is thought to be driven by sexual selection and may reinforce species separation. However, while the genetic basis of many phenotypic differences have been identified, we still lack knowledge about the genes underlying evolutionary differences in male genital organs and organ size more generally. The claspers (surstyli) are periphallic structures that play an important role in copulation in insects. Here we show that natural variation in clasper size and bristle number between Drosophila mauritiana and D. simulans is caused by evolutionary changes in tartan (trn), which encodes a transmembrane leucine-rich repeat domain protein that mediates cell-cell interactions and affinity differences. There are no fixed amino acid differences in trn between D. mauritiana and D. simulans but differences in the expression of this gene in developing genitalia suggest cis-regulatory changes in trn underlie the evolution of clasper morphology in these species. Finally, analysis of reciprocal hemizyotes that are genetically identical, except for which species the functional allele of trn is from, determined that the trn allele of D. mauritiana specifies larger claspers with more bristles than the allele of D. simulans. Therefore we have identified the first gene underlying evolutionary change in the size of a male genital organ, which will help to better understand the rapid diversification of these structures and the regulation and evolution of organ size more broadly.Significance StatementThe morphology of male genital organs evolves rapidly driven by sexual selection. However, little is known about the genes underlying genitalia differences between species. Identifying these genes is key to understanding how sexual selection acts on development to produce rapid phenotypic change. We have found that the gene tartan underlies differences between male Drosophila mauritiana and D. simulans in the size and bristle number of the claspers - genital projections that grasp the female during copulation. Moreover, since tartan encodes a protein that is involved in cell affinity, this may represent a new developmental mechanism for morphological change. Therefore, our study provides new insights into genetic and developmental bases for the rapid evolution of male genitalia and organ size more generally.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/0-100-users/page/34/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/31/" class="page-link">31</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/32/" class="page-link">32</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/33/" class="page-link">33</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/34/" class="page-link">34</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/0-100-users/page/35/" class="page-link">35</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/36/" class="page-link">36</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/37/" class="page-link">37</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/38/" class="page-link">38</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/39/" class="page-link">39</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/0-100-users/page/36/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/0-100-users/page/64/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
