<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/eqtl_catalogue_a_compendium_of_uniformly_processed_human_gene_expression_and_splicing_qtls_75134620/">
          <h4 class="font-weight-bold">eQTL Catalogue a compendium of uniformly processed human gene expression and splicing QTLs, bioRxiv, 2020-01-30</h4>
          <p>AbstractAn increasing number of gene expression quantitative trait locus (QTL) studies have made summary statistics publicly available, which can be used to gain insight into human complex traits by downstream analyses such as fine-mapping and colocalisation. However, differences between these datasets in their variants tested, allele codings, and in the transcriptional features quantified are a barrier to their widespread use. Here, we present the eQTL Catalogue, a resource which contains quality controlled, uniformly re-computed QTLs from 19 eQTL publications. In addition to gene expression QTLs, we have also identified QTLs at the level of exon expression, transcript usage, and promoter, splice junction and 3ʹ end usage. Our summary statistics can be downloaded by FTP or accessed via a REST API and are also accessible via the Open Targets Genetics Portal. We demonstrate how the eQTL Catalogue and GWAS Catalog APIs can be used to perform colocalisation analysis between GWAS and QTL results without downloading and reformatting summary statistics. New datasets will continuously be added to the eQTL Catalogue, enabling systematic interpretation of human GWAS associations across a large number of cell types and tissues. The eQTL Catalogue is available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpswww.ebi.ac.ukeqtl&gt;httpswww.ebi.ac.ukeqtl&lt;jatsext-link&gt;.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/haslr_fast_hybrid_assembly_of_long_reads_75551180/">
          <h4 class="font-weight-bold">HASLR Fast Hybrid Assembly of Long Reads, bioRxiv, 2020-01-28</h4>
          <p>AbstractThird generation sequencing technologies from platforms such as Oxford Nanopore Technologies and Pacific Biosciences have paved the way for building more contiguous assemblies and complete reconstruction of genomes. The larger effective length of the reads generated with these technologies has provided a mean to overcome the challenges of short to mid-range repeats. Currently, accurate long read assemblers are computationally expensive while faster methods are not as accurate. Therefore, there is still an unmet need for tools that are both fast and accurate for reconstructing small and large genomes. Despite the recent advances in third generation sequencing, researchers tend to generate second generation reads for many of the analysis tasks. Here, we present HASLR, a hybrid assembler which uses both second and third generation sequencing reads to efficiently generate accurate genome assemblies. Our experiments show that HASLR is not only the fastest assembler but also the one with the lowest number of misassemblies on all the samples compared to other tested assemblers. Furthermore, the generated assemblies in terms of contiguity and accuracy are on par with the other tools on most of the samples.AvailabilityHASLR is an open source tool available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsgithub.comvpc-ccghaslr&gt;httpsgithub.comvpc-ccghaslr&lt;jatsext-link&gt;.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/mapping_heritability_of_obesity_by_brain_cell_types_75587499/">
          <h4 class="font-weight-bold">Mapping heritability of obesity by brain cell types, bioRxiv, 2020-01-28</h4>
          <p>The underlying cell types mediating predisposition to obesity remain largely obscure. Here we first integrated recently published single-cell RNA-sequencing (scRNA-seq) data from &amp;gt;380 peripheral and nervous system cell types spanning 19 mouse organs with body mass index (BMI) genome-wide association study (GWAS) data from &amp;gt;450,000 individuals. Leveraging a novel strategy for integrating scRNA-seq data with GWAS data, we identified 22, exclusively neuronal, cell types from the subthalamus, midbrain, hippocampus, thalamus, cortex, pons, medulla, pallidum that were significantly enriched for BMI heritability (P&amp;lt;1.6×10-4). Using genes harboring coding mutations leading to syndromic forms of obesity, we replicate four midbrain cell types from the anterior pretectal nucleus, superior nucleus, periaqueductal gray and pallidum (P&amp;lt;1.7×10-4). Testing an additional set of 347 hypothalamic cell types, ventromedial hypothalamic steroidogenic-factor 1 (SF1) and cholecystokinin b receptor (CCKBR)-expressing neurons (P=4.9×10-5) previously implicated in energy homeostasis and glucose control and three cell types from the preoptic area of the hypothalamus and the lateral hypothalamus enriched for BMI GWAS associations (P&amp;lt;4.9×10-5). Together, our results suggest brain nuclei regulating integration of sensory stimuli, learning and memory are likely to play a key role in obesity and provide testable hypotheses for mechanistic follow-up studies.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_comprehensive_atlas_of_white_matter_tracts_in_the_chimpanzee_75481474/">
          <h4 class="font-weight-bold">A comprehensive atlas of white matter tracts in the chimpanzee, bioRxiv, 2020-01-26</h4>
          <p>AbstractChimpanzees (Pan troglodytes) are, along with bonobos, humans’ closest living relatives. The advent of diffusion tractography in recent years has allowed a resurgence of comparative neuroanatomical studies in humans and other primate species. Here, we offer, in comparative perspective, the first chimpanzee white matter atlas, coupled with surface projection maps of these major white matter tracts, constructed from in vivo chimpanzee diffusion-weighted scans. Comparative white matter atlases provide a useful tool for identifying neuroanatomical differences and similarities between humans and other primate species. Until now, comprehensive fascicular atlases have been created for humans (Homo sapiens), rhesus macaques (Macaca mulatta), and several other nonhuman primate species, but never in a nonhuman ape. Information on chimpanzee neuroanatomy is essential for understanding the anatomical specializations of white matter organization that are unique to the human lineage.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/comparison_of_visualisation_tools_for_single-cell_rnaseq_data_75084974/">
          <h4 class="font-weight-bold">Comparison of visualisation tools for single-cell RNAseq data, bioRxiv, 2020-01-26</h4>
          <p>In the last decade, single cell RNAseq (scRNAseq) datasets have grown from a single cell to millions of cells. Due to its high dimensionality, the scRNAseq data contains a lot of valuable information, however, it is not always feasible to visualise and share it in a scientific report or an article publication format. Recently, a lot of interactive analysis and visualisation tools have been developed to address this issue and facilitate knowledge transfer in the scientific community. In this study, we review and compare several of the currently available analysis and visualisation tools and benchmark those that allow to visualize the scRNAseq data on the web and share it with others. To address the problem of format compatibility for most visualisation tools, we have also developed a user-friendly R package, sceasy, which allows users to convert their own scRNAseq datasets into a specific data format for visualisation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/fine-scale_genomic_analyses_of_admixed_individuals_reveal_unrecognized_genetic_ancestry_components_in_argentina_native_american_african_and_european_genetic_ancestries_in_argentina_75431842/">
          <h4 class="font-weight-bold">Fine-scale genomic analyses of admixed individuals reveal unrecognized genetic ancestry components in Argentina Native American, African and European genetic ancestries in Argentina, bioRxiv, 2020-01-25</h4>
          <p>AbstractWe are at the dawn of the efforts to describe and understand the origins of genetic diversity in Argentina from high-throughput data. This knowledge is a primary step in the intent of deciphering the specific genetic bases of diseases and drug response in the country. Similarly to other populations across the Americas, genetic ancestry in Argentinean populations traces back into African, European and Native American ancestors, reflecting a complex demographic history with multiple migration and admixture events in pre- and post-colonial times. However, little is known about the sub-continental origins of these three main ancestries. We present new high-throughput genotyping data for 87 admixed individuals across Argentina. This data was combined to previously published data for admixed individuals in the region and then compared to different reference panels specifically built to run population structure analyses at a sub-continental level. Concerning the European and African ancestries, we confirmed previous results about their main origins, and we provide new insights into the presence of other origins that reflect historical records. As for the Native American ancestry, leveraging genotype data for archaeological samples in the region in order to gain temporal depth in our analyses, we could identify four Native American components segregating in modern Argentinean populations. Three of them are also found in modern South American populations and are specifically represented in Central ChilePatagonia, Lowlands and Central Andes geographic areas. The fourth one may be specific to the Central Western region of Argentina.Identifying such component has not been straightforward since it is not well represented in any genomic data from the literature. Altogether, we provide useful insights into the multiple population groups from different continents that have contributed to present-days genetic diversity in Argentina. We encourage the generation of massive genotype data locally to further describe the genetic structure in Argentina.Author SummaryThe human genetic diversity in Argentina reflects demographic mechanisms during which the European colonists invaded a territory where Native American populations were settled. During colonial period, the slave trade also prompted many African people to move to Argentina. Little is known about the origins of the Native American and African components in Argentinean populations nowadays.Genotyping data for 87 admixed individuals throughout Argentina was generated and data from the literature was re-analyzed to shed light on this question. We confirmed that most of the European genetic ancestry comes from the South, although several individuals are related to Northern Europeans. We found that African origins in Argentina trace back from different regions. As for the Native American ancestry, we identified that it can be divided into four main components that correspond to Central ChilePatagonia, Lowlands, Central Andes and Central Western region of Argentina. In order to understand the specificity of the genetic diversity in Argentina, we should not rely on knowledge generated in other populations. Instead, more effort is required to generate specific massive genomic knowledge at the local level.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2020">2020</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        
        <li class="page-item"><a href="../../../../tags/0-100-users/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/" class="page-link">1</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/0-100-users/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/6/" class="page-link">6</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/0-100-users/page/3/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/0-100-users/page/123/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
