<!DOCTYPE html>
<html><head>
  
  <title>2019 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/short-range_interactions_govern_cellular_dynamics_in_microbial_multi-genotype_systems_54544166/">
          <h4 class="font-weight-bold">Short-range interactions govern cellular dynamics in microbial multi-genotype systems, bioRxiv, 2019-01-27</h4>
          <p>Ecosystem processes result from interaction between organisms. When interactions are local, the spatial organization of organisms defines their network of interactions, and thus influences the system&#39;s functioning. This can be especially relevant for microbial systems, which often consist of spatially structured communities of cells connected by a dense interaction network. Here we measured the spatial interaction network between cells in microbial systems and identify the factors that determine it. Combining quantitative single-cell analysis of synthetic bacterial communities with mathematical modeling, we find that cells only interact with other cells in their immediate neighbourhood. This short interaction range impacts the functioning of the whole system by reducing its ability to perform metabolic processes collectively. Our experiments and models demonstrate that the spatial scale of cell-to-cell interaction plays a fundamental role in understanding and controlling natural communities, and in engineering microbial systems for specific purposes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/causal_relationships_between_obesity_and_the_leading_causes_of_death_in_women_and_men_54531769/">
          <h4 class="font-weight-bold">Causal relationships between obesity and the leading causes of death in women and men, bioRxiv, 2019-01-26</h4>
          <p>AbstractObesity traits are causally implicated with risk of cardiometabolic diseases. It remains unclear whether there are similar causal effects of obesity traits on other non-communicable diseases. Also, it is largely unexplored whether there are any sex-specific differences in the causal effects of obesity traits on cardiometabolic diseases and other leading causes of death.We constructed sex-specific genetic risk scores (GRS) for three obesity traits; body mass index (BMI), waist-hip ratio (WHR), and WHR adjusted for BMI, including 565, 324, and 337 genetic variants, respectively. These GRSs were then used as instrumental variables to assess associations between the obesity traits and leading causes of mortality in the UK Biobank using Mendelian randomization. We also investigated associations with potential mediators, including smoking, glycemic and blood pressure traits. Sex-differences were subsequently assessed by Cochran’s Q-test (Phet).A Mendelian randomization analysis of 228,466 women and 195,041 men showed that obesity causes coronary artery disease, stroke (particularly ischemic), chronic obstructive pulmonary disease, lung cancer, type 2 and 1 diabetes mellitus, non-alcoholic fatty liver disease, chronic liver disease, and acute and chronic renal failure. Higher BMI led to higher risk of type 2 diabetes in women than in men (Phet=1.4×10-5). Waist-hip-ratio led to a higher risk of chronic obstructive pulmonary disease (Phet=3.7×10-6) and higher risk of chronic renal failure (Phet=1.0×10-4) in men than women.Obesity traits have an etiological role in the majority of the leading global causes of death. Sex differences exist in the effects of obesity traits on risk of type 2 diabetes, chronic obstructive pulmonary disease, and renal failure, which may have downstream implications for public health.Author summaryObesity is increasing globally and has been linked to major causes of death, such as diabetes and heart disease. Still, the causal effects of obesity on other leading causes of death is relatively unexplored. It is also unclear if any such effects differ between men and women.Mendelian randomization is a method that explores causal relationships between traits using genetic data. Using Mendelian randomization, we investigated the effects of obesity traits on leading causes of death and assessed if any such effects differ between men and women.We found that obesity increases the risks of heart disease, stroke, chronic obstructive pulmonary disease, lung cancer, diabetes, kidney disease, non-alcoholic fatty liver disease and chronic liver disease. Higher body mass index led to a higher risk of type 2 diabetes in women than in men, whereas a higher waist-hip ratio increased risks of chronic obstructive pulmonary disease and chronic kidney disease more in men than in women.In summary, obesity traits are causally involved in the majority of the leading causes of death, and some obesity traits affect disease risk differently in men and women. This has potential implications for public health strategies and indicates that sex-specific preventative measure may be needed.AbbreviationsBMI, Body mass index; CAD coronary artery disease; CLD, chronic liver disease; COPD, chronic obstructive pulmonary disease; DBP, diastolic blood pressure; FG, fasting glucose; FI, fasting insulin; GIANT, Genetic Investigation of ANthropometric Traits; GRS, genetic risk score; GWAS, genomewide association study; MAGIC, the Meta-Analyses of Glucose and Insulin-related traits Consortium; MR, Mendelian randomization; NAFLD, non-alcoholic fatty liver disease; OR, odds ratio; T1D, type 1 diabetes; T2D, type 2 diabetes; SBP, systolic blood pressure; SD, standard deviation; SNP, single nucleotide polymorphism; WHO, the World Health Organization; WHR, waist-hip-ratio; WHRadjBMI, waist-hip-ratio adjusted for body mass index.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/epidemiology">epidemiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/causal_relevance_of_obesity_on_the_leading_causes_of_death_in_women_and_men_a_mendelian_randomization_study_54531769/">
          <h4 class="font-weight-bold">Causal relevance of obesity on the leading causes of death in women and men A Mendelian randomization study, bioRxiv, 2019-01-26</h4>
          <p>AbstractBackgroundObesity traits are causally implicated with risk of cardiometabolic diseases. It remains unclear whether there are similar causal effects of obesity traits on other non-communicable diseases. Also, it is largely unexplored whether there are any sex-specific differences in the causal effects of obesity traits on cardiometabolic diseases and other leading causes of death. We therefore tested associations of sex-specific genetic risk scores (GRSs) for body mass index (BMI), waist-hip-ratio (WHR), and WHR adjusted for BMI (WHRadjBMI) with leading causes of mortality, using a Mendelian randomization (MR) framework.Methods and FindingsWe constructed sex-specific GRSs for BMI, WHR, and WHRadjBMI, including 565, 324, and 338 genetic variants, respectively. These GRSs were then used as instrumental variables to assess associations between the obesity traits and leading causes of mortality using an MR design in up to 422,414 participants from the UK Biobank. We also investigated associations with potential mediators and risk factors, including smoking, glycemic and blood pressure traits. Sex-differences were subsequently assessed by Cochran’s Q-test (Phet).Up to 227,717 women and 194,697 men with mean (standard deviation) age 56.6 (7.9) and 57.0 (8.1) years, body mass index 27.0 (5.1) and 27.9 (4.2) kgm2 and waist-hip-ratio 0.82 (0.07) and 0.94 (0.07), respectively, were included. Mendelian randomization analysis showed that obesity causes coronary artery disease, stroke (particularly ischemic), chronic obstructive pulmonary disease, lung cancer, type 2 and 1 diabetes mellitus, non-alcoholic fatty liver disease, chronic liver disease, and acute and chronic renal failure. A 1 standard deviation higher body mass index led to higher risk of type 2 diabetes in women (OR 3.81; 95% CI 3.42-4.25, P=8.9×10−130) than in men (OR 2.78; 95% CI 2.57-3.02, P=1.0×10−133, Phet=5.1×10−6). Waist-hip-ratio led to a higher risk of chronic obstructive pulmonary disease (Phet=5.5×10−6) and higher risk of chronic renal failure (Phet=1.3×10−4) in men than women.A limitation of MR studies is potential bias if the genetic variants are directly associated with confounders (pleiotropy), but sensitivity analyses such as MR-Egger supported the main findings. Our study was also limited to people of European descent and results may differ in people of other ancestries.ConclusionsObesity traits have an etiological role in the majority of the leading global causes of death. Sex differences exist in the effects of obesity traits on risk of type 2 diabetes, chronic obstructive pulmonary disease, and renal failure, which may have implications on public health.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/epidemiology">epidemiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/endogenous_insulin_contributes_to_pancreatic_cancer_development_54463404/">
          <h4 class="font-weight-bold">Endogenous insulin contributes to pancreatic cancer development, bioRxiv, 2019-01-25</h4>
          <p>Obesity and early-stage type 2 diabetes (T2D) increase the risk for many cancers, including pancreatic ductal adenocarcinoma (PDAC). The mechanisms linking obesity and T2D to cancer have not been established, preventing targeted interventions. Arguments have been made that hyperinsulinemia, hyperglycemia, or inflammation could drive cancer initiation andor progression. Hyperinsulinemia is a cardinal feature of obesity and T2D, and is independently associated with PDAC incidence and mortality, even in non-obese people. Despite ample human epidemiological evidence linking hyperinsulinemia to PDAC, there is no direct in vivo evidence of a causal role for endogenous insulin in cancer in any system. Using mice with reduced insulin gene dosage, we show here that a modest reduction in endogenous insulin production leads to a ~50% reduction in pancreatic intraepithelial neoplasia (PanIN) pre-cancerous lesions in high fat diet-fed mice expressing the KrasG12D oncogene. The significant reduction in PanIN lesions occurred in the absence of changes in fasting glucose. Reduced insulin also led to a ~50% reduction in pancreatic fibrosis, suggesting that endogenous insulin drives PanIN development, in part, via its pro-fibrotic effects on the stroma surrounding acinar cells and PanIN. Collectively, our data indicate that endogenous insulin hypersecretion contributes causally to pancreatic cancer development. This suggests a modest reduction in fasting insulin via lifestyle interventions or therapeutics may be useful in cancer prevention.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cancer-biology">cancer-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/rna_interactions_with_ctcf_are_essential_for_its_proper_function_supplemental_figures_1-5_54452778/">
          <h4 class="font-weight-bold">RNA interactions with CTCF are essential for its proper function Supplemental Figures 1-5, bioRxiv, 2019-01-25</h4>
          <p>The function of the CCCTC-binding factor (CTCF) in the organization of the genome has become an important area of investigation, but the mechanisms of how CTCF dynamically contributes to genome organization is not clear. We previously discovered that CTCF binds to large numbers of endogenous RNAs; promoting its oligomerization. Here we found that inhibition of transcription or interfering with CTCF ability to bind RNA through mutations of two of its 11 zinc fingers that are not involved with CTCF binding to its cognate site in vitro, zinc finger-1 (ZF1) or -10 (ZF10), disrupt CTCF association to chromatin. These mutations alter gene expression profiles as CTCF mutants lose their ability to promote local insulation. Our results highlight the importance of RNA as a structural component of the genome, in part by affecting the association of CTCF with chromatin and likely its interaction with other factors.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/simultaneous_quantification_of_protein-dna_contacts_and_transcriptomes_in_single_cells_supplemental_figures_54446601/">
          <h4 class="font-weight-bold">Simultaneous quantification of protein-DNA contacts and transcriptomes in single cells Supplemental Figures, bioRxiv, 2019-01-25</h4>
          <p>The epigenome plays a critical role in regulating gene expression in mammalian cells. However, understanding how cell-to-cell heterogeneity in the epigenome influences gene expression variability remains a major challenge. Here we report a novel method for simultaneous single-cell quantification of protein-DNA contacts with DamID and transcriptomics (scDamID&amp;amp;T). This method enables quantifying the impact of protein-DNA contacts on gene expression from the same cell. By profiling lamina-associated domains (LADs) in human cells, we reveal different dependencies between genome-nuclear lamina (NL) association and gene expression in single cells. In addition, we introduce the E. coli methyltransferase, Dam, as an in vivo marker of chromatin accessibility in single cells and show that scDamID&amp;amp;T can be utilized as a general technology to identify cell types in silico while simultaneously determining the underlying gene-regulatory landscape. With this strategy the effect of chromatin states, transcription factor binding, and genome organization on the acquisition of cell-type specific transcriptional programs can be quantified.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/2019/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/2019/page/121/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/118/" class="page-link">118</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/119/" class="page-link">119</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/120/" class="page-link">120</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/121/" class="page-link">121</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/2019/page/122/" class="page-link">122</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/123/" class="page-link">123</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/124/" class="page-link">124</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/125/" class="page-link">125</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/126/" class="page-link">126</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/2019/page/123/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/2019/page/135/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
