<!DOCTYPE html>
<html><head>
  
  <title>100 200 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/resistance_gene_discovery_and_cloning_by_sequence_capture_and_association_genetics_31728278/">
          <h4 class="font-weight-bold">Resistance gene discovery and cloning by sequence capture and association genetics, bioRxiv, 2018-01-16</h4>
          <p>Genetic resistance is the most economic and environmentally sustainable approach for crop disease protection. Disease resistance (R) genes from wild relatives are a valuable resource for breeding resistant crops. However, introgression of R genes into crops is a lengthy process often associated with co-integration of deleterious linked genes1, 2 and pathogens can rapidly evolve to overcome R genes when deployed singly3. Introducing multiple cloned R genes into crops as a stack would avoid linkage drag and delay emergence of resistance-breaking pathogen races4. However, current R gene cloning methods require segregating or mutant progenies5–10, which are difficult to generate for many wild relatives due to poor agronomic traits. We exploited natural pan-genome variation in a wild diploid wheat by combining association genetics with R gene enrichment sequencing (AgRenSeq) to clone four stem rust resistance genes in &amp;lt;6 months. RenSeq combined with diversity panels is therefore a major advance in isolating R genes for engineering broad-spectrum resistance in crops.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/fast_automated_reconstruction_of_genome-scale_metabolic_models_for_microbial_species_and_communities_31608960/">
          <h4 class="font-weight-bold">Fast automated reconstruction of genome-scale metabolic models for microbial species and communities, bioRxiv, 2018-01-13</h4>
          <p>AbstractGenome-scale metabolic models are instrumental in uncovering operating principles of cellular metabolism and model-guided re-engineering. Recent applications of metabolic models have also demonstrated their usefulness in unraveling cross-feeding within microbial communities. Yet, the application of genome-scale models, especially to microbial communities, is lagging far behind the availability of sequenced genomes. This is largely due to the time-consuming steps of manual cura-tion required to obtain good quality models and thus physiologically meaningful simulation results. Here, we present an automated tool – CarveMe – for reconstruction of species and community level metabolic models. We introduce the concept of a universal model, which is manually curated and simulation-ready. Starting with this universal model and annotated genome sequences, CarveMe uses a top-down approach to build single-species and community models in a fast and scalable manner. We build reconstructions for two model organisms, Escherichia coli and Bacillus subtillis, as well as a collection of human gut bacteria, and show that CarveMe models perform similarly to manually curated models in reproducing experimental phenotypes. Finally, we demonstrate the scalability of CarveMe through reconstructing 5587 bacterial models. Overall, CarveMe provides an open-source and user-friendly tool towards broadening the use of metabolic modeling in studying microbial species and communities.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/direct_imaging_of_the_circular_chromosome_in_a_live_bacterium_31563978/">
          <h4 class="font-weight-bold">Direct imaging of the circular chromosome in a live bacterium, bioRxiv, 2018-01-12</h4>
          <p>New assays for quantitative imaging1–6 and sequencing7–11 have yielded great progress towards understanding the organizational principles of chromosomes. Yet, even for the well-studied model bacterium Escherichia coli, many basic questions remain unresolved regarding chromosomal (sub-)structure2,11, its mechanics1,2,12 and dynamics13,14, and the link between structure and function1,15,16. Here we resolve the spatial organization of the circular chromosome of bacteria by directly imaging the chromosome in live E. coli cells with a broadened cell shape. The chromosome was observed to exhibit a torus topology with a 4.2 μm toroidal length and 0.4 μm bundle thickness. On average, the DNA density along the chromosome shows dense right and left arms that branch from a lower-density origin of replication, and are connected at the terminus of replication by an ultrathin flexible string of DNA. At the single-cell level, the DNA density along the torus is found to be strikingly heterogeneous, with blob-like Mbp-size domains that undergo major dynamic rearrangements, splitting and merging at a minute timescale. We show that prominent domain boundaries at the terminus and origin of replication are induced by MatP proteins, while weaker transient domain boundaries are facilitated by the global transcription regulators HU and Fis. These findings provide an architectural basis for the understanding of the spatial organization of bacterial genomes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/limitation_of_alignment-free_tools_in_total_rna-seq_quantification_31565537/">
          <h4 class="font-weight-bold">Limitation of alignment-free tools in total RNA-seq quantification, bioRxiv, 2018-01-12</h4>
          <p>AbstractBackgroundAlignment-free RNA quantification tools have significantly increased the speed of RNA-seq analysis. However, it is unclear whether these state-of-the-art RNA-seq analysis pipelines can quantify small RNAs as accurately as they do with long RNAs in the context of total RNA quantification.ResultWe comprehensively tested and compared four RNA-seq pipelines on the accuracies of gene quantification and fold-change estimation on a novel total RNA benchmarking dataset, in which small non-coding RNAs are highly represented along with other long RNAs. The four RNA-seq pipelines were of two commonly-used alignment-free pipelines and two variants of alignment-based pipelines. We found that all pipelines showed high accuracies for quantifying the expressions of long and highly-abundant genes. However, alignment-free pipelines showed systematically poorer performances in quantifying lowly-abundant and small RNAs.ConclusionWe have shown that alignment-free and traditional alignment-based quantification methods performed similarly for common gene targets, such as protein-coding genes. However, we identified a potential pitfall in analyzing and quantifying lowly-expressed genes and small RNAs with alignment-free pipelines, especially when these small RNAs contain mutations.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/heterochromatin_drives_organization_of_conventional_and_inverted_nuclei_31471754/">
          <h4 class="font-weight-bold">Heterochromatin drives organization of conventional and inverted nuclei, bioRxiv, 2018-01-10</h4>
          <p>AbstractThe mammalian cell nucleus displays a remarkable spatial segregation of active euchromatic from inactive heterochromatic genomic regions. In conventional nuclei, euchromatin is localized in the nuclear interior and heterochromatin at the nuclear periphery. In contrast, rod photoreceptors in nocturnal mammals have inverted nuclei, with a dense heterochromatic core and a thin euchromatic outer shell. This inverted architecture likely converts rod nuclei into microlenses to facilitate nocturnal vision, and may relate to the absence of particular proteins that tether heterochromatin to the lamina. However, both the mechanism of inversion and the role of interactions between different types of chromatin and the lamina in nuclear organization remain unknown. To elucidate this mechanism we performed Hi-C and microscopy on cells with inverted nuclei and their conventional counterparts. Strikingly, despite the inversion evident in microscopy, both types of nuclei display similar Hi-C maps. To resolve this paradox we developed a polymer model of chromosomes and found a universal mechanism that reconciles Hi-C and microscopy for both inverted and conventional nuclei. Based solely on attraction between heterochromatic regions, this mechanism is sufficient to drive phase separation of euchromatin and heterochromatin and faithfully reproduces the 3D organization of inverted nuclei. When interactions between heterochromatin and the lamina are added, the same model recreates the conventional nuclear organization. To further test our models, we eliminated lamina interactions in models of conventional nuclei and found that this triggers a spontaneous process of inversion that qualitatively reproduces the pathway of morphological changes during nuclear inversion in vivo. Together, our experiments and modeling suggest that interactions among heterochromatic regions are central to phase separation of the active and inactive genome in inverted and conventional nuclei, while interactions with the lamina are essential for building the conventional architecture from these segregated phases. Ultimately our data suggest that an inverted organization constitutes the default state of nuclear architecture.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/insect_wings_and_body_wall_evolved_from_ancient_leg_segments_31454380/">
          <h4 class="font-weight-bold">Insect wings and body wall evolved from ancient leg segments, bioRxiv, 2018-01-10</h4>
          <p>AbstractThe origin of insect wings has long been debated. Central to this debate is whether wings evolved from an epipod (outgrowth, e.g., a gill) on ancestral crustacean leg segments, or represent a novel outgrowth from the dorsal body wall that co-opted some of the genes used to pattern the epipods. To determine whether wings can be traced to ancestral, pre-insect structures, or arose by co-option, comparisons are necessary between insects and arthropods more representative of the ancestral state, where the hypothesized proximal leg region is not fused to the body wall. To do so, we examined the function of five leg patterning genes in the crustacean Parhyale hawaiensis and compared this to previous functional data from insects. By comparing gene knockout phenotypes of leg patterning genes in a crustacean with those of insects, we show that two ancestral crustacean leg segments were incorporated into the insect body, moving the leg’s epipod dorsally, up onto the back to form insect wings. Thus, our data shows that much of the body wall of insects, including the entire wing, is derived from these two ancestral proximal leg segments. This model explains all observations in favor of either the body wall origin or proximal leg origin of insect wings. Thus, our results show that insect wings are not novel structures, but instead evolved from existing, ancestral structures.One Sentence SummaryCRISPR-Cas9 knockout of leg gap genes in a crustacean reveals that insect wings are not novel structures, they evolved from crustacean leg segments</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/90/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/87/" class="page-link">87</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/88/" class="page-link">88</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/89/" class="page-link">89</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/90/" class="page-link">90</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/91/" class="page-link">91</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/92/" class="page-link">92</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/93/" class="page-link">93</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/94/" class="page-link">94</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/95/" class="page-link">95</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/92/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/100-200-users/page/126/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
