<!DOCTYPE html>
<html><head>
  
  <title>Plant Biology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/optimization_of_t-dna_architecture_for_cas9-mediated_mutagenesis_in_arabidopsis_48543705/">
          <h4 class="font-weight-bold">Optimization of T-DNA architecture for Cas9-mediated mutagenesis in Arabidopsis, bioRxiv, 2018-09-17</h4>
          <p>ABSTRACTBacterial CRISPR systems have been widely adopted to create operator-specified site-specific nucleases. Such nuclease action commonly results in loss-of-function alleles, facilitating functional analysis of genes and gene families We conducted a systematic comparison of components and T-DNA architectures for CRISPR-mediated gene editing in Arabidopsis, testing multiple promoters, terminators, sgRNA backbones and Cas9 alleles. We identified a T-DNA architecture that usually results in stable (i.e. homozygous) mutations in the first generation after transformation. Notably, the transcription of sgRNA and Cas9 in head-to-head divergent orientation usually resulted in highly active lines. Our Arabidopsis data may prove useful for optimization of CRISPR methods in other plants.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/high-throughput_single-cell_transcriptome_profiling_of_plant_cell_types_47225996/">
          <h4 class="font-weight-bold">High-throughput single-cell transcriptome profiling of plant cell types, bioRxiv, 2018-08-29</h4>
          <p>AbstractSingle-cell transcriptome analysis of heterogeneous tissues can provide high-resolution windows into the genomic basis and spatiotemporal dynamics of developmental processes. Here we demonstrate the feasibility of high-throughput single-cell RNA sequencing of plant tissue using the Drop-seq approach. Profiling of &amp;gt;4,000 individual cells from the Arabidopsis root provides transcriptomes and marker genes for a diversity of cell types and illuminates the gene expression changes that occur across endodermis development.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genetic_diversity_patterns_and_domestication_origin_of_soybean_45029693/">
          <h4 class="font-weight-bold">Genetic Diversity Patterns and Domestication Origin of Soybean, bioRxiv, 2018-07-14</h4>
          <p>AbstractUnderstanding diversity and evolution of a crop is an essential step to implement a strategy to expand its germplasm base for crop improvement research. Samples intensively collected from Korea, which is a small but central region in the distribution geography of soybean, were genotyped to provide sufficient data to underpin genome-wide population genetic questions. After removing natural hybrids and duplicated or redundant accessions, we obtained a non-redundant set comprising 1,957 domesticated and 1,079 wild accessions to perform population structure analyses. Our analysis demonstrates that while wild soybean germplasm will require additional sampling from diverse indigenous areas to expand the germplasm base, the current domesticated soybean germplasm is saturated in terms of genetic diversity. We then showed that our genome-wide polymorphism map enabled us to detect genetic loci underling flower color, seed-coat color, and domestication syndrome. A representative soybean set consisting of 194 accessions were divided into one domesticated subpopulation and four wild subpopulations that could be traced back to their geographic collection areas. Population genomics analyses suggested that the monophyletic group of domesticated soybeans was originated in eastern Japan. The results were further substantiated by a phylogenetic tree constructed from domestication-associated single nucleotide polymorphisms identified in this study.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/crispr-cas9_interference_in_cassava_linked_to_the_evolution_of_editing-resistant_geminiviruses_40711777/">
          <h4 class="font-weight-bold">CRISPR-Cas9 interference in cassava linked to the evolution of editing-resistant geminiviruses, bioRxiv, 2018-05-04</h4>
          <p>ABSTRACTWe used CRISPR-Cas9 in the staple food crop cassava with the aim of engineering resistance to African cassava mosaic virus, a member of a widespread and important family of plant-pathogenic DNA viruses. We found that between 33 and 48% of edited virus genomes evolved a conserved single-nucleotide mutation that confers resistance to CRISPR-Cas9 cleavage. Our study highlights the potential for virus escape from this technology. Care should be taken to design CRISPR-Cas9 experiments that minimize the risk of virus escape.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/lrx-_and_fer-dependent_extracellular_sensing_coordinates_vacuolar_size_for_cytosol_homeostasis_30164181/">
          <h4 class="font-weight-bold">LRX- and FER-dependent extracellular sensing coordinates vacuolar size for cytosol homeostasis, bioRxiv, 2017-12-09</h4>
          <p>Cellular elongation requires the defined coordination of intra- and extracellular processes. The vacuole is the biggest plant organelle and its dimension has a role in limiting cell expansion (LÃ¶fke et al., 2015; Scheuring et al., 2016). We reveal that the increase in vacuolar occupancy enables cellular elongation with relatively little enlargement of the cytosole. It remains, however, completely unknown how the vacuolar size is coordinated with other growth-relevant processes. Intriguingly, we show that extracellular constraints impact on the intracellular expansion of the vacuole. The underlying cell wall sensing mechanism requires the interaction of the extracellular leucine-rich repeat extensin (LRX) with the receptor-like kinase Feronia (FER). Our data suggests that LRX links the plasma membrane localised FER with the cell wall, allowing this module to jointly sense and convey extracellular signals to the underlying cell. This mechanism coordinates cell wall acidificationloosening with the increase in vacuolar size, contributing cytosol homeostasis during plant cell expansion.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/k-mer_grammar_uncovers_maize_regulatory_architecture_29241814/">
          <h4 class="font-weight-bold">k-mer grammar uncovers maize regulatory architecture, bioRxiv, 2017-12-06</h4>
          <p>ABSTRACTOnly a small percentage of the genome sequence is involved in regulation of gene expression, but to biochemically identify this portion is expensive and laborious. In species like maize, with diverse intergenic regions and lots of repetitive elements, this is an especially challenging problem. While regulatory regions are rare, they do have characteristic chromatin contexts and sequence organization (the grammar) with which they can be identified. We developed a computational framework to exploit this sequence arrangement. The models learn to classify regulatory regions based on sequence features - k-mers. To do this, we borrowed two approaches from the field of natural language processing (1) âbag-of-wordsâ which is commonly used for differentially weighting key words in tasks like sentiment analyses, and (2) a vector-space model using word2vec (vector-k-mers), that captures semantic and linguistic relationships between words. We built âbag-of-k-mersâ and âvector-k-mersâ models that distinguish between regulatory and non-regulatory regions with an accuracy above 90%. Our âbag-of-k-mersâ achieved higher overall accuracy, while the âvector-k-mersâ models were more useful in highlighting key groups of sequences within the regulatory regions. These models now provide powerful tools to annotate regulatory regions in other maize lines beyond the reference, at low cost and with high accuracy.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/plant-biology/" rel="first" class="page-link">Â« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/plant-biology/page/3/" rel="prev" class="page-link">â¹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="/audiences/tags/plant-biology/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/plant-biology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/plant-biology/page/3/" class="page-link">3</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/plant-biology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/plant-biology/page/5/" class="page-link">5</a></li>
            
        

        
        <li class="page-item"><a href="/audiences/tags/plant-biology/page/5/" rel="next" class="page-link">Next âº</a></li>
        

        
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
