<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/charting_the_native_architecture_of_thylakoid_membranes_with_single-molecule_precision_65988221/">
          <h4 class="font-weight-bold">Charting the native architecture of thylakoid membranes with single-molecule precision, bioRxiv, 2019-09-05</h4>
          <p>Thylakoid membranes scaffold an assortment of large protein complexes that work together to harness the energy of light to produce oxygen, NADPH, and ATP. It has been a longstanding challenge to visualize how the intricate thylakoid network organizes these protein complexes to finely tune the photosynthetic reactions. Using cryo-electron tomography to analyze membrane surface topology, we have mapped the native molecular landscape of thylakoid membranes within green algae cells. Our tomograms provide insights into the molecular forces that drive thylakoid stacking and reveal that photosystems I and II are strictly segregated at the borders between appressed and non-appressed membrane domains. This new approach to charting thylakoid topology lays the foundation for dissecting photosynthetic regulation at the level of single protein complexes within the cell.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/comparison_of_bibliographic_data_sources_implications_for_the_robustness_of_university_rankings_65761508/">
          <h4 class="font-weight-bold">Comparison of bibliographic data sources Implications for the robustness of university rankings, bioRxiv, 2019-09-01</h4>
          <p>AbstractUniversities are increasingly evaluated, both internally and externally on the basis of their outputs. Often these are converted to simple, and frequently contested, rankings based on quantitative analysis of those outputs. These rankings can have substantial implications for student and staff recruitment, research income and perceived prestige of a university. Both internal and external analyses usually rely on a single data source to define the set of outputs assigned to a specific university. Although some differences between such databases are documented, few studies have explored them at the institutional scale and examined the implications of these differences for the metrics and rankings that are derived from them. We address this gap by performing detailed bibliographic comparisons between three key databases Web of Science (WoS), Scopus and, the recently relaunched Microsoft Academic (MSA). We analyse the differences between outputs with DOIs identified from each source for a sample of 155 universities and supplement this with a detailed manual analysis of the differences for fifteen universities. We find significant differences between the sources at the university level. Sources differ in the publication year of specific objects, the completeness of metadata, as well as in their coverage of disciplines, outlets, and publication type. We construct two simple rankings based on citation counts and open access status of the outputs for these universities and show dramatic changes in position based on the choice of bibliographic data sources. Those universities that experience the largest changes are frequently those from non-English speaking countries and those that are outside the top positions in international university rankings. Overall MSA has greater coverage than Scopus or WoS, but has less complete affiliation metadata. We suggest that robust evaluation measures need to consider the effect of choice of data sources and recommend an approach where data from multiple sources is integrated to provide a more robust dataset.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/scientific-communication-and-education">scientific-communication-and-education</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/interspecies_transcriptome_analyses_identify_genes_that_control_the_development_and_evolution_of_limb_skeletal_proportion_65742453/">
          <h4 class="font-weight-bold">Interspecies transcriptome analyses identify genes that control the development and evolution of limb skeletal proportion, bioRxiv, 2019-09-01</h4>
          <p>AbstractDespite the great diversity of vertebrate limb proportion and our deep understanding of the genetic mechanisms that drive skeletal elongation, little is known about how individual bones reach different lengths in any species. Here, we directly compare the transcriptomes of homologous growth cartilages of the mouse (Mus musculus) and bipedal jerboa (Jaculus jaculus), which has extremely long metatarsals of the feet and ‘mouse-like’ arms. When we intersected gene expression differences in metatarsals of the two species with expression differences in forearms, we found that about 10% of all orthologous genes are associated with disproportionate elongation of jerboa feet. Among these, Shox2, has gained expression in jerboa metatarsals where it is not expressed in other vertebrates that have been assessed. This transcription factor is necessary for proximal limb elongation, and we show that it is sufficient to increase mouse distal limb length. Unexpectedly, we also found evidence that jerboa foot elongation occurs in part by releasing latent growth potential that is repressed in mouse feet. In jerboa metatarsals, we observed higher expression of Crabp1, an antagonist of growth inhibitory retinoic acid, lower expression of Gdf10, an inhibitory TGFβ ligand, and lower expression of Mab21L2, a BMP signaling inhibitor that we show is sufficient to reduce limb bone elongation. By intersecting our data with prior expression analyses in other systems, we identify mechanisms that may both establish limb proportion during development and diversify proportion during evolution. The genes we identified here therefore provide a framework to understand the modular genetic control of skeletal growth and the remarkable malleability of vertebrate limb proportion.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/assessing_the_potential_of_environmental_dna_metabarcoding_for_monitoring_neotropical_mammals_a_case_study_in_the_amazon_and_atlantic_forest_brazil_65710239/">
          <h4 class="font-weight-bold">Assessing the potential of environmental DNA metabarcoding for monitoring Neotropical mammals a case study in the Amazon and Atlantic Forest, Brazil, bioRxiv, 2019-08-31</h4>
          <p>AbstractThe application of environmental DNA (eDNA) metabarcoding as a biomonitoring tool has greatly increased in the last decade. However, most studies have focused on aquatic macro-organisms in temperate areas (e.g., fishes). We apply eDNA metabarcoding to detect the mammalian community in two high-biodiversity regions of Brazil, the Amazon and Atlantic Forest. We identified critically endangered and endangered mammalian species in the Atlantic Forest and Amazon respectively and found congruence with species identified via camera trapping in the Atlantic Forest. In light of our results, we highlight the potential and challenges of eDNA monitoring for mammals in these high biodiverse areas.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/insight_into_the_genomic_history_of_the_near_east_from_whole-genome_sequences_and_genotypes_of_yemenis_65613878/">
          <h4 class="font-weight-bold">Insight into the genomic history of the Near East from whole-genome sequences and genotypes of Yemenis, bioRxiv, 2019-08-29</h4>
          <p>AbstractWe report high-coverage whole-genome sequencing data from 46 Yemeni individuals as well as genome-wide genotyping data from 169 Yemenis from diverse locations. We use this dataset to define the genetic diversity in Yemen and how it relates to people elsewhere in the Near East. Yemen is a vast region with substantial cultural and geographic diversity, but we found little genetic structure correlating with geography among the Yemenis – probably reflecting continuous movement of people between the regions. African ancestry from admixture in the past 800 years is widespread in Yemen and is the main contributor to the country’s limited genetic structure, with some individuals in Hudayda and Hadramout having up to 20% of their genetic ancestry from Africa. In contrast, individuals from Maarib appear to have been genetically isolated from the African gene flow and thus have genomes likely to reflect Yemen’s ancestry before the admixture. This ancestry was comparable to the ancestry present during the Bronze Age in the distant Northern regions of the Near East. After the Bronze Age, the South and North of the Near East therefore followed different genetic trajectories in the North the Levantines admixed with a Eurasian population carrying steppe ancestry whose impact never reached as far south as the Yemen, where people instead admixed with Africans leading to the genetic structure observed in the Near East today.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/an_easy-to-assemble_robust_and_lightweight_drive_implant_for_chronic_tetrode_recordings_in_freely_moving_animals_65414360/">
          <h4 class="font-weight-bold">An easy-to-assemble, robust, and lightweight drive implant for chronic tetrode recordings in freely moving animals, bioRxiv, 2019-08-25</h4>
          <p>AbstractTetrode arrays are the gold-standard method for neuronal recordings in many studies with behaving animals, especially for deep structures and chronic recordings. Here we outline an improved drive design for use in freely behaving animals. Our design makes use of recently developed technologies to reduce the complexity and build time of the drive while maintaining a low weight. The design also presents an improvement over many existing designs in terms of robustness and ease of use. We describe two variants a 16 tetrode implant weighing ∼2 g for mice, bats, tree shrews and similar animals, and a 64 tetrode implant weighing ∼16 g for rats, and similar animals.These designs were co-developed and optimized alongside a new class of drive-mounted feature-rich amplifier boards with ultra-thin RF tethers, as described in an upcoming paper (Newman, Zhang et al., in prep). This design significantly improves the data yield of chronic electrophysiology experiments.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/12/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/10/" class="page-link">10</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/11/" class="page-link">11</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/12/" class="page-link">12</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/0-100-users/page/13/" class="page-link">13</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/14/" class="page-link">14</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/15/" class="page-link">15</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/16/" class="page-link">16</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/17/" class="page-link">17</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/14/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/123/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
