<!DOCTYPE html>
<html><head>
  
  <title>0-100-users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.68.3" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.95d2483cd3c5d2bf47c1f36307a456787d60ad958c5efe44db9afc47012b1b05b8e1f64f6ac291650439322a6c38e86eb7cd6254255516d857c23f299fb0e666.js" integrity="sha512-ldJIPNPF0r9HwfNjB6RWeH1grZWMXv5E25r8RwErGwW44fZPasKRZQQ5MipsOOhut81iVCVVFthXwj8pn7DmZg=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/on_the_adaptive_behavior_of_head-fixed_flies_navigating_in_two-dimensional_visual_virtual_reality_50722645/">
          <h4 class="font-weight-bold">On the adaptive behavior of head-fixed flies navigating in two-dimensional, visual virtual reality, bioRxiv, 2018-11-05</h4>
          <p>AbstractA navigating animal’s sensory experience is shaped not just by its surroundings, but by its movements within them, which in turn are influenced by its past experiences. Studying the intertwined roles of sensation, experience and directed action in navigation has been made easier by the development of virtual reality (VR) environments for head-fixed animals, which allow for quantitative measurements of behavior in well-controlled sensory conditions. VR has long featured in studies of Drosophila melanogaster, but these experiments have typically relied on one-dimensional (1D) VR, effectively allowing the fly to change only its heading in a visual scene, and not its position. Here we explore how flies navigate in a two-dimensional (2D) visual VR environment that more closely resembles their experience during free behavior. We show that flies’ interaction with landmarks in 2D environments cannot be automatically derived from their behavior in simpler 1D environments. Using a novel paradigm, we then demonstrate that flies in 2D VR adapt their behavior in a visual environment in response to optogenetically delivered appetitive and aversive stimuli. Much like free-walking flies after encounters with food, head-fixed flies respond to optogenetic activation of sugar-sensing neurons by initiating a local search behavior. Finally, by pairing optogenetic activation of heat-sensing cells to the flies’ presence near visual landmarks of specific shapes, we elicit selective learned avoidance of landmarks associated with aversive “virtual heat”. These head-fixed paradigms set the stage for an interrogation of fly brain circuitry underlying flexible navigation in complex visual environments.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/animal-behavior-and-cognition">animal-behavior-and-cognition</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/tartan_underlies_the_evolution_of_male_drosophila_genital_morphology_50816397/">
          <h4 class="font-weight-bold">tartan underlies the evolution of male Drosophila genital morphology, bioRxiv, 2018-11-05</h4>
          <p>AbstractMale genital structures are among the most rapidly evolving morphological traits and are often the only features that can distinguish closely related species. This process is thought to be driven by sexual selection and may reinforce species separation. However, while the genetic basis of many phenotypic differences have been identified, we still lack knowledge about the genes underlying evolutionary differences in male genital organs and organ size more generally. The claspers (surstyli) are periphallic structures that play an important role in copulation in insects. Here we show that natural variation in clasper size and bristle number between Drosophila mauritiana and D. simulans is caused by evolutionary changes in tartan (trn), which encodes a transmembrane leucine-rich repeat domain protein that mediates cell-cell interactions and affinity differences. There are no fixed amino acid differences in trn between D. mauritiana and D. simulans but differences in the expression of this gene in developing genitalia suggest cis-regulatory changes in trn underlie the evolution of clasper morphology in these species. Finally, analysis of reciprocal hemizyotes that are genetically identical, except for which species the functional allele of trn is from, determined that the trn allele of D. mauritiana specifies larger claspers with more bristles than the allele of D. simulans. Therefore we have identified the first gene underlying evolutionary change in the size of a male genital organ, which will help to better understand the rapid diversification of these structures and the regulation and evolution of organ size more broadly.Significance StatementThe morphology of male genital organs evolves rapidly driven by sexual selection. However, little is known about the genes underlying genitalia differences between species. Identifying these genes is key to understanding how sexual selection acts on development to produce rapid phenotypic change. We have found that the gene tartan underlies differences between male Drosophila mauritiana and D. simulans in the size and bristle number of the claspers - genital projections that grasp the female during copulation. Moreover, since tartan encodes a protein that is involved in cell affinity, this may represent a new developmental mechanism for morphological change. Therefore, our study provides new insights into genetic and developmental bases for the rapid evolution of male genitalia and organ size more generally.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_risk-reward_tradeoff_of_high_ribosome_production_in_proliferating_cells_50556077/">
          <h4 class="font-weight-bold">A risk-reward tradeoff of high ribosome production in proliferating cells, bioRxiv, 2018-10-31</h4>
          <p>AbstractTo achieve maximal growth, cells must manage a massive economy of ribosomal proteins (r-proteins) and RNAs (rRNAs), which are required to produce thousands of new ribosomes every minute. Although ribosomes are essential in all cells, disruptions to ribosome biogenesis lead to heterogeneous phenotypes. Here, we modeled these perturbations in Saccharomyces cerevisiae and show that challenges to ribosome biogenesis result immediately in acute loss of proteostasis (protein folding homeostasis). Imbalances in the synthesis of r-proteins and rRNAs lead to the rapid aggregation of newly synthesized orphan r-proteins and compromise essential cellular processes. In response, proteostasis genes are activated by an Hsf1-dependent stress response pathway that is required for recovery from r-protein assembly stress. Importantly, we show that exogenously bolstering the proteostasis network increases cellular fitness in the face of challenges to ribosome assembly, demonstrating the direct contribution of orphan r-proteins to cellular phenotypes. Our results highlight ribosome assembly as a linchpin of cellular homeostasis, representing a key proteostasis vulnerability for rapidly proliferating cells that may be compromised by diverse genetic, environmental, and xenobiotic conditions that generate orphan r-proteins.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/immediate_visualization_of_recombination_events_and_chromosome_segregation_defects_in_fission_yeast_meiosis_50555793/">
          <h4 class="font-weight-bold">Immediate visualization of recombination events and chromosome segregation defects in fission yeast meiosis, bioRxiv, 2018-10-31</h4>
          <p>AbstractSchizosaccharomyces pombe, also known as fission yeast, is an established model for studying chromosome biological processes. Over the years research employing fission yeast has made important contributions to our knowledge about chromosome segregation during meiosis, as well as meiotic recombination and its regulation. Quantification of meiotic recombination frequency is not a straightforward undertaking, either requiring viable progeny for a genetic plating assay, or relying on laborious Southern blot analysis of recombination intermediates. Neither of these methods lends itself to high-throughput screens to identify novel meiotic factors. Here, we establish visual assays novel to Sz. pombe for characterizing chromosome segregation and meiotic recombination phenotypes. Genes expressing red, yellow, andor cyan fluorophores from spore-autonomous promoters have been integrated into the fission yeast genomes, either close to the centromere of chromosome I to monitor chromosome segregation, or on the arm of chromosome III to form a genetic interval at which recombination frequency can be determined. The visual recombination assay allows straightforward and immediate assessment of the genetic outcome of a single meiosis by epi-fluorescence microscopy without requiring tetrad dissection. We also demonstrate that the recombination frequency analysis can be automatized by utilizing imaging flow cytometry to enable high-throughput screens. These assays have several advantages over traditional methods for analysing meiotic phenotypes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_emergence_of_multiple_retinal_cell_types_through_efficient_coding_of_natural_movies_50556496/">
          <h4 class="font-weight-bold">The emergence of multiple retinal cell types through efficient coding of natural movies, bioRxiv, 2018-10-31</h4>
          <p>AbstractOne of the most striking aspects of early visual processing in the retina is the immediate parcellation of visual information into multiple parallel pathways, formed by different retinal ganglion cell types each tiling the entire visual field. Existing theories of efficient coding have been unable to account for the functional advantages of such cell-type diversity in encoding natural scenes. Here we go beyond previous theories to analyze how a simple linear retinal encoding model with different convolutional cell types efficiently encodes naturalistic spatiotemporal movies given a fixed firing rate budget. We find that optimizing the receptive fields and cell densities of two cell types makes them match the properties of the two main cell types in the primate retina, midget and parasol cells, in terms of spatial and temporal sensitivity, cell spacing, and their relative ratio. Moreover, our theory gives a precise account of how the ratio of midget to parasol cells decreases with retinal eccentricity. Also, we train a nonlinear encoding model with a rectifying nonlinearity to efficiently encode naturalistic movies, and again find emergent receptive fields resembling those of midget and parasol cells that are now further subdivided into ON and OFF types. Thus our work provides a theoretical justification, based on the efficient coding of natural movies, for the existence of the four most dominant cell types in the primate retina that together comprise 70% of all ganglion cells.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/specialized_and_spatially_organized_coding_of_sensory_motor_and_cognitive_variables_in_midbrain_dopamine_neurons_50453798/">
          <h4 class="font-weight-bold">Specialized and spatially organized coding of sensory, motor, and cognitive variables in midbrain dopamine neurons, bioRxiv, 2018-10-29</h4>
          <p>There is increased appreciation that dopamine (DA) neurons in the midbrain respond not only to reward 1,2 and reward-predicting cues 1,3,4, but also to other variables such as distance to reward 5, movements 6–11 and behavioral choices 12–15. Based on these findings, a major open question is how the responses to these diverse variables are organized across the population of DA neurons. In other words, do individual DA neurons multiplex multiple variables, or are subsets of neurons specialized in encoding specific behavioral variables? The reason that this fundamental question has been difficult to resolve is that recordings from large populations of individual DA neurons have not been performed in a behavioral task with sufficient complexity to examine these diverse variables simultaneously. To address this gap, we used 2-photon calcium imaging through an implanted lens to record activity of &amp;gt;300 midbrain DA neurons in the VTA during a complex decision-making task. As mice navigated in a virtual reality (VR) environment, DA neurons encoded an array of sensory, motor, and cognitive variables. These responses were functionally clustered, such that subpopulations of neurons transmitted information about a subset of behavioral variables, in addition to encoding reward. These functional clusters were spatially organized, such that neighboring neurons were more likely to be part of the same cluster. Taken together with the topography between DA neurons and their projections, this specialization and anatomical organization may aid downstream circuits in correctly interpreting the wide range of signals transmitted by DA neurons.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/61/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/58/" class="page-link">58</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/59/" class="page-link">59</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/60/" class="page-link">60</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/61/" class="page-link">61</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/62/" class="page-link">62</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/63/" class="page-link">63</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/64/" class="page-link">64</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/65/" class="page-link">65</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/66/" class="page-link">66</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/63/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/123/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
