<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/integrating_long-range_connectivity_information_into_de_bruijn_graphs_20914627/">
          <h4 class="font-weight-bold">Integrating long-range connectivity information into de Bruijn graphs, bioRxiv, 2017-06-09</h4>
          <p>AbstractMotivationThe de Bruijn graph is a simple and efficient data structure that is used in many areas of sequence analysis including genome assembly, read error correction and variant calling. The data structure has a single parameter k, is straightforward to implement and is tractable for large genomes with high sequencing depth. It also enables representation of multiple samples simultaneously to facilitate comparison. However, unlike the string graph, a de Bruijn graph does not retain long range information that is inherent in the read data. For this reason, applications that rely on de Bruijn graphs can produce sub-optimal results given their input.ResultsWe present a novel assembly graph data structure the Linked de Bruijn Graph (LdBG). Constructed by adding annotations on top of a de Bruijn graph, it stores long range connectivity information through the graph. We show that with error-free data it is possible to losslessly store and recover sequence from a Linked de Bruijn graph. With assembly simulations we demonstrate that the LdBG data structure outperforms both the de Bruijn graph and the String Graph Assembler (SGA). Finally we apply the LdBG to Klebsiella pneumoniae short read data to make large (12 kbp) variant calls, which we validate using PacBio sequencing data, and to characterise the genomic context of drug-resistance genes.AvailabilityLinked de Bruijn Graphs and associated algorithms are implemented as part of McCortex, available under the MIT license at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httphttpsgithub.commcveanmccortex&gt;httpsgithub.commcveanmccortex&lt;jatsext-link&gt;.Contactturner.isaac@gmail.com.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genome-wide_quantification_of_the_effects_of_dna_methylation_on_human_gene_regulation_20880241/">
          <h4 class="font-weight-bold">Genome-wide quantification of the effects of DNA methylation on human gene regulation, bioRxiv, 2017-06-08</h4>
          <p>AbstractChanges in DNA methylation are important in development and disease, but not all regulatory elements act in a methylation-dependent (MD) manner. Here, we developed mSTARR-seq, a high-throughput approach to quantify the effects of DNA methylation on regulatory element function. We assay MD activity in 14% of the euchromatic human genome, identify 2,143 MD regulatory elements, and predict MD activity using sequence and chromatin state information. We identify transcription factors associated with higher activity in unmethylated or methylated states, including an association between pioneer transcription factors and methylated DNA. Finally, we use mSTARR-seq to predict DNA methylation-gene expression correlations in primary cells. Our findings provide a map of MD regulatory activity across the human genome, facilitating interpretation of the many emerging associations between methylation and trait variation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/non-invasive_laminar_inference_with_meg_comparison_of_methods_and_source_inversion_algorithms_20883873/">
          <h4 class="font-weight-bold">Non-invasive laminar inference with MEG Comparison of methods and source inversion algorithms, bioRxiv, 2017-06-08</h4>
          <p>AbstractMagnetoencephalography (MEG) is a direct measure of neuronal current flow; its anatomical resolution is therefore not constrained by physiology but rather by data quality and the models used to explain these data. Recent simulation work has shown that it is possible to distinguish between signals arising in the deep and superficial cortical laminae given accurate knowledge of these surfaces with respect to the MEG sensors. This previous work has focused around a single inversion scheme (multiple sparse priors) and a single global parametric fit metric (free energy). In this paper we use several different source inversion algorithms and both local and global, as well as parametric and non-parametric fit metrics in order to demonstrate the robustness of the discrimination between layers. We find that only algorithms with some sparsity constraint can successfully be used to make laminar discrimination. Importantly, local t-statistics, global cross-validation and free energy all provide robust and mutually corroborating metrics of fit. We show that discrimination accuracy is affected by patch size estimates, cortical surface features, and lead field strength, which suggests several possible future improvements to this technique. This study demonstrates the possibility of determining the laminar origin of MEG sensor activity, and thus directly testing theories of human cognition that involve laminar- and frequency- specific mechanisms. This possibility can now be achieved using recent developments in high precision MEG, most notably the use of subject-specific head-casts, which allow for significant increases in data quality and therefore anatomically precise MEG recordings.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/resetting_the_yeast_epigenome_with_human_nucleosomes_20896431/">
          <h4 class="font-weight-bold">Resetting the yeast epigenome with human nucleosomes, bioRxiv, 2017-06-08</h4>
          <p>SummaryHumans and yeast are separated by a billion years of evolution, yet their conserved core histones retain central roles in gene regulation. Here, we “reset” yeast to use core human nucleosomes in lieu of their own, an exceedingly rare event which initially took twenty days. The cells adapt, however, by acquiring suppressor mutations in cell-division genes, or by acquiring certain aneuploidy states. Robust growth was also restored by converting five histone residues back to their yeast counterparts. We reveal that humanized nucleosomes in yeast are positioned according to endogenous yeast DNA sequence and chromatin-remodeling network, as judged by a yeast-like nucleosome repeat length. However, human nucleosomes have higher DNA occupancy and reduce RNA content. Adaptation to new biological conditions presented a special challenge for these cells due to slower chromatin remodeling. This humanized yeast poses many fundamental new questions about the nature of chromatin and how it is linked to many cell processes, and provides a platform to study histone variants via yeast epigenome reprogramming.Highlights&lt;jatslist list-type=simple&gt;&lt;jatslist-item&gt;- Only 1 in 107 yeast survive with fully human nucleosomes, but they rapidly evolve&lt;jatslist-item&gt;&lt;jatslist-item&gt;- Nucleosome positioning and nucleosome repeat length is not influenced by histone type&lt;jatslist-item&gt;&lt;jatslist-item&gt;- Human nucleosomes remodel slowly and delay yeast environmental adaptation&lt;jatslist-item&gt;&lt;jatslist-item&gt;- Human core nucleosomes are more repressive and globally reduce transcription in yeast&lt;jatslist-item&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/synthetic-biology">synthetic-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/distinct_neuronal_activity_patterns_induce_different_gene_expression_programs_20841401/">
          <h4 class="font-weight-bold">Distinct neuronal activity patterns induce different gene expression programs, bioRxiv, 2017-06-06</h4>
          <p>SUMMARYBrief and sustained neuronal activity patterns can have opposite effects on synaptic strength that both require activity-regulated gene (ARG) expression. However, whether distinct patterns of activity induce different sets of ARGs is unknown. In genome-scale experiments, we reveal that a neuron’s activity-pattern history can be predicted from the ARGs it expresses. Surprisingly, brief activity selectively induces a small subset of the ARG program that that corresponds precisely to the first of three temporal waves of genes induced by sustained activity. These first-wave genes are distinguished by an open chromatin state, proximity to rapidly activated enhancers, and a requirement for MAPKERK signaling for their induction. MAPKERK mediates rapid RNA polymerase recruitment to promoters, as well as enhancer RNA induction but not histone acetylation at enhancers. Thus, the same mechanisms that establish the multi-wave temporal structure of ARG induction also enable different sets of genes to be induced by distinct activity patterns.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genetic_identification_of_brain_cell_types_underlying_schizophrenia_20783477/">
          <h4 class="font-weight-bold">Genetic identification Of brain cell types underlying schizophrenia, bioRxiv, 2017-06-03</h4>
          <p>AbstractWith few exceptions, the marked advances in knowledge about the genetic basis for schizophrenia have not converged on findings that can be confidently used for precise experimental modeling. Applying knowledge of the cellular taxonomy of the brain from single-cell RNA-sequencing, we evaluated whether the genomic loci implicated in schizophrenia map onto specific brain cell types. The common variant genomic results consistently mapped to pyramidal cells, medium spiny neurons, and certain interneurons but far less consistently to embryonic, progenitor, or glial cells. These enrichments were due to distinct sets of genes specifically expressed in each of these cell types. Many of the diverse gene sets associated with schizophrenia (including antipsychotic targets) implicate the same brain cell types. Our results provide a parsimonious explanation the common-variant genetic results for schizophrenia point at a limited set of neurons, and the gene sets point to the same cells. While some of the genetic risk is associated with GABAergic interneurons, this risk largely does not overlap with that from projecting cells.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2017">2017</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/0-100-users/page/102/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/99/" class="page-link">99</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/100/" class="page-link">100</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/101/" class="page-link">101</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/102/" class="page-link">102</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/0-100-users/page/103/" class="page-link">103</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/104/" class="page-link">104</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/105/" class="page-link">105</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/106/" class="page-link">106</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/107/" class="page-link">107</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/0-100-users/page/104/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/0-100-users/page/123/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
