<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/environmental_dna_edna_metabarcoding_of_pond_water_as_a_tool_to_survey_conservation_and_management_priority_mammals_55274002/">
          <h4 class="font-weight-bold">Environmental DNA (eDNA) metabarcoding of pond water as a tool to survey conservation and management priority mammals, bioRxiv, 2019-02-11</h4>
          <p>Abstract&lt;jatslist list-type=order&gt;&lt;jatslist-item&gt;Environmental DNA (eDNA) metabarcoding is largely used to survey aquatic communities, but can also provide data on terrestrial taxa utilising aquatic habitats. However, the entry, dispersal, and detection of terrestrial species’ DNA within waterbodies is understudied.&lt;jatslist-item&gt;&lt;jatslist-item&gt;We evaluated eDNA metabarcoding of pond water for monitoring semi-aquatic, ground-dwelling, and arboreal mammals, and examined spatiotemporal variation in mammal eDNA signals using experiments in captive and wild conditions.&lt;jatslist-item&gt;&lt;jatslist-item&gt;We selected nine focal species of conservation and management concern European water vole, European otter, Eurasian beaver, European hedgehog, European badger, red deer, Eurasian lynx, red squirrel, and European pine marten. We hypothesised that eDNA signals (i.e. proportional read counts) would be stronger for semi-aquatic than terrestrial species, and at sites where mammals exhibited behaviours (e.g. swimming, urination). We tested this by sampling waterbodies in enclosures of captive focal species at specific sites where behaviours had been observed (‘directed’ sampling) and at equidistant intervals along the shoreline (‘stratified’ sampling). We then surveyed natural ponds (N = 6) where focal species were present using stratified water sampling, camera traps, and field signs. eDNA samples were metabarcoded using vertebrate-specific primers.&lt;jatslist-item&gt;&lt;jatslist-item&gt;All focal species were detected in captivity. eDNA signal strength did not differ between directed and stratified samples across or within species, between species lifestyles (i.e. semi-aquatic, ground-dwelling, arboreal), or according to behaviours. Therefore, eDNA was evenly distributed within artificial waterbodies. Conversely, eDNA was unevenly distributed in natural ponds. eDNA metabarcoding, camera trapping, and field signs detected beaver, red deer, and roe deer. Badger and red fox were recorded with cameras and field signs, but not eDNA metabarcoding. However, eDNA metabarcoding detected small mammals missed by cameras and field signs, e.g. water vole. Terrestrial mammal eDNA signals were weaker and detected in fewer samples than semi-aquatic mammal eDNA signals.&lt;jatslist-item&gt;&lt;jatslist-item&gt;eDNA metabarcoding has potential for inclusion in mammal monitoring schemes by enabling large-scale, multi-species distribution assessment for priority and difficult to survey species, and could provide early indication of range expansions or contractions. However, eDNA surveys need high spatiotemporal resolution and metabarcoding biases require further investigation before this tool is routinely implemented.&lt;jatslist-item&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/tissue_structure_accelerates_evolution_premalignant_sweeps_precede_neutral_expansion_55267528/">
          <h4 class="font-weight-bold">Tissue structure accelerates evolution premalignant sweeps precede neutral expansion, bioRxiv, 2019-02-11</h4>
          <p>Cancer has been hypothesized to be a caricature of the renewal process of the tissue of origin arising from (and maintained by) small subpopulations capable of continuous growth1. The strong influence of the tissue structure has been convincingly demonstrated in intestinal cancers where adenomas grow by the fission of stem-cell-maintained glands influenced by early expression of abnormal cell mobility in cancer progenitors2, 3. So-called “born to be bad” tumors arise from progenitors which may already possess the necessary driver mutations for malignancy4, 5 and metastasis6. These tumors subsequently evolve neutrally, thereby maximizing intratumoral heterogeneity and increasing the probability of therapeutic resistance. These findings have been nuanced by the advent of multi-region sequencing, which uses spatial and temporal patterns of genetic variation among competing tumor cell populations to shed light on the mode of tumor evolution (neutral or Darwinian) and also the tempo4, 7–11. Using a classic, well-studied model of tumor evolution (a passenger-driver mutation model12–16) we systematically alter spatial constraints and cell mixing rates to show how tissue structure influences functional (driver) mutations and genetic heterogeneity over time. This model approach explores a key mechanism behind both inter-patient and intratumoral tumor heterogeneity competition for space. Initial spatial constraints determine the emergent mode of evolution (Darwinian to neutral) without a change in cell-specific mutation rate or fitness effects. Driver acquisition during the Darwinian precancerous stage may be accelerated en route to neutral evolution by the combination of two factors spatial constraints and limited cellular mixing.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cancer-biology">cancer-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/socru_typing_of_genome_level_order_and_orientation_in_bacteria_55262268/">
          <h4 class="font-weight-bold">Socru Typing of genome level order and orientation in bacteria, bioRxiv, 2019-02-10</h4>
          <p>Genome rearrangements occur in bacteria between repeat sequences and impact growth and gene expression. Homologous recombination can occur between ribosomal operons, which are found in multiple copies in many bacteria. Inversion between indirect repeats and excisiontranslocation between direct repeats enable structural genome rearrangement. To identify what these rearrangements are by sequencing, reads of several thousand bases are required to span the ribosomal operons. With long read sequencing aiding the routine generation of complete bacterial assemblies, we have developed socru, a typing method for the order and orientation of genome fragments between ribosomal operons, defined against species-specific baselines. It allows for a single identifier to convey the order and orientation of genome level structure and 434 of the most common bacterial species are supported. Additionally, socru can be used to identify large scale misassemblies. Availability and implementation Socru is written in Python 3, runs on Linux and OSX systems and is available under the open source license GNU GPL 3 from httpsgithub.comquadram-institute-biosciencesocru.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/object_detection_networks_and_augmented_reality_for_cellular_detection_in_fluorescence_microscopy_acquisition_and_analysis_55202241/">
          <h4 class="font-weight-bold">Object Detection Networks and Augmented Reality for Cellular Detection in Fluorescence Microscopy Acquisition and Analysis, bioRxiv, 2019-02-09</h4>
          <p>AbstractIn this paper we demonstrate the application of object detection networks for the classification and localization of cells in fluorescence microscopy. We benchmark two leading object detection algorithms across multiple challenging 2-D microscopy datasets as well as develop and demonstrate an algorithm which can localize and image cells in 3-D, in real-time. Furthermore, we exploit the fast processing of these algorithms and develop a simple and effective Augmented Reality (AR) system for fluorescence microscopy systems. Object detection networks are well-known high performance networks famously applied to the task of identifying and localizing objects in photography images. Here we show their application and efficiency for localizing cells in fluorescence microscopy images. Object detection algorithms are typically trained on many thousands of images, which can be prohibitive within the biological sciences due to the cost of imaging and annotating large amounts of data. Through taking different cell types and assays as an example, we show that with some careful considerations it is possible to achieve very high performance with datasets with as few as 26 images present. Using our approach, it is possible for relatively non-skilled users to automate detection of cell classes with a variety of appearances and enable new avenues for automation of conventionally manual fluorescence microscopy acquisition pipelines.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_neurodevelopmental_origin_of_behavioral_individuality_54998417/">
          <h4 class="font-weight-bold">A neurodevelopmental origin of behavioral individuality, bioRxiv, 2019-02-06</h4>
          <p>The genome versus experience, or Nature versus Nurture, debate has dominated our understanding of individual behavioral variation. A third factor, namely variation in complex behavior potentially due to non-heritable developmental noise in brain development, has been largely ignored. Using the Drosophila vinegar fly we demonstrate a causal link between variation in brain wiring due to developmental noise, and behavioral individuality. A population of visual system neurons called DCNs shows non-heritable, inter-individual variation in rightleft wiring asymmetry, and control object orientation in freely walking flies. We show that DCN wiring asymmetry predicts individual object responses the greater the asymmetry, the better the individual orients. Silencing DCNs abolishes correlations between anatomy and behavior, while inducing visual asymmetry via monocular deprivation rescues object orientation in DCN-symmetric individuals.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/auxin_export_from_proximal_fruits_drives_arrest_in_competent_inflorescence_meristems_55042090/">
          <h4 class="font-weight-bold">Auxin export from proximal fruits drives arrest in competent inflorescence meristems, bioRxiv, 2019-02-06</h4>
          <p>A well-defined set of regulatory pathways control entry into the reproductive phase in flowering plants. Conversely, little is known about the mechanisms that control the end of the reproductive phase (floral arrest), despite this being a critical process for optimising fruit and seed production. Complete fruit removal or lack of fertile fruit-set in male sterile mutants, for example male sterile1 (ms1), prevents timely floral arrest in the model plant Arabidopsis. These observations formed the basis for Hensel and colleagues model in which end-of-flowering was proposed to result from a cumulative fruitseed-derived signal that caused simultaneous global proliferative arrest (GPA) in all inflorescences. Recent studies have suggested that end-of-flowering involves gene expression changes at the floral meristem which are at least in part controlled by the FRUITFULL-APETELA2 pathway, however there is limited understanding of how this process is controlled and the communication needed at the whole plant level. Here, we provide new information providing a framework for the fruit-to-meristem (F-M) communication implied by the GPA model. We show that floral arrest in Arabidopsis is not global and does not occur synchronously between branches, but rather that the arrest of each inflorescence is a local process, driven by auxin export from fruit proximal to the inflorescence meristem (IM). Furthermore, we show that inflorescence meristems are only competent for floral arrest once they reach a certain developmental age. Understanding the regulation of floral arrest is of major importance for the future manipulation of flowering to extend and maximise crop yields.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/48/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/45/" class="page-link">45</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/46/" class="page-link">46</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/47/" class="page-link">47</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/48/" class="page-link">48</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/0-100-users/page/49/" class="page-link">49</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/50/" class="page-link">50</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/51/" class="page-link">51</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/52/" class="page-link">52</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/0-100-users/page/53/" class="page-link">53</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/50/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/0-100-users/page/123/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
