<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/classification_of_single_particles_from_human_cell_extract_reveals_distinct_structures_31667604/">
          <h4 class="font-weight-bold">Classification of Single Particles from Human Cell Extract Reveals Distinct Structures, bioRxiv, 2018-01-15</h4>
          <p>SummaryMulti-protein complexes are necessary for nearly all cellular processes, and understanding their structure is required for elucidating their function. Current high-resolution strategies in structural biology are effective, but lag behind other fields (e.g. genomics and proteomics) due to their reliance on purified samples rather than characterizing heterogeneous mixtures. Here, we present a method combining single particle analysis by electron microscopy with protein identification by mass spectrometry to structurally characterize macromolecular complexes from extracts of human cells. We obtain three-dimensional structures of native proteasomes directly from ab initio classification of a heterogeneous mixture of protein complexes. In addition, we find an ~1 MDa size structure of unknown composition and reference our proteomics data to suggest possible identities. Our study shows the power of using a shotgun approach to electron microscopy (shotgun EM) when coupled with mass spectrometry as a tool to uncover the structures of macromolecular machines in parallel.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biochemistry">biochemistry</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/precise_temporal_regulation_of_alternative_splicing_during_neural_development_31667630/">
          <h4 class="font-weight-bold">Precise temporal regulation of alternative splicing during neural development, bioRxiv, 2018-01-15</h4>
          <p>AbstractAlternative splicing (AS) is a crucial step of gene expression that must be tightly controlled, but the precise timing of dynamic splicing switches during neural development and the underlying regulatory mechanisms are poorly understood. Here we systematically analyzed the temporal regulation of AS in a large number of transcriptome profiles of developing mouse cortices, in vivo purified neuronal subtypes, and neurons differentiated in vitro. Our analysis revealed early- and late-switch exons in genes with distinct functions, and these switches accurately define neuronal maturation stages. Integrative modeling suggests that these switches are under direct and combinatorial regulation by distinct sets of neuronal RNA-binding proteins including Nova, Rbfox, Mbnl and Ptbp. Surprisingly, various neuronal subtypes in the sensory systems lack Nova andor Rbfox expression. These neurons retain the “immature” splicing program in early-switch exons, affecting numerous synaptic genes. These results provide new insights into the organization and regulation of the neurodevelopmental transcriptome.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/a_systematic_review_of_drosophila_short-term-memory_genetics_meta-analysis_reveals_robust_reproducibility_31639518/">
          <h4 class="font-weight-bold">A systematic review of Drosophila short-term-memory genetics meta-analysis reveals robust reproducibility, bioRxiv, 2018-01-14</h4>
          <p>AbstractGeneticists use olfactory conditioning in Drosophila to identify learning genes; however, little is known about how these genes are integrated into short-term memory (STM) pathways. Here, we investigated the hypothesis that the STM evidence base is weak. We performed systematic review and meta-analysis of the field. Using metrics to quantify variation between discovery articles and follow-up studies, we found that seven genes were both highly replicated, and highly reproducible. However, ~80% of STM genes have never been replicated. While only a few studies investigated interactions, the reviewed genes could account for &amp;gt;1000% memory. This large summed effect size could indicate irreproducibility, many shared pathways, or that current assay protocols lack the specificity needed to identify core plasticity genes. Mechanistic theories of memory will require the convergence of evidence from system, circuit, cellular, molecular, and genetic experiments; systematic data synthesis is an essential tool for integrated neuroscience.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/animal-behavior-and-cognition">animal-behavior-and-cognition</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/high_accuracy_haplotype-derived_allele_frequencies_from_ultra-low_coverage_pool-seq_samples_31576116/">
          <h4 class="font-weight-bold">High accuracy haplotype-derived allele frequencies from ultra-low coverage pool-seq samples, bioRxiv, 2018-01-12</h4>
          <p>AbstractEvolve-and-resequence experiments leverage next-generation sequencing technology to track allele frequency dynamics of populations as they evolve. While previous work has shown that adaptive alleles can be detected by comparing frequency trajectories from many replicate populations, this power comes at the expense of high-coverage (&amp;gt;100x) sequencing of many pooled samples, which can be cost-prohibitive. Here we show that accurate estimates of allele frequencies can be achieved with very shallow sequencing depths (&amp;lt;5x) via inference of known founder haplotypes in small genomic windows. This technique can be used to efficiently estimate frequencies for any number of alleles in any model system. Using both experimentally-pooled and simulated samples of Drosophila melanogaster, we show that haplotype inference can improve allele frequency accuracy by orders of magnitude, and that high accuracy is maintained after up to 200 generations of recombination, even in the presence of missing data or incomplete founder knowledge. By reducing sequencing costs without sacrificing accuracy, our method enables analysis of samples from more time-points and replicates, increasing the statistical power to detect adaptive alleles.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/widespread_and_targeted_gene_expression_by_systemic_aav_vectors_production_purification_and_administration_31576148/">
          <h4 class="font-weight-bold">Widespread and targeted gene expression by systemic AAV vectors Production, purification, and administration, bioRxiv, 2018-01-12</h4>
          <p>ABSTRACTWe recently developed novel AAV capsids for efficient and noninvasive gene transfer across the central and peripheral nervous systems. In this protocol, we describe how to produce and systemically administer AAV-PHP viruses to label andor genetically manipulate cells in the mouse nervous system and organs including the heart. The procedure comprises three separate stages AAV production, intravenous delivery, and evaluation of transgene expression. The protocol spans eight days, excluding the time required to assess gene expression, and can be readily adopted by laboratories with standard molecular and cell culture capabilities. We provide guidelines for experimental design and choosing the capsid, cargo, and viral dose appropriate for the experimental aims. The procedures outlined here are adaptable to diverse biomedical applications, from anatomical and functional mapping to gene expression, silencing, and editing.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/a_metaproteomics_method_to_determine_carbon_sources_and_assimilation_pathways_of_species_in_microbial_communities_31461621/">
          <h4 class="font-weight-bold">A metaproteomics method to determine carbon sources and assimilation pathways of species in microbial communities, bioRxiv, 2018-01-10</h4>
          <p>AbstractMeasurements of the carbon stable isotope ratio (δ13C) are widely used in biology to address major questions regarding food sources and metabolic pathways used by organisms. Measurement of these so called stable carbon isotope fingerprints (SIFs) for microbes involved in biogeochemical cycling and microbiota of plants and animals have led to major discoveries in environmental microbiology. Currently, obtaining SIFs for microbial communities is challenging as the available methods either only provide limited taxonomic resolution, such as with the use of lipid biomarkers, or are limited in throughput, such as NanoSIMS imaging of single cells.Here we present “direct Protein-SIF” and the Calis-p software package (&lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpssourceforge.netprojectscalis-p&gt;httpssourceforge.netprojectscalis-p&lt;jatsext-link&gt;), which enable high-throughput measurements of accurate δ13C values for individual species within a microbial community. We benchmark the method using 20 pure culture microorganisms and show that the method reproducibly provides SIF values consistent with gold standard bulk measurements performed with an isotope ratio mass spectrometer. Using mock community samples, we show that SIF values can also be obtained for individual species within a microbial community. Finally, a case study of an obligate bacteria-animal symbiosis showed that direct Protein-SIF confirms previous physiological hypotheses and can provide unexpected new insights into the symbionts’ metabolism. This confirms the usefulness of this new approach to accurately determine δ13C values for different species in microbial community samples.&lt;jatssec id=s1&gt;SignificanceTo understand the roles that microorganisms play in diverse environments such as the open ocean and the human intestinal tract, we need an understanding of their metabolism and physiology. A variety of methods such as metagenomics and metaproteomics exist to assess the metabolism of environmental microorganisms based on gene content and gene expression. These methods often only provide indirect evidence for which substrates are used by a microorganism in a community. The direct Protein-SIF method that we developed allows linking microbial species in communities to the environmental carbon sources they consume by determining their stable carbon isotope signature. Direct Protein-SIF also allows assessing which carbon fixation pathway is used by autotrophic microorganisms that directly assimilate CO2.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/tags/0-100-users/page/48/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/45/" class="page-link">45</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/46/" class="page-link">46</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/47/" class="page-link">47</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/48/" class="page-link">48</a></li>
            
        
             
            <li class="page-item active"><a href="/tags/0-100-users/page/49/" class="page-link">49</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/50/" class="page-link">50</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/51/" class="page-link">51</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/52/" class="page-link">52</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/0-100-users/page/53/" class="page-link">53</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/0-100-users/page/50/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/0-100-users/page/64/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
