<!DOCTYPE html>
<html><head>
  
  <title>2019 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/non-homologous_end_joining_minimizes_errors_by_coordinating_dna_processing_with_ligation_56200760/">
          <h4 class="font-weight-bold">Non-homologous end joining minimizes errors by coordinating DNA processing with ligation, bioRxiv, 2019-03-01</h4>
          <p>Genome stability requires efficient and faithful repair of DNA double-strand breaks (DSBs). The predominant DSB repair pathway in human cells is non-homologous end-joining (NHEJ), which directly ligates DNA ends1–5. Broken DNA ends at DSBs are chemically diverse, and many are not compatible for direct ligation by the NHEJ-associated DNA Ligase IV (Lig4). To solve this problem, NHEJ end-processing enzymes including polymerases and nucleases modify ends until they are ligatable. How cells regulate end processing to minimize unnecessary genomic alterations6 during repair of pathological DSBs remains unknown. Using a biochemical system that recapitulates key features of cellular NHEJ, we previously demonstrated that DNA ends are initially tethered at a distance, followed by Lig4-mediated formation of a “short-range synaptic complex” in which DNA ends are closely aligned for ligation7. Here, we show that a wide variety of end-processing activities all depend on formation of the short-range complex. Moreover, using real-time single molecule imaging, we find that end processing occurs within the short-range complex. Confining end processing to the Lig4-dependent short-range synaptic complex promotes immediate ligation of compatible ends and ensures that incompatible ends are ligated as soon as they become compatible, thereby minimizing end processing. Our results elucidate how NHEJ exploits end processing to achieve versatility while minimizing errors that cause genome instability.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/slide-seq_a_scalable_technology_for_measuring_genome-wide_expression_at_high_spatial_resolution_56215967/">
          <h4 class="font-weight-bold">Slide-seq A Scalable Technology for Measuring Genome-Wide Expression at High Spatial Resolution, bioRxiv, 2019-03-01</h4>
          <p>The spatial organization of cells in tissue has a profound influence on their function, yet a high-throughput, genome-wide readout of gene expression with cellular resolution is lacking. Here, we introduce Slide-seq, a highly scalable method that enables facile generation of large volumes of unbiased spatial transcriptomes with 10 micron spatial resolution, comparable to the size of individual cells. In Slide-seq, RNA is transferred from freshly frozen tissue sections onto a surface covered in DNA-barcoded beads with known positions, allowing the spatial locations of the RNA to be inferred by sequencing. To demonstrate Slide-seq&#39;s utility, we localized cell types identified by large-scale scRNA-seq datasets within the cerebellum and hippocampus. We next systematically characterized spatial gene expression patterns in the Purkinje layer of mouse cerebellum, identifying new axes of variation across Purkinje cell compartments. Finally, we used Slide-seq to define the temporal evolution of cell-type-specific responses in a mouse model of traumatic brain injury. Slide-seq will accelerate biological discovery by enabling routine, high-resolution spatial mapping of gene expression.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_genomic_ecosystem_of_transposable_elements_in_maize_56197313/">
          <h4 class="font-weight-bold">The Genomic Ecosystem of Transposable Elements in Maize, bioRxiv, 2019-03-01</h4>
          <p>Transposable elements (TEs) constitute the majority of flowering plant DNA, reflecting their tremendous success in subverting, avoiding, and surviving the defenses of their host genomes to ensure their selfish replication. More than 85% of the sequence of the maize genome can be ascribed to past transposition, providing a major contribution to the structure of the genome. Evidence from individual loci has informed our understanding of how transposition has shaped the genome, and a number of individual TE insertions have been causally linked to dramatic phenotypic changes. But genome-wide analyses in maize and other taxa have frequently represented TEs as a relatively homogeneous class of fragmentary relics of past transposition, obscuring their evolutionary history and interaction with their host genome. Using an updated annotation of structurally intact TEs in the maize reference genome, we investigate the family-level ecological and evolutionary dynamics of TEs in maize. Integrating a variety of data, from descriptors of individual TEs like coding capacity, expression, and methylation, as well as similar features of the sequence they inserted into, we model the relationship between these attributes of the genomic environment and the survival of TE copies and families. Our analyses reveal a diversity of ecological strategies of TE families, each representing the evolution of a distinct ecological niche allowing survival of the TE family. In contrast to the wholesale relegation of all TEs to a single category of junk DNA, these differences generate a rich ecology of the genome, suggesting families of TEs that coexist in time and space compete and cooperate with each other. We conclude that while the impact of transposition is highly family- and context-dependent, a family-level understanding of the ecology of TEs in the genome can refine our ability to predict the role of TEs in generating genetic and phenotypic diversity.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_comprehensive_assessment_of_demographic_environmental_and_host_genetic_associations_with_gut_microbiome_diversity_in_healthy_individuals_56150428/">
          <h4 class="font-weight-bold">A Comprehensive Assessment of Demographic, Environmental and Host Genetic Associations with Gut Microbiome Diversity in Healthy Individuals, bioRxiv, 2019-02-28</h4>
          <p>Background The gut microbiome is an important determinant of human health. Its composition has been shown to be influenced by multiple environmental factors and likely by host genetic variation. In the framework of the Milieu Intérieur Consortium, a total of 1,000 healthy individuals of western European ancestry, with a 11 sex ratio and evenly stratified across five decades of life (age 20 - 69), were recruited. We generated 16S ribosomal RNA profiles from stool samples for 858 participants. We investigated genetic and non-genetic factors that contribute to individual differences in fecal microbiome composition.Results Among 110 demographic, clinical and environmental factors, 11 were identified as significantly correlated with α-diversity, β-diversity or abundance of specific microbial communities in multivariable models. Age and blood alanine aminotransferase levels showed the strongest associations with microbiome diversity. In total, all non-genetic factors explained 16.4% of the variance. We then searched for associations between &amp;gt;5 million single nucleotide polymorphisms and the same indicators of fecal microbiome diversity, including the significant non-genetic factors as covariates. No genome-wide significant associations were identified after correction for multiple testing. A small fraction of previously reported associations between human genetic variants and specific taxa could be replicated in our cohort, while no replication was observed for any of the diversity metrics.Conclusion In a well-characterized cohort of healthy individuals, we identified several non-genetic variables associated with fecal microbiome diversity. In contrast, host genetics only had a negligible influence. Demographic and environmental factors are thus the main contributors to fecal microbiome composition in healthy individuals.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/drug_screens_of_ngly1_deficiency_worm_and_fly_models_reveal_catecholamine_nrf2_and_anti-inflammatory_pathway_activation_as_clinical_approaches_56194351/">
          <h4 class="font-weight-bold">Drug screens of NGLY1 Deficiency worm and fly models reveal catecholamine, NRF2 and anti-inflammatory pathway activation as clinical approaches, bioRxiv, 2019-02-28</h4>
          <p>N-glycanase 1NGLY1 Deficiency is an ultra-rare and complex monogenic glycosylation disorder that affects fewer than 40 patients globally. NGLY1 Deficiency has been studied in model organisms such as yeast, worms, flies and mice. Proteasomal and mitochondrial homeostasisgene networks are controlled by the evolutionarily conserved transcriptional regulator Nrf1, whose activity requires deglycosylation by NGLY1. Hypersensitivity to the proteasome inhibitor bortezomib is a common phenotype observed in whole animal and cellular models of NGLY1Deficiency. Here we describe unbiased phenotypic drug screens to identify FDA approved drugs, generally recognized as safe natural products and novel chemical entities that rescue growth and development of NGLY1-deficient worm and fly larvae treated with a toxic dose of bortezomib. We used image-based larval size and number assays for use in screens of a 2,560-member drug repurposing library and a 20,240-member lead discovery library. A total of 91 validated hit compounds from primary invertebrate screens were tested in a human cell line in a NRF2 activity assay. NRF2 is a transcriptional regulator that regulates cellular redox homeostasis and it can compensate for loss of Nrf1. Plant-based polyphenols comprise the largest class of hit compounds and NRF2 inducers. Catecholamines and catecholamine receptor activators comprise the second largest class of hits. Steroidal and non-steroidal anti-inflammatory drugs comprise the third largest class. Only one compound was active in all assays and species the atypical antipsychotic and dopamine receptor agonist aripiprazole. Worm and fly models of NGLY1 Deficiency validate therapeutic rationales for activation of NRF2 and anti-inflammatory pathways based on results in mice and human cell models and suggest a novel therapeutic rationale for boosting catecholamine levels andor signaling in the brain.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/pharmacology-and-toxicology">pharmacology-and-toxicology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/feeding_kinematics_and_morphology_of_the_alligator_gar_atractosteus_spatula_lac%C3%A9p%C3%A8de_1803_feeding_mechanics_of_atractosteus_spatula_56154162/">
          <h4 class="font-weight-bold">Feeding Kinematics And Morphology Of The Alligator Gar (Atractosteus Spatula, Lacépède, 1803) Feeding Mechanics Of Atractosteus Spatula, bioRxiv, 2019-02-28</h4>
          <p>ABSTRACTModern (lepisosteid) gars are a small clade of seven species and two genera that occupy an important position on the actinopterygian phylogenetic tree as members of the Holostei (Amia &#43; gars), sister-group of the teleost radiation. Often referred to as “living fossils,” these taxa preserve many plesiomorphic characteristics used to interpret and reconstruct early osteichthyan feeding conditions. Less attention, however, has been paid to the functional implications of gar-specific morphology, thought to be related to an exclusively ram-based, lateral-snapping mode of prey capture. Previous studies of feeding kinematics in gars have focused solely on members of the narrow-snouted Lepisosteus genus, and here we expand that dataset to include a member of the broad-snouted sister-genus and largest species of gar, the alligator gar (Atractosteus spatula, Lacépède, 1803). High-speed videography reveals that the feeding system of alligator gars is capable of rapid expansion from anterior-to-posterior, precisely timed in a way that appears to counteract the effects of a bow-wave during ram-feeding and generate a unidirectional flow of water through the feeding system. Reconstructed cranial anatomy based on contrast-enhanced micro-CT data show that a lateral-sliding palatoquadrate, flexible intrasuspensorial joint, pivoting interhyal, and retractable pectoral girdle are all responsible for increasing the range of motion and expansive capabilities of the gar cranial linkage system. Muscular reconstructions and manipulation experiments show that, while the sternohyoideus is the primary input to the feeding system (similar to other “basal” actinopterygians), additional input from the hyoid constrictors and hypaxials play an important role in decoupling and modulating between the dual roles of the sternohyoideus hyoid retraction (jaw opening) and hyoid rotation (pharyngeal expansion) respectively. The data presented here demonstrate an intricate feeding mechanism, capable of precise control with plesiomorphic muscles, that represents one of the many ways the ancestral osteichthyan feeding mechanism has been modified for prey capture.RESEARCH HIGHLIGHTSAlligator gars use a surprisingly expansive cranial linkage system for prey capture that relies on specialized joints for increased mobility and is capable of precise modulation from anterior to posterior using plesiomorphic osteichthyan musculature.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/100/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/97/" class="page-link">97</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/98/" class="page-link">98</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/99/" class="page-link">99</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/100/" class="page-link">100</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/2019/page/101/" class="page-link">101</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/102/" class="page-link">102</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/103/" class="page-link">103</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/104/" class="page-link">104</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/105/" class="page-link">105</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/102/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/2019/page/135/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
