<!DOCTYPE html>
<html><head>
  
  <title>100 200 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/whole_organism_lineage_tracing_by_combinatorial_and_cumulative_genome_editing_7348322/">
          <h4 class="font-weight-bold">Whole organism lineage tracing by combinatorial and cumulative genome editing, bioRxiv, 2016-05-12</h4>
          <p>AbstractMulticellular systems develop from single cells through a lineage, but current lineage tracing approaches scale poorly to whole organisms. Here we use genome editing to progressively introduce and accumulate diverse mutations in a DNA barcode over multiple rounds of cell division. The barcode, an array of CRISPRCas9 target sites, records lineage relationships in the patterns of mutations shared between cells. In cell culture and zebrafish, we show that rates and patterns of editing are tunable, and that thousands of lineage-informative barcode alleles can be generated. We find that most cells in adult zebrafish organs derive from relatively few embryonic progenitors. Genome editing of synthetic target arrays for lineage tracing (GESTALT) will help generate large-scale maps of cell lineage in multicellular systems.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2016">2016</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/is_most_published_research_really_false_7043540/">
          <h4 class="font-weight-bold">Is most published research really false?, bioRxiv, 2016-04-28</h4>
          <p>AbstractThere has been an increasing concern that most published medical findings are false. But what does it mean to be false? Here we describe the range of definitions of false discoveries in the scientific literature. We summarize the philosophical, statistical, and experimental evidence for each type of false discovery. We discuss common underpinning problems with the scientific and data analytic practices and point to tools and behaviors that can be implemented to reduce the problems with published scientific results.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/scientific-communication-and-education">scientific-communication-and-education</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2016">2016</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/dengue_virus_antibodies_enhance_zika_virus_infection_6955826/">
          <h4 class="font-weight-bold">Dengue Virus Antibodies Enhance Zika Virus Infection, bioRxiv, 2016-04-26</h4>
          <p>AbstractBackgroundFor decades, human infections with Zika virus (ZIKV), a mosquito-transmitted flavivirus, were sporadic, associated with mild disease, and went underreported since symptoms were similar to other acute febrile diseases endemic in the same regions. Recent reports of severe disease associated with ZIKV, including Guillain-Barré syndrome and severe fetal abnormalities, have greatly heightened awareness. Given its recent history of rapid spread in immune naïve populations, it is anticipated that ZIKV will continue to spread in the Americas and globally in regions where competent Aedes mosquito vectors are found. Globally, dengue virus (DENV) is the most common mosquito-transmitted human flavivirus and is both well-established and the source of outbreaks in areas of recent ZIKV introduction. DENV and ZIKV are closely related, resulting in substantial antigenic overlap. Through a mechanism known as antibody-dependent enhancement (ADE), anti-DENV antibodies can enhance the infectivity of DENV for certain classes of immune cells, causing increased viral production that correlates with severe disease outcomes. Similarly, ZIKV has been shown to undergo ADE in response to antibodies generated by other flaviviruses. However, response to DENV antibodies has not yet been investigated.Methodology  Principal FindingsWe tested the neutralizing and enhancing potential of well-characterized broadly neutralizing human anti-DENV monoclonal antibodies (HMAbs) and human DENV immune sera against ZIKV using neutralization and ADE assays. We show that anti-DENV HMAbs, cross-react, do not neutralize, and greatly enhance ZIKV infection in vitro. DENV immune sera had varying degrees of neutralization against ZIKV and similarly enhanced ZIKV infection.Conclusions  SignificanceOur results suggest that pre-existing DENV immunity will enhance ZIKV infection in vivo and may increase disease severity. A clear understanding of the interplay between ZIKV and DENV will be critical in informing public health responses in regions where these viruses co-circulate and will be particularly valuable for ZIKV and DENV vaccine design and implementation strategies.Author SummaryRecent reports of severe disease, including developmental problems in newborns, have greatly heightened public health awareness of Zika virus (ZIKV), a mosquito-transmitted virus for which there is no vaccine or treatment. It is anticipated that ZIKV will continue to spread in the Americas and globally in regions where competent mosquitoes are found. Dengue virus (DENV), a closely related mosquito-transmitted virus is well-established in regions of recent ZIKV introduction and spread. It is increasingly common that individuals living in these regions may have had a prior DENV infection or may be infected with DENV and ZIKV at the same time. However, very little is known about the impact of DENV infections on ZIKV disease severity. In this study, we tested the ability of antibodies against DENV to prevent or enhance ZIKV infection in cell culture-based assays. We found that DENV antibodies can greatly enhance ZIKV infection in cells.Our results suggest that ZIKV infection in individuals that had a prior DENV infection may experience more severe clinical manifestations. The results of this study provide a better understanding of the interplay between ZIKV and DENV infections that can serve to inform public health responses and vaccine strategies.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2016">2016</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/first_detection_of_zika_virus_in_neotropical_primates_in_brazil_a_possible_new_reservoir_6845621/">
          <h4 class="font-weight-bold">First detection of Zika virus in neotropical primates in Brazil a possible new reservoir, bioRxiv, 2016-04-21</h4>
          <p>Samples from sera and oral swabs from fifteen marmosets (Callithrix jacchus) and nine capuchin-monkeys (Sapajus libidinosus) captured in Ceara State in Brazil were tested for Zika virus.  Samples were positive by Real time PCR and sequencing of the amplified product from a capuchin monkey showed 100% similarity to other ZIKV from South America. This is the first report on ZIKV detection among Neotropical primates.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2016">2016</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/high-resolution_interrogation_of_functional_elements_in_the_noncoding_genome_6776869/">
          <h4 class="font-weight-bold">High-resolution interrogation of functional elements in the noncoding genome, bioRxiv, 2016-04-19</h4>
          <p>The noncoding genome plays a major role in gene regulation and disease yet we lack tools for rapid identification and manipulation of noncoding elements. Here, we develop a large-scale CRISPR screen employing ~18,000 sgRNAs targeting &amp;gt;700 kb of noncoding sequence in an unbiased manner surrounding three genes (NF1, NF2, and CUL3) involved in resistance to the BRAF inhibitor vemurafenib in the BRAF-mutant melanoma cell line A375. We identify specific noncoding locations near genes that modulate drug resistance when mutated. These sites have predictive hallmarks of noncoding function, such as physical interaction with gene promoters, evolutionary conservation and tissue-specific chromatin accessibility. At a subset of identified elements at the CUL3 locus, we show that engineered mutations lead to a loss of gene expression associated with changes in transcription factor occupancy and in long-range and local epigenetic environments, implicating these sites in gene regulation and chemotherapeutic resistance. This demonstration of an unbiased mutagenesis screen across large noncoding regions expands the potential of pooled CRISPR screens for fundamental genomic discovery and for elucidating biologically relevant mechanisms of gene regulation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2016">2016</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/analysis_of_shared_heritability_in_common_disorders_of_the_brain_6759421/">
          <h4 class="font-weight-bold">Analysis of Shared Heritability in Common Disorders of the Brain, bioRxiv, 2016-04-17</h4>
          <p>AbstractDisorders of the brain exhibit considerable epidemiological comorbidity and frequently share symptoms, provoking debate about the extent of their etiologic overlap. We quantified the genetic sharing of 25 brain disorders based on summary statistics from genome-wide association studies of 215,683 patients and 657,164 controls, and their relationship to 17 phenotypes from 1,191,588 individuals. Psychiatric disorders show substantial sharing of common variant risk, while neurological disorders appear more distinct from one another. We observe limited evidence of sharing between neurological and psychiatric disorders, but do identify robust sharing between disorders and several cognitive measures, as well as disorders and personality types. We also performed extensive simulations to explore how power, diagnostic misclassification and phenotypic heterogeneity affect genetic correlations. These results highlight the importance of common genetic variation as a source of risk for brain disorders and the value of heritability-based methods in understanding their etiology.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2016">2016</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/100-200-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/100-200-users/page/121/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../../tags/100-200-users/page/118/" class="page-link">118</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/100-200-users/page/119/" class="page-link">119</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/100-200-users/page/120/" class="page-link">120</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/100-200-users/page/121/" class="page-link">121</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/100-200-users/page/122/" class="page-link">122</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/100-200-users/page/123/" class="page-link">123</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/100-200-users/page/124/" class="page-link">124</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/100-200-users/page/125/" class="page-link">125</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/100-200-users/page/126/" class="page-link">126</a></li>
            
        

        
        <li class="page-item"><a href="../../../../tags/100-200-users/page/123/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/100-200-users/page/126/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
