<!DOCTYPE html>
<html><head>
  
  <title>2019 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_history_of_measles_from_a_1912_genome_to_an_antique_origin_73413745/">
          <h4 class="font-weight-bold">The history of measles from a 1912 genome to an antique origin, bioRxiv, 2019-12-30</h4>
          <p>AbstractMany infectious diseases are thought to have emerged in humans after the Neolithic revolution. While it is broadly accepted that this also applies to measles, the exact date of emergence for this disease is controversial. Here, we sequenced the genome of a 1912 measles virus and used selection-aware molecular clock modeling to determine the divergence date of measles virus and rinderpest virus. This divergence date represents the earliest possible date for the establishment of measles in human populations. Our analyses show that the measles virus potentially arose as early as the 4th century BCE, rekindling the recently challenged hypothesis of an antique origin of this disease.One Sentence SummaryMeasles virus diverged from rinderpest virus in the 4th century BCE, which is compatible with an emergence of measles during Antiquity.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/perturbation_of_effector_and_regulatory_t_cell_subsets_in_myalgic_encephalomyelitischronic_fatigue_syndrome_mecfs_73284635/">
          <h4 class="font-weight-bold">Perturbation of effector and regulatory T cell subsets in Myalgic EncephalomyelitisChronic Fatigue Syndrome (MECFS), bioRxiv, 2019-12-26</h4>
          <p>AbstractMyalgic encephalomyelitischronic fatigue syndrome (MECFS) is a debilitating disorder of unknown etiology, and diagnosis of the disease is largely based on clinical symptoms. We hypothesized that immunological disruption is the major driver of this disease and analyzed a large cohort of MECFS patient or control blood samples for differences in T cell subset frequencies and functions. We found that the ratio of CD4&#43; to CD8&#43; T cells and the proportion of CD8&#43; effector memory T cells were increased, whereas NK cells were reduced in MECFS patients younger than 50 years old compared to a healthy control group. Remarkably, major differences were observed in Th1, Th2, Th17 and mucosal-associated invariant T (MAIT) T cell subset functions across all ages of patients compared to healthy subjects. While CCR6&#43; Th17 cells in MECFS secreted less IL-17 compared to controls, their overall frequency was higher. Similarly, MAIT cells from patients secreted lower IFNγ, GranzymeA and IL-17 upon activation. Together, these findings suggest chronic stimulation of these T cell populations in MECFS patients. In contrast, the frequency of regulatory T cells (Tregs), which control excessive immune activation, was higher in MECFS patients. Finally, using a machine learning algorithm called random forest, we determined that the set of T cell parameters analyzed could identify more than 90% of the subjects in the MECFS cohort as patients (93% true positive rate or sensitivity). In conclusion, these multiple and major perturbations or dysfunctions in T cell subsets in MECFS patients suggest potential chronic infections or microbiome dysbiosis. These findings also have implications for development of MECFS specific immune biomarkers and reveal potential targets for novel therapeutic interventions.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/on-demand_spatiotemporal_programming_of_collective_cell_migration_via_bioelectric_stimulation_73177656/">
          <h4 class="font-weight-bold">On-demand spatiotemporal programming of collective cell migration via bioelectric stimulation, bioRxiv, 2019-12-23</h4>
          <p>Directed cell migration is critical across biological processes spanning healing to cancer invasion, yet no tools allow such migration to be interactively guided. We present a new bioreactor that harnesses electrotaxis—directed cell migration along electric field gradients—by integrating multiple independent electrodes under computer control to dynamically program electric field patterns, and hence steer cell migration. Using this platform, we programmed and characterized multiple precise, two-dimensional collective migration maneuvers in renal epithelia and primary skin keratinocyte ensembles. First, we demonstrated on-demand, 90-degree collective turning. Next, we developed a universal electrical stimulation scheme capable of programming arbitrary 2D migration maneuvers such as precise angular turns and directing cells to migrate in a complete circle. Our stimulation scheme proves that cells effectively timeaverage electric field cues, helping to elucidate the transduction time scales in electrotaxis. Together, this work represents a fundamentally different platform for controlling cell migration with broad utility across fields.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioengineering">bioengineering</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/ultra-high_throughput_single-cell_rna_sequencing_by_combinatorial_fluidic_indexing_72967508/">
          <h4 class="font-weight-bold">Ultra-high throughput single-cell RNA sequencing by combinatorial fluidic indexing, bioRxiv, 2019-12-19</h4>
          <p>AbstractCell atlas projects and single-cell CRISPR screens hit the limits of current technology, as they require cost-effective profiling for millions of individual cells. To satisfy these enormous throughput requirements, we developed “single-cell combinatorial fluidic indexing” (scifi) and applied it to single-cell RNA sequencing. The resulting scifi-RNA-seq assay combines one-step combinatorial pre-indexing of single-cell transcriptomes with subsequent single-cell RNA-seq using widely available droplet microfluidics. Pre-indexing allows us to load multiple cells per droplet, which increases the throughput of droplet-based single-cell RNA-seq up to 15-fold, and it provides a straightforward way of multiplexing hundreds of samples in a single scifi-RNA-seq experiment. Compared to multi-round combinatorial indexing, scifi-RNA-seq provides an easier, faster, and more efficient workflow, thereby enabling massive-scale scRNA-seq experiments for a broad range of applications ranging from population genomics to drug screens with scRNA-seq readout. We benchmarked scifi-RNA-seq on various human and mouse cell lines, and we demonstrated its feasibility for human primary material by profiling TCR activation in T cells.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/unconventional_kinetochore_kinases_kkt2_and_kkt3_have_a_unique_zinc_finger_that_promotes_their_kinetochore_localization_72748577/">
          <h4 class="font-weight-bold">Unconventional kinetochore kinases KKT2 and KKT3 have a unique zinc finger that promotes their kinetochore localization, bioRxiv, 2019-12-14</h4>
          <p>AbstractChromosome segregation in eukaryotes is driven by the kinetochore, a macromolecular protein complex that assembles onto centromeric DNA and binds spindle microtubules. Cells must tightly control the number and position of kinetochores so that all chromosomes assemble a single kinetochore. A central player in this process is the centromere-specific histone H3 variant CENP-A, which localizes specifically within centromeres and promotes kinetochore assembly. However, CENP-A is absent from several eukaryotic lineages including kinetoplastids, a group of evolutionarily divergent eukaryotes that have an unconventional set of kinetochore proteins. It remains unknown how kinetoplastids specify kinetochore positions or promote kinetochore assembly in the absence of CENP-A. Here we studied two homologous kinetoplastid kinases (KKT2 and KKT3) that localize constitutively at centromeres. KKT2 and KKT3 central domains were sufficient for centromere localization in Trypanosoma brucei. Crystal structures of the KKT2 central domain from two divergent kinetoplastids revealed a unique zinc finger domain, which promotes its kinetochore localization in T. brucei. Mutations in the equivalent zinc finger domain of KKT3 abolished its kinetochore localization and function. This study lays the foundation for understanding the mechanism of kinetochore specification and assembly in kinetoplastids.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genomic_evidence_for_global_ocean_plankton_biogeography_shaped_by_large-scale_current_systems_72345163/">
          <h4 class="font-weight-bold">Genomic evidence for global ocean plankton biogeography shaped by large-scale current systems, bioRxiv, 2019-12-07</h4>
          <p>AbstractBiogeographical studies have traditionally focused on readily visible organisms, but recent technological advances are enabling analyses of the large-scale distribution of microscopic organisms, whose biogeographical patterns have long been debated1,2. The most prominent global biogeography of marine plankton was derived by Longhurst3 based on parameters principally associated with photosynthetic plankton. Localized studies of selected plankton taxa or specific organismal sizes1,4–7 have mapped community structure and begun to assess the roles of environment and ocean current transport in shaping these patterns2,8. Here we assess global plankton biogeography and its relation to the biological, chemical and physical context of the ocean (the ‘seascape’) by analyzing 24 terabases of metagenomic sequence data and 739 million metabarcodes from the Tara Oceans expedition in light of environmental data and simulated ocean current transport. In addition to significant local heterogeneity, viral, prokaryotic and eukaryotic plankton communities all display near steady-state, large-scale, size-dependent biogeographical patterns. Correlation analyses between plankton transport time and metagenomic or environmental dissimilarity reveal the existence of basin-scale biological and environmental continua emerging within the main current systems. Across oceans, there is a measurable, continuous change within communities and environmental factors up to an average of 1.5 years of travel time. Modulation of plankton communities during transport varies with organismal size, such that the distribution of smaller plankton best matches Longhurst biogeochemical provinces, whereas larger plankton group into larger provinces. Together these findings provide an integrated framework to interpret plankton community organization in its physico-chemical context, paving the way to a better understanding of oceanic ecosystem functioning in a changing global environment.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/ecology">ecology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        

        
             
            <li class="page-item active"><a href="/audiences/tags/2019/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/5/" class="page-link">5</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/2019/page/2/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/2019/page/135/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
