<!DOCTYPE html>
<html><head>
  
  <title>Genomics | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/islands_of_retroelements_are_the_major_components_of_drosophila_centromeres_54834204/">
          <h4 class="font-weight-bold">Islands of retroelements are the major components of Drosophila centromeres, bioRxiv, 2019-02-02</h4>
          <p>Centromeres are essential chromosomal regions that mediate kinetochore assembly and spindle attachments during cell division. Despite their functional conservation, centromeres are amongst the most rapidly evolving genomic regions and can shape karyotype evolution and speciation across taxa. Although significant progress has been made in identifying centromere-associated proteins, the highly repetitive centromeres of metazoans have been refractory to DNA sequencing and assembly, leaving large gaps in our understanding of their functional organization and evolution. Here, we identify the sequence composition and organization of the centromeres of Drosophila melanogaster by combining long-read sequencing, chromatin immunoprecipitation for the centromeric histone CENP-A, and high-resolution chromatin fiber imaging. Contrary to previous models that heralded satellite repeats as the major functional components, we demonstrate that functional centromeres form on islands of complex DNA sequences enriched in retroelements that are flanked by large arrays of satellite repeats. Each centromere displays distinct size and arrangement of its DNA elements but is similar in composition overall. We discover that a specific retroelement, G2Jockey-3, is the most highly enriched sequence in CENP-A chromatin and is the only element shared among all centromeres. G2Jockey-3 is also associated with CENP-A in the sister species Drosophila simulans, revealing an unexpected conservation despite the reported turnover of centromeric satellite DNA. Our work reveals the DNA sequence identity of the active centromeres of a premier model organism and implicates retroelements as conserved features of centromeric DNA.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/self-reporting_transposons_enable_simultaneous_readout_of_gene_expression_and_transcription_factor_binding_in_single_cells_54834788/">
          <h4 class="font-weight-bold">Self-reporting transposons enable simultaneous readout of gene expression and transcription factor binding in single cells, bioRxiv, 2019-02-02</h4>
          <p>In situ assays of transcription factor (TF) binding are confounded by cellular heterogeneity and represent averaged profiles in complex tissues. Single cell RNA-seq (scRNA-seq) is capable of resolving different cell types based on gene expression profiles, but no technology exists to directly link specific cell types to the binding pattern of TFs in those cell types. Here, we present self-reporting transposons (SRTs) and their use in single cell calling cards (scCC), a novel assay for simultaneously capturing gene expression profiles and mapping TF binding sites in single cells. First, we show how the genomic locations of SRTs can be recovered from mRNA. Next, we demonstrate that SRTs deposited by the piggyBac transposase can be used to map the genome-wide localization of the TFs SP1, through a direct fusion of the two proteins, and BRD4, through its native affinity for piggyBac. We then present the scCC method, which maps SRTs from scRNA-seq libraries, thus enabling concomitant identification of cell types and TF binding sites in those same cells. As a proof-of-concept, we show recovery of cell type-specific BRD4 and SP1 binding sites from cultured cells. Finally, we map Brd4 binding sites in the mouse cortex at single cell resolution, thus establishing a new technique for studying TF biology in situ.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genotype-phenotype_relationships_in_children_with_copy_number_variants_associated_with_high_neuropsychiatric_risk_findings_from_the_case-control_imagine-id_cohort_in_the_united_kingdom_54765026/">
          <h4 class="font-weight-bold">Genotype-phenotype relationships in children with Copy Number Variants associated with high neuropsychiatric risk Findings from the case-control IMAGINE-ID cohort in the United Kingdom, bioRxiv, 2019-01-31</h4>
          <p>AbstractBackgroundA variety of Copy Number Variants are associated with a high risk of neurodevelopmental and psychiatric disorders (ND-CNVs). We aimed to characterise the impact of ND-CNVs on childhood development and investigate whether different ND-CNVs lead to distinct and specific patterns of cognitive and behavioural outcomes.Methods258 children with ND-CNVs (13 CNVs across 9 loci) were systematically assessed for psychiatric disorders as well as broader traits of neurodevelopmental, cognitive and psychopathological origin. A comparison was made with 106 control siblings, in order to test the hypothesis that phenotypes would differ by genotype, both quantitatively, in terms of severity, and qualitatively in the pattern of associated impairments.Outcomes79.8% of ND-CNVs carriers met criteria for one or more psychiatric disorders (OR=13.8 compared to controls) the risk of ADHD (OR=6.9), ODD (OR=3.6), anxiety disorders (OR=2.9), and ASD traits (OR=44.1) was particularly high. ND-CNVs carriers were impaired across all neurodevelopmental, cognitive, and psychopathological traits relative to controls. Only moderate quantitative and qualitative differences in phenotypic profile were found between genotypes. In general, the range of phenotypes was broadly similar for all ND-CNV genotypes. Traits did show some evidence of genotypic specificity, however the specific genotype accounted for a low proportion of variance in outcome (5-20% depending on trait).InterpretationThe 13 ND-CNVs studied have a similar range of adverse effects on childhood neurodevelopment, despite subtle quantitative and qualitative differences. Our findings suggest that genomic risk for neuropsychiatric disorder has pleiotropic effects on multiple processes and neural circuits, and provides important implications for research into genotype-phenotype relationships within psychiatry.FundingThe Medical Research Council and the Waterloo FoundationResearch in contextEvidence before this studySeveral Copy Number Variants (CNVs) have been associated with high risk of development of child and adult neuropsychiatric disorders. Increasingly young children with developmental delay referred for genetic testing are being diagnosed with neurodevelopmental and psychiatric risk CNVs (referred to as ND-CNVs hereafter). It remains unclear whether different genotypes are associated with specific cognitive and behavioural phenotypes or whether these outcomes are non-specific. We searched PubMed for English language studies published from database inception until January 10th, 2019 that investigated the relationship between CNVs and cognitive and behavioural outcomes. Search terms included “CNV”, “genomics”, “1q21.1”, “2p16.3”, “NRXN1”, “9q34”, “Kleefstra Syndrome”, “15q11.2”, “15q13.3”, “16p11.2”, “22q11.2”, “psychiatry”, and “cognition”. Preliminary studies have indicated that deletions and duplications at the same loci may differ in cognitive and behavioural phenotypes. However, to date, there have been limited studies that contrasted the phenotypes of ND-CNVs across several loci on a range of cognitive and behavioural domains.Added value of this studyWe found that young people carrying a ND-CNV were at considerably increased risk for neuropsychiatric disorder and impairments across a range of neurodevelopmental, psychopathological, cognitive, social, sleep and motor traits. Within ND-CNV carriers, comparisons between genotypes indicated moderate quantitative and qualitative differences in overall phenotypic profile, with evidence that severity of impairment was similar across all genotypes for some traits (e.g. mood problems, sleep impairments, peer problems, and sustained attention) whereas for other traits there was evidence of genotype specific effects on severity (e.g., IQ, spatial planning, processing speed, subclinical psychotic experiences, ASD traits, motor coordination total psychiatric symptomatology, particularly anxiety, ADHD, and conduct related traits). However the proportion of variance explained by genotype was low, 5-20% depending on trait, indicating that overall ND-CNVs lead to similar neurodevelopmental outcomes. It is important that genotype-phenotype relationships are viewed through a developmental lens as some phenotypic outcomes were found to be associated with age.Implications of all the available evidenceOur work highlights that children who carry a ND-CNV represent a patient group that warrants clinical and educational attention for a broad range of cognitive and behavioural impairments and that commonalities in clinically relevant neurodevelopmental impairments exist across ND-CNVs. This group of young people could benefit from the development of a general care pathway, to which genotype-specific recommendations can be added where needed. Our work indicates that the relationship between genotype and neurodevelopmental phenotype is complex and that future research will need to take a global systems approach and not be narrowly focused on single phenotypes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genotype-phenotype_relationships_in_children_with_copy_number_variants_associated_with_high_neuropsychiatric_risk_findings_from_the_intellectual_disability__mental_health_assessing_the_genomic_impact_on_neurodevelopment_imagine-id_study_54765026/">
          <h4 class="font-weight-bold">Genotype-phenotype relationships in children with copy number variants associated with high neuropsychiatric risk Findings from the Intellectual Disability &amp; Mental Health Assessing the Genomic Impact on Neurodevelopment (IMAGINE-ID) study, bioRxiv, 2019-01-31</h4>
          <p>AbstractBackgroundA variety of copy number variants are associated with a high risk of neurodevelopmental and psychiatric disorders (ND-CNVs). Different ND-CNVs could lead to distinct and specific patterns of cognitive and behavioural outcomes, but supporting evidence is currently lacking.Methods258 children with ND-CNVs (13 CNVs across 9 loci) were systematically assessed for psychiatric disorders as well as broader traits of neurodevelopmental, cognitive and psychopathological origin. A comparison was made with 106 non-carrier control siblings, in order to test the hypothesis that phenotypes would differ by genotype, both quantitatively, in terms of severity, and qualitatively in the pattern of associated impairments.Outcomes79.8% of ND-CNVs carriers met criteria for one or more psychiatric disorders (OR=13.8 compared to controls) the risk of ADHD (OR=6.9), ODD (OR=3.6), anxiety disorders (OR=2.9), and ASD traits (OR=44.1) was particularly high. ND-CNVs carriers were impaired across all neurodevelopmental, cognitive, and psychopathological traits relative to controls. Only moderate quantitative and qualitative differences in phenotypic profile were found between genotypes. In general, the range of phenotypes was broadly similar for all ND-CNV genotypes. Traits did show some evidence of genotypic specificity, however the specific genotype accounted for a low proportion of variance in outcome (5-20% depending on trait).InterpretationThe 13 ND-CNVs studied have a similar range of adverse effects on childhood neurodevelopment, despite subtle quantitative and qualitative differences. Our findings suggest that genomic risk for neuropsychiatric disorder has pleiotropic effects on multiple processes and neural circuits, and provides important implications for research into genotype-phenotype relationships within psychiatry.FundingThe Medical Research Council and the Medical Research FoundationResearch in contextEvidence before this studySeveral copy number variants (CNVs) have been associated with high risk of development of child and adult neuropsychiatric disorders. Increasingly young children with developmental delay referred for genetic testing are being diagnosed with neurodevelopmental and psychiatric risk CNVs (referred to as ND-CNVs hereafter). It remains unclear whether different genotypes are associated with specific cognitive and behavioural phenotypes or whether these outcomes are non-specific. We searched PubMed for studies published from database inception until January 10th, 2019 that investigated the relationship between CNVs and cognitive and behavioural outcomes. Search terms included “CNV”, “genomics”, “1q21.1”, “2p16.3”, “NRXN1”, “9q34”, “Kleefstra Syndrome”, “15q11.2”, “15q13.3”, “16p11.2”, “22q11.2”, “psychiatry”, and “cognition”. Preliminary studies have indicated that deletions and duplications at the same loci may differ in cognitive and behavioural phenotypes. However, to date, there have been limited studies that contrasted the phenotypes of ND-CNVs across several loci on a range of cognitive and behavioural domains.Added value of this studyWe found that young people carrying a ND-CNV were at considerably increased risk for neuropsychiatric disorder and impairments across a range of neurodevelopmental, psychopathological, cognitive, social, sleep and motor traits. Within ND-CNV carriers, comparisons between genotypes indicated moderate quantitative and qualitative differences in overall phenotypic profile, with evidence that severity of impairment was similar across all genotypes for some traits (e.g. mood problems, sleep impairments, peer problems, and sustained attention) whereas for other traits there was evidence of genotype specific effects on severity (e.g., IQ, spatial planning, processing speed, subclinical psychotic experiences, ASD traits, motor coordination total psychiatric symptomatology, particularly anxiety, ADHD, and conduct related traits). However the proportion of variance explained by genotype was low, 5-20% depending on trait, indicating that overall ND-CNVs lead to similar neurodevelopmental outcomes. It is important that genotype-phenotype relationships are viewed through a developmental lens as some phenotypic outcomes were found to be associated with age.Implications of all the available evidenceChildren who carry a ND-CNV represent a patient group that warrants clinical and educational attention for a broad range of cognitive and behavioural impairments. Although qualitative and quantitative differences exist between ND-CNVs, our findings point to commonalities in clinical outcomes with neurodevelopmental impairments being present across all ND-CNVs. This group of young people could benefit from the development of a general intervention plan, to which genotype-specific recommendations can be added where needed. Our findings do not support a model whereby different ND-CNVs represent discrete forms of neuropsychiatric disorder and suggest that multiple processes and neural circuits are affected by ND-CNVs. The pleiotropic effects of ND-CNVs emphasises that research aiming to identify causal pathways between genetic variation and psychiatric outcomes via intermediary (or endo-)phenotypes needs to take a global perspective and not be narrowly focused on single phenotypes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/is_it_time_to_change_the_reference_genome_54673121/">
          <h4 class="font-weight-bold">Is it time to change the reference genome?, bioRxiv, 2019-01-30</h4>
          <p>The use of the human reference genome has shaped methods and data across modern genomics.  This has offered many benefits while creating a few constraints. In the following piece, we outline the history, properties, and pitfalls of the current human reference genome. In a few illustrative analyses, we focus on its use for variant-calling, highlighting its nearness to a type specimen. We suggest that switching to a consensus reference offers important advantages over the current reference with few disadvantages.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/evaluating_potential_drug_targets_through_human_loss-of-function_genetic_variation_54627010/">
          <h4 class="font-weight-bold">Evaluating potential drug targets through human loss-of-function genetic variation, bioRxiv, 2019-01-29</h4>
          <p>AbstractHuman genetics has informed the clinical development of new drugs, and is beginning to influence the selection of new drug targets. Large-scale DNA sequencing studies have created a catalogue of naturally occurring genetic variants predicted to cause loss of function in human genes, which in principle should provide powerful in vivo models of human genetic “knockouts” to complement model organism knockout studies and inform drug development. Here, we consider the use of predicted loss-of-function (pLoF) variation catalogued in the Genome Aggregation Database (gnomAD) for the evaluation of genes as potential drug targets. Many drug targets, including the targets of highly successful inhibitors such as aspirin and statins, are under natural selection at least as extreme as known haploinsufficient genes, with pLoF variants almost completely depleted from the population. Thus, metrics of gene essentiality should not be used to eliminate genes from consideration as potential targets. The identification of individual humans harboring “knockouts” (biallelic gene inactivation), followed by individual recall and deep phenotyping, is highly valuable to study gene function. In most genes, pLoF alleles are sufficiently rare that ascertainment will be largely limited to heterozygous individuals in outbred populations. Sampling of diverse bottlenecked populations and consanguineous individuals will aid in identification of total “knockouts”. Careful filtering and curation of pLoF variants in a gene of interest is necessary in order to identify true LoF individuals for follow-up, and the positional distribution or frequency of true LoF variants may reveal important disease biology. Our analysis suggests that the value of pLoF variant data for drug discovery lies in deep curation informed by the nature of the drug and its indication, as well as the biology of the gene, followed by recall-by-genotype studies in targeted populations.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/genomics/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/genomics/page/22/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/genomics/page/19/" class="page-link">19</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/genomics/page/20/" class="page-link">20</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/genomics/page/21/" class="page-link">21</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/genomics/page/22/" class="page-link">22</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/genomics/page/23/" class="page-link">23</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/genomics/page/24/" class="page-link">24</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/genomics/page/25/" class="page-link">25</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/genomics/page/26/" class="page-link">26</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/genomics/page/27/" class="page-link">27</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/genomics/page/24/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/genomics/page/64/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
