<!DOCTYPE html>
<html><head>
  
  <title>2017 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/ultrafast_glutamate_sensors_resolve_high-frequency_release_at_schaffer_collateral_synapses_30393707/">
          <h4 class="font-weight-bold">Ultrafast glutamate sensors resolve high-frequency release at Schaffer collateral synapses, bioRxiv, 2017-12-14</h4>
          <p>ABSTRACTGlutamatergic synapses display a rich repertoire of plasticity mechanisms on many different time scales, involving dynamic changes in the efficacy of transmitter release as well as changes in the number and function of postsynaptic glutamate receptors. The genetically encoded glutamate sensor iGluSnFR enables visualization of glutamate release from presynaptic terminals at frequencies up to ∼10 Hz. However, to resolve glutamate dynamics during high frequency bursts, faster indicators are required. Here we report the development of fast (iGluf) and ultrafast (iGluu) variants with comparable brightness, but increased Kd for glutamate (137 μM and 600 μM, respectively). Compared to iGluSnFR, iGluu has a 6-fold faster dissociation rate in vitro and 5-fold faster kinetics in synapses. Fitting a three-state model to kinetic data, we identify the large conformational change after glutamate binding as the rate-limiting step. In rat hippocampal slice culture stimulated at 100 Hz, we find that iGluu is sufficiently fast to resolve individual glutamate release events, revealing that glutamate is rapidly cleared from the synaptic cleft. Depression of iGluu responses during 100 Hz trains correlates with depression of postsynaptic EPSPs, indicating that depression during high frequency stimulation is purely presynaptic in origin. At individual boutons, the recovery from depression could be predicted from the amount of glutamate released on the second pulse (paired pulse facilitationdepression), demonstrating differential frequency-dependent filtering of spike trains at Schaffer collateral boutons.Significance StatementExcitatory synapses convert presynaptic action potentials into chemical signals that are sensed by postsynaptic glutamate receptors. To eavesdrop on synaptic transmission, genetically encoded fluorescent sensors for glutamate have been developed. However, even the best available sensors lag behind the very fast glutamate dynamics in the synaptic cleft. Here we report the development of an ultrafast genetically encoded glutamate sensor, iGluu, which allowed us to image glutamate clearance and synaptic depression during 100 Hz spike trains. We found that only boutons showing paired-pulse facilitation were able to rapidly recover from depression. Thus, presynaptic boutons act as frequency-specific filters to transmit select features of the spike train to specific postsynaptic cells.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_revised_understanding_of_tribolium_morphogenesis_further_reconciles_short_and_long_germ_development_30343453/">
          <h4 class="font-weight-bold">A revised understanding of Tribolium morphogenesis further reconciles short and long germ development, bioRxiv, 2017-12-13</h4>
          <p>AbstractIn Drosophila melanogaster, the germband forms directly on the egg surface and solely consists of embryonic tissue. In contrast, most insect embryos undergo a complicated set of tissue rearrangements to generate a condensed, multi-layered germband. The ventral side of the germband is embryonic, while the dorsal side is thought to be an extraembryonic tissue called the amnion. While this tissue organisation has been accepted for decades, and has been widely reported in insects, its accuracy has not been directly tested in any species. Using live cell tracking and differential cell labelling in the short germ beetle Tribolium castaneum, I show that most of the cells previously thought to be amnion actually give rise to large parts of the embryo. This process occurs via the dorsal-to-ventral flow of cells and contributes to germband extension. In addition, I show that true ‘amnion’ cells in Tribolium originate from a small region of the blastoderm. Together, my findings show that development in the short germ embryos of Tribolium and the long germ embryos of Drosophila is more similar than previously proposed. Dorsal-to-ventral cell flow also occurs in Drosophila during germband extension, and I argue that the flow is driven by a conserved set of underlying morphogenetic events in both species. Furthermore, the revised Tribolium fatemap that I present is far more similar to that of Drosophila than the classic Tribolium fatemap. Lastly, my findings show that there is no qualitative difference between the tissue structure of the cellularised blastoderm and the shortintermediate germ germband. As such, the same tissue patterning mechanisms could function continuously throughout the cellularised blastoderm and germband stages, and easily shift between them over evolutionary time.Author summaryIn many animals, certain groups of cells in the embryo do not directly contribute to adult structures. Instead, these cells generate so-called ‘extra-embryonic tissues’ that support and facilitate development, but degenerate prior to birthhatching. In most insect species, embryos are described as having two major extra-embryonic tissues; the serosa, which encapsulates the entire embryo and yolk, and the amnion, which covers one side of the embryo. This tissue structure has been widely reported for over a century, but detailed studies on the amnion are lacking. Working in the beetle Tribolium castaneum, I used long-term fluorescent live imaging, cell tracking and differential cell labelling to investigate amnion development. In contrast to our current understanding, I show that most cells previously thought to be amnion actually form large parts of the embryo. In addition, I show how these cells ‘flow’ as a whole tissue and contribute to elongation of the embryo, and how only a relatively small number of cells form the actual amnion. Lastly, I describe how my findings show that despite exhibiting substantial differences in overall structure, embryos of Tribolium and the fruit fly, Drosophila melanogaster, utilise a conserved set of morphogenetic processes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/pathway_enrichment_analysis_of_-omics_data_30331843/">
          <h4 class="font-weight-bold">Pathway enrichment analysis of -omics data, bioRxiv, 2017-12-13</h4>
          <p>ABSTRACTPathway enrichment analysis helps gain mechanistic insight into large gene lists typically resulting from genome scale (–omics) experiments. It identifies biological pathways that are enriched in the gene list more than expected by chance. We explain pathway enrichment analysis and present a practical step-by-step guide to help interpret gene lists resulting from RNA-seq and genome sequencing experiments. The protocol comprises three major steps define a gene list from genome scale data, determine statistically enriched pathways, and visualize and interpret the results. We focus on differentially expressed genes and mutated cancer genes, however the described principles can be applied to diverse –omics data. The protocol is designed for biologists with no prior bioinformatics training and uses freely available software including gProfiler, GSEA, Cytoscape and Enrichment Map.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/rarefaction_alpha_diversity_and_statistics_30290617/">
          <h4 class="font-weight-bold">Rarefaction, alpha diversity, and statistics, bioRxiv, 2017-12-12</h4>
          <p>AbstractUnderstanding the drivers of microbial diversity is a fundamental question in microbial ecology. Extensive literature discusses different methods for describing microbial diversity and documenting its effects on ecosystem function. However, it is widely believed that diversity depends on the number of reads that are sequenced. I discuss a statistical perspective on diversity, framing the diversity of an environment as an unknown parameter, and discussing the bias and variance of plug-in and rarefied estimates. I argue that by failing to account for both bias and variance, we invalidate analysis of alpha diversity. I describe the state of the statistical literature for addressing these problems, and suggest that measurement error modeling can address issues with variance, but bias corrections need to be utilized as well. I encourage microbial ecologists to avoid motivating their investigations with alpha diversity analyses that do not use valid statistical methodology.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/robust_manipulation_of_the_behavior_of_drosophila_melanogaster_by_a_fungal_pathogen_in_the_laboratory_30276844/">
          <h4 class="font-weight-bold">Robust manipulation of the behavior of Drosophila melanogaster by a fungal pathogen in the laboratory, bioRxiv, 2017-12-11</h4>
          <p>AbstractMany microbes induce striking behavioral changes in their animal hosts, but how they achieve this is poorly understood, especially at the molecular level. Mechanistic understanding has been largely constrained by the lack of a model system with advanced tools for molecular manipulation. We recently discovered a strain of the behavior-manipulating fungal pathogen Entomophthora muscae infecting wild Drosophila, and established methods to infect D. melanogaster in the lab. Lab-infected flies manifest the moribund behaviors characteristic of E. muscae infection hours before death, they climb upward, extend their proboscides and affix in place, then raise their wings, clearing a path for infectious spores to launch from their abdomens. We found that E. muscae invades the fly nervous system, suggesting a direct means by which the fungus could induce behavioral changes. Given the vast molecular toolkit available for D. melanogaster, we believe this new system will enable rapid progress in understanding the mechanistic basis of E. muscae’s behavioral manipulation in the fly.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/animal-behavior-and-cognition">animal-behavior-and-cognition</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_ancestral_animal_genetic_toolkit_revealed_by_diverse_choanoflagellate_transcriptomes_30269992/">
          <h4 class="font-weight-bold">The ancestral animal genetic toolkit revealed by diverse choanoflagellate transcriptomes, bioRxiv, 2017-12-11</h4>
          <p>AbstractThe changes in gene content that preceded the origin of animals can be reconstructed by comparison with their sister group, the choanoflagellates. However, only two choanoflagellate genomes are currently available, providing poor coverage of their diversity. We sequenced transcriptomes of 19 additional choanoflagellate species to produce a comprehensive reconstruction of the gains and losses that shaped the ancestral animal gene repertoire. We find roughly 1,700 gene families with origins on the animal stem lineage, of which only a core set of 36 are conserved across animals. We find more than 350 gene families that were previously thought to be animal-specific actually evolved before the animal-choanoflagellate divergence, including Notch and Delta, Toll-like receptors, and glycosaminoglycan hydrolases that regulate animal extracellular matrix (ECM). In the choanoflagellate Salpingoeca helianthica, we show that a glycosaminoglycan hydrolase modulates rosette colony size, suggesting a link between ECM regulation and morphogenesis in choanoflagellates and animals.Data AvailabilityRaw sequencing reads NCBI BioProject PRJNA419411 (19 choanoflagellate transcriptomes), PRJNA420352 (S. rosetta polyA selection test)Transcriptome assemblies, annotations, and gene families &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsdx.doi.org10.6084m9.figshare.5686984&gt;httpsdx.doi.org10.6084m9.figshare.5686984&lt;jatsext-link&gt;Protocols &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsdx.doi.org10.17504protocols.io.kwscxee&gt;httpsdx.doi.org10.17504protocols.io.kwscxee&lt;jatsext-link&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/2017/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/2017/page/9/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2017/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2017/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2017/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2017/page/9/" class="page-link">9</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/2017/page/10/" class="page-link">10</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2017/page/11/" class="page-link">11</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2017/page/12/" class="page-link">12</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2017/page/13/" class="page-link">13</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2017/page/14/" class="page-link">14</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/2017/page/11/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/2017/page/30/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
