<!DOCTYPE html>
<html><head>
  
  <title>200-500-users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.68.3" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.95d2483cd3c5d2bf47c1f36307a456787d60ad958c5efe44db9afc47012b1b05b8e1f64f6ac291650439322a6c38e86eb7cd6254255516d857c23f299fb0e666.js" integrity="sha512-ldJIPNPF0r9HwfNjB6RWeH1grZWMXv5E25r8RwErGwW44fZPasKRZQQ5MipsOOhut81iVCVVFthXwj8pn7DmZg=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/bright_and_photostable_chemigenetic_indicators_for_extended_in_vivo_voltage_imaging_49309055/">
          <h4 class="font-weight-bold">Bright and photostable chemigenetic indicators for extended in vivo voltage imaging, bioRxiv, 2018-10-06</h4>
          <p>Imaging changes in membrane potential using genetically encoded fluorescent voltage indicators (GEVIs) has great potential for monitoring neuronal activity with high spatial and temporal resolution. Brightness and photostability of fluorescent proteins and rhodopsins have limited the utility of existing GEVIs. We engineered a novel GEVI, Voltron, that utilizes bright and photostable synthetic dyes instead of protein-based fluorophores, extending the combined duration of imaging and number of neurons imaged simultaneously by more than tenfold relative to existing GEVIs. We used Voltron for in vivo voltage imaging in mice, zebrafish, and fruit flies. In mouse cortex, Voltron allowed single-trial recording of spikes and subthreshold voltage signals from dozens of neurons simultaneously, over 15 minutes of continuous imaging. In larval zebrafish, Voltron enabled the precise correlation of spike timing with behavior.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/bazam_a_rapid_method_for_read_extraction_and_realignment_of_high_throughput_sequencing_data_49189297/">
          <h4 class="font-weight-bold">Bazam A rapid method for read extraction and realignment of high throughput sequencing data, bioRxiv, 2018-10-04</h4>
          <p>AbstractBackgroundAs costs of high throughput sequencing have fallen, we are seeing vast quantities of short read genomic data being generated. Often, the data is exchanged and stored as aligned reads, which provides high compression and convenient access for many analyses. However, aligned data becomes outdated as new reference genomes and alignment methods become available. Moreover, some applications cannot utilise pre-aligned reads at all, necessitating conversion back to raw format (FASTQ) before they can be used. In both cases, the process of extraction and realignment is expensive and time consuming.FindingsWe describe Bazam, a tool that efficiently extracts the original paired FASTQ from reads stored in aligned form (BAM or CRAM format). Bazam extracts reads in a format that directly allows realignment with popular aligners with high concurrency. Through eliminating steps and increasing the accessible concurrency, Bazam facilitates up to a 90% reduction in the time required for realignment compared to standard methods. Bazam can support selective extraction of read pairs from focused genomic regions, further increasing efficiency for targeted analyses. Bazam is additionally suitable as a base for other applications that require efficient paired read information, such as quality control, structural variant calling and alignment comparison.ConclusionsBazam offers significant improvements for users needing to realign genomic data.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/high-performance_gfp-based_calcium_indicators_for_imaging_activity_in_neuronal_populations_and_microcompartments_49189339/">
          <h4 class="font-weight-bold">High-performance GFP-based calcium indicators for imaging activity in neuronal populations and microcompartments, bioRxiv, 2018-10-04</h4>
          <p>AbstractCalcium imaging with genetically encoded calcium indicators (GECIs) is routinely used to measure neural activity in intact nervous systems. GECIs are frequently used in one of two different modes to track activity in large populations of neuronal cell bodies, or to follow dynamics in subcellular compartments such as axons, dendrites and individual synaptic compartments. Despite major advances, calcium imaging is still limited by the biophysical properties of existing GECIs, including affinity, signal-to-noise ratio, rise and decay kinetics, and dynamic range. Using structure-guided mutagenesis and neuron-based screening, we optimized the green fluorescent protein-based GECI GCaMP6 for different modes of in vivo imaging. The jGCaMP7 sensors provide improved detection of individual spikes (jGCaMP7s,f), imaging in neurites and neuropil (jGCaMP7b), and tracking large populations of neurons using 2-photon (jGCaMP7s,f) or wide-field (jGCaMP7c) imaging.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/there_is_no_single_functional_atlas_even_for_a_single_individual_parcellation_of_the_human_brain_is_state_dependent_49104845/">
          <h4 class="font-weight-bold">There is no single functional atlas even for a single individual Parcellation of the human brain is state dependent, bioRxiv, 2018-10-02</h4>
          <p>AbstractThe goal of human brain mapping has long been to delineate the functional subunits in the brain and elucidate the functional role of each of these brain regions. Recent work has focused on whole-brain parcellation of functional Magnetic Resonance Imaging (fMRI) data to identify these subunits and create a functional atlas. Functional connectivity approaches to understand the brain at the network level require such an atlas to assess connections between parcels and extract network properties. While no single functional atlas has emerged as the dominant atlas to date, there remains an underlying assumption that such an atlas exists. Using fMRI data from a highly sampled subject as well as two independent replication data sets, we demonstrate that functional parcellations based on fMRI connectivity data reconfigure substantially and in a meaningful manner, according to brain state.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/collective_intercellular_communication_through_ultra-fast_hydrodynamic_trigger_waves_48929251/">
          <h4 class="font-weight-bold">Collective intercellular communication through ultra-fast hydrodynamic trigger waves, bioRxiv, 2018-09-27</h4>
          <p>The biophysical relationships between sensors and actuators have been fundamental to the development of complex life forms; abundant flows are generated and persist in aquatic environments by swimming organisms, while responding promptly to external stimuli is key to survival. Here, akin to a chain reaction, we present the discovery of hydrodynamic trigger waves in cellular communities of the protist Spirostomum ambiguum, propagating hundreds of times faster than the swimming speed. Coiling its cytoskeleton, Spirostomum can contract its long body by 50% within milliseconds, with accelerations reaching 14g-forces. Surprisingly, a single cellular contraction (transmitter) is shown to generate long-ranged vortex flows at intermedi- ate Reynolds numbers, which can trigger neighbouring cells, in turn. To measure the sensitivity to hydrodynamic signals (receiver), we further present a high-throughput suction-flow device to probe mechanosensitive ion channel gating by back-calculating the microscopic forces on the cell mem- brane. These ultra-fast hydrodynamic trigger waves are analysed and modelled quantitatively in a universal framework of antenna and percolation theory. A phase transition is revealed, requiring a critical colony density to sustain collective communication. Our results suggest that this signalling could help organise cohabiting communities over large distances, influencing long-term behaviour through gene expression, comparable to quorum sensing. More immediately, as contractions release toxins, synchronised discharges could also facilitate the repulsion of large predators, or conversely immobilise large prey. We postulate that beyond protists numerous other freshwater and marine organisms could coordinate with variations of hydrodynamic trigger waves.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/trans_effects_on_gene_expression_can_drive_omnigenic_inheritance_48842970/">
          <h4 class="font-weight-bold">Trans effects on gene expression can drive omnigenic inheritance, bioRxiv, 2018-09-25</h4>
          <p>Early genome-wide association studies (GWAS) led to the surprising discovery that, for typical complex traits, the most significant genetic variants contribute only a small fraction of the estimated heritability. Instead, it has become clear that a huge number of common variants, each with tiny effects, explain most of the heritability. Previously, we argued that these patterns conﬂict with standard conceptual models, and that new models are needed. Here we provide a formal model in which genetic contributions to complex traits can be partitioned into direct effects from core genes, and indirect effects from peripheral genes acting as trans-regulators. We argue that the central importance of peripheral genes is a direct consequence of the large contribution of trans-acting variation to gene expression variation. In particular, we propose that if the core genes for a trait are co-regulated – as seems likely – then the effects of peripheral variation can be amplified by these co-regulated networks such that nearly all of the genetic variance is driven by peripheral genes. Thus our model proposes a framework for understanding key features of the architecture of complex traits.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/200-500-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/200-500-users/page/31/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/200-500-users/page/28/" class="page-link">28</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/200-500-users/page/29/" class="page-link">29</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/200-500-users/page/30/" class="page-link">30</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/200-500-users/page/31/" class="page-link">31</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/200-500-users/page/32/" class="page-link">32</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/200-500-users/page/33/" class="page-link">33</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/200-500-users/page/34/" class="page-link">34</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/200-500-users/page/35/" class="page-link">35</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/200-500-users/page/36/" class="page-link">36</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/200-500-users/page/33/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/200-500-users/page/56/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
