<!DOCTYPE html>
<html><head>
  
  <title>neuroscience | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.68.3" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.95d2483cd3c5d2bf47c1f36307a456787d60ad958c5efe44db9afc47012b1b05b8e1f64f6ac291650439322a6c38e86eb7cd6254255516d857c23f299fb0e666.js" integrity="sha512-ldJIPNPF0r9HwfNjB6RWeH1grZWMXv5E25r8RwErGwW44fZPasKRZQQ5MipsOOhut81iVCVVFthXwj8pn7DmZg=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/hormonal_and_neural_correlates_of_care_in_active_versus_observing_poison_frog_parents_66442344/">
          <h4 class="font-weight-bold">Hormonal and neural correlates of care in active versus observing poison frog parents, bioRxiv, 2019-09-11</h4>
          <p>The occasional reversal of sex-typical behavior suggests that many of the neural circuits underlying behavior are conserved between males and females and can be activated in response to the appropriate social condition or stimulus. Most poison frog species (Family Dendrobatidae) exhibit male uniparental care, but flexible compensation has been observed in some species, where females will take over parental care duties when males disappear. We investigated hormonal and neural correlates of sex-typical and sex-reversed parental care in a typically male uniparental species, the Dyeing Poison Frog (Dendrobates tinctorius). We first characterized hormone levels and whole brain gene expression across parental care stages during sex-typical care. Surprisingly, hormonal changes and brain gene expression differences associated with active parental behavior in males were mirrored in their non-caregiving female partners. To further explore the disconnect between neuroendocrine patterns and behavior, we characterized hormone levels and neural activity patterns in females performing sex-reversed parental care. In contrast to hormone and gene expression patterns, we found that patterns of neural activity were linked to the active performance of parental behavior, with sex-reversed tadpole transporting females exhibiting neural activity patterns more similar to those of transporting males than non-caregiving females. We suggest that parallels in hormones and brain gene expression in active and observing parents are related to females ability to flexibly take over parental care in the absence of their male partners.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/how_neurons_move_during_action_potentials_66443380/">
          <h4 class="font-weight-bold">How neurons move during action potentials, bioRxiv, 2019-09-11</h4>
          <p>AbstractNeurons undergo nanometer-scale deformations during action potentials, and the underlying mechanism has been actively debated for decades. Previous observations were limited to a single spot or the cell boundary, while movement across the entire neuron during the action potential remained unclear.We report full-field imaging of cellular deformations accompanying the action potential in mammalian neuron somas (−1.8nm~1.3nm) and neurites (−0.7nm~0.9nm), using fast quantitative phase imaging with a temporal resolution of 0.1ms and an optical pathlength sensitivity of &amp;lt;4pm per pixel. Spike-triggered average, synchronized to electrical recording, demonstrates that the time course of the optical phase changes matches the dynamics of the electrical signal, with the optical signal revealing the intracellular potential rather than its time derivative detected via extracellular electrodes. Using 3D cellular morphology extracted via confocal microscopy, we demonstrate that the voltage-dependent changes in the membrane tension induced by ionic repulsion can explain the magnitude, time course and spatial features of the phase imaging. Our full-field observations of the spike-induced deformations in mammalian neurons opens the door to non-invasive label-free imaging of neural signaling.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/direct-fit_to_nature_an_evolutionary_perspective_on_biological_and_artificial_neural_networks_66394854/">
          <h4 class="font-weight-bold">Direct-fit to nature an evolutionary perspective on biological (and artificial) neural networks, bioRxiv, 2019-09-10</h4>
          <p>AbstractEvolution is a blind fitting process by which organisms, over generations, adapt to the niches of an ever-changing environment. Does the mammalian brain use similar brute-force fitting processes to learn how to perceive and act upon the world? Recent advances in training deep neural networks has exposed the power of optimizing millions of synaptic weights to map millions of observations along ecologically relevant objective functions. This class of models has dramatically outstripped simpler, more intuitive models, operating robustly in real-life contexts spanning perception, language, and action coordination. These models do not learn an explicit, human-interpretable representation of the underlying structure of the data; rather, they use local computations to interpolate over task-relevant manifolds in a high-dimensional parameter space. Furthermore, counterintuitively, over-parameterized models, similarly to evolutionary processes, can be simple and parsimonious as they provide a versatile, robust solution for learning a diverse set of functions. In contrast to traditional scientific models, where the ultimate goal is interpretability, over-parameterized models eschew interpretability in favor of solving real-life problems or tasks. We contend that over-parameterized blind fitting presents a radical challenge to many of the underlying assumptions and practices in computational neuroscience and cognitive psychology. At the same time, this shift in perspective informs longstanding debates and establishes unexpected links with evolution, ecological psychology, and artificial life.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/brainspace_a_toolbox_for_the_analysis_of_macroscale_gradients_in_neuroimaging_and_connectomics_datasets_66231552/">
          <h4 class="font-weight-bold">BrainSpace a toolbox for the analysis of macroscale gradients in neuroimaging and connectomics datasets, bioRxiv, 2019-09-09</h4>
          <p>AbstractUnderstanding how higher order cognitive function emerges from the underlying brain structure depends on quantifying how the behaviour of discrete regions are integrated within the broader cortical landscape. Recent work has established that this macroscale brain organization and function can be quantified in a compact manner through the use of multivariate machine learning approaches that identify manifolds often described as cortical gradients. By quantifying topographic principles of macroscale organization, cortical gradients lend an analytical framework to study structural and functional brain organization across species, throughout development and aging, and its perturbations in disease. More generally, its macroscale perspective on brain organization offers novel possibilities to investigate the complex relationships between brain structure, function, and cognition in a quantified manner. Here, we present a compact workflow and open-access toolbox that allows for (i) the identification of gradients (from structural or functional imaging data), (ii) their alignment (across subjects or modalities), and (iii) their visualization (in embedding or cortical space). Our toolbox also allows for controlled association studies between gradients with other brain-level features, adjusted with respect to several null models that account for spatial autocorrelation. The toolbox is implemented in both Python and Matlab, programming languages widely used by the neuroimaging and network neuroscience communities. Several use-case examples and validation experiments demonstrate the usage and consistency of our tools for the analysis of functional and microstructural gradients across different spatial scales.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/widespread_receptor_driven_modulation_in_peripheral_olfactory_coding_66231824/">
          <h4 class="font-weight-bold">Widespread Receptor Driven Modulation in Peripheral Olfactory Coding, bioRxiv, 2019-09-09</h4>
          <p>AbstractWe utilized swept confocally aligned planar excitation (SCAPE) microscopy to measure odor-driven activity simultaneously in many (&amp;gt;10,000) olfactory sensory neurons distributed over large areas of intact mouse olfactory epithelium. This approach allowed us to investigate the responses to mixtures or blends of odors and their components, a more realistic stimulus than monomolecular odors. In up to 38% of responding cells, responses to a mixture of odors were different - absent, smaller or larger - than what would be expected from the sum of the individual components. Further investigation revealed instances of both antagonism and allosteric enhancement in the primary olfactory sensory neurons. All 10 of the odor compounds tested were found to act as both agonists and antagonists at different receptors. We present a hypothetical scheme for how modulation at the peripheral receptors increases the capability of the olfactory system to recognize patterns of complex odor mixtures. The widespread modulation of primary sensory receptors argues against a simple combinatorial code and should motivate a search for alternative coding strategies.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/stimjim_open_source_hardware_for_precise_electrical_stimulation_65925120/">
          <h4 class="font-weight-bold">Stimjim open source hardware for precise electrical stimulation, bioRxiv, 2019-09-05</h4>
          <p>AbstractElectrical stimulation is a simple and powerful tool to perturb and evoke neuronal activity in order to understand the function of neurons and neural circuits. Despite this, devices that can provide precise current or voltage stimulation are expensive and closed-source. Here, we introduce Stimjim, a capable and inexpensive ($200 USD) open-source instrument for electrical stimulation that combines both function generation and electrical isolation. Stimjim provides microsecond temporal resolution with microampere or millivolt scale precision on two electrically isolated output channels. We demonstrate Stimjim’s utility both in vitro by precisely stimulating brain slices, and in vivo by training mice to perform intracranial self-stimulation (ICSS) for brain stimulation reward. During ICSS, Stimjim enables the experimenter to smoothly tune the strength of reward-seeking behavior by varying either the output frequency or amplitude. We envision Stimjim will enable new kinds of experiments due to its open-source and scalable nature.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/4/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/4/" class="page-link">4</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/9/" class="page-link">9</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/6/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/neuroscience/page/53/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
