<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/single-cell_in_situ_transcriptomic_map_of_astrocyte_cortical_layer_diversity_49173691/">
          <h4 class="font-weight-bold">Single-cell in situ transcriptomic map of astrocyte cortical layer diversity, bioRxiv, 2018-10-03</h4>
          <p>AbstractDuring organogenesis, patterns and gradients of gene expression underlie organization and diversified cell specification to generate complex tissue architecture. While the cerebral cortex is organized into six excitatory neuronal layers, it is unclear whether glial cells are diversified to mimic neuronal laminae or show distinct layering. To determine the molecular architecture of the mammalian cortex, we developed a high content pipeline that can quantify single-cell gene expression in situ. The Large-area Spatial Transcriptomic (LaST) map confirmed expected cortical neuron layer organization and also revealed a novel neuronal identity signature. Screening 46 candidate genes for astrocyte diversity across the cortex, we identified grey matter superficial, mid and deep astrocyte identities in gradient layer patterns that were distinct from neurons. Astrocyte layers formed in early postnatal cortex and mostly persisted in adult mouse and human cortex. Mutations that shifted neuronal post-mitotic identity or organization were sufficient to alter glial layering, indicating an instructive role for neuronal cues. In normal mouse cortex, astrocyte layer patterns showed area diversity between functionally distinct cortical regions. These findings indicate that excitatory neurons and astrocytes cells are organized into distinct lineage-associated laminae, which give rise to higher order neuroglial complexity of cortical architecture.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/quiescent_cells_actively_replenish_cenp-a_nucleosomes_to_maintain_centromere_identity_and_proliferative_potential_49147569/">
          <h4 class="font-weight-bold">Quiescent cells actively replenish CENP-A nucleosomes to maintain centromere identity and proliferative potential, bioRxiv, 2018-10-02</h4>
          <p>SummaryCentromeres provide a robust model for epigenetic inheritance as they are specified by sequence-independent mechanisms involving the histone H3-variant CENP-A. Prevailing models indicate that the high intrinsic stability of CENP-A nucleosomes maintains centromere identity indefinitely. Here, we demonstrate that CENP-A is not stable at centromeres, but is instead gradually and continuously incorporated in quiescent cells including G0-arrested tissue culture cells and prophase I-arrested oocytes. Quiescent CENP-A incorporation involves the canonical CENP-A deposition machinery, but displays distinct requirements from cell cycle-dependent deposition. We demonstrate that Plk1 is required specifically for G1 CENP-A deposition, whereas transcription promotes CENP-A incorporation in quiescent oocytes. Preventing CENP-A deposition during quiescence results in significantly reduced CENP-A levels and perturbs chromosome segregation following the resumption of cell division. In contrast to quiescent cells, terminally differentiated cells fail to maintain CENP-A levels. Our work reveals that quiescent cells actively maintain centromere identity providing an indicator of proliferative potential.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_mouse_tissue_atlas_of_small_non-coding_rna_49028091/">
          <h4 class="font-weight-bold">A mouse tissue atlas of small non-coding RNA, bioRxiv, 2018-09-29</h4>
          <p>SUMMARYSmall non-coding RNAs (ncRNAs) play a vital role in a broad range of biological processes both in health and disease. A comprehensive quantitative reference of small ncRNA expression would significantly advance our understanding of ncRNA roles in shaping tissue functions. Here, we systematically profiled the levels of five ncRNA classes (miRNA, snoRNA, snRNA, scaRNA and tRNA fragments) across eleven mouse tissues by deep sequencing. Using fourteen biological replicates spanning both sexes, we identified that ~ 30% of small ncRNAs are distributed across the body in a tissue-specific manner with some are also being sexually dimorphic. We found that miRNAs are subject to “arm switching” between healthy tissues and that tRNA fragments are retained within tissues in both a gene- and a tissue-specific manner. Out of eleven profiled tissues we confirmed that brain contains the largest number of unique small ncRNA transcripts, some of which were previously annotated while others are identified for the first time in this study. Furthermore, by combining these findings with single-cell ATAC-seq data, we were able to connect identified brain-specific ncRNA with their cell types of origin. These results yield the most comprehensive characterization of specific and ubiquitous small RNAs in individual murine tissues to date, and we expect that this data will be a resource for the further identification of ncRNAs involved in tissue-function in health and dysfunction in disease.HIGHLIGHTS&lt;jatslist list-type=simple&gt;&lt;jatslist-item&gt;-An atlas of tissue levels of multiple small ncRNA classes generated from 14 biological replicates of both sexes across 11 tissues&lt;jatslist-item&gt;&lt;jatslist-item&gt;-Distinct distribution patterns of miRNA arms and tRNA fragments across tissues suggest the existence of tissue-specific mechanisms of ncRNA cleavage and retention&lt;jatslist-item&gt;&lt;jatslist-item&gt;-miRNA expression is sex specific in healthy tissues&lt;jatslist-item&gt;&lt;jatslist-item&gt;-Small RNA-seq and scATAC-seq data integration produce a detailed map of cell-type specific ncRNA profiles in the mouse brain&lt;jatslist-item&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/pathogen_detection_and_microbiome_analysis_of_infected_wheat_using_a_portable_dna_sequencer_49017775/">
          <h4 class="font-weight-bold">Pathogen Detection and Microbiome Analysis of Infected Wheat Using a Portable DNA Sequencer, bioRxiv, 2018-09-29</h4>
          <p>ABSTRACTFungal diseases of plants are responsible for major losses in agriculture, highlighting the need for rapid and accurate identification of plant pathogens. Disease outcomes are often defined not only by the main pathogen but are influenced by diverse microbial communities known as the microbiome at sites of infection. Here we present the first use of whole genome shot-gun sequencing with a portable DNA sequencing device as a method for the detection of fungal pathogens from wheat (Triticum aestivum) in a standard molecular biology laboratory. The data revealed that our method is robust and applicable to the diagnosis of fungal diseases including wheat stripe rust (caused by Puccinia striiformis f. sp. tritici), septoria tritici blotch (caused by Zymoseptoria tritici) and yellow leaf spot (caused by Pyrenophora tritici repentis). We also identified the bacterial genus Pseudomonas co-present with Puccinia and Zymoseptoria but not Pyrenophora infections. One limitation of the method is the over-representation of redundant wheat genome sequences from samples. This could be addressed by long-range amplicon-based sequencing approaches in future studies, which specifically target non-host organisms. Our work outlines a new approach for detection of a broad range of plant pathogens and associated microbes using a portable sequencer in a standard laboratory, providing the basis for future development of an on-site disease monitoring system.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/reconstructing_the_human_first_trimester_fetalmaternal_interface_using_single_cell_transcriptomics_49017765/">
          <h4 class="font-weight-bold">Reconstructing the human first trimester fetal–maternal interface using single cell transcriptomics, bioRxiv, 2018-09-29</h4>
          <p>SummaryDuring the early weeks of human pregnancy, the fetal placenta implants into the uterine mucosa (decidua) where placental trophoblast cells intermingle and communicate with maternal cells. Here, we profile transcriptomes of ∼50,000 single cells from this unique microenvironment, sampling matched first trimester maternal blood and decidua, and fetal cells from the placenta itself. We define the cellular composition of human decidua, revealing five distinct subsets of decidual fibroblasts with differing growth factors and hormone production profiles, and show that fibroblast states define two distinct decidual layers. Among decidual NK cells, we resolve three subsets, each with a different immunomodulatory and chemokine profile. We develop a repository of ligand-receptor pairs (&lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpwww.CellPhoneDB.org&gt;www.CellPhoneDB.org&lt;jatsext-link&gt;) and a statistical tool to predict the probability of cell-cell interactions via these pairs, highlighting specific interactions between decidual NK cells and invading fetal extravillous trophoblast cells, maternal immune and stromal cells. Our single cell atlas of the maternal-fetal interface reveals the cellular organization and interactions critical for placentation and reproductive success.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_ucsc_repeat_browser_allows_discovery_and_visualization_of_evolutionary_conflict_across_repeat_families_48981990/">
          <h4 class="font-weight-bold">The UCSC Repeat Browser allows discovery and visualization of evolutionary conflict across repeat families, bioRxiv, 2018-09-28</h4>
          <p>ABSTRACTBackgroundNearly half the human genome consists of repeat elements, most of which are retrotransposons, and many of these sequences play important biological roles. However repeat elements pose several unique challenges to current bioinformatic analyses and visualization tools, as short repeat sequences can map to multiple genomic loci resulting in their misclassification and misinterpretation. In fact, sequence data mapping to repeat elements are often discarded from analysis pipelines. Therefore, there is a continued need for standardized tools and techniques to interpret genomic data of repeats.ResultsWe present the UCSC Repeat Browser, which consists of a complete set of human repeat reference sequences derived from the gold standard repeat database RepeatMasker. The UCSC Repeat Browser contains mapped annotations from the human genome to these references, and presents all of them as a comprehensive interface to facilitate work with repetitive elements. Furthermore, it provides processed tracks of multiple publicly available datasets of biological interest to the repeat community, including ChIP-SEQ datasets for KRAB Zinc Finger Proteins (KZNFs) – a family of proteins known to bind and repress certain classes of repeats. Here we show how the UCSC Repeat Browser in combination with these datasets, as well as RepeatMasker annotations in several non-human primates, can be used to trace the independent trajectories of species-specific evolutionary conflicts.ConclusionsThe UCSC Repeat Browser allows easy and intuitive visualization of genomic data on consensus repeat elements, circumventing the problem of multi-mapping, in which sequencing reads of repeat elements map to multiple locations on the human genome. By developing a reference consensus, multiple datasets and annotation tracks can easily be overlaid to reveal complex evolutionary histories of repeats in a single interactive window. Specifically, we use this approach to retrace the history of several primate specific LINE-1 families across apes, and discover several species-specific routes of evolution that correlate with the emergence and binding of KZNFs.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/70/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/67/" class="page-link">67</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/68/" class="page-link">68</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/69/" class="page-link">69</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/70/" class="page-link">70</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/71/" class="page-link">71</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/72/" class="page-link">72</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/73/" class="page-link">73</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/74/" class="page-link">74</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/75/" class="page-link">75</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/72/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/0-100-users/page/123/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
