<!DOCTYPE html>
<html><head>
  
  <title>Genetics | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genetic_associations_with_mathematics_tracking_and_persistence_in_secondary_school_58599187/">
          <h4 class="font-weight-bold">Genetic Associations with Mathematics Tracking and Persistence in Secondary School, bioRxiv, 2019-04-05</h4>
          <p>Maximizing the flow of students through the science, technology, engineering, and math (STEM) pipeline is important to promoting human capital development and reducing economic inequality1. A critical juncture in the STEM pipeline is the highly-cumulative sequence of secondary school math courses2–5. Students from disadvantaged schools are less likely to complete advanced math courses, but debate continues about why6,7. Here, we address this question using student polygenic scores, which are DNA-based indicators of propensity to succeed in education8. We integrated genetic and official school transcript data from over 3,000 European-ancestry students from U.S. high schools. We used polygenic scores as a molecular tracer to understand how the flow of students through the high school math pipeline differs in socioeconomically advantaged versus disadvantaged schools. Students with higher education polygenic scores were tracked to more advanced math already at the beginning of high school and persisted in math for more years. Molecular tracer analyses revealed that the dynamics of the math pipeline differed by school advantage. Compared to disadvantaged schools, advantaged schools tracked more students with high polygenic scores into advanced math classes at the start of high school, and they buffered students with low polygenic scores from dropping out of math. Across all schools, even students with exceptional polygenic scores (top 2%) were unlikely to take the most advanced math classes, suggesting substantial room for improvement in the development of potential STEM talent. These results link new molecular genetic discoveries to a common target of educational-policy reforms.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/allododecaploid_yeasts_synthetic_hybrids_of_six_species_58388689/">
          <h4 class="font-weight-bold">Allododecaploid yeasts synthetic hybrids of six species, bioRxiv, 2019-04-03</h4>
          <p>AbstractPolyploidy generates diversity by increasing the number of copies of each chromosome. Many plants, animals, fungi, and other eukaryotes are ancient or recent polyploids, including some of the best-known evolutionary radiations, crops, and industrial organisms. Polyploidy facilitates differentiation and adaptation to new environments, but the tools to test its limits are lacking. Here we develop an iterative Hybrid Production (iHyPr) method to produce allododecaploid yeast strains with a base ploidy of 12n. Chromosomal instability increased dramatically as additional copies of the genome were added. These six-species hybrids rapidly improved their fitness during adaptive laboratory evolution. This new method for making synthetic hybrids will enable basic research on polyploidy, cancer, and chromosome biology, as well as more applied research on biofuels, bioproducts, and synthetic biology.One sentence summaryWe constructed six-species synthetic hybrids and showed that they were chromosomally unstable but able to adapt rapidly.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genetic_compensation_triggered_by_mutant_mrna_degradation_58448166/">
          <h4 class="font-weight-bold">Genetic compensation triggered by mutant mRNA degradation, Nature, 2019-04-03</h4>
          <p>Genetic robustness, or the ability of an organism to maintain fitness in the presence of harmful mutations, can be achieved via protein feedback loops. Previous work has suggested that organisms may also respond to mutations by transcriptional adaptation, a process by which related gene(s) are upregulated independently of protein feedback loops. However, the prevalence of transcriptional adaptation and its underlying molecular mechanisms are unknown. Here, by analysing several models of transcriptional adaptation in zebrafish and mouse, we uncover a requirement for mutant mRNA degradation. Alleles that fail to transcribe the mutated gene do not exhibit transcriptional adaptation, and these alleles give rise to more severe phenotypes than alleles displaying mutant mRNA decay. Transcriptome analysis in alleles displaying mutant mRNA decay reveals the upregulation of a substantial proportion of the genes that exhibit sequence similarity with the mutated gene&#39;s mRNA, suggesting a sequence-dependent mechanism. These findings have implications for our understanding of disease-causing mutations, and will help in the design of mutant alleles with minimal transcriptional adaptation-derived compensation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/nature">nature</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/synthetic_hybrids_of_six_yeast_species_58388689/">
          <h4 class="font-weight-bold">Synthetic hybrids of six yeast species, bioRxiv, 2019-04-03</h4>
          <p>AbstractAllopolyploidy generates diversity by increasing the number of copies and sources of chromosomes. Many of the best-known evolutionary radiations, crops, and industrial organisms are ancient or recent allopolyploids. Allopolyploidy promotes differentiation and facilitates adaptation to new environments, but the tools to test its limits are lacking. Here we develop an iterative method to combine the genomes of multiple budding yeast species, generating Saccharomyces allopolyploids of an unprecedented scale. Chromosomal instability and cell size increased dramatically as additional copies of the genome were added, but we were able to construct synthetic hybrids of up to six species. The six-species hybrids initially grew slowly, but they rapidly adapted when selection to a novel environment was applied, even as they retained traits from multiple species. These new synthetic yeast hybrids have potential applications for the study of polyploidy, genome stability, chromosome segregation, cancer, and bioenergy.One sentence summaryWe constructed six-species synthetic hybrids and showed that they were chromosomally unstable but able to adapt rapidly.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/crispr-cas9_gene_editing_in_lizards_through_microinjection_of_unfertilized_oocytes_58188400/">
          <h4 class="font-weight-bold">CRISPR-Cas9 Gene Editing in Lizards Through Microinjection of Unfertilized Oocytes, bioRxiv, 2019-04-02</h4>
          <p>AbstractCRISPR-Cas9 mediated gene editing has enabled the direct manipulation of gene function in many species. However, the reproductive biology of reptiles presents unique barriers for the use of this technology, and there are currently no reptiles with effective methods for targeted mutagenesis. Here we present a new approach that enables the efficient production of CRISPR-Cas9 induced mutations in Anolis lizards, an important model for studies of reptile evolution and development.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_mutational_landscape_of_a_prion-like_domain_58188578/">
          <h4 class="font-weight-bold">The mutational landscape of a prion-like domain, bioRxiv, 2019-04-02</h4>
          <p>AbstractSpecific insoluble protein aggregates are the hallmarks of many neurodegenerative diseases1–5. For example, cytoplasmic aggregates of the RNA-binding protein TDP-43 are observed in 97% of cases of Amyotrophic Lateral Sclerosis (ALS)6,7. However, it is still unclear for ALS and other diseases whether it is the insoluble aggregates or other forms of the mutated proteins that cause these diseases that are actually toxic to cells8–13. Here we address this question for TDP-43 by systematically mutating14 the protein and quantifying the effects on cellular toxicity. We generated &amp;gt;50,000 mutations in the intrinsically disordered prion-like domain (PRD) and observed that changes in hydrophobicity and aggregation potential are highly predictive of changes in toxicity. Surprisingly, however, increased hydrophobicity and cytoplasmic aggregation actually reduce cellular toxicity. Mutations have their strongest effects in a central region of the PRD, with variants that increase toxicity promoting the formation of more dynamic liquid-like condensates. The genetic interactions in double mutants reveal that specific structures exist in this ‘unstructured’ region in vivo. Our results demonstrate that deep mutagenesis is a powerful approach for probing the sequence-function relationships of intrinsically disordered proteins as well as their in vivo structural conformations. Moreover, we show that aggregation of TDP-43 is not harmful but actually protects cells, most likely by titrating the protein away from a toxic liquid-like phase.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/genetics/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/genetics/page/9/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/genetics/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/genetics/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/genetics/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/genetics/page/9/" class="page-link">9</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/genetics/page/10/" class="page-link">10</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/genetics/page/11/" class="page-link">11</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/genetics/page/12/" class="page-link">12</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/genetics/page/13/" class="page-link">13</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/genetics/page/14/" class="page-link">14</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/genetics/page/11/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/genetics/page/36/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
