<!DOCTYPE html>
<html><head>
  
  <title>100 200 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_high_abortion_cost_of_human_reproduction_45207903/">
          <h4 class="font-weight-bold">The high abortion cost of human reproduction, bioRxiv, 2018-07-18</h4>
          <p>Information from many large data bases and published studies was integrated to estimate the age-specific spontaneous abortion rate in an economically-developed human population. Accuracy was tested with published data from a diverse array of studies. Spontaneous abortion was found to be i) the predominant outcome of fertilization and ii) a natural and inevitable part of human reproduction at all ages. The decision to reproduce is inextricably coupled with the production of spontaneous abortions with high probability, and the decision to have a large family leads to many spontaneous abortions with virtual certainty. The lifetime number of spontaneous abortions was estimated for a “canonical” woman (constrained to have average age at marriage, first birth, inter-birth intervals, and family size) in two populations one with and the other without effective birth control (including free access to elective abortions). Birth control was found to reduce lifetime abortions more than 6-fold.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/physiology">physiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/speed_breeding_in_growth_chambers_and_glasshouses_for_crop_breeding_and_model_plant_research_45092716/">
          <h4 class="font-weight-bold">Speed breeding in growth chambers and glasshouses for crop breeding and model plant research, bioRxiv, 2018-07-16</h4>
          <p>1.AbstractTo meet the challenge of feeding a growing population, breeders and scientists are continuously looking for ways to increase genetic gain in crop breeding. One way this can be achieved is through “speed breeding” (SB), which shortens the breeding cycle and accelerates research studies through rapid generation advancement. The SB method can be carried out in a number of ways, one of which involves extending the duration of a plant’s daily exposure to light (photoperiod) combined with early seed harvest in order to cycle quickly from seed to seed, thereby reducing the generation times for some long-day (LD) or day-neutral crops. Here we present glasshouse and growth chamber-based SB protocols with supporting data from experimentation with several crop species. These protocols describe the growing conditions, including soil media composition, lighting, temperature and spacing, which promote rapid growth of spring and winter bread wheat, durum wheat, barley, oat, various members of the Brassica family, chickpea, pea, grasspea, quinoa and the model grass Brachypodium distachyon. Points of flexibility within the protocols are highlighted, including how plant density can be increased to efficiently scale-up plant numbers for single seed descent (SSD) purposes. Conversely, instructions on how to perform SB on a small-scale by creating a benchtop SB growth cabinet that enables optimization of parameters at a low cost are provided. We also outline the procedure for harvesting and germinating premature wheat, barley and pea seed to reduce generation time. Finally, we provide troubleshooting suggestions to avoid potential pitfalls.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/what_is_a_cognitive_map_organising_knowledge_for_flexible_behaviour_44855365/">
          <h4 class="font-weight-bold">What is a cognitive map? Organising knowledge for flexible behaviour, bioRxiv, 2018-07-10</h4>
          <p>AbstractIt is proposed that a cognitive map encoding the relationships between entities in the world supports flexible behaviour, but the majority of the neural evidence for such a system comes from studies of spatial navigation. Recent work describing neuronal parallels between spatial and non-spatial behaviours has rekindled the notion of a systematic organisation of knowledge across multiple domains. We review experimental evidence and theoretical frameworks that point to principles unifying these apparently disparate functions. These principles describe how to learn and use abstract, generalisable knowledge and suggest map-like representations observed in a spatial context may be an instance of general coding mechanisms capable of organising knowledge of all kinds. We highlight how artificial agents endowed with such principles exhibit flexible behaviour and learn map-like representations observed in the brain. Finally, we speculate on how these principles may offer insight into the extreme generalisations, abstractions and inferences that characterise human cognition.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/single-cell_isoform_rna_sequencing_scisor-seq_across_thousands_of_cells_reveals_isoforms_of_cerebellar_cell_types_44708839/">
          <h4 class="font-weight-bold">Single-cell isoform RNA sequencing (ScISOr-Seq) across thousands of cells reveals isoforms of cerebellar cell types, bioRxiv, 2018-07-09</h4>
          <p>AbstractFull-length isoform sequencing has advanced our knowledge of isoform biology1–11. However, apart from applying full-length isoform sequencing to very few single cells12,13, isoform sequencing has been limited to bulk tissue, cell lines, or sorted cells. Single splicing events have been described for &amp;lt;=200 single cells with great statistical success14,15, but these methods do not describe full-length mRNAs. Single cell short-read 3’ sequencing has allowed identification of many cell sub-types16–23, but full-length isoforms for these cell types have not been profiled. Using our new method of single-cell-isoform-RNA-sequencing (ScISOr-Seq) we determine isoform-expression in thousands of individual cells from a heterogeneous bulk tissue (cerebellum), without specific antibody-fluorescence activated cell sorting. We elucidate isoform usage in high-level cell types such as neurons, astrocytes and microglia and finer sub-types, such as Purkinje cells and Granule cells, including the combination patterns of distant splice sites6–9,24,25, which for individual molecules requires long reads. We produce an enhanced genome annotation revealing cell-type specific expression of known and 16,872 novel (with respect to mouse Gencode version 10) isoforms (see isoformatlas.com).ScISOr-Seq describes isoforms from &amp;gt;1,000 single cells from bulk tissue without cell sorting by leveraging two technologies in three steps In step one, we employ microfluidics to produce amplified full-length cDNAs barcoded for their cell of origin. This cDNA is split into two pools one pool for 3’ sequencing to measure gene expression (step 2) and another pool for long-read sequencing and isoform expression (step 3). In step two, short-read 3’-sequencing provides molecular counts for each gene and cell, which allows clustering cells and assigning a cell type using cell-type specific markers. In step three, an aliquot of the same cDNAs (each barcoded for the individual cell of origin) is sequenced using Pacific Biosciences (“PacBio”)1,2,4,5,26 or Oxford Nanopore3. Since these long reads carry the single-cell barcodes identified in step two, one can determine the individual cell from which each long read originates. Since most single cells are assigned to a named cluster, we can also assign the cell’s cluster name (e.g. “Purkinje cell” or “astrocyte”) to the long read in question (Fig 1A) – without losing the cell of origin of each long read.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/super-mendelian_inheritance_mediated_by_crisprcas9_in_the_female_mouse_germline_44494359/">
          <h4 class="font-weight-bold">Super-Mendelian inheritance mediated by CRISPRCas9 in the female mouse germline, bioRxiv, 2018-07-04</h4>
          <p>AbstractA gene drive biases the transmission of a particular allele of a gene such that it is inherited at a greater frequency than by random assortment. Recently, a highly efficient gene drive was developed in insects, which leverages the sequence-targeted DNA cleavage activity of CRISPRCas9 and endogenous homology directed repair mechanisms to convert heterozygous genotypes to homozygosity. If implemented in laboratory rodents, this powerful system would enable the rapid assembly of genotypes that involve multiple genes (e.g., to model multigenic human diseases). Such complex genetic models are currently precluded by time, cost, and a requirement for a large number of animals to obtain a few individuals of the desired genotype. However, the efficiency of a CRISPRCas9 gene drive system in mammals has not yet been determined. Here, we utilize an active genetic “CopyCat” element embedded in the mouse Tyrosinase gene to detect genotype conversions after Cas9 activity in the embryo and in the germline. Although Cas9 efficiently induces double strand DNA breaks in the early embryo and is therefore highly mutagenic, these breaks are not resolved by homology directed repair. However, when Cas9 expression is limited to the developing female germline, resulting double strand breaks are resolved by homology directed repair that copies the CopyCat allele from the donor to the receiver chromosome and leads to its super-Mendelian inheritance. These results demonstrate that the CRISPRCas9 gene drive mechanism can be implemented to simplify complex genetic crosses in laboratory mice and also contribute valuable data to the ongoing debate about applications to combat invasive rodent populations in island communities.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/super-resolution_fight_club_a_broad_assessment_of_2d__3d_single-molecule_localization_microscopy_software_44493765/">
          <h4 class="font-weight-bold">Super-resolution fight club A broad assessment of 2D &amp; 3D single-molecule localization microscopy software, bioRxiv, 2018-07-04</h4>
          <p>ABSTRACTWith the widespread uptake of 2D and 3D single molecule localization microscopy, a large set of different data analysis packages have been developed to generate super-resolution images. To guide researchers on the optimal analytical software for their experiments, we have designed, in a large community effort, a competition to extensively characterise and rank these options. We generated realistic simulated datasets for popular imaging modalities – 2D, astigmatic 3D, biplane 3D, and double helix 3D – and evaluated 36 participant packages against these data. This provides the first broad assessment of 3D single molecule localization microscopy software, provides a holistic view of how the latest 2D and 3D single molecule localization software perform in realistic conditions, and ultimately provides insight into the current limits of the field.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/100-200-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/100-200-users/page/81/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/78/" class="page-link">78</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/79/" class="page-link">79</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/80/" class="page-link">80</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/81/" class="page-link">81</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/100-200-users/page/82/" class="page-link">82</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/83/" class="page-link">83</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/84/" class="page-link">84</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/85/" class="page-link">85</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/86/" class="page-link">86</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/100-200-users/page/83/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/100-200-users/page/126/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
