<!DOCTYPE html>
<html><head>
  
  <title>Reports | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_hippocampus_is_necessary_for_the_sleep-dependent_consolidation_of_a_task_that_does_not_require_the_hippocampus_for_initial_learning_50127132/">
          <h4 class="font-weight-bold">The hippocampus is necessary for the sleep-dependent consolidation of a task that does not require the hippocampus for initial learning, bioRxiv, 2018-10-23</h4>
          <p>AbstractDuring sleep, the hippocampus plays an active role in consolidating memories that depend on it for initial encoding. There are hints in the literature that the hippocampus may have a broader influence, contributing to the consolidation of memories that may not initially require the area. We tested this possibility by evaluating learning and consolidation of the motor sequence task (MST) in hippocampal amnesics and demographically matched control participants. While the groups showed similar initial learning, only controls exhibited evidence of sleep-dependent consolidation. These results demonstrate that the hippocampus can be required for normal consolidation of a task without being required for its acquisition, suggesting that the area plays a broader role in coordinating sleep-dependent memory consolidation than has previously been assumed.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/adding_function_to_the_genome_of_african_salmonella_st313_50075876/">
          <h4 class="font-weight-bold">Adding function to the genome of African Salmonella ST313, bioRxiv, 2018-10-22</h4>
          <p>AbstractSalmonella Typhimurium ST313 causes invasive nontyphoidal Salmonella (iNTS) disease in sub-Saharan Africa, targeting susceptible HIV&#43;, malarial or malnourished individuals. An in-depth genomic comparison between the ST313 isolate D23580, and the well-characterized ST19 isolate 474 that causes gastroenteritis across the globe, revealed extensive synteny. To understand how the 856 nucleotide variations generated phenotypic differences, we devised a large-scale experimental approach that involved the global gene expression analysis of strains D23580 and 474 grown in sixteen infection-relevant growth conditions. Comparison of transcriptional patterns identified virulence and metabolic genes that were differentially expressed between D23580 versus 474, many of which were validated by proteomics. We also uncovered the S. Typhimurium D23580 and 474 genes that showed expression differences during infection of murine macrophages. Our comparative transcriptomic data are presented in a new enhanced version of the Salmonella expression compendium SalComD23580 &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpbioinf.gen.tcd.iecgi-binsalcom_v2.pl&gt;bioinf.gen.tcd.iecgi-binsalcom_v2.pl&lt;jatsext-link&gt;. We discovered that the ablation of melibiose utilization was caused by 3 independent SNP mutations in D23580 that are shared across ST313 lineage 2, suggesting that the ability to catabolise this carbon source has been negatively selected during ST313 evolution. The data revealed a novel plasmid maintenance system involving a plasmid-encoded CysS cysteinyl-tRNA synthetase, highlighting the power of large-scale comparative multi-condition analyses to pinpoint key phenotypic differences between bacterial pathovariants.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/navigome_navigating_the_human_phenome_50081743/">
          <h4 class="font-weight-bold">Navigome Navigating the Human Phenome, bioRxiv, 2018-10-22</h4>
          <p>AbstractWe now have access to a sufficient number of genome-wide association studies (GWAS) to cluster phenotypes into genetic-informed categories and to navigate the “phenome” space of human traits. Using a collection of 465 GWAS, we generated genetic correlations, pathways, gene-wise and tissue-wise associations using MAGMA and S-PrediXcan for 465 human traits. Testing 7267 biological pathways, we found that only 898 were significantly associated with any trait. Similarly, out of ~20,000 tested protein-coding genes, 12,311 genes exhibited an association. Based on the genetic correlations between all traits, we constructed a phenome map using t-distributed stochastic neighbor embedding (t-SNE), where each of the 465 traits can be visualized as an individual point. This map reveals well-defined clusters of traits such as educationhigh longevity, lower longevity, height, body composition, and depressionanxietyneuroticism. These clusters are enriched in specific groups of pathways, such as lipid pathways in the lower longevity cluster, and neuronal pathways for body composition or education clusters. The map and all other analyses are available in the Navigome web interface (&lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsphenviz.navigome.com&gt;httpsphenviz.navigome.com&lt;jatsext-link&gt;).</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/polygenic_scores_for_major_depressive_disorder_and_depressive_symptoms_predict_response_to_lithium_in_patients_with_bipolar_disorder_50081597/">
          <h4 class="font-weight-bold">Polygenic scores for major depressive disorder and depressive symptoms predict response to lithium in patients with bipolar disorder, bioRxiv, 2018-10-22</h4>
          <p>AbstractBackgroundLithium is a first-line medication for bipolar disorder (BD), but only ~30% of patients respond optimally to the drug. Since genetic factors are known to mediate lithium treatment response, we hypothesized whether polygenic susceptibility to the spectrum of depression traits is associated with treatment outcomes in patients with BD. In addition, we explored the potential molecular underpinnings of this relationship.MethodsWeighted polygenic scores (PGSs) were computed for major depressive disorder (MDD) and depressive symptoms (DS) in BD patients from the Consortium on Lithium Genetics (ConLi&#43;Gen; n=2,586) who received lithium treatment. Lithium treatment outcome was assessed using the ALDA scale. Summary statistics from genome-wide association studies (GWAS) in MDD (130,664 cases and 330,470 controls) and DS (n=161,460) were used for PGS weighting. Associations between PGSs of depression traits and lithium treatment response were assessed by binary logistic regression. We also performed a cross-trait meta-GWAS, followed by Ingenuity® Pathway Analysis.OutcomesBD patients with a low polygenic load for depressive traits were more likely to respond well to lithium, compared to patients with high polygenic load (MDD OR =1.64 [95%CI 1.26-2.15], lowest vs highest PGS quartiles; DS OR=1.53 [95%CI 1.18-2.00]). Associations were significant for type 1, but not type 2 BD. Cross-trait GWAS and functional characterization implicated voltage-gated potassium channels, insulin-related pathways, mitogen-activated protein-kinase (MAPK) signaling, and miRNA expression.InterpretationGenetic loading to depression traits in BD patients lower their odds of responding optimally to lithium. Our findings support the emerging concept of a lithium-responsive biotype in BD.FundingSee attached details</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/predicting_the_future_with_multi-scale_successor_representations_50072632/">
          <h4 class="font-weight-bold">Predicting the Future with Multi-scale Successor Representations, bioRxiv, 2018-10-22</h4>
          <p>AbstractThe successor representation (SR) is a candidate principle for generalization in reinforcement learning, computational accounts of memory, and the structure of neural representations in the hippocampus. Given a sequence of states, the SR learns a predictive representation for every given state that encodes how often, on average, each upcoming state is expected to be visited, even if it is multiple steps ahead. A discount or scale parameter determines how many steps into the future SR’s generalizations reach, enabling rapid value computation, subgoal discovery, and flexible decision-making in large trees. However, SR with a single scale could discard information for predicting both the sequential order of and the distance between states, which are common problems in navigation for animals and artificial agents. Here we propose a solution an ensemble of SRs with multiple scales. We show that the derivative of multi-scale SR can reconstruct both the sequence of expected future states and estimate distance to goal. This derivative can be computed linearly we show that a multi-scale SR ensemble is the Laplace transform of future states, and the inverse of this Laplace transform is a biologically plausible linear estimation of the derivative. Multi-scale SR and its derivative could lead to a common principle for how the medial temporal lobe supports both map-based and vector-based navigation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_population_history_of_northeastern_siberia_since_the_pleistocene_50079518/">
          <h4 class="font-weight-bold">The population history of northeastern Siberia since the Pleistocene, bioRxiv, 2018-10-22</h4>
          <p>ABSTRACTFar northeastern Siberia has been occupied by humans for more than 40 thousand years. Yet, owing to a scarcity of early archaeological sites and human remains, its population history and relationship to ancient and modern populations across Eurasia and the Americas are poorly understood. Here, we analyze 34 ancient genome sequences, including two from fragmented milk teeth found at the ~31.6 thousand-year-old (kya) Yana RHS site, the earliest and northernmost Pleistocene human remains found. These genomes reveal complex patterns of past population admixture and replacement events throughout northeastern Siberia, with evidence for at least three large-scale human migrations into the region. The first inhabitants, a previously unknown population of “Ancient North Siberians” (ANS), represented by Yana RHS, diverged ~38 kya from Western Eurasians, soon after the latter split from East Asians. Between 20 and 11 kya, the ANS population was largely replaced by peoples with ancestry related to present-day East Asians, giving rise to ancestral Native Americans and “Ancient Paleosiberians” (AP), represented by a 9.8 kya skeleton from Kolyma River. AP are closely related to the Siberian ancestors of Native Americans, and ancestral to contemporary communities such as Koryaks and Itelmen. Paleoclimatic modelling shows evidence for a refuge during the last glacial maximum (LGM) in southeastern Beringia, suggesting Beringia as a possible location for the admixture forming both ancestral Native Americans and AP. Between 11 and 4 kya, AP were in turn largely replaced by another group of peoples with ancestry from East Asia, the “Neosiberians” from which many contemporary Siberians derive. We detect gene flow events in both directions across the Bering Strait during this time, influencing the genetic composition of Inuit, as well as Na Dene-speaking Northern Native Americans, whose Siberian-related ancestry components is closely related to AP. Our analyses reveal that the population history of northeastern Siberia was highly dynamic throughout the Late Pleistocene and Holocene. The pattern observed in northeastern Siberia, with earlier, once widespread populations being replaced by distinct peoples, seems to have taken place across northern Eurasia, as far west as Scandinavia.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../reports/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../reports/page/171/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/168/" class="page-link">168</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/169/" class="page-link">169</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/170/" class="page-link">170</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/171/" class="page-link">171</a></li>
            
        
             
            <li class="page-item active"><a href="../../../reports/page/172/" class="page-link">172</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/173/" class="page-link">173</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/174/" class="page-link">174</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/175/" class="page-link">175</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/176/" class="page-link">176</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../reports/page/173/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../reports/page/320/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
