<!DOCTYPE html>
<html><head>
  
  <title>Biophysics | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/collective_intercellular_communication_through_ultra-fast_hydrodynamic_trigger_waves_48929251/">
          <h4 class="font-weight-bold">Collective intercellular communication through ultra-fast hydrodynamic trigger waves, bioRxiv, 2018-09-27</h4>
          <p>The biophysical relationships between sensors and actuators have been fundamental to the development of complex life forms; abundant flows are generated and persist in aquatic environments by swimming organisms, while responding promptly to external stimuli is key to survival. Here, akin to a chain reaction, we present the discovery of hydrodynamic trigger waves in cellular communities of the protist Spirostomum ambiguum, propagating hundreds of times faster than the swimming speed. Coiling its cytoskeleton, Spirostomum can contract its long body by 50% within milliseconds, with accelerations reaching 14g-forces. Surprisingly, a single cellular contraction (transmitter) is shown to generate long-ranged vortex flows at intermedi- ate Reynolds numbers, which can trigger neighbouring cells, in turn. To measure the sensitivity to hydrodynamic signals (receiver), we further present a high-throughput suction-flow device to probe mechanosensitive ion channel gating by back-calculating the microscopic forces on the cell mem- brane. These ultra-fast hydrodynamic trigger waves are analysed and modelled quantitatively in a universal framework of antenna and percolation theory. A phase transition is revealed, requiring a critical colony density to sustain collective communication. Our results suggest that this signalling could help organise cohabiting communities over large distances, influencing long-term behaviour through gene expression, comparable to quorum sensing. More immediately, as contractions release toxins, synchronised discharges could also facilitate the repulsion of large predators, or conversely immobilise large prey. We postulate that beyond protists numerous other freshwater and marine organisms could coordinate with variations of hydrodynamic trigger waves.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/relion-3_new_tools_for_automated_high-resolution_cryo-em_structure_determination_48653558/">
          <h4 class="font-weight-bold">RELION-3 new tools for automated high-resolution cryo-EM structure determination, bioRxiv, 2018-09-19</h4>
          <p>AbstractHere, we describe the third major release of relion. CPU-based vector acceleration has been added in addition to GPU support, which provides flexibility in use of resources and avoids memory limitations. Reference-free autopicking with Laplacian-of-Gaussian filtering and execution of jobs from python allows non-interactive processing during acquisition, including 2D-classification, de novo model generation and 3D-classification. Perparticle refinement of CTF parameters and correction of estimated beam tilt provides higher-resolution reconstructions when particles are at different heights in the ice, andor coma-free alignment has not been optimal. Ewald sphere curvature correction improves resolution for large particles. We illustrate these developments with publicly available data sets together with a Bayesian approach to beam-induced motion correction it leads to resolution improvements of 0.2-0.7 Å compared to previous relion versions.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_deadly_touch_protein_denaturation_at_the_water-air_interface_and_how_to_prevent_it_47092133/">
          <h4 class="font-weight-bold">The deadly touch protein denaturation at the water-air interface and how to prevent it, bioRxiv, 2018-08-26</h4>
          <p>ABSTRACTElectron cryo-microscopy analyzes the structure of proteins and protein complexes in vitrified solution. Proteins tend to adsorb to the air-water interface in unsupported films of aqueous solution, which can result in partial or complete denaturation of the protein. We investigated the structure of yeast fatty acid synthase at the air-water interface by electron cryo-tomography and single-particle image processing. Around 90% of complexes adsorbed to the air-water interface are partly denatured. We show that the unfolded regions are those facing the air-water interface. Denaturation by contact with air may happen at any stage of specimen preparation. Denaturation at the air-water interface is completely avoided when the complex is plunge-frozen on a substrate of hydrophilized graphene.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/determining_cellular_ctcf_and_cohesin_abundances_to_constrain_3d_genome_models_45155888/">
          <h4 class="font-weight-bold">Determining cellular CTCF and cohesin abundances to constrain 3D genome models, bioRxiv, 2018-07-18</h4>
          <p>Achieving a quantitative and predictive understanding of 3D genome architecture remains a major challenge, as it requires quantitative measurements of the key proteins involved. Here we report the quantification of CTCF and cohesin, two causal regulators of topological associating domains (TADs) in mammalian cells. Extending our previous imaging studies (Hansen 2017), we estimate bounds on the density of putatively DNA loop-extruding cohesin complexes and CTCF binding site occupancy. Furthermore, co-immunoprecipitation studies of an endogenously tagged subunit (Rad21) suggest the presence of cohesin dimers andor oligomers. Finally, based on our cell lines with accurately measured protein abundances, we report a method to conveniently determine the number of molecules of any Halo-tagged protein in the cell. We anticipate that our results and the established tool for measuring cellular protein abundances will advance a more quantitative understanding of 3D genome organization, and facilitate protein quantification, key for understanding diverse biological processes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/super-resolution_fight_club_a_broad_assessment_of_2d__3d_single-molecule_localization_microscopy_software_44493765/">
          <h4 class="font-weight-bold">Super-resolution fight club A broad assessment of 2D &amp; 3D single-molecule localization microscopy software, bioRxiv, 2018-07-04</h4>
          <p>ABSTRACTWith the widespread uptake of 2D and 3D single molecule localization microscopy, a large set of different data analysis packages have been developed to generate super-resolution images. To guide researchers on the optimal analytical software for their experiments, we have designed, in a large community effort, a competition to extensively characterise and rank these options. We generated realistic simulated datasets for popular imaging modalities – 2D, astigmatic 3D, biplane 3D, and double helix 3D – and evaluated 36 participant packages against these data. This provides the first broad assessment of 3D single molecule localization microscopy software, provides a holistic view of how the latest 2D and 3D single molecule localization software perform in realistic conditions, and ultimately provides insight into the current limits of the field.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/real-time_cryo-em_data_pre-processing_with_warp_43706631/">
          <h4 class="font-weight-bold">Real-time cryo-EM data pre-processing with Warp, bioRxiv, 2018-06-14</h4>
          <p>The acquisition of cryo-electron microscopy (cryo-EM) data from biological specimens is currently largely uncoupled from subsequent data evaluation, correction and processing. Therefore, the acquisition strategy is difficult to optimize during data collection, often leading to suboptimal microscope usage and disappointing results. Here we provide Warp, a software for real-time evaluation, correction, and processing of cryo-EM data during their acquisition. Warp evaluates and monitors key parameters for each recorded micrograph or tomographic tilt series in real time. Warp also rapidly corrects micrographs for global and local motion, and estimates the local defocus with the use of novel algorithms. The software further includes a deep learning-based particle picking algorithm that rivals human accuracy to make the pre-processing pipeline truly automated. The output from Warp can be directly fed into established tools for particle classification and 3D image reconstruction. In a benchmarking study we show that Warp automatically processed a published cryo-EM data set for influenza virus hemagglutinin, leading to an improvement of the nominal resolution from 3.9 Å to 3.2 Å. Warp is easy to install, computationally inexpensive, and has an intuitive and streamlined user interface.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/biophysics/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/biophysics/page/5/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/biophysics/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/biophysics/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/biophysics/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/biophysics/page/5/" class="page-link">5</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/biophysics/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/biophysics/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/biophysics/page/8/" class="page-link">8</a></li>
            
        

        
        <li class="page-item"><a href="../../../../tags/biophysics/page/7/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/biophysics/page/8/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
