<!DOCTYPE html>
<html><head>
  
  <title>100 200 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_transposable_element_insertion_is_the_switch_between_alternative_life_history_strategies_48826894/">
          <h4 class="font-weight-bold">A transposable element insertion is the switch between alternative life history strategies, bioRxiv, 2018-09-24</h4>
          <p>Tradeoffs affect resource allocation during development and result in fitness consequences that drive the evolution of life history strategies. Yet despite their importance, we know little about the mechanisms underlying life history tradeoffs in wild populations. Many species of Colias butterflies exhibit an alternative life history strategy (ALHS) where females divert resources from wing pigment synthesis to reproductive and somatic development. Due to this reallocation, a wing color polymorphism is associated with the ALHS individuals have either yelloworange or white wings. Here we map the genetic basis of the ALHS switch in Colias crocea to a transposable element insertion downstream of the Colias homolog of BarH-1, a homeobox transcription factor. Using CRISPRCas9 gene editing, antibody staining, and electron microscopy we find morph-specific specific expression of BarH-1 suppresses the formation of pigment granules in wing scales. Lipid and transcriptome analyses reveal physiological differences associated with the ALHS. These findings characterize a novel mechanism for a female-limited ALHS and show that the switch arises via recruitment of a transcription factor previously known for its function in cell fate determination in pigment cells of the retina.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/mito-tag_mice_enable_rapid_isolation_and_multimodal_profiling_of_mitochondria_from_specific_cell_types_in_vivo_48845489/">
          <h4 class="font-weight-bold">MITO-Tag Mice enable rapid isolation and multimodal profiling of mitochondria from specific cell types in vivo, bioRxiv, 2018-09-24</h4>
          <p>ABSTRACTMitochondria are metabolic organelles that are essential for mammalian life, but the dynamics of mitochondrial metabolism within mammalian tissues in vivo remains incompletely understood. While whole-tissue metabolite profiling has been useful for studying metabolism in vivo, such an approach lacks resolution at the cellular and subcellular level. In vivo methods for interrogating organellar metabolites in specific cell-types within mammalian tissues have been limited. To address this, we built on prior work in which we exploited a mitochondrially-localized 3XHA epitope-tag (“MITO-Tag”) for the fast isolation of mitochondria from cultured cells to now generate “MITO-Tag Mice.” Affording spatiotemporal control over MITO-Tag expression, these transgenic animals enable the rapid, cell-type-specific immunoisolation of mitochondria from tissues, which we verified using a combination of proteomic and metabolomic approaches. Using MITO-Tag Mice and targeted and untargeted metabolite profiling, we identified changes during fasted and refed conditions in a diverse array of mitochondrial metabolites in hepatocytes and found metabolites that behaved differently at the mitochondrial versus whole-tissue level. MITO-Tag Mice should have utility for studying mitochondrial physiology and our strategy should be generally applicable for studying other mammalian organelles in specific cell-types in vivo.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/how_to_make_a_rodent_giant_genomic_basis_and_tradeoffs_of_gigantism_in_the_capybara_the_worlds_largest_rodent_48807334/">
          <h4 class="font-weight-bold">How to make a rodent giant Genomic basis and tradeoffs of gigantism in the capybara, the world’s largest rodent, bioRxiv, 2018-09-23</h4>
          <p>AbstractGigantism is the result of one lineage within a clade evolving extremely large body size relative to its small-bodied ancestors, a phenomenon observed numerous times in animals. Theory predicts that the evolution of giants should be constrained by two tradeoffs. First, because body size is negatively correlated with population size, purifying selection is expected to be less efficient in species of large body size, leading to a genome-wide elevation of the ratio of non-synonymous to synonymous substitution rates (dNdS) or mutation load. Second, gigantism is achieved through higher number of cells and higher rates of cell proliferation, thus increasing the likelihood of cancer. However, the incidence of cancer in gigantic animals is lower than the theoretical expectation, a phenomenon referred to as Peto’s Paradox. To explore the genetic basis of gigantism in rodents and uncover genomic signatures of gigantism-related tradeoffs, we sequenced the genome of the capybara, the world’s largest living rodent. We found that dNdS is elevated genome wide in the capybara, relative to other rodents, implying a higher mutation load. Conversely, a genome-wide scan for adaptive protein evolution in the capybara highlighted several genes involved in growth regulation by the insulininsulin-like growth factor signaling (IIS) pathway. Capybara-specific gene-family expansions included a putative novel anticancer adaptation that involves T cell-mediated tumor suppression, offering a potential resolution to Peto’s Paradox in this lineage. Gene interaction network analyses also revealed that size regulators function simultaneously as growth factors and oncogenes, creating an evolutionary conflict. Based on our findings, we hypothesize that gigantism in the capybara likely involved three evolutionary steps 1) Increase in body size by cell proliferation through the ISS pathway, 2) coupled evolution of growth-regulatory and cancer-suppression mechanisms, possibly driven by intragenomic conflict, and 3) establishment of the T cell-mediated tumor suppression pathway as an anticancer adaptation. Interestingly, increased mutation load appears to be an inevitable outcome of an increase in body size.Author SummaryThe existence of gigantic animals presents an evolutionary puzzle. Larger animals have more cells and undergo exponentially more cell divisions, thus, they should have enormous rates of cancer. Moreover, large animals also have smaller populations making them vulnerable to extinction. So, how do gigantic animals such as elephants and blue whales protect themselves from cancer, and what are the consequences of evolving a large size on the ‘genetic health’ of a species? To address these questions we sequenced the genome of the capybara, the world’s largest rodent, and performed comparative genomic analyses to identify the genes and pathways involved in growth regulation and cancer suppression. We found that the insulin-signaling pathway was involved in the evolution of gigantism in the capybara. We also found a putative novel anticancer mechanism mediated by the detection of tumors by T-cells, offering a potential solution to how capybaras mitigated the tradeoff imposed by cancer. Furthermore, we show that capybara genome harbors a higher proportion of slightly deleterious mutations relative to all other rodent genomes. Overall, this study provides insights at the genomic level into the evolution of a complex and extreme phenotype, and offers a detailed picture of how the evolution of a giant body size in the capybara has shaped its genome.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/revealing_multi-scale_population_structure_in_large_cohorts_48806746/">
          <h4 class="font-weight-bold">Revealing multi-scale population structure in large cohorts, bioRxiv, 2018-09-23</h4>
          <p>Genetic structure in large cohorts results from technical, sampling and demographic variation. Visualisation is therefore a first step in most genomic analyses. However, existing data exploration methods struggle with unbalanced sampling and the many scales of population structure. We investigate an approach to dimension reduction of genomic data that combines principal components analysis (PCA) with uniform manifold approximation and projection (UMAP) to succinctly illustrate population structure in large cohorts and capture their relationships on local and global scales. Using data from large-scale genomic datasets, we demonstrate that PCA-UMAP effectively clusters closely related individuals while placing them in a global continuum of genetic variation. This approach reveals previously overlooked subpopulations within the American Hispanic population and fine-scale relationships between geography, genotypes, and phenotypes in the UK population. This opens new lines of investigation for demographic research and statistical genetics. Given its small computational cost, PCA-UMAP also provides a general-purpose approach to exploratory analysis in population-scale datasets.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_multi-domain_task_battery_reveals_functional_boundaries_in_the_human_cerebellum_48724700/">
          <h4 class="font-weight-bold">A Multi-Domain Task Battery Reveals Functional Boundaries in the Human Cerebellum, bioRxiv, 2018-09-21</h4>
          <p>AbstractThere is compelling evidence that the human cerebellum is engaged in a wide array of motor and cognitive tasks. A fundamental question centers on whether the cerebellum is organized into distinct functional sub-regions. To address this question, we employed a rich task battery, designed to tap into a broad range of cognitive processes. During four functional magnetic resonance imaging (fMRI) sessions, participants performed a battery of 26 diverse tasks comprising 47 unique conditions. Using the data from this multi-domain task battery (MDTB), we derived a comprehensive functional parcellation of the cerebellar cortex and evaluated it by predicting functional boundaries in a novel set of tasks. The new parcellation successfully identified distinct functional sub-regions, providing significant improvements over existing parcellations derived from task-free data. Lobular boundaries, commonly used to summarize functional data, did not coincide with functional subdivisions. This multi-domain task approach offers novel insights into the functional heterogeneity of the cerebellar cortex.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/objective_versus_self-reported_energy_intake_changes_during_low-carbohydrate_and_low-fat_diets_48703297/">
          <h4 class="font-weight-bold">Objective versus Self-Reported Energy Intake Changes During Low-Carbohydrate and Low-Fat Diets, bioRxiv, 2018-09-21</h4>
          <p>AbstractObjectiveTo examine objective versus self-reported energy intake changes (ΔEI) during a 12-month diet intervention.MethodsWe calculated ΔEI in subjects who participated in a 1-year randomized low-carbohydrate versus low-fat diet trial using repeated body weight measurements as inputs to an objective mathematical model (ΔEIModel) and compared these values with self-reported energy intake changes assessed by repeated 24-hr recalls (ΔEI24hrRecall).ResultsΔEI24hrRecall indicated a relatively persistent state of calorie restriction ≥500 kcald throughout the year with no significant differences between diets. ΔEIModel demonstrated large early decreases in calorie intake &amp;gt;800 kcald followed by an exponential return to approximately 100 kcald below baseline at the end of the year. The low-carbohydrate diet resulted in ΔEIModel that was 162±53 kcald lower than the low-fat diet over the first 3 months (p=0.002), but no significant diet differences were found at later times. Weight loss at 12 months was significantly related to ΔEIModel at all time intervals for both diets (p&amp;lt;0.0001).ConclusionsSelf-reported measurements of ΔEI were inaccurate. Model-based calculations of ΔEI found that instructions to follow the low-carbohydrate diet resulted in greater calorie restriction than the low-fat diet in the early phases of the intervention, but these diet differences were not sustained.What is already known about this subject?&lt;jatslist list-type=bullet&gt;&lt;jatslist-item&gt;Diet assessments that rely on self-report, such as 24hr dietary recall, are known to underestimate actual energy intake as measured by doubly labeled water. However, it is possible that repeated self-reported measurements could accurately detect changes in energy intake over time if the absolute bias of self-reported of measurements is approximately constant for each subject.&lt;jatslist-item&gt;What this study adds&lt;jatslist list-type=bullet&gt;&lt;jatslist-item&gt;We compared energy intake changes measured using repeated 24hr dietary recall measurements collected over the course of the 1-year Diet Intervention Examining The Factors Interacting with Treatment Success (DIETFITS) trial versus energy intake changes calculated using repeated body weight measurements as inputs to a validated mathematical model.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Whereas self-reported measurements indicated a relatively persistent state of calorie restriction, objective model-based measurements demonstrated a large early calorie restriction followed by an exponential rise in energy intake towards the pre-intervention baseline.&lt;jatslist-item&gt;&lt;jatslist-item&gt;Model-based calculations, but not self-reported measurements, found that low-carbohydrate diets led to significantly greater early decreases in energy intake compared to low-fat diets, but long-term energy intake changes were not significantly different.&lt;jatslist-item&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/physiology">physiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/100-200-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/100-200-users/page/49/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/46/" class="page-link">46</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/47/" class="page-link">47</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/48/" class="page-link">48</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/49/" class="page-link">49</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/100-200-users/page/50/" class="page-link">50</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/51/" class="page-link">51</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/52/" class="page-link">52</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/53/" class="page-link">53</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/54/" class="page-link">54</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/100-200-users/page/51/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/100-200-users/page/68/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
