<!DOCTYPE html>
<html><head>
  
  <title>Cell Biology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/quiescent_cells_actively_replenish_cenp-a_nucleosomes_to_maintain_centromere_identity_and_proliferative_potential_49147569/">
          <h4 class="font-weight-bold">Quiescent cells actively replenish CENP-A nucleosomes to maintain centromere identity and proliferative potential, bioRxiv, 2018-10-02</h4>
          <p>SummaryCentromeres provide a robust model for epigenetic inheritance as they are specified by sequence-independent mechanisms involving the histone H3-variant CENP-A. Prevailing models indicate that the high intrinsic stability of CENP-A nucleosomes maintains centromere identity indefinitely. Here, we demonstrate that CENP-A is not stable at centromeres, but is instead gradually and continuously incorporated in quiescent cells including G0-arrested tissue culture cells and prophase I-arrested oocytes. Quiescent CENP-A incorporation involves the canonical CENP-A deposition machinery, but displays distinct requirements from cell cycle-dependent deposition. We demonstrate that Plk1 is required specifically for G1 CENP-A deposition, whereas transcription promotes CENP-A incorporation in quiescent oocytes. Preventing CENP-A deposition during quiescence results in significantly reduced CENP-A levels and perturbs chromosome segregation following the resumption of cell division. In contrast to quiescent cells, terminally differentiated cells fail to maintain CENP-A levels. Our work reveals that quiescent cells actively maintain centromere identity providing an indicator of proliferative potential.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/transient_intracellular_acidification_regulates_the_core_transcriptional_heat_shock_response_48185096/">
          <h4 class="font-weight-bold">Transient intracellular acidification regulates the core transcriptional heat shock response, bioRxiv, 2018-09-12</h4>
          <p>AbstractCellular stress induces rapid expression of genes encoding molecular chaperones. In many eukaryotes, stress also triggers transient intracellular acidification which, by unknown mechanisms, is associated with increased survival. Here, using budding yeast as a model, we discover that preventing cells from transiently acidifying during heat shock compromises induction of molecular chaperones and fitness. Prevention of acidification during stress and recovery silences induction of a canonical heat-shock protein altogether. The association between acidification, induction, and growth holds at the population and single-cell levels. Hinting at the molecular basis of these effects, the failure to acidify specifically suppresses induction of genes regulated by the conserved heat shock transcription factor Hsf1. Our results establish a central role for intracellular pH in the eukaryotic transcriptional stress response, and implicate pH-sensitive stress-sensing proteins, rather than misfolded proteins, in the activation of Hsf1 under physiological heat shock conditions.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/systematic_assessment_of_gfp_tag_position_on_protein_localization_and_growth_fitness_in_yeast_44413054/">
          <h4 class="font-weight-bold">Systematic assessment of GFP tag position on protein localization and growth fitness in yeast, bioRxiv, 2018-07-02</h4>
          <p>AbstractWhile protein tags are ubiquitously utilized in molecular biology, they harbor the potential to interfere with functional traits of their fusion counterparts. Systematic evaluation of the effect of protein tags on localization and function would promote accurate use of tags in experimental setups. Here we examine the effect of Green Fluorescent Protein (GFP) tagging at either the N or C terminus of budding yeast proteins on localization and functionality. We use a competition-based approach to decipher the relative fitness of two strains tagged on the same protein but on opposite termini and from that infer the correct, physiological localization for each protein and the optimal position for tagging. Our study provides a first of a kind systematic assessment of the effect of tags on the functionality of proteins and provides step towards broad investigation of protein fusion libraries.Highlights&lt;jatslist list-type=bullet&gt;&lt;jatslist-item&gt;Protein tags are widely used in molecular biology although they may interfere with protein function.&lt;jatslist-item&gt;&lt;jatslist-item&gt;The subcellular localization of hundreds of proteins in yeast is different when tagged at the N or the C terminus.&lt;jatslist-item&gt;&lt;jatslist-item&gt;A competition based assay enables systematic deciphering of correct tagging terminus for essential proteins.&lt;jatslist-item&gt;&lt;jatslist-item&gt;The presented approach can be used to derive physiologically relevant tagged libraries.&lt;jatslist-item&gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/spatial_organization_of_rho_gtpase_signaling_by_rhogefrhogap_proteins_44087803/">
          <h4 class="font-weight-bold">Spatial Organization of Rho GTPase signaling by RhoGEFRhoGAP proteins, bioRxiv, 2018-06-24</h4>
          <p>AbstractRho GTPases control cell morphogenesis and thus fundamental processes in all eukaryotes. They are regulated by 145 RhoGEF and RhoGAP multi-domain proteins in humans. How the Rho signaling system is organized to generate localized responses in cells and prevent their spreading is not understood. Here, we systematically characterized the substrate specificities, localization and interactome of the RhoGEFsRhoGAPs and revealed their critical role in contextualizing and spatially delimiting Rho signaling. They localize to multiple compartments providing positional information, are extensively interconnected to jointly coordinate their signaling networks and are widely autoinhibited to remain sensitive to local activation. RhoGAPs exhibit lower substrate specificity than RhoGEFs and may contribute to preserving Rho activity gradients. Our approach led us to uncover a multi-RhoGEF complex downstream of G-protein-coupled receptors controlling a Cdc42RhoA crosstalk. The spatial organization of Rho signaling thus differs from other small GTPases and expands the repertoire of mechanisms governing localized signaling activity.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/automating_multimodal_microscopy_with_nanoj-fluidics_41348528/">
          <h4 class="font-weight-bold">Automating multimodal microscopy with NanoJ-Fluidics, bioRxiv, 2018-05-14</h4>
          <p>AbstractFluorescence microscopy can reveal all aspects of cellular mechanisms, from molecular details to dynamics, thanks to approaches such as super-resolution and live-cell imaging. Each of its modalities requires specific sample preparation and imaging conditions to obtain high-quality, artefact-free images, ultimately providing complementary information. Combining and multiplexing microscopy approaches is crucial to understand cellular events, but requires elaborate workflows involving multiple sample preparation steps. We present a robust fluidics approach to automate complex sequences of treatment, labelling and imaging of live and fixed cells. Our open-source NanoJ-Fluidics system is based on low-cost LEGO hardware controlled by ImageJ-based software and can be directly adapted to any microscope, providing easy-to-implement high-content, multimodal imaging with high reproducibility. We demonstrate its capacity to carry out complex sequences of experiments such as super-resolved live-to-fixed imaging to study actin dynamics; highly-multiplexed STORM and DNA-PAINT acquisitions of multiple targets; and event-driven fixation microscopy to study the role of adhesion contacts in mitosis.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/label-free_prediction_of_three-dimensional_fluorescence_images_from_transmitted_light_microscopy_34891101/">
          <h4 class="font-weight-bold">Label-free prediction of three-dimensional fluorescence images from transmitted light microscopy, bioRxiv, 2018-03-27</h4>
          <p>Understanding living cells as integrated systems, a challenge central to modern biology, is complicated by limitations of available imaging methods. While fluorescence microscopy can resolve subcellular structure in living cells, it is expensive, slow, and damaging to cells. Here, we present a label-free method for predicting 3D fluorescence directly from transmitted light images and demonstrate that it can be used to generate multi-structure, integrated images.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/cell-biology/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/cell-biology/page/7/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../../tags/cell-biology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/cell-biology/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/cell-biology/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/cell-biology/page/7/" class="page-link">7</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/cell-biology/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/cell-biology/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/cell-biology/page/10/" class="page-link">10</a></li>
            
        

        
        <li class="page-item"><a href="../../../../tags/cell-biology/page/9/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/cell-biology/page/10/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
