<!DOCTYPE html>
<html><head>
  
  <title>100 200 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_case_for_and_against_double-blind_reviews_52725309/">
          <h4 class="font-weight-bold">The Case For and Against Double-blind Reviews, bioRxiv, 2018-12-16</h4>
          <p>To date, the majority of authors on scientific publications have been men. While much of this gender bias can be explained by historic sexism and discrimination, there is concern that women may still be disadvantaged by the peer review process if reviewers&#39; unconscious biases lead them to reject publications with female authors more often. One potential solution to this perceived gender bias in the reviewing process is for journals to adopt double-blind reviews whereby neither the authors nor the reviewers are aware of each other&#39;s identities and genders. To test the efficacy of double-blind reviews, we assigned gender to every authorship of every paper published in 5 different journals with different peer review processes (double-blind vs. single blind) and subject matter (birds vs. behavioral ecology) from 2010-2018 (n = 4865 papers). While female authorships comprised only 35% of the total, the double-blind journal Behavioral Ecology did not have more female authorships than its single-blind counterparts. Interestingly, the incidence of female authorship is higher at behavioral ecology journals (Behavioral Ecology and Behavioral Ecology and Sociobiology) than in the ornithology journals (Auk, Condor, Ibis), for papers on all topics as well as those on birds. These analyses suggest that double-blind review does not currently increase the incidence of female authorship in the journals studied here. We conclude, at least for these journals, that double-blind review does not benefit female authors and may, in the long run, be detrimental.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/scientific-communication-and-education">scientific-communication-and-education</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/comprehensive_immune_monitoring_of_clinical_trials_to_advance_human_immunotherapy_52608023/">
          <h4 class="font-weight-bold">Comprehensive Immune Monitoring of Clinical Trials to Advance Human Immunotherapy, bioRxiv, 2018-12-13</h4>
          <p>SummaryThe success of immunotherapy has led to a myriad of new clinical trials. Connected to these trials are efforts to discover biomarkers providing mechanistic insight and predictive signatures for personalization. Still, the plethora of immune monitoring technologies can face investigator bias, missing unanticipated cellular responses in limited clinical material. We here present a mass cytometry workflow for standardized, systems-level biomarker discovery in immunotherapy trials. To broadly enumerate human immune cell identity and activity, we established and extensively assessed a reference panel of 33 antibodies to cover major cell subsets, simultaneously quantifying activation and immune checkpoint molecules in a single assay. The resulting assay enumerated ≥ 98% of peripheral immune cells with ≥ 4 positively identifying antigens. Robustness and reproducibility were demonstrated on multiple samples types, across research centers and by orthogonal measurements. Using automated analysis, we monitored complex immune dynamics, identifying signatures in bone-marrow transplantation associated graft-versus-host disease. This validated and available workflow ensures comprehensive immunophenotypic analysis, data comparability and will accelerate biomarker discovery in immunomodulatory therapeutics.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/immunology">immunology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_fluorescent_reporter_enables_instantaneous_measurement_of_cell_cycle_speed_in_live_cells_52553188/">
          <h4 class="font-weight-bold">A fluorescent reporter enables instantaneous measurement of cell cycle speed in live cells, bioRxiv, 2018-12-12</h4>
          <p>AbstractPeriodicity is a fundamental property of biological oscillators such as the mitotic cell cycle. In this context, periodicity refers to the time interval between the same phases of two consecutive cell cycles. The length of this interval, or the cell cycle speed, varies widely depending on cell type and the pathophysiological conditions. The relevance of cell cycle speed in various biological contexts has not been well-studied, partially due to the lack of experimental approaches that capture this parameter. Here, we describe a genetically encoded live-cell reporter of cell cycle speed. This reporter is based on the color-changing Fluorescent Timer (FT) protein, which emits blue fluorescence when newly synthesized before maturing into a red fluorescent protein. Its ability to report cell cycle speed exploits the different half-life of the blue vs. red form of the same molecule, as predicted by mathematical modeling. When a Histone H2B-FT fusion protein is expressed at steady-state in heterogeneously dividing cells, faster-cycling cells can be distinguished from slower-cycling ones by differences in their intracellular ratio between the blue and red fluorescent wavelengths. Cell cycle perturbation experiments demonstrate that the H2B-FT is a bona fide reporter of cell cycle speed in multiple cultured cell lines. In vivo, the bluered profile faithfully tracked with known proliferation kinetics of various hematopoietic stem and progenitor cells, when expressed either from lentiviral vectors or from a targeted knock-in allele. As the H2B-FT is compatible with flow cytometry, it provides a strategy to physically separate subpopulations of live cells cycling at different rates for downstream analysis. We anticipate this system to be useful in diverse cell types and tissue contexts for dissecting the role of cell cycle speed in development and disease.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genetic_determination_of_stomatal_patterning_in_winter_wheat_triticum_aestivum_l._52515038/">
          <h4 class="font-weight-bold">Genetic determination of stomatal patterning in winter wheat (Triticum aestivum L.), bioRxiv, 2018-12-11</h4>
          <p>Leaf stomata are microscopic pores mediating plant-environment interactions. Their role in carbon uptake and transpiration make them prime candidates for improving water use efficiency (WUE). Stomatal density (SD), the number of stomata per unit area, has been shown to be negatively correlated with WUE. However, little is known about the genetic basis of SD in wheat (Triticum aestivum L.), and to what extant genetic variation exists in contemporary wheat germplasm. Here, we evaluated stomatal patterning over two growing seasons in a set of 333 wheat lines, representing the European winter wheat germplasm. Stomatal patterning was mainly determined by two underlying traits, the distance between files of stomata and the distance between stomata within a file. By haplotype association mapping, quantitative trait loci for SD were consistently detected in both seasons on wheat chromosomes (CHR) 2A, 3A and 7B. The single nucleotide polymorphism markers most significantly associated with SD coincided with the genes INDUCER OF CBF EXPRESSION 1 (ICE1) and STOMATAL CYTOKINESIS-DEFECTIVE 1 (SCD1) on CHR 3A, and genes involved in ethylene and auxin signaling on CHR 2A and 7B, respectively. Our study unlocks the phenotypic and genotypic variation for stomatal patterning traits in contemporary wheat germplasm. It provides gene targets for functional validation and practical tools to manipulate SD using marker-assisted selection for crop improvement.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/real-time_capture_of_horizontal_gene_transfers_from_gut_microbiota_by_engineered_crispr-cas_acquisition_52487723/">
          <h4 class="font-weight-bold">Real-time capture of horizontal gene transfers from gut microbiota by engineered CRISPR-Cas acquisition, bioRxiv, 2018-12-11</h4>
          <p>AbstractHorizontal gene transfer (HGT) is central to the adaptation and evolution of bacteria. However, our knowledge about the flow of genetic material within complex microbiomes is lacking; most studies of HGT rely on bioinformatic analyses of genetic elements maintained on evolutionary timescales or experimental measurements of phenotypically trackable markers (e.g. antibiotic resistance). Consequently, our knowledge of the capacity and dynamics of HGT in complex communities is limited. Here, we utilize the CRISPR-Cas spacer acquisition process to detect HGT events from complex microbiota in real-time and at nucleotide resolution. In this system, a recording strain is exposed to a microbial sample, spacers are acquired from foreign transferred elements and permanently stored in genomic CRISPR arrays. Subsequently, sequencing and analysis of these spacers enables identification of the transferred elements. This approach allowed us to quantify transfer frequencies of individual mobile elements without the need for phenotypic markers or post-transfer replication. We show that HGT in human clinical fecal samples can be extensive and rapid, often involving multiple different plasmid types, with the IncX type being the most actively transferred. Importantly, the vast majority of transferred elements did not carry readily selectable phenotypic markers, highlighting the utility of our approach to reveal previously hidden real-time dynamics of mobile gene pools within complex microbiomes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/using_dna_from_mothers_and_children_to_study_parental_investment_in_childrens_educational_attainment_52436750/">
          <h4 class="font-weight-bold">Using DNA from mothers and children to study parental investment in children&#39;s educational attainment, bioRxiv, 2018-12-10</h4>
          <p>This study tested implications of new genetic discoveries for understanding the association between parental investment and children&#39;s educational attainment. A novel design matched genetic data from 860 British mothers and their children with home-visit measures of parenting the E-Risk Study. Three findings emerged. First, both mothers&#39; and children&#39;s education-associated genetics, summarized in a genome-wide polygenic score, predicted parenting -- a gene-environment correlation. Second, accounting for genetic influences slightly reduced associations between parenting and children&#39;s attainment -- indicating some genetic confounding. Third, mothers&#39; genetics influenced children&#39;s attainment over and above genetic mother-to-child transmission, via cognitively-stimulating parenting -- an environmentally-mediated effect. Findings imply that, when interpreting parents&#39; effects on children, environmentalists must consider genetic transmission, but geneticists must also consider environmental transmission.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/100-200-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/100-200-users/page/36/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/33/" class="page-link">33</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/34/" class="page-link">34</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/35/" class="page-link">35</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/36/" class="page-link">36</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/100-200-users/page/37/" class="page-link">37</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/38/" class="page-link">38</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/39/" class="page-link">39</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/40/" class="page-link">40</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/41/" class="page-link">41</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/100-200-users/page/38/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/100-200-users/page/68/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
