<!DOCTYPE html>
<html><head>
  
  <title>Genetics | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/evidence_of_large_genetic_influences_on_dog_ownership_in_the_swedish_twin_registry_has_implications_for_understanding_domestication_and_health_associations_60582021/">
          <h4 class="font-weight-bold">Evidence of large genetic influences on dog ownership in the Swedish Twin Registry has implications for understanding domestication and health associations, Scientific Reports, 2019-05-17</h4>
          <p>Dogs were the first domesticated animal and, according to the archaeological evidence, have had a close relationship with humans for at least 15,000 years. Today, dogs are common pets in our society and have been linked to increased well-being and improved health outcomes in their owners. A dog in the family during childhood is associated with ownership in adult life. The underlying factors behind this association could be related to experiences or to genetic influences. We aimed to investigate the heritability of dog ownership in a large twin sample including all twins in the Swedish Twin Registry born between 1926 and 1996 and alive in 2006. Information about dog ownership was available from 2001 to 2016 from national dog registers. The final data set included 85,542 twins from 50,507 twin pairs with known zygosity, where information on both twins were available in 35,035 pairs. Structural equation modeling was performed to estimate additive genetic effects (the heritability), commonshared environmental, and uniquenon-shared environmental effects. We found that additive genetic factors largely contributed to dog ownership, with heritability estimated at 57% for females and 51% for males. An effect of shared environmental factors was only observed in early adulthood. In conclusion, we show a strong genetic contribution to dog ownership in adulthood in a large twin study. We see two main implications of this finding (1) genetic variation may have contributed to our ability to domesticate dogs and other animals and (2) potential pleiotropic effects of genetic variation affecting dog ownership should be considered in studies examining health impacts of dog ownership.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/scientific%20reports">scientific reports</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/gene_knock-ins_in_drosophila_using_homology-independent_insertion_of_universal_donor_plasmids_60546926/">
          <h4 class="font-weight-bold">Gene knock-ins in Drosophila using homology-independent insertion of universal donor plasmids, bioRxiv, 2019-05-16</h4>
          <p>AbstractSite-specific insertion of DNA into endogenous genes (knock-in) is a powerful method to study gene function. However, traditional methods for knock-in require laborious cloning of long homology arms for homology-directed repair. Here, we report a simplified method in Drosophila melanogaster to insert large DNA elements into any gene using homology-independent repair. This method, known as CRISPaint, employs CRISPR-Cas9 and non-homologous end joining (NHEJ) to linearize and insert donor plasmid DNA into a target genomic cut site. The inclusion of commonly used elements such as GFP on donor plasmids makes them universal, abolishing the need to create gene-specific homology arms and greatly reducing user workload. Using this method, we show robust gene-specific integration of donor plasmids in cultured cells and the fly germ line. Furthermore, we use this method to analyze gene function by fluorescently tagging endogenous proteins, disrupting gene function, and generating reporters of gene expression. Finally, we assemble a collection of donor plasmids for germ line knock-in that contain commonly used insert sequences. This method simplifies the generation of site-specific large DNA insertions in Drosophila cell lines and fly strains, and better enables researchers to dissect gene function in vivo.SummaryWe report a new homology-independent genomic knock-in method in Drosophila to insert large DNA elements into any target gene. Using CRISPR-Cas9 and non-homologous end joining (NHEJ), an entire donor plasmid is inserted into the genome without the need for homology arms. This approach eliminates the burden associated with designing and constructing traditional donor plasmids. We demonstrate its usefulness in cultured cells and in vivo to fluorescently tag endogenous proteins, generate reporters of gene expression, and disrupt gene function.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_large-scale_resource_for_tissue-specific_crispr_mutagenesis_in_drosophila_60399962/">
          <h4 class="font-weight-bold">A large-scale resource for tissue-specific CRISPR mutagenesis in Drosophila, bioRxiv, 2019-05-13</h4>
          <p>SUMMARYGenetic screens are powerful tools for the functional annotation of genomes. In the context of multicellular organisms, interrogation of gene function is greatly facilitated by methods that allow spatial and temporal control of gene abrogation. Here, we describe a large-scale transgenic short guide (sg) RNA library for efficient CRISPR-based disruption of specific target genes in a constitutive or conditional manner. The library consists currently of more than 2600 plasmids and 1400 fly lines with a focus on targeting kinases, phosphatases and transcription factors, each expressing two sgRNAs under control of the Gal4UAS system. We show that conditional CRISPR mutagenesis is robust across many target genes and can be efficiently employed in various somatic tissues, as well as the germline. In order to prevent artefacts commonly associated with excessive amounts of Cas9 protein, we have developed a series of novel UAS-Cas9 transgenes, which allow fine tuning of Cas9 expression to achieve high gene editing activity without detectable toxicity. Functional assays, as well as direct sequencing of genomic sgRNA target sites, indicates that the vast majority of transgenic sgRNA lines mediate efficient gene disruption. Furthermore, we conducted the so far largest fully transgenic CRISPR screen in any metazoan organism, which further supported the high efficiency and accuracy of our library and revealed many so far uncharacterized genes essential for development.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_tomato_pan-genome_uncovers_new_genes_and_a_rare_allele_regulating_fruit_flavor_60397352/">
          <h4 class="font-weight-bold">The tomato pan-genome uncovers new genes and a rare allele regulating fruit flavor, Nature Genetics, 2019-05-13</h4>
          <p>TomLoxC</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/nature%20genetics">nature genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/variable_prediction_accuracy_of_polygenic_scores_within_an_ancestry_group_60037925/">
          <h4 class="font-weight-bold">Variable prediction accuracy of polygenic scores within an ancestry group, bioRxiv, 2019-05-07</h4>
          <p>AbstractFields as diverse as human genetics and sociology are increasingly using polygenic scores based on genome-wide association studies (GWAS) for phenotypic prediction. However, recent work has shown that polygenic scores have limited portability across groups of different genetic ancestries, restricting the contexts in which they can be used reliably and potentially creating serious inequities in future clinical applications. Using the UK Biobank data, we demonstrate that even within a single ancestry group, the prediction accuracy of polygenic scores depends on characteristics such as the age or sex composition of the individuals in which the GWAS and the prediction were conducted, and on the GWAS study design. Our findings highlight both the complexities of interpreting polygenic scores and underappreciated obstacles to their broad use.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/why_are_education_socioeconomic_position_and_intelligence_genetically_correlated_60037668/">
          <h4 class="font-weight-bold">Why are education, socioeconomic position and intelligence genetically correlated?, bioRxiv, 2019-05-07</h4>
          <p>AbstractGenetic associations and correlations are perceived as confirmation that genotype influences one or more phenotypes respectively. However, genetic correlations can arise from non-biological or indirect mechanisms including population stratification, dynastic effects, and assortative mating. In this paper, we outline these mechanisms and demonstrate available tools and analytic methods that can be used to assess their presence in estimates of genetic correlations and genetic associations. Using educational attainment and parental socioeconomic position data as an exemplar, we demonstrate that both heritability and genetic correlation estimates may be inflated by these indirect mechanisms. The results highlight the limitations of between-individual estimates obtained from samples of unrelated individuals and the potential value of family-based studies. Use of the highlighted tools in combination with within-sibling or mother-father-offspring trio data may offer researchers greater opportunity to explore the underlying mechanisms behind genetic associations and correlations and identify the underlying causes of estimate inflation.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/genetics/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/genetics/page/6/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/genetics/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/genetics/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/genetics/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/genetics/page/6/" class="page-link">6</a></li>
            
        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/genetics/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/genetics/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/genetics/page/9/" class="page-link">9</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/genetics/page/10/" class="page-link">10</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/genetics/page/11/" class="page-link">11</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/genetics/page/8/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/genetics/page/36/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
