<!DOCTYPE html>
<html><head>
  
  <title>Reports | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/plotsofdata__a_web_app_for_visualizing_data_together_with_its_summaries_48929262/">
          <h4 class="font-weight-bold">PlotsOfData – a web app for visualizing data together with its summaries, bioRxiv, 2018-09-27</h4>
          <p>AbstractReporting of the actual data in graphs and plots increases transparency and enables independent evaluation. On the other hand, data summaries are often used in graphs since they aid interpretation. State-of-the art data visualizations can be made with the ggplot2 package, which uses the ideas of a ‘grammar of graphics’ to generate a graphic from multiple layers of data. However, ggplot2 requires coding skills and an understanding of the tidy data structure. To democratize state-of-the-art data visualization of raw data with a selection of statistical summaries, a web app was written using Rshiny that uses the ggplot2 package for generating plots. A multilayered approach together with adjustable transparency offers a unique flexibility, enabling users can to choose how to display the data and which of the data summaries to add. Four data summaries are provided, mean, median, boxplot, violinplot, to accommodate several types of data distributions. In addition, 95% confidence intervals can be added for visual inferences. By adjusting the transparency of the layers, the visualization of the raw data together with the summary can be tuned for optimal presentation and interpretation. The app is dubbed PlotsOfData and is available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpshuygens.science.uva.nlPlotsOfData&gt;httpshuygens.science.uva.nlPlotsOfData&lt;jatsext-link&gt;&lt;jatsfig id=ufig1 position=float orientation=portrait fig-type=figure&gt;&lt;jatsgraphic xmlnsxlink=httpwww.w3.org1999xlink xlinkhref=426767v3_ufig1 position=float orientation=portrait &gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/scientific-communication-and-education">scientific-communication-and-education</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/towards_inferring_causal_gene_regulatory_networks_from_single_cell_expression_measurements_48894487/">
          <h4 class="font-weight-bold">Towards inferring causal gene regulatory networks from single cell expression Measurements, bioRxiv, 2018-09-27</h4>
          <p>AbstractSingle-cell transcriptome sequencing now routinely samples thousands of cells, potentially providing enough data to reconstruct causal gene regulatory networks from observational data. Here, we present Scribe, a toolkit for detecting and visualizing causal regulatory interactions between genes and explore the potential for single-cell experiments to power network reconstruction. Scribe employs Restricted Directed Information to determine causality by estimating the strength of information transferred from a potential regulator to its downstream target. We apply Scribe and other leading approaches for causal network reconstruction to several types of single-cell measurements and show that there is a dramatic drop in performance for pseudotime” ordered single-cell data compared to true time series data. We demonstrate that performing causal inference requires temporal coupling between measurements. We show that methods such as “RNA velocity” restore some degree of coupling through an analysis of chromaffin cell fate commitment. These analyses therefore highlight an important shortcoming in experimental and computational methods for analyzing gene regulation at single-cell resolution and point the way towards overcoming it.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/genomics">genomics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/efficient_generation_of_endogenous_fluorescent_reporters_by_nested_crispr_in_caenorhabditis_elegans_48920754/">
          <h4 class="font-weight-bold">Efficient generation of endogenous fluorescent reporters by Nested CRISPR in Caenorhabditis elegans, bioRxiv, 2018-09-26</h4>
          <p>AbstractCRISPR-based genome editing methods in model organisms are evolving at an extraordinary speed. Whereas the generation of deletion or missense mutants is quite straightforward, the production of endogenous fluorescent reporters is still inefficient. The use of plasmids with selection markers is an effective methodology, but often requires laborious and complicated cloning steps. We have established a cloning-free ribonucleoprotein-driven Nested CRISPR method that robustly produces endogenous fluorescent reporters. This methodology is based on the division of the GFP and mCherry sequences in three fragments. In the first step we use ssDNA donors (≤200 bp) to insert 5’ and 3’ fragments in the place of interest. In the second step, we use these sequences as homology regions for Homology Directed Repair (HDR) with a dsDNA donor (PCR product, ≈700 bp) including the middle fragment, thus completing the fluorescent protein sequence. This method is advantageous because the first step with ssDNA donors is known to be very efficient, and the second step, uses universal reagents, including validated PCR products and crRNAs, to create fluorescent reporters reaching reliable editing efficiencies as high as 40%. We have also used Nested CRISPR in a non-essential gene to produce a deletion mutant in the first step and a transcriptional reporter in the second step.In the search of modifications to optimize the method, we tested synthetic sgRNAs, but we did not observe a significant increase in the efficacy compared to independently adding tracrRNA and crRNA to the injection mix. Conveniently, we also found that both steps of Nested CRISPR could be performed in a single injection. Finally, we discuss the utility of Nested CRISPR for targeted insertion of long DNA fragments in other systems and prospects of this method in the future.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/exploiting_selection_at_linked_sites_to_infer_the_rate_and_strength_of_adaptation_48908263/">
          <h4 class="font-weight-bold">Exploiting selection at linked sites to infer the rate and strength of adaptation, bioRxiv, 2018-09-26</h4>
          <p>Genomic data encodes past evolutionary events and has the potential to reveal the strength, rate, and biological drivers of adaptation. However, robust estimation of adaptation rate (alpha) and adaptation strength remains a challenging problem because evolutionary processes such as demography, linkage, and non-neutral polymorphism can confound inference. Here, we exploit the influence of background selection to reduce the fixation rate of weakly-beneficial alleles to jointly infer the strength and rate of adaptation. We develop a novel MK-based method (ABC-MK) to infer adaptation rate and strength, and estimate alpha=0.135 in human protein-coding sequences, 72% of which is contributed by weakly adaptive variants. We show that in this adaptation regime alpha is reduced approx. 25% by linkage genome-wide. Moreover, we show that virus-interacting proteins (VIPs) undergo adaptation that is both stronger and nearly twice as frequent as the genome average (alpha=0.224, 56% due to strongly-beneficial alleles). Our results suggest that while most adaptation in human proteins is weakly-beneficial, adaptation to viruses is often strongly-beneficial. Our method provides a robust framework for estimating adaptation rate and strength across species.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/slow_transcriptional_elongation_causes_embryonic_lethality_and_perturbs_kinetic_coupling_of_long_neural_genes_48912400/">
          <h4 class="font-weight-bold">Slow transcriptional elongation causes embryonic lethality and perturbs kinetic coupling of long neural genes, bioRxiv, 2018-09-26</h4>
          <p>The rate of RNA Polymerase II (RNAPII) elongation has an important role in the control of Alternative splicing (AS); however, the in vivo consequences of an altered elongation rate are unknown. Here, we generated mouse embryonic stem cells (ESCs) knocked-in for a slow elongating form of RNAPII. We show that a reduced transcriptional elongation rate results in early embryonic lethality in mice and impairs the differentiation of ESCs into the neural lineage. This is accompanied by changes in splicing and in gene expression in ESCs and along the pathway of neuronal differentiation. In particular, we found a crucial role for RNAPII elongation rate in transcription and splicing of long neuronal genes involved in synapse signaling. The impact of the kinetic coupling of RNAPII elongation rate with AS is more predominant in ESC-differentiated neurons than in pluripotent cells. Our results demonstrate the requirement for an appropriate transcriptional elongation rate to ensure proper gene expression and to regulate AS during development.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/molecular-biology">molecular-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/stronger_and_higher_proportion_of_beneficial_amino_acid_changing_mutations_in_humans_compared_to_mice_and_flies_48909889/">
          <h4 class="font-weight-bold">Stronger and higher proportion of beneficial amino acid changing mutations in humans compared to mice and flies, bioRxiv, 2018-09-26</h4>
          <p>ABSTRACTQuantifying and comparing the amount of adaptive evolution among different species is key to understanding evolutionary processes. Previous studies have shown differences in adaptive evolution across species, however their specific causes remain elusive. Here, we use improved modeling of weakly deleterious mutations and the demographic history of the outgroup species and estimate that 30–34% of nonsynonymous substitutions between humans and outgroup species have been fixed by positive selection. This estimate is much higher than previous estimates, which did not account for the population size of the outgroup species. Next, we directly estimate the proportion and selection coefficients of newly arising strongly beneficial nonsynonymous mutations in humans, mice, and D. melanogaster by examining patterns of polymorphism and divergence. We develop a novel composite likelihood framework to test whether these parameters differ across species. Overall, we reject a model with the same proportion and the same selection coefficients of beneficial mutations across species, and estimate that humans have a higher proportion of beneficial mutations compared to Drosophila and mice. We demonstrate that this result cannot be attributed to biased gene conversion. In summary, we find the proportion of beneficial mutations is higher in humans than in D. melanogaster or mice, suggesting that organismal complexity, which increases the number of steps required in adaptive walks, may be a key predictor of the amount of adaptive evolution within a species.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/evolutionary-biology">evolutionary-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../tags/2018">2018</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../reports/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../reports/page/186/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/183/" class="page-link">183</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/184/" class="page-link">184</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/185/" class="page-link">185</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/186/" class="page-link">186</a></li>
            
        
             
            <li class="page-item active"><a href="../../../reports/page/187/" class="page-link">187</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/188/" class="page-link">188</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/189/" class="page-link">189</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/190/" class="page-link">190</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../reports/page/191/" class="page-link">191</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../reports/page/188/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../reports/page/320/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
