<!DOCTYPE html>
<html><head>
  
  <title>Systems Biology | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/high-throughput_single-cell_proteomics_quantifies_the_emergence_of_macrophage_heterogeneity_61883494/">
          <h4 class="font-weight-bold">High-throughput single-cell proteomics quantifies the emergence of macrophage heterogeneity, bioRxiv, 2019-06-10</h4>
          <p>AbstractThe fate and physiology of individual cells are controlled by networks of proteins. Yet, our ability to quantitatively analyze protein networks in single cells has remained limited. To overcome this barrier, we developed SCoPE2. It integrates concepts from Single-Cell ProtEomics by Mass Spectrometry (SCoPE-MS) with automated and miniaturized sample preparation, substantially lowering cost and hands-on time. SCoPE2 uses data-driven analytics to optimize instrument parameters for sampling more ion copies per protein, thus supporting quantification with improved count statistics. These advances enabled us to analyze the emergence of cellular heterogeneity as homogeneous monocytes differentiated into macrophage-like cells in the absence of polarizing cytokines. We used SCoPE2 to quantify over 2,000 proteins in 356 single monocytes and macrophages in about 85 hours of instrument time, and the quantified proteins allowed us to discern single cells by cell type. Furthermore, the data uncovered a continuous gradient of proteome states for the macrophage-like cells, suggesting that macrophage heterogeneity may emerge even in the absence of polarizing cytokines. Our methodology lays the foundation for quantitative analysis of protein networks at single-cell resolution.&lt;jatsfig id=ufig1 position=float fig-type=figure orientation=portrait&gt;&lt;jatsgraphic xmlnsxlink=httpwww.w3.org1999xlink xlinkhref=665307v2_ufig1 position=float orientation=portrait &gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/chromatinization_of_escherichia_coli_with_archaeal_histones_61507230/">
          <h4 class="font-weight-bold">Chromatinization of Escherichia coli with archaeal histones, bioRxiv, 2019-06-04</h4>
          <p>ABSTRACTNucleosomes restrict DNA accessibility throughout eukaryotic genomes, with repercussions for replication, transcription, and other DNA-templated processes. How this globally restrictive organization emerged from a presumably more open ancestral state remains poorly understood. Here, to better understand the challenges associated with establishing globally restrictive chromatin, we express histones in a naïve bacterial system that has not evolved to deal with nucleosomal structures Escherichia coli. We find that histone proteins from the archaeon Methanothermus fervidus assemble on the E. coli chromosome in vivo and protect DNA from micrococcal nuclease digestion, allowing us to map binding footprints genome-wide. We provide evidence that nucleosome occupancy along the E. coli genome tracks intrinsic sequence preferences but is disturbed by ongoing transcription and replication. Notably, we show that higher nucleosome occupancy at promoters and across gene bodies is associated with lower transcript levels, consistent with local repressive effects. Surprisingly, however, this sudden enforced chromatinization has only mild repercussions for growth, suggesting that histones can become established as ubiquitous chromatin proteins without interfering critically with key DNA-templated processes. Our results have implications for the evolvability of transcriptional ground states and highlight chromatinization by archaeal histones as a potential avenue for controlling genome accessibility in synthetic prokaryotic systems.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/a_reference_map_of_the_human_protein_interactome_58852789/">
          <h4 class="font-weight-bold">A reference map of the human protein interactome, bioRxiv, 2019-04-11</h4>
          <p>AbstractGlobal insights into cellular organization and function require comprehensive understanding of interactome networks. Similar to how a reference genome sequence revolutionized human genetics, a reference map of the human interactome network is critical to fully understand genotype-phenotype relationships. Here we present the first human “all-by-all” binary reference interactome map, or “HuRI”. With ~53,000 high-quality protein-protein interactions (PPIs), HuRI is approximately four times larger than the information curated from small-scale studies available in the literature. Integrating HuRI with genome, transcriptome and proteome data enables the study of cellular function within essentially any physiological or pathological cellular context. We demonstrate the use of HuRI in identifying specific subcellular roles of PPIs and protein function modulation via splicing during brain development. Inferred tissue-specific networks reveal general principles for the formation of cellular context-specific functions and elucidate potential molecular mechanisms underlying tissue-specific phenotypes of Mendelian diseases. HuRI thus represents an unprecedented, systematic reference linking genomic variation to phenotypic outcomes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/metabolic_activity_affects_response_of_single_cells_to_a_nutrient_switch_in_structured_populations_57196911/">
          <h4 class="font-weight-bold">Metabolic activity affects response of single cells to a nutrient switch in structured populations, bioRxiv, 2019-03-18</h4>
          <p>AbstractMicrobes live in ever-changing environments where they need to adapt their metabolism to different nutrient conditions. Many studies have characterized the response of genetically identical cells to nutrient switches in homogenous cultures, however in nature microbes often live in spatially structured groups such as biofilms where cells can create metabolic gradients by consuming and releasing nutrients. Consequently, cells experience different local microenvironments and vary in their phenotype. How does this phenotypic variation affect the ability of cells to cope with nutrient switches? Here we address this question by growing dense populations of Escherichia coli in microfluidic chambers and studying a switch from glucose to acetate at the single cell level. Before the switch, cells vary in their metabolic activity some grow on glucose while others cross-feed on acetate. After the switch, only few cells can resume growth after a period of lag. The probability to resume growth depends on a cells’ phenotype prior to the switch it is highest for cells crossfeeding on acetate, while it depends in a non-monotonic way on growth rate for cells growing on glucose. Our results suggest that the strong phenotypic variation in spatially structured populations might enhance their ability to cope with fluctuating environments.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/ion_beam_subcellular_tomography_55924043/">
          <h4 class="font-weight-bold">Ion beam subcellular tomography, bioRxiv, 2019-02-23</h4>
          <p>Multiplexed ion beam imaging (MIBI) has been previously used to profile multiple parameters in two dimensions in single cells within tissue slices. Here, a mathematical and technical framework for three-dimensional subcellular MIBI is presented. We term the approach ion beam tomography (IBT) wherein ion beam images are acquired iteratively across successive, multiple scans and later compiled into a 3D format. For IBT, cells were imaged at 0.2-4 pA ion current across 1,000 axial scans. Consecutive subsets of ion beam images were binned over 3 to 20 slices (above and below) to create a resolved image, wherein binning was incremented one slice at a time to yield an enhanced multi-depth data without loss of depth resolution. Algorithmic deconvolution, tailored for ion beams, was then applied to the transformed ion image series using a hybrid deblurring algorithm and an ion beam current-dependent point-spread function. Three-dimensional processing was implemented by segmentation, mesh, molecular neighborhoods, and association maps. In cultured cancer cells and tissues, IBT enabled accessible visualization of three-dimensional volumetric distributions of genomic regions, RNA transcripts, and protein factors with 65-nm lateral and 5-nm axial resolution. IBT also enabled label-free elemental mapping of cells, allowing point of source cellular component measurements not possible for most optical microscopy targets. Detailed multiparameter imaging of subcellular features at near macromolecular resolution should now be made possible by the IBT tools and reagents provided here to open novel venues for interrogating subcellular biology.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/subcellular_localization_of_drug_distribution_by_super-resolution_ion_beam_imaging_55904025/">
          <h4 class="font-weight-bold">Subcellular localization of drug distribution by super-resolution ion beam imaging, bioRxiv, 2019-02-22</h4>
          <p>Technologies that visualize multiple biomolecules at the nanometer scale in cells will enable deeper understanding of biological processes that proceed at the molecular scale. Current fluorescence-based methods for microscopy are constrained by a combination of spatial resolution limitations, limited parameters per experiment, and detector systems for the wide variety of biomolecules found in cells. We present here super-resolution ion beam imaging (srIBI), a secondary ion mass spectrometry approach capable of high-parameter imaging in 3D of targeted biological entities and exogenously added small molecules. Uniquely, the atomic constituents of the biomolecules themselves can often be used in our system as the tag. We visualized the subcellular localization of the chemotherapy drug cisplatin simultaneously with localization of five other nuclear structures, with further carbon elemental mapping and secondary electron visualization, down to ~30 nm lateral resolution. Cisplatin was preferentially enriched in nuclear speckles and excluded from closed-chromatin regions, indicative of a role for cisplatin in active regions of chromatin. These data highlight how multiplexed super-resolution techniques, such as srIBI, will enable studies of biomolecule distributions in biologically relevant subcellular microenvironments.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        

        
             
            <li class="page-item active"><a href="/tags/systems-biology/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/systems-biology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/systems-biology/page/3/" class="page-link">3</a></li>
            
        

        
        <li class="page-item"><a href="/tags/systems-biology/page/2/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/systems-biology/page/3/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
