<!DOCTYPE html>
<html><head>
  
  <title>Reports | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/reproducible_bioinformatics_project_a_community_for_reproducible_bioinformatics_analysis_pipelines_30938817/">
          <h4 class="font-weight-bold">Reproducible Bioinformatics Project A community for reproducible bioinformatics analysis pipelines, bioRxiv, 2017-12-27</h4>
          <p>AbstractBackgroundReproducibility of a research is a key element in the modern science and it is mandatory for any industrial application. It represents the ability of replicating an experiment independently by the location and the operator. Therefore, a study can be considered reproducible only if all used data are available and the exploited computational analysis workflow is clearly described. However, today for reproducing a complex bioinformatics analysis, the raw data and a list of tools used in the workflow could be not enough to guarantee the reproducibility of the results obtained. Indeed, different releases of the same tools andor of the system libraries (exploited by such tools) might lead to sneaky reproducibility issues.ResultsTo address this challenge, we established the Reproducible Bioinformatics Project (RBP), which is a non-profit and open-source project, whose aim is to provide a schema and an infrastructure, based on docker images and R package, to provide reproducible results in Bioinformatics. One or more Docker images are then defined for a workflow (typically one for each task), while the workflow implementation is handled via R-functions embedded in a package available at github repository. Thus, a bioinformatician participating to the project has firstly to integrate herhis workflow modules into Docker image(s) exploiting an Ubuntu docker image developed ad hoc by RPB to make easier this task. Secondly, the workflow implementation must be realized in R according to an R-skeleton function made available by RPB to guarantee homogeneity and reusability among different RPB functions. Moreover shehe has to provide the R vignette explaining the package functionality together with an example dataset which can be used to improve the user confidence in the workflow utilization.ConclusionsReproducible Bioinformatics Project provides a general schema and an infrastructure to distribute robust and reproducible workflows. Thus, it guarantees to final users the ability to repeat consistently any analysis independently by the used UNIX-like architecture.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/equivalent_high-resolution_identification_of_neuronal_cell_types_with_single-nucleus_and_single-cell_rna-sequencing_30918316/">
          <h4 class="font-weight-bold">Equivalent high-resolution identification of neuronal cell types with single-nucleus and single-cell RNA-sequencing, bioRxiv, 2017-12-26</h4>
          <p>Transcriptional profiling of complex tissues by RNA-sequencing of single nuclei presents some advantages over whole cell analysis. It enables unbiased cellular coverage, lack of cell isolation-based transcriptional effects, and application to archived frozen specimens. Using a well-matched pair of single-nucleus RNA-seq (snRNA-seq) and single-cell RNA-seq (scRNA-seq) SMART-Seq v4 datasets from mouse visual cortex, we demonstrate that similarly high-resolution clustering of closely related neuronal types can be achieved with both methods if intronic sequences are included in nuclear RNA-seq analysis. More transcripts are detected in individual whole cells (~11,000 genes) than nuclei (~7,000 genes), but the majority of genes have similar detection across cells and nuclei. We estimate that the nuclear proportion of total cellular mRNA varies from 20% to over 50% for large and small pyramidal neurons, respectively. Together, these results illustrate the high information content of nuclear RNA for characterization of cellular diversity in brain tissues.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/rethinking_dopamine_as_generalized_prediction_error_30920703/">
          <h4 class="font-weight-bold">Rethinking dopamine as generalized prediction error, bioRxiv, 2017-12-26</h4>
          <p>AbstractMidbrain dopamine neurons are commonly thought to report a reward prediction error, as hypothesized by reinforcement learning theory. While this theory has been highly successful, several lines of evidence suggest that dopamine activity also encodes sensory prediction errors unrelated to reward. Here we develop a new theory of dopamine function that embraces a broader conceptualization of prediction errors. By signaling errors in both sensory and reward predictions, dopamine supports a form of reinforcement learning that lies between model-based and model-free algorithms. This account remains consistent with current canon regarding the correspondence between dopamine transients and reward prediction errors, while also accounting for new data suggesting a role for these signals in phenomena such as sensory preconditioning and identity unblocking, which ostensibly draw upon knowledge beyond reward predictions.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/the_null_additivity_of_multi-drug_combinations_30901769/">
          <h4 class="font-weight-bold">The null additivity of multi-drug combinations, bioRxiv, 2017-12-26</h4>
          <p>AbstractFrom natural ecology 1–4 to clinical therapy 5–8, cells are often exposed to mixtures of multiple drugs. Two competing null models are used to predict the combined effect of drugs response additivity (Bliss) and dosage additivity (Loewe) 9–11. Here, noting that these models diverge with increased number of drugs, we contrast their predictions with measurements of Escherichia coli growth under combinations of up to 10 different antibiotics. As the number of drugs increases, Bliss maintains accuracy while Loewe systematically loses its predictive power. The total dosage required for growth inhibition, which Loewe predicts should be fixed, steadily increases with the number of drugs, following a square root scaling. This scaling is explained by an approximation to Bliss where, inspired by RA Fisher’s classical geometric model 12, dosages of independent drugs adds up as orthogonal vectors rather than linearly. This dose-orthogonality approximation provides results similar to Bliss, yet uses the dosage language as Loewe and is hence easier to implement and intuit. The rejection of dosage additivity in favor of effect additivity and dosage orthogonality provides a framework for understanding how multiple drugs and stressors add up in nature and the clinic.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/contact-dependent_cell-cell_communications_drive_morphological_invariance_during_ascidian_embryogenesis_30875487/">
          <h4 class="font-weight-bold">Contact-dependent cell-cell communications drive morphological invariance during ascidian embryogenesis, bioRxiv, 2017-12-25</h4>
          <p>ABSTRACTCanalization of developmental processes ensures the reproducibility and robustness of embryogenesis within each species. In its extreme form, found in ascidians, early embryonic cell lineages are invariant between embryos within and between species, despite rapid genomic divergence. To resolve this paradox, we used live light-sheet imaging to quantify individual cell behaviors in digitalized embryos and explore the forces that canalize their development. This quantitative approach revealed that individual cell geometries and cell contacts are strongly constrained, and that these constraints are tightly linked to the control of fate specification by local cell inductions. While in vertebrates ligand concentration usually controls cell inductions, we found that this role is fulfilled in ascidians by the area of contacts between signaling and responding cells. We propose that the duality between geometric and genetic control of inductions contributes to the counterintuitive inverse correlation between geometric and genetic variability during embryogenesis.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/developmental-biology">developmental-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/deepmhc_deep_convolutional_neural_networks_for_high-performance_peptide-mhc_binding_affinity_prediction_30875476/">
          <h4 class="font-weight-bold">DeepMHC Deep Convolutional Neural Networks for High-performance peptide-MHC Binding Affinity Prediction, bioRxiv, 2017-12-25</h4>
          <p>AbstractConvolutional neural networks (CNN) have been shown to outperform conventional methods in DNA-protien binding specificity prediction. However, whether we can transfer this success to protien-peptide binding affinity prediction depends on appropriate design of the CNN architectue that calls for thorough understanding how to match the architecture to the problem. Here we propose DeepMHC, a deep convolutional neural network (CNN) based protein-peptide binding prediction algorithm for achieving better performance in MHC-I peptide binding affinity prediction than conventional algorithms. Our model takes only raw binding peptide sequences as input without needing any human-designed features and othe physichochemical or evolutionary information of the amino acids. Our CNN models are shown to be able to learn non-linear relationships among the amino acid positions of the peptides to achieve highly competitive performance on most of the IEDB benchmark datasets with a single model architecture and without using any consensus or composite ensemble classifier models. By systematically exploring the best CNN architecture, we identified critical design considerations in CNN architecture development for peptide-MHC binding prediction.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2017">2017</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/reports/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/reports/page/141/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/reports/page/138/" class="page-link">138</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/139/" class="page-link">139</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/140/" class="page-link">140</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/141/" class="page-link">141</a></li>
            
        
             
            <li class="page-item active"><a href="/reports/page/142/" class="page-link">142</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/143/" class="page-link">143</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/144/" class="page-link">144</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/145/" class="page-link">145</a></li>
            
        
            
             
            <li class="page-item"><a href="/reports/page/146/" class="page-link">146</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/reports/page/143/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/reports/page/172/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
