<!DOCTYPE html>
<html><head>
  
  <title>Microbiology | audiences</title>
  
  <link rel="stylesheet" href='css/bootstrap.min.css'>
  <link rel="stylesheet" href='css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/intestinal_delta-6-desaturase_activity_determines_host_range_for_toxoplasma_sexual_reproduction_62936604/">
          <h4 class="font-weight-bold">Intestinal delta-6-desaturase activity determines host range for Toxoplasma sexual reproduction, bioRxiv, 2019-07-01</h4>
          <p>AbstractMany eukaryotic microbes have complex lifecycles that include both sexual and asexual phases with strict species-specificity. While the asexual cycle of the protistan parasite Toxoplasma gondii can occur in any warm-blooded mammal, the sexual cycle is restricted to the feline intestine1. The molecular determinants that identify cats as the definitive host for T. gondii are unknown. Here, we defined the mechanism of species specificity for T. gondii sexual development and break the species barrier to allow the sexual cycle to occur in mice. We determined that T. gondii sexual development occurs when cultured feline intestinal epithelial cells are supplemented with linoleic acid. Felines are the only mammals that lack delta-6-desaturase activity in their intestines, which is required for linoleic acid metabolism, resulting in systemic excess of linoleic acid2, 3. We found that inhibition of murine delta-6-desaturase and supplementation of their diet with linoleic acid allowed T. gondii sexual development in mice. This mechanism of species specificity is the first defined for a parasite sexual cycle. This work highlights how host diet and metabolism shape coevolution with microbes. The key to unlocking the species boundaries for other eukaryotic microbes may also rely on the lipid composition of their environments as we see increasing evidence for the importance of host lipid metabolism during parasitic lifecycles4, 5. Pregnant women are advised against handling cat litter as maternal infection with T. gondii can be transmitted to the fetus with potentially lethal outcomes. Knowing the molecular components that create a conducive environment for T. gondii sexual reproduction will allow for development of therapeutics that prevent shedding of T. gondii parasites. Finally, given the current reliance on companion animals to study T. gondii sexual development, this work will allow the T. gondii field to use of alternative models in future studies.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/highly_multiplexed_spatial_mapping_of_microbial_communities_62519865/">
          <h4 class="font-weight-bold">Highly Multiplexed Spatial Mapping of Microbial Communities, bioRxiv, 2019-06-22</h4>
          <p>ABSTRACTMapping the complex biogeography of microbial communities in situ with high taxonomic and spatial resolution poses a major challenge because of the high density and rich diversity of species in environmental microbiomes and the limitations of optical imaging technology. Here, we introduce High Phylogenetic Resolution microbiome mapping by Fluorescence In-Situ Hybridization (HiPR-FISH), a versatile and cost-effective technology that uses binary encoding and spectral imaging and machine learning based decoding to create micron-scale maps of the locations and identities of hundreds of microbial species in complex communities. We demonstrate the ability of 10-bit HiPR-FISH to distinguish 1023 E. coli strains, each fluorescently labeled with a unique binary barcode. HiPR-FISH, in conjunction with custom algorithms for automated probe design and segmentation of single-cells in the native context of tissues, reveals the intricate spatial architectures formed by bacteria in the human oral plaque microbiome and disruption of spatial networks in the mouse gut microbiome in response to antibiotic treatment. HiPR-FISH provides a framework for analyzing the spatial organization of microbial communities in tissues and the environment at single cell resolution.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/on_the_emergence_of_candida_auris_climate_change_azoles_swamps_and_birds_61471872/">
          <h4 class="font-weight-bold">On the emergence of Candida auris climate change, azoles, swamps and birds, bioRxiv, 2019-06-04</h4>
          <p>The most enigmatic aspect of the rise of Candida auris as a human pathogen is that it emerged simultaneously in three continents with each clade being genetically distinct. Although new pathogenic fungal species are described regularly, these are mostly species associated with single cases in individuals who are immunosuppressed. In this study, we used phylogenetic analysis to compare C. auris with temperature susceptibility of close relatives and use these results to argue that it may be the first example of a new fungal disease emerging from climate change with the caveat that many other factors could have contributed.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/consistent_metagenome-derived_metrics_verify_and_define_bacterial_species_boundaries_61070986/">
          <h4 class="font-weight-bold">Consistent metagenome-derived metrics verify and define bacterial species boundaries, bioRxiv, 2019-05-25</h4>
          <p>AbstractLongstanding questions relate to the existence of naturally distinct bacterial species and genetic approaches to distinguish them. Bacterial genomes in public databases form distinct groups, but these databases are subject to isolation and deposition biases. We compared 5,203 bacterial genomes from 1,457 environmental metagenomic samples to test for distinct clouds of diversity, and evaluated metrics that could be used to define the species boundary. Bacterial genomes from the human gut, soil, and the ocean all exhibited gaps in whole-genome average nucleotide identities (ANI) near the previously suggested species threshold of 95% ANI. While genome-wide ratios of non-synonymous and synonymous nucleotide differences (dNdS) decrease until ANI values approach ∼98%, estimates for homologous recombination approached zero at ∼95% ANI, supporting breakdown of recombination due to sequence divergence as a species-forming force. We evaluated 107 genome-based metrics for their ability to distinguish species when full genomes are not recovered. Full length 16S rRNA genes were least useful because they were under-recovered from metagenomes, but many ribosomal proteins displayed both high metagenomic recoverability and species-discrimination power. Taken together, our results verify the existence of sequence-discrete microbial species in metagenome-derived genomes and highlight the usefulness of ribosomal genes for gene-level species discrimination.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/root_development_is_maintained_by_specific_bacteria-bacteria_interactions_within_a_complex_microbiome_60956302/">
          <h4 class="font-weight-bold">Root development is maintained by specific bacteria-bacteria interactions within a complex microbiome, bioRxiv, 2019-05-24</h4>
          <p>AbstractPlants grow within a complex web of species interacting with each other and with the plant via a wide repertoire of chemical signals. To model plant-microbe-microbe-environment interactions, we inoculated seedlings with a defined 185-member bacterial synthetic community (SynCom), and manipulated the abiotic environment to enable classification of the SynCom to modules of co-occurring strains. We deconstructed the SynCom based on these modules, identifying a single bacterial genus, Variovorax, which reverts phenotypic effects on root development induced by a wide diversity of bacterial strains and by the entire 185-member community. Variovorax use mechanisms related to auxin and ethylene manipulation to balance this ecologically realistic root community’s effects on root development. We demonstrate metabolic signal interference within a complex model community, defining Variovorax as determinants of bacteria-plant communication networks.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="/reports/allelic_polymorphism_shapes_collective_phenotypes_in_evolving_pseudomonas_aeruginosa_populations_60781879/">
          <h4 class="font-weight-bold">Allelic polymorphism shapes collective phenotypes in evolving Pseudomonas aeruginosa populations, bioRxiv, 2019-05-22</h4>
          <p>AbstractPseudomonas aeruginosa is an opportunistic pathogen that chronically infects cystic fibrosis (CF) lungs by forming antibiotic resistant biofilms. Emergence of phenotypically diverse isolates within P. aeruginosa infections has previously been reported, however, the impact of heterogeneity on social behaviors is poorly understood. Here we test how diversity impacts on traits by evolving the strain PAO1 in biofilms grown in a synthetic sputum medium for 50 days. We measured social trait production and used a metagenomic approach to analyze and assess genomic changes over the duration of the evolution experiment. We found that (i) evolutionary trajectories were reproducible in independently evolving populations; (ii) over 60% of genomic diversity occurred within the first 10 days of selection. We then focused on quorum sensing (QS), a well-studied P. aeruginosa trait that is commonly mutated in strains isolated from CF lungs. We found that at the population level (i) evolution in sputum medium selected for decreased production of QS and QS-dependent traits due to accumulation of a SNP in the lasR gene; (ii) there was a significant correlation between lasR mutant frequency, the loss of protease and the 3O-C12-HSL signal, and an increase in resistance to clinically relevant β-lactam antibiotics, despite no previous antibiotic exposure. Overall, our findings provide insights into the impact of allele polymorphism on the collective phenotypes of diverse P. aeruginosa populations. Further, we demonstrate that P. aeruginosa population and evolutionary dynamics can impact on traits important for virulence and could also provide reasons for antibiotic resistance found in CF infections.SignificancePseudomonas aeruginosa is a major cause of chronic infection of the lungs in individuals with cystic fibrosis (CF). Epidemic P. aeruginosa strains dominate and displace others in the lung and become phenotypically and genotypically diverse over time. How this diversity impacts on clinically relevant traits such as virulence or antibiotic resistance has received little attention. Here we demonstrate that the accumulation and frequency of genetic variants within heterogenous P. aeruginosa populations, results in population level changes in important traits including quorum sensing and antimicrobial resistance. Our work suggests that in the future, we should consider metagenomic and metaphenotypic assessments of P. aeruginosa populations collected from CF patients, rather than focusing on single random colonies isolated from infection.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        

        
             
            <li class="page-item active"><a href="/tags/microbiology/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/microbiology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/microbiology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/microbiology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/tags/microbiology/page/5/" class="page-link">5</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/tags/microbiology/page/2/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/tags/microbiology/page/11/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
