<!DOCTYPE html>
<html><head>
  
  <title>Microbiology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/predicting_commercially_available_antiviral_drugs_that_may_act_on_the_novel_coronavirus_2019-ncov_wuhan_china_through_a_drug-target_interaction_deep_learning_model_75023403/">
          <h4 class="font-weight-bold">Predicting commercially available antiviral drugs that may act on the novel coronavirus (2019-nCoV), Wuhan, China through a drug-target interaction deep learning model, bioRxiv, 2020-02-03</h4>
          <p>AbstractThe infection of a novel coronavirus found in Wuhan of China (2019-nCoV) is rapidly spreading, and the incidence rate is increasing worldwide. Due to the lack of effective treatment options for 2019-nCoV, various strategies are being tested in China, including drug repurposing. In this study, we used our pretrained deep learning-based drug-target interaction model called Molecule Transformer-Drug Target Interaction (MT-DTI) to identify commercially available drugs that could act on viral proteins of 2019-nCoV. The result showed that atazanavir, an antiretroviral medication used to treat and prevent the human immunodeficiency virus (HIV), is the best chemical compound, showing a inhibitory potency with Kd of 94.94 nM against the 2019-nCoV 3C-like proteinase, followed by efavirenz (199.17 nM), ritonavir (204.05 nM), and dolutegravir (336.91 nM). Interestingly, lopinavir, ritonavir, and darunavir are all designed to target viral proteinases. However, in our prediction, they may also bind to the replication complex components of 2019-nCoV with an inhibitory potency with Kd &amp;lt; 1000 nM. In addition, we also found that several antiviral agents, such as Kaletra, could be used for the treatment of 2019-nCoV, although there is no real-world evidence supporting the prediction. Overall, we suggest that the list of antiviral drugs identified by the MT-DTI model should be considered, when establishing effective treatment strategies for 2019-nCoV.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/discovery_of_a_novel_coronavirus_associated_with_the_recent_pneumonia_outbreak_in_humans_and_its_potential_bat_origin_74429171/">
          <h4 class="font-weight-bold">Discovery of a novel coronavirus associated with the recent pneumonia outbreak in humans and its potential bat origin, bioRxiv, 2020-01-23</h4>
          <p>Since the SARS outbreak 18 years ago, a large number of severe acute respiratory syndrome related coronaviruses (SARSr-CoV) have been discovered in their natural reservoir host, bats1-4. Previous studies indicated that some of those bat SARSr-CoVs have the potential to infect humans5-7. Here we report the identification and characterization of a novel coronavirus (nCoV-2019) which caused an epidemic of acute respiratory syndrome in humans, in Wuhan, China. The epidemic, started from December 12th, 2019, has caused 198 laboratory confirmed infections with three fatal cases by January 20th, 2020. Full-length genome sequences were obtained from five patients at the early stage of the outbreak. They are almost identical to each other and share 79.5% sequence identify to SARS-CoV. Furthermore, it was found that nCoV-2019 is 96% identical at the whole genome level to a bat coronavirus. The pairwise protein sequence analysis of seven conserved non-structural proteins show that this virus belongs to the species of SARSr-CoV. The nCoV-2019 virus was then isolated from the bronchoalveolar lavage fluid of a critically ill patient, which can be neutralized by sera from several patients. Importantly, we have confirmed that this novel CoV uses the same cell entry receptor, ACE2, as SARS-CoV.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/500&#43;-users">500&#43;-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genomic_and_protein_structure_modelling_analysis_depicts_the_origin_and_infectivity_of_2019-ncov_a_new_coronavirus_which_caused_a_pneumonia_outbreak_in_wuhan_china_74728787/">
          <h4 class="font-weight-bold">Genomic and protein structure modelling analysis depicts the origin and infectivity of 2019-nCoV, a new coronavirus which caused a pneumonia outbreak in Wuhan, China, bioRxiv, 2020-01-22</h4>
          <p>AbstractDetailed genomic and structure-based analysis of a new coronavirus, namely 2019-nCoV, showed that the new virus is a new type of bat coronavirus and is genetically fairly distant from the human SARS coronavirus. Structure analysis of the spike (S) protein of this new virus showed that its S protein only binds weakly to the ACE2 receptor on human cells whereas the human SARS coronavirus exhibits strongly affinity to the ACE receptor. These findings suggest that the new virus does not readily transmit between humans and should theoretically not able to cause very serious human infection. These data are important to guide design of infection control policy and inform the public on the nature of threat imposed by 2019-nCov when results of direct laboratory tests on this virus are not expected to be available in the near future.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/ferrosomes_are_iron_storage_organelles_formed_by_broadly_conserved_gene_clusters_in_bacteria_and_archaea_73911718/">
          <h4 class="font-weight-bold">Ferrosomes are iron storage organelles formed by broadly conserved gene clusters in bacteria and archaea, bioRxiv, 2020-01-12</h4>
          <p>Cellular iron homeostasis is vital and maintained through tight regulation of iron import, efflux, storage, and detoxification1â€“3. The most common modes of iron storage employ proteinaceous compartments that are composed of ferritin or related proteins4,5. While lipid-bounded iron compartments have also been described, the basis for their formation and function remains unknown. Here, we focus on one such compartment, the ferrosome, which had been previously observed in the anaerobic bacterium Desulfovibrio magneticus6. We identify three ferrosome-associated (Fez) proteins, encoded by a putative operon, that are associated with and responsible for forming ferrosomes in D. magneticus. Fez proteins include FezB, a P1B-6-ATPase found in phylogenetically and metabolically diverse species of bacteria and archaea with anaerobic lifestyles. In the majority of these species, two to ten genes define a cluster that encodes FezB. We show that two other species, Rhodopseudomonas palustris and Shewanella putrefaciens, make ferrosomes in anaerobic conditions through the action of their six-gene fez operon. Additionally, we find that the S. putrefaciens fez operon is sufficient for ferrosome formation in Escherichia coli. Using S. putrefaciens as a model, we find that ferrosomes likely play a role in the anaerobic adaptation to iron starvation. Overall, this work establishes ferrosomes as a new class of lipid-bounded iron storage organelles and sets the stage for studying ferrosome formation and structure in diverse microorganisms.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/dissecting_the_collateral_damage_of_antibiotics_on_gut_microbes_73812213/">
          <h4 class="font-weight-bold">Dissecting the collateral damage of antibiotics on gut microbes, bioRxiv, 2020-01-10</h4>
          <p>AbstractAntibiotics are used for fighting pathogens, but also target our commensal bacteria as a side effect, disturbing the gut microbiota composition and causing dysbiosis and disease1-3. Despite this well-known collateral damage, the activity spectrum of the different antibiotic classes on gut bacteria remains poorly characterized. Having monitored the activities of &amp;gt;1,000 marketed drugs on 38 representative species of the healthy human gut microbiome4, we here characterize further the 144 antibiotics therein, representing all major classes. We determined &amp;gt;800 Minimal Inhibitory Concentrations (MICs) and extended the antibiotic profiling to 10 additional species to validate these results and link to available data on antibiotic breakpoints for gut microbes. Antibiotic classes exhibited distinct inhibition spectra, including generation-dependent effects by quinolones and phylogeny-independence by Î²-lactams. Macrolides and tetracyclines, two prototypic classes of bacteriostatic protein synthesis inhibitors, inhibited almost all commensals tested. We established that both kill different subsets of prevalent commensal bacteria, and cause cell lysis in specific cases. This species-specific activity challenges the long-standing divide of antibiotics into bactericidal and bacteriostatic, and provides a possible explanation for the strong impact of macrolides on the gut microbiota composition in animals5-8 and humans9-11. To mitigate the collateral damage of macrolides and tetracyclines on gut commensals, we exploited the fact that drug combinations have species-specific outcomes in bacteria12 and sought marketed drugs, which could antagonize the activity of these antibiotics in abundant gut commensal species. By screening &amp;gt;1,000 drugs, we identified several such antidotes capable of protecting gut species from these antibiotics without compromising their activity against relevant pathogens. Altogether, this study broadens our understanding of antibiotic action on gut commensals, uncovers a previously unappreciated and broad bactericidal effect of prototypical bacteriostatic antibiotics on gut bacteria, and opens avenues for preventing the collateral damage caused by antibiotics on human gut commensals.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/increased_antibiotic_susceptibility_in_neisseria_gonorrhoeae_through_adaptation_to_the_cervical_environment_73769545/">
          <h4 class="font-weight-bold">Increased antibiotic susceptibility in Neisseria gonorrhoeae through adaptation to the cervical environment, bioRxiv, 2020-01-09</h4>
          <p>AbstractNeisseria gonorrhoeae is an urgent public health threat due to rapidly increasing incidence and antibiotic resistance. In contrast with the trend of increasing resistance, clinical isolates that have reverted to susceptibility regularly appear, prompting questions about which pressures compete with antibiotics to shape gonococcal evolution. Here, we used genome-wide association on the largest collection of N. gonorrhoeae isolates to date (n=4882) to identify loss-of-function (LOF) mutations in the efflux pump mtrCDE operon as a mechanism of increased antibiotic susceptibility and demonstrate that these mutations are overrepresented in cervical isolates relative to urethral isolates (odds ratio (OR) = 3.74, 95% CI [1.98-6.70]). In support of a model in which pump expression incurs a fitness cost in this niche, cervical isolates were also enriched relative to urethral isolates in LOF mutations in the mtrCDE activator mtrA (OR = 8.60, 95% CI [4.96-14.57]) and in farA, a subunit of the FarAB efflux pump (OR = 6.25, 95% CI [3.90-9.83]). In total, approximately 1 in 3 cervical isolates (36.4%) contained a LOF mutation in either the efflux pump components mtrC or farA or the activator mtrA. Our findings extend beyond N. gonorrhoeae to other Neisseria mtrC LOF mutations are rare (&amp;lt;1%) in the primarily nasopharyngeal-colonizing N. meningitidis in a collection of 14,798 genomes but enriched in a heterosexual urethritis-associated lineage (8.6%, p = 9.90Ã—10-5), indicating that efflux pump downregulation contributes broadly to the adaptation of pathogenic Neisseria to the female urogenital tract. Overall, our findings highlight the impact of integrating microbial population genomics with host metadata and demonstrate how host environmental pressures can lead to increased antibiotic susceptibility.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../tags/2020">2020</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        

        
             
            <li class="page-item active"><a href="../../tags/microbiology/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="../../tags/microbiology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="../../tags/microbiology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="../../tags/microbiology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="../../tags/microbiology/page/5/" class="page-link">5</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../tags/microbiology/page/2/" rel="next" class="page-link">Next â€º</a></li>
        

        
        
        <li class="page-item"><a href="../../tags/microbiology/page/17/" rel="last" class="page-link">Last Â»</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
