<!DOCTYPE html>
<html><head>
  
  <title>100 200 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/direct_imaging_of_liquid_domains_in_membranes_by_cryo_electron_tomography_75079476/">
          <h4 class="font-weight-bold">Direct Imaging of Liquid Domains in Membranes by Cryo Electron Tomography, bioRxiv, 2020-02-06</h4>
          <p>ABSTRACTImages of micron-scale domains in lipid bilayers have provided the gold standard of model-free evidence to understand the domains’ shapes, sizes, and distributions. Corresponding techniques to directly and quantitatively assess smaller (nanoscale and submicron) liquid domains have been lacking, leading to an inability to answer key questions. For example, researchers commonly seek to correlate activities of membrane proteins with attributes of the domains in which they reside; doing so hinges on identification and characterization of membrane domains. Although some features of membrane domains can be probed by indirect methods, these methods are often constrained by the limitation that data must be analyzed in the context of models that require multiple assumptions or parameters. Here, we address this challenge by developing and testing two new methods of identifying submicron domains in biomimetic membranes. Both methods leverage cryo-electron tomograms of ternary membranes under native solution conditions. The first method is optimized for probe-free applications domains are directly distinguished from the surrounding membrane by their thickness. This technique measures area fractions of domains with quantitative accuracy, in excellent agreement with known phase diagrams. The second method is optimized for applications in which a single label is deployed for imaging membranes by both high-resolution cryo-electron tomography and diffraction-limited optical microscopy. For this method, we test a panel of probes, find that a trimeric mCherry label performs best, and specify criteria for developing future high-performance, dual-use probes. These developments have led to the first direct and quantitative imaging of submicron membrane domains under native conditions.SIGNIFICANCE STATEMENTFluorescence micrographs that capture the sizes, shapes, and distributions of liquid domains in model membranes have provided high standards of evidence to prove (and disprove) theories of how micron-scale domains form and grow. Corresponding theories about smaller domains have remained untested, partly because experimental methods of identifying submicron domains in vesicles under native solvent conditions have not been available. Here we introduce two such methods. Both leverage cryo-electron tomography to observe membrane features far smaller than the diffraction limit of light. The first method is probe-free and identifies differences in thicknesses between liquid domains and their surrounding membranes. The second method identifies membrane regions labeled by an electron-dense, fluorescent protein, which enables direct comparison of fluorescence micrographs with cryo-electron tomograms.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/direct_label-free_imaging_of_nanodomains_in_biomimetic_and_biological_membranes_by_cryogenic_electron_microscopy_75079451/">
          <h4 class="font-weight-bold">Direct label-free imaging of nanodomains in biomimetic and biological membranes by cryogenic electron microscopy, bioRxiv, 2020-02-06</h4>
          <p>ABSTRACTThe nanoscale organization of biological membranes into structurally and compositionally distinct lateral domains is believed to be central to membrane function. The nature of this organization has remained elusive due to a lack of methods to directly probe nanoscopic membrane features. We show here that cryogenic electron microscopy (cryoEM) can be used to directly image coexisting nanoscopic domains in synthetic and bio-derived membranes without extrinsic probes. Analyzing a series of single-component liposomes composed of synthetic lipids of varying lengths, we demonstrate that cryoEM can distinguish bilayer thickness differences as small as 0.5 Å, comparable to the resolution of small-angle scattering methods. Simulated images from computational models reveal that features in cryoEM images result from a complex interplay between the atomic distribution normal to the plane of the bilayer and imaging parameters. Simulations of phase separated bilayers were used to predict two sources of contrast between coexisting ordered and disordered phases within a single liposome, namely differences in membrane thickness and molecular density. We observe both sources of contrast in biomimetic membranes composed of saturated lipids, unsaturated lipids, and cholesterol. When extended to isolated mammalian plasma membranes, these methods reveal similar nanoscale lateral heterogeneities. The methods reported here for direct, probe-free imaging of nanodomains in unperturbed membranes open new avenues for investigation of nanoscopic membrane organization.SIGNIFICANCEWe have used cryoEM to achieve direct, probe-free imaging of lateral domains in biomimetic lipid membranes under native conditions and to characterize differences in their structures. First, measurements of membrane thickness in laterally uniform single-component membranes show that cryoEM is capable of sub-angstrom resolution of interleaflet membrane thickness. All-atom simulations are used to predict the cryo-EM appearance of submicron domains in vesicles with coexisting liquid domains and these are quantitatively validated by direct imaging of phase separated membranes. We then extend this approach to observe nanoscopic domains in isolated cellular membranes, comprising the first direct imaging of nanodomains in biomembranes.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biophysics">biophysics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/predicting_commercially_available_antiviral_drugs_that_may_act_on_the_novel_coronavirus_2019-ncov_wuhan_china_through_a_drug-target_interaction_deep_learning_model_75023403/">
          <h4 class="font-weight-bold">Predicting commercially available antiviral drugs that may act on the novel coronavirus (2019-nCoV), Wuhan, China through a drug-target interaction deep learning model, bioRxiv, 2020-02-03</h4>
          <p>AbstractThe infection of a novel coronavirus found in Wuhan of China (2019-nCoV) is rapidly spreading, and the incidence rate is increasing worldwide. Due to the lack of effective treatment options for 2019-nCoV, various strategies are being tested in China, including drug repurposing. In this study, we used our pretrained deep learning-based drug-target interaction model called Molecule Transformer-Drug Target Interaction (MT-DTI) to identify commercially available drugs that could act on viral proteins of 2019-nCoV. The result showed that atazanavir, an antiretroviral medication used to treat and prevent the human immunodeficiency virus (HIV), is the best chemical compound, showing a inhibitory potency with Kd of 94.94 nM against the 2019-nCoV 3C-like proteinase, followed by efavirenz (199.17 nM), ritonavir (204.05 nM), and dolutegravir (336.91 nM). Interestingly, lopinavir, ritonavir, and darunavir are all designed to target viral proteinases. However, in our prediction, they may also bind to the replication complex components of 2019-nCoV with an inhibitory potency with Kd &amp;lt; 1000 nM. In addition, we also found that several antiviral agents, such as Kaletra, could be used for the treatment of 2019-nCoV, although there is no real-world evidence supporting the prediction. Overall, we suggest that the list of antiviral drugs identified by the MT-DTI model should be considered, when establishing effective treatment strategies for 2019-nCoV.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/complete_genome_characterisation_of_a_novel_coronavirus_associated_with_severe_human_respiratory_disease_in_wuhan_china_74813373/">
          <h4 class="font-weight-bold">Complete genome characterisation of a novel coronavirus associated with severe human respiratory disease in Wuhan, China, bioRxiv, 2020-01-26</h4>
          <p>Emerging and re-emerging infectious diseases, such as SARS, MERS, Zika and highly pathogenic influenza present a major threat to public health1–3. Despite intense research effort, how, when and where novel diseases appear are still the source of considerable uncertainly. A severe respiratory disease was recently reported in the city of Wuhan, Hubei province, China. At the time of writing, at least 62 suspected cases have been reported since the first patient was hospitalized on December 12nd 2019. Epidemiological investigation by the local Center for Disease Control and Prevention (CDC) suggested that the outbreak was associated with a sea food market in Wuhan. We studied seven patients who were workers at the market, and collected bronchoalveolar lavage fluid (BALF) from one patient who exhibited a severe respiratory syndrome including fever, dizziness and cough, and who was admitted to Wuhan Central Hospital on December 26th 2019. Next generation metagenomic RNA sequencing4 identified a novel RNA virus from the family Coronaviridae designed WH-Human-1 coronavirus (WHCV).Phylogenetic analysis of the complete viral genome (29,903 nucleotides) revealed that WHCV was most closely related (89.1% nucleotide similarity similarity) to a group of Severe Acute Respiratory Syndrome (SARS)-like coronaviruses (genus Betacoronavirus, subgenus Sarbecovirus) previously sampled from bats in China and that have a history of genomic recombination. This outbreak highlights the ongoing capacity of viral spill-over from animals to cause severe disease in humans.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/pathology">pathology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_mathematical_model_for_simulating_the_transmission_of_wuhan_novel_coronavirus_74254422/">
          <h4 class="font-weight-bold">A mathematical model for simulating the transmission of Wuhan novel Coronavirus, bioRxiv, 2020-01-20</h4>
          <p>AbstractAs reported by the World Health Organization, a novel coronavirus (2019-nCoV) was identified as the causative virus of Wuhan pneumonia of unknown etiology by Chinese authorities on 7 January, 2020. In this study, we developed a Bats-Hosts-Reservoir-People transmission network model for simulating the potential transmission from the infection source (probable be bats) to the human infection. Since the Bats-Hosts-Reservoir network was hard to explore clearly and public concerns were focusing on the transmission from a seafood market (reservoir) to people, we simplified the model as Reservoir-People transmission network model. The basic reproduction number (R0) was calculated from the RP model to assess the transmissibility of the 2019-nCoV.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/aav_ablates_neurogenesis_in_the_adult_murine_hippocampus_74254179/">
          <h4 class="font-weight-bold">AAV Ablates Neurogenesis in the Adult Murine Hippocampus, bioRxiv, 2020-01-20</h4>
          <p>ABSTRACTRecombinant adeno-associated virus (rAAV) has been widely used as a viral vector across mammalian biology and has been shown to be safe and effective in human gene therapy. We demonstrate that neural progenitor cells (NPCs) and immature dentate granule cells (DGCs) within the adult murine hippocampus are particularly sensitive to rAAV-induced cell death. Cell loss is dose dependent and nearly complete at experimentally relevant viral titers. rAAV-induced cell death is rapid and persistent, with loss of BrdU-labeled cells within 18 hours post-injection and no evidence of recovery of adult neurogenesis at 3 months post-injection. The remaining mature DGCs appear hyperactive 4 weeks post-injection based on immediate early gene expression, consistent with previous studies investigating the effects of attenuating adult neurogenesis. In vitro application of AAV or electroporation of AAV2 inverted terminal repeats (ITRs) is sufficient to induce cell death. Efficient transduction of the dentate gyrus (DG)—without ablating adult neurogenesis—can be achieved by injection of rAAV2-retro serotyped virus into CA3. rAAV2-retro results in efficient retrograde labeling of mature DGCs and permits in vivo 2-photon calcium imaging of dentate activity while leaving adult neurogenesis intact. These findings expand on recent reports implicating rAAV-linked toxicity in stem cells and other cell types and suggest that future work using rAAV as an experimental tool in the DG and as a gene therapy for diseases of the central nervous system (CNS) should be carefully evaluated.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2020">2020</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        

        
             
            <li class="page-item active"><a href="/audiences/tags/100-200-users/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/100-200-users/page/5/" class="page-link">5</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/100-200-users/page/2/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/100-200-users/page/126/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
