<!DOCTYPE html>
<html><head>
  
  <title>Reports | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/gephebase_a_database_of_genotype-phenotype_relationships_for_natural_and_domesticated_variation_in_eukaryotes_59510592/">
          <h4 class="font-weight-bold">Gephebase, a Database of Genotype-Phenotype Relationships for natural and domesticated variation in Eukaryotes, bioRxiv, 2019-04-25</h4>
          <p>AbstractGephebase is a manually-curated database compiling our accumulated knowledge of the genes and mutations that underlie natural, domesticated and experimental phenotypic variation in all Eukaryotes — mostly animals, plants and yeasts. Gephebase aims to compile studies where the genotype-phenotype association (based on linkage mapping, association mapping or a candidate gene approach) is relatively well supported or understood. Human disease and aberrant mutant phenotypes in laboratory model organisms are not included in Gephebase and can be found in other databases (eg. OMIM, OMIA, Monarch Initiative). Gephebase contains more than 1700 entries. Each entry corresponds to an allelic difference at a given gene and its associated phenotypic change(s) between two species or between two individuals of the same species, and is enriched with molecular details, taxonomic information, and bibliographic information. Users can easily browse entries for their topic of interest and perform searches at various levels, whether phenotypic, genetic, taxonomic or bibliographic (eg. transposable elements, cis-regulatory mutations, snakes, carotenoid content, an author name). Data can be searched using keywords and boolean operators and is exportable in spreadsheet format. This database allows to perform meta-analysis to extract general information and global trends about evolution, genetics, and the field of evolutionary genetics itself. Gephebase should also help breeders, conservationists and others to identify the most promising target genes for traits of interest, with potential applications such as crop improvement, parasite and pest control, bioconservation, and genetic diagnostic. It is freely available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpwww.gephebase.org&gt;www.gephebase.org&lt;jatsext-link&gt;.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/neighbor_predation_linked_to_natural_competence_fosters_the_transfer_of_large_genomic_regions_in_vibrio_cholerae_59471825/">
          <h4 class="font-weight-bold">Neighbor predation linked to natural competence fosters the transfer of large genomic regions in Vibrio cholerae, bioRxiv, 2019-04-25</h4>
          <p>AbstractNatural competence for transformation is a primary mode of horizontal gene transfer (HGT). Competent bacteria are able to absorb free DNA from their surroundings and exchange this DNA against pieces of their own genome when sufficiently homologous. And while it is known that transformation contributes to evolution and pathogen emergence in bacteria, there are still questions regarding the general prevalence of non-degraded DNA with sufficient coding capacity. In this context, we previously showed that the naturally competent bacterium Vibrio cholerae uses its type VI secretion system (T6SS) to actively acquire DNA from non-kin neighbors under chitin-colonizing conditions. We therefore sought to further explore the role of the T6SS in acquiring DNA, the condition of the DNA released through T6SS-mediated killing versus passive cell lysis, and the extent of the transfers that occur due to these conditions. To do this, we herein measured the frequency and the extent of genetic exchanges in bacterial co-cultures on competence-inducing chitin under various DNA-acquisition conditions. We show that competent V. cholerae strains acquire DNA fragments with an average and maximum length exceeding 50 kbp and 150 kbp, respectively, and that the T6SS is of prime importance for such HGT events. Collectively, our data support the notion that the environmental lifestyle of V. cholerae fosters HGT and that the coding capacity of the exchanged genetic material is sufficient to significantly accelerate bacterial evolution.Significance StatementDNA shuffled from one organism to another in an inheritable manner is a common feature of prokaryotes. It is a significant mechanism by which bacteria acquire new phenotypes, for example by first absorbing foreign DNA and then recombining it into their genome. In this study, we show the remarkable extent of the exchanged genetic material, frequently exceeding 150 genes in a seemingly single transfer event, in Vibrio cholerae. We also show that to best preserve its length and quality, bacteria mainly acquire this DNA by killing adjacent, healthy neighbors then immediately absorbing the released DNA before it can be degraded. These new insights into this prey-killing DNA acquisition process shed light on how bacterial species evolve in the wild.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/rna_transcribed_from_heterochromatic_simple-tandem_repeats_are_required_for_male_fertility_and_histone-protamine_exchange_in_drosophila_melanogaster_59473338/">
          <h4 class="font-weight-bold">RNA transcribed from heterochromatic simple-tandem repeats are required for male fertility and histone-protamine exchange in Drosophila melanogaster, bioRxiv, 2019-04-25</h4>
          <p>AbstractLong arrays of simple, tandemly repeated DNA sequences (known as satellites) are enriched in centromeres1 and pericentromeric regions2, and contribute to chromosome segregation and other heterochromatin functions3,4. Surprisingly, satellite DNAs are expressed in many multicellular eukaryotes, and their aberrant transcription may contribute to carcinogenesis and cellular toxicity5-7. Satellite transcription andor RNAs may also promote centromere and heterochromatin activities 8-12. However, we lack direct evidence that satellite DNA transcripts are required for normal cell or organismal functions. Here, we show that satellite RNAs derived from AAGAG tandem repeats are transcribed in many cell types throughout Drosophila melanogaster development, enriched in neuronal tissues and testes, localized within heterochromatic regions, and important for viability. Strikingly, we find that AAGAG transcripts are necessary for male fertility and are specifically required for normal histone-protamine exchange and sperm chromatin organization. Since AAGAG RNA-dependent events happen late in spermatogenesis when the transcripts are not detected, we speculate that AAGAG RNA functions in primary spermatocytes to ‘prime’ post-meiosis steps in sperm maturation. In addition to demonstrating specific essential functions for AAGAG RNAs, comparisons between closely related Drosophila species suggest that satellite repeats and their transcription evolve quickly to generate new functions.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/cell-biology">cell-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_gut_microbiota_influences_how_circulating_immune_cells_in_humans_change_from_one_day_to_the_next_59473373/">
          <h4 class="font-weight-bold">The gut microbiota influences how circulating immune cells in humans change from one day to the next, bioRxiv, 2019-04-25</h4>
          <p>ABSTRACTThe gut microbiota influences the development and homeostasis of the mammalian immune system1–3, can alter immune cell compositions in mice4–7, and is associated with responses to immunotherapy that rely on the activity of peripheral immune cells8–12. Still, our understanding of how the microbiota modulates circulatory immune cells remains limited, particularly in humans where a lack of manipulative experiments makes inference challenging. Here we overcome this challenge by studying hundreds of hospitalized—and closely monitored—bone marrow transplantation patients as they recover from chemotherapy-induced immune ablation. This aggressive treatment causes large shifts in both circulatory immune cell and microbiota populations, allowing the relationships between the two to be studied simultaneously over time with unprecedented resolution. We analyzed daily changes in white blood cell counts from 2,235 patients, and 10,680 longitudinal fecal microbiota samples to identify bacterial genera consistently associated with those changes. Bayesian inference and validation across different patient cohorts revealed consistent associations between intestinal bacteria and peripheral immune cell dynamics in the context of immunomodulatory medications, clinical metadata and homeostatic feedbacks between peripheral immune cells. The quantification of validated microbiota associations enabled us to contrast the potency of fermentatively active, obligate anaerobic bacteria with that of medications with known immunomodulatory mechanism, and this way estimate the microbiota potential to alter peripheral immune cell dynamics directly in patients. Our analysis establishes and quantifies the link between the intestinal microbiota and immune cell dynamics in humans, with implications for microbiota-driven modulation of immunity and immunotherapies that rely on circulatory immune cells.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/microbiology">microbiology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/population_imaging_of_neural_activity_in_awake_behaving_mice_in_multiple_brain_regions_59416424/">
          <h4 class="font-weight-bold">Population imaging of neural activity in awake behaving mice in multiple brain regions, bioRxiv, 2019-04-24</h4>
          <p>AbstractA longstanding goal in neuroscience has been to image membrane voltage, with high temporal precision and sensitivity, in awake behaving mammals. Here, we report a genetically encoded voltage indicator, SomArchon, which exhibits millisecond response times and compatibility with optogenetic control, and which increases the sensitivity, signal-to-noise ratio, and number of neurons observable, by manyfold over previous reagents. SomArchon only requires conventional one-photon microscopy to achieve these high performance characteristics. These improvements enable population analysis of neural activity, both at the subthreshold and spiking levels, in multiple brain regions – cortex, hippocampus, and striatum – of awake behaving mice. Using SomArchon, we detect both positive and negative responses of striatal neurons during movement, highlighting the power of voltage imaging to reveal bidirectional modulation. We also examine how the intracellular subthreshold theta oscillations of hippocampal neurons govern spike output, finding that nearby cells can exhibit highly correlated subthreshold activities, even as they generate highly divergent spiking patterns.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/the_mads-box_transcription_factor_pheres1_controls_imprinting_in_the_endosperm_by_binding_to_domesticated_transposons_59417475/">
          <h4 class="font-weight-bold">The MADS-box transcription factor PHERES1 controls imprinting in the endosperm by binding to domesticated transposons, bioRxiv, 2019-04-24</h4>
          <p>AbstractMADS-box transcription factors are ubiquitous in eukaryotic organisms and play major roles during plant development. Nevertheless, their function in seed development remains largely unknown. Here we show that the imprinted Arabidopsis thaliana MADS-box TF PHERES1 (PHE1) is a master regulator of paternally expressed imprinted genes, as well as of non-imprinted key regulators of endosperm development. PHE1 binding sites show distinct epigenetic modifications on maternal and paternal alleles, correlating with parental-specific transcriptional activity. Importantly, we show that the CArG-box-like DNA-binding motifs bound by PHE1 have been distributed by RCHelitron transposable elements. Our data provide an example of molecular domestication of these elements, which by distributing PHE1 binding sites throughout the genome, have facilitated the recruitment of crucial endosperm regulators into a single transcriptional network.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/reports/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/reports/page/80/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/77/" class="page-link">77</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/78/" class="page-link">78</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/79/" class="page-link">79</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/80/" class="page-link">80</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/reports/page/81/" class="page-link">81</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/82/" class="page-link">82</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/83/" class="page-link">83</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/84/" class="page-link">84</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/reports/page/85/" class="page-link">85</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/reports/page/82/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/reports/page/320/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
