<!DOCTYPE html>
<html><head>
  
  <title>2019 | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.04bd3499e23e665b761954011cfb58a5988ae72a56495d664077f360ae1c92545ea2a31eb009c0215fd62914601dd0a6fcc660a502754b40526292d8429fcdbc.js" integrity="sha512-BL00meI&#43;Zlt2GVQBHPtYpZiK5ypWSV1mQHfzYK4cklReoqMesAnAIV/WKRRgHdCm/MZgpQJ1S0BSYpLYQp/NvA=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="/audiences/search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/measuring_shared_responses_across_subjects_using_intersubject_correlation_58618123/">
          <h4 class="font-weight-bold">Measuring shared responses across subjects using intersubject correlation, bioRxiv, 2019-04-06</h4>
          <p>AbstractOur capacity to jointly represent information about the world underpins our social experience. By leveraging one individual’s brain activity to model another’s, we can measure shared information across brains—even in dynamic, naturalistic scenarios where an explicit response model may be unobtainable. Introducing experimental manipulations allows us to measure, for example, shared responses between speakers and listeners, or between perception and recall. In this tutorial, we develop the logic of intersubject correlation (ISC) analysis and discuss the family of neuroscientific questions that stem from this approach. We also extend this logic to spatially distributed response patterns and functional network estimation. We provide a thorough and accessible treatment of methodological considerations specific to ISC analysis, and outline best practices.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/genetic_associations_with_mathematics_tracking_and_persistence_in_secondary_school_58599187/">
          <h4 class="font-weight-bold">Genetic Associations with Mathematics Tracking and Persistence in Secondary School, bioRxiv, 2019-04-05</h4>
          <p>Maximizing the flow of students through the science, technology, engineering, and math (STEM) pipeline is important to promoting human capital development and reducing economic inequality1. A critical juncture in the STEM pipeline is the highly-cumulative sequence of secondary school math courses2–5. Students from disadvantaged schools are less likely to complete advanced math courses, but debate continues about why6,7. Here, we address this question using student polygenic scores, which are DNA-based indicators of propensity to succeed in education8. We integrated genetic and official school transcript data from over 3,000 European-ancestry students from U.S. high schools. We used polygenic scores as a molecular tracer to understand how the flow of students through the high school math pipeline differs in socioeconomically advantaged versus disadvantaged schools. Students with higher education polygenic scores were tracked to more advanced math already at the beginning of high school and persisted in math for more years. Molecular tracer analyses revealed that the dynamics of the math pipeline differed by school advantage. Compared to disadvantaged schools, advantaged schools tracked more students with high polygenic scores into advanced math classes at the start of high school, and they buffered students with low polygenic scores from dropping out of math. Across all schools, even students with exceptional polygenic scores (top 2%) were unlikely to take the most advanced math classes, suggesting substantial room for improvement in the development of potential STEM talent. These results link new molecular genetic discoveries to a common target of educational-policy reforms.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/200-500-users">200-500-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/geosmin_attracts_aedes_aegypti_mosquitoes_to_oviposition_sites_58558884/">
          <h4 class="font-weight-bold">Geosmin attracts Aedes aegypti mosquitoes to oviposition sites, bioRxiv, 2019-04-05</h4>
          <p>Geosmin is one of the most recognizable and common microbial smells on the planet. Some insects, like mosquitoes, require microbial-rich environments for their progeny, whereas for other insects such microbes may prove dangerous. In the vinegar fly Drosophila melanogaster, geosmin is decoded in a remarkably precise fashion and induces aversion, presumably signaling the presence of harmful microbes. We have here investigated the effect of geosmin on the behavior of the yellow fever mosquito Aedes aegypti. In contrast to flies, geosmin is not aversive in mosquitoes but stimulates egg-laying site selection. Female mosquitoes could associate geosmin with microbes, including cyanobacteria consumed by larvae, who also find geosmin – as well as geosmin producing cyanobacteria – attractive. Using in vivo multiphoton imaging from mosquitoes with pan-neural expression of the calcium reporter GCaMP6s, we show that Ae. aegypti code geosmin in a similar fashion to flies, i.e. with extreme sensitivity and with a high degree of selectivity. We further demonstrate that geosmin can be used as bait under field conditions, and finally we show that geosmin, which is both expensive and difficult to obtain, can be substituted by beetroot peel extract, providing a cheap and viable mean of mosquito control and surveillance in developing countries.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/inferring_the_function_performed_by_a_recurrent_neural_network_58604703/">
          <h4 class="font-weight-bold">Inferring the function performed by a recurrent neural network, bioRxiv, 2019-04-05</h4>
          <p>AbstractA central goal in systems neuroscience is to understand the functions performed by neural circuits. Previous top-down models addressed this question by comparing the behaviour of an ideal model circuit, optimised to perform a given function, with neural recordings. However, this requires guessing in advance what function is being performed, which may not be possible for many neural systems. Here, we propose an alternative approach that uses recorded neural responses to directly infer the function performed by a neural network. We assume that the goal of the network can be expressed via a reward function, which describes how desirable each state of the network is for carrying out a given objective. This allows us to frame the problem of optimising each neuron’s responses by viewing neurons as agents in a reinforcement learning (RL) paradigm; likewise the problem of inferring the reward function from the observed dynamics can be treated using inverse RL. Our framework encompasses previous influential theories of neural coding, such as efficient coding and attractor network models, as special cases, given specific choices of reward function. Finally, we can use the reward function inferred from recorded neural responses to make testable predictions about how the network dynamics will adapt depending on contextual changes, such as cell death andor varying input statistics, so as to carry out the same underlying function with different constraints.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/pbs3_is_the_missing_link_in_plant-specific_isochorismate-derived_salicylic_acid_biosynthesis_58613331/">
          <h4 class="font-weight-bold">PBS3 is the missing link in plant-specific isochorismate-derived salicylic acid biosynthesis, bioRxiv, 2019-04-05</h4>
          <p>AbstractThe phytohormone salicylic acid (SA) is a central regulator of plant immunity. Despite such functional importance, our knowledge of its biosynthesis is incomplete. Previous work showed that SA is synthesized from chorismic acid in plastids. The bulk of pathogen-induced SA derives from isochorismate generated by the catalytic activity of ISOCHORISMATE SYNTHASE1 (ICS1). How and in which cellular compartment isochorismate is converted to SA is unknown. Here we show that the pathway downstream of isochorismate requires only two additional proteins the plastidial isochorismate exporter ENHANCED DISEASE SUSCEPTIBILITY5 (EDS5) and the cytosolic amido-transferase AvrPphB SUSCEPTIBLE3 (PBS3). PBS3 catalyzes the conjugation of glutamate to isochorismate. The reaction product isochorismate-9-glutamate spontaneously decomposes into enolpyruvyl-N-glutamate and SA. This previously unknown reaction mechanism appears to be conserved throughout the plant kingdom.One Sentence SummarySalicylic acid is synthesized via isochorismate-9-glutamate by PBS3.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/plant-biology">plant-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/pirate_a_fast_and_scalable_pangenomics_toolbox_for_clustering_diverged_orthologues_in_bacteria_58599091/">
          <h4 class="font-weight-bold">PIRATE A fast and scalable pangenomics toolbox for clustering diverged orthologues in bacteria, bioRxiv, 2019-04-05</h4>
          <p>AbstractCataloguing the distribution of genes within natural bacterial populations is essential for understanding evolutionary processes and the genetic basis of adaptation. Here we present a pangenomics toolbox, PIRATE (Pangenome Iterative Refinement And Threshold Evaluation), which identifies and classifies orthologous gene families in bacterial pangenomes over a wide range of sequence similarity thresholds. PIRATE builds upon recent scalable software developments to allow for the rapid interrogation of thousands of isolates. PIRATE clusters genes (or other annotated features) over a wide range of amino-acid or nucleotide identity thresholds and uses the clustering information to rapidly classify paralogous gene families into either putative fissionfusion events or gene duplications. Furthermore, PIRATE orders the pangenome using a directed graph, provides a measure of allelic variation and estimates sequence divergence for each gene family. We demonstrate that PIRATE scales linearly with both number of samples and computation resources, allowing for analysis of large genomic datasets, and compares favorably to other popular tools. PIRATE provides a robust framework for analysing bacterial pangenomes, from largely clonal to panmictic species.AvailabilityPIRATE is implemented in Perl and is freely available under an GNU GPL 3 open source license from &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsgithub.comSionBaylissPIRATE&gt;httpsgithub.comSionBaylissPIRATE&lt;jatsext-link&gt;.&lt;jatssec sec-type=supplementary-material&gt;Supplementary InformationSupplementary data is available online.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="/audiences/tags/2019/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="/audiences/tags/2019/page/36/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/33/" class="page-link">33</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/34/" class="page-link">34</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/35/" class="page-link">35</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/36/" class="page-link">36</a></li>
            
        
             
            <li class="page-item active"><a href="/audiences/tags/2019/page/37/" class="page-link">37</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/38/" class="page-link">38</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/39/" class="page-link">39</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/40/" class="page-link">40</a></li>
            
        
            
             
            <li class="page-item"><a href="/audiences/tags/2019/page/41/" class="page-link">41</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="/audiences/tags/2019/page/38/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="/audiences/tags/2019/page/87/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://www.linkedin.com/in/jedidiah/" class="fa fa-linkedin"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
