<!DOCTYPE html>
<html><head>
  
  <title>cancer-biology | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.68.3" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.95d2483cd3c5d2bf47c1f36307a456787d60ad958c5efe44db9afc47012b1b05b8e1f64f6ac291650439322a6c38e86eb7cd6254255516d857c23f299fb0e666.js" integrity="sha512-ldJIPNPF0r9HwfNjB6RWeH1grZWMXv5E25r8RwErGwW44fZPasKRZQQ5MipsOOhut81iVCVVFthXwj8pn7DmZg=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="https://carjed.github.io/audiences/">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="https://carjed.github.io/audiences/reports" title="">reports</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/clonal_evolution_of_acute_myeloid_leukemia_revealed_by_high-throughput_single-cell_genomics_75412873/">
          <h4 class="font-weight-bold">Clonal Evolution of Acute Myeloid Leukemia Revealed by High-Throughput Single-Cell Genomics, bioRxiv, 2020-02-08</h4>
          <p>SummaryOne of the pervasive features of cancer is the diversity of mutations found in malignant cells within the same tumor; a phenomenon called clonal diversity or intratumor heterogeneity. Clonal diversity allows tumors to adapt to the selective pressure of treatment and likely contributes to the development of treatment resistance and cancer recurrence. Thus, the ability to precisely delineate the clonal substructure of a tumor, including the evolutionary history of its development and the co-occurrence of its mutations, is necessary to understand and overcome treatment resistance. However, DNA sequencing of bulk tumor samples cannot accurately resolve complex clonal architectures. Here, we performed high-throughput single-cell DNA sequencing to quantitatively assess the clonal architecture of acute myeloid leukemia (AML). We sequenced a total of 556,951 cells from 77 patients with AML for 19 genes known to be recurrently mutated in AML. The data revealed clonal relationship among AML driver mutations and identified mutations that often co-occurred (e.g., NPM1FLT3-ITD, DNMT3ANPM1, SRSF2IDH2, and WT1FLT3-ITD) and those that were mutually exclusive (e.g., NRASKRAS, FLT3-D835ITD, and IDH1IDH2) at single-cell resolution. Reconstruction of the tumor phylogeny uncovered history of tumor development that is characterized by linear and branching clonal evolution patterns with latter involving functional convergence of separately evolved clones. Analysis of longitudinal samples revealed remodeling of clonal architecture in response to therapeutic pressure that is driven by clonal selection. Furthermore, in this AML cohort, higher clonal diversity (≥4 subclones) was associated with significantly worse overall survival. These data portray clonal relationship, architecture, and evolution of AML driver genes with unprecedented resolution, and illuminate the role of clonal diversity in therapeutic resistance, relapse and clinical outcome in AML.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/cancer-biology">cancer-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/frequent_extrachromosomal_oncogene_amplification_drives_aggressive_tumors_71372515/">
          <h4 class="font-weight-bold">Frequent extrachromosomal oncogene amplification drives aggressive tumors, bioRxiv, 2019-11-29</h4>
          <p>Extrachromosomal DNA (ecDNA) amplification promotes high oncogene copy number, intratumoral genetic heterogeneity, and accelerated tumor evolution1–3, but its frequency and clinical impact are not well understood. Here we show, using computational analysis of whole-genome sequencing data from 1,979 cancer patients, that ecDNA amplification occurs in at least 26% of human cancers, of a wide variety of histological types, but not in whole blood or normal tissue. We demonstrate a highly significant enrichment for oncogenes on amplified ecDNA and that the most common recurrent oncogene amplifications arise on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number matched linear DNA, coupled with enhanced chromatin accessibility. Patients whose tumors have ecDNA-based oncogene amplification showed increase of cell proliferation signature activity, greater likelihood of lymph node spread at initial diagnosis, and significantly shorter survival, even when controlled for tissue type, than do patients whose cancers are not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification plays a central role in driving the poor outcome for patients with some of the most aggressive forms of cancers.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/cancer-biology">cancer-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/non-oncology_drugs_are_a_source_of_previously_unappreciated_anti-cancer_activity_64761272/">
          <h4 class="font-weight-bold">Non-oncology drugs are a source of previously unappreciated anti-cancer activity, bioRxiv, 2019-08-09</h4>
          <p>ABSTRACTAnti-cancer uses of non-oncology drugs have been found on occasion, but such discoveries have been serendipitous and rare. We sought to create a public resource containing the growth inhibitory activity of 4,518 drugs tested across 578 human cancer cell lines. To accomplish this, we used PRISM, which involves drug treatment of molecularly barcoded cell lines in pools. Relative barcode abundance following treatment thus reflects cell line viability. We found that an unexpectedly large number of non-oncology drugs selectively inhibited subsets of cancer cell lines. Moreover, the killing activity of the majority of these drugs was predictable based on the molecular features of the cell lines. Follow-up of several of these compounds revealed novel mechanisms. For example, compounds that kill by inducing PDE3A-SLFN12 complex formation; vanadium-containing compounds whose killing is dependent on the sulfate transporter SLC26A2; the alcohol dependence drug disulfiram, which kills cells with low expression of metallothioneins; and the anti-inflammatory drug tepoxalin, whose killing is dependent on high expression of the multi-drug resistance gene ABCB1. These results illustrate the potential of the PRISM drug repurposing resource as a starting point for new oncology therapeutic development. The resource is available at &lt;jatsext-link xmlnsxlink=httpwww.w3.org1999xlink ext-link-type=uri xlinkhref=httpsdepmap.org&gt;httpsdepmap.org&lt;jatsext-link&gt;.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/cancer-biology">cancer-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/deep_learning_detects_virus_presence_in_cancer_histology_63190390/">
          <h4 class="font-weight-bold">Deep learning detects virus presence in cancer histology, bioRxiv, 2019-07-06</h4>
          <p>AbstractOncogenic viruses like human papilloma virus (HPV) or Epstein Barr virus (EBV) are a major cause of human cancer. Viral oncogenesis has a direct impact on treatment decisions because virus-associated tumors can demand a lower intensity of chemotherapy and radiation or can be more susceptible to immune check-point inhibition. However, molecular tests for HPV and EBV are not ubiquitously available.We hypothesized that the histopathological features of virus-driven and non-virus driven cancers are sufficiently different to be detectable by artificial intelligence (AI) through deep learning-based analysis of images from routine hematoxylin and eosin (HE) stained slides. We show that deep transfer learning can predict presence of HPV in head and neck cancer with a patient-level 3-fold cross validated area-under-the-curve (AUC) of 0.89 [0.82; 0.94]. The same workflow was used for Epstein-Barr virus (EBV) driven gastric cancer achieving a cross-validated AUC of 0.80 [0.70; 0.92] and a similar performance in external validation sets. Reverse-engineering our deep neural networks, we show that the key morphological features can be made understandable to humans.This workflow could enable a fast and low-cost method to identify virus-induced cancer in clinical trials or clinical routine. At the same time, our approach for feature visualization allows pathologists to look into the black box of deep learning, enabling them to check the plausibility of computer-based image classification.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/cancer-biology">cancer-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/serum_flt3_ligand_is_biomarker_of_progenitor_cell_mass_and_prognosis_in_acute_myeloid_leukemia_57683166/">
          <h4 class="font-weight-bold">Serum Flt3 ligand is biomarker of progenitor cell mass and prognosis in acute myeloid leukemia, bioRxiv, 2019-03-25</h4>
          <p>AbstractFms-like tyrosine kinase 3 (Flt3) is a hematopoietic growth factor receptor expressed on lymphomyeloid progenitors and frequently, by AML blasts. Its ligand, Flt3L, has non-hematopoietic and lymphoid origins, is detectable during homeostasis and increases to high levels in states of hypoplasia due to genetic defects or treatment with cytoreductive agents. Measurement of Flt3L by ELISA reveals that Flt3&#43;AML, is associated with depletion of Flt3L to undetectable levels. After induction chemotherapy, Flt3L is restored in patients entering CR, but remains depressed in those with refractory disease. Weekly sampling reveals marked differences in the kinetics of Flt3L response during the first 6 weeks of treatment, proportionate to the clearance of blasts and cellularity of the BM. In the UK NCRI AML17 trial, Flt3L was measured at day 26 in a subgroup of 135 patients with Flt3 mutation randomized to the tyrosine kinase inhibitor lestaurtinib. In these patients, attainment of CR was associated with higher Flt3L at day 26 (Mann-Whitney p &amp;lt; 0.0001). Day 26 Flt3L was also associated with survival Flt3L ≤ 291pgml was associated with inferior event-free survival; and, Flt3L &amp;gt;1185pgml was associated with higher overall survival (p = 0.0119). Serial measurement of Flt3L in patients who had received a hematopoietic stem cell transplant for AML further illustrated the potential value of declining Flt3L to identify relapse. Together these observations suggest that measurement of Flt3L provides a non-invasive estimate of progenitor cell mass in most patients with AML, with the potential to inform clinical decisions.Graphical abstract&lt;jatsfig id=ufig1 position=float fig-type=figure orientation=portrait&gt;&lt;jatsgraphic xmlnsxlink=httpwww.w3.org1999xlink xlinkhref=588319_ufig1 position=float orientation=portrait &gt;</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/cancer-biology">cancer-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/100-200-users">100-200-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/spatial_structure_governs_the_mode_of_tumour_evolution_57587794/">
          <h4 class="font-weight-bold">Spatial structure governs the mode of tumour evolution, bioRxiv, 2019-03-24</h4>
          <p>AbstractCharacterizing the mode – the way, manner, or pattern – of evolution in tumours is important for clinical forecasting and optimizing cancer treatment. DNA sequencing studies have inferred various modes, including branching, punctuated and neutral evolution, but it is unclear why a particular pattern predominates in any given tumour.1, 2 Here we propose that differences in tumour architecture alone can explain the variety of observed patterns. We examine this hypothesis using spatially explicit population genetic models and demonstrate that, within biologically relevant parameter ranges, human tumours are expected to exhibit four distinct onco-evolutionary modes (oncoevotypes) rapid clonal expansion (predicted in leukaemia); progressive diversification (in colorectal adenomas and early-stage colorectal carcinomas); branching evolution (in invasive glandular tumours); and effectively almost neutral evolution (in certain non-glandular and poorly differentiated solid tumours). We thus provide a simple, mechanistic explanation for a wide range of empirical observations. Oncoevotypes are governed by the mode of cell dispersal and the range of cell-cell interaction, which we show are essential factors in accurately characterizing, forecasting and controlling tumour evolution.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/cancer-biology">cancer-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="https://carjed.github.io/audiences/tags/2019">2019</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        

        

        
             
            <li class="page-item active"><a href="https://carjed.github.io/audiences/tags/cancer-biology/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/cancer-biology/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/cancer-biology/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/cancer-biology/page/4/" class="page-link">4</a></li>
            
        
            
             
            <li class="page-item"><a href="https://carjed.github.io/audiences/tags/cancer-biology/page/5/" class="page-link">5</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        

        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/cancer-biology/page/2/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="https://carjed.github.io/audiences/tags/cancer-biology/page/6/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
