<!DOCTYPE html>
<html><head>
  
  <title>0 100 Users | audiences</title>
  
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/bootstrap.min.css'>
  <link rel="stylesheet" href='https://carjed.github.io/audiences/css/mondrian.css'>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="generator" content="Hugo 0.52" />
  

<style>

  #grid .list-item.page:hover {
    background: white;
  }
  main.single.page h1 {
    border-color: white;
  }

  #grid .list-item.reports:hover {
    background: gold;
  }
  main.single.reports h1 {
    border-color: gold;
  }

  #grid .list-item.summary:hover {
    background: limegreen;
  }
  main.single.summary h1 {
    border-color: limegreen;
  }

</style>
  









<script src="https://carjed.github.io/audiences/js/bundle.min.f25ff27688046eb318bdc22977150cde493b69297eb61c33d467745adf840d358a2427ed01d7dc1e0bd54c4d5831f1ed5e9bc1e1f7187533a24e36d63659607f.js" integrity="sha512-8l/ydogEbrMYvcIpdxUM3kk7aSl&#43;thwz1Gd0Wt&#43;EDTWKJCftAdfcHgvVTE1YMfHtXpvB4fcYdTOiTjbWNllgfw=="></script>

  <style></style>
  
  
</head>
<body>
  <div class="container-fluid h-100 d-flex flex-column">
<header class="row flex-shrink-0 justify-content-center justify-content-md-start">
  <ul class="nav px-2">
    <li class="p-2">
      <a class="text-muted text-decoration-none" href="../../../../">audiences</a>
    </li>
    
      <li class="p-2">
        <a class="text-light" href="reports" title="">reports</a>
      </li>
    
      <li class="p-2">
        <a class="text-light" href="search" title="">search</a>
      </li>
    
  </nav>
</header>

<main id="grid" class="row list flex-wrap h-99 text-center">
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/skd3_human_clpb_is_a_potent_mitochondrial_protein_disaggregase_that_is_inactivated_by_3-methylglutaconic_aciduria-linked_mutations_74220710/">
          <h4 class="font-weight-bold">Skd3 (human CLPB) is a potent mitochondrial protein disaggregase that is inactivated by 3-methylglutaconic aciduria-linked mutations, bioRxiv, 2020-01-19</h4>
          <p>ABSTRACTCells have evolved specialized protein disaggregases to reverse toxic protein aggregation and restore protein functionality. In nonmetazoan eukaryotes, the AAA&#43; disaggregase Hsp78 resolubilizes and reactivates proteins in mitochondria. Curiously, metazoa lack Hsp78. Hence, whether metazoan mitochondria reactivate aggregated proteins is unknown. Here, we establish that a mitochondrial AAA&#43; protein, Skd3 (human CLPB), couples ATP hydrolysis to protein disaggregation and reactivation. The Skd3 ankyrin-repeat domain combines with conserved AAA&#43; elements to enable stand-alone disaggregase activity. A mitochondrial inner-membrane protease, PARL, removes an autoinhibitory peptide from Skd3 to greatly enhance disaggregase activity. Indeed, PARL-activated Skd3 dissolves α-synuclein fibrils connected to Parkinson’s disease. Human cells lacking Skd3 exhibit reduced solubility of various mitochondrial proteins, including anti-apoptotic Hax1. Importantly, Skd3 variants linked to 3-methylglutaconic aciduria, a severe mitochondrial disorder, display diminished disaggregase activity (but not always reduced ATPase activity), which predicts disease severity. Thus, Skd3 is a potent protein disaggregase critical for human health.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biochemistry">biochemistry</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/distinguishing_the_neural_correlates_of_perceptual_awareness_and_post-perceptual_processing_74118318/">
          <h4 class="font-weight-bold">Distinguishing the neural correlates of perceptual awareness and post-perceptual processing, bioRxiv, 2020-01-17</h4>
          <p>AbstractTo identify the neural correlates of perceptual awareness, researchers often compare the differences in neural activation between conditions in which an observer is or is not aware of a target stimulus. While intuitive, this approach often contains a critical limitation In order to link brain activity with perceptual awareness, observers traditionally report the contents of their perceptual experience. However, relying on observers’ reports is problematic because it makes it difficult to know if the neural responses being measured are associated with conscious perception per se or with post-perceptual processes involved in the reporting task (i.e., working memory, decision-making, etc.). To address this issue, we combined a standard visual masking paradigm with a recently developed “no-report” paradigm. In the visual masking paradigm, observers saw images of animals and objects that were visible or invisible depending on their proximity to masks. Meanwhile, on half of the trials, observers reported the contents of their perceptual experience (i.e., report condition), while on the other half of trials they refrained from reporting about their experiences (i.e., no-report condition). We used electroencephalography (EEG) to examine how visibility interacts with reporting by measuring the P3b event related potential (ERP), one of the proposed canonical “signatures” of conscious processing. Overall, we found a robust P3b in the report condition, but no P3b whatsoever in the no-report condition. This finding suggests that the P3b itself is not a neural signature of conscious processing and highlights the importance of carefully distinguishing the neural correlates of perceptual awareness from post-perceptual processing.Significance statementWhat are the neural signatures that differentiate conscious and unconscious processing in the brain? Perhaps the most well-established candidate signature is the P3b event-related potential (ERP), a late slow wave that appears when observers are aware of a stimulus, but disappears when a stimulus fails to reach awareness. Here, however, we found that the P3b does not track what observers are perceiving but instead tracks what observers are reporting. When observers are aware of simple visual stimuli, the P3b is nowhere to be found unless observers are reporting the contents of their experience. These results challenge the well-established notion of the P3b as a neural marker of awareness and highlight the need for new approaches to the neuroscience of consciousness.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/theoretical_and_empirical_quantification_of_the_accuracy_of_polygenic_scores_in_ancestry_divergent_populations_74072185/">
          <h4 class="font-weight-bold">Theoretical and empirical quantification of the accuracy of polygenic scores in ancestry divergent populations, bioRxiv, 2020-01-16</h4>
          <p>AbstractPolygenic scores (PGS) have been widely used to predict complex traits and risk of diseases using variants identified from genome-wide association studies (GWASs). To date, most GWASs have been conducted in populations of European ancestry, which limits the use of GWAS-derived PGS in non-European populations. Here, we develop a new theory to predict the relative accuracy (RA, relative to the accuracy in populations of the same ancestry as the discovery population) of PGS across ancestries. We used simulations and real data from the UK Biobank to evaluate our results. We found across various simulation scenarios that the RA of PGS based on trait-associated SNPs can be predicted accurately from modelling linkage disequilibrium (LD), minor allele frequencies (MAF), cross-population correlations of SNP effect sizes and heritability. Altogether, we find that LD and MAF differences between ancestries explain alone up to ~70% of the loss of RA using European-based PGS in African ancestry for traits like body mass index and height. Our results suggest that causal variants underlying common genetic variation identified in European ancestry GWASs are mostly shared across continents.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/genetics">genetics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/a_behavioral_polymorphism_caused_by_a_single_gene_inside_a_supergene_74019247/">
          <h4 class="font-weight-bold">A behavioral polymorphism caused by a single gene inside a supergene, bioRxiv, 2020-01-15</h4>
          <p>AbstractBehavioral evolution relies on genetic changes, yet few social behaviors can be traced to specific genetic sequences in vertebrates. Here, we show experimental evidence that differentiation of a single gene has contributed to divergent behavioral phenotypes in the white-throated sparrow, a common North American songbird. In this species, one of two alleles of ESR1, encoding estrogen receptor α (ERα), has been captured inside a differentiating supergene that segregates with an aggressive phenotype, such that ESR1 expression predicts aggression. Here, we show that the aggressive phenotype associated with the supergene is prevented by ESR1 knockdown in a single brain region. Next, we show that in a free-living population, aggression is predicted by allelic imbalance favoring the supergene allele. Cis-regulatory variation between the two alleles affects transcription factor binding sites, DNA methylation, and rates of transcription. This work provides a rare illustration of how genotypic divergence has led to behavioral phenotypic divergence in a vertebrate.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/neuroscience">neuroscience</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/algorithmic_learning_for_auto-deconvolution_of_gc-ms_data_to_enable_molecular_networking_within_gnps_74033548/">
          <h4 class="font-weight-bold">Algorithmic Learning for Auto-deconvolution of GC-MS Data to Enable Molecular Networking within GNPS, bioRxiv, 2020-01-15</h4>
          <p>AbstractGas chromatography-mass spectrometry (GC-MS) represents an analytical technique with significant practical societal impact. Spectral deconvolution is an essential step for interpreting GC-MS data. No public GC-MS repositories that also enable repository-scale analysis exist, in part because deconvolution requires significant user input. We therefore engineered a scalable machine learning workflow for the Global Natural Product Social Molecular Networking (GNPS) analysis platform to enable the mass spectrometry community to store, process, share, annotate, compare, and perform molecular networking of GC-MS data. The workflow performs auto-deconvolution of compound fragmentation patterns via unsupervised non-negative matrix factorization, using a Fast Fourier Transform-based strategy to overcome scalability limitations. We introduce a “balance score” that quantifies the reproducibility of fragmentation patterns across all samples. We demonstrate the utility of the platform with breathomics analysis applied to the early detection of oesophago-gastric cancer, and by creating the first molecular spatial map of the human volatilome.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/bioinformatics">bioinformatics</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2020">2020</a>
          
        
      </div>
    </div>
  
    <div class="list-item d-flex lead justify-content-center align-items-center reports">
      <div>
        <a class="text-decoration-none" href="https://carjed.github.io/audiences/reports/drug_mechanism-of-action_discovery_through_the_integration_of_pharmacological_and_crispr_screens_74033007/">
          <h4 class="font-weight-bold">Drug mechanism-of-action discovery through the integration of pharmacological and CRISPR screens, bioRxiv, 2020-01-15</h4>
          <p>AbstractLow success rates during drug development are due in part to the difficulty of defining drug mechanism-of-action and molecular markers of therapeutic activity. Here, we integrated 199,219 drug sensitivity measurements for 397 unique anti-cancer drugs and genome-wide CRISPR loss-of-function screens in 484 cell lines to systematically investigate in cellular drug mechanism-of-action. We observed an enrichment for positive associations between drug sensitivity and knockout of their nominal targets, and by leveraging protein-protein networks we identified pathways that mediate drug response. This revealed an unappreciated role of mitochondrial E3 ubiquitin-protein ligase MARCH5 in sensitivity to MCL1 inhibitors. We also estimated drug on-target and off-target activity, informing on specificity, potency and toxicity. Linking drug and gene dependency together with genomic datasets uncovered contexts in which molecular networks when perturbed mediate cancer cell loss-of-fitness, and thereby provide independent and orthogonal evidence of biomarkers for drug development. This study illustrates how integrating cell line drug sensitivity with CRISPR loss-of-function screens can elucidate mechanism-of-action to advance drug development.</p>
        </a>
        
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/biorxiv">biorxiv</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/systems-biology">systems-biology</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/0-100-users">0-100-users</a>
          
            <a class="text-body text-decoration-none border-bottom border-dark" href="../../../../tags/2020">2020</a>
          
        
      </div>
    </div>
  
</main>
<br>
<main>
  


<nav aria-label="page navigation">
    <ul class="pagination">
        
        
        <li class="page-item"><a href="../../../../tags/0-100-users/" rel="first" class="page-link">« First</a></li>
        

        
        <li class="page-item"><a href="../../../../tags/0-100-users/page/4/" rel="prev" class="page-link">‹ Prev</a></li>
        

        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/" class="page-link">1</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/2/" class="page-link">2</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/3/" class="page-link">3</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/4/" class="page-link">4</a></li>
            
        
             
            <li class="page-item active"><a href="../../../../tags/0-100-users/page/5/" class="page-link">5</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/6/" class="page-link">6</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/7/" class="page-link">7</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/8/" class="page-link">8</a></li>
            
        
            
             
            <li class="page-item"><a href="../../../../tags/0-100-users/page/9/" class="page-link">9</a></li>
            
        
             
            
            <li class="page-item disabled"><a class="page-link">...</a></li>
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        

        
        <li class="page-item"><a href="../../../../tags/0-100-users/page/6/" rel="next" class="page-link">Next ›</a></li>
        

        
        
        <li class="page-item"><a href="../../../../tags/0-100-users/page/123/" rel="last" class="page-link">Last »</a></li>
        
    </ul>
</nav>


</main>

&nbsp;
<hr />
<p style="text-align: center;">Created with the <a href="https://github.com/carjed/audiences">audiences framework </a> by <a href="https://github.com/carjed/">Jedidiah Carlson</a></p>
<p style="text-align: center;">Powered by <a href="https://gohugo.io/">Hugo </a></p>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">


<p style="text-align: center;">
    <a href="https://twitter.com/JedMSP?lang=en" class="fa fa-twitter"></a>
    <a href="https://github.com/carjed/" class="fa fa-github"></a>
</p>

&nbsp;

</div>
  
</body>
</html>
